{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Exploration of Gateway to research data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/23/24 09:24:53] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Resolved project path as: <span style=\"color: #800080; text-decoration-color: #800080\">/home/dampudia/projects/</span><span style=\"color: #ff00ff; text-decoration-color: #ff00ff\">alphafold-impact.</span>    <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">__init__.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#146\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">146</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         To set a different path, run <span style=\"color: #008000; text-decoration-color: #008000\">'%reload_kedro &lt;project_root&gt;'</span>            <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">               </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/23/24 09:24:53]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Resolved project path as: \u001b[35m/home/dampudia/projects/\u001b[0m\u001b[95malphafold-impact.\u001b[0m    \u001b]8;id=162969;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\u001b\\\u001b[2m__init__.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=260825;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#146\u001b\\\u001b[2m146\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         To set a different path, run \u001b[32m'%reload_kedro \u001b[0m\u001b[32m<\u001b[0m\u001b[32mproject_root\u001b[0m\u001b[32m>\u001b[0m\u001b[32m'\u001b[0m            \u001b[2m               \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/23/24 09:24:53] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Kedro project alphafold-impact                                         <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">__init__.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#115\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">115</span></a>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/23/24 09:24:53]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Kedro project alphafold-impact                                         \u001b]8;id=388036;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\u001b\\\u001b[2m__init__.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=301469;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#115\u001b\\\u001b[2m115\u001b[0m\u001b]8;;\u001b\\\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Defined global variable <span style=\"color: #008000; text-decoration-color: #008000\">'context'</span>, <span style=\"color: #008000; text-decoration-color: #008000\">'session'</span>, <span style=\"color: #008000; text-decoration-color: #008000\">'catalog'</span> and            <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">__init__.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#116\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">116</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"color: #008000; text-decoration-color: #008000\">'pipelines'</span>                                                            <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">               </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m                   \u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Defined global variable \u001b[32m'context'\u001b[0m, \u001b[32m'session'\u001b[0m, \u001b[32m'catalog'\u001b[0m and            \u001b]8;id=141626;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\u001b\\\u001b[2m__init__.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=269001;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#116\u001b\\\u001b[2m116\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[32m'pipelines'\u001b[0m                                                            \u001b[2m               \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/23/24 09:24:54] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Registered line magic <span style=\"color: #008000; text-decoration-color: #008000\">'run_viz'</span>                                        <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">__init__.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#122\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">122</span></a>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/23/24 09:24:54]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Registered line magic \u001b[32m'run_viz'\u001b[0m                                        \u001b]8;id=4874;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py\u001b\\\u001b[2m__init__.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=341351;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/ipython/__init__.py#122\u001b\\\u001b[2m122\u001b[0m\u001b]8;;\u001b\\\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "\u001b[1;35mDataTransformerRegistry.enable\u001b[0m\u001b[1m(\u001b[0m\u001b[32m'default'\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%load_ext kedro.ipython\n",
    "\n",
    "import pandas as pd\n",
    "import altair as alt\n",
    "import yaml\n",
    "\n",
    "alt.data_transformers.disable_max_rows()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.institutions</span>              <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m                   \u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.institutions\u001b[0m              \u001b]8;id=187165;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=79549;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/23/24 09:24:59] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.publications</span>              <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/23/24 09:24:59]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.publications\u001b[0m              \u001b]8;id=641970;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=309902;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/23/24 09:25:22] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.citations.alphafold</span>       <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/23/24 09:25:22]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.citations.alphafold\u001b[0m       \u001b]8;id=156067;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=655;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gtr_institutions = catalog.load(\"oa.data_processing.gtr.institutions\")\n",
    "gtr_publications = catalog.load(\"oa.data_processing.gtr.publications\")\n",
    "gtr_citations = catalog.load('oa.data_processing.gtr.citations.alphafold')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Publications over time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-7cf3c751e8de483dafb45e0398094f16.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-7cf3c751e8de483dafb45e0398094f16.vega-embed details,\n",
       "  #altair-viz-7cf3c751e8de483dafb45e0398094f16.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-7cf3c751e8de483dafb45e0398094f16\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-7cf3c751e8de483dafb45e0398094f16\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-7cf3c751e8de483dafb45e0398094f16\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-38b04dc026b73419f7d294b9a9ac1e77\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"axis\": {\"grid\": false, \"labelAngle\": -45, \"labelFontSize\": 8, \"tickCount\": 157258}, \"field\": \"year_month\", \"title\": \"Publication Month\", \"type\": \"ordinal\"}, \"y\": {\"field\": \"counts\", \"title\": \"Number of Publications\", \"type\": \"quantitative\"}}, \"height\": 400, \"title\": \"Monthly Distribution of GTR Publications\", \"width\": 1800, \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-38b04dc026b73419f7d294b9a9ac1e77\": [{\"year_month\": \"2008-01\", \"counts\": 134}, {\"year_month\": \"2008-02\", \"counts\": 75}, {\"year_month\": \"2008-03\", \"counts\": 89}, {\"year_month\": \"2008-04\", \"counts\": 93}, {\"year_month\": \"2008-05\", \"counts\": 91}, {\"year_month\": \"2008-06\", \"counts\": 82}, {\"year_month\": \"2008-07\", \"counts\": 126}, {\"year_month\": \"2008-08\", \"counts\": 98}, {\"year_month\": \"2008-09\", \"counts\": 117}, {\"year_month\": \"2008-10\", \"counts\": 116}, {\"year_month\": \"2008-11\", \"counts\": 119}, {\"year_month\": \"2008-12\", \"counts\": 148}, {\"year_month\": \"2009-01\", \"counts\": 389}, {\"year_month\": \"2009-02\", \"counts\": 164}, {\"year_month\": \"2009-03\", \"counts\": 205}, {\"year_month\": \"2009-04\", \"counts\": 214}, {\"year_month\": \"2009-05\", \"counts\": 216}, {\"year_month\": \"2009-06\", \"counts\": 265}, {\"year_month\": \"2009-07\", \"counts\": 235}, {\"year_month\": \"2009-08\", \"counts\": 210}, {\"year_month\": \"2009-09\", \"counts\": 235}, {\"year_month\": \"2009-10\", \"counts\": 260}, {\"year_month\": \"2009-11\", \"counts\": 223}, {\"year_month\": \"2009-12\", \"counts\": 271}, {\"year_month\": \"2010-01\", \"counts\": 637}, {\"year_month\": \"2010-02\", \"counts\": 299}, {\"year_month\": \"2010-03\", \"counts\": 342}, {\"year_month\": \"2010-04\", \"counts\": 282}, {\"year_month\": \"2010-05\", \"counts\": 291}, {\"year_month\": \"2010-06\", \"counts\": 312}, {\"year_month\": \"2010-07\", \"counts\": 445}, {\"year_month\": \"2010-08\", \"counts\": 276}, {\"year_month\": \"2010-09\", \"counts\": 354}, {\"year_month\": \"2010-10\", \"counts\": 346}, {\"year_month\": \"2010-11\", \"counts\": 369}, {\"year_month\": \"2010-12\", \"counts\": 366}, {\"year_month\": \"2011-01\", \"counts\": 814}, {\"year_month\": \"2011-02\", \"counts\": 336}, {\"year_month\": \"2011-03\", \"counts\": 410}, {\"year_month\": \"2011-04\", \"counts\": 357}, {\"year_month\": \"2011-05\", \"counts\": 407}, {\"year_month\": \"2011-06\", \"counts\": 403}, {\"year_month\": \"2011-07\", \"counts\": 400}, {\"year_month\": \"2011-08\", \"counts\": 432}, {\"year_month\": \"2011-09\", \"counts\": 453}, {\"year_month\": \"2011-10\", \"counts\": 378}, {\"year_month\": \"2011-11\", \"counts\": 418}, {\"year_month\": \"2011-12\", \"counts\": 395}, {\"year_month\": \"2012-01\", \"counts\": 990}, {\"year_month\": \"2012-02\", \"counts\": 465}, {\"year_month\": \"2012-03\", \"counts\": 568}, {\"year_month\": \"2012-04\", \"counts\": 467}, {\"year_month\": \"2012-05\", \"counts\": 497}, {\"year_month\": \"2012-06\", \"counts\": 484}, {\"year_month\": \"2012-07\", \"counts\": 474}, {\"year_month\": \"2012-08\", \"counts\": 459}, {\"year_month\": \"2012-09\", \"counts\": 551}, {\"year_month\": \"2012-10\", \"counts\": 521}, {\"year_month\": \"2012-11\", \"counts\": 527}, {\"year_month\": \"2012-12\", \"counts\": 501}, {\"year_month\": \"2013-01\", \"counts\": 1397}, {\"year_month\": \"2013-02\", \"counts\": 557}, {\"year_month\": \"2013-03\", \"counts\": 639}, {\"year_month\": \"2013-04\", \"counts\": 623}, {\"year_month\": \"2013-05\", \"counts\": 629}, {\"year_month\": \"2013-06\", \"counts\": 617}, {\"year_month\": \"2013-07\", \"counts\": 623}, {\"year_month\": \"2013-08\", \"counts\": 640}, {\"year_month\": \"2013-09\", \"counts\": 678}, {\"year_month\": \"2013-10\", \"counts\": 666}, {\"year_month\": \"2013-11\", \"counts\": 670}, {\"year_month\": \"2013-12\", \"counts\": 659}, {\"year_month\": \"2014-01\", \"counts\": 1544}, {\"year_month\": \"2014-02\", \"counts\": 673}, {\"year_month\": \"2014-03\", \"counts\": 792}, {\"year_month\": \"2014-04\", \"counts\": 721}, {\"year_month\": \"2014-05\", \"counts\": 677}, {\"year_month\": \"2014-06\", \"counts\": 810}, {\"year_month\": \"2014-07\", \"counts\": 710}, {\"year_month\": \"2014-08\", \"counts\": 772}, {\"year_month\": \"2014-09\", \"counts\": 808}, {\"year_month\": \"2014-10\", \"counts\": 803}, {\"year_month\": \"2014-11\", \"counts\": 755}, {\"year_month\": \"2014-12\", \"counts\": 783}, {\"year_month\": \"2015-01\", \"counts\": 1805}, {\"year_month\": \"2015-02\", \"counts\": 732}, {\"year_month\": \"2015-03\", \"counts\": 832}, {\"year_month\": \"2015-04\", \"counts\": 851}, {\"year_month\": \"2015-05\", \"counts\": 830}, {\"year_month\": \"2015-06\", \"counts\": 972}, {\"year_month\": \"2015-07\", \"counts\": 932}, {\"year_month\": \"2015-08\", \"counts\": 908}, {\"year_month\": \"2015-09\", \"counts\": 973}, {\"year_month\": \"2015-10\", \"counts\": 970}, {\"year_month\": \"2015-11\", \"counts\": 921}, {\"year_month\": \"2015-12\", \"counts\": 965}, {\"year_month\": \"2016-01\", \"counts\": 2164}, {\"year_month\": \"2016-02\", \"counts\": 1016}, {\"year_month\": \"2016-03\", \"counts\": 1035}, {\"year_month\": \"2016-04\", \"counts\": 1026}, {\"year_month\": \"2016-05\", \"counts\": 965}, {\"year_month\": \"2016-06\", \"counts\": 1069}, {\"year_month\": \"2016-07\", \"counts\": 1025}, {\"year_month\": \"2016-08\", \"counts\": 987}, {\"year_month\": \"2016-09\", \"counts\": 1056}, {\"year_month\": \"2016-10\", \"counts\": 1038}, {\"year_month\": \"2016-11\", \"counts\": 961}, {\"year_month\": \"2016-12\", \"counts\": 973}, {\"year_month\": \"2017-01\", \"counts\": 1998}, {\"year_month\": \"2017-02\", \"counts\": 1048}, {\"year_month\": \"2017-03\", \"counts\": 998}, {\"year_month\": \"2017-04\", \"counts\": 919}, {\"year_month\": \"2017-05\", \"counts\": 1021}, {\"year_month\": \"2017-06\", \"counts\": 1058}, {\"year_month\": \"2017-07\", \"counts\": 1065}, {\"year_month\": \"2017-08\", \"counts\": 1066}, {\"year_month\": \"2017-09\", \"counts\": 1072}, {\"year_month\": \"2017-10\", \"counts\": 1075}, {\"year_month\": \"2017-11\", \"counts\": 1098}, {\"year_month\": \"2017-12\", \"counts\": 1075}, {\"year_month\": \"2018-01\", \"counts\": 2194}, {\"year_month\": \"2018-02\", \"counts\": 1121}, {\"year_month\": \"2018-03\", \"counts\": 1108}, {\"year_month\": \"2018-04\", \"counts\": 1075}, {\"year_month\": \"2018-05\", \"counts\": 1025}, {\"year_month\": \"2018-06\", \"counts\": 1091}, {\"year_month\": \"2018-07\", \"counts\": 1204}, {\"year_month\": \"2018-08\", \"counts\": 1099}, {\"year_month\": \"2018-09\", \"counts\": 1121}, {\"year_month\": \"2018-10\", \"counts\": 1281}, {\"year_month\": \"2018-11\", \"counts\": 1137}, {\"year_month\": \"2018-12\", \"counts\": 982}, {\"year_month\": \"2019-01\", \"counts\": 2280}, {\"year_month\": \"2019-02\", \"counts\": 1351}, {\"year_month\": \"2019-03\", \"counts\": 1196}, {\"year_month\": \"2019-04\", \"counts\": 1163}, {\"year_month\": \"2019-05\", \"counts\": 1173}, {\"year_month\": \"2019-06\", \"counts\": 1157}, {\"year_month\": \"2019-07\", \"counts\": 1212}, {\"year_month\": \"2019-08\", \"counts\": 1151}, {\"year_month\": \"2019-09\", \"counts\": 1173}, {\"year_month\": \"2019-10\", \"counts\": 1227}, {\"year_month\": \"2019-11\", \"counts\": 1254}, {\"year_month\": \"2019-12\", \"counts\": 1202}, {\"year_month\": \"2020-01\", \"counts\": 2418}, {\"year_month\": \"2020-02\", \"counts\": 1233}, {\"year_month\": \"2020-03\", \"counts\": 1275}, {\"year_month\": \"2020-04\", \"counts\": 1227}, {\"year_month\": \"2020-05\", \"counts\": 1188}, {\"year_month\": \"2020-06\", \"counts\": 1313}, {\"year_month\": \"2020-07\", \"counts\": 1401}, {\"year_month\": \"2020-08\", \"counts\": 1169}, {\"year_month\": \"2020-09\", \"counts\": 1396}, {\"year_month\": \"2020-10\", \"counts\": 1486}, {\"year_month\": \"2020-11\", \"counts\": 1304}, {\"year_month\": \"2020-12\", \"counts\": 1445}, {\"year_month\": \"2021-01\", \"counts\": 2409}, {\"year_month\": \"2021-02\", \"counts\": 1343}, {\"year_month\": \"2021-03\", \"counts\": 1449}, {\"year_month\": \"2021-04\", \"counts\": 1187}, {\"year_month\": \"2021-05\", \"counts\": 1305}, {\"year_month\": \"2021-06\", \"counts\": 1272}, {\"year_month\": \"2021-07\", \"counts\": 1304}, {\"year_month\": \"2021-08\", \"counts\": 1127}, {\"year_month\": \"2021-09\", \"counts\": 1246}, {\"year_month\": \"2021-10\", \"counts\": 1278}, {\"year_month\": \"2021-11\", \"counts\": 1230}, {\"year_month\": \"2021-12\", \"counts\": 1271}, {\"year_month\": \"2022-01\", \"counts\": 1941}, {\"year_month\": \"2022-02\", \"counts\": 1128}, {\"year_month\": \"2022-03\", \"counts\": 1142}, {\"year_month\": \"2022-04\", \"counts\": 914}, {\"year_month\": \"2022-05\", \"counts\": 986}, {\"year_month\": \"2022-06\", \"counts\": 983}, {\"year_month\": \"2022-07\", \"counts\": 942}, {\"year_month\": \"2022-08\", \"counts\": 993}, {\"year_month\": \"2022-09\", \"counts\": 952}, {\"year_month\": \"2022-10\", \"counts\": 967}, {\"year_month\": \"2022-11\", \"counts\": 963}, {\"year_month\": \"2022-12\", \"counts\": 865}, {\"year_month\": \"2023-01\", \"counts\": 1237}, {\"year_month\": \"2023-02\", \"counts\": 744}, {\"year_month\": \"2023-03\", \"counts\": 499}, {\"year_month\": \"2023-04\", \"counts\": 335}, {\"year_month\": \"2023-05\", \"counts\": 318}, {\"year_month\": \"2023-06\", \"counts\": 295}, {\"year_month\": \"2023-07\", \"counts\": 315}, {\"year_month\": \"2023-08\", \"counts\": 267}, {\"year_month\": \"2023-09\", \"counts\": 173}, {\"year_month\": \"2023-10\", \"counts\": 48}, {\"year_month\": \"2023-11\", \"counts\": 23}, {\"year_month\": \"2023-12\", \"counts\": 22}, {\"year_month\": \"2024-01\", \"counts\": 3}, {\"year_month\": \"2024-02\", \"counts\": 3}, {\"year_month\": \"2024-04\", \"counts\": 1}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Assuming gtr_publications is already defined and loaded\n",
    "pubs = gtr_publications.copy()\n",
    "pubs['publication_date'] = pd.to_datetime(pubs['publication_date'])\n",
    "\n",
    "# keep publications from 2008 onwards\n",
    "pubs = pubs[pubs['publication_date'] >= '2008-01-01']\n",
    "\n",
    "pubs['year_month'] = pubs['publication_date'].dt.strftime('%Y-%m')\n",
    "\n",
    "# Group by 'year_month' and count the publications\n",
    "monthly_counts = pubs.groupby('year_month').size().reset_index(name='counts')\n",
    "\n",
    "# Create a histogram using the aggregated data\n",
    "chart = alt.Chart(monthly_counts).mark_bar().encode(\n",
    "    x=alt.X('year_month:O', title='Publication Month', axis=alt.Axis(tickCount=pubs.shape[0], grid=False, labelAngle=-45, labelFontSize=8)),\n",
    "    y=alt.Y('counts:Q', title='Number of Publications')\n",
    ").properties(\n",
    "    title='Monthly Distribution of GTR Publications',\n",
    "    width=1800,  # Adjusted for better readability\n",
    "    height=400\n",
    ")\n",
    "\n",
    "chart"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Distribution over institutions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>author_id</th>\n",
       "      <th>institution_id</th>\n",
       "      <th>institution_name</th>\n",
       "      <th>country_code</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>W3123896073</td>\n",
       "      <td>A5029121397</td>\n",
       "      <td>I157614274</td>\n",
       "      <td>University of Cape Town</td>\n",
       "      <td>ZA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>W3123896073</td>\n",
       "      <td>A5029121397</td>\n",
       "      <td>I46305939</td>\n",
       "      <td>University of Zielona Góra</td>\n",
       "      <td>PL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>W3123896073</td>\n",
       "      <td>A5029121397</td>\n",
       "      <td>I121797337</td>\n",
       "      <td>Leiden University</td>\n",
       "      <td>NL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>W3123896073</td>\n",
       "      <td>A5045208440</td>\n",
       "      <td>I4210105583</td>\n",
       "      <td>Royal Observatory</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>W3123896073</td>\n",
       "      <td>A5045208440</td>\n",
       "      <td>I98677209</td>\n",
       "      <td>University of Edinburgh</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880343</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>A5085798363</td>\n",
       "      <td>I4210156067</td>\n",
       "      <td>Institute for Research on Combustion</td>\n",
       "      <td>IT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880344</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>A5076148122</td>\n",
       "      <td>I4210156067</td>\n",
       "      <td>Institute for Research on Combustion</td>\n",
       "      <td>IT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880345</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>A5085943471</td>\n",
       "      <td>I71267560</td>\n",
       "      <td>University of Naples Federico II</td>\n",
       "      <td>IT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880346</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>A5084132907</td>\n",
       "      <td>I82284825</td>\n",
       "      <td>Cranfield University</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880347</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>A5089317030</td>\n",
       "      <td>I82284825</td>\n",
       "      <td>Cranfield University</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1880348 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "\n",
       "            paper_id    author_id institution_id  \\\n",
       "\u001b[1;36m0\u001b[0m        W3123896073  A5029121397     I157614274   \n",
       "\u001b[1;36m1\u001b[0m        W3123896073  A5029121397      I46305939   \n",
       "\u001b[1;36m2\u001b[0m        W3123896073  A5029121397     I121797337   \n",
       "\u001b[1;36m3\u001b[0m        W3123896073  A5045208440    I4210105583   \n",
       "\u001b[1;36m4\u001b[0m        W3123896073  A5045208440      I98677209   \n",
       "\u001b[33m...\u001b[0m              \u001b[33m...\u001b[0m          \u001b[33m...\u001b[0m            \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m1880343\u001b[0m  W2528595972  A5085798363    I4210156067   \n",
       "\u001b[1;36m1880344\u001b[0m  W2528595972  A5076148122    I4210156067   \n",
       "\u001b[1;36m1880345\u001b[0m  W2528595972  A5085943471      I71267560   \n",
       "\u001b[1;36m1880346\u001b[0m  W2528595972  A5084132907      I82284825   \n",
       "\u001b[1;36m1880347\u001b[0m  W2528595972  A5089317030      I82284825   \n",
       "\n",
       "                             institution_name country_code  \n",
       "\u001b[1;36m0\u001b[0m                     University of Cape Town           ZA  \n",
       "\u001b[1;36m1\u001b[0m                  University of Zielona Góra           PL  \n",
       "\u001b[1;36m2\u001b[0m                           Leiden University           NL  \n",
       "\u001b[1;36m3\u001b[0m                           Royal Observatory           GB  \n",
       "\u001b[1;36m4\u001b[0m                     University of Edinburgh           GB  \n",
       "\u001b[33m...\u001b[0m                                       \u001b[33m...\u001b[0m          \u001b[33m...\u001b[0m  \n",
       "\u001b[1;36m1880343\u001b[0m  Institute for Research on Combustion           IT  \n",
       "\u001b[1;36m1880344\u001b[0m  Institute for Research on Combustion           IT  \n",
       "\u001b[1;36m1880345\u001b[0m      University of Naples Federico II           IT  \n",
       "\u001b[1;36m1880346\u001b[0m                  Cranfield University           GB  \n",
       "\u001b[1;36m1880347\u001b[0m                  Cranfield University           GB  \n",
       "\n",
       "\u001b[1m[\u001b[0m\u001b[1;36m1880348\u001b[0m rows x \u001b[1;36m5\u001b[0m columns\u001b[1m]\u001b[0m"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_institutions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-08cb94d590024a37b361d6daab69a7ec.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-08cb94d590024a37b361d6daab69a7ec.vega-embed details,\n",
       "  #altair-viz-08cb94d590024a37b361d6daab69a7ec.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-08cb94d590024a37b361d6daab69a7ec\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-08cb94d590024a37b361d6daab69a7ec\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-08cb94d590024a37b361d6daab69a7ec\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-1ea32775fc1474d9469ac7fccccad879\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"color\": {\"field\": \"Region\", \"title\": \"Region\", \"type\": \"nominal\"}, \"tooltip\": [{\"field\": \"institution_name\", \"type\": \"nominal\"}, {\"field\": \"count\", \"type\": \"quantitative\"}, {\"field\": \"Region\", \"type\": \"nominal\"}], \"x\": {\"field\": \"count\", \"title\": \"Number of Papers/Authors\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"institution_name\", \"sort\": \"-x\", \"title\": \"Institution Name\", \"type\": \"nominal\"}}, \"height\": 1000, \"title\": \"Distribution of Institutions in GB vs RoW\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-1ea32775fc1474d9469ac7fccccad879\": [{\"institution_id\": \"I1118541\", \"institution_name\": \"University of East Anglia\", \"country_code\": \"GB\", \"count\": 5186, \"Region\": \"GB\"}, {\"institution_id\": \"I122411786\", \"institution_name\": \"California Institute of Technology\", \"country_code\": \"US\", \"count\": 5815, \"Region\": \"RoW\"}, {\"institution_id\": \"I126231945\", \"institution_name\": \"Queen's University Belfast\", \"country_code\": \"GB\", \"count\": 8888, \"Region\": \"GB\"}, {\"institution_id\": \"I1286704778\", \"institution_name\": \"Rutherford Appleton Laboratory\", \"country_code\": \"GB\", \"count\": 6446, \"Region\": \"GB\"}, {\"institution_id\": \"I1294671590\", \"institution_name\": \"French National Centre for Scientific Research\", \"country_code\": \"FR\", \"count\": 27222, \"Region\": \"RoW\"}, {\"institution_id\": \"I130828816\", \"institution_name\": \"University of Leeds\", \"country_code\": \"GB\", \"count\": 19149, \"Region\": \"GB\"}, {\"institution_id\": \"I1314696892\", \"institution_name\": \"Fermi National Accelerator Laboratory\", \"country_code\": \"US\", \"count\": 4266, \"Region\": \"RoW\"}, {\"institution_id\": \"I142263535\", \"institution_name\": \"University of Nottingham\", \"country_code\": \"GB\", \"count\": 18436, \"Region\": \"GB\"}, {\"institution_id\": \"I146655781\", \"institution_name\": \"University of Liverpool\", \"country_code\": \"GB\", \"count\": 16624, \"Region\": \"GB\"}, {\"institution_id\": \"I153648349\", \"institution_name\": \"University of Leicester\", \"country_code\": \"GB\", \"count\": 7796, \"Region\": \"GB\"}, {\"institution_id\": \"I154526488\", \"institution_name\": \"Inserm\", \"country_code\": \"FR\", \"count\": 4042, \"Region\": \"RoW\"}, {\"institution_id\": \"I162608824\", \"institution_name\": \"University of Sussex\", \"country_code\": \"GB\", \"count\": 4872, \"Region\": \"GB\"}, {\"institution_id\": \"I166337079\", \"institution_name\": \"Queen Mary University of London\", \"country_code\": \"GB\", \"count\": 7997, \"Region\": \"GB\"}, {\"institution_id\": \"I16835326\", \"institution_name\": \"University of St Andrews\", \"country_code\": \"GB\", \"count\": 7880, \"Region\": \"GB\"}, {\"institution_id\": \"I181647926\", \"institution_name\": \"University of Strathclyde\", \"country_code\": \"GB\", \"count\": 7960, \"Region\": \"GB\"}, {\"institution_id\": \"I183935753\", \"institution_name\": \"King's College London\", \"country_code\": \"GB\", \"count\": 24027, \"Region\": \"GB\"}, {\"institution_id\": \"I190082696\", \"institution_name\": \"Durham University\", \"country_code\": \"GB\", \"count\": 11011, \"Region\": \"GB\"}, {\"institution_id\": \"I195460627\", \"institution_name\": \"University of Aberdeen\", \"country_code\": \"GB\", \"count\": 4449, \"Region\": \"GB\"}, {\"institution_id\": \"I197940685\", \"institution_name\": \"National Oceanography Centre\", \"country_code\": \"GB\", \"count\": 4337, \"Region\": \"GB\"}, {\"institution_id\": \"I19820366\", \"institution_name\": \"Chinese Academy of Sciences\", \"country_code\": \"CN\", \"count\": 4256, \"Region\": \"RoW\"}, {\"institution_id\": \"I204730241\", \"institution_name\": \"Universit\\u00e9 Paris Cit\\u00e9\", \"country_code\": \"FR\", \"count\": 5774, \"Region\": \"RoW\"}, {\"institution_id\": \"I23923803\", \"institution_name\": \"University of Exeter\", \"country_code\": \"GB\", \"count\": 11386, \"Region\": \"GB\"}, {\"institution_id\": \"I241749\", \"institution_name\": \"University of Cambridge\", \"country_code\": \"GB\", \"count\": 55042, \"Region\": \"GB\"}, {\"institution_id\": \"I2738703131\", \"institution_name\": \"Atomic Energy and Alternative Energies Commission\", \"country_code\": \"FR\", \"count\": 6528, \"Region\": \"RoW\"}, {\"institution_id\": \"I277688954\", \"institution_name\": \"University of Paris-Saclay\", \"country_code\": \"FR\", \"count\": 4237, \"Region\": \"RoW\"}, {\"institution_id\": \"I2799300731\", \"institution_name\": \"Norwich Research Park\", \"country_code\": \"GB\", \"count\": 4695, \"Region\": \"GB\"}, {\"institution_id\": \"I28290843\", \"institution_name\": \"University of Surrey\", \"country_code\": \"GB\", \"count\": 5916, \"Region\": \"GB\"}, {\"institution_id\": \"I28407311\", \"institution_name\": \"University of Manchester\", \"country_code\": \"GB\", \"count\": 34429, \"Region\": \"GB\"}, {\"institution_id\": \"I34931013\", \"institution_name\": \"National Institute for Health Research\", \"country_code\": \"GB\", \"count\": 4134, \"Region\": \"GB\"}, {\"institution_id\": \"I36234482\", \"institution_name\": \"University of Bristol\", \"country_code\": \"GB\", \"count\": 27822, \"Region\": \"GB\"}, {\"institution_id\": \"I39555362\", \"institution_name\": \"University of Warwick\", \"country_code\": \"GB\", \"count\": 14138, \"Region\": \"GB\"}, {\"institution_id\": \"I39586589\", \"institution_name\": \"Swansea University\", \"country_code\": \"GB\", \"count\": 5869, \"Region\": \"GB\"}, {\"institution_id\": \"I39804081\", \"institution_name\": \"Sorbonne Universit\\u00e9\", \"country_code\": \"FR\", \"count\": 5458, \"Region\": \"RoW\"}, {\"institution_id\": \"I40120149\", \"institution_name\": \"University of Oxford\", \"country_code\": \"GB\", \"count\": 53971, \"Region\": \"GB\"}, {\"institution_id\": \"I4210089966\", \"institution_name\": \"London School of Hygiene & Tropical Medicine\", \"country_code\": \"GB\", \"count\": 7285, \"Region\": \"GB\"}, {\"institution_id\": \"I4210128565\", \"institution_name\": \"CEA Saclay\", \"country_code\": \"FR\", \"count\": 4119, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210129701\", \"institution_name\": \"Institute of High Energy Physics\", \"country_code\": \"AT\", \"count\": 7883, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210133362\", \"institution_name\": \"Institut National de Physique Nucl\\u00e9aire et de Physique des Particules\", \"country_code\": \"FR\", \"count\": 4158, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210145102\", \"institution_name\": \"Institut Polytechnique de Paris\", \"country_code\": \"FR\", \"count\": 5629, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210151618\", \"institution_name\": \"National Hospital for Neurology and Neurosurgery\", \"country_code\": \"GB\", \"count\": 4983, \"Region\": \"GB\"}, {\"institution_id\": \"I4210158597\", \"institution_name\": \"MRC Epidemiology Unit\", \"country_code\": \"GB\", \"count\": 5445, \"Region\": \"GB\"}, {\"institution_id\": \"I43439940\", \"institution_name\": \"University of Southampton\", \"country_code\": \"GB\", \"count\": 23292, \"Region\": \"GB\"}, {\"institution_id\": \"I45129253\", \"institution_name\": \"University College London\", \"country_code\": \"GB\", \"count\": 57852, \"Region\": \"GB\"}, {\"institution_id\": \"I47508984\", \"institution_name\": \"Imperial College London\", \"country_code\": \"GB\", \"count\": 47726, \"Region\": \"GB\"}, {\"institution_id\": \"I51601045\", \"institution_name\": \"University of Bath\", \"country_code\": \"GB\", \"count\": 5999, \"Region\": \"GB\"}, {\"institution_id\": \"I52099693\", \"institution_name\": \"University of York\", \"country_code\": \"GB\", \"count\": 9562, \"Region\": \"GB\"}, {\"institution_id\": \"I67311998\", \"institution_name\": \"European Organization for Nuclear Research\", \"country_code\": \"CH\", \"count\": 6198, \"Region\": \"RoW\"}, {\"institution_id\": \"I67415387\", \"institution_name\": \"Lancaster University\", \"country_code\": \"GB\", \"count\": 6527, \"Region\": \"GB\"}, {\"institution_id\": \"I71052956\", \"institution_name\": \"University of Reading\", \"country_code\": \"GB\", \"count\": 4886, \"Region\": \"GB\"}, {\"institution_id\": \"I7882870\", \"institution_name\": \"University of Glasgow\", \"country_code\": \"GB\", \"count\": 18717, \"Region\": \"GB\"}, {\"institution_id\": \"I79510175\", \"institution_name\": \"Cardiff University\", \"country_code\": \"GB\", \"count\": 14901, \"Region\": \"GB\"}, {\"institution_id\": \"I79619799\", \"institution_name\": \"University of Birmingham\", \"country_code\": \"GB\", \"count\": 15417, \"Region\": \"GB\"}, {\"institution_id\": \"I84884186\", \"institution_name\": \"Newcastle University\", \"country_code\": \"GB\", \"count\": 13306, \"Region\": \"GB\"}, {\"institution_id\": \"I90344618\", \"institution_name\": \"Medical Research Council\", \"country_code\": \"GB\", \"count\": 9826, \"Region\": \"GB\"}, {\"institution_id\": \"I91136226\", \"institution_name\": \"University of Sheffield\", \"country_code\": \"GB\", \"count\": 17211, \"Region\": \"GB\"}, {\"institution_id\": \"I98677209\", \"institution_name\": \"University of Edinburgh\", \"country_code\": \"GB\", \"count\": 31828, \"Region\": \"GB\"}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "institution_counts = (\n",
    "    gtr_institutions.groupby(\n",
    "        [\"institution_id\", \"institution_name\", \"country_code\"]\n",
    "    )\n",
    "    .size()\n",
    "    .reset_index(name=\"count\")\n",
    ")\n",
    "institution_counts[\"Region\"] = institution_counts[\"country_code\"].apply(\n",
    "    lambda x: \"GB\" if x == \"GB\" else \"RoW\"\n",
    ")\n",
    "\n",
    "# keep only institutions with more than 4000 papers/authors\n",
    "institution_counts = institution_counts[institution_counts[\"count\"] > 4000]\n",
    "\n",
    "chart = (\n",
    "    alt.Chart(institution_counts)\n",
    "    .mark_bar()\n",
    "    .encode(\n",
    "        x=alt.X(\"count:Q\", title=\"Number of Papers/Authors\"),\n",
    "        y=alt.Y(\"institution_name:N\", title=\"Institution Name\", sort=\"-x\"),\n",
    "        color=alt.Color(\"Region:N\", title=\"Region\"),\n",
    "        tooltip=[\"institution_name\", \"count\", \"Region\"],\n",
    "    )\n",
    "    .properties(title=\"Distribution of Institutions in GB vs RoW\", height=1000)\n",
    ")\n",
    "\n",
    "chart"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>work_id</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>publication_date</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>W1996468795</td>\n",
       "      <td>Measurement of an excess of&lt;mml:math xmlns:mml...</td>\n",
       "      <td>Based on the full BaBar data sample, we report...</td>\n",
       "      <td>https://doi.org/10.1103/physrevd.88.072012</td>\n",
       "      <td>2013-10-31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>W1532821088</td>\n",
       "      <td>Measurement of the Higgs boson mass from the&lt;m...</td>\n",
       "      <td>An improved measurement of the mass of the Hig...</td>\n",
       "      <td>https://doi.org/10.1103/physrevd.90.052004</td>\n",
       "      <td>2014-09-09</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>W2025070190</td>\n",
       "      <td>Electron reconstruction and identification eff...</td>\n",
       "      <td>Many of the interesting physics processes to b...</td>\n",
       "      <td>https://doi.org/10.1140/epjc/s10052-014-2941-0</td>\n",
       "      <td>2014-07-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>W2202434937</td>\n",
       "      <td>Search for new phenomena in dijet mass and ang...</td>\n",
       "      <td>This Letter describes a model-agnostic search ...</td>\n",
       "      <td>https://doi.org/10.1016/j.physletb.2016.01.032</td>\n",
       "      <td>2016-03-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>W1811257610</td>\n",
       "      <td>Measurement of event-plane correlations in&lt;mml...</td>\n",
       "      <td>A measurement of event-plane correlations invo...</td>\n",
       "      <td>https://doi.org/10.1103/physrevc.90.024905</td>\n",
       "      <td>2014-08-12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157794</th>\n",
       "      <td>W2792494745</td>\n",
       "      <td>Two-dimensional heat and mass transfer and the...</td>\n",
       "      <td>Transport of heat and mass and the thermodynam...</td>\n",
       "      <td>https://doi.org/10.1016/j.cep.2018.02.025</td>\n",
       "      <td>2018-04-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157795</th>\n",
       "      <td>W4213414592</td>\n",
       "      <td>Mental health and life satisfaction among 10–1...</td>\n",
       "      <td>In many countries, including in the United Kin...</td>\n",
       "      <td>https://doi.org/10.1186/s12889-022-12752-6</td>\n",
       "      <td>2022-02-23</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157796</th>\n",
       "      <td>W3042807857</td>\n",
       "      <td>PhISCS-BnB: a fast branch and bound algorithm ...</td>\n",
       "      <td>Recent advances in single-cell sequencing (SCS...</td>\n",
       "      <td>https://doi.org/10.1093/bioinformatics/btaa464</td>\n",
       "      <td>2020-07-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157797</th>\n",
       "      <td>W3207988091</td>\n",
       "      <td>Effect of CRTH2 antagonism on the response to ...</td>\n",
       "      <td>The chemoattractant receptor-homologous molecu...</td>\n",
       "      <td>https://doi.org/10.1136/thoraxjnl-2021-217429</td>\n",
       "      <td>2021-10-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157798</th>\n",
       "      <td>W2528595972</td>\n",
       "      <td>Fragmentation of biomass-templated CaO-based p...</td>\n",
       "      <td>The use of biomass templating materials with a...</td>\n",
       "      <td>https://doi.org/10.1016/j.fuel.2016.09.061</td>\n",
       "      <td>2017-01-01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>157799 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "\n",
       "            work_id                                              title  \\\n",
       "\u001b[1;36m0\u001b[0m       W1996468795  Measurement of an excess of<mml:math xmlns:mml\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m1\u001b[0m       W1532821088  Measurement of the Higgs boson mass from the<m\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m2\u001b[0m       W2025070190  Electron reconstruction and identification eff\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m3\u001b[0m       W2202434937  Search for new phenomena in dijet mass and ang\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m4\u001b[0m       W1811257610  Measurement of event-plane correlations in<mml\u001b[33m...\u001b[0m   \n",
       "\u001b[33m...\u001b[0m             \u001b[33m...\u001b[0m                                                \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157794\u001b[0m  W2792494745  Two-dimensional heat and mass transfer and the\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157795\u001b[0m  W4213414592  Mental health and life satisfaction among \u001b[1;36m10\u001b[0m–\u001b[1;36m1\u001b[0m\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157796\u001b[0m  W3042807857  PhISCS-BnB: a fast branch and bound algorithm \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157797\u001b[0m  W3207988091  Effect of CRTH2 antagonism on the response to \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157798\u001b[0m  W2528595972  Fragmentation of biomass-templated CaO-based p\u001b[33m...\u001b[0m   \n",
       "\n",
       "                                                 abstract  \\\n",
       "\u001b[1;36m0\u001b[0m       Based on the full BaBar data sample, we report\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m1\u001b[0m       An improved measurement of the mass of the Hig\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m2\u001b[0m       Many of the interesting physics processes to b\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m3\u001b[0m       This Letter describes a model-agnostic search \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m4\u001b[0m       A measurement of event-plane correlations invo\u001b[33m...\u001b[0m   \n",
       "\u001b[33m...\u001b[0m                                                   \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157794\u001b[0m  Transport of heat and mass and the thermodynam\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157795\u001b[0m  In many countries, including in the United Kin\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157796\u001b[0m  Recent advances in single-cell sequencing \u001b[1m(\u001b[0mSCS\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157797\u001b[0m  The chemoattractant receptor-homologous molecu\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m157798\u001b[0m  The use of biomass templating materials with a\u001b[33m...\u001b[0m   \n",
       "\n",
       "                                                   doi publication_date  \n",
       "\u001b[1;36m0\u001b[0m           \u001b[4;94mhttps://doi.org/10.1103/physrevd.88.072012\u001b[0m       \u001b[1;36m2013\u001b[0m-\u001b[1;36m10\u001b[0m-\u001b[1;36m31\u001b[0m  \n",
       "\u001b[1;36m1\u001b[0m           \u001b[4;94mhttps://doi.org/10.1103/physrevd.90.052004\u001b[0m       \u001b[1;36m2014\u001b[0m-\u001b[1;36m09\u001b[0m-\u001b[1;36m09\u001b[0m  \n",
       "\u001b[1;36m2\u001b[0m       \u001b[4;94mhttps://doi.org/10.1140/epjc/s10052-014-2941-0\u001b[0m       \u001b[1;36m2014\u001b[0m-\u001b[1;36m07\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m3\u001b[0m       \u001b[4;94mhttps://doi.org/10.1016/j.physletb.2016.01.032\u001b[0m       \u001b[1;36m2016\u001b[0m-\u001b[1;36m03\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m4\u001b[0m           \u001b[4;94mhttps://doi.org/10.1103/physrevc.90.024905\u001b[0m       \u001b[1;36m2014\u001b[0m-\u001b[1;36m08\u001b[0m-\u001b[1;36m12\u001b[0m  \n",
       "\u001b[33m...\u001b[0m                                                \u001b[33m...\u001b[0m              \u001b[33m...\u001b[0m  \n",
       "\u001b[1;36m157794\u001b[0m       \u001b[4;94mhttps://doi.org/10.1016/j.cep.2018.02.025\u001b[0m       \u001b[1;36m2018\u001b[0m-\u001b[1;36m04\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m157795\u001b[0m      \u001b[4;94mhttps://doi.org/10.1186/s12889-022-12752-6\u001b[0m       \u001b[1;36m2022\u001b[0m-\u001b[1;36m02\u001b[0m-\u001b[1;36m23\u001b[0m  \n",
       "\u001b[1;36m157796\u001b[0m  \u001b[4;94mhttps://doi.org/10.1093/bioinformatics/btaa464\u001b[0m       \u001b[1;36m2020\u001b[0m-\u001b[1;36m07\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m157797\u001b[0m   \u001b[4;94mhttps://doi.org/10.1136/thoraxjnl-2021-217429\u001b[0m       \u001b[1;36m2021\u001b[0m-\u001b[1;36m10\u001b[0m-\u001b[1;36m29\u001b[0m  \n",
       "\u001b[1;36m157798\u001b[0m      \u001b[4;94mhttps://doi.org/10.1016/j.fuel.2016.09.061\u001b[0m       \u001b[1;36m2017\u001b[0m-\u001b[1;36m01\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\n",
       "\u001b[1m[\u001b[0m\u001b[1;36m157799\u001b[0m rows x \u001b[1;36m5\u001b[0m columns\u001b[1m]\u001b[0m"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_publications"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# merge citations with publications using work_id\n",
    "citations = gtr_citations.copy()\n",
    "\n",
    "citations = citations.merge(gtr_publications, on='work_id', how='left')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "\n",
       "af_indirect_citations\n",
       "\u001b[1;36m0\u001b[0m     \u001b[1;36m157506\u001b[0m\n",
       "\u001b[1;36m1\u001b[0m        \u001b[1;36m196\u001b[0m\n",
       "\u001b[1;36m2\u001b[0m         \u001b[1;36m48\u001b[0m\n",
       "\u001b[1;36m3\u001b[0m         \u001b[1;36m18\u001b[0m\n",
       "\u001b[1;36m4\u001b[0m         \u001b[1;36m13\u001b[0m\n",
       "\u001b[1;36m5\u001b[0m          \u001b[1;36m8\u001b[0m\n",
       "\u001b[1;36m8\u001b[0m          \u001b[1;36m5\u001b[0m\n",
       "\u001b[1;36m13\u001b[0m         \u001b[1;36m2\u001b[0m\n",
       "\u001b[1;36m9\u001b[0m          \u001b[1;36m1\u001b[0m\n",
       "\u001b[1;36m11\u001b[0m         \u001b[1;36m1\u001b[0m\n",
       "\u001b[1;36m6\u001b[0m          \u001b[1;36m1\u001b[0m\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_citations.af_indirect_citations.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-04ef56fa34ff4b5ebceed0673fa21760.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-04ef56fa34ff4b5ebceed0673fa21760.vega-embed details,\n",
       "  #altair-viz-04ef56fa34ff4b5ebceed0673fa21760.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-04ef56fa34ff4b5ebceed0673fa21760\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-04ef56fa34ff4b5ebceed0673fa21760\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-04ef56fa34ff4b5ebceed0673fa21760\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-41ecc575f49c00b4f7bc796c69d5c31c\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"tooltip\": [{\"field\": \"title\", \"type\": \"nominal\"}, {\"field\": \"af_indirect_citations\", \"type\": \"quantitative\"}], \"x\": {\"field\": \"af_indirect_citations\", \"title\": \"Number of Papers\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"title\", \"sort\": \"-x\", \"title\": \"Paper Title\", \"type\": \"nominal\"}}, \"height\": 2400, \"title\": \"Count of Papers that Indirectly Cite Alphafold\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-41ecc575f49c00b4f7bc796c69d5c31c\": [{\"work_id\": \"W4310959824\", \"doi_x\": \"10.1093/plcell/koac346\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Beyond transcription: compelling open questions in plant RNA biology\", \"abstract\": \"The study of RNAs has become one of the most influential research fields in contemporary biology and biomedicine. In the last few years, new sequencing technologies have produced an explosion of new and exciting discoveries in the field but have also given rise to many open questions. Defining these questions, together with old, long-standing gaps in our knowledge, is the spirit of this article. The breadth of topics within RNA biology research is vast, and every aspect of the biology of these molecules contains countless exciting open questions. Here, we asked 12 groups to discuss their most compelling question among some plant RNA biology topics. The following vignettes cover RNA alternative splicing; RNA dynamics; RNA translation; RNA structures; R-loops; epitranscriptomics; long non-coding RNAs; small RNA production and their functions in crops; small RNAs during gametogenesis and in cross-kingdom RNA interference; and RNA-directed DNA methylation. In each section, we will present the current state-of-the-art in plant RNA biology research before asking the questions that will surely motivate future discoveries in the field. We hope this article will spark a debate about the future perspective on RNA biology and provoke novel reflections in the reader.\", \"doi_y\": \"https://doi.org/10.1093/plcell/koac346\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4292240021\", \"doi_x\": \"10.1016/j.cell.2022.07.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Bacteriophages benefit from mobilizing pathogenicity islands encoding immune systems against competitors\", \"abstract\": \"Bacteria encode sophisticated anti-phage systems that are diverse and versatile and display high genetic mobility. How this variability and mobility occurs remains largely unknown. Here, we demonstrate that a widespread family of pathogenicity islands, the phage-inducible chromosomal islands (PICIs), carry an impressive arsenal of defense mechanisms, which can be disseminated intra- and inter-generically by helper phages. These defense systems provide broad immunity, blocking not only phage reproduction, but also plasmid and non-cognate PICI transfer. Our results demonstrate that phages can mobilize PICI-encoded immunity systems to use them against other mobile genetic elements, which compete with the phages for the same bacterial hosts. Therefore, despite the cost, mobilization of PICIs may be beneficial for phages, PICIs, and bacteria in nature. Our results suggest that PICIs are important players controlling horizontal gene transfer and that PICIs and phages establish mutualistic interactions that drive bacterial ecology and evolution.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.07.014\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4293567428\", \"doi_x\": \"10.1107/s2059798322007987\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"abstract\": \"Nowadays, progress in the determination of three-dimensional macromolecular structures from diffraction images is achieved partly at the cost of increasing data volumes. This is due to the deployment of modern high-speed, high-resolution detectors, the increased complexity and variety of crystallographic software, the use of extensive databases and high-performance computing. This limits what can be accomplished with personal, offline, computing equipment in terms of both productivity and maintainability. There is also an issue of long-term data maintenance and availability of structure-solution projects as the links between experimental observations and the final results deposited in the PDB. In this article, CCP4 Cloud, a new front-end of the CCP4 software suite, is presented which mitigates these effects by providing an online, cloud-based environment for crystallographic computation. CCP4 Cloud was developed for the efficient delivery of computing power, database services and seamless integration with web resources. It provides a rich graphical user interface that allows project sharing and long-term storage for structure-solution projects, and can be linked to data-producing facilities. The system is distributed with the CCP4 software suite version 7.1 and higher, and an online publicly available instance of CCP4 Cloud is provided by CCP4.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322007987\", \"publication_date\": \"2022-08-30\"}, {\"work_id\": \"W4297489083\", \"doi_x\": \"10.1083/jcb.202112068\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Seipin concentrates distinct neutral lipids via interactions with their acyl chain carboxyl esters\", \"abstract\": \"Lipid droplets (LDs) are essential for cellular lipid homeostasis by storing diverse neutral lipids (NLs), such as triacylglycerol (TAG), steryl esters (SE), and retinyl esters (RE). A proper assembly of TAG-containing LDs at the ER requires Seipin, a conserved protein often mutated in lipodystrophies. Here, we show that the yeast Seipin Sei1 and its partner Ldb16 also promote the storage of other NL in LDs. Importantly, this role of Sei1/Ldb16 is evolutionarily conserved as expression of human-Seipin restored normal SE-containing LDs in yeast Seipin mutants. As in the case of TAG, the formation of SE-containing LDs requires interactions between hydroxyl-residues in human Seipin or yeast Ldb16 with NL carboxyl esters. These findings provide a universal mechanism for Seipin-mediated LD formation and suggest a model for how Seipin distinguishes NLs from aliphatic phospholipid acyl chains in the center of the membrane bilayer.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202112068\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4378070558\", \"doi_x\": \"10.1021/acssynbio.3c00231\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"abstract\": \"Synthetic biology applications would benefit from protein modules of reduced complexity that function orthogonally to cellular components. As many subcellular processes depend on peptide\\u2013protein or protein\\u2013protein interactions, de novo designed polypeptides that can bring together other proteins controllably are particularly useful. Thanks to established sequence-to-structure relationships, helical bundles provide good starting points for such designs. Typically, however, such designs are tested in vitro and function in cells is not guaranteed. Here, we describe the design, characterization, and application of de novo helical hairpins that heterodimerize to form 4-helix bundles in cells. Starting from a rationally designed homodimer, we construct a library of helical hairpins and identify complementary pairs using bimolecular fluorescence complementation in E. coli. We characterize some of the pairs using biophysics and X-ray crystallography to confirm heterodimeric 4-helix bundles. Finally, we demonstrate the function of an exemplar pair in regulating transcription in both E. coli and mammalian cells.\", \"doi_y\": \"https://doi.org/10.1021/acssynbio.3c00231\", \"publication_date\": \"2023-05-24\"}, {\"work_id\": \"W4292295879\", \"doi_x\": \"10.1039/d2cs00387b\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Discovery of small molecule ligands for the von Hippel-Lindau (VHL) E3 ligase and their use as inhibitors and PROTAC degraders\", \"abstract\": \"The von Hippel-Lindau (VHL) Cullin RING E3 ligase is an essential enzyme in the ubiquitin-proteasome system that recruits substrates such as the hypoxia inducible factor for ubiquitination and subsequent proteasomal degradation. The ubiquitin-proteasome pathway can be hijacked toward non-native neo-substrate proteins using proteolysis targeting chimeras (PROTACs), bifunctional molecules designed to simultaneously bind to an E3 ligase and a target protein to induce target ubiquitination and degradation. The availability of high-quality small-molecule ligands with good binding affinity for E3 ligases is fundamental for PROTAC development. Lack of good E3 ligase ligands as starting points to develop PROTAC degraders was initially a stumbling block to the development of the field. Herein, the journey towards the design of small-molecule ligands binding to VHL is presented. We cover the structure-based design of VHL ligands, their application as inhibitors in their own right, and their implementation into rationally designed, potent PROTAC degraders of various target proteins. We highlight the key findings and learnings that have provided strong foundations for the remarkable development of targeted protein degradation, and that offer a blueprint for designing new ligands for E3 ligases beyond VHL.\", \"doi_y\": \"https://doi.org/10.1039/d2cs00387b\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4292651681\", \"doi_x\": \"10.1073/pnas.2205041119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Repeat sequences limit the effectiveness of lateral gene transfer and favored the evolution of meiotic sex in early eukaryotes\", \"abstract\": \"The transition from prokaryotic lateral gene transfer to eukaryotic meiotic sex is poorly understood. Phylogenetic evidence suggests that it was tightly linked to eukaryogenesis, which involved an unprecedented rise in both genome size and the density of genetic repeats. Expansion of genome size raised the severity of Muller's ratchet, while limiting the effectiveness of lateral gene transfer (LGT) at purging deleterious mutations. In principle, an increase in recombination length combined with higher rates of LGT could solve this problem. Here, we show using a computational model that this solution fails in the presence of genetic repeats prevalent in early eukaryotes. The model demonstrates that dispersed repeat sequences allow ectopic recombination, which leads to the loss of genetic information and curtails the capacity of LGT to prevent mutation accumulation. Increasing recombination length in the presence of repeat sequences exacerbates the problem. Mutational decay can only be resisted with homology along extended sequences of DNA. We conclude that the transition to homologous pairing along linear chromosomes was a key innovation in meiotic sex, which was instrumental in the expansion of eukaryotic genomes and morphological complexity.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2205041119\", \"publication_date\": \"2022-08-22\"}, {\"work_id\": \"W4381548756\", \"doi_x\": \"10.1038/s41467-023-38944-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cyclic di-AMP traps proton-coupled K+ transporters of the KUP family in an inward-occluded conformation\", \"abstract\": \"Cyclic di-AMP is the only known essential second messenger in bacteria and archaea, regulating different proteins indispensable for numerous physiological processes. In particular, it controls various potassium and osmolyte transporters involved in osmoregulation. In Bacillus subtilis, the K+/H+ symporter KimA of the KUP family is inactivated by c-di-AMP. KimA sustains survival at potassium limitation at low external pH by mediating potassium ion uptake. However, at elevated intracellular K+ concentrations, further K+ accumulation would be toxic. In this study, we reveal the molecular basis of how c-di-AMP binding inhibits KimA. We report cryo-EM structures of KimA with bound c-di-AMP in detergent solution and reconstituted in amphipols. By combining structural data with functional assays and molecular dynamics simulations we reveal how c-di-AMP modulates transport. We show that an intracellular loop in the transmembrane domain interacts with c-di-AMP bound to the adjacent cytosolic domain. This reduces the mobility of transmembrane helices at the cytosolic side of the K+ binding site and therefore traps KimA in an inward-occluded conformation.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-38944-1\", \"publication_date\": \"2023-06-21\"}, {\"work_id\": \"W4321016016\", \"doi_x\": \"10.1016/j.tips.2022.12.002\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"abstract\": \"Up to 1.5 million people die yearly from fungal disease, but the repertoire of antifungal drug classes is minimal and the incidence of drug resistance is rising rapidly. This dilemma was recently declared by the World Health Organization as a global health emergency, but the discovery of new antifungal drug classes remains excruciatingly slow. This process could be accelerated by focusing on novel targets, such as G protein-coupled receptor (GPCR)-like proteins, that have a high likelihood of being druggable and have well-defined biology and roles in disease. We discuss recent successes in understanding the biology of virulence and in structure determination of yeast GPCRs, and highlight new approaches that might pay significant dividends in the urgent search for novel antifungal drugs.\", \"doi_y\": \"https://doi.org/10.1016/j.tips.2022.12.002\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4307342094\", \"doi_x\": \"10.1111/nph.18573\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic and functional analysis of phage\\u2010mediated horizontal gene transfer in <i>Pseudomonas syringae</i> on the plant surface\", \"abstract\": \"Many strains of Pseudomonas colonise plant surfaces, including the cherry canker pathogens, Pseudomonas syringae pathovars syringae and morsprunorum. We have examined the genomic diversity of P. syringae in the cherry phyllosphere and focused on the role of prophages in transfer of genes encoding Type 3 secreted effector (T3SE) proteins contributing to the evolution of virulence. Phylogenomic analysis was carried out on epiphytic pseudomonads in the UK orchards. Significant differences in epiphytic populations occurred between regions. Nonpathogenic strains were found to contain reservoirs of T3SE genes. Members of P. syringae phylogroups 4 and 10 were identified for the first time from Prunus. Using bioinformatics, we explored the presence of the gene encoding T3SE HopAR1 within related prophage sequences in diverse P. syringae strains including cherry epiphytes and pathogens. Results indicated that horizontal gene transfer (HGT) of this effector between phylogroups may have involved phage. Prophages containing hopAR1 were demonstrated to excise, circularise and transfer the gene on the leaf surface. The phyllosphere provides a dynamic environment for prophage-mediated gene exchange and the potential for the emergence of new more virulent pathotypes. Our results suggest that genome-based epidemiological surveillance of environmental populations will allow the timely application of control measures to prevent damaging diseases.\", \"doi_y\": \"https://doi.org/10.1111/nph.18573\", \"publication_date\": \"2022-12-02\"}, {\"work_id\": \"W4281754639\", \"doi_x\": \"10.1016/j.cell.2022.06.005\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Antibody escape of SARS-CoV-2 Omicron BA.4 and BA.5 from vaccine and BA.1 serum\", \"abstract\": \"The Omicron lineage of SARS-CoV-2, which was first described in November 2021, spread rapidly to become globally dominant and has split into a number of sublineages. BA.1 dominated the initial wave but has been replaced by BA.2 in many countries. Recent sequencing from South Africa\\u2019s Gauteng region uncovered two new sublineages, BA.4 and BA.5, which are taking over locally, driving a new wave. BA.4 and BA.5 contain identical spike sequences, and although closely related to BA.2, they contain further mutations in the receptor-binding domain of their spikes. Here, we study the neutralization of BA.4/5 using a range of vaccine and naturally immune serum and panels of monoclonal antibodies. BA.4/5 shows reduced neutralization by the serum from individuals vaccinated with triple doses of AstraZeneca or Pfizer vaccine compared with BA.1 and BA.2. Furthermore, using the serum from BA.1 vaccine breakthrough infections, there are, likewise, significant reductions in the neutralization of BA.4/5, raising the possibility of repeat Omicron infections.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.06.005\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4385344743\", \"doi_x\": \"10.1007/s40588-023-00195-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Conserved and Divergent Features of pH Sensing in Major Fungal Pathogens\", \"abstract\": \"Abstract Purpose of Review For human fungal pathogens, sensory perception of extracellular pH is essential for colonisation of mammalian tissues and immune evasion. The molecular complexes that perceive and transmit the fungal pH signal are membrane-proximal and essential for virulence and are therefore of interest as novel antifungal drug targets. Intriguingly, the sensory machinery has evolved divergently in different fungal pathogens, yet spatial co-ordination of cellular components is conserved. Recent Findings The recent discovery of a novel pH sensor in the basidiomycete pathogen Cryptococcus neformans highlights that, although the molecular conservation of fungal pH sensors is evolutionarily restricted, their subcellular localisation and coupling to essential components of the cellular ESCRT machinery are consistent features of the cellular pH sensing and adaptation mechanism. In both basidiomycetes and ascomycetes, the lipid composition of the plasma membrane to which pH sensing complexes are localised appears to have pivotal functional importance. Endocytosis of pH-sensing complexes occurs in multiple fungal species, but its relevance for signal transduction appears not to be universal. Summary Our overview of current understanding highlights conserved and divergent mechanisms of the pH sensing machinery in model and pathogenic fungal species, as well as important unanswered questions that must be addressed to inform the future study of such sensing mechanisms and to devise therapeutic strategies for manipulating them.\", \"doi_y\": \"https://doi.org/10.1007/s40588-023-00195-5\", \"publication_date\": \"2023-07-28\"}, {\"work_id\": \"W4321380826\", \"doi_x\": \"10.1039/d2tb02781j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Synthetic engineering of a new biocatalyst encapsulating [NiFe]-hydrogenases for enhanced hydrogen production\", \"abstract\": \"By incorporating [NiFe]-hydrogenases into a proteinaceous carboxysome shell, we generate a novel biocatalyst that has improved production of clean hydrogen, oxygen tolerance, and thermostability, highlighting its great potential in biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1039/d2tb02781j\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385839904\", \"doi_x\": \"10.3390/v15081744\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evaluation of Nafamostat as Chemoprophylaxis for SARS-CoV-2 Infection in Hamsters\", \"abstract\": \"The successful development of a chemoprophylaxis against SARS-CoV-2 could provide a tool for infection prevention that is implementable alongside vaccination programmes. Nafamostat is a serine protease inhibitor that inhibits SARS-CoV-2 entry in vitro, but it has not been characterised for chemoprophylaxis in animal models. Clinically, nafamostat is limited to intravenous delivery and has an extremely short plasma half-life. This study sought to determine whether intranasal dosing of nafamostat at 5 mg/kg twice daily was able to prevent the airborne transmission of SARS-CoV-2 from infected to uninfected Syrian Golden hamsters. SARS-CoV-2 RNA was detectable in the throat swabs of the water-treated control group 4 days after cohabitation with a SARS-CoV-2 inoculated hamster. However, throat swabs from the intranasal nafamostat-treated hamsters remained SARS-CoV-2 RNA negative for the full 4 days of cohabitation. Significantly lower SARS-CoV-2 RNA concentrations were seen in the nasal turbinates of the nafamostat-treated group compared to the control (p = 0.001). A plaque assay quantified a significantly lower concentration of infectious SARS-CoV-2 in the lungs of the nafamostat-treated group compared to the control (p = 0.035). When taken collectively with the pathological changes observed in the lungs and nasal mucosa, these data are strongly supportive of the utility of intranasally delivered nafamostat for the prevention of SARS-CoV-2 infection.\", \"doi_y\": \"https://doi.org/10.3390/v15081744\", \"publication_date\": \"2023-08-15\"}, {\"work_id\": \"W4307918867\", \"doi_x\": \"10.1002/ange.202213053\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"abstract\": \"Abstract Abyssomicin C and its atropisomer are potent inhibitors of bacterial folate metabolism. They possess complex polycyclic structures, and their biosynthesis has been shown to involve several unusual enzymatic transformations. Using a combination of synthesis and in vitro assays we reveal that AbyV, a cytochrome P450 enzyme from the aby gene cluster, catalyses a key late\\u2010stage epoxidation required for the installation of the characteristic ether\\u2010bridged core of abyssomicin C. The X\\u2010ray crystal structure of AbyV has been determined, which in combination with molecular dynamics simulations provides a structural framework for our functional data. This work demonstrates the power of combining selective carbon\\u201013 labelling with NMR spectroscopy as a sensitive tool to interrogate enzyme\\u2010catalysed reactions in vitro with no need for purification.\", \"doi_y\": \"https://doi.org/10.1002/ange.202213053\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4206493948\", \"doi_x\": \"10.1002/cbic.202100688\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular Determinants of Carbocation Cyclisation in Bacterial Monoterpene Synthases\", \"abstract\": \"Abstract Monoterpene synthases are often promiscuous enzymes, yielding product mixtures rather than pure compounds due to the nature of the branched reaction mechanism involving reactive carbocations. Two previously identified bacterial monoterpene synthases, a linalool synthase (bLinS) and a cineole synthase (bCinS), produce nearly pure linalool and cineole from geranyl diphosphate, respectively. We used a combined experimental and computational approach to identify critical residues involved in bacterial monoterpenoid synthesis. Phe77 is essential for bCinS activity, guiding the linear carbocation intermediate towards the formation of the cyclic \\u03b1\\u2010terpinyl intermediate; removal of the aromatic ring results in variants that produce acyclic products only. Computational chemistry confirmed the importance of Phe77 in carbocation stabilisation. Phe74, Phe78 and Phe179 are involved in maintaining the active site shape in bCinS without a specific role for the aromatic ring. Phe295 in bLinS, and the equivalent Ala301 in bCinS, are essential for linalool and cineole formation, respectively. Where Phe295 places steric constraints on the carbocation intermediates, Ala301 is essential for bCinS initial cyclisation and activity. Our multidisciplinary approach gives unique insights into how carefully placed amino acid residues in the active site can direct carbocations down specific paths, by placing steric constraints or offering stabilisation via cation\\u2010\\u03c0 interactions.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202100688\", \"publication_date\": \"2022-01-19\"}, {\"work_id\": \"W4361301517\", \"doi_x\": \"10.1021/acssynbio.3c00064\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Rational Design of Phosphorylation-Responsive Coiled Coil-Peptide Assemblies\", \"abstract\": \"De novo peptides and proteins that switch state in response to chemical and physical cues would advance protein design and synthetic biology. Here we report two designed systems that disassemble and reassemble upon site-specific phosphorylation and dephosphorylation, respectively. As starting points, we use hyperthermostable de novo antiparallel and parallel coiled-coil heterotetramers, i.e., A2B2 systems, to afford control in downstream applications. The switches are incorporated by adding protein kinase A phosphorylation sites, R-R-X-S, with the phosphoacceptor serine residues placed to maximize disruption of the coiled-coil interfaces. The unphosphorylated peptides assemble as designed and unfold reversibly when heated. Addition of kinase to the assembled states unfolds them with half-lives of \\u22645 min. Phosphorylation is reversed by Lambda Protein Phosphatase resulting in tetramer reassembly. We envisage that the new de novo designed coiled-coil components, the switches, and a mechanistic model for them will be useful in synthetic biology, biomaterials, and biotechnology applications.\", \"doi_y\": \"https://doi.org/10.1021/acssynbio.3c00064\", \"publication_date\": \"2023-03-29\"}, {\"work_id\": \"W4293168376\", \"doi_x\": \"10.1038/s41467-022-32774-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A ribosomally synthesised and post-translationally modified peptide containing a \\u03b2-enamino acid and a macrocyclic motif\", \"abstract\": \"Abstract Ribosomally synthesized and post-translationally modified peptides (RiPPs) are structurally complex natural products with diverse bioactivities. Here we report discovery of a RiPP, kintamdin, for which the structure is determined through spectroscopy, spectrometry and genomic analysis to feature a bis -thioether macrocyclic ring and a \\u03b2-enamino acid residue. Biosynthetic investigation demonstrated that its pathway relies on four dedicated proteins: phosphotransferase KinD, Lyase KinC, kinase homolog KinH and flavoprotein KinI, which share low homologues to enzymes known in other RiPP biosynthesis. During the posttranslational modifications, KinCD is responsible for the formation of the characteristic dehydroamino acid residues including the \\u03b2-enamino acid residue, followed by oxidative decarboxylation on the C-terminal Cys and subsequent cyclization to provide the bis -thioether ring moiety mediated by coordinated action of KinH and KinI. Finally, conserved genomic investigation allows further identification of two kintamdin-like peptides among the kin -like BGCs, suggesting the occurrence of RiPPs from actinobacteria.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-32774-3\", \"publication_date\": \"2022-08-26\"}, {\"work_id\": \"W4313455881\", \"doi_x\": \"10.1101/2022.12.20.521207\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Photovoltaic enzymes by design and evolution\", \"abstract\": \"ABSTRACT The global energy crisis challenges us to develop more efficient strategies for the sustainable production of energy. Given the excellent efficiency of the natural photosynthetic apparatus, biohybrid photovoltaic devices present an attractive solution for solar energy conversion. However, their composition, stability, and complexity can limit their inclusion into photovoltaic devices. Here, we combined computational design and directed evolution to overcome these limitations and create tailor-made photoenzymes. Photo-biocatalysts were designed by introducing photosensitizer binding sites into heme-containing helical bundle proteins. The designed binding sites were specific for the target photosensitizer and readily transplanted into other helical bundles. The best design was highly evolvable and reached nanomolar ligand affinity after mutagenesis and screening. The evolved enzyme generated 2.6 times higher photocurrents than the photosensitizer alone, primarily driven by increased photostability. Evolvability is a unique advantage of our protein-based approach over abiological photovoltaic and will be critical to developing efficient biohybrid systems. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.12.20.521207\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4311291543\", \"doi_x\": \"10.1039/d2sc04072g\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of complexes III and IV\", \"abstract\": \"Cardiolipin interacts more strongly than a disease-related lipid, monolysocardiolipin, at the interface of Complex III and Complex IV in mitochondrial respiratory supercomplexes.\", \"doi_y\": \"https://doi.org/10.1039/d2sc04072g\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4312202064\", \"doi_x\": \"10.1186/s13059-022-02837-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Implicating genes, pleiotropy, and sexual dimorphism at blood lipid loci through multi-ancestry meta-analysis\", \"abstract\": \"Genetic variants within nearly 1000 loci are known to contribute to modulation of blood lipid levels. However, the biological pathways underlying these associations are frequently unknown, limiting understanding of these findings and hindering downstream translational efforts such as drug target discovery.To expand our understanding of the underlying biological pathways and mechanisms controlling blood lipid levels, we leverage a large multi-ancestry meta-analysis (N = 1,654,960) of blood lipids to prioritize putative causal genes for 2286 lipid associations using six gene prediction approaches. Using phenome-wide association (PheWAS) scans, we identify relationships of genetically predicted lipid levels to other diseases and conditions. We confirm known pleiotropic associations with cardiovascular phenotypes and determine novel associations, notably with cholelithiasis risk. We perform sex-stratified GWAS meta-analysis of lipid levels and show that 3-5% of autosomal lipid-associated loci demonstrate sex-biased effects. Finally, we report 21 novel lipid loci identified on the X chromosome. Many of the sex-biased autosomal and X chromosome lipid loci show pleiotropic associations with sex hormones, emphasizing the role of hormone regulation in lipid metabolism.Taken together, our findings provide insights into the biological mechanisms through which associated variants lead to altered lipid levels and potentially cardiovascular disease risk.\", \"doi_y\": \"https://doi.org/10.1186/s13059-022-02837-1\", \"publication_date\": \"2022-12-27\"}, {\"work_id\": \"W3206920052\", \"doi_x\": \"10.1042/bcj20210535\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"abstract\": \"Optimising the function of a protein of length N amino acids by directed evolution involves navigating a 'search space' of possible sequences of some 20N. Optimising the expression levels of P proteins that materially affect host performance, each of which might also take 20 (logarithmically spaced) values, implies a similar search space of 20P. In this combinatorial sense, then, the problems of directed protein evolution and of host engineering are broadly equivalent. In practice, however, they have different means for avoiding the inevitable difficulties of implementation. The spare capacity exhibited in metabolic networks implies that host engineering may admit substantial increases in flux to targets of interest. Thus, we rehearse the relevant issues for those wishing to understand and exploit those modern genome-wide host engineering tools and thinking that have been designed and developed to optimise fluxes towards desirable products in biotechnological processes, with a focus on microbial systems. The aim throughput is 'making such biology predictable'. Strategies have been aimed at both transcription and translation, especially for regulatory processes that can affect multiple targets. However, because there is a limit on how much protein a cell can produce, increasing kcat in selected targets may be a better strategy than increasing protein expression levels for optimal host engineering.\", \"doi_y\": \"https://doi.org/10.1042/bcj20210535\", \"publication_date\": \"2021-10-21\"}, {\"work_id\": \"W4288988319\", \"doi_x\": \"10.1016/j.molp.2022.07.012\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"abstract\": \"Species of the genus Phytophthora, the plant killer, cause disease and reduce yields in many crop plants. Although many Resistance to Phytophthora infestans (Rpi) genes effective against potato late blight have been cloned, few have been cloned against other Phytophthora species. Most Rpi genes encode nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptor proteins that recognize RXLR (Arg-X-Leu-Arg) effectors. However, whether NLR proteins can recognize RXLR effectors from multiple Phytophthora species has rarely been investigated. Here, we identified a new RXLR-WY effector AVRamr3 from P. infestans that is recognized by Rpi-amr3 from a wild Solanaceae species Solanum americanum. Rpi-amr3 associates with AVRamr3 in planta. AVRamr3 is broadly conserved in many different Phytophthora species, and the recognition of AVRamr3 homologs by Rpi-amr3 activates resistance against multiple Phytophthora pathogens, including the tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is thus the first characterized resistance gene that acts against P. parasitica or P. palmivora. These findings suggest a novel path to redeploy known R genes against different important plant pathogens. Species of the genus Phytophthora, the plant killer, cause disease and reduce yields in many crop plants. Although many Resistance to Phytophthora infestans (Rpi) genes effective against potato late blight have been cloned, few have been cloned against other Phytophthora species. Most Rpi genes encode nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptor proteins that recognize RXLR (Arg-X-Leu-Arg) effectors. However, whether NLR proteins can recognize RXLR effectors from multiple Phytophthora species has rarely been investigated. Here, we identified a new RXLR-WY effector AVRamr3 from P. infestans that is recognized by Rpi-amr3 from a wild Solanaceae species Solanum americanum. Rpi-amr3 associates with AVRamr3 in planta. AVRamr3 is broadly conserved in many different Phytophthora species, and the recognition of AVRamr3 homologs by Rpi-amr3 activates resistance against multiple Phytophthora pathogens, including the tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is thus the first characterized resistance gene that acts against P. parasitica or P. palmivora. These findings suggest a novel path to redeploy known R genes against different important plant pathogens. IntroductionSpecies in the oomycete genus Phytophthora cause many devastating plant diseases. For example, P. infestans, P. parasitica, P. cactorum, P. ramorum, P. sojae, P. palmivora, and P. megakarya cause potato and tomato late blight, tobacco black shank disease, strawberry crown and leather rot, sudden oak death, soybean root and stem rot, and cacao black pod disease, respectively. P. infestans and P. sojae infect few plant species, while others such as P. parasitica, P. ramorum, and P. palmivora have a broad host range (Kamoun et al., 2015Kamoun S. Furzer O. Jones J.D.G. Judelson H.S. Ali G.S. Dalio R.J.D. Roy S.G. Schena L. Zambounis A. Panabi\\u00e8res F. et al.The Top 10 oomycete pathogens in molecular plant pathology.Mol. Plant Pathol. 2015; 16: 413-434Crossref PubMed Scopus (451) Google Scholar).Plant immunity involves detection of pathogen-derived molecules by either cell-surface pattern recognition immune receptors or intracellular nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptors, which activate either pattern-triggered immunity (PTI) or effector-triggered immunity (ETI), respectively (Jones and Dangl, 2006Jones J.D.G. Dangl J.L. The plant immune system.Nature. 2006; 444: 323-329Crossref PubMed Scopus (8005) Google Scholar). So far, more than 20 Resistance to P. infestans (Rpi) genes were cloned from wild Solanum species that confer resistance against potato late blight (Vleeshouwers et al., 2011Vleeshouwers V.G.A.A. Raffaele S. Vossen J.H. Champouret N. Oliva R. Segretin M.E. Rietman H. Cano L.M. Lokossou A. Kessel G. et al.Understanding and exploiting late blight resistance in the age of effectors.Annu. Rev. Phytopathol. 2011; 49: 507-531Crossref PubMed Scopus (276) Google Scholar). Several Resistance genes against P. sojae (Rps) have also been mapped in different soybean accessions, and a few were cloned (Sahoo et al., 2017Sahoo D.K. Abeysekara N.S. Cianzio S.R. Robertson A.E. Bhattacharyya M.K. A novel Phytophthora sojae resistance Rps12 gene mapped to a genomic region that contains several Rps genes.PLoS ONE. 2017; 12: e0169950Crossref PubMed Scopus (40) Google Scholar; Wang et al., 2021Wang W. Chen L. Fengler K. Bolar J. Llaca V. Wang X. Clark C.B. Fleury T.J. Myrvold J. Oneal D. et al.A giant NLR gene confers broad-spectrum resistance to Phytophthora sojae in soybean.Nat. Commun. 2021; 12: 6263-6268Crossref PubMed Scopus (8) Google Scholar). In tobacco, the black shank resistance genes Phl, Php, and Ph were genetically mapped, but not yet cloned; these confer race-specific resistance to P. parasitica (also known as [aka] P. nicotianae) isolates (Gallup and Shew, 2010Gallup C.A. Shew H.D. Occurrence of race 3 of Phytophthora nicotianae in North Carolina, the causal agent of black shank of tobacco.Plant Dis. 2010; 94: 557-562Crossref PubMed Scopus (23) Google Scholar; Bao et al., 2019Bao Y. Ding N. Qin Q. Wu X. Martinez N. Miller R. Zaitlin D. Li D. Yang S. Genetic mapping of the Ph gene conferring disease resistance to black shank in tobacco.Mol. Breeding. 2019; 39: 122https://doi.org/10.1007/s11032-019-1036-xCrossref Scopus (3) Google Scholar). For P. palmivora, some resistant cacao (Theobroma cacao) accessions were identified, but no dominant R genes have been defined or cloned (Thevenin et al., 2012Thevenin J.-M. Rossi V. Ducamp M. Doare F. Condina V. Lachenaud P. Numerous clones resistant to Phytophthora palmivora in the \\u201cGuiana\\u201d genetic group of Theobroma cacao L.PLoS One. 2012; 7: e40915Crossref PubMed Scopus (19) Google Scholar). In summary, apart from Rpi genes, very few R genes against Phytophthora pathogens have been cloned.Solanum americanum and Solanum nigrum are wild Solanaceae species and are highly resistant to P. infestans (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar, Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). Two Rpi genes of coiled-coil (CC) type, Rpi-amr3 and Rpi-amr1, were cloned from different S. americanum accessions; both confer late blight resistance in cultivated potato (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar, Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). S. nigrum is a hexaploid species that was thought to be a \\u201cnon-host\\u201d plant of P. infestans. No Rpi gene had been cloned from S. nigrum until we reported the functional Rpi-amr1 homolog Rpi-nig1 (Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar).In oomycetes, the recognized effectors are usually secreted RXLR (Arg-X-Leu-Arg, X represents any amino acid)-EER (Glu-Glu-Arg) proteins that are translocated into plant cells (Rehmany et al., 2005Rehmany A.P. Gordon A. Rose L.E. Allen R.L. Armstrong M.R. Whisson S.C. Kamoun S. Tyler B.M. Birch P.R.J. Beynon J.L. Differential recognition of highly divergent downy mildew avirulence gene alleles by RPP1 resistance genes from two Arabidopsis lines.Plant Cell. 2005; 17: 1839-1850Crossref PubMed Scopus (319) Google Scholar; Wang et al., 2019Wang S. McLellan H. Bukharova T. He Q. Murphy F. Shi J. Sun S. van Weymers P. Ren Y. Thilliez G. et al.Phytophthora infestans RXLR effectors act in concert at diverse subcellular locations to enhance host colonization.J. Exp. Bot. 2019; 70: 343-356Crossref PubMed Scopus (34) Google Scholar). Dozens of Avirulence (Avr) genes encoding recognized effectors from Phytophthora species have been identified, and they are typically fast-evolving and lineage-specific molecules (Jiang et al., 2008Jiang R.H.Y. Tripathy S. Govers F. Tyler B.M. RXLR effector reservoir in two Phytophthora species is dominated by a single rapidly evolving superfamily with more than 700 members.Proc. Natl. Acad. Sci. USA. 2008; 105: 4874-4879Crossref PubMed Scopus (304) Google Scholar). Recently, AVRamr1 (PITG_07569), the recognized effector of Rpi-amr1, was identified by a cDNA pathogen enrichment sequencing approach (Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar). Surprisingly, AVRamr1 homologs were identified from P. parasitica and P. cactorum genomes, and both are recognized by all Rpi-amr1 variants (Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). Similarly, AVR3a-like effectors were found in different Phytophthora species, including P. capsici and P. sojae, and the recognition of AVR3a homologs correlates with P. capsici or P. sojae resistance in Nicotiana species and soybean (Shan et al., 2004Shan W. Cao M. Leung D. Tyler B.M. The Avr1b locus of Phytophthora sojae encodes an elicitor and a regulator required for avirulence on soybean plants carrying resistance gene Rps1b.Mol. Plant Microbe Interact. 2004; 17: 394-403Crossref PubMed Scopus (241) Google Scholar; Vega-Arregu\\u00edn et al., 2014Vega-Arregu\\u00edn J.C. Jalloh A. Bos J.I. Moffett P. Recognition of an Avr3a homologue plays a major role in mediating nonhost resistance to Phytophthora capsici in Nicotiana species.Mol. Plant Microbe Interact. 2014; 27: 770-780Crossref PubMed Scopus (48) Google Scholar). In addition, AVRblb2 homologs from P. andina and P. mirabilis trigger a hypersensitive response (HR) with Rpi-blb2 (Oliva et al., 2015Oliva R.F. Cano L.M. Raffaele S. Win J. Bozkurt T.O. Belhaj K. Oh S.-K. THINES M. Kamoun S. A recent expansion of the RXLR effector gene Avrblb2 is maintained in global populations of Phytophthora infestans indicating different contributions to virulence.Mol. Plant Microbe Interact. 2015; 28: 901-912Crossref PubMed Scopus (25) Google Scholar). Remarkably, a single N336Y mutation in R3a expands its recognition specificity to a P. capsici AVR3a homolog (Segretin et al., 2014Segretin M.E. Pais M. Franceschetti M. Chaparro-Garcia A. Bos J.I.B. Banfield M.J. Kamoun S. Single amino acid mutations in the potato immune receptor R3a expand response to Phytophthora effectors.Mol. Plant Microbe Interact. 2014; 27: 624-637Crossref PubMed Scopus (80) Google Scholar). These reports raise intriguing questions. Could RXLR effectors be widely conserved molecules among different Phytophthora species? Could these effectors be recognized by the same plant immune receptor? Of particular interest, could such effector recognition capacity enable disease resistance?Here we show that Rpi-amr3 confers resistance to all tested late blight isolates in both the field and laboratory conditions. We also identified AVRamr3, a novel AVR protein from P. infestans, by screening an RXLR effector library. AVRamr3 is a broadly conserved effector found in 13 different Phytophthora species. We also show functional Rpi-amr3 genes are widely distributed among S. americanum and S. nigrum accessions. The recognition of AVRamr3 not only enables resistance to a wide range of P. infestans isolates but also to other economically important Phytophthora pathogens such as tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is the first reported R gene that confers resistance against P. parasitica and P. palmivora.ResultsRpi-amr3 confers late blight resistance in the laboratory and field conditionsRpi-amr3 was cloned by SMRT-RenSeq and reported to confer resistance against two Phytophthora infestans isolates, 88069 and 06_3928A, in a diploid potato line (Line 26, Solynta B.V.) (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar) in the laboratory conditions. However, whether Rpi-amr3 confers broad-spectrum and field resistance to late blight was not reported.To address this, we transformed Rpi-amr3 with its native promoter and terminator into a favored UK potato cultivar cv. Maris Piper. Two lines (SLJ24895-5C and SLJ24895-9A) (Supplemental Figure 1B) were selected for a field experiment in 2017, and SLJ24895-5C was further tested in the field in 2018 (Figure 1A ). These field trials indicate that Rpi-amr3 confers protection against potato late blight in field conditions, while the wild-type Maris Piper control lines were infected completely within \\u223c3 weeks once disease symptoms appeared (Figures 1A and 1B). As a result, the tuber yield of the Rpi-amr3 transgenic lines was significantly higher than the wild-type Maris Piper lines (Figures 1C and 1D). To determine the P. infestans genotypes present in the field trial, we sampled the infected leaves and genotyped them by SSR markers; most of the isolates corresponded to a dominant UK strain 6_A1 (aka Pink6) (Supplemental Table 1).To further evaluate the resistance spectrum of Rpi-amr3, we performed detached leaf assay (DLA) on SLJ24895-5C, with wild-type Maris Piper and Rpi-amr1 transgenic Maris Piper as controls. Seventeen P. infestans isolates with different origins and races were tested (Figure 1E, Supplemental Table 2). Our results show that Rpi-amr3 confers resistance against all tested isolates in potato in laboratory conditions but with different efficacy, and Rpi-amr3-mediated resistance is weaker than Rpi-amr1 in potato (Figure 1E). We also generated Rpi-amr3 stably transformed N. benthamiana lines. Two homozygous T2 lines, 13.3 and 16.5, were tested with nine P. infestans isolates. Both Rpi-amr3 transgenic N. benthamiana lines confer complete resistance to all tested P. infestans isolates (Supplemental Table 2).These data show that Rpi-amr3 confers potato late blight resistance in both the laboratory and field conditions.Avramr3 encodes a conserved RXLR-WY effector proteinTo identify the effector recognized by Rpi-amr3, we screened an RXLR effector library (Rietman, 2011Rietman H. Putting the Phytophthora Infestans Genome Sequence at Work: Multiple Novel Avirulence and Potato Resistance Gene Candidates Revealed.2011Google Scholar; Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar) of 311 RXLR effectors by Agrobacterium tumefaciens-mediated co-expression with Rpi-amr3 in N. benthamiana. Most of these effectors (296/311) do not induce a HR when expressed alone or co-expressed with Rpi-amr3; 14 effectors are auto-active in N. benthamiana, and we found PITG_21190 specifically induces an HR with Rpi-amr3 (Figure 2A, Supplemental Table 3); therefore, we concluded that PITG_21190 is Avramr3. Avramr3 encodes a 339-amino acid (aa) protein with a signal peptide followed by RXLR, EER motifs, and four predicted WY motifs (Win et al., 2012Win J. Krasileva K.V. Kamoun S. Shirasu K. Staskawicz B.J. Banfield M.J. Sequence divergent RXLR effectors share a structural fold conserved across plant pathogenic oomycete Species.PLoS Pathog. 2012; 8: e1002400Crossref PubMed Scopus (106) Google Scholar) (Figure 2C). To characterize the expression profile of Avramr3, eight P. infestans isolates (T30-4, 88069, NL01096, 06_3928A, 6_A1, EC1, US23, and 99183) were used to inoculate a susceptible potato cultivar Maris Piper; RNA was isolated 2 days after the infection for RT\\u2013PCR. Our data show that Avramr3 are expressed in all eight isolates at 2 days after infection (Supplemental Figure 1A).Figure 2Identification and characterization of AVRamr3.Show full caption(A) Co-expression of Rpi-amr3:HA and AVRamr3:HIS-FLAG triggers cell death on N. benthamiana; Rpi-amr1 and AVRamr1 were used as controls. The photos were taken 3 days after infiltration; Agrobacterium strain GV3101(pMP90) carrying Rpi-amr3:HA or AVRamr3:HIS-FLAG constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results.(B) Co-expression of Rpi-amr3:HA and AVRamr3 truncations. All truncations are tagged with a C-terminal HIS-FLAG tag. T3, T4, T8, and T10 trigger cell death when co-expressed with Rpi-amr3, but not T1, T2, T5, T6, T7, and T9. Full-length AVRamr3:HIS-FLAG was used as control. OD600 = 0.5. Three biological replicates were performed with the same results.(C) Cartoon of AVRamr3 (PITG_21190), a protein with 339 aa with a signal peptide (lemon), RXLR-EER motif (green), and an effector domain (red) with four predicted WY motifs (details are shown in Supplemental Figure 4). T1\\u2013T10 indicates the AVRamr3 truncations used in HR assays. Those that induce HR after co-expression with Rpi-amr3 are marked by orange bars, and otherwise by blue.(D) Rpi-amr3::HA and AVRamr3::HIS-FLAG constructs were used for a bidirectional co-immunoprecipitation experiment, with Rpi-amr1-HA and AVRamr1::HIS-FLAG used as control. After HA pull-down of Rpi-amr3::HA or Rpi-amr1::HA, only AVRamr3::HIS-FLAG is associated with Rpi-amr3::HA. After Flag pull-down of AVRamr3::HIS-FLAG or AVRamr1-HIS-FLAG, only Rpi-amr3::HA is associated with AVRamr3::HIS-FLAG. Agrobacterium strain GV3101(pMP90) carrying different constructs was used for transient expression in the nrc2/3/4 knockout N. benthamiana line (210.4.3) to abolish the cell death phenotype. OD600 = 0.5. Three biological replicates were performed with the same results.(E) Rpi-amr3::Cluc and AVRamr3::Nluc constructs were used to test their interaction in planta; Rpi-amr1::Cluc and AVRamr1::Nluc were used as controls. The luciferase signal can be detected only on Rpi-amr3::Cluc and AVRamr3::Nluc co-expression. The nrc2/3/4 knockout N. benthamiana line (210.4.3) was used to abolish the cell death phenotype.View Large Image Figure ViewerDownload Hi-res image Download (PPT)Many RXLR effectors are encoded by fast-evolving, multiple-member family genes with extensive sequence polymorphism, such as the Avr2 and Avrblb2 families (Gilroy et al., 2011Gilroy E.M. Breen S. Whisson S.C. Squires J. Hein I. Kaczmarek M. Turnbull D. Boevink P.C. Lokossou A. Cano L.M. et al.Presence/absence, differential expression and sequence polymorphisms between PiAVR2 and PiAVR2-like in Phytophthora infestans determine virulence on R2 plants.New Phytol. 2011; 191: 763-776Crossref PubMed Scopus (103) Google Scholar; Oliva et al., 2015Oliva R.F. Cano L.M. Raffaele S. Win J. Bozkurt T.O. Belhaj K. Oh S.-K. THINES M. Kamoun S. A recent expansion of the RXLR effector gene Avrblb2 is maintained in global populations of Phytophthora infestans indicating different contributions to virulence.Mol. Plant Microbe Interact. 2015; 28: 901-912Crossref PubMed Scopus (25) Google Scholar). To study the sequence polymorphism of Avramr3, we identified 17 additional Avramr3 homologs from 11 isolates from published databases (KR_1, 3928A, EC1, 6_A1, and US23) (Lee et al., 2020Lee Y. Cho K.-S. Seo J.-H. Sohn K.H. Prokchorchik M. Improved genome sequence and gene annotation resource for the potato late blight pathogen Phytophthora infestans.Mol. Plant Microbe Interact. 2020; 33: 1025-1028https://doi.org/10.1094/MPMI-02-20-0023-ACrossref PubMed Scopus (4) Google Scholar; Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar) or cloned by PCR (EC1, NL01096, NL14538, 88069, PIC99183, and PIC99177) (Supplemental Figure 2). The sequence alignment shows Avramr3 is a highly conserved RXLR effector among P. infestans isolates, with only two polymorphic amino acids found among the 18 AVRamr3 homologs (Supplemental Figure 2).To define the domain responsible for recognition by Rpi-amr3, we fused 10 truncated Avramr3 fragments with HIS-FLAG tags and cloned into an expression vector with 35S promoter (T1\\u2013T10; Figure 2C; Supplemental Figure 3) and transiently co-expressed with Rpi-amr3 in N. benthamiana. Six AVRamr3 truncations (T1, T2, T5, T6, T7, and T9) cannot be recognized by Rpi-amr3 (Figure 2B). The protein levels of HIS-FLAG-tagged T6 and T7 are lower than others (Supplemental Figure 3A); therefore, we generated GFP-tagged constructs. The expression of T6-GFP is comparable with AVRamr3-GFP, but T7-GFP is not stable (Supplemental Figure 3B and 3C). We found four AVRamr3 truncations (T3, T4, T8, and T10) can be recognized by Rpi-amr3. T10 (111\\u2013240 aa), which carries the second and third WY motifs, is the minimal region to be recognized by Rpi-amr3, but not the adjacent T9 protein (130\\u2013258 aa) (Figure 2B). This suggests these 130 aa of AVRamr3 T10 are sufficient for recognition by Rpi-amr3 and initiation of HR.Rpi-amr3 is dependent on the helper NLRs NRC2, NRC3, and NRC4In Solanaceae, the functionality of many CC-NLR proteins requires helper NLR proteins of the NRC class (Wu et al., 2017Wu C.-H. Abd-El-Haliem A. Bozkurt T.O. Belhaj K. Terauchi R. Vossen J.H. Kamoun S. NLR network mediates immunity to diverse plant pathogens.Proc. Natl. Acad. Sci. USA. 2017; 114: 8113-8118Crossref PubMed Scopus (169) Google Scholar). To test whether Rpi-amr3 is NRC dependent, we co-expressed Rpi-amr3 and Avramr3 in NRC knockout N. benthamiana lines (nrc2/3_1.3.1, nrc4_185.9.1.3, nrc2/3/4_210.4.3) (Adachi et al., 2019Adachi H. Contreras M.P. Harant A. Wu C.-H. Derevnina L. Sakai T. Duggan C. Moratto E. Bozkurt T.O. Maqbool A. et al.An N-terminal motif in NLR immune receptors is functionally conserved across distantly related plant species.Elife. 2019; 8: 121Crossref Scopus (69) Google Scholar; Wu et al., 2020Wu C.-H. Adachi H. De la Concepcion J.C. Castells-Graells R. Nekrasov V. Kamoun S. Kamoun S. NRC4 Gene cluster Is not essential for bacterial flagellin-triggered immunity.Plant Physiol. 2020; 182: 455-459Crossref PubMed Scopus (4) Google Scholar; Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar), as with wild-type N. benthamiana. We found HR on the nrc2/3_1.3.1 and nrc4_185.9.1.3 knockout lines, but not the nrc2/3/4_210.4.3 knockout lines. Similarly, only nrc2/3/4_210.4.3 knockout lines show susceptibility to P. infestans after Rpi-amr3 transient expression (Supplemental Figure 4). These data suggest both Rpi-amr3-mediated effector recognition and resistance are supported by either NRC2, NRC3, or NRC4.Rpi-amr3 associates with AVRamr3 in plantaTo date, most Rpi proteins recognize their cognate effectors in an indirect manner, apart from the RB and IPI-O effectors (Chen et al., 2012Chen Y. Liu Z. Halterman D.A. Molecular determinants of resistance activation and suppression by Phytophthora infestans effector IPI-O.PLoS Pathog. 2012; 8: e1002595Crossref PubMed Scopus (71) Google Scholar; Zhao and Song, 2021Zhao J. Song J. NLR immune receptor RB is differentially targeted by two homologous but functionally distinct effector proteins.Plant Commun. 2021; 2: 100236Abstract Full Text Full Text PDF PubMed Scopus (2) Google Scholar). To test the interaction between Rpi-amr3 and AVRamr3, we generated and transiently co-expressed Rpi-amr3:HA and AVRamr3:HIS-FLAG epitope-tagged constructs in nrc2/3/4 knockout N. benthamiana leaves to avoid cell death. Protein was then extracted, and bi-directional co-immunoprecipitations (Co-IPs) were performed. These Co-IPs indicate that Rpi-amr3 associates with AVRamr3 bidirectionally (Figure 2D). We also tested their interaction using a split-luciferase assay. Rpi-amr3:C-luciferase (Cluc) and AVRamr3:N-luciferase (Nluc) constructs were generated and transiently expressed in the nrc2/3/4 knockout N. benthamiana. Luciferase signal was detected only when Rpi-amr3:Cluc and AVRamr3:Nluc were co-expressed (Figure 2E), but not in the negative controls. These data suggest Rpi-amr3 associates with AVRamr3 in planta, but do not exclude the possible involvement of additional proteins.Avramr3 orthologs occur in multiple Phytophthora speciesTo study the evolution of Avramr3 in Phytophthora species, we searched for Avramr3 homologs from published Phytophthora and Hyaloperonospora arabidopsidis genomes. Surprisingly, we found Avramr3 homologs in many Phytophthora genomes, including P. parasitica, P. cactorum, P. palmivora, P. pluvialis, P. megakarya, P. litchii, P. ramorum, P. lateralis, P. sojae, P. capsici, and P. cinnamomi and in H. arabidopsidis. Most of the Avramr3 homologs are located at a syntenic locus (Figure 3A ). Notably, the P. infestans Avramr3-containing contig was not fully assembled; it lacks sequences on the 5\\u2032 side of Avramr3 (Figure 3A).Figure 3AVRamr3 is a conserved effector among different Phytophthora species.Show full caption(A) The synteny map of Avramr3 loci from 12 different Phytophthora genomes. The Avramr3 loci were extracted from different genomes, annotated by the gene prediction tool in EumicrobeDB, and then analyzed and visualized by Clinker. Avramr3 homologs are shown by purple triangles and indicated by a black arrow; the flanking genes with homology are represented by the corresponding colors. The Phytophthora clades are adapted from the Phytophthora database (Rahman et al., 2014Rahman M.Z. Uematsu S. Takeuchi T. Shirai K. Ishiguro Y. Suga H. Kageyama K. Two new species, Phytophthora nagaii sp. nov. and P. fragariaefolia sp. nov., causing serious diseases on rose and strawberry plants, respectively, in Japan.J. Gen. Plant Pathol. 2014; 80: 348-365Crossref Scopus (13) Google Scholar).(B) Expression of AVRamr3 homologs with HIS-FLAG tag alone does not trigger cell death on Nicotiana benthamiana. Agrobacterium strain GV3101(pMP90) carrying different constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results.(C) Co-expression of HIS-FLAG-tagged AVRamr3 homologs with Rpi-amr3::GFP in N. benthamiana. The AVRamr3 homologs from P. infestans (Pi), P. parasitica (Pp), P. cactorum (Pc), P. palmivora (Ppal), P. megakarya (Pmeg), P. litchii (Plit), P. sojae (Ps), P. lateralis (Plat), and P. pluvialis (Pplu) induce cell death after co-expression with Rpi-amr3::GFP, but not AVRamr3 homologs from P. ramorum (Pr), P. capsici (Pcap), and H. arabidopsidis (Hpa). The AVRamr3 homolog from P. cinnamomi (Pcin) shows an intermediate cell death. Agrobacterium strain GV3101(pMP90) carrying different constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results. The protein expression of the AVRamr3 homologs with HIS-FLAG tag was shown in Supplemental Figure 6.View Large Image Figure ViewerDownload Hi-res image Download (PPT)To test whether those AVRamr3 homologs from different Phytophthora species are also recognized by Rpi-amr3, we synthesized and cloned them into an expression vector with the 35S promoter and performed transient expression assays in N. benthamiana. Expressing the effectors alone does not trigger HR in N. benthamiana (Figure 3B), but AVRamr3 homologs from P. parasitica, P. cactorum, P. palmivora, P. megakarya, P. litchii, P. sojae, P. lateralis, and P. pluvialis can induce HR when co-expressed with Rpi-amr3, respectively. The AVRamr3 homolog from P. cinnamomi triggers a weaker HR compared with other recognized AVRamr3 homologs, and the AVRamr3 homologs from P. ramorum, P. capsici, and H. arabidopsidis (Figure 3C) do not trigger Rpi-amr3-dependent HR. All these AVRamr3 homologs carry multiple WY motifs, but many polymorphic amino acids are present among these homologs. We\", \"doi_y\": \"https://doi.org/10.1016/j.molp.2022.07.012\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4315433413\", \"doi_x\": \"10.1021/acs.jproteome.2c00637\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Proteomics Standards Initiative at Twenty Years: Current Activities and Future Work\", \"abstract\": \"The Human Proteome Organization (HUPO) Proteomics Standards Initiative (PSI) has been successfully developing guidelines, data formats, and controlled vocabularies (CVs) for the proteomics community and other fields supported by mass spectrometry since its inception 20 years ago. Here we describe the general operation of the PSI, including its leadership, working groups, yearly workshops, and the document process by which proposals are thoroughly and publicly reviewed in order to be ratified as PSI standards. We briefly describe the current state of the many existing PSI standards, some of which remain the same as when originally developed, some of which have undergone subsequent revisions, and some of which have become obsolete. Then the set of proposals currently being developed are described, with an open call to the community for participation in the forging of the next generation of standards. Finally, we describe some synergies and collaborations with other organizations and look to the future in how the PSI will continue to promote the open sharing of data and thus accelerate the progress of the field of proteomics.\", \"doi_y\": \"https://doi.org/10.1021/acs.jproteome.2c00637\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W3200383688\", \"doi_x\": \"10.1042/bcj20210375\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Impact of Type II LRRK2 inhibitors on signaling and mitophagy\", \"abstract\": \"Much effort has been devoted to the development of selective inhibitors of the LRRK2 as a potential treatment for LRRK2 driven Parkinson's disease. In this study, we first compare the properties of Type I (GSK3357679A and MLi-2) and Type II (GZD-824, Rebastinib and Ponatinib) kinase inhibitors that bind to the closed or open conformations of the LRRK2 kinase domain, respectively. We show that Type I and Type II inhibitors suppress phosphorylation of Rab10 and Rab12, key physiological substrates of LRRK2 and also promote mitophagy, a process suppressed by LRRK2. Type II inhibitors also display higher potency towards wild-type LRRK2 compared with pathogenic mutants. Unexpectedly, we find that Type II inhibitors, in contrast with Type I compounds, fail to induce dephosphorylation of a set of well-studied LRRK2 biomarker phosphorylation sites at the N-terminal region of LRRK2, including Ser935. These findings emphasize that the biomarker phosphorylation sites on LRRK2 are likely reporting on the open vs closed conformation of LRRK2 kinase and that only inhibitors which stabilize the closed conformation induce dephosphorylation of these biomarker sites. Finally, we demonstrate that the LRRK2[A2016T] mutant which is resistant to MLi-2 Type 1 inhibitor, also induces resistance to GZD-824 and Rebastinib suggesting this mutation could be exploited to distinguish off target effects of Type II inhibitors. Our observations provide a framework of knowledge to aid with the development of more selective Type II LRRK2 inhibitors.\", \"doi_y\": \"https://doi.org/10.1042/bcj20210375\", \"publication_date\": \"2021-10-06\"}, {\"work_id\": \"W4311092866\", \"doi_x\": \"10.1016/j.ceca.2022.102675\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"PI(3,5)P2 and NAADP: Team players or lone warriors? \\u2013 New insights into TPC activation modes\", \"abstract\": \"NAADP (nicotinic acid adenine dinucleotide phosphate) is a second messenger, releasing Ca2+ from acidic calcium stores such as endosomes and lysosomes. PI(3,5)P2 (phosphatidylinositol 3,5-bisphosphate) is a phospho-inositide, residing on endolysosomal membranes and likewise releasing Ca2+ from endosomes and lysosomes. Both compounds have been shown to activate endolysosomal two-pore channels (TPCs) in mammalian cells. However, their effects on ion permeability as demonstrated specifically for TPC2 differ. While PI(3,5)P2 elicits predominantly Na+-selective currents, NAADP increases the Ca2+ permeability of the channel. What happens when both compounds are applied simultaneously was unclear until recently.\", \"doi_y\": \"https://doi.org/10.1016/j.ceca.2022.102675\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4293116279\", \"doi_x\": \"10.1093/ve/veac080\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The origins and molecular evolution of SARS-CoV-2 lineage B.1.1.7 in the UK\", \"abstract\": \"The first SARS-CoV-2 variant of concern (VOC) to be designated was lineage B.1.1.7, later labelled by the World Health Organization as Alpha. Originating in early autumn but discovered in December 2020, it spread rapidly and caused large waves of infections worldwide. The Alpha variant is notable for being defined by a long ancestral phylogenetic branch with an increased evolutionary rate, along which only two sequences have been sampled. Alpha genomes comprise a well-supported monophyletic clade within which the evolutionary rate is typical of SARS-CoV-2. The Alpha epidemic continued to grow despite the continued restrictions on social mixing across the UK and the imposition of new restrictions, in particular, the English national lockdown in November 2020. While these interventions succeeded in reducing the absolute number of cases, the impact of these non-pharmaceutical interventions was predominantly to drive the decline of the SARS-CoV-2 lineages that preceded Alpha. We investigate the only two sampled sequences that fall on the branch ancestral to Alpha. We find that one is likely to be a true intermediate sequence, providing information about the order of mutational events that led to Alpha. We explore alternate hypotheses that can explain how Alpha acquired a large number of mutations yet remained largely unobserved in a region of high genomic surveillance: an under-sampled geographical location, a non-human animal population, or a chronically infected individual. We conclude that the latter provides the best explanation of the observed behaviour and dynamics of the variant, although the individual need not be immunocompromised, as persistently infected immunocompetent hosts also display a higher within-host rate of evolution. Finally, we compare the ancestral branches and mutation profiles of other VOCs and find that Delta appears to be an outlier both in terms of the genomic locations of its defining mutations and a lack of the rapid evolutionary rate on its ancestral branch. As new variants, such as Omicron, continue to evolve (potentially through similar mechanisms), it remains important to investigate the origins of other variants to identify ways to potentially disrupt their evolution and emergence.\", \"doi_y\": \"https://doi.org/10.1093/ve/veac080\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4308834822\", \"doi_x\": \"10.1093/protein/gzac013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering enzyme activity using an expanded amino acid alphabet\", \"abstract\": \"Enzyme design and engineering strategies are typically constrained by the limited size of nature's genetic alphabet, comprised of only 20 canonical amino acids. In recent years, site-selective incorporation of non-canonical amino acids (ncAAs) via an expanded genetic code has emerged as a powerful means of inserting new functional components into proteins, with hundreds of structurally diverse ncAAs now available. Here, we highlight how the emergence of an expanded repertoire of amino acids has opened new avenues in enzyme design and engineering. ncAAs have been used to probe complex biological mechanisms, augment enzyme function and, most ambitiously, embed new catalytic mechanisms into protein active sites that would be challenging to access within the constraints of nature's genetic code. We predict that the studies reviewed in this article, along with further advances in genetic code expansion technology, will establish ncAA incorporation as an increasingly important tool for biocatalysis in the coming years.\", \"doi_y\": \"https://doi.org/10.1093/protein/gzac013\", \"publication_date\": \"2022-11-12\"}, {\"work_id\": \"W4317212025\", \"doi_x\": \"10.1038/s41579-022-00841-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"SARS-CoV-2 variant biology: immune escape, transmission and fitness\", \"abstract\": \"In late 2020, after circulating for almost a year in the human population, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) exhibited a major step change in its adaptation to humans. These highly mutated forms of SARS-CoV-2 had enhanced rates of transmission relative to previous variants and were termed 'variants of concern' (VOCs). Designated Alpha, Beta, Gamma, Delta and Omicron, the VOCs emerged independently from one another, and in turn each rapidly became dominant, regionally or globally, outcompeting previous variants. The success of each VOC relative to the previously dominant variant was enabled by altered intrinsic functional properties of the virus and, to various degrees, changes to virus antigenicity conferring the ability to evade a primed immune response. The increased virus fitness associated with VOCs is the result of a complex interplay of virus biology in the context of changing human immunity due to both vaccination and prior infection. In this Review, we summarize the literature on the relative transmissibility and antigenicity of SARS-CoV-2 variants, the role of mutations at the furin spike cleavage site and of non-spike proteins, the potential importance of recombination to virus success, and SARS-CoV-2 evolution in the context of T cells, innate immunity and population immunity. SARS-CoV-2 shows a complicated relationship among virus antigenicity, transmission and virulence, which has unpredictable implications for the future trajectory and disease burden of COVID-19.\", \"doi_y\": \"https://doi.org/10.1038/s41579-022-00841-7\", \"publication_date\": \"2023-01-18\"}, {\"work_id\": \"W4306641915\", \"doi_x\": \"10.1073/pnas.2210559119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"abstract\": \"Exocytosis plays an important role in plant-microbe interactions, in both pathogenesis and symbiosis. Exo70 proteins are integral components of the exocyst, an octameric complex that mediates tethering of vesicles to membranes in eukaryotes. Although plant Exo70s are known to be targeted by pathogen effectors, the underpinning molecular mechanisms and the impact of this interaction on infection are poorly understood. Here, we show the molecular basis of the association between the effector AVR-Pii of the blast fungus Maganaporthe oryzae and rice Exo70 alleles OsExo70F2 and OsExo70F3, which is sensed by the immune receptor pair Pii via an integrated RIN4/NOI domain. The crystal structure of AVR-Pii in complex with OsExo70F2 reveals that the effector binds to a conserved hydrophobic pocket in Exo70, defining an effector/target binding interface. Structure-guided and random mutagenesis validates the importance of AVR-Pii residues at the Exo70 binding interface to sustain protein association and disease resistance in rice when challenged with fungal strains expressing effector mutants. Furthermore, the structure of AVR-Pii defines a zinc-finger effector fold (ZiF) distinct from the MAX (Magnaporthe Avrs and ToxB-like) fold previously described for a majority of characterized M. oryzae effectors. Our data suggest that blast fungus ZiF effectors bind a conserved Exo70 interface to manipulate plant exocytosis and that these effectors are also baited by plant immune receptors, pointing to new opportunities for engineering disease resistance.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2210559119\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4298120764\", \"doi_x\": \"10.1002/bies.202200151\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Controlling contacts\\u2014Molecular mechanisms to regulate organelle membrane tethering\", \"abstract\": \"In recent years, membrane contact sites (MCS), which mediate interactions between virtually all subcellular organelles, have been extensively characterized and shown to be essential for intracellular communication. In this review essay, we focus on an emerging topic: the regulation of MCS. Focusing on the tether proteins themselves, we discuss some of the known mechanisms which can control organelle tethering events and identify apparent common regulatory hubs, such as the VAP interface at the endoplasmic reticulum (ER). We also highlight several currently hypothetical concepts, including the idea of tether oligomerization and redox regulation playing a role in MCS formation. We identify gaps in our current understanding, such as the identity of the majority of kinases/phosphatases involved in tether modification and conclude that a holistic approach-incorporating the formation of multiple MCS, regulated by interconnected regulatory modulators-may be required to fully appreciate the true complexity of these fascinating intracellular communication systems.\", \"doi_y\": \"https://doi.org/10.1002/bies.202200151\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4307812279\", \"doi_x\": \"10.1016/j.trac.2022.116808\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Advances in methods to analyse cardiolipin and their clinical applications\", \"abstract\": \"Cardiolipin (CL) is a mitochondria-exclusive phospholipid, primarily localised within the inner mitochondrial membrane, that plays an essential role in mitochondrial architecture and function. Aberrant CL content, structure, and localisation have all been linked to impaired mitochondrial activity and are observed in the pathophysiology of cancer and neurological, cardiovascular, and metabolic disorders. The detection, quantification, and localisation of CL species is a valuable tool to investigate mitochondrial dysfunction and the pathophysiological mechanisms underpinning several human disorders. CL is measured using liquid chromatography, usually combined with mass spectrometry, mass spectrometry imaging, shotgun lipidomics, ion mobility spectrometry, fluorometry, and radiolabelling. This review summarises available methods to analyse CL, with a particular focus on modern mass spectrometry, and evaluates their advantages and limitations. We provide guidance aimed at selecting the most appropriate technique, or combination of techniques, when analysing CL in different model systems, and highlight the clinical contexts in which measuring CL is relevant.\", \"doi_y\": \"https://doi.org/10.1016/j.trac.2022.116808\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4379467311\", \"doi_x\": \"10.1073/pnas.2302580120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"abstract\": \"AmiA and AmiB are peptidoglycan-hydrolyzing enzymes from Escherichia coli that are required to break the peptidoglycan layer during bacterial cell division and maintain integrity of the cell envelope. In vivo, the activity of AmiA and AmiB is tightly controlled through their interactions with the membrane-bound FtsEX-EnvC complex. Activation of AmiA and AmiB requires access to a groove in the amidase-activating LytM domain of EnvC which is gated by ATP-driven conformational changes in FtsEX-EnvC complex. Here, we present a high-resolution structure of the isolated AmiA protein, confirming that it is autoinhibited in the same manner as AmiB and AmiC, and a complex of the AmiB enzymatic domain bound to the activating EnvC LytM domain. In isolation, the active site of AmiA is blocked by an autoinhibitory helix that binds directly to the catalytic zinc and fills the volume expected to accommodate peptidoglycan binding. In the complex, binding of the EnvC LytM domain induces a conformational change that displaces the amidase autoinhibitory helix and reorganizes the active site for activity. Our structures, together with complementary mutagenesis work, defines the conformational changes required to activate AmiA and/or AmiB through their interaction with their cognate activator EnvC.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2302580120\", \"publication_date\": \"2023-06-05\"}, {\"work_id\": \"W4220732247\", \"doi_x\": \"10.1101/2022.03.07.22271833\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"GWAS and meta-analysis identifies multiple new genetic mechanisms underlying severe Covid-19\", \"abstract\": \"Abstract Pulmonary inflammation drives critical illness in Covid-19, 1;2 creating a clinically homogeneous extreme phenotype, which we have previously shown to be highly efficient for discovery of genetic associations. 3;4 Despite the advanced stage of illness, we have found that immunomodulatory therapies have strong beneficial effects in this group. 1;5 Further genetic discoveries may identify additional therapeutic targets to modulate severe disease. 6 In this new data release from the GenOMICC (Genetics Of Mortality in Critical Care) study we include new microarray genotyping data from additional critically-ill cases in the UK and Brazil, together with cohorts of severe Covid-19 from the ISARIC4C 7 and SCOURGE 8 studies, and meta-analysis with previously-reported data. We find an additional 14 new genetic associations. Many are in potentially druggable targets, in inflammatory signalling (JAK1, PDE4A), monocyte-macrophage differentiation (CSF2), immunometabolism (SLC2A5, AK5), and host factors required for viral entry and replication (TMPRSS2, RAB2A). As with our previous work, these results provide tractable therapeutic targets for modulation of harmful host-mediated inflammation in Covid-19.\", \"doi_y\": \"https://doi.org/10.1101/2022.03.07.22271833\", \"publication_date\": \"2022-03-07\"}, {\"work_id\": \"W4206153788\", \"doi_x\": \"10.1016/j.cell.2021.12.046\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"abstract\": \"On 24th November 2021, the sequence of a new SARS-CoV-2 viral isolate Omicron-B.1.1.529 was announced, containing far more mutations in Spike (S) than previously reported variants. Neutralization titers of Omicron by sera from vaccinees and convalescent subjects infected with early pandemic Alpha, Beta, Gamma, or Delta are substantially reduced, or the sera failed to neutralize. Titers against Omicron are boosted by third vaccine doses and are high in both vaccinated individuals and those infected by Delta. Mutations in Omicron knock out or substantially reduce neutralization by most of the large panel of potent monoclonal antibodies and antibodies under commercial development. Omicron S has structural changes from earlier viruses and uses mutations that confer tight binding to ACE2 to unleash evolution driven by immune escape. This leads to a large number of mutations in the ACE2 binding site and rebalances receptor affinity to that of earlier pandemic viruses.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2021.12.046\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4280488864\", \"doi_x\": \"10.1101/2022.04.25.489359\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"abstract\": \"Abstract Plant pathogens compromise crop yields. Plants have evolved robust innate immunity that depends in part on intracellular Nucleotide-binding, Leucine Rich-Repeat (NLR) immune receptors that activate defense responses upon detection of pathogen-derived effectors. Most \\u201csensor\\u201d NLRs that detect effectors require the activity of \\u201chelper\\u201d NLRs, but how helper NLRs support sensor NLR function is poorly understood. Many Solanaceae NLRs require the NRC (NLR-Required for Cell death) class of helper NLRs. We show here that Rpi-amr3, a sensor NLR from Solanum americanum , detects AVRamr3 from the potato late blight pathogen, Phytophthora infestans , and activates oligomerization of the helper NLR NRC2 into a high-molecular weight resistosome. The NRC2 resistosome also forms upon recognition of P. infestans effector AVRamr1 by another sensor NLR, Rpi-amr1. The ATP-binding motif of Rpi-amr3 is required for NRC2 resistosome formation, but not for interaction with the cognate effector. The NRC2 resistosome can be activated by AVRamr3 homologs from other Phytophthora species. Mechanistic understanding of NRC resistosome formation will underpin engineering crops with durable disease resistance.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.25.489359\", \"publication_date\": \"2022-04-25\"}, {\"work_id\": \"W4224311483\", \"doi_x\": \"10.26508/lsa.202101255\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The value of genotype-specific reference for transcriptome analyses in barley\", \"abstract\": \"It is increasingly apparent that although different genotypes within a species share \\\"core\\\" genes, they also contain variable numbers of \\\"specific\\\" genes and different structures of \\\"core\\\" genes that are only present in a subset of individuals. Using a common reference genome may thus lead to a loss of genotype-specific information in the assembled Reference Transcript Dataset (RTD) and the generation of erroneous, incomplete or misleading transcriptomics analysis results. In this study, we assembled genotype-specific RTD (sRTD) and common reference-based RTD (cRTD) from RNA-seq data of cultivated Barke and Morex barley, respectively. Our quantitative evaluation showed that the sRTD has a significantly higher diversity of transcripts and alternative splicing events, whereas the cRTD missed 40% of transcripts present in the sRTD and it only has \\u223c70% accurate transcript assemblies. We found that the sRTD is more accurate for transcript quantification as well as differential expression analysis. However, gene-level quantification is less affected, which may be a reasonable compromise when a high-quality genotype-specific reference is not available.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202101255\", \"publication_date\": \"2022-04-22\"}, {\"work_id\": \"W4319293755\", \"doi_x\": \"10.1128/mbio.02667-22\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Insights into the Roles of Lipoteichoic Acids and MprF in Bacillus subtilis\", \"abstract\": \"Gram-positive bacterial cells are protected from the environment by a cell envelope that is comprised of a thick layer of peptidoglycan that maintains cell shape and teichoic acid polymers whose biological function remains unclear. In Bacillus subtilis, the loss of all class A penicillin-binding proteins (aPBPs), which function in peptidoglycan synthesis, is conditionally lethal. Here, we show that this lethality is associated with an alteration of lipoteichoic acids (LTAs) and the accumulation of the major autolysin LytE in the cell wall. Our analysis provides further evidence that the length and abundance of LTAs act to regulate the cellular level and activity of autolytic enzymes, specifically LytE. Importantly, we identify a novel function for the aminoacyl-phosphatidylglycerol synthase MprF in the modulation of LTA biosynthesis in both B. subtilis and Staphylococcus aureus. This finding has implications for our understanding of antimicrobial resistance (particularly to daptomycin) in clinically relevant bacteria and the involvement of MprF in the virulence of pathogens such as methicillin-resistant S. aureus (MRSA). IMPORTANCE In Gram-positive bacteria such as Bacillus subtilis and Staphylococcus aureus, the cell envelope is a structure that protects the cells from the environment but is also dynamic in that it must be modified in a controlled way to allow cell growth. In this study, we show that lipoteichoic acids (LTAs), which are anionic polymers attached to the membrane, have a direct role in modulating the cellular abundance of cell wall-degrading enzymes. We also find that the apparent length of the LTA is modulated by the activity of the enzyme MprF, previously implicated in modifications of the cell membrane leading to resistance to antimicrobial peptides. These findings are important contributions to our understanding of how bacteria balance cell wall synthesis and degradation to permit controlled growth and division. These results also have implications for the interpretation of antibiotic resistance, particularly for the clinical treatment of MRSA infections.\", \"doi_y\": \"https://doi.org/10.1128/mbio.02667-22\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4284886202\", \"doi_x\": \"10.1038/s41564-022-01143-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 Omicron is an immune escape variant with an altered cell entry pathway\", \"abstract\": \"Vaccines based on the spike protein of SARS-CoV-2 are a cornerstone of the public health response to COVID-19. The emergence of hypermutated, increasingly transmissible variants of concern (VOCs) threaten this strategy. Omicron (B.1.1.529), the fifth VOC to be described, harbours multiple amino acid mutations in spike, half of which lie within the receptor-binding domain. Here we demonstrate substantial evasion of neutralization by Omicron BA.1 and BA.2 variants in vitro using sera from individuals vaccinated with ChAdOx1, BNT162b2 and mRNA-1273. These data were mirrored by a substantial reduction in real-world vaccine effectiveness that was partially restored by booster vaccination. The Omicron variants BA.1 and BA.2 did not induce cell syncytia in vitro and favoured a TMPRSS2-independent endosomal entry pathway, these phenotypes mapping to distinct regions of the spike protein. Impaired cell fusion was determined by the receptor-binding domain, while endosomal entry mapped to the S2 domain. Such marked changes in antigenicity and replicative biology may underlie the rapid global spread and altered pathogenicity of the Omicron variant.\", \"doi_y\": \"https://doi.org/10.1038/s41564-022-01143-7\", \"publication_date\": \"2022-07-07\"}, {\"work_id\": \"W4376959094\", \"doi_x\": \"10.1038/s41467-023-38484-8\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"abstract\": \"Siderophores belonging to the ferrichrome family are essential for the viability of fungal species and play a key role for virulence of numerous pathogenic fungi. Despite their biological significance, our understanding of how these iron-chelating cyclic hexapeptides are assembled by non-ribosomal peptide synthetase (NRPS) enzymes remains poorly understood, primarily due to the nonlinearity exhibited by the domain architecture. Herein, we report the biochemical characterization of the SidC NRPS, responsible for construction of the intracellular siderophore ferricrocin. In vitro reconstitution of purified SidC reveals its ability to produce ferricrocin and its structural variant, ferrichrome. Application of intact protein mass spectrometry uncovers several non-canonical events during peptidyl siderophore biosynthesis, including inter-modular loading of amino acid substrates and an adenylation domain capable of poly-amide bond formation. This work expands the scope of NRPS programming, allows biosynthetic assignment of ferrichrome NRPSs, and sets the stage for reprogramming towards novel hydroxamate scaffolds.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-38484-8\", \"publication_date\": \"2023-05-17\"}, {\"work_id\": \"W4386001747\", \"doi_x\": \"10.1038/s42003-023-05239-6\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"abstract\": \"Abstract Insulin-like Growth Factor-2 (IGF2) is important for the regulation of human embryonic growth and development, and for adults\\u2019 physiology. Incorrect processing of the IGF2 precursor, pro-IGF2(156), leads to the formation of two IGF2 proforms, big-IGF2(87) and big-IGF2(104). Unprocessed and mainly non-glycosylated IGF2 proforms are found at abnormally high levels in certain diseases, but their mode of action is still unclear. Here, we found that pro-IGF2(156) has the lowest ability to form its inactivating complexes with IGF-Binding Proteins and has higher proliferative properties in cells than IGF2 and other IGF prohormones. We also showed that big-IGF2(104) has a seven-fold higher binding affinity for the IGF2 receptor than IGF2, and that pro-IGF2(87) binds and activates specific receptors and stimulates cell growth similarly to the mature IGF2. The properties of these pro-IGF2 forms, especially of pro-IGF2(156) and big-IGF2(104), indicate them as hormones that may be associated with human diseases related to the accumulation of IGF-2 proforms in the circulation.\", \"doi_y\": \"https://doi.org/10.1038/s42003-023-05239-6\", \"publication_date\": \"2023-08-19\"}, {\"work_id\": \"W4224287290\", \"doi_x\": \"10.1016/s1473-3099(22)00141-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Severity of omicron variant of concern and effectiveness of vaccine boosters against symptomatic disease in Scotland (EAVE II): a national cohort study with nested test-negative design\", \"abstract\": \"Since its emergence in November, 2021, in southern Africa, the SARS-CoV-2 omicron variant of concern (VOC) has rapidly spread across the world. We aimed to investigate the severity of omicron and the extent to which booster vaccines are effective in preventing symptomatic infection.In this study, using the Scotland-wide Early Pandemic Evaluation and Enhanced Surveillance of COVID-19 (EAVE II) platform, we did a cohort analysis with a nested test-negative design incident case-control study covering the period Nov 1-Dec 19, 2021, to provide initial estimates of omicron severity and the effectiveness of vaccine boosters against symptomatic disease relative to 25 weeks or more after the second vaccine dose. Primary care data derived from 940 general practices across Scotland were linked to laboratory data and hospital admission data. We compared outcomes between infection with the delta VOC (defined as S-gene positive) and the omicron VOC (defined as S-gene negative). We assessed effectiveness against symptomatic SARS-CoV-2 infection, with infection confirmed through a positive RT-PCR.By Dec 19, 2021, there were 23 840 S-gene-negative cases in Scotland, which were predominantly among those aged 20-39 years (11 732 [49\\u00b72%]). The proportion of S-gene-negative cases that were possible reinfections was more than ten times that of S-gene-positive cases (7\\u00b76% vs 0\\u00b77%; p<0\\u00b70001). There were 15 hospital admissions in S-gene-negative individuals, giving an adjusted observed-to-expected admissions ratio of 0\\u00b732 (95% CI 0\\u00b719-0\\u00b752). The booster vaccine dose was associated with a 57% (54-60) reduction in the risk of symptomatic S-gene-negative infection relative to individuals who tested positive 25 weeks or more after the second vaccine dose.These early national data suggest that omicron is associated with a two-thirds reduction in the risk of COVID-19 hospitalisation compared with delta. Although offering the greatest protection against delta, the booster dose of vaccination offers substantial additional protection against the risk of symptomatic COVID-19 for omicron compared with 25 weeks or more after the second vaccine dose.Health Data Research UK, National Core Studies, Public Health Scotland, Scottish Government, UK Research and Innovation, and University of Edinburgh.\", \"doi_y\": \"https://doi.org/10.1016/s1473-3099(22)00141-4\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4321765801\", \"doi_x\": \"10.1021/acs.chemrev.2c00600\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"abstract\": \"The investigation of macromolecular biomolecules with ion mobility mass spectrometry (IM-MS) techniques has provided substantial insights into the field of structural biology over the past two decades. An IM-MS workflow applied to a given target analyte provides mass, charge, and conformation, and all three of these can be used to discern structural information. While mass and charge are determined in mass spectrometry (MS), it is the addition of ion mobility that enables the separation of isomeric and isobaric ions and the direct elucidation of conformation, which has reaped huge benefits for structural biology. In this review, where we focus on the analysis of proteins and their complexes, we outline the typical features of an IM-MS experiment from the preparation of samples, the creation of ions, and their separation in different mobility and mass spectrometers. We describe the interpretation of ion mobility data in terms of protein conformation and how the data can be compared with data from other sources with the use of computational tools. The benefit of coupling mobility analysis to activation via collisions with gas or surfaces or photons photoactivation is detailed with reference to recent examples. And finally, we focus on insights afforded by IM-MS experiments when applied to the study of conformationally dynamic and intrinsically disordered proteins.\", \"doi_y\": \"https://doi.org/10.1021/acs.chemrev.2c00600\", \"publication_date\": \"2023-02-24\"}, {\"work_id\": \"W4224078886\", \"doi_x\": \"10.1016/j.molcel.2022.03.030\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"H3K4 methylation by SETD1A/BOD1L facilitates RIF1-dependent NHEJ\", \"abstract\": \"The 53BP1-RIF1-shieldin pathway maintains genome stability by suppressing nucleolytic degradation of DNA ends at double-strand breaks (DSBs). Although RIF1 interacts with damaged chromatin via phospho-53BP1 and facilitates recruitment of the shieldin complex to DSBs, it is unclear whether other regulatory cues contribute to this response. Here, we implicate methylation of histone H3 at lysine 4 by SETD1A-BOD1L in the recruitment of RIF1 to DSBs. Compromising SETD1A or BOD1L expression or deregulating H3K4 methylation allows uncontrolled resection of DNA ends, impairs end-joining of dysfunctional telomeres, and abrogates class switch recombination. Moreover, defects in RIF1 localization to DSBs are evident in patient cells bearing loss-of-function mutations in SETD1A. Loss of SETD1A-dependent RIF1 recruitment in BRCA1-deficient cells restores homologous recombination and leads to resistance to poly(ADP-ribose)polymerase inhibition, reinforcing the clinical relevance of these observations. Mechanistically, RIF1 binds directly to methylated H3K4, facilitating its recruitment to, or stabilization at, DSBs.\", \"doi_y\": \"https://doi.org/10.1016/j.molcel.2022.03.030\", \"publication_date\": \"2022-05-01\"}, {\"work_id\": \"W4386743501\", \"doi_x\": \"10.1038/s41557-023-01321-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Assembling membraneless organelles from de novo designed proteins\", \"abstract\": \"Abstract Recent advances in de novo protein design have delivered a diversity of discrete de novo protein structures and complexes. A new challenge for the field is to use these designs directly in cells to intervene in biological processes and augment natural systems. The bottom-up design of self-assembled objects such as microcompartments and membraneless organelles is one such challenge. Here we describe the design of genetically encoded polypeptides that form membraneless organelles in Escherichia coli . To do this, we combine de novo \\u03b1-helical sequences, intrinsically disordered linkers and client proteins in single-polypeptide constructs. We tailor the properties of the helical regions to shift protein assembly from arrested assemblies to dynamic condensates. The designs are characterized in cells and in vitro using biophysical methods and soft-matter physics. Finally, we use the designed polypeptide to co-compartmentalize a functional enzyme pair in E. coli , improving product formation close to the theoretical limit.\", \"doi_y\": \"https://doi.org/10.1038/s41557-023-01321-y\", \"publication_date\": \"2023-09-14\"}, {\"work_id\": \"W4317434672\", \"doi_x\": \"10.1038/s44222-022-00007-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Lateral flow test engineering and lessons learned from COVID-19\", \"abstract\": \"The acceptability and feasibility of large-scale testing with lateral flow tests (LFTs) for clinical and public health purposes has been demonstrated during the COVID-19 pandemic. LFTs can detect analytes in a variety of samples, providing a rapid read-out, which allows self-testing and decentralized diagnosis. In this Review, we examine the changing LFT landscape with a focus on lessons learned from COVID-19. We discuss the implications of LFTs for decentralized testing of infectious diseases, including diseases of epidemic potential, the \\u2018silent pandemic\\u2019 of antimicrobial resistance, and other acute and chronic infections. Bioengineering approaches will play a key part in increasing the sensitivity and specificity of LFTs, improving sample preparation, incorporating nucleic acid amplification and detection, and enabling multiplexing, digital connection and green manufacturing, with the aim of creating the next generation of high-accuracy, easy-to-use, affordable and digitally connected LFTs. We conclude with recommendations, including the building of a global network of LFT research and development hubs to facilitate and strengthen future diagnostic resilience. The feasibility of large-scale testing with lateral flow tests has been demonstrated in the COVID-19 pandemic. This Review examines lessons learned from the COVID-19 pandemic to inform the design and bioengineering of next-generation lateral flow tests to strengthen future diagnostic resilience.\", \"doi_y\": \"https://doi.org/10.1038/s44222-022-00007-3\", \"publication_date\": \"2023-01-19\"}, {\"work_id\": \"W4386488775\", \"doi_x\": \"10.1039/d3cc02909c\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Considerations for defining +80 Da mass shifts in mass spectrometry-based proteomics: phosphorylation and beyond\", \"abstract\": \"Post-translational modifications (PTMs) are ubiquitous and key to regulating protein function. Understanding the dynamics of individual PTMs and their biological roles requires robust characterisation. Mass spectrometry (MS) is the method of choice for the identification and quantification of protein modifications. This article focusses on the MS-based analysis of those covalent modifications that induce a mass shift of +80 Da, notably phosphorylation and sulfation, given the challenges associated with their discrimination and pinpointing the sites of modification on a polypeptide chain. Phosphorylation in particular is highly abundant, dynamic and can occur on numerous residues to invoke specific functions, hence robust characterisation is crucial to understanding biological relevance. Showcasing our work in the context of other developments in the field, we highlight approaches for enrichment and site localisation of phosphorylated (canonical and non-canonical) and sulfated peptides, as well as modification analysis in the context of intact proteins (top down proteomics) to explore combinatorial roles. Finally, we discuss the application of native ion-mobility MS to explore the effect of these PTMs on protein structure and ligand binding.\", \"doi_y\": \"https://doi.org/10.1039/d3cc02909c\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4315853020\", \"doi_x\": \"10.1088/1361-6420/acb2ba\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"abstract\": \"We consider the problem of recovering the three-dimensional atomic structure of a flexible macromolecule from a heterogeneous cryo-EM dataset. The dataset contains noisy tomographic projections of the electrostatic potential of the macromolecule, taken from different viewing directions, and in the heterogeneous case, each image corresponds to a different conformation of the macromolecule. Under the assumption that the macromolecule can be modelled as a chain, or discrete curve (as it is for instance the case for a protein backbone with a single chain of amino-acids), we introduce a method to estimate the deformation of the atomic model with respect to a given conformation, which is assumed to be known a priori. Our method consists on estimating the torsion and bond angles of the atomic model in each conformation as a linear combination of the eigenfunctions of the Laplace operator in the manifold of conformations. These eigenfunctions can be approximated by means of a well-known technique in manifold learning, based on the construction of a graph Laplacian using the cryo-EM dataset. Finally, we test our approach with synthetic datasets, for which we recover the atomic model of two-dimensional and three-dimensional flexible structures from noisy tomographic projections.\", \"doi_y\": \"https://doi.org/10.1088/1361-6420/acb2ba\", \"publication_date\": \"2023-01-30\"}, {\"work_id\": \"W4311838198\", \"doi_x\": \"10.1101/2022.12.03.22282974\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Non-generalizability of biomarkers for mortality in SARS-CoV-2: a meta-analyses series\", \"abstract\": \"ABSTRACT Rationale Sophisticated prognostic scores have been proposed for SARS-CoV-2 but do not always perform consistently. We conducted these meta-analyses to uncover why and to investigate the impact of vaccination and variants. Methods We searched the PubMed database for the keywords \\u2018SARS-CoV-2\\u2019 with \\u2018biomarker\\u2019 and \\u2018mortality\\u2019 for the baseline tranche (01/12/2020-30/06/2021) and either \\u2018SARS-CoV-2\\u2019 or \\u2018Covid19\\u2019 with \\u2018biomarker\\u2019 and either \\u2018vaccination\\u2019 or \\u2018variant\\u2019 from 01/12/2020 to 31/10/2023. To aggregate the data, the meta library in R was used, and a random effects model fitted to obtain pooled AUCs and 95% confidence intervals for the European/North American, Asian, and overall datasets. Results Biomarker effectiveness varies significantly in different continents. Admission CRP levels were a good prognostic marker for mortality due to wild-type virus in Asian countries, with a pooled area under curve (AUC) of 0.83 (95%CI 0.80-0.85), but only an average predictor of mortality in Europe/North America, with a pooled AUC of 0.67 (95%CI 0.63-0.71, P&lt; 0.0001). We observed the same pattern for D-dimer and IL-6. This variability explains why the proposed prognostic scores did not perform evenly. Notably, urea and troponin had pooled AUCs \\u22650.78 regardless of location, implying that end-organ damage at presentation is a key prognostic factor. The inflammatory biomarkers (CRP, D-dimer and IL-6) have generally declined in effectiveness in the vaccinated and variant cohorts. We note a significant lag from the pandemic advent to data availability and this has no doubt impacted on patient care. Conclusions Biomarker efficacies vary considerably by region. It is imperative that the infrastructure for collecting clinical data should be put in place ahead of a future pandemic.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.03.22282974\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4308448193\", \"doi_x\": \"10.1126/science.abq6656\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"On-demand cell-autonomous gene therapy for brain circuit disorders\", \"abstract\": \"Several neurodevelopmental and neuropsychiatric disorders are characterized by intermittent episodes of pathological activity. Although genetic therapies offer the ability to modulate neuronal excitability, a limiting factor is that they do not discriminate between neurons involved in circuit pathologies and \\\"healthy\\\" surrounding or intermingled neurons. We describe a gene therapy strategy that down-regulates the excitability of overactive neurons in closed loop, which we tested in models of epilepsy. We used an immediate early gene promoter to drive the expression of Kv1.1 potassium channels specifically in hyperactive neurons, and only for as long as they exhibit abnormal activity. Neuronal excitability was reduced by seizure-related activity, leading to a persistent antiepileptic effect without interfering with normal behaviors. Activity-dependent gene therapy is a promising on-demand cell-autonomous treatment for brain circuit disorders.\", \"doi_y\": \"https://doi.org/10.1126/science.abq6656\", \"publication_date\": \"2022-11-04\"}, {\"work_id\": \"W4310957810\", \"doi_x\": \"10.1093/nar/gkac1080\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"abstract\": \"Abstract The MGnify platform (https://www.ebi.ac.uk/metagenomics) facilitates the assembly, analysis and archiving of microbiome-derived nucleic acid sequences. The platform provides access to taxonomic assignments and functional annotations for nearly half a million analyses covering metabarcoding, metatranscriptomic, and metagenomic datasets, which are derived from a wide range of different environments. Over the past 3 years, MGnify has not only grown in terms of the number of datasets contained but also increased the breadth of analyses provided, such as the analysis of long-read sequences. The MGnify protein database now exceeds 2.4 billion non-redundant sequences predicted from metagenomic assemblies. This collection is now organised into a relational database making it possible to understand the genomic context of the protein through navigation back to the source assembly and sample metadata, marking a major improvement. To extend beyond the functional annotations already provided in MGnify, we have applied deep learning-based annotation methods. The technology underlying MGnify's Application Programming Interface (API) and website has been upgraded, and we have enabled the ability to perform downstream analysis of the MGnify data through the introduction of a coupled Jupyter Lab environment.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1080\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4319071338\", \"doi_x\": \"10.1136/jnnp-2022-330504\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>UNC13A</i>in amyotrophic lateral sclerosis: from genetic association to therapeutic target\", \"abstract\": \"Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease with limited treatment options and an incompletely understood pathophysiology. Although genomewide association studies (GWAS) have advanced our understanding of the disease, the precise manner in which risk polymorphisms contribute to disease pathogenesis remains unclear. Of relevance, GWAS have shown that a polymorphism (rs12608932) in the UNC13A gene is associated with risk for both ALS and frontotemporal dementia (FTD). Homozygosity for the C-allele at rs12608932 modifies the ALS phenotype, as these patients are more likely to have bulbar-onset disease, cognitive impairment and FTD at baseline as well as shorter survival. UNC13A is expressed in neuronal tissue and is involved in maintaining synaptic active zones, by enabling the priming and docking of synaptic vesicles. In the absence of functional TDP-43, risk variants in UNC13A lead to the inclusion of a cryptic exon in UNC13A messenger RNA, subsequently leading to nonsense mediated decay, with loss of functional protein. Depletion of UNC13A leads to impaired neurotransmission. Recent discoveries have identified UNC13A as a potential target for therapy development in ALS, with a confirmatory trial with lithium carbonate in UNC13A cases now underway and future approaches with antisense oligonucleotides currently under consideration. Considering UNC13A is a potent phenotypic modifier, it may also impact clinical trial outcomes. This present review describes the path from the initial discovery of UNC13A as a risk gene in ALS to the current therapeutic options being explored and how knowledge of its distinct phenotype needs to be taken into account in future trials.\", \"doi_y\": \"https://doi.org/10.1136/jnnp-2022-330504\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4229452091\", \"doi_x\": \"10.1038/s41587-022-01316-z\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"An open invitation to the Understudied Proteins Initiative\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41587-022-01316-z\", \"publication_date\": \"2022-05-09\"}, {\"work_id\": \"W4284666963\", \"doi_x\": \"10.1038/s41589-022-01071-x\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"abstract\": \"The abundance of recorded protein sequence data stands in contrast to the small number of experimentally verified functional annotation. Here we screened a million-membered metagenomic library at ultrahigh throughput in microfluidic droplets for \\u03b2-glucuronidase activity. We identified SN243, a genuine \\u03b2-glucuronidase with little homology to previously studied enzymes of this type, as a glycoside hydrolase 3 family member. This glycoside hydrolase family contains only one recently added \\u03b2-glucuronidase, showing that a functional metagenomic approach can shed light on assignments that are currently 'unpredictable' by bioinformatics. Kinetic analyses of SN243 characterized it as a promiscuous catalyst and structural analysis suggests regions of divergence from homologous glycoside hydrolase 3 members creating a wide-open active site. With a screening throughput of >107 library members per day, picolitre-volume microfluidic droplets enable functional assignments that complement current enzyme database dictionaries and provide bridgeheads for the annotation of unexplored sequence space.\", \"doi_y\": \"https://doi.org/10.1038/s41589-022-01071-x\", \"publication_date\": \"2022-07-07\"}, {\"work_id\": \"W4385349885\", \"doi_x\": \"10.1021/acscatal.3c01906\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Comment on: \\u201cComputer Simulations Reveal an Entirely Entropic Activation Barrier for the Chemical Step in a Designer Enzyme\\u201d\", \"abstract\": \"Activation heat capacity has been proposed as an important factor in enzyme evolution and thermoadaptation. We previously demonstrated that the emergence of curved activity\\u2013temperature profiles during the evolution of a designer enzyme was due to the selective rigidification of its transition state ensemble that induced an activation heat capacity. \\u00c5qvist challenged our findings with molecular dynamics simulations suggesting that a change in the rate-limiting step underlies the experimental observations. As we describe here, \\u00c5qvist\\u2019s model is not consistent with the experimental trends observed for the chemical step of the catalyzed reaction (kcat). We suggest that this discrepancy arises because the simulations performed by \\u00c5qvist were limited by restraints and short simulation times, which do not allow sampling of the motions responsible for the observed activation heat capacity.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c01906\", \"publication_date\": \"2023-07-28\"}, {\"work_id\": \"W4382046174\", \"doi_x\": \"10.1016/j.tim.2023.06.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural diversity and modularity of photosynthetic RC\\u2212LH1 complexes\", \"abstract\": \"Bacterial photosynthesis is essential for sustaining life on Earth as it aids in carbon assimilation, atmospheric composition, and ecosystem maintenance. Many bacteria utilize anoxygenic photosynthesis to convert sunlight into chemical energy while producing organic matter. The core machinery of anoxygenic photosynthesis performed by purple photosynthetic bacteria and Chloroflexales is the reaction center\\u2212light-harvesting 1 (RC\\u2212LH1) pigment\\u2013protein supercomplex. In this review, we discuss recent structural studies of RC\\u2212LH1 core complexes based on the advancement in structural biology techniques. These studies have provided fundamental insights into the assembly mechanisms, structural variations, and modularity of RC\\u2212LH1 complexes across different bacterial species, highlighting their functional adaptability. Understanding the natural architectures of RC\\u2212LH1 complexes will facilitate the design and engineering of artificial photosynthetic systems, which can enhance photosynthetic efficiency and potentially find applications in sustainable energy production and carbon capture.\", \"doi_y\": \"https://doi.org/10.1016/j.tim.2023.06.002\", \"publication_date\": \"2024-01-01\"}, {\"work_id\": \"W4285082691\", \"doi_x\": \"10.1016/j.sbi.2022.102420\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Frontiers in metalloprotein crystallography and cryogenic electron microscopy\", \"abstract\": \"Metalloproteins comprise at least a third of all proteins that utilize redox properties of transition metals on their own or as parts of cofactors. The development of third generation storage ring sources and X-ray free-electron lasers with femtosecond pulses in the first decade of the 21st century has transformed metalloprotein crystallography. In the past decade, cryogenic-electron microscopy single-particle analysis, which does not require crystallization of biological samples has been extensively utilized, particularly for membrane-bound metalloprotein systems. Here, we explore recent frontiers in metalloprotein crystallography and cryogenic electron microscopy, organized for convenience under three metalloprotein-centered biological cycles, focusing on contributions from each technique, their synergy and the ability to preserve metals' redox states when subjected to a particular probe.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2022.102420\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4319593844\", \"doi_x\": \"10.1038/s42003-023-04488-9\", \"af_direct_citation\": 1, \"af_indirect_citations\": 13, \"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"abstract\": \"Deep-learning (DL) methods like DeepMind's AlphaFold2 (AF2) have led to substantial improvements in protein structure prediction. We analyse confident AF2 models from 21 model organisms using a new classification protocol (CATH-Assign) which exploits novel DL methods for structural comparison and classification. Of ~370,000 confident models, 92% can be assigned to 3253 superfamilies in our CATH domain superfamily classification. The remaining cluster into 2367 putative novel superfamilies. Detailed manual analysis on 618 of these, having at least one human relative, reveal extremely remote homologies and further unusual features. Only 25 novel superfamilies could be confirmed. Although most models map to existing superfamilies, AF2 domains expand CATH by 67% and increases the number of unique 'global' folds by 36% and will provide valuable insights on structure function relationships. CATH-Assign will harness the huge expansion in structural data provided by DeepMind to rationalise evolutionary changes driving functional divergence.\", \"doi_y\": \"https://doi.org/10.1038/s42003-023-04488-9\", \"publication_date\": \"2023-02-08\"}, {\"work_id\": \"W4313907594\", \"doi_x\": \"10.1021/acs.analchem.2c05105\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The New Era of High-Throughput Nanoelectrochemistry\", \"abstract\": \"ADVERTISEMENT RETURN TO ISSUEPREVReviewNEXTThe New Era of High-Throughput NanoelectrochemistryXiangdong XuXiangdong XuDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.More by Xiangdong XuView Biographyhttps://orcid.org/0000-0002-1120-4661, Dimitrios ValavanisDimitrios ValavanisDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.More by Dimitrios ValavanisView Biographyhttps://orcid.org/0000-0002-8777-664X, Paolo CiocciPaolo CiocciUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, FranceMore by Paolo CiocciView Biography, Samuel ConfederatSamuel ConfederatSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.More by Samuel ConfederatView Biography, Fabio MarcuccioFabio MarcuccioSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.Faculty of Medicine, Imperial College London, London SW7 2AZ, United KingdomMore by Fabio MarcuccioView Biographyhttps://orcid.org/0000-0003-4816-2896, Jean-Fran\\u00e7ois LemineurJean-Fran\\u00e7ois LemineurUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, FranceMore by Jean-Fran\\u00e7ois LemineurView Biography, Paolo Actis*Paolo ActisSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.*[email protected]More by Paolo ActisView Biographyhttps://orcid.org/0000-0002-7146-1854, Fr\\u00e9d\\u00e9ric Kanoufi*Fr\\u00e9d\\u00e9ric KanoufiUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, France*[email protected]More by Fr\\u00e9d\\u00e9ric KanoufiView Biographyhttps://orcid.org/0000-0002-9784-2380, and Patrick R. Unwin*Patrick R. UnwinDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.*[email protected]More by Patrick R. UnwinView Biographyhttps://orcid.org/0000-0003-3106-2178Cite this: Anal. Chem. 2023, 95, 1, 319\\u2013356Publication Date (Web):January 10, 2023Publication History Received15 November 2022Published online10 January 2023Published inissue 10 January 2023https://doi.org/10.1021/acs.analchem.2c05105Copyright \\u00a9 2023 The Authors. Published by American Chemical SocietyRIGHTS & PERMISSIONSACS AuthorChoiceCC: Creative CommonsBY: Credit must be given to the creatorArticle Views1582Altmetric-Citations1LEARN ABOUT THESE METRICSArticle Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.Citations are the number of other articles citing this article, calculated by Crossref and updated daily. Find more information about Crossref citation counts.The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on the Altmetric Attention Score and how the score is calculated. Share Add toView InAdd Full Text with ReferenceAdd Description ExportRISCitationCitation and abstractCitation and referencesMore Options Share onFacebookTwitterWechatLinked InReddit PDF (5 MB) Get e-AlertsSUBJECTS:Ions,Nanoparticles,Nanopores,Probes,Surface charge Get e-Alerts\", \"doi_y\": \"https://doi.org/10.1021/acs.analchem.2c05105\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W4297494206\", \"doi_x\": \"10.1002/pro.4439\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"abstract\": \"Abstract The archiving and dissemination of protein and nucleic acid structures as well as their structural, functional and biophysical annotations is an essential task that enables the broader scientific community to conduct impactful research in multiple fields of the life sciences. The Protein Data Bank in Europe (PDBe; pdbe.org ) team develops and maintains several databases and web services to address this fundamental need. From data archiving as a member of the Worldwide PDB consortium (wwPDB; wwpdb.org ), to the PDBe Knowledge Base (PDBe\\u2010KB; pdbekb.org ), we provide data, data\\u2010access mechanisms, and visualizations that facilitate basic and applied research and education across the life sciences. Here, we provide an overview of the structural data and annotations that we integrate and make freely available. We describe the web services and data visualization tools we offer, and provide information on how to effectively use or even further develop them. Finally, we discuss the direction of our data services, and how we aim to tackle new challenges that arise from the recent, unprecedented advances in the field of structure determination and protein structure modeling.\", \"doi_y\": \"https://doi.org/10.1002/pro.4439\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W4385196381\", \"doi_x\": \"10.1073/pnas.2306046120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"An expandable, modular de novo protein platform for precision redox engineering\", \"abstract\": \"The electron-conducting circuitry of life represents an as-yet untapped resource of exquisite, nanoscale biomolecular engineering. Here, we report the characterization and structure of a de novo diheme \\u201cmaquette\\u201d protein, 4D2, which we subsequently use to create an expanded, modular platform for heme protein design. A well-folded monoheme variant was created by computational redesign, which was then utilized for the experimental validation of continuum electrostatic redox potential calculations. This demonstrates how fundamental biophysical properties can be predicted and fine-tuned. 4D2 was then extended into a tetraheme helical bundle, representing a 7 nm molecular wire. Despite a molecular weight of only 24 kDa, electron cryomicroscopy illustrated a remarkable level of detail, indicating the positioning of the secondary structure and the heme cofactors. This robust, expressible, highly thermostable and readily designable modular platform presents a valuable resource for redox protein design and the future construction of artificial electron-conducting circuitry.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2306046120\", \"publication_date\": \"2023-07-24\"}, {\"work_id\": \"W4294074096\", \"doi_x\": \"10.1126/science.adc8969\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"abstract\": \"Cyclic adenosine diphosphate (ADP)-ribose (cADPR) isomers are signaling molecules produced by bacterial and plant Toll/interleukin-1 receptor (TIR) domains via nicotinamide adenine dinucleotide (oxidized form) (NAD+) hydrolysis. We show that v-cADPR (2'cADPR) and v2-cADPR (3'cADPR) isomers are cyclized by O-glycosidic bond formation between the ribose moieties in ADPR. Structures of 2'cADPR-producing TIR domains reveal conformational changes that lead to an active assembly that resembles those of Toll-like receptor adaptor TIR domains. Mutagenesis reveals a conserved tryptophan that is essential for cyclization. We show that 3'cADPR is an activator of ThsA effector proteins from the bacterial antiphage defense system termed Thoeris and a suppressor of plant immunity when produced by the effector HopAM1. Collectively, our results reveal the molecular basis of cADPR isomer production and establish 3'cADPR in bacteria as an antiviral and plant immunity-suppressing signaling molecule.\", \"doi_y\": \"https://doi.org/10.1126/science.adc8969\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4223585765\", \"doi_x\": \"10.3390/cells11081307\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"How Pyroptosis Contributes to Inflammation and Fibroblast-Macrophage Cross-Talk in Rheumatoid Arthritis\", \"abstract\": \"About thirty years ago, a new form of pro-inflammatory lytic cell death was observed and termed pyroptosis. Only in 2015, gasdermins were defined as molecules that create pores at the plasma membrane and drive pyroptosis. Today, we know that gasdermin-mediated death is an important antimicrobial defence mechanism in bacteria, yeast and mammals as it destroys the intracellular niche for pathogen replication. However, excessive and uncontrolled cell death also contributes to immunopathology in several chronic inflammatory diseases, including arthritis. In this review, we discuss recent findings where pyroptosis contributes to tissue damage and inflammation with a main focus on injury-induced and autoimmune arthritis. We also review novel functions and regulatory mechanisms of the pyroptotic executors gasdermins. Finally, we discuss possible models of how pyroptosis may contribute to the cross-talk between fibroblast and macrophages, and also how this cross-talk may regulate inflammation by modulating inflammasome activation and pyroptosis induction.\", \"doi_y\": \"https://doi.org/10.3390/cells11081307\", \"publication_date\": \"2022-04-12\"}, {\"work_id\": \"W4385273065\", \"doi_x\": \"10.3389/fnins.2023.1236815\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"How neurons maintain their axons long-term: an integrated view of axon biology and pathology\", \"abstract\": \"Axons are processes of neurons, up to a metre long, that form the essential biological cables wiring nervous systems. They must survive, often far away from their cell bodies and up to a century in humans. This requires self-sufficient cell biology including structural proteins, organelles, and membrane trafficking, metabolic, signalling, translational, chaperone, and degradation machinery\\u2014all maintaining the homeostasis of energy, lipids, proteins, and signalling networks including reactive oxygen species and calcium. Axon maintenance also involves specialised cytoskeleton including the cortical actin-spectrin corset, and bundles of microtubules that provide the highways for motor-driven transport of components and organelles for virtually all the above-mentioned processes. Here, we aim to provide a conceptual overview of key aspects of axon biology and physiology, and the homeostatic networks they form. This homeostasis can be derailed, causing axonopathies through processes of ageing, trauma, poisoning, inflammation or genetic mutations. To illustrate which malfunctions of organelles or cell biological processes can lead to axonopathies, we focus on axonopathy-linked subcellular defects caused by genetic mutations. Based on these descriptions and backed up by our comprehensive data mining of genes linked to neural disorders, we describe the \\u2018dependency cycle of local axon homeostasis\\u2019 as an integrative model to explain why very different causes can trigger very similar axonopathies, providing new ideas that can drive the quest for strategies able to battle these devastating diseases.\", \"doi_y\": \"https://doi.org/10.3389/fnins.2023.1236815\", \"publication_date\": \"2023-07-26\"}, {\"work_id\": \"W4220885070\", \"doi_x\": \"10.1038/s41586-022-04534-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structure, substrate recognition and initiation of hyaluronan synthase\", \"abstract\": \"Hyaluronan is an acidic heteropolysaccharide comprising alternating N-acetylglucosamine and glucuronic acid sugars that is ubiquitously expressed in the vertebrate extracellular matrix1. The high-molecular-mass polymer modulates essential physiological processes in health and disease, including cell differentiation, tissue homeostasis and angiogenesis2. Hyaluronan is synthesized by a membrane-embedded processive glycosyltransferase, hyaluronan synthase (HAS), which catalyses the synthesis and membrane translocation of hyaluronan from uridine diphosphate-activated precursors3,4. Here we describe five cryo-electron microscopy structures of a viral HAS homologue at different states during substrate binding and initiation of polymer synthesis. Combined with biochemical analyses and molecular dynamics simulations, our data reveal how HAS selects its substrates, hydrolyses the first substrate to prime the synthesis reaction, opens a hyaluronan-conducting transmembrane channel, ensures alternating substrate polymerization and coordinates hyaluronan inside its transmembrane pore. Our research suggests a detailed model for the formation of an acidic extracellular heteropolysaccharide and provides insights into the biosynthesis of one of the most abundant and essential glycosaminoglycans in the human body.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-04534-2\", \"publication_date\": \"2022-03-30\"}, {\"work_id\": \"W4319063252\", \"doi_x\": \"10.1371/journal.pbio.3001959\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"LRRC15 mediates an accessory interaction with the SARS-CoV-2 spike protein\", \"abstract\": \"The interactions between Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) and human host factors enable the virus to propagate infections that lead to Coronavirus Disease 2019 (COVID-19). The spike protein is the largest structural component of the virus and mediates interactions essential for infection, including with the primary angiotensin-converting enzyme 2 (ACE2) receptor. We performed two independent cell-based systematic screens to determine whether there are additional proteins by which the spike protein of SARS-CoV-2 can interact with human cells. We discovered that in addition to ACE2, expression of LRRC15 also causes spike protein binding. This interaction is distinct from other known spike attachment mechanisms such as heparan sulfates or lectin receptors. Measurements of orthologous coronavirus spike proteins implied the interaction was functionally restricted to SARS-CoV-2 by accessibility. We localized the interaction to the C-terminus of the S1 domain and showed that LRRC15 shares recognition of the ACE2 receptor binding domain. From analyzing proteomics and single-cell transcriptomics, we identify LRRC15 expression as being common in human lung vasculature cells and fibroblasts. Levels of LRRC15 were greatly elevated by inflammatory signals in the lungs of COVID-19 patients. Although infection assays demonstrated that LRRC15 alone is not sufficient to permit viral entry, we present evidence that it can modulate infection of human cells. This unexpected interaction merits further investigation to determine how SARS-CoV-2 exploits host LRRC15 and whether it could account for any of the distinctive features of COVID-19.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001959\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4366189236\", \"doi_x\": \"10.3390/antiox12040938\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"abstract\": \"Coenzyme A (CoA) is an important cellular metabolite that is critical for metabolic processes and the regulation of gene expression. Recent discovery of the antioxidant function of CoA has highlighted its protective role that leads to the formation of a mixed disulfide bond with protein cysteines, which is termed protein CoAlation. To date, more than 2000 CoAlated bacterial and mammalian proteins have been identified in cellular responses to oxidative stress, with the majority being involved in metabolic pathways (60%). Studies have shown that protein CoAlation is a widespread post-translational modification which modulates the activity and conformation of the modified proteins. The induction of protein CoAlation by oxidative stress was found to be rapidly reversed after the removal of oxidizing agents from the medium of cultured cells. In this study, we developed an enzyme-linked immunosorbent assay (ELISA)-based deCoAlation assay to detect deCoAlation activity from Bacillus subtilis and Bacillus megaterium lysates. We then used a combination of ELISA-based assay and purification strategies to show that deCoAlation is an enzyme-driven mechanism. Using mass-spectrometry and deCoAlation assays, we identified B. subtilis YtpP (thioredoxin-like protein) and thioredoxin A (TrxA) as enzymes that can remove CoA from different substrates. With mutagenesis studies, we identified YtpP and TrxA catalytic cysteine residues and proposed a possible deCoAlation mechanism for CoAlated methionine sulfoxide reducatse A (MsrA) and peroxiredoxin 5 (PRDX5) proteins, which results in the release of both CoA and the reduced form of MsrA or PRDX5. Overall, this paper reveals the deCoAlation activity of YtpP and TrxA and opens doors to future studies on the CoA-mediated redox regulation of CoAlated proteins under various cellular stress conditions.\", \"doi_y\": \"https://doi.org/10.3390/antiox12040938\", \"publication_date\": \"2023-04-15\"}, {\"work_id\": \"W4282929382\", \"doi_x\": \"10.1111/mmi.14953\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"abstract\": \"Gram-negative pathogens like Burkholderia pseudomallei use trimeric autotransporter adhesins such as BpaC as key molecules in their pathogenicity. Our 1.4 \\u00c5 crystal structure of the membrane-proximal part of the BpaC head domain shows that the domain is exclusively made of left-handed parallel \\u03b2-roll repeats. This, the largest such structure solved, has two unique features. First, the core, rather than being composed of the canonical hydrophobic Ile and Val, is made up primarily of the hydrophilic Thr and Asn, with two different solvent channels. Second, comparing BpaC to all other left-handed parallel \\u03b2-roll structures showed that the position of the head domain in the protein correlates with the number and type of charged residues. In BpaC, only negatively charged residues face the solvent-in stark contrast to the primarily positive surface charge of the left-handed parallel \\u03b2-roll \\\"type\\\" protein, YadA. We propose extending the definitions of these head domains to include the BpaC-like head domain as a separate subtype, based on its unusual sequence, position, and charge. We speculate that the function of left-handed parallel \\u03b2-roll structures may differ depending on their position in the structure.\", \"doi_y\": \"https://doi.org/10.1111/mmi.14953\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4307953698\", \"doi_x\": \"10.1038/s41579-022-00809-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"SARS-CoV-2 variant evasion of monoclonal antibodies based on in vitro studies\", \"abstract\": \"Monoclonal antibodies (mAbs) offer a treatment option for individuals with severe COVID-19 and are especially important in high-risk individuals where vaccination is not an option. Given the importance of understanding the evolution of resistance to mAbs by SARS-CoV-2, we reviewed the available in vitro neutralization data for mAbs against live variants and viral constructs containing spike mutations of interest. Unfortunately, evasion of mAb-induced protection is being reported with new SARS-CoV-2 variants. The magnitude of neutralization reduction varied greatly among mAb-variant pairs. For example, sotrovimab retained its neutralization capacity against Omicron BA.1 but showed reduced efficacy against BA.2, BA.4 and BA.5, and BA.2.12.1. At present, only bebtelovimab has been reported to retain its efficacy against all SARS-CoV-2 variants considered here. Resistance to mAb neutralization was dominated by the action of epitope single amino acid substitutions in the spike protein. Although not all observed epitope mutations result in increased mAb evasion, amino acid substitutions at non-epitope positions and combinations of mutations also contribute to evasion of neutralization. This Review highlights the implications for the rational design of viral genomic surveillance and factors to consider for the development of novel mAb therapies.\", \"doi_y\": \"https://doi.org/10.1038/s41579-022-00809-7\", \"publication_date\": \"2022-10-28\"}, {\"work_id\": \"W4308593405\", \"doi_x\": \"10.3389/fmolb.2022.970391\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"abstract\": \"Human equilibrative nucleoside transporters represent a major pharmaceutical target for cardiac, cancer and viral therapies. Understanding the molecular basis for transport is crucial for the development of improved therapeutics through structure-based drug design. ENTs have been proposed to utilise an alternating access mechanism of action, similar to that of the major facilitator superfamily. However, ENTs lack functionally-essential features of that superfamily, suggesting that they may use a different transport mechanism. Understanding the molecular basis of their transport requires insight into diverse conformational states. Differences between intermediate states may be discrete and mediated by subtle gating interactions, such as salt bridges. We identified four variants of human equilibrative nucleoside transporter isoform 1 (hENT1) at the large intracellular loop (ICL6) and transmembrane helix 7 (TM7) that stabilise the apo -state (\\u2206T m 0.7\\u20131.5\\u00b0C). Furthermore, we showed that variants K263A (ICL6) and I282V (TM7) specifically stabilise the inhibitor-bound state of hENT1 (\\u2206\\u2206T m 5.0 \\u00b1 1.7\\u00b0C and 3.0 \\u00b1 1.8\\u00b0C), supporting the role of ICL6 in hENT1 gating. Finally, we showed that, in comparison with wild type, variant T336A is de stabilised by nitrobenzylthioinosine (\\u2206\\u2206T m -4.7 \\u00b1 1.1\\u00b0C) and binds it seven times worse. This residue may help determine inhibitor and substrate sensitivity. Residue K263 is not present in the solved structures, highlighting the need for further structural data that include the loop regions.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.970391\", \"publication_date\": \"2022-11-08\"}, {\"work_id\": \"W4313280753\", \"doi_x\": \"10.15252/embj.2022111519\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Sensor <scp>NLR</scp> immune proteins activate oligomerization of their <scp>NRC</scp> helpers in response to plant pathogens\", \"abstract\": \"Abstract Nucleotide\\u2010binding domain leucine\\u2010rich repeat (NLR) immune receptors are important components of plant and metazoan innate immunity that can function as individual units or as pairs or networks. Upon activation, NLRs form multiprotein complexes termed resistosomes or inflammasomes. Although metazoan paired NLRs, such as NAIP/NLRC4, form hetero\\u2010complexes upon activation, the molecular mechanisms underpinning activation of plant paired NLRs, especially whether they associate in resistosome hetero\\u2010complexes, is unknown. In asterid plant species, the NLR required for cell death (NRC) immune receptor network is composed of multiple resistance protein sensors and downstream helpers that confer immunity against diverse plant pathogens. Here, we show that pathogen effector\\u2010activation of the NLR proteins Rx (confers virus resistance), and Bs2 (confers bacterial resistance) leads to oligomerization of their helper NLR, NRC2. Activated Rx does not oligomerize or enter into a stable complex with the NRC2 oligomer and remains cytoplasmic. In contrast, activated NRC2 oligomers accumulate in membrane\\u2010associated puncta. We propose an activation\\u2010and\\u2010release model for NLRs in the NRC immune receptor network. This points to a distinct activation model compared with mammalian paired NLRs.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111519\", \"publication_date\": \"2022-12-29\"}, {\"work_id\": \"W4362521293\", \"doi_x\": \"10.26508/lsa.202301954\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>arfA</i>antisense RNA regulates MscL excretory activity\", \"abstract\": \"Excretion of cytoplasmic protein (ECP) is a commonly observed phenomenon in bacteria, and this partial extracellular localisation of the intracellular proteome has been implicated in a variety of stress response mechanisms. In response to hypoosmotic shock and ribosome stalling in Escherichia coli, ECP is dependent upon the presence of the large-conductance mechanosensitive channel and the alternative ribosome-rescue factor A gene products. However, it is not known if a mechanistic link exists between the corresponding genes and the respective stress response pathways. Here, we report that the corresponding mscL and arfA genes are commonly co-located on the genomes of Gammaproteobacteria and display overlap in their respective 3' UTR and 3' CDS. We show this unusual genomic arrangement permits an antisense RNA-mediated regulatory control between mscL and arfA, and this modulates MscL excretory activity in E. coli These findings highlight a mechanistic link between osmotic, translational stress responses and ECP in E. coli, further elucidating the previously unknown regulatory function of arfA sRNA.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202301954\", \"publication_date\": \"2023-04-03\"}, {\"work_id\": \"W4310055674\", \"doi_x\": \"10.1107/s2059798322010415\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"abstract\": \"Determination of protein structures typically entails building a model that satisfies the collected experimental observations and its deposition in the Protein Data Bank. Experimental limitations can lead to unavoidable uncertainties during the process of model building, which result in the introduction of errors into the deposited model. Many metrics are available for model validation, but most are limited to consideration of the physico-chemical aspects of the model or its match to the experimental data. The latest advances in the field of deep learning have enabled the increasingly accurate prediction of inter-residue distances, an advance which has played a pivotal role in the recent improvements observed in the field of protein ab initio modelling. Here, new validation methods are presented based on the use of these precise inter-residue distance predictions, which are compared with the distances observed in the protein model. Sequence-register errors are particularly clearly detected and the register shifts required for their correction can be reliably determined. The method is available in the ConKit package (https://www.conkit.org).\", \"doi_y\": \"https://doi.org/10.1107/s2059798322010415\", \"publication_date\": \"2022-11-25\"}, {\"work_id\": \"W4225516213\", \"doi_x\": \"10.1093/bib/bbac187\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"abstract\": \"Mutations in human proteins lead to diseases. The structure of these proteins can help understand the mechanism of such diseases and develop therapeutics against them. With improved deep learning techniques, such as RoseTTAFold and AlphaFold, we can predict the structure of proteins even in the absence of structural homologs. We modeled and extracted the domains from 553 disease-associated human proteins without known protein structures or close homologs in the Protein Databank. We noticed that the model quality was higher and the Root mean square deviation (RMSD) lower between AlphaFold and RoseTTAFold models for domains that could be assigned to CATH families as compared to those which could only be assigned to Pfam families of unknown structure or could not be assigned to either. We predicted ligand-binding sites, protein-protein interfaces and conserved residues in these predicted structures. We then explored whether the disease-associated missense mutations were in the proximity of these predicted functional sites, whether they destabilized the protein structure based on ddG calculations or whether they were predicted to be pathogenic. We could explain 80% of these disease-associated mutations based on proximity to functional sites, structural destabilization or pathogenicity. When compared to polymorphisms, a larger percentage of disease-associated missense mutations were buried, closer to predicted functional sites, predicted as destabilizing and pathogenic. Usage of models from the two state-of-the-art techniques provide better confidence in our predictions, and we explain 93 additional mutations based on RoseTTAFold models which could not be explained based solely on AlphaFold models.\", \"doi_y\": \"https://doi.org/10.1093/bib/bbac187\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4304889335\", \"doi_x\": \"10.1016/j.jbc.2022.102589\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Herpes simplex virus 1 protein pUL21 alters ceramide metabolism by activating the interorganelle transport protein CERT\", \"abstract\": \"Herpes simplex virus (HSV)-1 dramatically alters the architecture and protein composition of cellular membranes during infection, but its effects upon membrane lipid composition remain unclear. HSV-1 pUL21 is a virus-encoded protein phosphatase adaptor that promotes dephosphorylation of multiple cellular and virus proteins, including the cellular ceramide (Cer) transport protein CERT. CERT mediates nonvesicular Cer transport from the endoplasmic reticulum to the trans-Golgi network, whereupon Cer is converted to sphingomyelin (SM) and other sphingolipids that play important roles in cellular proliferation, signaling, and membrane trafficking. Here, we use click chemistry to profile the kinetics of sphingolipid metabolism, showing that pUL21-mediated dephosphorylation activates CERT and accelerates Cer-to-SM conversion. Purified pUL21 and full-length CERT interact with submicromolar affinity, and we solve the solution structure of the pUL21 C-terminal domain in complex with the CERT Pleckstrin homology and steroidogenic acute regulatory-related lipid transfer domains using small-angle X-ray scattering. We identify a single amino acid mutation on the surface of pUL21 that disrupts CERT binding in vitro and in cultured cells. This residue is highly conserved across the genus Simplexvirus. In addition, we identify a pUL21 residue essential for binding to HSV-1 pUL16. Sphingolipid profiling demonstrates that Cer-to-SM conversion is severely diminished in the context of HSV-1 infection, a defect that is compounded when infecting with a virus encoding the mutated form of pUL21 that lacks the ability to activate CERT. However, virus replication and spread in cultured keratinocytes or epithelial cells is not significantly altered when pUL21-mediated CERT dephosphorylation is abolished. Collectively, we demonstrate that HSV-1 modifies sphingolipid metabolism via specific protein-protein interactions.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102589\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4280528488\", \"doi_x\": \"10.1146/annurev-virology-100120-010228\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"abstract\": \"Advances in our understanding of prokaryotic antiphage defense mechanisms in the past few years have revealed a multitude of new cyclic nucleotide signaling molecules that play a crucial role in switching infected cells into an antiviral state. Defense pathways including type III CRISPR (clustered regularly interspaced palindromic repeats), CBASS (cyclic nucleotide-based antiphage signaling system), PYCSAR (pyrimidine cyclase system for antiphage resistance), and Thoeris all use cyclic nucleotides as second messengers to activate a diverse range of effector proteins. These effectors typically degrade or disrupt key cellular components such as nucleic acids, membranes, or metabolites, slowing down viral replication kinetics at great cost to the infected cell. Mechanisms to manipulate the levels of cyclic nucleotides are employed by cells to regulate defense pathways and by viruses to subvert them. Here we review the discovery and mechanism of the key pathways, signaling molecules and effectors, parallels and differences between the systems, open questions, and prospects for future research in this area.\", \"doi_y\": \"https://doi.org/10.1146/annurev-virology-100120-010228\", \"publication_date\": \"2022-09-29\"}, {\"work_id\": \"W4291020693\", \"doi_x\": \"10.1038/s41586-022-05070-9\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"abstract\": \"Cyclic nucleotide signalling is a key component of antiviral defence in all domains of life. Viral detection activates a nucleotide cyclase to generate a second messenger, resulting in activation of effector proteins. This is exemplified by the metazoan cGAS-STING innate immunity pathway1, which originated in bacteria2. These defence systems require a sensor domain to bind the cyclic nucleotide and are often coupled with an effector domain that, when activated, causes cell death by destroying essential biomolecules3. One example is the Toll/interleukin-1 receptor (TIR) domain, which degrades the essential cofactor NAD+ when activated in response to infection in plants and bacteria2,4,5 or during programmed nerve cell death6. Here we show that a bacterial antiviral defence system generates a cyclic tri-adenylate that binds to a TIR-SAVED effector, acting as the 'glue' to allow assembly of an extended superhelical solenoid structure. Adjacent TIR subunits interact to organize and complete a composite active site, allowing NAD+ degradation. Activation requires extended filament formation, both in vitro and in vivo. Our study highlights an example of large-scale molecular assembly controlled by cyclic nucleotides and reveals key details of the mechanism of TIR enzyme activation.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-05070-9\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4385255907\", \"doi_x\": \"10.1111/2041-210x.14166\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Why should we compare morphological and molecular disparity?\", \"abstract\": \"Abstract Indices of morphological disparity seek to summarise the highly multivariate morphological variation across groups of species within clades, time bins or other groups. Morphological variation can be quantified using geometric morphometric, outline or surface\\u2010based methods. These are most effective when morphological differences are relatively modest and there are numerous ubiquitous landmarks and phase aligned features of shape variation. The most disparate samples, such as those across classes and phyla, typically necessitate the use of discrete characters. Unfortunately, such characters are often compiled subjectively in a manner reflecting the level of morphological and taxonomic focus and the intensity of taxon sampling. Sampling intensity is often highly variable within a single data set, especially in repurposed and amalgamated cladistic matrices. Here, we propose indices of molecular disparity analogous to those of morphological disparity. Despite numerous shortcomings discussed here, molecular sequence data can be obtained in a more objective, automated and scalable manner than morphological data. Comparisons of the morphological and molecular disparity of subclades in 16 large data sets suggest that molecular disparity is less susceptible to sampling biases than morphological disparity. Moreover, distance matrices inferred from individual genes tend to correlate strongly with each other and with distances from all concatenated genes. By contrast, morphological and molecular disparity are typically not significantly correlated across subclades, such that comparisons for groups can help to give a fuller picture of their evolution. For example, within mammals, Afrotheria have conspicuously high morphological disparity but modest molecular disparity, suggesting unusually high morphological plasticity. Even more strikingly, the molecular disparity of rodents is over five times that for Artiodactyla, despite having only half of their morphological disparity. These contrasts suggest the differential operation of geometric, biomechanical, ontogenetic and environmental constraints on form. Given the increasing abundance of total evidence datasets in the literature and the widespread and sometimes uncritical repurposing of discrete morphological matrices, we propose the comparison of morphological and molecular disparity as a useful tool to understand subclade evolution more fully.\", \"doi_y\": \"https://doi.org/10.1111/2041-210x.14166\", \"publication_date\": \"2023-07-24\"}, {\"work_id\": \"W2978205448\", \"doi_x\": \"10.1101/793943\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Energy landscape steering in SecYEG mediates dynamic coupling in ATP driven protein translocation\", \"abstract\": \"Abstract The Sec translocon is a highly conserved membrane complex for transport of polypeptides across, or into, lipid bilayers. In bacteria, the core protein-channel SecYEG resides in the inner-membrane, through which secretion is powered by the cytosolic ATPase SecA. Here, we use single-molecule fluorescence to interrogate the dynamic state of SecYEG throughout the hydrolytic cycle of SecA. We show that the SecYEG channel fluctuates between open and closed states faster ( \\u223c 20-fold during transport) than ATP turnover; while the nucleotide status of SecA modulates the rates of opening and closure. Interestingly, a SecY variant (PrlA4), exhibiting faster protein transport, but unaffected ATPase rates, increases the dwell time in the open state, facilitating pre-protein diffusion through the pore; thereby improving the efficiency of translocation. Thus, contrary to prevailing structure-based models, SecYEG plays an integral part in the translocation mechanism through dynamic allosteric coupling in which SecA \\u2018steers\\u2019 the energy landscape of the protein-channel.\", \"doi_y\": \"https://doi.org/10.1101/793943\", \"publication_date\": \"2019-10-05\"}, {\"work_id\": \"W4320723690\", \"doi_x\": \"10.3389/fcvm.2023.1094563\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Affimers targeting proteins in the cardiomyocyte Z-disc: Novel tools that improve imaging of heart tissue\", \"abstract\": \"Dilated Cardiomyopathy is a common form of heart failure. Determining how this disease affects the structure and organization of cardiomyocytes in the human heart is important in understanding how the heart becomes less effective at contraction. Here we isolated and characterised Affimers (small non-antibody binding proteins) to Z-disc proteins ACTN2 (\\u03b1-actinin-2), ZASP (also known as LIM domain binding protein 3 or LDB3) and the N-terminal region of the giant protein titin (TTN Z1-Z2). These proteins are known to localise in both the sarcomere Z-discs and the transitional junctions, found close to the intercalated discs that connect adjacent cardiomyocytes. We use cryosections of left ventricles from two patients diagnosed with end-stage Dilated Cardiomyopathy who underwent Orthotopic Heart Transplantation and were whole genome sequenced. We describe how Affimers substantially improve the resolution achieved by confocal and STED microscopy compared to conventional antibodies. We quantified the expression of ACTN2, ZASP and TTN proteins in two patients with dilated cardiomyopathy and compared them with a sex- and age-matched healthy donor. The small size of the Affimer reagents, combined with a small linkage error (the distance from the epitope to the dye label covalently bound to the Affimer) revealed new structural details in Z-discs and intercalated discs in the failing samples. Affimers are thus useful for analysis of changes to cardiomyocyte structure and organisation in diseased hearts.\", \"doi_y\": \"https://doi.org/10.3389/fcvm.2023.1094563\", \"publication_date\": \"2023-02-14\"}, {\"work_id\": \"W4316926589\", \"doi_x\": \"10.1093/bioinformatics/btad029\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"abstract\": \"CATH is a protein domain classification resource that exploits an automated workflow of structure and sequence comparison alongside expert manual curation to construct a hierarchical classification of evolutionary and structural relationships. The aim of this study was to develop algorithms for detecting remote homologues missed by state-of-the-art hidden Markov model (HMM)-based approaches. The method developed (CATHe) combines a neural network with sequence representations obtained from protein language models. It was assessed using a dataset of remote homologues having less than 20% sequence identity to any domain in the training set.The CATHe models trained on 1773 largest and 50 largest CATH superfamilies had an accuracy of 85.6 \\u00b1 0.4% and 98.2 \\u00b1 0.3%, respectively. As a further test of the power of CATHe to detect more remote homologues missed by HMMs derived from CATH domains, we used a dataset consisting of protein domains that had annotations in Pfam, but not in CATH. By using highly reliable CATHe predictions (expected error rate <0.5%), we were able to provide CATH annotations for 4.62 million Pfam domains. For a subset of these domains from Homo sapiens, we structurally validated 90.86% of the predictions by comparing their corresponding AlphaFold2 structures with structures from the CATH superfamilies to which they were assigned.The code for the developed models is available on https://github.com/vam-sin/CATHe, and the datasets developed in this study can be accessed on https://zenodo.org/record/6327572.Supplementary data are available at Bioinformatics online.\", \"doi_y\": \"https://doi.org/10.1093/bioinformatics/btad029\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4287510390\", \"doi_x\": \"10.1038/s41467-022-32004-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"abstract\": \"Carboxysomes are a family of bacterial microcompartments in cyanobacteria and chemoautotrophs. They encapsulate Ribulose 1,5-bisphosphate carboxylase/oxygenase (Rubisco) and carbonic anhydrase catalyzing carbon fixation inside a proteinaceous shell. How Rubisco complexes pack within the carboxysomes is unknown. Using cryo-electron tomography, we determine the distinct 3D organization of Rubisco inside two distant \\u03b1-carboxysomes from a marine \\u03b1-cyanobacterium Cyanobium sp. PCC 7001 where Rubiscos are organized in three concentric layers, and from a chemoautotrophic bacterium Halothiobacillus neapolitanus where they form intertwining spirals. We further resolve the structures of native Rubisco as well as its higher-order assembly at near-atomic resolutions by subtomogram averaging. The structures surprisingly reveal that the authentic intrinsically disordered linker protein CsoS2 interacts with Rubiscos in native carboxysomes but functions distinctively in the two \\u03b1-carboxysomes. In contrast to the uniform Rubisco-CsoS2 association in the Cyanobium \\u03b1-carboxysome, CsoS2 binds only to the Rubiscos close to the shell in the Halo \\u03b1-carboxysome. Our findings provide critical knowledge of the assembly principles of \\u03b1-carboxysomes, which may aid in the rational design and repurposing of carboxysome structures for new functions.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-32004-w\", \"publication_date\": \"2022-07-25\"}, {\"work_id\": \"W3173575423\", \"doi_x\": \"10.1002/prot.26214\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"abstract\": \"The assessment of CASP models for utility in molecular replacement is a measure of their use in a valuable real-world application. In CASP7, the metric for molecular replacement assessment involved full likelihood-based molecular replacement searches; however, this restricted the assessable targets to crystal structures with only one copy of the target in the asymmetric unit, and to those where the search found the correct pose. In CASP10, full molecular replacement searches were replaced by likelihood-based rigid-body refinement of models superimposed on the target using the LGA algorithm, with the metric being the refined log-likelihood-gain (LLG) score. This enabled multi-copy targets and very poor models to be evaluated, but a significant further issue remained: the requirement of diffraction data for assessment. We introduce here the relative-expected-LLG (reLLG), which is independent of diffraction data. This reLLG is also independent of any crystal form, and can be calculated regardless of the source of the target, be it X-ray, NMR or cryo-EM. We calibrate the reLLG against the LLG for targets in CASP14, showing that it is a robust measure of both model and group ranking. Like the LLG, the reLLG shows that accurate coordinate error estimates add substantial value to predicted models. We find that refinement by CASP groups can often convert an inadequate initial model into a successful MR search model. Consistent with findings from others, we show that the AlphaFold2 models are sufficiently good, and reliably so, to surpass other current model generation strategies for attempting molecular replacement phasing.\", \"doi_y\": \"https://doi.org/10.1002/prot.26214\", \"publication_date\": \"2021-08-21\"}, {\"work_id\": \"W4321116997\", \"doi_x\": \"10.1016/j.jmb.2023.168016\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"MGnify Genomes: A Resource for Biome-specific Microbial Genome Catalogues\", \"abstract\": \"An increasingly common output arising from the analysis of shotgun metagenomic datasets is the generation of metagenome-assembled genomes (MAGs), with tens of thousands of MAGs now described in the literature. However, the discovery and comparison of these MAG collections is hampered by the lack of uniformity in their generation, annotation and storage. To address this, we have developed MGnify Genomes, a growing collection of biome-specific non-redundant microbial genome catalogues generated using MAGs and publicly available isolate genomes. Genomes within a biome-specific catalogue are organised into species clusters. For species that contain multiple conspecific genomes, the highest quality genome is selected as the representative, always prioritising an isolate genome over a MAG. The species representative sequences and annotations can be visualised on the MGnify website and the full catalogue and associated analysis outputs can be downloaded from MGnify servers. A suite of online search tools is provided allowing users to compare their own sequences, ranging from a gene to sets of genomes, against the catalogues. Seven biomes are available currently, comprising over 300,000 genomes that represent 11,048 non-redundant species, and include 36 taxonomic classes not currently represented by cultured genomes. MGnify Genomes is available at https://www.ebi.ac.uk/metagenomics/browse/genomes/.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168016\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4386033679\", \"doi_x\": \"10.1038/s41467-023-40695-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"TRAIP resolves DNA replication-transcription conflicts during the S-phase of unperturbed cells\", \"abstract\": \"Cell division is the basis for the propagation of life and requires accurate duplication of all genetic information. DNA damage created during replication (replication stress) is a major cause of cancer, premature aging and a spectrum of other human disorders. Over the years, TRAIP E3 ubiquitin ligase has been shown to play a role in various cellular processes that govern genome integrity and faultless segregation. TRAIP is essential for cell viability, and mutations in TRAIP ubiquitin ligase activity lead to primordial dwarfism in patients. Here, we have determined the mechanism of inhibition of cell proliferation in TRAIP-depleted cells. We have taken advantage of the auxin induced degron system to rapidly degrade TRAIP within cells and to dissect the importance of various functions of TRAIP in different stages of the cell cycle. We conclude that upon rapid TRAIP degradation, specifically in S-phase, cells cease to proliferate, arrest in G2 stage of the cell cycle and undergo senescence. Our findings reveal that TRAIP works in S-phase to prevent DNA damage at transcription start sites, caused by replication-transcription conflicts.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40695-y\", \"publication_date\": \"2023-08-21\"}, {\"work_id\": \"W4294671716\", \"doi_x\": \"10.1021/acs.molpharmaceut.2c00453\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Understanding the Stabilizing Effect of Histidine on mAb Aggregation: A Molecular Dynamics Study\", \"abstract\": \"Histidine, a widely used buffer in monoclonal antibody (mAb) formulations, is known to reduce antibody aggregation. While experimental studies suggest a nonelectrostatic, nonstructural (relating to secondary structure preservation) origin of the phenomenon, the underlying microscopic mechanism behind the histidine action is still unknown. Understanding this mechanism will help evaluate and predict the stabilizing effect of this buffer under different experimental conditions and for different mAbs. We have used all-atom molecular dynamics simulations and contact-based free energy calculations to investigate molecular-level interactions between the histidine buffer and mAbs, which lead to the observed stability of therapeutic formulations in the presence of histidine. We reformulate the Spatial Aggregation Propensity index by including the buffer-protein interactions. The buffer adsorption on the protein surface leads to lower exposure of the hydrophobic regions to water. Our analysis indicates that the mechanism behind the stabilizing action of histidine is connected to the shielding of the solvent-exposed hydrophobic regions on the protein surface by the buffer molecules.\", \"doi_y\": \"https://doi.org/10.1021/acs.molpharmaceut.2c00453\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4376141439\", \"doi_x\": \"10.1002/ange.202218783\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Darobactin B Stabilises a Lateral\\u2010Closed Conformation of the BAM Complex in <i>E. coli</i> Cells\", \"abstract\": \"Abstract The \\u03b2\\u2010barrel assembly machinery (BAM complex) is essential for outer membrane protein (OMP) folding in Gram\\u2010negative bacteria, and represents a promising antimicrobial target. Several conformational states of BAM have been reported, but all have been obtained under conditions which lack the unique features and complexity of the outer membrane (OM). Here, we use Pulsed Electron\\u2010Electron Double Resonance (PELDOR, or DEER) spectroscopy distance measurements to interrogate the conformational ensemble of the BAM complex in E. coli cells. We show that BAM adopts a broad ensemble of conformations in the OM, while in the presence of the antibiotic darobactin B (DAR\\u2010B), BAM\\u2032s conformational equilibrium shifts to a restricted ensemble consistent with the lateral closed state. Our in\\u2010cell PELDOR findings are supported by new cryoEM structures of BAM in the presence and absence of DAR\\u2010B. This work demonstrates the utility of PELDOR to map conformational changes in BAM within its native cellular environment.\", \"doi_y\": \"https://doi.org/10.1002/ange.202218783\", \"publication_date\": \"2023-05-31\"}, {\"work_id\": \"W4290730357\", \"doi_x\": \"10.1080/19336950.2022.2098661\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"abstract\": \"Mechanosensitive ion channels are integral membrane proteins ubiquitously present in bacteria, archaea, and eukarya. They act as molecular sensors of mechanical stress to serve vital functions such as touch, hearing, osmotic pressure, proprioception and balance, while their malfunction is often associated with pathologies. Amongst them, the structurally distinct MscL and MscS channels from bacteria are the most extensively studied. MscS-like channels have been found in plants and Schizosaccharomyces pombe, where they regulate intracellular Ca2+ and cell volume under hypo-osmotic conditions. Here we characterize two MscS-like putative channels, named MscA and MscB, from the model filamentous fungus Aspergillus nidulans. Orthologues of MscA and MscB are present in most fungi, including relative plant and animal pathogens. MscA/MscB and other fungal MscS-like proteins share the three transmembrane helices and the extended C-terminal cytosolic domain that form the structural fingerprint of MscS-like channels with at least three additional transmembrane segments than Escherichia coli MscS. We show that MscA and MscB localize in Endoplasmic Reticulum and the Plasma Membrane, respectively, whereas their overexpression leads to increased CaCl2 toxicity or/and reduction of asexual spore formation. Our findings contribute to understanding the role of MscS-like channels in filamentous fungi and relative pathogens.\", \"doi_y\": \"https://doi.org/10.1080/19336950.2022.2098661\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4282925859\", \"doi_x\": \"10.1038/s42003-022-03502-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"abstract\": \"Abstract Correct folding of outer membrane proteins (OMPs) into the outer membrane of Gram-negative bacteria depends on delivery of unfolded OMPs to the \\u03b2-barrel assembly machinery (BAM). How unfolded substrates are presented to BAM remains elusive, but the major OMP chaperone SurA is proposed to play a key role. Here, we have used hydrogen deuterium exchange mass spectrometry (HDX-MS), crosslinking, in vitro folding and binding assays and computational modelling to show that the core domain of SurA and one of its two PPIase domains are key to the SurA-BAM interaction and are required for maximal catalysis of OMP folding. We reveal that binding causes changes in BAM and SurA conformation and/or dynamics distal to the sites of binding, including at the BamA \\u03b21-\\u03b216 seam. We propose a model for OMP biogenesis in which SurA plays a crucial role in OMP delivery and primes BAM to accept substrates for folding.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03502-w\", \"publication_date\": \"2022-06-08\"}, {\"work_id\": \"W4312213393\", \"doi_x\": \"10.1101/2022.12.26.22283911\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Exome sequencing reveals aggregates of rare variants in glycosyltransferase and other genes influencing immunoglobulin G and transferrin glycosylation\", \"abstract\": \"Abstract It is often difficult to be certain which genes underlie the effects seen in association studies. However, variants that disrupt the protein, such as predicted loss of function (pLoF) and missense variants, provide a shortcut to identify genes with a clear biological link to the phenotype of interest. Glycosylation is one of the most common post-translationalmodifications of proteins, and an important biomarker of both disease and its progression. Here, we utilised the power of genetic isolates, gene-based aggregation tests and intermediate phenotypes to assess the effect of rare (MAF&lt;5%) pLoF and missense variants from whole exome sequencing on the N-glycome of plasma transferrin (N=1907) and immunoglobulin G (N=4912), and their effect on diseases. We identified significant gene-based associations for transferrin glycosylation at 5 genes (p&lt;8.06\\u00d710 \\u22128 ) and for IgG glycan traits at 4 genes (p&lt;1.19\\u00d710 \\u22127 ). Associations in three of these genes ( FUT8, MGAT3 and RFXAP ) are driven by multiple rare variants simultaneously contributing to protein glycosylation. Association at ST6GAL1 , with a 300-fold up-drifted variant in the Orkney Islands, was detectable by a single-point exome-wide association analysis. Glycome-associated aggregate associations are located in genes already known to have a biological link to protein glycosylation ( FUT6, FUT8 for transferrin; FUT8, MGAT3 and ST6GAL1 for IgG) but also in genes which have not been previously reported (e.g. RFXAP for IgG). To assess the potential impact of rare variants associated with glycosylation on other traits, we queried public repositories of gene-based tests, discovering a potential connection between transferrin glycosylation, MSR1 , galectin-3, insulin-like growth factor 1 and diabetes. However, the exact mechanism behind these connections requires further elucidation.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.26.22283911\", \"publication_date\": \"2022-12-27\"}, {\"work_id\": \"W4291014905\", \"doi_x\": \"10.1021/acs.jmedchem.2c00804\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"abstract\": \"G-protein-coupled receptor 84 (GPR84) is a proinflammatory orphan G-protein-coupled receptor implicated in several inflammatory and fibrotic diseases. Several agonist and antagonist ligands have been developed that target GPR84; however, a noncompetitive receptor blocker that was progressed to phase II clinical trials failed to demonstrate efficacy. New high-quality antagonists are required to investigate the pathophysiological role of GPR84 and to validate GPR84 as a therapeutic target. We previously reported the discovery of a novel triazine GPR84 competitive antagonist 1. Here, we describe an extensive structure-activity relationship (SAR) of antagonist 1 and also present in silico docking with supporting mutagenesis studies that reveals a potential binding pose for this type of orthosteric antagonist. Lead compound 42 is a potent GPR84 antagonist with a favorable pharmacokinetic (PK) profile suitable for further drug development.\", \"doi_y\": \"https://doi.org/10.1021/acs.jmedchem.2c00804\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4291801098\", \"doi_x\": \"10.1101/2022.08.12.503723\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evolutionary dynamics of genome size and content during the adaptive radiation of Heliconiini butterflies\", \"abstract\": \"Abstract Heliconius butterflies, a speciose genus of M\\u00fcllerian mimics, represent a classic example of an adaptive radiation that includes a range of derived dietary, life history, physiological and neural traits. However, key lineages within the genus, and across the broader Heliconiini tribe, lack genomic resources, limiting our understanding of how adaptive and neutral processes shaped genome evolution during their radiation. We have generated highly contiguous genome assemblies for nine new Heliconiini, 29 additional reference-assembled genomes, and improve 10 existing assemblies. Altogether, we provide a major new dataset of annotated genomes for a total of 63 species, including 58 species within the Heliconiini tribe. We use this extensive dataset to generate a robust and dated heliconiine phylogeny, describe major patterns of introgression, explore the evolution of genome architecture, and the genomic basis of key innovations in this enigmatic group, including an assessment of the evolution of putative regulatory regions at the Heliconius stem. Our work illustrates how the increased resolution provided by such dense genomic sampling improves our power to generate and test gene-phenotype hypotheses, and precisely characterize how genomes evolve.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.12.503723\", \"publication_date\": \"2022-08-15\"}, {\"work_id\": \"W4379928262\", \"doi_x\": \"10.1371/journal.ppat.1011433\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The host phylogeny determines viral infectivity and replication across Staphylococcus host species\", \"abstract\": \"Virus host shifts, where a virus transmits to and infects a novel host species, are a major source of emerging infectious disease. Genetic similarity between eukaryotic host species has been shown to be an important determinant of the outcome of virus host shifts, but it is unclear if this is the case for prokaryotes where anti-virus defences can be transmitted by horizontal gene transfer and evolve rapidly. Here, we measure the susceptibility of 64 strains of Staphylococcaceae bacteria (48 strains of Staphylococcus aureus and 16 non-S. aureus species spanning 2 genera) to the bacteriophage ISP, which is currently under investigation for use in phage therapy. Using three methods-plaque assays, optical density (OD) assays, and quantitative (q)PCR-we find that the host phylogeny explains a large proportion of the variation in susceptibility to ISP across the host panel. These patterns were consistent in models of only S. aureus strains and models with a single representative from each Staphylococcaceae species, suggesting that these phylogenetic effects are conserved both within and among host species. We find positive correlations between susceptibility assessed using OD and qPCR and variable correlations between plaque assays and either OD or qPCR, suggesting that plaque assays alone may be inadequate to assess host range. Furthermore, we demonstrate that the phylogenetic relationships between bacterial hosts can generally be used to predict the susceptibility of bacterial strains to phage infection when the susceptibility of closely related hosts is known, although this approach produced large prediction errors in multiple strains where phylogeny was uninformative. Together, our results demonstrate the ability of bacterial host evolutionary relatedness to explain differences in susceptibility to phage infection, with implications for the development of ISP both as a phage therapy treatment and as an experimental system for the study of virus host shifts.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011433\", \"publication_date\": \"2023-06-08\"}, {\"work_id\": \"W4296400863\", \"doi_x\": \"10.1002/1873-3468.14500\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protein metalation in a nutshell\", \"abstract\": \"Metalation, the acquisition of metals by proteins, must avoid mis\\u2010metalation with tighter binding metals. This is illustrated by four selected proteins that require different metals: all show similar ranked orders of affinity for bioavailable metals, as described in a universal affinity series (the Irving\\u2013Williams series). Crucially, cellular protein metalation occurs in competition with other metal binding sites. The strength of this competition defines the intracellular availability of each metal: its magnitude has been estimated by calibrating a cells' set of DNA\\u2010binding, metal\\u2010sensing, transcriptional regulators. This has established that metal availabilities (as free energies for forming metal complexes) are maintained to the inverse of the universal series. The tightest binding metals are least available. With these availabilities, correct metalation is achieved.\", \"doi_y\": \"https://doi.org/10.1002/1873-3468.14500\", \"publication_date\": \"2022-09-26\"}, {\"work_id\": \"W4310059371\", \"doi_x\": \"10.1038/s41467-022-34972-5\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"abstract\": \"Abstract The Klebsiella jumbo myophage \\u03d5Kp24 displays an unusually complex arrangement of tail fibers interacting with a host cell. In this study, we combine cryo-electron microscopy methods, protein structure prediction methods, molecular simulations, microbiological and machine learning approaches to explore the capsid, tail, and tail fibers of \\u03d5Kp24. We determine the structure of the capsid and tail at 4.1 \\u00c5 and 3.0 \\u00c5 resolution. We observe the tail fibers are branched and rearranged dramatically upon cell surface attachment. This complex configuration involves fourteen putative tail fibers with depolymerase activity that provide \\u03d5Kp24 with the ability to infect a broad panel of capsular polysaccharide (CPS) types of Klebsiella pneumoniae . Our study provides structural and functional insight into how \\u03d5Kp24 adapts to the variable surfaces of capsulated bacterial pathogens, which is useful for the development of phage therapy approaches against pan-drug resistant K. pneumoniae strains.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-34972-5\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4224950216\", \"doi_x\": \"10.1016/s0140-6736(22)00462-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Comparative analysis of the risks of hospitalisation and death associated with SARS-CoV-2 omicron (B.1.1.529) and delta (B.1.617.2) variants in England: a cohort study\", \"abstract\": \"BackgroundThe omicron variant (B.1.1.529) of SARS-CoV-2 has demonstrated partial vaccine escape and high transmissibility, with early studies indicating lower severity of infection than that of the delta variant (B.1.617.2). We aimed to better characterise omicron severity relative to delta by assessing the relative risk of hospital attendance, hospital admission, or death in a large national cohort.MethodsIndividual-level data on laboratory-confirmed COVID-19 cases resident in England between Nov 29, 2021, and Jan 9, 2022, were linked to routine datasets on vaccination status, hospital attendance and admission, and mortality. The relative risk of hospital attendance or admission within 14 days, or death within 28 days after confirmed infection, was estimated using proportional hazards regression. Analyses were stratified by test date, 10-year age band, ethnicity, residential region, and vaccination status, and were further adjusted for sex, index of multiple deprivation decile, evidence of a previous infection, and year of age within each age band. A secondary analysis estimated variant-specific and vaccine-specific vaccine effectiveness and the intrinsic relative severity of omicron infection compared with delta (ie, the relative risk in unvaccinated cases).FindingsThe adjusted hazard ratio (HR) of hospital attendance (not necessarily resulting in admission) with omicron compared with delta was 0\\u00b756 (95% CI 0\\u00b754\\u20130\\u00b758); for hospital admission and death, HR estimates were 0\\u00b741 (0\\u00b739\\u20130\\u00b743) and 0\\u00b731 (0\\u00b726\\u20130\\u00b737), respectively. Omicron versus delta HR estimates varied with age for all endpoints examined. The adjusted HR for hospital admission was 1\\u00b710 (0\\u00b785\\u20131\\u00b742) in those younger than 10 years, decreasing to 0\\u00b725 (0\\u00b721\\u20130\\u00b730) in 60\\u201369-year-olds, and then increasing to 0\\u00b747 (0\\u00b740\\u20130\\u00b756) in those aged at least 80 years. For both variants, past infection gave some protection against death both in vaccinated (HR 0\\u00b747 [0\\u00b732\\u20130\\u00b768]) and unvaccinated (0\\u00b718 [0\\u00b706\\u20130\\u00b757]) cases. In vaccinated cases, past infection offered no additional protection against hospital admission beyond that provided by vaccination (HR 0\\u00b796 [0\\u00b788\\u20131\\u00b704]); however, for unvaccinated cases, past infection gave moderate protection (HR 0\\u00b755 [0\\u00b748\\u20130\\u00b763]). Omicron versus delta HR estimates were lower for hospital admission (0\\u00b730 [0\\u00b728\\u20130\\u00b732]) in unvaccinated cases than the corresponding HR estimated for all cases in the primary analysis. Booster vaccination with an mRNA vaccine was highly protective against hospitalisation and death in omicron cases (HR for hospital admission 8\\u201311 weeks post-booster vs unvaccinated: 0\\u00b722 [0\\u00b720\\u20130\\u00b724]), with the protection afforded after a booster not being affected by the vaccine used for doses 1 and 2.InterpretationThe risk of severe outcomes following SARS-CoV-2 infection is substantially lower for omicron than for delta, with higher reductions for more severe endpoints and significant variation with age. Underlying the observed risks is a larger reduction in intrinsic severity (in unvaccinated individuals) counterbalanced by a reduction in vaccine effectiveness. Documented previous SARS-CoV-2 infection offered some protection against hospitalisation and high protection against death in unvaccinated individuals, but only offered additional protection in vaccinated individuals for the death endpoint. Booster vaccination with mRNA vaccines maintains over 70% protection against hospitalisation and death in breakthrough confirmed omicron infections.FundingMedical Research Council, UK Research and Innovation, Department of Health and Social Care, National Institute for Health Research, Community Jameel, and Engineering and Physical Sciences Research Council.\", \"doi_y\": \"https://doi.org/10.1016/s0140-6736(22)00462-7\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W4296693414\", \"doi_x\": \"10.1038/s41586-022-05335-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A designed photoenzyme for enantioselective [2+2] cycloadditions\", \"abstract\": \"The ability to program new modes of catalysis into proteins would allow the development of enzyme families with functions beyond those found in nature. To this end, genetic code expansion methodology holds particular promise, as it allows the site-selective introduction of new functional elements into proteins as noncanonical amino acid side chains1-4. Here we exploit an expanded genetic code to develop a photoenzyme that operates by means of triplet energy transfer (EnT) catalysis, a versatile mode of reactivity in organic synthesis that is not accessible to biocatalysis at present5-12. Installation of a genetically encoded photosensitizer into the beta-propeller scaffold of DA_20_00 (ref. 13) converts a de novo Diels-Alderase into a photoenzyme for [2+2] cycloadditions (EnT1.0). Subsequent development and implementation of a platform for photoenzyme evolution afforded an efficient and enantioselective enzyme (EnT1.3, up to 99% enantiomeric excess (e.e.)) that can promote intramolecular and bimolecular cycloadditions, including transformations that have proved challenging to achieve selectively with small-molecule catalysts. EnT1.3 performs >300 turnovers and, in contrast to small-molecule photocatalysts, can operate effectively under aerobic conditions and at ambient temperatures. An X-ray crystal structure of an EnT1.3-product complex shows how multiple functional components work in synergy to promote efficient and selective photocatalysis. This study opens up a wealth of new excited-state chemistry in protein active sites and establishes the framework for developing a new generation of enantioselective photocatalysts.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-05335-3\", \"publication_date\": \"2022-09-21\"}, {\"work_id\": \"W4220882361\", \"doi_x\": \"10.1111/1751-7915.14034\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The fluorescent protein iLOV as a reporter for screening of high\\u2010yield production of antimicrobial peptides in <i>Pichia pastoris</i>\", \"abstract\": \"The methylotrophic yeast Pichia pastoris is commonly used for the production of recombinant proteins at scale. The identification of an optimally overexpressing strain following transformation can be time and reagent consuming. Fluorescent reporters like GFP have been used to assist identification of superior producers, but their relatively big size, maturation requirements and narrow temperature range restrict their applications. Here, we introduce the use of iLOV, a flavin-based fluorescent protein, as a fluorescent marker to identify P. pastoris high-yielding strains easily and rapidly. The use of this fluorescent protein as a fusion partner is exemplified by the production of the antimicrobial peptide NI01, a difficult target to overexpress in its native form. iLOV fluorescence correlated well with protein expression level and copy number of the chromosomally integrated gene. An easy and simple medium-throughput plate-based screen directly following transformation is demonstrated for low complexity screening, while a high-throughput method using fluorescence-activated cell sorting (FACS) allowed for comprehensive library screening. Both codon optimization of the iLOV_NI01 fusion cassettes and different integration strategies into the P. pastoris genome were tested to produce and isolate a high-yielding strain. Checking the genetic stability, process reproducibility and following the purification of the active native peptide are eased by visualization of and efficient cleavage from the iLOV reporter. We show that this system can be used for expression and screening of several different antimicrobial peptides recombinantly produced in P. pastoris.\", \"doi_y\": \"https://doi.org/10.1111/1751-7915.14034\", \"publication_date\": \"2022-03-21\"}, {\"work_id\": \"W4321099736\", \"doi_x\": \"10.1016/j.medj.2023.02.004\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evolution of long-term vaccine-induced and hybrid immunity in healthcare workers after different COVID-19 vaccine regimens\", \"abstract\": \"Both infection and vaccination, alone or in combination, generate antibody and T cell responses against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). However, the maintenance of such responses-and hence protection from disease-requires careful characterization. In a large prospective study of UK healthcare workers (HCWs) (Protective Immunity from T Cells in Healthcare Workers [PITCH], within the larger SARS-CoV-2 Immunity and Reinfection Evaluation [SIREN] study), we previously observed that prior infection strongly affected subsequent cellular and humoral immunity induced after long and short dosing intervals of BNT162b2 (Pfizer/BioNTech) vaccination.Here, we report longer follow-up of 684 HCWs in this cohort over 6-9 months following two doses of BNT162b2 or AZD1222 (Oxford/AstraZeneca) vaccination and up to 6 months following a subsequent mRNA booster vaccination.We make three observations: first, the dynamics of humoral and cellular responses differ; binding and neutralizing antibodies declined, whereas T and memory B cell responses were maintained after the second vaccine dose. Second, vaccine boosting restored immunoglobulin (Ig) G levels; broadened neutralizing activity against variants of concern, including Omicron BA.1, BA.2, and BA.5; and boosted T cell responses above the 6-month level after dose 2. Third, prior infection maintained its impact driving larger and broader T cell responses compared with never-infected people, a feature maintained until 6 months after the third dose.Broadly cross-reactive T cell responses are well maintained over time-especially in those with combined vaccine and infection-induced immunity (\\\"hybrid\\\" immunity)-and may contribute to continued protection against severe disease.Department for Health and Social Care, Medical Research Council.\", \"doi_y\": \"https://doi.org/10.1016/j.medj.2023.02.004\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4309809681\", \"doi_x\": \"10.1021/acs.analchem.2c03164\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"abstract\": \"Microfluidic water-in-oil emulsion droplets are becoming a mainstay of experimental biology, where they replace the classical test tube. In most applications, such as ultrahigh-throughput directed evolution, the droplet content is identical for all compartmentalized assay reactions. When emulsion droplets are used for kinetics or other functional assays, though, concentration dependencies of initial rates that define Michaelis\\u2013Menten parameters are required. Droplet-on-demand systems satisfy this need, but extracting large amounts of data is challenging. Here, we introduce a multiplexed droplet absorbance detector, which\\u2500coupled to semi-automated droplet generation\\u2500forms a tubing-based droplet-on-demand system able to generate and extract quantitative datasets from defined concentration gradients across multiple series of droplets for multiple time points. The emergence of a product is detected by reading the absorbance of the droplet sets at multiple, adjustable time points by reversing the flow direction after each detection, so that the droplets pass a line scan camera multiple times. Detection multiplexing allows absorbance values at 12 distinct positions to be measured, and enzyme kinetics are recorded for label-free concentration gradients that are composed of about 60 droplets each, covering as many concentrations. With a throughput of around 8640 data points per hour, a 10-fold improvement compared to the previously reported single point detection method is achieved. In a single experiment, 12 full datasets of high-resolution and high-accuracy Michaelis\\u2013Menten kinetics were determined to demonstrate the potential for enzyme characterization for glycosidase substrates covering a range in enzymatic hydrolysis of 7 orders of magnitude in kcat/KM. The straightforward setup, high throughput, excellent data quality, and wide dynamic range that allows coverage of diverse activities suggest that this system may serve as a miniaturized spectrophotometer for detailed analysis of clones emerging from large-scale combinatorial experiments.\", \"doi_y\": \"https://doi.org/10.1021/acs.analchem.2c03164\", \"publication_date\": \"2022-11-23\"}, {\"work_id\": \"W4285605412\", \"doi_x\": \"10.1016/j.envpol.2022.119757\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genetic sequencing detected the SARS-CoV-2 delta variant in wastewater a month prior to the first COVID-19 case in Ahmedabad (India)\", \"abstract\": \"Wastewater-based genomic surveillance can identify a huge majority of variants shed by the infected individuals within a population, which goes beyond genomic surveillance based on clinical samples (i.e., symptomatic patients only). We analyzed four samples to detect key mutations in the SARS-CoV-2 genome and track circulating variants in Ahmedabad during the first wave (Sep/Nov 2020) and before the second wave (in Feb 2021) of COVID-19 in India. The analysis identified a total of 34 mutations in the spike protein across samples categorized into 23 types. The spike protein mutations were linked to the VOC-21APR-02; B.1.617.2 lineage (Delta variant) with 57% frequency in wastewater samples of Feb 2021. The key spike protein mutations were T19R, L452R, T478K, D614G, & P681R and deletions at 22029 (6 bp), 28248 (6 bp), & 28271 (1 bp). Interestingly, these mutations were not seen in the samples from Sep/Nov 2020 but did appear before the massive second wave of COVID-19 cases, which in India started in early April 2021. In fact, genetic traces of the Delta variant were found in samples of early Feb 2021, more than a month before the first clinically confirmed case of this in March 2021 in Ahmedabad, Gujarat. The present work describes the circulating of SARS-CoV-2 variants in Ahmedabad and confirms the consequential value of wastewater surveillance for the early detection of variants of concerns (VOCs). Such monitoring must be included as a major component of future health protection systems.\", \"doi_y\": \"https://doi.org/10.1016/j.envpol.2022.119757\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4307843054\", \"doi_x\": \"10.1093/nar/gkac958\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Ensembl 2023\", \"abstract\": \"Abstract Ensembl (https://www.ensembl.org) has produced high-quality genomic resources for vertebrates and model organisms for more than twenty years. During that time, our resources, services and tools have continually evolved in line with both the publicly available genome data and the downstream research and applications that utilise the Ensembl platform. In recent years we have witnessed a dramatic shift in the genomic landscape. There has been a large increase in the number of high-quality reference genomes through global biodiversity initiatives. In parallel, there have been major advances towards pangenome representations of higher species, where many alternative genome assemblies representing different breeds, cultivars, strains and haplotypes are now available. In order to support these efforts and accelerate downstream research, it is our goal at Ensembl to create high-quality annotations, tools and services for species across the tree of life. Here, we report our resources for popular reference genomes, the dramatic growth of our annotations (including haplotypes from the first human pangenome graphs), updates to the Ensembl Variant Effect Predictor (VEP), interactive protein structure predictions from AlphaFold DB, and the beta release of our new website.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac958\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4362716961\", \"doi_x\": \"10.1002/adma.202300589\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Multi\\u2010Stimuli Responsive and Mechano\\u2010Actuated Biomimetic Membrane Nanopores Self\\u2010Assembled from DNA\", \"abstract\": \"Abstract In bioinspired design, biological templates are mimicked in structure and function by highly controllable synthetic means. Of interest are static barrel\\u2010like nanopores that enable molecular transport across membranes for use in biosensing, sequencing, and biotechnology. However, biological ion channels offer additional functions such as dynamic changes of the entire pore shape between open and closed states, and triggering of dynamic processes with biochemical and physical stimuli. To better capture this complexity, this report presents multi\\u2010stimuli and mechano\\u2010responsive biomimetic nanopores which are created with DNA nanotechnology. The nanopores switch between open and closed states, whereby specific binding of DNA and protein molecules as stimuli locks the pores in the open state. Furthermore, the physical stimulus of high transmembrane voltage switches the pores into a closed state. In addition, the pore diameters are larger and more tunable than those of natural templates. These multi\\u2010stimuli\\u2010responsive and mechanically actuated nanopores mimic several aspects of complex biological channels yet offer easier control over pore size, shape and stimulus response. The designer pores are expected to be applied in biosensing and synthetic biology.\", \"doi_y\": \"https://doi.org/10.1002/adma.202300589\", \"publication_date\": \"2023-06-02\"}, {\"work_id\": \"W4322741830\", \"doi_x\": \"10.3390/genes14030615\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Review of Recurrently Mutated Genes in Craniosynostosis Supports Expansion of Diagnostic Gene Panels\", \"abstract\": \"Craniosynostosis, the premature fusion of the cranial sutures, affects ~1 in 2000 children. Although many patients with a genetically determined cause harbor a variant in one of just seven genes or have a chromosomal abnormality, over 60 genes are known to be recurrently mutated, thus comprising a long tail of rarer diagnoses. Genome sequencing for the diagnosis of rare diseases is increasingly used in clinical settings, but analysis of the data is labor intensive and involves a trade-off between achieving high sensitivity or high precision. PanelApp, a crowd-sourced disease-focused set of gene panels, was designed to enable prioritization of variants in known disease genes for a given pathology, allowing enhanced identification of true-positives. For heterogeneous disorders like craniosynostosis, these panels must be regularly updated to ensure that diagnoses are not being missed. We provide a systematic review of genetic literature on craniosynostosis over the last 5 years, including additional results from resequencing a 42-gene panel in 617 affected individuals. We identify 16 genes (representing a 25% uplift) that should be added to the list of bona fide craniosynostosis disease genes and discuss the insights that these new genes provide into pathophysiological mechanisms of craniosynostosis.\", \"doi_y\": \"https://doi.org/10.3390/genes14030615\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4229026778\", \"doi_x\": \"10.3390/cells11091568\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Nutrient-Response Pathways in Healthspan and Lifespan Regulation\", \"abstract\": \"Cellular, small invertebrate and vertebrate models are a driving force in biogerontology studies. Using various models, such as yeasts, appropriate tissue culture cells, Drosophila, the nematode Caenorhabditis elegans and the mouse, has tremendously increased our knowledge around the relationship between diet, nutrient-response signaling pathways and lifespan regulation. In recent years, combinatorial drug treatments combined with mutagenesis, high-throughput screens, as well as multi-omics approaches, have provided unprecedented insights in cellular metabolism, development, differentiation, and aging. Scientists are, therefore, moving towards characterizing the fine architecture and cross-talks of growth and stress pathways towards identifying possible interventions that could lead to healthy aging and the amelioration of age-related diseases in humans. In this short review, we briefly examine recently uncovered knowledge around nutrient-response pathways, such as the Insulin Growth Factor (IGF) and the mechanistic Target of Rapamycin signaling pathways, as well as specific GWAS and some EWAS studies on lifespan and age-related disease that have enhanced our current understanding within the aging and biogerontology fields. We discuss what is learned from the rich and diverse generated data, as well as challenges and next frontiers in these scientific disciplines.\", \"doi_y\": \"https://doi.org/10.3390/cells11091568\", \"publication_date\": \"2022-05-06\"}, {\"work_id\": \"W4296802427\", \"doi_x\": \"10.1371/journal.pgen.1010414\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The helper NLR immune protein NRC3 mediates the hypersensitive cell death caused by the cell-surface receptor Cf-4\", \"abstract\": \"Cell surface pattern recognition receptors (PRRs) activate immune responses that can include the hypersensitive cell death. However, the pathways that link PRRs to the cell death response are poorly understood. Here, we show that the cell surface receptor-like protein Cf-4 requires the intracellular nucleotide-binding domain leucine-rich repeat containing receptor (NLR) NRC3 to trigger a confluent cell death response upon detection of the fungal effector Avr4 in leaves of Nicotiana benthamiana. This NRC3 activity requires an intact N-terminal MADA motif, a conserved signature of coiled-coil (CC)-type plant NLRs that is required for resistosome-mediated immune responses. A chimeric protein with the N-terminal \\u03b11 helix of Arabidopsis ZAR1 swapped into NRC3 retains the capacity to mediate Cf-4 hypersensitive cell death. Pathogen effectors acting as suppressors of NRC3 can suppress Cf-4-triggered hypersensitive cell-death. Our findings link the NLR resistosome model to the hypersensitive cell death caused by a cell surface PRR.\", \"doi_y\": \"https://doi.org/10.1371/journal.pgen.1010414\", \"publication_date\": \"2022-09-22\"}, {\"work_id\": \"W4315752260\", \"doi_x\": \"10.1002/adma.202210098\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Ion\\u2010Selective Microporous Polymer Membranes with Hydrogen\\u2010Bond and Salt\\u2010Bridge Networks for Aqueous Organic Redox Flow Batteries\", \"abstract\": \"Abstract Redox flow batteries (RFBs) have great potential for long\\u2010duration grid\\u2010scale energy storage. Ion\\u2010conducting membranes are a crucial component in RFBs, allowing charge\\u2010carrying ions to transport while preventing the cross\\u2010mixing of redox couples. Commercial Nafion membranes are widely used in RFBs, but their unsatisfactory ionic and molecular selectivity, as well as high costs, limit the performance and the widespread deployment of this technology. To extend the longevity and reduce the cost of RFB systems, inexpensive ion\\u2010selective membranes that concurrently deliver low ionic resistance and high selectivity toward redox\\u2010active species are highly desired. Here, high\\u2010performance RFB membranes are fabricated from blends of carboxylate\\u2010 and amidoxime\\u2010functionalized polymers of intrinsic microporosity, which exploit the beneficial properties of both polymers. The enthalpy\\u2010driven formation of cohesive interchain interactions, including hydrogen bonds and salt bridges, facilitates the microscopic miscibility of the blends, while ionizable functional groups within the sub\\u2010nanometer pores allow optimization of membrane ion\\u2010transport functions. The resulting microporous membranes demonstrate fast cation conduction with low crossover of redox\\u2010active molecular species, enabling improved power ratings and reduced capacity fade in aqueous RFBs using anthraquinone and ferrocyanide as redox couples.\", \"doi_y\": \"https://doi.org/10.1002/adma.202210098\", \"publication_date\": \"2023-02-05\"}, {\"work_id\": \"W4307770903\", \"doi_x\": \"10.1002/anie.202213053\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"abstract\": \"Abstract Abyssomicin C and its atropisomer are potent inhibitors of bacterial folate metabolism. They possess complex polycyclic structures, and their biosynthesis has been shown to involve several unusual enzymatic transformations. Using a combination of synthesis and in vitro assays we reveal that AbyV, a cytochrome P450 enzyme from the aby gene cluster, catalyses a key late\\u2010stage epoxidation required for the installation of the characteristic ether\\u2010bridged core of abyssomicin C. The X\\u2010ray crystal structure of AbyV has been determined, which in combination with molecular dynamics simulations provides a structural framework for our functional data. This work demonstrates the power of combining selective carbon\\u201013 labelling with NMR spectroscopy as a sensitive tool to interrogate enzyme\\u2010catalysed reactions in vitro with no need for purification.\", \"doi_y\": \"https://doi.org/10.1002/anie.202213053\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W3202009088\", \"doi_x\": \"10.1098/rsob.210222\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Intrinsically disordered proteins: modes of binding with emphasis on disordered domains\", \"abstract\": \"Our notions of protein function have long been determined by the protein structure-function paradigm. However, the idea that protein function is dictated by a prerequisite complementarity of shapes at the binding interface is becoming increasingly challenged. Interactions involving intrinsically disordered proteins (IDPs) have indicated a significant degree of disorder present in the bound state, ranging from static disorder to complete disorder, termed 'random fuzziness'. This review assesses the anatomy of an IDP and relates how its intrinsic properties permit promiscuity and allow for the various modes of interaction. Furthermore, a mechanistic overview of the types of disordered domains is detailed, while also relating to a recent example and the kinetic and thermodynamic principles governing its formation.\", \"doi_y\": \"https://doi.org/10.1098/rsob.210222\", \"publication_date\": \"2021-10-01\"}, {\"work_id\": \"W4223575014\", \"doi_x\": \"10.1016/j.jbc.2022.101932\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"abstract\": \"GPR84 is an immune cell\\u2013expressed, proinflammatory receptor currently being assessed as a therapeutic target in conditions including fibrosis and inflammatory bowel disease. Although it was previously shown that the orthosteric GPR84 activators 2-HTP and 6-OAU promoted its interactions with arrestin-3, a G protein\\u2013biased agonist DL-175 did not. Here, we show that replacement of all 21 serine and threonine residues within i-loop 3 of GPR84, but not the two serines in the C-terminal tail, eliminated the incorporation of [32P] and greatly reduced receptor\\u2013arrestin-3 interactions promoted by 2-HTP. GPR84 was phosphorylated constitutively on residues Ser221 and Ser224, while various other amino acids are phosphorylated in response to 2-HTP. Consistent with this, an antiserum able to identify pSer221/pSer224 recognized GPR84 from cells treated with and without activators, whereas an antiserum able to identify pThr263/pThr264 only recognized GPR84 after exposure to 2-HTP and not DL-175. Two distinct GPR84 antagonists as well as inhibition of G protein\\u2013coupled receptor kinase 2/3 prevented phosphorylation of pThr263/pThr264, but neither strategy affected constitutive phosphorylation of Ser221/Ser224. Furthermore, mutation of residues Thr263 and Thr264 to alanine generated a variant of GPR84 also limited in 2-HTP\\u2013induced interactions with arrestin-2 and -3. By contrast, this mutant was unaffected in its capacity to reduce cAMP levels. Taken together, these results define a key pair of threonine residues, regulated only by subsets of GPR84 small molecule activators and by GRK2/3 that define effective interactions with arrestins and provide novel tools to monitor the phosphorylation and functional status of GPR84. GPR84 is an immune cell\\u2013expressed, proinflammatory receptor currently being assessed as a therapeutic target in conditions including fibrosis and inflammatory bowel disease. Although it was previously shown that the orthosteric GPR84 activators 2-HTP and 6-OAU promoted its interactions with arrestin-3, a G protein\\u2013biased agonist DL-175 did not. Here, we show that replacement of all 21 serine and threonine residues within i-loop 3 of GPR84, but not the two serines in the C-terminal tail, eliminated the incorporation of [32P] and greatly reduced receptor\\u2013arrestin-3 interactions promoted by 2-HTP. GPR84 was phosphorylated constitutively on residues Ser221 and Ser224, while various other amino acids are phosphorylated in response to 2-HTP. Consistent with this, an antiserum able to identify pSer221/pSer224 recognized GPR84 from cells treated with and without activators, whereas an antiserum able to identify pThr263/pThr264 only recognized GPR84 after exposure to 2-HTP and not DL-175. Two distinct GPR84 antagonists as well as inhibition of G protein\\u2013coupled receptor kinase 2/3 prevented phosphorylation of pThr263/pThr264, but neither strategy affected constitutive phosphorylation of Ser221/Ser224. Furthermore, mutation of residues Thr263 and Thr264 to alanine generated a variant of GPR84 also limited in 2-HTP\\u2013induced interactions with arrestin-2 and -3. By contrast, this mutant was unaffected in its capacity to reduce cAMP levels. Taken together, these results define a key pair of threonine residues, regulated only by subsets of GPR84 small molecule activators and by GRK2/3 that define effective interactions with arrestins and provide novel tools to monitor the phosphorylation and functional status of GPR84. GPR84 is a nominally orphan G protein\\u2013coupled receptor (GPCR) that can be activated by both medium-chain fatty acids (MCFAs) and a range of synthetic ligands that act as either orthosteric or allosteric activators (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 2Wojciechowicz M.L. Ma'ayan A. GPR84: An immune response dial?.Nat. Rev. Drug Discov. 2020; 19: 374Crossref PubMed Scopus (11) Google Scholar, 3Chen L.H. Zhang Q. Xie X. Nan F.J. Modulation of the G-protein-coupled receptor 84 (GPR84) by agonists and antagonists.J. Med. Chem. 2020; 63: 15399-15409Crossref PubMed Scopus (6) Google Scholar). The pattern of expression across a variety of both peripheral and central immune cells (4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar, 5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar, 7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar), and that mRNA and protein corresponding to the receptor are strongly upregulated in response to proinflammatory challenges (6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar, 7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar), has promoted interest in GPR84 as a therapeutic target in a range of areas (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 2Wojciechowicz M.L. Ma'ayan A. GPR84: An immune response dial?.Nat. Rev. Drug Discov. 2020; 19: 374Crossref PubMed Scopus (11) Google Scholar). These include both ulcerative colitis and idiopathic pulmonary fibrosis (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar). Although focus for both these indications has been on the development and assessment of antagonist ligands (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 3Chen L.H. Zhang Q. Xie X. Nan F.J. Modulation of the G-protein-coupled receptor 84 (GPR84) by agonists and antagonists.J. Med. Chem. 2020; 63: 15399-15409Crossref PubMed Scopus (6) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar), it has been suggested that agonists of GPR84 might also find use in, for example, the treatment of atherosclerosis (9Gaidarov I. Anthony T. Gatlin J. Chen X. Mills D. Solomon M. Han S. Semple G. Unett D.J. Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor.Pharmacol. Res. 2018; 131: 185-198Crossref PubMed Scopus (23) Google Scholar). No matter the disease area, understanding of the extent of upregulation of the receptor in disease settings and the degree of activation of the receptor in such situations is vital. Progress in the production and characterization of a [3H]radiolabeled agonist (10K\\u00f6se M. Pillaiyar T. Namasivayam V. De Filippo E. Sylvester K. Ulven T. von K\\u00fcgelgen I. M\\u00fcller C.E. An agonist radioligand for the proinflammatory lipid-activated G protein-coupled receptor GPR84 providing structural insights.J. Med. Chem. 2020; 63: 2391-2410Crossref PubMed Scopus (10) Google Scholar) and [3H]radiolabeled antagonists (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar, 11Jenkins L. Marsango S. Mancini S. Mahmud Z.A. Morrison A. McElroy S.P. Bennett K.A. Barnes M. Tobin A.B. Tikhonova I.G. Milligan G. Discovery and characterization of novel antagonists of the proinflammatory orphan receptor GPR84.ACS Pharmacol. Transl. Sci. 2021; 4: 1598-1613Crossref PubMed Scopus (1) Google Scholar) can potentially provide insights into the former of these questions, but other reagents and biosensors will be needed to probe the degree of receptor activation. A broadly applicable feature of GPCRs following agonist occupancy is that phosphorylation of various hydroxy-amino acids within intracellular sequences of the receptor is stimulated and this promotes the effectiveness of interactions of the receptor with arrestin isoforms (12Wanka L. Behr V. Beck-Sickinger A.G. Arrestin-dependent internalization of rhodopsin-like G protein-coupled receptors.Biol. Chem. 2021; 403: 133-149Crossref PubMed Scopus (0) Google Scholar, 13Sulon S.M. Benovic J.L. Targeting G protein-coupled receptor kinases (GRKs) to G protein-coupled receptors.Curr. Opin. Endocr. Metab. Res. 2021; 16: 56-65Crossref PubMed Scopus (2) Google Scholar). This can limit the ability of the receptor to interact with G proteins and hence restrict signaling. It can also potentially promote other distinct signaling cascades and generally promotes movement of the receptor away from the cell surface and into intracellular locations. The patterns and locations of the amino acids that become phosphorylated in an agonist-dependent manner have been extensively mapped for a range of GPCRs (14Mann A. Illing S. Miess E. Schulz S. Different mechanisms of homologous and heterologous mu-opioid receptor phosphorylation.Br. J. Pharmacol. 2015; 172: 311-316Crossref PubMed Scopus (30) Google Scholar, 15Kliewer A. Reinscheid R.K. Schulz S. Emerging paradigms of G protein-coupled receptor dephosphorylation.Trends Pharmacol. Sci. 2017; 38: 621-636Abstract Full Text Full Text PDF PubMed Scopus (14) Google Scholar, 16Kliewer A. Schmiedel F. Sianati S. Bailey A. Bateman J.T. Levitt E.S. Williams J.T. Christie M.J. Schulz S. Phosphorylation-deficient G-protein-biased \\u03bc-opioid receptors improve analgesia and diminish tolerance but worsen opioid side effects.Nat. Commun. 2019; 10: 367Crossref PubMed Scopus (123) Google Scholar, 17Matthees E.S.F. Haider R.S. Hoffmann C. Drube J. differential regulation of GPCRs-are GRK expression levels the key?.Front. Cell Dev. Biol. 2021; 9: 687489Crossref PubMed Scopus (5) Google Scholar, 18Seyedabadi M. Gharghabi M. Gurevich E.V. Gurevich V.V. Receptor-arrestin interactions: The GPCR perspective.Biomolecules. 2021; 11: 218Crossref PubMed Scopus (15) Google Scholar), and both mass spectrometry and mutagenesis have contributed to this effort. Information taken from such studies can then be used to generate antisera able to identify specifically individual phosphorylated amino acids or groups of such amino acids (14Mann A. Illing S. Miess E. Schulz S. Different mechanisms of homologous and heterologous mu-opioid receptor phosphorylation.Br. J. Pharmacol. 2015; 172: 311-316Crossref PubMed Scopus (30) Google Scholar, 15Kliewer A. Reinscheid R.K. Schulz S. Emerging paradigms of G protein-coupled receptor dephosphorylation.Trends Pharmacol. Sci. 2017; 38: 621-636Abstract Full Text Full Text PDF PubMed Scopus (14) Google Scholar, 16Kliewer A. Schmiedel F. Sianati S. Bailey A. Bateman J.T. Levitt E.S. Williams J.T. Christie M.J. Schulz S. Phosphorylation-deficient G-protein-biased \\u03bc-opioid receptors improve analgesia and diminish tolerance but worsen opioid side effects.Nat. Commun. 2019; 10: 367Crossref PubMed Scopus (123) Google Scholar, 19Butcher A.J. Hudson B.D. Shimpukade B. Alvarez-Curto E. Prihandoko R. Ulven T. Milligan G. Tobin A.B. Concomitant action of structural elements and receptor phosphorylation determines arrestin-3 interaction with the free fatty acid receptor FFA4.J. Biol. Chem. 2014; 289: 18451-18465Abstract Full Text Full Text PDF PubMed Scopus (48) Google Scholar, 20Butcher A.J. Bradley S.J. Prihandoko R. Brooke S.M. Mogg A. Bourgognon J.M. Macedo-Hatch T. Edwards J.M. Bottrill A.R. Challiss R.A. Broad L.M. Felder C.C. Tobin A.B. An antibody biosensor establishes the activation of the M1 muscarinic acetylcholine receptor during learning and memory.J. Biol. Chem. 2016; 291: 8862-8875Abstract Full Text Full Text PDF PubMed Scopus (19) Google Scholar). These can then potentially be used as surrogate reagents to detect and demonstrate the extent of receptor activation in situ. Among ligands that activate GPR84, the recently described molecule 3-(2-((4-chloronaphthalen-1-yl)oxy)ethyl)pyridine 1-oxide (DL-175) is reported to be functionally \\u201cbiased,\\u201d in that, although a potent regulator of G protein\\u2013mediated signaling, it is not effective in promoting arrestin interactions (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar) and has distinct characteristics in signaling compared with the agonist 6-n-octylaminouracil (6-OAU) (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar). Because we anticipated that DL-175 and 6-OAU might well produce a different pattern or extent of phosphorylation of GPR84, to obtain a fuller understanding of such potential variation caused by GPR84 activators, herein in addition we have assessed both 2-(hexylthio)pyrimidine-4,6 diol (2-HTP) as a potent orthosteric activator (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar) and the allosteric activator PSB-16771 (22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar). This is a more potent analogue of the originally defined allosteric activator of GPR84, 3,3\\u2032-diindolylmethane (23Takeda S. Yamamoto A. Okada T. Matsumura E. Nose E. Kogure K. Kojima S. Haga T. Identification of surrogate ligands for orphan G protein-coupled receptors.Life Sci. 2003; 74: 367-377Crossref PubMed Scopus (49) Google Scholar). We show that, although both 2-HTP and 6-OAU produce effective phosphorylation of human GPR84 at residues within the third intracellular loop, neither DL-175 nor PSB-16771 does so. Moreover, although each of 2-HTP, 6-OAU, and DL-175 acts as orthosteric agonists of GPR84, we use homology modeling and mutagenesis to provide a molecular basis for their differences in function and bias. In addition, based on mass spectrometry, we identify residues in the third intracellular loop of GPR84 that are phosphorylated constitutively. Production of antisera that identify specific sites of both constitutive and agonist-regulated phosphorylation allowed us to demonstrate that phosphorylation at amino acids Thr263 and/or Thr264 is required to generate highly effective interactions with arrestin-2 and arrestin-3 and that these amino acids become phosphorylated in a GRK2/3-dependent manner. By contrast, neither Ser221 nor Ser224, which are constitutively phosphorylated and not affected by inhibition of GRK2/3, plays a major role in arrestin-2 or arrestin-3-interactions. However, the phosphorylation status of neither of these sets of amino acids impacts on the ability of the receptor to regulate cellular levels of cAMP. Of importance, as GPR84 is markedly upregulated in a wide range of immune cells and in a variety of proinflammatory settings (4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar, 5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar) the activation status detection of GPR84 by the agonist-regulated antiserum may provide an important biosensor in model systems of GPR84 regulation and function. To study mechanisms of signal transduction induced by activation of the proinflammatory GPCR GPR84 we selected a pair of orthosteric agonists, 2-HTP (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar) (also designated \\u201ccompound 1\\u201d (24Mahmud Z.A. Jenkins L. Ulven T. Lab\\u00e9gu\\u00e8re F. Gosmini R. De Vos S. Hudson B.D. Tikhonova I.G. Milligan G. Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84.Sci. Rep. 2017; 7: 17953Crossref PubMed Scopus (21) Google Scholar), or ZQ-16 (25Liu Y. Zhang Q. Chen L.H. Yang H. Lu W. Xie X. Nan F.J. Design and synthesis of 2-alkylpyrimidine-4,6-diol and 6-alkylpyridine-2,4-diol as potent GPR84 agonists.ACS Med. Chem. Lett. 2016; 7: 579-583Crossref PubMed Scopus (28) Google Scholar)) and 6-OAU (5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 26Pillaiyar T. K\\u00f6se M. Namasivayam V. Sylvester K. Borges G. Thimm D. von K\\u00fcgelgen I. M\\u00fcller C.E. 6-(Ar)Alkylamino-Substituted uracil derivatives: Lipid mimetics with potent activity at the orphan G protein-coupled receptor 84 (GPR84).ACS Omega. 2018; 3: 3365-3383Crossref PubMed Scopus (15) Google Scholar), as well as the allosteric activator PSB-16671 (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar) (Fig. 1A). In addition, we selected DL-175, a compound optimized following a virtual screen and reported to display marked G protein signaling compared with arrestin-interaction bias at GPR84 (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar) (Fig. 1A). To initiate studies, we expressed stably in Flp-In TREx 293 cells human GPR84 with enhanced Yellow Fluorescent Protein linked in-frame to the intracellular C-terminal tail of the receptor (hGPR84-eYFP) (24Mahmud Z.A. Jenkins L. Ulven T. Lab\\u00e9gu\\u00e8re F. Gosmini R. De Vos S. Hudson B.D. Tikhonova I.G. Milligan G. Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84.Sci. Rep. 2017; 7: 17953Crossref PubMed Scopus (21) Google Scholar). This cell system allows doxycycline-induced expression of constructs located at the Flp-In TREx locus (27Ward R.J. Alvarez-Curto E. Milligan G. Using the Flp-In\\u2122 T-Rex\\u2122 system to regulate GPCR expression.Methods Mol. Biol. 2011; 746: 21-37Crossref PubMed Scopus (41) Google Scholar). GPR84 is known to be able to interact with pertussis toxin\\u2013sensitive members of the Gi-G protein family (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar). In membranes prepared from such doxycycline-induced cells the ability of each ligand to cause activation of Gi-G proteins was measured via [35S]GTP\\u03b3S binding assays. Each compound promoted an increase in binding of [35S]GTP\\u03b3S in a concentration-dependent fashion with rank order 2-HTP > PSB-16671 > 6-OAU = DL-175 (Fig. 1B), and each ligand displayed similar efficacy at maximally effective concentrations (Fig. 1B). Activation of GPR84 with certain, but not all, ligands is reported to promote interaction with arrestin isoforms (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar, 28M\\u00e5rtensson J. Sundqvist M. Manandhar A. Leremias L. Zhang L. Ulven T. Xie X. Bj\\u00f6rkman L. Forsman H. The two formyl peptide receptors differently regulate GPR84-mediated neutrophil NADPH oxidase activity.J. Innate Immun. 2021; 31: 1-15Google Scholar). When hGPR84-eYFP was transiently coexpressed in human embryonic kidney (HEK) 293T cells with arrestin-3 fused to nanoluciferase, 2-HTP effectively promoted proximity and potential interactions between the receptor and this arrestin construct (Fig. 1C). 6-OAU was also able to promote interactions between hGPR84-eYFP and arrestin-3-nanoluciferase. Once more 2-HTP was substantially more potent than 6-OAU, although it was not possible to add 6-OAU at sufficiently high concentrations to allow full concentration\\u2013response curves to be defined (Fig. 1C) because both compounds displayed substantially lower potency than in the [35S]GTP\\u03b3S binding studies (compare Fig. 1, B and C). In contrast, DL-175 did not promote measurable interactions between hGPR84-eYFP and arrestin-3-nanoluciferase at any concentration that was practical to assess (Fig. 1C) and neither did PSB-16671 (Fig. 1C). However, as anticipated for a positive allosteric modulator at GPR84 (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar) PSB-16671 substantially increased (p < 0.0001) the potency of 2-HTP in such arrestin-3 interaction studies (Fig. 1D), and this was also the case for 6-OAU (p < 0.01) (Fig. 1E). By contrast, coaddition of DL-175 did not increase the observed potency of 2-HTP (Fig. 1D) or 6-OAU (Fig. 1E). Interaction of a GPCR with an arrestin is often dependent upon phosphorylation of the receptor that occurs subsequent to agonist occupancy of the receptor (12Wanka L. Behr V. Beck-Sickinger A.G. Arrestin-dependent internalization of rhodopsin-like G protein-coupled receptors.Biol. Chem. 2021; 403: 133-149Crossref PubMed Scopus (0) Google Scholar). To examine this directly we labeled Flp-In TREx 293 cells induced to express hGPR84-eYFP with [32P] orthophosphate and then treated the cells with vehicle or a concentration of 2-HTP (3.7 \\u00d7 10-6 M) determined to produce an EC90 effect in the arrestin-3 interaction BRET assay. Lysates of these cells were immunoprecipitated with an anti-Green Fluorescent Protein (GFP) antiserum (that also identifies eYFP) and then resolved by SDS-PAGE. After drying, such gels were subsequently exposed to X-ray film. No substantial incorporation of [32P] was detected in vehicle-treated samples. However, clear incorporation of [32P] into polypeptide(s) with molecular mass corresponding to some 80 kDa was observed following treatment with 2-HTP, with close to maximal incorporation of [32P] achieved within 5 min of agonist exposure (Fig. 2A). 6-OAU (6.4 \\u00d7 10\\u22125 M) also effectively promoted incorporation of [32P] into hGPR84-eYFP (Fig. 2B). Because neither DL-175 nor PSB-16671 produced effective interactions of hGPR84-eYFP with arrestin-3, and therefore no EC90 concentration could be defined, for [32P]-phosphorylation studies we employed concentrations of these ligands that were 100 times greater than their EC90 concentrations in [35S]GTP\\u03b3S binding studies on the basis that, as shown earlier, 2-HTP and 6-OAU were to this extent less potent in the arrestin-3 interaction studies than in [35S]GTP\\u03b3S binding studies. However, even at these concentrations, no significant incorporation of [32P] into hGPR84-eYFP was observed in samples treated with either DL-175 (9.8 \\u00d7 10\\u22125 M) or PSB-16671 (1.3 \\u00d7 10\\u22125 M) (Fig. 2B). As expected, the ability of 2-HTP to promote phosphorylation of hGPR84-eYFP was prevented by coincubation with either of two chemically distinct GPR84 antagonists, GLPG1205 (8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar) and compound 837 (11Jenkins L. Marsango S. Mancini S. Mahmud Z.A. Morrison A. McElroy S.P. Bennett K.A. Barnes M. Tobin A.B. Tikhonova I.G. Milligan G. Discovery and characterization of novel antagonists of the proinflammatory orphan receptor GPR84.ACS Pharmacol. Transl. Sci. 2021; 4: 1598-1613Crossref PubMed Scopus (1) Google Scholar) (each at 1 \\u00d7 10\\u22125 M) (Fig. 2C). Because agonist-mediated phosphorylation of many GPCRs reflects interactions with members of the GRK family we assessed the effect of the GRK2/3 inhibitor compound 101 (1 \\u00d7 10\\u22125 M) (29Thal D.M. Yeow R.Y. Schoenau C. Huber J. Tesmer J.J. Molecular mechanism of selectivity among G protein-coupled receptor kinase 2 inhibitors.Mol. Pharmacol. 2011; 80: 294-303Crossref PubMed Scopus (81) Google Scholar, 30Lowe J.D. Sanderson H.S. Cooke A.E. Ostovar M. Tsisanova E. Withey S.L. Chavkin C. Husbands S.M. Kelly E. Henderson G. Bailey C.P. Role of G Protein-Coupled receptor kinases 2 and 3 in \\u03bc-opioid receptor desensitization and internalization.Mol. Pharmacol. 2015; 88: 347-356Crossref PubMed Scopus (48) Google Scholar). [32P] phosphorylation of hGPR84-eYFP in respons\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.101932\", \"publication_date\": \"2022-05-01\"}, {\"work_id\": \"W4293211899\", \"doi_x\": \"10.15252/embr.202154322\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"abstract\": \"The emergence of SARS-CoV-2 variants has exacerbated the COVID-19 global health crisis. Thus far, all variants carry mutations in the spike glycoprotein, which is a critical determinant of viral transmission being responsible for attachment, receptor engagement and membrane fusion, and an important target of immunity. Variants frequently bear truncations of flexible loops in the N-terminal domain (NTD) of spike; the functional importance of these modifications has remained poorly characterised. We demonstrate that NTD deletions are important for efficient entry by the Alpha and Omicron variants and that this correlates with spike stability. Phylogenetic analysis reveals extensive NTD loop length polymorphisms across the sarbecoviruses, setting an evolutionary precedent for loop remodelling. Guided by these analyses, we demonstrate that variations in NTD loop length, alone, are sufficient to modulate virus entry. We propose that variations in NTD loop length act to fine-tune spike; this may provide a mechanism for SARS-CoV-2 to navigate a complex selection landscape encompassing optimisation of essential functionality, immune-driven antigenic variation and ongoing adaptation to a new host.\", \"doi_y\": \"https://doi.org/10.15252/embr.202154322\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4283821860\", \"doi_x\": \"10.1038/s41591-022-01883-3\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"abstract\": \"Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management.\", \"doi_y\": \"https://doi.org/10.1038/s41591-022-01883-3\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4386769118\", \"doi_x\": \"10.1103/physrevb.108.115134\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation energy of the spin-polarized electron liquid studied using quantum Monte Carlo simulations\", \"abstract\": \"Variational and diffusion quantum Monte Carlo (VMC and DMC) methods with Slater-Jastrow-backflow trial wave functions are used to study the spin-polarized three-dimensional uniform electron fluid. We report ground state VMC and DMC energies in the density range $0.5\\\\ensuremath{\\\\le}{r}_{\\\\text{s}}\\\\ensuremath{\\\\le}20$. Finite-size errors are corrected using canonical-ensemble twist-averaged boundary conditions and extrapolation of the twist-averaged energy per particle calculated at three system sizes $(N=113, 259, \\\\mathrm{and} 387)$ to the thermodynamic limit of infinite system size. The DMC energies in the thermodynamic limit are used to parametrize a local spin density approximation correlation function for inhomogeneous electron systems.\", \"doi_y\": \"https://doi.org/10.1103/physrevb.108.115134\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4323656741\", \"doi_x\": \"10.1073/pnas.2220921120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Broader functions of TIR domains in Arabidopsis immunity\", \"abstract\": \"TIR domains are NAD-degrading enzymes that function during immune signaling in prokaryotes, plants, and animals. In plants, most TIR domains are incorporated into intracellular immune receptors termed TNLs. In Arabidopsis, TIR-derived small molecules bind and activate EDS1 heterodimers, which in turn activate RNLs, a class of cation channel\\u2013forming immune receptors. RNL activation drives cytoplasmic Ca 2+ influx, transcriptional reprogramming, pathogen resistance, and host cell death. We screened for mutants that suppress an RNL activation mimic allele and identified a TNL, SADR1. Despite being required for the function of an autoactivated RNL, SADR1 is not required for defense signaling triggered by other tested TNLs. SADR1 is required for defense signaling initiated by some transmembrane pattern recognition receptors and contributes to the unbridled spread of cell death in lesion simulating disease 1 . Together with RNLs, SADR1 regulates defense gene expression at infection site borders, likely in a non-cell autonomous manner. RNL mutants that cannot sustain this pattern of gene expression are unable to prevent disease spread beyond localized infection sites, suggesting that this pattern corresponds to a pathogen containment mechanism. SADR1 potentiates RNL-driven immune signaling not only through the activation of EDS1 but also partially independently of EDS1. We studied EDS1-independent TIR function using nicotinamide, an NADase inhibitor. Nicotinamide decreased defense induction from transmembrane pattern recognition receptors and decreased calcium influx, pathogen growth restriction, and host cell death following intracellular immune receptor activation. We demonstrate that TIR domains can potentiate calcium influx and defense and are thus broadly required for Arabidopsis immunity.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2220921120\", \"publication_date\": \"2023-03-09\"}, {\"work_id\": \"W4313424363\", \"doi_x\": \"10.15252/embj.2022111484\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"abstract\": \"Plant pathogens compromise crop yields. Plants have evolved robust innate immunity that depends in part on intracellular Nucleotide-binding, Leucine rich-Repeat (NLR) immune receptors that activate defense responses upon detection of pathogen-derived effectors. Most \\\"sensor\\\" NLRs that detect effectors require the activity of \\\"helper\\\" NLRs, but how helper NLRs support sensor NLR function is poorly understood. Many Solanaceae NLRs require NRC (NLR-Required for Cell death) class of helper NLRs. We show here that Rpi-amr3, a sensor NLR from Solanum americanum, detects AVRamr3 from the potato late blight pathogen, Phytophthora infestans, and activates oligomerization of helper NLRs NRC2 and NRC4 into high-molecular-weight resistosomes. In contrast, recognition of P. infestans effector AVRamr1 by another sensor NLR Rpi-amr1 induces formation of only the NRC2 resistosome. The activated NRC2 oligomer becomes enriched in membrane fractions. ATP-binding motifs of both Rpi-amr3 and NRC2 are required for NRC2 resistosome formation, but not for the interaction of Rpi-amr3 with its cognate effector. NRC2 resistosome can be activated by Rpi-amr3 upon detection of AVRamr3 homologs from other Phytophthora species. Mechanistic understanding of NRC resistosome formation will underpin engineering crops with durable disease resistance.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111484\", \"publication_date\": \"2023-01-02\"}, {\"work_id\": \"W4285803545\", \"doi_x\": \"10.1101/2022.07.17.500290\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The snapdragon genomes reveal the evolutionary dynamics of the<i>S</i>locus supergene\", \"abstract\": \"Abstract The multi-allelic S -locus, containing a pistil S-RNase and dozens of S -locus F-box ( SLF ), underlies genetic control of self-incompatibility (SI) in Antirrhinum hispanicum . The genus Antirrhinum , harboring such a SI system has been used as a model to study self-incompatibility extensively. However, there have been limited studies on the genomic organization of the S -locus supergene due to a lack of high-quality genomic data. Here, we present the chromosome-level reference and haplotype-resolved genome assemblies of a self-incompatible Antirrhinum hispanicum line, AhS 7 S 8 . Alongside with the draft genome of Misopates orontium , comparative genomics reveals that A.hispanicum diverged from its self-compatible cousin 12.3 million years ago (Mya). Expanded gene families enriched in distinct functional terms implied different evolutionary trajectories of outcrossing and selfing species. For the first time, two complete A.hispanicum S -haplotypes spanning \\u223c1.2Mb and containing 32 SLFs were reconstructed, while most of the SLFs derived from retroelement-mediated proximal or tandem duplication approximately 122 Mya. Moreover, we detected a candidate cis -transcription factor associated with regulating SLF s expression, and two miRNAs may control the expression of this transcription factor. Inter-specific S -locus and intra-specific S -haplotype comparisons revealed the dynamic nature and polymorphism of the S -locus supergene mediated by continuous gene duplication, segmental translocation or loss, and TE-involved transposition events. Our data provides an excellent resource for future research on the evolutionary studies on S-RNase -based self-incompatibility system.\", \"doi_y\": \"https://doi.org/10.1101/2022.07.17.500290\", \"publication_date\": \"2022-07-18\"}, {\"work_id\": \"W4280617578\", \"doi_x\": \"10.1016/j.cell.2022.05.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Potent cross-reactive antibodies following Omicron breakthrough in vaccinees\", \"abstract\": \"Highly transmissible Omicron variants of SARS-CoV-2 currently dominate globally. Here, we compare neutralization of Omicron BA.1, BA.1.1, and BA.2. BA.2 RBD has slightly higher ACE2 affinity than BA.1 and slightly reduced neutralization by vaccine serum, possibly associated with its increased transmissibility. Neutralization differences between sub-lineages for mAbs (including therapeutics) mostly arise from variation in residues bordering the ACE2 binding site; however, more distant mutations S371F (BA.2) and R346K (BA.1.1) markedly reduce neutralization by therapeutic antibody Vir-S309. In-depth structure-and-function analyses of 27 potent RBD-binding mAbs isolated from vaccinated volunteers following breakthrough Omicron-BA.1 infection reveals that they are focused in two main clusters within the RBD, with potent right-shoulder antibodies showing increased prevalence. Selection and somatic maturation have optimized antibody potency in less-mutated epitopes and recovered potency in highly mutated epitopes. All 27 mAbs potently neutralize early pandemic strains, and many show broad reactivity with variants of concern.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.05.014\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4289528710\", \"doi_x\": \"10.1016/j.celrep.2022.111220\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 spike N-terminal domain modulates TMPRSS2-dependent viral entry and fusogenicity\", \"abstract\": \"The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike N-terminal domain (NTD) remains poorly characterized despite enrichment of mutations in this region across variants of concern (VOCs). Here, we examine the contribution of the NTD to infection and cell-cell fusion by constructing chimeric spikes bearing B.1.617 lineage (Delta and Kappa variants) NTDs and generating spike pseudotyped lentivirus. We find that the Delta NTD on a Kappa or wild-type (WT) background increases S1/S2 cleavage efficiency and virus entry, specifically in lung cells and airway organoids, through use of TMPRSS2. Delta exhibits increased cell-cell fusogenicity that could be conferred to WT and Kappa spikes by Delta NTD transfer. However, chimeras of Omicron BA.1 and BA.2 spikes with a Delta NTD do not show more efficient TMPRSS2 use or fusogenicity. We conclude that the NTD allosterically modulates S1/S2 cleavage and spike-mediated functions in a spike context-dependent manner, and allosteric interactions may be lost when combining regions from more distantly related VOCs.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111220\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4385067096\", \"doi_x\": \"10.1021/acscatal.3c01609\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Versatile Product Detection via Coupled Assays for Ultrahigh-Throughput Screening of Carbohydrate-Active Enzymes in Microfluidic Droplets\", \"abstract\": \"Enzyme discovery and directed evolution are the two major contemporary approaches for the improvement of industrial processes by biocatalysis in various fields. Customization of catalysts for improvement of single enzyme reactions or de novo reaction development is often complex and tedious. The success of screening campaigns relies on the fraction of sequence space that can be sampled, whether for evolving a particular enzyme or screening metagenomes. Ultrahigh-throughput screening (uHTS) based on in vitro compartmentalization in water-in-oil emulsion of picoliter droplets generated in microfluidic systems allows screening rates >1 kHz (or >107 per day). Screening for carbohydrate-active enzymes (CAZymes) catalyzing biotechnologically valuable reactions in this format presents an additional challenge because the released carbohydrates are difficult to monitor in high throughput. Activated substrates with large optically active hydrophobic leaving groups provide a generic optical readout, but the molecular recognition properties of sugars will be altered by the incorporation of such fluoro- or chromophores and their typically higher reactivity, as leaving groups with lowered pKa values compared to native substrates make the observation of promiscuous reactions more likely. To overcome these issues, we designed microdroplet assays in which optically inactive carbohydrate products are made visible by specific cascades: the primary reaction of an unlabeled substrate leads to an optical signal downstream. Successfully implementing such assays at the picoliter droplet scale allowed us to detect glucose, xylose, glucuronic acid, and arabinose as final products of complex oligosaccharide degradation by glycoside hydrolases by absorbance measurements. Enabling the use of uHTS for screening CAZyme reactions that have been thus far elusive will chart a route toward faster and easier development of specific and efficient biocatalysts for biovalorization, directing enzyme discovery by challenging catalysts for reaction with natural rather than model substrates.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c01609\", \"publication_date\": \"2023-07-21\"}, {\"work_id\": \"W4321076815\", \"doi_x\": \"10.1021/acscentsci.2c01349\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"O-Linked Sialoglycans Modulate the Proteolysis of SARS-CoV-2 Spike and Likely Contribute to the Mutational Trajectory in Variants of Concern\", \"abstract\": \"The emergence of a polybasic cleavage motif for the protease furin in SARS-CoV-2 spike has been established as a major factor for human viral transmission. The region N-terminal to that motif is extensively mutated in variants of concern (VOCs). Besides furin, spikes from these variants appear to rely on other proteases for maturation, including TMPRSS2. Glycans near the cleavage site have raised questions about proteolytic processing and the consequences of variant-borne mutations. Here, we identify that sialic acid-containing O-linked glycans on Thr678 of SARS-CoV-2 spike influence furin and TMPRSS2 cleavage and posit O-linked glycosylation as a likely driving force for the emergence of VOC mutations. We provide direct evidence that the glycosyltransferase GalNAc-T1 primes glycosylation at Thr678 in the living cell, an event that is suppressed by mutations in the VOCs Alpha, Delta, and Omicron. We found that the sole incorporation of N-acetylgalactosamine did not impact furin activity in synthetic O-glycopeptides, but the presence of sialic acid reduced the furin rate by up to 65%. Similarly, O-glycosylation with a sialylated trisaccharide had a negative impact on TMPRSS2 cleavage. With a chemistry-centered approach, we substantiate O-glycosylation as a major determinant of spike maturation and propose disruption of O-glycosylation as a substantial driving force for VOC evolution.\", \"doi_y\": \"https://doi.org/10.1021/acscentsci.2c01349\", \"publication_date\": \"2023-02-16\"}, {\"work_id\": \"W4323036511\", \"doi_x\": \"10.1093/genetics/iyad031\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The Gene Ontology knowledgebase in 2023\", \"abstract\": \"The Gene Ontology (GO) knowledgebase (http://geneontology.org) is a comprehensive resource concerning the functions of genes and gene products (proteins and noncoding RNAs). GO annotations cover genes from organisms across the tree of life as well as viruses, though most gene function knowledge currently derives from experiments carried out in a relatively small number of model organisms. Here, we provide an updated overview of the GO knowledgebase, as well as the efforts of the broad, international consortium of scientists that develops, maintains, and updates the GO knowledgebase. The GO knowledgebase consists of three components: (1) the GO-a computational knowledge structure describing the functional characteristics of genes; (2) GO annotations-evidence-supported statements asserting that a specific gene product has a particular functional characteristic; and (3) GO Causal Activity Models (GO-CAMs)-mechanistic models of molecular \\\"pathways\\\" (GO biological processes) created by linking multiple GO annotations using defined relations. Each of these components is continually expanded, revised, and updated in response to newly published discoveries and receives extensive QA checks, reviews, and user feedback. For each of these components, we provide a description of the current contents, recent developments to keep the knowledgebase up to date with new discoveries, and guidance on how users can best make use of the data that we provide. We conclude with future directions for the project.\", \"doi_y\": \"https://doi.org/10.1093/genetics/iyad031\", \"publication_date\": \"2023-03-03\"}, {\"work_id\": \"W3195161208\", \"doi_x\": \"10.1002/prot.26216\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<scp>Cryo\\u2010EM</scp> targets in <scp>CASP14</scp>\", \"abstract\": \"Structures of seven CASP14 targets were determined using cryo-electron microscopy (cryo-EM) technique with resolution between 2.1 and 3.8 \\u00c5. We provide an evaluation of the submitted models versus the experimental data (cryo-EM density maps) and experimental reference structures built into the maps. The accuracy of models is measured in terms of coordinate-to-density and coordinate-to-coordinate fit. A-posteriori refinement of the most accurate models in their corresponding cryo-EM density resulted in structures that are close to the reference structure, including some regions with better fit to the density. Regions that were found to be less \\\"refineable\\\" correlate well with regions of high diversity between the CASP models and low goodness-of-fit to density in the reference structure.\", \"doi_y\": \"https://doi.org/10.1002/prot.26216\", \"publication_date\": \"2021-09-16\"}, {\"work_id\": \"W4285389980\", \"doi_x\": \"10.1038/s41594-022-00799-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular basis for the regulation of human glycogen synthase by phosphorylation and glucose-6-phosphate\", \"abstract\": \"Glycogen synthase (GYS1) is the central enzyme in muscle glycogen biosynthesis. GYS1 activity is inhibited by phosphorylation of its amino (N) and carboxyl (C) termini, which is relieved by allosteric activation of glucose-6-phosphate (Glc6P). We present cryo-EM structures at 3.0-4.0 \\u00c5 resolution of phosphorylated human GYS1, in complex with a minimal interacting region of glycogenin, in the inhibited, activated and catalytically competent states. Phosphorylations of specific terminal residues are sensed by different arginine clusters, locking the GYS1 tetramer in an inhibited state via intersubunit interactions. The Glc6P activator promotes conformational change by disrupting these interactions and increases the flexibility of GYS1, such that it is poised to adopt a catalytically competent state when the sugar donor UDP-glucose (UDP-glc) binds. We also identify an inhibited-like conformation that has not transitioned into the activated state, in which the locking interaction of phosphorylation with the arginine cluster impedes subsequent conformational changes due to Glc6P binding. Our results address longstanding questions regarding the mechanism of human GYS1 regulation.\", \"doi_y\": \"https://doi.org/10.1038/s41594-022-00799-3\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4323546397\", \"doi_x\": \"10.1038/s41586-023-05781-7\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"abstract\": \"Diverse aerobic bacteria use atmospheric H2 as an energy source for growth and survival1. This globally significant process regulates the composition of the atmosphere, enhances soil biodiversity and drives primary production in extreme environments2,3. Atmospheric H2 oxidation is attributed to uncharacterized members of the [NiFe] hydrogenase superfamily4,5. However, it remains unresolved how these enzymes overcome the extraordinary catalytic challenge of oxidizing picomolar levels of H2 amid ambient levels of the catalytic poison O2 and how the derived electrons are transferred to the respiratory chain1. Here we determined the cryo-electron microscopy structure of the Mycobacterium smegmatis hydrogenase Huc and investigated its mechanism. Huc is a highly efficient oxygen-insensitive enzyme that couples oxidation of atmospheric H2 to the hydrogenation of the respiratory electron carrier menaquinone. Huc uses narrow hydrophobic gas channels to selectively bind atmospheric H2 at the expense of O2, and 3 [3Fe-4S] clusters modulate the properties of the enzyme so that atmospheric H2 oxidation is energetically feasible. The Huc catalytic subunits form an octameric 833 kDa complex around a membrane-associated stalk, which transports and reduces menaquinone 94 \\u00c5 from the membrane. These findings provide a mechanistic basis for the biogeochemically and ecologically important process of atmospheric H2 oxidation, uncover a mode of energy coupling dependent on long-range quinone transport, and pave the way for the development of catalysts that oxidize H2 in ambient air.\", \"doi_y\": \"https://doi.org/10.1038/s41586-023-05781-7\", \"publication_date\": \"2023-03-08\"}, {\"work_id\": \"W4281945460\", \"doi_x\": \"10.1038/s41586-022-04456-z\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The road to fully programmable protein catalysis\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-04456-z\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4311858108\", \"doi_x\": \"10.3390/toxins14120863\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Crystal Structure of Bacillus thuringiensis Tpp80Aa1 and Its Interaction with Galactose-Containing Glycolipids\", \"abstract\": \"Tpp80Aa1 from Bacillus thuringiensis is a Toxin_10 family protein (Tpp) with reported action against Culex mosquitoes. Here, we demonstrate an expanded target range, showing Tpp80Aa1 is also active against the larvae of Anopheles gambiae and Aedes aegypti mosquitoes. We report the first crystal structure of Tpp80Aa1 at a resolution of 1.8 \\u00c5, which shows Tpp80Aa1 consists of two domains: an N-terminal \\u03b2-trefoil domain resembling a ricin B lectin and a C-terminal putative pore-forming domain sharing structural similarity with the aerolysin family. Similar to other Tpp family members, we observe Tpp80Aa1 binds to the mosquito midgut, specifically the posterior midgut and the gastric caecum. We also identify that Tpp80Aa1 can interact with galactose-containing glycolipids and galactose, and this interaction is critical for exerting full insecticidal action against mosquito target cell lines.\", \"doi_y\": \"https://doi.org/10.3390/toxins14120863\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4308469543\", \"doi_x\": \"10.1016/j.csbj.2022.11.004\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"abstract\": \"Coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 is an ongoing pandemic that causes significant health/socioeconomic burden. Variants of concern (VOCs) have emerged affecting transmissibility, disease severity and re-infection risk. Studies suggest that the - N-terminal domain (NTD) of the spike protein may have a role in facilitating virus entry via sialic-acid receptor binding. Furthermore, most VOCs include novel NTD variants. Despite global sequence and structure similarity, most sialic-acid binding pockets in NTD vary across coronaviruses. Our work suggests ongoing evolutionary tuning of the sugar-binding pockets and recent analyses have shown that NTD insertions in VOCs tend to lie close to loops. We extended the structural characterisation of these sugar-binding pockets and explored whether variants could enhance sialic acid-binding. We found that recent NTD insertions in VOCs (i.e., Gamma, Delta and Omicron variants) and emerging variants of interest (VOIs) (i.e., Iota, Lambda and Theta variants) frequently lie close to sugar-binding pockets. For some variants, including the recent Omicron VOC, we find increases in predicted sialic acid-binding energy, compared to the original SARS-CoV-2, which may contribute to increased transmission. These binding observations are supported by molecular dynamics simulations (MD). We examined the similarity of NTD across Betacoronaviruses to determine whether the sugar-binding pockets are sufficiently similar to be exploited in drug design. Whilst most pockets are too structurally variable, we detected a previously unknown highly structurally conserved pocket which can be investigated in pursuit of a generic pan-Betacoronavirus drug. Our structure-based analyses help rationalise the effects of VOCs and provide hypotheses for experiments. Our findings suggest a strong need for experimental monitoring of changes in NTD of VOCs.\", \"doi_y\": \"https://doi.org/10.1016/j.csbj.2022.11.004\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4200390169\", \"doi_x\": \"10.1101/2021.12.03.471045\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"abstract\": \"Summary On the 24 th November 2021 the sequence of a new SARS CoV-2 viral isolate spreading rapidly in Southern Africa was announced, containing far more mutations in Spike (S) than previously reported variants. Neutralization titres of Omicron by sera from vaccinees and convalescent subjects infected with early pandemic as well as Alpha, Beta, Gamma, Delta are substantially reduced or fail to neutralize. Titres against Omicron are boosted by third vaccine doses and are high in cases both vaccinated and infected by Delta. Mutations in Omicron knock out or substantially reduce neutralization by most of a large panel of potent monoclonal antibodies and antibodies under commercial development. Omicron S has structural changes from earlier viruses, combining mutations conferring tight binding to ACE2 to unleash evolution driven by immune escape, leading to a large number of mutations in the ACE2 binding site which rebalance receptor affinity to that of early pandemic viruses.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.03.471045\", \"publication_date\": \"2021-12-07\"}, {\"work_id\": \"W4315499137\", \"doi_x\": \"10.3389/fpls.2022.1076298\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"\\u03b2-1,4-Xylan backbone synthesis in higher plants: How complex can it be?\", \"abstract\": \"Xylan is a hemicellulose present in the cell walls of all land plants. Glycosyltransferases of the GT43 (IRX9/IRX9L and IRX14/IRX14L) and GT47 (IRX10/IRX10L) families are involved in the biosynthesis of its \\u03b2-1,4-linked xylose backbone, which can be further modified by acetylation and sugar side chains. However, it remains unclear how the different enzymes work together to synthesize the xylan backbone. A xylan synthesis complex (XSC) has been described in the monocots wheat and asparagus, and co-expression of asparagus AoIRX9, AoIRX10 and AoIRX14A is required to form a catalytically active complex for secondary cell wall xylan biosynthesis. Here, we argue that an equivalent XSC exists for the synthesis of the primary cell wall of the eudicot Arabidopsis thaliana, consisting of IRX9L, IRX10L and IRX14. This would suggest the existence of distinct XSCs for primary and secondary cell wall xylan synthesis, reminiscent of the distinct cellulose synthesis complexes (CSCs) of the primary and secondary cell wall. In contrast to the CSC, in which each CESA protein has catalytic activity, the XSC seems to contain proteins with non-catalytic function with each component bearing potentially unique but crucial roles. Moreover, the core XSC formed by a combination of IRX9/IRX9L, IRX10/IRX10L and IRX14/IRX14L might not be stable in its composition during transit from the endoplasmic reticulum to the Golgi apparatus. Instead, potential dynamic changes of the XSC might be a means of regulating xylan biosynthesis to facilitate coordinated deposition of tailored polysaccharides in the plant cell wall.\", \"doi_y\": \"https://doi.org/10.3389/fpls.2022.1076298\", \"publication_date\": \"2023-01-11\"}, {\"work_id\": \"W4360813182\", \"doi_x\": \"10.1093/femsec/fiad027\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Compensatory mutations reducing the fitness cost of plasmid carriage occur in plant rhizosphere communities\", \"abstract\": \"Plasmids drive bacterial evolutionary innovation by transferring ecologically important functions between lineages, but acquiring a plasmid often comes at a fitness cost to the host cell. Compensatory mutations, which ameliorate the cost of plasmid carriage, promote plasmid maintenance in simplified laboratory media across diverse plasmid-host associations. Whether such compensatory evolution can occur in more complex communities inhabiting natural environmental niches where evolutionary paths may be more constrained is, however, unclear. Here, we show a substantial fitness cost of carrying the large conjugative plasmid pQBR103 in Pseudomonas fluorescens SBW25 in the plant rhizosphere. This plasmid fitness cost could be ameliorated by compensatory mutations affecting the chromosomal global regulatory system gacA/gacS, which arose rapidly in plant rhizosphere communities and were exclusive to plasmid carriers. These findings expand our understanding of the importance of compensatory evolution in plasmid dynamics beyond simplified lab media. Compensatory mutations contribute to plasmid survival in bacterial populations living within complex microbial communities in their environmental niche.\", \"doi_y\": \"https://doi.org/10.1093/femsec/fiad027\", \"publication_date\": \"2023-03-23\"}, {\"work_id\": \"W4296502374\", \"doi_x\": \"10.1039/d2sc04479j\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"abstract\": \"Rules for designing 4-helix bundles are defined, tested, and used to generate de novo peptide assemblies and a single-chain protein.\", \"doi_y\": \"https://doi.org/10.1039/d2sc04479j\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4385782400\", \"doi_x\": \"10.1016/j.funbio.2023.08.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fungal spore swelling and germination are restricted by the macrophage phagolysosome\", \"abstract\": \"Many species of medically important fungi are prolific in the formation of asexual spores. Spores undergo a process of active swelling and cell wall remodelling before a germ tube is formed and filamentous growth ensues. Highly elongated germ tubes are known to be difficult to phagocytose and pose particular challenges for immune phagocytes. However, the significance of the earliest stages of spore germination during immune cell interactions has not been investigated and yet this is likely to be important for defence against sporogenous fungal pathogens. We show here that macrophages restrict the early phases of the spore germination process of Aspergillus fumigatus and Mucor circinelloides including the initial phase of spore swelling, spore germination and early polarised growth. Macrophages are therefore adept at retarding germination as well as subsequent vegetative growth which is likely to be critical for immune surveillance and protection against sporulating fungi.\", \"doi_y\": \"https://doi.org/10.1016/j.funbio.2023.08.002\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W3164905191\", \"doi_x\": \"10.1002/prot.26185\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Evaluation of model refinement in <scp>CASP14</scp>\", \"abstract\": \"We report here an assessment of the model refinement category of the 14th round of Critical Assessment of Structure Prediction (CASP14). As before, predictors submitted up to five ranked refinements, along with associated residue-level error estimates, for targets that had a wide range of starting quality. The ability of groups to accurately rank their submissions and to predict coordinate error varied widely. Overall, only four groups out-performed a \\\"na\\u00efve predictor\\\" corresponding to the resubmission of the starting model. Among the top groups, there are interesting differences of approach and in the spread of improvements seen: some methods are more conservative, others more adventurous. Some targets were \\\"double-barreled\\\" for which predictors were offered a high-quality AlphaFold 2 (AF2)-derived prediction alongside another of lower quality. The AF2-derived models were largely unimprovable, many of their apparent errors being found to reside at domain and, especially, crystal lattice contacts. Refinement is shown to have a mixed impact overall on structure-based function annotation methods to predict nucleic acid binding, spot catalytic sites, and dock protein structures.\", \"doi_y\": \"https://doi.org/10.1002/prot.26185\", \"publication_date\": \"2021-07-29\"}, {\"work_id\": \"W4210399978\", \"doi_x\": \"10.1016/j.matbio.2022.01.005\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"abstract\": \"TGF\\u03b2 superfamily members are potent growth factors in the extracellular matrix with essential roles in all aspects of cellular behaviour. Latent TGF\\u03b2 binding proteins (LTBPs) are co-expressed with TGF\\u03b2, essential for correct folding and secretion of the growth factor, to form large latent complexes. These large latent complexes bind extracellular proteins such as fibrillin for sequestration of TGF\\u03b2 in the matrix, essential for normal tissue function, and dysregulated TGF\\u03b2 signalling is a hallmark of many fibrillinopathies. Transglutaminase-2 (TG2) cross-linking of LTBPs is known to play a role in TGF\\u03b2 activation but the underlying molecular mechanisms are not resolved. Here we show that fibrillin is a matrix substrate for TG2 and that TG2 cross-linked complexes can be formed between fibrillin and LTBP-1 and -3, and their latent TGF\\u03b2 complexes. The structure of the fibrillin-LTBP1 complex shows that the two elongated proteins interact in a perpendicular arrangement which would allow them to form distal interactions between the matrix and the cell surface. Formation of the cross-link with fibrillin does not change the interaction between latent TGF\\u03b2 and integrin \\u03b1V\\u03b26 but does increase TGF\\u03b2 activation in cell-based assays. The activating effect may be due to direction of the latent complexes to the cell surface by fibrillin, as competition with heparan sulphate can ameliorate the activating effect. Together, these data support that TGF\\u03b2 activation can be enhanced by covalent tethering of LTBPs to the matrix via fibrillin.\", \"doi_y\": \"https://doi.org/10.1016/j.matbio.2022.01.005\", \"publication_date\": \"2022-03-01\"}, {\"work_id\": \"W4200542864\", \"doi_x\": \"10.1101/2021.12.09.472018\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"abstract\": \"Abstract Bacterial metabolosomes are a family of protein organelles in bacteria. Elucidating how thousands of proteins self-assemble to form functional metabolosomes is essential for understanding their significance in cellular metabolism and pathogenesis. Here we investigate the de novo biogenesis of propanediol-utilization (Pdu) metabolosomes and characterize the roles of the key constituents in generation and intracellular positioning of functional metabolosomes. Our results demonstrate that the Pdu metabolosome undertakes both \\u201cShell first\\u201d and \\u201cCargo first\\u201d assembly pathways, unlike the \\u03b2-carboxysome structural analog which only involves the \\u201cCargo first\\u201d strategy. Shell and cargo assemblies occur independently at the cell poles. The internal cargo core is formed through the ordered assembly of multiple enzyme complexes, and exhibits liquid-like properties within the metabolosome architecture. Our findings provide mechanistic insight into the molecular principles driving bacterial metabolosome assembly and expand our understanding of liquid-like organelle biogenesis.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.09.472018\", \"publication_date\": \"2021-12-09\"}, {\"work_id\": \"W4281491296\", \"doi_x\": \"10.1038/s41467-022-30608-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"abstract\": \"Bacterial metabolosomes are a family of protein organelles in bacteria. Elucidating how thousands of proteins self-assemble to form functional metabolosomes is essential for understanding their significance in cellular metabolism and pathogenesis. Here we investigate the de novo biogenesis of propanediol-utilization (Pdu) metabolosomes and characterize the roles of the key constituents in generation and intracellular positioning of functional metabolosomes. Our results demonstrate that the Pdu metabolosome undertakes both \\\"Shell first\\\" and \\\"Cargo first\\\" assembly pathways, unlike the \\u03b2-carboxysome structural analog which only involves the \\\"Cargo first\\\" strategy. Shell and cargo assemblies occur independently at the cell poles. The internal cargo core is formed through the ordered assembly of multiple enzyme complexes, and exhibits liquid-like properties within the metabolosome architecture. Our findings provide mechanistic insight into the molecular principles driving bacterial metabolosome assembly and expand our understanding of liquid-like organelle biogenesis.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-30608-w\", \"publication_date\": \"2022-05-25\"}, {\"work_id\": \"W4322764495\", \"doi_x\": \"10.1021/jacsau.2c00569\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"abstract\": \"Glycosaminoglycans (GAGs) are complex polysaccharides exhibiting a vast structural diversity and fulfilling various functions mediated by thousands of interactions in the extracellular matrix, at the cell surface, and within the cells where they have been detected in the nucleus. It is known that the chemical groups attached to GAGs and GAG conformations comprise \\\"glycocodes\\\" that are not yet fully deciphered. The molecular context also matters for GAG structures and functions, and the influence of the structure and functions of the proteoglycan core proteins on sulfated GAGs and vice versa warrants further investigation. The lack of dedicated bioinformatic tools for mining GAG data sets contributes to a partial characterization of the structural and functional landscape and interactions of GAGs. These pending issues will benefit from the development of new approaches reviewed here, namely (i) the synthesis of GAG oligosaccharides to build large and diverse GAG libraries, (ii) GAG analysis and sequencing by mass spectrometry (e.g., ion mobility-mass spectrometry), gas-phase infrared spectroscopy, recognition tunnelling nanopores, and molecular modeling to identify bioactive GAG sequences, biophysical methods to investigate binding interfaces, and to expand our knowledge and understanding of glycocodes governing GAG molecular recognition, and (iii) artificial intelligence for in-depth investigation of GAGomic data sets and their integration with proteomics.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.2c00569\", \"publication_date\": \"2023-03-02\"}, {\"work_id\": \"W4213126762\", \"doi_x\": \"10.1111/febs.16396\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Probing heavy metal binding to phycobiliproteins\", \"abstract\": \"Blue-green algae, also known as cyanobacteria, contain some of the most efficient light-harvesting complexes known. These large, colourful complexes consist of phycobiliproteins which are extremely valuable in the cosmetics, food, nutraceutical and pharmaceutical industries. Additionally, the colourful and fluorescent properties of phycobiliproteins can be modulated by metal ions, making them highly attractive as heavy metal sensors and heavy metal scavengers. Although the overall quenching ability metal ions have on phycobiliproteins is known, the mechanism of heavy metal binding to phycobiliproteins is not fully understood, limiting their widespread quantitative applications. Here, we show using high-resolution native mass spectrometry that phycobiliprotein complexes bind metal ions in different manners. Through monitoring the binding equilibria and metal-binding stoichiometry, we show in particular copper and silver to have drastic, yet different effects on phycobiliprotein structure, both copper and silver modulate the overall complex properties. Together, the data reveals the mechanisms by which metal ions can modulate phycobiliprotein properties which can be used as a basis for the future design of metal-related phycobiliprotein applications.\", \"doi_y\": \"https://doi.org/10.1111/febs.16396\", \"publication_date\": \"2022-02-22\"}, {\"work_id\": \"W4220725479\", \"doi_x\": \"10.1128/aem.00255-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"abstract\": \"The methylotrophic OM43 clade are Gammaproteobacteria that comprise some of the smallest free-living cells known and have highly streamlined genomes. OM43 represents an important microbial link between marine primary production and remineralization of carbon back to the atmosphere. Bacteriophages shape microbial communities and are major drivers of mortality and global marine biogeochemistry. Recent cultivation efforts have brought the first viruses infecting members of the OM43 clade into culture. Here, we characterize a novel myophage infecting OM43 called Melnitz. Melnitz was isolated independently from water samples from a subtropical ocean gyre (Sargasso Sea) and temperate coastal (Western English Channel) systems. Metagenomic recruitment from global ocean viromes confirmed that Melnitz is globally ubiquitous, congruent with patterns of host abundance. Bacteria with streamlined genomes such as OM43 and the globally dominant SAR11 clade use riboswitches as an efficient method to regulate metabolism. Melnitz encodes a two-piece tmRNA (ssrA), controlled by a glutamine riboswitch, providing evidence that riboswitch use also occurs for regulation during phage infection of streamlined heterotrophs. Virally encoded tRNAs and ssrA found in Melnitz were phylogenetically more closely related to those found within the alphaproteobacterial SAR11 clade and their associated myophages than those within their gammaproteobacterial hosts. This suggests the possibility of an ancestral host transition event between SAR11 and OM43. Melnitz and a related myophage that infects SAR11 were unable to infect hosts of the SAR11 and OM43, respectively, suggesting host transition rather than a broadening of host range. IMPORTANCE Isolation and cultivation of viruses are the foundations on which the mechanistic understanding of virus-host interactions and parameterization of bioinformatic tools for viral ecology are based. This study isolated and characterized the first myophage known to infect the OM43 clade, expanding our knowledge of this understudied group of microbes. The nearly identical genomes of four strains of Melnitz isolated from different marine provinces and the global abundance estimations from metagenomic data suggest that this viral population is globally ubiquitous. Genome analysis revealed several unusual features in Melnitz and related genomes recovered from viromes, such as a curli operon and virally encoded tmRNA controlled by a glutamine riboswitch, neither of which are found in the host. Further phylogenetic analysis of shared genes indicates that this group of viruses infecting the gammaproteobacterial OM43 shares a recent common ancestor with viruses infecting the abundant alphaproteobacterial SAR11 clade. Host ranges are affected by compatible cell surface receptors, successful circumvention of superinfection exclusion systems, and the presence of required accessory proteins, which typically limits phages to singular narrow groups of closely related bacterial hosts. This study provides intriguing evidence that for streamlined heterotrophic bacteria, virus-host transitioning may not be necessarily restricted to phylogenetically related hosts but is a function of shared physical and biochemical properties of the cell.\", \"doi_y\": \"https://doi.org/10.1128/aem.00255-22\", \"publication_date\": \"2022-04-12\"}, {\"work_id\": \"W4290033835\", \"doi_x\": \"10.1101/2022.08.03.502475\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Multi-organ functions of yolk sac during human early development\", \"abstract\": \"Abstract The yolk sac (YS) represents an evolutionarily-conserved extraembryonic structure that ensures timely delivery of nutritional support and oxygen to the developing embryo. However, the YS remains ill-defined in humans. We therefore assemble a complete single cell 3D map of human YS from 3-8 post conception weeks by integrating multiomic protein and gene expression data. We reveal the YS as a site of primitive and definitive haematopoiesis including a YS-specific accelerated route to macrophage production, a source of nutritional/metabolic support and a regulator of oxygen-carrying capacity. We reconstruct the emergence of primitive haematopoietic stem and progenitor cells from YS hemogenic endothelium and their decline upon stromal support modulation as intraembryonic organs specialise to assume these functions. The YS therefore functions as \\u2018three organs in one\\u2019 revealing a multifaceted relay of vital organismal functions as pregnancy proceeds. One Sentence Summary Human yolk sac is a key staging post in a relay of vital organismal functions during human pregnancy.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.03.502475\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4381470302\", \"doi_x\": \"10.1128/spectrum.04858-22\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Absence of Staphylococcus aureus in Wild Populations of Fish Supports a Spillover Hypothesis\", \"abstract\": \"Staphylococcus aureus is a human and livestock commensal but also an important pathogen responsible for high human mortality rates and economic losses in farming. Recent studies show that S. aureus is common in wild animals, including fish.\", \"doi_y\": \"https://doi.org/10.1128/spectrum.04858-22\", \"publication_date\": \"2023-08-17\"}, {\"work_id\": \"W4312064800\", \"doi_x\": \"10.1007/s11120-022-00990-z\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Absolute quantification of cellular levels of photosynthesis-related proteins in Synechocystis sp. PCC 6803\", \"abstract\": \"Quantifying cellular components is a basic and important step for understanding how a cell works, how it responds to environmental changes, and for re-engineering cells to produce valuable metabolites and increased biomass. We quantified proteins in the model cyanobacterium Synechocystis sp. PCC 6803 given the general importance of cyanobacteria for global photosynthesis, for synthetic biology and biotechnology research, and their ancestral relationship to the chloroplasts of plants. Four mass spectrometry methods were used to quantify cellular components involved in the biosynthesis of chlorophyll, carotenoid and bilin pigments, membrane assembly, the light reactions of photosynthesis, fixation of carbon dioxide and nitrogen, and hydrogen and sulfur metabolism. Components of biosynthetic pathways, such as those for chlorophyll or for photosystem II assembly, range between 1000 and 10,000 copies per cell, but can be tenfold higher for CO2 fixation enzymes. The most abundant subunits are those for photosystem I, with around 100,000 copies per cell, approximately 2 to fivefold higher than for photosystem II and ATP synthase, and 5-20 fold more than for the cytochrome b6f complex. Disparities between numbers of pathway enzymes, between components of electron transfer chains, and between subunits within complexes indicate possible control points for biosynthetic processes, bioenergetic reactions and for the assembly of multisubunit complexes.\", \"doi_y\": \"https://doi.org/10.1007/s11120-022-00990-z\", \"publication_date\": \"2022-12-21\"}, {\"work_id\": \"W4225103662\", \"doi_x\": \"10.1186/s40694-022-00138-7\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"abstract\": \"Non-ribosomal peptide synthetase-like (NRPS-like) enzymes are highly enriched in fungal genomes and can be discriminated into reducing and non-reducing enzymes. Non-reducing NRPS-like enzymes possess a C-terminal thioesterase domain that catalyses the condensation of two identical aromatic \\u03b1-keto acids under the formation of enzyme-specific substrate-interconnecting core structures such as terphenylquinones, furanones, butyrolactones or dioxolanones. Ascocoryne sarcoides produces large quantities of ascocorynin, which structurally resembles a terphenylquinone produced from the condensation of p-hydroxyphenylpyruvate and phenylpyruvate. Since the parallel use of two different substrates by a non-reducing NRPS-like enzyme appeared as highly unusual, we investigated the biosynthesis of ascocorynin in A. sarcoides.Here, we searched the genome of A. sarcoides for genes coding for non-reducing NRPS-like enzymes. A single candidate gene was identified that was termed acyN. Heterologous gene expression confirmed that AcyN is involved in ascocorynin production but only produces the non-hydroxylated precursor polyporic acid. Although acyN is embedded in an ascocorynin biosynthesis gene cluster, a gene encoding a monooxygenase required for the hydroxylation of polyporic acid was not present. Expression analyses of all monooxygenase-encoding genes from A. sarcoides identified a single candidate that showed the same expression pattern as acyN. Accordingly, heterologous co-expression of acyN and the monooxygenase gene resulted in the production of ascocorynin. Structural modelling of the monooxygenase suggests that the hydrophobic substrate polyporic acid enters the monooxygenase from a membrane facing entry site and is converted into the more hydrophilic product ascocorynin, which prevents its re-entry for a second round of hydroxylation.This study characterises the first naturally occurring polyporic acid synthetase from an ascomycete. It confirms the high substrate and product specificity of this non-reducing NRPS-like enzyme and highlights the requirement of a monooxygenase to produce the terphenylquinone ascocorynin.\", \"doi_y\": \"https://doi.org/10.1186/s40694-022-00138-7\", \"publication_date\": \"2022-04-27\"}, {\"work_id\": \"W4289868872\", \"doi_x\": \"10.3389/fmicb.2022.948383\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"A new class of biological ion-driven rotary molecular motors with 5:2 symmetry\", \"abstract\": \"Several new structures of three types of protein complexes, obtained by cryo-electron microscopy (cryo-EM) and published between 2019 and 2021, identify a new family of natural molecular wheels, the \\\"5:2 rotary motors.\\\" These span the cytoplasmic membranes of bacteria, and their rotation is driven by ion flow into the cell. They consist of a pentameric wheel encircling a dimeric axle within the cytoplasmic membrane of both Gram-positive and gram-negative bacteria. The axles extend into the periplasm, and the wheels extend into the cytoplasm. Rotation of these wheels has never been observed directly; it is inferred from the symmetry of the complexes and from the roles they play within the larger systems that they are known to power. In particular, the new structure of the stator complex of the Bacterial Flagellar Motor, MotA5B2, is consistent with a \\\"wheels within wheels\\\" model of the motor. Other 5:2 rotary motors are believed to share the core rotary function and mechanism, driven by ion-motive force at the cytoplasmic membrane. Their structures diverge in their periplasmic and cytoplasmic parts, reflecting the variety of roles that they perform. This review focuses on the structures of 5:2 rotary motors and their proposed mechanisms and functions. We also discuss molecular rotation in general and its relation to the rotational symmetry of molecular complexes.\", \"doi_y\": \"https://doi.org/10.3389/fmicb.2022.948383\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4311547974\", \"doi_x\": \"10.1038/s41467-022-35454-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Multi-omics identify falling LRRC15 as a COVID-19 severity marker and persistent pro-thrombotic signals in convalescence\", \"abstract\": \"Patients with end-stage kidney disease (ESKD) are at high risk of severe COVID-19. Here, we perform longitudinal blood sampling of ESKD haemodialysis patients with COVID-19, collecting samples pre-infection, serially during infection, and after clinical recovery. Using plasma proteomics, and RNA-sequencing and flow cytometry of immune cells, we identify transcriptomic and proteomic signatures of COVID-19 severity, and find distinct temporal molecular profiles in patients with severe disease. Supervised learning reveals that the plasma proteome is a superior indicator of clinical severity than the PBMC transcriptome. We show that a decreasing trajectory of plasma LRRC15, a proposed co-receptor for SARS-CoV-2, is associated with a more severe clinical course. We observe that two months after the acute infection, patients still display dysregulated gene expression related to vascular, platelet and coagulation pathways, including PF4 (platelet factor 4), which may explain the prolonged thrombotic risk following COVID-19.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-35454-4\", \"publication_date\": \"2022-12-15\"}, {\"work_id\": \"W4379600803\", \"doi_x\": \"10.1093/nar/gkad478\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Reverse transcriptases prime DNA synthesis\", \"abstract\": \"Abstract The discovery of reverse transcriptases (RTs) challenged the central dogma by establishing that genetic information can also flow from RNA to DNA. Although they act as DNA polymerases, RTs are distantly related to replicases that also possess de novo primase activity. Here we identify that CRISPR associated RTs (CARTs) directly prime DNA synthesis on both RNA and DNA. We demonstrate that RT-dependent priming is utilized by some CRISPR-Cas complexes to synthesise new spacers and integrate these into CRISPR arrays. Expanding our analyses, we show that primer synthesis activity is conserved in representatives of other major RT classes, including group II intron RT, telomerase and retroviruses. Together, these findings establish a conserved innate ability of RTs to catalyse de novo DNA primer synthesis, independently of accessory domains or alternative priming mechanisms, which likely plays important roles in a wide variety of biological pathways.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkad478\", \"publication_date\": \"2023-06-06\"}, {\"work_id\": \"W4321440644\", \"doi_x\": \"10.1126/scisignal.abm7134\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Disruptions in endocytic traffic contribute to the activation of the NLRP3 inflammasome\", \"abstract\": \"Inflammation driven by the NLRP3 inflammasome is coordinated through multiple signaling pathways and is regulated by subcellular organelles. Here, we tested the hypothesis that NLRP3 senses disrupted endosome trafficking to trigger inflammasome formation and inflammatory cytokine secretion. NLRP3-activating stimuli disrupted endosome trafficking and triggered localization of NLRP3 to vesicles positive for endolysosomal markers and for the inositol lipid PI4P. Chemical disruption of endosome trafficking sensitized macrophages to the NLRP3 activator imiquimod, driving enhanced inflammasome activation and cytokine secretion. Together, these data suggest that NLRP3 can sense disruptions in the trafficking of endosomal cargoes, which may explain in part the spatial activation of the NLRP3 inflammasome. These data highlight mechanisms that could be exploited in the therapeutic targeting of NLRP3.\", \"doi_y\": \"https://doi.org/10.1126/scisignal.abm7134\", \"publication_date\": \"2023-02-21\"}, {\"work_id\": \"W4324376276\", \"doi_x\": \"10.3389/fimmu.2023.1146702\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 antibody responses associate with sex, age and disease severity in previously uninfected people admitted to hospital with COVID-19: An ISARIC4C prospective study\", \"abstract\": \"The SARS-CoV-2 pandemic enables the analysis of immune responses induced against a novel coronavirus infecting immunologically na\\u00efve individuals. This provides an opportunity for analysis of immune responses and associations with age, sex and disease severity. Here we measured an array of solid-phase binding antibody and viral neutralising Ab (nAb) responses in participants (n=337) of the ISARIC4C cohort and characterised their correlation with peak disease severity during acute infection and early convalescence. Overall, the responses in a Double Antigen Binding Assay (DABA) for antibody to the receptor binding domain (anti-RBD) correlated well with IgM as well as IgG responses against viral spike, S1 and nucleocapsid protein (NP) antigens. DABA reactivity also correlated with nAb. As we and others reported previously, there is greater risk of severe disease and death in older men, whilst the sex ratio was found to be equal within each severity grouping in younger people. In older males with severe disease (mean age 68 years), peak antibody levels were found to be delayed by one to two weeks compared with women, and nAb responses were delayed further. Additionally, we demonstrated that solid-phase binding antibody responses reached higher levels in males as measured via DABA and IgM binding against Spike, NP and S1 antigens. In contrast, this was not observed for nAb responses. When measuring SARS-CoV-2 RNA transcripts (as a surrogate for viral shedding) in nasal swabs at recruitment, we saw no significant differences by sex or disease severity status. However, we have shown higher antibody levels associated with low nasal viral RNA indicating a role of antibody responses in controlling viral replication and shedding in the upper airway. In this study, we have shown discernible differences in the humoral immune responses between males and females and these differences associate with age as well as with resultant disease severity.\", \"doi_y\": \"https://doi.org/10.3389/fimmu.2023.1146702\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4302276392\", \"doi_x\": \"10.1093/cid/ciac796\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Booster Vaccination Against SARS-CoV-2 Induces Potent Immune Responses in People With Human Immunodeficiency Virus\", \"abstract\": \"People with human immunodeficiency virus (HIV) on antiretroviral therapy (ART) with good CD4 T-cell counts make effective immune responses following vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). There are few data on longer term responses and the impact of a booster dose.Adults with HIV were enrolled into a single arm open label study. Two doses of ChAdOx1 nCoV-19 were followed 12 months later by a third heterologous vaccine dose. Participants had undetectable viraemia on ART and CD4 counts >350 cells/\\u00b5L. Immune responses to the ancestral strain and variants of concern were measured by anti-spike immunoglobulin G (IgG) enzyme-linked immunosorbent assay (ELISA), MesoScale Discovery (MSD) anti-spike platform, ACE-2 inhibition, activation induced marker (AIM) assay, and T-cell proliferation.In total, 54 participants received 2 doses of ChAdOx1 nCoV-19. 43 received a third dose (42 with BNT162b2; 1 with mRNA-1273) 1 year after the first dose. After the third dose, total anti-SARS-CoV-2 spike IgG titers (MSD), ACE-2 inhibition, and IgG ELISA results were significantly higher compared to Day 182 titers (P < .0001 for all 3). SARS-CoV-2 specific CD4+ T-cell responses measured by AIM against SARS-CoV-2 S1 and S2 peptide pools were significantly increased after a third vaccine compared to 6 months after a first dose, with significant increases in proliferative CD4+ and CD8+ T-cell responses to SARS-CoV-2 S1 and S2 after boosting. Responses to Alpha, Beta, Gamma, and Delta variants were boosted, although to a lesser extent for Omicron.In PWH receiving a third vaccine dose, there were significant increases in B- and T-cell immunity, including to known variants of concern (VOCs).\", \"doi_y\": \"https://doi.org/10.1093/cid/ciac796\", \"publication_date\": \"2022-10-05\"}, {\"work_id\": \"W3214131546\", \"doi_x\": \"10.1093/nar/gkab988\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"abstract\": \"Abstract The Protein Data Bank in Europe \\u2013 Knowledge Base (PDBe-KB, https://pdbe-kb.org) is an open collaboration between world-leading specialist data resources contributing functional and biophysical annotations derived from or relevant to the Protein Data Bank (PDB). The goal of PDBe-KB is to place macromolecular structure data in their biological context by developing standardised data exchange formats and integrating functional annotations from the contributing partner resources into a knowledge graph that can provide valuable biological insights. Since we described PDBe-KB in 2019, there have been significant improvements in the variety of available annotation data sets and user functionality. Here, we provide an overview of the consortium, highlighting the addition of annotations such as predicted covalent binders, phosphorylation sites, effects of mutations on the protein structure and energetic local frustration. In addition, we describe a library of reusable web-based visualisation components and introduce new features such as a bulk download data service and a novel superposition service that generates clusters of superposed protein chains weekly for the whole PDB archive.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkab988\", \"publication_date\": \"2021-11-10\"}, {\"work_id\": \"W4378952892\", \"doi_x\": \"10.1107/s2059798323003595\", \"af_direct_citation\": 1, \"af_indirect_citations\": 9, \"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"abstract\": \"The Collaborative Computational Project No. 4 (CCP4) is a UK-led international collective with a mission to develop, test, distribute and promote software for macromolecular crystallography. The CCP4 suite is a multiplatform collection of programs brought together by familiar execution routines, a set of common libraries and graphical interfaces. The CCP4 suite has experienced several considerable changes since its last reference article, involving new infrastructure, original programs and graphical interfaces. This article, which is intended as a general literature citation for the use of the CCP4 software suite in structure determination, will guide the reader through such transformations, offering a general overview of the new features and outlining future developments. As such, it aims to highlight the individual programs that comprise the suite and to provide the latest references to them for perusal by crystallographers around the world.\", \"doi_y\": \"https://doi.org/10.1107/s2059798323003595\", \"publication_date\": \"2023-05-30\"}, {\"work_id\": \"W4212951442\", \"doi_x\": \"10.3390/v14020441\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Na\\u00efve Human Macrophages Are Refractory to SARS-CoV-2 Infection and Exhibit a Modest Inflammatory Response Early in Infection\", \"abstract\": \"Involvement of macrophages in the SARS-CoV-2-associated cytokine storm, the excessive secretion of inflammatory/anti-viral factors leading to the acute respiratory distress syndrome (ARDS) in COVID-19 patients, is unclear. In this study, we sought to characterize the interplay between the virus and primary human monocyte-derived macrophages (MDM). MDM were stimulated with recombinant IFN-\\u03b1 and/or infected with either live or UV-inactivated SARS-CoV-2 or with two reassortant influenza viruses containing external genes from the H1N1 PR8 strain and heterologous internal genes from a highly pathogenic avian H5N1 or a low pathogenic human seasonal H1N1 strain. Virus replication was monitored by qRT-PCR for the E viral gene for SARS-CoV-2 or M gene for influenza and TCID50 or plaque assay, and cytokine levels were assessed semiquantitatively with qRT-PCR and a proteome cytokine array. We report that MDM are not susceptible to SARS-CoV-2 whereas both influenza viruses replicated in MDM, albeit abortively. We observed a modest cytokine response in SARS-CoV-2 exposed MDM with notable absence of IFN-\\u03b2 induction, which was instead strongly induced by the influenza viruses. Pre-treatment of MDM with IFN-\\u03b1 enhanced proinflammatory cytokine expression upon exposure to virus. Together, the findings concur that the hyperinflammation observed in SARS-CoV-2 infection is not driven by macrophages.\", \"doi_y\": \"https://doi.org/10.3390/v14020441\", \"publication_date\": \"2022-02-21\"}, {\"work_id\": \"W4289132939\", \"doi_x\": \"10.1101/2022.07.28.500969\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High throughput steady-state enzyme kinetics measured in a parallel droplet generation and absorbance detection platform\", \"abstract\": \"ABSTRACT Microfluidic water-in-oil emulsion droplets are becoming a mainstay of experimental biology, where they replace the classical test tube. In most applications (e.g. in ultrahigh throughput directed evolution) the droplet content is identical for all compartmentalized assay reactions. When emulsion droplets are used for kinetics or other functional assays, though, concentration dependencies (e.g. of initial rates for Michaelis-Menten plots) are required. Droplet-on-demand systems satisfy this need but extracting large amounts of data is challenging. Here we introduce a multiplexed droplet absorbance detector which, coupled to semi-automated droplet generation, forms a tubing-based droplet-on-demand system able to generate and extract quantitative datasets from defined concentration gradients across multiple series of droplets for multiple time points. The emergence of product is detected by reading the absorbance of the droplet sets at multiple, adjustable time points (reversing the flow direction after each detection, so that the droplets pass a line scan camera multiple times). Detection multiplexing allows absorbance values at twelve distinct positions to be measured and enzyme kinetics are recorded for label-free concentration gradients (composed of about 60 droplets each, covering as many concentrations). With a throughput of around 8640 data points per hour, a 10-fold improvement compared to the previously reported single point detection method is achieved. In a single experiment, twelve full datasets of high-resolution and high accuracy Michaelis-Menten kinetics were determined to demonstrate the potential for enzyme characterization for glycosidase substrates covering a range in enzymatic hydrolysis of seven orders of magnitude in k cat /K M . The straightforward set-up, high throughput, excellent data quality, wide dynamic range that allows coverage of diverse activities suggest that this system may serve as a miniaturized spectrophotometer to for detailed analysis of study clones emerging from large-scale combinatorial experiments. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.07.28.500969\", \"publication_date\": \"2022-07-31\"}, {\"work_id\": \"W4385927801\", \"doi_x\": \"10.1038/s41467-023-40779-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"C16orf72/HAPSTR1/TAPR1 functions with BRCA1/Senataxin to modulate replication-associated R-loops and confer resistance to PARP disruption\", \"abstract\": \"Abstract While the toxicity of PARP inhibitors to cells with defects in homologous recombination (HR) is well established, other synthetic lethal interactions with PARP1/PARP2 disruption are poorly defined. To inform on these mechanisms we conducted a genome-wide screen for genes that are synthetic lethal with PARP1/2 gene disruption and identified C16orf72/HAPSTR1/TAPR1 as a novel modulator of replication-associated R-loops. C16orf72 is critical to facilitate replication fork restart, suppress DNA damage and maintain genome stability in response to replication stress. Importantly, C16orf72 and PARP1/2 function in parallel pathways to suppress DNA:RNA hybrids that accumulate at stalled replication forks. Mechanistically, this is achieved through an interaction of C16orf72 with BRCA1 and the RNA/DNA helicase Senataxin to facilitate their recruitment to RNA:DNA hybrids and confer resistance to PARP inhibitors. Together, this identifies a C16orf72/Senataxin/BRCA1-dependent pathway to suppress replication-associated R-loop accumulation, maintain genome stability and confer resistance to PARP inhibitors.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40779-9\", \"publication_date\": \"2023-08-17\"}, {\"work_id\": \"W4291169794\", \"doi_x\": \"10.1111/pim.12946\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fungal resilience and host\\u2013pathogen interactions: Future perspectives and opportunities\", \"abstract\": \"We are constantly exposed to the threat of fungal infection. The outcome-clearance, commensalism or infection-depends largely on the ability of our innate immune defences to clear infecting fungal cells versus the success of the fungus in mounting compensatory adaptive responses. As each seeks to gain advantage during these skirmishes, the interactions between host and fungal pathogen are complex and dynamic. Nevertheless, simply compromising the physiological robustness of fungal pathogens reduces their ability to evade antifungal immunity, their virulence, and their tolerance against antifungal therapy. In this article I argue that this physiological robustness is based on a 'Resilience Network' which mechanistically links and controls fungal growth, metabolism, stress resistance and drug tolerance. The elasticity of this network probably underlies the phenotypic variability of fungal isolates and the heterogeneity of individual cells within clonal populations. Consequently, I suggest that the definition of the fungal Resilience Network represents an important goal for the future which offers the clear potential to reveal drug targets that compromise drug tolerance and synergise with current antifungal therapies.\", \"doi_y\": \"https://doi.org/10.1111/pim.12946\", \"publication_date\": \"2022-08-27\"}, {\"work_id\": \"W4210823904\", \"doi_x\": \"10.1101/2022.02.07.478301\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Identification of Unusual Oxysterols Biosynthesised in Human Pregnancy by Charge-Tagging and Liquid Chromatography - Mass Spectrometry\", \"abstract\": \"Abstract The aim of this study was to identify sterols, oxysterols and any down-stream metabolites in placenta, umbilical cord blood plasma, maternal plasma and amniotic fluid to enhance our knowledge of the involvement of these molecules in pregnancy. We confirm the identification of 20S- hydroxycholesterol in human placenta, previously reported in a single publication, and propose a pathway from 22R-hydroxycholesterol to a C 27 bile acid of probable structure 3\\u03b2,20R,22R- trihydroxycholest-5-en-(25R)26-oic acid. The pathway is evident not only in placenta, but pathway intermediates are also found in umbilical cord plasma, maternal plasma and amniotic fluid but not non-pregnant women.\", \"doi_y\": \"https://doi.org/10.1101/2022.02.07.478301\", \"publication_date\": \"2022-02-08\"}, {\"work_id\": \"W4282913766\", \"doi_x\": \"10.1101/2022.06.09.22276196\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Accelerated waning of the humoral response to SARS-CoV-2 vaccines in obesity\", \"abstract\": \"ABSTRACT Obesity is associated with an increased risk of severe Covid-19. However, the effectiveness of SARS-CoV-2 vaccines in people with obesity is unknown. Here we studied the relationship between body mass index (BMI), hospitalization and mortality due to Covid-19 amongst 3.5 million people in Scotland. Vaccinated people with severe obesity (BMI&gt;40 kg/m 2 ) were significantly more likely to experience hospitalization or death from Covid-19. Excess risk increased with time since vaccination. To investigate the underlying mechanisms, we conducted a prospective longitudinal study of the immune response in a clinical cohort of vaccinated people with severe obesity. Compared with normal weight people, six months after their second vaccine dose, significantly more people with severe obesity had unquantifiable titres of neutralizing antibody against authentic SARS-CoV-2 virus, reduced frequencies of antigen-experienced SARS-CoV-2 Spike-binding B cells, and a dissociation between anti-Spike antibody levels and neutralizing capacity. Neutralizing capacity was restored by a third dose of vaccine, but again declined more rapidly in people with severe obesity. We demonstrate that waning of SARS-CoV-2 vaccine-induced humoral immunity is accelerated in people with severe obesity and associated with increased hospitalization and mortality from breakthrough infections. Given the prevalence of obesity, our findings have significant implications for global public health.\", \"doi_y\": \"https://doi.org/10.1101/2022.06.09.22276196\", \"publication_date\": \"2022-06-14\"}, {\"work_id\": \"W4385666772\", \"doi_x\": \"10.1016/j.devcel.2023.07.012\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"S-acylation of the Wnt receptor Frizzled-5 by zDHHC5 controls its cellular localization and synaptogenic activity in the rodent hippocampus\", \"abstract\": \"Proper localization of receptors for synaptic organizing factors is crucial for synapse formation. Wnt proteins promote synapse assembly through Frizzled (Fz) receptors. In hippocampal neurons, the surface and synaptic localization of Fz5 is regulated by neuronal activity, but the mechanisms involved remain poorly understood. Here, we report that all Fz receptors can be post-translationally modified by S-acylation and that Fz5 is S-acylated on three C-terminal cysteines by zDHHC5. S-acylation is essential for Fz5 localization to the cell surface, axons, and presynaptic sites. Notably, S-acylation-deficient Fz5 is internalized faster, affecting its association with signalosome components at the cell surface. S-acylation-deficient Fz5 also fails to activate canonical and divergent canonical Wnt pathways. Fz5 S-acylation levels are regulated by the pattern of neuronal activity. In vivo studies demonstrate that S-acylation-deficient Fz5 expression fails to induce presynaptic assembly. Our studies show that S-acylation of Frizzled receptors is a mechanism controlling their localization and function.\", \"doi_y\": \"https://doi.org/10.1016/j.devcel.2023.07.012\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W3207256498\", \"doi_x\": \"10.1002/cbic.202100484\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"abstract\": \"Abstract Terpenoids are a highly diverse group of natural products with considerable industrial interest. Increasingly, engineered microbes are used for the production of terpenoids to replace natural extracts and chemical synthesis. Terpene synthases (TSs) show a high level of functional plasticity and are responsible for the vast structural diversity observed in natural terpenoids. Their relatively inert active sites guide intrinsically reactive linear carbocation intermediates along one of many cyclisation paths via exertion of subtle steric and electrostatic control. Due to the absence of a strong protein interaction with these intermediates, there is a remarkable lack of sequence\\u2010function relationship within the TS family, making product\\u2010outcome predictions from sequences alone challenging. This, in combination with the fact that many TSs produce multiple products from a single substrate hampers the design and use of TSs in the biomanufacturing of terpenoids. This review highlights recent advances in genome mining, computational modelling, high\\u2010throughput screening, and machine\\u2010learning that will allow more predictive engineering of these fascinating enzymes in the near future.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202100484\", \"publication_date\": \"2021-11-03\"}, {\"work_id\": \"W4226169846\", \"doi_x\": \"10.1128/jvi.02024-21\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>Birnaviridae</i> Virus Factories Show Features of Liquid-Liquid Phase Separation and Are Distinct from Paracrystalline Arrays of Virions Observed by Electron Microscopy\", \"abstract\": \"To gain more information about the nature of Birnaviridae virus factories (VFs), we used a recombinant infectious bursal disease virus (IBDV) expressing split-GFP11 tagged to the polymerase (VP1) that we have previously shown is a marker for VFs in infected cells expressing GFP1-10. We found that VFs colocalized with 5-ethynyl uridine in the presence of actinomycin, demonstrating they contained newly synthesized viral RNA, and VFs were visible in infected cells that were fixed and permeabilized with digitonin, demonstrating that they were not membrane bound. Fluorescence recovery after photobleaching (FRAP) a region of interest within the VFs occurred rapidly, recovering from approximately 25% to 87% the original intensity over 146 s, and VFs were dissolved by 1,6-hexanediol treatment, demonstrating they showed properties consistent with liquid-liquid phase separation. There was a lower colocalization of the VF GFP signal with the capsid protein VP2 (Manders' coefficient [MC] 0.6), compared to VP3 (MC, 0.9), which prompted us to investigate the VF ultrastructure by transmission electron microscopy (TEM). In infected cells, paracrystalline arrays (PAs) of virions were observed in the cytoplasm, as well as discrete electron dense regions. Using correlative light and electron microscopy (CLEM), we observed that the electron dense regions correlated with the GFP signal of the VFs, which were distinct from the PAs. In summary, Birnaviridae VFs contain newly synthesized viral RNA, are not bound by a membrane, show properties consistent with liquid-liquid phase separation, and are distinct from the PAs observed by TEM. IMPORTANCE Members of the Birnaviridae infect birds, fish and insects, and are responsible for diseases of significant economic importance to the poultry industry and aquaculture. Despite their importance, how they replicate in cells remains poorly understood. Here, we show that the Birnaviridae virus factories are not membrane bound, demonstrate properties consistent with liquid-liquid phase separation, and are distinct from the paracrystalline arrays of virions observed by transmission electron microscopy, enhancing our fundamental knowledge of virus replication that could be used to develop strategies to control disease, or optimize their therapeutic application.\", \"doi_y\": \"https://doi.org/10.1128/jvi.02024-21\", \"publication_date\": \"2022-03-23\"}, {\"work_id\": \"W4310965502\", \"doi_x\": \"10.1093/nar/gkac1098\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"EMBL\\u2019s European Bioinformatics Institute (EMBL-EBI) in 2022\", \"abstract\": \"The European Molecular Biology Laboratory's European Bioinformatics Institute (EMBL-EBI) is one of the world's leading sources of public biomolecular data. Based at the Wellcome Genome Campus in Hinxton, UK, EMBL-EBI is one of six sites of the European Molecular Biology Laboratory (EMBL), Europe's only intergovernmental life sciences organisation. This overview summarises the status of services that EMBL-EBI data resources provide to scientific communities globally. The scale, openness, rich metadata and extensive curation of EMBL-EBI added-value databases makes them particularly well-suited as training sets for deep learning, machine learning and artificial intelligence applications, a selection of which are described here. The data resources at EMBL-EBI can catalyse such developments because they offer sustainable, high-quality data, collected in some cases over decades and made openly availability to any researcher, globally. Our aim is for EMBL-EBI data resources to keep providing the foundations for tools and research insights that transform fields across the life sciences.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1098\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4296807180\", \"doi_x\": \"10.1242/jcs.260288\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"abstract\": \"ABSTRACT Tropomyosins are structurally conserved \\u03b1-helical coiled-coil proteins that bind along the length of filamentous actin (F-actin) in fungi and animals. Tropomyosins play essential roles in the stability of actin filaments and in regulating myosin II contractility. Despite the crucial role of tropomyosin in actin cytoskeletal regulation, in vivo investigations of tropomyosin are limited, mainly due to the suboptimal live-cell imaging tools currently available. Here, we report on an mNeonGreen (mNG)-tagged tropomyosin, with native promoter and linker length configuration, that clearly reports tropomyosin dynamics in Schizosaccharomyces pombe (Cdc8), Schizosaccharomyces japonicus (Cdc8) and Saccharomyces cerevisiae (Tpm1 and Tpm2). We also describe a fluorescent probe to visualize mammalian tropomyosin (TPM2 isoform). Finally, we generated a camelid nanobody against S. pombe Cdc8, which mimics the localization of mNG\\u2013Cdc8 in vivo. Using these tools, we report the presence of tropomyosin in previously unappreciated patch-like structures in fission and budding yeasts, show flow of tropomyosin (F-actin) cables to the cytokinetic actomyosin ring and identify rearrangements of the actin cytoskeleton during mating. These powerful tools and strategies will aid better analyses of tropomyosin and F-actin cables in vivo.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260288\", \"publication_date\": \"2022-09-15\"}, {\"work_id\": \"W4200347144\", \"doi_x\": \"10.1016/j.mcpro.2021.100186\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Subcellular Transcriptomics and Proteomics: A Comparative Methods Review\", \"abstract\": \"The internal environment of cells is molecularly crowded, which requires spatial organization via subcellular compartmentalization. These compartments harbor specific conditions for molecules to perform their biological functions, such as coordination of the cell cycle, cell survival, and growth. This compartmentalization is also not static, with molecules trafficking between these subcellular neighborhoods to carry out their functions. For example, some biomolecules are multifunctional, requiring an environment with differing conditions or interacting partners, and others traffic to export such molecules. Aberrant localization of proteins or RNA species has been linked to many pathological conditions, such as neurological, cancer, and pulmonary diseases. Differential expression studies in transcriptomics and proteomics are relatively common, but the majority have overlooked the importance of subcellular information. In addition, subcellular transcriptomics and proteomics data do not always colocate because of the biochemical processes that occur during and after translation, highlighting the complementary nature of these fields. In this review, we discuss and directly compare the current methods in spatial proteomics and transcriptomics, which include sequencing- and imaging-based strategies, to give the reader an overview of the current tools available. We also discuss current limitations of these strategies as well as future developments in the field of spatial -omics.\", \"doi_y\": \"https://doi.org/10.1016/j.mcpro.2021.100186\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4376134661\", \"doi_x\": \"10.1038/s41591-023-02343-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Accelerated waning of the humoral response to COVID-19 vaccines in obesity\", \"abstract\": \"Obesity is associated with an increased risk of severe Coronavirus Disease 2019 (COVID-19) infection and mortality. COVID-19 vaccines reduce the risk of serious COVID-19 outcomes; however, their effectiveness in people with obesity is incompletely understood. We studied the relationship among body mass index (BMI), hospitalization and mortality due to COVID-19 among 3.6 million people in Scotland using the Early Pandemic Evaluation and Enhanced Surveillance of COVID-19 (EAVE II) surveillance platform. We found that vaccinated individuals with severe obesity (BMI > 40 kg/m2) were 76% more likely to experience hospitalization or death from COVID-19 (adjusted rate ratio of 1.76 (95% confidence interval (CI), 1.60-1.94). We also conducted a prospective longitudinal study of a cohort of 28 individuals with severe obesity compared to 41 control individuals with normal BMI (BMI 18.5-24.9 kg/m2). We found that 55% of individuals with severe obesity had unquantifiable titers of neutralizing antibody against authentic severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus compared to 12% of individuals with normal BMI (P = 0.0003) 6 months after their second vaccine dose. Furthermore, we observed that, for individuals with severe obesity, at any given anti-spike and anti-receptor-binding domain (RBD) antibody level, neutralizing capacity was lower than that of individuals with a normal BMI. Neutralizing capacity was restored by a third dose of vaccine but again declined more rapidly in people with severe obesity. We demonstrate that waning of COVID-19 vaccine-induced humoral immunity is accelerated in individuals with severe obesity. As obesity is associated with increased hospitalization and mortality from breakthrough infections, our findings have implications for vaccine prioritization policies.\", \"doi_y\": \"https://doi.org/10.1038/s41591-023-02343-2\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4220875554\", \"doi_x\": \"10.7554/elife.72825\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Allele-specific gene expression can underlie altered transcript abundance in zebrafish mutants\", \"abstract\": \"In model organisms, RNA-sequencing (RNA-seq) is frequently used to assess the effect of genetic mutations on cellular and developmental processes. Typically, animals heterozygous for a mutation are crossed to produce offspring with different genotypes. Resultant embryos are grouped by genotype to compare homozygous mutant embryos to heterozygous and wild-type siblings. Genes that are differentially expressed between the groups are assumed to reveal insights into the pathways affected by the mutation. Here we show that in zebrafish, differentially expressed genes are often over-represented on the same chromosome as the mutation due to different levels of expression of alleles from different genetic backgrounds. Using an incross of haplotype-resolved wild-type fish, we found evidence of widespread allele-specific expression, which appears as differential expression when comparing embryos homozygous for a region of the genome to their siblings. When analysing mutant transcriptomes, this means that the differential expression of genes on the same chromosome as a mutation of interest may not be caused by that mutation. Typically, the genomic location of a differentially expressed gene is not considered when interpreting its importance with respect to the phenotype. This could lead to pathways being erroneously implicated or overlooked due to the noise of spurious differentially expressed genes on the same chromosome as the mutation. These observations have implications for the interpretation of RNA-seq experiments involving outbred animals and non-inbred model organisms.\", \"doi_y\": \"https://doi.org/10.7554/elife.72825\", \"publication_date\": \"2022-02-17\"}, {\"work_id\": \"W4306152028\", \"doi_x\": \"10.3390/ijms232012237\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"How Arrestins and GRKs Regulate the Function of Long Chain Fatty Acid Receptors\", \"abstract\": \"FFA1 and FFA4, two G protein-coupled receptors that are activated by long chain fatty acids, play crucial roles in mediating many biological functions in the body. As a result, these fatty acid receptors have gained considerable attention due to their potential to be targeted for the treatment of type-2 diabetes. However, the relative contribution of canonical G protein-mediated signalling versus the effects of agonist-induced phosphorylation and interactions with \\u03b2-arrestins have yet to be fully defined. Recently, several reports have highlighted the ability of \\u03b2-arrestins and GRKs to interact with and modulate different functions of both FFA1 and FFA4, suggesting that it is indeed important to consider these interactions when studying the roles of FFA1 and FFA4 in both normal physiology and in different disease settings. Here, we discuss what is currently known and show the importance of understanding fully how \\u03b2-arrestins and GRKs regulate the function of long chain fatty acid receptors.\", \"doi_y\": \"https://doi.org/10.3390/ijms232012237\", \"publication_date\": \"2022-10-13\"}, {\"work_id\": \"W4317721966\", \"doi_x\": \"10.3389/fmolb.2023.1113762\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Perspective: Structure determination of protein-ligand complexes at room temperature using X-ray diffraction approaches\", \"abstract\": \"The interaction between macromolecular proteins and small molecule ligands is an essential component of cellular function. Such ligands may include enzyme substrates, molecules involved in cellular signalling or pharmaceutical drugs. Together with biophysical techniques used to assess the thermodynamic and kinetic properties of ligand binding to proteins, methodology to determine high-resolution structures that enable atomic level interactions between protein and ligand(s) to be directly visualised is required. Whilst such structural approaches are well established with high throughput X-ray crystallography routinely used in the pharmaceutical sector, they provide only a static view of the complex. Recent advances in X-ray structural biology methods offer several new possibilities that can examine protein-ligand complexes at ambient temperature rather than under cryogenic conditions, enable transient binding sites and interactions to be characterised using time-resolved approaches and combine spectroscopic measurements from the same crystal that the structures themselves are determined. This Perspective reviews several recent developments in these areas and discusses new possibilities for applications of these advanced methodologies to transform our understanding of protein-ligand interactions.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2023.1113762\", \"publication_date\": \"2023-01-23\"}, {\"work_id\": \"W4298109826\", \"doi_x\": \"10.3389/fpls.2022.967203\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"abstract\": \"OPINION article Front. Plant Sci., 30 September 2022Sec. Photosynthesis and Photobiology Volume 13 - 2022 | https://doi.org/10.3389/fpls.2022.967203\", \"doi_y\": \"https://doi.org/10.3389/fpls.2022.967203\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4319598121\", \"doi_x\": \"10.3390/antiox12020401\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Methionine Sulfoxide Reductases Suppress the Formation of the [PSI+] Prion and Protein Aggregation in Yeast\", \"abstract\": \"Prions are self-propagating, misfolded forms of proteins associated with various neurodegenerative diseases in mammals and heritable traits in yeast. How prions form spontaneously into infectious amyloid-like structures without underlying genetic changes is poorly understood. Previous studies have suggested that methionine oxidation may underlie the switch from a soluble protein to the prion form. In this current study, we have examined the role of methionine sulfoxide reductases (MXRs) in protecting against de novo formation of the yeast [PSI+] prion, which is the amyloid form of the Sup35 translation termination factor. We show that [PSI+] formation is increased during normal and oxidative stress conditions in mutants lacking either one of the yeast MXRs (Mxr1, Mxr2), which protect against methionine oxidation by reducing the two epimers of methionine-S-sulfoxide. We have identified a methionine residue (Met124) in Sup35 that is important for prion formation, confirming that direct Sup35 oxidation causes [PSI+] prion formation. [PSI+] formation was less pronounced in mutants simultaneously lacking both MXR isoenzymes, and we show that the morphology and biophysical properties of protein aggregates are altered in this mutant. Taken together, our data indicate that methionine oxidation triggers spontaneous [PSI+] prion formation, which can be alleviated by methionine sulfoxide reductases.\", \"doi_y\": \"https://doi.org/10.3390/antiox12020401\", \"publication_date\": \"2023-02-07\"}, {\"work_id\": \"W4322615507\", \"doi_x\": \"10.1146/annurev-arplant-070122-032532\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"abstract\": \"Chloroplasts are the defining plant organelles with responsibility for photosynthesis and other vital functions. To deliver these functions, they possess a complex proteome comprising thousands of largely nucleus-encoded proteins. Composition of the proteome is controlled by diverse processes affecting protein translocation and degradation-our focus here. Most chloroplast proteins are imported from the cytosol via multiprotein translocons in the outer and inner envelope membranes (the TOC and TIC complexes, respectively), or via one of several noncanonical pathways, and then sorted by different systems to organellar subcompartments. Chloroplast proteolysis is equally complex, involving the concerted action of internal proteases of prokaryotic origin and the nucleocytosolic ubiquitin-proteasome system (UPS). The UPS degrades unimported proteins in the cytosol and chloroplast-resident proteins via chloroplast-associated protein degradation (CHLORAD). The latter targets the TOC apparatus to regulate protein import, as well as numerous internal proteins directly, to reconfigure chloroplast functions in response to developmental and environmental signals.\", \"doi_y\": \"https://doi.org/10.1146/annurev-arplant-070122-032532\", \"publication_date\": \"2023-05-22\"}, {\"work_id\": \"W4306835822\", \"doi_x\": \"10.1101/2022.10.19.512840\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The N-terminal executioner domains of NLR immune receptors are functionally conserved across major plant lineages\", \"abstract\": \"ABSTRACT Nucleotide-binding domain and leucine-rich repeat (NLR) proteins are a prominent class of intracellular immune receptors that are present across diverse plant lineages. However, our understanding of plant NLR structure and function is limited to the evolutionarily young flowering plant clade (angiosperms). Here, we describe an extended spectrum of NLR diversity across major plant lineages and demonstrate functional conservation of N-terminal \\u2018executioner\\u2019 domains that trigger immune responses. We show that broadly distributed CC (coiled-coil) and TIR (toll/interleukin-1 receptor) domains retain executioner function through trans-lineage activation of immune-related cell death in the model angiosperm Nicotiana benthamiana . Further examination of a CC subfamily specific to non-flowering lineages uncovered an essential N-terminal MAEPL motif with functional similarity to resistosome-forming CC-NLRs. Ectopic activation of the MAEPL-type CC in the divergent liverwort Marchantia polymorpha led to profound growth inhibition, defense gene activation, and signatures of cell death resembling CC activity in flowering plants. Moreover, comparative macroevolutionary transcriptomics in Marchantia and Nicotiana identified conserved CC responsive genes, providing further insight into the core aspects of CC function shared between flowering and non-flowering plants. Our findings highlight the need to understand NLR structure and function across the full spectrum of plant diversity.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.19.512840\", \"publication_date\": \"2022-10-19\"}, {\"work_id\": \"W4285491711\", \"doi_x\": \"10.1073/pnas.2205412119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation between the binding affinity and the conformational entropy of nanobody SARS-CoV-2 spike protein complexes\", \"abstract\": \"Camelid single-domain antibodies, also known as nanobodies, can be readily isolated from na\\u00efve libraries for specific targets but often bind too weakly to their targets to be immediately useful. Laboratory-based genetic engineering methods to enhance their affinity, termed maturation, can deliver useful reagents for different areas of biology and potentially medicine. Using the receptor binding domain (RBD) of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike protein and a na\\u00efve library, we generated closely related nanobodies with micromolar to nanomolar binding affinities. By analyzing the structure-activity relationship using X-ray crystallography, cryoelectron microscopy, and biophysical methods, we observed that higher conformational entropy losses in the formation of the spike protein-nanobody complex are associated with tighter binding. To investigate this, we generated structural ensembles of the different complexes from electron microscopy maps and correlated the conformational fluctuations with binding affinity. This insight guided the engineering of a nanobody with improved affinity for the spike protein.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2205412119\", \"publication_date\": \"2022-07-15\"}, {\"work_id\": \"W4386697797\", \"doi_x\": \"10.1088/1748-605x/acf976\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Human vascularised synovium-on-a-chip: a mechanically stimulated, microfluidic model to investigate synovial inflammation and monocyte recruitment\", \"abstract\": \"Healthy synovium is critical for joint homeostasis. Synovial inflammation (synovitis) is implicated in the onset, progression and symptomatic presentation of arthritic joint diseases such as rheumatoid arthritis and osteoarthritis. Thus, the synovium is a promising target for the development of novel, disease-modifying therapeutics. However, target exploration is hampered by a lack of good pre-clinical models that accurately replicate human physiology and that are developed in a way that allows for widespread uptake. The current study presents a multi-channel, microfluidic, organ-on-a-chip (OOAC) model, comprising a 3D configuration of the human synovium and its associated vasculature, with biomechanical and inflammatory stimulation, built upon a commercially available OOAC platform. Healthy human fibroblast-like synoviocytes (hFLS) were co-cultured with human umbilical vein endothelial cells (HUVECs) with appropriate matrix proteins, separated by a flexible, porous membrane. The model was developed within the Emulate organ-chip platform enabling the application of physiological biomechanical stimulation in the form of fluid shear and cyclic tensile strain. The hFLS exhibited characteristic morphology, cytoskeletal architecture and matrix protein deposition. Synovial inflammation was initiated through the addition of interleukin-1\\u03b2(IL-1\\u03b2) into the synovium channel resulting in the increased secretion of inflammatory and catabolic mediators, interleukin-6 (IL-6), prostaglandin E2 (PGE2), matrix metalloproteinase 1 (MMP-1), as well as the synovial fluid constituent protein, hyaluronan. Enhanced expression of the inflammatory marker, intercellular adhesion molecule-1 (ICAM-1), was observed in HUVECs in the vascular channel, accompanied by increased attachment of circulating monocytes. This vascularised human synovium-on-a-chip model recapitulates a number of the functional characteristics of both healthy and inflamed human synovium. Thus, this model offers the first human synovium organ-chip suitable for widespread adoption to understand synovial joint disease mechanisms, permit the identification of novel therapeutic targets and support pre-clinical testing of therapies.\", \"doi_y\": \"https://doi.org/10.1088/1748-605x/acf976\", \"publication_date\": \"2023-10-10\"}, {\"work_id\": \"W4323276421\", \"doi_x\": \"10.1016/j.jbc.2023.104579\", \"af_direct_citation\": 1, \"af_indirect_citations\": 11, \"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"abstract\": \"Protein science is being transformed by powerful computational methods for structure prediction and design: AlphaFold2 can predict many natural protein structures from sequence, and other AI methods are enabling the de novo design of new structures. This raises a question: how much do we understand the underlying sequence-to-structure/function relationships being captured by these methods? This perspective presents our current understanding of one class of protein assembly, the \\u03b1-helical coiled coils. At first sight, these are straightforward: sequence repeats of hydrophobic (h) and polar (p) residues, (hpphppp)n, direct the folding and assembly of amphipathic \\u03b1 helices into bundles. However, many different bundles are possible: they can have two or more helices (different oligomers); the helices can have parallel, antiparallel, or mixed arrangements (different topologies); and the helical sequences can be the same (homomers) or different (heteromers). Thus, sequence-to-structure relationships must be present within the hpphppp repeats to distinguish these states. I discuss the current understanding of this problem at three levels: first, physics gives a parametric framework to generate the many possible coiled-coil backbone structures. Second, chemistry provides a means to explore and deliver sequence-to-structure relationships. Third, biology shows how coiled coils are adapted and functionalized in nature, inspiring applications of coiled coils in synthetic biology. I argue that the chemistry is largely understood; the physics is partly solved, though the considerable challenge of predicting even relative stabilities of different coiled-coil states remains; but there is much more to explore in the biology and synthetic biology of coiled coils. Protein science is being transformed by powerful computational methods for structure prediction and design: AlphaFold2 can predict many natural protein structures from sequence, and other AI methods are enabling the de novo design of new structures. This raises a question: how much do we understand the underlying sequence-to-structure/function relationships being captured by these methods? This perspective presents our current understanding of one class of protein assembly, the \\u03b1-helical coiled coils. At first sight, these are straightforward: sequence repeats of hydrophobic (h) and polar (p) residues, (hpphppp)n, direct the folding and assembly of amphipathic \\u03b1 helices into bundles. However, many different bundles are possible: they can have two or more helices (different oligomers); the helices can have parallel, antiparallel, or mixed arrangements (different topologies); and the helical sequences can be the same (homomers) or different (heteromers). Thus, sequence-to-structure relationships must be present within the hpphppp repeats to distinguish these states. I discuss the current understanding of this problem at three levels: first, physics gives a parametric framework to generate the many possible coiled-coil backbone structures. Second, chemistry provides a means to explore and deliver sequence-to-structure relationships. Third, biology shows how coiled coils are adapted and functionalized in nature, inspiring applications of coiled coils in synthetic biology. I argue that the chemistry is largely understood; the physics is partly solved, though the considerable challenge of predicting even relative stabilities of different coiled-coil states remains; but there is much more to explore in the biology and synthetic biology of coiled coils. As a graduate student in the late 1980s, I was drawn to the challenge of the protein-folding problem and to using protein design as a means of testing our understanding of protein structure and folding. Even then, we suspected that solutions to these problems would come through bioinformatics. However, we did not realize how long it would take for solutions to come or the form that they would take in terms of combining big data, large computer power, and artificial intelligence, specifically, machine learning. Of course, I refer to the recent successes AlphaFold2 and RoseTTAfold in predicting protein structure from sequence alone (1AlQuraishi M. AlphaFold at CASP13.Bioinformatics. 2019; 35: 4862-4865Crossref PubMed Scopus (153) Google Scholar, 2Baek M. DiMaio F. Anishchenko I. Dauparas J. Ovchinnikov S. Lee G.R. et al.Accurate prediction of protein structures and interactions using a three-track neural network.Science. 2021; 373: 871-876Crossref PubMed Scopus (1186) Google Scholar, 3Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (6259) Google Scholar, 4Varadi M. Anyango S. Deshpande M. Nair S. Natassia C. Yordanova G. et al.AlphaFold protein structure database: massively expanding the structural coverage of protein-sequence space with high-accuracy models.Nucl. Acids Res. 2022; 50: D439-D444Crossref PubMed Scopus (1001) Google Scholar). Like many, when I look at an AlphaFold2 model for a complex protein assembly, I am awestruck by the structural solutions that it finds. However, while AlphaFold2 and RoseTTAfold provide solutions to the protein-folding problem\\u2014and, to be sure, these and other methods will improve to provide models for ever-more complex protein structures and assemblies\\u2014they are just that, solutions. In themselves, they do not necessarily, at least not at present, provide an understanding of the physics and chemistry that drives and directs protein folding and assembly, which, in turn, are responsible for protein function. Back in the 1980s and 1990s, it was the desire to understand protein folding that drove interest in the field; it was not just to find solutions to the problem per se. In short, we wanted to understand the physico-chemical principles that underpin protein structure, folding, assembly, and stability. In other words, we sought to decipher the underlying sequence-to-structure relationships for these properties. This perspective is about how far we have progressed in understanding protein folding and design in these terms. Well, at least for one particular protein structure\\u2014the \\u03b1-helical coiled coil. At first sight, this is a relatively straightforward peptide and protein assembly in which two or more \\u03b1-helical chains wrap around each other to form supercoiled or rope-like structures, Figure 1. At that time, coiled-coil\\u2013forming leucine-zipper peptides, Figure 1A, became models for protein folding and design (5Landschulz W.H. Johnson P.F. Mcknight S.L. The leucine zipper - a hypothetical structure common to a new class of DNA-binding proteins.Science. 1988; 240: 1759-1764Crossref PubMed Google Scholar, 6O\\u2019Shea E.K. Klemm J.D. Kim P.S. Alber T. X-ray structure of the GCN4 leucine zipper, a 2-stranded, parallel coiled coil.Science. 1991; 254: 539-544Crossref PubMed Google Scholar, 7O\\u2019Shea E.K. Rutkowski R. Kim P.S. Evidence that the leucine zipper is a coiled coil.Science. 1989; 243: 538-542Crossref PubMed Google Scholar). I was drawn to the folding and design of leucine zippers as a post-doc with Tom Alber in the early 1990s. This was because of the apparent simplicity of these coiled coils and the thought that unless we could understand such simple protein folds and assemblies, we would have no hope with more complex globular proteins. Therefore, I adopted coiled coils as a model for developing an understanding of sequence-to-structure relationships in proteins. An aim of this perspective is to capture some of that journey, which has been contributed to by many scientists in many groups over the past few decades. However, it is not a complete review of coiled-coil structure, biology, or even design. Such an article would be redundant because many excellent reviews are already available (8Lupas A. Coiled coils: new structures and new functions.Trends Biochem. Sci. 1996; 21: 375-382Abstract Full Text PDF PubMed Scopus (0) Google Scholar, 9Mason J.M. Arndt K.M. Coiled coil domains: stability, specificity, and biological implications.Chembiochem. 2004; 5: 170-176Crossref PubMed Scopus (546) Google Scholar, 10Burkhard P. Stetefeld J. Strelkov S.V. Coiled coils: a highly versatile protein folding motif.Trends Cell Biol. 2001; 11: 82-88Abstract Full Text Full Text PDF PubMed Scopus (850) Google Scholar, 11Rose A. Meier I. Scaffolds, levers, rods and springs: diverse cellular functions of long coiled-coil proteins.Cell Mol. Life Sci. 2004; 61: 1996-2009Crossref PubMed Google Scholar, 12Woolfson D.N. The design of coiled-coil structures and assemblies.Adv. Protein Chem. 2005; 70: 79-112Crossref PubMed Scopus (477) Google Scholar, 13Lupas A.N. Bassler J. Coiled coils - a model system for the 21st century.Trends Biochem. Sci. 2017; 42: 130-140Abstract Full Text Full Text PDF PubMed Scopus (127) Google Scholar, 14Woolfson D.N. Coiled-coil design: updated and upgraded.Subcell Biochem. 2017; 82: 35-61Crossref PubMed Scopus (90) Google Scholar), as are others covering newer topics such as the applications of natural and designed coiled coils in biotechnology and synthetic biology (15Lapenta F. Aupic J. Strmsek Z. Jerala R. Coiled coil protein origami: from modular design principles towards biotechnological applications.Chem. Soc. Rev. 2018; 47: 3530-3542Crossref PubMed Google Scholar, 16Beesley J.L. Woolfson D.N. The de novo design of alpha-helical peptides for supramolecular self-assembly.Curr. Opin. Biotech. 2019; 58: 175-182Crossref PubMed Scopus (0) Google Scholar, 17Rink W.M. Thomas F. De novo designed alpha-helical coiled-coil peptides as scaffolds for chemical reactions.Chem-eur J. 2019; 25: 1665-1677Crossref PubMed Scopus (9) Google Scholar, 18Utterstrom J. Naeimipour S. Selegard R. Aili D. Coiled coil-based therapeutics and drug delivery systems.Adv. Drug Deliv. Rev. 2021; 170: 26-43Crossref PubMed Scopus (0) Google Scholar). Instead, I want to give some sense of three things: first, of the journey and the joy of discovery, which have led to our current understanding of this relatively straightforward protein structure; second, that in some respects\\u2014for instance, the physics and chemistry of coiled-coil folding and assembly\\u2014our understanding is complete or very near to it; and third, despite this understanding, we still have much to learn, particularly on the biology of coiled coils. This article is my perspective on the amassed understanding of coiled-coil proteins. Indeed, it closes some of my own research questions; namely, what are the sequence-to-structure relationships that govern coiled-coil folding and assembly, and how do these allow us to design de novo coiled coils with confidence? However, as some research chapters close, others open. For coiled-coil research, new challenges include the following: achieving fully quantitative (free-energy) predictions for coiled-coil structure, stability, and partner selection; gaining a deeper understanding of coiled-coil dynamics and plasticity and how this relates to coiled-coil function. And, as one of the best understood protein folds, how can de novo coiled-coil peptides and proteins be used in biotechnology and synthetic biology to address real-world applications, for instance, in health and the environment. I am certain that the methods, principles, and understanding that have been developed over the past few decades will provide a foundation for these and other endeavors. And to come full circle, it is clear that new artificial intelligence/machine learning\\u2013based methods for protein-structure prediction will contribute here. Indeed, I see these tools as fantastic hypothesis generators for structural molecular and cell biology, including for processes that involve relatively simple, but adaptable and versatile coiled coils. For more complete historical perspectives on the conceptual origins of \\u03b1-helical coiled coils, please see reviews by Squire and Parry and by Lupas et al. (19Lupas A.N. Gruber M. The structure of alpha-helical coiled coils.Adv. Protein Chem. 2005; 70: 37-78Crossref PubMed Scopus (578) Google Scholar, 20Lupas A.N. Bassler J. Dunin-Horkawicz S. The structure and topology of alpha-helical coiled coils.Subcell Biochem. 2017; 82: 95-129Crossref PubMed Scopus (59) Google Scholar, 21Squire J.M. Parry D.A. Fibrous protein structures: hierarchy, history and heroes.Subcell Biochem. 2017; 82: 1-33Crossref PubMed Scopus (8) Google Scholar). The coiled coil came out of physics: Francis Crick\\u2019s description of the coiled coil, as he first named it, is mathematical (22Crick F.H.C. The fourier transform of a coiled-coil.Acta Crystallogr. 1953; 6: 685-689Crossref Google Scholar, 23Crick F.H.C. The packing of alpha-helices - simple coiled-coils.Acta Crystallogr. 1953; 6: 689-697Crossref Google Scholar). Moreover, at that time in the early 1950s, there was little experimental data or confirmed details for any protein structure. So, what followed\\u2014namely, the first description of any protein structure, the concepts of helical nets and knobs-into-holes (KIH) packing, heptad repeats, and what we now refer to as the Crick Equations\\u2014was extremely insightful. Incidentally, Crick published this work in the same year that he and James Watson proposed the double helix for the structure of DNA (24Watson J.D. Crick F.H.C. Molecular structure of nucleic acids - a structure for deoxyribose nucleic acid.Nature. 1953; 171: 737-738Crossref PubMed Scopus (0) Google Scholar). Crick started with Linus Pauling\\u2019s \\u03b1 helix and its regularity (Fig. 2A) (25Pauling L. Corey R.B. Branson H.R. The structure of proteins - 2 hydrogen-bonded helical configurations of the polypeptide chain.Pro. Natl. Acad. Sci. U. S. A. 1951; 37: 205-211Crossref PubMed Google Scholar). As a consequence of steric constraints in polypeptide chains\\u2014as later formalized by Ramachandran in his famous plot (26Ramachandran G.N. Ramakrishnan C. Sasisekharan V. Stereochemistry of polypeptide chain configurations.J. Mol. Biol. 1963; 7: 95-99Crossref PubMed Scopus (2715) Google Scholar, 27Ramachandran G.N. Venkatachalam C.M. Krimm S. Stereochemical criteria for polypeptide and protein chain conformations .3. Helical and hydrogen-bonded polypeptide chains.Biophys. J. 1966; 6: 849-872Abstract Full Text PDF PubMed Google Scholar)\\u2014the \\u03b1 helix has precisely 3.6 residues per turn (Fig. 2A). Crick reasoned that two or more such helices could interact tightly via seams of interacting side chains spaced three and four residues apart along polypeptide chains\\u2014i.e., with an average spacing of 3.5 residues\\u2014to match the 3.6 residues per turn as closely as possible. Now, we annotate these repeats abcdefg with the key interacting side chains falling at the a and d sites. Crick visualized this with helical-net diagrams (Fig. 2B). Highlighting the 3,4 spacing on one of these reveals the seam as the line of connected diamond shapes. Two such seams can interlace to bring the helices into intimate contact (Fig. 2C): the diamonds on one helix form \\u2018holes\\u2019 into which side chains from the other helix can slot. In this way, Crick coined the terms \\u2018heptad repeat\\u2019 (from the Greek for seven) and KIH packing for these sequence and structural features, respectively. These are now the hallmarks for coiled-coil peptides and proteins, and they provide a firm basis for understanding them. The term \\u2018coiled coil\\u2019 is a consequence of these sequence patterns and the structural packing. This is because 3.5 is less than 3.6. As a result, on a helical net, the seam slants to the left (Fig. 2B). Thus, for two nets to interlace requires one to be offset at an angle to the other (Fig. 2C). In a 3D \\u03b1 helix, the seam precesses around the surface of the helix with the opposite sense to the handedness of the helix. Thus, for two \\u03b1 helices to maintain contact, they must wrap around each other like the strands of a rope. As \\u03b1 helices made from l-amino acids are right handed, heptad-based coiled coils are left-handed ropes. This overall assembly\\u2014which is a quaternary structure and not a tertiary structure\\u2014is also helical. Strictly, it is a superhelix. This is why Crick called the envisaged structures coiled coils; \\u201ccoils\\u201d refers to the \\u03b1 helices, and \\u201ccoiled\\u201d refers to the superhelix. From Pauling\\u2019s \\u03b1-helical parameters\\u20143.6 residues per turn and a rise per residues of 1.5 \\u00c5\\u2014Crick predicted that the crossing angle between the two helices would be \\u224820\\u02da, which gives a superhelical pitch of \\u2248126 residues or \\u2248186 \\u00c5 (Box 1). (Note: these values correspond to those observed in experimentally determined coiled coils (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar)). This is formalized in the Crick Equations. Thus, the coiled coil is inherently parametric, which makes it physics, predictable, modelable and, as, we will see, ultimately designable (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar). Indeed, this formalism has led to a swathe of coiled-coil modeling and design programs (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar, 30Harbury P.B. Tidor B. Kim P.S. Repacking protein cores with backbone freedom - structure prediction for coiled coils.Proc. Natl. Acad. Sci. U. S. A. 1995; 92: 8408-8412Crossref PubMed Scopus (0) Google Scholar, 31Offer G. Hicks M.R. Woolfson D.N. Generalized crick equations for modeling noncanonical coiled coils.J. Struct. Biol. 2002; 137: 41-53Crossref PubMed Scopus (51) Google Scholar, 32Dunin-Horkawicz S. Lupas A.N. Measuring the conformational space of square four-helical bundles with the program samCC.J. Struct. Biol. 2010; 170: 226-235Crossref PubMed Scopus (33) Google Scholar, 33Huang P.S. Oberdorfer G. Xu C.F. Pei X.Y. Nannenga B.L. Rogers J.M. et al.High thermodynamic stability of parametrically designed helical bundles.Science. 2014; 346: 481-485Crossref PubMed Scopus (192) Google Scholar, 34Thomson A.R. Wood C.W. Burton A.J. Bartlett G.J. Sessions R.B. Brady R.L. et al.Computational design of water-soluble alpha-helical barrels.Science. 2014; 346: 485-488Crossref PubMed Scopus (0) Google Scholar, 35Wood C.W. Heal J.W. Thomson A.R. Bartlett G.J. Ibarra A.A. Brady R.L. et al.ISAMBARD: an open-source computational environment for biomolecular analysis, modelling and design.Bioinformatics. 2017; 33: 3043-3050Crossref PubMed Scopus (33) Google Scholar, 36Wood C.W. Woolfson D.N. CCBuilder 2.0: powerful and accessible coiled-coil modeling.Protein Sci. 2018; 27: 103-111Crossref PubMed Scopus (69) Google Scholar). Several of these are accessible and easy-to-use computational tools on the internet (Box 2). Some allow coiled-coil backbones to be built quickly and accurately (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 31Offer G. Hicks M.R. Woolfson D.N. Generalized crick equations for modeling noncanonical coiled coils.J. Struct. Biol. 2002; 137: 41-53Crossref PubMed Scopus (51) Google Scholar), and others allow full atomistic modeling of coiled coils including side chains (29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar, 33Huang P.S. Oberdorfer G. Xu C.F. Pei X.Y. Nannenga B.L. Rogers J.M. et al.High thermodynamic stability of parametrically designed helical bundles.Science. 2014; 346: 481-485Crossref PubMed Scopus (192) Google Scholar, 35Wood C.W. Heal J.W. Thomson A.R. Bartlett G.J. Ibarra A.A. Brady R.L. et al.ISAMBARD: an open-source computational environment for biomolecular analysis, modelling and design.Bioinformatics. 2017; 33: 3043-3050Crossref PubMed Scopus (33) Google Scholar, 36Wood C.W. Woolfson D.N. CCBuilder 2.0: powerful and accessible coiled-coil modeling.Protein Sci. 2018; 27: 103-111Crossref PubMed Scopus (69) Google Scholar). Furthermore, there are tools for assessing the quality of such models and experimental structures parametrically and energetically (32Dunin-Horkawicz S. Lupas A.N. Measuring the conformational space of square four-helical bundles with the program samCC.J. Struct. Biol. 2010; 170: 226-235Crossref PubMed Scopus (33) Google Scholar, 37Walshaw J. Woolfson D.N. Socket: a program for identifying and analysing coiled-coil motifs within protein structures.J. Mol. Biol. 2001; 307: 1427-1450Crossref PubMed Scopus (306) Google Scholar, 38Strelkov S.V. Burkhard P. Analysis of alpha-helical coiled coils with the program TWISTER reveals a structural mechanism for stutter compensation.J. Struct. Biol. 2002; 137: 54-64Crossref PubMed Scopus (212) Google Scholar, 39Kumar P. Woolfson D.N. Socket2: a program for locating, visualizing and analyzing coiled-coil interfaces in protein structures.Bioinformatics. 2021; 37: 4575-4577Crossref PubMed Scopus (12) Google Scholar). This has made coiled-coil modeling, engineering, and design accessible to nonexperts, which is a significant advance. In turn, this has enabled the exploration of coiled-coil assemblies beyond the confines of natural proteins (34Thomson A.R. Wood C.W. Burton A.J. Bartlett G.J. Sessions R.B. Brady R.L. et al.Computational design of water-soluble alpha-helical barrels.Science. 2014; 346: 485-488Crossref PubMed Scopus (0) Google Scholar, 40Dawson W.M. Martin F.J.O. Rhys G.G. Shelley K.L. Brady R.L. Woolfson D.N. Coiled coils 9-to-5: rational de novo design of alpha-helical barrels with tunable oligomeric states.Chem. Sci. 2021; 12: 6923-6928Crossref PubMed Google Scholar).Box 1From \\u03b1-helical to coiled-coil parametersThe following exercise can be done by considering projections of the \\u03b1 helix as helical nets or helical wheels.First, I need to dispel a key misconception made by students and professors alike. Pauling\\u2019s \\u03b1 helix has 3.6 residues per turn with a very low tolerance of variation (29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar); indeed, this is one of the few fixed physical constants in biology (52Kumar P. Paterson N.G. Clayden J. Woolfson D.N. De novo design of discrete, stable 310-helix peptide assemblies.Nature. 2022; 607: 387-392Crossref PubMed Scopus (0) Google Scholar). This is because the energy well for the \\u03b1 helix is narrow and deep (53Kuster D.J. Liu C.Y. Fang Z. Ponder J.W. Marshall G.R. High-resolution crystal structures of protein helices reconciled with three-centered hydrogen bonds and multipole electrostatics.PLoS One. 2015; 10: e0123146Crossref PubMed Scopus (23) Google Scholar), and small deviations from \\u03b1-helical parameters incur large energy penalties. Indeed, the nearby 310 and \\u03c0 helixes are rare in protein structures and difficult to design (52Kumar P. Paterson N.G. Clayden J. Woolfson D.N. De novo design of discrete, stable 310-helix peptide assemblies.Nature. 2022; 607: 387-392Crossref PubMed Scopus (0) Google Scholar). As a result, the \\u03b1 helix does not somehow collapse to 3.5 residues per turn in coiled coils. If it did, we would not have coiled coils: the average sequence repeat of 3.5 and a helical repeat of 3.5 would match, and the helices would pack straight and not wrap around each other. I suspect that the convenience of 7-residue helical wheels (Fig. 2D) rather than the more accurate 18-residues helical wheels (Fig. 2E) has contributed to this misconception. Crick\\u2019s helical nets are a more faithful mapping of an \\u03b1-helical surface in 2D (Fig. 2B).With the C\\u03b1 atoms of an \\u03b1 helix projected on an 18-residue helical wheel, successive residues are separated by 100\\u02da. Thus, the seven residues of a heptad repeat would span out 700\\u02da. This is 20\\u02da short of two full turns (2 \\u00d7 360\\u02da = 720\\u02da and 7.2 residues) of the \\u03b1 helix. 20\\u02da goes into 360\\u02da 18 times. Therefore, 18 heptad repeats are required for the interacting seam to make one complete revolution of a helix and to bring the helices back into sync. This defines the pitch of the coiled coil. Hence, the pitch is 18 \\u00d7 7 = 126 residues. Given that the \\u03b1 helix has a rise per residue of 1.5 \\u00c5, these would span 189 \\u00c5 of a straight \\u03b1 helix. However, each \\u03b1 helix is inclined by \\u224810\\u02da relative to the superhelical axis. Therefore, the rise per residue along this coiled-coil axis is 1.5 \\u00d7 cos(10\\u02da) = 1.48 \\u00c5, and the ideal superhelical pitch is \\u2248186 \\u00c5 (Fig. 2F). In addition to the rise per residue and the coiled-coil pitch, just two other parameters are needed to define and generate regular coiled-coil backbones. These are the radius of the coiled coil and the interface or Crick angle, Figure 2G.Finally for this Box, we should also credit Pauling. Crick considered only 7-residue repeats. However, Pauling et al. considered other repeats in \\u03b1-helical conformations, specifically, those with 11, 15, and 18 residues (25Pauling L. Corey R.B. Branson H.R. The structure of proteins - 2 hydrogen-bonded helical configurations of the polypeptide chain.Pro. Natl. Acad. Sci. U. S. A. 1951; 37: 205-211Crossref PubMed Google Scholar, 54Pauling L. Corey R.B. The structure of synthetic polypeptides.Proc. Natl. Acad. Sci. U. S. A. 1951; 37: 241-250Crossref PubMed Scopus (0) Google Scholar). Along with 7-residue repeats, these are compatible with combinations of 3- and 4-residue spacings; namely, 3-4, 3-4-4, 3-4-4-4, and 3-4-3-4-4, respectively. However, they have different average spacings between the interacting residues of 3.5, 3.67, 3.75, and 3.6, respectively. As the \\u03b1-helical structural repeat is fixed at 3.6 residues per turn, when these repeats are realized in packed coiled coils, they lead to different superhelical twists. These can also be calculated using the considerations laid out above. They result in further left-handed, some right-handed, and even \\u201cstraight\\u201d coiled coils (19Lupas A.N. Gruber M. The structure of alpha-helical coiled coils.Adv. Protein Chem. 2005; 70: 37-78Crossref PubMed Scopus (578) Google Scholar, 55Brown J.H. Cohen C. Parry D.A.D. Heptad breaks in alpha-helical coiled coils: stutters and stammers.Proteins-Struct. Funct. Genet. 1996; 26: 134-145Crossref PubMed Scopus (0) Google Scholar, 56Hicks M.R. Holberton D.V. Kowalczyk C. Woolfson D.N. Coiled-coil assembly by peptides with non-heptad sequence motifs.Fold Des. 1997; 2: 149-158Abstract Full Text Full Text PDF PubMed Google Scholar, 57Harbury P.B. Plecs J.J. Tidor B. Alber T. Kim P.S. High-resolution protein design with backbone freedom.Science. 1998; 282: 1462-1467Crossref PubMed Google Scholar, 58Hicks M.R. Walshaw J. Woolfson D.N. Investigating the tolerance of coiled-coil peptides to nonheptad sequence inserts.J. Struct. Biol. 2002; 137: 73-81Crossref PubMed Scopus (26) Google Scholar).Box 2Tools and resources for predicting, building, analyzing, and visualizing coiled-coil structuresPredictionSeveral tools for predicting coiled-coil structure from sequence are brought together and implemented at Andrei Lupas\\u2019s Max Planck Institute Bioinformatics Toolkit (59Gabler F. Nam S.Z. Till S. Mirdita M. Steinegger M. Soding J. et al.Protein sequence analysis using the MPI bioinformatics toolkit.Curr. Protoc. Bioinform. 2020; 72: e108Crossref PubMed Scopus (196) Google Scholar) (https://toolkit.tuebingen.mpg.de/). These include the following: Lupas\\u2019s original COILS program (60Lupas A. Vandyke M. Stock J. Predicting coiled coils from protein sequences.Science. 1991; 252: 1162-1164Crossref PubMed Google Scholar) (now PCOIL), DeepCoil (61Ludwiczak J. Winski A. Szczepaniak K. Alva V. Dunin-Horkawicz S. DeepCoil-a fast and accurate prediction of coiled-coil domains in protein sequences.Bioinformatics. 2019; 35: 2790-2795Crossref PubMed Scopus (0) Google Scholar), and MARCOIL (62Delorenzi M. Speed T. An HMM model for coiled-coil domains and a comparison with PSSM-based predictions.Bioinformatics. 2002; 18: 617-625Crossref PubMed Google Scholar). Regarding predicting oligomeric states from sequence, in light of recent advances in coiled-coil design and protein-structure prediction, generally, there is considerable room for improvement here. However, the following are currently available: LOGICOIL (63Vincent T.L. Green P.J. Woolfson D.N. LOGICOIL-multi-state prediction of coiled-coil oligomeric state.Bioinformatics. 2013; 29: 69-76Crossref PubMed Scopus (0) Google Scholar) (http://coiledcoils.chm.bris.ac.uk/LOGICOIL/) and Multicoil2 (64Trigg J. Gutwin K. Keating A.E. Berger B. Multicoil2: predicting coiled coils and their oligomerization states from sequence in the twilight zone.PLoS One. 2011; 6: e23519Crossref PubMed Scopus (60) Google Scholar) (http://cb.csail.mit.edu/cb/multicoil2/cgi-bin/multicoil2.cgi). AlphaFold2 (3Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (6259) Google Scholar) also predi\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.104579\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4384435971\", \"doi_x\": \"10.1016/j.sbi.2023.102652\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Mechanism of primer synthesis by Primase-Polymerases\", \"abstract\": \"Members of the primase-polymerase (Prim-Pol) superfamily are found in all domains of life and play diverse roles in genome stability, including primer synthesis during DNA replication, lesion repair and damage tolerance. This review focuses primarily on Prim-Pol members capable of de novo primer synthesis that have experimentally derived structural models available. We discuss the mechanism of DNA primer synthesis initiation by Prim-Pol catalytic domains, based on recent structural and functional studies. We also describe a general model for primer initiation that also includes the ancillary domains/subunits, which stimulate the initiation of primer synthesis.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102652\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4307563852\", \"doi_x\": \"10.1085/jgp.202213210\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"abstract\": \"The flux of ions through a channel is most commonly regulated by changes that result in steric occlusion of its pore. However, ion permeation can also be prevented by formation of a desolvation barrier created by hydrophobic residues that line the pore. As a result of relatively minor structural changes, confined hydrophobic regions in channels may undergo transitions between wet and dry states to gate the pore closed without physical constriction of the permeation pathway. This concept is referred to as hydrophobic gating, and many examples of this process have been demonstrated. However, the term is also now being used in a much broader context that often deviates from its original meaning. In this Viewpoint, we explore the formal definition of a hydrophobic gate, discuss examples of this process compared with other gating mechanisms that simply exploit hydrophobic residues and/or lipids in steric closure of the pore, and describe the best practice for identification of a hydrophobic gate.\", \"doi_y\": \"https://doi.org/10.1085/jgp.202213210\", \"publication_date\": \"2022-10-26\"}, {\"work_id\": \"W4296078744\", \"doi_x\": \"10.1126/sciadv.abp9660\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"abstract\": \"Despite continuing progress in kinesin enzyme mechanochemistry and emerging understanding of the cargo recognition machinery, it is not known how these functions are coupled and controlled by the \\u03b1-helical coiled coils encoded by a large component of kinesin protein sequences. Here, we combine computational structure prediction with single-particle negative-stain electron microscopy to reveal the coiled-coil architecture of heterotetrameric kinesin-1 in its compact state. An unusual flexion in the scaffold enables folding of the complex, bringing the kinesin heavy chain-light chain interface into close apposition with a tetrameric assembly formed from the region of the molecule previously assumed to be the folding hinge. This framework for autoinhibition is required to uncover how engagement of cargo and other regulatory factors drives kinesin-1 activation.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.abp9660\", \"publication_date\": \"2022-09-16\"}, {\"work_id\": \"W4386801812\", \"doi_x\": \"10.1021/acscatal.3c03047\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Decoding Catalysis by Terpene Synthases\", \"abstract\": \"The review by Christianson, published in 2017 on the twentieth anniversary of the emergence of the field, summarizes the foundational discoveries and key advances in terpene synthase/cyclase (TS) biocatalysis (Christianson, D. W. Chem Rev2017, 117 (17), 11570-11648. DOI: 10.1021/acs.chemrev.7b00287). Here, we review the TS literature published since then, bringing the field up to date and looking forward to what could be the near future of TS rational design. Many revealing discoveries have been made in recent years, building on the knowledge and fundamental principles uncovered during those initial two decades of study. We use these to explore TS reaction chemistry and see how a combined experimental and computational approach helps to decipher the complexities of TS catalysis. Revealed are a suite of catalytic motifs which control product outcome in TSs, some obvious, some more subtle. We examine each in detail, using the most recent papers and insights to illustrate how exactly this fascinating class of enzymes takes a single acyclic substrate and turns it into the many thousands of complex terpenoids found in Nature. We then explore some of the recent strategies for TS engineering, including machine learning and other data-driven approaches. From this, rational and predictive engineering of TSs, \\\"designer terpene synthases\\\", will begin to emerge as a realistic goal.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c03047\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4296642242\", \"doi_x\": \"10.1002/prot.26429\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"abstract\": \"Bacteria are equipped with a diverse set of regulatory tools that allow them to quickly adapt to their environment. The RimK system allows for Pseudomonas spp. to adapt through post-transcriptional regulation by altering the ribosomal subunit RpsF. RimK is found in a wide range of bacteria with a conserved amino acid sequence, however, the genetic context and the role of this protein is highly diverse. By solving and comparing the structures of RimK homologs from two related but functionally divergent systems, we uncovered key structural differences that likely contribute to the different activity levels of each of these homologs. Moreover, we were able to clearly resolve the active site of this protein for the first time, resolving binding of the glutamate substrate. This work advances our understanding of how subtle differences in protein sequence and structure can have profound effects on protein activity, which can in turn result in widespread mechanistic changes.\", \"doi_y\": \"https://doi.org/10.1002/prot.26429\", \"publication_date\": \"2022-10-06\"}, {\"work_id\": \"W4319081171\", \"doi_x\": \"10.1101/2023.02.02.526626\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"abstract\": \"Abstract Mitochondrial ATP synthases form rows of dimers, which induce membrane curvature to give cristae their characteristic lamellar or tubular morphology. The angle formed between the central stalks of ATP synthase dimers varies between species. Using cryo-electron tomography and sub-tomogram averaging, we determined the structure of the ATP synthase dimer from the nematode worm C. elegans and show that the dimer angle differs from previously determined structures. The consequences of this species-specific difference at the dimer interface were investigated by comparing C. elegans and S. cerevisiae mitochondrial morphology. We reveal that C. elegans has a larger ATP synthase dimer angle with more lamellar (flatter) cristae when compared to yeast. The underlying cause of this difference was investigated by generating an atomic model of the C. elegans ATP synthase dimer by homology modelling. A comparison of our C. elegans model to an existing S. cerevisiae structure reveals the presence of extensions and rearrangements in C. elegans subunits associated with maintaining the dimer interface. We speculate that increasing dimer angles could provide an advantage for species that inhabit variable-oxygen environments by forming flatter more energetically efficient cristae.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.02.526626\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4308059022\", \"doi_x\": \"10.1182/blood.2021014668\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Single-cell methods in myeloproliferative neoplasms: old questions, new technologies\", \"abstract\": \"Myeloproliferative neoplasms (MPN) are a group of clonal stem cell-derived hematopoietic malignancies driven by aberrant Janus kinase-signal transducer and activator of transcription proteins (JAK/STAT) signaling. Although these are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intratumoral heterogeneity driven by the interplay of genetic and nongenetic factors. Their evolution is determined by factors that enable certain cellular subsets to outcompete others. Therefore, techniques that resolve cellular heterogeneity at the single-cell level are ideally placed to provide new insights into MPN biology. With these insights comes the potential to uncover new approaches to predict the clinical course and treat these cancers, ultimately improving outcomes for patients. MPNs present a particularly tractable model of cancer evolution, because most patients present in an early disease phase and only a small proportion progress to aggressive disease. Therefore, it is not surprising that many groundbreaking technological advances in single-cell omics have been pioneered by their application in MPNs. In this review article, we explore how single-cell approaches have provided transformative insights into MPN disease biology, which are broadly applicable across human cancers, and discuss how these studies might be swiftly translated into clinical pathways and may eventually underpin precision medicine.\", \"doi_y\": \"https://doi.org/10.1182/blood.2021014668\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W4309048142\", \"doi_x\": \"10.1128/mbio.02541-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"abstract\": \"The human pathogen Pseudomonas aeruginosa (Pa) is one of the most frequent and severe causes of nosocomial infection. This organism is also a major cause of airway infections in people with cystic fibrosis (CF). Pa is known to have a remarkable metabolic plasticity, allowing it to thrive under diverse environmental conditions and ecological niches; yet, little is known about the central metabolic pathways that sustain its growth during infection or precisely how these pathways operate. In this work, we used a combination of 'omics approaches (transcriptomics, proteomics, metabolomics, and 13C-fluxomics) and reverse genetics to provide systems-level insight into how the infection-relevant organic acids succinate and propionate are metabolized by Pa. Moreover, through structural and kinetic analysis of the 2-methylcitrate synthase (2-MCS; PrpC) and its paralogue citrate (CIT) synthase (GltA), we show how these two crucial enzymatic steps are interconnected in Pa organic acid assimilation. We found that Pa can rapidly adapt to the loss of GltA function by acquiring mutations in a transcriptional repressor, which then derepresses prpC expression. Our findings provide a clear example of how \\\"underground metabolism,\\\" facilitated by enzyme substrate promiscuity, \\\"rewires\\\" Pa metabolism, allowing it to overcome the loss of a crucial enzyme. This pathogen-specific knowledge is critical for the advancement of a model-driven framework to target bacterial central metabolism. IMPORTANCE Pseudomonas aeruginosa is an opportunistic human pathogen that, due to its unrivalled resistance to antibiotics, ubiquity in the built environment, and aggressiveness in infection scenarios, has acquired the somewhat dubious accolade of being designated a \\\"critical priority pathogen\\\" by the WHO. In this work, we uncover the pathways and mechanisms used by P. aeruginosa to grow on a substrate that is abundant at many infection sites: propionate. We found that if the organism is prevented from metabolizing propionate, the substrate turns from being a convenient nutrient source into a potent poison, preventing bacterial growth. We further show that one of the enzymes involved in these reactions, 2-methylcitrate synthase (PrpC), is promiscuous and can moonlight for another essential enzyme in the cell (citrate synthase). Indeed, mutations that abolish citrate synthase activity (which would normally prevent the cell from growing) can be readily overcome if the cell acquires additional mutations that increase the expression of PrpC. This is a nice example of the evolutionary utility of so-called \\\"underground metabolism.\\\"\", \"doi_y\": \"https://doi.org/10.1128/mbio.02541-22\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4213296721\", \"doi_x\": \"10.3389/fmolb.2022.834011\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"abstract\": \"Since pH sensitivity has a fundamental role in biology, much effort has been committed to establishing physical models to rationalize and predict pH dependence from molecular structures. Two of the key challenges are to accurately calculate ionizable group solvation and hydration and then to apply this modeling to all conformations relevant to the process in question. Explicit solvent methods coupled to molecular dynamics simulation are increasingly complementing lower resolution implicit solvent techniques, but equally, the scale of biological data acquisition leaves a role for high-throughput modeling. Additionally, determination of ranges of structures for a system allows sampling of key stages in solvation. In a review of the area, it is emphasized that pH sensors in biology beyond the most obvious candidate (histidine side chain, with an unshifted p K a near neutral pH) should be considered; that modeling can benefit from other concepts in bioinformatics, in particular modulation of interactions and function in families of homologs; and that it can also be beneficial to incorporate as many experimental structures as possible, to mitigate against small variations in conformation and to analyze larger, functional, conformational changes. These aspects are then demonstrated with new work on the spike protein of SARS-CoV-2, looking at the pH dependence of variants, including prediction of a change in the balance of locked, closed, and open forms at neutral pH for the Omicron variant spike protein.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.834011\", \"publication_date\": \"2022-02-18\"}, {\"work_id\": \"W4282583104\", \"doi_x\": \"10.1002/cbic.202200171\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"abstract\": \"The carbon backbone of biotin is constructed from the C7 di-acid pimelate, which is converted to an acyl-CoA thioester by an ATP-dependent, pimeloyl-CoA synthetase (PCAS, encoded by BioW). The acyl-thioester is condensed with \\u029f-alanine in a decarboxylative, Claisen-like reaction to form an aminoketone (8-amino-7-oxononanoic acid, AON). This step is catalysed by the pyridoxal 5'-phosphate (PLP)-dependent enzyme (AON synthase, AONS, encoded by BioF). Distinct versions of Bacillus subtilis BioW (BsBioW) and E. coli BioF (EcBioF) display strict substrate specificity. In contrast, a BioW-BioF fusion from Corynebacterium amycolatum (CaBioWF) accepts a wider range of mono- and di-fatty acids. Analysis of the active site of the BsBioW : pimeloyl-adenylate complex suggested a key role for a Phe (F192) residue in the CaBioW domain; a F192Y mutant restored the substrate specificity to pimelate. This surprising substrate flexibility also extends to the CaBioF domain, which accepts \\u029f-alanine, \\u029f-serine and glycine. Structural models of the CaBioWF fusion provide insight into how both domains interact with each other and suggest the presence of an intra-domain tunnel. The CaBioWF fusion catalyses conversion of various fatty acids and amino acids to a range of AON derivatives. Such unexpected, natural broad substrate scope suggests that the CaBioWF fusion is a versatile biocatalyst that can be used to prepare a number of aminoketone analogues.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202200171\", \"publication_date\": \"2022-07-13\"}, {\"work_id\": \"W4288051747\", \"doi_x\": \"10.1038/s41589-022-01088-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A new dawn beyond lysine ubiquitination\", \"abstract\": \"The ubiquitin system has become synonymous with the modification of lysine residues. However, the substrate scope and diversity of the conjugation machinery have been underappreciated, bringing us to an epoch in ubiquitin system research. The striking discoveries of metazoan enzymes dedicated toward serine and threonine ubiquitination have revealed the important role of nonlysine ubiquitination in endoplasmic reticulum\\u2013associated degradation, immune signaling and neuronal processes, while reports of nonproteinaceous substrates have extended ubiquitination beyond the proteome. Bacterial effectors that bypass the canonical ubiquitination machinery and form unprecedented linkage chemistry further redefine long-standing dogma. While chemical biology approaches have advanced our understanding of the canonical ubiquitin system, further study of noncanonical ubiquitination has been hampered by a lack of suitable tools. This Perspective aims to consolidate and contextualize recent discoveries and to propose potential applications of chemical biology, which will be instrumental in unraveling this new frontier of ubiquitin research. Emerging evidence shows that ubiquitination can occur on nonlysine residues and nonproteinaceous substrates. This perspective summarizes the recent discoveries in noncanonical ubiquitination and advocates the development of chemical biology tools for new substrate identification and mechanistic dissection.\", \"doi_y\": \"https://doi.org/10.1038/s41589-022-01088-2\", \"publication_date\": \"2022-07-27\"}, {\"work_id\": \"W4360843077\", \"doi_x\": \"10.1016/j.jmb.2023.168060\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"abstract\": \"In 2019, we released Missense3D which identifies stereochemical features that are disrupted by a missense variant, such as introducing a buried charge. Missense3D analyses the effect of a missense variant on a single structure and thus may fail to identify as damaging surface variants disrupting a protein interface i.e., a protein-protein interaction (PPI) site. Here we present Missense3D-PPI designed to predict missense variants at PPI interfaces. Our development dataset comprised of 1,279 missense variants (pathogenic n = 733, benign n = 546) in 434 proteins and 545 experimental structures of PPI complexes. Benchmarking of Missense3D-PPI was performed after dividing the dataset in training (320 benign and 320 pathogenic variants) and testing (226 benign and 413 pathogenic). Structural features affecting PPI, such as disruption of interchain bonds and introduction of unbalanced charged interface residues, were analysed to assess the impact of the variant at PPI. The performance of Missense3D-PPI was superior to that of Missense3D: sensitivity 44 % versus 8% and accuracy 58% versus 40%, p = 4.23 \\u00d7 10-16. However, the specificity of Missense3D-PPI was lower compared to Missense3D (84% versus 98%). On our dataset, Missense3D-PPI's accuracy was superior to BeAtMuSiC (p = 3.4 \\u00d7 10-5), mCSM-PPI2 (p = 1.5 \\u00d7 10-12) and MutaBind2 (p = 0.0025). Missense3D-PPI represents a valuable tool for predicting the structural effect of missense variants on biological protein networks and is available at the Missense3D web portal (http://missense3d.bc.ic.ac.uk).\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168060\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4318049405\", \"doi_x\": \"10.1515/hsz-2022-0314\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Super-resolution microscopy and studies of peroxisomes\", \"abstract\": \"Fluorescence microscopy is an important tool for studying cellular structures such as organelles. Unfortunately, many details in the corresponding images are hidden due to the resolution limit of conventional lens-based far-field microscopy. An example is the study of peroxisomes, where important processes such as molecular organization during protein important can simply not be studied with conventional far-field microscopy methods. A remedy is super-resolution fluorescence microscopy, which is nowadays a well-established technique for the investigation of inner-cellular structures but has so far to a lesser extent been applied to the study of peroxisomes. To help advancing the latter, we here give an overview over the different super-resolution microscopy approaches and their potentials and challenges in cell-biological research, including labelling issues and a focus on studies on peroxisomes. Here, we also highlight experiments beyond simple imaging such as observations of diffusion dynamics of peroxisomal proteins.\", \"doi_y\": \"https://doi.org/10.1515/hsz-2022-0314\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W3198923619\", \"doi_x\": \"10.1002/prot.26237\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"abstract\": \"Critical assessment of structure prediction (CASP) is a community experiment to advance methods of computing three-dimensional protein structure from amino acid sequence. Core components are rigorous blind testing of methods and evaluation of the results by independent assessors. In the most recent experiment (CASP14), deep-learning methods from one research group consistently delivered computed structures rivaling the corresponding experimental ones in accuracy. In this sense, the results represent a solution to the classical protein-folding problem, at least for single proteins. The models have already been shown to be capable of providing solutions for problematic crystal structures, and there are broad implications for the rest of structural biology. Other research groups also substantially improved performance. Here, we describe these results and outline some of the many implications. Other related areas of CASP, including modeling of protein complexes, structure refinement, estimation of model accuracy, and prediction of inter-residue contacts and distances, are also described.\", \"doi_y\": \"https://doi.org/10.1002/prot.26237\", \"publication_date\": \"2021-10-07\"}, {\"work_id\": \"W4385443358\", \"doi_x\": \"10.1021/acs.jmedchem.3c01114\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"5-Substituted Pyridine-2,4-dicarboxylate Derivatives Have Potential for Selective Inhibition of Human Jumonji-C Domain-Containing Protein 5\", \"abstract\": \"Jumonji-C domain-containing protein 5 (JMJD5) is a 2-oxoglutarate (2OG)-dependent oxygenase that plays important roles in development, circadian rhythm, and cancer through unclear mechanisms. JMJD5 has been reported to have activity as a histone protease, as an N\\u03b5-methyl lysine demethylase, and as an arginine residue hydroxylase. Small-molecule JMJD5-selective inhibitors will be useful for investigating its (patho)physiological roles. Following the observation that the broad-spectrum 2OG oxygenase inhibitor pyridine-2,4-dicarboxylic acid (2,4-PDCA) is a 2OG-competing JMJD5 inhibitor, we report that 5-aminoalkyl-substituted 2,4-PDCA derivatives are potent JMJD5 inhibitors manifesting selectivity for JMJD5 over other human 2OG oxygenases. Crystallographic analyses with five inhibitors imply induced fit binding and reveal that the 2,4-PDCA C5 substituent orients into the JMJD5 substrate-binding pocket. Cellular studies indicate that the lead compounds display similar phenotypes as reported for clinically observed JMJD5 variants, which have a reduced catalytic activity compared to wild-type JMJD5.\", \"doi_y\": \"https://doi.org/10.1021/acs.jmedchem.3c01114\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4225146277\", \"doi_x\": \"10.1016/j.tibtech.2022.03.012\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Quality by Design for enabling RNA platform production processes\", \"abstract\": \"RNA-based products have emerged as one of the most promising and strategic technologies for global vaccination, infectious disease control, and future therapy development. The assessment of critical quality attributes (CQAs), product-process interactions, relevant process analytical technologies, and process modeling capabilities can feed into a robust Quality by Design (QbD) framework for future development, design, and control of manufacturing processes. QbD implementation will help the RNA technology reach its full potential and will be central to the development, pre-qualification, and regulatory approval of rapid response, disease-agnostic RNA platform production processes.\", \"doi_y\": \"https://doi.org/10.1016/j.tibtech.2022.03.012\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4385778709\", \"doi_x\": \"10.14814/phy2.15793\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Stanniocalcin\\u20102 inhibits skeletal muscle growth and is upregulated in functional overload\\u2010induced hypertrophy\", \"abstract\": \"Stanniocalcin-2 (STC2) has recently been implicated in human muscle mass variability by genetic analysis. Biochemically, STC2 inhibits the proteolytic activity of the metalloproteinase PAPP-A, which promotes muscle growth by upregulating the insulin-like growth factor (IGF) axis. The aim was to examine if STC2 affects skeletal muscle mass and to assess how the IGF axis mediates muscle hypertrophy induced by functional overload.We compared muscle mass and muscle fiber morphology between Stc2-/- (n = 21) and wild-type (n = 15) mice. We then quantified IGF1, IGF2, IGF binding proteins -4 and -5 (IGFBP-4, IGFBP-5), PAPP-A and STC2 in plantaris muscles of wild-type mice subjected to 4-week unilateral overload (n = 14).Stc2-/- mice showed up to 10% larger muscle mass compared with wild-type mice. This increase was mediated by greater cross-sectional area of muscle fibers. Overload increased plantaris mass and components of the IGF axis, including quantities of IGF1 (by 2.41-fold, p = 0.0117), IGF2 (1.70-fold, p = 0.0461), IGFBP-4 (1.48-fold, p = 0.0268), PAPP-A (1.30-fold, p = 0.0154) and STC2 (1.28-fold, p = 0.019).Here we provide evidence that STC2 is an inhibitor of muscle growth upregulated, along with other components of the IGF axis, during overload-induced muscle hypertrophy.\", \"doi_y\": \"https://doi.org/10.14814/phy2.15793\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4311502524\", \"doi_x\": \"10.15252/embj.2022111372\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"NDP52 acts as a redox sensor in PINK1/Parkin\\u2010mediated mitophagy\", \"abstract\": \"Abstract Mitophagy, the elimination of mitochondria via the autophagy\\u2010lysosome pathway, is essential for the maintenance of cellular homeostasis. The best characterised mitophagy pathway is mediated by stabilisation of the protein kinase PINK1 and recruitment of the ubiquitin ligase Parkin to damaged mitochondria. Ubiquitinated mitochondrial surface proteins are recognised by autophagy receptors including NDP52 which initiate the formation of an autophagic vesicle around the mitochondria. Damaged mitochondria also generate reactive oxygen species (ROS) which have been proposed to act as a signal for mitophagy, however the mechanism of ROS sensing is unknown. Here we found that oxidation of NDP52 is essential for the efficient PINK1/Parkin\\u2010dependent mitophagy. We identified redox\\u2010sensitive cysteine residues involved in disulphide bond formation and oligomerisation of NDP52 on damaged mitochondria. Oligomerisation of NDP52 facilitates the recruitment of autophagy machinery for rapid mitochondrial degradation. We propose that redox sensing by NDP52 allows mitophagy to function as a mechanism of oxidative stress response.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111372\", \"publication_date\": \"2022-12-14\"}, {\"work_id\": \"W4319459491\", \"doi_x\": \"10.1371/journal.pbio.3001962\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"abstract\": \"Macroautophagy/autophagy is an intracellular degradation process central to cellular homeostasis and defense against pathogens in eukaryotic cells. Regulation of autophagy relies on hierarchical binding of autophagy cargo receptors and adaptors to ATG8/LC3 protein family members. Interactions with ATG8/LC3 are typically facilitated by a conserved, short linear sequence, referred to as the ATG8/LC3 interacting motif/region (AIM/LIR), present in autophagy adaptors and receptors as well as pathogen virulence factors targeting host autophagy machinery. Since the canonical AIM/LIR sequence can be found in many proteins, identifying functional AIM/LIR motifs has proven challenging. Here, we show that protein modelling using Alphafold-Multimer (AF2-multimer) identifies both canonical and atypical AIM/LIR motifs with a high level of accuracy. AF2-multimer can be modified to detect additional functional AIM/LIR motifs by using protein sequences with mutations in primary AIM/LIR residues. By combining protein modelling data from AF2-multimer with phylogenetic analysis of protein sequences and protein-protein interaction assays, we demonstrate that AF2-multimer predicts the physiologically relevant AIM motif in the ATG8-interacting protein 2 (ATI-2) as well as the previously uncharacterized noncanonical AIM motif in ATG3 from potato (Solanum tuberosum). AF2-multimer also identified the AIM/LIR motifs in pathogen-encoded virulence factors that target ATG8 members in their plant and human hosts, revealing that cross-kingdom ATG8-LIR/AIM associations can also be predicted by AF2-multimer. We conclude that the AF2-guided discovery of autophagy adaptors/receptors will substantially accelerate our understanding of the molecular basis of autophagy in all biological kingdoms.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001962\", \"publication_date\": \"2023-02-08\"}, {\"work_id\": \"W4366691203\", \"doi_x\": \"10.3390/microorganisms11041077\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Structural Analysis of Bacillus subtilis Sigma Factors\", \"abstract\": \"Bacteria use an array of sigma factors to regulate gene expression during different stages of their life cycles. Full-length, atomic-level structures of sigma factors have been challenging to obtain experimentally as a result of their many regions of intrinsic disorder. AlphaFold has now supplied plausible full-length models for most sigma factors. Here we discuss the current understanding of the structures and functions of sigma factors in the model organism, Bacillus subtilis, and present an X-ray crystal structure of a region of B. subtilis SigE, a sigma factor that plays a critical role in the developmental process of spore formation.\", \"doi_y\": \"https://doi.org/10.3390/microorganisms11041077\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4306353420\", \"doi_x\": \"10.1093/nar/gkac896\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"abstract\": \"RNA G-quadruplex (rG4) is a vital RNA tertiary structure motif that involves the base pairs on both Hoogsteen and Watson-Crick faces of guanines. rG4 is of great importance in the post-transcriptional regulation of gene expression. Experimental technologies have advanced to identify in vitro and in vivo rG4s across diverse transcriptomes. Building on these recent advances, here we present G4Atlas, the first transcriptome-wide G-quadruplex database, in which we have collated, classified, and visualized transcriptome rG4 experimental data, generated from rG4-seq, chemical profiling and ligand-binding methods. Our comprehensive database includes transcriptome-wide rG4s generated from 82 experimental treatments and 238 samples across ten species. In addition, we have also included RNA secondary structure prediction information across both experimentally identified and unidentified rG4s to enable users to display any potential competitive folding between rG4 and RNA secondary structures. As such, G4Atlas will enable users to explore the general functions of rG4s in diverse biological processes. In addition, G4Atlas lays the foundation for further data-driven deep learning algorithms to examine rG4 structural features.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac896\", \"publication_date\": \"2022-10-16\"}, {\"work_id\": \"W4319333664\", \"doi_x\": \"10.1038/s41467-023-36372-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Plasma FIB milling for the determination of structures in situ\", \"abstract\": \"Abstract Structural biology studies inside cells and tissues require methods to thin vitrified specimens to electron transparency. Until now, focused ion beams based on gallium have been used. However, ion implantation, changes to surface chemistry and an inability to access high currents limit gallium application. Here, we show that plasma-coupled ion sources can produce cryogenic lamellae of vitrified human cells in a robust and automated manner, with quality sufficient for pseudo-atomic structure determination. Lamellae were produced in a prototype microscope equipped for long cryogenic run times (&gt; 1 week) and with multi-specimen support fully compatible with modern-day transmission electron microscopes. We demonstrate that plasma ion sources can be used for structural biology within cells, determining a structure in situ to 4.9 \\u00c5, and characterise the resolution dependence on particle distance from the lamella edge. We describe a workflow upon which different plasmas can be examined to further streamline lamella fabrication.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-36372-9\", \"publication_date\": \"2023-02-06\"}, {\"work_id\": \"W4200558169\", \"doi_x\": \"10.1101/2021.12.20.473413\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrated view and comparative analysis of baseline protein expression in mouse and rat tissues\", \"abstract\": \"Abstract The increasingly large amount of proteomics data in the public domain enables, among other applications, the combined analyses of datasets to create comparative protein expression maps covering different organisms and different biological conditions. Here we have reanalysed public proteomics datasets from mouse and rat tissues (14 and 9 datasets, respectively), to assess baseline protein abundance. Overall, the aggregated dataset contained 23 individual datasets, including a total of 211 samples coming from 34 different tissues across 14 organs, comprising 9 mouse and 3 rat strains, respectively. In all cases, we studied the distribution of canonical proteins between the different organs. The number of canonical proteins per dataset ranged from 273 (tendon) and 9,715 (liver) in mouse, and from 101 (tendon) and 6,130 (kidney) in rat. Then, we studied how protein abundances compared across different datasets and organs for both species. As a key point we carried out a comparative analysis of protein expression between mouse, rat and human tissues. We observed a high level of correlation of protein expression among orthologs between all three species in brain, kidney, heart and liver samples, whereas the correlation of protein expression was generally slightly lower between organs within the same species. Protein expression results have been integrated into the resource Expression Atlas for widespread dissemination. Author summary We have reanalysed 23 baseline mass spectrometry-based public proteomics datasets stored in the PRIDE database. Overall, the aggregated dataset contained 211 samples, coming from 34 different tissues across 14 organs, comprising 9 mouse and 3 rat strains, respectively. We analysed the distribution of protein expression across organs in both species. We also studied how protein abundances compared across different datasets and organs for both species. Then we performed gene ontology and pathway enrichment analyses to identify enriched biological processes and pathways across organs. We also carried out a comparative analysis of baseline protein expression across mouse, rat and human tissues, observing a high level of expression correlation among orthologs in all three species, in brain, kidney, heart and liver samples. To disseminate these findings, we have integrated the protein expression results into the resource Expression Atlas.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.20.473413\", \"publication_date\": \"2021-12-21\"}, {\"work_id\": \"W4312224088\", \"doi_x\": \"10.1038/s41467-022-35375-2\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"abstract\": \"Abstract Histone modifications are deposited by chromatin modifying enzymes and read out by proteins that recognize the modified state. BRD4-NUT is an oncogenic fusion protein of the acetyl lysine reader BRD4 that binds to the acetylase p300 and enables formation of long-range intra- and interchromosomal interactions. We here examine how acetylation reading and writing enable formation of such interactions. We show that NUT contains an acidic transcriptional activation domain that binds to the TAZ2 domain of p300. We use NMR to investigate the structure of the complex and found that the TAZ2 domain has an autoinhibitory role for p300. NUT-TAZ2 interaction or mutations found in cancer that interfere with autoinhibition by TAZ2 allosterically activate p300. p300 activation results in a self-organizing, acetylation-dependent feed-forward reaction that enables long-range interactions by bromodomain multivalent acetyl-lysine binding. We discuss the implications for chromatin organisation, gene regulation and dysregulation in disease.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-35375-2\", \"publication_date\": \"2022-12-15\"}, {\"work_id\": \"W4380576774\", \"doi_x\": \"10.1042/cs20220061\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Circadian regulation of pulmonary disease: the importance of timing\", \"abstract\": \"Circadian regulation causes the activity of biological processes to vary over a 24-h cycle. The pathological effects of this variation are predominantly studied using two different approaches: pre-clinical models or observational clinical studies. Both these approaches have provided useful insights into how underlying circadian mechanisms operate and specifically which are regulated by the molecular oscillator, a key time-keeping mechanism in the body. This review compares and contrasts findings from these two approaches in the context of four common respiratory diseases (asthma, chronic obstructive pulmonary disease, pulmonary fibrosis, and respiratory infection). Potential methods used to identify and measure human circadian oscillations are also discussed as these will be useful outcome measures in future interventional human trials that target circadian mechanisms.\", \"doi_y\": \"https://doi.org/10.1042/cs20220061\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4281909669\", \"doi_x\": \"10.1002/rmv.2373\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Recent insights into SARS\\u2010CoV\\u20102 omicron variant\", \"abstract\": \"The SARS-CoV-2 omicron variant (B.1.1.529) was first identified in Botswana and South Africa, and its emergence has been associated with a steep increase in the number of SARS-CoV-2 infections. The omicron variant has subsequently spread very rapidly across the world, resulting in the World Health Organization classification as a variant of concern on 26 November 2021. Since its emergence, great efforts have been made by research groups around the world that have rapidly responded to fill our gaps in knowledge for this novel variant. A growing body of data has demonstrated that the omicron variant shows high transmissibility, robust binding to human angiotensin-converting enzyme 2 receptor, attenuated viral replication, and causes less severe disease in COVID-19 patients. Further, the variant has high environmental stability, high resistance against most therapeutic antibodies, and partial escape neutralisation by antibodies from convalescent patients or vaccinated individuals. With the pandemic ongoing, there is a need for the distillation of literature from primary research into an accessible format for the community. In this review, we summarise the key discoveries related to the SARS-CoV-2 omicron variant, highlighting the gaps in knowledge that guide the field's ongoing and future work.\", \"doi_y\": \"https://doi.org/10.1002/rmv.2373\", \"publication_date\": \"2022-06-04\"}, {\"work_id\": \"W4308842536\", \"doi_x\": \"10.1016/j.pbi.2022.102311\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"abstract\": \"Crop yield and global food security are under constant threat from plant pathogens with the potential to cause epidemics. Traditional breeding for disease resistance can be too slow to counteract these emerging threats, resulting in the need to retool the plant immune system using bioengineered made-to-order immune receptors. Efforts to engineer immune receptors have focused primarily on nucleotide-binding domain and leucine-rich repeat (NLR) immune receptors and proof-of-principles studies. Based upon a near-exhaustive literature search of previously engineered plant immune systems we distil five emerging principles in the design of bioengineered made-to-order plant NLRs and describe approaches based on other components. These emerging principles are anticipated to assist the functional understanding of plant immune receptors, as well as bioengineering novel disease resistance specificities.\", \"doi_y\": \"https://doi.org/10.1016/j.pbi.2022.102311\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4366588851\", \"doi_x\": \"10.1038/s41594-023-00950-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fibrillin microfibril structure identifies long-range effects of inherited pathogenic mutations affecting a key regulatory latent TGF\\u03b2-binding site\", \"abstract\": \"Genetic mutations in fibrillin microfibrils cause serious inherited diseases, such as Marfan syndrome and Weill-Marchesani syndrome (WMS). These diseases typically show major dysregulation of tissue development and growth, particularly in skeletal long bones, but links between the mutations and the diseases are unknown. Here we describe a detailed structural analysis of native fibrillin microfibrils from mammalian tissue by cryogenic electron microscopy. The major bead region showed pseudo eightfold symmetry where the amino and carboxy termini reside. On the basis of this structure, we show that a WMS deletion mutation leads to the induction of a structural rearrangement that blocks interaction with latent TGF\\u03b2-binding protein-1 at a remote site. Separate deletion of this binding site resulted in the assembly of shorter fibrillin microfibrils with structural alterations. The integrin \\u03b1v\\u03b23-binding site was also mapped onto the microfibril structure. These results establish that in complex extracellular assemblies, such as fibrillin microfibrils, mutations may have long-range structural consequences leading to the disruption of growth factor signaling and the development of disease.\", \"doi_y\": \"https://doi.org/10.1038/s41594-023-00950-8\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4306746309\", \"doi_x\": \"10.1016/j.celrep.2022.111522\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic landscape of drug response reveals mediators of anthelmintic resistance\", \"abstract\": \"Like other pathogens, parasitic helminths can rapidly evolve resistance to drug treatment. Understanding the genetic basis of anthelmintic drug resistance in parasitic nematodes is key to tracking its spread and improving the efficacy and sustainability of parasite control. Here, we use an in vivo genetic cross between drug-susceptible and multi-drug-resistant strains of Haemonchus contortus in a natural host-parasite system to simultaneously map resistance loci for the three major classes of anthelmintics. This approach identifies new alleles for resistance to benzimidazoles and levamisole and implicates the transcription factor cky-1 in ivermectin resistance. This gene is within a locus under selection in ivermectin-resistant populations worldwide; expression analyses and functional validation using knockdown experiments support that cky-1 is associated with ivermectin survival. Our work demonstrates the feasibility of high-resolution forward genetics in a parasitic nematode and identifies variants for the development of molecular diagnostics to combat drug resistance in the field.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111522\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4313279513\", \"doi_x\": \"10.1021/acs.jproteome.2c00406\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrated View of Baseline Protein Expression in Human Tissues\", \"abstract\": \"The availability of proteomics datasets in the public domain, and in the PRIDE database, in particular, has increased dramatically in recent years. This unprecedented large-scale availability of data provides an opportunity for combined analyses of datasets to get organism-wide protein abundance data in a consistent manner. We have reanalyzed 24 public proteomics datasets from healthy human individuals to assess baseline protein abundance in 31 organs. We defined tissue as a distinct functional or structural region within an organ. Overall, the aggregated dataset contains 67 healthy tissues, corresponding to 3,119 mass spectrometry runs covering 498 samples from 489 individuals. We compared protein abundances between different organs and studied the distribution of proteins across these organs. We also compared the results with data generated in analogous studies. Additionally, we performed gene ontology and pathway-enrichment analyses to identify organ-specific enriched biological processes and pathways. As a key point, we have integrated the protein abundance results into the resource Expression Atlas, where they can be accessed and visualized either individually or together with gene expression data coming from transcriptomics datasets. We believe this is a good mechanism to make proteomics data more accessible for life scientists.\", \"doi_y\": \"https://doi.org/10.1021/acs.jproteome.2c00406\", \"publication_date\": \"2022-12-28\"}, {\"work_id\": \"W4366819790\", \"doi_x\": \"10.1111/tra.12885\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Out of the <scp>ESCPE</scp> room: Emerging roles of endosomal <scp>SNX\\u2010BARs</scp> in receptor transport and host\\u2013pathogen interaction\", \"abstract\": \"Abstract Several functions of the human cell, such as sensing nutrients, cell movement and interaction with the surrounding environment, depend on a myriad of transmembrane proteins and their associated proteins and lipids (collectively termed \\u201ccargoes\\u201d). To successfully perform their tasks, cargo must be sorted and delivered to the right place, at the right time, and in the right amount. To achieve this, eukaryotic cells have evolved a highly organized sorting platform, the endosomal network. Here, a variety of specialized multiprotein complexes sort cargo into itineraries leading to either their degradation or their recycling to various organelles for further rounds of reuse. A key sorting complex is the Endosomal SNX\\u2010BAR Sorting Complex for Promoting Exit (ESCPE\\u20101) that promotes the recycling of an array of cargos to the plasma membrane and/or the trans \\u2010Golgi network. ESCPE\\u20101 recognizes a hydrophobic\\u2010based sorting motif in numerous cargoes and orchestrates their packaging into tubular carriers that pinch off from the endosome and travel to the target organelle. A wide range of pathogens mimic this sorting motif to hijack ESCPE\\u20101 transport to promote their invasion and survival within infected cells. In other instances, ESCPE\\u20101 exerts restrictive functions against pathogens by limiting their replication and infection. In this review, we discuss ESCPE\\u20101 assembly and functions, with a particular focus on recent advances in the understanding of its role in membrane trafficking, cellular homeostasis and host\\u2013pathogen interaction.\", \"doi_y\": \"https://doi.org/10.1111/tra.12885\", \"publication_date\": \"2023-04-23\"}, {\"work_id\": \"W4319310661\", \"doi_x\": \"10.1038/s43588-022-00391-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High-throughput property-driven generative design of functional organic molecules\", \"abstract\": \"The design of molecules and materials with tailored properties is challenging, as candidate molecules must satisfy multiple competing requirements that are often difficult to measure or compute. While molecular structures, produced through generative deep learning, will satisfy those patterns, they often only possess specific target properties by chance and not by design, which makes molecular discovery via this route inefficient. In this work, we predict molecules with (pareto)-optimal properties by combining a generative deep learning model that predicts three dimensional conformations of molecules with a supervised deep learning model that takes these as inputs and predicts their electronic structure. Optimization of (multiple) molecular properties is achieved by screening newly generated molecules for desirable electronic properties and reusing hit molecules to retrain the generative model with a bias. The approach is demonstrated to find optimal molecules for organic electronics applications. Our method is generally applicable and eliminates the need for quantum chemical calculations during predictions, making it suitable for high-throughput screening in materials and catalyst design.\", \"doi_y\": \"https://doi.org/10.1038/s43588-022-00391-1\", \"publication_date\": \"2023-02-06\"}, {\"work_id\": \"W4210908551\", \"doi_x\": \"10.1101/2022.02.07.479382\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Pitfalls of machine learning models for protein-protein interactions\", \"abstract\": \"Abstract Protein-protein interactions (PPIs) are essential to understanding biological pathways as well as their roles in development and disease. Computational tools, based on classic machine learning, have been successful at predicting PPIs in silico , but the lack of consistent and reliable frameworks for this task has led to network models that are difficult to compare and discrepancies between algorithms that remain unexplained. To better understand the underlying inference mechanisms that underpin these models, we designed an open-source framework for benchmarking that accounts for a range of biological and statistical pitfalls while facilitating reproducibility. We use it to shed light on the impact of network topology and how different algorithms deal with highly connected proteins. By studying functional genomics-based and sequence-based models on human PPIs, we show their complementarity as the former performs best on lone proteins while the latter specialises in interactions involving hubs. We also show that algorithm design has little impact on performance with functional genomic data. We replicate our results between both human and S. cerevisiae data and demonstrate that models using functional genomics are better suited to PPI prediction across species. With rapidly increasing amounts of sequence and functional genomics data, our study provides a principled foundation for future construction, comparison and application of PPI networks.\", \"doi_y\": \"https://doi.org/10.1101/2022.02.07.479382\", \"publication_date\": \"2022-02-08\"}, {\"work_id\": \"W4291010252\", \"doi_x\": \"10.1101/2022.08.11.503608\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The<i>Solanum americanum</i>pangenome and effectoromics reveal new resistance genes against potato late blight\", \"abstract\": \"Abstract Late blight caused by the oomycete pathogen Phytophthora infestans continues to cause major worldwide losses in potato and tomato. Most accessions of Solanum americanum , a globally distributed, wild Solanaceae plant, are highly resistant to late blight. We generated high-quality reference genomes of four S. americanum accessions, re-sequenced 52 accessions, and we defined variation in the NLR immune receptor genes (the S. americanum NLRome). We further screened for variation in recognition of \\u223c315 P. infestans RXLR effectors in 52 S. americanum accessions. Using these genotypic and phenotypic data, we cloned three novel NLR-encoding genes Rpi-amr4, Rpi-amr16 and Rpi-amr17 , and determined their corresponding RXLR effector genes Avramr4 ( PITG_22825 ), Avramr16 ( PITG_02860 ) and Avramr17 ( PITG_04373 ) from P. infestans . These genomic resources and methodology will support efforts to convert potato into a \\u201cnonhost\\u201d of late blight and can be applied to diseases of other crops.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.11.503608\", \"publication_date\": \"2022-08-11\"}, {\"work_id\": \"W4307492594\", \"doi_x\": \"10.1093/nar/gkac925\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"abstract\": \"Type I CRISPR systems are the most common CRISPR type found in bacteria. They use a multisubunit effector, guided by crRNA, to detect and bind dsDNA targets, forming an R-loop and recruiting the Cas3 enzyme to facilitate target DNA destruction, thus providing immunity against mobile genetic elements. Subtypes have been classified into families A-G, with type I-G being the least well understood. Here, we report the composition, structure and function of the type I-G Cascade CRISPR effector from Thioalkalivibrio sulfidiphilus, revealing key new molecular details. The unique Csb2 subunit processes pre-crRNA, remaining bound to the 3' end of the mature crRNA, and seven Cas7 subunits form the backbone of the effector. Cas3 associates stably with the effector complex via the Cas8g subunit and is important for target DNA recognition. Structural analysis by cryo-Electron Microscopy reveals a strikingly curved backbone conformation with Cas8g spanning the belly of the structure. These biochemical and structural insights shed new light on the diversity of type I systems and open the way to applications in genome engineering.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac925\", \"publication_date\": \"2022-10-28\"}, {\"work_id\": \"W4324354022\", \"doi_x\": \"10.1103/physrevb.107.l121105\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation energy of the paramagnetic electron gas at the thermodynamic limit\", \"abstract\": \"The variational and diffusion quantum Monte Carlo methods are used to calculate the correlation energy of the paramagnetic three-dimensional homogeneous electron gas at intermediate to high density. Ground state energies in finite cells are determined using Slater-Jastrow-backflow trial wave functions, and finite-size errors are removed using twist-averaged boundary conditions and extrapolation of the energy per particle to the thermodynamic limit of infinite system size. Our correlation energies in the thermodynamic limit are lower (i.e., more negative, and therefore more accurate according to the variational principle) than previous results, and can be used for the parameterization of density functionals to be applied to high-density systems.\", \"doi_y\": \"https://doi.org/10.1103/physrevb.107.l121105\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4205296878\", \"doi_x\": \"10.1101/2022.01.18.476751\", \"af_direct_citation\": 1, \"af_indirect_citations\": 6, \"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"abstract\": \"Abstract In the recent CASP (Critical Assessment of Structure Prediction) competition, AlphaFold2 performed outstandingly. Its worst predictions were for NMR structures, which has two alternative explanations: either the NMR structures were poor, implying that AlphaFold may be more accurate than NMR; or there is a genuine difference between crystal and solution structures. Here, we use the program ANSURR, which measures the accuracy of solution structures, and show that one of the NMR structures was indeed poor. We then compare AlphaFold predictions to NMR structures, and show that AlphaFold tends to be more accurate than NMR ensembles, in particular correctly more rigid in loops. There are however some cases where the NMR ensembles are more accurate. These tend to be dynamic structures where AlphaFold had low confidence. We suggest that AlphaFold could be used as the model for NMR structure refinements, and that AlphaFold structures validated by ANSURR require no further refinement.\", \"doi_y\": \"https://doi.org/10.1101/2022.01.18.476751\", \"publication_date\": \"2022-01-20\"}, {\"work_id\": \"W4303437380\", \"doi_x\": \"10.1101/2022.10.04.510780\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mutation of the ALS/FTD-associated RNA-binding protein FUS alters axonal cytoskeletal organisation\", \"abstract\": \"Summary Aberrant condensation and localisation of the RNA-binding protein fused in sarcoma (FUS) occur in variants of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). ALS is also associated with cytoskeletal defects, genetically and through observations of compromised axonal transport. Here, we asked whether compromised axonal cytoskeletal organisation is an early feature of FUS-associated ALS/FTD. We used an ALS-associated mutant FUS(P525L) and the FTD-mimic hypomethylated FUS, FUS(16R), to investigate the common and distinct cytoskeletal changes found in these two reported Xenopus models. Combining a novel atomic force microscopy (AFM)-based approach for in vitro cytoskeletal characterisation and in vivo axonal branching analysis, we found that mutant FUS reduced actin density in the dynamically remodelling growth cone, and reduced axonal branch complexity. We furthermore found evidence of an axon looping defect for FUS(P525L). Therefore, we show that compromised actin remodelling is potentially an important early event in FUS-associated pathogenesis. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.10.04.510780\", \"publication_date\": \"2022-10-07\"}, {\"work_id\": \"W4382728082\", \"doi_x\": \"10.1016/j.molcel.2023.06.013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular basis for the reversible ADP-ribosylation of guanosine bases\", \"abstract\": \"Modification of nucleic acids by ADP-ribosylation is catalyzed by various ADP-ribosyltransferases, including the DarT enzyme. The latter is part of the bacterial toxin-antitoxin (TA) system DarTG, which was shown to provide control of DNA replication and bacterial growth as well as protection against bacteriophages. Two subfamilies have been identified, DarTG1 and DarTG2, which are distinguished by their associated antitoxins. While DarTG2 catalyzes reversible ADP-ribosylation of thymidine bases employing a macrodomain as antitoxin, the DNA ADP-ribosylation activity of DarTG1 and the biochemical function of its antitoxin, a NADAR domain, are as yet unknown. Using structural and biochemical approaches, we show that DarT1-NADAR is a TA system for reversible ADP-ribosylation of guanosine bases. DarT1 evolved the ability to link ADP-ribose to the guanine amino group, which is specifically hydrolyzed by NADAR. We show that guanine de-ADP-ribosylation is also conserved among eukaryotic and non-DarT-associated NADAR members, indicating a wide distribution of reversible guanine modifications beyond DarTG systems.\", \"doi_y\": \"https://doi.org/10.1016/j.molcel.2023.06.013\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4315752818\", \"doi_x\": \"10.1242/jcs.260462\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Multiple interactions of the dynein-2 complex with the IFT-B complex are required for effective intraflagellar transport\", \"abstract\": \"The dynein-2 complex must be transported anterogradely within cilia to then drive retrograde trafficking of the intraflagellar transport (IFT) machinery containing IFT-A and IFT-B complexes. Here, we screened for potential interactions between the dynein-2 and IFT-B complexes and found multiple interactions among the dynein-2 and IFT-B subunits. In particular, WDR60 (also known as DYNC2I1) and the DYNC2H1-DYNC2LI1 dimer from dynein-2, and IFT54 (also known as TRAF3IP1) and IFT57 from IFT-B contribute to the dynein-2-IFT-B interactions. WDR60 interacts with IFT54 via a conserved region N-terminal to its light chain-binding regions. Expression of the WDR60 constructs in WDR60-knockout (KO) cells revealed that N-terminal truncation mutants lacking the IFT54-binding site fail to rescue abnormal phenotypes of WDR60-KO cells, such as aberrant accumulation of the IFT machinery around the ciliary tip and on the distal side of the transition zone. However, a WDR60 construct specifically lacking just the IFT54-binding site substantially restored the ciliary defects. In line with the current docking model of dynein-2 with the anterograde IFT trains, these results indicate that extensive interactions involving multiple subunits from the dynein-2 and IFT-B complexes participate in their connection.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260462\", \"publication_date\": \"2023-02-07\"}, {\"work_id\": \"W4362520906\", \"doi_x\": \"10.7554/elife.86049\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"abstract\": \"The phylum of Apicomplexa groups intracellular parasites that employ substrate-dependent gliding motility to invade host cells, egress from the infected cells, and cross biological barriers. The glideosome-associated connector (GAC) is a conserved protein essential to this process. GAC facilitates the association of actin filaments with surface transmembrane adhesins and the efficient transmission of the force generated by myosin translocation of actin to the cell surface substrate. Here, we present the crystal structure of Toxoplasma gondii GAC and reveal a unique, supercoiled armadillo repeat region that adopts a closed ring conformation. Characterisation of the solution properties together with membrane and F-actin binding interfaces suggests that GAC adopts several conformations from closed to open and extended. A multi-conformational model for assembly and regulation of GAC within the glideosome is proposed.\", \"doi_y\": \"https://doi.org/10.7554/elife.86049\", \"publication_date\": \"2023-04-04\"}, {\"work_id\": \"W4305015942\", \"doi_x\": \"10.1016/j.cell.2022.09.025\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"abstract\": \"Neural migration is a critical step during brain development that requires the interactions of cell-surface guidance receptors. Cancer cells often hijack these mechanisms to disseminate. Here, we reveal crystal structures of Uncoordinated-5 receptor D (Unc5D) in complex with morphogen receptor glypican-3 (GPC3), forming an octameric glycoprotein complex. In the complex, four Unc5D molecules pack into an antiparallel bundle, flanked by four GPC3 molecules. Central glycan-glycan interactions are formed by N-linked glycans emanating from GPC3 (N241 in human) and C-mannosylated tryptophans of the Unc5D thrombospondin-like domains. MD simulations, mass spectrometry and structure-based mutants validate the crystallographic data. Anti-GPC3 nanobodies enhance or weaken Unc5-GPC3 binding and, together with mutant proteins, show that Unc5/GPC3 guide migrating pyramidal neurons in the mouse cortex, and cancer cells in an embryonic xenograft neuroblastoma model. The results demonstrate a conserved structural mechanism of cell guidance, where finely balanced Unc5-GPC3 interactions regulate cell migration.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.09.025\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4386419407\", \"doi_x\": \"10.1002/fes3.498\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Reviving grain quality in wheat through non\\u2010destructive phenotyping techniques like hyperspectral imaging\", \"abstract\": \"Abstract A long\\u2010term goal of breeders and researchers is to develop crop varieties that can resist environmental stressors and produce high yields. However, prioritising yield often compromises improvement of other key traits, including grain quality, which is tedious and time\\u2010consuming to measure because of the frequent involvement of destructive phenotyping methods. Recently, non\\u2010destructive methods such as hyperspectral imaging (HSI) have gained attention in the food industry for studying wheat grain quality. HSI can quantify variations in individual grains, helping to differentiate high\\u2010quality grains from those of low quality. In this review, we discuss the reduction of wheat genetic diversity underlying grain quality traits due to modern breeding, key traits for grain quality, traditional methods for studying grain quality and the application of HSI to study grain quality traits in wheat and its scope in breeding. Our critical review of literature on wheat domestication, grain quality traits and innovative technology introduces approaches that could help improve grain quality in wheat.\", \"doi_y\": \"https://doi.org/10.1002/fes3.498\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4283802940\", \"doi_x\": \"10.3389/fphy.2022.910381\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Extending the Capabilities of Data-Driven Reduced-Order Models to Make Predictions for Unseen Scenarios: Applied to Flow Around Buildings\", \"abstract\": \"We present a data-driven or non-intrusive reduced-order model (NIROM) which is capable of making predictions for a significantly larger domain than the one used to generate the snapshots or training data. This development relies on the combination of a novel way of sampling the training data (which frees the NIROM from its dependency on the original problem domain) and a domain decomposition approach (which partitions unseen geometries in a manner consistent with the sub-sampling approach). The method extends current capabilities of reduced-order models to generalise, i.e., to make predictions for unseen scenarios. The method is applied to a 2D test case which simulates the chaotic time-dependent flow of air past buildings at a moderate Reynolds number using a computational fluid dynamics (CFD) code. The procedure for 3D problems is similar, however, a 2D test case is considered sufficient here, as a proof-of-concept. The reduced-order model consists of a sampling technique to obtain the snapshots; a convolutional autoencoder for dimensionality reduction; an adversarial network for prediction; all set within a domain decomposition framework. The autoencoder is chosen for dimensionality reduction as it has been demonstrated in the literature that these networks can compress information more efficiently than traditional (linear) approaches based on singular value decomposition. In order to keep the predictions realistic, properties of adversarial networks are exploited. To demonstrate its ability to generalise, once trained, the method is applied to a larger domain which has a different arrangement of buildings. Statistical properties of the flows from the reduced-order model are compared with those from the CFD model in order to establish how realistic the predictions are.\", \"doi_y\": \"https://doi.org/10.3389/fphy.2022.910381\", \"publication_date\": \"2022-07-04\"}, {\"work_id\": \"W4365443842\", \"doi_x\": \"10.1002/pro.4640\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering mutually orthogonal <scp>PylRS</scp>/<scp>tRNA</scp> pairs for dual encoding of functional histidine analogues\", \"abstract\": \"The availability of an expanded genetic code opens exciting new opportunities in enzyme design and engineering. In this regard histidine analogues have proven particularly versatile, serving as ligands to augment metalloenzyme function and as catalytic nucleophiles in designed enzymes. The ability to genetically encode multiple functional residues could greatly expand the range of chemistry accessible within enzyme active sites. Here, we develop mutually orthogonal translation components to selectively encode two structurally similar histidine analogues. Transplanting known mutations from a promiscuous Methanosarcina mazei pyrrolysyl-tRNA synthetase (MmPylRSIFGFF ) into a single domain PylRS from Methanomethylophilus alvus (MaPylRSIFGFF ) provided a variant with improved efficiency and specificity for 3-methyl-L-histidine (MeHis) incorporation. The MaPylRSIFGFF clone was further characterized using in vitro biochemical assays and x-ray crystallography. We subsequently engineered the orthogonal MmPylRS for activity and selectivity for 3-(3-pyridyl)-L-alanine (3-Pyr), which was used in combination with MaPylRSIFGFF to produce proteins containing both 3-Pyr and MeHis. Given the versatile roles played by histidine in enzyme mechanisms, we anticipate that the tools developed within this study will underpin the development of enzymes with new and enhanced functions.\", \"doi_y\": \"https://doi.org/10.1002/pro.4640\", \"publication_date\": \"2023-04-25\"}, {\"work_id\": \"W4321483489\", \"doi_x\": \"10.1016/j.jmb.2023.168021\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"abstract\": \"ModelCIF (github.com/ihmwg/ModelCIF) is a data information framework developed for and by computational structural biologists to enable delivery of Findable, Accessible, Interoperable, and Reusable (FAIR) data to users worldwide. ModelCIF describes the specific set of attributes and metadata associated with macromolecular structures modeled by solely computational methods and provides an extensible data representation for deposition, archiving, and public dissemination of predicted three-dimensional (3D) models of macromolecules. It is an extension of the Protein Data Bank Exchange / macromolecular Crystallographic Information Framework (PDBx/mmCIF), which is the global data standard for representing experimentally-determined 3D structures of macromolecules and associated metadata. The PDBx/mmCIF framework and its extensions (e.g., ModelCIF) are managed by the Worldwide Protein Data Bank partnership (wwPDB, wwpdb.org) in collaboration with relevant community stakeholders such as the wwPDB ModelCIF Working Group (wwpdb.org/task/modelcif). This semantically rich and extensible data framework for representing computed structure models (CSMs) accelerates the pace of scientific discovery. Herein, we describe the architecture, contents, and governance of ModelCIF, and tools and processes for maintaining and extending the data standard. Community tools and software libraries that support ModelCIF are also described.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168021\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4385064091\", \"doi_x\": \"10.1371/journal.ppat.1011529\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Thrombin cleavage of the hepatitis E virus polyprotein at multiple conserved locations is required for genome replication\", \"abstract\": \"The genomes of positive-sense RNA viruses encode polyproteins that are essential for mediating viral replication. These viral polyproteins must undergo proteolysis (also termed polyprotein processing) to generate functional protein units. This proteolysis can be performed by virally-encoded proteases as well as host cellular proteases, and is generally believed to be a key step in regulating viral replication. Hepatitis E virus (HEV) is a leading cause of acute viral hepatitis. The positive-sense RNA genome is translated to generate a polyprotein, termed pORF1, which is necessary and sufficient for viral genome replication. However, the mechanism of polyprotein processing in HEV remains to be determined. In this study, we aimed to understand processing of this polyprotein and its role in viral replication using a combination of in vitro translation experiments and HEV sub-genomic replicons. Our data suggest no evidence for a virally-encoded protease or auto-proteolytic activity, as in vitro translation predominantly generates unprocessed viral polyprotein precursors. However, seven cleavage sites within the polyprotein (suggested by bioinformatic analysis) are susceptible to the host cellular protease, thrombin. Using two sub-genomic replicon systems, we demonstrate that mutagenesis of these sites prevents replication, as does pharmacological inhibition of serine proteases including thrombin. Overall, our data supports a model where HEV uses host proteases to support replication and could have evolved to be independent of a virally-encoded protease for polyprotein processing.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011529\", \"publication_date\": \"2023-07-21\"}, {\"work_id\": \"W4320709386\", \"doi_x\": \"10.1021/acsami.2c21187\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Surface-Functionalized Metal\\u2013Organic Frameworks for Binding Coronavirus Proteins\", \"abstract\": \"Since the outbreak of SARS-CoV-2, a multitude of strategies have been explored for the means of protection and shielding against virus particles: filtration equipment (PPE) has been widely used in daily life. In this work, we explore another approach in the form of deactivating coronavirus particles through selective binding onto the surface of metal-organic frameworks (MOFs) to further the fight against the transmission of respiratory viruses. MOFs are attractive materials in this regard, as their rich pore and surface chemistry can easily be modified on demand. The surfaces of three MOFs, UiO-66(Zr), UiO-66-NH2(Zr), and UiO-66-NO2(Zr), have been functionalized with repurposed antiviral agents, namely, folic acid, nystatin, and tenofovir, to enable specific interactions with the external spike protein of the SARS virus. Protein binding studies revealed that this surface modification significantly improved the binding affinity toward glycosylated and non-glycosylated proteins for all three MOFs. Additionally, the pores for the surface-functionalized MOFs can adsorb water, making them suitable for locally dehydrating microbial aerosols. Our findings highlight the immense potential of MOFs in deactivating respiratory coronaviruses to be better equipped to fight future pandemics.\", \"doi_y\": \"https://doi.org/10.1021/acsami.2c21187\", \"publication_date\": \"2023-02-14\"}, {\"work_id\": \"W4224126867\", \"doi_x\": \"10.1101/2022.04.14.488335\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Parallel recruitment pathways contribute to synaptonemal complex assembly during mammalian meiosis\", \"abstract\": \"Abstract During meiosis, the synaptonemal complex (SC) assembles between paired chromosomes, binding them together in close apposition, and facilitating recombination. SC assembly is thought to occur through the hierarchical zipper-like recruitment of axial elements, followed by transverse filaments and then central elements. However, the rapidity of SC formation in mammals has hitherto hindered investigation of its assembly mechanisms and their relationship with recombination. Using super-resolution imaging of separation-of-function mouse mutants, we show that, contrary to the hierarchical assembly model, central element protein SYCE2 is recruited to recombination sites early in SC assembly, and independently of SYCP1-containing transverse filaments. Further, SYCE2-TEX12 binds DNA in vitro , and SYCE2-containing bridges physically link paired chromosomes at recombination sites prior to transverse filament recruitment and chromosome synapsis. These data suggest that mammals integrate parallel recruitment pathways to assemble a mature SC: one recruiting central element proteins to recombination sites, and another recruiting transverse filaments to chromosomes.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.14.488335\", \"publication_date\": \"2022-04-14\"}, {\"work_id\": \"W4320481144\", \"doi_x\": \"10.1101/2023.02.12.528179\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Light sensitive short hypocotyl (LSH) confer symbiotic nodule identity in the legume Medicago truncatula\", \"abstract\": \"Summary Legumes grow specialized root nodules that are distinct from lateral roots in morphology and function, with nodules intracellularly hosting beneficial nitrogen-fixing bacteria that provide the plant with a nitrogen source. We have previously shown that a lateral root-like program underpins nodule initiation, but there must be additional developmental programs that confer nodule identity. Here, we show that two members of the LIGHT SENSITIVE SHORT HYPOCOTYL (LSH) transcription factor family, predominantly known to define organ boundaries and meristem complexity in the shoot, function as regulators of nodule organ identity. LSH1/LSH2 function upstream of and together with the known nodule regulators Nuclear Factor Y - A1 and NODULE ROOT1/2. The principal outcome of LSH1/LSH2 function is the production of cells able to accommodate nitrogen-fixing bacteria, the unique nodule feature. We conclude that the coordinate recruitment of a pre-existing primordium identity program, in parallel to a root initiation program, underpins the divergence between lateral roots and nodules.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.12.528179\", \"publication_date\": \"2023-02-13\"}, {\"work_id\": \"W4308197865\", \"doi_x\": \"10.3390/ijms232113455\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Generation and Characterization of Native and Sialic Acid-Deficient IgE\", \"abstract\": \"Efficient characterization of IgE antibodies and their glycan structures is required for understanding their function in allergy and in the emerging AllergoOncology field for antibody immunotherapy. We report the generation, glyco-profiling and functional analysis of native and sialic acid-deficient glyco-engineered human IgE. The antibodies produced from human embryonic kidney cells were purified via a human IgE class-specific affinity matrix and structural integrity was confirmed by SDS-PAGE and size-exclusion chromatography (SEC). Purified IgEs specific for the tumor-associated antigens Chondroitin Sulfate Proteoglycan 4 (CSPG4-IgE) and Human Epidermal Growth Factor Receptor 2 (HER2-IgE) were devoid of by-products such as free light chains. Using neuraminidase-A, we generated sialic acid-deficient CSPG4-IgE as example glyco-engineered antibody. Comparative glycan analyses of native and glyco-engineered IgEs by Hydrophilic interaction liquid chromatography (HILIC)-high performance liquid chromatography (HPLC) indicated loss of sialic acid terminal residues and differential glycan profiles. Native and glyco-engineered CSPG4-IgEs recognized Fc receptors on the surface of human Fc\\u03b5RI-expressing rat basophilic leukemia RBL-SX38 cells, and of CD23/Fc\\u03b5RII-expressing human RPMI-8866 B-lymphocytes and bound to CSPG4-expressing A2058 human melanoma cells, confirming Fab-mediated recognition. When cross-linked on the cell surface, both IgEs triggered RBL-SX38 degranulation. We demonstrate efficient generation and functional competence of recombinant native and sialic acid-deficient IgEs.\", \"doi_y\": \"https://doi.org/10.3390/ijms232113455\", \"publication_date\": \"2022-11-03\"}, {\"work_id\": \"W4378211582\", \"doi_x\": \"10.1016/j.jmb.2023.168160\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Computational Resources for Molecular Biology 2023\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168160\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4319842310\", \"doi_x\": \"10.1039/d2cb00231k\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A primer to directed evolution: current methodologies and future directions\", \"abstract\": \"This review summarises the methods available for directed evolution, including mutagenesis and variant selection techniques. The advantages and disadvantages of each technique are presented, and future challenges in the field are discussed.\", \"doi_y\": \"https://doi.org/10.1039/d2cb00231k\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4313486378\", \"doi_x\": \"10.1126/sciadv.add3479\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"abstract\": \"Gene transcription is carried out by RNA polymerase (RNAP) and requires the conversion of the initial closed promoter complex, where DNA is double stranded, to a transcription-competent open promoter complex, where DNA is opened up. In bacteria, RNAP relies on \\u03c3 factors for its promoter specificities. Using a special form of sigma factor (\\u03c354), which forms a stable closed complex and requires its activator that belongs to the AAA+ ATPases (ATPases associated with diverse cellular activities), we obtained cryo-electron microscopy structures of transcription initiation complexes that reveal a previously unidentified process of DNA melting opening. The \\u03c354 amino terminus threads through the locally opened up DNA and then becomes enclosed by the AAA+ hexameric ring in the activator-bound intermediate complex. Our structures suggest how ATP hydrolysis by the AAA+ activator could remove the \\u03c354 inhibition while helping to open up DNA, using \\u03c354 amino-terminal peptide as a pry bar.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.add3479\", \"publication_date\": \"2022-12-23\"}, {\"work_id\": \"W4290667627\", \"doi_x\": \"10.1371/journal.ppat.1010706\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic evidence of contemporary hybridization between Schistosoma species\", \"abstract\": \"Hybridization between different species of parasites is increasingly being recognised as a major public and veterinary health concern at the interface of infectious diseases biology, evolution, epidemiology and ultimately control. Recent research has revealed that viable hybrids and introgressed lineages between Schistosoma spp. are prevalent across Africa and beyond, including those with zoonotic potential. However, it remains unclear whether these hybrid lineages represent recent hybridization events, suggesting hybridization is ongoing, and/or whether they represent introgressed lineages derived from ancient hybridization events. In human schistosomiasis, investigation is hampered by the inaccessibility of adult-stage worms due to their intravascular location, an issue which can be circumvented by post-mortem of livestock at abattoirs for Schistosoma spp. of known zoonotic potential. To characterise the composition of naturally-occurring schistosome hybrids, we performed whole-genome sequencing of 21 natural livestock infective schistosome isolates. To facilitate this, we also assembled a de novo chromosomal-scale draft assembly of Schistosoma curassoni. Genomic analyses identified isolates of S. bovis, S. curassoni and hybrids between the two species, all of which were early generation hybrids with multiple generations found within the same host. These results show that hybridization is an ongoing process within natural populations with the potential to further challenge elimination efforts against schistosomiasis.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1010706\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4310464829\", \"doi_x\": \"10.1039/d2cc04823j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Enzyme self-label-bound ATTO700 in single-molecule and super-resolution microscopy\", \"abstract\": \"Near-infrared ATTO700 is evaluated as an acceptor in SNAP- and Halo-tag protein labelling for ensemble and single molecule F\\u00f6rster Resonance Energy Transfer (FRET) and in stimulated emission by depletion (STED) super resolution microscopy.\", \"doi_y\": \"https://doi.org/10.1039/d2cc04823j\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4308616064\", \"doi_x\": \"10.1093/nar/gkac993\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"InterPro in 2022\", \"abstract\": \"Abstract The InterPro database (https://www.ebi.ac.uk/interpro/) provides an integrative classification of protein sequences into families, and identifies functionally important domains and conserved sites. Here, we report recent developments with InterPro (version 90.0) and its associated software, including updates to data content and to the website. These developments extend and enrich the information provided by InterPro, and provide a more user friendly access to the data. Additionally, we have worked on adding Pfam website features to the InterPro website, as the Pfam website will be retired in late 2022. We also show that InterPro's sequence coverage has kept pace with the growth of UniProtKB. Moreover, we report the development of a card game as a method of engaging the non-scientific community. Finally, we discuss the benefits and challenges brought by the use of artificial intelligence for protein structure prediction.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac993\", \"publication_date\": \"2022-11-09\"}, {\"work_id\": \"W4315754439\", \"doi_x\": \"10.1128/jvi.00872-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"abstract\": \"The interferon system is part of the innate immune response that inhibits viruses and other pathogens. This system emerged approximately 500 million years ago in early vertebrates.\", \"doi_y\": \"https://doi.org/10.1128/jvi.00872-22\", \"publication_date\": \"2023-01-31\"}, {\"work_id\": \"W4306857359\", \"doi_x\": \"10.1016/j.jbc.2022.102624\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A bovine antibody possessing an ultralong complementarity-determining region CDRH3 targets a highly conserved epitope in sarbecovirus spike proteins\", \"abstract\": \"Broadly neutralizing antibodies have huge potential as novel antiviral therapeutics due to their ability to recognize highly conserved epitopes that are seldom mutated in viral variants. A subset of bovine antibodies possess an ultralong complementarity-determining region (CDR)H3 that is highly adept at recognizing such conserved epitopes, but their reactivity against Sarbecovirus Spike proteins has not been explored previously. Here, we use a SARS-na\\u00efve library to isolate a broadly reactive bovine CDRH3 that binds the receptor-binding domain of SARS-CoV, SARS-CoV-2, and all SARS-CoV-2 variants. We show further that it neutralizes viruses pseudo-typed with SARS-CoV Spike, but this is not by competition with angiotensin-converting enzyme 2 (ACE2) binding. Instead, using differential hydrogen-deuterium exchange mass spectrometry, we demonstrate that it recognizes the major site of vulnerability of Sarbecoviruses. This glycan-shielded cryptic epitope becomes available only transiently via interdomain movements of the Spike protein such that antibody binding triggers destruction of the prefusion complex. This proof of principle study demonstrates the power of in vitro expressed bovine antibodies with ultralong CDRH3s for the isolation of novel, broadly reactive tools to combat emerging pathogens and to identify key epitopes for vaccine development.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102624\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4380365679\", \"doi_x\": \"10.1016/j.str.2023.05.012\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"abstract\": \"\\u2022We show that NMR structures typically lack hydrogen bonds\\u2022We present a protocol for gradual inclusion of hydrogen bond restraints\\u2022The resulting structures are better using many measures of quality\\u2022ANSURR provides a measure for when the NMR structure calculation is completed Protein structures calculated using NMR data are less accurate and less well-defined than they could be. Here we use the program ANSURR to show that this deficiency is at least in part due to a lack of hydrogen bond restraints. We describe a protocol to introduce hydrogen bond restraints into the structure calculation of the SH2 domain from SH2B1 in a systematic and transparent way and show that the structures generated are more accurate and better defined as a result. We also show that ANSURR can be used as a guide to know when the structure calculation is good enough to stop. Protein structures calculated using NMR data are less accurate and less well-defined than they could be. Here we use the program ANSURR to show that this deficiency is at least in part due to a lack of hydrogen bond restraints. We describe a protocol to introduce hydrogen bond restraints into the structure calculation of the SH2 domain from SH2B1 in a systematic and transparent way and show that the structures generated are more accurate and better defined as a result. We also show that ANSURR can be used as a guide to know when the structure calculation is good enough to stop. We recently introduced a new method, ANSURR (Accuracy of NMR structures Using RCI and Rigidity), which permits a simple and robust indication of the accuracy of NMR structures.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar,3Fowler N.J. Williamson M.P. The accuracy of protein structures in solution determined by AlphaFold and NMR.Structure. 2022; 30: 1-9https://doi.org/10.1101/2022.01.18.476751Crossref PubMed Scopus (0) Google Scholar We noted2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar that when estimated in this way, the accuracy of NMR protein structures improved until about 2005 but has not changed much since; and we suggested that one reason for the lack of improvement is that there has been until now no reliable way to calculate accuracy, and therefore to measure the success of technical improvements. With the introduction of ANSURR, we have a tool that increases our confidence in measuring whether new methods are effective. In this paper, we take the first step in implementing this idea, by addressing one experimental restraint that has proved difficult to implement, namely hydrogen bond restraints (HBRs). The most widely used restraints for NMR structure calculation are nuclear Overhauser effects (NOEs). These are easily obtained from nuclear Overhauser effect spectroscopy (NOESY) spectra and have (in principle) a simple interpretation, in that under appropriate conditions, an NOE peak denotes the close distance between two protons, with an intensity proportional to r\\u22126, where r is the internuclear distance.4Neuhaus D. Williamson M.P. The Nuclear Overhauser Effect in Structural and Conformational Analysis, 2nd Edition. Wiley-VCH, 2000Google Scholar Another common restraint is to use backbone chemical shifts to produce restraints on dihedral angles, for example using TALOS-N.5Shen Y. Bax A. Protein backbone and sidechain torsion angles predicted from NMR chemical shifts using artificial neural networks.J. Biomol. NMR. 2013; 56: 227-241https://doi.org/10.1007/s10858-013-9741-yCrossref PubMed Scopus (740) Google Scholar Hydrogen bonds are one of the few other common restraints and are very strong, because the distances are short and have a small range between upper and lower bounds. Restraint violations therefore typically become unfavorable with relatively small changes in geometry. For this reason, HBRs should be applied with caution, because they effectively force the relevant groups to form a hydrogen bond. Hydrogen bonds are more difficult restraints to apply than NOEs because they cannot usually be obtained directly from experimental data. NOE restraints come directly from NOESY spectra and dihedral restraints from backbone chemical shifts. It is possible to observe hydrogen bonds directly from observation of 3J across the hydrogen bond.6Cordier F. Grzesiek S. Direct observation of hydrogen bonds in proteins by interresidue 3hJNC ' scalar couplings.J. Am. Chem. Soc. 1999; 121: 1601-1602https://doi.org/10.1021/ja983945dCrossref Scopus (370) Google Scholar However, the coupling constant is small and for most proteins the signal is too weak to be detected. Therefore the presence of a hydrogen bond has to be inferred. Hydrogen bonds in regular secondary structure can be predicted based on NOE patterns in the vicinity,7W\\u00fcthrich K. Billeter M. Braun W. Polypeptide secondary structure determination by nuclear magnetic resonance observation of short proton-proton distances.J. Mol. Biol. 1984; 180: 715-740https://doi.org/10.1016/0022-2836(84)90034-2Crossref PubMed Scopus (747) Google Scholar but a more secure identification of hydrogen bonds relies on secondary factors such as slow amide exchange of the amide proton8Wagner G. Kumar A. W\\u00fcthrich K. Systematic application of two-dimensional 1H NMR techniques for studies of proteins. 2. Combined use of correlated spectroscopy and nuclear Overhauser spectroscopy for sequential ssignments of backbone resonances and elucidation of polypeptide secondary structures.Eur. J. Biochem. 1981; 114: 375-384https://doi.org/10.1111/j.1432-1033.1981.tb05157.xCrossref PubMed Scopus (272) Google Scholar,9Williamson M.P. Havel T.F. W\\u00fcthrich K. Solution conformation of proteinase inhibitor IIA from bull seminal plasma by 1H nuclear magnetic resonance and distance geometry.J. Mol. Biol. 1985; 182: 295-315Crossref PubMed Scopus (532) Google Scholar or on the amide proton temperature coefficient.10Baxter N.J. Williamson M.P. Temperature dependence of 1H chemical shifts in proteins.J. Biomol. NMR. 1997; 9: 359-369Crossref PubMed Google Scholar Neither of these is completely reliable, meaning that any inference of a hydrogen bond is potentially wrong. Significantly, amide exchange and temperature coefficients identify the amide group but do not identify the carbonyl donor. Therefore for most hydrogen bonds, identification of the donor is based on inferences not direct observation. For these reasons, NMR spectroscopists have generally been cautious in applying HBRs in protein structure calculation. There is no \\u201cstandard protocol,\\u201d but typically HBRs are only applied toward the end of the calculation, the suspicion being that it may not be scientifically justified to use them as part of the standard structure calculation. There has thus emerged a sense that HBRs are not as \\u201crespectable\\u201d as NOEs or dihedral restraints, as a result of which, the details of hydrogen bond restraints are often deeply buried in papers, and it is often very hard to work out from the literature exactly how HBRs were generated and used. This is a pity, because HBRs are such powerful restraints, and therefore very useful if used correctly. The main aim of this work is to propose a justifiable and transparent protocol. We describe a robust iterative approach to the addition of HBRs, monitored using ANSURR, and demonstrate the effect of HBRs on the structure calculation of the SH2 domain from SH2B1. We show that the accuracy of the structures improves as HBRs are added. SH2B1 is a signaling scaffold protein that acts downstream of the insulin and leptin receptors11Maures T.J. Kurzer J.H. Carter-Su C. SH2B1 (SH2-B) and JAK2: a multifunctional adaptor protein and kinase made for each other.Trends Endocrinol. Metabol. 2007; 18: 38-45Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar and regulates the activity of JAK2 kinase. Mutations to its SH2 domain have been shown to disrupt the leptin signaling pathway12Li Z. Zhou Y. Carter-Su C. Myers Jr., M.G. Rui L. SH2B1 enhances leptin signaling by both janus kinase 2 Tyr813 phosphorylation-dependent and -independent mechanisms.Mol. Endocrinol. 2007; 21: 2270-2281https://doi.org/10.1210/me.2007-0111Crossref PubMed Scopus (86) Google Scholar; deletion leads to an obese phenotype, and severely disrupts signaling downstream of the insulin receptor.13Morris D.L. Cho K.W. Zhou Y. Rui L. SH2B1 enhances insulin sensitivity by both stimulating the insulin receptor and inhibiting tyrosine dephosphorylation of insulin receptor substrate proteins.Diabetes. 2009; 58: 2039-2047https://doi.org/10.2337/db08-1388Crossref PubMed Scopus (72) Google Scholar SH2B1 consists of a dimerization domain, a PH domain, and an SH2 domain, which together form less than half of the protein, the rest being intrinsically disordered.11Maures T.J. Kurzer J.H. Carter-Su C. SH2B1 (SH2-B) and JAK2: a multifunctional adaptor protein and kinase made for each other.Trends Endocrinol. Metabol. 2007; 18: 38-45Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar Part of the function of SH2 is to bind to JAK2 kinase via its phosphorylated pTyr813 and modulate its kinase activity.14Rui L. Gunter D.R. Herrington J. Carter-Su C. Differential binding to and regulation of JAK2 by the SH2 domain and N-terminal region of SH2-B \\u03b2.Mol. Cell Biol. 2000; 20: 3168-3177https://doi.org/10.1128/mcb.20.9.3168-3177.2000Crossref PubMed Scopus (0) Google Scholar It is thus a protein with important biological effects. It is also an example of a well-studied domain with a crystal structure. For the purposes of this study this is useful because the crystal structure provides us with a useful guide to accuracy. The aim of this study is to present an iterative approach to introducing hydrogen bond restraints (HBRs) into an NMR structure calculation, and to investigate whether this improves the accuracy of the structure. Our primary measure of accuracy is ANSURR. ANSURR calculates the rigidity of a protein structure using the algorithm Floppy Inclusions and Rigid Substructure Topography (FIRST)15Jacobs D.J. Rader A.J. Kuhn L.A. Thorpe M.F. Protein flexibility predictions using graph theory.Proteins. 2001; 44: 150-165https://doi.org/10.1002/prot.1081Crossref PubMed Scopus (609) Google Scholar and compares it to the rigidity derived from experimental backbone chemical shifts via the random coil index (RCI).16Berjanskii M.V. Wishart D.S. Application of the random coil index to studying protein flexibility.J. Biomol. NMR. 2008; 40: 31-48https://doi.org/10.1007/s10858-007-9208-0Crossref PubMed Scopus (90) Google Scholar The similarity obtained by comparing these two measures provides the accuracy. The comparison uses two measures: correlation, which calculates the extent to which peaks and troughs in rigidity along the sequence match, and is primarily a measure of the correctness of local secondary structure; and RMSD, which compares rigidity along the sequence (Figure S1). Our previous studies1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar showed that although the correlation for NMR structures is often good, the RMSD is often poor, in almost all cases because NMR structures are too floppy in comparison to the rigidity indicated by backbone chemical shifts. We suggested that a major cause of this floppiness is that NMR structures have too few hydrogen bonds, in comparison to either crystal structures2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar or AlphaFold structures.3Fowler N.J. Williamson M.P. The accuracy of protein structures in solution determined by AlphaFold and NMR.Structure. 2022; 30: 1-9https://doi.org/10.1101/2022.01.18.476751Crossref PubMed Scopus (0) Google Scholar This was the rationale behind the present study. Before starting these structure calculations, we therefore undertook a more thorough study of the relationship between hydrogen bond content and accuracy. A set of 215 pairs of published NMR and X-ray structures was assembled (Table S1). This set was used to compute the average hydrogen bond correctness of each NMR ensemble, defined as the average percentage of hydrogen bonds in an X-ray structure that appear in an NMR ensemble. This calculation relies on the assumption that the hydrogen bonds in a crystal structure are also present in solution, which is clearly not completely true, but is a good approximation.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar We computed the average ANSURR score (the sum of correlation and RMSD scores, and thus a ranked score out of 200) for each ensemble and compared this to the average hydrogen bond correctness. Figure 1A shows that NMR structures with higher hydrogen bond correctness tend to be more accurate as measured by ANSURR, and also indicates that the hydrogen bonds in X-ray structures generally represent those that appear in solution. Plotting the number of hydrogen bonds per residue in each ensemble against hydrogen bond correctness (Figure 1B) demonstrates that poor scoring NMR ensembles are primarily inaccurate because they lack hydrogen bonds, rather than because they contain incorrect hydrogen bonds. We conclude that the accuracy of NMR structures will be improved by incorporation of HBRs but only if they are correct. This is important, because it implies that ANSURR scores will not improve on addition of incorrect HBRs. In the subsequent text, we therefore explore how to add HBRs, monitoring the accuracy of the resultant structures using ANSURR but also comparing to other measures. A plasmid for expression of mouse SH2 was constructed and inserted into Escherichia coli. The expressed protein is 118 residues long, with an 8-residue His-tag at the N-terminus, followed by the sequence D9QPLS \\u2026, with D9 corresponding to D519 of mature mouse SH2B. The protein was expressed and labeled uniformly with 15N and 13C. One aim of this project was to test out the automated assignment/structure calculation in FLYA/CYANA.17L\\u00f3pez-M\\u00e9ndez B. G\\u00fcntert P. Automated protein structure determination from NMR spectra.J. Am. Chem. Soc. 2006; 128: 13112-13122Crossref PubMed Scopus (136) Google Scholar The NMR spectrum was therefore assigned automatically using CYANA, as well as manually. The NMR spectra used are listed in Table S2, and the final manually assigned HSQC is shown in Figure S2. A comparison of the two assignments showed that 99% of the automated backbone chemical shift assignments (1HN, 15N, 1H\\u03b1, 13C\\u03bf, 13C\\u03b1, and 13C\\u03b2) of SH2 (9\\u2013118) agreed with the manual assignment. The only resonances that were not assigned correctly by the automated calculation are the amide chemical shifts of Trp17 and Gln61, which are two of the four resonances that had missing signals (but assigned H and N) in the manual assignment. For aliphatic methyl and methylene sidechain groups, the agreement was 97%. On the other hand, many of the NOE-based chemical shift assignments did not match the manual assignment. This includes the amide sidechains of Arg, Asn, and Gln, and many aromatics: only 53% of these were assigned correctly (Figure S3). CYANA continues to check and refine its assignments throughout the structure calculations, and it is possible that some of these incorrect assignments would have been corrected. On the other hand, the accuracy of the automated structure calculation relies heavily on the accuracy of the first iteration, which in turn depends on the accuracy of the initial set of assignments.18G\\u00fcntert P. Buchner L. Combined automated NOE assignment and structure calculation with CYANA.J. Biomol. NMR. 2015; 62: 453-471https://doi.org/10.1007/s10858-015-9924-9Crossref PubMed Scopus (233) Google Scholar The clear implication is that assignments should be checked manually before doing automated structure calculations.19Williamson M.P. Craven C.J. Automated protein structure calculation from NMR data.J. Biomol. NMR. 2009; 43: 131-143Crossref PubMed Scopus (0) Google Scholar A manually checked set of assignments have been deposited in BioMagResBank20Ulrich E.L. Akutsu H. Doreleijers J.F. Harano Y. Ioannidis Y.E. Lin J. Livny M. Mading S. Maziuk D. Miller Z. et al.BioMagResBank. Nucleic Acids Res. 2008; 36: D402-D408Crossref PubMed Scopus (1314) Google Scholar under code 51342. Structure calculation was conducted using the standard automated routines within CYANA. We used the default CYANA calculation of 20 structures. We then repeated this calculation 60 times changing the random seed each time, in order to obtain a better \\u201cconsensus bundle\\u201d of structures,21Buchner L. G\\u00fcntert P. Increased reliability of nuclear magnetic resonance protein structures by consensus structure bundles.Structure. 2015; 23: 425-434https://doi.org/10.1016/j.str.2014.11.014Abstract Full Text Full Text PDF PubMed Scopus (21) Google Scholar giving a total of 1200 calculated structures per iteration. The reason for carrying out so many calculations was to generate a reliable set of HBRs, with good discrimination between correct and incorrect HBRs. The figure of 60 was determined by calculating the ratio of true positive HBRs (that is, HBRs identified that are also present in the crystal structure) to false positive HBRs (HBRs identified that are not in the crystal structure). This ratio improved with the number of calculations, reaching a plateau at 60 (Figure S4). HBRs were identified from these structures and used as restraints for the next iteration, as described below. The initial round of CYANA calculations produced an ensemble with a correct fold but low accuracy (as measured by ANSURR), and had poor convergence, a large RMSD between structures, and poor energies, all symptoms of a weak restraint set. We therefore improved the structures by iteratively and systematically including backbone hydrogen bond restraints, in such a way as to avoid incorrectly biasing the structure calculation (Figure 2). The protocol aims to generate HBRs that are present in a significant fraction of ensemble structures, supported by experimental temperature coefficients and/or amide exchange rates, and are consistent with other restraints. Iteration 6 was introduced as a way of removing any HBRs that were incompatible with the experimental restraints. The complete process identified 47 hydrogen bond restraints, mostly in regular secondary structure. The final set of HBRs is illustrated in Figure 3. Not surprisingly, the first HBRs to be identified came from the helices and from the \\u201ccore\\u201d \\u03b2-strands (residues 41\\u201369). The shorter \\u03b2-sheet HBRs tended to be added later. Many of the HBRs that do not form part of regular secondary structures (for example, HBRs from the amide protons of G14, Y15, G104, and V109) were identified in the final stages. More details are available in Table S3, which also shows that most of the HBRs that were added in an early iteration and subsequently removed are present in the crystal structure. At each stage in the calculation, the structures were analyzed to measure whether the additional HBRs were improving the quality of the ensemble. Our primary analytical tool was ANSURR. Figure 4 shows how ANSURR scores steadily increase as the number of HBRs increases, suggesting that inclusion of hydrogen bond restraints does improve accuracy. The best scoring models in the final iteration 6 approach the accuracy of AlphaFold222Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. Tunyasuvunakool K. Bates R. \\u017d\\u00eddek A. Potapenko A. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589https://doi.org/10.1038/s41586-021-03819-2Crossref PubMed Scopus (6941) Google Scholar and Rosetta-Fold23Baek M. DiMaio F. Anishchenko I. Dauparas J. Ovchinnikov S. Lee G.R. Wang J. Cong Q. Kinch L.N. Schaeffer R.D. et al.Accurate prediction of protein structures and interactions using a three-track neural network.Science. 2021; 373: 871-876https://doi.org/10.1126/science.abj8754Crossref PubMed Scopus (1299) Google Scholar (Figure 4A), although the average values remain worse. Note that the structures shown here come directly from CYANA and have not yet been refined in explicit solvent. The final iteration was designed to remove HBRs that do not have good supporting evidence. This iteration removed 4 out of 51 HBRs, leading to a drop in ANSURR score as well as some of the other measures of accuracy discussed below. The implication is that we probably removed correct HBRs, so this final step may have been over-cautious. Figures 4B\\u20134G depicts the ANSURR output for the best scoring model from each iteration. Figures 4H and 4I show ANSURR output for the X-ray structure (PDB: 5w3r) and the best scoring AlphaFold2 prediction, respectively. The crystal structure has a very good ANSURR score, but appears to be more rigid than the \\u201ctrue\\u201d solution structure around residues 70\\u201390, judging by the rigidities shown in Figure 4H. With this exception, it would appear to be an excellent guide to the hydrogen bonds present in solution. In support, in Figure S5 we present an analysis of the HBRs at each iteration compared to those in the crystal structure, and show that they become progressively closer to those in the crystal structure. We note that the amino acid sequence of the proteins used for NMR and crystallography24McKercher M.A. Guan X. Tan Z. Wuttke D.S. Diversity in peptide recognition by the SH2 domain of SH2B1.Proteins. 2018; 86: 164-176https://doi.org/10.1002/prot.25420Crossref PubMed Scopus (4) Google Scholar (mouse and human, respectively) differ in 5 locations (Figure S6). It is of interest that the region for which NMR does worst is residues 46\\u201355, which is too flexible in the NMR structures but is better in the X-ray structure. In the crystal, there is a bound phosphate here. This is where phosphotyrosine binds. The residues in contact with the phosphate in the crystal structure (residues 47\\u201349) appear to be too rigid, suggesting that the phosphate is more tightly bound in the crystal than in solution, or perhaps that the flexibility according to chemical shifts represents an average of a bound state and unbound state. We have compared backbone chemical shifts for protein in 50 mM Tris buffer and in 50 mM phosphate buffer, pH 6 (Figure S7). It is striking that almost all of the large chemical shift changes surround the location of the phosphate ion in the crystal structure, implying that phosphate binds here in 50 mM phosphate buffer. This may serve to rigidify the region, and explain why the NMR structure, which was determined in tris buffer, has low rigidity here. The previous section showed that inclusion of HBRs leads to a steady improvement in ANSURR score: i.e., a more accurate structure. Here, we report on other commonly used measures of quality, to characterize the structural improvement in more detail. The results are shown in Figure 5. The magnitude of NOE violations increases gradually as HBRs are included, although it remains low throughout (Figure 5A). The calculation of an NMR structure is a balance between forces pulling in different directions: for example, van der Waals violations push atoms apart, while NOE violations pull them together. For this reason, it is generally found that addition of a new type of restraint leads to a slight increase in violations of existing restraints.25Schwieters C.D. Bermejo G.A. Clore G.M. A three-dimensional potential of mean force to improve backbone and sidechain hydrogen bond geometry in Xplor-NIH protein structure determination.Protein Sci. 2020; 29: 100-110https://doi.org/10.1002/pro.3745Crossref PubMed Scopus (9) Google Scholar We therefore do not see this result as surprising or indicative of structural errors. In a similar way, the magnitude of dihedral angle violations has a small but insignificant increase (Figure 5B), as do van der Waals violations (data not shown). The RMSD of backbone atoms to the crystal structure improves on initial addition of HBRs, up to iteration 4 after which there is little change (Figure 5C). This can be understood by the argument that HBRs are excellent for improving local geometry, but make little difference to the positions of backbone atoms. It is widely agreed that the template modeling score (TM-score) is a more accurate measure of global similarity than RMSD,26Zhang Y. Skolnick J. Scoring function for automated assessment of protein structure template quality.Proteins. 2004; 57: 702-710https://doi.org/10.1002/prot.20264Crossref PubMed Scopus (1268) Google Scholar and we see a more clear trend for TM-score than for RMSD (Figure 5D). TM-score slightly increases up until iteration 4. Residual dipolar couplings (RDCs) can be used either to validate the quality of NMR structures or as restraints to improve NMR structures, but not both at the same time (unless steps are taken to separate them into different groups).27Chiliveri S.C. Robertson A.J. Shen Y. Torchia D.A. Bax A. Advances in NMR spectroscopy of weakly aligned biomolecular systems.Chem. Rev. 2022; 122: 9307-9330https://doi.org/10.1021/acs.chemrev.1c00730Crossref PubMed Scopus (6) Google Scholar Here we measured a set of 1DNH as a simple validation measure. Figure 5E shows the RDC Q factor,28Cornilescu G. Marquardt J.L. Ottiger M. Bax A. Validation of protein structure from anisotropic carbonyl chemical shifts in a dilute liquid crystalline phase.J. Am. Chem. Soc. 1998; 120: 6836-6837https://doi.org/10.1021/ja9812610Crossref Scopus (787) Google Scholar which improves up to iteration 4. The chemical shifts of amide proteins should be a sensitive measure of structural accuracy, because they are strongly affected by hydrogen bonding.29Asakura T. Taoka K. Demura M. Williamson M.P. The relationship between amide proton chemical shifts and secondary structure in proteins.J. Biomol. NMR. 1995; 6: 227-236Crossref PubMed Scopus (115) Google Scholar We therefore calculated the expected amide proton shifts using SHIFTX230Han B. Liu Y. Ginzinger S.W. Wishart D.S. SHIFTX2: significantly improved protein chemical shift prediction.J. Biomol. NMR. 2011; 50: 43-57Crossref PubMed Scopus (468) Google Scholar and compared them to the experimental shifts (Figure 5F), which improve steadily as HBRs are added. The distribution of Ramachandran angles also improves slightly up to iteration 4 (Figure 5G). Finally, we have calculated the backbone ensemble RMSD (Figure 5H). This is a measure of precision rather than accuracy, but our survey of PDB structures indicated that it does correlate with accuracy, presumably because adding more (correct) restraints serves to improve both accuracy and precision at the same time2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar; and indeed, RMSD decreases steadily as HBRs are added. Thus in summary, the gradual addition of 47 HBRs results in an improvement in accuracy, as measured by a wide range of parameters, with the improvement in accuracy slowing down as the number of HBRs increases. These metrics clearly show that there is overall no increased penalty to the structure quality from their inclusion: in other words, the HBRs are correct and do not conflict with the other structural restraints. The accuracy clearly improves until iteration 4. Iteration 5 results in an improvement on some measures but a deterioration in others, while iteration 6 (in which some HBRs were removed for lack of good evidence) represents a deterioration in almost all measures, the exception being precision (Figure 5H). It is well established that refinement in solvent is required in order to produce good quality structures.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,31Linge J.P. Williams M.A. Spronk C.A.E.M. Bonvin A.M.J.J. Nilges M. Refinement of protein structures in explicit solvent.Proteins. 2003; 50: 496-506https://doi.org/10.1002/prot.10299Crossref PubMed Scopus (544) Google Scholar The 1,200 structures produced in each iteration above were therefore refined in water using CNS.32Br\\u00fcnger A.T. Adams P.D. Clore G.M. DeLano W.L. Gros P. Grosse-Kunstleve R.W. Jiang J.S. Kuszewski J. Nilges M. Pannu N.S. et al.Crystallography & NMR system: a new software suite for macromolecular structure determination.Acta Crystallogr. D. 1998; 54: 905-921Crossref PubMed Scopus (16979) Google Scholar There is a reasonable correlation between the energies before and after refinement (Figure S8). The distance, dihedral a\", \"doi_y\": \"https://doi.org/10.1016/j.str.2023.05.012\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4380085975\", \"doi_x\": \"10.1038/s41467-023-39140-x\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A 2.2 \\u00c5 cryoEM structure of a quinol-dependent NO Reductase shows close similarity to respiratory oxidases\", \"abstract\": \"Abstract Quinol-dependent nitric oxide reductases (qNORs) are considered members of the respiratory heme-copper oxidase superfamily, are unique to bacteria, and are commonly found in pathogenic bacteria where they play a role in combating the host immune response. qNORs are also essential enzymes in the denitrification pathway, catalysing the reduction of nitric oxide to nitrous oxide. Here, we determine a 2.2 \\u00c5 cryoEM structure of qNOR from Alcaligenes xylosoxidans , an opportunistic pathogen and a denitrifying bacterium of importance in the nitrogen cycle. This high-resolution structure provides insight into electron, substrate, and proton pathways, and provides evidence that the quinol binding site not only contains the conserved His and Asp residues but also possesses a critical Arg (Arg720) observed in cytochrome bo 3 , a respiratory quinol oxidase.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-39140-x\", \"publication_date\": \"2023-06-09\"}, {\"work_id\": \"W4308878080\", \"doi_x\": \"10.3389/fendo.2022.1031013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Identification of unusual oxysterols biosynthesised in human pregnancy by charge-tagging and liquid chromatography - mass spectrometry\", \"abstract\": \"The aim of this study was to identify oxysterols and any down-stream metabolites in placenta, umbilical cord blood plasma, maternal plasma and amniotic fluid to enhance our knowledge of the involvement of these molecules in pregnancy. We confirm the identification of 20S-hydroxycholesterol in human placenta, previously reported in a single publication, and propose a pathway from 22R-hydroxycholesterol to a C 27 bile acid of probable structure 3\\u03b2,20R,22R-trihydroxycholest-5-en-(25R)26-oic acid. The pathway is evident not only in placenta, but pathway intermediates are also found in umbilical cord plasma, maternal plasma and amniotic fluid but not non-pregnant women.\", \"doi_y\": \"https://doi.org/10.3389/fendo.2022.1031013\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W4205120081\", \"doi_x\": \"10.1101/2022.01.14.476343\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"abstract\": \"Abstract Tpp49Aa1 from Lysinibacillus sphaericus is a Toxin_10 family protein that \\u2013 in combination with Cry48Aa1, a 3-domain crystal protein - has potent mosquitocidal activity, specifically against Culex quinquefasciatus mosquitoes. MHz serial femtosecond crystallography at a nano-focused X-ray free electron laser, allowed rapid and high-quality data collection to determine the Tpp49Aa1 structure at 1.62 \\u00c5 resolution from native nanocrystals. This revealed the packing of Tpp49Aa1 within these nanocrystals, isolated from sporulated bacteria, as a homodimer with a large intermolecular interface, shedding light on natural crystallization. Complementary experiments conducted at varied pH also enabled investigations of the early structural events leading up to the dissolution of natural Tpp49Aa1 crystals. Using modelling, we propose a potential interaction between Tpp49Aa1 and Cry48Aa1 that may play a role in their codependency and broaden our understanding of this two-component system. We expand the known target range, demonstrating Tpp49Aa1/Cry48Aa1 susceptibility of larvae from Anopheles stephensi, Aedes albopictus and Culex tarsalis \\u2013 substantially increasing the potential use of this toxin pair in mosquito control. Further functional insights are gained using Culex cell lines to characterise cellular models for future investigations into Cry48Aa1/Tpp49Aa1 mechanism of action and to demonstrate transient detrimental effects of individual toxin components. Significance Statement The Tpp49Aa1/Cry48Aa1 protein pair kills mosquito larvae. Innovative use of nano-focused X-ray free electron laser to match the size of natural Tpp49Aa1 nanocrystals and the highest beam intensity available in any XFEL for high-throughput data collection, allowed structural resolution to 1.62 \\u00c5. Tpp proteins show a range of interactions with different partners to elicit toxicity. To gain insight into Tpp49Aa1, its interaction with Cry48Aa1 was modelled. We also establish cell-based assays of Tpp49Aa1/Cry48Aa1 activity. We expand the known target range to include three more mosquito species: Anopheles stephensi, Aedes albopictus and Culex tarsalis . This study will underpin future Tpp mode of action investigations and aid insecticide optimization against mosquito vectors of emerging diseases such as West Nile Virus and malaria.\", \"doi_y\": \"https://doi.org/10.1101/2022.01.14.476343\", \"publication_date\": \"2022-01-15\"}, {\"work_id\": \"W4318071688\", \"doi_x\": \"10.1042/bcj20220480\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"abstract\": \"Gram-negative bacteria are surrounded by two protein-rich membranes with a peptidoglycan layer sandwiched between them. Together they form the envelope (or cell wall), crucial for energy production, lipid biosynthesis, structural integrity, and for protection against physical and chemical environmental challenges. To achieve envelope biogenesis, periplasmic and outer-membrane proteins (OMPs) must be transported from the cytosol and through the inner-membrane, via the ubiquitous SecYEG protein-channel. Emergent proteins either fold in the periplasm or cross the peptidoglycan (PG) layer towards the outer-membrane for insertion through the \\u03b2-barrel assembly machinery (BAM). Trafficking of hydrophobic proteins through the periplasm is particularly treacherous given the high protein density and the absence of energy (ATP or chemiosmotic potential). Numerous molecular chaperones assist in the prevention and recovery from aggregation, and of these SurA is known to interact with BAM, facilitating delivery to the outer-membrane. However, it is unclear how proteins emerging from the Sec-machinery are received and protected from aggregation and proteolysis prior to an interaction with SurA. Through biochemical analysis and electron microscopy we demonstrate the binding capabilities of the unoccupied and substrate-engaged SurA to the inner-membrane translocation machinery complex of SecYEG-SecDF-YidC - aka the holo-translocon (HTL). Supported by AlphaFold predictions, we suggest a role for periplasmic domains of SecDF in chaperone recruitment to the protein translocation exit site in SecYEG. We propose that this immediate interaction with the enlisted chaperone helps to prevent aggregation and degradation of nascent envelope proteins, facilitating their safe passage to the periplasm and outer-membrane.\", \"doi_y\": \"https://doi.org/10.1042/bcj20220480\", \"publication_date\": \"2023-02-27\"}, {\"work_id\": \"W4323294987\", \"doi_x\": \"10.1080/10409238.2023.2181309\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"abstract\": \"Understanding how Nature accomplishes the reduction of inert nitrogen gas to form metabolically tractable ammonia at ambient temperature and pressure has challenged scientists for more than a century. Such an understanding is a key aspect toward accomplishing the transfer of the genetic determinants of biological nitrogen fixation to crop plants as well as for the development of improved synthetic catalysts based on the biological mechanism. Over the past 30 years, the free-living nitrogen-fixing bacterium Azotobacter vinelandii emerged as a preferred model organism for mechanistic, structural, genetic, and physiological studies aimed at understanding biological nitrogen fixation. This review provides a contemporary overview of these studies and places them within the context of their historical development.\", \"doi_y\": \"https://doi.org/10.1080/10409238.2023.2181309\", \"publication_date\": \"2022-11-02\"}, {\"work_id\": \"W4294311651\", \"doi_x\": \"10.1021/acs.biomac.2c00781\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Probing the Internal pH and Permeability of a Carboxysome Shell\", \"abstract\": \"The carboxysome is a protein-based nanoscale organelle in cyanobacteria and many proteobacteria, which encapsulates the key CO2-fixing enzymes ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco) and carbonic anhydrase (CA) within a polyhedral protein shell. The intrinsic self-assembly and architectural features of carboxysomes and the semipermeability of the protein shell provide the foundation for the accumulation of CO2 within carboxysomes and enhanced carboxylation. Here, we develop an approach to determine the interior pH conditions and inorganic carbon accumulation within an \\u03b1-carboxysome shell derived from a chemoautotrophic proteobacterium Halothiobacillus neapolitanus and evaluate the shell permeability. By incorporating a pH reporter, pHluorin2, within empty \\u03b1-carboxysome shells produced in Escherichia coli, we probe the interior pH of the protein shells with and without CA. Our in vivo and in vitro results demonstrate a lower interior pH of \\u03b1-carboxysome shells than the cytoplasmic pH and buffer pH, as well as the modulation of the interior pH in response to changes in external environments, indicating the shell permeability to bicarbonate ions and protons. We further determine the saturated HCO3\\u2013 concentration of 15 mM within \\u03b1-carboxysome shells and show the CA-mediated increase in the interior CO2 level. Uncovering the interior physiochemical microenvironment of carboxysomes is crucial for understanding the mechanisms underlying carboxysomal shell permeability and enhancement of Rubisco carboxylation within carboxysomes. Such fundamental knowledge may inform reprogramming carboxysomes to improve metabolism and recruit foreign enzymes for enhanced catalytical performance.\", \"doi_y\": \"https://doi.org/10.1021/acs.biomac.2c00781\", \"publication_date\": \"2022-09-02\"}, {\"work_id\": \"W3178087467\", \"doi_x\": \"10.1002/prot.26171\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High\\u2010accuracy protein structure prediction in <scp>CASP14</scp>\", \"abstract\": \"The application of state-of-the-art deep-learning approaches to the protein modeling problem has expanded the \\\"high-accuracy\\\" category in CASP14 to encompass all targets. Building on the metrics used for high-accuracy assessment in previous CASPs, we evaluated the performance of all groups that submitted models for at least 10 targets across all difficulty classes, and judged the usefulness of those produced by AlphaFold2 (AF2) as molecular replacement search models with AMPLE. Driven by the qualitative diversity of the targets submitted to CASP, we also introduce DipDiff as a new measure for the improvement in backbone geometry provided by a model versus available templates. Although a large leap in high-accuracy is seen due to AF2, the second-best method in CASP14 out-performed the best in CASP13, illustrating the role of community-based benchmarking in the development and evolution of the protein structure prediction field.\", \"doi_y\": \"https://doi.org/10.1002/prot.26171\", \"publication_date\": \"2021-07-14\"}, {\"work_id\": \"W3194187547\", \"doi_x\": \"10.1002/prot.26223\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Computational models in the service of X\\u2010ray and <scp>cryo\\u2010</scp>electron microscopy structure determination\", \"abstract\": \"Critical assessment of structure prediction (CASP) conducts community experiments to determine the state of the art in computing protein structure from amino acid sequence. The process relies on the experimental community providing information about not yet public or about to be solved structures, for use as targets. For some targets, the experimental structure is not solved in time for use in CASP. Calculated structure accuracy improved dramatically in this round, implying that models should now be much more useful for resolving many sorts of experimental difficulties. To test this, selected models for seven unsolved targets were provided to the experimental groups. These models were from the AlphaFold2 group, who overall submitted the most accurate predictions in CASP14. Four targets were solved with the aid of the models, and, additionally, the structure of an already solved target was improved. An a posteriori analysis showed that, in some cases, models from other groups would also be effective. This paper provides accounts of the successful application of models to structure determination, including molecular replacement for X-ray crystallography, backbone tracing and sequence positioning in a cryo-electron microscopy structure, and correction of local features. The results suggest that, in future, there will be greatly increased synergy between computational and experimental approaches to structure determination.\", \"doi_y\": \"https://doi.org/10.1002/prot.26223\", \"publication_date\": \"2021-09-06\"}, {\"work_id\": \"W4366992750\", \"doi_x\": \"10.1111/nph.18921\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Manipulation of plant immunity via an <scp>mRNA</scp> decapping pathogen effector\", \"abstract\": \"This article is a Commentary on McCombe et al . (2023), 239 : 222\\u2013239 .\", \"doi_y\": \"https://doi.org/10.1111/nph.18921\", \"publication_date\": \"2023-04-25\"}, {\"work_id\": \"W4385620824\", \"doi_x\": \"10.1111/jeu.12994\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<scp>AAK1</scp>\\u2010like: A putative pseudokinase with potential roles in cargo uptake in bloodstream form <i>Trypanosoma brucei</i> parasites\", \"abstract\": \"Abstract Selection and internalization of cargo via clathrin\\u2010mediated endocytosis requires adaptor protein complexes. One complex, AP\\u20102, acts during cargo selection at the plasma membrane. African trypanosomes lack all components of the AP\\u20102 complex, except for a recently identified orthologue of the AP\\u20102\\u2010associated protein kinase 1, AAK1. In characterized eukaryotes, AAK1 phosphorylates the \\u03bc2 subunit of the AP\\u20102 complex to enhance cargo recognition and uptake into clathrin\\u2010coated vesicles. Here, we show that kinetoplastids encode not one, but two AAK1 orthologues: one (AAK1L2) is absent from salivarian trypanosomes, while the other (AAK1L1) lacks important kinase\\u2010specific residues in a range of trypanosomes. These AAK1L1 and AAK1L2 novelties reinforce suggestions of functional divergence in endocytic uptake within salivarian trypanosomes. Despite this, we show that AAK1L1 null mutant Trypanosoma brucei , while viable, display slowed proliferation, morphological abnormalities including swelling of the flagellar pocket, and altered cargo uptake. In summary, our data suggest an unconventional role for a putative pseudokinase during endocytosis and/or vesicular trafficking in T. brucei , independent of AP\\u20102.\", \"doi_y\": \"https://doi.org/10.1111/jeu.12994\", \"publication_date\": \"2023-08-07\"}, {\"work_id\": \"W4307842580\", \"doi_x\": \"10.1016/j.gim.2022.09.010\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Brain monoamine vesicular transport disease caused by homozygous SLC18A2 variants: A study in 42 affected individuals\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.gim.2022.09.010\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4378640426\", \"doi_x\": \"10.1210/endocr/bqad075\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Hypothalamic PVN CRH neurons signal through PVN GABA neurons to suppress GnRH pulse generator frequency in female mice\", \"abstract\": \"Corticotropin-releasing hormone (CRH) neurons in the paraventricular nucleus of the hypothalamus (PVN) are central to the stress response. Chemogenetic activation of PVN CRH neurons decreases LH pulse frequency but the mechanism is unknown. In the present study, optogenetic stimulation of PVN CRH neurons suppressed LH pulse frequency in estradiol-replaced ovariectomized CRH-cre mice, and this effect was augmented or attenuated by intra-PVN GABAA or GABAB receptor antagonism, respectively. PVN CRH neurons signal to local GABA neurons, which may provide a possible indirect mechanism by which PVN CRH neurons suppress LH pulse frequency. Optogenetic stimulation of potential PVN GABAergic projection terminals in the hypothalamic arcuate nucleus in ovariectomized estradiol-replaced Vgat-cre-tdTomato mice via an optic fiber implanted in the arcuate nucleus suppressed LH pulse frequency. To further determine whether PVN CRH neurons signal through PVN GABA neurons to suppress LH pulsatility, we combined recombinase mice with intersectional vectors to selectively target these neurons. CRH-cre::Vgat-FlpO mice expressing the stimulatory opsin ChRmine in non-GABAergic CRH neurons alone or in combination with the inhibitory opsin NpHR3.3 in non-CRH-expressing GABA neurons in the PVN were used. Optogenetic stimulation of non-GABAergic CRH neurons suppressed pulsatile LH secretion; however, LH pulse frequency was not affected when CRH neurons were stimulated and PVN GABA neurons were simultaneously inhibited. Together, these studies demonstrate that suppression of LH pulse frequency in response to PVN CRH neuronal activation is mediated by GABAergic signalling intrinsic to the PVN and may incorporate PVN GABAergic projection to the hypothalamic GnRH pulse generator.\", \"doi_y\": \"https://doi.org/10.1210/endocr/bqad075\", \"publication_date\": \"2023-04-17\"}, {\"work_id\": \"W4309496690\", \"doi_x\": \"10.26508/lsa.202201388\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The \\u03b2I domain promotes active \\u03b21 integrin clustering into mature adhesion sites\", \"abstract\": \"Modulation of integrin function is required in many physiological and pathological settings, such as angiogenesis and cancer. Integrin allosteric changes, clustering, and trafficking cooperate to regulate cell adhesion and motility on extracellular matrix proteins via mechanisms that are partly defined. By exploiting four monoclonal antibodies recognizing distinct conformational epitopes, we show that in endothelial cells (ECs), the extracellular \\u03b2I domain, but not the hybrid or I-EGF2 domain of active \\u03b21 integrins, promotes their FAK-regulated clustering into tensin 1-containing fibrillar adhesions and impairs their endocytosis. In this regard, the \\u03b2I domain-dependent clustering of active \\u03b21 integrins is necessary to favor fibronectin-elicited directional EC motility, which cannot be effectively promoted by \\u03b21 integrin conformational activation alone.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202201388\", \"publication_date\": \"2022-11-21\"}, {\"work_id\": \"W4225822689\", \"doi_x\": \"10.1242/jcs.259234\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Visualising the cytoskeletal machinery in neuronal growth cones using cryo-electron tomography\", \"abstract\": \"ABSTRACT Neurons extend axons to form the complex circuitry of the mature brain. This depends on the coordinated response and continuous remodelling of the microtubule and F-actin networks in the axonal growth cone. Growth cone architecture remains poorly understood at nanoscales. We therefore investigated mouse hippocampal neuron growth cones using cryo-electron tomography to directly visualise their three-dimensional subcellular architecture with molecular detail. Our data showed that the hexagonal arrays of actin bundles that form filopodia penetrate and terminate deep within the growth cone interior. We directly observed the modulation of these and other growth cone actin bundles by alteration of individual F-actin helical structures. Microtubules with blunt, slightly flared or gently curved ends predominated in the growth cone, frequently contained lumenal particles and exhibited lattice defects. Investigation of the effect of absence of doublecortin, a neurodevelopmental cytoskeleton regulator, on growth cone cytoskeleton showed no major anomalies in overall growth cone organisation or in F-actin subpopulations. However, our data suggested that microtubules sustained more structural defects, highlighting the importance of microtubule integrity during growth cone migration.\", \"doi_y\": \"https://doi.org/10.1242/jcs.259234\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W4294844275\", \"doi_x\": \"10.1371/journal.ppat.1010807\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Hypoxia inducible factors regulate infectious SARS-CoV-2, epithelial damage and respiratory symptoms in a hamster COVID-19 model\", \"abstract\": \"Understanding the host pathways that define susceptibility to Severe-acute-respiratory-syndrome-coronavirus-2 (SARS-CoV-2) infection and disease are essential for the design of new therapies. Oxygen levels in the microenvironment define the transcriptional landscape, however the influence of hypoxia on virus replication and disease in animal models is not well understood. In this study, we identify a role for the hypoxic inducible factor (HIF) signalling axis to inhibit SARS-CoV-2 infection, epithelial damage and respiratory symptoms in the Syrian hamster model. Pharmacological activation of HIF with the prolyl-hydroxylase inhibitor FG-4592 significantly reduced infectious virus in the upper and lower respiratory tract. Nasal and lung epithelia showed a reduction in SARS-CoV-2 RNA and nucleocapsid expression in treated animals. Transcriptomic and pathological analysis showed reduced epithelial damage and increased expression of ciliated cells. Our study provides new insights on the intrinsic antiviral properties of the HIF signalling pathway in SARS-CoV-2 replication that may be applicable to other respiratory pathogens and identifies new therapeutic opportunities.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1010807\", \"publication_date\": \"2022-09-06\"}, {\"work_id\": \"W4361279418\", \"doi_x\": \"10.22331/q-2023-03-30-966\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Characterization of variational quantum algorithms using free fermions\", \"abstract\": \"We study variational quantum algorithms from the perspective of free fermions. By deriving the explicit structure of the associated Lie algebras, we show that the Quantum Approximate Optimization Algorithm (QAOA) on a one-dimensional lattice \\u2013 with and without decoupled angles \\u2013 is able to prepare all fermionic Gaussian states respecting the symmetries of the circuit. Leveraging these results, we numerically study the interplay between these symmetries and the locality of the target state, and find that an absence of symmetries makes nonlocal states easier to prepare. An efficient classical simulation of Gaussian states, with system sizes up to<mml:math xmlns:mml=\\\"http://www.w3.org/1998/Math/MathML\\\"><mml:mn>80</mml:mn></mml:math>and deep circuits, is employed to study the behavior of the circuit when it is overparameterized. In this regime of optimization, we find that the number of iterations to converge to the solution scales linearly with system size. Moreover, we observe that the number of iterations to converge to the solution decreases exponentially with the depth of the circuit, until it saturates at a depth which is quadratic in system size. Finally, we conclude that the improvement in the optimization can be explained in terms of better local linear approximations provided by the gradients.\", \"doi_y\": \"https://doi.org/10.22331/q-2023-03-30-966\", \"publication_date\": \"2023-03-30\"}, {\"work_id\": \"W4378528560\", \"doi_x\": \"10.3390/ijms24119359\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A Unique Mode of Coenzyme A Binding to the Nucleotide Binding Pocket of Human Metastasis Suppressor NME1\", \"abstract\": \"Coenzyme A (CoA) is a key cellular metabolite which participates in diverse metabolic pathways, regulation of gene expression and the antioxidant defense mechanism. Human NME1 (hNME1), which is a moonlighting protein, was identified as a major CoA-binding protein. Biochemical studies showed that hNME1 is regulated by CoA through both covalent and non-covalent binding, which leads to a decrease in the hNME1 nucleoside diphosphate kinase (NDPK) activity. In this study, we expanded the knowledge on previous findings by focusing on the non-covalent mode of CoA binding to the hNME1. With X-ray crystallography, we solved the CoA bound structure of hNME1 (hNME1-CoA) and determined the stabilization interactions CoA forms within the nucleotide-binding site of hNME1. A hydrophobic patch stabilizing the CoA adenine ring, while salt bridges and hydrogen bonds stabilizing the phosphate groups of CoA were observed. With molecular dynamics studies, we extended our structural analysis by characterizing the hNME1-CoA structure and elucidating possible orientations of the pantetheine tail, which is absent in the X-ray structure due to its flexibility. Crystallographic studies suggested the involvement of arginine 58 and threonine 94 in mediating specific interactions with CoA. Site-directed mutagenesis and CoA-based affinity purifications showed that arginine 58 mutation to glutamate (R58E) and threonine 94 mutation to aspartate (T94D) prevent hNME1 from binding to CoA. Overall, our results reveal a unique mode by which hNME1 binds CoA, which differs significantly from that of ADP binding: the \\u03b1- and \\u03b2-phosphates of CoA are oriented away from the nucleotide-binding site, while 3'-phosphate faces catalytic histidine 118 (H118). The interactions formed by the CoA adenine ring and phosphate groups contribute to the specific mode of CoA binding to hNME1.\", \"doi_y\": \"https://doi.org/10.3390/ijms24119359\", \"publication_date\": \"2023-05-27\"}, {\"work_id\": \"W4309858933\", \"doi_x\": \"10.1080/21505594.2022.2150452\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pathogenicity and virulence of <i>Clostridioides difficile</i>\", \"abstract\": \"Clostridioides difficile is the most common cause of nosocomial antibiotic-associated diarrhea, and is responsible for a spectrum of diseases characterized by high levels of recurrence, morbidity, and mortality. Treatment is complex, since antibiotics constitute both the main treatment and the major risk factor for infection. Worryingly, resistance to multiple antibiotics is becoming increasingly widespread, leading to the classification of this pathogen as an urgent threat to global health. As a consummate opportunist, C. difficile is well equipped for promoting disease, owing to its arsenal of virulence factors: transmission of this anaerobe is highly efficient due to the formation of robust endospores, and an array of adhesins promote gut colonization. C. difficile produces multiple toxins acting upon gut epithelia, resulting in manifestations typical of diarrheal disease, and severe inflammation in a subset of patients. This review focuses on such virulence factors, as well as the importance of antimicrobial resistance and genome plasticity in enabling pathogenesis and persistence of this important pathogen.\", \"doi_y\": \"https://doi.org/10.1080/21505594.2022.2150452\", \"publication_date\": \"2023-01-04\"}, {\"work_id\": \"W4386248168\", \"doi_x\": \"10.1098/rsob.230166\", \"af_direct_citation\": 0, \"af_indirect_citations\": 5, \"title\": \"A unifying mechanism for protein transport through the core bacterial Sec machinery\", \"abstract\": \"Encapsulation and compartmentalization are fundamental to the evolution of cellular life, but they also pose a challenge: how to partition the molecules that perform biological functions\\u2014the proteins\\u2014across impermeable barriers into sub-cellular organelles, and to the outside. The solution lies in the evolution of specialized machines, translocons, found in every biological membrane, which act both as gate and gatekeeper across and into membrane bilayers. Understanding how these translocons operate at the molecular level has been a long-standing ambition of cell biology, and one that is approaching its denouement; particularly in the case of the ubiquitous Sec system. In this review, we highlight the fruits of recent game-changing technical innovations in structural biology, biophysics and biochemistry to present a largely complete mechanism for the bacterial version of the core Sec machinery. We discuss the merits of our model over alternative proposals and identify the remaining open questions. The template laid out by the study of the Sec system will be of immense value for probing the many other translocons found in diverse biological membranes, towards the ultimate goal of altering or impeding their functions for pharmaceutical or biotechnological purposes.\", \"doi_y\": \"https://doi.org/10.1098/rsob.230166\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4386019098\", \"doi_x\": \"10.1002/bies.202300076\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The evolution of selective autophagy as a mechanism of oxidative stress response\", \"abstract\": \"Ageing is associated with a decline in autophagy and elevated reactive oxygen species (ROS), which can breach the capacity of antioxidant systems. Resulting oxidative stress can cause further cellular damage, including DNA breaks and protein misfolding. This poses a challenge for longevous organisms, including humans. In this review, we hypothesise that in the course of human evolution selective autophagy receptors (SARs) acquired the ability to sense and respond to localised oxidative stress. We posit that in the vicinity of protein aggregates and dysfunctional mitochondria oxidation of key cysteine residues in SARs induces their oligomerisation which initiates autophagy. The degradation of damaged cellular components thus could reduce ROS production and restore redox homeostasis. This evolutionarily acquired function of SARs may represent one of the biological adaptations that contributed to longer lifespan. Inversely, loss of this mechanism can lead to age-related diseases associated with impaired autophagy and oxidative stress.\", \"doi_y\": \"https://doi.org/10.1002/bies.202300076\", \"publication_date\": \"2023-08-21\"}, {\"work_id\": \"W4322764113\", \"doi_x\": \"10.1126/science.abn4116\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NLR immune receptor\\u2013nanobody fusions confer plant disease resistance\", \"abstract\": \"Plant pathogens cause recurrent epidemics, threatening crop yield and global food security. Efforts to retool the plant immune system have been limited to modifying natural components and can be nullified by the emergence of new pathogen strains. Made-to-order synthetic plant immune receptors provide an opportunity to tailor resistance to pathogen genotypes present in the field. In this work, we show that plant nucleotide-binding, leucine-rich repeat immune receptors (NLRs) can be used as scaffolds for nanobody (single-domain antibody fragment) fusions that bind fluorescent proteins (FPs). These fusions trigger immune responses in the presence of the corresponding FP and confer resistance against plant viruses expressing FPs. Because nanobodies can be raised against most molecules, immune receptor-nanobody fusions have the potential to generate resistance against plant pathogens and pests delivering effectors inside host cells.\", \"doi_y\": \"https://doi.org/10.1126/science.abn4116\", \"publication_date\": \"2023-03-03\"}, {\"work_id\": \"W4385693523\", \"doi_x\": \"10.1073/pnas.2305393120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"abstract\": \"Toxin-antitoxin (TA) systems are a large group of small genetic modules found in prokaryotes and their mobile genetic elements. Type II TAs are encoded as bicistronic (two-gene) operons that encode two proteins: a toxin and a neutralizing antitoxin. Using our tool NetFlax (standing for Network-FlaGs for toxins and antitoxins), we have performed a large-scale bioinformatic analysis of proteinaceous TAs, revealing interconnected clusters constituting a core network of TA-like gene pairs. To understand the structural basis of toxin neutralization by antitoxins, we have predicted the structures of 3,419 complexes with AlphaFold2. Together with mutagenesis and functional assays, our structural predictions provide insights into the neutralizing mechanism of the hyperpromiscuous Panacea antitoxin domain. In antitoxins composed of standalone Panacea, the domain mediates direct toxin neutralization, while in multidomain antitoxins the neutralization is mediated by other domains, such as PAD1, Phd-C, and ZFD. We hypothesize that Panacea acts as a sensor that regulates TA activation. We have experimentally validated 16 NetFlax TA systems and used domain annotations and metabolic labeling assays to predict their potential mechanisms of toxicity (such as membrane disruption, and inhibition of cell division or protein synthesis) as well as biological functions (such as antiphage defense). We have validated the antiphage activity of a RosmerTA system encoded by Gordonia phage Kita, and used fluorescence microscopy to confirm its predicted membrane-depolarizing activity. The interactive version of the NetFlax TA network that includes structural predictions can be accessed at http://netflax.webflags.se/ .\", \"doi_y\": \"https://doi.org/10.1073/pnas.2305393120\", \"publication_date\": \"2023-08-09\"}, {\"work_id\": \"W4285036285\", \"doi_x\": \"10.1039/d2cc02360a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pulse dipolar EPR for determining nanomolar binding affinities\", \"abstract\": \"Protein interaction studies often require very low concentrations and highly sensitive biophysical methods. Here, we demonstrate that pulse dipolar electron paramagnetic resonance spectroscopy allows measuring dissociation constants in the nanomolar range. This approach is appealing for concentration-limited biomolecular systems and medium-to-high-affinity binding studies, demonstrated here at 50 nanomolar protein concentration.\", \"doi_y\": \"https://doi.org/10.1039/d2cc02360a\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4386366735\", \"doi_x\": \"10.1016/j.jbc.2023.105218\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"abstract\": \"Human G protein-coupled receptor 35 is regulated by agonist-mediated phosphorylation of a set of five phospho-acceptor amino acids within its C-terminal tail. Alteration of both Ser300 and Ser303 to alanine in the GPR35a isoform greatly reduces the ability of receptor agonists to promote interactions with arrestin adapter proteins. Here, we have integrated the use of cell lines genome edited to lack expression of combinations of G protein receptor kinases (GRKs), selective small molecule inhibitors of subsets of these kinases, and antisera able to specifically identify either human GPR35a or mouse GPR35 only when Ser300 and Ser303 (orce; the equivalent residues in mouse GPR35) have become phosphorylated to demonstrate that GRK5 and GRK6 cause agonist-dependent phosphorylation of these residues. Extensions of these studies demonstrated the importance of the GRK5/6-mediated phosphorylation of these amino acids for agonist-induced internalization of the receptor. Homology and predictive modeling of the interaction of human GPR35 with GRKs showed that the N terminus of GRK5 is likely to dock in the same methionine pocket on the intracellular face of GPR35 as the C terminus of the \\u03b15 helix of G\\u03b113 and, that while this is also the case for GRK6, GRK2 and GRK3 are unable to do so effectively. These studies provide unique and wide-ranging insights into modes of regulation of GPR35, a receptor that is currently attracting considerable interest as a novel therapeutic target in diseases including ulcerative colitis.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.105218\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4309506674\", \"doi_x\": \"10.1093/nar/gkac1052\", \"af_direct_citation\": 0, \"af_indirect_citations\": 5, \"title\": \"UniProt: the Universal Protein Knowledgebase in 2023\", \"abstract\": \"The aim of the UniProt Knowledgebase is to provide users with a comprehensive, high-quality and freely accessible set of protein sequences annotated with functional information. In this publication we describe enhancements made to our data processing pipeline and to our website to adapt to an ever-increasing information content. The number of sequences in UniProtKB has risen to over 227 million and we are working towards including a reference proteome for each taxonomic group. We continue to extract detailed annotations from the literature to update or create reviewed entries, while unreviewed entries are supplemented with annotations provided by automated systems using a variety of machine-learning techniques. In addition, the scientific community continues their contributions of publications and annotations to UniProt entries of their interest. Finally, we describe our new website (https://www.uniprot.org/), designed to enhance our users' experience and make our data easily accessible to the research community. This interface includes access to AlphaFold structures for more than 85% of all entries as well as improved visualisations for subcellular localisation of proteins.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1052\", \"publication_date\": \"2022-11-21\"}, {\"work_id\": \"W4386138222\", \"doi_x\": \"10.1038/s41467-023-40483-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Structural basis of peptidoglycan synthesis by E. coli RodA-PBP2 complex\", \"abstract\": \"Peptidoglycan (PG) is an essential structural component of the bacterial cell wall that is synthetized during cell division and elongation. PG forms an extracellular polymer crucial for cellular viability, the synthesis of which is the target of many antibiotics. PG assembly requires a glycosyltransferase (GT) to generate a glycan polymer using a Lipid II substrate, which is then crosslinked to the existing PG via a transpeptidase (TP) reaction. A Shape, Elongation, Division and Sporulation (SEDS) GT enzyme and a Class B Penicillin Binding Protein (PBP) form the core of the multi-protein complex required for PG assembly. Here we used single particle cryo-electron microscopy to determine the structure of a cell elongation-specific E. coli RodA-PBP2 complex. We combine this information with biochemical, genetic, spectroscopic, and computational analyses to identify the Lipid II binding sites and propose a mechanism for Lipid II polymerization. Our data suggest a hypothesis for the movement of the glycan strand from the Lipid II polymerization site of RodA towards the TP site of PBP2, functionally linking these two central enzymatic activities required for cell wall peptidoglycan biosynthesis.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40483-8\", \"publication_date\": \"2023-08-24\"}, {\"work_id\": \"W4316469669\", \"doi_x\": \"10.1093/hmg/ddad007\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Single-cell RNA sequencing reveals transcriptional changes of human choroidal and retinal pigment epithelium cells during fetal development, in healthy adult and intermediate age-related macular degeneration\", \"abstract\": \"Abstract Age-related macular degeneration (AMD) is the most prevalent cause of blindness in the developed world. Vision loss in the advanced stages of the disease is caused by atrophy of retinal photoreceptors, overlying retinal pigment epithelium (RPE) and choroidal endothelial cells. The molecular events that underline the development of these cell types from in utero to adult as well as the progression to intermediate and advanced stages AMD are not yet fully understood. We performed single-cell RNA-sequencing (RNA-Seq) of human fetal and adult RPE\\u2013choroidal tissues, profiling in detail all the cell types and elucidating cell type-specific proliferation, differentiation and immunomodulation events that occur up to midgestation. Our data demonstrate that progression from the fetal to adult state is characterized by an increase in expression of genes involved in the oxidative stress response and detoxification from heavy metals, suggesting a better defence against oxidative stress in the adult RPE\\u2013choroid tissue. Single-cell comparative transcriptional analysis between a patient with intermediate AMD and an unaffected subject revealed a reduction in the number of RPE cells and melanocytes in the macular region of the AMD patient. Together these findings may suggest a macular loss of RPE cells and melanocytes in the AMD patients, but given the complex processing of tissues required for single-cell RNA-Seq that is prone to technical artefacts, these findings need to be validated by additional techniques in a larger number of AMD patients and controls.\", \"doi_y\": \"https://doi.org/10.1093/hmg/ddad007\", \"publication_date\": \"2023-01-16\"}, {\"work_id\": \"W4296323389\", \"doi_x\": \"10.15252/embj.2022111015\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"abstract\": \"Abstract Protein UFMylation, i.e., post\\u2010translational modification with ubiquitin\\u2010fold modifier 1 (UFM1), is essential for cellular and endoplasmic reticulum homeostasis. Despite its biological importance, we have a poor understanding of how UFM1 is conjugated onto substrates. Here, we use a rebuilding approach to define the minimal requirements of protein UFMylation. We find that the reported cognate E3 ligase UFL1 is inactive on its own and instead requires the adaptor protein UFBP1 to form an active E3 ligase complex. Structure predictions suggest the UFL1/UFBP1 complex to be made up of winged helix (WH) domain repeats. We show that UFL1/UFBP1 utilizes a scaffold\\u2010type E3 ligase mechanism that activates the UFM1\\u2010conjugating E2 enzyme, UFC1, for aminolysis. Further, we characterize a second adaptor protein CDK5RAP3 that binds to and forms an integral part of the ligase complex. Unexpectedly, we find that CDK5RAP3 inhibits UFL1/UFBP1 ligase activity in vitro . Results from reconstituting ribosome UFMylation suggest that CDK5RAP3 functions as a substrate adaptor that directs UFMylation to the ribosomal protein RPL26. In summary, our reconstitution approach reveals the biochemical basis of UFMylation and regulatory principles of this atypical E3 ligase complex.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111015\", \"publication_date\": \"2022-09-19\"}, {\"work_id\": \"W4307357511\", \"doi_x\": \"10.1101/2022.10.21.513200\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A systems approach evaluating the impact of SARS-CoV-2 variant of concern mutations on CD8+ T cell responses\", \"abstract\": \"Abstract T cell recognition of SARS-CoV-2 antigens after vaccination and/or natural infection has played a central role in resolving SARS-CoV-2 infections and generating adaptive immune memory. However, the clinical impact of SARS-CoV-2-specific T cell responses is variable and the mechanisms underlying T cell interaction with target antigens are not fully understood. This is especially true given the virus\\u2019 rapid evolution, which leads to new variants with immune escape capacity. In this study, we used the Omicron variant as a model organism and took a systems approach to evaluate the impact of mutations on CD8+ T cell immunogenicity. We computed an \\u2018immunogenicity potential\\u2019 score for each SARS-CoV-2 peptide antigen from the ancestral strain and Omicron, capturing both antigen presentation and T cell recognition probabilities. By comparing ancestral vs. Omicron immunogenicity scores, we reveal a divergent and heterogeneous landscape of impact for CD8+ T cell recognition of mutated targets in Omicron variants. While T cell recognition of Omicron peptides is broadly preserved, we observed mutated peptides with deteriorated immunogenicity that may assist breakthrough infection in some individuals. We then combined our scoring scheme with an in-silico mutagenesis, to characterise the position- and residue-specific theoretical mutational impact on immunogenicity. While we predict many escape trajectories from the theoretical landscape of substitutions, our study suggests that Omicron mutations in T cell epitopes did not develop under cell-mediated pressure. Our study provides a generalisable platform for fostering a deeper understanding of existing and novel variant impact on antigen-specific vaccine- and/or infection-induced T cell immunity.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.21.513200\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4290695246\", \"doi_x\": \"10.1021/acsinfecdis.2c00204\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Two Years into the COVID-19 Pandemic: Lessons Learned\", \"abstract\": \"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a highly transmissible and virulent human-infecting coronavirus that emerged in late December 2019 in Wuhan, China, causing a respiratory disease called coronavirus disease 2019 (COVID-19), which has massively impacted global public health and caused widespread disruption to daily life. The crisis caused by COVID-19 has mobilized scientists and public health authorities across the world to rapidly improve our knowledge about this devastating disease, shedding light on its management and control, and spawned the development of new countermeasures. Here we provide an overview of the state of the art of knowledge gained in the last 2 years about the virus and COVID-19, including its origin and natural reservoir hosts, viral etiology, epidemiology, modes of transmission, clinical manifestations, pathophysiology, diagnosis, treatment, prevention, emerging variants, and vaccines, highlighting important differences from previously known highly pathogenic coronaviruses. We also discuss selected key discoveries from each topic and underline the gaps of knowledge for future investigations.\", \"doi_y\": \"https://doi.org/10.1021/acsinfecdis.2c00204\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4380870595\", \"doi_x\": \"10.1038/s41564-023-01413-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mechanistic basis for potent neutralization of Sin Nombre hantavirus by a human monoclonal antibody\", \"abstract\": \"Rodent-borne hantaviruses are prevalent worldwide and upon spillover to human populations, cause severe disease for which no specific treatment is available. A potent antibody response is key for recovery from hantavirus infection. Here we study a highly neutralizing human monoclonal antibody, termed SNV-42, which was derived from a memory B cell isolated from an individual with previous Sin Nombre virus (SNV) infection. Crystallographic analysis demonstrates that SNV-42 targets the Gn subcomponent of the tetrameric (Gn-Gc)4 glycoprotein assembly that is relevant for viral entry. Integration of our 1.8 \\u00c5 structure with the (Gn-Gc)4 ultrastructure arrangement indicates that SNV-42 targets the membrane-distal region of the virus envelope. Comparison of the SNV-42 paratope encoding variable genes with inferred germline gene segments reveals high sequence conservation, suggesting that germline-encoded antibodies inhibit SNV. Furthermore, mechanistic assays reveal that SNV-42 interferes with both receptor recognition and fusion during host-cell entry. This work provides a molecular-level blueprint for understanding the human neutralizing antibody response to hantavirus infection.\", \"doi_y\": \"https://doi.org/10.1038/s41564-023-01413-y\", \"publication_date\": \"2023-06-15\"}, {\"work_id\": \"W4311547358\", \"doi_x\": \"10.1042/bcj20220474\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"abstract\": \"Pseudokinases, so named because they lack one or more conserved canonical amino acids that define their catalytically active relatives, have evolved a variety of biological functions in both prokaryotic and eukaryotic organisms. Human PSKH2 is closely related to the canonical kinase PSKH1, which maps to the CAMK family of protein kinases. Primates encode PSKH2 in the form of a pseudokinase, which is predicted to be catalytically inactive due to loss of the invariant catalytic Asp residue. Although the biological role(s) of vertebrate PSKH2 proteins remains unclear, we previously identified species-level adaptions in PSKH2 that have led to the appearance of kinase or pseudokinase variants in vertebrate genomes alongside a canonical PSKH1 paralog. In this paper we confirm that, as predicted, PSKH2 lacks detectable protein phosphotransferase activity, and exploit structural informatics, biochemistry and cellular proteomics to begin to characterise vertebrate PSKH2 orthologues. AlphaFold 2-based structural analysis predicts functional roles for both the PSKH2 N- and C-regions that flank the pseudokinase domain core, and cellular truncation analysis confirms that the N-terminal domain, which contains a conserved myristoylation site, is required for both stable human PSKH2 expression and localisation to a membrane-rich subcellular fraction containing mitochondrial proteins. Using mass spectrometry-based proteomics, we confirm that human PSKH2 is part of a cellular mitochondrial protein network, and that its expression is regulated through client-status within the HSP90/Cdc37 molecular chaperone system. HSP90 interactions are mediated through binding to the PSKH2 C-terminal tail, leading us to predict that this region might act as both a cis and trans regulatory element, driving outputs linked to the PSKH2 pseudokinase domain that are important for functional signalling.\", \"doi_y\": \"https://doi.org/10.1042/bcj20220474\", \"publication_date\": \"2023-01-23\"}, {\"work_id\": \"W4319662875\", \"doi_x\": \"10.1126/science.ade8840\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"abstract\": \"Certain inhibitor of apoptosis (IAP) family members are sentinel proteins that prevent untimely cell death by inhibiting caspases. Antagonists, including second mitochondria-derived activator of caspases (SMAC), regulate IAPs and drive cell death. Baculoviral IAP repeat-containing protein 6 (BIRC6), a giant IAP with dual E2 and E3 ubiquitin ligase activity, regulates programmed cell death through unknown mechanisms. We show that BIRC6 directly restricts executioner caspase-3 and -7 and ubiquitinates caspase-3, -7, and -9, working exclusively with noncanonical E1, UBA6. Notably, we show that SMAC suppresses both mechanisms. Cryo-electron microscopy structures of BIRC6 alone and in complex with SMAC reveal that BIRC6 is an antiparallel dimer juxtaposing the substrate-binding module against the catalytic domain. Furthermore, we discover that SMAC multisite binding to BIRC6 results in a subnanomolar affinity interaction, enabling SMAC to competitively displace caspases, thus antagonizing BIRC6 anticaspase function.\", \"doi_y\": \"https://doi.org/10.1126/science.ade8840\", \"publication_date\": \"2023-03-17\"}, {\"work_id\": \"W4313527937\", \"doi_x\": \"10.1088/2633-1357/acadb9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Morphology exploration of pollen using deep learning latent space\", \"abstract\": \"Abstract The structure of pollen has evolved depending on its local environment, competition, and ecology. As pollen grains are generally of size 10\\u2013100 microns with nanometre-scale substructure, scanning electron microscopy is an important microscopy technique for imaging and analysis. Here, we use style transfer deep learning to allow exploration of latent w-space of scanning electron microscope images of pollen grains and show the potential for using this technique to understand evolutionary pathways and characteristic structural traits of pollen grains.\", \"doi_y\": \"https://doi.org/10.1088/2633-1357/acadb9\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4310954939\", \"doi_x\": \"10.1111/jipb.13422\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA methylation dynamics during germline development\", \"abstract\": \"DNA methylation plays essential homeostatic functions in eukaryotic genomes. In animals, DNA methylation is also developmentally regulated and, in turn, regulates development. In the past two decades, huge research effort has endorsed the understanding that DNA methylation plays a similar role in plant development, especially during sexual reproduction. The power of whole-genome sequencing and cell isolation techniques, as well as bioinformatics tools, have enabled recent studies to reveal dynamic changes in DNA methylation during germline development. Furthermore, the combination of these technological advances with genetics, developmental biology and cell biology tools has revealed functional methylation reprogramming events that control gene and transposon activities in flowering plant germlines. In this review, we discuss the major advances in our knowledge of DNA methylation dynamics during male and female germline development in flowering plants.\", \"doi_y\": \"https://doi.org/10.1111/jipb.13422\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4289667563\", \"doi_x\": \"10.1016/j.celrep.2022.111168\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"abstract\": \"An essential first step in the post-translational modification of proteins with UFM1, UFMylation, is the proteolytic cleavage of pro-UFM1 to expose a C-terminal glycine. Of the two UFM1-specific proteases (UFSPs) identified in humans, only UFSP2 is reported to be active, since the annotated sequence of UFSP1 lacks critical catalytic residues. Nonetheless, efficient UFM1 maturation occurs in cells lacking UFSP2, suggesting the presence of another active protease. We herein identify UFSP1 translated from a non-canonical start site to be this protease. Cells lacking both UFSPs show complete loss of UFMylation resulting from an absence of mature UFM1. While UFSP2, but not UFSP1, removes UFM1 from the ribosomal subunit RPL26, UFSP1 acts earlier in the pathway to mature UFM1 and cleave a potential autoinhibitory modification on UFC1, thereby controlling activation of UFMylation. In summary, our studies reveal important distinctions in substrate specificity and localization-dependent functions for the two proteases in regulating UFMylation.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111168\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4385684007\", \"doi_x\": \"10.1098/rsif.2023.0187\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Modelling the viral dynamics of the SARS-CoV-2 Delta and Omicron variants in different cell types\", \"abstract\": \"We use viral kinetic models fitted to viral load data from in vitro studies to explain why the SARS-CoV-2 Omicron variant replicates faster than the Delta variant in nasal cells, but slower than Delta in lung cells, which could explain Omicron's higher transmission potential and lower severity. We find that in both nasal and lung cells, viral infectivity is higher for Omicron but the virus production rate is higher for Delta, with an estimated approximately 200-fold increase in infectivity and 100-fold decrease in virus production when comparing Omicron with Delta in nasal cells. However, the differences are unequal between cell types, and ultimately lead to the basic reproduction number and growth rate being higher for Omicron in nasal cells, and higher for Delta in lung cells. In nasal cells, Omicron alone can enter via a TMPRSS2-independent pathway, but it is primarily increased efficiency of TMPRSS2-dependent entry which accounts for Omicron's increased activity. This work paves the way for using within-host mathematical models to understand the transmission potential and severity of future variants.\", \"doi_y\": \"https://doi.org/10.1098/rsif.2023.0187\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4307407432\", \"doi_x\": \"10.1107/s2059798322009986\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"abstract\": \"Crystallographic fragment screens provide an efficient and effective way to identify small-molecule ligands of a crystallized protein. Due to their low molecular weight, such hits tend to have low, often unquantifiable, affinity for their target, complicating the twin challenges of validating the hits as authentic solution-phase ligands of the target and identifying the `best' hit(s) for further elaboration. In this article, approaches that address these challenges are assessed. Using retrospective analysis of a recent ATAD2 hit-identification campaign, alongside other examples of successful fragment-screening campaigns, it is suggested that hit validation and prioritization are best achieved by a `triangulation' approach in which the results of multiple available biochemical and biophysical techniques are correlated to develop qualitative structure-activity relationships (SARs). Such qualitative SARs may indeed be the only means by which to navigate a project through the tunnel of uncertainty that prevails before on-scale biophysical, biochemical and/or biological measurements become possible.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322009986\", \"publication_date\": \"2022-10-27\"}, {\"work_id\": \"W4210284779\", \"doi_x\": \"10.3389/fmed.2021.836826\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Viral Load in COVID-19 Patients: Implications for Prognosis and Vaccine Efficacy in the Context of Emerging SARS-CoV-2 Variants\", \"abstract\": \"The worldwide spread of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has caused an unprecedented public health crisis in the 21st century. As the pandemic evolves, the emergence of SARS-CoV-2 has been characterized by the emergence of new variants of concern (VOCs), which resulted in a catastrophic impact on SARS-CoV-2 infection. In light of this, research groups around the world are unraveling key aspects of the associated illness, coronavirus disease 2019 (COVID-19). A cumulative body of data has indicated that the SARS-CoV-2 viral load may be a determinant of the COVID-19 severity. Here we summarize the main characteristics of the emerging variants of SARS-CoV-2, discussing their impact on viral transmissibility, viral load, disease severity, vaccine breakthrough, and lethality among COVID-19 patients. We also provide a rundown of the rapidly expanding scientific evidence from clinical studies and animal models that indicate how viral load could be linked to COVID-19 prognosis and vaccine efficacy among vaccinated individuals, highlighting the differences compared to unvaccinated individuals.\", \"doi_y\": \"https://doi.org/10.3389/fmed.2021.836826\", \"publication_date\": \"2022-01-31\"}, {\"work_id\": \"W4361766435\", \"doi_x\": \"10.1039/d2sm01225a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Diversity of viscoelastic properties of an engineered muscle-inspired protein hydrogel\", \"abstract\": \"Through a multi-modal rheology approach, we show that muscle-inspired hydrogels exhibit a surprising diversity of viscoelastic response from power-law rheology at low strains to stiffening and energy dissipation at high strains.\", \"doi_y\": \"https://doi.org/10.1039/d2sm01225a\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4382133299\", \"doi_x\": \"10.1016/j.crmeth.2023.100508\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Affinity-enhanced RNA-binding domains as tools to understand RNA recognition\", \"abstract\": \"Understanding how the RNA-binding domains of a protein regulator are used to recognize its RNA targets is a key problem in RNA biology, but RNA-binding domains with very low affinity do not perform well in the methods currently available to characterize protein-RNA interactions. Here, we propose to use conservative mutations that enhance the affinity of RNA-binding domains to overcome this limitation. As a proof of principle, we have designed and validated an affinity-enhanced K-homology (KH) domain mutant of the fragile X syndrome protein FMRP, a key regulator of neuronal development, and used this mutant to determine the domain\\u2019s sequence preference and to explain FMRP recognition of specific RNA motifs in the cell. Our results validate our concept and our nuclear magnetic resonance (NMR)-based workflow. While effective mutant design requires an understanding of the underlying principles of RNA recognition by the relevant domain type, we expect the method will be used effectively in many RNA-binding domains.\", \"doi_y\": \"https://doi.org/10.1016/j.crmeth.2023.100508\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4317434079\", \"doi_x\": \"10.1083/jcb.202301035\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mammalian fertilization: Does sperm IZUMO1 mediate fusion as well as adhesion?\", \"abstract\": \"The molecular mechanism of sperm-egg fusion is a long-standing mystery in reproduction. Brukman and colleagues (2022. J. Cell Biol.https://doi.org/10.1083/jcb.202207147) now provide evidence that the sperm surface protein IZUMO1, which is essential for mammalian fertilization, can induce membrane fusion in cultured cells.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202301035\", \"publication_date\": \"2023-01-19\"}, {\"work_id\": \"W4310112261\", \"doi_x\": \"10.1021/acs.biomac.2c01097\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Poly(vinyl alcohol) Molecular Bottlebrushes Nucleate Ice\", \"abstract\": \"Ice binding proteins (IBP) have evolved to limit the growth of ice but also to promote ice formation by ice-nucleating proteins (INPs). IBPs, which modulate these seemingly distinct processes, often have high sequence similarities, and molecular size/assembly is hypothesized to be a crucial determinant. There are only a few synthetic materials that reproduce INP function, and rational design of ice nucleators has not been achieved due to outstanding questions about the mechanisms of ice binding. Poly(vinyl alcohol) (PVA) is a water-soluble synthetic polymer well known to effectively block ice recrystallization, by binding to ice. Here, we report the synthesis of a polymeric ice nucleator, which mimics the dense assembly of IBPs, using confined ice-binding polymers in a high-molar-mass molecular bottlebrush. Poly(vinyl alcohol)-based molecular bottlebrushes with different side-chain densities were synthesized via a combination of ring-opening metathesis polymerization (ROMP) and reversible addition-fragmentation chain-transfer (RAFT) polymerization, using \\\"grafting-to\\\" and \\\"grafting-through\\\" approaches. The facile preparation of the PVA bottlebrushes was performed via selective hydrolysis of the acetate of the poly(vinyl acetate) (PVAc) side chains of the PVAc bottlebrush precursors. Ice-binding polymer side-chain density was shown to be crucial for nucleation activity, with less dense brushes resulting in colder nucleation than denser brushes. This bio-inspired approach provides a synthetic framework for probing heterogeneous ice nucleation and a route toward defined synthetic nucleators for biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1021/acs.biomac.2c01097\", \"publication_date\": \"2022-11-28\"}, {\"work_id\": \"W4284992638\", \"doi_x\": \"10.1016/j.jbc.2022.102247\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural features of the protein kinase domain and targeted binding by small-molecule inhibitors\", \"abstract\": \"Protein kinases are key components in cellular signaling pathways as they carry out the phosphorylation of proteins, primarily on Ser, Thr, and Tyr residues. The catalytic activity of protein kinases is regulated, and they can be thought of as molecular switches that are controlled through protein-protein interactions and post-translational modifications. Protein kinases exhibit diverse structural mechanisms of regulation and have been fascinating subjects for structural biologists from the first crystal structure of a protein kinase over 30 years ago, to recent insights into kinase assemblies enabled by the breakthroughs in cryo-EM. Protein kinases are high-priority targets for drug discovery in oncology and other disease settings, and kinase inhibitors have transformed the outcomes of specific groups of patients. Most kinase inhibitors are ATP competitive, deriving potency by occupying the deep hydrophobic pocket at the heart of the kinase domain. Selectivity of inhibitors depends on exploiting differences between the amino acids that line the ATP site and exploring the surrounding pockets that are present in inactive states of the kinase. More recently, allosteric pockets outside the ATP site are being targeted to achieve high selectivity and to overcome resistance to current therapeutics. Here, we review the key regulatory features of the protein kinase family, describe the different types of kinase inhibitors, and highlight examples where the understanding of kinase regulatory mechanisms has gone hand in hand with the development of inhibitors.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102247\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4309862298\", \"doi_x\": \"10.7554/elife.76387\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"abstract\": \"EROS (essential for reactive oxygen species) protein is indispensable for expression of gp91phox, the catalytic core of the phagocyte NADPH oxidase. EROS deficiency in humans is a novel cause of the severe immunodeficiency, chronic granulomatous disease, but its mechanism of action was unknown until now. We elucidate the role of EROS, showing it acts at the earliest stages of gp91phox maturation. It binds the immature 58 kDa gp91phox directly, preventing gp91phox degradation and allowing glycosylation via the oligosaccharyltransferase machinery and the incorporation of the heme prosthetic groups essential for catalysis. EROS also regulates the purine receptors P2X7 and P2X1 through direct interactions, and P2X7 is almost absent in EROS-deficient mouse and human primary cells. Accordingly, lack of murine EROS results in markedly abnormal P2X7 signalling, inflammasome activation, and T cell responses. The loss of both ROS and P2X7 signalling leads to resistance to influenza infection in mice. Our work identifies EROS as a highly selective chaperone for key proteins in innate and adaptive immunity and a rheostat for immunity to infection. It has profound implications for our understanding of immune physiology, ROS dysregulation, and possibly gene therapy.\", \"doi_y\": \"https://doi.org/10.7554/elife.76387\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4221136828\", \"doi_x\": \"10.1038/s41576-022-00465-w\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA methylation-based predictors of health: applications and statistical considerations\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41576-022-00465-w\", \"publication_date\": \"2022-03-18\"}, {\"work_id\": \"W4378781380\", \"doi_x\": \"10.1073/pnas.2302858120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structure of the priming arabinosyltransferase AftA required for AG biosynthesis of <i>Mycobacterium tuberculosis</i>\", \"abstract\": \"Arabinogalactan (AG) is an essential cell wall component in mycobacterial species, including the deadly human pathogen Mycobacterium tuberculosis. It plays a pivotal role in forming the rigid mycolyl-AG-peptidoglycan core for in vitro growth. AftA is a membrane-bound arabinosyltransferase and a key enzyme involved in AG biosynthesis which bridges the assembly of the arabinan chain to the galactan chain. It is known that AftA catalyzes the transfer of the first arabinofuranosyl residue from the donor decaprenyl-monophosphoryl-arabinose to the mature galactan chain (i.e., priming); however, the priming mechanism remains elusive. Herein, we report the cryo-EM structure of Mtb AftA. The detergent-embedded AftA assembles as a dimer with an interface maintained by both the transmembrane domain (TMD) and the soluble C-terminal domain (CTD) in the periplasm. The structure shows a conserved glycosyltransferase-C fold and two cavities converging at the active site. A metal ion participates in the interaction of TMD and CTD of each AftA molecule. Structural analyses combined with functional mutagenesis suggests a priming mechanism catalyzed by AftA in Mtb AG biosynthesis. Our data further provide a unique perspective into anti-TB drug discovery.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2302858120\", \"publication_date\": \"2023-05-30\"}, {\"work_id\": \"W4223926836\", \"doi_x\": \"10.1371/journal.pbio.3001601\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"abstract\": \"Coat complexes coordinate cargo recognition through cargo adaptors with biogenesis of transport carriers during integral membrane protein trafficking. Here, we combine biochemical, structural, and cellular analyses to establish the mechanistic basis through which SNX27-Retromer, a major endosomal cargo adaptor, couples to the membrane remodeling endosomal SNX-BAR sorting complex for promoting exit 1 (ESCPE-1). In showing that the SNX27 FERM (4.1/ezrin/radixin/moesin) domain directly binds acidic-Asp-Leu-Phe (aDLF) motifs in the SNX1/SNX2 subunits of ESCPE-1, we propose a handover model where SNX27-Retromer captured cargo proteins are transferred into ESCPE-1 transport carriers to promote endosome-to-plasma membrane recycling. By revealing that assembly of the SNX27:Retromer:ESCPE-1 coat evolved in a stepwise manner during early metazoan evolution, likely reflecting the increasing complexity of endosome-to-plasma membrane recycling from the ancestral opisthokont to modern animals, we provide further evidence of the functional diversification of yeast pentameric Retromer in the recycling of hundreds of integral membrane proteins in metazoans.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001601\", \"publication_date\": \"2022-04-13\"}, {\"work_id\": \"W4377161612\", \"doi_x\": \"10.1021/acssuschemeng.3c00243\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Versatile Chemo-Biocatalytic Cascade Driven by a Thermophilic and Irreversible C\\u2013C Bond-Forming \\u03b1-Oxoamine Synthase\", \"abstract\": \"We report a chemo-biocatalytic cascade for the synthesis of substituted pyrroles, driven by the action of an irreversible, thermostable, pyridoxal 5\\u2032-phosphate (PLP)-dependent, C\\u2013C bond-forming biocatalyst (ThAOS). The ThAOS catalyzes the Claisen-like condensation between various amino acids and acyl-CoA substrates to generate a range of \\u03b1-aminoketones. These products are reacted with \\u03b2-keto esters in an irreversible Knorr pyrrole reaction. The determination of the 1.6 \\u00c5 resolution crystal structure of the PLP-bound form of ThAOS lays the foundation for future engineering and directed evolution. This report establishes the AOS family as useful and versatile C\\u2013C bond-forming biocatalysts.\", \"doi_y\": \"https://doi.org/10.1021/acssuschemeng.3c00243\", \"publication_date\": \"2023-05-16\"}, {\"work_id\": \"W4361801564\", \"doi_x\": \"10.1039/d3ra01708g\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mechanoenzymatic reactions for the hydrolysis of PET\", \"abstract\": \"Recent advances in the enzymatic degradation of poly(ethylene terphthalate) (PET) have led to a number of PET hydrolytic enzymes and mutants being developed. With the amount of PET building up in the natural world, there is a pressing need to develop scalable methods of breaking down the polymer into its monomers for recycling or other uses. Mechanoenzymatic reactions have gained traction recently as a green and efficient alternative to traditional biocatalytic reactions. For the first time we report increased yields of PET degradation by whole cell PETase enzymes by up to 27-fold by utilising ball milling cycles of reactive aging, when compared with typical solution-based reactions. This methodology leads to up to a 2600-fold decrease in the solvent required when compared with other leading degradation reactions in the field and a 30-fold decrease in comparison to reported industrial scale PET hydrolysis reactions.\", \"doi_y\": \"https://doi.org/10.1039/d3ra01708g\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4367399594\", \"doi_x\": \"10.1016/j.sbi.2023.102600\", \"af_direct_citation\": 1, \"af_indirect_citations\": 13, \"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"abstract\": \"We provide an overview of the methods that can be used for protein structure-based evaluation of missense variants. The algorithms can be broadly divided into those that calculate the difference in free energy (\\u0394\\u0394G) between the wild type and variant structures and those that use structural features to predict the damaging effect of a variant without providing a \\u0394\\u0394G. A wide range of machine learning approaches have been employed to develop those algorithms. We also discuss challenges and opportunities for variant interpretation in view of the recent breakthrough in three-dimensional structural modelling using deep learning.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102600\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4293824663\", \"doi_x\": \"10.1242/jcs.260456\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Meeting report \\u2013 Cell dynamics: host\\u2013pathogen interface\", \"abstract\": \"ABSTRACT Two years into the most significant infectious disease event of our generation, infections have populated every conversation and in-depth understanding of host\\u2013pathogen interactions has, perhaps, never been more important. In a successful return to in-person conferences, the host\\u2013pathogen interface was the focus of the third Cell Dynamics meeting, which took place at the glorious Wotton House in Surrey, UK. The meeting organised by Michaela Gack, Maximiliano Gutierrez, Dominique Soldati-Favre and Michael Way gathered an international group of scientists who shared their recent discoveries and views on numerous aspects, including cell-autonomous defence mechanisms, pathogen interactions with host cytoskeletal or membrane dynamics, and cellular immune regulation. More than 30 years into the beginning of cellular microbiology as a field, the meeting exhibited the unique aspect of the host\\u2013pathogen interface in uncovering the fundamentals of both pathogens and their hosts.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260456\", \"publication_date\": \"2022-08-15\"}, {\"work_id\": \"W4206635163\", \"doi_x\": \"10.3390/genes13010158\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Inorganic Nitrogen Transport and Assimilation in Pea (Pisum sativum)\", \"abstract\": \"The genome sequences of several legume species are now available allowing the comparison of the nitrogen (N) transporter inventories with non-legume species. A survey of the genes encoding inorganic N transporters and the sensing and assimilatory families in pea, revealed similar numbers of genes encoding the primary N assimilatory enzymes to those in other types of plants. Interestingly, we find that pea and Medicago truncatula have fewer members of the NRT2 nitrate transporter family. We suggest that this difference may result from a decreased dependency on soil nitrate acquisition, as legumes have the capacity to derive N from a symbiotic relationship with diazotrophs. Comparison with M. truncatula, indicates that only one of three NRT2s in pea is likely to be functional, possibly indicating less N uptake before nodule formation and N-fixation starts. Pea seeds are large, containing generous amounts of N-rich storage proteins providing a reserve that helps seedling establishment and this may also explain why fewer high affinity nitrate transporters are required. The capacity for nitrate accumulation in the vacuole is another component of assimilation, as it can provide a storage reservoir that supplies the plant when soil N is depleted. Comparing published pea tissue nitrate concentrations with other plants, we find that there is less accumulation of nitrate, even in non-nodulated plants, and that suggests a lower capacity for vacuolar storage. The long-distance transported form of organic N in the phloem is known to be specialized in legumes, with increased amounts of organic N molecules transported, like ureides, allantoin, asparagine and amides in pea. We suggest that, in general, the lower tissue and phloem nitrate levels compared with non-legumes may also result in less requirement for high affinity nitrate transporters. The pattern of N transporter and assimilatory enzyme distribution in pea is discussed and compared with non-legumes with the aim of identifying future breeding targets.\", \"doi_y\": \"https://doi.org/10.3390/genes13010158\", \"publication_date\": \"2022-01-17\"}, {\"work_id\": \"W4306641762\", \"doi_x\": \"10.1073/pnas.2210109119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"abstract\": \"The genomes of some purple photosynthetic bacteria contain a multigene puc family encoding a series of \\u03b1- and \\u03b2-polypeptides that together form a heterogeneous antenna of light-harvesting 2 (LH2) complexes. To unravel this complexity, we generated four sets of puc deletion mutants in Rhodopseudomonas palustris , each encoding a single type of pucBA gene pair and enabling the purification of complexes designated as PucA-LH2, PucB-LH2, PucD-LH2, and PucE-LH2. The structures of all four purified LH2 complexes were determined by cryogenic electron microscopy (cryo-EM) at resolutions ranging from 2.7 to 3.6 \\u00c5. Uniquely, each of these complexes contains a hitherto unknown polypeptide, \\u03b3, that forms an extended undulating ribbon that lies in the plane of the membrane and that encloses six of the nine LH2 \\u03b1\\u03b2-subunits. The \\u03b3-subunit, which is located near to the cytoplasmic side of the complex, breaks the C9 symmetry of the LH2 complex and binds six extra bacteriochlorophylls (BChls) that enhance the 800-nm absorption of each complex. The structures show that all four complexes have two complete rings of BChls, conferring absorption bands centered at 800 and 850 nm on the PucA-LH2, PucB-LH2, and PucE-LH2 complexes, but, unusually, the PucD-LH2 antenna has only a single strong near-infared (NIR) absorption peak at 803 nm. Comparison of the cryo-EM structures of these LH2 complexes reveals altered patterns of hydrogen bonds between LH2 \\u03b1\\u03b2-side chains and the bacteriochlorin rings, further emphasizing the major role that H bonds play in spectral tuning of bacterial antenna complexes.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2210109119\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4226515274\", \"doi_x\": \"10.7554/elife.67667\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Unifying the known and unknown microbial coding sequence space\", \"abstract\": \"Genes of unknown function are among the biggest challenges in molecular biology, especially in microbial systems, where 40-60% of the predicted genes are unknown. Despite previous attempts, systematic approaches to include the unknown fraction into analytical workflows are still lacking. Here, we present a conceptual framework, its translation into the computational workflow AGNOSTOS and a demonstration on how we can bridge the known-unknown gap in genomes and metagenomes. By analyzing 415,971,742 genes predicted from 1749 metagenomes and 28,941 bacterial and archaeal genomes, we quantify the extent of the unknown fraction, its diversity, and its relevance across multiple organisms and environments. The unknown sequence space is exceptionally diverse, phylogenetically more conserved than the known fraction and predominantly taxonomically restricted at the species level. From the 71 M genes identified to be of unknown function, we compiled a collection of 283,874 lineage-specific genes of unknown function for Cand. Patescibacteria (also known as Candidate Phyla Radiation, CPR), which provides a significant resource to expand our understanding of their unusual biology. Finally, by identifying a target gene of unknown function for antibiotic resistance, we demonstrate how we can enable the generation of hypotheses that can be used to augment experimental data.It is estimated that scientists do not know what half of microbial genes actually do. When these genes are discovered in microorganisms grown in the lab or found in environmental samples, it is not possible to identify what their roles are. Many of these genes are excluded from further analyses for these reasons, meaning that the study of microbial genes tends to be limited to genes that have already been described. These limitations hinder research into microbiology, because information from newly discovered genes cannot be integrated to better understand how these organisms work. Experiments to understand what role these genes have in the microorganisms are labor-intensive, so new analytical strategies are needed. To do this, Vanni et al. developed a new framework to categorize genes with unknown roles, and a computational workflow to integrate them into traditional analyses. When this approach was applied to over 400 million microbial genes (both with known and unknown roles), it showed that the share of genes with unknown functions is only about 30 per cent, smaller than previously thought. The analysis also showed that these genes are very diverse, revealing a huge space for future research and potential applications. Combining their approach with experimental data, Vanni et al. were able to identify a gene with a previously unknown purpose that could be involved in antibiotic resistance. This system could be useful for other scientists studying microorganisms to get a more complete view of microbial systems. In future, it may also be used to analyze the genetics of other organisms, such as plants and animals.\", \"doi_y\": \"https://doi.org/10.7554/elife.67667\", \"publication_date\": \"2022-03-31\"}, {\"work_id\": \"W4292241897\", \"doi_x\": \"10.1016/j.chom.2022.07.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structure of trimeric pre-fusion rabies virus glycoprotein in complex with two protective antibodies\", \"abstract\": \"Rabies virus (RABV) causes lethal encephalitis and is responsible for approximately 60,000 deaths per year. As the sole virion-surface protein, the rabies virus glycoprotein (RABV-G) mediates host-cell entry. RABV-G's pre-fusion trimeric conformation displays epitopes bound by protective neutralizing antibodies that can be induced by vaccination or passively administered for post-exposure prophylaxis. We report a 2.8-\\u00c5 structure of a RABV-G trimer in the pre-fusion conformation, in complex with two neutralizing and protective monoclonal antibodies, 17C7 and 1112-1, that recognize distinct epitopes. One of these antibodies is a licensed prophylactic (17C7, Rabishield), which we show locks the protein in pre-fusion conformation. Targeted mutations can similarly stabilize RABV-G in the pre-fusion conformation, a key step toward structure-guided vaccine design. These data reveal the higher-order architecture of a key therapeutic target and the structural basis of neutralization by antibodies binding two key antigenic sites, and this will facilitate the development of improved vaccines and prophylactic antibodies.\", \"doi_y\": \"https://doi.org/10.1016/j.chom.2022.07.014\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4225390182\", \"doi_x\": \"10.1107/s2059798322003576\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"abstract\": \"Crystallographers have an array of search-model options for structure solution by molecular replacement (MR). The well established options of homologous experimental structures and regular secondary-structure elements or motifs are increasingly supplemented by computational modelling. Such modelling may be carried out locally or may use pre-calculated predictions retrieved from databases such as the EBI AlphaFold database. MrParse is a new pipeline to help to streamline the decision process in MR by consolidating bioinformatic predictions in one place. When reflection data are provided, MrParse can rank any experimental homologues found using eLLG, which indicates the likelihood that a given search model will work in MR. Inbuilt displays of predicted secondary structure, coiled-coil and transmembrane regions further inform the choice of MR protocol. MrParse can also identify and rank homologues in the EBI AlphaFold database, a function that will also interest other structural biologists and bioinformaticians.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322003576\", \"publication_date\": \"2022-04-26\"}, {\"work_id\": \"W4280558961\", \"doi_x\": \"10.1016/j.jmb.2022.167625\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Computational Resources for Molecular Biology 2022\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2022.167625\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4386470019\", \"doi_x\": \"10.1371/journal.pntd.0011618\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Exploring the antimicrobial peptidome of nematodes through phylum-spanning in silico analyses highlights novel opportunities for pathogen control\", \"abstract\": \"Antimicrobial Peptides (AMPs) are key constituents of the invertebrate innate immune system and provide critical protection against microbial threat. Nematodes display diverse life strategies where they are exposed to heterogenous, microbe rich, environments highlighting their need for an innate immune system. Within the Ecdysozoa, arthropod AMPs have been well characterised, however nematode-derived AMP knowledge is limited. In this study the distribution and abundance of putative AMP-encoding genes was examined in 134 nematode genomes providing the most comprehensive profile of AMP candidates within phylum Nematoda. Through genome and transcriptome analyses we reveal that phylum Nematoda is a rich source of putative AMP diversity and demonstrate (i) putative AMP group profiles that are influenced by nematode lifestyle where free-living nematodes appear to display enriched putative AMP profiles relative to parasitic species; (ii) major differences in the putative AMP profiles between nematode clades where Clade 9/V and 10/IV species possess expanded putative AMP repertoires; (iii) AMP groups with highly restricted profiles (e.g. Cecropins and Diapausins) and others [e.g. Nemapores and Glycine Rich Secreted Peptides (GRSPs)] which are more widely distributed; (iv) complexity in the distribution and abundance of CS\\u03b1\\u03b2 subgroup members; and (v) that putative AMPs are expressed in host-facing life stages and biofluids of key nematode parasites. These data indicate that phylum Nematoda displays diversity in putative AMPs and underscores the need for functional characterisation to reveal their role and importance to nematode biology and host-nematode-microbiome interactions.\", \"doi_y\": \"https://doi.org/10.1371/journal.pntd.0011618\", \"publication_date\": \"2023-09-06\"}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "indirect_citations = citations[citations.af_indirect_citations > 0]\n",
    "\n",
    "indirect_citations_chart = alt.Chart(indirect_citations).mark_bar().encode(\n",
    "    x=alt.X('af_indirect_citations:Q', title='Number of Papers'),\n",
    "    y=alt.Y('title:N', title='Paper Title', sort='-x'),\n",
    "    tooltip=['title', 'af_indirect_citations']\n",
    ").properties(\n",
    "    title='Count of Papers that Indirectly Cite Alphafold',\n",
    "    height=2400\n",
    ")\n",
    "\n",
    "indirect_citations_chart"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-929efe7d55314efc95a862810413d027.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-929efe7d55314efc95a862810413d027.vega-embed details,\n",
       "  #altair-viz-929efe7d55314efc95a862810413d027.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-929efe7d55314efc95a862810413d027\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-929efe7d55314efc95a862810413d027\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-929efe7d55314efc95a862810413d027\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-c391c6972bbede857abeb33059a61785\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"color\": {\"field\": \"type\", \"scale\": {\"domain\": [\"Indirect\", \"Direct\"], \"range\": [\"blue\", \"red\"]}, \"title\": \"Citation Type\", \"type\": \"nominal\"}, \"tooltip\": [{\"field\": \"title\", \"type\": \"nominal\"}, {\"aggregate\": \"sum\", \"field\": \"count\", \"type\": \"quantitative\"}, {\"field\": \"type\", \"type\": \"nominal\"}], \"x\": {\"field\": \"count\", \"title\": \"Number of Citations\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"title\", \"sort\": \"-x\", \"title\": \"Paper Title\", \"type\": \"nominal\"}}, \"height\": 3200, \"title\": \"Count of Papers Citing Alphafold (Direct and Indirect)\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-c391c6972bbede857abeb33059a61785\": [{\"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 11}, {\"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 9}, {\"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Computational Resources for Molecular Biology 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Computational Resources for Molecular Biology 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 6}, {\"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"A unifying mechanism for protein transport through the core bacterial Sec machinery\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"UniProt: the Universal Protein Knowledgebase in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Structural Analysis of Bacillus subtilis Sigma Factors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structural basis of peptidoglycan synthesis by E. coli RodA-PBP2 complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Mechanism of primer synthesis by Primase-Polymerases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Decoding Catalysis by Terpene Synthases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"How neurons maintain their axons long-term: an integrated view of axon biology and pathology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Sensor <scp>NLR</scp> immune proteins activate oligomerization of their <scp>NRC</scp> helpers in response to plant pathogens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Pitfalls of machine learning models for protein-protein interactions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Evaluation of model refinement in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Computational models in the service of X\\u2010ray and <scp>cryo\\u2010</scp>electron microscopy structure determination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Photovoltaic enzymes by design and evolution\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"SARS-CoV-2 variant biology: immune escape, transmission and fitness\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"An expandable, modular de novo protein platform for precision redox engineering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"SARS-CoV-2 variant evasion of monoclonal antibodies based on in vitro studies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"A new class of biological ion-driven rotary molecular motors with 5:2 symmetry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"NDP52 acts as a redox sensor in PINK1/Parkin\\u2010mediated mitophagy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Broader functions of TIR domains in Arabidopsis immunity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"How Pyroptosis Contributes to Inflammation and Fibroblast-Macrophage Cross-Talk in Rheumatoid Arthritis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"\\u03b2-1,4-Xylan backbone synthesis in higher plants: How complex can it be?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Potent cross-reactive antibodies following Omicron breakthrough in vaccinees\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Herpes simplex virus 1 protein pUL21 alters ceramide metabolism by activating the interorganelle transport protein CERT\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure of the priming arabinosyltransferase AftA required for AG biosynthesis of <i>Mycobacterium tuberculosis</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Considerations for defining +80 Da mass shifts in mass spectrometry-based proteomics: phosphorylation and beyond\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Discovery of small molecule ligands for the von Hippel-Lindau (VHL) E3 ligase and their use as inhibitors and PROTAC degraders\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"EMBL\\u2019s European Bioinformatics Institute (EMBL-EBI) in 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Energy landscape steering in SecYEG mediates dynamic coupling in ATP driven protein translocation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Review of Recurrently Mutated Genes in Craniosynostosis Supports Expansion of Diagnostic Gene Panels\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Multiple interactions of the dynein-2 complex with the IFT-B complex are required for effective intraflagellar transport\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Versatile Product Detection via Coupled Assays for Ultrahigh-Throughput Screening of Carbohydrate-Active Enzymes in Microfluidic Droplets\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanoenzymatic reactions for the hydrolysis of PET\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Affimers targeting proteins in the cardiomyocyte Z-disc: Novel tools that improve imaging of heart tissue\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The fluorescent protein iLOV as a reporter for screening of high\\u2010yield production of antimicrobial peptides in <i>Pichia pastoris</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Multi-omics identify falling LRRC15 as a COVID-19 severity marker and persistent pro-thrombotic signals in convalescence\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Meeting report \\u2013 Cell dynamics: host\\u2013pathogen interface\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Unifying the known and unknown microbial coding sequence space\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Absolute quantification of cellular levels of photosynthesis-related proteins in Synechocystis sp. PCC 6803\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The road to fully programmable protein catalysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Understanding the Stabilizing Effect of Histidine on mAb Aggregation: A Molecular Dynamics Study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Two Years into the COVID-19 Pandemic: Lessons Learned\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Thrombin cleavage of the hepatitis E virus polyprotein at multiple conserved locations is required for genome replication\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Modelling the viral dynamics of the SARS-CoV-2 Delta and Omicron variants in different cell types\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mammalian fertilization: Does sperm IZUMO1 mediate fusion as well as adhesion?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Antibody escape of SARS-CoV-2 Omicron BA.4 and BA.5 from vaccine and BA.1 serum\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The N-terminal executioner domains of NLR immune receptors are functionally conserved across major plant lineages\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Beyond transcription: compelling open questions in plant RNA biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"The Gene Ontology knowledgebase in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanistic basis for potent neutralization of Sin Nombre hantavirus by a human monoclonal antibody\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Reverse transcriptases prime DNA synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protein metalation in a nutshell\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Probing heavy metal binding to phycobiliproteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Probing the Internal pH and Permeability of a Carboxysome Shell\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Manipulation of plant immunity via an <scp>mRNA</scp> decapping pathogen effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Processing of NODULE INCEPTION controls the transition to nitrogen fixation in root nodules\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Repeat sequences limit the effectiveness of lateral gene transfer and favored the evolution of meiotic sex in early eukaryotes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Quality by Design for enabling RNA platform production processes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Redeployment of automated <i>MrBUMP</i> search-model identification for map fitting in cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Methionine Sulfoxide Reductases Suppress the Formation of the [PSI+] Prion and Protein Aggregation in Yeast\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Recent insights into SARS\\u2010CoV\\u20102 omicron variant\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Rational Design of Phosphorylation-Responsive Coiled Coil-Peptide Assemblies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Proteomics Standards Initiative at Twenty Years: Current Activities and Future Work\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pulse dipolar EPR for determining nanomolar binding affinities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mechanism of glycogen synthase inactivation and interaction with glycogenin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Non-canonical <i>Staphylococcus aureus</i> pathogenicity island repression\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plasma FIB milling for the determination of structures in situ\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Potential Role of the Host-Derived Cell-Wall Binding Domain of Endolysin CD16/50L as a Molecular Anchor in Preservation of Uninfected Clostridioides difficile for New Rounds of Phage Infection\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Multi-organ functions of yolk sac during human early development\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Out of the <scp>ESCPE</scp> room: Emerging roles of endosomal <scp>SNX\\u2010BARs</scp> in receptor transport and host\\u2013pathogen interaction\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Optimized expression and purification of adipose triglyceride lipase improved hydrolytic and transacylation activities in vitro\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"On-demand cell-autonomous gene therapy for brain circuit disorders\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Multi\\u2010Stimuli Responsive and Mechano\\u2010Actuated Biomimetic Membrane Nanopores Self\\u2010Assembled from DNA\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mutation of the ALS/FTD-associated RNA-binding protein FUS alters axonal cytoskeletal organisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"O-Linked Sialoglycans Modulate the Proteolysis of SARS-CoV-2 Spike and Likely Contribute to the Mutational Trajectory in Variants of Concern\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Nutrient-Response Pathways in Healthspan and Lifespan Regulation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mutations in the chloroplast inner envelope protein TIC100 impair and repair chloroplast protein import and impact retrograde signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"NLR immune receptor\\u2013nanobody fusions confer plant disease resistance\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Na\\u00efve Human Macrophages Are Refractory to SARS-CoV-2 Infection and Exhibit a Modest Inflammatory Response Early in Infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Non canonical scaffold-type ligase complex mediates protein UFMylation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Molecular Determinants of Carbocation Cyclisation in Bacterial Monoterpene Synthases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Poly(vinyl alcohol) Molecular Bottlebrushes Nucleate Ice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Plasmids manipulate bacterial behaviour through translational regulatory crosstalk\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Non-generalizability of biomarkers for mortality in SARS-CoV-2: a meta-analyses series\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Plants to mine metals and remediate land\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plant vernalization proteins contain unusual PHD superdomains without histone H3 binding activity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Molecular basis for the regulation of human glycogen synthase by phosphorylation and glucose-6-phosphate\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Molecular basis for the reversible ADP-ribosylation of guanosine bases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Perspective: Structure determination of protein-ligand complexes at room temperature using X-ray diffraction approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pathogenicity and virulence of <i>Clostridioides difficile</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Parallel recruitment pathways contribute to synaptonemal complex assembly during mammalian meiosis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PI(3,5)P2 and NAADP: Team players or lone warriors? \\u2013 New insights into TPC activation modes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PDBx/mmCIF Ecosystem: Foundational Semantic Tools for Structural Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Morphology exploration of pollen using deep learning latent space\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural Conservation of Insulin/IGF Signalling Axis at the Insulin Receptors Level in<i>Drosophila</i>and humans\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Reviving grain quality in wheat through non\\u2010destructive phenotyping techniques like hyperspectral imaging\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The origins and molecular evolution of SARS-CoV-2 lineage B.1.1.7 in the UK\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The host phylogeny determines viral infectivity and replication across Staphylococcus host species\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The helper NLR immune protein NRC3 mediates the hypersensitive cell death caused by the cell-surface receptor Cf-4\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The evolution of selective autophagy as a mechanism of oxidative stress response\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The PCDDB (Protein Circular Dichroism Data Bank): A Bioinformatics Resource for Protein Characterisations and Methods Development\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The New Era of High-Throughput Nanoelectrochemistry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The IUPHAR/BPS guide to PHARMACOLOGY in 2022: curating pharmacology for COVID-19, malaria and antibacterials\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The DNA replication initiation protein DnaD recognises a specific strand of the <i>Bacillus subtilis</i> chromosome origin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Crystal Structure of Bacillus thuringiensis Tpp80Aa1 and Its Interaction with Galactose-Containing Glycolipids\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"TRAIP resolves DNA replication-transcription conflicts during the S-phase of unperturbed cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Synthetic engineering of a new biocatalyst encapsulating [NiFe]-hydrogenases for enhanced hydrogen production\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The limitations of the current protein classification tools in identifying lipolytic features in putative bacterial lipase sequences\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The road to fully programmable protein catalysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Root angle is controlled by<i>EGT1</i>in cereal crops employing an antigravitropic mechanism\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The snapdragon genomes reveal the evolutionary dynamics of the<i>S</i>locus supergene\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"mPPases create a conserved anionic membrane fingerprint as identified via multi-scale simulations\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Why should we compare morphological and molecular disparity?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Visualising the cytoskeletal machinery in neuronal growth cones using cryo-electron tomography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Viral Load in COVID-19 Patients: Implications for Prognosis and Vaccine Efficacy in the Context of Emerging SARS-CoV-2 Variants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Versatile Chemo-Biocatalytic Cascade Driven by a Thermophilic and Irreversible C\\u2013C Bond-Forming \\u03b1-Oxoamine Synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Unifying the known and unknown microbial coding sequence space\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Understudied proteins: opportunities and challenges for functional proteomics\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Understanding the evolution of immune genes in jawed vertebrates\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Unconventional secretion of \\u03b1-synuclein mediated by palmitoylated DNAJC5 oligomers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The<i>Solanum americanum</i>pangenome and effectoromics reveal new resistance genes against potato late blight\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The \\u03b2I domain promotes active \\u03b21 integrin clustering into mature adhesion sites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The value of genotype-specific reference for transcriptome analyses in barley\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Synthetic Sansanmycin Analogues as Potent <i>Mycobacterium tuberculosis</i> Translocase I Inhibitors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Surface-Functionalized Metal\\u2013Organic Frameworks for Binding Coronavirus Proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Super-resolution microscopy and studies of peroxisomes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Subcellular Transcriptomics and Proteomics: A Comparative Methods Review\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Stanniocalcin\\u20102 inhibits skeletal muscle growth and is upregulated in functional overload\\u2010induced hypertrophy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Single-particle cryo-EM analysis of the shell architecture and internal organization of an intact \\u03b1-carboxysome\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Single-cell methods in myeloproliferative neoplasms: old questions, new technologies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Single-cell RNA sequencing reveals transcriptional changes of human choroidal and retinal pigment epithelium cells during fetal development, in healthy adult and intermediate age-related macular degeneration\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Severity of omicron variant of concern and effectiveness of vaccine boosters against symptomatic disease in Scotland (EAVE II): a national cohort study with nested test-negative design\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Seipin concentrates distinct neutral lipids via interactions with their acyl chain carboxyl esters\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 spike N-terminal domain modulates TMPRSS2-dependent viral entry and fusogenicity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 antibody responses associate with sex, age and disease severity in previously uninfected people admitted to hospital with COVID-19: An ISARIC4C prospective study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron is an immune escape variant with an altered cell entry pathway\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"S-acylation of the Wnt receptor Frizzled-5 by zDHHC5 controls its cellular localization and synaptogenic activity in the rodent hippocampus\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure, substrate recognition and initiation of hyaluronan synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure of trimeric pre-fusion rabies virus glycoprotein in complex with two protective antibodies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural features of the protein kinase domain and targeted binding by small-molecule inhibitors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural diversity and modularity of photosynthetic RC\\u2212LH1 complexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis of lipoprotein recognition by the bacterial Lol trafficking chaperone LolA\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of lipopolysaccharide maturation by the O-antigen ligase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for proton coupled cystine transport by cystinosin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for membrane attack complex inhibition by CD59\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for binding of the renal carcinoma target hypoxia\\u2010inducible factor 2\\u03b1 to prolyl hydroxylase domain 2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"5-Substituted Pyridine-2,4-dicarboxylate Derivatives Have Potential for Selective Inhibition of Human Jumonji-C Domain-Containing Protein 5\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"MGnify Genomes: A Resource for Biome-specific Microbial Genome Catalogues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterization of variational quantum algorithms using free fermions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Conformational changes and CO2-induced channel gating in connexin26\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational Resources for Molecular Biology 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational Resources for Molecular Biology 2022\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Compensatory mutations reducing the fitness cost of plasmid carriage occur in plant rhizosphere communities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Comparative analysis of the risks of hospitalisation and death associated with SARS-CoV-2 omicron (B.1.1.529) and delta (B.1.617.2) variants in England: a cohort study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Comment on: \\u201cComputer Simulations Reveal an Entirely Entropic Activation Barrier for the Chemical Step in a Designer Enzyme\\u201d\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Circumventing the packaging limit of AAV-mediated gene replacement therapy for neurological disorders\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Circadian regulation of pulmonary disease: the importance of timing\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterization of a glycan-binding complex of minor pilins completes the analysis of <i>Streptococcus sanguinis</i> type 4 pili subunits\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cation ATPase (ATP4) Orthologue Replacement in the Malaria Parasite Plasmodium knowlesi Reveals Species-Specific Responses to ATP4-Targeting Drugs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of complexes III and IV\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"CMTr mediated 2\\u2032-<i>O</i>-ribose methylation status of cap-adjacent nucleotides across animals\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"C16orf72/HAPSTR1/TAPR1 functions with BRCA1/Senataxin to modulate replication-associated R-loops and confer resistance to PARP disruption\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Brain monoamine vesicular transport disease caused by homozygous SLC18A2 variants: A study in 42 affected individuals\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Booster Vaccination Against SARS-CoV-2 Induces Potent Immune Responses in People With Human Immunodeficiency Virus\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Conserved and Divergent Features of pH Sensing in Major Fungal Pathogens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Controlling contacts\\u2014Molecular mechanisms to regulate organelle membrane tethering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlation between the binding affinity and the conformational entropy of nanobody SARS-CoV-2 spike protein complexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlation energy of the paramagnetic electron gas at the thermodynamic limit\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Deep-Learning-Assisted Focused Ion Beam Nanofabrication\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Decoding Catalysis by Terpene Synthases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Darobactin B Stabilises a Lateral\\u2010Closed Conformation of the BAM Complex in <i>E. coli</i> Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA methylation-based predictors of health: applications and statistical considerations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA methylation dynamics during germline development\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cyclic di-AMP traps proton-coupled K+ transporters of the KUP family in an inward-occluded conformation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CryoEM structure of the outer membrane secretin channel pIV from the f1 filamentous bacteriophage\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structures of human fucosidase FucA1 reveal insight into substrate recognition and catalysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the human Kv3.1 channel reveals gating control by the cytoplasmic T1 domain\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the four-subunit <i>Rhodobacter sphaeroides</i> cytochrome <i>bc</i> <sub>1</sub> complex in styrene maleic acid nanodiscs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the Smc5/6 holo-complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Correlation energy of the spin-polarized electron liquid studied using quantum Monte Carlo simulations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biallelic variants in OGDH encoding oxoglutarate dehydrogenase lead to a neurodevelopmental disorder characterized by global developmental delay, movement disorder, and metabolic abnormalities\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Bacteriophages benefit from mobilizing pathogenicity islands encoding immune systems against competitors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A new dawn beyond lysine ubiquitination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A founder<i>UMOD</i>variant is a common cause of hereditary nephropathy in the British population\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A designed photoenzyme for enantioselective [2+2] cycloadditions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A common TMPRSS2 variant has a protective effect against severe COVID-19\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A bovine antibody possessing an ultralong complementarity-determining region CDRH3 targets a highly conserved epitope in sarbecovirus spike proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Unique Mode of Coenzyme A Binding to the Nucleotide Binding Pocket of Human Metastasis Suppressor NME1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A 2.2 \\u00c5 cryoEM structure of a quinol-dependent NO Reductase shows close similarity to respiratory oxidases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<scp>Cryo\\u2010EM</scp> targets in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<scp>AAK1</scp>\\u2010like: A putative pseudokinase with potential roles in cargo uptake in bloodstream form <i>Trypanosoma brucei</i> parasites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>arfA</i>antisense RNA regulates MscL excretory activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>UNC13A</i>in amyotrophic lateral sclerosis: from genetic association to therapeutic target\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>Toxoplasma</i> ceramide synthases: a curious case of gene duplication, divergence and key functionality\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>Dapper:</i> Deploying Service Function Chains in the Programmable Data Plane Via Deep Reinforcement Learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A new dimethylsulfoniopropionate lyase of the cupin superfamily in marine bacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel fold for acyltransferase-3 (AT3) proteins provides a framework for transmembrane acyl-group transfer\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Assembling membraneless organelles from de novo designed proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"An open invitation to the Understudied Proteins Initiative\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"An eggshell-localised annexin plays a key role in the coordination of the life cycle of a plant-parasitic nematode with its host\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Altered TMPRSS2 usage by SARS-CoV-2 Omicron impacts infectivity and fusogenicity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Allele-specific gene expression can underlie altered transcript abundance in zebrafish mutants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Affinity-enhanced RNA-binding domains as tools to understand RNA recognition\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Advances in methods to analyse cardiolipin and their clinical applications\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Accelerated waning of the humoral response to SARS-CoV-2 vaccines in obesity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Accelerated waning of the humoral response to COVID-19 vaccines in obesity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Absence of Staphylococcus aureus in Wild Populations of Fish Supports a Spillover Hypothesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A versatile cortical pattern-forming circuit based on Rho, F-actin, Ect2, and RGA-3/4\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A systems approach evaluating the impact of SARS-CoV-2 variant of concern mutations on CD8+ T cell responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A robust deep learning platform to predict CD8+ T-cell epitopes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A ribosomally synthesised and post-translationally modified peptide containing a \\u03b2-enamino acid and a macrocyclic motif\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A primer to directed evolution: current methodologies and future directions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Diene incorporation by a dehydratase domain variant in modular polyketide synthases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dipolar Order Parameters in Large Systems With Fast Spinning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Implicating genes, pleiotropy, and sexual dimorphism at blood lipid loci through multi-ancestry meta-analysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Impact of Type II LRRK2 inhibitors on signaling and mitophagy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Immune-mediated inflammation across disease boundaries: breaking down research silos\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Identification of unusual oxysterols biosynthesised in human pregnancy by charge-tagging and liquid chromatography - mass spectrometry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Identification of the target and mode of action for the prokaryotic nucleotide excision repair inhibitor ATBC\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Identification of Unusual Oxysterols Biosynthesised in Human Pregnancy by Charge-Tagging and Liquid Chromatography - Mass Spectrometry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Hypoxia inducible factors regulate infectious SARS-CoV-2, epithelial damage and respiratory symptoms in a hamster COVID-19 model\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Hypothalamic PVN CRH neurons signal through PVN GABA neurons to suppress GnRH pulse generator frequency in female mice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Human vascularised synovium-on-a-chip: a mechanically stimulated, microfluidic model to investigate synovial inflammation and monocyte recruitment\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"How Arrestins and GRKs Regulate the Function of Long Chain Fatty Acid Receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High\\u2010accuracy protein structure prediction in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-throughput property-driven generative design of functional organic molecules\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"High throughput steady-state enzyme kinetics measured in a parallel droplet generation and absorbance detection platform\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Head-to-tail polymerization by VEL proteins underpins cold-induced Polycomb silencing in flowering control\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"HPV E6 inhibits E6AP to regulate epithelial homeostasis by modulating keratinocyte differentiation commitment and YAP1 activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"H3K4 methylation by SETD1A/BOD1L facilitates RIF1-dependent NHEJ\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>Birnaviridae</i> Virus Factories Show Features of Liquid-Liquid Phase Separation and Are Distinct from Paracrystalline Arrays of Virions Observed by Electron Microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Inorganic Nitrogen Transport and Assimilation in Pea (Pisum sativum)\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Light sensitive short hypocotyl (LSH) confer symbiotic nodule identity in the legume Medicago truncatula\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Lateral flow test engineering and lessons learned from COVID-19\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"LRRC15 mediates an accessory interaction with the SARS-CoV-2 spike protein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ion\\u2010Selective Microporous Polymer Membranes with Hydrogen\\u2010Bond and Salt\\u2010Bridge Networks for Aqueous Organic Redox Flow Batteries\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intrinsically disordered proteins: modes of binding with emphasis on disordered domains\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Interrogation of an Enzyme Library Reveals the Catalytic Plasticity of Naturally Evolved [4+2] Cyclases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"InterPro in 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"InterPro in 2022\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Integrated view and comparative analysis of baseline protein expression in mouse and rat tissues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Integrated View of Baseline Protein Expression in Human Tissues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Insights into the Roles of Lipoteichoic Acids and MprF in Bacillus subtilis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Growth-coupled enzyme engineering through manipulation of redox cofactor regeneration\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Disruptions in endocytic traffic contribute to the activation of the NLRP3 inflammasome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exploring the antimicrobial peptidome of nematodes through phylum-spanning in silico analyses highlights novel opportunities for pathogen control\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evolutionary dynamics of genome size and content during the adaptive radiation of Heliconiini butterflies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evolution of long-term vaccine-induced and hybrid immunity in healthcare workers after different COVID-19 vaccine regimens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evaluation of Nafamostat as Chemoprophylaxis for SARS-CoV-2 Infection in Hamsters\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Enzyme self-label-bound ATTO700 in single-molecule and super-resolution microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ensembl 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ensembl 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Engineering mutually orthogonal <scp>PylRS</scp>/<scp>tRNA</scp> pairs for dual encoding of functional histidine analogues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Engineering enzyme activity using an expanded amino acid alphabet\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Editorial: Experiments and Simulations: A Pas de Deux to Unravel Biological Function\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Diversity of viscoelastic properties of an engineered muscle-inspired protein hydrogel\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exome sequencing reveals aggregates of rare variants in glycosyltransferase and other genes influencing immunoglobulin G and transferrin glycosylation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Extending the Capabilities of Data-Driven Reduced-Order Models to Make Predictions for Unseen Scenarios: Applied to Flow Around Buildings\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic landscape of drug response reveals mediators of anthelmintic resistance\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fibrillin microfibril structure identifies long-range effects of inherited pathogenic mutations affecting a key regulatory latent TGF\\u03b2-binding site\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic evidence of contemporary hybridization between Schistosoma species\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic and functional analysis of phage\\u2010mediated horizontal gene transfer in <i>Pseudomonas syringae</i> on the plant surface\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic sequencing detected the SARS-CoV-2 delta variant in wastewater a month prior to the first COVID-19 case in Ahmedabad (India)\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Generation and Characterization of Native and Sialic Acid-Deficient IgE\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Generating images of hydrated pollen grains using deep learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"GWAS and meta-analysis identifies multiple new genetic mechanisms underlying severe Covid-19\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Fungal spore swelling and germination are restricted by the macrophage phagolysosome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fungal resilience and host\\u2013pathogen interactions: Future perspectives and opportunities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Full-length antithrombin frameshift variant with aberrant C-terminus causes endoplasmic reticulum retention with a dominant-negative effect\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Frontiers in metalloprotein crystallography and cryogenic electron microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Fragment-Based Drug Discovery by NMR. Where Are the Successes and Where can It Be Improved?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Create two separate DataFrames for indirect and direct citations\n",
    "indirect_df = citations[['title', 'af_indirect_citations']].rename(columns={'af_indirect_citations': 'count'})\n",
    "indirect_df['type'] = 'Indirect'\n",
    "\n",
    "direct_df = citations[['title', 'af_direct_citation']].rename(columns={'af_direct_citation': 'count'})\n",
    "direct_df['type'] = 'Direct'\n",
    "\n",
    "# Concatenate the two DataFrames\n",
    "combined_df = pd.concat([indirect_df, direct_df])\n",
    "\n",
    "# keep those that have at least one citation either directly or indirectly\n",
    "combined_df = combined_df[combined_df['count'] > 0]\n",
    "\n",
    "# Add a color column based on the type\n",
    "combined_df['color'] = combined_df['type'].map({'Indirect': 'blue', 'Direct': 'red'})\n",
    "\n",
    "# Group by title and type, and sum the counts\n",
    "grouped_df = combined_df.groupby(['title', 'type', 'color']).sum().reset_index()\n",
    "\n",
    "# Sort by the summed count\n",
    "grouped_df = grouped_df.sort_values(by='count', ascending=False)\n",
    "\n",
    "# Create the chart\n",
    "chart = alt.Chart(grouped_df).mark_bar().encode(\n",
    "    x=alt.X('count:Q', title='Number of Citations'),\n",
    "    y=alt.Y('title:N', title='Paper Title', sort='-x'),\n",
    "    color=alt.Color('type:N', title='Citation Type', scale=alt.Scale(domain=['Indirect', 'Direct'], range=['blue', 'red'])),\n",
    "    tooltip=['title', 'sum(count)', 'type']\n",
    ").properties(\n",
    "    title='Count of Papers Citing Alphafold (Direct and Indirect)',\n",
    "    height=3200\n",
    ")\n",
    "\n",
    "chart\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".igl_alphafold",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
