{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Exploration of Gateway to research data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The kedro.ipython extension is already loaded. To reload it, use:\n",
      "  %reload_ext kedro.ipython\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "\u001b[1;35mDataTransformerRegistry.enable\u001b[0m\u001b[1m(\u001b[0m\u001b[32m'default'\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%load_ext kedro.ipython\n",
    "\n",
    "import pandas as pd\n",
    "import altair as alt\n",
    "import yaml\n",
    "\n",
    "alt.data_transformers.disable_max_rows()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/25/24 17:20:25] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.institutions</span>              <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/25/24 17:20:25]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.institutions\u001b[0m              \u001b]8;id=149580;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=747239;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/25/24 17:20:31] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.publications</span>              <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/25/24 17:20:31]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.publications\u001b[0m              \u001b]8;id=73646;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=590574;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">[01/25/24 17:21:03] </span><span style=\"color: #000080; text-decoration-color: #000080\">INFO    </span> Loading data from <span style=\"color: #ff8700; text-decoration-color: #ff8700\">oa.data_processing.gtr.citations.alphafold</span>       <a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">data_catalog.py</span></a><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">:</span><a href=\"file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\" target=\"_blank\"><span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">482</span></a>\n",
       "<span style=\"color: #7fbfbf; text-decoration-color: #7fbfbf\">                    </span>         <span style=\"font-weight: bold\">(</span>ParquetDataset<span style=\"font-weight: bold\">)</span><span style=\"color: #808000; text-decoration-color: #808000\">...</span>                                                <span style=\"color: #7f7f7f; text-decoration-color: #7f7f7f\">                   </span>\n",
       "</pre>\n"
      ],
      "text/plain": [
       "\u001b[2;36m[01/25/24 17:21:03]\u001b[0m\u001b[2;36m \u001b[0m\u001b[34mINFO    \u001b[0m Loading data from \u001b[38;5;208moa.data_processing.gtr.citations.alphafold\u001b[0m       \u001b]8;id=465179;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py\u001b\\\u001b[2mdata_catalog.py\u001b[0m\u001b]8;;\u001b\\\u001b[2m:\u001b[0m\u001b]8;id=289394;file:///home/dampudia/miniconda3/envs/.igl_alphafold/lib/python3.11/site-packages/kedro/io/data_catalog.py#482\u001b\\\u001b[2m482\u001b[0m\u001b]8;;\u001b\\\n",
       "\u001b[2;36m                    \u001b[0m         \u001b[1m(\u001b[0mParquetDataset\u001b[1m)\u001b[0m\u001b[33m...\u001b[0m                                                \u001b[2m                   \u001b[0m\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gtr_institutions = catalog.load(\"oa.data_processing.gtr.institutions\")\n",
    "gtr_publications = catalog.load(\"oa.data_processing.gtr.publications\")\n",
    "gtr_citations = catalog.load('oa.data_processing.gtr.citations.alphafold')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Publications over time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-1549e40bbe5747dd95e75e9e03093975.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-1549e40bbe5747dd95e75e9e03093975.vega-embed details,\n",
       "  #altair-viz-1549e40bbe5747dd95e75e9e03093975.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-1549e40bbe5747dd95e75e9e03093975\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-1549e40bbe5747dd95e75e9e03093975\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-1549e40bbe5747dd95e75e9e03093975\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-e3390bc9be16e3b56911ff28e4b65cac\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"x\": {\"axis\": {\"grid\": false, \"labelAngle\": -45, \"labelFontSize\": 8, \"tickCount\": 452041}, \"field\": \"year_month\", \"title\": \"Publication Month\", \"type\": \"ordinal\"}, \"y\": {\"field\": \"counts\", \"title\": \"Number of Publications\", \"type\": \"quantitative\"}}, \"height\": 400, \"title\": \"Monthly Distribution of GTR Publications\", \"width\": 1800, \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-e3390bc9be16e3b56911ff28e4b65cac\": [{\"year_month\": \"2008-01\", \"counts\": 1136}, {\"year_month\": \"2008-02\", \"counts\": 505}, {\"year_month\": \"2008-03\", \"counts\": 526}, {\"year_month\": \"2008-04\", \"counts\": 565}, {\"year_month\": \"2008-05\", \"counts\": 575}, {\"year_month\": \"2008-06\", \"counts\": 545}, {\"year_month\": \"2008-07\", \"counts\": 637}, {\"year_month\": \"2008-08\", \"counts\": 633}, {\"year_month\": \"2008-09\", \"counts\": 683}, {\"year_month\": \"2008-10\", \"counts\": 687}, {\"year_month\": \"2008-11\", \"counts\": 682}, {\"year_month\": \"2008-12\", \"counts\": 751}, {\"year_month\": \"2009-01\", \"counts\": 2023}, {\"year_month\": \"2009-02\", \"counts\": 833}, {\"year_month\": \"2009-03\", \"counts\": 926}, {\"year_month\": \"2009-04\", \"counts\": 954}, {\"year_month\": \"2009-05\", \"counts\": 876}, {\"year_month\": \"2009-06\", \"counts\": 1162}, {\"year_month\": \"2009-07\", \"counts\": 1068}, {\"year_month\": \"2009-08\", \"counts\": 958}, {\"year_month\": \"2009-09\", \"counts\": 1076}, {\"year_month\": \"2009-10\", \"counts\": 1110}, {\"year_month\": \"2009-11\", \"counts\": 958}, {\"year_month\": \"2009-12\", \"counts\": 1078}, {\"year_month\": \"2010-01\", \"counts\": 2890}, {\"year_month\": \"2010-02\", \"counts\": 1105}, {\"year_month\": \"2010-03\", \"counts\": 1279}, {\"year_month\": \"2010-04\", \"counts\": 1165}, {\"year_month\": \"2010-05\", \"counts\": 1128}, {\"year_month\": \"2010-06\", \"counts\": 1274}, {\"year_month\": \"2010-07\", \"counts\": 1490}, {\"year_month\": \"2010-08\", \"counts\": 1186}, {\"year_month\": \"2010-09\", \"counts\": 1368}, {\"year_month\": \"2010-10\", \"counts\": 1336}, {\"year_month\": \"2010-11\", \"counts\": 1400}, {\"year_month\": \"2010-12\", \"counts\": 1327}, {\"year_month\": \"2011-01\", \"counts\": 3336}, {\"year_month\": \"2011-02\", \"counts\": 1210}, {\"year_month\": \"2011-03\", \"counts\": 1459}, {\"year_month\": \"2011-04\", \"counts\": 1251}, {\"year_month\": \"2011-05\", \"counts\": 1431}, {\"year_month\": \"2011-06\", \"counts\": 1492}, {\"year_month\": \"2011-07\", \"counts\": 1337}, {\"year_month\": \"2011-08\", \"counts\": 1371}, {\"year_month\": \"2011-09\", \"counts\": 1528}, {\"year_month\": \"2011-10\", \"counts\": 1433}, {\"year_month\": \"2011-11\", \"counts\": 1461}, {\"year_month\": \"2011-12\", \"counts\": 1390}, {\"year_month\": \"2012-01\", \"counts\": 3829}, {\"year_month\": \"2012-02\", \"counts\": 1484}, {\"year_month\": \"2012-03\", \"counts\": 1690}, {\"year_month\": \"2012-04\", \"counts\": 1433}, {\"year_month\": \"2012-05\", \"counts\": 1585}, {\"year_month\": \"2012-06\", \"counts\": 1619}, {\"year_month\": \"2012-07\", \"counts\": 1602}, {\"year_month\": \"2012-08\", \"counts\": 1534}, {\"year_month\": \"2012-09\", \"counts\": 1705}, {\"year_month\": \"2012-10\", \"counts\": 1718}, {\"year_month\": \"2012-11\", \"counts\": 1662}, {\"year_month\": \"2012-12\", \"counts\": 1595}, {\"year_month\": \"2013-01\", \"counts\": 4672}, {\"year_month\": \"2013-02\", \"counts\": 1698}, {\"year_month\": \"2013-03\", \"counts\": 1904}, {\"year_month\": \"2013-04\", \"counts\": 1890}, {\"year_month\": \"2013-05\", \"counts\": 1940}, {\"year_month\": \"2013-06\", \"counts\": 1868}, {\"year_month\": \"2013-07\", \"counts\": 1958}, {\"year_month\": \"2013-08\", \"counts\": 1818}, {\"year_month\": \"2013-09\", \"counts\": 2069}, {\"year_month\": \"2013-10\", \"counts\": 2102}, {\"year_month\": \"2013-11\", \"counts\": 1943}, {\"year_month\": \"2013-12\", \"counts\": 1979}, {\"year_month\": \"2014-01\", \"counts\": 4996}, {\"year_month\": \"2014-02\", \"counts\": 1814}, {\"year_month\": \"2014-03\", \"counts\": 2097}, {\"year_month\": \"2014-04\", \"counts\": 2152}, {\"year_month\": \"2014-05\", \"counts\": 1955}, {\"year_month\": \"2014-06\", \"counts\": 2272}, {\"year_month\": \"2014-07\", \"counts\": 2194}, {\"year_month\": \"2014-08\", \"counts\": 2111}, {\"year_month\": \"2014-09\", \"counts\": 2352}, {\"year_month\": \"2014-10\", \"counts\": 2289}, {\"year_month\": \"2014-11\", \"counts\": 2086}, {\"year_month\": \"2014-12\", \"counts\": 2164}, {\"year_month\": \"2015-01\", \"counts\": 5325}, {\"year_month\": \"2015-02\", \"counts\": 2066}, {\"year_month\": \"2015-03\", \"counts\": 2467}, {\"year_month\": \"2015-04\", \"counts\": 2348}, {\"year_month\": \"2015-05\", \"counts\": 2342}, {\"year_month\": \"2015-06\", \"counts\": 2601}, {\"year_month\": \"2015-07\", \"counts\": 2541}, {\"year_month\": \"2015-08\", \"counts\": 2404}, {\"year_month\": \"2015-09\", \"counts\": 2718}, {\"year_month\": \"2015-10\", \"counts\": 2673}, {\"year_month\": \"2015-11\", \"counts\": 2506}, {\"year_month\": \"2015-12\", \"counts\": 2574}, {\"year_month\": \"2016-01\", \"counts\": 5928}, {\"year_month\": \"2016-02\", \"counts\": 2591}, {\"year_month\": \"2016-03\", \"counts\": 2545}, {\"year_month\": \"2016-04\", \"counts\": 2498}, {\"year_month\": \"2016-05\", \"counts\": 2522}, {\"year_month\": \"2016-06\", \"counts\": 2690}, {\"year_month\": \"2016-07\", \"counts\": 2741}, {\"year_month\": \"2016-08\", \"counts\": 2561}, {\"year_month\": \"2016-09\", \"counts\": 2861}, {\"year_month\": \"2016-10\", \"counts\": 2763}, {\"year_month\": \"2016-11\", \"counts\": 2594}, {\"year_month\": \"2016-12\", \"counts\": 2557}, {\"year_month\": \"2017-01\", \"counts\": 5549}, {\"year_month\": \"2017-02\", \"counts\": 2530}, {\"year_month\": \"2017-03\", \"counts\": 2767}, {\"year_month\": \"2017-04\", \"counts\": 2401}, {\"year_month\": \"2017-05\", \"counts\": 2655}, {\"year_month\": \"2017-06\", \"counts\": 2662}, {\"year_month\": \"2017-07\", \"counts\": 2769}, {\"year_month\": \"2017-08\", \"counts\": 2722}, {\"year_month\": \"2017-09\", \"counts\": 2836}, {\"year_month\": \"2017-10\", \"counts\": 2894}, {\"year_month\": \"2017-11\", \"counts\": 2835}, {\"year_month\": \"2017-12\", \"counts\": 2714}, {\"year_month\": \"2018-01\", \"counts\": 5778}, {\"year_month\": \"2018-02\", \"counts\": 2777}, {\"year_month\": \"2018-03\", \"counts\": 2752}, {\"year_month\": \"2018-04\", \"counts\": 2850}, {\"year_month\": \"2018-05\", \"counts\": 2690}, {\"year_month\": \"2018-06\", \"counts\": 2945}, {\"year_month\": \"2018-07\", \"counts\": 3144}, {\"year_month\": \"2018-08\", \"counts\": 2816}, {\"year_month\": \"2018-09\", \"counts\": 3006}, {\"year_month\": \"2018-10\", \"counts\": 3236}, {\"year_month\": \"2018-11\", \"counts\": 2907}, {\"year_month\": \"2018-12\", \"counts\": 2741}, {\"year_month\": \"2019-01\", \"counts\": 5973}, {\"year_month\": \"2019-02\", \"counts\": 3156}, {\"year_month\": \"2019-03\", \"counts\": 3100}, {\"year_month\": \"2019-04\", \"counts\": 2882}, {\"year_month\": \"2019-05\", \"counts\": 3178}, {\"year_month\": \"2019-06\", \"counts\": 3042}, {\"year_month\": \"2019-07\", \"counts\": 3192}, {\"year_month\": \"2019-08\", \"counts\": 2989}, {\"year_month\": \"2019-09\", \"counts\": 3065}, {\"year_month\": \"2019-10\", \"counts\": 3152}, {\"year_month\": \"2019-11\", \"counts\": 3234}, {\"year_month\": \"2019-12\", \"counts\": 3002}, {\"year_month\": \"2020-01\", \"counts\": 6289}, {\"year_month\": \"2020-02\", \"counts\": 3009}, {\"year_month\": \"2020-03\", \"counts\": 3205}, {\"year_month\": \"2020-04\", \"counts\": 3013}, {\"year_month\": \"2020-05\", \"counts\": 3030}, {\"year_month\": \"2020-06\", \"counts\": 3437}, {\"year_month\": \"2020-07\", \"counts\": 3414}, {\"year_month\": \"2020-08\", \"counts\": 3074}, {\"year_month\": \"2020-09\", \"counts\": 3581}, {\"year_month\": \"2020-10\", \"counts\": 3761}, {\"year_month\": \"2020-11\", \"counts\": 3328}, {\"year_month\": \"2020-12\", \"counts\": 3669}, {\"year_month\": \"2021-01\", \"counts\": 6702}, {\"year_month\": \"2021-02\", \"counts\": 3319}, {\"year_month\": \"2021-03\", \"counts\": 3884}, {\"year_month\": \"2021-04\", \"counts\": 3089}, {\"year_month\": \"2021-05\", \"counts\": 3298}, {\"year_month\": \"2021-06\", \"counts\": 3397}, {\"year_month\": \"2021-07\", \"counts\": 3467}, {\"year_month\": \"2021-08\", \"counts\": 3087}, {\"year_month\": \"2021-09\", \"counts\": 3390}, {\"year_month\": \"2021-10\", \"counts\": 3411}, {\"year_month\": \"2021-11\", \"counts\": 3241}, {\"year_month\": \"2021-12\", \"counts\": 3393}, {\"year_month\": \"2022-01\", \"counts\": 5681}, {\"year_month\": \"2022-02\", \"counts\": 2901}, {\"year_month\": \"2022-03\", \"counts\": 3364}, {\"year_month\": \"2022-04\", \"counts\": 2702}, {\"year_month\": \"2022-05\", \"counts\": 3014}, {\"year_month\": \"2022-06\", \"counts\": 3084}, {\"year_month\": \"2022-07\", \"counts\": 2927}, {\"year_month\": \"2022-08\", \"counts\": 2956}, {\"year_month\": \"2022-09\", \"counts\": 2995}, {\"year_month\": \"2022-10\", \"counts\": 3048}, {\"year_month\": \"2022-11\", \"counts\": 2933}, {\"year_month\": \"2022-12\", \"counts\": 2826}, {\"year_month\": \"2023-01\", \"counts\": 3726}, {\"year_month\": \"2023-02\", \"counts\": 2233}, {\"year_month\": \"2023-03\", \"counts\": 1718}, {\"year_month\": \"2023-04\", \"counts\": 1207}, {\"year_month\": \"2023-05\", \"counts\": 1159}, {\"year_month\": \"2023-06\", \"counts\": 1098}, {\"year_month\": \"2023-07\", \"counts\": 1118}, {\"year_month\": \"2023-08\", \"counts\": 1002}, {\"year_month\": \"2023-09\", \"counts\": 664}, {\"year_month\": \"2023-10\", \"counts\": 197}, {\"year_month\": \"2023-11\", \"counts\": 122}, {\"year_month\": \"2023-12\", \"counts\": 97}, {\"year_month\": \"2024-01\", \"counts\": 40}, {\"year_month\": \"2024-02\", \"counts\": 7}, {\"year_month\": \"2024-03\", \"counts\": 1}, {\"year_month\": \"2024-04\", \"counts\": 3}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Assuming gtr_publications is already defined and loaded\n",
    "pubs = gtr_publications.copy()\n",
    "pubs['publication_date'] = pd.to_datetime(pubs['publication_date'])\n",
    "\n",
    "# keep publications from 2008 onwards\n",
    "pubs = pubs[pubs['publication_date'] >= '2008-01-01']\n",
    "\n",
    "pubs['year_month'] = pubs['publication_date'].dt.strftime('%Y-%m')\n",
    "\n",
    "# Group by 'year_month' and count the publications\n",
    "monthly_counts = pubs.groupby('year_month').size().reset_index(name='counts')\n",
    "\n",
    "# Create a histogram using the aggregated data\n",
    "chart = alt.Chart(monthly_counts).mark_bar().encode(\n",
    "    x=alt.X('year_month:O', title='Publication Month', axis=alt.Axis(tickCount=pubs.shape[0], grid=False, labelAngle=-45, labelFontSize=8)),\n",
    "    y=alt.Y('counts:Q', title='Number of Publications')\n",
    ").properties(\n",
    "    title='Monthly Distribution of GTR Publications',\n",
    "    width=1800,  # Adjusted for better readability\n",
    "    height=400\n",
    ")\n",
    "\n",
    "chart"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Distribution over institutions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>work_id</th>\n",
       "      <th>author_id</th>\n",
       "      <th>institution_id</th>\n",
       "      <th>institution_name</th>\n",
       "      <th>country_code</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>A5075519270</td>\n",
       "      <td>I40120149</td>\n",
       "      <td>University of Oxford</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>A5075519270</td>\n",
       "      <td>I4210146410</td>\n",
       "      <td>Science Oxford</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>A5065572308</td>\n",
       "      <td>I40120149</td>\n",
       "      <td>University of Oxford</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>A5065572308</td>\n",
       "      <td>I4210146410</td>\n",
       "      <td>Science Oxford</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>A5078802488</td>\n",
       "      <td>I40120149</td>\n",
       "      <td>University of Oxford</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233157</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>A5056240345</td>\n",
       "      <td>I45129253</td>\n",
       "      <td>University College London</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233158</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>A5055823700</td>\n",
       "      <td>I4210122016</td>\n",
       "      <td>Wellcome / EPSRC Centre for Interventional and...</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233159</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>A5055823700</td>\n",
       "      <td>I45129253</td>\n",
       "      <td>University College London</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233160</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>A5077630267</td>\n",
       "      <td>I4210122016</td>\n",
       "      <td>Wellcome / EPSRC Centre for Interventional and...</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4233161</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>A5077630267</td>\n",
       "      <td>I45129253</td>\n",
       "      <td>University College London</td>\n",
       "      <td>GB</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4233162 rows Ã— 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "\n",
       "             work_id    author_id institution_id  \\\n",
       "\u001b[1;36m0\u001b[0m        W2107167859  A5075519270      I40120149   \n",
       "\u001b[1;36m1\u001b[0m        W2107167859  A5075519270    I4210146410   \n",
       "\u001b[1;36m2\u001b[0m        W2107167859  A5065572308      I40120149   \n",
       "\u001b[1;36m3\u001b[0m        W2107167859  A5065572308    I4210146410   \n",
       "\u001b[1;36m4\u001b[0m        W2107167859  A5078802488      I40120149   \n",
       "\u001b[33m...\u001b[0m              \u001b[33m...\u001b[0m          \u001b[33m...\u001b[0m            \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m4233157\u001b[0m  W3212985413  A5056240345      I45129253   \n",
       "\u001b[1;36m4233158\u001b[0m  W3212985413  A5055823700    I4210122016   \n",
       "\u001b[1;36m4233159\u001b[0m  W3212985413  A5055823700      I45129253   \n",
       "\u001b[1;36m4233160\u001b[0m  W3212985413  A5077630267    I4210122016   \n",
       "\u001b[1;36m4233161\u001b[0m  W3212985413  A5077630267      I45129253   \n",
       "\n",
       "                                          institution_name country_code  \n",
       "\u001b[1;36m0\u001b[0m                                     University of Oxford           GB  \n",
       "\u001b[1;36m1\u001b[0m                                           Science Oxford           GB  \n",
       "\u001b[1;36m2\u001b[0m                                     University of Oxford           GB  \n",
       "\u001b[1;36m3\u001b[0m                                           Science Oxford           GB  \n",
       "\u001b[1;36m4\u001b[0m                                     University of Oxford           GB  \n",
       "\u001b[33m...\u001b[0m                                                    \u001b[33m...\u001b[0m          \u001b[33m...\u001b[0m  \n",
       "\u001b[1;36m4233157\u001b[0m                          University College London           GB  \n",
       "\u001b[1;36m4233158\u001b[0m  Wellcome \u001b[35m/\u001b[0m EPSRC Centre for Interventional and\u001b[33m...\u001b[0m           GB  \n",
       "\u001b[1;36m4233159\u001b[0m                          University College London           GB  \n",
       "\u001b[1;36m4233160\u001b[0m  Wellcome \u001b[35m/\u001b[0m EPSRC Centre for Interventional and\u001b[33m...\u001b[0m           GB  \n",
       "\u001b[1;36m4233161\u001b[0m                          University College London           GB  \n",
       "\n",
       "\u001b[1m[\u001b[0m\u001b[1;36m4233162\u001b[0m rows x \u001b[1;36m5\u001b[0m columns\u001b[1m]\u001b[0m"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_institutions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-d42b08a1260a42c4a8304912388afa8a.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-d42b08a1260a42c4a8304912388afa8a.vega-embed details,\n",
       "  #altair-viz-d42b08a1260a42c4a8304912388afa8a.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-d42b08a1260a42c4a8304912388afa8a\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-d42b08a1260a42c4a8304912388afa8a\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-d42b08a1260a42c4a8304912388afa8a\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-aa98da3b20229559bfb7711090c8440b\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"color\": {\"field\": \"Region\", \"title\": \"Region\", \"type\": \"nominal\"}, \"tooltip\": [{\"field\": \"institution_name\", \"type\": \"nominal\"}, {\"field\": \"count\", \"type\": \"quantitative\"}, {\"field\": \"Region\", \"type\": \"nominal\"}], \"x\": {\"field\": \"count\", \"title\": \"Number of Papers/Authors\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"institution_name\", \"sort\": \"-x\", \"title\": \"Institution Name\", \"type\": \"nominal\"}}, \"height\": 1000, \"title\": \"Distribution of Institutions in GB vs RoW\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-aa98da3b20229559bfb7711090c8440b\": [{\"institution_id\": \"I100288624\", \"institution_name\": \"John Innes Centre\", \"country_code\": \"GB\", \"count\": 5478, \"Region\": \"GB\"}, {\"institution_id\": \"I102197404\", \"institution_name\": \"University of Paris-Sud\", \"country_code\": \"FR\", \"count\": 4316, \"Region\": \"RoW\"}, {\"institution_id\": \"I110002522\", \"institution_name\": \"University of Essex\", \"country_code\": \"GB\", \"count\": 4364, \"Region\": \"GB\"}, {\"institution_id\": \"I1118541\", \"institution_name\": \"University of East Anglia\", \"country_code\": \"GB\", \"count\": 12117, \"Region\": \"GB\"}, {\"institution_id\": \"I114457229\", \"institution_name\": \"University of Geneva\", \"country_code\": \"CH\", \"count\": 5449, \"Region\": \"RoW\"}, {\"institution_id\": \"I121797337\", \"institution_name\": \"Leiden University\", \"country_code\": \"NL\", \"count\": 5280, \"Region\": \"RoW\"}, {\"institution_id\": \"I122411786\", \"institution_name\": \"California Institute of Technology\", \"country_code\": \"US\", \"count\": 9466, \"Region\": \"RoW\"}, {\"institution_id\": \"I123387679\", \"institution_name\": \"Uppsala University\", \"country_code\": \"SE\", \"count\": 4121, \"Region\": \"RoW\"}, {\"institution_id\": \"I124055696\", \"institution_name\": \"University of Copenhagen\", \"country_code\": \"DK\", \"count\": 7828, \"Region\": \"RoW\"}, {\"institution_id\": \"I124357947\", \"institution_name\": \"University of London\", \"country_code\": \"GB\", \"count\": 5303, \"Region\": \"GB\"}, {\"institution_id\": \"I126231945\", \"institution_name\": \"Queen's University Belfast\", \"country_code\": \"GB\", \"count\": 20045, \"Region\": \"GB\"}, {\"institution_id\": \"I1286704778\", \"institution_name\": \"Rutherford Appleton Laboratory\", \"country_code\": \"GB\", \"count\": 10363, \"Region\": \"GB\"}, {\"institution_id\": \"I1294671590\", \"institution_name\": \"French National Centre for Scientific Research\", \"country_code\": \"FR\", \"count\": 50457, \"Region\": \"RoW\"}, {\"institution_id\": \"I129604602\", \"institution_name\": \"University of Sydney\", \"country_code\": \"AU\", \"count\": 5376, \"Region\": \"RoW\"}, {\"institution_id\": \"I1299303238\", \"institution_name\": \"National Institutes of Health\", \"country_code\": \"US\", \"count\": 4352, \"Region\": \"RoW\"}, {\"institution_id\": \"I1303153112\", \"institution_name\": \"European Bioinformatics Institute\", \"country_code\": \"GB\", \"count\": 5605, \"Region\": \"GB\"}, {\"institution_id\": \"I1306266525\", \"institution_name\": \"Goddard Space Flight Center\", \"country_code\": \"US\", \"count\": 4654, \"Region\": \"RoW\"}, {\"institution_id\": \"I130828816\", \"institution_name\": \"University of Leeds\", \"country_code\": \"GB\", \"count\": 42556, \"Region\": \"GB\"}, {\"institution_id\": \"I1314696892\", \"institution_name\": \"Fermi National Accelerator Laboratory\", \"country_code\": \"US\", \"count\": 5460, \"Region\": \"RoW\"}, {\"institution_id\": \"I132053463\", \"institution_name\": \"Universit\\u00e9 Libre de Bruxelles\", \"country_code\": \"BE\", \"count\": 4421, \"Region\": \"RoW\"}, {\"institution_id\": \"I1324925910\", \"institution_name\": \"St Thomas' Hospital\", \"country_code\": \"GB\", \"count\": 4012, \"Region\": \"GB\"}, {\"institution_id\": \"I1334627681\", \"institution_name\": \"Jet Propulsion Laboratory\", \"country_code\": \"US\", \"count\": 4369, \"Region\": \"RoW\"}, {\"institution_id\": \"I1336263701\", \"institution_name\": \"Wellcome Centre for Human Genetics\", \"country_code\": \"GB\", \"count\": 4343, \"Region\": \"GB\"}, {\"institution_id\": \"I133731052\", \"institution_name\": \"University of Helsinki\", \"country_code\": \"FI\", \"count\": 5965, \"Region\": \"RoW\"}, {\"institution_id\": \"I133960621\", \"institution_name\": \"Roslin Institute\", \"country_code\": \"GB\", \"count\": 5618, \"Region\": \"GB\"}, {\"institution_id\": \"I136199984\", \"institution_name\": \"Harvard University\", \"country_code\": \"US\", \"count\": 10734, \"Region\": \"RoW\"}, {\"institution_id\": \"I138689650\", \"institution_name\": \"University of Padua\", \"country_code\": \"IT\", \"count\": 4005, \"Region\": \"RoW\"}, {\"institution_id\": \"I138801177\", \"institution_name\": \"University of Ulster\", \"country_code\": \"GB\", \"count\": 4210, \"Region\": \"GB\"}, {\"institution_id\": \"I141584323\", \"institution_name\": \"University of Hertfordshire\", \"country_code\": \"GB\", \"count\": 4210, \"Region\": \"GB\"}, {\"institution_id\": \"I141945490\", \"institution_name\": \"University of British Columbia\", \"country_code\": \"CA\", \"count\": 4760, \"Region\": \"RoW\"}, {\"institution_id\": \"I142263535\", \"institution_name\": \"University of Nottingham\", \"country_code\": \"GB\", \"count\": 43859, \"Region\": \"GB\"}, {\"institution_id\": \"I143804889\", \"institution_name\": \"Loughborough University\", \"country_code\": \"GB\", \"count\": 10005, \"Region\": \"GB\"}, {\"institution_id\": \"I145311948\", \"institution_name\": \"Johns Hopkins University\", \"country_code\": \"US\", \"count\": 5849, \"Region\": \"RoW\"}, {\"institution_id\": \"I145872427\", \"institution_name\": \"Radboud University Nijmegen\", \"country_code\": \"NL\", \"count\": 5156, \"Region\": \"RoW\"}, {\"institution_id\": \"I146655781\", \"institution_name\": \"University of Liverpool\", \"country_code\": \"GB\", \"count\": 36287, \"Region\": \"GB\"}, {\"institution_id\": \"I148283060\", \"institution_name\": \"Lawrence Berkeley National Laboratory\", \"country_code\": \"US\", \"count\": 5004, \"Region\": \"RoW\"}, {\"institution_id\": \"I153648349\", \"institution_name\": \"University of Leicester\", \"country_code\": \"GB\", \"count\": 15954, \"Region\": \"GB\"}, {\"institution_id\": \"I154526488\", \"institution_name\": \"Inserm\", \"country_code\": \"FR\", \"count\": 10759, \"Region\": \"RoW\"}, {\"institution_id\": \"I157614274\", \"institution_name\": \"University of Cape Town\", \"country_code\": \"ZA\", \"count\": 5110, \"Region\": \"RoW\"}, {\"institution_id\": \"I161548249\", \"institution_name\": \"Bangor University\", \"country_code\": \"GB\", \"count\": 4398, \"Region\": \"GB\"}, {\"institution_id\": \"I161593684\", \"institution_name\": \"Stockholm University\", \"country_code\": \"SE\", \"count\": 4062, \"Region\": \"RoW\"}, {\"institution_id\": \"I162608824\", \"institution_name\": \"University of Sussex\", \"country_code\": \"GB\", \"count\": 10815, \"Region\": \"GB\"}, {\"institution_id\": \"I165143802\", \"institution_name\": \"University of Queensland\", \"country_code\": \"AU\", \"count\": 5582, \"Region\": \"RoW\"}, {\"institution_id\": \"I165779595\", \"institution_name\": \"University of Melbourne\", \"country_code\": \"AU\", \"count\": 5544, \"Region\": \"RoW\"}, {\"institution_id\": \"I165932596\", \"institution_name\": \"National University of Singapore\", \"country_code\": \"SG\", \"count\": 4421, \"Region\": \"RoW\"}, {\"institution_id\": \"I166337079\", \"institution_name\": \"Queen Mary University of London\", \"country_code\": \"GB\", \"count\": 20942, \"Region\": \"GB\"}, {\"institution_id\": \"I16835326\", \"institution_name\": \"University of St Andrews\", \"country_code\": \"GB\", \"count\": 16898, \"Region\": \"GB\"}, {\"institution_id\": \"I169199633\", \"institution_name\": \"Aston University\", \"country_code\": \"GB\", \"count\": 4319, \"Region\": \"GB\"}, {\"institution_id\": \"I169381384\", \"institution_name\": \"University of Groningen\", \"country_code\": \"NL\", \"count\": 4752, \"Region\": \"RoW\"}, {\"institution_id\": \"I177639307\", \"institution_name\": \"University of Dundee\", \"country_code\": \"GB\", \"count\": 8977, \"Region\": \"GB\"}, {\"institution_id\": \"I17974374\", \"institution_name\": \"Universidade de S\\u00e3o Paulo\", \"country_code\": \"BR\", \"count\": 4217, \"Region\": \"RoW\"}, {\"institution_id\": \"I181647926\", \"institution_name\": \"University of Strathclyde\", \"country_code\": \"GB\", \"count\": 18388, \"Region\": \"GB\"}, {\"institution_id\": \"I183935753\", \"institution_name\": \"King's College London\", \"country_code\": \"GB\", \"count\": 57554, \"Region\": \"GB\"}, {\"institution_id\": \"I184558857\", \"institution_name\": \"Royal Holloway University of London\", \"country_code\": \"GB\", \"count\": 4184, \"Region\": \"GB\"}, {\"institution_id\": \"I184942183\", \"institution_name\": \"University of Oslo\", \"country_code\": \"NO\", \"count\": 4106, \"Region\": \"RoW\"}, {\"institution_id\": \"I185261750\", \"institution_name\": \"University of Toronto\", \"country_code\": \"CA\", \"count\": 6072, \"Region\": \"RoW\"}, {\"institution_id\": \"I187531555\", \"institution_name\": \"Lund University\", \"country_code\": \"SE\", \"count\": 4410, \"Region\": \"RoW\"}, {\"institution_id\": \"I190082696\", \"institution_name\": \"Durham University\", \"country_code\": \"GB\", \"count\": 21910, \"Region\": \"GB\"}, {\"institution_id\": \"I193662353\", \"institution_name\": \"Utrecht University\", \"country_code\": \"NL\", \"count\": 4317, \"Region\": \"RoW\"}, {\"institution_id\": \"I195460627\", \"institution_name\": \"University of Aberdeen\", \"country_code\": \"GB\", \"count\": 11067, \"Region\": \"GB\"}, {\"institution_id\": \"I197940685\", \"institution_name\": \"National Oceanography Centre\", \"country_code\": \"GB\", \"count\": 7116, \"Region\": \"GB\"}, {\"institution_id\": \"I19820366\", \"institution_name\": \"Chinese Academy of Sciences\", \"country_code\": \"CN\", \"count\": 10221, \"Region\": \"RoW\"}, {\"institution_id\": \"I201448701\", \"institution_name\": \"University of Washington\", \"country_code\": \"US\", \"count\": 5184, \"Region\": \"RoW\"}, {\"institution_id\": \"I20231570\", \"institution_name\": \"Peking University\", \"country_code\": \"CN\", \"count\": 4749, \"Region\": \"RoW\"}, {\"institution_id\": \"I204136569\", \"institution_name\": \"The Open University\", \"country_code\": \"GB\", \"count\": 6224, \"Region\": \"GB\"}, {\"institution_id\": \"I204337017\", \"institution_name\": \"Aarhus University\", \"country_code\": \"DK\", \"count\": 5459, \"Region\": \"RoW\"}, {\"institution_id\": \"I204730241\", \"institution_name\": \"Universit\\u00e9 Paris Cit\\u00e9\", \"country_code\": \"FR\", \"count\": 11102, \"Region\": \"RoW\"}, {\"institution_id\": \"I205783295\", \"institution_name\": \"Cornell University\", \"country_code\": \"US\", \"count\": 4834, \"Region\": \"RoW\"}, {\"institution_id\": \"I20581793\", \"institution_name\": \"University of Kent\", \"country_code\": \"GB\", \"count\": 5818, \"Region\": \"GB\"}, {\"institution_id\": \"I21491767\", \"institution_name\": \"Aix-Marseille University\", \"country_code\": \"FR\", \"count\": 4305, \"Region\": \"RoW\"}, {\"institution_id\": \"I223822909\", \"institution_name\": \"Heidelberg University\", \"country_code\": \"DE\", \"count\": 8095, \"Region\": \"RoW\"}, {\"institution_id\": \"I23923803\", \"institution_name\": \"University of Exeter\", \"country_code\": \"GB\", \"count\": 25017, \"Region\": \"GB\"}, {\"institution_id\": \"I241749\", \"institution_name\": \"University of Cambridge\", \"country_code\": \"GB\", \"count\": 115618, \"Region\": \"GB\"}, {\"institution_id\": \"I2738703131\", \"institution_name\": \"Atomic Energy and Alternative Energies Commission\", \"country_code\": \"FR\", \"count\": 10937, \"Region\": \"RoW\"}, {\"institution_id\": \"I277688954\", \"institution_name\": \"University of Paris-Saclay\", \"country_code\": \"FR\", \"count\": 6864, \"Region\": \"RoW\"}, {\"institution_id\": \"I27837315\", \"institution_name\": \"University of Michigan\\u2013Ann Arbor\", \"country_code\": \"US\", \"count\": 6718, \"Region\": \"RoW\"}, {\"institution_id\": \"I2799300731\", \"institution_name\": \"Norwich Research Park\", \"country_code\": \"GB\", \"count\": 8929, \"Region\": \"GB\"}, {\"institution_id\": \"I2799553609\", \"institution_name\": \"Rothamsted Research\", \"country_code\": \"GB\", \"count\": 4924, \"Region\": \"GB\"}, {\"institution_id\": \"I2799725268\", \"institution_name\": \"Manchester Academic Health Science Centre\", \"country_code\": \"GB\", \"count\": 7676, \"Region\": \"GB\"}, {\"institution_id\": \"I2800129641\", \"institution_name\": \"Great Ormond Street Hospital\", \"country_code\": \"GB\", \"count\": 4675, \"Region\": \"GB\"}, {\"institution_id\": \"I2801081054\", \"institution_name\": \"The Francis Crick Institute\", \"country_code\": \"GB\", \"count\": 4540, \"Region\": \"GB\"}, {\"institution_id\": \"I2801237587\", \"institution_name\": \"London Centre for Nanotechnology\", \"country_code\": \"GB\", \"count\": 4604, \"Region\": \"GB\"}, {\"institution_id\": \"I2801748203\", \"institution_name\": \"Hammersmith Hospital\", \"country_code\": \"GB\", \"count\": 4059, \"Region\": \"GB\"}, {\"institution_id\": \"I2801952686\", \"institution_name\": \"Erasmus MC\", \"country_code\": \"NL\", \"count\": 4030, \"Region\": \"RoW\"}, {\"institution_id\": \"I2802476451\", \"institution_name\": \"Wellcome Sanger Institute\", \"country_code\": \"GB\", \"count\": 8525, \"Region\": \"GB\"}, {\"institution_id\": \"I2802711169\", \"institution_name\": \"Diamond Light Source\", \"country_code\": \"GB\", \"count\": 4357, \"Region\": \"GB\"}, {\"institution_id\": \"I2802742124\", \"institution_name\": \"John Radcliffe Hospital\", \"country_code\": \"GB\", \"count\": 6000, \"Region\": \"GB\"}, {\"institution_id\": \"I28166907\", \"institution_name\": \"Karolinska Institutet\", \"country_code\": \"SE\", \"count\": 5859, \"Region\": \"RoW\"}, {\"institution_id\": \"I28290843\", \"institution_name\": \"University of Surrey\", \"country_code\": \"GB\", \"count\": 13832, \"Region\": \"GB\"}, {\"institution_id\": \"I28407311\", \"institution_name\": \"University of Manchester\", \"country_code\": \"GB\", \"count\": 72343, \"Region\": \"GB\"}, {\"institution_id\": \"I31746571\", \"institution_name\": \"UNSW Sydney\", \"country_code\": \"AU\", \"count\": 4330, \"Region\": \"RoW\"}, {\"institution_id\": \"I32062511\", \"institution_name\": \"Heriot-Watt University\", \"country_code\": \"GB\", \"count\": 9355, \"Region\": \"GB\"}, {\"institution_id\": \"I32971472\", \"institution_name\": \"Yale University\", \"country_code\": \"US\", \"count\": 4359, \"Region\": \"RoW\"}, {\"institution_id\": \"I34931013\", \"institution_name\": \"National Institute for Health Research\", \"country_code\": \"GB\", \"count\": 9109, \"Region\": \"GB\"}, {\"institution_id\": \"I35440088\", \"institution_name\": \"ETH Zurich\", \"country_code\": \"CH\", \"count\": 5098, \"Region\": \"RoW\"}, {\"institution_id\": \"I36234482\", \"institution_name\": \"University of Bristol\", \"country_code\": \"GB\", \"count\": 59774, \"Region\": \"GB\"}, {\"institution_id\": \"I36258959\", \"institution_name\": \"University of California, San Diego\", \"country_code\": \"US\", \"count\": 4126, \"Region\": \"RoW\"}, {\"institution_id\": \"I39555362\", \"institution_name\": \"University of Warwick\", \"country_code\": \"GB\", \"count\": 29425, \"Region\": \"GB\"}, {\"institution_id\": \"I39586589\", \"institution_name\": \"Swansea University\", \"country_code\": \"GB\", \"count\": 12320, \"Region\": \"GB\"}, {\"institution_id\": \"I39804081\", \"institution_name\": \"Sorbonne Universit\\u00e9\", \"country_code\": \"FR\", \"count\": 10608, \"Region\": \"RoW\"}, {\"institution_id\": \"I40120149\", \"institution_name\": \"University of Oxford\", \"country_code\": \"GB\", \"count\": 119204, \"Region\": \"GB\"}, {\"institution_id\": \"I40347166\", \"institution_name\": \"University of Chicago\", \"country_code\": \"US\", \"count\": 4084, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210089966\", \"institution_name\": \"London School of Hygiene & Tropical Medicine\", \"country_code\": \"GB\", \"count\": 20290, \"Region\": \"GB\"}, {\"institution_id\": \"I4210092773\", \"institution_name\": \"UK Centre for Ecology & Hydrology\", \"country_code\": \"GB\", \"count\": 5698, \"Region\": \"GB\"}, {\"institution_id\": \"I4210094962\", \"institution_name\": \"Culham Science Centre\", \"country_code\": \"GB\", \"count\": 5401, \"Region\": \"GB\"}, {\"institution_id\": \"I4210105583\", \"institution_name\": \"Royal Observatory\", \"country_code\": \"GB\", \"count\": 4223, \"Region\": \"GB\"}, {\"institution_id\": \"I4210112368\", \"institution_name\": \"National Centre for Atmospheric Science\", \"country_code\": \"GB\", \"count\": 4430, \"Region\": \"GB\"}, {\"institution_id\": \"I4210124175\", \"institution_name\": \"Center for Astrophysics Harvard & Smithsonian\", \"country_code\": \"US\", \"count\": 5975, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210126463\", \"institution_name\": \"UK Dementia Research Institute\", \"country_code\": \"GB\", \"count\": 5605, \"Region\": \"GB\"}, {\"institution_id\": \"I4210128565\", \"institution_name\": \"CEA Saclay\", \"country_code\": \"FR\", \"count\": 6350, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210129701\", \"institution_name\": \"Institute of High Energy Physics\", \"country_code\": \"AT\", \"count\": 7765, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210133362\", \"institution_name\": \"Institut National de Physique Nucl\\u00e9aire et de Physique des Particules\", \"country_code\": \"FR\", \"count\": 5765, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210145102\", \"institution_name\": \"Institut Polytechnique de Paris\", \"country_code\": \"FR\", \"count\": 9144, \"Region\": \"RoW\"}, {\"institution_id\": \"I4210148293\", \"institution_name\": \"MRC Lifecourse Epidemiology Unit\", \"country_code\": \"GB\", \"count\": 4207, \"Region\": \"GB\"}, {\"institution_id\": \"I4210151618\", \"institution_name\": \"National Hospital for Neurology and Neurosurgery\", \"country_code\": \"GB\", \"count\": 10130, \"Region\": \"GB\"}, {\"institution_id\": \"I4210156194\", \"institution_name\": \"Addenbrooke's Hospital\", \"country_code\": \"GB\", \"count\": 5161, \"Region\": \"GB\"}, {\"institution_id\": \"I4210157240\", \"institution_name\": \"Institute of Structural and Molecular Biology\", \"country_code\": \"GB\", \"count\": 4211, \"Region\": \"GB\"}, {\"institution_id\": \"I4210158597\", \"institution_name\": \"MRC Epidemiology Unit\", \"country_code\": \"GB\", \"count\": 7909, \"Region\": \"GB\"}, {\"institution_id\": \"I43439940\", \"institution_name\": \"University of Southampton\", \"country_code\": \"GB\", \"count\": 50887, \"Region\": \"GB\"}, {\"institution_id\": \"I45129253\", \"institution_name\": \"University College London\", \"country_code\": \"GB\", \"count\": 130094, \"Region\": \"GB\"}, {\"institution_id\": \"I47508984\", \"institution_name\": \"Imperial College London\", \"country_code\": \"GB\", \"count\": 106338, \"Region\": \"GB\"}, {\"institution_id\": \"I5023651\", \"institution_name\": \"McGill University\", \"country_code\": \"CA\", \"count\": 4204, \"Region\": \"RoW\"}, {\"institution_id\": \"I51601045\", \"institution_name\": \"University of Bath\", \"country_code\": \"GB\", \"count\": 16430, \"Region\": \"GB\"}, {\"institution_id\": \"I52099693\", \"institution_name\": \"University of York\", \"country_code\": \"GB\", \"count\": 21593, \"Region\": \"GB\"}, {\"institution_id\": \"I56007636\", \"institution_name\": \"Keele University\", \"country_code\": \"GB\", \"count\": 4173, \"Region\": \"GB\"}, {\"institution_id\": \"I56590836\", \"institution_name\": \"Monash University\", \"country_code\": \"AU\", \"count\": 4071, \"Region\": \"RoW\"}, {\"institution_id\": \"I5681781\", \"institution_name\": \"University of Adelaide\", \"country_code\": \"AU\", \"count\": 4579, \"Region\": \"RoW\"}, {\"institution_id\": \"I59433898\", \"institution_name\": \"Brunel University London\", \"country_code\": \"GB\", \"count\": 4926, \"Region\": \"GB\"}, {\"institution_id\": \"I63072094\", \"institution_name\": \"University of Portsmouth\", \"country_code\": \"GB\", \"count\": 4820, \"Region\": \"GB\"}, {\"institution_id\": \"I63098007\", \"institution_name\": \"Liverpool John Moores University\", \"country_code\": \"GB\", \"count\": 4527, \"Region\": \"GB\"}, {\"institution_id\": \"I63966007\", \"institution_name\": \"Massachusetts Institute of Technology\", \"country_code\": \"US\", \"count\": 5401, \"Region\": \"RoW\"}, {\"institution_id\": \"I67311998\", \"institution_name\": \"European Organization for Nuclear Research\", \"country_code\": \"CH\", \"count\": 8627, \"Region\": \"RoW\"}, {\"institution_id\": \"I67415387\", \"institution_name\": \"Lancaster University\", \"country_code\": \"GB\", \"count\": 14770, \"Region\": \"GB\"}, {\"institution_id\": \"I71052956\", \"institution_name\": \"University of Reading\", \"country_code\": \"GB\", \"count\": 11598, \"Region\": \"GB\"}, {\"institution_id\": \"I74801974\", \"institution_name\": \"The University of Tokyo\", \"country_code\": \"JP\", \"count\": 5803, \"Region\": \"RoW\"}, {\"institution_id\": \"I78577930\", \"institution_name\": \"Columbia University\", \"country_code\": \"US\", \"count\": 4428, \"Region\": \"RoW\"}, {\"institution_id\": \"I7882870\", \"institution_name\": \"University of Glasgow\", \"country_code\": \"GB\", \"count\": 40389, \"Region\": \"GB\"}, {\"institution_id\": \"I79510175\", \"institution_name\": \"Cardiff University\", \"country_code\": \"GB\", \"count\": 32638, \"Region\": \"GB\"}, {\"institution_id\": \"I79576946\", \"institution_name\": \"University of Pennsylvania\", \"country_code\": \"US\", \"count\": 4677, \"Region\": \"RoW\"}, {\"institution_id\": \"I79619799\", \"institution_name\": \"University of Birmingham\", \"country_code\": \"GB\", \"count\": 37817, \"Region\": \"GB\"}, {\"institution_id\": \"I8204097\", \"institution_name\": \"Ludwig-Maximilians-Universit\\u00e4t M\\u00fcnchen\", \"country_code\": \"DE\", \"count\": 5630, \"Region\": \"RoW\"}, {\"institution_id\": \"I82284825\", \"institution_name\": \"Cranfield University\", \"country_code\": \"GB\", \"count\": 5671, \"Region\": \"GB\"}, {\"institution_id\": \"I84884186\", \"institution_name\": \"Newcastle University\", \"country_code\": \"GB\", \"count\": 34593, \"Region\": \"GB\"}, {\"institution_id\": \"I87048295\", \"institution_name\": \"Wellcome Trust\", \"country_code\": \"GB\", \"count\": 5765, \"Region\": \"GB\"}, {\"institution_id\": \"I887064364\", \"institution_name\": \"University of Amsterdam\", \"country_code\": \"NL\", \"count\": 5201, \"Region\": \"RoW\"}, {\"institution_id\": \"I899635006\", \"institution_name\": \"Universit\\u00e9 Grenoble Alpes\", \"country_code\": \"FR\", \"count\": 5210, \"Region\": \"RoW\"}, {\"institution_id\": \"I90344618\", \"institution_name\": \"Medical Research Council\", \"country_code\": \"GB\", \"count\": 19123, \"Region\": \"GB\"}, {\"institution_id\": \"I91136226\", \"institution_name\": \"University of Sheffield\", \"country_code\": \"GB\", \"count\": 41995, \"Region\": \"GB\"}, {\"institution_id\": \"I95457486\", \"institution_name\": \"University of California, Berkeley\", \"country_code\": \"US\", \"count\": 8138, \"Region\": \"RoW\"}, {\"institution_id\": \"I97018004\", \"institution_name\": \"Stanford University\", \"country_code\": \"US\", \"count\": 6848, \"Region\": \"RoW\"}, {\"institution_id\": \"I98677209\", \"institution_name\": \"University of Edinburgh\", \"country_code\": \"GB\", \"count\": 68183, \"Region\": \"GB\"}, {\"institution_id\": \"I99464096\", \"institution_name\": \"KU Leuven\", \"country_code\": \"BE\", \"count\": 6631, \"Region\": \"RoW\"}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "institution_counts = (\n",
    "    gtr_institutions.groupby(\n",
    "        [\"institution_id\", \"institution_name\", \"country_code\"]\n",
    "    )\n",
    "    .size()\n",
    "    .reset_index(name=\"count\")\n",
    ")\n",
    "institution_counts[\"Region\"] = institution_counts[\"country_code\"].apply(\n",
    "    lambda x: \"GB\" if x == \"GB\" else \"RoW\"\n",
    ")\n",
    "\n",
    "# keep only institutions with more than 4000 papers/authors\n",
    "institution_counts = institution_counts[institution_counts[\"count\"] > 4000]\n",
    "\n",
    "chart = (\n",
    "    alt.Chart(institution_counts)\n",
    "    .mark_bar()\n",
    "    .encode(\n",
    "        x=alt.X(\"count:Q\", title=\"Number of Papers/Authors\"),\n",
    "        y=alt.Y(\"institution_name:N\", title=\"Institution Name\", sort=\"-x\"),\n",
    "        color=alt.Color(\"Region:N\", title=\"Region\"),\n",
    "        tooltip=[\"institution_name\", \"count\", \"Region\"],\n",
    "    )\n",
    "    .properties(title=\"Distribution of Institutions in GB vs RoW\", height=1000)\n",
    ")\n",
    "\n",
    "chart"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>work_id</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>publication_date</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>W2107167859</td>\n",
       "      <td>Designing logical codon reassignment â€“ Expandi...</td>\n",
       "      <td>Over the last decade, the ability to genetical...</td>\n",
       "      <td>https://doi.org/10.1039/c4sc01534g</td>\n",
       "      <td>2015-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>W2027950306</td>\n",
       "      <td>&lt;scp&gt;RNA&lt;/scp&gt;â€&lt;scp&gt;S&lt;/scp&gt;eq bulked segregant...</td>\n",
       "      <td>The identification of genetic markers linked t...</td>\n",
       "      <td>https://doi.org/10.1111/pbi.12281</td>\n",
       "      <td>2014-11-08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>W2625129035</td>\n",
       "      <td>Validation practices for satelliteâ€based Earth...</td>\n",
       "      <td>Abstract Assessing the inherent uncertainties ...</td>\n",
       "      <td>https://doi.org/10.1002/2017rg000562</td>\n",
       "      <td>2017-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>W2121853553</td>\n",
       "      <td>Iridium-catalysed amine alkylation with alcoho...</td>\n",
       "      <td>Amines have been directly alkylated with alcoh...</td>\n",
       "      <td>https://doi.org/10.1039/b923083a</td>\n",
       "      <td>2010-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>W2766693751</td>\n",
       "      <td>SDSS-IV MaNGA: evidence of the importance of A...</td>\n",
       "      <td>We present new evidence for AGN feedback in a ...</td>\n",
       "      <td>https://doi.org/10.1093/mnras/sty202</td>\n",
       "      <td>2018-01-29</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457154</th>\n",
       "      <td>W2936701156</td>\n",
       "      <td>Ultrasonic Testing of Laboratory Samples Repre...</td>\n",
       "      <td></td>\n",
       "      <td>https://doi.org/10.1784/insi.2019.61.4.187</td>\n",
       "      <td>2019-04-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457155</th>\n",
       "      <td>W3081449123</td>\n",
       "      <td>Decoupled Spatial Distribution of PAHs Degrade...</td>\n",
       "      <td>Abstract Knowing the structure and distributio...</td>\n",
       "      <td>https://doi.org/10.1029/2020jg005659</td>\n",
       "      <td>2020-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457156</th>\n",
       "      <td>W4294024768</td>\n",
       "      <td>SILAC-based quantitative proteomics to investi...</td>\n",
       "      <td>Abstract Background Macrophages play a central...</td>\n",
       "      <td>https://doi.org/10.1186/s12950-022-00309-8</td>\n",
       "      <td>2022-09-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457157</th>\n",
       "      <td>W4252167073</td>\n",
       "      <td>Sensitive and Selective Detection of DNA Fragm...</td>\n",
       "      <td>A colorimetric assay for the detection of DNA ...</td>\n",
       "      <td>https://doi.org/10.26434/chemrxiv.9248825.v2</td>\n",
       "      <td>2020-07-06</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457158</th>\n",
       "      <td>W3212985413</td>\n",
       "      <td>Simulation and Beyond â€“ Principles of Effectiv...</td>\n",
       "      <td>Simulation training provides a safe, non-judgm...</td>\n",
       "      <td>https://doi.org/10.1016/j.bpobgyn.2021.10.004</td>\n",
       "      <td>2021-11-01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>457159 rows Ã— 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "\n",
       "            work_id                                              title  \\\n",
       "\u001b[1;36m0\u001b[0m       W2107167859  Designing logical codon reassignment â€“ Expandi\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m1\u001b[0m       W2027950306  \u001b[1m<\u001b[0m\u001b[1;95mscp\u001b[0m\u001b[39m>RNA<\u001b[0m\u001b[35m/\u001b[0m\u001b[95mscp\u001b[0m\u001b[39m>â€<scp>S<\u001b[0m\u001b[35m/\u001b[0m\u001b[95mscp\u001b[0m\u001b[1m>\u001b[0meq bulked segregant\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m2\u001b[0m       W2625129035  Validation practices for satelliteâ€based Earth\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m3\u001b[0m       W2121853553  Iridium-catalysed amine alkylation with alcoho\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m4\u001b[0m       W2766693751  SDSS-IV MaNGA: evidence of the importance of A\u001b[33m...\u001b[0m   \n",
       "\u001b[33m...\u001b[0m             \u001b[33m...\u001b[0m                                                \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457154\u001b[0m  W2936701156  Ultrasonic Testing of Laboratory Samples Repre\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457155\u001b[0m  W3081449123  Decoupled Spatial Distribution of PAHs Degrade\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457156\u001b[0m  W4294024768  SILAC-based quantitative proteomics to investi\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457157\u001b[0m  W4252167073  Sensitive and Selective Detection of DNA Fragm\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457158\u001b[0m  W3212985413  Simulation and Beyond â€“ Principles of Effectiv\u001b[33m...\u001b[0m   \n",
       "\n",
       "                                                 abstract  \\\n",
       "\u001b[1;36m0\u001b[0m       Over the last decade, the ability to genetical\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m1\u001b[0m       The identification of genetic markers linked t\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m2\u001b[0m       Abstract Assessing the inherent uncertainties \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m3\u001b[0m       Amines have been directly alkylated with alcoh\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m4\u001b[0m       We present new evidence for AGN feedback in a \u001b[33m...\u001b[0m   \n",
       "\u001b[33m...\u001b[0m                                                   \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457154\u001b[0m                                                      \n",
       "\u001b[1;36m457155\u001b[0m  Abstract Knowing the structure and distributio\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457156\u001b[0m  Abstract Background Macrophages play a central\u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457157\u001b[0m  A colorimetric assay for the detection of DNA \u001b[33m...\u001b[0m   \n",
       "\u001b[1;36m457158\u001b[0m  Simulation training provides a safe, non-judgm\u001b[33m...\u001b[0m   \n",
       "\n",
       "                                                  doi publication_date  \n",
       "\u001b[1;36m0\u001b[0m                  \u001b[4;94mhttps://doi.org/10.1039/c4sc01534g\u001b[0m       \u001b[1;36m2015\u001b[0m-\u001b[1;36m01\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m1\u001b[0m                   \u001b[4;94mhttps://doi.org/10.1111/pbi.12281\u001b[0m       \u001b[1;36m2014\u001b[0m-\u001b[1;36m11\u001b[0m-\u001b[1;36m08\u001b[0m  \n",
       "\u001b[1;36m2\u001b[0m                \u001b[4;94mhttps://doi.org/10.1002/2017rg000562\u001b[0m       \u001b[1;36m2017\u001b[0m-\u001b[1;36m09\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m3\u001b[0m                    \u001b[4;94mhttps://doi.org/10.1039/b923083a\u001b[0m       \u001b[1;36m2010\u001b[0m-\u001b[1;36m01\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m4\u001b[0m                \u001b[4;94mhttps://doi.org/10.1093/mnras/sty202\u001b[0m       \u001b[1;36m2018\u001b[0m-\u001b[1;36m01\u001b[0m-\u001b[1;36m29\u001b[0m  \n",
       "\u001b[33m...\u001b[0m                                               \u001b[33m...\u001b[0m              \u001b[33m...\u001b[0m  \n",
       "\u001b[1;36m457154\u001b[0m     \u001b[4;94mhttps://doi.org/10.1784/insi.2019.61.4.187\u001b[0m       \u001b[1;36m2019\u001b[0m-\u001b[1;36m04\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m457155\u001b[0m           \u001b[4;94mhttps://doi.org/10.1029/2020jg005659\u001b[0m       \u001b[1;36m2020\u001b[0m-\u001b[1;36m09\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m457156\u001b[0m     \u001b[4;94mhttps://doi.org/10.1186/s12950-022-00309-8\u001b[0m       \u001b[1;36m2022\u001b[0m-\u001b[1;36m09\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\u001b[1;36m457157\u001b[0m   \u001b[4;94mhttps://doi.org/10.26434/chemrxiv.9248825.v2\u001b[0m       \u001b[1;36m2020\u001b[0m-\u001b[1;36m07\u001b[0m-\u001b[1;36m06\u001b[0m  \n",
       "\u001b[1;36m457158\u001b[0m  \u001b[4;94mhttps://doi.org/10.1016/j.bpobgyn.2021.10.004\u001b[0m       \u001b[1;36m2021\u001b[0m-\u001b[1;36m11\u001b[0m-\u001b[1;36m01\u001b[0m  \n",
       "\n",
       "\u001b[1m[\u001b[0m\u001b[1;36m457159\u001b[0m rows x \u001b[1;36m5\u001b[0m columns\u001b[1m]\u001b[0m"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_publications"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "# merge citations with publications using work_id\n",
    "citations = gtr_citations.copy()\n",
    "\n",
    "citations = citations.merge(gtr_publications, on='work_id', how='left')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "\n",
       "af_direct_citation\n",
       "\u001b[1;36m0\u001b[0m    \u001b[1;36m456773\u001b[0m\n",
       "\u001b[1;36m1\u001b[0m       \u001b[1;36m386\u001b[0m\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_citations.af_direct_citation.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "\n",
       "af_indirect_citations\n",
       "\u001b[1;36m0\u001b[0m     \u001b[1;36m456437\u001b[0m\n",
       "\u001b[1;36m1\u001b[0m        \u001b[1;36m492\u001b[0m\n",
       "\u001b[1;36m2\u001b[0m        \u001b[1;36m119\u001b[0m\n",
       "\u001b[1;36m3\u001b[0m         \u001b[1;36m38\u001b[0m\n",
       "\u001b[1;36m4\u001b[0m         \u001b[1;36m30\u001b[0m\n",
       "\u001b[1;36m5\u001b[0m         \u001b[1;36m13\u001b[0m\n",
       "\u001b[1;36m8\u001b[0m         \u001b[1;36m10\u001b[0m\n",
       "\u001b[1;36m7\u001b[0m          \u001b[1;36m7\u001b[0m\n",
       "\u001b[1;36m13\u001b[0m         \u001b[1;36m3\u001b[0m\n",
       "\u001b[1;36m10\u001b[0m         \u001b[1;36m3\u001b[0m\n",
       "\u001b[1;36m9\u001b[0m          \u001b[1;36m2\u001b[0m\n",
       "\u001b[1;36m6\u001b[0m          \u001b[1;36m2\u001b[0m\n",
       "\u001b[1;36m12\u001b[0m         \u001b[1;36m1\u001b[0m\n",
       "\u001b[1;36m11\u001b[0m         \u001b[1;36m1\u001b[0m\n",
       "\u001b[1;36m16\u001b[0m         \u001b[1;36m1\u001b[0m\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gtr_citations.af_indirect_citations.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a.vega-embed details,\n",
       "  #altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-6d25d63b2f604f6eb66c9cb9dea8e31a\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-9869a8234c631dab1170c0a5a25f7112\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"tooltip\": [{\"field\": \"title\", \"type\": \"nominal\"}, {\"field\": \"af_indirect_citations\", \"type\": \"quantitative\"}], \"x\": {\"field\": \"af_indirect_citations\", \"title\": \"Number of Papers\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"title\", \"sort\": \"-x\", \"title\": \"Paper Title\", \"type\": \"nominal\"}}, \"height\": 6400, \"title\": \"Count of Papers that Indirectly Cite Alphafold\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-9869a8234c631dab1170c0a5a25f7112\": [{\"work_id\": \"W4310959824\", \"doi_x\": \"10.1093/plcell/koac346\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Beyond transcription: compelling open questions in plant RNA biology\", \"abstract\": \"The study of RNAs has become one of the most influential research fields in contemporary biology and biomedicine. In the last few years, new sequencing technologies have produced an explosion of new and exciting discoveries in the field but have also given rise to many open questions. Defining these questions, together with old, long-standing gaps in our knowledge, is the spirit of this article. The breadth of topics within RNA biology research is vast, and every aspect of the biology of these molecules contains countless exciting open questions. Here, we asked 12 groups to discuss their most compelling question among some plant RNA biology topics. The following vignettes cover RNA alternative splicing; RNA dynamics; RNA translation; RNA structures; R-loops; epitranscriptomics; long non-coding RNAs; small RNA production and their functions in crops; small RNAs during gametogenesis and in cross-kingdom RNA interference; and RNA-directed DNA methylation. In each section, we will present the current state-of-the-art in plant RNA biology research before asking the questions that will surely motivate future discoveries in the field. We hope this article will spark a debate about the future perspective on RNA biology and provoke novel reflections in the reader.\", \"doi_y\": \"https://doi.org/10.1093/plcell/koac346\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4315499137\", \"doi_x\": \"10.3389/fpls.2022.1076298\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"\\u03b2-1,4-Xylan backbone synthesis in higher plants: How complex can it be?\", \"abstract\": \"Xylan is a hemicellulose present in the cell walls of all land plants. Glycosyltransferases of the GT43 (IRX9/IRX9L and IRX14/IRX14L) and GT47 (IRX10/IRX10L) families are involved in the biosynthesis of its \\u03b2-1,4-linked xylose backbone, which can be further modified by acetylation and sugar side chains. However, it remains unclear how the different enzymes work together to synthesize the xylan backbone. A xylan synthesis complex (XSC) has been described in the monocots wheat and asparagus, and co-expression of asparagus AoIRX9, AoIRX10 and AoIRX14A is required to form a catalytically active complex for secondary cell wall xylan biosynthesis. Here, we argue that an equivalent XSC exists for the synthesis of the primary cell wall of the eudicot Arabidopsis thaliana, consisting of IRX9L, IRX10L and IRX14. This would suggest the existence of distinct XSCs for primary and secondary cell wall xylan synthesis, reminiscent of the distinct cellulose synthesis complexes (CSCs) of the primary and secondary cell wall. In contrast to the CSC, in which each CESA protein has catalytic activity, the XSC seems to contain proteins with non-catalytic function with each component bearing potentially unique but crucial roles. Moreover, the core XSC formed by a combination of IRX9/IRX9L, IRX10/IRX10L and IRX14/IRX14L might not be stable in its composition during transit from the endoplasmic reticulum to the Golgi apparatus. Instead, potential dynamic changes of the XSC might be a means of regulating xylan biosynthesis to facilitate coordinated deposition of tailored polysaccharides in the plant cell wall.\", \"doi_y\": \"https://doi.org/10.3389/fpls.2022.1076298\", \"publication_date\": \"2023-01-11\"}, {\"work_id\": \"W4307256177\", \"doi_x\": \"10.1101/2022.10.19.512842\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High-resolution promoter interaction analysis in Type 3 Innate Lymphoid Cells implicates Batten Disease gene<i>CLN3</i>in Crohn\\u2019s Disease aetiology\", \"abstract\": \"Abstract Innate lymphoid cells (ILCs) are rare tissue-resident \\u201chelper\\u201d lymphocytes that do not express diversified antigen receptors. Type 3 ILCs (ILC3s) are an important class of these cells enriched in the respiratory and intestinal mucosa, where they regulate inflammation and mucosal homeostasis. To gain insight into the cis-regulatory circuitries underlying ILC3 function, we used high-resolution Capture Hi-C to profile promoter-anchored chromosomal contacts in human primary ILC3s. Combining significant interaction detection with the Activity-By-Contact approach adapted to Capture Hi-C, we reveal a multitude of contacts between promoters and distal regulatory elements and obtain evidence for distinct regulatory wiring of alternative promoters. We find that promoter-interacting regions in ILC3s are enriched for genetic variants associated with multiple immune diseases. Focusing on Crohn\\u2019s disease (CD), in which ILC3s are established mediators, we devised a Bayesian approach that incorporates multivariate fine-mapping to link CD-associated genetic variants with putative target genes. We identify known and previously unimplicated genes in conferring genetic risk of CD through activity in ILC3s. This includes the C LN3 gene that is mutated in most cases of the neurodegenerative disorder Batten disease. Using Cln3 mutant mice, we show that CLN3 is a putative negative regulator of IL-17 production in an inflammatory subset of ILC3s. This finding suggests a functional role for CLN3 in ILC3 biology, with mechanistic implications for Crohn\\u2019s and Batten diseases.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.19.512842\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4319593844\", \"doi_x\": \"10.1038/s42003-023-04488-9\", \"af_direct_citation\": 1, \"af_indirect_citations\": 13, \"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"abstract\": \"Deep-learning (DL) methods like DeepMind's AlphaFold2 (AF2) have led to substantial improvements in protein structure prediction. We analyse confident AF2 models from 21 model organisms using a new classification protocol (CATH-Assign) which exploits novel DL methods for structural comparison and classification. Of ~370,000 confident models, 92% can be assigned to 3253 superfamilies in our CATH domain superfamily classification. The remaining cluster into 2367 putative novel superfamilies. Detailed manual analysis on 618 of these, having at least one human relative, reveal extremely remote homologies and further unusual features. Only 25 novel superfamilies could be confirmed. Although most models map to existing superfamilies, AF2 domains expand CATH by 67% and increases the number of unique 'global' folds by 36% and will provide valuable insights on structure function relationships. CATH-Assign will harness the huge expansion in structural data provided by DeepMind to rationalise evolutionary changes driving functional divergence.\", \"doi_y\": \"https://doi.org/10.1038/s42003-023-04488-9\", \"publication_date\": \"2023-02-08\"}, {\"work_id\": \"W4229373389\", \"doi_x\": \"10.1016/j.str.2022.04.005\", \"af_direct_citation\": 1, \"af_indirect_citations\": 7, \"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"abstract\": \"In the recent Critical Assessment of Structure Prediction (CASP) competition, AlphaFold2 performed outstandingly. Its worst predictions were for nuclear magnetic resonance (NMR) structures, which has two alternative explanations: either the NMR structures were poor, implying that Alpha-Fold may be more accurate than NMR, or there is a genuine difference between crystal and solution structures. Here, we use the program Accuracy of NMR Structures Using RCI and Rigidity (ANSURR), which measures the accuracy of solution structures, and show that one of the NMR structures was indeed poor. We then compare Alpha-Fold predictions to NMR structures and show that Alpha-Fold tends to be more accurate than NMR ensembles. There are, however, some cases where the NMR ensembles are more accurate. These tend to be dynamic structures, where Alpha-Fold had low confidence. We suggest that Alpha-Fold could be used as the model for NMR-structure refinements and that Alpha-Fold structures validated by ANSURR may require no further refinement.\", \"doi_y\": \"https://doi.org/10.1016/j.str.2022.04.005\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W3214131546\", \"doi_x\": \"10.1093/nar/gkab988\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"abstract\": \"Abstract The Protein Data Bank in Europe \\u2013 Knowledge Base (PDBe-KB, https://pdbe-kb.org) is an open collaboration between world-leading specialist data resources contributing functional and biophysical annotations derived from or relevant to the Protein Data Bank (PDB). The goal of PDBe-KB is to place macromolecular structure data in their biological context by developing standardised data exchange formats and integrating functional annotations from the contributing partner resources into a knowledge graph that can provide valuable biological insights. Since we described PDBe-KB in 2019, there have been significant improvements in the variety of available annotation data sets and user functionality. Here, we provide an overview of the consortium, highlighting the addition of annotations such as predicted covalent binders, phosphorylation sites, effects of mutations on the protein structure and energetic local frustration. In addition, we describe a library of reusable web-based visualisation components and introduce new features such as a bulk download data service and a novel superposition service that generates clusters of superposed protein chains weekly for the whole PDB archive.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkab988\", \"publication_date\": \"2021-11-10\"}, {\"work_id\": \"W4224287290\", \"doi_x\": \"10.1016/s1473-3099(22)00141-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Severity of omicron variant of concern and effectiveness of vaccine boosters against symptomatic disease in Scotland (EAVE II): a national cohort study with nested test-negative design\", \"abstract\": \"Since its emergence in November, 2021, in southern Africa, the SARS-CoV-2 omicron variant of concern (VOC) has rapidly spread across the world. We aimed to investigate the severity of omicron and the extent to which booster vaccines are effective in preventing symptomatic infection.In this study, using the Scotland-wide Early Pandemic Evaluation and Enhanced Surveillance of COVID-19 (EAVE II) platform, we did a cohort analysis with a nested test-negative design incident case-control study covering the period Nov 1-Dec 19, 2021, to provide initial estimates of omicron severity and the effectiveness of vaccine boosters against symptomatic disease relative to 25 weeks or more after the second vaccine dose. Primary care data derived from 940 general practices across Scotland were linked to laboratory data and hospital admission data. We compared outcomes between infection with the delta VOC (defined as S-gene positive) and the omicron VOC (defined as S-gene negative). We assessed effectiveness against symptomatic SARS-CoV-2 infection, with infection confirmed through a positive RT-PCR.By Dec 19, 2021, there were 23 840 S-gene-negative cases in Scotland, which were predominantly among those aged 20-39 years (11 732 [49\\u00b72%]). The proportion of S-gene-negative cases that were possible reinfections was more than ten times that of S-gene-positive cases (7\\u00b76% vs 0\\u00b77%; p<0\\u00b70001). There were 15 hospital admissions in S-gene-negative individuals, giving an adjusted observed-to-expected admissions ratio of 0\\u00b732 (95% CI 0\\u00b719-0\\u00b752). The booster vaccine dose was associated with a 57% (54-60) reduction in the risk of symptomatic S-gene-negative infection relative to individuals who tested positive 25 weeks or more after the second vaccine dose.These early national data suggest that omicron is associated with a two-thirds reduction in the risk of COVID-19 hospitalisation compared with delta. Although offering the greatest protection against delta, the booster dose of vaccination offers substantial additional protection against the risk of symptomatic COVID-19 for omicron compared with 25 weeks or more after the second vaccine dose.Health Data Research UK, National Core Studies, Public Health Scotland, Scottish Government, UK Research and Innovation, and University of Edinburgh.\", \"doi_y\": \"https://doi.org/10.1016/s1473-3099(22)00141-4\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4386743501\", \"doi_x\": \"10.1038/s41557-023-01321-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Assembling membraneless organelles from de novo designed proteins\", \"abstract\": \"Abstract Recent advances in de novo protein design have delivered a diversity of discrete de novo protein structures and complexes. A new challenge for the field is to use these designs directly in cells to intervene in biological processes and augment natural systems. The bottom-up design of self-assembled objects such as microcompartments and membraneless organelles is one such challenge. Here we describe the design of genetically encoded polypeptides that form membraneless organelles in Escherichia coli . To do this, we combine de novo \\u03b1-helical sequences, intrinsically disordered linkers and client proteins in single-polypeptide constructs. We tailor the properties of the helical regions to shift protein assembly from arrested assemblies to dynamic condensates. The designs are characterized in cells and in vitro using biophysical methods and soft-matter physics. Finally, we use the designed polypeptide to co-compartmentalize a functional enzyme pair in E. coli , improving product formation close to the theoretical limit.\", \"doi_y\": \"https://doi.org/10.1038/s41557-023-01321-y\", \"publication_date\": \"2023-09-14\"}, {\"work_id\": \"W4385344743\", \"doi_x\": \"10.1007/s40588-023-00195-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Conserved and Divergent Features of pH Sensing in Major Fungal Pathogens\", \"abstract\": \"Abstract Purpose of Review For human fungal pathogens, sensory perception of extracellular pH is essential for colonisation of mammalian tissues and immune evasion. The molecular complexes that perceive and transmit the fungal pH signal are membrane-proximal and essential for virulence and are therefore of interest as novel antifungal drug targets. Intriguingly, the sensory machinery has evolved divergently in different fungal pathogens, yet spatial co-ordination of cellular components is conserved. Recent Findings The recent discovery of a novel pH sensor in the basidiomycete pathogen Cryptococcus neformans highlights that, although the molecular conservation of fungal pH sensors is evolutionarily restricted, their subcellular localisation and coupling to essential components of the cellular ESCRT machinery are consistent features of the cellular pH sensing and adaptation mechanism. In both basidiomycetes and ascomycetes, the lipid composition of the plasma membrane to which pH sensing complexes are localised appears to have pivotal functional importance. Endocytosis of pH-sensing complexes occurs in multiple fungal species, but its relevance for signal transduction appears not to be universal. Summary Our overview of current understanding highlights conserved and divergent mechanisms of the pH sensing machinery in model and pathogenic fungal species, as well as important unanswered questions that must be addressed to inform the future study of such sensing mechanisms and to devise therapeutic strategies for manipulating them.\", \"doi_y\": \"https://doi.org/10.1007/s40588-023-00195-5\", \"publication_date\": \"2023-07-28\"}, {\"work_id\": \"W4280617578\", \"doi_x\": \"10.1016/j.cell.2022.05.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Potent cross-reactive antibodies following Omicron breakthrough in vaccinees\", \"abstract\": \"Highly transmissible Omicron variants of SARS-CoV-2 currently dominate globally. Here, we compare neutralization of Omicron BA.1, BA.1.1, and BA.2. BA.2 RBD has slightly higher ACE2 affinity than BA.1 and slightly reduced neutralization by vaccine serum, possibly associated with its increased transmissibility. Neutralization differences between sub-lineages for mAbs (including therapeutics) mostly arise from variation in residues bordering the ACE2 binding site; however, more distant mutations S371F (BA.2) and R346K (BA.1.1) markedly reduce neutralization by therapeutic antibody Vir-S309. In-depth structure-and-function analyses of 27 potent RBD-binding mAbs isolated from vaccinated volunteers following breakthrough Omicron-BA.1 infection reveals that they are focused in two main clusters within the RBD, with potent right-shoulder antibodies showing increased prevalence. Selection and somatic maturation have optimized antibody potency in less-mutated epitopes and recovered potency in highly mutated epitopes. All 27 mAbs potently neutralize early pandemic strains, and many show broad reactivity with variants of concern.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.05.014\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4281643745\", \"doi_x\": \"10.1111/tpj.15856\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"ROS production and signalling in chloroplasts: cornerstones and evolving concepts\", \"abstract\": \"SUMMARY Reactive oxygen species (ROS) such as singlet oxygen, superoxide (O 2 \\u25cf\\u2212 ) and hydrogen peroxide (H 2 O 2 ) are the markers of living cells. Oxygenic photosynthesis produces ROS in abundance, which act as a readout of a functional electron transport system and metabolism. The concept that photosynthetic ROS production is a major driving force in chloroplast to nucleus retrograde signalling is embedded in the literature, as is the role of chloroplasts as environmental sensors. The different complexes and components of the photosynthetic electron transport chain (PETC) regulate O 2 \\u25cf\\u2212 production in relation to light energy availability and the redox state of the stromal Cys\\u2010based redox systems. All of the ROS generated in chloroplasts have the potential to act as signals and there are many sulphhydryl\\u2010containing proteins and peptides in chloroplasts that have the potential to act as H 2 O 2 sensors and function in signal transduction. While ROS may directly move out of the chloroplasts to other cellular compartments, ROS signalling pathways can only be triggered if appropriate ROS\\u2010sensing proteins are present at or near the site of ROS production. Chloroplast antioxidant systems serve either to propagate these signals or to remove excess ROS that cannot effectively be harnessed in signalling. The key challenge is to understand how regulated ROS delivery from the PETC to the Cys\\u2010based redox machinery is organised to transmit redox signals from the environment to the nucleus. Redox changes associated with stromal carbohydrate metabolism also play a key role in chloroplast signalling pathways.\", \"doi_y\": \"https://doi.org/10.1111/tpj.15856\", \"publication_date\": \"2022-06-28\"}, {\"work_id\": \"W4293567428\", \"doi_x\": \"10.1107/s2059798322007987\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"abstract\": \"Nowadays, progress in the determination of three-dimensional macromolecular structures from diffraction images is achieved partly at the cost of increasing data volumes. This is due to the deployment of modern high-speed, high-resolution detectors, the increased complexity and variety of crystallographic software, the use of extensive databases and high-performance computing. This limits what can be accomplished with personal, offline, computing equipment in terms of both productivity and maintainability. There is also an issue of long-term data maintenance and availability of structure-solution projects as the links between experimental observations and the final results deposited in the PDB. In this article, CCP4 Cloud, a new front-end of the CCP4 software suite, is presented which mitigates these effects by providing an online, cloud-based environment for crystallographic computation. CCP4 Cloud was developed for the efficient delivery of computing power, database services and seamless integration with web resources. It provides a rich graphical user interface that allows project sharing and long-term storage for structure-solution projects, and can be linked to data-producing facilities. The system is distributed with the CCP4 software suite version 7.1 and higher, and an online publicly available instance of CCP4 Cloud is provided by CCP4.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322007987\", \"publication_date\": \"2022-08-30\"}, {\"work_id\": \"W4308593405\", \"doi_x\": \"10.3389/fmolb.2022.970391\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"abstract\": \"Human equilibrative nucleoside transporters represent a major pharmaceutical target for cardiac, cancer and viral therapies. Understanding the molecular basis for transport is crucial for the development of improved therapeutics through structure-based drug design. ENTs have been proposed to utilise an alternating access mechanism of action, similar to that of the major facilitator superfamily. However, ENTs lack functionally-essential features of that superfamily, suggesting that they may use a different transport mechanism. Understanding the molecular basis of their transport requires insight into diverse conformational states. Differences between intermediate states may be discrete and mediated by subtle gating interactions, such as salt bridges. We identified four variants of human equilibrative nucleoside transporter isoform 1 (hENT1) at the large intracellular loop (ICL6) and transmembrane helix 7 (TM7) that stabilise the apo -state (\\u2206T m 0.7\\u20131.5\\u00b0C). Furthermore, we showed that variants K263A (ICL6) and I282V (TM7) specifically stabilise the inhibitor-bound state of hENT1 (\\u2206\\u2206T m 5.0 \\u00b1 1.7\\u00b0C and 3.0 \\u00b1 1.8\\u00b0C), supporting the role of ICL6 in hENT1 gating. Finally, we showed that, in comparison with wild type, variant T336A is de stabilised by nitrobenzylthioinosine (\\u2206\\u2206T m -4.7 \\u00b1 1.1\\u00b0C) and binds it seven times worse. This residue may help determine inhibitor and substrate sensitivity. Residue K263 is not present in the solved structures, highlighting the need for further structural data that include the loop regions.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.970391\", \"publication_date\": \"2022-11-08\"}, {\"work_id\": \"W4293877662\", \"doi_x\": \"10.1016/j.csbj.2022.08.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Streptomyces coelicolor macrodomain hydrolase SCO6735 cleaves thymidine-linked ADP-ribosylation of DNA\", \"abstract\": \"ADP-ribosylation is an ancient, highly conserved, and reversible covalent modification critical for a variety of endogenous processes in both prokaryotes and eukaryotes. ADP-ribosylation targets proteins, nucleic acids, and small molecules (including antibiotics). ADP-ribosylation signalling involves enzymes that add ADP-ribose to the target molecule, the (ADP-ribosyl)transferases; and those that remove it, the (ADP-ribosyl)hydrolases. Recently, the toxin/antitoxin pair DarT/DarG composed of a DNA ADP-ribosylating toxin, DarT, and (ADP-ribosyl)hydrolase antitoxin, DarG, was described. DarT modifies thymidine in single-stranded DNA in a sequence-specific manner while DarG reverses this modification, thereby rescuing cells from DarT toxicity. We studied the DarG homologue SCO6735 which is highly conserved in all Streptomyces species and known to be associated with antibiotic production in the bacterium S. coelicolor. SCO6735 shares a high structural similarity with the bacterial DarG and human TARG1. Like DarG and TARG1, SCO6735 can also readily reverse thymidine-linked ADP-ribosylation catalysed by DarT in vitro and in cells. SCO6735 active site analysis including molecular dynamic simulations of its complex with ADP-ribosylated thymidine suggests a novel catalytic mechanism of DNA-(ADP-ribose) hydrolysis. Moreover, a comparison of SCO6735 structure with ALC1-like homologues revealed an evolutionarily conserved feature characteristic for this subclass of macrodomain hydrolases.\", \"doi_y\": \"https://doi.org/10.1016/j.csbj.2022.08.002\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4322154584\", \"doi_x\": \"10.1101/2023.02.23.23286106\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>IFT74</i>variants cause skeletal ciliopathy and motile cilia defects in mice and humans\", \"abstract\": \"Abstract Motile and non-motile cilia are critical to mammalian development and health. Assembly of these organelles depends on proteins synthesized in the cell body and transported into the cilium by intraflagellar transport (IFT). A series of human and mouse IFT74 variants were studied to understand the function of this IFT subunit. Humans missing exon 2, which codes for the first 40 residues, presented an unusual combination of ciliary chondrodysplasia and mucociliary clearance disorders while individuals carrying biallelic splice site variants developed a lethal skeletal chondrodysplasia. In mice, variants thought to remove all Ift74 function, completely block ciliary assembly and result in midgestational lethality. A mouse allele that removes the first 40 amino acids, analogous to the human exon 2 deletion, results in a motile cilia phenotype with mild skeletal abnormalities. In vitro studies suggest that the first 40 amino acids of IFT74 are dispensable for binding of other IFT subunits but are important for tubulin binding. Higher demands on tubulin transport in motile cilia compared to primary cilia could account for the motile cilia phenotype observed in human and mice.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.23.23286106\", \"publication_date\": \"2023-02-26\"}, {\"work_id\": \"W4297494206\", \"doi_x\": \"10.1002/pro.4439\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"abstract\": \"Abstract The archiving and dissemination of protein and nucleic acid structures as well as their structural, functional and biophysical annotations is an essential task that enables the broader scientific community to conduct impactful research in multiple fields of the life sciences. The Protein Data Bank in Europe (PDBe; pdbe.org ) team develops and maintains several databases and web services to address this fundamental need. From data archiving as a member of the Worldwide PDB consortium (wwPDB; wwpdb.org ), to the PDBe Knowledge Base (PDBe\\u2010KB; pdbekb.org ), we provide data, data\\u2010access mechanisms, and visualizations that facilitate basic and applied research and education across the life sciences. Here, we provide an overview of the structural data and annotations that we integrate and make freely available. We describe the web services and data visualization tools we offer, and provide information on how to effectively use or even further develop them. Finally, we discuss the direction of our data services, and how we aim to tackle new challenges that arise from the recent, unprecedented advances in the field of structure determination and protein structure modeling.\", \"doi_y\": \"https://doi.org/10.1002/pro.4439\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W4319788261\", \"doi_x\": \"10.1080/17469899.2023.2175672\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrating artificial intelligence into an ophthalmologist\\u2019s workflow: obstacles and opportunities\", \"abstract\": \"Introduction Demand in clinical services within the field of ophthalmology is predicted to rise over the future years. Artificial intelligence, in particular, machine learning-based systems, have demonstrated significant potential in optimizing medical diagnostics, predictive analysis, and management of clinical conditions. Ophthalmology has been at the forefront of this digital revolution, setting precedents for integration of these systems into clinical workflows.Areas covered This review discusses integration of machine learning tools within ophthalmology clinical practices. We discuss key issues around ethical consideration, regulation, and clinical governance. We also highlight challenges associated with clinical adoption, sustainability, and discuss the importance of interoperability.Expert opinion Clinical integration is considered one of the most challenging stages within the implementation process. Successful integration necessitates a collaborative approach from multiple stakeholders around a structured governance framework, with emphasis on standardization across healthcare providers and equipment and software developers.\", \"doi_y\": \"https://doi.org/10.1080/17469899.2023.2175672\", \"publication_date\": \"2023-01-02\"}, {\"work_id\": \"W4380987458\", \"doi_x\": \"10.1016/j.bbadva.2023.100096\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Near atmospheric carbon dioxide activates plant ubiquitin cross-linking\", \"abstract\": \"Background: Identifying CO2-binding proteins is vital for our knowledge of CO2-regulated molecular processes. The carbamate post-translational modification is a reversible CO2-mediated adduct that can form on neutral N-terminal \\u03b1-amino or lysine \\u03b5-amino groups. Methods: We have developed triethyloxonium ion (TEO) as a chemical proteomics tool to trap the carbamate post-translational modification on protein covalently. We use 13C-NMR and TEO and identify ubiquitin as a plant CO2-binding protein. Results: We observe the carbamate post-translational modification on the Arabidopsis thaliana ubiquitin N-terminus and \\u03b5-amino groups of lysines 6, 33, and 48. We show that biologically relevant near atmospheric PCO2 levels increase ubiquitin conjugation dependent on lysine 6. We further demonstrate that CO2 increases the ubiquitin E2 ligase (AtUBC5) charging step via the transthioesterification reaction in which Ub is transferred from the E1 ligase active site to the E2 active site. Conclusions and General Significance: Therefore, plant ubiquitin is a CO2-binding protein, and the carbamate post-translational modification represents a potential mechanism through which plant cells can respond to fluctuating PCO2.\", \"doi_y\": \"https://doi.org/10.1016/j.bbadva.2023.100096\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4315853020\", \"doi_x\": \"10.1088/1361-6420/acb2ba\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"abstract\": \"We consider the problem of recovering the three-dimensional atomic structure of a flexible macromolecule from a heterogeneous cryo-EM dataset. The dataset contains noisy tomographic projections of the electrostatic potential of the macromolecule, taken from different viewing directions, and in the heterogeneous case, each image corresponds to a different conformation of the macromolecule. Under the assumption that the macromolecule can be modelled as a chain, or discrete curve (as it is for instance the case for a protein backbone with a single chain of amino-acids), we introduce a method to estimate the deformation of the atomic model with respect to a given conformation, which is assumed to be known a priori. Our method consists on estimating the torsion and bond angles of the atomic model in each conformation as a linear combination of the eigenfunctions of the Laplace operator in the manifold of conformations. These eigenfunctions can be approximated by means of a well-known technique in manifold learning, based on the construction of a graph Laplacian using the cryo-EM dataset. Finally, we test our approach with synthetic datasets, for which we recover the atomic model of two-dimensional and three-dimensional flexible structures from noisy tomographic projections.\", \"doi_y\": \"https://doi.org/10.1088/1361-6420/acb2ba\", \"publication_date\": \"2023-01-30\"}, {\"work_id\": \"W4210908551\", \"doi_x\": \"10.1101/2022.02.07.479382\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Pitfalls of machine learning models for protein-protein interactions\", \"abstract\": \"Abstract Protein-protein interactions (PPIs) are essential to understanding biological pathways as well as their roles in development and disease. Computational tools, based on classic machine learning, have been successful at predicting PPIs in silico , but the lack of consistent and reliable frameworks for this task has led to network models that are difficult to compare and discrepancies between algorithms that remain unexplained. To better understand the underlying inference mechanisms that underpin these models, we designed an open-source framework for benchmarking that accounts for a range of biological and statistical pitfalls while facilitating reproducibility. We use it to shed light on the impact of network topology and how different algorithms deal with highly connected proteins. By studying functional genomics-based and sequence-based models on human PPIs, we show their complementarity as the former performs best on lone proteins while the latter specialises in interactions involving hubs. We also show that algorithm design has little impact on performance with functional genomic data. We replicate our results between both human and S. cerevisiae data and demonstrate that models using functional genomics are better suited to PPI prediction across species. With rapidly increasing amounts of sequence and functional genomics data, our study provides a principled foundation for future construction, comparison and application of PPI networks.\", \"doi_y\": \"https://doi.org/10.1101/2022.02.07.479382\", \"publication_date\": \"2022-02-08\"}, {\"work_id\": \"W3173575423\", \"doi_x\": \"10.1002/prot.26214\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"abstract\": \"The assessment of CASP models for utility in molecular replacement is a measure of their use in a valuable real-world application. In CASP7, the metric for molecular replacement assessment involved full likelihood-based molecular replacement searches; however, this restricted the assessable targets to crystal structures with only one copy of the target in the asymmetric unit, and to those where the search found the correct pose. In CASP10, full molecular replacement searches were replaced by likelihood-based rigid-body refinement of models superimposed on the target using the LGA algorithm, with the metric being the refined log-likelihood-gain (LLG) score. This enabled multi-copy targets and very poor models to be evaluated, but a significant further issue remained: the requirement of diffraction data for assessment. We introduce here the relative-expected-LLG (reLLG), which is independent of diffraction data. This reLLG is also independent of any crystal form, and can be calculated regardless of the source of the target, be it X-ray, NMR or cryo-EM. We calibrate the reLLG against the LLG for targets in CASP14, showing that it is a robust measure of both model and group ranking. Like the LLG, the reLLG shows that accurate coordinate error estimates add substantial value to predicted models. We find that refinement by CASP groups can often convert an inadequate initial model into a successful MR search model. Consistent with findings from others, we show that the AlphaFold2 models are sufficiently good, and reliably so, to surpass other current model generation strategies for attempting molecular replacement phasing.\", \"doi_y\": \"https://doi.org/10.1002/prot.26214\", \"publication_date\": \"2021-08-21\"}, {\"work_id\": \"W4319310661\", \"doi_x\": \"10.1038/s43588-022-00391-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High-throughput property-driven generative design of functional organic molecules\", \"abstract\": \"The design of molecules and materials with tailored properties is challenging, as candidate molecules must satisfy multiple competing requirements that are often difficult to measure or compute. While molecular structures, produced through generative deep learning, will satisfy those patterns, they often only possess specific target properties by chance and not by design, which makes molecular discovery via this route inefficient. In this work, we predict molecules with (pareto)-optimal properties by combining a generative deep learning model that predicts three dimensional conformations of molecules with a supervised deep learning model that takes these as inputs and predicts their electronic structure. Optimization of (multiple) molecular properties is achieved by screening newly generated molecules for desirable electronic properties and reusing hit molecules to retrain the generative model with a bias. The approach is demonstrated to find optimal molecules for organic electronics applications. Our method is generally applicable and eliminates the need for quantum chemical calculations during predictions, making it suitable for high-throughput screening in materials and catalyst design.\", \"doi_y\": \"https://doi.org/10.1038/s43588-022-00391-1\", \"publication_date\": \"2023-02-06\"}, {\"work_id\": \"W4309271418\", \"doi_x\": \"10.1111/1462-2920.16280\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Characterization of the molecular mechanisms of silicon uptake in coccolithophores\", \"abstract\": \"Coccolithophores are an important group of calcifying marine phytoplankton. Although coccolithophores are not silicified, some species exhibit a requirement for Si in the calcification process. These species also possess a novel protein (SITL) that resembles the SIT family of Si transporters found in diatoms. However, the nature of Si transport in coccolithophores is not yet known, making it difficult to determine the wider role of Si in coccolithophore biology. Here, we show that coccolithophore SITLs act as Na+ -coupled Si transporters when expressed in heterologous systems and exhibit similar characteristics to diatom SITs. We find that CbSITL from Coccolithus braarudii is transcriptionally regulated by Si availability and is expressed in environmental coccolithophore populations. However, the Si requirement of C. braarudii and other coccolithophores is very low, with transport rates of exogenous Si below the level of detection in sensitive assays of Si transport. As coccoliths contain only low levels of Si, we propose that Si acts to support the calcification process, rather than forming a structural component of the coccolith itself. Si is therefore acting as a micronutrient in coccolithophores and natural populations are only likely to experience Si limitation in circumstances where dissolved silicon (DSi) is depleted to extreme levels.\", \"doi_y\": \"https://doi.org/10.1111/1462-2920.16280\", \"publication_date\": \"2022-11-22\"}, {\"work_id\": \"W4312526408\", \"doi_x\": \"10.1093/bioadv/vbac087\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"OAF: a new member of the BRICHOS family\", \"abstract\": \"The 10 known BRICHOS domain-containing proteins in humans have been linked to an unusually long list of pathologies, including cancer, obesity and two amyloid-like diseases. BRICHOS domains themselves have been described as intramolecular chaperones that act to prevent amyloid-like aggregation of their proteins' mature polypeptides. Using structural comparison of coevolution-based AlphaFold models and sequence conservation, we identified the Out at First (OAF) protein as a new member of the BRICHOS family in humans. OAF is an experimentally uncharacterized protein that has been proposed as a candidate biomarker for clinical management of coronavirus disease 2019 infections. Our analysis revealed how structural comparison of AlphaFold models can discover remote homology relationships and lead to a better understanding of BRICHOS domain molecular mechanism.Supplementary data are available at Bioinformatics Advances online.\", \"doi_y\": \"https://doi.org/10.1093/bioadv/vbac087\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4378902334\", \"doi_x\": \"10.1016/j.molimm.2023.05.006\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Crystal structures of the human IgD Fab reveal insights into CH1 domain diversity\", \"abstract\": \"Antibodies of the IgD isotype remain the least well characterized of the mammalian immunoglobulin isotypes. Here we report three-dimensional structures for the Fab region of IgD, based on four different crystal structures, at resolutions of 1.45\\u20132.75 \\u00c5. These IgD Fab crystals provide the first high-resolution views of the unique C\\u03b41 domain. Structural comparisons identify regions of conformational diversity within the C\\u03b41 domain, as well as among the homologous domains of C\\u03b11, C\\u03b31 and C\\u03bc1. The IgD Fab structure also possesses a unique conformation of the upper hinge region, which may contribute to the overall disposition of the very long linker sequence between the Fab and Fc regions found in human IgD. Structural similarities observed between IgD and IgG, and differences with IgA and IgM, are consistent with predicted evolutionary relationships for the mammalian antibody isotypes.\", \"doi_y\": \"https://doi.org/10.1016/j.molimm.2023.05.006\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4319657030\", \"doi_x\": \"10.1016/j.sbi.2022.102524\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mucin networks: Dynamic structural assemblies controlling mucus function\", \"abstract\": \"Contrary to first appearances, mucus structural biology is not an oxymoron. Though mucus hydrogels derive their characteristics largely from intrinsically disordered, heavily glycosylated polypeptide segments, the secreted mucin glycoproteins that constitute mucus undergo an orderly assembly process controlled by folded domains at their termini. Recent structural studies revealed how mucin complexes promote disulphide-mediated polymerization to produce the mucus gel scaffold. Additional protein-protein and protein-glycan interactions likely tune the mesoscale properties, stability, and activities of mucins. Evidence is emerging that even intrinsically disordered glycosylated segments have specific structural roles in the production and properties of mucus. Though soft-matter biophysical approaches to understanding mucus remain highly relevant, high-resolution structural studies of mucins and other mucus components are providing new perspectives on these vital, protective hydrogels.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2022.102524\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4313889417\", \"doi_x\": \"10.1101/2023.01.06.23284281\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Estimating clinical risk in gene regions from population sequencing cohort data\", \"abstract\": \"While pathogenic variants significantly increase disease risk in many genes, it is still challenging to estimate the clinical impact of rare missense variants more generally. Even in genes such as BRCA2 or PALB2 , large cohort studies find no significant association between breast cancer and rare germline missense variants collectively. Here we introduce REGatta, a method to improve the estimation of clinical risk in gene segments. We define gene regions using the density of pathogenic diagnostic reports, and then calculate the relative risk in each of these regions using 109,581 exome sequences from women in the UK Biobank. We apply this method in seven established breast cancer genes, and identify regions in each gene with statistically significant differences in breast cancer incidence for rare missense carriers. Even in genes with no significant difference at the gene level, this approach significantly separates rare missense variant carriers at higher or lower risk ( BRCA2 regional model OR=1.46 [1.12, 1.79], p=0.0036 vs. BRCA2 gene model OR=0.96 [0.85,1.07] p=0.4171). We find high concordance between these regional risk estimates and high-throughput functional assays of variant impact. We compare with existing methods and the use of protein domains (Pfam) as regions, and find REGatta better identifies individuals at elevated or reduced risk. These regions provide useful priors which can potentially be used to improve risk assessment and clinical management.\", \"doi_y\": \"https://doi.org/10.1101/2023.01.06.23284281\", \"publication_date\": \"2023-01-09\"}, {\"work_id\": \"W4361801564\", \"doi_x\": \"10.1039/d3ra01708g\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mechanoenzymatic reactions for the hydrolysis of PET\", \"abstract\": \"Recent advances in the enzymatic degradation of poly(ethylene terphthalate) (PET) have led to a number of PET hydrolytic enzymes and mutants being developed. With the amount of PET building up in the natural world, there is a pressing need to develop scalable methods of breaking down the polymer into its monomers for recycling or other uses. Mechanoenzymatic reactions have gained traction recently as a green and efficient alternative to traditional biocatalytic reactions. For the first time we report increased yields of PET degradation by whole cell PETase enzymes by up to 27-fold by utilising ball milling cycles of reactive aging, when compared with typical solution-based reactions. This methodology leads to up to a 2600-fold decrease in the solvent required when compared with other leading degradation reactions in the field and a 30-fold decrease in comparison to reported industrial scale PET hydrolysis reactions.\", \"doi_y\": \"https://doi.org/10.1039/d3ra01708g\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4309522729\", \"doi_x\": \"10.1101/2022.11.21.517365\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>arfA</i>antisense RNA regulates MscL excretory activity\", \"abstract\": \"Abstract Bacteria adapt to acute changes in their environment by processing multiple input stimuli through signal integration and crosstalk to allow fine tuning of gene expression in response to stress. The response to hypoosmotic shock and ribosome stalling occurs through the action of mechanosensitive channels and ribosome rescue mechanisms respectively. However, it is not known if a mechanistic link exists between these stress response pathways. Here we report that the corresponding Large-conductance mechanosensitive channel ( mscL ) and Alternative ribosome-rescue factor A ( arfA ) genes are commonly co-located on the genomes of Gammaproteobacteria and display overlap in their respective 3\\u2019 UTR and 3\\u2019 CDS. We show this unusual genomic arrangement permits an antisense RNA mediated regulatory control between mscL and arfA and this modulates MscL excretory activity in E. coli . These findings highlight a mechanistic link between osmotic and translational stress responses in E. coli , and further elucidates the previously unknown regulatory function of arfA sRNA.\", \"doi_y\": \"https://doi.org/10.1101/2022.11.21.517365\", \"publication_date\": \"2022-11-21\"}, {\"work_id\": \"W4317528588\", \"doi_x\": \"10.1016/j.ijpharm.2023.122628\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Machine learning using multi-modal data predicts the production of selective laser sintered 3D printed drug products\", \"abstract\": \"Three-dimensional (3D) printing is drastically redefining medicine production, offering digital precision and personalized design opportunities. One emerging 3D printing technology is selective laser sintering (SLS), which is garnering attention for its high precision, and compatibility with a wide range of pharmaceutical materials, including low-solubility compounds. However, the full potential of SLS for medicines is yet to be realized, requiring expertise and considerable time-consuming and resource-intensive trial-and-error research. Machine learning (ML), a subset of artificial intelligence, is an in silico tool that is accomplishing remarkable breakthroughs in several sectors for its ability to make highly accurate predictions. Therefore, the present study harnessed ML to predict the printability of SLS formulations. Using a dataset of 170 formulations from 78 materials, ML models were developed from inputs that included the formulation composition and characterization data retrieved from Fourier-transformed infrared spectroscopy (FT-IR), X-ray powder diffraction (XRPD) and differential scanning calorimetry (DSC). Multiple ML models were explored, including supervised and unsupervised approaches. The results revealed that ML can achieve high accuracies, by using the formulation composition leading to a maximum F1 score of 81.9%. Using the FT-IR, XRPD and DSC data as inputs resulted in an F1 score of 84.2%, 81.3%, and 80.1%, respectively. A subsequent ML pipeline was built to combine the predictions from FT-IR, XRPD and DSC into one consensus model, where the F1 score was found to further increase to 88.9%. Therefore, it was determined for the first time that ML predictions of 3D printability benefit from multi-modal data, combining numeric, spectral, thermogram and diffraction data. The study lays the groundwork for leveraging existing characterization data for developing high-performing computational models to accelerate formulation development.\", \"doi_y\": \"https://doi.org/10.1016/j.ijpharm.2023.122628\", \"publication_date\": \"2023-02-01\"}, {\"work_id\": \"W4221027706\", \"doi_x\": \"10.1093/molbev/msac061\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Selection Analysis Identifies Clusters of Unusual Mutational Changes in Omicron Lineage BA.1 That Likely Impact Spike Function\", \"abstract\": \"Among the 30 nonsynonymous nucleotide substitutions in the Omicron S-gene are 13 that have only rarely been seen in other SARS-CoV-2 sequences. These mutations cluster within three functionally important regions of the S-gene at sites that will likely impact (1) interactions between subunits of the Spike trimer and the predisposition of subunits to shift from down to up configurations, (2) interactions of Spike with ACE2 receptors, and (3) the priming of Spike for membrane fusion. We show here that, based on both the rarity of these 13 mutations in intrapatient sequencing reads and patterns of selection at the codon sites where the mutations occur in SARS-CoV-2 and related sarbecoviruses, prior to the emergence of Omicron the mutations would have been predicted to decrease the fitness of any virus within which they occurred. We further propose that the mutations in each of the three clusters therefore cooperatively interact to both mitigate their individual fitness costs, and, in combination with other mutations, adaptively alter the function of Spike. Given the evident epidemic growth advantages of Omicron overall previously known SARS-CoV-2 lineages, it is crucial to determine both how such complex and highly adaptive mutation constellations were assembled within the Omicron S-gene, and why, despite unprecedented global genomic surveillance efforts, the early stages of this assembly process went completely undetected.\", \"doi_y\": \"https://doi.org/10.1093/molbev/msac061\", \"publication_date\": \"2022-03-24\"}, {\"work_id\": \"W4316654730\", \"doi_x\": \"10.1016/j.molmet.2023.101674\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Thioalbamide inhibits FoF1-ATPase in breast cancer cells and reduces tumor proliferation and invasiveness in breast cancer in vivo models\", \"abstract\": \"Thioalbamide is a ribosomally synthesized and post-translationally modified peptide (RiPP) belonging to the family of thioamitides, a rare class of microbial specialized metabolites with unusual post-translational modifications and promising biological activities. Recent studies have demonstrated the ability of thioalbamide to exert highly selective cytotoxic effects on tumor cells by affecting their energy metabolism, thus causing abnormal ROS production and triggering apoptosis. This study is aimed to investigate the molecular mechanisms underlying the antitumor activity of thioalbamide in order to identify its exact molecular target. Wild type MCF-7 and MDA-MB-231 breast cancer cell lines as well as cancer cells deprived of mitochondrial DNA (\\u03c10 cells) were employed in order to assess thioalbamide effects on tumor bioenergetics. In this regard, metabolic profile was evaluated by a Seahorse XFe96 analyzer, and the activity of the enzyme complexes involved in oxidative phosphorylation was quantified by spectrophotometric assays. Thioalbamide effects on tumor invasiveness were assessed by gelatin zymography experiments and invasion assays. In vivo experiments were carried out on breast cancer xenograft and \\u201cexperimental metastasis\\u201d mouse models. Experiments carried out on \\u03c10 breast cancer cells, together with Seahorse analysis and the application of spectrophotometric enzymatic assays, highlighted the ability of thioalbamide to affect the mitochondrial respiration process, and allowed to propose the FoF1-ATPase complex as its main molecular target in breast cancer cells. Additionally, thioalbamide-mediated OXPHOS inhibition was shown, for the first time, to reduce tumor invasiveness by inhibiting metalloproteinase-9 secretion. Furthermore, this study has confirmed the antitumor potential of thioalbamide in two different in vivo models. In particular, experiments on MCF-7 and MDA-MB-231 xenograft mouse models have confirmed in vivo its high anti-proliferative and pro-apoptotic activity, while experiments on MDA-MB-231 \\u2033experimental metastasis\\u201d mouse models have highlighted its ability to inhibit breast cancer cell invasiveness. Overall, our results shed more light on the molecular mechanisms underlying the pharmacological potential of thioamidated peptides, thus reducing the gap that separates this rare class of microbial metabolites from clinical studies, which could validate them as effective tools for cancer treatment.\", \"doi_y\": \"https://doi.org/10.1016/j.molmet.2023.101674\", \"publication_date\": \"2023-02-01\"}, {\"work_id\": \"W4319004578\", \"doi_x\": \"10.3390/biom13020277\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"KinFams: De-Novo Classification of Protein Kinases Using CATH Functional Units\", \"abstract\": \"Protein kinases are important targets for treating human disorders, and they are the second most targeted families after G-protein coupled receptors. Several resources provide classification of kinases into evolutionary families (based on sequence homology); however, very few systematically classify functional families (FunFams) comprising evolutionary relatives that share similar functional properties. We have developed the FunFam-MARC (Multidomain ARchitecture-based Clustering) protocol, which uses multi-domain architectures of protein kinases and specificity-determining residues for functional family classification. FunFam-MARC predicts 2210 kinase functional families (KinFams), which have increased functional coherence, in terms of EC annotations, compared to the widely used KinBase classification. Our protocol provides a comprehensive classification for kinase sequences from >10,000 organisms. We associate human KinFams with diseases and drugs and identify 28 druggable human KinFams, i.e., enriched in clinically approved drugs. Since relatives in the same druggable KinFam tend to be structurally conserved, including the drug-binding site, these KinFams may be valuable for shortlisting therapeutic targets. Information on the human KinFams and associated 3D structures from AlphaFold2 are provided via our CATH FTP website and Zenodo. This gives the domain structure representative of each KinFam together with information on any drug compounds available. For 32% of the KinFams, we provide information on highly conserved residue sites that may be associated with specificity.\", \"doi_y\": \"https://doi.org/10.3390/biom13020277\", \"publication_date\": \"2023-02-02\"}, {\"work_id\": \"W4293313205\", \"doi_x\": \"10.1002/prp2.994\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Combined docking and machine learning identify key molecular determinants of ligand pharmacological activity on \\u03b22 adrenoceptor\", \"abstract\": \"G protein-coupled receptors (GPCRs) are valuable therapeutic targets for many diseases. A central question of GPCR drug discovery is to understand what determines the agonism or antagonism of ligands that bind them. Ligands exert their action via the interactions in the ligand binding pocket. We hypothesized that there is a common set of receptor interactions made by ligands of diverse structures that mediate their action and that among a large dataset of different ligands, the functionally important interactions will be over-represented. We computationally docked ~2700 known \\u03b22AR ligands to multiple \\u03b22AR structures, generating ca 75 000 docking poses and predicted all atomic interactions between the receptor and the ligand. We used machine learning (ML) techniques to identify specific interactions that correlate with the agonist or antagonist activity of these ligands. We demonstrate with the application of ML methods that it is possible to identify the key interactions associated with agonism or antagonism of ligands. The most representative interactions for agonist ligands involve K972.68\\u00d767 , F194ECL2 , S2035.42\\u00d743 , S2045.43\\u00d744 , S2075.46\\u00d7641 , H2966.58\\u00d758 , and K3057.32\\u00d731 . Meanwhile, the antagonist ligands made interactions with W2866.48\\u00d748 and Y3167.43\\u00d742 , both residues considered to be important in GPCR activation. The interpretation of ML analysis in human understandable form allowed us to construct an exquisitely detailed structure-activity relationship that identifies small changes to the ligands that invert their pharmacological activity and thus helps to guide the drug discovery process. This approach can be readily applied to any drug target.\", \"doi_y\": \"https://doi.org/10.1002/prp2.994\", \"publication_date\": \"2022-08-26\"}, {\"work_id\": \"W4288802398\", \"doi_x\": \"10.1038/s42003-022-03745-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural determinants of DNA recognition by the NO sensor NsrR and related Rrf2-type [FeS]-transcription factors\", \"abstract\": \"Several transcription factors of the Rrf2 family use an iron-sulfur cluster to regulate DNA binding through effectors such as nitric oxide (NO), cellular redox status and iron levels. [4Fe-4S]-NsrR from Streptomyces coelicolor (ScNsrR) modulates expression of three different genes via reaction and complex formation with variable amounts of NO, which results in detoxification of this gas. Here, we report the crystal structure of ScNsrR complexed with an hmpA1 gene operator fragment and compare it with those previously reported for [2Fe-2S]-RsrR/rsrR and apo-IscR/hyA complexes. Important structural differences reside in the variation of the DNA minor and major groove widths. In addition, different DNA curvatures and different interactions with the protein sensors are observed. We also report studies of NsrR binding to four hmpA1 variants, which indicate that flexibility in the central region is not a key binding determinant. Our study explores the promotor binding specificities of three closely related transcriptional regulators.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03745-7\", \"publication_date\": \"2022-07-30\"}, {\"work_id\": \"W4323351421\", \"doi_x\": \"10.1101/2023.03.05.531151\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Discovery and characterization of non-canonical E2 conjugating enzymes\", \"abstract\": \"Abstract E2 conjugating enzymes (E2s) play a central role in the enzymatic cascade that leads to the attachment of ubiquitin to a substrate. This process, termed ubiquitylation is fundamental for maintaining cellular homeostasis and impacts almost all cellular process. By interacting with multiple E3 ligases, E2s direct the ubiquitylation landscape within the cell. Since its discovery, ubiquitylation has been regarded as a post-translational modification that specifically targets lysine side chains (canonical ubiquitylation). We used MALDI-TOF Mass Spectrometry to discover and characterize a family of E2s that are instead able to conjugate ubiquitin to serine and/or threonine. We employed protein modelling and prediction tools to identify the catalytic determinants that these E2s use to interact with ubiquitin as well as their substrates. Our results join a stream of recent literature that challenges the definition of ubiquitylation as an exquisitely lysine-specific modification and provide crucial insights into the missing E2 element responsible for non-canonical ubiquitylation. Teaser E2 conjugating enzymes (E2s) play a fundamental role in the attachment of ubiquitin to its substrate. Most E2s can form an isopeptide bond between the ubiquitin C- terminus and a lysine present on the substrate. We identified a family of E2s, UBE2Q1 and UBE2Q2, able to target amino acids other than lysine. Currently nothing is known about their mechanism of action and what substrates they are targeting, even though genetic ablation of UBE2Q1 produce substantial infertility in mice. Here we answer the question about what the key residues beneath their peculiar activity are. We discovered that UBE2Q1 target the lysine-free cytoplasmic domain of the Golgi resident protein Beta-1,4-galactosyltransferase 1, providing an interesting precedent for the role of non-canonical ubiquitylation in eukaryotic cells.\", \"doi_y\": \"https://doi.org/10.1101/2023.03.05.531151\", \"publication_date\": \"2023-03-05\"}, {\"work_id\": \"W4308365540\", \"doi_x\": \"10.1093/femsre/fuac044\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>In situ, in vivo</i>, and <i>in vitro</i> approaches for studying AMR plasmid conjugation in the gut microbiome\", \"abstract\": \"Antimicrobial resistance (AMR) is a global threat, with evolution and spread of resistance to frontline antibiotics outpacing the development of novel treatments. The spread of AMR is perpetuated by transfer of antimicrobial resistance genes (ARGs) between bacteria, notably those encoded by conjugative plasmids. The human gut microbiome is a known 'melting pot' for plasmid conjugation, with ARG transfer in this environment widely documented. There is a need to better understand the factors affecting the incidence of these transfer events, and to investigate methods of potentially counteracting the spread of ARGs. This review describes the use and potential of three approaches to studying conjugation in the human gut: observation of in situ events in hospitalized patients, modelling of the microbiome in vivo predominantly in rodent models, and the use of in vitro models of various complexities. Each has brought unique insights to our understanding of conjugation in the gut. The use and development of these systems, and combinations thereof, will be pivotal in better understanding the significance, prevalence, and manipulability of horizontal gene transfer in the gut microbiome.\", \"doi_y\": \"https://doi.org/10.1093/femsre/fuac044\", \"publication_date\": \"2022-11-07\"}, {\"work_id\": \"W4288096495\", \"doi_x\": \"10.7554/elife.79602\", \"af_direct_citation\": 0, \"af_indirect_citations\": 7, \"title\": \"Large protein complex interfaces have evolved to promote cotranslational assembly\", \"abstract\": \"Assembly pathways of protein complexes should be precise and efficient to minimise misfolding and unwanted interactions with other proteins in the cell. One way to achieve this efficiency is by seeding assembly pathways during translation via the cotranslational assembly of subunits. While recent evidence suggests that such cotranslational assembly is widespread, little is known about the properties of protein complexes associated with the phenomenon. Here, using a combination of proteome-specific protein complex structures and publicly available ribosome profiling data, we show that cotranslational assembly is particularly common between subunits that form large intermolecular interfaces. To test whether large interfaces have evolved to promote cotranslational assembly, as opposed to cotranslational assembly being a non-adaptive consequence of large interfaces, we compared the sizes of first and last translated interfaces of heteromeric subunits in bacterial, yeast, and human complexes. When considering all together, we observe the N-terminal interface to be larger than the C-terminal interface 54% of the time, increasing to 64% when we exclude subunits with only small interfaces, which are unlikely to cotranslationally assemble. This strongly suggests that large interfaces have evolved as a means to maximise the chance of successful cotranslational subunit binding.\", \"doi_y\": \"https://doi.org/10.7554/elife.79602\", \"publication_date\": \"2022-07-28\"}, {\"work_id\": \"W4220916195\", \"doi_x\": \"10.1101/2022.03.17.484746\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structure and functional mapping of the KRAB-KAP1 repressor complex\", \"abstract\": \"Abstract Transposable elements (TEs) are a genetic reservoir from which new genes and regulatory elements can emerge. Expression of TEs can be pathogenic, however, and is tightly regulated. KRAB domain-containing zinc finger proteins (KRAB-ZFPs) recruit co-repressor KRAB-associated protein 1 (KAP1/TRIM28) to regulate many TEs but how KRAB-ZFPs and KAP1 interact remains unclear. Here, we report the crystal structure of the KAP1 tripartite motif (TRIM) in complex with the KRAB domain from a human KRAB-ZFP, ZNF93. Structure-guided mutations in the KAP1-KRAB binding interface abolished repressive activity in an epigenetic TE silencing assay. Deposition of H3K9me3 over thousands of loci is lost genome-wide in cells expressing a KAP1 variant with mutations that abolish KRAB binding. Our work identifies and functionally validates the KRAB-KAP1 molecular interface, which forms the nexus of a transcriptional control axis vital to vertebrates and underpins programmable gene repression by CRISPRi.\", \"doi_y\": \"https://doi.org/10.1101/2022.03.17.484746\", \"publication_date\": \"2022-03-17\"}, {\"work_id\": \"W4297311560\", \"doi_x\": \"10.1101/2022.09.25.509416\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Supercharged PGR5-dependent cyclic electron transfer compensates for mis-regulated chloroplast ATP synthase\", \"abstract\": \"Abstract The light reactions of photosynthesis couple electron and proton transfers across the thylakoid membrane, generating NADPH, and proton motive force (pmf) that powers the endergonic synthesis of ATP by ATP synthase. ATP and NADPH are required for CO 2 fixation into carbohydrates by the Calvin-Benson-Bassham cycle (CBBC). The dominant \\u0394pH component of the pmf also plays a photoprotective role in regulating photosystem II (PSII) light harvesting efficiency, through non-photochemical quenching (NPQ), and cytochrome b 6 f (cyt b 6 f ) to photosystem I (PSI) electron transfer, via photosynthetic control. \\u0394pH can be adjusted by increasing the proton influx into the thylakoid lumen via upregulation of cyclic electron transfer (CET) or decreasing proton efflux via downregulation of ATP synthase conductivity (gH + ). The interplay and relative contributions of these two elements of \\u0394pH control to photoprotection are not well understood. Here, we show that an Arabidopsis ATP synthase mutant ( hope2 ) with 40% higher proton efflux, has supercharged CET. Double crosses of hope2 with the CET-deficient pgr5 and ndho lines reveal that PGR5-dependent CET is the major pathway contributing to higher proton influx. PGR5-dependent CET allows hope2 to maintain wild-type levels of \\u0394pH, CO 2 fixation and NPQ, however photosynthetic control remains absent, and PSI is acceptor-side limited. Therefore, high CET in the absence of ATP synthase regulation is insufficient for PSI photoprotection.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.25.509416\", \"publication_date\": \"2022-09-26\"}, {\"work_id\": \"W4385830496\", \"doi_x\": \"10.1098/rsob.230220\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The metabolic control of DNA replication: mechanism and function\", \"abstract\": \"Metabolism and DNA replication are the two most fundamental biological functions in life. The catabolic branch of metabolism breaks down nutrients to produce energy and precursors used by the anabolic branch of metabolism to synthesize macromolecules. DNA replication consumes energy and precursors for faithfully copying genomes, propagating the genetic material from generation to generation. We have exquisite understanding of the mechanisms that underpin and regulate these two biological functions. However, the molecular mechanism coordinating replication to metabolism and its biological function remains mostly unknown. Understanding how and why living organisms respond to fluctuating nutritional stimuli through cell-cycle dynamic changes and reproducibly and distinctly temporalize DNA synthesis in a wide-range of growth conditions is important, with wider implications across all domains of life. After summarizing the seminal studies that founded the concept of the metabolic control of replication, we review data linking metabolism to replication from bacteria to humans. Molecular insights underpinning these links are then presented to propose that the metabolic control of replication uses signalling systems gearing metabolome homeostasis to orchestrate replication temporalization. The remarkable replication phenotypes found in mutants of this control highlight its importance in replication regulation and potentially genetic stability and tumorigenesis.\", \"doi_y\": \"https://doi.org/10.1098/rsob.230220\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4381377872\", \"doi_x\": \"10.1101/2023.06.19.545293\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Bimodular architecture of bacterial effector SAP05 drives ubiquitin-independent targeted protein degradation\", \"abstract\": \"Abstract In eukaryotes, targeted protein degradation (TPD) typically depends on a series of interactions among ubiquitin ligases that transfer ubiquitin molecules to substrates leading to degradation by the 26S proteasome. We previously discovered that the bacterial effector protein SAP05 mediates ubiquitin-independent TPD. SAP05 forms a ternary complex via interactions with the von Willebrand Factor Type A (vWA) domain of the proteasomal ubiquitin receptor Rpn10 and the Zinc-finger (ZnF) domains of the SQUAMOSA-PROMOTER BINDING PROTEIN-LIKE (SPL) and GATA BINDING FACTOR (GATA) transcription factors (TFs). This leads to direct TPD of the TFs by the 26S proteasome. Here, we report the crystal structures of the SAP05-vWA complex at 2.17 \\u00c5 resolution and of the SAP05-ZnF(SPL5) complex at 2.20 \\u00c5 resolution. Structural analyses revealed that SAP05 displays a unique bimodular architecture with two distinct non-overlapping surfaces, a \\u2018loop surface\\u2019 with three protruding loops that form electrostatic interactions with ZnF, and a \\u2018sheet surface\\u2019 featuring two \\u03b2-sheets, loops and \\u237a-helices that establish polar interactions with vWA. SAP05 binding to ZnF TFs involves single amino acids responsible for multiple contacts, while SAP05 binding to vWA is more stable due to the necessity of multiple mutations to break the interaction. In addition, positioning of the SAP05 complex on the 26S proteasome points to a mechanism of protein degradation. Collectively, our findings demonstrate how a small bacterial bimodular protein can bypass the canonical UPS cellular proteolysis pathway, enabling ubiquitin-independent TPD in eukaryotic cells. This knowledge holds significant potential for the creation of novel TPD technologies.\", \"doi_y\": \"https://doi.org/10.1101/2023.06.19.545293\", \"publication_date\": \"2023-06-19\"}, {\"work_id\": \"W4386366735\", \"doi_x\": \"10.1016/j.jbc.2023.105218\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"abstract\": \"Human G protein-coupled receptor 35 is regulated by agonist-mediated phosphorylation of a set of five phospho-acceptor amino acids within its C-terminal tail. Alteration of both Ser300 and Ser303 to alanine in the GPR35a isoform greatly reduces the ability of receptor agonists to promote interactions with arrestin adapter proteins. Here, we have integrated the use of cell lines genome edited to lack expression of combinations of G protein receptor kinases (GRKs), selective small molecule inhibitors of subsets of these kinases, and antisera able to specifically identify either human GPR35a or mouse GPR35 only when Ser300 and Ser303 (orce; the equivalent residues in mouse GPR35) have become phosphorylated to demonstrate that GRK5 and GRK6 cause agonist-dependent phosphorylation of these residues. Extensions of these studies demonstrated the importance of the GRK5/6-mediated phosphorylation of these amino acids for agonist-induced internalization of the receptor. Homology and predictive modeling of the interaction of human GPR35 with GRKs showed that the N terminus of GRK5 is likely to dock in the same methionine pocket on the intracellular face of GPR35 as the C terminus of the \\u03b15 helix of G\\u03b113 and, that while this is also the case for GRK6, GRK2 and GRK3 are unable to do so effectively. These studies provide unique and wide-ranging insights into modes of regulation of GPR35, a receptor that is currently attracting considerable interest as a novel therapeutic target in diseases including ulcerative colitis.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.105218\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4319600908\", \"doi_x\": \"10.3390/cancers15041040\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"E2 Partner Tunes the Ubiquitylation Specificity of Arkadia E3 Ubiquitin Ligase\", \"abstract\": \"Arkadia (RNF111) is a positive regulator of the TGF-\\u03b2 signaling that mediates the proteasome-dependent degradation of negative factors of the pathway. It is classified as an E3 ubiquitin ligase and a SUMO-targeted ubiquitin ligase (STUBL), implicated in various pathological conditions including cancer and fibrosis. The enzymatic (ligase) activity of Arkadia is located at its C-terminus and involves the RING domain. Notably, E3 ligases require E2 enzymes to perform ubiquitylation. However, little is known about the cooperation of Arkadia with various E2 enzymes and the type of ubiquitylation that they mediate. In the present work, we study the interaction of Arkadia with the E2 partners UbcH5B and UbcH13, as well as UbcH7. Through NMR spectroscopy, we found that the E2-Arkadia interaction surface is similar in all pairs examined. Nonetheless, the requirements and factors that determine an enzymatically active E2-Arkadia complex differ in each case. Furthermore, we revealed that the cooperation of Arkadia with different E2s results in either monoubiquitylation or polyubiquitin chain formation via K63, K48, or K11 linkages, which can determine the fate of the substrate and lead to distinct biological outcomes.\", \"doi_y\": \"https://doi.org/10.3390/cancers15041040\", \"publication_date\": \"2023-02-07\"}, {\"work_id\": \"W4366828133\", \"doi_x\": \"10.3389/fbioe.2023.1178536\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Improving carbon monoxide tolerance of Cupriavidus necator H16 through adaptive laboratory evolution\", \"abstract\": \"Background: The toxic gas carbon monoxide (CO) is abundantly present in synthesis gas (syngas) and certain industrial waste gases that can serve as feedstocks for the biological production of industrially significant chemicals and fuels. For efficient bacterial growth to occur, and to increase productivity and titres, a high resistance to the gas is required. The aerobic bacterium Cupriavidus necator H16 can grow on CO2 + H2, although it cannot utilise CO as a source of carbon and energy. This study aimed to increase its CO resistance through adaptive laboratory evolution. Results: To increase the tolerance of C. necator to CO, the organism was continually subcultured in the presence of CO both heterotrophically and autotrophically. Ten individual cultures were evolved heterotrophically with fructose in this manner and eventually displayed a clear growth advantage over the wild type strain. Next-generation sequencing revealed several mutations, including a single point mutation upstream of a cytochrome bd ubiquinol oxidase operon (cydA2B2), which was present in all evolved isolates. When a subset of these mutations was engineered into the parental H16 strain, only the cydA2B2 upstream mutation enabled faster growth in the presence of CO. Expression analysis, mutation, overexpression and complementation suggested that cydA2B2 transcription is upregulated in the evolved isolates, resulting in increased CO tolerance under heterotrophic but not autotrophic conditions. However, through subculturing on a syngas-like mixture with increasing CO concentrations, C. necator could also be evolved to tolerate high CO concentrations under autotrophic conditions. A mutation in the gene for the soluble [NiFe]-hydrogenase subunit hoxH was identified in the evolved isolates. When the resulting amino acid change was engineered into the parental strain, autotrophic CO resistance was conferred. A strain constitutively expressing cydA2B2 and the mutated hoxH gene exhibited high CO tolerance under both heterotrophic and autotrophic conditions. Conclusion:C. necator was evolved to tolerate high concentrations of CO, a phenomenon which was dependent on the terminal respiratory cytochrome bd ubiquinol oxidase when grown heterotrophically and the soluble [NiFe]-hydrogenase when grown autotrophically. A strain exhibiting high tolerance under both conditions was created and presents a promising chassis for syngas-based bioproduction processes.\", \"doi_y\": \"https://doi.org/10.3389/fbioe.2023.1178536\", \"publication_date\": \"2023-04-24\"}, {\"work_id\": \"W4311948686\", \"doi_x\": \"10.1101/2022.12.15.519894\", \"af_direct_citation\": 0, \"af_indirect_citations\": 8, \"title\": \"Codon language embeddings provide strong signals for protein engineering\", \"abstract\": \"Abstract Protein representations from deep language models have yielded state-of-the-art performance across many tasks in computational protein engineering. In recent years, progress has primarily focused on parameter count, with recent models\\u2019 capacities surpassing the size of the very datasets they were trained on. Here, we propose an alternative direction. We show that large language models trained on codons, instead of amino acid sequences, provide high-quality representations that outperform comparable state-of-the-art models across a variety of tasks. In some tasks, like species recognition, prediction of protein and transcript abundance, or melting point estimation, we show that a language model trained on codons outperforms every other published protein language model, including some that contain over 50 times more parameters. These results suggest that, in addition to commonly studied scale and model complexity, the information content of biological data provides an orthogonal direction to improve the power of machine learning in biology.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.15.519894\", \"publication_date\": \"2022-12-19\"}, {\"work_id\": \"W4225377946\", \"doi_x\": \"10.1038/s42003-022-03325-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pseudotyped Bat Coronavirus RaTG13 is efficiently neutralised by convalescent sera from SARS-CoV-2 infected patients\", \"abstract\": \"RaTG13 is a close relative of SARS-CoV-2, the virus responsible for the COVID-19 pandemic, sharing 96% sequence similarity at the genome-wide level. The spike receptor binding domain (RBD) of RaTG13 contains a number of amino acid substitutions when compared to SARS-CoV-2, likely impacting affinity for the ACE2 receptor. Antigenic differences between the viruses are less well understood, especially whether RaTG13 spike can be efficiently neutralised by antibodies generated from infection with, or vaccination against, SARS-CoV-2. Using RaTG13 and SARS-CoV-2 pseudotypes we compared neutralisation using convalescent sera from previously infected patients or vaccinated healthcare workers. Surprisingly, our results revealed that RaTG13 was more efficiently neutralised than SARS-CoV-2. In addition, neutralisation assays using spike mutants harbouring single and combinatorial amino acid substitutions within the RBD demonstrated that both spike proteins can tolerate multiple changes without dramatically reducing neutralisation. Moreover, introducing the 484 K mutation into RaTG13 resulted in increased neutralisation, in contrast to the same mutation in SARS-CoV-2 (E484K). This is despite E484K having a well-documented role in immune evasion in variants of concern (VOC) such as B.1.351 (Beta). These results indicate that the future spill-over of RaTG13 and/or related sarbecoviruses could be mitigated using current SARS-CoV-2-based vaccination strategies.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03325-9\", \"publication_date\": \"2022-05-03\"}, {\"work_id\": \"W4287838589\", \"doi_x\": \"10.1073/pnas.2204539119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 impairs interferon production via NSP2-induced repression of mRNA translation\", \"abstract\": \"Viruses evade the innate immune response by suppressing the production or activity of cytokines such as type I interferons (IFNs). Here we report the discovery of a mechanism by which the SARS-CoV-2 virus coopts an intrinsic cellular machinery to suppress the production of the key immunostimulatory cytokine IFN-\\u03b2. We reveal that the SARS-CoV-2 encoded nonstructural protein 2 (NSP2) directly interacts with the cellular GIGYF2 protein. This interaction enhances the binding of GIGYF2 to the mRNA cap-binding protein 4EHP, thereby repressing the translation of the Ifnb1 mRNA. Depletion of GIGYF2 or 4EHP significantly enhances IFN-\\u03b2 production, which inhibits SARS-CoV-2 replication. Our findings reveal a target for rescuing the antiviral innate immune response to SARS-CoV-2 and other RNA viruses.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2204539119\", \"publication_date\": \"2022-07-25\"}, {\"work_id\": \"W4311256441\", \"doi_x\": \"10.1016/j.preteyeres.2022.101154\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"10q26 \\u2013 The enigma in age-related macular degeneration\", \"abstract\": \"Despite comprehensive research efforts over the last decades, the pathomechanisms of age-related macular degeneration (AMD) remain far from being understood. Large-scale genome wide association studies (GWAS) were able to provide a defined set of genetic aberrations which contribute to disease risk, with the strongest contributors mapping to distinct regions on chromosome 1 and 10. While the chromosome 1 locus comprises factors of the complement system with well-known functions, the role of the 10q26-locus in AMD-pathophysiology remains enigmatic. 10q26 harbors a cluster of three functional genes, namely PLEKHA1, ARMS2 and HTRA1, with most of the AMD-associated genetic variants mapping to the latter two genes. High linkage disequilibrium between ARMS2 and HTRA1 has kept association studies from reliably defining the risk-causing gene for long and only very recently the genetic risk region has been narrowed to ARMS2, suggesting that this is the true AMD gene at this locus. However, genetic associations alone do not suffice to prove causality and one or more of the 14 SNPs on this haplotype may be involved in long-range control of gene expression, leaving HTRA1 and PLEKHA1 still suspects in the pathogenic pathway. Both, ARMS2 and HTRA1 have been linked to extracellular matrix homeostasis, yet their exact molecular function as well as their role in AMD pathogenesis remains to be uncovered. The transcriptional regulation of the 10q26 locus adds an additional level of complexity, given, that gene-regulatory as well as epigenetic alterations may influence expression levels from 10q26 in diseased individuals. Here, we provide a comprehensive overview on the 10q26 locus and its three gene products on various levels of biological complexity and discuss current and future research strategies to shed light on one of the remaining enigmatic spots in the AMD landscape.\", \"doi_y\": \"https://doi.org/10.1016/j.preteyeres.2022.101154\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4289667563\", \"doi_x\": \"10.1016/j.celrep.2022.111168\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"abstract\": \"An essential first step in the post-translational modification of proteins with UFM1, UFMylation, is the proteolytic cleavage of pro-UFM1 to expose a C-terminal glycine. Of the two UFM1-specific proteases (UFSPs) identified in humans, only UFSP2 is reported to be active, since the annotated sequence of UFSP1 lacks critical catalytic residues. Nonetheless, efficient UFM1 maturation occurs in cells lacking UFSP2, suggesting the presence of another active protease. We herein identify UFSP1 translated from a non-canonical start site to be this protease. Cells lacking both UFSPs show complete loss of UFMylation resulting from an absence of mature UFM1. While UFSP2, but not UFSP1, removes UFM1 from the ribosomal subunit RPL26, UFSP1 acts earlier in the pathway to mature UFM1 and cleave a potential autoinhibitory modification on UFC1, thereby controlling activation of UFMylation. In summary, our studies reveal important distinctions in substrate specificity and localization-dependent functions for the two proteases in regulating UFMylation.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111168\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4317749977\", \"doi_x\": \"10.1038/s41570-022-00456-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA synthesis technologies to close the gene writing gap\", \"abstract\": \"Synthetic DNA is of increasing demand across many sectors of research and commercial activities. Engineering biology, therapy, data storage and nanotechnology are set for rapid developments if DNA can be provided at scale and low cost. Stimulated by successes in next generation sequencing and gene editing technologies, DNA synthesis is already a burgeoning industry. However, the synthesis of >200 bp sequences remains unaffordable. To overcome these limitations and start writing DNA as effectively as it is read, alternative technologies have been developed including molecular assembly and cloning methods, template-independent enzymatic synthesis, microarray and rolling circle amplification techniques. Here, we review the progress in developing and commercializing these technologies, which are exemplified by innovations from leading companies. We discuss pros and cons of each technology, the need for oversight and regulatory policies for DNA synthesis as a whole and give an overview of DNA synthesis business models. There is increasing demand for synthetic DNA. However, our ability to make, or write, DNA lags behind our ability to sequence, or read, it. This Review discusses commercialized DNA synthesis technologies in the pursuit of closing the DNA writing gap.\", \"doi_y\": \"https://doi.org/10.1038/s41570-022-00456-9\", \"publication_date\": \"2023-01-23\"}, {\"work_id\": \"W4327711803\", \"doi_x\": \"10.1126/sciadv.ade8487\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Plant and prokaryotic TIR domains generate distinct cyclic ADPR NADase products\", \"abstract\": \"Toll/interleukin-1 receptor (TIR) domain proteins function in cell death and immunity. In plants and bacteria, TIR domains are often enzymes that produce isomers of cyclic adenosine 5'-diphosphate-ribose (cADPR) as putative immune signaling molecules. The identity and functional conservation of cADPR isomer signals is unclear. A previous report found that a plant TIR could cross-activate the prokaryotic Thoeris TIR-immune system, suggesting the conservation of plant and prokaryotic TIR-immune signals. Here, we generate autoactive Thoeris TIRs and test the converse hypothesis: Do prokaryotic Thoeris TIRs also cross-activate plant TIR immunity? Using in planta and in vitro assays, we find that Thoeris and plant TIRs generate overlapping sets of cADPR isomers and further clarify how plant and Thoeris TIRs activate the Thoeris system via producing 3'cADPR. This study demonstrates that the TIR signaling requirements for plant and prokaryotic immune systems are distinct and that TIRs across kingdoms generate a diversity of small-molecule products.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.ade8487\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4386784386\", \"doi_x\": \"10.1186/s12934-023-02184-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"High-throughput process development from gene cloning to protein production\", \"abstract\": \"Abstract In the post-genomic era, the demand for faster and more efficient protein production has increased, both in public laboratories and industry. In addition, with the expansion of protein sequences in databases, the range of possible enzymes of interest for a given application is also increasing. Faced with peer competition, budgetary, and time constraints, companies and laboratories must find ways to develop a robust manufacturing process for recombinant protein production. In this review, we explore high-throughput technologies for recombinant protein expression and present a holistic high-throughput process development strategy that spans from genes to proteins. We discuss the challenges that come with this task, the limitations of previous studies, and future research directions.\", \"doi_y\": \"https://doi.org/10.1186/s12934-023-02184-1\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4311584214\", \"doi_x\": \"10.7554/elife.84702\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Teaching old drugs new tricks\", \"abstract\": \"Understanding the mechanism by which streptomycin binds to the small subunit of the mitoribosome may help researchers design less toxic derivatives of this antibiotic.\", \"doi_y\": \"https://doi.org/10.7554/elife.84702\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4324028664\", \"doi_x\": \"10.1073/pnas.2301358120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Rgs1 is a regulator of effector gene expression during plant infection by the rice blast fungus <i>Magnaporthe oryzae</i>\", \"abstract\": \"To cause rice blast disease, the filamentous fungus Magnaporthe oryzae secretes a battery of effector proteins into host plant tissue to facilitate infection. Effector-encoding genes are expressed only during plant infection and show very low expression during other developmental stages. How effector gene expression is regulated in such a precise manner during invasive growth by M. oryzae is not known. Here, we report a forward-genetic screen to identify regulators of effector gene expression, based on the selection of mutants that show constitutive effector gene expression. Using this simple screen, we identify Rgs1, a regulator of G-protein signaling (RGS) protein that is necessary for appressorium development, as a novel transcriptional regulator of effector gene expression, which acts prior to plant infection. We show that an N-terminal domain of Rgs1, possessing transactivation activity, is required for effector gene regulation and acts in an RGS-independent manner. Rgs1 controls the expression of at least 60 temporally coregulated effector genes, preventing their transcription during the prepenetration stage of development prior to plant infection. A regulator of appressorium morphogenesis is therefore also required for the orchestration of pathogen gene expression required for invasive growth by M. oryzae during plant infection.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2301358120\", \"publication_date\": \"2023-03-13\"}, {\"work_id\": \"W4367676465\", \"doi_x\": \"10.12688/wellcomeopenres.18937.2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structure of the transmembrane protein 2 (TMEM2) ectodomain and its apparent lack of hyaluronidase activity\", \"abstract\": \"<ns4:p><ns4:bold>Background: </ns4:bold>Hyaluronic acid (HA) is a major polysaccharide component of the extracellular matrix. HA has essential functions in tissue architecture and the regulation of cell behaviour. HA turnover needs to be finely balanced. Increased HA degradation is associated with cancer, inflammation, and other pathological situations. Transmembrane protein 2 (TMEM2) is a cell surface protein that has been reported to degrade HA into ~5 kDa fragments and play an essential role in systemic HA turnover.</ns4:p><ns4:p> <ns4:bold>Methods:</ns4:bold> We produced the soluble TMEM2 ectodomain (residues 106-1383; sTMEM2) in human embryonic kidney cells (HEK293) and determined its structure using X-ray crystallography. We tested sTMEM2 hyaluronidase activity using fluorescently labelled HA and size fractionation of reaction products. We tested HA binding in solution and using a glycan microarray.</ns4:p><ns4:p> <ns4:bold>Results:</ns4:bold> Our crystal structure of sTMEM2 confirms a remarkably accurate prediction by AlphaFold. sTMEM2 contains a parallel \\u03b2-helix typical of other polysaccharide-degrading enzymes, but an active site cannot be assigned with confidence. A lectin-like domain is inserted into the \\u03b2-helix and predicted to be functional in carbohydrate binding. A second lectin-like domain at the C-terminus is unlikely to bind carbohydrates. We did not observe HA binding in two assay formats, suggesting a modest affinity at best. Unexpectedly, we were unable to observe any HA degradation by sTMEM2. Our negative results set an upper limit for <ns4:italic>k</ns4:italic><ns4:sub>cat</ns4:sub> of approximately 10<ns4:sup>-5</ns4:sup> min<ns4:sup>-1</ns4:sup>.</ns4:p><ns4:p> <ns4:bold>Conclusions:</ns4:bold> Although sTMEM2 contains domain types consistent with its suggested role in TMEM2 degradation, its hyaluronidase activity was undetectable. HA degradation by TMEM2 may require additional proteins and/or localisation at the cell surface.</ns4:p>\", \"doi_y\": \"https://doi.org/10.12688/wellcomeopenres.18937.2\", \"publication_date\": \"2023-05-02\"}, {\"work_id\": \"W4221026494\", \"doi_x\": \"10.1038/s41467-022-28503-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A late-stage assembly checkpoint of the human mitochondrial ribosome large subunit\", \"abstract\": \"Many cellular processes, including ribosome biogenesis, are regulated through post-transcriptional RNA modifications. Here, a genome-wide analysis of the human mitochondrial transcriptome shows that 2'-O-methylation is limited to residues of the mitoribosomal large subunit (mtLSU) 16S mt-rRNA, introduced by MRM1, MRM2 and MRM3, with the modifications installed by the latter two proteins being interdependent. MRM2 controls mitochondrial respiration by regulating mitoribosome biogenesis. In its absence, mtLSU particles (visualized by cryo-EM at the resolution of 2.6 \\u00c5) present disordered RNA domains, partial occupancy of bL36m and bound MALSU1:L0R8F8:mtACP anti-association module, allowing five mtLSU biogenesis intermediates with different intersubunit interface configurations to be placed along the assembly pathway. However, mitoribosome biogenesis does not depend on the methyltransferase activity of MRM2. Disruption of the MRM2 Drosophila melanogaster orthologue leads to mitochondria-related developmental arrest. This work identifies a key checkpoint during mtLSU assembly, essential to maintain mitochondrial homeostasis.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-28503-5\", \"publication_date\": \"2022-02-17\"}, {\"work_id\": \"W4297520297\", \"doi_x\": \"10.1101/2022.09.27.509674\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Benchmarking strategies for cross-species integration of single-cell RNA sequencing data\", \"abstract\": \"Abstract The growing number of available single cell gene expression datasets from different species creates opportunities to explore evolutionary relationships between cell types across species. Cross-species integration of single-cell RNA-sequencing data has been particularly informative in this context. However, in order to do so robustly it is essential to have rigorous benchmarking and appropriate guidelines to ensure that integration results truly reflect biology. We benchmarked 28 combinations of gene homology mapping methods and data integration algorithms in a variety of biological settings. We examined the capability of each strategy to perform species-mixing of known homologous cell types and to preserve biological heterogeneity using 9 established metrics. We also developed a new biology conservation metric to address the maintenance of cell type distinguishability. Overall, scANVI, scVI and SeuratV4 methods achieved a balance between species-mixing and biology conservation. For evolutionarily distant species, including in-paralogs was beneficial. SAMap outperformed when integrating whole-body atlases between species with challenging gene homology annotation. We provided our freely available cross-species integration and assessment pipeline to help analyse new data and develop new algorithms.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.27.509674\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W4306290708\", \"doi_x\": \"10.1101/2022.10.14.511843\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Precisely patterned nanofibers made from extendable protein multiplexes\", \"abstract\": \"Abstract Molecular systems with coincident cyclic and superhelical symmetry axes have considerable advantages for materials design as they can be readily lengthened or shortened by changing the length of the constituent monomers. Among proteins, alpha helical coiled coils have such symmetric extendable architectures, but are limited by the relatively fixed geometry and flexibility of the helical protomers. Here, we describe a systematic approach to generating modular and rigid repeat protein oligomers with coincident C 2 to C 8 and superhelical symmetry axes that can be readily extended by repeat propagation. From these building blocks, we demonstrate that a wide range of unbounded fibers can be systematically designed by introducing hydrophilic surface patches that force staggering of the monomers; the geometry of such fibers can be precisely tuned by varying the number of repeat units in the monomer and the placement of the hydrophilic patches.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.14.511843\", \"publication_date\": \"2022-10-14\"}, {\"work_id\": \"W4378348751\", \"doi_x\": \"10.1021/jacsau.3c00119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Two Distinct Thermodynamic Gradients for Cellular Metalation of Vitamin B<sub>12</sub>\", \"abstract\": \"The acquisition of CoII by the corrin component of vitamin B12 follows one of two distinct pathways, referred to as early or late CoII insertion. The late insertion pathway exploits a CoII metallochaperone (CobW) from the COG0523 family of G3E GTPases, while the early insertion pathway does not. This provides an opportunity to contrast the thermodynamics of metalation in a metallochaperone-requiring and a metallochaperone-independent pathway. In the metallochaperone-independent route, sirohydrochlorin (SHC) associates with the CbiK chelatase to form CoII-SHC. CoII-buffered enzymatic assays indicate that SHC binding enhances the thermodynamic gradient for CoII transfer from the cytosol to CbiK. In the metallochaperone-dependent pathway, hydrogenobyrinic acid a,c-diamide (HBAD) associates with the CobNST chelatase to form CoII-HBAD. Here, CoII-buffered enzymatic assays indicate that CoII transfer from the cytosol to HBAD-CobNST must somehow traverse a highly unfavorable thermodynamic gradient for CoII binding. Notably, there is a favorable gradient for CoII transfer from the cytosol to the MgIIGTP-CobW metallochaperone, but further transfer of CoII from the GTP-bound metallochaperone to the HBAD-CobNST chelatase complex is thermodynamically unfavorable. However, after nucleotide hydrolysis, CoII transfer from the chaperone to the chelatase complex is calculated to become favorable. These data reveal that the CobW metallochaperone can overcome an unfavorable thermodynamic gradient for CoII transfer from the cytosol to the chelatase by coupling this process to GTP hydrolysis.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.3c00119\", \"publication_date\": \"2023-05-10\"}, {\"work_id\": \"W4306407174\", \"doi_x\": \"10.1038/s41467-022-33792-x\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-COV-2 antibody responses to AZD1222 vaccination in West Africa\", \"abstract\": \"Real-world data on vaccine-elicited neutralising antibody responses for two-dose AZD1222 in African populations are limited. We assessed baseline SARS-CoV-2 seroprevalence and levels of protective neutralizing antibodies prior to vaccination rollout using binding antibodies analysis coupled with pseudotyped virus neutralisation assays in two cohorts from West Africa: Nigerian healthcare workers (n = 140) and a Ghanaian community cohort (n = 527) pre and post vaccination. We found 44 and 28% of pre-vaccination participants showed IgG anti-N positivity, increasing to 59 and 39% respectively with anti-receptor binding domain (RBD) IgG-specific antibodies. Previous IgG anti-N positivity significantly increased post two-dose neutralizing antibody titres in both populations. Serological evidence of breakthrough infection was observed in 8/49 (16%). Neutralising antibodies were observed to wane in both populations, especially in anti-N negative participants with an observed waning rate of 20% highlighting the need for a combination of additional markers to characterise previous infection. We conclude that AZD1222 is immunogenic in two independent West African cohorts with high background seroprevalence and incidence of breakthrough infection in 2021. Waning titres post second dose indicates the need for booster dosing after AZD1222 in the African setting despite hybrid immunity from previous infection.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-33792-x\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4296100791\", \"doi_x\": \"10.3389/fimmu.2022.953949\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"mRNA vaccination drives differential mucosal neutralizing antibody profiles in na\\u00efve and SARS-CoV-2 previously-infected individuals\", \"abstract\": \"Two doses of BNT162b2 mRNA vaccine induces a strong systemic SARS-CoV-2 specific humoral response. However, SARS-CoV-2 airborne transmission makes mucosal immune response a crucial first line of defense. Therefore, we characterized SARS-CoV-2-specific IgG responses induced by BNT162b2 vaccine, as well as IgG responses to other pathogenic and seasonal human coronaviruses in oral fluid and plasma from 200 UK healthcare workers who were na\\u00efve (N=62) or previously infected with SARS-CoV-2 (N=138) using a pan-coronavirus multiplex binding immunoassay (Meso Scale Discovery \\u00ae ). Additionally, we investigated the impact of historical SARS-CoV-2 infection on vaccine-induced IgG, IgA and neutralizing responses in selected oral fluid samples before vaccination, after a first and second dose of BNT162b2, as well as following a third dose of mRNA vaccine or breakthrough infections using the same immunoassay and an ACE2 inhibition assay. Prior to vaccination, we found that spike-specific IgG levels in oral fluid positively correlated with IgG levels in plasma from previously-infected individuals (Spearman r=0.6858, p&amp;lt;0.0001) demonstrating that oral fluid could be used as a proxy for the presence of plasma SARS-CoV-2 IgG. However, the sensitivity was lower in oral fluid (0.85, 95% CI 0.77-0.91) than in plasma (0.94, 95% CI 0.88-0.97). Similar kinetics of mucosal and systemic spike-specific IgG levels were observed following vaccination in na\\u00efve and previously-infected individuals, respectively. In addition, a significant enhancement of OC43 and HKU1 spike-specific IgG levels was observed in previously-infected individuals following one vaccine dose in oral fluid (OC43 S: p&amp;lt;0.0001; HKU1 S: p=0.0423) suggesting cross-reactive IgG responses to seasonal beta coronaviruses. Mucosal spike-specific IgA responses were induced by mRNA vaccination particularly in previously-infected individuals (71%) but less frequently in na\\u00efve participants (23%). Neutralizing responses to SARS-CoV-2 ancestral and variants of concerns were detected following vaccination in na\\u00efve and previously-infected participants, with likely contribution from both IgG and IgA in previously-infected individuals (correlations between neutralizing responses and IgG: Spearman r=0.5642, p&amp;lt;0.0001; IgA: Spearman r=0.4545, p=0.0001). We also observed that breakthrough infections or a third vaccine dose enhanced mucosal antibody levels and neutralizing responses. These data contribute to show that a previous SARS-CoV-2 infection tailors the mucosal antibody profile induced by vaccination.\", \"doi_y\": \"https://doi.org/10.3389/fimmu.2022.953949\", \"publication_date\": \"2022-09-08\"}, {\"work_id\": \"W4361255804\", \"doi_x\": \"10.1371/journal.ppat.1011283\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Characterisation of the antiviral RNA interference response to Toscana virus in sand fly cells\", \"abstract\": \"Toscana virus (TOSV) ( Bunyavirales , Phenuiviridae , Phlebovirus , Toscana phlebovirus ) and other related human pathogenic arboviruses are transmitted by phlebotomine sand flies. TOSV has been reported in nations bordering the Mediterranean Sea among other regions. Infection can result in febrile illness as well as meningitis and encephalitis. Understanding vector-arbovirus interactions is crucial to improving our knowledge of how arboviruses spread, and in this context, immune responses that control viral replication play a significant role. Extensive research has been conducted on mosquito vector immunity against arboviruses, with RNA interference (RNAi) and specifically the exogenous siRNA (exo-siRNA) pathway playing a critical role. However, the antiviral immunity of phlebotomine sand flies is less well understood. Here we were able to show that the exo-siRNA pathway is active in a Phlebotomus papatasi -derived cell line. Following TOSV infection, distinctive 21 nucleotide virus-derived small interfering RNAs (vsiRNAs) were detected. We also identified the exo-siRNA effector Ago2 in this cell line, and silencing its expression rendered the exo-siRNA pathway largely inactive. Thus, our data show that this pathway is active as an antiviral response against a sand fly transmitted bunyavirus, TOSV.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011283\", \"publication_date\": \"2023-03-30\"}, {\"work_id\": \"W4376879190\", \"doi_x\": \"10.1016/j.ejmech.2023.115500\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Small molecule-mediated targeting of microRNAs for drug discovery: Experiments, computational techniques, and disease implications\", \"abstract\": \"Small molecules have been providing medical breakthroughs for human diseases for more than a century. Recently, identifying small molecule inhibitors that target microRNAs (miRNAs) has gained importance, despite the challenges posed by labour-intensive screening experiments and the significant efforts required for medicinal chemistry optimization. Numerous experimentally-verified cases have demonstrated the potential of miRNA-targeted small molecule inhibitors for disease treatment. This new approach is grounded in their posttranscriptional regulation of the expression of disease-associated genes. Reversing dysregulated gene expression using this mechanism may help control dysfunctional pathways. Furthermore, the ongoing improvement of algorithms has allowed for the integration of computational strategies built on top of laboratory-based data, facilitating a more precise and rational design and discovery of lead compounds. To complement the use of extensive pharmacogenomics data in prioritising potential drugs, our previous work introduced a computational approach based on only molecular sequences. Moreover, various computational tools for predicting molecular interactions in biological networks using similarity-based inference techniques have been accumulated in established studies. However, there are a limited number of comprehensive reviews covering both computational and experimental drug discovery processes. In this review, we outline a cohesive overview of both biological and computational applications in miRNA-targeted drug discovery, along with their disease implications and clinical significance. Finally, utilizing drug-target interaction (DTIs) data from DrugBank, we showcase the effectiveness of deep learning for obtaining the physicochemical characterization of DTIs.\", \"doi_y\": \"https://doi.org/10.1016/j.ejmech.2023.115500\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4287218665\", \"doi_x\": \"10.3389/ffunb.2022.908343\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NIH4215: A mutation-prone thiamine auxotrophic clinical Aspergillus fumigatus isolate\", \"abstract\": \"Aspergillus fumigatus is the main cause of life-threatening invasive aspergillosis. Despite the availability of various antifungals, therapy remains challenging and requires further studies. Accordingly, the clinical A. fumigatus isolate NIH4215 deriving from a fatal case of human pulmonary aspergillosis has frequently been used in drug efficacy studies. Unexpectedly, our initial attempts to generate a bioluminescent reporter of strain NIH4215 for in vivo drug efficacy studies failed, as NIH4215 was unable to grow on defined minimal medium. Subsequent analyses discovered a previously undescribed thiamine auxotrophy of strain NIH4215 and transformation with thiamine biosynthesis genes from A. fumigatus strain Af293 identified the nmt1 gene as cause of the thiamine auxotrophy. Sequencing of the defective nmt1 gene revealed the loss of a cysteine codon within an essential iron-binding motif. Subsequently, the wild-type nmt1 gene was successfully used to generate a bioluminescent reporter strain in NIH4215 by simultaneously deleting the akuB locus. The resulting bioluminescent \\u0394 akuB strains showed a high frequency of homologous integration as confirmed by generation of pyrG and niaD deletion mutants. When tested in a Galleria mellonella infection model, neither thiamine auxotrophy nor the deletion of the akuB locus had a significant effect on virulence. However, besides thiamine auxotrophy, sectors with altered morphology and albino mutants frequently arose on colony edges of strain NIH4215 and its derivatives, and stable albino mutants were successfully isolated. A proposed increased mutation rate of NIH4215 was confirmed by screening for spontaneous occurrence of fluoorotic acid resistant mutants. Independent mutations in the pyrG and pyrE gene were identified in the fluoroorotic acid resistant NIH4215 isolates and the frequency of mutation was by at least one order of magnitude higher than that observed for the clinical A. fumigatus isolate CBS144.89. In summary, despite its virulence in animal models, strain NIH4215 is a thiamine auxotroph and prone to accumulate mutations. Our results suggest that thiamine biosynthesis is dispensable for host infection and mutation-prone strains such as NIH4215 could potentially facilitate the evolution of azole resistant strains as increasingly observed in the environment.\", \"doi_y\": \"https://doi.org/10.3389/ffunb.2022.908343\", \"publication_date\": \"2022-07-25\"}, {\"work_id\": \"W4304775444\", \"doi_x\": \"10.1101/2022.10.10.511629\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Inferring disease architecture and predictive ability with LDpred2-auto\", \"abstract\": \"Abstract LDpred2 is a widely used Bayesian method for building polygenic scores (PGS). LDpred2-auto can infer the two parameters from the LDpred model, the SNP heritability h 2 and polygenicity p , so that it does not require an additional validation dataset to choose best-performing parameters. The main aim of this paper is to properly validate the use of LDpred2-auto for inferring multiple genetic parameters. Here, we present a new version of LDpred2-auto that adds an optional third parameter \\u03b1 to its model, for modeling negative selection. We then validate the inference of these three parameters (or two, when using the previous model). We also show that LDpred2-auto provides per-variant probabilities of being causal that are well calibrated, and can therefore be used for fine-mapping purposes. We also derive a new formula to infer the out-of-sample predictive performance r 2 of the resulting PGS directly from the Gibbs sampler of LDpred2-auto. Finally, we extend the set of HapMap3 variants recommended to use with LDpred2 with 37% more variants to improve the coverage of this set, and show that this new set of variants captures 12% more heritability and provides 6% more predictive performance, on average, in UK Biobank analyses.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.10.511629\", \"publication_date\": \"2022-10-12\"}, {\"work_id\": \"W4362457738\", \"doi_x\": \"10.1146/annurev-biochem-052521-040313\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Inseparable Relationship Between Cholesterol and Hedgehog Signaling\", \"abstract\": \"Ligands of the Hedgehog (HH) pathway are paracrine signaling molecules that coordinate tissue development in metazoans. A remarkable feature of HH signaling is the repeated use of cholesterol in steps spanning ligand biogenesis, secretion, dispersal, and reception on target cells. A cholesterol molecule covalently attached to HH ligands is used as a molecular baton by transfer proteins to guide their secretion, spread, and reception. On target cells, a signaling circuit composed of a cholesterol transporter and sensor regulates transmission of HH signals across the plasma membrane to the cytoplasm. The repeated use of cholesterol in signaling supports the view that the HH pathway likely evolved by coopting ancient systems to regulate the abundance or organization of sterol-like lipids in membranes.\", \"doi_y\": \"https://doi.org/10.1146/annurev-biochem-052521-040313\", \"publication_date\": \"2023-06-20\"}, {\"work_id\": \"W4311529002\", \"doi_x\": \"10.1101/2022.12.02.518813\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protective human IgE responses are promoted by comparable life-cycle dependent Tegument Allergen-Like protein expression in<i>Schistosoma haematobium</i>and<i>Schistosoma mansoni</i>infection\", \"abstract\": \"Abstract Schistosoma haematobium is the most prevalent of the human-infecting schistosome species, causing significant morbidity in endemically exposed populations. Despite this, it has been relatively understudied compared to its fellow species, S. mansoni . Here we provide the first comprehensive characterization of the S. haematobium Tegument Allergen-Like protein family, a key protein family directly linked to protective immunity in S. mansoni infection. Comparable with observations for S. mansoni , parasite phylogenetic analysis and relative gene expression combined with host serological analysis support a cross-reactive relationship between S. haematobium TAL proteins, exposed to the host immune system as adult worms die, and closely related proteins, exposed during penetration by the infecting cercarial and early schistosomulae stages. Specifically, our results strengthen the evidence for host immunity driven by cross-reactivity between family members TAL3 and TAL5, establishing it for the first time for S. haematobium infection. Furthermore, we build upon this relationship to include the involvement of an additional member of the TAL protein family, TAL11 for both schistosome species. Finally, we show a close association between experience of infection and intensity of transmission and the development of protective IgE responses to these antigens, thus improving our knowledge of the mechanisms by which protective host immune responses develop. This knowledge will be critical in understanding how control efforts such as mass drug administration campaigns influence the development of host immunity and subsequent patterns of infection and disease within endemic populations. Author Summary S. haematobium is the most prevalent of the human infecting schistosomes. Along with S. mansoni , it is responsible for the majority of schistosomiasis cases that are borne by the populations of sub-Saharan Africa, where the global burden of this infection is centered. Here, we provide insight into the IgE antibody response that protects against these infections. Through utilization of in silico analysis and transcriptional studies of parasite life stages, in combination with immuno-epidemiological studies, we explore the relationship between host immune protection and a parasite protein family named the Tegument Allergen-Like (TAL) proteins. Our results show that several members of the TAL protein family are important in host protection to both these major schistosome species. For the first time we demonstrate that a progressive cross-reactive TAL-IgE response occurs against S. haematobium , similar to that previous observed in S. mansoni infection. We additionally expand upon previous knowledge for S. mansoni , identifying further complexity in the cross-reactive relationship between TAL family members, providing evidence of a key role for family member TAL11 in induction of the protective host immune response.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.02.518813\", \"publication_date\": \"2022-12-05\"}, {\"work_id\": \"W4376643492\", \"doi_x\": \"10.1073/pnas.2302584120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The SHDRA syndrome-associated gene <i>TMEM260</i> encodes a protein-specific O-mannosyltransferase\", \"abstract\": \"Mutations in the TMEM260 gene cause structural heart defects and renal anomalies syndrome, but the function of the encoded protein remains unknown. We previously reported wide occurrence of O-mannose glycans on extracellular immunoglobulin, plexin, transcription factor (IPT) domains found in the hepatocyte growth factor receptor (cMET), macrophage-stimulating protein receptor (RON), and plexin receptors, and further demonstrated that two known protein O-mannosylation systems orchestrated by the POMT1/2 and transmembrane and tetratricopeptide repeat-containing proteins 1-4 gene families were not required for glycosylation of these IPT domains. Here, we report that the TMEM260 gene encodes an ER-located protein O-mannosyltransferase that selectively glycosylates IPT domains. We demonstrate that disease-causing TMEM260 mutations impair O-mannosylation of IPT domains and that TMEM260 knockout in cells results in receptor maturation defects and abnormal growth of 3D cell models. Thus, our study identifies the third protein-specific O-mannosylation pathway in mammals and demonstrates that O-mannosylation of IPT domains serves critical functions during epithelial morphogenesis. Our findings add a new glycosylation pathway and gene to a growing group of congenital disorders of glycosylation.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2302584120\", \"publication_date\": \"2023-05-15\"}, {\"work_id\": \"W4386764235\", \"doi_x\": \"10.1021/acs.jpcb.3c03887\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Asymmetric Periodic Boundary Conditions for All-Atom Molecular Dynamics and Coarse-Grained Simulations of Nucleic Acids\", \"abstract\": \"Periodic boundary conditions are commonly applied in molecular dynamics simulations in the microcanonical (NVE), canonical (NVT), and isothermal-isobaric (NpT) ensembles. In their simplest application, a biological system of interest is placed in the middle of a solvation box, which is chosen 'sufficiently large' to minimize any numerical artifacts associated with the periodic boundary conditions. This practical approach brings limitations to the size of biological systems that can be simulated. Here, we study simulations of effectively infinitely long nucleic acids, which are solvated in the directions perpendicular to the polymer chain, while periodic boundary conditions are also applied along the polymer chain. We study the effects of these asymmetric periodic boundary conditions (APBC) on the simulated results, including the mechanical properties of biopolymers and the properties of the surrounding solvent. To get some further insights into the advantages of using the APBC, a coarse-grained worm-like chain model is first studied, illustrating how the persistence length can be extracted from the local properties of the polymer chain, which are less affected by the APBC than some global averages. This is followed by all-atom molecular dynamics simulations of DNA in ionic solutions, where we use the APBC to investigate sequence-dependent properties of DNA molecules and properties of the surrounding solvent.\", \"doi_y\": \"https://doi.org/10.1021/acs.jpcb.3c03887\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4303184143\", \"doi_x\": \"10.1093/bioadv/vbac072\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Reciprocal best structure hits: using AlphaFold models to discover distant homologues\", \"abstract\": \"Abstract Motivation The conventional methods to detect homologous protein pairs use the comparison of protein sequences. But the sequences of two homologous proteins may diverge significantly and consequently may be undetectable by standard approaches. The release of the AlphaFold 2.0 software enables the prediction of highly accurate protein structures and opens many opportunities to advance our understanding of protein functions, including the detection of homologous protein structure pairs. Results In this proof-of-concept work, we search for the closest homologous protein pairs using the structure models of five model organisms from the AlphaFold database. We compare the results with homologous protein pairs detected by their sequence similarity and show that the structural matching approach finds a similar set of results. In addition, we detect potential novel homologs solely with the structural matching approach, which can help to understand the function of uncharacterized proteins and make previously overlooked connections between well-characterized proteins. We also observe limitations of our implementation of the structure-based approach, particularly when handling highly disordered proteins or short protein structures. Our work shows that high accuracy protein structure models can be used to discover homologous protein pairs, and we expose areas for improvement of this structural matching approach. Availability and Implementation Information to the discovered homologous protein pairs can be found at the following URL: https://doi.org/10.17863/CAM.87873. The code can be accessed here: https://github.com/VivianMonzon/Reciprocal_Best_Structure_Hits. Supplementary information Supplementary data are available at Bioinformatics Advances online.\", \"doi_y\": \"https://doi.org/10.1093/bioadv/vbac072\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4284965432\", \"doi_x\": \"10.1016/j.gde.2022.101946\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Ex situ-armus: experimental models for combating respiratory dysfunction\", \"abstract\": \"Ex situ experimental models have become a main stay in pulmonary research. Organoids and explant systems have uncovered novel stem cell subsets, served as disease models, delineated cell fate transitions, and aided high throughput pre-clinical drug screening. Integration of gene-editing and bioengineering approaches have further generated novel avenues for regenerative medicine and transplantation strategies. In this article, we highlight recent studies, aided by ex situ systems, which have contributed to significant advances in our understanding of the human lower respiratory tract. We present key observations from these studies to gain improved insights into human disease. We conclude this article with a summary of existing challenges and potential technological advances to successfully mirror human tissue physiology.\", \"doi_y\": \"https://doi.org/10.1016/j.gde.2022.101946\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4311204813\", \"doi_x\": \"10.1042/ebc20220190\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Structural mass spectrometry approaches to understand multidrug efflux systems\", \"abstract\": \"Multidrug efflux pumps are ubiquitous across both eukaryotes and prokaryotes, and have major implications in antimicrobial and multidrug resistance. They reside within cellular membranes and have proven difficult to study owing to their hydrophobic character and relationship with their compositionally complex lipid environment. Advances in structural mass spectrometry (MS) techniques have made it possible to study these systems to elucidate critical information on their structure-function relationships. For example, MS techniques can report on protein structural dynamics, stoichiometry, connectivity, solvent accessibility, and binding interactions with ligands, lipids, and other proteins. This information proving powerful when used in conjunction with complementary structural biology methods and molecular dynamics (MD) simulations. In the present review, aimed at those not experts in MS techniques, we report on the current uses of MS in studying multidrug efflux systems, practical considerations to consider, and the future direction of the field. In the first section, we highlight the importance of studying multidrug efflux proteins, and introduce a range of different MS techniques and explain what information they yield. In the second section, we review recent studies that have utilised MS techniques to study and characterise a range of different multidrug efflux systems.\", \"doi_y\": \"https://doi.org/10.1042/ebc20220190\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4316096804\", \"doi_x\": \"10.1038/s42003-023-04443-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A genome-wide association study identifies distinct variants associated with pulmonary function among European and African ancestries from the UK Biobank\", \"abstract\": \"Pulmonary function is an indicator of well-being, and pulmonary pathologies are the third major cause of death worldwide. We analysed the UK Biobank genome-wide association summary statistics of pulmonary function for Europeans and individuals of recent African descent to identify variants associated with the trait in the two ancestries. Here, we show 627 variants in Europeans and 3 in Africans associated with three pulmonary function parameters. In addition to the 110 variants in Europeans previously reported to be associated with phenotypes related to pulmonary function, we identify 279 novel loci, including an ISX intergenic variant rs369476290 on chromosome 22 in Africans. Remarkably, we find no shared variants among Africans and Europeans. Furthermore, enrichment analyses of variants separately for each ancestry background reveal significant enrichment for terms related to pulmonary phenotypes in Europeans but not Africans. Further analysis of studies of pulmonary phenotypes reveals that individuals of European background are disproportionally overrepresented in datasets compared to Africans, with the gap widening over the past five years. Our findings extend our understanding of the different variants that modify the pulmonary function in Africans and Europeans, a promising finding for future GWASs and medical studies.\", \"doi_y\": \"https://doi.org/10.1038/s42003-023-04443-8\", \"publication_date\": \"2023-01-14\"}, {\"work_id\": \"W4382137668\", \"doi_x\": \"10.3390/ijms241310613\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Energy Landscapes and Heat Capacity Signatures for Monomers and Dimers of Amyloid-Forming Hexapeptides\", \"abstract\": \"Amyloid formation is a hallmark of various neurodegenerative disorders. In this contribution, energy landscapes are explored for various hexapeptides that are known to form amyloids. Heat capacity (CV) analysis at low temperature for these hexapeptides reveals that the low energy structures contributing to the first heat capacity feature above a threshold temperature exhibit a variety of backbone conformations for amyloid-forming monomers. The corresponding control sequences do not exhibit such structural polymorphism, as diagnosed via end-to-end distance and a dihedral angle defined for the monomer. A similar heat capacity analysis for dimer conformations obtained using basin-hopping global optimisation shows clear features in end-to-end distance versus dihedral correlation plots, where amyloid-forming sequences exhibit a preference for larger end-to-end distances and larger positive dihedrals. These results hold true for sequences taken from tau, amylin, insulin A chain, a de novo designed peptide, and various control sequences. While there is a little overall correlation between the aggregation propensity and the temperature at which the low-temperature CV feature occurs, further analysis suggests that the amyloid-forming sequences exhibit the key CV feature at a lower temperature compared to control sequences derived from the same protein.\", \"doi_y\": \"https://doi.org/10.3390/ijms241310613\", \"publication_date\": \"2023-06-25\"}, {\"work_id\": \"W4382982445\", \"doi_x\": \"10.3390/toxins15070430\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Dynamics and Molecular Interactions of GPI-Anchored CD59\", \"abstract\": \"CD59 is a GPI-anchored cell surface receptor that serves as a gatekeeper to controlling pore formation. It is the only membrane-bound inhibitor of the complement membrane attack complex (MAC), an immune pore that can damage human cells. While CD59 blocks MAC pores, the receptor is co-opted by bacterial pore-forming proteins to target human cells. Recent structures of CD59 in complexes with binding partners showed dramatic differences in the orientation of its ectodomain relative to the membrane. Here, we show how GPI-anchored CD59 can satisfy this diversity in binding modes. We present a PyLipID analysis of coarse-grain molecular dynamics simulations of a CD59-inhibited MAC to reveal residues of complement proteins (C6:Y285, C6:R407 C6:K412, C7:F224, C8\\u03b2:F202, C8\\u03b2:K326) that likely interact with lipids. Using modules of the MDAnalysis package to investigate atomistic simulations of GPI-anchored CD59, we discover properties of CD59 that encode the flexibility necessary to bind both complement proteins and bacterial virulence factors.\", \"doi_y\": \"https://doi.org/10.3390/toxins15070430\", \"publication_date\": \"2023-06-30\"}, {\"work_id\": \"W4313644976\", \"doi_x\": \"10.1101/2023.01.05.522861\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"ChemGAPP; A Package for Chemical Genomics Analysis and Phenotypic Profiling\", \"abstract\": \"Abstract High-throughput chemical genomic screens produce informative datasets, providing valuable insights into unknown gene function on a genome-wide level. However, there is currently no comprehensive analytic package publicly available. We developed and benchmarked ChemGAPP to bridge this gap. ChemGAPP allows integration of various steps in a streamlined and user-friendly format, including rigorous quality control measures to curate screening data. ChemGAPP provides three sub-packages for different chemical-genomic screens: ChemGAPP Big for handling large-scale high-throughput screens; ChemGAPP Small, designed for small-scale screen analysis and ChemGAPP GI for genetic interaction screen analysis. ChemGAPP is available at https://github.com/HannahMDoherty/ChemGAPP .\", \"doi_y\": \"https://doi.org/10.1101/2023.01.05.522861\", \"publication_date\": \"2023-01-06\"}, {\"work_id\": \"W4384500170\", \"doi_x\": \"10.1371/journal.pbio.3001815\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Attenuation hotspots in neurotropic human astroviruses\", \"abstract\": \"During the last decade, the detection of neurotropic astroviruses has increased dramatically. The MLB genogroup of astroviruses represents a genetically distinct group of zoonotic astroviruses associated with gastroenteritis and severe neurological complications in young children, the immunocompromised, and the elderly. Using different virus evolution approaches, we identified dispensable regions in the 3' end of the capsid-coding region responsible for attenuation of MLB astroviruses in susceptible cell lines. To create recombinant viruses with identified deletions, MLB reverse genetics (RG) and replicon systems were developed. Recombinant truncated MLB viruses resulted in imbalanced RNA synthesis and strong attenuation in iPSC-derived neuronal cultures confirming the location of neurotropism determinants. This approach can be used for the development of vaccine candidates using attenuated astroviruses that infect humans, livestock animals, and poultry.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001815\", \"publication_date\": \"2023-07-17\"}, {\"work_id\": \"W4284712597\", \"doi_x\": \"10.1016/j.pt.2022.06.002\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Improving helminth genome resources in the post-genomic era\", \"abstract\": \"Genomics resources for helminth parasites are growing rapidly; however, it is still difficult to fully assemble a genome so most genome resources vary significantly in completeness and contiguity.Genome annotations rely heavily on computational prediction and shared orthology between closely (and sometimes not so closely) related species, which can result in incomplete and incorrect annotations and may bias against some genes, such as those that are species-specific or phylogenetically restricted.New approaches are required to generate, improve, and extend these valuable resources.Community-based efforts could build on and exploit different areas of specific expertise, from single genes to whole gene families, to provide iterative improvement of centralised resources.Coordinated annotation will provide new opportunities for interaction and collaboration among the parasitology community. Rapid advancement in high-throughput sequencing and analytical approaches has seen a steady increase in the generation of genomic resources for helminth parasites. Now, helminth genomes and their annotations are a cornerstone of numerous efforts to compare genetic and transcriptomic variation, from single cells to populations of globally distributed parasites, to genome modifications to understand gene function. Our understanding of helminths is increasingly reliant on these genomic resources, which are primarily static once published and vary widely in quality and completeness between species. This article seeks to highlight the cause and effect of this variation and argues for the continued improvement of these genomic resources \\u2013 even after their publication \\u2013 which is necessary to provide a more accurate and complete understanding of the biology of these important pathogens. Rapid advancement in high-throughput sequencing and analytical approaches has seen a steady increase in the generation of genomic resources for helminth parasites. Now, helminth genomes and their annotations are a cornerstone of numerous efforts to compare genetic and transcriptomic variation, from single cells to populations of globally distributed parasites, to genome modifications to understand gene function. Our understanding of helminths is increasingly reliant on these genomic resources, which are primarily static once published and vary widely in quality and completeness between species. This article seeks to highlight the cause and effect of this variation and argues for the continued improvement of these genomic resources \\u2013 even after their publication \\u2013 which is necessary to provide a more accurate and complete understanding of the biology of these important pathogens. In 2022, the first gapless human genome assembly was published [1.Nurk S. et al.The complete sequence of a human genome.Science. 2022; 376: 44-53Crossref PubMed Scopus (168) Google Scholar], a little more than 20 years after the landmark completion of the draft assembly was announced [2.International Human Genome Sequencing Consortium Finishing the euchromatic sequence of the human genome.Nature. 2004; 431: 931-945Crossref PubMed Scopus (3472) Google Scholar]. This publicly available resource \\u2013 the result of multinational collaborative efforts \\u2013 is arguably one of our most important scientific achievements. The impact of this project extends far beyond the genome sequence itself; technological and analytical advancement made during and after the Human Genome Project has driven the ever-increasing accessibility of \\u2018genomics\\u2019 \\u2013 the study of the structure, composition, function, and evolution of genomes \\u2013 to other, nonhuman species. Such advances now allow genome-sequencing projects to be undertaken all over the world on potentially any organism with accessible DNA, from routine experiments found in any modest molecular biology laboratory to sequencing in extreme environments [3.Castro-Wallace S.L. et al.Nanopore DNA sequencing and genome assembly on the International Space Station.Sci. Rep. 2017; 7: 18022Crossref PubMed Scopus (177) Google Scholar] or in response to disease [4.Volz E. et al.Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England.Nature. 2021; 593: 266-269Crossref PubMed Scopus (463) Google Scholar], to cataloguing the genomes of all life on Earth [5.Lewin H.A. et al.Earth BioGenome Project: Sequencing life for the future of life.Proc. Natl. Acad. Sci. U. S. A. 2018; 115: 4325-4333Crossref PubMed Scopus (352) Google Scholar]. The study of parasitic worms, collectively termed helminths, has benefited significantly from this genomic revolution. Often considered \\u2018neglected\\u2019 parasites, helminths affect over a billion people living in some of the lowest-resource countries worldwide and significantly impact global livestock and agriculture industries. Despite this neglect, several sustained efforts have been made to generate genomic resources for helminths, with broad aims including the understanding of parasite biology and life-history traits [6.International Helminth Genomes Consortium Comparative genomics of the major parasitic worms.Nat. Genet. 2019; 51: 163-174Crossref PubMed Scopus (216) Google Scholar], mining new drug targets [7.Cotton J.A. et al.The genome of Onchocerca volvulus, agent of river blindness.Nat. Microbiol. 2016; 2: 16216Crossref PubMed Scopus (62) Google Scholar], and characterising genetic traits such as drug resistance [8.Doyle S.R. Cotton J.A. Genome-wide approaches to investigate anthelmintic resistance.Trends Parasitol. 2019; 35: 289-301Abstract Full Text Full Text PDF PubMed Scopus (37) Google Scholar]. The principal repository for these data, WormBase ParaSite [9.Howe K.L. et al.WormBase ParaSite \\u2212 a comprehensive resource for helminth genomics.Mol. Biochem. Parasitol. 2017; 215: 2-10Crossref PubMed Scopus (277) Google Scholar], now contains 210 genomes and annotations (see Glossary) from 169 species of nematodes and flatworms (WormBase ParaSite release 17), representing some of the most pathogenic helminth species of humans, animals, and plants as well as closely related species for comparison. These genome resources are the starting point of many molecular and computational experiments and form the scaffold upon which many \\u2018omic experiments and analyses rely. Although intentionally provocative and somewhat oversimplified, there is some truth to this statement; a genome assembly and its annotation are a hypothesis of the structure and content of the genomic DNA within an organism from which they are derived but, despite best efforts, rarely resemble the complete product. For example, biologically, a genome is defined as all of the DNA within a cell and is found assembled into discrete chromosomes but, more often than not, genomes are represented informatically by contigs and scaffolds that number in the thousands or more and vary significantly in their \\u2018completeness\\u2019 (see Figure 1 for a comparison of helminth genome assemblies and BUSCO [10.Manni M. et al.BUSCO Update: novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes.Mol. Biol. Evol. 2021; 38: 4647-4654Crossref PubMed Scopus (283) Google Scholar] completeness available from WormBase ParaSite release 17; 80.9% of genome assemblies are in 1000 or more contigs/scaffolds). Despite recent technological and analytical advancements that make it cheaper, easier, and more likely to produce highly contiguous and even chromosome-scale assemblies, it is still challenging to generate finished genomes for eukaryotic species such as helminths. A key challenge is that there is not one molecular/sequencing/analytical approach optimised for all organisms. Although helminth genomes are not considered to be large or particularly complex for a eukaryotic organism (the median genome assembly size is 122.18 Mb, and range from 38.18 Mb for Meloidogyne graminicola to 1.25 Gb for Spirometra erinaceieuropaei), helminth-focused genome projects are further challenged by a range of factors. These challenges include: (i) vast differences in morphology and tissue composition, both within species (i.e., between life stages) and between species, may require different approaches to DNA extraction and lead to differences in extraction efficiencies [11.Ayana M. et al.Comparison of four DNA extraction and three preservation protocols for the molecular detection and quantification of soil-transmitted helminths in stool.PLoS Negl. Trop. Dis. 2019; 13e0007778Google Scholar,12.Doyle S.R. et al.Evaluation of DNA extraction methods on individual helminth egg and larval stages for whole-genome sequencing.Front. Genet. 2019; 10: 826Crossref PubMed Scopus (17) Google Scholar]; (ii) the inaccessibility of parasite DNA, for example, larger mature life stages with more DNA are also likely to be in vivo parasitic stages, whereas for some species, accessible juvenile stages such as eggs and larvae are typically microscopic and may be environmentally resistant, further limiting DNA extraction yield [12]; (iii) further inaccessibility to DNA based on parasite host and geographic location, for example, despite infecting over a billion people worldwide, mature stages of human-infective helminths can be incredibly difficult to access particularly in low-resource settings where they are most commonly found [13.Doyle S.R. et al.Population genomics of ancient and modern Trichuris trichiura.bioRxiv. 2021; https://doi.org/10.1101/2021.10.21.464505Crossref PubMed Scopus (0) Google Scholar], and even if available, it can be difficult to preserve high-quality samples (tissue or DNA) if access to a laboratory or cold-chain is limited; and (iv) the high degree of genetic heterogeneity among individuals of many helminth species which, if pooling of samples is required to generate sufficient DNA for sequencing, can lead to assembly artefacts [14.Doyle S.R. et al.Genomic and transcriptomic variation defines the chromosome-scale assembly of Haemonchus contortus, a model gastrointestinal worm.Commun. Biol. 2020; 3: 656Crossref PubMed Scopus (36) Google Scholar]. Some of these challenges can be addressed and potentially overcome by specific technological developments that now allow very low-input DNA concentrations for sequencing [15.Kingan S.B. et al.a high-quality de novo genome assembly from a single mosquito using pacbio sequencing.Genes. 2019; 10: 62Crossref Scopus (66) Google Scholar], particularly if an individual helminth specimen can be sequenced. Nonetheless, one or more of these issues will have undoubtedly limited the ability of each of the existing helminth genome assembly projects to achieve a contiguous reference genome. The genome assembly is only one part of the puzzle. Once an assembly is \\u2018finished\\u2019, efforts are then made to determine the coordinates of coding, noncoding and other feature content of the genome, which is often followed by an attempt to identify and characterise the function of these annotations. Several computational pipelines have been developed for annotating helminth genomes (for example, see [6.International Helminth Genomes Consortium Comparative genomics of the major parasitic worms.Nat. Genet. 2019; 51: 163-174Crossref PubMed Scopus (216) Google Scholar] for a description of the 50 Helminths Genome Consortium pipeline, applied at scale). In contrast, other approaches use one or a combination of annotation tools (for example, Augustus [16.Hoff K.J. Stanke M. Predicting genes in single genomes with AUGUSTUS.Curr. Protoc. Bioinform. 2019; 65e57PubMed Google Scholar], Braker [17.Br\\u016fna T. et al.BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database.NAR Genom. Bioinform. 2021; 3lqaa108PubMed Google Scholar], Maker [18.Holt C. Yandell M. MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.BMC Bioinform. 2011; 12: 491Crossref PubMed Scopus (987) Google Scholar], PASA [19.Haas B.J. et al.Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.Nucleic Acids Res. 2003; 31: 5654-5666Crossref PubMed Scopus (950) Google Scholar]) which can use and combine various additional datasets, for example, RNA-seq data or homologous protein sequences from a closely related species, as evidence to identify putative features within the genome. The quality of these predictions is highly dependent on the quality of the assembly, the type and amount of evidence used, and how representative that evidence is of the features in the genome. For example, mapped RNA-seq data are broadly considered to be good evidence to identify intron\\u2013exon splice junctions of gene models; however, there is a bias towards genes that are more highly expressed, whereas lowly expressed genes or genes not expressed in the life stage/condition that the RNA-seq data were derived from will not be predicted as reliably. While many of the computational prediction pipelines will perform well, given sufficient evidence, even under an ideal scenario there will still be errors in the annotation [20.Salzberg S.L. Next-generation genome annotation: we still struggle to get it right.Genome Biol. 2019; 20: 92Crossref PubMed Scopus (105) Google Scholar]. Many of these errors will not be evident until an annotation is manually inspected, gene-by-gene, using available evidence in a genome browser; given that the median coding gene content of helminth genomes is 16 380 features (ranging from 8140 coding genes for Onchocerca flexuosa to 101 269 for Meloidogyne arenaria), this is not a trivial task. Unfortunately, most genome annotations available will not have been generated under an ideal scenario, and even fewer will be thoroughly interrogated manually. Examples of genome annotation errors include gene models that may be fragmented or multiple genes incorrectly merged, genes broken by technical errors such as fragmented assemblies or sequencing errors that produce stop codons or frameshifts, or missing genes due to a lack of evidence or divergence from the training dataset. Such errors complicate analyses of orthology typically used to define evolutionary and putative functional relationships of genes within and between species, and likely confounds estimates of shared and species-specific annotations [21.Weisman C.M. et al.Mixing genome annotation methods in a comparative analysis inflates the apparent number of lineage-specific genes.Curr. Biol. 2022; 32: 1-8Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar]. Many examples in the literature describe the cloning and validation of a gene structure in response to identifying an error in a gene model from a genomic resource; unfortunately, these updated experimentally validated models are rarely used to correct the original resource. Further challenges are associated with the functional characterisation of gene models of non-model organisms such as helminths. It has recently been estimated that over 47% of gene families lack any functional annotation [6.International Helminth Genomes Consortium Comparative genomics of the major parasitic worms.Nat. Genet. 2019; 51: 163-174Crossref PubMed Scopus (216) Google Scholar], rendering many as \\u2018hypothetical\\u2019 [22.International Molecular Helminthology Annotation Network (IMHAN) et al.Tackling hypotheticals in helminth genomes.Trends Parasitol. 2018; 34: 179-183Abstract Full Text Full Text PDF PubMed Scopus (17) Google Scholar], and those that do have functional information will have relied heavily on computation inference based on homology with a closely related species. Such information, or lack thereof, can make interpreting genomic data difficult; analyses that point to a specific region of interest within the genome may reveal a list of genes with no known function, whereas approaches such as gene set enrichment will be biased towards genes with known functions and common gene ontologies. Because incorrect annotations are not immediately obvious, particularly at scale, research will repeatedly be based on the original genome resource and, consequently, not benefit from new information identified. Furthermore, even if the limitations are known, the effort required to update these data by an individual is large and, therefore, impractical, especially for genome-wide analyses that rely on the whole genome and annotation. More genomics data are not necessarily the solution, and the direct study of gene function \\u2013 particularly of genes with novel, parasite-specific roles \\u2013 is still needed [23.Viney M. The failure of genomics in biology.Trends Parasitol. 2014; 30: 319-321Abstract Full Text Full Text PDF PubMed Scopus (12) Google Scholar]. Such analyses of gene function, including deciphering the functional role of hypothetical proteins, may be supported or even enhanced by novel computational approaches such as 3D structure predictions by AlphaFold [24.Jumper J. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (4221) Google Scholar]. However, without improvements in the coordination and feedback of information between the functional validation of genes and the genomes from which they are based, progress towards a better understanding of helminth functional biology will ultimately be limited. Now, in the \\u2018post-genomic\\u2019 era, the rate and scale of new data generated for helminths are steadily growing [25.McVeigh P. Post-genomic progress in helminth parasitology.Parasitology. 2020; 147: 835-840Crossref PubMed Scopus (13) Google Scholar]. Genome-led analyses now include, for example, characterising transcription at single-cell [26.Diaz Soria C.L. et al.Single-cell atlas of the first intra-mammalian developmental stage of the human parasite Schistosoma mansoni.Nat. Commun. 2020; 11: 6411Crossref PubMed Scopus (22) Google Scholar, 27.Wendt G. et al.A single-cell RNA-seq atlas of Schistosoma mansoni identifies a key regulator of blood feeding.Science. 2020; 369: 1644-1649Crossref PubMed Scopus (48) Google Scholar, 28.Li P. et al.Single-cell analysis of Schistosoma mansoni identifies a conserved genetic program controlling germline stem cell fate.Nat. Commun. 2021; 12: 485Crossref PubMed Scopus (14) Google Scholar], temporal [29.Rawlinson K.A. et al.Daily rhythms in gene expression of the human parasite Schistosoma mansoni.BMC Biol. 2021; 19: 255Crossref PubMed Scopus (5) Google Scholar], and spatially defined resolutions [30.Airs P.M. et al.Spatial transcriptomics reveals antiparasitic targets associated with essential behaviors in the human parasite Brugia malayi.PLoS Pathog. 2022; 18e1010399Crossref PubMed Scopus (2) Google Scholar], large-scale RNAi screens to understand gene function and reveal novel drug targets [31.Wang J. et al.Large-scale RNAi screening uncovers therapeutic targets in the parasite Schistosoma mansoni.Science. 2020; 369: 1649-1653Crossref PubMed Scopus (28) Google Scholar], extensive gene family characterisation, for example, chemoreceptors [32.Wheeler N.J. et al.Genetic and functional diversification of chemosensory pathway receptors in mosquito-borne filarial nematodes.PLoS Biol. 2020; 18e3000723Crossref PubMed Scopus (17) Google Scholar] or small noncoding RNAs [33.Fontenla S. et al.Lost and found: Piwi and Argonaute pathways in flatworms.Front. Cell. Infect. Microbiol. 2021; 11653695Crossref Scopus (4) Google Scholar], and targeted genome modification and transgenesis are becoming more efficient in a range of species [34.Gang S.S. et al.Targeted mutagenesis in a human-parasitic nematode.PLoS Pathog. 2017; 13e1006675Crossref Scopus (69) Google Scholar, 35.Ittiprasert W. et al.Programmed genome editing of the omega-1 ribonuclease of the blood fluke, Schistosoma mansoni.eLife. 2019; 8e41337Crossref Google Scholar, 36.Arunsan P. et al.Programmed knockout mutation of liver fluke granulin attenuates virulence of infection-induced hepatobiliary morbidity.eLife. 2019; 8e41463Crossref PubMed Scopus (40) Google Scholar, 37.Sankaranarayanan G. et al.Large CRISPR-Cas-induced deletions in the oxamniquine resistance locus of the human parasite.Wellcome Open Res. 2020; 5: 178Crossref PubMed Scopus (11) Google Scholar]. These genomic analyses are also increasingly being complemented by other \\u2018omic approaches such as proteomics [38.Wang T. et al.Somatic proteome of Haemonchus contortus.Int. J. Parasitol. 2019; 49: 311-320Crossref PubMed Scopus (27) Google Scholar], peptidomics [39.Buzy A. et al.Peptidomics of Haemonchus contortus.ACS Omega. 2021; 6: 10288-10305Crossref PubMed Scopus (2) Google Scholar], metabolomics [40.Kokova D. Mayboroda O.A. Twenty years on: metabolomics in helminth research.Trends Parasitol. 2019; 35: 282-288Abstract Full Text Full Text PDF PubMed Scopus (13) Google Scholar], and lipidomics [41.Wang T. et al.Helminth lipidomics: technical aspects and future prospects.Curr. Res. Parasitol. Vector Borne Dis. 2021; 1100018Google Scholar]. However, generating \\u2018omics data that heavily rely on existing genomics resources is potentially problematic. The first problem is that many approaches are based on the assumption that the reference genome and annotation are essentially \\u2018correct\\u2019 (see the previous subheading); if one or both are incomplete or erroneous, then biological insights will surely be missed or, worse, misinterpreted. The second problem is that these information-rich datasets have significant potential to add value to the reference genome and annotation but rarely, if ever, are used to do so. A genome and annotation are largely static resources that, once published, do not change; new information generated \\u2013 whether primary data or a synthesis of literature \\u2013 remain distinct and are not typically used to improve or enrich the genome resource upon which they are based. Addressing these two points has been the focus of research efforts for a limited number of helminth species, which have seen iterative improvement in the genomic resources over time from fragmented draft assemblies to chromosome-scale and reference-quality assemblies (Figure 2). These improvements have led to fundamental new insights into parasite biology that could not have been addressed using a draft genome, including the resolution of drug resistance QTL in Haemonchus contortus [42.Doyle S.R. et al.Population genomic and evolutionary modelling analyses reveal a single major QTL for ivermectin drug resistance in the pathogenic nematode, Haemonchus contortus.BMC Genom. 2019; 20: 218Crossref PubMed Scopus (37) Google Scholar,43.Doyle S.R. et al.Genomic landscape of drug response reveals novel mediators of anthelmintic resistance.bioRxiv. 2022; https://doi.org/10.1101/2021.11.12.465712Crossref Scopus (0) Google Scholar], the evolution of sex chromosomes of Brugia malayi (and related filarial species) [44.Foster J.M. et al.Sex chromosome evolution in parasitic nematodes of humans.Nat. Commun. 2020; 11: 1964Crossref PubMed Scopus (17) Google Scholar] and delineation of separate Z and W sex chromosomes of Schistosoma mansoni [45.Buddenborg S.K. et al.Assembled chromosomes of the blood fluke Schistosoma mansoni provide insight into the evolution of its ZW sex-determination system.bioRxiv. 2021; https://doi.org/10.1101/2021.08.13.456314Crossref Scopus (0) Google Scholar], and the unusual centromere and telomere structures found at the ends of chromosomes in Hymenolepis microstoma [46.Olson P.D. et al.Complete representation of a tapeworm genome reveals chromosomes capped by centromeres, necessitating a dual role in segregation and protection.BMC Biol. 2020; 18: 165Crossref PubMed Scopus (11) Google Scholar]. These examples demonstrate that improvements to existing resources can be achieved with measurable effects and novel insights made. Therefore, the question remains: how could (or should) this be done more broadly, and by whom? (Box 1 outlines the access, synthesis, and extension of genomic resources for helminths). The examples of genome improvement, from draft to a complete representation of the genome and associated data (as described above and in Figure 2), have been achieved by discrete research groups working over many years and often motivated by a specific research question. For a limited number of helminth genomes \\u2013 currently, B. malayi, Onchocerca volvulus, Strongyloides ratti, and Trichuris muris \\u2013 dedicated manual curation of their annotations are ongoing as part of the WormBase core genome group (which also includes Caenorhabditis elegans, and other Caenorhabditis spp., and Pristionchus pacificus), or from individual curation efforts (Box 2 describes the manual curation of the H. contortus genome annotation). For the remaining 95% of genomes in WormBase ParaSite, the scope for improvement seems less certain, especially given that relying on dedicated curators is unlikely to scale for the large number and diversity of species for which curation is required. However, the communities that use these resources are far greater in size and cumulative expertise; although communities contribute knowledge via the publication of research based on the genome resource, this knowledge remains separate from the genome resource itself. This is a missed opportunity that needs to be taken advantage of. What is currently lacking are the mechanisms by which new data and knowledge can easily be transferred and integrated into dynamic resources that can change and improve over time. Such a mechanism should function at both small and large scales, whether curating the structure of a single-gene model or describing the function of a whole gene family or more. Although such a mechanism, when considered as a final product, may seem complex and therefore unachievable, there are already easy-to-use tools available for collaborative curation (e.g., Web Apollo [47.Dunn N.A. et al.Apollo: democratizing genome annotation.PLoS Comput. Biol. 2019; 15e1006790Crossref PubMed Scopus (99) Google Scholar]) and initiatives underway that exploit the use of both experts and non-experts alike in the shared goal of improving genome resources for specific nematode species (see Box 2 for examples of initiatives to improve the Trichuris trichiura and P. pacificus annotations). More broadly, WormBase ParaSite is an existing platform and centralised repository of helminth genomic data and can both integrate new and synthesise available data, providing that new and improved genomes and annotations are made publicly available and open access without restriction.Box 1Access, synthesis and extension of genomic resources for helminths1.Depositing genetic and genomic data in open-access repositories\\u2022NCBIi: DNA/protein sequences, largely focussed on genes, proteins, and genomes.\\u2022ENAii & SRAiii: raw high-throughput sequencing reads, genomes, annotations.\\u2022WormBase ParaSiteiv: the major repository of genomes and annotations of parasitic and related free-living nematodes and platyhelminths.2.Synthesis of available resources\\u2022WormBase ParaSiteivoAlthough WormBase ParaSite is primarily a repository for genomes and annotations, inferred protein sequences from the genomes and annotations are transferred to UniProtv, the major repository of protein sequence and functional information for all specieso(Re-)integrates genome annotations at each release and as new genomes/annotations become available. WormBase ParaSite release 17 has 210 genomes from 169 species.oRuns the Ensembl Compara pipeline to determine orthology between gene sets of different species, refining relationships between genesets and species as new data is added.\\u2022AlphaFold [24.Jumper J. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (4221) Google Scholar]vi has been used to generate predicted 3D protein structures for select worm proteomes, including: C. elegans, B. malayi, Dracunculus medinensis, Onchocerca volvulus, S. mansoni, Strongyloides stercoralis, T. trichiura, and Wuchereria bancrofti\\u2022Tools: gVAL [48.Howe K. et al.Significantly improving the quality of genome assemblies through curation.GigaScience. 2021; 10giaa153Crossref Scopus (66) Google Scholar], G-OnRamp [49.Sargent L. et al.G-OnRamp: Generating genome browsers to facilitate undergraduate-driven collaborative genome annotation.PLoS Comput. Biol. 2020; 16e1007863Crossref PubMed Scopus (3) Google Scholar].3.Development and extension of available resources\\u2022Dedicated manual curation by WormBasevii.oCurrently, six free-living and four parasitic species.oCuration includes semi-automated, machine-learning components to integrate new literature and variants to complement manual curation, as reviewed in [50.Davis P. et al.WormBase in 2022-data, processes, and tools for analyzing Caenorhabditis elegans.Genetics. 2022; 220iyac003Crossref PubMed Scopus (17) Google Scholar].\\u2022Introduction of new genome technologies to improve contiguity [e.g., high-throughput chromosome conformation capture (HiC), optical mapping] and new annotation data (e.g., long-read RNAseq, proteomics).oExamples: S. mansoni [45.Buddenborg S.K. et al.Assembled chromosomes of the blood fluke Schistosoma mansoni provide insight into the evolution of its ZW sex-determination system.bioRxiv. 2021; https://doi.org/10.1101/2021.08.13.456314Crossref Scopus (0) Google Scholar], Schistosoma japonicum [51.Luo F. et al.An improved genome assembly of the fluke Schistosoma japonicum.PLoS Negl. Trop. Dis. 2019; 13e0007612Crossref Scopus (29) Google Scholar], Schistosoma haematobium [52.Stroehlein A.J. et al.Chromosome-level genome of Schistosoma haematobium underpins genome-wide explorations of molecular variation.PLoS Pathog. 2022; 18e1010288Crossref PubMed Scopus (3) Google Scholar], Hymenolepis microstoma [46.Olson P.D. et al.Complete representation of a tapeworm genome reveals chromosomes capped by centromeres, necessitating a dual role in segregation and protection.BMC Biol. 2020; 18: 165Crossref PubMed Sc\", \"doi_y\": \"https://doi.org/10.1016/j.pt.2022.06.002\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4381929719\", \"doi_x\": \"10.1016/j.sbi.2023.102640\", \"af_direct_citation\": 1, \"af_indirect_citations\": 7, \"title\": \"Protein diversification through post-translational modifications, alternative splicing, and gene duplication\", \"abstract\": \"Proteins provide the basis for cellular function. Having multiple versions of the same protein within a single organism provides a way of regulating its activity or developing novel functions. Post-translational modifications of proteins, by means of adding/removing chemical groups to amino acids, allow for a well-regulated and controlled way of generating functionally distinct protein species. Alternative splicing is another method with which organisms possibly generate new isoforms. Additionally, gene duplication events throughout evolution generate multiple paralogs of the same genes, resulting in multiple versions of the same protein within an organism. In this review, we discuss recent advancements in the study of these three methods of protein diversification and provide illustrative examples of how they affect protein structure and function.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102640\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4281382514\", \"doi_x\": \"10.1242/bio.059199\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Modelling the structure of Short Gastrulation and generation of a toolkit for studying its function in <i>Drosophila</i>\", \"abstract\": \"A BMP gradient is essential for patterning the dorsal-ventral axis of invertebrate and vertebrate embryos. The extracellular BMP binding protein Short Gastrulation (Sog) in Drosophila plays a key role in BMP gradient formation. In this study, we combine genome editing, structural and developmental approaches to study Sog function in Drosophila. We generate a sog knockout fly stock, which allows simple reintegration of altered versions of the sog coding sequence. As proof-of-principle, we test the requirement for two cysteine residues that were previously identified as targets for palmitoylation, which has been proposed to enhance Sog secretion. However, we show that the sogC27,28S mutant is viable with only very mild phenotypes, indicating that these residues and their potential modification are not critical for Sog secretion in vivo. Additionally, we use experimental negative stain EM imaging and hydrodynamic data to validate the AlphaFold structure prediction for Sog. The model suggests a more compact shape than the vertebrate ortholog Chordin and conformational flexibility between the C-terminal von Willebrand C domains. We discuss how this altered compactness may contribute to mechanistic differences in Sog and Chordin function during BMP gradient formation. This article has an associated First Person interview with the first author of the paper.\", \"doi_y\": \"https://doi.org/10.1242/bio.059199\", \"publication_date\": \"2022-06-10\"}, {\"work_id\": \"W4319662875\", \"doi_x\": \"10.1126/science.ade8840\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"abstract\": \"Certain inhibitor of apoptosis (IAP) family members are sentinel proteins that prevent untimely cell death by inhibiting caspases. Antagonists, including second mitochondria-derived activator of caspases (SMAC), regulate IAPs and drive cell death. Baculoviral IAP repeat-containing protein 6 (BIRC6), a giant IAP with dual E2 and E3 ubiquitin ligase activity, regulates programmed cell death through unknown mechanisms. We show that BIRC6 directly restricts executioner caspase-3 and -7 and ubiquitinates caspase-3, -7, and -9, working exclusively with noncanonical E1, UBA6. Notably, we show that SMAC suppresses both mechanisms. Cryo-electron microscopy structures of BIRC6 alone and in complex with SMAC reveal that BIRC6 is an antiparallel dimer juxtaposing the substrate-binding module against the catalytic domain. Furthermore, we discover that SMAC multisite binding to BIRC6 results in a subnanomolar affinity interaction, enabling SMAC to competitively displace caspases, thus antagonizing BIRC6 anticaspase function.\", \"doi_y\": \"https://doi.org/10.1126/science.ade8840\", \"publication_date\": \"2023-03-17\"}, {\"work_id\": \"W4366982959\", \"doi_x\": \"10.1038/s41467-023-37490-0\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering \\u03b1-carboxysomes into plant chloroplasts to support autotrophic photosynthesis\", \"abstract\": \"The growth in world population, climate change, and resource scarcity necessitate a sustainable increase in crop productivity. Photosynthesis in major crops is limited by the inefficiency of the key CO2-fixing enzyme Rubisco, owing to its low carboxylation rate and poor ability to discriminate between CO2 and O2. In cyanobacteria and proteobacteria, carboxysomes function as the central CO2-fixing organelles that elevate CO2 levels around encapsulated Rubisco to enhance carboxylation. There is growing interest in engineering carboxysomes into crop chloroplasts as a potential route for improving photosynthesis and crop yields. Here, we generate morphologically correct carboxysomes in tobacco chloroplasts by transforming nine carboxysome genetic components derived from a proteobacterium. The chloroplast-expressed carboxysomes display a structural and functional integrity comparable to native carboxysomes and support autotrophic growth and photosynthesis of the transplastomic plants at elevated CO2. Our study provides proof-of-concept for a route to engineering fully functional CO2-fixing modules and entire CO2-concentrating mechanisms into chloroplasts to improve crop photosynthesis and productivity.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-37490-0\", \"publication_date\": \"2023-04-25\"}, {\"work_id\": \"W4308653706\", \"doi_x\": \"10.1021/acs.jcim.2c01153\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Open Force Field BespokeFit: Automating Bespoke Torsion Parametrization at Scale\", \"abstract\": \"The development of accurate transferable force fields is key to realizing the full potential of atomistic modeling in the study of biological processes such as protein-ligand binding for drug discovery. State-of-the-art transferable force fields, such as those produced by the Open Force Field Initiative, use modern software engineering and automation techniques to yield accuracy improvements. However, force field torsion parameters, which must account for many stereoelectronic and steric effects, are considered to be less transferable than other force field parameters and are therefore often targets for bespoke parametrization. Here, we present the Open Force Field QCSubmit and BespokeFit software packages that, when combined, facilitate the fitting of torsion parameters to quantum mechanical reference data at scale. We demonstrate the use of QCSubmit for simplifying the process of creating and archiving large numbers of quantum chemical calculations, by generating a dataset of 671 torsion scans for druglike fragments. We use BespokeFit to derive individual torsion parameters for each of these molecules, thereby reducing the root-mean-square error in the potential energy surface from 1.1 kcal/mol, using the original transferable force field, to 0.4 kcal/mol using the bespoke version. Furthermore, we employ the bespoke force fields to compute the relative binding free energies of a congeneric series of inhibitors of the TYK2 protein, and demonstrate further improvements in accuracy, compared to the base force field (MUE reduced from 0.560.390.77 to 0.420.280.59 kcal/mol and R2 correlation improved from 0.720.350.87 to 0.930.840.97).\", \"doi_y\": \"https://doi.org/10.1021/acs.jcim.2c01153\", \"publication_date\": \"2022-11-09\"}, {\"work_id\": \"W3186179742\", \"doi_x\": \"10.1126/science.abj8754\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Accurate prediction of protein structures and interactions using a three-track neural network\", \"abstract\": \"DeepMind presented notably accurate predictions at the recent 14th Critical Assessment of Structure Prediction (CASP14) conference. We explored network architectures that incorporate related ideas and obtained the best performance with a three-track network in which information at the one-dimensional (1D) sequence level, the 2D distance map level, and the 3D coordinate level is successively transformed and integrated. The three-track network produces structure predictions with accuracies approaching those of DeepMind in CASP14, enables the rapid solution of challenging x-ray crystallography and cryo-electron microscopy structure modeling problems, and provides insights into the functions of proteins of currently unknown structure. The network also enables rapid generation of accurate protein-protein complex models from sequence information alone, short-circuiting traditional approaches that require modeling of individual subunits followed by docking. We make the method available to the scientific community to speed biological research.\", \"doi_y\": \"https://doi.org/10.1126/science.abj8754\", \"publication_date\": \"2021-08-20\"}, {\"work_id\": \"W4226315274\", \"doi_x\": \"10.1099/jgv.0.001716\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The crystal structure of vaccinia virus protein E2 and perspectives on the prediction of novel viral protein folds\", \"abstract\": \"The morphogenesis of vaccinia virus (VACV, family Poxviridae ), the smallpox vaccine, is a complex process involving multiple distinct cellular membranes and resulting in multiple different forms of infectious virion. Efficient release of enveloped virions, which promote systemic spread of infection within hosts, requires the VACV protein E2 but the molecular basis of E2 function remains unclear and E2 lacks sequence homology to any well-characterised family of proteins. We solved the crystal structure of VACV E2 to 2.3 \\u00c5 resolution, revealing that it comprises two domains with novel folds: an N-terminal annular (ring) domain and a C-terminal globular (head) domain. The C-terminal head domain displays weak structural homology with cellular (pseudo)kinases but lacks conserved surface residues or kinase features, suggesting that it is not enzymatically active, and possesses a large surface basic patch that might interact with phosphoinositide lipid headgroups. Recent deep learning methods have revolutionised our ability to predict the three-dimensional structures of proteins from primary sequence alone. VACV E2 is an exemplar \\u2018difficult\\u2019 viral protein target for structure prediction, being comprised of multiple novel domains and lacking sequence homologues outside Poxviridae . AlphaFold2 nonetheless succeeds in predicting the structures of the head and ring domains with high and moderate accuracy, respectively, allowing accurate inference of multiple structural properties. The advent of highly accurate virus structure prediction marks a step-change in structural virology and beckons a new era of structurally-informed molecular virology.\", \"doi_y\": \"https://doi.org/10.1099/jgv.0.001716\", \"publication_date\": \"2022-01-21\"}, {\"work_id\": \"W4324060687\", \"doi_x\": \"10.3390/ijms24065374\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"COVID-19-Induced Myocarditis: Pathophysiological Roles of ACE2 and Toll-like Receptors\", \"abstract\": \"The clinical manifestations of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection responsible for coronavirus disease 2019 (COVID-19) commonly include dyspnoea and fatigue, and they primarily involve the lungs. However, extra-pulmonary organ dysfunctions, particularly affecting the cardiovascular system, have also been observed following COVID-19 infection. In this context, several cardiac complications have been reported, including hypertension, thromboembolism, arrythmia and heart failure, with myocardial injury and myocarditis being the most frequent. These secondary myocardial inflammatory responses appear to be associated with a poorer disease course and increased mortality in patients with severe COVID-19. In addition, numerous episodes of myocarditis have been reported as a complication of COVID-19 mRNA vaccinations, especially in young adult males. Changes in the cell surface expression of angiotensin-converting enzyme 2 (ACE2) and direct injury to cardiomyocytes resulting from exaggerated immune responses to COVID-19 are just some of the mechanisms that may explain the pathogenesis of COVID-19-induced myocarditis. Here, we review the pathophysiological mechanisms underlying myocarditis associated with COVID-19 infection, with a particular focus on the involvement of ACE2 and Toll-like receptors (TLRs).\", \"doi_y\": \"https://doi.org/10.3390/ijms24065374\", \"publication_date\": \"2023-03-11\"}, {\"work_id\": \"W4382139037\", \"doi_x\": \"10.1016/j.cell.2023.05.049\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Pathogen protein modularity enables elaborate mimicry of a host phosphatase\", \"abstract\": \"Pathogens produce diverse effector proteins to manipulate host cellular processes. However, how functional diversity is generated in an effector repertoire is poorly understood. Many effectors in the devastating plant pathogen Phytophthora contain tandem repeats of the \\u201c(L)WY\\u201d motif, which are structurally conserved but variable in sequences. Here, we discovered a functional module formed by a specific (L)WY-LWY combination in multiple Phytophthora effectors, which efficiently recruits the serine/threonine protein phosphatase 2A (PP2A) core enzyme in plant hosts. Crystal structure of an effector-PP2A complex shows that the (L)WY-LWY module enables hijacking of the host PP2A core enzyme to form functional holoenzymes. While sharing the PP2A-interacting module at the amino terminus, these effectors possess divergent C-terminal LWY units and regulate distinct sets of phosphoproteins in the host. Our results highlight the appropriation of an essential host phosphatase through molecular mimicry by pathogens and diversification promoted by protein modularity in an effector repertoire.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2023.05.049\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4312210016\", \"doi_x\": \"10.1177/25152564221134328\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Sequence Analysis and Structural Predictions of Lipid Transfer Bridges in the Repeating Beta Groove (RBG) Superfamily Reveal Past and Present Domain Variations Affecting Form, Function and Interactions of VPS13, ATG2, SHIP164, Hobbit and Tweek\", \"abstract\": \"Lipid transfer between organelles requires proteins that shield the hydrophobic portions of lipids as they cross the cytoplasm. In the last decade a new structural form of lipid transfer protein (LTP) has been found: long hydrophobic grooves made of beta-sheet that bridge between organelles at membrane contact sites. Eukaryotes have five families of bridge-like LTPs: VPS13, ATG2, SHIP164, Hobbit and Tweek. These are unified into a single superfamily through their bridges being composed of just one domain, called the repeating beta groove (RBG) domain, which builds into rod shaped multimers with a hydrophobic-lined groove and hydrophilic exterior. Here, sequences and predicted structures of the RBG superfamily were analyzed in depth. Phylogenetics showed that the last eukaryotic common ancestor contained all five RBG proteins, with duplicated VPS13s. The current set of long RBG protein appears to have arisen in even earlier ancestors from shorter forms with 4 RBG domains. The extreme ends of most RBG proteins have amphipathic helices that might be an adaptation for direct or indirect bilayer interaction, although this has yet to be tested. The one exception to this is the C-terminus of SHIP164, which instead has a coiled-coil. Finally, the exterior surfaces of the RBG bridges are shown to have conserved residues along most of their length, indicating sites for partner interactions almost all of which are unknown. These findings can inform future cell biological and biochemical experiments.\", \"doi_y\": \"https://doi.org/10.1177/25152564221134328\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4280496699\", \"doi_x\": \"10.1101/2022.04.25.489459\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"A Feed-forward Pathway Drives LRRK2 kinase Membrane Recruitment and Activation\", \"abstract\": \"Abstract Activating mutations in the Leucine Rich Repeat Kinase 2 (LRRK2) cause Parkinson\\u2019s disease and previously we showed that activated LRRK2 phosphorylates a subset of Rab GTPases (Steger et al., 2017). Moreover, Golgi-associated Rab29 can recruit LRRK2 to the surface of the Golgi and activate it there for both auto- and Rab substrate phosphorylation. Here we define the precise Rab29 binding region of the LRRK2 Armadillo domain between residues 360-450 and show that this domain, termed \\u201cSite #1\\u201d, can also bind additional LRRK2 substrates, Rab8A and Rab10. Moreover, we identify a distinct, N-terminal, higher affinity interaction interface between LRRK2 phosphorylated Rab8 and Rab10 termed \\u201cSite #2\\u201d, that can retain LRRK2 on membranes in cells to catalyze multiple, subsequent phosphorylation events. Kinase inhibitor washout experiments demonstrate that rapid recovery of kinase activity in cells depends on the ability of LRRK2 to associate with phosphorylated Rab proteins, and phosphorylated Rab8A stimulates LRRK2 phosphorylation of Rab10 in vitro. Reconstitution of purified LRRK2 recruitment onto planar lipid bilayers decorated with Rab10 protein demonstrates cooperative association of only active LRRK2 with phospho-Rab10-containing membrane surfaces. These experiments reveal a feed-forward pathway that provides spatial control and membrane activation of LRRK2 kinase activity.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.25.489459\", \"publication_date\": \"2022-04-25\"}, {\"work_id\": \"W4309934767\", \"doi_x\": \"10.3390/ijms232314686\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"First Insights into the Venom Composition of Two Ecuadorian Coral Snakes\", \"abstract\": \"Micrurus is a medically relevant genus of venomous snakes composed of 85 species. Bites caused by coral snakes are rare, but they are usually associated with very severe and life-threatening clinical manifestations. Ecuador is a highly biodiverse country with a complex natural environment, which is home to approximately 20% of identified Micrurus species. Additionally, it is on the list of Latin American countries with the highest number of snakebites. However, there is no local antivenom available against the Ecuadorian snake venoms, and the biochemistry of these venoms has been poorly explored. Only a limited number of samples collected in the country from the Viperidae family were recently characterised. Therefore, this study addressed the compositional patterns of two coral snake venoms from Ecuador, M. helleri and M. mipartitus, using venomics strategies, integrating sample fractionation, gel electrophoresis, and mass spectrometry. Chromatographic and electrophoretic profiles of these snake venoms revealed interspecific variability, which was ascertained by mass spectrometry. The two venoms followed the recently recognised dichotomic toxin expression trends displayed by Micrurus species: M. helleri venom contains a high proportion (72%) of phospholipase A2, whereas M. mipartitus venom is dominated by three-finger toxins (63%). A few additional protein families were also detected in these venoms. Overall, these results provide the first comprehensive views on the composition of two Ecuadorian coral snake venoms and expand the knowledge of Micrurus venom phenotypes. These findings open novel perspectives to further research the functional aspects of these biological cocktails of PLA2s and 3FTxs and stress the need for the preclinical evaluation of the currently used antivenoms for therapeutic purposes in Ecuador.\", \"doi_y\": \"https://doi.org/10.3390/ijms232314686\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4291652457\", \"doi_x\": \"10.1073/pnas.2206069119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Deep learning predicts DNA methylation regulatory variants in the human brain and elucidates the genetics of psychiatric disorders\", \"abstract\": \"There is growing evidence for the role of DNA methylation (DNAm) quantitative trait loci (mQTLs) in the genetics of complex traits, including psychiatric disorders. However, due to extensive linkage disequilibrium (LD) of the genome, it is challenging to identify causal genetic variations that drive DNAm levels by population-based genetic association studies. This limits the utility of mQTLs for fine-mapping risk loci underlying psychiatric disorders identified by genome-wide association studies (GWAS). Here we present INTERACT, a deep learning model that integrates convolutional neural networks with transformer, to predict effects of genetic variations on DNAm levels at CpG sites in the human brain. We show that INTERACT-derived DNAm regulatory variants are not confounded by LD, are concentrated in regulatory genomic regions in the human brain, and are convergent with mQTL evidence from genetic association analysis. We further demonstrate that predicted DNAm regulatory variants are enriched for heritability of brain-related traits and improve polygenic risk prediction for schizophrenia across diverse ancestry samples. Finally, we applied predicted DNAm regulatory variants for fine-mapping schizophrenia GWAS risk loci to identify potential novel risk genes. Our study shows the power of a deep learning approach to identify functional regulatory variants that may elucidate the genetic basis of complex traits.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2206069119\", \"publication_date\": \"2022-08-15\"}, {\"work_id\": \"W4383816911\", \"doi_x\": \"10.3389/fimmu.2023.1177691\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Are the integrin binding motifs within SARS CoV-2 spike protein and MHC class II alleles playing the key role in COVID-19?\", \"abstract\": \"The previous studies on the RGD motif (aa403-405) within the SARS CoV-2 spike (S) protein receptor binding domain (RBD) suggest that the RGD motif binding integrin(s) may play an important role in infection of the host cells. We also discussed the possible role of two other integrin binding motifs that are present in S protein: LDI (aa585-587) and ECD (661-663), the motifs used by some other viruses in the course of infection. The MultiFOLD models for protein structure analysis have shown that the ECD motif is clearly accessible in the S protein, whereas the RGD and LDI motifs are partially accessible. Furthermore, the amino acids that are present in Epstein-Barr virus protein (EBV) gp42 playing very important role in binding to the HLA-DRB1 molecule and in the subsequent immune response evasion, are also present in the S protein heptad repeat-2. Our MultiFOLD model analyses have shown that these amino acids are clearly accessible on the surface in each S protein chain as monomers and in the homotrimer complex and bind to HLA-DRB1 \\u03b2 chain. Therefore, they may have the identical role in SARS CoV-2 immune evasion as in EBV infection. The prediction analyses of the MHC class II binding peptides within the S protein have shown that the RGD motif is present in the core 9-mer peptide IRGDEVRQI within the two HLA-DRB1*03:01 and HLA-DRB3*01.01 strong binding 15-mer peptides suggesting that RGD motif may be the potential immune epitope. Accordingly, infected HLA-DRB1*03:01 or HLA-DRB3*01.01 positive individuals may develop high affinity anti-RGD motif antibodies that react with the RGD motif in the host proteins, like fibrinogen, thrombin or von Willebrand factor, affecting haemostasis or participating in autoimmune disorders.\", \"doi_y\": \"https://doi.org/10.3389/fimmu.2023.1177691\", \"publication_date\": \"2023-07-10\"}, {\"work_id\": \"W4379469725\", \"doi_x\": \"10.1016/j.cub.2023.03.037\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Contrasting self-recognition rejection systems for self-incompatibility in Brassica and Papaver\", \"abstract\": \"Self-incompatibility (SI) plays a pivotal role in whether self-pollen is accepted or rejected. Most SI systems employ two tightly linked loci encoding highly polymorphic pollen (male) and pistil (female) S-determinants that control whether self-pollination is successful or not. In recent years our knowledge of the signalling networks and cellular mechanisms involved has improved considerably, providing an important contribution to our understanding of the diverse mechanisms used by plant cells to recognise each other and elicit responses. Here, we compare and contrast two important SI systems employed in the Brassicaceae and Papaveraceae. Both use 'self-recognition' systems, but their genetic control and S-determinants are quite different. We describe the current knowledge about the receptors and ligands, and the downstream signals and responses utilized to prevent self-seed set. What emerges is a common theme involving the initiation of destructive pathways that block the key processes that are required for compatible pollen-pistil interactions.\", \"doi_y\": \"https://doi.org/10.1016/j.cub.2023.03.037\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4200013746\", \"doi_x\": \"10.1101/2021.12.15.472780\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Detecting and quantifying clonal selection in somatic stem cells\", \"abstract\": \"Abstract As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the underlying driver event. Moreover, SCIFER infers the self-renewal and mutation dynamics of the tissue\\u2019s stem cells, and, if selection is present, the size and growth rate of the largest selected clone. We benchmark SCIFER with published data and then probe bone marrow of 22 non-leukemic individuals for clonal hematopoiesis (CH), identifying CH with known and unknown driver events. Unexpectedly, we find accelerated division of all stem cells in CH, compared to age-matched non-CH individuals, suggesting that the bone marrow environment alters stem cell dynamics in individuals with CH. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.15.472780\", \"publication_date\": \"2021-12-16\"}, {\"work_id\": \"W4226091362\", \"doi_x\": \"10.1093/bib/bbac165\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural landscapes of PPI interfaces\", \"abstract\": \"Proteins are capable of highly specific interactions and are responsible for a wide range of functions, making them attractive in the pursuit of new therapeutic options. Previous studies focusing on overall geometry of protein-protein interfaces, however, concluded that PPI interfaces were generally flat. More recently, this idea has been challenged by their structural and thermodynamic characterisation, suggesting the existence of concave binding sites that are closer in character to traditional small-molecule binding sites, rather than exhibiting complete flatness. Here, we present a large-scale analysis of binding geometry and physicochemical properties of all protein-protein interfaces available in the Protein Data Bank. In this review, we provide a comprehensive overview of the protein-protein interface landscape, including evidence that even for overall larger, more flat interfaces that utilize discontinuous interacting regions, small and potentially druggable pockets are utilized at binding sites.\", \"doi_y\": \"https://doi.org/10.1093/bib/bbac165\", \"publication_date\": \"2022-06-02\"}, {\"work_id\": \"W4319656652\", \"doi_x\": \"10.1371/journal.ppat.1011106\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Plant immune receptor pathways as a united front against pathogens\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011106\", \"publication_date\": \"2023-02-09\"}, {\"work_id\": \"W4292239272\", \"doi_x\": \"10.1089/genbio.2022.0017\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Deep Learning Concepts and Applications for Synthetic Biology\", \"abstract\": \"Synthetic biology has a natural synergy with deep learning. It can be used to generate large data sets to train models, for example by using DNA synthesis, and deep learning models can be used to inform design, such as by generating novel parts or suggesting optimal experiments to conduct. Recently, research at the interface of engineering biology and deep learning has highlighted this potential through successes including the design of novel biological parts, protein structure prediction, automated analysis of microscopy data, optimal experimental design, and biomolecular implementations of artificial neural networks. In this review, we present an overview of synthetic biology-relevant classes of data and deep learning architectures. We also highlight emerging studies in synthetic biology that capitalize on deep learning to enable novel understanding and design, and discuss challenges and future opportunities in this space.\", \"doi_y\": \"https://doi.org/10.1089/genbio.2022.0017\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4224213858\", \"doi_x\": \"10.3390/ijms23094568\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"PTPRD and DCC Are Novel BACE1 Substrates Differentially Expressed in Alzheimer\\u2019s Disease: A Data Mining and Bioinformatics Study\", \"abstract\": \"The \\u03b2-site Amyloid precursor protein Cleaving Enzyme 1 (BACE1) is an extensively studied therapeutic target for Alzheimer's disease (AD), owing to its role in the production of neurotoxic amyloid beta (A\\u03b2) peptides. However, despite numerous BACE1 inhibitors entering clinical trials, none have successfully improved AD pathogenesis, despite effectively lowering A\\u03b2 concentrations. This can, in part, be attributed to an incomplete understanding of BACE1, including its physiological functions and substrate specificity. We propose that BACE1 has additional important physiological functions, mediated through substrates still to be identified. Thus, to address this, we computationally analysed a list of 533 BACE1 dependent proteins, identified from the literature, for potential BACE1 substrates, and compared them against proteins differentially expressed in AD. We identified 15 novel BACE1 substrates that were specifically altered in AD. To confirm our analysis, we validated Protein tyrosine phosphatase receptor type D (PTPRD) and Netrin receptor DCC (DCC) using Western blotting. These findings shed light on the BACE1 inhibitor failings and could enable the design of substrate-specific inhibitors to target alternative BACE1 substrates. Furthermore, it gives us a greater understanding of the roles of BACE1 and its dysfunction in AD.\", \"doi_y\": \"https://doi.org/10.3390/ijms23094568\", \"publication_date\": \"2022-04-20\"}, {\"work_id\": \"W4362460408\", \"doi_x\": \"10.1021/jacs.3c00587\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Investigating Native Metal Ion Binding Sites in Mammalian Histidine-Rich Glycoprotein\", \"abstract\": \"Mammalian histidine-rich glycoprotein (HRG) is a highly versatile and abundant blood plasma glycoprotein with a diverse range of ligands that is involved in regulating many essential biological processes, including coagulation, cell adhesion, and angiogenesis. Despite its biomedical importance, structural information on the multi-domain protein is sparse, not least due to intrinsically disordered regions that elude high-resolution structural characterization. Binding of divalent metal ions, particularly ZnII, to multiple sites within the HRG protein is of critical functional importance and exerts a regulatory role. However, characterization of the ZnII binding sites of HRG is a challenge; their number and composition as well as their affinities and stoichiometries of binding are currently not fully understood. In this study, we explored modern electron paramagnetic resonance (EPR) spectroscopy methods supported by protein secondary and tertiary structure prediction to assemble a holistic picture of native HRG and its interaction with metal ions. To the best of our knowledge, this is the first time that this suite of EPR techniques has been applied to count and characterize endogenous metal ion binding sites in a native mammalian protein of unknown structure.\", \"doi_y\": \"https://doi.org/10.1021/jacs.3c00587\", \"publication_date\": \"2023-03-31\"}, {\"work_id\": \"W4296129041\", \"doi_x\": \"10.1101/2022.09.15.508082\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Altered plasma membrane abundance of the sulfatide-binding protein NF155 links glycosphingolipid imbalances to demyelination\", \"abstract\": \"ABSTRACT Myelin is a multi-layered membrane that tightly wraps neuronal axons enabling efficient, high-speed signal propagation. The axon and myelin sheath form tight contacts, mediated by specific plasma membrane proteins and lipids, and disruption of these contacts causes devastating demyelinating diseases. Using two cell-based models of demyelinating sphingolipidoses, we demonstrate that altered lipid metabolism changes the abundance of specific plasma membrane proteins. These altered membrane proteins have known roles in cell adhesion and signalling, with several implicated in neurological diseases. The cell surface abundance of the adhesion molecule Neurofascin, a protein critical for the maintenance of myelin-axon contacts, changes following disruption to sphingolipid metabolism. This provides a direct molecular link between altered lipid abundance and myelin stability. We show that the Neurofascin isoform NF155, but not NF186, interacts directly and specifically with the sphingolipid sulfatide via multiple binding sites and that this interaction requires the full-length extracellular domain of NF155. We demonstrate that NF155 adopts an S-shaped conformation and preferrentially binds sulfatide-containing membranes in cis , with important implications for protein arrangement in the tight axon-myelin space. Our work links glycosphingolipid imbalances to disturbance of membrane protein abundance and demonstrates how this may be driven by direct protein-lipid interactions, providing a mechanistic framework to understand the pathogenesis of galactosphingolipidoses.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.15.508082\", \"publication_date\": \"2022-09-15\"}, {\"work_id\": \"W4319825963\", \"doi_x\": \"10.15252/embj.2022112053\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Shuffled ATG8 interacting motifs form an ancestral bridge between UFMylation and autophagy\", \"abstract\": \"Abstract UFMylation involves the covalent modification of substrate proteins with UFM1 (Ubiquitin\\u2010fold modifier 1) and is important for maintaining ER homeostasis. Stalled translation triggers the UFMylation of ER\\u2010bound ribosomes and activates C53\\u2010mediated autophagy to clear toxic polypeptides. C53 contains noncanonical shuffled ATG8\\u2010interacting motifs (sAIMs) that are essential for ATG8 interaction and autophagy initiation. However, the mechanistic basis of sAIM\\u2010mediated ATG8 interaction remains unknown. Here, we show that C53 and sAIMs are conserved across eukaryotes but secondarily lost in fungi and various algal lineages. Biochemical assays showed that the unicellular alga Chlamydomonas reinhardtii has a functional UFMylation pathway, refuting the assumption that UFMylation is linked to multicellularity. Comparative structural analyses revealed that both UFM1 and ATG8 bind sAIMs in C53, but in a distinct way. Conversion of sAIMs into canonical AIMs impaired binding of C53 to UFM1, while strengthening ATG8 binding. Increased ATG8 binding led to the autoactivation of the C53 pathway and sensitization of Arabidopsis thaliana to ER stress. Altogether, our findings reveal an ancestral role of sAIMs in UFMylation\\u2010dependent fine\\u2010tuning of C53\\u2010mediated autophagy activation.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022112053\", \"publication_date\": \"2023-02-10\"}, {\"work_id\": \"W4385690713\", \"doi_x\": \"10.1038/s41467-023-40541-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A HIF independent oxygen-sensitive pathway for controlling cholesterol synthesis\", \"abstract\": \"Cholesterol biosynthesis is a highly regulated, oxygen-dependent pathway, vital for cell membrane integrity and growth. In fungi, the dependency on oxygen for sterol production has resulted in a shared transcriptional response, resembling prolyl hydroxylation of Hypoxia Inducible Factors (HIFs) in metazoans. Whether an analogous metazoan pathway exists is unknown. Here, we identify Sterol Regulatory Element Binding Protein 2 (SREBP2), the key transcription factor driving sterol production in mammals, as an oxygen-sensitive regulator of cholesterol synthesis. SREBP2 degradation in hypoxia overrides the normal sterol-sensing response, and is HIF independent. We identify MARCHF6, through its NADPH-mediated activation in hypoxia, as the main ubiquitin ligase controlling SREBP2 stability. Hypoxia-mediated degradation of SREBP2 protects cells from statin-induced cell death by forcing cells to rely on exogenous cholesterol uptake, explaining why many solid organ tumours become auxotrophic for cholesterol. Our findings therefore uncover an oxygen-sensitive pathway for governing cholesterol synthesis through regulated SREBP2-dependent protein degradation.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40541-1\", \"publication_date\": \"2023-08-09\"}, {\"work_id\": \"W4319232691\", \"doi_x\": \"10.1101/2023.02.03.526954\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Native Mass Spectrometry of Complexes Formed by Molecular Glues Reveals Stoichiometric Rearrangement of E3 Ligases\", \"abstract\": \"Abstract In this application of native mass spectrometry (nMS) to investigate complexes formed by molecular glues (MGs), we have demonstrated its efficiency in delineating stoichiometric rearrangements of E3 ligases that occur during targeted protein degradation (TPD). MGs stabilise interactions between an E3 ligase and a protein of interest (POI) targeted for degradation, and these ternary interactions are challenging to characterise. We have shown that nMS can unambiguously identify complexes formed between the CRBN:DDB1 E3 ligase and the POI GSPT1 upon the addition of lenalidomide, pomalidomide or thalidomide. Ternary complex formation was also identified involving the DCAF15:DDA1:DDB1 E3 ligase in the presence of MG (E7820 or indisulam) and POI RBM39. Moreover, we uncovered that the DCAF15:DDA1:DDB1 E3 ligase self-associates into dimers and trimers when analysed alone at low salt concentrations (100 mM ammonium acetate) which dissociate into single copies of the complex at higher salt concentrations (500 mM ammonium acetate), or upon the addition of MG and POI, forming a 1:1:1 ternary complex. This work demonstrates the strength of nMS in TPD research, reveals novel binding mechanisms of the DCAF15 E3 ligase, and highlights the potential effect of salt concentrations on protein complexes during structural analysis.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.03.526954\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4224046030\", \"doi_x\": \"10.1101/2022.04.13.488131\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The meiotic LINC complex component KASH5 is an activating adaptor for cytoplasmic dynein\", \"abstract\": \"Abstract Cytoplasmic dynein-driven movement of chromosomes during prophase I of mammalian meiosis is essential for synapsis and genetic exchange. Dynein connects to chromosome telomeres via KASH5 and SUN1 or SUN2, which together span the nuclear envelope. Here, we show that KASH5 promotes dynein motility in vitro, and cytosolic KASH5 inhibits dynein\\u2019s interphase functions. KASH5 interacts with either dynein light intermediate chain (DYNC1LI1 or DYNC1LI2) via a conserved helix in the LIC C-terminal, and this region is also needed for dynein\\u2019s recruitment to other cellular membranes. KASH5\\u2019s N-terminal EF-hands are essential, as the interaction with dynein is disrupted by mutation of key calcium-binding residues, although it is not regulated by cellular calcium levels. Dynein can be recruited to KASH5 at the nuclear envelope independently of dynactin, while LIS1 is essential for dynactin incorporation into the KASH5-dynein complex. Altogether, we show that the trans-membrane protein KASH5 is an activating adaptor for dynein, and shed light on the hierarchy of assembly of KASH5-dynein-dynactin complexes.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.13.488131\", \"publication_date\": \"2022-04-13\"}, {\"work_id\": \"W4317212025\", \"doi_x\": \"10.1038/s41579-022-00841-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"SARS-CoV-2 variant biology: immune escape, transmission and fitness\", \"abstract\": \"In late 2020, after circulating for almost a year in the human population, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) exhibited a major step change in its adaptation to humans. These highly mutated forms of SARS-CoV-2 had enhanced rates of transmission relative to previous variants and were termed \\u2018variants of concern\\u2019 (VOCs). Designated Alpha, Beta, Gamma, Delta and Omicron, the VOCs emerged independently from one another, and in turn each rapidly became dominant, regionally or globally, outcompeting previous variants. The success of each VOC relative to the previously dominant variant was enabled by altered intrinsic functional properties of the virus and, to various degrees, changes to virus antigenicity conferring the ability to evade a primed immune response. The increased virus fitness associated with VOCs is the result of a complex interplay of virus biology in the context of changing human immunity due to both vaccination and prior infection. In this Review, we summarize the literature on the relative transmissibility and antigenicity of SARS-CoV-2 variants, the role of mutations at the furin spike cleavage site and of non-spike proteins, the potential importance of recombination to virus success, and SARS-CoV-2 evolution in the context of T cells, innate immunity and population immunity. SARS-CoV-2 shows a complicated relationship among virus antigenicity, transmission and virulence, which has unpredictable implications for the future trajectory and disease burden of COVID-19. In this Review, the authors summarize the mutations harboured by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants of concern. They describe the impact of mutations on virus infectivity and transmissibility, and discuss SARS-CoV-2 evolution in the context of T cells, innate immunity and population immunity.\", \"doi_y\": \"https://doi.org/10.1038/s41579-022-00841-7\", \"publication_date\": \"2023-01-18\"}, {\"work_id\": \"W4285589504\", \"doi_x\": \"10.1038/s41467-022-31847-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Recovery of Lutacidiplasmatales archaeal order genomes suggests convergent evolution in Thermoplasmatota\", \"abstract\": \"The Terrestrial Miscellaneous Euryarchaeota Group has been identified in various environments, and the single genome investigated thus far suggests that these archaea are anaerobic sulfite reducers. We assemble 35 new genomes from this group that, based on genome analysis, appear to possess aerobic and facultative anaerobic lifestyles and may oxidise rather than reduce sulfite. We propose naming this order (representing 16 genera) \\\"Lutacidiplasmatales\\\" due to their occurrence in various acidic environments and placement within the phylum Thermoplasmatota. Phylum-level analysis reveals that Thermoplasmatota evolution had been punctuated by several periods of high levels of novel gene family acquisition. Several essential metabolisms, such as aerobic respiration and acid tolerance, were likely acquired independently by divergent lineages through convergent evolution rather than inherited from a common ancestor. Ultimately, this study describes the terrestrially prevalent Lutacidiciplasmatales and highlights convergent evolution as an important driving force in the evolution of archaeal lineages.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-31847-7\", \"publication_date\": \"2022-07-15\"}, {\"work_id\": \"W4313907594\", \"doi_x\": \"10.1021/acs.analchem.2c05105\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The New Era of High-Throughput Nanoelectrochemistry\", \"abstract\": \"ADVERTISEMENT RETURN TO ISSUEPREVReviewNEXTThe New Era of High-Throughput NanoelectrochemistryXiangdong XuXiangdong XuDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.More by Xiangdong XuView Biographyhttps://orcid.org/0000-0002-1120-4661, Dimitrios ValavanisDimitrios ValavanisDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.More by Dimitrios ValavanisView Biographyhttps://orcid.org/0000-0002-8777-664X, Paolo CiocciPaolo CiocciUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, FranceMore by Paolo CiocciView Biography, Samuel ConfederatSamuel ConfederatSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.More by Samuel ConfederatView Biography, Fabio MarcuccioFabio MarcuccioSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.Faculty of Medicine, Imperial College London, London SW7 2AZ, United KingdomMore by Fabio MarcuccioView Biographyhttps://orcid.org/0000-0003-4816-2896, Jean-Fran\\u00e7ois LemineurJean-Fran\\u00e7ois LemineurUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, FranceMore by Jean-Fran\\u00e7ois LemineurView Biography, Paolo Actis*Paolo ActisSchool of Electronic and Electrical Engineering and Pollard Institute, University of Leeds, Leeds LS2 9JT, U.K.Bragg Centre for Materials Research, University of Leeds, Leeds LS2 9JT, U.K.*[email protected]More by Paolo ActisView Biographyhttps://orcid.org/0000-0002-7146-1854, Fr\\u00e9d\\u00e9ric Kanoufi*Fr\\u00e9d\\u00e9ric KanoufiUniversit\\u00e9 Paris Cit\\u00e9, ITODYS, CNRS, F-75013 Paris, France*[email protected]More by Fr\\u00e9d\\u00e9ric KanoufiView Biographyhttps://orcid.org/0000-0002-9784-2380, and Patrick R. Unwin*Patrick R. UnwinDepartment of Chemistry, University of Warwick, Coventry CV4 7AL, U.K.*[email protected]More by Patrick R. UnwinView Biographyhttps://orcid.org/0000-0003-3106-2178Cite this: Anal. Chem. 2023, 95, 1, 319\\u2013356Publication Date (Web):January 10, 2023Publication History Received15 November 2022Published online10 January 2023Published inissue 10 January 2023https://doi.org/10.1021/acs.analchem.2c05105Copyright \\u00a9 2023 The Authors. Published by American Chemical SocietyRIGHTS & PERMISSIONSACS AuthorChoiceCC: Creative CommonsBY: Credit must be given to the creatorArticle Views1582Altmetric-Citations1LEARN ABOUT THESE METRICSArticle Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.Citations are the number of other articles citing this article, calculated by Crossref and updated daily. Find more information about Crossref citation counts.The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on the Altmetric Attention Score and how the score is calculated. Share Add toView InAdd Full Text with ReferenceAdd Description ExportRISCitationCitation and abstractCitation and referencesMore Options Share onFacebookTwitterWechatLinked InReddit PDF (5 MB) Get e-AlertsSUBJECTS:Ions,Nanoparticles,Nanopores,Probes,Surface charge Get e-Alerts\", \"doi_y\": \"https://doi.org/10.1021/acs.analchem.2c05105\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W4317434672\", \"doi_x\": \"10.1038/s44222-022-00007-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Lateral flow test engineering and lessons learned from COVID-19\", \"abstract\": \"The acceptability and feasibility of large-scale testing with lateral flow tests (LFTs) for clinical and public health purposes has been demonstrated during the COVID-19 pandemic. LFTs can detect analytes in a variety of samples, providing a rapid read-out, which allows self-testing and decentralized diagnosis. In this Review, we examine the changing LFT landscape with a focus on lessons learned from COVID-19. We discuss the implications of LFTs for decentralized testing of infectious diseases, including diseases of epidemic potential, the \\u2018silent pandemic\\u2019 of antimicrobial resistance, and other acute and chronic infections. Bioengineering approaches will play a key part in increasing the sensitivity and specificity of LFTs, improving sample preparation, incorporating nucleic acid amplification and detection, and enabling multiplexing, digital connection and green manufacturing, with the aim of creating the next generation of high-accuracy, easy-to-use, affordable and digitally connected LFTs. We conclude with recommendations, including the building of a global network of LFT research and development hubs to facilitate and strengthen future diagnostic resilience. The feasibility of large-scale testing with lateral flow tests has been demonstrated in the COVID-19 pandemic. This Review examines lessons learned from the COVID-19 pandemic to inform the design and bioengineering of next-generation lateral flow tests to strengthen future diagnostic resilience.\", \"doi_y\": \"https://doi.org/10.1038/s44222-022-00007-3\", \"publication_date\": \"2023-01-19\"}, {\"work_id\": \"W4386488775\", \"doi_x\": \"10.1039/d3cc02909c\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Considerations for defining +80 Da mass shifts in mass spectrometry-based proteomics: phosphorylation and beyond\", \"abstract\": \"Post-translational modifications (PTMs) are ubiquitous and key to regulating protein function. Understanding the dynamics of individual PTMs and their biological roles requires robust characterisation. Mass spectrometry (MS) is the method of choice for the identification and quantification of protein modifications. This article focusses on the MS-based analysis of those covalent modifications that induce a mass shift of +80 Da, notably phosphorylation and sulfation, given the challenges associated with their discrimination and pinpointing the sites of modification on a polypeptide chain. Phosphorylation in particular is highly abundant, dynamic and can occur on numerous residues to invoke specific functions, hence robust characterisation is crucial to understanding biological relevance. Showcasing our work in the context of other developments in the field, we highlight approaches for enrichment and site localisation of phosphorylated (canonical and non-canonical) and sulfated peptides, as well as modification analysis in the context of intact proteins (top down proteomics) to explore combinatorial roles. Finally, we discuss the application of native ion-mobility MS to explore the effect of these PTMs on protein structure and ligand binding.\", \"doi_y\": \"https://doi.org/10.1039/d3cc02909c\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4298101475\", \"doi_x\": \"10.1124/pharmrev.120.000281\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Angiotensin AT<sub>2</sub>Receptor: From a Binding Site to a Novel Therapeutic Target\", \"abstract\": \"Discovered more than 30 years ago, the angiotensin AT2 receptor (AT2R) has evolved from a binding site with unknown function to a firmly established major effector within the protective arm of the renin-angiotensin system (RAS) and a target for new drugs in development. The AT2R represents an endogenous protective mechanism that can be manipulated in the majority of preclinical models to alleviate lung, renal, cardiovascular, metabolic, cutaneous, and neural diseases as well as cancer. This article is a comprehensive review summarizing our current knowledge of the AT2R, from its discovery to its position within the RAS and its overall functions. This is followed by an in-depth look at the characteristics of the AT2R, including its structure, intracellular signaling, homo- and heterodimerization, and expression. AT2R-selective ligands, from endogenous peptides to synthetic peptides and nonpeptide molecules that are used as research tools, are discussed. Finally, we summarize the known physiological roles of the AT2R and its abundant protective effects in multiple experimental disease models and expound on AT2R ligands that are undergoing development for clinical use. The present review highlights the controversial aspects and gaps in our knowledge of this receptor and illuminates future perspectives for AT2R research. SIGNIFICANCE STATEMENT: The angiotensin AT2 receptor (AT2R) is now regarded as a fully functional and important component of the renin-angiotensin system, with the potential of exerting protective actions in a variety of diseases. This review provides an in-depth view of the AT2R, which has progressed from being an enigma to becoming a therapeutic target.\", \"doi_y\": \"https://doi.org/10.1124/pharmrev.120.000281\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4379467311\", \"doi_x\": \"10.1073/pnas.2302580120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"abstract\": \"AmiA and AmiB are peptidoglycan-hydrolyzing enzymes from Escherichia coli that are required to break the peptidoglycan layer during bacterial cell division and maintain integrity of the cell envelope. In vivo, the activity of AmiA and AmiB is tightly controlled through their interactions with the membrane-bound FtsEX-EnvC complex. Activation of AmiA and AmiB requires access to a groove in the amidase-activating LytM domain of EnvC which is gated by ATP-driven conformational changes in FtsEX-EnvC complex. Here, we present a high-resolution structure of the isolated AmiA protein, confirming that it is autoinhibited in the same manner as AmiB and AmiC, and a complex of the AmiB enzymatic domain bound to the activating EnvC LytM domain. In isolation, the active site of AmiA is blocked by an autoinhibitory helix that binds directly to the catalytic zinc and fills the volume expected to accommodate peptidoglycan binding. In the complex, binding of the EnvC LytM domain induces a conformational change that displaces the amidase autoinhibitory helix and reorganizes the active site for activity. Our structures, together with complementary mutagenesis work, defines the conformational changes required to activate AmiA and/or AmiB through their interaction with their cognate activator EnvC.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2302580120\", \"publication_date\": \"2023-06-05\"}, {\"work_id\": \"W4302004223\", \"doi_x\": \"10.1021/acscatal.2c02954\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Coupled Natural Fusion Enzymes in a Novel Biocatalytic Cascade Convert Fatty Acids to Amines\", \"abstract\": \"Tambjamine YP1 is a pyrrole-containing natural product. Analysis of the enzymes encoded in the Pseudoalteromonas tunicata \\\"tam\\\" biosynthetic gene cluster (BGC) identified a unique di-domain biocatalyst (PtTamH). Sequence and bioinformatic analysis predicts that PtTamH comprises an N-terminal, pyridoxal 5'-phosphate (PLP)-dependent transaminase (TA) domain fused to a NADH-dependent C-terminal thioester reductase (TR) domain. Spectroscopic and chemical analysis revealed that the TA domain binds PLP, utilizes l-Glu as an amine donor, accepts a range of fatty aldehydes (C7-C14 with a preference for C12), and produces the corresponding amines. The previously characterized PtTamA from the \\\"tam\\\" BGC is an ATP-dependent, di-domain enzyme comprising a class I adenylation domain fused to an acyl carrier protein (ACP). Since recombinant PtTamA catalyzes the activation and thioesterification of C12 acid to the holo-ACP domain, we hypothesized that C12 ACP is the natural substrate for PtTamH. PtTamA and PtTamH were successfully coupled together in a biocatalytic cascade that converts fatty acids (FAs) to amines in one pot. Moreover, a structural model of PtTamH provides insights into how the TA and TR domains are organized. This work not only characterizes the formation of the tambjamine YP1 tail but also suggests that PtTamA and PtTamH could be useful biocatalysts for FA to amine functional group conversion.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.2c02954\", \"publication_date\": \"2022-10-05\"}, {\"work_id\": \"W4291020693\", \"doi_x\": \"10.1038/s41586-022-05070-9\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"abstract\": \"Cyclic nucleotide signalling is a key component of antiviral defence in all domains of life. Viral detection activates a nucleotide cyclase to generate a second messenger, resulting in activation of effector proteins. This is exemplified by the metazoan cGAS-STING innate immunity pathway1, which originated in bacteria2. These defence systems require a sensor domain to bind the cyclic nucleotide and are often coupled with an effector domain that, when activated, causes cell death by destroying essential biomolecules3. One example is the Toll/interleukin-1 receptor (TIR) domain, which degrades the essential cofactor NAD+ when activated in response to infection in plants and bacteria2,4,5 or during programmed nerve cell death6. Here we show that a bacterial antiviral defence system generates a cyclic tri-adenylate that binds to a TIR-SAVED effector, acting as the 'glue' to allow assembly of an extended superhelical solenoid structure. Adjacent TIR subunits interact to organize and complete a composite active site, allowing NAD+ degradation. Activation requires extended filament formation, both in vitro and in vivo. Our study highlights an example of large-scale molecular assembly controlled by cyclic nucleotides and reveals key details of the mechanism of TIR enzyme activation.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-05070-9\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4366189236\", \"doi_x\": \"10.3390/antiox12040938\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"abstract\": \"Coenzyme A (CoA) is an important cellular metabolite that is critical for metabolic processes and the regulation of gene expression. Recent discovery of the antioxidant function of CoA has highlighted its protective role that leads to the formation of a mixed disulfide bond with protein cysteines, which is termed protein CoAlation. To date, more than 2000 CoAlated bacterial and mammalian proteins have been identified in cellular responses to oxidative stress, with the majority being involved in metabolic pathways (60%). Studies have shown that protein CoAlation is a widespread post-translational modification which modulates the activity and conformation of the modified proteins. The induction of protein CoAlation by oxidative stress was found to be rapidly reversed after the removal of oxidizing agents from the medium of cultured cells. In this study, we developed an enzyme-linked immunosorbent assay (ELISA)-based deCoAlation assay to detect deCoAlation activity from Bacillus subtilis and Bacillus megaterium lysates. We then used a combination of ELISA-based assay and purification strategies to show that deCoAlation is an enzyme-driven mechanism. Using mass-spectrometry and deCoAlation assays, we identified B. subtilis YtpP (thioredoxin-like protein) and thioredoxin A (TrxA) as enzymes that can remove CoA from different substrates. With mutagenesis studies, we identified YtpP and TrxA catalytic cysteine residues and proposed a possible deCoAlation mechanism for CoAlated methionine sulfoxide reducatse A (MsrA) and peroxiredoxin 5 (PRDX5) proteins, which results in the release of both CoA and the reduced form of MsrA or PRDX5. Overall, this paper reveals the deCoAlation activity of YtpP and TrxA and opens doors to future studies on the CoA-mediated redox regulation of CoAlated proteins under various cellular stress conditions.\", \"doi_y\": \"https://doi.org/10.3390/antiox12040938\", \"publication_date\": \"2023-04-15\"}, {\"work_id\": \"W4382934283\", \"doi_x\": \"10.1101/2023.07.03.547448\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Large-scale deorphanization of<i>Nematostella vectensis</i>neuropeptide GPCRs supports the independent expansion of bilaterian and cnidarian peptidergic systems\", \"abstract\": \"Abstract Neuropeptides are ancient signaling molecules in animals but only few peptide receptors are known outside bilaterians. Cnidarians possess a large number of G protein-coupled receptors (GPCRs) \\u2013 the most common receptors of bilaterian neuropeptides \\u2013 but most of these remain orphan with no known ligands. We searched for neuropeptides in the sea anemone Nematostella vectensis and created a library of 64 peptides derived from 33 precursors. In a large-scale pharmacological screen with these peptides and 161 N. vectensis GPCRs, we identified 31 receptors specifically activated by one of 14 peptides. Mapping GPCR and neuropeptide expression to single-cell sequencing data revealed how cnidarian tissues are extensively wired by multilayer peptidergic networks. Phylogenetic analysis identified no direct orthology to bilaterian peptidergic systems and supports the independent expansion of neuropeptide signaling in cnidarians from a few ancestral peptide-receptor pairs.\", \"doi_y\": \"https://doi.org/10.1101/2023.07.03.547448\", \"publication_date\": \"2023-07-03\"}, {\"work_id\": \"W4293005050\", \"doi_x\": \"10.1038/s41563-022-01348-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Ascendancy of semi-synthetic biomaterials from design towards democratization\", \"abstract\": \"Semi-synthetic goldilocks material design integrates the tunable characteristics of synthetic materials and the refined complexity of natural components, enabling for the progress of biomaterials across length scales. Accelerated translational success may thus be possible for more personalized and accessible products.\", \"doi_y\": \"https://doi.org/10.1038/s41563-022-01348-5\", \"publication_date\": \"2022-08-24\"}, {\"work_id\": \"W4386033679\", \"doi_x\": \"10.1038/s41467-023-40695-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"TRAIP resolves DNA replication-transcription conflicts during the S-phase of unperturbed cells\", \"abstract\": \"Cell division is the basis for the propagation of life and requires accurate duplication of all genetic information. DNA damage created during replication (replication stress) is a major cause of cancer, premature aging and a spectrum of other human disorders. Over the years, TRAIP E3 ubiquitin ligase has been shown to play a role in various cellular processes that govern genome integrity and faultless segregation. TRAIP is essential for cell viability, and mutations in TRAIP ubiquitin ligase activity lead to primordial dwarfism in patients. Here, we have determined the mechanism of inhibition of cell proliferation in TRAIP-depleted cells. We have taken advantage of the auxin induced degron system to rapidly degrade TRAIP within cells and to dissect the importance of various functions of TRAIP in different stages of the cell cycle. We conclude that upon rapid TRAIP degradation, specifically in S-phase, cells cease to proliferate, arrest in G2 stage of the cell cycle and undergo senescence. Our findings reveal that TRAIP works in S-phase to prevent DNA damage at transcription start sites, caused by replication-transcription conflicts.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40695-y\", \"publication_date\": \"2023-08-21\"}, {\"work_id\": \"W4293211899\", \"doi_x\": \"10.15252/embr.202154322\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"abstract\": \"The emergence of SARS-CoV-2 variants has exacerbated the COVID-19 global health crisis. Thus far, all variants carry mutations in the spike glycoprotein, which is a critical determinant of viral transmission being responsible for attachment, receptor engagement and membrane fusion, and an important target of immunity. Variants frequently bear truncations of flexible loops in the N-terminal domain (NTD) of spike; the functional importance of these modifications has remained poorly characterised. We demonstrate that NTD deletions are important for efficient entry by the Alpha and Omicron variants and that this correlates with spike stability. Phylogenetic analysis reveals extensive NTD loop length polymorphisms across the sarbecoviruses, setting an evolutionary precedent for loop remodelling. Guided by these analyses, we demonstrate that variations in NTD loop length, alone, are sufficient to modulate virus entry. We propose that variations in NTD loop length act to fine-tune spike; this may provide a mechanism for SARS-CoV-2 to navigate a complex selection landscape encompassing optimisation of essential functionality, immune-driven antigenic variation and ongoing adaptation to a new host.\", \"doi_y\": \"https://doi.org/10.15252/embr.202154322\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W3206920052\", \"doi_x\": \"10.1042/bcj20210535\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"abstract\": \"Optimising the function of a protein of length N amino acids by directed evolution involves navigating a 'search space' of possible sequences of some 20N. Optimising the expression levels of P proteins that materially affect host performance, each of which might also take 20 (logarithmically spaced) values, implies a similar search space of 20P. In this combinatorial sense, then, the problems of directed protein evolution and of host engineering are broadly equivalent. In practice, however, they have different means for avoiding the inevitable difficulties of implementation. The spare capacity exhibited in metabolic networks implies that host engineering may admit substantial increases in flux to targets of interest. Thus, we rehearse the relevant issues for those wishing to understand and exploit those modern genome-wide host engineering tools and thinking that have been designed and developed to optimise fluxes towards desirable products in biotechnological processes, with a focus on microbial systems. The aim throughput is 'making such biology predictable'. Strategies have been aimed at both transcription and translation, especially for regulatory processes that can affect multiple targets. However, because there is a limit on how much protein a cell can produce, increasing kcat in selected targets may be a better strategy than increasing protein expression levels for optimal host engineering.\", \"doi_y\": \"https://doi.org/10.1042/bcj20210535\", \"publication_date\": \"2021-10-21\"}, {\"work_id\": \"W4321180449\", \"doi_x\": \"10.1101/2023.02.16.23285748\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"CD4+ and CD8+ T cell and antibody correlates of protection against Delta vaccine breakthrough infection: A nested case-control study within the PITCH study\", \"abstract\": \"ABSTRACT T cell correlates of protection against SARS-CoV-2 infection after vaccination (\\u2018vaccine breakthrough\\u2019) are incompletely defined, especially the specific contributions of CD4+ and CD8+ T cells. We studied 279 volunteers in the Protective Immunity from T Cells in Healthcare Workers (PITCH) UK study, including 32 cases (with SARS-CoV-2 positive testing after two vaccine doses during the Delta-dominant era) and 247 controls (no positive test nor anti-nucleocapsid seroconversion during this period). 28 days after second vaccination, before all breakthroughs occurred, cases had lower ancestral S- and RBD-specific immunoglobulin G titres and S1- and S2-specific T cell interferon gamma (IFN\\u03b3) responses compared with controls. In a subset of matched cases and controls, cases had lower CD4+ and CD8+ IFN\\u03b3 and tumour necrosis factor responses to Delta S peptides with reduced CD8+ responses to Delta versus ancestral peptides compared with controls. Our findings support a protective role for T cells against Delta breakthrough infection.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.16.23285748\", \"publication_date\": \"2023-02-17\"}, {\"work_id\": \"W4224078886\", \"doi_x\": \"10.1016/j.molcel.2022.03.030\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"H3K4 methylation by SETD1A/BOD1L facilitates RIF1-dependent NHEJ\", \"abstract\": \"The 53BP1-RIF1-shieldin pathway maintains genome stability by suppressing nucleolytic degradation of DNA ends at double-strand breaks (DSBs). Although RIF1 interacts with damaged chromatin via phospho-53BP1 and facilitates recruitment of the shieldin complex to DSBs, it is unclear whether other regulatory cues contribute to this response. Here, we implicate methylation of histone H3 at lysine 4 by SETD1A-BOD1L in the recruitment of RIF1 to DSBs. Compromising SETD1A or BOD1L expression or deregulating H3K4 methylation allows uncontrolled resection of DNA ends, impairs end-joining of dysfunctional telomeres, and abrogates class switch recombination. Moreover, defects in RIF1 localization to DSBs are evident in patient cells bearing loss-of-function mutations in SETD1A. Loss of SETD1A-dependent RIF1 recruitment in BRCA1-deficient cells restores homologous recombination and leads to resistance to poly(ADP-ribose)polymerase inhibition, reinforcing the clinical relevance of these observations. Mechanistically, RIF1 binds directly to methylated H3K4, facilitating its recruitment to, or stabilization at, DSBs.\", \"doi_y\": \"https://doi.org/10.1016/j.molcel.2022.03.030\", \"publication_date\": \"2022-05-01\"}, {\"work_id\": \"W4284666963\", \"doi_x\": \"10.1038/s41589-022-01071-x\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"abstract\": \"The abundance of recorded protein sequence data stands in contrast to the small number of experimentally verified functional annotation. Here we screened a million-membered metagenomic library at ultrahigh throughput in microfluidic droplets for \\u03b2-glucuronidase activity. We identified SN243, a genuine \\u03b2-glucuronidase with little homology to previously studied enzymes of this type, as a glycoside hydrolase 3 family member. This glycoside hydrolase family contains only one recently added \\u03b2-glucuronidase, showing that a functional metagenomic approach can shed light on assignments that are currently 'unpredictable' by bioinformatics. Kinetic analyses of SN243 characterized it as a promiscuous catalyst and structural analysis suggests regions of divergence from homologous glycoside hydrolase 3 members creating a wide-open active site. With a screening throughput of >107 library members per day, picolitre-volume microfluidic droplets enable functional assignments that complement current enzyme database dictionaries and provide bridgeheads for the annotation of unexplored sequence space.\", \"doi_y\": \"https://doi.org/10.1038/s41589-022-01071-x\", \"publication_date\": \"2022-07-07\"}, {\"work_id\": \"W4226515274\", \"doi_x\": \"10.7554/elife.67667\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Unifying the known and unknown microbial coding sequence space\", \"abstract\": \"Genes of unknown function are among the biggest challenges in molecular biology, especially in microbial systems, where 40-60% of the predicted genes are unknown. Despite previous attempts, systematic approaches to include the unknown fraction into analytical workflows are still lacking. Here, we present a conceptual framework, its translation into the computational workflow AGNOSTOS and a demonstration on how we can bridge the known-unknown gap in genomes and metagenomes. By analyzing 415,971,742 genes predicted from 1749 metagenomes and 28,941 bacterial and archaeal genomes, we quantify the extent of the unknown fraction, its diversity, and its relevance across multiple organisms and environments. The unknown sequence space is exceptionally diverse, phylogenetically more conserved than the known fraction and predominantly taxonomically restricted at the species level. From the 71 M genes identified to be of unknown function, we compiled a collection of 283,874 lineage-specific genes of unknown function for Cand. Patescibacteria (also known as Candidate Phyla Radiation, CPR), which provides a significant resource to expand our understanding of their unusual biology. Finally, by identifying a target gene of unknown function for antibiotic resistance, we demonstrate how we can enable the generation of hypotheses that can be used to augment experimental data.It is estimated that scientists do not know what half of microbial genes actually do. When these genes are discovered in microorganisms grown in the lab or found in environmental samples, it is not possible to identify what their roles are. Many of these genes are excluded from further analyses for these reasons, meaning that the study of microbial genes tends to be limited to genes that have already been described. These limitations hinder research into microbiology, because information from newly discovered genes cannot be integrated to better understand how these organisms work. Experiments to understand what role these genes have in the microorganisms are labor-intensive, so new analytical strategies are needed. To do this, Vanni et al. developed a new framework to categorize genes with unknown roles, and a computational workflow to integrate them into traditional analyses. When this approach was applied to over 400 million microbial genes (both with known and unknown roles), it showed that the share of genes with unknown functions is only about 30 per cent, smaller than previously thought. The analysis also showed that these genes are very diverse, revealing a huge space for future research and potential applications. Combining their approach with experimental data, Vanni et al. were able to identify a gene with a previously unknown purpose that could be involved in antibiotic resistance. This system could be useful for other scientists studying microorganisms to get a more complete view of microbial systems. In future, it may also be used to analyze the genetics of other organisms, such as plants and animals.\", \"doi_y\": \"https://doi.org/10.7554/elife.67667\", \"publication_date\": \"2022-03-31\"}, {\"work_id\": \"W4315752260\", \"doi_x\": \"10.1002/adma.202210098\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Ion\\u2010Selective Microporous Polymer Membranes with Hydrogen\\u2010Bond and Salt\\u2010Bridge Networks for Aqueous Organic Redox Flow Batteries\", \"abstract\": \"Abstract Redox flow batteries (RFBs) have great potential for long\\u2010duration grid\\u2010scale energy storage. Ion\\u2010conducting membranes are a crucial component in RFBs, allowing charge\\u2010carrying ions to transport while preventing the cross\\u2010mixing of redox couples. Commercial Nafion membranes are widely used in RFBs, but their unsatisfactory ionic and molecular selectivity, as well as high costs, limit the performance and the widespread deployment of this technology. To extend the longevity and reduce the cost of RFB systems, inexpensive ion\\u2010selective membranes that concurrently deliver low ionic resistance and high selectivity toward redox\\u2010active species are highly desired. Here, high\\u2010performance RFB membranes are fabricated from blends of carboxylate\\u2010 and amidoxime\\u2010functionalized polymers of intrinsic microporosity, which exploit the beneficial properties of both polymers. The enthalpy\\u2010driven formation of cohesive interchain interactions, including hydrogen bonds and salt bridges, facilitates the microscopic miscibility of the blends, while ionizable functional groups within the sub\\u2010nanometer pores allow optimization of membrane ion\\u2010transport functions. The resulting microporous membranes demonstrate fast cation conduction with low crossover of redox\\u2010active molecular species, enabling improved power ratings and reduced capacity fade in aqueous RFBs using anthraquinone and ferrocyanide as redox couples.\", \"doi_y\": \"https://doi.org/10.1002/adma.202210098\", \"publication_date\": \"2023-02-05\"}, {\"work_id\": \"W4377161612\", \"doi_x\": \"10.1021/acssuschemeng.3c00243\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Versatile Chemo-Biocatalytic Cascade Driven by a Thermophilic and Irreversible C\\u2013C Bond-Forming \\u03b1-Oxoamine Synthase\", \"abstract\": \"We report a chemo-biocatalytic cascade for the synthesis of substituted pyrroles, driven by the action of an irreversible, thermostable, pyridoxal 5\\u2032-phosphate (PLP)-dependent, C\\u2013C bond-forming biocatalyst (ThAOS). The ThAOS catalyzes the Claisen-like condensation between various amino acids and acyl-CoA substrates to generate a range of \\u03b1-aminoketones. These products are reacted with \\u03b2-keto esters in an irreversible Knorr pyrrole reaction. The determination of the 1.6 \\u00c5 resolution crystal structure of the PLP-bound form of ThAOS lays the foundation for future engineering and directed evolution. This report establishes the AOS family as useful and versatile C\\u2013C bond-forming biocatalysts.\", \"doi_y\": \"https://doi.org/10.1021/acssuschemeng.3c00243\", \"publication_date\": \"2023-05-16\"}, {\"work_id\": \"W4311861069\", \"doi_x\": \"10.1111/1751-7915.14193\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Herbicolin A production and its modulation by quorum sensing in a<i>Pantoea agglomerans</i>rhizobacterium bioactive against a broad spectrum of plant\\u2010pathogenic fungi\", \"abstract\": \"Global population growth makes it necessary to increase agricultural production yields. However, climate change impacts and diseases caused by plant pathogens are challenging modern agriculture. Therefore, it is necessary to look for alternatives to the excessive use of chemical fertilizers and pesticides. The plant microbiota plays an essential role in plant nutrition and health, and offers enormous potential to meet future challenges of agriculture. In this context, here we characterized the antifungal properties of the rhizosphere bacterium Pantoea agglomerans 9Rz4, which is active against a broad spectrum of plant pathogenic fungi. Chemical analyses revealed that strain 9Rz4 produces the antifungal herbicolin A and its biosynthetic gene cluster was identified and characterized. We found that the only acyl-homoserine lactone-based quorum sensing system of 9Rz4 modulates herbicolin A gene cluster expression. No role of plasmid carriage in the production of herbicolin A was observed. Plant assays revealed that herbicolin A biosynthesis does not affect the root colonization ability of P. agglomerans 9Rz4. Current legislative restrictions are aimed at reducing the use of chemical pesticides in agriculture, and the results derived from this study may lay the foundations for the development of novel biopesticides from rhizosphere microorganisms.\", \"doi_y\": \"https://doi.org/10.1111/1751-7915.14193\", \"publication_date\": \"2022-12-18\"}, {\"work_id\": \"W4298120764\", \"doi_x\": \"10.1002/bies.202200151\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Controlling contacts\\u2014Molecular mechanisms to regulate organelle membrane tethering\", \"abstract\": \"In recent years, membrane contact sites (MCS), which mediate interactions between virtually all subcellular organelles, have been extensively characterized and shown to be essential for intracellular communication. In this review essay, we focus on an emerging topic: the regulation of MCS. Focusing on the tether proteins themselves, we discuss some of the known mechanisms which can control organelle tethering events and identify apparent common regulatory hubs, such as the VAP interface at the endoplasmic reticulum (ER). We also highlight several currently hypothetical concepts, including the idea of tether oligomerization and redox regulation playing a role in MCS formation. We identify gaps in our current understanding, such as the identity of the majority of kinases/phosphatases involved in tether modification and conclude that a holistic approach-incorporating the formation of multiple MCS, regulated by interconnected regulatory modulators-may be required to fully appreciate the true complexity of these fascinating intracellular communication systems.\", \"doi_y\": \"https://doi.org/10.1002/bies.202200151\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4385067096\", \"doi_x\": \"10.1021/acscatal.3c01609\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Versatile Product Detection via Coupled Assays for Ultrahigh-Throughput Screening of Carbohydrate-Active Enzymes in Microfluidic Droplets\", \"abstract\": \"Enzyme discovery and directed evolution are the two major contemporary approaches for the improvement of industrial processes by biocatalysis in various fields. Customization of catalysts for improvement of single enzyme reactions or de novo reaction development is often complex and tedious. The success of screening campaigns relies on the fraction of sequence space that can be sampled, whether for evolving a particular enzyme or screening metagenomes. Ultrahigh-throughput screening (uHTS) based on in vitro compartmentalization in water-in-oil emulsion of picoliter droplets generated in microfluidic systems allows screening rates >1 kHz (or >107 per day). Screening for carbohydrate-active enzymes (CAZymes) catalyzing biotechnologically valuable reactions in this format presents an additional challenge because the released carbohydrates are difficult to monitor in high throughput. Activated substrates with large optically active hydrophobic leaving groups provide a generic optical readout, but the molecular recognition properties of sugars will be altered by the incorporation of such fluoro- or chromophores and their typically higher reactivity, as leaving groups with lowered pKa values compared to native substrates make the observation of promiscuous reactions more likely. To overcome these issues, we designed microdroplet assays in which optically inactive carbohydrate products are made visible by specific cascades: the primary reaction of an unlabeled substrate leads to an optical signal downstream. Successfully implementing such assays at the picoliter droplet scale allowed us to detect glucose, xylose, glucuronic acid, and arabinose as final products of complex oligosaccharide degradation by glycoside hydrolases by absorbance measurements. Enabling the use of uHTS for screening CAZyme reactions that have been thus far elusive will chart a route toward faster and easier development of specific and efficient biocatalysts for biovalorization, directing enzyme discovery by challenging catalysts for reaction with natural rather than model substrates.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c01609\", \"publication_date\": \"2023-07-21\"}, {\"work_id\": \"W4385839904\", \"doi_x\": \"10.3390/v15081744\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evaluation of Nafamostat as Chemoprophylaxis for SARS-CoV-2 Infection in Hamsters\", \"abstract\": \"The successful development of a chemoprophylaxis against SARS-CoV-2 could provide a tool for infection prevention that is implementable alongside vaccination programmes. Nafamostat is a serine protease inhibitor that inhibits SARS-CoV-2 entry in vitro, but it has not been characterised for chemoprophylaxis in animal models. Clinically, nafamostat is limited to intravenous delivery and has an extremely short plasma half-life. This study sought to determine whether intranasal dosing of nafamostat at 5 mg/kg twice daily was able to prevent the airborne transmission of SARS-CoV-2 from infected to uninfected Syrian Golden hamsters. SARS-CoV-2 RNA was detectable in the throat swabs of the water-treated control group 4 days after cohabitation with a SARS-CoV-2 inoculated hamster. However, throat swabs from the intranasal nafamostat-treated hamsters remained SARS-CoV-2 RNA negative for the full 4 days of cohabitation. Significantly lower SARS-CoV-2 RNA concentrations were seen in the nasal turbinates of the nafamostat-treated group compared to the control (p = 0.001). A plaque assay quantified a significantly lower concentration of infectious SARS-CoV-2 in the lungs of the nafamostat-treated group compared to the control (p = 0.035). When taken collectively with the pathological changes observed in the lungs and nasal mucosa, these data are strongly supportive of the utility of intranasally delivered nafamostat for the prevention of SARS-CoV-2 infection.\", \"doi_y\": \"https://doi.org/10.3390/v15081744\", \"publication_date\": \"2023-08-15\"}, {\"work_id\": \"W4225390182\", \"doi_x\": \"10.1107/s2059798322003576\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"abstract\": \"Crystallographers have an array of search-model options for structure solution by molecular replacement (MR). The well established options of homologous experimental structures and regular secondary-structure elements or motifs are increasingly supplemented by computational modelling. Such modelling may be carried out locally or may use pre-calculated predictions retrieved from databases such as the EBI AlphaFold database. MrParse is a new pipeline to help to streamline the decision process in MR by consolidating bioinformatic predictions in one place. When reflection data are provided, MrParse can rank any experimental homologues found using eLLG, which indicates the likelihood that a given search model will work in MR. Inbuilt displays of predicted secondary structure, coiled-coil and transmembrane regions further inform the choice of MR protocol. MrParse can also identify and rank homologues in the EBI AlphaFold database, a function that will also interest other structural biologists and bioinformaticians.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322003576\", \"publication_date\": \"2022-04-26\"}, {\"work_id\": \"W4312095111\", \"doi_x\": \"10.1042/bst20220851\", \"af_direct_citation\": 0, \"af_indirect_citations\": 8, \"title\": \"Lumenal components of cytoplasmic microtubules\", \"abstract\": \"The lumen of cytoplasmic microtubules is a poorly explored expanse of intracellular space. Although typically represented in textbooks as a hollow tube, studies over several decades have shown that the microtubule lumen is occupied by a range of morphologically diverse components. These are predominantly globular particles of varying sizes which appear to exist either in isolation, bind to the microtubule wall, or form discontinuous columns that extend through the lumenal space. Actin filaments with morphologies distinct from the canonical cytoplasmic forms have also now been found within the microtubule lumen. In this review, we examine the historic literature that observed these lumenal components in tissues from diverse species and integrate it with recent cryo-electron tomography studies that have begun to identify lumenal proteins. We consider their cell and tissue distribution, possible mechanisms of incorporation, and potential functions. It is likely that continuing work in this area will open a new frontier in cytoskeletal biology.\", \"doi_y\": \"https://doi.org/10.1042/bst20220851\", \"publication_date\": \"2022-12-16\"}, {\"work_id\": \"W4380200308\", \"doi_x\": \"10.1016/j.pt.2023.05.007\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protein disulfide isomerases \\u2013 a way to tackle malaria\", \"abstract\": \"Protein disulfide isomerases (PDIs) ensure that specific substrate proteins are correctly folded. PDI activity plays an essential role in malaria transmission. Here we provide an overview of the role of PDIs in malaria-causing Plasmodium parasites and outline why PDI inhibition could be a novel way to treat malaria and prevent transmission.\", \"doi_y\": \"https://doi.org/10.1016/j.pt.2023.05.007\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4224950216\", \"doi_x\": \"10.1016/s0140-6736(22)00462-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Comparative analysis of the risks of hospitalisation and death associated with SARS-CoV-2 omicron (B.1.1.529) and delta (B.1.617.2) variants in England: a cohort study\", \"abstract\": \"BackgroundThe omicron variant (B.1.1.529) of SARS-CoV-2 has demonstrated partial vaccine escape and high transmissibility, with early studies indicating lower severity of infection than that of the delta variant (B.1.617.2). We aimed to better characterise omicron severity relative to delta by assessing the relative risk of hospital attendance, hospital admission, or death in a large national cohort.MethodsIndividual-level data on laboratory-confirmed COVID-19 cases resident in England between Nov 29, 2021, and Jan 9, 2022, were linked to routine datasets on vaccination status, hospital attendance and admission, and mortality. The relative risk of hospital attendance or admission within 14 days, or death within 28 days after confirmed infection, was estimated using proportional hazards regression. Analyses were stratified by test date, 10-year age band, ethnicity, residential region, and vaccination status, and were further adjusted for sex, index of multiple deprivation decile, evidence of a previous infection, and year of age within each age band. A secondary analysis estimated variant-specific and vaccine-specific vaccine effectiveness and the intrinsic relative severity of omicron infection compared with delta (ie, the relative risk in unvaccinated cases).FindingsThe adjusted hazard ratio (HR) of hospital attendance (not necessarily resulting in admission) with omicron compared with delta was 0\\u00b756 (95% CI 0\\u00b754\\u20130\\u00b758); for hospital admission and death, HR estimates were 0\\u00b741 (0\\u00b739\\u20130\\u00b743) and 0\\u00b731 (0\\u00b726\\u20130\\u00b737), respectively. Omicron versus delta HR estimates varied with age for all endpoints examined. The adjusted HR for hospital admission was 1\\u00b710 (0\\u00b785\\u20131\\u00b742) in those younger than 10 years, decreasing to 0\\u00b725 (0\\u00b721\\u20130\\u00b730) in 60\\u201369-year-olds, and then increasing to 0\\u00b747 (0\\u00b740\\u20130\\u00b756) in those aged at least 80 years. For both variants, past infection gave some protection against death both in vaccinated (HR 0\\u00b747 [0\\u00b732\\u20130\\u00b768]) and unvaccinated (0\\u00b718 [0\\u00b706\\u20130\\u00b757]) cases. In vaccinated cases, past infection offered no additional protection against hospital admission beyond that provided by vaccination (HR 0\\u00b796 [0\\u00b788\\u20131\\u00b704]); however, for unvaccinated cases, past infection gave moderate protection (HR 0\\u00b755 [0\\u00b748\\u20130\\u00b763]). Omicron versus delta HR estimates were lower for hospital admission (0\\u00b730 [0\\u00b728\\u20130\\u00b732]) in unvaccinated cases than the corresponding HR estimated for all cases in the primary analysis. Booster vaccination with an mRNA vaccine was highly protective against hospitalisation and death in omicron cases (HR for hospital admission 8\\u201311 weeks post-booster vs unvaccinated: 0\\u00b722 [0\\u00b720\\u20130\\u00b724]), with the protection afforded after a booster not being affected by the vaccine used for doses 1 and 2.InterpretationThe risk of severe outcomes following SARS-CoV-2 infection is substantially lower for omicron than for delta, with higher reductions for more severe endpoints and significant variation with age. Underlying the observed risks is a larger reduction in intrinsic severity (in unvaccinated individuals) counterbalanced by a reduction in vaccine effectiveness. Documented previous SARS-CoV-2 infection offered some protection against hospitalisation and high protection against death in unvaccinated individuals, but only offered additional protection in vaccinated individuals for the death endpoint. Booster vaccination with mRNA vaccines maintains over 70% protection against hospitalisation and death in breakthrough confirmed omicron infections.FundingMedical Research Council, UK Research and Innovation, Department of Health and Social Care, National Institute for Health Research, Community Jameel, and Engineering and Physical Sciences Research Council.\", \"doi_y\": \"https://doi.org/10.1016/s0140-6736(22)00462-7\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W3199283396\", \"doi_x\": \"10.3390/ph14100953\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The Novel Application of Geometric Morphometrics with Principal Component Analysis to Existing G Protein-Coupled Receptor (GPCR) Structures\", \"abstract\": \"The G protein-coupled receptor (GPCR) superfamily is a large group of membrane proteins which, because of their vast involvement in cell signalling pathways, are implicated in a plethora of disease states and are therefore considered to be key drug targets. Despite advances in techniques to study these receptors, current prophylaxis is often limited due to the challenging nature of their dynamic, complex structures. Greater knowledge and understanding of their intricate structural rearrangements will therefore undoubtedly aid structure-based drug design against GPCRs. Disciplines such as anthropology and palaeontology often use geometric morphometrics to measure variation between shapes and we have therefore applied this technique to analyse GPCR structures in a three-dimensional manner, using principal component analysis. Our aim was to create a novel system able to discriminate between GPCR structures and discover variation between them, correlated with a variety of receptor characteristics. This was conducted by assessing shape changes at the extra- and intracellular faces of the transmembrane helix bundle, analysing the XYZ coordinates of the amino acids at those positions. We have demonstrated that GPCR structures can be classified based on characteristics such as activation state, bound ligands and fusion proteins, with the most significant results focussed at the intracellular face. Conversely, our analyses provide evidence that thermostabilising mutations do not cause significant differences when compared to non-mutated GPCRs. We believe that this is the first time geometric morphometrics has been applied to membrane proteins on this scale, and believe it can be used as a future tool in sense-checking newly resolved structures and planning experimental design.\", \"doi_y\": \"https://doi.org/10.3390/ph14100953\", \"publication_date\": \"2021-09-23\"}, {\"work_id\": \"W4291014905\", \"doi_x\": \"10.1021/acs.jmedchem.2c00804\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"abstract\": \"G-protein-coupled receptor 84 (GPR84) is a proinflammatory orphan G-protein-coupled receptor implicated in several inflammatory and fibrotic diseases. Several agonist and antagonist ligands have been developed that target GPR84; however, a noncompetitive receptor blocker that was progressed to phase II clinical trials failed to demonstrate efficacy. New high-quality antagonists are required to investigate the pathophysiological role of GPR84 and to validate GPR84 as a therapeutic target. We previously reported the discovery of a novel triazine GPR84 competitive antagonist 1. Here, we describe an extensive structure-activity relationship (SAR) of antagonist 1 and also present in silico docking with supporting mutagenesis studies that reveals a potential binding pose for this type of orthosteric antagonist. Lead compound 42 is a potent GPR84 antagonist with a favorable pharmacokinetic (PK) profile suitable for further drug development.\", \"doi_y\": \"https://doi.org/10.1021/acs.jmedchem.2c00804\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4307918867\", \"doi_x\": \"10.1002/ange.202213053\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"abstract\": \"Abstract Abyssomicin C and its atropisomer are potent inhibitors of bacterial folate metabolism. They possess complex polycyclic structures, and their biosynthesis has been shown to involve several unusual enzymatic transformations. Using a combination of synthesis and in vitro assays we reveal that AbyV, a cytochrome P450 enzyme from the aby gene cluster, catalyses a key late\\u2010stage epoxidation required for the installation of the characteristic ether\\u2010bridged core of abyssomicin C. The X\\u2010ray crystal structure of AbyV has been determined, which in combination with molecular dynamics simulations provides a structural framework for our functional data. This work demonstrates the power of combining selective carbon\\u201013 labelling with NMR spectroscopy as a sensitive tool to interrogate enzyme\\u2010catalysed reactions in vitro with no need for purification.\", \"doi_y\": \"https://doi.org/10.1002/ange.202213053\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4380870595\", \"doi_x\": \"10.1038/s41564-023-01413-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mechanistic basis for potent neutralization of Sin Nombre hantavirus by a human monoclonal antibody\", \"abstract\": \"Rodent-borne hantaviruses are prevalent worldwide and upon spillover to human populations, cause severe disease for which no specific treatment is available. A potent antibody response is key for recovery from hantavirus infection. Here we study a highly neutralizing human monoclonal antibody, termed SNV-42, which was derived from a memory B cell isolated from an individual with previous Sin Nombre virus (SNV) infection. Crystallographic analysis demonstrates that SNV-42 targets the Gn subcomponent of the tetrameric (Gn-Gc)4 glycoprotein assembly that is relevant for viral entry. Integration of our 1.8 \\u00c5 structure with the (Gn-Gc)4 ultrastructure arrangement indicates that SNV-42 targets the membrane-distal region of the virus envelope. Comparison of the SNV-42 paratope encoding variable genes with inferred germline gene segments reveals high sequence conservation, suggesting that germline-encoded antibodies inhibit SNV. Furthermore, mechanistic assays reveal that SNV-42 interferes with both receptor recognition and fusion during host-cell entry. This work provides a molecular-level blueprint for understanding the human neutralizing antibody response to hantavirus infection.\", \"doi_y\": \"https://doi.org/10.1038/s41564-023-01413-y\", \"publication_date\": \"2023-06-15\"}, {\"work_id\": \"W4311291543\", \"doi_x\": \"10.1039/d2sc04072g\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of complexes III and IV\", \"abstract\": \"Cardiolipin interacts more strongly than a disease-related lipid, monolysocardiolipin, at the interface of Complex III and Complex IV in mitochondrial respiratory supercomplexes.\", \"doi_y\": \"https://doi.org/10.1039/d2sc04072g\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4317600508\", \"doi_x\": \"10.1073/pnas.2209964120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"TMEM161B regulates cerebral cortical gyration, Sonic Hedgehog signaling, and ciliary structure in the developing central nervous system\", \"abstract\": \"Sonic hedgehog signaling regulates processes of embryonic development across multiple tissues, yet factors regulating context-specific Shh signaling remain poorly understood. Exome sequencing of families with polymicrogyria (disordered cortical folding) revealed multiple individuals with biallelic deleterious variants in TMEM161B, which encodes a multi-pass transmembrane protein of unknown function. Tmem161b null mice demonstrated holoprosencephaly, craniofacial midline defects, eye defects, and spinal cord patterning changes consistent with impaired Shh signaling, but were without limb defects, suggesting a CNS-specific role of Tmem161b. Tmem161b depletion impaired the response to Smoothened activation in vitro and disrupted cortical histogenesis in vivo in both mouse and ferret models, including leading to abnormal gyration in the ferret model. Tmem161b localizes non-exclusively to the primary cilium, and scanning electron microscopy revealed shortened, dysmorphic, and ballooned ventricular zone cilia in the Tmem161b null mouse, suggesting that the Shh-related phenotypes may reflect ciliary dysfunction. Our data identify TMEM161B as a regulator of cerebral cortical gyration, as involved in primary ciliary structure, as a regulator of Shh signaling, and further implicate Shh signaling in human gyral development.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2209964120\", \"publication_date\": \"2023-01-20\"}, {\"work_id\": \"W4386801812\", \"doi_x\": \"10.1021/acscatal.3c03047\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Decoding Catalysis by Terpene Synthases\", \"abstract\": \"The review by Christianson, published in 2017 on the twentieth anniversary of the emergence of the field, summarizes the foundational discoveries and key advances in terpene synthase/cyclase (TS) biocatalysis (Christianson, D. W. Chem Rev2017, 117 (17), 11570-11648. DOI: 10.1021/acs.chemrev.7b00287). Here, we review the TS literature published since then, bringing the field up to date and looking forward to what could be the near future of TS rational design. Many revealing discoveries have been made in recent years, building on the knowledge and fundamental principles uncovered during those initial two decades of study. We use these to explore TS reaction chemistry and see how a combined experimental and computational approach helps to decipher the complexities of TS catalysis. Revealed are a suite of catalytic motifs which control product outcome in TSs, some obvious, some more subtle. We examine each in detail, using the most recent papers and insights to illustrate how exactly this fascinating class of enzymes takes a single acyclic substrate and turns it into the many thousands of complex terpenoids found in Nature. We then explore some of the recent strategies for TS engineering, including machine learning and other data-driven approaches. From this, rational and predictive engineering of TSs, \\\"designer terpene synthases\\\", will begin to emerge as a realistic goal.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c03047\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4200542864\", \"doi_x\": \"10.1101/2021.12.09.472018\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"abstract\": \"Abstract Bacterial metabolosomes are a family of protein organelles in bacteria. Elucidating how thousands of proteins self-assemble to form functional metabolosomes is essential for understanding their significance in cellular metabolism and pathogenesis. Here we investigate the de novo biogenesis of propanediol-utilization (Pdu) metabolosomes and characterize the roles of the key constituents in generation and intracellular positioning of functional metabolosomes. Our results demonstrate that the Pdu metabolosome undertakes both \\u201cShell first\\u201d and \\u201cCargo first\\u201d assembly pathways, unlike the \\u03b2-carboxysome structural analog which only involves the \\u201cCargo first\\u201d strategy. Shell and cargo assemblies occur independently at the cell poles. The internal cargo core is formed through the ordered assembly of multiple enzyme complexes, and exhibits liquid-like properties within the metabolosome architecture. Our findings provide mechanistic insight into the molecular principles driving bacterial metabolosome assembly and expand our understanding of liquid-like organelle biogenesis.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.09.472018\", \"publication_date\": \"2021-12-09\"}, {\"work_id\": \"W4316926589\", \"doi_x\": \"10.1093/bioinformatics/btad029\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"abstract\": \"CATH is a protein domain classification resource that exploits an automated workflow of structure and sequence comparison alongside expert manual curation to construct a hierarchical classification of evolutionary and structural relationships. The aim of this study was to develop algorithms for detecting remote homologues missed by state-of-the-art hidden Markov model (HMM)-based approaches. The method developed (CATHe) combines a neural network with sequence representations obtained from protein language models. It was assessed using a dataset of remote homologues having less than 20% sequence identity to any domain in the training set.The CATHe models trained on 1773 largest and 50 largest CATH superfamilies had an accuracy of 85.6 \\u00b1 0.4% and 98.2 \\u00b1 0.3%, respectively. As a further test of the power of CATHe to detect more remote homologues missed by HMMs derived from CATH domains, we used a dataset consisting of protein domains that had annotations in Pfam, but not in CATH. By using highly reliable CATHe predictions (expected error rate <0.5%), we were able to provide CATH annotations for 4.62 million Pfam domains. For a subset of these domains from Homo sapiens, we structurally validated 90.86% of the predictions by comparing their corresponding AlphaFold2 structures with structures from the CATH superfamilies to which they were assigned.The code for the developed models is available on https://github.com/vam-sin/CATHe, and the datasets developed in this study can be accessed on https://zenodo.org/record/6327572.Supplementary data are available at Bioinformatics online.\", \"doi_y\": \"https://doi.org/10.1093/bioinformatics/btad029\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4311448319\", \"doi_x\": \"10.1101/2022.12.12.520052\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Macromolecular crowding is surprisingly unable to deform the structure of a model biomolecular condensate\", \"abstract\": \"Abstract The crowded interior of a living cell makes experiments on simpler in vitro systems attractive. Although these reveal interesting phenomena, their biological relevance can be questionable. A topical example is the phase separation of intrinsically-disordered proteins into biomolecular condensates, which is proposed to underlie the membraneless compartmentalisation of many cellular functions. How a cell reliably controls biochemical reactions in compartments open to the compositionally-varying cytoplasm is an important question for understanding cellular homeostasis. Computer simulations are often used to study the phase behaviour of model biomolecular condensates, but the number of relevant parameters explodes as the number of protein components increases. It is unfeasible to exhaustively simulate such models for all parameter combinations, although interesting phenomena are almost certainly hidden in the jungle of their high-dimensional parameter space. Here we have studied the phase behaviour of a model biomolecular condensate in the presence of a polymeric crowding agent. We used a novel compute framework to execute dozens of simultaneous simulations spanning the protein/crowder concentration space. We then combined the results into a graphical representation for human interpretation, which provided an efficient way to search the model\\u2019s high-dimensional parameter space. We found that steric repulsion from the crowder drives a near-critical system across the phase boundary, but the molecular arrangement within the resulting biomolecular condensate is rather insensitive to the crowder concentration and molecular weight. We propose that a cell may use the local cytoplasmic concentration to assist formation of biomolecular condensates, while relying on the dense phase reliably providing a stable, structured, fluid milieu for cellular biochemistry despite being open to its changing environment.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.12.520052\", \"publication_date\": \"2022-12-14\"}, {\"work_id\": \"W4301394817\", \"doi_x\": \"10.1101/2022.10.03.508498\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Modelling Membrane Reshaping by Staged Polymerization of ESCRT-III Filaments\", \"abstract\": \"Abstract ESCRT-III filaments are composite cytoskeletal polymers that can constrict and cut cell membranes from the inside of the membrane neck. Membrane-bound ESCRT-III filaments undergo a series of dramatic composition and geometry changes in the presence of an ATP-consuming Vps4 enzyme, which causes stepwise changes in the membrane morphology. We set out to understand the physical mechanisms involved in translating the changes in ESCRT-III polymer composition into membrane deformation. We have built a coarse-grained model in which ESCRT-III polymers of different geometries and mechanical properties are allowed to copolymerise and bind to a deformable membrane. By modelling ATP-driven stepwise depolymerisation of specific polymers, we identify mechanical regimes in which changes in filament composition trigger the associated membrane transition from a flat to a buckled state, and then to a tubule state that eventually undergoes scission to release a small cargo-loaded vesicle. We then characterise how the location and kinetics of polymer loss affects the extent of membrane deformation and the efficiency of membrane neck scission. Our results identify the near-minimal mechanical conditions for the operation of shape-shifting composite polymers that sever membrane necks. Author summary ESCRT-III proteins have the unique ability to cut membrane necks from within, which is needed for a vast number of cell remodelling events including the release of cargo-containing vesicles. ESCRT-III proteins exist in different forms, which can assemble into spiral and helical homopolymers of different curvatures, and they have been suggested to polymerize and depolymerize with each other in a staged manner to deform and cut membranes. We developed a computer model to explore the physical mechanisms behind vesicle budding driven by the staged assembly and disassembly of multiple elastic filaments. We identified rules that determine the outcomes of membrane remodelling, which depend on the relative physical features of the distinct filaments, the dynamics of their disassembly, and on the presence of cargo; thereby providing experimentally testable predictions. Our study provides new physical insights into the ESCRT-III-mediated vesicle budding process, at a single subunit level, and identifies the general design principles of nanomachines built from shapeshifting copolymers, which might also be realized in synthetic systems.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.03.508498\", \"publication_date\": \"2022-10-04\"}, {\"work_id\": \"W4312202064\", \"doi_x\": \"10.1186/s13059-022-02837-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Implicating genes, pleiotropy, and sexual dimorphism at blood lipid loci through multi-ancestry meta-analysis\", \"abstract\": \"Abstract Background Genetic variants within nearly 1000 loci are known to contribute to modulation of blood lipid levels. However, the biological pathways underlying these associations are frequently unknown, limiting understanding of these findings and hindering downstream translational efforts such as drug target discovery. Results To expand our understanding of the underlying biological pathways and mechanisms controlling blood lipid levels, we leverage a large multi-ancestry meta-analysis ( N = 1,654,960) of blood lipids to prioritize putative causal genes for 2286 lipid associations using six gene prediction approaches. Using phenome-wide association (PheWAS) scans, we identify relationships of genetically predicted lipid levels to other diseases and conditions. We confirm known pleiotropic associations with cardiovascular phenotypes and determine novel associations, notably with cholelithiasis risk. We perform sex-stratified GWAS meta-analysis of lipid levels and show that 3\\u20135% of autosomal lipid-associated loci demonstrate sex-biased effects. Finally, we report 21 novel lipid loci identified on the X chromosome. Many of the sex-biased autosomal and X chromosome lipid loci show pleiotropic associations with sex hormones, emphasizing the role of hormone regulation in lipid metabolism. Conclusions Taken together, our findings provide insights into the biological mechanisms through which associated variants lead to altered lipid levels and potentially cardiovascular disease risk.\", \"doi_y\": \"https://doi.org/10.1186/s13059-022-02837-1\", \"publication_date\": \"2022-12-27\"}, {\"work_id\": \"W4296642242\", \"doi_x\": \"10.1002/prot.26429\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"abstract\": \"Bacteria are equipped with a diverse set of regulatory tools that allow them to quickly adapt to their environment. The RimK system allows for Pseudomonas spp. to adapt through post-transcriptional regulation by altering the ribosomal subunit RpsF. RimK is found in a wide range of bacteria with a conserved amino acid sequence, however, the genetic context and the role of this protein is highly diverse. By solving and comparing the structures of RimK homologs from two related but functionally divergent systems, we uncovered key structural differences that likely contribute to the different activity levels of each of these homologs. Moreover, we were able to clearly resolve the active site of this protein for the first time, resolving binding of the glutamate substrate. This work advances our understanding of how subtle differences in protein sequence and structure can have profound effects on protein activity, which can in turn result in widespread mechanistic changes.\", \"doi_y\": \"https://doi.org/10.1002/prot.26429\", \"publication_date\": \"2022-10-06\"}, {\"work_id\": \"W4323036511\", \"doi_x\": \"10.1093/genetics/iyad031\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The Gene Ontology knowledgebase in 2023\", \"abstract\": \"The Gene Ontology (GO) knowledgebase (http://geneontology.org) is a comprehensive resource concerning the functions of genes and gene products (proteins and noncoding RNAs). GO annotations cover genes from organisms across the tree of life as well as viruses, though most gene function knowledge currently derives from experiments carried out in a relatively small number of model organisms. Here, we provide an updated overview of the GO knowledgebase, as well as the efforts of the broad, international consortium of scientists that develops, maintains, and updates the GO knowledgebase. The GO knowledgebase consists of three components: (1) the GO-a computational knowledge structure describing the functional characteristics of genes; (2) GO annotations-evidence-supported statements asserting that a specific gene product has a particular functional characteristic; and (3) GO Causal Activity Models (GO-CAMs)-mechanistic models of molecular \\\"pathways\\\" (GO biological processes) created by linking multiple GO annotations using defined relations. Each of these components is continually expanded, revised, and updated in response to newly published discoveries and receives extensive QA checks, reviews, and user feedback. For each of these components, we provide a description of the current contents, recent developments to keep the knowledgebase up to date with new discoveries, and guidance on how users can best make use of the data that we provide. We conclude with future directions for the project.\", \"doi_y\": \"https://doi.org/10.1093/genetics/iyad031\", \"publication_date\": \"2023-03-03\"}, {\"work_id\": \"W4288385858\", \"doi_x\": \"10.1042/bst20220405\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Unravelling mechanisms that govern meiotic crossover formation in wheat\", \"abstract\": \"Wheat is a major cereal crop that possesses a large allopolyploid genome formed through hybridisation of tetraploid and diploid progenitors. During meiosis, crossovers (COs) are constrained in number to 1-3 per chromosome pair that are predominantly located towards the chromosome ends. This reduces the probability of advantageous traits recombining onto the same chromosome, thus limiting breeding. Therefore, understanding the underlying factors controlling meiotic recombination may provide strategies to unlock the genetic potential in wheat. In this mini-review, we will discuss the factors associated with restricted CO formation in wheat, such as timing of meiotic events, chromatin organisation, pre-meiotic DNA replication and dosage of CO genes, as a means to modulate recombination.\", \"doi_y\": \"https://doi.org/10.1042/bst20220405\", \"publication_date\": \"2022-07-28\"}, {\"work_id\": \"W4288988319\", \"doi_x\": \"10.1016/j.molp.2022.07.012\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"abstract\": \"Species of the genus Phytophthora, the plant killer, cause disease and reduce yields in many crop plants. Although many Resistance to Phytophthora infestans (Rpi) genes effective against potato late blight have been cloned, few have been cloned against other Phytophthora species. Most Rpi genes encode nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptor proteins that recognize RXLR (Arg-X-Leu-Arg) effectors. However, whether NLR proteins can recognize RXLR effectors from multiple Phytophthora species has rarely been investigated. Here, we identified a new RXLR-WY effector AVRamr3 from P. infestans that is recognized by Rpi-amr3 from a wild Solanaceae species Solanum americanum. Rpi-amr3 associates with AVRamr3 in planta. AVRamr3 is broadly conserved in many different Phytophthora species, and the recognition of AVRamr3 homologs by Rpi-amr3 activates resistance against multiple Phytophthora pathogens, including the tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is thus the first characterized resistance gene that acts against P. parasitica or P. palmivora. These findings suggest a novel path to redeploy known R genes against different important plant pathogens. Species of the genus Phytophthora, the plant killer, cause disease and reduce yields in many crop plants. Although many Resistance to Phytophthora infestans (Rpi) genes effective against potato late blight have been cloned, few have been cloned against other Phytophthora species. Most Rpi genes encode nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptor proteins that recognize RXLR (Arg-X-Leu-Arg) effectors. However, whether NLR proteins can recognize RXLR effectors from multiple Phytophthora species has rarely been investigated. Here, we identified a new RXLR-WY effector AVRamr3 from P. infestans that is recognized by Rpi-amr3 from a wild Solanaceae species Solanum americanum. Rpi-amr3 associates with AVRamr3 in planta. AVRamr3 is broadly conserved in many different Phytophthora species, and the recognition of AVRamr3 homologs by Rpi-amr3 activates resistance against multiple Phytophthora pathogens, including the tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is thus the first characterized resistance gene that acts against P. parasitica or P. palmivora. These findings suggest a novel path to redeploy known R genes against different important plant pathogens. IntroductionSpecies in the oomycete genus Phytophthora cause many devastating plant diseases. For example, P. infestans, P. parasitica, P. cactorum, P. ramorum, P. sojae, P. palmivora, and P. megakarya cause potato and tomato late blight, tobacco black shank disease, strawberry crown and leather rot, sudden oak death, soybean root and stem rot, and cacao black pod disease, respectively. P. infestans and P. sojae infect few plant species, while others such as P. parasitica, P. ramorum, and P. palmivora have a broad host range (Kamoun et al., 2015Kamoun S. Furzer O. Jones J.D.G. Judelson H.S. Ali G.S. Dalio R.J.D. Roy S.G. Schena L. Zambounis A. Panabi\\u00e8res F. et al.The Top 10 oomycete pathogens in molecular plant pathology.Mol. Plant Pathol. 2015; 16: 413-434Crossref PubMed Scopus (451) Google Scholar).Plant immunity involves detection of pathogen-derived molecules by either cell-surface pattern recognition immune receptors or intracellular nucleotide-binding domain, leucine-rich repeat-containing (NLR) immune receptors, which activate either pattern-triggered immunity (PTI) or effector-triggered immunity (ETI), respectively (Jones and Dangl, 2006Jones J.D.G. Dangl J.L. The plant immune system.Nature. 2006; 444: 323-329Crossref PubMed Scopus (8005) Google Scholar). So far, more than 20 Resistance to P. infestans (Rpi) genes were cloned from wild Solanum species that confer resistance against potato late blight (Vleeshouwers et al., 2011Vleeshouwers V.G.A.A. Raffaele S. Vossen J.H. Champouret N. Oliva R. Segretin M.E. Rietman H. Cano L.M. Lokossou A. Kessel G. et al.Understanding and exploiting late blight resistance in the age of effectors.Annu. Rev. Phytopathol. 2011; 49: 507-531Crossref PubMed Scopus (276) Google Scholar). Several Resistance genes against P. sojae (Rps) have also been mapped in different soybean accessions, and a few were cloned (Sahoo et al., 2017Sahoo D.K. Abeysekara N.S. Cianzio S.R. Robertson A.E. Bhattacharyya M.K. A novel Phytophthora sojae resistance Rps12 gene mapped to a genomic region that contains several Rps genes.PLoS ONE. 2017; 12: e0169950Crossref PubMed Scopus (40) Google Scholar; Wang et al., 2021Wang W. Chen L. Fengler K. Bolar J. Llaca V. Wang X. Clark C.B. Fleury T.J. Myrvold J. Oneal D. et al.A giant NLR gene confers broad-spectrum resistance to Phytophthora sojae in soybean.Nat. Commun. 2021; 12: 6263-6268Crossref PubMed Scopus (8) Google Scholar). In tobacco, the black shank resistance genes Phl, Php, and Ph were genetically mapped, but not yet cloned; these confer race-specific resistance to P. parasitica (also known as [aka] P. nicotianae) isolates (Gallup and Shew, 2010Gallup C.A. Shew H.D. Occurrence of race 3 of Phytophthora nicotianae in North Carolina, the causal agent of black shank of tobacco.Plant Dis. 2010; 94: 557-562Crossref PubMed Scopus (23) Google Scholar; Bao et al., 2019Bao Y. Ding N. Qin Q. Wu X. Martinez N. Miller R. Zaitlin D. Li D. Yang S. Genetic mapping of the Ph gene conferring disease resistance to black shank in tobacco.Mol. Breeding. 2019; 39: 122https://doi.org/10.1007/s11032-019-1036-xCrossref Scopus (3) Google Scholar). For P. palmivora, some resistant cacao (Theobroma cacao) accessions were identified, but no dominant R genes have been defined or cloned (Thevenin et al., 2012Thevenin J.-M. Rossi V. Ducamp M. Doare F. Condina V. Lachenaud P. Numerous clones resistant to Phytophthora palmivora in the \\u201cGuiana\\u201d genetic group of Theobroma cacao L.PLoS One. 2012; 7: e40915Crossref PubMed Scopus (19) Google Scholar). In summary, apart from Rpi genes, very few R genes against Phytophthora pathogens have been cloned.Solanum americanum and Solanum nigrum are wild Solanaceae species and are highly resistant to P. infestans (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar, Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). Two Rpi genes of coiled-coil (CC) type, Rpi-amr3 and Rpi-amr1, were cloned from different S. americanum accessions; both confer late blight resistance in cultivated potato (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar, Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). S. nigrum is a hexaploid species that was thought to be a \\u201cnon-host\\u201d plant of P. infestans. No Rpi gene had been cloned from S. nigrum until we reported the functional Rpi-amr1 homolog Rpi-nig1 (Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar).In oomycetes, the recognized effectors are usually secreted RXLR (Arg-X-Leu-Arg, X represents any amino acid)-EER (Glu-Glu-Arg) proteins that are translocated into plant cells (Rehmany et al., 2005Rehmany A.P. Gordon A. Rose L.E. Allen R.L. Armstrong M.R. Whisson S.C. Kamoun S. Tyler B.M. Birch P.R.J. Beynon J.L. Differential recognition of highly divergent downy mildew avirulence gene alleles by RPP1 resistance genes from two Arabidopsis lines.Plant Cell. 2005; 17: 1839-1850Crossref PubMed Scopus (319) Google Scholar; Wang et al., 2019Wang S. McLellan H. Bukharova T. He Q. Murphy F. Shi J. Sun S. van Weymers P. Ren Y. Thilliez G. et al.Phytophthora infestans RXLR effectors act in concert at diverse subcellular locations to enhance host colonization.J. Exp. Bot. 2019; 70: 343-356Crossref PubMed Scopus (34) Google Scholar). Dozens of Avirulence (Avr) genes encoding recognized effectors from Phytophthora species have been identified, and they are typically fast-evolving and lineage-specific molecules (Jiang et al., 2008Jiang R.H.Y. Tripathy S. Govers F. Tyler B.M. RXLR effector reservoir in two Phytophthora species is dominated by a single rapidly evolving superfamily with more than 700 members.Proc. Natl. Acad. Sci. USA. 2008; 105: 4874-4879Crossref PubMed Scopus (304) Google Scholar). Recently, AVRamr1 (PITG_07569), the recognized effector of Rpi-amr1, was identified by a cDNA pathogen enrichment sequencing approach (Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar). Surprisingly, AVRamr1 homologs were identified from P. parasitica and P. cactorum genomes, and both are recognized by all Rpi-amr1 variants (Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar). Similarly, AVR3a-like effectors were found in different Phytophthora species, including P. capsici and P. sojae, and the recognition of AVR3a homologs correlates with P. capsici or P. sojae resistance in Nicotiana species and soybean (Shan et al., 2004Shan W. Cao M. Leung D. Tyler B.M. The Avr1b locus of Phytophthora sojae encodes an elicitor and a regulator required for avirulence on soybean plants carrying resistance gene Rps1b.Mol. Plant Microbe Interact. 2004; 17: 394-403Crossref PubMed Scopus (241) Google Scholar; Vega-Arregu\\u00edn et al., 2014Vega-Arregu\\u00edn J.C. Jalloh A. Bos J.I. Moffett P. Recognition of an Avr3a homologue plays a major role in mediating nonhost resistance to Phytophthora capsici in Nicotiana species.Mol. Plant Microbe Interact. 2014; 27: 770-780Crossref PubMed Scopus (48) Google Scholar). In addition, AVRblb2 homologs from P. andina and P. mirabilis trigger a hypersensitive response (HR) with Rpi-blb2 (Oliva et al., 2015Oliva R.F. Cano L.M. Raffaele S. Win J. Bozkurt T.O. Belhaj K. Oh S.-K. THINES M. Kamoun S. A recent expansion of the RXLR effector gene Avrblb2 is maintained in global populations of Phytophthora infestans indicating different contributions to virulence.Mol. Plant Microbe Interact. 2015; 28: 901-912Crossref PubMed Scopus (25) Google Scholar). Remarkably, a single N336Y mutation in R3a expands its recognition specificity to a P. capsici AVR3a homolog (Segretin et al., 2014Segretin M.E. Pais M. Franceschetti M. Chaparro-Garcia A. Bos J.I.B. Banfield M.J. Kamoun S. Single amino acid mutations in the potato immune receptor R3a expand response to Phytophthora effectors.Mol. Plant Microbe Interact. 2014; 27: 624-637Crossref PubMed Scopus (80) Google Scholar). These reports raise intriguing questions. Could RXLR effectors be widely conserved molecules among different Phytophthora species? Could these effectors be recognized by the same plant immune receptor? Of particular interest, could such effector recognition capacity enable disease resistance?Here we show that Rpi-amr3 confers resistance to all tested late blight isolates in both the field and laboratory conditions. We also identified AVRamr3, a novel AVR protein from P. infestans, by screening an RXLR effector library. AVRamr3 is a broadly conserved effector found in 13 different Phytophthora species. We also show functional Rpi-amr3 genes are widely distributed among S. americanum and S. nigrum accessions. The recognition of AVRamr3 not only enables resistance to a wide range of P. infestans isolates but also to other economically important Phytophthora pathogens such as tobacco black shank disease and cacao black pod disease pathogens P. parasitica and P. palmivora. Rpi-amr3 is the first reported R gene that confers resistance against P. parasitica and P. palmivora.ResultsRpi-amr3 confers late blight resistance in the laboratory and field conditionsRpi-amr3 was cloned by SMRT-RenSeq and reported to confer resistance against two Phytophthora infestans isolates, 88069 and 06_3928A, in a diploid potato line (Line 26, Solynta B.V.) (Witek et al., 2016Witek K. Jupe F. Witek A.I. Baker D. Clark M.D. Jones J.D.G. Accelerated cloning of a potato late blight\\u2013resistance gene using RenSeq and SMRT sequencing.Nat. Biotechnol. 2016; 34: 656-660Crossref PubMed Scopus (138) Google Scholar) in the laboratory conditions. However, whether Rpi-amr3 confers broad-spectrum and field resistance to late blight was not reported.To address this, we transformed Rpi-amr3 with its native promoter and terminator into a favored UK potato cultivar cv. Maris Piper. Two lines (SLJ24895-5C and SLJ24895-9A) (Supplemental Figure 1B) were selected for a field experiment in 2017, and SLJ24895-5C was further tested in the field in 2018 (Figure 1A ). These field trials indicate that Rpi-amr3 confers protection against potato late blight in field conditions, while the wild-type Maris Piper control lines were infected completely within \\u223c3 weeks once disease symptoms appeared (Figures 1A and 1B). As a result, the tuber yield of the Rpi-amr3 transgenic lines was significantly higher than the wild-type Maris Piper lines (Figures 1C and 1D). To determine the P. infestans genotypes present in the field trial, we sampled the infected leaves and genotyped them by SSR markers; most of the isolates corresponded to a dominant UK strain 6_A1 (aka Pink6) (Supplemental Table 1).To further evaluate the resistance spectrum of Rpi-amr3, we performed detached leaf assay (DLA) on SLJ24895-5C, with wild-type Maris Piper and Rpi-amr1 transgenic Maris Piper as controls. Seventeen P. infestans isolates with different origins and races were tested (Figure 1E, Supplemental Table 2). Our results show that Rpi-amr3 confers resistance against all tested isolates in potato in laboratory conditions but with different efficacy, and Rpi-amr3-mediated resistance is weaker than Rpi-amr1 in potato (Figure 1E). We also generated Rpi-amr3 stably transformed N. benthamiana lines. Two homozygous T2 lines, 13.3 and 16.5, were tested with nine P. infestans isolates. Both Rpi-amr3 transgenic N. benthamiana lines confer complete resistance to all tested P. infestans isolates (Supplemental Table 2).These data show that Rpi-amr3 confers potato late blight resistance in both the laboratory and field conditions.Avramr3 encodes a conserved RXLR-WY effector proteinTo identify the effector recognized by Rpi-amr3, we screened an RXLR effector library (Rietman, 2011Rietman H. Putting the Phytophthora Infestans Genome Sequence at Work: Multiple Novel Avirulence and Potato Resistance Gene Candidates Revealed.2011Google Scholar; Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar) of 311 RXLR effectors by Agrobacterium tumefaciens-mediated co-expression with Rpi-amr3 in N. benthamiana. Most of these effectors (296/311) do not induce a HR when expressed alone or co-expressed with Rpi-amr3; 14 effectors are auto-active in N. benthamiana, and we found PITG_21190 specifically induces an HR with Rpi-amr3 (Figure 2A, Supplemental Table 3); therefore, we concluded that PITG_21190 is Avramr3. Avramr3 encodes a 339-amino acid (aa) protein with a signal peptide followed by RXLR, EER motifs, and four predicted WY motifs (Win et al., 2012Win J. Krasileva K.V. Kamoun S. Shirasu K. Staskawicz B.J. Banfield M.J. Sequence divergent RXLR effectors share a structural fold conserved across plant pathogenic oomycete Species.PLoS Pathog. 2012; 8: e1002400Crossref PubMed Scopus (106) Google Scholar) (Figure 2C). To characterize the expression profile of Avramr3, eight P. infestans isolates (T30-4, 88069, NL01096, 06_3928A, 6_A1, EC1, US23, and 99183) were used to inoculate a susceptible potato cultivar Maris Piper; RNA was isolated 2 days after the infection for RT\\u2013PCR. Our data show that Avramr3 are expressed in all eight isolates at 2 days after infection (Supplemental Figure 1A).Figure 2Identification and characterization of AVRamr3.Show full caption(A) Co-expression of Rpi-amr3:HA and AVRamr3:HIS-FLAG triggers cell death on N. benthamiana; Rpi-amr1 and AVRamr1 were used as controls. The photos were taken 3 days after infiltration; Agrobacterium strain GV3101(pMP90) carrying Rpi-amr3:HA or AVRamr3:HIS-FLAG constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results.(B) Co-expression of Rpi-amr3:HA and AVRamr3 truncations. All truncations are tagged with a C-terminal HIS-FLAG tag. T3, T4, T8, and T10 trigger cell death when co-expressed with Rpi-amr3, but not T1, T2, T5, T6, T7, and T9. Full-length AVRamr3:HIS-FLAG was used as control. OD600 = 0.5. Three biological replicates were performed with the same results.(C) Cartoon of AVRamr3 (PITG_21190), a protein with 339 aa with a signal peptide (lemon), RXLR-EER motif (green), and an effector domain (red) with four predicted WY motifs (details are shown in Supplemental Figure 4). T1\\u2013T10 indicates the AVRamr3 truncations used in HR assays. Those that induce HR after co-expression with Rpi-amr3 are marked by orange bars, and otherwise by blue.(D) Rpi-amr3::HA and AVRamr3::HIS-FLAG constructs were used for a bidirectional co-immunoprecipitation experiment, with Rpi-amr1-HA and AVRamr1::HIS-FLAG used as control. After HA pull-down of Rpi-amr3::HA or Rpi-amr1::HA, only AVRamr3::HIS-FLAG is associated with Rpi-amr3::HA. After Flag pull-down of AVRamr3::HIS-FLAG or AVRamr1-HIS-FLAG, only Rpi-amr3::HA is associated with AVRamr3::HIS-FLAG. Agrobacterium strain GV3101(pMP90) carrying different constructs was used for transient expression in the nrc2/3/4 knockout N. benthamiana line (210.4.3) to abolish the cell death phenotype. OD600 = 0.5. Three biological replicates were performed with the same results.(E) Rpi-amr3::Cluc and AVRamr3::Nluc constructs were used to test their interaction in planta; Rpi-amr1::Cluc and AVRamr1::Nluc were used as controls. The luciferase signal can be detected only on Rpi-amr3::Cluc and AVRamr3::Nluc co-expression. The nrc2/3/4 knockout N. benthamiana line (210.4.3) was used to abolish the cell death phenotype.View Large Image Figure ViewerDownload Hi-res image Download (PPT)Many RXLR effectors are encoded by fast-evolving, multiple-member family genes with extensive sequence polymorphism, such as the Avr2 and Avrblb2 families (Gilroy et al., 2011Gilroy E.M. Breen S. Whisson S.C. Squires J. Hein I. Kaczmarek M. Turnbull D. Boevink P.C. Lokossou A. Cano L.M. et al.Presence/absence, differential expression and sequence polymorphisms between PiAVR2 and PiAVR2-like in Phytophthora infestans determine virulence on R2 plants.New Phytol. 2011; 191: 763-776Crossref PubMed Scopus (103) Google Scholar; Oliva et al., 2015Oliva R.F. Cano L.M. Raffaele S. Win J. Bozkurt T.O. Belhaj K. Oh S.-K. THINES M. Kamoun S. A recent expansion of the RXLR effector gene Avrblb2 is maintained in global populations of Phytophthora infestans indicating different contributions to virulence.Mol. Plant Microbe Interact. 2015; 28: 901-912Crossref PubMed Scopus (25) Google Scholar). To study the sequence polymorphism of Avramr3, we identified 17 additional Avramr3 homologs from 11 isolates from published databases (KR_1, 3928A, EC1, 6_A1, and US23) (Lee et al., 2020Lee Y. Cho K.-S. Seo J.-H. Sohn K.H. Prokchorchik M. Improved genome sequence and gene annotation resource for the potato late blight pathogen Phytophthora infestans.Mol. Plant Microbe Interact. 2020; 33: 1025-1028https://doi.org/10.1094/MPMI-02-20-0023-ACrossref PubMed Scopus (4) Google Scholar; Lin et al., 2020Lin X. Song T. Fairhead S. Witek K. Jouet A. Jupe F. Witek A.I. Karki H.S. Vleeshouwers V.G.A.A. Hein I. et al.Identification of Avramr1 from Phytophthora infestans using long read and cDNA pathogen-enrichment sequencing (PenSeq).Mol. Plant Pathol. 2020; 21: 1502-1512Crossref PubMed Scopus (10) Google Scholar) or cloned by PCR (EC1, NL01096, NL14538, 88069, PIC99183, and PIC99177) (Supplemental Figure 2). The sequence alignment shows Avramr3 is a highly conserved RXLR effector among P. infestans isolates, with only two polymorphic amino acids found among the 18 AVRamr3 homologs (Supplemental Figure 2).To define the domain responsible for recognition by Rpi-amr3, we fused 10 truncated Avramr3 fragments with HIS-FLAG tags and cloned into an expression vector with 35S promoter (T1\\u2013T10; Figure 2C; Supplemental Figure 3) and transiently co-expressed with Rpi-amr3 in N. benthamiana. Six AVRamr3 truncations (T1, T2, T5, T6, T7, and T9) cannot be recognized by Rpi-amr3 (Figure 2B). The protein levels of HIS-FLAG-tagged T6 and T7 are lower than others (Supplemental Figure 3A); therefore, we generated GFP-tagged constructs. The expression of T6-GFP is comparable with AVRamr3-GFP, but T7-GFP is not stable (Supplemental Figure 3B and 3C). We found four AVRamr3 truncations (T3, T4, T8, and T10) can be recognized by Rpi-amr3. T10 (111\\u2013240 aa), which carries the second and third WY motifs, is the minimal region to be recognized by Rpi-amr3, but not the adjacent T9 protein (130\\u2013258 aa) (Figure 2B). This suggests these 130 aa of AVRamr3 T10 are sufficient for recognition by Rpi-amr3 and initiation of HR.Rpi-amr3 is dependent on the helper NLRs NRC2, NRC3, and NRC4In Solanaceae, the functionality of many CC-NLR proteins requires helper NLR proteins of the NRC class (Wu et al., 2017Wu C.-H. Abd-El-Haliem A. Bozkurt T.O. Belhaj K. Terauchi R. Vossen J.H. Kamoun S. NLR network mediates immunity to diverse plant pathogens.Proc. Natl. Acad. Sci. USA. 2017; 114: 8113-8118Crossref PubMed Scopus (169) Google Scholar). To test whether Rpi-amr3 is NRC dependent, we co-expressed Rpi-amr3 and Avramr3 in NRC knockout N. benthamiana lines (nrc2/3_1.3.1, nrc4_185.9.1.3, nrc2/3/4_210.4.3) (Adachi et al., 2019Adachi H. Contreras M.P. Harant A. Wu C.-H. Derevnina L. Sakai T. Duggan C. Moratto E. Bozkurt T.O. Maqbool A. et al.An N-terminal motif in NLR immune receptors is functionally conserved across distantly related plant species.Elife. 2019; 8: 121Crossref Scopus (69) Google Scholar; Wu et al., 2020Wu C.-H. Adachi H. De la Concepcion J.C. Castells-Graells R. Nekrasov V. Kamoun S. Kamoun S. NRC4 Gene cluster Is not essential for bacterial flagellin-triggered immunity.Plant Physiol. 2020; 182: 455-459Crossref PubMed Scopus (4) Google Scholar; Witek et al., 2021Witek K. Lin X. Karki H.S. Jupe F. Witek A.I. Steuernagel B. Stam R. van Oosterhout C. Fairhead S. Heal R. et al.A complex resistance locus in Solanum americanum recognizes a conserved Phytophthora effector.Nat. Plants. 2021; 7: 198-208Crossref PubMed Scopus (14) Google Scholar), as with wild-type N. benthamiana. We found HR on the nrc2/3_1.3.1 and nrc4_185.9.1.3 knockout lines, but not the nrc2/3/4_210.4.3 knockout lines. Similarly, only nrc2/3/4_210.4.3 knockout lines show susceptibility to P. infestans after Rpi-amr3 transient expression (Supplemental Figure 4). These data suggest both Rpi-amr3-mediated effector recognition and resistance are supported by either NRC2, NRC3, or NRC4.Rpi-amr3 associates with AVRamr3 in plantaTo date, most Rpi proteins recognize their cognate effectors in an indirect manner, apart from the RB and IPI-O effectors (Chen et al., 2012Chen Y. Liu Z. Halterman D.A. Molecular determinants of resistance activation and suppression by Phytophthora infestans effector IPI-O.PLoS Pathog. 2012; 8: e1002595Crossref PubMed Scopus (71) Google Scholar; Zhao and Song, 2021Zhao J. Song J. NLR immune receptor RB is differentially targeted by two homologous but functionally distinct effector proteins.Plant Commun. 2021; 2: 100236Abstract Full Text Full Text PDF PubMed Scopus (2) Google Scholar). To test the interaction between Rpi-amr3 and AVRamr3, we generated and transiently co-expressed Rpi-amr3:HA and AVRamr3:HIS-FLAG epitope-tagged constructs in nrc2/3/4 knockout N. benthamiana leaves to avoid cell death. Protein was then extracted, and bi-directional co-immunoprecipitations (Co-IPs) were performed. These Co-IPs indicate that Rpi-amr3 associates with AVRamr3 bidirectionally (Figure 2D). We also tested their interaction using a split-luciferase assay. Rpi-amr3:C-luciferase (Cluc) and AVRamr3:N-luciferase (Nluc) constructs were generated and transiently expressed in the nrc2/3/4 knockout N. benthamiana. Luciferase signal was detected only when Rpi-amr3:Cluc and AVRamr3:Nluc were co-expressed (Figure 2E), but not in the negative controls. These data suggest Rpi-amr3 associates with AVRamr3 in planta, but do not exclude the possible involvement of additional proteins.Avramr3 orthologs occur in multiple Phytophthora speciesTo study the evolution of Avramr3 in Phytophthora species, we searched for Avramr3 homologs from published Phytophthora and Hyaloperonospora arabidopsidis genomes. Surprisingly, we found Avramr3 homologs in many Phytophthora genomes, including P. parasitica, P. cactorum, P. palmivora, P. pluvialis, P. megakarya, P. litchii, P. ramorum, P. lateralis, P. sojae, P. capsici, and P. cinnamomi and in H. arabidopsidis. Most of the Avramr3 homologs are located at a syntenic locus (Figure 3A ). Notably, the P. infestans Avramr3-containing contig was not fully assembled; it lacks sequences on the 5\\u2032 side of Avramr3 (Figure 3A).Figure 3AVRamr3 is a conserved effector among different Phytophthora species.Show full caption(A) The synteny map of Avramr3 loci from 12 different Phytophthora genomes. The Avramr3 loci were extracted from different genomes, annotated by the gene prediction tool in EumicrobeDB, and then analyzed and visualized by Clinker. Avramr3 homologs are shown by purple triangles and indicated by a black arrow; the flanking genes with homology are represented by the corresponding colors. The Phytophthora clades are adapted from the Phytophthora database (Rahman et al., 2014Rahman M.Z. Uematsu S. Takeuchi T. Shirai K. Ishiguro Y. Suga H. Kageyama K. Two new species, Phytophthora nagaii sp. nov. and P. fragariaefolia sp. nov., causing serious diseases on rose and strawberry plants, respectively, in Japan.J. Gen. Plant Pathol. 2014; 80: 348-365Crossref Scopus (13) Google Scholar).(B) Expression of AVRamr3 homologs with HIS-FLAG tag alone does not trigger cell death on Nicotiana benthamiana. Agrobacterium strain GV3101(pMP90) carrying different constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results.(C) Co-expression of HIS-FLAG-tagged AVRamr3 homologs with Rpi-amr3::GFP in N. benthamiana. The AVRamr3 homologs from P. infestans (Pi), P. parasitica (Pp), P. cactorum (Pc), P. palmivora (Ppal), P. megakarya (Pmeg), P. litchii (Plit), P. sojae (Ps), P. lateralis (Plat), and P. pluvialis (Pplu) induce cell death after co-expression with Rpi-amr3::GFP, but not AVRamr3 homologs from P. ramorum (Pr), P. capsici (Pcap), and H. arabidopsidis (Hpa). The AVRamr3 homolog from P. cinnamomi (Pcin) shows an intermediate cell death. Agrobacterium strain GV3101(pMP90) carrying different constructs was used in this experiment. OD600 = 0.5. Three biological replicates were performed with the same results. The protein expression of the AVRamr3 homologs with HIS-FLAG tag was shown in Supplemental Figure 6.View Large Image Figure ViewerDownload Hi-res image Download (PPT)To test whether those AVRamr3 homologs from different Phytophthora species are also recognized by Rpi-amr3, we synthesized and cloned them into an expression vector with the 35S promoter and performed transient expression assays in N. benthamiana. Expressing the effectors alone does not trigger HR in N. benthamiana (Figure 3B), but AVRamr3 homologs from P. parasitica, P. cactorum, P. palmivora, P. megakarya, P. litchii, P. sojae, P. lateralis, and P. pluvialis can induce HR when co-expressed with Rpi-amr3, respectively. The AVRamr3 homolog from P. cinnamomi triggers a weaker HR compared with other recognized AVRamr3 homologs, and the AVRamr3 homologs from P. ramorum, P. capsici, and H. arabidopsidis (Figure 3C) do not trigger Rpi-amr3-dependent HR. All these AVRamr3 homologs carry multiple WY motifs, but many polymorphic amino acids are present among these homologs. We\", \"doi_y\": \"https://doi.org/10.1016/j.molp.2022.07.012\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4293824663\", \"doi_x\": \"10.1242/jcs.260456\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Meeting report \\u2013 Cell dynamics: host\\u2013pathogen interface\", \"abstract\": \"ABSTRACT Two years into the most significant infectious disease event of our generation, infections have populated every conversation and in-depth understanding of host\\u2013pathogen interactions has, perhaps, never been more important. In a successful return to in-person conferences, the host\\u2013pathogen interface was the focus of the third Cell Dynamics meeting, which took place at the glorious Wotton House in Surrey, UK. The meeting organised by Michaela Gack, Maximiliano Gutierrez, Dominique Soldati-Favre and Michael Way gathered an international group of scientists who shared their recent discoveries and views on numerous aspects, including cell-autonomous defence mechanisms, pathogen interactions with host cytoskeletal or membrane dynamics, and cellular immune regulation. More than 30 years into the beginning of cellular microbiology as a field, the meeting exhibited the unique aspect of the host\\u2013pathogen interface in uncovering the fundamentals of both pathogens and their hosts.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260456\", \"publication_date\": \"2022-08-15\"}, {\"work_id\": \"W4290730357\", \"doi_x\": \"10.1080/19336950.2022.2098661\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"abstract\": \"Mechanosensitive ion channels are integral membrane proteins ubiquitously present in bacteria, archaea, and eukarya. They act as molecular sensors of mechanical stress to serve vital functions such as touch, hearing, osmotic pressure, proprioception and balance, while their malfunction is often associated with pathologies. Amongst them, the structurally distinct MscL and MscS channels from bacteria are the most extensively studied. MscS-like channels have been found in plants and Schizosaccharomyces pombe, where they regulate intracellular Ca2+ and cell volume under hypo-osmotic conditions. Here we characterize two MscS-like putative channels, named MscA and MscB, from the model filamentous fungus Aspergillus nidulans. Orthologues of MscA and MscB are present in most fungi, including relative plant and animal pathogens. MscA/MscB and other fungal MscS-like proteins share the three transmembrane helices and the extended C-terminal cytosolic domain that form the structural fingerprint of MscS-like channels with at least three additional transmembrane segments than Escherichia coli MscS. We show that MscA and MscB localize in Endoplasmic Reticulum and the Plasma Membrane, respectively, whereas their overexpression leads to increased CaCl2 toxicity or/and reduction of asexual spore formation. Our findings contribute to understanding the role of MscS-like channels in filamentous fungi and relative pathogens.\", \"doi_y\": \"https://doi.org/10.1080/19336950.2022.2098661\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4323294987\", \"doi_x\": \"10.1080/10409238.2023.2181309\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"abstract\": \"Understanding how Nature accomplishes the reduction of inert nitrogen gas to form metabolically tractable ammonia at ambient temperature and pressure has challenged scientists for more than a century. Such an understanding is a key aspect toward accomplishing the transfer of the genetic determinants of biological nitrogen fixation to crop plants as well as for the development of improved synthetic catalysts based on the biological mechanism. Over the past 30 years, the free-living nitrogen-fixing bacterium Azotobacter vinelandii emerged as a preferred model organism for mechanistic, structural, genetic, and physiological studies aimed at understanding biological nitrogen fixation. This review provides a contemporary overview of these studies and places them within the context of their historical development.\", \"doi_y\": \"https://doi.org/10.1080/10409238.2023.2181309\", \"publication_date\": \"2022-11-02\"}, {\"work_id\": \"W4291010252\", \"doi_x\": \"10.1101/2022.08.11.503608\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The<i>Solanum americanum</i>pangenome and effectoromics reveal new resistance genes against potato late blight\", \"abstract\": \"Abstract Late blight caused by the oomycete pathogen Phytophthora infestans continues to cause major worldwide losses in potato and tomato. Most accessions of Solanum americanum , a globally distributed, wild Solanaceae plant, are highly resistant to late blight. We generated high-quality reference genomes of four S. americanum accessions, re-sequenced 52 accessions, and we defined variation in the NLR immune receptor genes (the S. americanum NLRome). We further screened for variation in recognition of \\u223c315 P. infestans RXLR effectors in 52 S. americanum accessions. Using these genotypic and phenotypic data, we cloned three novel NLR-encoding genes Rpi-amr4, Rpi-amr16 and Rpi-amr17 , and determined their corresponding RXLR effector genes Avramr4 ( PITG_22825 ), Avramr16 ( PITG_02860 ) and Avramr17 ( PITG_04373 ) from P. infestans . These genomic resources and methodology will support efforts to convert potato into a \\u201cnonhost\\u201d of late blight and can be applied to diseases of other crops.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.11.503608\", \"publication_date\": \"2022-08-11\"}, {\"work_id\": \"W4311134609\", \"doi_x\": \"10.1016/j.immuni.2022.12.005\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Immunoglobulin germline gene polymorphisms influence the function of SARS-CoV-2 neutralizing antibodies\", \"abstract\": \"The human immunoglobulin heavy-chain (IGH) locus is exceptionally polymorphic, with high levels of allelic and structural variation. Thus, germline IGH genotypes are personal, which may influence responses to infection and vaccination. For an improved understanding of inter-individual differences in antibody responses, we isolated SARS-CoV-2 spike-specific monoclonal antibodies from convalescent health care workers, focusing on the IGHV1-69 gene, which has the highest level of allelic variation of all IGHV genes. The IGHV1-69\\u221720-using CAB-I47 antibody and two similar antibodies isolated from an independent donor were critically dependent on allele usage. Neutralization was retained when reverting the V region to the germline IGHV1-69\\u221720 allele but lost when reverting to other IGHV1-69 alleles. Structural data confirmed that two germline-encoded polymorphisms, R50 and F55, in the IGHV1-69 gene were required for high-affinity receptor-binding domain interaction. These results demonstrate that polymorphisms in IGH genes can influence the function of SARS-CoV-2 neutralizing antibodies.\", \"doi_y\": \"https://doi.org/10.1016/j.immuni.2022.12.005\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4386271358\", \"doi_x\": \"10.1111/mmi.15144\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A bioinformatic workflow for in silico secretome prediction with the lignocellulose degrading ascomycete fungus <i>Parascedosporium putredinis</i> NO1\", \"abstract\": \"The increasing availability of microbial genome sequences provides a reservoir of information for the identification of new microbial enzymes. Genes encoding proteins engaged in extracellular processes are of particular interest as these mediate the interactions microbes have with their environments. However, proteomic analysis of secretomes is challenging and often captures intracellular proteins released through cell death and lysis. Secretome prediction workflows from sequence data are commonly used to filter proteins identified through proteomics but are often simplified to a single step and are not evaluated bioinformatically for their effectiveness. Here, a workflow to predict a fungal secretome was designed and applied to the coding regions of the Parascedosporium putredinis NO1 genome. This ascomycete fungus is an exceptional lignocellulose degrader from which a new lignin-degrading enzyme has previously been identified. The 'secretome isolation' workflow is based on two strategies of localisation prediction and secretion prediction each utilising multiple available tools. The workflow produced three final secretomes with increasing levels of stringency. All three secretomes showed increases in functional annotations for extracellular processes and reductions in annotations for intracellular processes. Multiple sequences isolated as part of the secretome lacked any functional annotation and made exciting candidates for novel enzyme discovery.\", \"doi_y\": \"https://doi.org/10.1111/mmi.15144\", \"publication_date\": \"2023-08-30\"}, {\"work_id\": \"W4315433413\", \"doi_x\": \"10.1021/acs.jproteome.2c00637\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Proteomics Standards Initiative at Twenty Years: Current Activities and Future Work\", \"abstract\": \"The Human Proteome Organization (HUPO) Proteomics Standards Initiative (PSI) has been successfully developing guidelines, data formats, and controlled vocabularies (CVs) for the proteomics community and other fields supported by mass spectrometry since its inception 20 years ago. Here we describe the general operation of the PSI, including its leadership, working groups, yearly workshops, and the document process by which proposals are thoroughly and publicly reviewed in order to be ratified as PSI standards. We briefly describe the current state of the many existing PSI standards, some of which remain the same as when originally developed, some of which have undergone subsequent revisions, and some of which have become obsolete. Then the set of proposals currently being developed are described, with an open call to the community for participation in the forging of the next generation of standards. Finally, we describe some synergies and collaborations with other organizations and look to the future in how the PSI will continue to promote the open sharing of data and thus accelerate the progress of the field of proteomics.\", \"doi_y\": \"https://doi.org/10.1021/acs.jproteome.2c00637\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W4296078744\", \"doi_x\": \"10.1126/sciadv.abp9660\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"abstract\": \"Despite continuing progress in kinesin enzyme mechanochemistry and emerging understanding of the cargo recognition machinery, it is not known how these functions are coupled and controlled by the \\u03b1-helical coiled coils encoded by a large component of kinesin protein sequences. Here, we combine computational structure prediction with single-particle negative-stain electron microscopy to reveal the coiled-coil architecture of heterotetrameric kinesin-1 in its compact state. An unusual flexion in the scaffold enables folding of the complex, bringing the kinesin heavy chain-light chain interface into close apposition with a tetrameric assembly formed from the region of the molecule previously assumed to be the folding hinge. This framework for autoinhibition is required to uncover how engagement of cargo and other regulatory factors drives kinesin-1 activation.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.abp9660\", \"publication_date\": \"2022-09-16\"}, {\"work_id\": \"W4315786113\", \"doi_x\": \"10.3390/biom13010157\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pollen Coat Proteomes of Arabidopsis thaliana, Arabidopsis lyrata, and Brassica oleracea Reveal Remarkable Diversity of Small Cysteine-Rich Proteins at the Pollen-Stigma Interface\", \"abstract\": \"The pollen coat is the outermost domain of the pollen grain and is largely derived from the anther tapetum, which is a secretory tissue that degenerates late in pollen development. By being localised at the interface of the pollen-stigma interaction, the pollen coat plays a central role in mediating early pollination events, including molecular recognition. Amongst species of the Brassicaceae, a growing body of data has revealed that the pollen coat carries a range of proteins, with a number of small cysteine-rich proteins (CRPs) being identified as important regulators of the pollen-stigma interaction. By utilising a state-of-the-art liquid chromatography/tandem mass spectrometry (LC-MS/MS) approach, rich pollen coat proteomic profiles were obtained for Arabidopsis thaliana, Arabidopsis lyrata, and Brassica oleracea, which greatly extended previous datasets. All three proteomes revealed a strikingly large number of small CRPs that were not previously reported as pollen coat components. The profiling also uncovered a wide range of other protein families, many of which were enriched in the pollen coat proteomes and had functions associated with signal transduction, cell walls, lipid metabolism and defence. These proteomes provide an excellent source of molecular targets for future investigations into the pollen-stigma interaction and its potential evolutionary links to plant-pathogen interactions.\", \"doi_y\": \"https://doi.org/10.3390/biom13010157\", \"publication_date\": \"2023-01-12\"}, {\"work_id\": \"W4308448193\", \"doi_x\": \"10.1126/science.abq6656\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"On-demand cell-autonomous gene therapy for brain circuit disorders\", \"abstract\": \"Several neurodevelopmental and neuropsychiatric disorders are characterized by intermittent episodes of pathological activity. Although genetic therapies offer the ability to modulate neuronal excitability, a limiting factor is that they do not discriminate between neurons involved in circuit pathologies and \\\"healthy\\\" surrounding or intermingled neurons. We describe a gene therapy strategy that down-regulates the excitability of overactive neurons in closed loop, which we tested in models of epilepsy. We used an immediate early gene promoter to drive the expression of Kv1.1 potassium channels specifically in hyperactive neurons, and only for as long as they exhibit abnormal activity. Neuronal excitability was reduced by seizure-related activity, leading to a persistent antiepileptic effect without interfering with normal behaviors. Activity-dependent gene therapy is a promising on-demand cell-autonomous treatment for brain circuit disorders.\", \"doi_y\": \"https://doi.org/10.1126/science.abq6656\", \"publication_date\": \"2022-11-04\"}, {\"work_id\": \"W4377093432\", \"doi_x\": \"10.1107/s2059798323003613\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cysteine synthase: multiple structures of a key enzyme in cysteine synthesis and a potential drug target for Chagas disease and leishmaniasis\", \"abstract\": \"Chagas disease is a neglected tropical disease (NTD) caused by Trypanosoma cruzi , whilst leishmaniasis, which is caused by over 20 species of Leishmania , represents a group of NTDs endemic to most countries in the tropical and subtropical belt of the planet. These diseases remain a significant health problem both in endemic countries and globally. These parasites and other trypanosomatids, including T. theileri , a bovine pathogen, rely on cysteine biosynthesis for the production of trypanothione, which is essential for parasite survival in hosts. The de novo pathway of cysteine biosynthesis requires the conversion of O -acetyl-L-serine into L-cysteine, which is catalysed by cysteine synthase (CS). These enzymes present potential for drug development against T. cruzi , Leishmania spp. and T. theileri . To enable these possibilities, biochemical and crystallographic studies of CS from T. cruzi ( Tc CS), L. infantum ( Li CS) and T. theileri ( Tth CS) were conducted. Crystal structures of the three enzymes were determined at resolutions of 1.80 \\u00c5 for Tc CS, 1.75 \\u00c5 for Li CS and 2.75 \\u00c5 for Tth CS. These three homodimeric structures show the same overall fold and demonstrate that the active-site geometry is conserved, supporting a common reaction mechanism. Detailed structural analysis revealed reaction intermediates of the de novo pathway ranging from an apo structure of Li CS and holo structures of both Tc CS and Tth CS to the substrate-bound structure of Tc CS. These structures will allow exploration of the active site for the design of novel inhibitors. Additionally, unexpected binding sites discovered at the dimer interface represent new potential for the development of protein\\u2013protein inhibitors.\", \"doi_y\": \"https://doi.org/10.1107/s2059798323003613\", \"publication_date\": \"2023-05-19\"}, {\"work_id\": \"W4384435971\", \"doi_x\": \"10.1016/j.sbi.2023.102652\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Mechanism of primer synthesis by Primase-Polymerases\", \"abstract\": \"Members of the primase-polymerase (Prim-Pol) superfamily are found in all domains of life and play diverse roles in genome stability, including primer synthesis during DNA replication, lesion repair and damage tolerance. This review focuses primarily on Prim-Pol members capable of de novo primer synthesis that have experimentally derived structural models available. We discuss the mechanism of DNA primer synthesis initiation by Prim-Pol catalytic domains, based on recent structural and functional studies. We also describe a general model for primer initiation that also includes the ancillary domains/subunits, which stimulate the initiation of primer synthesis.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102652\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4297489083\", \"doi_x\": \"10.1083/jcb.202112068\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Seipin concentrates distinct neutral lipids via interactions with their acyl chain carboxyl esters\", \"abstract\": \"Lipid droplets (LDs) are essential for cellular lipid homeostasis by storing diverse neutral lipids (NLs), such as triacylglycerol (TAG), steryl esters (SE), and retinyl esters (RE). A proper assembly of TAG-containing LDs at the ER requires Seipin, a conserved protein often mutated in lipodystrophies. Here, we show that the yeast Seipin Sei1 and its partner Ldb16 also promote the storage of other NL in LDs. Importantly, this role of Sei1/Ldb16 is evolutionarily conserved as expression of human-Seipin restored normal SE-containing LDs in yeast Seipin mutants. As in the case of TAG, the formation of SE-containing LDs requires interactions between hydroxyl-residues in human Seipin or yeast Ldb16 with NL carboxyl esters. These findings provide a universal mechanism for Seipin-mediated LD formation and suggest a model for how Seipin distinguishes NLs from aliphatic phospholipid acyl chains in the center of the membrane bilayer.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202112068\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4286780066\", \"doi_x\": \"10.1101/2022.07.22.500997\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mining the human gut microbiome identifies mycobacterial d-arabinan degrading enzymes\", \"abstract\": \"Abstract Division and degradation of bacterial cell walls requires coordinated action of a myriad of enzymes. This particularly applies to the elaborate cell walls of acid-fast organisms such as Mycobacterium tuberculosis , which consist of a multi-layered cell wall that contains an unusual glycan called arabinogalactan. Enzymes that cleave the d -arabinan core of this structure have not previously been identified in any organism. We have interrogated the diverse carbohydrate degrading enzymes expressed by the human gut microbiota and uncovered four families of glycoside hydrolases with the capability to degrade the d -arabinan or d -galactan components of arabinogalactan. Using novel exo- d -galactofuranosidases from gut bacteria we generated enriched d -arabinan and used it to identify D. gadei as a D-arabinan degrader. This enabled the discovery of endo- and exo-acting enzymes that cleave D-arabinan. We have identified new members of the DUF2961 family (GH172), and a novel family of glycoside hydrolases (DUF4185) that display endo- d -arabinofuranase activity. The DUF4185 enzymes are conserved in mycobacteria and found in many microbes, suggesting that the ability to degrade mycobacterial glycans plays an important role in the biology of diverse organisms. All mycobacteria encode two conserved endo- d -arabinanases that display different preferences for the d -arabinan-containing cell wall components arabinogalactan and lipoarabinomannan, suggesting they are important for cell wall modification and/or degradation. The discovery of these enzymes will support future studies into the structure and function of the mycobacterial cell wall.\", \"doi_y\": \"https://doi.org/10.1101/2022.07.22.500997\", \"publication_date\": \"2022-07-23\"}, {\"work_id\": \"W4322764495\", \"doi_x\": \"10.1021/jacsau.2c00569\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"abstract\": \"Glycosaminoglycans (GAGs) are complex polysaccharides exhibiting a vast structural diversity and fulfilling various functions mediated by thousands of interactions in the extracellular matrix, at the cell surface, and within the cells where they have been detected in the nucleus. It is known that the chemical groups attached to GAGs and GAG conformations comprise \\\"glycocodes\\\" that are not yet fully deciphered. The molecular context also matters for GAG structures and functions, and the influence of the structure and functions of the proteoglycan core proteins on sulfated GAGs and vice versa warrants further investigation. The lack of dedicated bioinformatic tools for mining GAG data sets contributes to a partial characterization of the structural and functional landscape and interactions of GAGs. These pending issues will benefit from the development of new approaches reviewed here, namely (i) the synthesis of GAG oligosaccharides to build large and diverse GAG libraries, (ii) GAG analysis and sequencing by mass spectrometry (e.g., ion mobility-mass spectrometry), gas-phase infrared spectroscopy, recognition tunnelling nanopores, and molecular modeling to identify bioactive GAG sequences, biophysical methods to investigate binding interfaces, and to expand our knowledge and understanding of glycocodes governing GAG molecular recognition, and (iii) artificial intelligence for in-depth investigation of GAGomic data sets and their integration with proteomics.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.2c00569\", \"publication_date\": \"2023-03-02\"}, {\"work_id\": \"W4292240021\", \"doi_x\": \"10.1016/j.cell.2022.07.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Bacteriophages benefit from mobilizing pathogenicity islands encoding immune systems against competitors\", \"abstract\": \"Bacteria encode sophisticated anti-phage systems that are diverse and versatile and display high genetic mobility. How this variability and mobility occurs remains largely unknown. Here, we demonstrate that a widespread family of pathogenicity islands, the phage-inducible chromosomal islands (PICIs), carry an impressive arsenal of defense mechanisms, which can be disseminated intra- and inter-generically by helper phages. These defense systems provide broad immunity, blocking not only phage reproduction, but also plasmid and non-cognate PICI transfer. Our results demonstrate that phages can mobilize PICI-encoded immunity systems to use them against other mobile genetic elements, which compete with the phages for the same bacterial hosts. Therefore, despite the cost, mobilization of PICIs may be beneficial for phages, PICIs, and bacteria in nature. Our results suggest that PICIs are important players controlling horizontal gene transfer and that PICIs and phages establish mutualistic interactions that drive bacterial ecology and evolution.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.07.014\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4283362019\", \"doi_x\": \"10.1021/acs.biomac.2c00516\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Controlled Bioactive Delivery Using Degradable Electroactive Polymers\", \"abstract\": \"Biomaterials capable of precisely controlling the delivery of agrochemicals/biologics/drugs/fragrances have significant markets in the agriscience/healthcare industries. Here, we report the development of degradable electroactive polymers and their application for the controlled delivery of a clinically relevant drug (the anti-inflammatory dexamethasone phosphate, DMP). Electroactive copolymers composed of blocks of polycaprolactone (PCL) and naturally occurring electroactive pyrrole oligomers (e.g., bilirubin, biliverdin, and hemin) were prepared and solution-processed to produce films (optionally doped with DMP). A combination of in silico/in vitro/in vivo studies demonstrated the cytocompatibility of the polymers. The release of DMP in response to the application of an electrical stimulus was observed to be enhanced by ca. 10-30% relative to the passive release from nonstimulated samples in vitro. Such stimuli-responsive biomaterials have the potential for integration devices capable of delivering a variety of molecules for technical/medical applications.\", \"doi_y\": \"https://doi.org/10.1021/acs.biomac.2c00516\", \"publication_date\": \"2022-06-24\"}, {\"work_id\": \"W4290991908\", \"doi_x\": \"10.3389/fmolb.2022.889667\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Towards Reverse Vaccinology for Bovine TB: High Throughput Expression of Full Length Recombinant Mycobacterium bovis Proteins\", \"abstract\": \"Bovine tuberculosis caused by Mycobacterium bovis, is a significant global pathogen causing economic loss in livestock and zoonotic TB in man. Several vaccine approaches are in development including reverse vaccinology which uses an unbiased approach to select open reading frames (ORF) of potential vaccine candidates, produce them as recombinant proteins and assesses their immunogenicity by direct immunization. To provide feasibility data for this approach we have cloned and expressed 123 ORFs from the M. bovis genome, using a mixture of E. coli and insect cell expression. We used a concatenated open reading frames design to reduce the number of clones required and single chain fusion proteins for protein pairs known to interact, such as the members of the PPE-PE family. Over 60% of clones showed soluble expression in one or the other host and most allowed rapid purification of the tagged bTB protein from the host cell background. The catalogue of recombinant proteins represents a resource that may be suitable for test immunisations in the development of an effective bTB vaccine.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.889667\", \"publication_date\": \"2022-08-11\"}, {\"work_id\": \"W4309496690\", \"doi_x\": \"10.26508/lsa.202201388\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The \\u03b2I domain promotes active \\u03b21 integrin clustering into mature adhesion sites\", \"abstract\": \"Modulation of integrin function is required in many physiological and pathological settings, such as angiogenesis and cancer. Integrin allosteric changes, clustering, and trafficking cooperate to regulate cell adhesion and motility on extracellular matrix proteins via mechanisms that are partly defined. By exploiting four monoclonal antibodies recognizing distinct conformational epitopes, we show that in endothelial cells (ECs), the extracellular \\u03b2I domain, but not the hybrid or I-EGF2 domain of active \\u03b21 integrins, promotes their FAK-regulated clustering into tensin 1-containing fibrillar adhesions and impairs their endocytosis. In this regard, the \\u03b2I domain-dependent clustering of active \\u03b21 integrins is necessary to favor fibronectin-elicited directional EC motility, which cannot be effectively promoted by \\u03b21 integrin conformational activation alone.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202201388\", \"publication_date\": \"2022-11-21\"}, {\"work_id\": \"W4384627282\", \"doi_x\": \"10.1099/mgen.0.001065\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A hybrid and poly-polish workflow for the complete and accurate assembly of phage genomes: a case study of ten przondoviruses\", \"abstract\": \"Bacteriophages (phages) within the genus Przondovirus are T7-like podoviruses belonging to the subfamily Studiervirinae, within the family Autographiviridae, and have a highly conserved genome organisation. The genomes of these phages range from 37 to 42 kb in size, encode 50-60 genes and are characterised by the presence of direct terminal repeats (DTRs) flanking the linear chromosome. These DTRs are often deleted during short-read-only and hybrid assemblies. Moreover, long-read-only assemblies are often littered with sequencing and/or assembly errors and require additional curation. Here, we present the isolation and characterisation of ten novel przondoviruses targeting Klebsiella spp. We describe HYPPA, a HYbrid and Poly-polish Phage Assembly workflow, which utilises long-read assemblies in combination with short-read sequencing to resolve phage DTRs and correcting errors, negating the need for laborious primer walking and Sanger sequencing validation. Our assembly workflow utilised Oxford Nanopore Technologies for long-read sequencing for its accessibility, making it the more relevant long-read sequencing technology at this time, and Illumina DNA Prep for short-read sequencing, representing the most commonly used technologies globally. Our data demonstrate the importance of careful curation of phage assemblies before publication, and prior to using them for comparative genomics.\", \"doi_y\": \"https://doi.org/10.1099/mgen.0.001065\", \"publication_date\": \"2023-07-18\"}, {\"work_id\": \"W4226191556\", \"doi_x\": \"10.1093/cvr/cvac065\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Inducible apelin receptor knockdown reduces differentiation efficiency and contractility of hESC-derived cardiomyocytes\", \"abstract\": \"The apelin receptor, a G protein-coupled receptor, has emerged as a key regulator of cardiovascular development, physiology, and disease. However, there is a lack of suitable human in vitro models to investigate the apelinergic system in cardiovascular cell types. For the first time we have used human embryonic stem cell-derived cardiomyocytes (hESC-CMs) and a novel inducible knockdown system to examine the role of the apelin receptor in both cardiomyocyte development and to determine the consequences of loss of apelin receptor function as a model of disease.Expression of the apelin receptor and its ligands in hESCs and hESC-CMs was determined. hESCs carrying a tetracycline-inducible short hairpin RNA targeting the apelin receptor were generated using the sOPTiKD system. Phenotypic assays characterized the consequences of either apelin receptor knockdown before hESC-CM differentiation (early knockdown) or in 3D engineered heart tissues as a disease model (late knockdown). hESC-CMs expressed the apelin signalling system at a similar level to the adult heart. Early apelin receptor knockdown decreased cardiomyocyte differentiation efficiency and prolonged voltage sensing, associated with asynchronous contraction. Late apelin receptor knockdown had detrimental consequences on 3D engineered heart tissue contractile properties, decreasing contractility and increasing stiffness.We have successfully knocked down the apelin receptor, using an inducible system, to demonstrate a key role in hESC-CM differentiation. Knockdown in 3D engineered heart tissues recapitulated the phenotype of apelin receptor down-regulation in a failing heart, providing a potential platform for modelling heart failure and testing novel therapeutic strategies.\", \"doi_y\": \"https://doi.org/10.1093/cvr/cvac065\", \"publication_date\": \"2022-05-16\"}, {\"work_id\": \"W4380577696\", \"doi_x\": \"10.1002/prot.26532\", \"af_direct_citation\": 0, \"af_indirect_citations\": 5, \"title\": \"Estimation of model accuracy in <scp>CASP15</scp> using the M<scp>odFOLDdock</scp> server\", \"abstract\": \"Abstract In CASP15, there was a greater emphasis on multimeric modeling than in previous experiments, with assembly structures nearly doubling in number (41 up from 22) since the previous round. CASP15 also included a new estimation of model accuracy (EMA) category in recognition of the importance of objective quality assessment (QA) for quaternary structure models. ModFOLDdock is a multimeric model QA server developed by the McGuffin group at the University of Reading, which brings together a range of single\\u2010model, clustering, and deep learning methods to form a consensus of approaches. For CASP15, three variants of ModFOLDdock were developed to optimize for the different facets of the quality estimation problem. The standard ModFOLDdock variant produced predicted scores optimized for positive linear correlations with the observed scores. The ModFOLDdockR variant produced predicted scores optimized for ranking, that is, the top\\u2010ranked models have the highest accuracy. In addition, the ModFOLDdockS variant used a quasi\\u2010single model approach to score each model on an individual basis. The scores from all three variants achieved strongly positive Pearson correlation coefficients with the CASP observed scores (oligo\\u2010lDDT) in excess of 0.70, which were maintained across both homomeric and heteromeric model populations. In addition, at least one of the ModFOLDdock variants was consistently ranked in the top two methods across all three EMA categories. Specifically, for overall global fold prediction accuracy, ModFOLDdock placed second and ModFOLDdockR placed third; for overall interface quality prediction accuracy, ModFOLDdockR, ModFOLDdock, and ModFOLDdockS were placed above all other predictor methods, and ModFOLDdockR and ModFOLDdockS were placed second and third respectively for individual residue confidence scores. The ModFOLDdock server is available at: https://www.reading.ac.uk/bioinf/ModFOLDdock/ . ModFOLDdock is also available as part of the MultiFOLD docker package: https://hub.docker.com/r/mcguffin/multifold .\", \"doi_y\": \"https://doi.org/10.1002/prot.26532\", \"publication_date\": \"2023-06-14\"}, {\"work_id\": \"W4294844275\", \"doi_x\": \"10.1371/journal.ppat.1010807\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Hypoxia inducible factors regulate infectious SARS-CoV-2, epithelial damage and respiratory symptoms in a hamster COVID-19 model\", \"abstract\": \"Understanding the host pathways that define susceptibility to Severe-acute-respiratory-syndrome-coronavirus-2 (SARS-CoV-2) infection and disease are essential for the design of new therapies. Oxygen levels in the microenvironment define the transcriptional landscape, however the influence of hypoxia on virus replication and disease in animal models is not well understood. In this study, we identify a role for the hypoxic inducible factor (HIF) signalling axis to inhibit SARS-CoV-2 infection, epithelial damage and respiratory symptoms in the Syrian hamster model. Pharmacological activation of HIF with the prolyl-hydroxylase inhibitor FG-4592 significantly reduced infectious virus in the upper and lower respiratory tract. Nasal and lung epithelia showed a reduction in SARS-CoV-2 RNA and nucleocapsid expression in treated animals. Transcriptomic and pathological analysis showed reduced epithelial damage and increased expression of ciliated cells. Our study provides new insights on the intrinsic antiviral properties of the HIF signalling pathway in SARS-CoV-2 replication that may be applicable to other respiratory pathogens and identifies new therapeutic opportunities.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1010807\", \"publication_date\": \"2022-09-06\"}, {\"work_id\": \"W4281945460\", \"doi_x\": \"10.1038/s41586-022-04456-z\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The road to fully programmable protein catalysis\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-04456-z\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4307812279\", \"doi_x\": \"10.1016/j.trac.2022.116808\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Advances in methods to analyse cardiolipin and their clinical applications\", \"abstract\": \"Cardiolipin (CL) is a mitochondria-exclusive phospholipid, primarily localised within the inner mitochondrial membrane, that plays an essential role in mitochondrial architecture and function. Aberrant CL content, structure, and localisation have all been linked to impaired mitochondrial activity and are observed in the pathophysiology of cancer and neurological, cardiovascular, and metabolic disorders. The detection, quantification, and localisation of CL species is a valuable tool to investigate mitochondrial dysfunction and the pathophysiological mechanisms underpinning several human disorders. CL is measured using liquid chromatography, usually combined with mass spectrometry, mass spectrometry imaging, shotgun lipidomics, ion mobility spectrometry, fluorometry, and radiolabelling. This review summarises available methods to analyse CL, with a particular focus on modern mass spectrometry, and evaluates their advantages and limitations. We provide guidance aimed at selecting the most appropriate technique, or combination of techniques, when analysing CL in different model systems, and highlight the clinical contexts in which measuring CL is relevant.\", \"doi_y\": \"https://doi.org/10.1016/j.trac.2022.116808\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4294074096\", \"doi_x\": \"10.1126/science.adc8969\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"abstract\": \"Cyclic adenosine diphosphate (ADP)-ribose (cADPR) isomers are signaling molecules produced by bacterial and plant Toll/interleukin-1 receptor (TIR) domains via nicotinamide adenine dinucleotide (oxidized form) (NAD+) hydrolysis. We show that v-cADPR (2'cADPR) and v2-cADPR (3'cADPR) isomers are cyclized by O-glycosidic bond formation between the ribose moieties in ADPR. Structures of 2'cADPR-producing TIR domains reveal conformational changes that lead to an active assembly that resembles those of Toll-like receptor adaptor TIR domains. Mutagenesis reveals a conserved tryptophan that is essential for cyclization. We show that 3'cADPR is an activator of ThsA effector proteins from the bacterial antiphage defense system termed Thoeris and a suppressor of plant immunity when produced by the effector HopAM1. Collectively, our results reveal the molecular basis of cADPR isomer production and establish 3'cADPR in bacteria as an antiviral and plant immunity-suppressing signaling molecule.\", \"doi_y\": \"https://doi.org/10.1126/science.adc8969\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4280560734\", \"doi_x\": \"10.1101/2022.04.25.489342\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Sensor NLR immune proteins activate oligomerization of their NRC helper\", \"abstract\": \"Abstract Nucleotide-binding domain and leucine-rich repeat (NLR) immune receptors are important components of plant and metazoan innate immunity that can function as individual units or as pairs or networks. Upon activation, NLRs form multiprotein complexes termed resistosomes or inflammasomes. Whereas metazoan paired NLRs, such as NAIP/NLRC4, activate into hetero-complexes, the molecular mechanisms underpinning activation of plant paired NLRs, especially whether they associate in resistosome hetero-complexes is unknown. In asterid plant species, the NLR required for cell death (NRC) immune receptor network is composed of multiple resistance protein sensors and downstream helpers that confer immunity against diverse plant pathogens. Here, we show that pathogen effector-activation of the NLR proteins Rx (confers virus resistance) and Bs2 (confers bacterial resistance) leads to oligomerization of the helper NLR NRC2. Activated Rx does not oligomerize or enter into a stable complex with the NRC2 oligomer and remains cytoplasmic. In contrast, activated NRC2 oligomers accumulate in membrane-associated puncta. We propose an activation-and-release model for NLRs in the NRC immune receptor network. This points to a distinct activation model compared to mammalian paired NLRs.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.25.489342\", \"publication_date\": \"2022-04-25\"}, {\"work_id\": \"W4285218681\", \"doi_x\": \"10.1039/d1sc06466e\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"<i>N</i>-Methyl deuterated rhodamines for protein labelling in sensitive fluorescence microscopy\", \"abstract\": \"Rhodamine fluorophores are setting benchmarks in fluorescence microscopy. Herein, we report the deuterium (d12) congeners of tetramethyl(silicon)rhodamine, obtained by isotopic labelling of the four methyl groups, show improved photophysical parameters (i.e. brightness, lifetimes) and reduced chemical bleaching. We explore this finding for SNAP- and Halo-tag labelling in live cells, and highlight enhanced properties in several applications, such as fluorescence activated cell sorting, fluorescence lifetime microscopy, stimulated emission depletion nanoscopy and single-molecule F\\u00f6rster-resonance energy transfer. We finally extend this idea to other dye families and envision deuteration as a generalizable concept to improve existing and to develop new chemical biology probes.\", \"doi_y\": \"https://doi.org/10.1039/d1sc06466e\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4301373369\", \"doi_x\": \"10.1101/2022.10.03.22280042\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Environmental and genetic predictors of human cardiovascular ageing\", \"abstract\": \"ABSTRACT Cardiovascular ageing is a process that begins early in life and leads to a progressive change in structure and decline in function due to accumulated damage across diverse cell types, tissues and organs contributing to multi-morbidity. Damaging biophysical, metabolic and immunological factors exceed endogenous repair mechanisms resulting in a pro-fibrotic state, cellular senescence and end-organ damage, however the genetic architecture of cardiovascular ageing is not known. Here we used machine learning approaches to quantify cardiovascular age from image-derived traits of vascular function, cardiac motion and myocardial fibrosis, as well as conduction traits from electrocardiograms, in 39,559 participants of UK Biobank. Cardiovascular ageing was found to be significantly associated with common or rare variants in genes regulating sarcomere homeostasis, myocardial immunomodulation, and tissue responses to biophysical stress. Ageing is accelerated by cardiometabolic risk factors and we also identified prescribed medications that were potential modifiers of ageing. Through large-scale modelling of ageing across multiple traits our results reveal insights into the mechanisms driving premature cardiovascular ageing and reveal potential molecular targets to attenuate age-related processes.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.03.22280042\", \"publication_date\": \"2022-10-04\"}, {\"work_id\": \"W4321765801\", \"doi_x\": \"10.1021/acs.chemrev.2c00600\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"abstract\": \"The investigation of macromolecular biomolecules with ion mobility mass spectrometry (IM-MS) techniques has provided substantial insights into the field of structural biology over the past two decades. An IM-MS workflow applied to a given target analyte provides mass, charge, and conformation, and all three of these can be used to discern structural information. While mass and charge are determined in mass spectrometry (MS), it is the addition of ion mobility that enables the separation of isomeric and isobaric ions and the direct elucidation of conformation, which has reaped huge benefits for structural biology. In this review, where we focus on the analysis of proteins and their complexes, we outline the typical features of an IM-MS experiment from the preparation of samples, the creation of ions, and their separation in different mobility and mass spectrometers. We describe the interpretation of ion mobility data in terms of protein conformation and how the data can be compared with data from other sources with the use of computational tools. The benefit of coupling mobility analysis to activation via collisions with gas or surfaces or photons photoactivation is detailed with reference to recent examples. And finally, we focus on insights afforded by IM-MS experiments when applied to the study of conformationally dynamic and intrinsically disordered proteins.\", \"doi_y\": \"https://doi.org/10.1021/acs.chemrev.2c00600\", \"publication_date\": \"2023-02-24\"}, {\"work_id\": \"W4225146277\", \"doi_x\": \"10.1016/j.tibtech.2022.03.012\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Quality by Design for enabling RNA platform production processes\", \"abstract\": \"RNA-based products have emerged as one of the most promising and strategic technologies for global vaccination, infectious disease control, and future therapy development. The assessment of critical quality attributes (CQAs), product-process interactions, relevant process analytical technologies, and process modeling capabilities can feed into a robust Quality by Design (QbD) framework for future development, design, and control of manufacturing processes. QbD implementation will help the RNA technology reach its full potential and will be central to the development, pre-qualification, and regulatory approval of rapid response, disease-agnostic RNA platform production processes.\", \"doi_y\": \"https://doi.org/10.1016/j.tibtech.2022.03.012\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4311092866\", \"doi_x\": \"10.1016/j.ceca.2022.102675\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"PI(3,5)P2 and NAADP: Team players or lone warriors? \\u2013 New insights into TPC activation modes\", \"abstract\": \"NAADP (nicotinic acid adenine dinucleotide phosphate) is a second messenger, releasing Ca2+ from acidic calcium stores such as endosomes and lysosomes. PI(3,5)P2 (phosphatidylinositol 3,5-bisphosphate) is a phospho-inositide, residing on endolysosomal membranes and likewise releasing Ca2+ from endosomes and lysosomes. Both compounds have been shown to activate endolysosomal two-pore channels (TPCs) in mammalian cells. However, their effects on ion permeability as demonstrated specifically for TPC2 differ. While PI(3,5)P2 elicits predominantly Na+-selective currents, NAADP increases the Ca2+ permeability of the channel. What happens when both compounds are applied simultaneously was unclear until recently.\", \"doi_y\": \"https://doi.org/10.1016/j.ceca.2022.102675\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W3207256498\", \"doi_x\": \"10.1002/cbic.202100484\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"abstract\": \"Abstract Terpenoids are a highly diverse group of natural products with considerable industrial interest. Increasingly, engineered microbes are used for the production of terpenoids to replace natural extracts and chemical synthesis. Terpene synthases (TSs) show a high level of functional plasticity and are responsible for the vast structural diversity observed in natural terpenoids. Their relatively inert active sites guide intrinsically reactive linear carbocation intermediates along one of many cyclisation paths via exertion of subtle steric and electrostatic control. Due to the absence of a strong protein interaction with these intermediates, there is a remarkable lack of sequence\\u2010function relationship within the TS family, making product\\u2010outcome predictions from sequences alone challenging. This, in combination with the fact that many TSs produce multiple products from a single substrate hampers the design and use of TSs in the biomanufacturing of terpenoids. This review highlights recent advances in genome mining, computational modelling, high\\u2010throughput screening, and machine\\u2010learning that will allow more predictive engineering of these fascinating enzymes in the near future.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202100484\", \"publication_date\": \"2021-11-03\"}, {\"work_id\": \"W4319293755\", \"doi_x\": \"10.1128/mbio.02667-22\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Insights into the Roles of Lipoteichoic Acids and MprF in Bacillus subtilis\", \"abstract\": \"Gram-positive bacterial cells are protected from the environment by a cell envelope that is comprised of a thick layer of peptidoglycan that maintains cell shape and teichoic acid polymers whose biological function remains unclear. In Bacillus subtilis, the loss of all class A penicillin-binding proteins (aPBPs), which function in peptidoglycan synthesis, is conditionally lethal. Here, we show that this lethality is associated with an alteration of lipoteichoic acids (LTAs) and the accumulation of the major autolysin LytE in the cell wall. Our analysis provides further evidence that the length and abundance of LTAs act to regulate the cellular level and activity of autolytic enzymes, specifically LytE. Importantly, we identify a novel function for the aminoacyl-phosphatidylglycerol synthase MprF in the modulation of LTA biosynthesis in both B. subtilis and Staphylococcus aureus. This finding has implications for our understanding of antimicrobial resistance (particularly to daptomycin) in clinically relevant bacteria and the involvement of MprF in the virulence of pathogens such as methicillin-resistant S. aureus (MRSA). IMPORTANCE In Gram-positive bacteria such as Bacillus subtilis and Staphylococcus aureus, the cell envelope is a structure that protects the cells from the environment but is also dynamic in that it must be modified in a controlled way to allow cell growth. In this study, we show that lipoteichoic acids (LTAs), which are anionic polymers attached to the membrane, have a direct role in modulating the cellular abundance of cell wall-degrading enzymes. We also find that the apparent length of the LTA is modulated by the activity of the enzyme MprF, previously implicated in modifications of the cell membrane leading to resistance to antimicrobial peptides. These findings are important contributions to our understanding of how bacteria balance cell wall synthesis and degradation to permit controlled growth and division. These results also have implications for the interpretation of antibiotic resistance, particularly for the clinical treatment of MRSA infections.\", \"doi_y\": \"https://doi.org/10.1128/mbio.02667-22\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4224311483\", \"doi_x\": \"10.26508/lsa.202101255\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The value of genotype-specific reference for transcriptome analyses in barley\", \"abstract\": \"It is increasingly apparent that although different genotypes within a species share \\\"core\\\" genes, they also contain variable numbers of \\\"specific\\\" genes and different structures of \\\"core\\\" genes that are only present in a subset of individuals. Using a common reference genome may thus lead to a loss of genotype-specific information in the assembled Reference Transcript Dataset (RTD) and the generation of erroneous, incomplete or misleading transcriptomics analysis results. In this study, we assembled genotype-specific RTD (sRTD) and common reference-based RTD (cRTD) from RNA-seq data of cultivated Barke and Morex barley, respectively. Our quantitative evaluation showed that the sRTD has a significantly higher diversity of transcripts and alternative splicing events, whereas the cRTD missed 40% of transcripts present in the sRTD and it only has \\u223c70% accurate transcript assemblies. We found that the sRTD is more accurate for transcript quantification as well as differential expression analysis. However, gene-level quantification is less affected, which may be a reasonable compromise when a high-quality genotype-specific reference is not available.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202101255\", \"publication_date\": \"2022-04-22\"}, {\"work_id\": \"W4296047174\", \"doi_x\": \"10.1101/2022.09.14.507918\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic variation during culture-adaptation of genetically complex<i>Plasmodium falciparum</i>clinical isolates\", \"abstract\": \"Abstract Experimental studies on the biology of malaria parasites have been mostly based on laboratory-adapted lines, but there is limited understanding of how these may differ from parasites in natural infections. Loss-of-function mutants have previously been shown to emerge during culture of some Plasmodium falciparum clinical isolates, in analyses that focused on single-genotype infections. The present study included a broader array of isolates, mostly representing multiple-genotype infections which are more typical in areas where malaria is highly endemic. Genome sequence data from multiple time points during several months of culture adaptation of 28 West African isolates were analysed, including previously available sequences along with new genome sequences from additional isolates and timepoints. Some genetically complex isolates eventually became fixed over time to single surviving genotypes in culture, whereas others retained diversity although proportions of genotypes varied over time. Drug-resistance allele frequencies did not show overall directional changes, suggesting that resistance-associated costs are not the main causes of fitness differences among parasites in culture. Loss-of-function mutants emerged during culture in several of the multiple-genotype isolates, affecting genes (including AP2-HS, EPAC and SRPK1 ) for which loss-of-function mutants were previously seen to emerge in single-genotype isolates. Parasite clones were derived by limiting dilution from six of the isolates, and sequencing identified de novo variants not detected in the bulk isolate sequences. Interestingly, most of these were nonsense mutants and frameshifts disrupting the coding sequence of EPAC , the gene with the largest number of independent nonsense mutants previously identified in laboratory-adapted lines. Analysis of Identity-By-Descent to explore relatedness among clones revealed co-occurring non-identical sibling parasites, illustrative of the natural genetic structure within parasite populations.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.14.507918\", \"publication_date\": \"2022-09-15\"}, {\"work_id\": \"W4310668128\", \"doi_x\": \"10.1016/j.cell.2022.11.010\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"IFT-A structure reveals carriages for membrane protein transport into cilia\", \"abstract\": \"Intraflagellar transport (IFT) trains are massive molecular machines that traffic proteins between cilia and the cell body. Each IFT train is a dynamic polymer of two large complexes (IFT-A and -B) and motor proteins, posing a formidable challenge to mechanistic understanding. Here, we reconstituted the complete human IFT-A complex and obtained its structure using cryo-EM. Combined with AlphaFold prediction and genome-editing studies, our results illuminate how IFT-A polymerizes, interacts with IFT-B, and uses an array of \\u03b2-propeller and TPR domains to create \\\"carriages\\\" of the IFT train that engage TULP adaptor proteins. We show that IFT-A\\u22c5TULP carriages are essential for cilia localization of diverse membrane proteins, as well as ICK-the key kinase regulating IFT train turnaround. These data establish a structural link between IFT-A's distinct functions, provide a blueprint for IFT-A in the train, and shed light on how IFT evolved from a proto-coatomer ancestor.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.11.010\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4367174996\", \"doi_x\": \"10.1093/nar/gkad297\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Prediction of protein structures, functions and interactions using the IntFOLD7, MultiFOLD and ModFOLDdock servers\", \"abstract\": \"Abstract The IntFOLD server based at the University of Reading has been a leading method over the past decade in providing free access to accurate prediction of protein structures and functions. In a post-AlphaFold2 world, accurate models of tertiary structures are widely available for even more protein targets, so there has been a refocus in the prediction community towards the accurate modelling of protein-ligand interactions as well as modelling quaternary structure assemblies. In this paper, we describe the latest improvements to IntFOLD, which maintains its competitive structure prediction performance by including the latest deep learning methods while also integrating accurate model quality estimates and 3D models of protein-ligand interactions. Furthermore, we also introduce our two new server methods: MultiFOLD for accurately modelling both tertiary and quaternary structures, with performance which has been independently verified to outperform the standard AlphaFold2 methods, and ModFOLDdock, which provides world-leading quality estimates for quaternary structure models. The IntFOLD7, MultiFOLD and ModFOLDdock servers are available at: https://www.reading.ac.uk/bioinf/.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkad297\", \"publication_date\": \"2023-04-27\"}, {\"work_id\": \"W4225756883\", \"doi_x\": \"10.1016/bs.mie.2021.11.010\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Application of antibiotic-derived fluorescent probes to bacterial studies\", \"abstract\": \"Fluorescent probes are extensively applied as useful tools for imaging and determining dynamic processes in bacterial cells. In particular, antibiotic-derived fluorescent probes which can visualize the presence or the localization of antibiotics within bacteria through the monitoring of changes in fluorescence signal, are particularly useful. They form an emerging set of tools for studying the mode of action of their parent antibiotics and examining bacterial resistance and persistence, with the long-term goal of developing fresh approaches to the treatment of drug-resistant bacterial infections. In this chapter, we discuss the applications of antibiotic-based fluorescent probes to visualize bacteria, focusing on the techniques we have utilized to study their localization, penetration and efflux. We describe detailed protocols for analysis of bacteria using microscopy, flow cytometry, and plate reader-based methods based on these probes.\", \"doi_y\": \"https://doi.org/10.1016/bs.mie.2021.11.010\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4291010968\", \"doi_x\": \"10.1007/s00125-022-05779-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Live enteroviruses, but not other viruses, detected in human pancreas at the onset of type 1 diabetes in the DiViD study\", \"abstract\": \"Enterovirus (EV) infection of pancreatic islet cells is one possible factor contributing to type 1 diabetes development. We have reported the presence of EV genome by PCR and of EV proteins by immunohistochemistry in pancreatic sections. Here we explore multiple human virus species in the Diabetes Virus Detection (DiViD) study cases using innovative methods, including virus passage in cell cultures.Six recent-onset type 1 diabetes patients (age 24-35) were included in the DiViD study. Minimal pancreatic tail resection was performed under sterile conditions. Eleven live cases (age 43-83) of pancreatic carcinoma without diabetes served as control cases. In the present study, we used EV detection methods that combine virus growth in cell culture, gene amplification and detection of virus-coded proteins by immunofluorescence. Pancreas homogenates in cell culture medium were incubated with EV-susceptible cell lines for 3 days. Two to three blind passages were performed. DNA and RNA were extracted from both pancreas tissue and cell cultures. Real-time PCR was used for detecting 20 different viral agents other than EVs (six herpesviruses, human polyomavirus [BK virus and JC virus], parvovirus B19, hepatitis B virus, hepatitis C virus, hepatitis A virus, mumps, rubella, influenza A/B, parainfluenza 1-4, respiratory syncytial virus, astrovirus, norovirus, rotavirus). EV genomes were detected by endpoint PCR using five primer pairs targeting the partially conserved 5' untranslated region genome region of the A, B, C and D species. Amplicons were sequenced. The expression of EV capsid proteins was evaluated in cultured cells using a panel of EV antibodies.Samples from six of six individuals with type 1 diabetes (cases) and two of 11 individuals without diabetes (control cases) contained EV genomes (p<0.05). In contrast, genomes of 20 human viruses other than EVs could be detected only once in an individual with diabetes (Epstein-Barr virus) and once in an individual without diabetes (parvovirus B19). EV detection was confirmed by immunofluorescence of cultured cells incubated with pancreatic extracts: viral antigens were expressed in the cytoplasm of approximately 1% of cells. Notably, infection could be transmitted from EV-positive cell cultures to uninfected cell cultures using supernatants filtered through 100 nm membranes, indicating that infectious agents of less than 100 nm were present in pancreases. Due to the slow progression of infection in EV-carrying cell cultures, cytopathic effects were not observed by standard microscopy but were recognised by measuring cell viability. Sequences of 5' untranslated region amplicons were compatible with EVs of the B, A and C species. Compared with control cell cultures exposed to EV-negative pancreatic extracts, EV-carrying cell cultures produced significantly higher levels of IL-6, IL-8 and monocyte chemoattractant protein-1 (MCP1).Sensitive assays confirm that the pancreases of all DiViD cases contain EVs but no other viruses. Analogous EV strains have been found in pancreases of two of 11 individuals without diabetes. The detected EV strains can be passaged in series from one cell culture to another in the form of poorly replicating live viruses encoding antigenic proteins recognised by multiple EV-specific antibodies. Thus, the early phase of type 1 diabetes is associated with a low-grade infection by EVs, but not by other viral agents.\", \"doi_y\": \"https://doi.org/10.1007/s00125-022-05779-2\", \"publication_date\": \"2022-08-12\"}, {\"work_id\": \"W4382133299\", \"doi_x\": \"10.1016/j.crmeth.2023.100508\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Affinity-enhanced RNA-binding domains as tools to understand RNA recognition\", \"abstract\": \"Understanding how the RNA-binding domains of a protein regulator are used to recognize its RNA targets is a key problem in RNA biology, but RNA-binding domains with very low affinity do not perform well in the methods currently available to characterize protein-RNA interactions. Here, we propose to use conservative mutations that enhance the affinity of RNA-binding domains to overcome this limitation. As a proof of principle, we have designed and validated an affinity-enhanced K-homology (KH) domain mutant of the fragile X syndrome protein FMRP, a key regulator of neuronal development, and used this mutant to determine the domain\\u2019s sequence preference and to explain FMRP recognition of specific RNA motifs in the cell. Our results validate our concept and our nuclear magnetic resonance (NMR)-based workflow. While effective mutant design requires an understanding of the underlying principles of RNA recognition by the relevant domain type, we expect the method will be used effectively in many RNA-binding domains.\", \"doi_y\": \"https://doi.org/10.1016/j.crmeth.2023.100508\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4225775936\", \"doi_x\": \"10.1038/s43705-022-00110-x\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"RNA-viromics reveals diverse communities of soil RNA viruses with the potential to affect grassland ecosystems across multiple trophic levels\", \"abstract\": \"The distribution and diversity of RNA viruses in soil ecosystems are largely unknown, despite their significant impact on public health, ecosystem functions, and food security. Here, we characterise soil RNA viral communities along an altitudinal productivity gradient of peat, managed grassland and coastal soils. We identified 3462 viral contigs in RNA viromes from purified virus-like-particles in five soil-types and assessed their spatial distribution, phylogenetic diversity and potential host ranges. Soil types exhibited minimal similarity in viral community composition, but with >10-fold more viral contigs shared between managed grassland soils when compared with peat or coastal soils. Phylogenetic analyses predicted soil RNA viral communities are formed from viruses of bacteria, plants, fungi, vertebrates and invertebrates, with only 12% of viral contigs belonging to the bacteria-infecting Leviviricetes class. 11% of viral contigs were found to be most closely related to members of the Ourmiavirus genus, suggesting that members of this clade of plant viruses may be far more widely distributed and diverse than previously thought. These results contrast with soil DNA viromes which are typically dominated by bacteriophages. RNA viral communities, therefore, have the potential to exert influence on inter-kingdom interactions across terrestrial biomes.\", \"doi_y\": \"https://doi.org/10.1038/s43705-022-00110-x\", \"publication_date\": \"2022-04-08\"}, {\"work_id\": \"W4386059414\", \"doi_x\": \"10.1039/d3sc03020b\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"<b>Stabilisation of the RirA [4Fe-4S] cluster results in loss of iron-sensing function</b>\", \"abstract\": \"The regulator RirA senses Fe through its [4Fe\\u20134S] cluster, which may have only three Cys ligands from the protein. Addition of a likely fourth Asp ligand stabilised the cluster, rendering it unable to sense Fe and locked into its DNA-binding state.\", \"doi_y\": \"https://doi.org/10.1039/d3sc03020b\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4307342094\", \"doi_x\": \"10.1111/nph.18573\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic and functional analysis of phage\\u2010mediated horizontal gene transfer in <i>Pseudomonas syringae</i> on the plant surface\", \"abstract\": \"Many strains of Pseudomonas colonise plant surfaces, including the cherry canker pathogens, Pseudomonas syringae pathovars syringae and morsprunorum. We have examined the genomic diversity of P. syringae in the cherry phyllosphere and focused on the role of prophages in transfer of genes encoding Type 3 secreted effector (T3SE) proteins contributing to the evolution of virulence. Phylogenomic analysis was carried out on epiphytic pseudomonads in the UK orchards. Significant differences in epiphytic populations occurred between regions. Nonpathogenic strains were found to contain reservoirs of T3SE genes. Members of P. syringae phylogroups 4 and 10 were identified for the first time from Prunus. Using bioinformatics, we explored the presence of the gene encoding T3SE HopAR1 within related prophage sequences in diverse P. syringae strains including cherry epiphytes and pathogens. Results indicated that horizontal gene transfer (HGT) of this effector between phylogroups may have involved phage. Prophages containing hopAR1 were demonstrated to excise, circularise and transfer the gene on the leaf surface. The phyllosphere provides a dynamic environment for prophage-mediated gene exchange and the potential for the emergence of new more virulent pathotypes. Our results suggest that genome-based epidemiological surveillance of environmental populations will allow the timely application of control measures to prevent damaging diseases.\", \"doi_y\": \"https://doi.org/10.1111/nph.18573\", \"publication_date\": \"2022-12-02\"}, {\"work_id\": \"W4304608986\", \"doi_x\": \"10.1093/synbio/ysac023\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"basicsynbio and the BASIC SEVA collection: software and vectors for an established DNA assembly method\", \"abstract\": \"Abstract Standardized deoxyribonucleic acid (DNA) assembly methods utilizing modular components provide a powerful framework to explore designs and iterate through Design\\u2013Build\\u2013Test\\u2013Learn cycles. Biopart Assembly Standard for Idempotent Cloning (BASIC) DNA assembly uses modular parts and linkers, is highly accurate, easy to automate, free for academic and commercial use and enables hierarchical assemblies through an idempotent format. These features enable applications including pathway engineering, ribosome binding site (RBS) tuning, fusion protein engineering and multiplexed guide ribonucleic acid (RNA) expression. In this work, we present basicsynbio, open-source software encompassing a Web App (https://basicsynbio.web.app/) and Python Package (https://github.com/LondonBiofoundry/basicsynbio), enabling BASIC construct design via simple drag-and-drop operations or programmatically. With basicsynbio, users can access commonly used BASIC parts and linkers while designing new parts and assemblies with exception handling for common errors. Users can export sequence data and create instructions for manual or acoustic liquid-handling platforms. Instruction generation relies on the BasicBuild Open Standard, which is parsed for bespoke workflows and is serializable in JavaScript Object Notation for transfer and storage. We demonstrate basicsynbio, assembling 30 vectors using sequences including modules from the Standard European Vector Architecture (SEVA). The BASIC SEVA vector collection is compatible with BASIC and Golden Gate using BsaI. Vectors contain one of six antibiotic resistance markers and five origins of replication from different compatibility groups. The collection is available via Addgene under an OpenMTA agreement. Furthermore, vector sequences are available from within the basicsynbio application programming interface with other collections of parts and linkers, providing a powerful environment for designing assemblies for bioengineering applications. Graphical Abstract\", \"doi_y\": \"https://doi.org/10.1093/synbio/ysac023\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4380551816\", \"doi_x\": \"10.1093/bioadv/vbad078\", \"af_direct_citation\": 1, \"af_indirect_citations\": 9, \"title\": \"Improvement of protein tertiary and quaternary structure predictions using the ReFOLD refinement method and the AlphaFold2 recycling process\", \"abstract\": \"Abstract Motivation The accuracy gap between predicted and experimental structures has been significantly reduced following the development of AlphaFold2 (AF2). However, for many targets, AF2 models still have room for improvement. In previous CASP experiments, highly computationally intensive MD simulation-based methods have been widely used to improve the accuracy of single 3D models. Here, our ReFOLD pipeline was adapted to refine AF2 predictions while maintaining high model accuracy at a modest computational cost. Furthermore, the AF2 recycling process was utilized to improve 3D models by using them as custom template inputs for tertiary and quaternary structure predictions. Results According to the Molprobity score, 94% of the generated 3D models by ReFOLD were improved. AF2 recycling showed an improvement rate of 87.5% (using MSAs) and 81.25% (using single sequences) for monomeric AF2 models and 100% (MSA) and 97.8% (single sequence) for monomeric non-AF2 models, as measured by the average change in lDDT. By the same measure, the recycling of multimeric models showed an improvement rate of as much as 80% for AF2-Multimer (AF2M) models and 94% for non-AF2M models. Availability and implementation Refinement using AlphaFold2-Multimer recycling is available as part of the MultiFOLD docker package (https://hub.docker.com/r/mcguffin/multifold). The ReFOLD server is available at https://www.reading.ac.uk/bioinf/ReFOLD/ and the modified scripts can be downloaded from https://www.reading.ac.uk/bioinf/downloads/. Supplementary information Supplementary data are available at Bioinformatics Advances online.\", \"doi_y\": \"https://doi.org/10.1093/bioadv/vbad078\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4292074325\", \"doi_x\": \"10.3390/ijms23168877\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Comprehensive Collection and Prediction of ABC Transmembrane Protein Structures in the AI Era of Structural Biology\", \"abstract\": \"The number of unique transmembrane (TM) protein structures doubled in the last four years, which can be attributed to the revolution of cryo-electron microscopy. In addition, AlphaFold2 (AF2) also provided a large number of predicted structures with high quality. However, if a specific protein family is the subject of a study, collecting the structures of the family members is highly challenging in spite of existing general and protein domain-specific databases. Here, we demonstrate this and assess the applicability and usability of automatic collection and presentation of protein structures via the ABC protein superfamily. Our pipeline identifies and classifies transmembrane ABC protein structures using the PFAM search and also aims to determine their conformational states based on special geometric measures, conftors. Since the AlphaFold database contains structure predictions only for single polypeptide chains, we performed AF2-Multimer predictions for human ABC half transporters functioning as dimers. Our AF2 predictions warn of possibly ambiguous interpretation of some biochemical data regarding interaction partners and call for further experiments and experimental structure determination. We made our predicted ABC protein structures available through a web application, and we joined the 3D-Beacons Network to reach the broader scientific community through platforms such as PDBe-KB.\", \"doi_y\": \"https://doi.org/10.3390/ijms23168877\", \"publication_date\": \"2022-08-09\"}, {\"work_id\": \"W4318620410\", \"doi_x\": \"10.1128/aem.01704-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Thermophilic Carboxylesterases from Hydrothermal Vents of the Volcanic Island of Ischia Active on Synthetic and Biobased Polymers and Mycotoxins\", \"abstract\": \"Hydrothermal vents are geographically widespread and host microorganisms with robust enzymes useful in various industrial applications. We examined microbial communities and carboxylesterases of two terrestrial hydrothermal vents of the volcanic island of Ischia (Italy) predominantly composed of Firmicutes, Proteobacteria, and Bacteroidota. High-temperature enrichment cultures with the polyester plastics polyhydroxybutyrate and polylactic acid (PLA) resulted in an increase of Thermus and Geobacillus species and to some extent Fontimonas and Schleiferia species. The screening at 37 to 70\\u00b0C of metagenomic fosmid libraries from above enrichment cultures identified three hydrolases (IS10, IS11, and IS12), all derived from yet-uncultured Chloroflexota and showing low sequence identity (33 to 56%) to characterized enzymes. Enzymes expressed in Escherichia coli exhibited maximal esterase activity at 70 to 90\\u00b0C, with IS11 showing the highest thermostability (90% activity after 20-min incubation at 80\\u00b0C). IS10 and IS12 were highly substrate promiscuous and hydrolyzed all 51 monoester substrates tested. Enzymes were active with PLA, polyethylene terephthalate model substrate, and mycotoxin T-2 (IS12). IS10 and IS12 had a classical \\u03b1/\\u03b2-hydrolase core domain with a serine hydrolase catalytic triad (Ser155, His280, and Asp250) in their hydrophobic active sites. The crystal structure of IS11 resolved at 2.92 \\u00c5 revealed the presence of a N-terminal \\u03b2-lactamase-like domain and C-terminal lipocalin domain. The catalytic cleft of IS11 included catalytic Ser68, Lys71, Tyr160, and Asn162, whereas the lipocalin domain enclosed the catalytic cleft like a lid and contributed to substrate binding. Our study identified novel thermotolerant carboxylesterases with a broad substrate range, including polyesters and mycotoxins, for potential applications in biotechnology. IMPORTANCE High-temperature-active microbial enzymes are important biocatalysts for many industrial applications, including recycling of synthetic and biobased polyesters increasingly used in textiles, fibers, coatings and adhesives. Here, we identified three novel thermotolerant carboxylesterases (IS10, IS11, and IS12) from high-temperature enrichment cultures from Ischia hydrothermal vents and incubated with biobased polymers. The identified metagenomic enzymes originated from uncultured Chloroflexota and showed low sequence similarity to known carboxylesterases. Active sites of IS10 and IS12 had the largest effective volumes among the characterized prokaryotic carboxylesterases and exhibited high substrate promiscuity, including hydrolysis of polyesters and mycotoxin T-2 (IS12). Though less promiscuous than IS10 and IS12, IS11 had a higher thermostability with a high temperature optimum (80 to 90\\u00b0C) for activity and hydrolyzed polyesters, and its crystal structure revealed an unusual lipocalin domain likely involved in substrate binding. The polyesterase activity of these enzymes makes them attractive candidates for further optimization and potential application in plastics recycling.\", \"doi_y\": \"https://doi.org/10.1128/aem.01704-22\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4213126762\", \"doi_x\": \"10.1111/febs.16396\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Probing heavy metal binding to phycobiliproteins\", \"abstract\": \"Blue-green algae, also known as cyanobacteria, contain some of the most efficient light-harvesting complexes known. These large, colourful complexes consist of phycobiliproteins which are extremely valuable in the cosmetics, food, nutraceutical and pharmaceutical industries. Additionally, the colourful and fluorescent properties of phycobiliproteins can be modulated by metal ions, making them highly attractive as heavy metal sensors and heavy metal scavengers. Although the overall quenching ability metal ions have on phycobiliproteins is known, the mechanism of heavy metal binding to phycobiliproteins is not fully understood, limiting their widespread quantitative applications. Here, we show using high-resolution native mass spectrometry that phycobiliprotein complexes bind metal ions in different manners. Through monitoring the binding equilibria and metal-binding stoichiometry, we show in particular copper and silver to have drastic, yet different effects on phycobiliprotein structure, both copper and silver modulate the overall complex properties. Together, the data reveals the mechanisms by which metal ions can modulate phycobiliprotein properties which can be used as a basis for the future design of metal-related phycobiliprotein applications.\", \"doi_y\": \"https://doi.org/10.1111/febs.16396\", \"publication_date\": \"2022-02-22\"}, {\"work_id\": \"W4317721966\", \"doi_x\": \"10.3389/fmolb.2023.1113762\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Perspective: Structure determination of protein-ligand complexes at room temperature using X-ray diffraction approaches\", \"abstract\": \"The interaction between macromolecular proteins and small molecule ligands is an essential component of cellular function. Such ligands may include enzyme substrates, molecules involved in cellular signalling or pharmaceutical drugs. Together with biophysical techniques used to assess the thermodynamic and kinetic properties of ligand binding to proteins, methodology to determine high-resolution structures that enable atomic level interactions between protein and ligand(s) to be directly visualised is required. Whilst such structural approaches are well established with high throughput X-ray crystallography routinely used in the pharmaceutical sector, they provide only a static view of the complex. Recent advances in X-ray structural biology methods offer several new possibilities that can examine protein-ligand complexes at ambient temperature rather than under cryogenic conditions, enable transient binding sites and interactions to be characterised using time-resolved approaches and combine spectroscopic measurements from the same crystal that the structures themselves are determined. This Perspective reviews several recent developments in these areas and discusses new possibilities for applications of these advanced methodologies to transform our understanding of protein-ligand interactions.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2023.1113762\", \"publication_date\": \"2023-01-23\"}, {\"work_id\": \"W4296802427\", \"doi_x\": \"10.1371/journal.pgen.1010414\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The helper NLR immune protein NRC3 mediates the hypersensitive cell death caused by the cell-surface receptor Cf-4\", \"abstract\": \"Cell surface pattern recognition receptors (PRRs) activate immune responses that can include the hypersensitive cell death. However, the pathways that link PRRs to the cell death response are poorly understood. Here, we show that the cell surface receptor-like protein Cf-4 requires the intracellular nucleotide-binding domain leucine-rich repeat containing receptor (NLR) NRC3 to trigger a confluent cell death response upon detection of the fungal effector Avr4 in leaves of Nicotiana benthamiana. This NRC3 activity requires an intact N-terminal MADA motif, a conserved signature of coiled-coil (CC)-type plant NLRs that is required for resistosome-mediated immune responses. A chimeric protein with the N-terminal \\u03b11 helix of Arabidopsis ZAR1 swapped into NRC3 retains the capacity to mediate Cf-4 hypersensitive cell death. Pathogen effectors acting as suppressors of NRC3 can suppress Cf-4-triggered hypersensitive cell-death. Our findings link the NLR resistosome model to the hypersensitive cell death caused by a cell surface PRR.\", \"doi_y\": \"https://doi.org/10.1371/journal.pgen.1010414\", \"publication_date\": \"2022-09-22\"}, {\"work_id\": \"W4382937218\", \"doi_x\": \"10.1038/s41565-023-01431-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Next-generation protein-based materials capture and preserve projectiles from supersonic impacts\", \"abstract\": \"Extreme energy-dissipating materials are essential for a range of applications. The military and police force require ballistic armour to ensure the safety of their personnel, while the aerospace industry requires materials that enable the capture, preservation and study of hypervelocity projectiles. However, current industry standards display at least one inherent limitation, such as weight, breathability, stiffness, durability and failure to preserve captured projectiles. To resolve these limitations, we have turned to nature, using proteins that have evolved over millennia to enable effective energy dissipation. Specifically, a recombinant form of the mechanosensitive protein talin was incorporated into a monomeric unit and crosslinked, resulting in a talin shock-absorbing material (TSAM). When subjected to 1.5 km s-1 supersonic shots, TSAMs were shown to absorb the impact and capture and preserve the projectile.\", \"doi_y\": \"https://doi.org/10.1038/s41565-023-01431-1\", \"publication_date\": \"2023-07-03\"}, {\"work_id\": \"W4378952892\", \"doi_x\": \"10.1107/s2059798323003595\", \"af_direct_citation\": 1, \"af_indirect_citations\": 9, \"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"abstract\": \"The Collaborative Computational Project No. 4 (CCP4) is a UK-led international collective with a mission to develop, test, distribute and promote software for macromolecular crystallography. The CCP4 suite is a multiplatform collection of programs brought together by familiar execution routines, a set of common libraries and graphical interfaces. The CCP4 suite has experienced several considerable changes since its last reference article, involving new infrastructure, original programs and graphical interfaces. This article, which is intended as a general literature citation for the use of the CCP4 software suite in structure determination, will guide the reader through such transformations, offering a general overview of the new features and outlining future developments. As such, it aims to highlight the individual programs that comprise the suite and to provide the latest references to them for perusal by crystallographers around the world.\", \"doi_y\": \"https://doi.org/10.1107/s2059798323003595\", \"publication_date\": \"2023-05-30\"}, {\"work_id\": \"W4379600803\", \"doi_x\": \"10.1093/nar/gkad478\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Reverse transcriptases prime DNA synthesis\", \"abstract\": \"Abstract The discovery of reverse transcriptases (RTs) challenged the central dogma by establishing that genetic information can also flow from RNA to DNA. Although they act as DNA polymerases, RTs are distantly related to replicases that also possess de novo primase activity. Here we identify that CRISPR associated RTs (CARTs) directly prime DNA synthesis on both RNA and DNA. We demonstrate that RT-dependent priming is utilized by some CRISPR-Cas complexes to synthesise new spacers and integrate these into CRISPR arrays. Expanding our analyses, we show that primer synthesis activity is conserved in representatives of other major RT classes, including group II intron RT, telomerase and retroviruses. Together, these findings establish a conserved innate ability of RTs to catalyse de novo DNA primer synthesis, independently of accessory domains or alternative priming mechanisms, which likely plays important roles in a wide variety of biological pathways.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkad478\", \"publication_date\": \"2023-06-06\"}, {\"work_id\": \"W4386142056\", \"doi_x\": \"10.3390/applbiosci2030029\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Deregulations of RNA Pol II Subunits in Cancer\", \"abstract\": \"Deregulated transcription is a well-known characteristic of cancer cells, with differentially expressed genes being a common feature of several cancers. Often, deregulated transcription is a consequence of alterations in transcription factors (TFs), which play a crucial role in gene expression and can act as tumour suppressors or proto-oncogenes. In eukaryotic organisms, transcription is carried out by three distinct RNA polymerase complexes: Pol I, Pol II, and Pol III. Pol II, specifically, is responsible for transcribing messenger RNA (mRNA), the protein coding part of the genome, as well as long non-coding RNAs (lncRNAs). While there is considerable research on the impact of specific deregulated transcription factors in cancer development, there is a lack of studies focusing on defects within the RNA polymerase complexes and their subunits. This review aims to shed light in particular on the Pol II complex and highlight the deregulation of its subunits that have a significant impact on tumour development, prognosis, and survival. By providing a comprehensive overview of our current understanding of Pol II subunits in cancer, this review emphasizes the importance of further research in this area. It suggests that exploring these subunits\\u2019 deregulations could lead to the identification of valuable biomarkers and potential therapeutic targets, making it a topic of collective interest.\", \"doi_y\": \"https://doi.org/10.3390/applbiosci2030029\", \"publication_date\": \"2023-08-24\"}, {\"work_id\": \"W4311202139\", \"doi_x\": \"10.1042/ebc20220095\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"The increasing role of structural proteomics in cyanobacteria\", \"abstract\": \"Abstract Cyanobacteria, also known as blue\\u2013green algae, are ubiquitous organisms on the planet. They contain tremendous protein machineries that are of interest to the biotechnology industry and beyond. Recently, the number of annotated cyanobacterial genomes has expanded, enabling structural studies on known gene-coded proteins to accelerate. This review focuses on the advances in mass spectrometry (MS) that have enabled structural proteomics studies to be performed on the proteins and protein complexes within cyanobacteria. The review also showcases examples whereby MS has revealed critical mechanistic information behind how these remarkable machines within cyanobacteria function.\", \"doi_y\": \"https://doi.org/10.1042/ebc20220095\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4308183560\", \"doi_x\": \"10.1038/s42004-022-00754-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"An open-source molecular builder and free energy preparation workflow\", \"abstract\": \"Automated free energy calculations for the prediction of binding free energies of congeneric series of ligands to a protein target are growing in popularity, but building reliable initial binding poses for the ligands is challenging. Here, we introduce the open-source FEgrow workflow for building user-defined congeneric series of ligands in protein binding pockets for input to free energy calculations. For a given ligand core and receptor structure, FEgrow enumerates and optimises the bioactive conformations of the grown functional group(s), making use of hybrid machine learning/molecular mechanics potential energy functions where possible. Low energy structures are optionally scored using the gnina convolutional neural network scoring function, and output for more rigorous protein-ligand binding free energy predictions. We illustrate use of the workflow by building and scoring binding poses for ten congeneric series of ligands bound to targets from a standard, high quality dataset of protein-ligand complexes. Furthermore, we build a set of 13 inhibitors of the SARS-CoV-2 main protease from the literature, and use free energy calculations to retrospectively compute their relative binding free energies. FEgrow is freely available at https://github.com/cole-group/FEgrow , along with a tutorial.\", \"doi_y\": \"https://doi.org/10.1038/s42004-022-00754-9\", \"publication_date\": \"2022-10-27\"}, {\"work_id\": \"W4378716282\", \"doi_x\": \"10.1038/s41589-023-01341-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Spontaneously established syntrophic yeast communities improve bioproduction\", \"abstract\": \"Nutritional codependence (syntrophy) has underexplored potential to improve biotechnological processes by using cooperating cell types. So far, design of yeast syntrophic communities has required extensive genetic manipulation, as the co-inoculation of most eukaryotic microbial auxotrophs does not result in cooperative growth. Here we employ high-throughput phenotypic screening to systematically test pairwise combinations of auxotrophic Saccharomyces cerevisiae deletion mutants. Although most coculture pairs do not enter syntrophic growth, we identify 49 pairs that spontaneously form syntrophic, synergistic communities. We characterized the stability and growth dynamics of nine cocultures and demonstrated that a pair of tryptophan auxotrophs grow by exchanging a pathway intermediate rather than end products. We then introduced a malonic semialdehyde biosynthesis pathway split between different pairs of auxotrophs, which resulted in increased production. Our results report the spontaneous formation of stable syntrophy in S. cerevisiae auxotrophs and illustrate the biotechnological potential of dividing labor in a cooperating intraspecies community.\", \"doi_y\": \"https://doi.org/10.1038/s41589-023-01341-2\", \"publication_date\": \"2023-05-29\"}, {\"work_id\": \"W4310009650\", \"doi_x\": \"10.1101/2022.11.23.517669\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Poly ADP-Ribose Signaling is Dysregulated in Huntington\\u2019s Disease Patients\", \"abstract\": \"Abstract Huntington\\u2019s disease (HD) is an autosomal dominant genetic neurodegenerative disease caused by a CAG expansion in the Huntingtin (HTT) gene, translating to an expanded polyglutamine tract in the huntingtin (HTT) protein. Age at disease onset is correlated to CAG repeat length, but varies by decades between individuals with identical repeat lengths. Genome-wide association studies link HD modification to DNA repair and mitochondrial health pathways. Recent clinical studies show elevated DNA damage in HD, even at the premanifest stage of disease. One of the major DNA repair nodes influencing neurodegenerative disease is the PARP pathway. Accumulation of poly ADP-ribose (PAR), produced by PARP1 and PARP2, has been implicated in the pathology of Alzheimer\\u2019s and Parkinson\\u2019s diseases, as well as autosomal recessive cerebellar ataxia. We report that HD mutation carriers have lower cerebrospinal fluid PAR levels than healthy controls, starting at the premanifest stage. Patient-derived fibroblasts have reduced PARP1/2 activity and elevated DNA damage, while elevated PAR levels are only revealed upon inhibition of PAR degradation. These phenotypes are rescued by moderate huntingtin level reduction via the huntingtin-lowering splice modulator drug, LMI070 (Branaplam). As a direct mechanism, we have defined a PAR-binding motif in huntingtin, detected huntingtin complexed with PARylated proteins in human cells during stress, and localized huntingtin to mitotic chromosomes upon inhibition of PAR degradation. Direct huntingtin PAR binding was measured by fluorescence polarization and visualized by atomic force microscopy. These results provide insight into a very early molecular mechanism of HD, suggesting possible targets in HD to design early preventive therapies.\", \"doi_y\": \"https://doi.org/10.1101/2022.11.23.517669\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4281754639\", \"doi_x\": \"10.1016/j.cell.2022.06.005\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Antibody escape of SARS-CoV-2 Omicron BA.4 and BA.5 from vaccine and BA.1 serum\", \"abstract\": \"The Omicron lineage of SARS-CoV-2, which was first described in November 2021, spread rapidly to become globally dominant and has split into a number of sublineages. BA.1 dominated the initial wave but has been replaced by BA.2 in many countries. Recent sequencing from South Africa's Gauteng region uncovered two new sublineages, BA.4 and BA.5, which are taking over locally, driving a new wave. BA.4 and BA.5 contain identical spike sequences, and although closely related to BA.2, they contain further mutations in the receptor-binding domain of their spikes. Here, we study the neutralization of BA.4/5 using a range of vaccine and naturally immune serum and panels of monoclonal antibodies. BA.4/5 shows reduced neutralization by the serum from individuals vaccinated with triple doses of AstraZeneca or Pfizer vaccine compared with BA.1 and BA.2. Furthermore, using the serum from BA.1 vaccine breakthrough infections, there are, likewise, significant reductions in the neutralization of BA.4/5, raising the possibility of repeat Omicron infections.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.06.005\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4285036285\", \"doi_x\": \"10.1039/d2cc02360a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pulse dipolar EPR for determining nanomolar binding affinities\", \"abstract\": \"Protein interaction studies often require very low concentrations and highly sensitive biophysical methods. Here, we demonstrate that pulse dipolar electron paramagnetic resonance spectroscopy allows measuring dissociation constants in the nanomolar range. This approach is appealing for concentration-limited biomolecular systems and medium-to-high-affinity binding studies, demonstrated here at 50 nanomolar protein concentration.\", \"doi_y\": \"https://doi.org/10.1039/d2cc02360a\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4318542342\", \"doi_x\": \"10.1093/jxb/erad041\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"A novel mutant allele of <i>AtCNGC15</i> reveals a dual function of nuclear calcium release in the root meristem\", \"abstract\": \"Calcium release to the nucleoplasm of root meristem cells was demonstrated to modulate root development. The calcium channel encoded by cyclic nucleotide-gated channel (CNGC) 15 localizes at the nuclear envelope in young Arabidopsis seedlings. In contrast, at later stages of root growth, overexpression analysis showed that AtCNGC15 can relocalize to the plasma membrane to mediate primary nitrate-induced gene expression. This raises the question as to whether nuclear localized AtCNGC15 is required for root apical meristem development in young Arabidopsis seedlings, and whether nitrate signalling occurs independently of nuclear localized AtCNGC15 at this developmental stage. In this study, we characterize a novel mutant allele of AtCNGC15 and demonstrate that the mutation of a highly conserved aspartic acid in the C-linker domain is sufficient to impair the gating of AtCNCG15. We demonstrate that AtCNGC15 mediates the nuclear calcium release that modulates root apical meristem development and nitrate-induced LBD39 expression. We also show that, in the presence of nitrate, the relocalization of AtCNGC15 at the plasma membrane occurs specifically in the columella cells. Our results further suggest that the induction of LBD37, LBD38, and LBD39 in the presence of nitrate is modulated by different inputs of cytoplasmic or nuclear calcium release.\", \"doi_y\": \"https://doi.org/10.1093/jxb/erad041\", \"publication_date\": \"2023-01-30\"}, {\"work_id\": \"W4296077745\", \"doi_x\": \"10.1101/2022.09.16.508273\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The patterned assembly and stepwise Vps4-mediated disassembly of composite ESCRT-III polymers drives archaeal cell division\", \"abstract\": \"Abstract ESCRT-III family proteins form composite polymers that deform and cut membrane tubes in the context of a wide range of cell biological processes across the tree of life. In reconstituted systems sequential changes in the composition of ESCRT-III polymers induced by the AAA ATPase Vps4 have been shown to remodel membranes. However, it is not known how composite ESCRT-III polymers are organised and remodelled in space and time in cells. Here, taking advantage of the relative simplicity of the ESCRT-III-dependent division system in Sulfolobus acidocaldarius , one of the closest experimentally tractable prokaryotic relative of eukaryotes, we use super-resolution microscopy and computational modelling to show how CdvB/CdvB1/CdvB2 proteins form a precisely patterned composite ESCRT-III division ring which undergoes stepwise Vps4-dependent disassembly and contracts to cut cells into two. These observations lead us to suggest sequential changes in a patterned composite polymer as a general mechanism of ESCRT-III-dependent membrane remodelling.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.16.508273\", \"publication_date\": \"2022-09-16\"}, {\"work_id\": \"W4307486835\", \"doi_x\": \"10.1101/2022.10.26.513919\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Tropomyosin 1-I/C co-ordinates kinesin-1 and dynein motors during<i>oskar</i>mRNA transport\", \"abstract\": \"Abstract Dynein and kinesin motors mediate long-range intracellular transport, translocating towards microtubule minus and plus ends, respectively. Cargoes often undergo bidirectional transport by binding to both motors simultaneously. However, it is not known how motor activities are coordinated in such circumstances. In Drosophila , sequential activities of the dynein-dynactin-BicD-Egalitarian (DDBE) complex and of kinesin-1 deliver oskar mRNA from nurse cells to the oocyte, and within the oocyte to the posterior pole. Here, through in vitro reconstitution, we show that Tm1-I/C, a Tropomyosin-1 isoform, links kinesin-1 in an inactive state to DDBE-associated oskar mRNA. NMR spectroscopy, small-angle X-ray scattering and structural modeling indicate that Tm1-I/C suppresses kinesin-1 activity by stabilizing its autoinhibited conformation, thus preventing a tug-of-war between the opposite polarity motors until kinesin-1 is activated in the oocyte. Our work reveals a novel strategy ensuring sequential activity of microtubule motors.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.26.513919\", \"publication_date\": \"2022-10-26\"}, {\"work_id\": \"W3200383688\", \"doi_x\": \"10.1042/bcj20210375\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Impact of Type II LRRK2 inhibitors on signaling and mitophagy\", \"abstract\": \"Much effort has been devoted to the development of selective inhibitors of the LRRK2 as a potential treatment for LRRK2 driven Parkinson's disease. In this study, we first compare the properties of Type I (GSK3357679A and MLi-2) and Type II (GZD-824, Rebastinib and Ponatinib) kinase inhibitors that bind to the closed or open conformations of the LRRK2 kinase domain, respectively. We show that Type I and Type II inhibitors suppress phosphorylation of Rab10 and Rab12, key physiological substrates of LRRK2 and also promote mitophagy, a process suppressed by LRRK2. Type II inhibitors also display higher potency towards wild-type LRRK2 compared with pathogenic mutants. Unexpectedly, we find that Type II inhibitors, in contrast with Type I compounds, fail to induce dephosphorylation of a set of well-studied LRRK2 biomarker phosphorylation sites at the N-terminal region of LRRK2, including Ser935. These findings emphasize that the biomarker phosphorylation sites on LRRK2 are likely reporting on the open vs closed conformation of LRRK2 kinase and that only inhibitors which stabilize the closed conformation induce dephosphorylation of these biomarker sites. Finally, we demonstrate that the LRRK2[A2016T] mutant which is resistant to MLi-2 Type 1 inhibitor, also induces resistance to GZD-824 and Rebastinib suggesting this mutation could be exploited to distinguish off target effects of Type II inhibitors. Our observations provide a framework of knowledge to aid with the development of more selective Type II LRRK2 inhibitors.\", \"doi_y\": \"https://doi.org/10.1042/bcj20210375\", \"publication_date\": \"2021-10-06\"}, {\"work_id\": \"W4378781380\", \"doi_x\": \"10.1073/pnas.2302858120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structure of the priming arabinosyltransferase AftA required for AG biosynthesis of <i>Mycobacterium tuberculosis</i>\", \"abstract\": \"Arabinogalactan (AG) is an essential cell wall component in mycobacterial species, including the deadly human pathogen Mycobacterium tuberculosis. It plays a pivotal role in forming the rigid mycolyl-AG-peptidoglycan core for in vitro growth. AftA is a membrane-bound arabinosyltransferase and a key enzyme involved in AG biosynthesis which bridges the assembly of the arabinan chain to the galactan chain. It is known that AftA catalyzes the transfer of the first arabinofuranosyl residue from the donor decaprenyl-monophosphoryl-arabinose to the mature galactan chain (i.e., priming); however, the priming mechanism remains elusive. Herein, we report the cryo-EM structure of Mtb AftA. The detergent-embedded AftA assembles as a dimer with an interface maintained by both the transmembrane domain (TMD) and the soluble C-terminal domain (CTD) in the periplasm. The structure shows a conserved glycosyltransferase-C fold and two cavities converging at the active site. A metal ion participates in the interaction of TMD and CTD of each AftA molecule. Structural analyses combined with functional mutagenesis suggests a priming mechanism catalyzed by AftA in Mtb AG biosynthesis. Our data further provide a unique perspective into anti-TB drug discovery.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2302858120\", \"publication_date\": \"2023-05-30\"}, {\"work_id\": \"W4386019098\", \"doi_x\": \"10.1002/bies.202300076\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The evolution of selective autophagy as a mechanism of oxidative stress response\", \"abstract\": \"Ageing is associated with a decline in autophagy and elevated reactive oxygen species (ROS), which can breach the capacity of antioxidant systems. Resulting oxidative stress can cause further cellular damage, including DNA breaks and protein misfolding. This poses a challenge for longevous organisms, including humans. In this review, we hypothesise that in the course of human evolution selective autophagy receptors (SARs) acquired the ability to sense and respond to localised oxidative stress. We posit that in the vicinity of protein aggregates and dysfunctional mitochondria oxidation of key cysteine residues in SARs induces their oligomerisation which initiates autophagy. The degradation of damaged cellular components thus could reduce ROS production and restore redox homeostasis. This evolutionarily acquired function of SARs may represent one of the biological adaptations that contributed to longer lifespan. Inversely, loss of this mechanism can lead to age-related diseases associated with impaired autophagy and oxidative stress.\", \"doi_y\": \"https://doi.org/10.1002/bies.202300076\", \"publication_date\": \"2023-08-21\"}, {\"work_id\": \"W4382011861\", \"doi_x\": \"10.1042/bsr20221986\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Primase-polymerases: how to make a primer from scratch\", \"abstract\": \"To pass on genetic information to the next generation, cells must faithfully replicate their genomes to provide copies for each daughter cell. To synthesise these duplicates, cells employ specialised enzymes called DNA polymerases, which rapidly and accurately replicate nucleic acid polymers. However, most polymerases lack the ability to directly initiate DNA synthesis and required specialised replicases called primases to make short polynucleotide primers, from which they then extend. Replicative primases (eukaryotes and archaea) belong to a functionally diverse enzyme superfamily known as Primase-Polymerases (Prim-Pols), with orthologues present throughout all domains of life. Characterised by a conserved catalytic Prim-Pol domain, these enzymes have evolved various roles in DNA metabolism, including DNA replication, repair, and damage tolerance. Many of these biological roles are fundamentally underpinned by the ability of Prim-Pols to generate primers de novo. This review examines our current understanding of the catalytic mechanisms utilised by Prim-Pols to initiate primer synthesis.\", \"doi_y\": \"https://doi.org/10.1042/bsr20221986\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4385927801\", \"doi_x\": \"10.1038/s41467-023-40779-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"C16orf72/HAPSTR1/TAPR1 functions with BRCA1/Senataxin to modulate replication-associated R-loops and confer resistance to PARP disruption\", \"abstract\": \"Abstract While the toxicity of PARP inhibitors to cells with defects in homologous recombination (HR) is well established, other synthetic lethal interactions with PARP1/PARP2 disruption are poorly defined. To inform on these mechanisms we conducted a genome-wide screen for genes that are synthetic lethal with PARP1/2 gene disruption and identified C16orf72/HAPSTR1/TAPR1 as a novel modulator of replication-associated R-loops. C16orf72 is critical to facilitate replication fork restart, suppress DNA damage and maintain genome stability in response to replication stress. Importantly, C16orf72 and PARP1/2 function in parallel pathways to suppress DNA:RNA hybrids that accumulate at stalled replication forks. Mechanistically, this is achieved through an interaction of C16orf72 with BRCA1 and the RNA/DNA helicase Senataxin to facilitate their recruitment to RNA:DNA hybrids and confer resistance to PARP inhibitors. Together, this identifies a C16orf72/Senataxin/BRCA1-dependent pathway to suppress replication-associated R-loop accumulation, maintain genome stability and confer resistance to PARP inhibitors.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40779-9\", \"publication_date\": \"2023-08-17\"}, {\"work_id\": \"W4308372964\", \"doi_x\": \"10.1002/cbic.202200450\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The Chromatin Regulator HMGA1a Undergoes Phase Separation in the Nucleus**\", \"abstract\": \"Abstract The protein high mobility group A1 (HMGA1) is an important regulator of chromatin organization and function. However, the mechanisms by which it exerts its biological function are not fully understood. Here, we report that the HMGA isoform, HMGA1a, nucleates into foci that display liquid\\u2010like properties in the nucleus, and that the protein readily undergoes phase separation to form liquid condensates in vitro. By bringing together machine\\u2010leaning modelling, cellular and biophysical experiments and multiscale simulations, we demonstrate that phase separation of HMGA1a is promoted by protein\\u2010DNA interactions, and has the potential to be modulated by post\\u2010transcriptional effects such as phosphorylation. We further show that the intrinsically disordered C\\u2010terminal tail of HMGA1a significantly contributes to its phase separation through electrostatic interactions via AT hooks 2 and 3. Our work sheds light on HMGA1 phase separation as an emergent biophysical factor in regulating chromatin structure.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202200450\", \"publication_date\": \"2022-11-30\"}, {\"work_id\": \"W3215144478\", \"doi_x\": \"10.1101/2021.11.17.468998\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Characterizing and explaining impact of disease-associated mutations in proteins without known structures or structural homologues\", \"abstract\": \"Abstract Mutations in human proteins lead to diseases. The structure of these proteins can help understand the mechanism of such diseases and develop therapeutics against them. With improved deep learning techniques such as RoseTTAFold and AlphaFold, we can predict the structure of proteins even in the absence of structural homologues. We modeled and extracted the domains from 553 disease-associated human proteins without known protein structures or close homologues in the Protein Databank (PDB). We noticed that the model quality was higher and the RMSD lower between AlphaFold and RoseTTAFold models for domains that could be assigned to CATH families as compared to those which could only be assigned to Pfam families of unknown structure or could not be assigned to either. We predicted ligand-binding sites, protein-protein interfaces, conserved residues in these predicted structures. We then explored whether the disease-associated missense mutations were in the proximity of these predicted functional sites, if they destabilized the protein structure based on ddG calculations or if they were predicted to be pathogenic. We could explain 80% of these disease-associated mutations based on proximity to functional sites, structural destabilization or pathogenicity. When compared to polymorphisms a larger percentage of disease associated missense mutations were buried, closer to predicted functional sites, predicted as destabilising and/or pathogenic. Usage of models from the two state-of-the-art techniques provide better confidence in our predictions, and we explain 93 additional mutations based on RoseTTAFold models which could not be explained based solely on AlphaFold models.\", \"doi_y\": \"https://doi.org/10.1101/2021.11.17.468998\", \"publication_date\": \"2021-11-19\"}, {\"work_id\": \"W4366588851\", \"doi_x\": \"10.1038/s41594-023-00950-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fibrillin microfibril structure identifies long-range effects of inherited pathogenic mutations affecting a key regulatory latent TGF\\u03b2-binding site\", \"abstract\": \"Genetic mutations in fibrillin microfibrils cause serious inherited diseases, such as Marfan syndrome and Weill-Marchesani syndrome (WMS). These diseases typically show major dysregulation of tissue development and growth, particularly in skeletal long bones, but links between the mutations and the diseases are unknown. Here we describe a detailed structural analysis of native fibrillin microfibrils from mammalian tissue by cryogenic electron microscopy. The major bead region showed pseudo eightfold symmetry where the amino and carboxy termini reside. On the basis of this structure, we show that a WMS deletion mutation leads to the induction of a structural rearrangement that blocks interaction with latent TGF\\u03b2-binding protein-1 at a remote site. Separate deletion of this binding site resulted in the assembly of shorter fibrillin microfibrils with structural alterations. The integrin \\u03b1v\\u03b23-binding site was also mapped onto the microfibril structure. These results establish that in complex extracellular assemblies, such as fibrillin microfibrils, mutations may have long-range structural consequences leading to the disruption of growth factor signaling and the development of disease.\", \"doi_y\": \"https://doi.org/10.1038/s41594-023-00950-8\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4285082691\", \"doi_x\": \"10.1016/j.sbi.2022.102420\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Frontiers in metalloprotein crystallography and cryogenic electron microscopy\", \"abstract\": \"Metalloproteins comprise at least a third of all proteins that utilize redox properties of transition metals on their own or as parts of cofactors. The development of third generation storage ring sources and X-ray free-electron lasers with femtosecond pulses in the first decade of the 21st century has transformed metalloprotein crystallography. In the past decade, cryogenic-electron microscopy single-particle analysis, which does not require crystallization of biological samples has been extensively utilized, particularly for membrane-bound metalloprotein systems. Here, we explore recent frontiers in metalloprotein crystallography and cryogenic electron microscopy, organized for convenience under three metalloprotein-centered biological cycles, focusing on contributions from each technique, their synergy and the ability to preserve metals' redox states when subjected to a particular probe.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2022.102420\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4289868872\", \"doi_x\": \"10.3389/fmicb.2022.948383\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"A new class of biological ion-driven rotary molecular motors with 5:2 symmetry\", \"abstract\": \"Several new structures of three types of protein complexes, obtained by cryo-electron microscopy (cryo-EM) and published between 2019 and 2021, identify a new family of natural molecular wheels, the \\u201c5:2 rotary motors.\\u201d These span the cytoplasmic membranes of bacteria, and their rotation is driven by ion flow into the cell. They consist of a pentameric wheel encircling a dimeric axle within the cytoplasmic membrane of both Gram-positive and gram-negative bacteria. The axles extend into the periplasm, and the wheels extend into the cytoplasm. Rotation of these wheels has never been observed directly; it is inferred from the symmetry of the complexes and from the roles they play within the larger systems that they are known to power. In particular, the new structure of the stator complex of the Bacterial Flagellar Motor, MotA 5 B 2 , is consistent with a \\u201cwheels within wheels\\u201d model of the motor. Other 5:2 rotary motors are believed to share the core rotary function and mechanism, driven by ion-motive force at the cytoplasmic membrane. Their structures diverge in their periplasmic and cytoplasmic parts, reflecting the variety of roles that they perform. This review focuses on the structures of 5:2 rotary motors and their proposed mechanisms and functions. We also discuss molecular rotation in general and its relation to the rotational symmetry of molecular complexes.\", \"doi_y\": \"https://doi.org/10.3389/fmicb.2022.948383\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4223585765\", \"doi_x\": \"10.3390/cells11081307\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"How Pyroptosis Contributes to Inflammation and Fibroblast-Macrophage Cross-Talk in Rheumatoid Arthritis\", \"abstract\": \"About thirty years ago, a new form of pro-inflammatory lytic cell death was observed and termed pyroptosis. Only in 2015, gasdermins were defined as molecules that create pores at the plasma membrane and drive pyroptosis. Today, we know that gasdermin-mediated death is an important antimicrobial defence mechanism in bacteria, yeast and mammals as it destroys the intracellular niche for pathogen replication. However, excessive and uncontrolled cell death also contributes to immunopathology in several chronic inflammatory diseases, including arthritis. In this review, we discuss recent findings where pyroptosis contributes to tissue damage and inflammation with a main focus on injury-induced and autoimmune arthritis. We also review novel functions and regulatory mechanisms of the pyroptotic executors gasdermins. Finally, we discuss possible models of how pyroptosis may contribute to the cross-talk between fibroblast and macrophages, and also how this cross-talk may regulate inflammation by modulating inflammasome activation and pyroptosis induction.\", \"doi_y\": \"https://doi.org/10.3390/cells11081307\", \"publication_date\": \"2022-04-12\"}, {\"work_id\": \"W4290033835\", \"doi_x\": \"10.1101/2022.08.03.502475\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Multi-organ functions of yolk sac during human early development\", \"abstract\": \"Abstract The yolk sac (YS) represents an evolutionarily-conserved extraembryonic structure that ensures timely delivery of nutritional support and oxygen to the developing embryo. However, the YS remains ill-defined in humans. We therefore assemble a complete single cell 3D map of human YS from 3-8 post conception weeks by integrating multiomic protein and gene expression data. We reveal the YS as a site of primitive and definitive haematopoiesis including a YS-specific accelerated route to macrophage production, a source of nutritional/metabolic support and a regulator of oxygen-carrying capacity. We reconstruct the emergence of primitive haematopoietic stem and progenitor cells from YS hemogenic endothelium and their decline upon stromal support modulation as intraembryonic organs specialise to assume these functions. The YS therefore functions as \\u2018three organs in one\\u2019 revealing a multifaceted relay of vital organismal functions as pregnancy proceeds. One Sentence Summary Human yolk sac is a key staging post in a relay of vital organismal functions during human pregnancy.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.03.502475\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4385073959\", \"doi_x\": \"10.1007/s10974-023-09653-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Zebrafish as a model for cardiac disease; Cryo-EM structure of native cardiac thin filaments from Danio Rerio\", \"abstract\": \"Actin, tropomyosin and troponin, the proteins that comprise the contractile apparatus of the cardiac thin filament, are highly conserved across species. We have used cryo-EM to study the three-dimensional structure of the zebrafish cardiac thin and actin filaments. With 70% of human genes having an obvious zebrafish orthologue, and conservation of 85% of disease-causing genes, zebrafish are a good animal model for the study of human disease. Our structure of the zebrafish thin filament reveals the molecular interactions between the constituent proteins, showing that the fundamental organisation of the complex is the same as that reported in the human reconstituted thin filament. A reconstruction of zebrafish cardiac F-actin demonstrates no deviations from human cardiac actin over an extended length of 14 actin subunits. Modelling zebrafish homology models into our maps enabled us to compare, in detail, the similarity with human models. The structural similarities of troponin-T in particular, a region known to contain a hypertrophic cardiomyopathy 'hotspot', confirm the suitability of zebrafish to study these disease-causing mutations.\", \"doi_y\": \"https://doi.org/10.1007/s10974-023-09653-5\", \"publication_date\": \"2023-07-22\"}, {\"work_id\": \"W4380085975\", \"doi_x\": \"10.1038/s41467-023-39140-x\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A 2.2 \\u00c5 cryoEM structure of a quinol-dependent NO Reductase shows close similarity to respiratory oxidases\", \"abstract\": \"Abstract Quinol-dependent nitric oxide reductases (qNORs) are considered members of the respiratory heme-copper oxidase superfamily, are unique to bacteria, and are commonly found in pathogenic bacteria where they play a role in combating the host immune response. qNORs are also essential enzymes in the denitrification pathway, catalysing the reduction of nitric oxide to nitrous oxide. Here, we determine a 2.2 \\u00c5 cryoEM structure of qNOR from Alcaligenes xylosoxidans , an opportunistic pathogen and a denitrifying bacterium of importance in the nitrogen cycle. This high-resolution structure provides insight into electron, substrate, and proton pathways, and provides evidence that the quinol binding site not only contains the conserved His and Asp residues but also possesses a critical Arg (Arg720) observed in cytochrome bo 3 , a respiratory quinol oxidase.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-39140-x\", \"publication_date\": \"2023-06-09\"}, {\"work_id\": \"W4311858108\", \"doi_x\": \"10.3390/toxins14120863\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Crystal Structure of Bacillus thuringiensis Tpp80Aa1 and Its Interaction with Galactose-Containing Glycolipids\", \"abstract\": \"Tpp80Aa1 from Bacillus thuringiensis is a Toxin_10 family protein (Tpp) with reported action against Culex mosquitoes. Here, we demonstrate an expanded target range, showing Tpp80Aa1 is also active against the larvae of Anopheles gambiae and Aedes aegypti mosquitoes. We report the first crystal structure of Tpp80Aa1 at a resolution of 1.8 \\u00c5, which shows Tpp80Aa1 consists of two domains: an N-terminal \\u03b2-trefoil domain resembling a ricin B lectin and a C-terminal putative pore-forming domain sharing structural similarity with the aerolysin family. Similar to other Tpp family members, we observe Tpp80Aa1 binds to the mosquito midgut, specifically the posterior midgut and the gastric caecum. We also identify that Tpp80Aa1 can interact with galactose-containing glycolipids and galactose, and this interaction is critical for exerting full insecticidal action against mosquito target cell lines.\", \"doi_y\": \"https://doi.org/10.3390/toxins14120863\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4296693414\", \"doi_x\": \"10.1038/s41586-022-05335-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A designed photoenzyme for enantioselective [2+2] cycloadditions\", \"abstract\": \"The ability to program new modes of catalysis into proteins would allow the development of enzyme families with functions beyond those found in nature. To this end, genetic code expansion methodology holds particular promise, as it allows the site-selective introduction of new functional elements into proteins as noncanonical amino acid side chains1-4. Here we exploit an expanded genetic code to develop a photoenzyme that operates by means of triplet energy transfer (EnT) catalysis, a versatile mode of reactivity in organic synthesis that is not accessible to biocatalysis at present5-12. Installation of a genetically encoded photosensitizer into the beta-propeller scaffold of DA_20_00 (ref. 13) converts a de novo Diels-Alderase into a photoenzyme for [2+2] cycloadditions (EnT1.0). Subsequent development and implementation of a platform for photoenzyme evolution afforded an efficient and enantioselective enzyme (EnT1.3, up to 99% enantiomeric excess (e.e.)) that can promote intramolecular and bimolecular cycloadditions, including transformations that have proved challenging to achieve selectively with small-molecule catalysts. EnT1.3 performs >300 turnovers and, in contrast to small-molecule photocatalysts, can operate effectively under aerobic conditions and at ambient temperatures. An X-ray crystal structure of an EnT1.3-product complex shows how multiple functional components work in synergy to promote efficient and selective photocatalysis. This study opens up a wealth of new excited-state chemistry in protein active sites and establishes the framework for developing a new generation of enantioselective photocatalysts.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-05335-3\", \"publication_date\": \"2022-09-21\"}, {\"work_id\": \"W4310642534\", \"doi_x\": \"10.1021/jacs.2c06233\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Effects of Dynamical Degrees of Freedom on Magnetic Compass Sensitivity: A Comparison of Plant and Avian Cryptochromes\", \"abstract\": \"The magnetic compass of migratory birds is thought to rely on a radical pair reaction inside the blue-light photoreceptor protein cryptochrome. The sensitivity of such a sensor to weak external magnetic fields is determined by a variety of magnetic interactions, including electron-nuclear hyperfine interactions. Here, we investigate the implications of thermal motion, focusing on fluctuations in the dihedral and librational angles of flavin adenine dinucleotide (FAD) and tryptophan (Trp) radicals in cryptochrome 4a from European robin (Erithacus rubecula, ErCry4a) and pigeon (Columba livia, ClCry4a) and cryptochrome 1 from the plant Arabidopsis thaliana (AtCry1). Molecular dynamics simulations and density functional theory-derived hyperfine interactions are used to calculate the quantum yield of radical pair recombination dependent on the direction of the geomagnetic field. This quantity and various dynamical parameters are compared for [FAD\\u2022- Trp\\u2022+] in ErCry4a, ClCry4a, and AtCry1, with TrpC or TrpD being the third and fourth components of the tryptophan triad/tetrad in the respective proteins. We find that (i) differences in the average dihedral angles in the radical pairs are small, (ii) the librational motions of TrpC\\u2022+ in the avian cryptochromes are appreciably smaller than in AtCry1, (iii) the rapid vibrational motions of the radicals leading to strong fluctuations in the hyperfine couplings affect the spin dynamics depending on the usage of instantaneous or time-averaged interactions. Future investigations of radical pair compass sensitivity should therefore not be based on single snapshots of the protein structure but should include the ensemble properties of the hyperfine interactions.\", \"doi_y\": \"https://doi.org/10.1021/jacs.2c06233\", \"publication_date\": \"2022-12-02\"}, {\"work_id\": \"W4380576774\", \"doi_x\": \"10.1042/cs20220061\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Circadian regulation of pulmonary disease: the importance of timing\", \"abstract\": \"Circadian regulation causes the activity of biological processes to vary over a 24-h cycle. The pathological effects of this variation are predominantly studied using two different approaches: pre-clinical models or observational clinical studies. Both these approaches have provided useful insights into how underlying circadian mechanisms operate and specifically which are regulated by the molecular oscillator, a key time-keeping mechanism in the body. This review compares and contrasts findings from these two approaches in the context of four common respiratory diseases (asthma, chronic obstructive pulmonary disease, pulmonary fibrosis, and respiratory infection). Potential methods used to identify and measure human circadian oscillations are also discussed as these will be useful outcome measures in future interventional human trials that target circadian mechanisms.\", \"doi_y\": \"https://doi.org/10.1042/cs20220061\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4385607238\", \"doi_x\": \"10.1016/j.jsb.2023.108009\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structure determination and analysis of titin A-band fibronectin type III domains provides insights for disease-linked variants and protein oligomerisation\", \"abstract\": \"Titin is the largest protein found in nature and spans half a sarcomere in vertebrate striated muscle. The protein has multiple functions, including in the organisation of the thick filament and acting as a molecular spring during the muscle contraction cycle. Missense variants in titin have been linked to both cardiac and skeletal myopathies. Titin is primarily composed of tandem repeats of immunoglobulin and fibronectin type III (Fn3) domains in a variety of repeat patterns; however, the vast majority of these domains have not had their high-resolution structure determined experimentally. Here, we present the crystal structures of seven wild type titin Fn3 domains and two harbouring rare missense variants reported in hypertrophic cardiomyopathy (HCM) patients. All domains present the typical Fn3 fold, with the domains harbouring variants reported in HCM patients retaining the wild-type conformation. The effect on domain folding and stability were assessed for five rare missense variants found in HCM patients: four caused thermal destabilization of between 7 and 13 \\u00b0C and one prevented the folding of its domain. The structures also allowed us to locate the positions of residues whose mutations have been linked to congenital myopathies and rationalise how they convey their deleterious effects. We find no evidence of physiological homodimer formation, excluding one hypothesised mechanism as to how titin variants could exert pathological effects.\", \"doi_y\": \"https://doi.org/10.1016/j.jsb.2023.108009\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4316469669\", \"doi_x\": \"10.1093/hmg/ddad007\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Single-cell RNA sequencing reveals transcriptional changes of human choroidal and retinal pigment epithelium cells during fetal development, in healthy adult and intermediate age-related macular degeneration\", \"abstract\": \"Abstract Age-related macular degeneration (AMD) is the most prevalent cause of blindness in the developed world. Vision loss in the advanced stages of the disease is caused by atrophy of retinal photoreceptors, overlying retinal pigment epithelium (RPE) and choroidal endothelial cells. The molecular events that underline the development of these cell types from in utero to adult as well as the progression to intermediate and advanced stages AMD are not yet fully understood. We performed single-cell RNA-sequencing (RNA-Seq) of human fetal and adult RPE\\u2013choroidal tissues, profiling in detail all the cell types and elucidating cell type-specific proliferation, differentiation and immunomodulation events that occur up to midgestation. Our data demonstrate that progression from the fetal to adult state is characterized by an increase in expression of genes involved in the oxidative stress response and detoxification from heavy metals, suggesting a better defence against oxidative stress in the adult RPE\\u2013choroid tissue. Single-cell comparative transcriptional analysis between a patient with intermediate AMD and an unaffected subject revealed a reduction in the number of RPE cells and melanocytes in the macular region of the AMD patient. Together these findings may suggest a macular loss of RPE cells and melanocytes in the AMD patients, but given the complex processing of tissues required for single-cell RNA-Seq that is prone to technical artefacts, these findings need to be validated by additional techniques in a larger number of AMD patients and controls.\", \"doi_y\": \"https://doi.org/10.1093/hmg/ddad007\", \"publication_date\": \"2023-01-16\"}, {\"work_id\": \"W4229026778\", \"doi_x\": \"10.3390/cells11091568\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Nutrient-Response Pathways in Healthspan and Lifespan Regulation\", \"abstract\": \"Cellular, small invertebrate and vertebrate models are a driving force in biogerontology studies. Using various models, such as yeasts, appropriate tissue culture cells, Drosophila, the nematode Caenorhabditis elegans and the mouse, has tremendously increased our knowledge around the relationship between diet, nutrient-response signaling pathways and lifespan regulation. In recent years, combinatorial drug treatments combined with mutagenesis, high-throughput screens, as well as multi-omics approaches, have provided unprecedented insights in cellular metabolism, development, differentiation, and aging. Scientists are, therefore, moving towards characterizing the fine architecture and cross-talks of growth and stress pathways towards identifying possible interventions that could lead to healthy aging and the amelioration of age-related diseases in humans. In this short review, we briefly examine recently uncovered knowledge around nutrient-response pathways, such as the Insulin Growth Factor (IGF) and the mechanistic Target of Rapamycin signaling pathways, as well as specific GWAS and some EWAS studies on lifespan and age-related disease that have enhanced our current understanding within the aging and biogerontology fields. We discuss what is learned from the rich and diverse generated data, as well as challenges and next frontiers in these scientific disciplines.\", \"doi_y\": \"https://doi.org/10.3390/cells11091568\", \"publication_date\": \"2022-05-06\"}, {\"work_id\": \"W4307230602\", \"doi_x\": \"10.1002/pro.4482\", \"af_direct_citation\": 1, \"af_indirect_citations\": 7, \"title\": \"<scp>RCSB</scp> Protein Data bank: Tools for visualizing and understanding biological macromolecules in <scp>3D</scp>\", \"abstract\": \"Now in its 52nd year of continuous operations, the Protein Data Bank (PDB) is the premiere open-access global archive housing three-dimensional (3D) biomolecular structure data. It is jointly managed by the Worldwide Protein Data Bank (wwPDB) partnership. The Research Collaboratory for Structural Bioinformatics Protein Data Bank (RCSB PDB) is funded by the National Science Foundation, National Institutes of Health, and US Department of Energy and serves as the US data center for the wwPDB. RCSB PDB is also responsible for the security of PDB data in its role as wwPDB-designated Archive Keeper. Every year, RCSB PDB serves tens of thousands of depositors of 3D macromolecular structure data (coming from macromolecular crystallography, nuclear magnetic resonance spectroscopy, electron microscopy, and micro-electron diffraction). The RCSB PDB research-focused web portal (RCSB.org) makes PDB data available at no charge and without usage restrictions to many millions of PDB data consumers around the world. The RCSB PDB training, outreach, and education web portal (PDB101.RCSB.org) serves nearly 700 K educators, students, and members of the public worldwide. This invited Tools Issue contribution describes how RCSB PDB (i) is organized; (ii) works with wwPDB partners to process new depositions; (iii) serves as the wwPDB-designated Archive Keeper; (iv) enables exploration and 3D visualization of PDB data via RCSB.org; and (v) supports training, outreach, and education via PDB101.RCSB.org. New tools and features at RCSB.org are presented using examples drawn from high-resolution structural studies of proteins relevant to treatment of human cancers by targeting immune checkpoints.\", \"doi_y\": \"https://doi.org/10.1002/pro.4482\", \"publication_date\": \"2022-11-16\"}, {\"work_id\": \"W4311287562\", \"doi_x\": \"10.1093/gigascience/giac118\", \"af_direct_citation\": 1, \"af_indirect_citations\": 10, \"title\": \"3D-Beacons: decreasing the gap between protein sequences and structures through a federated network of protein structure data resources\", \"abstract\": \"While scientists can often infer the biological function of proteins from their 3-dimensional quaternary structures, the gap between the number of known protein sequences and their experimentally determined structures keeps increasing. A potential solution to this problem is presented by ever more sophisticated computational protein modeling approaches. While often powerful on their own, most methods have strengths and weaknesses. Therefore, it benefits researchers to examine models from various model providers and perform comparative analysis to identify what models can best address their specific use cases. To make data from a large array of model providers more easily accessible to the broader scientific community, we established 3D-Beacons, a collaborative initiative to create a federated network with unified data access mechanisms. The 3D-Beacons Network allows researchers to collate coordinate files and metadata for experimentally determined and theoretical protein models from state-of-the-art and specialist model providers and also from the Protein Data Bank.\", \"doi_y\": \"https://doi.org/10.1093/gigascience/giac118\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4313279513\", \"doi_x\": \"10.1021/acs.jproteome.2c00406\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrated View of Baseline Protein Expression in Human Tissues\", \"abstract\": \"The availability of proteomics datasets in the public domain, and in the PRIDE database, in particular, has increased dramatically in recent years. This unprecedented large-scale availability of data provides an opportunity for combined analyses of datasets to get organism-wide protein abundance data in a consistent manner. We have reanalyzed 24 public proteomics datasets from healthy human individuals to assess baseline protein abundance in 31 organs. We defined tissue as a distinct functional or structural region within an organ. Overall, the aggregated dataset contains 67 healthy tissues, corresponding to 3,119 mass spectrometry runs covering 498 samples from 489 individuals. We compared protein abundances between different organs and studied the distribution of proteins across these organs. We also compared the results with data generated in analogous studies. Additionally, we performed gene ontology and pathway-enrichment analyses to identify organ-specific enriched biological processes and pathways. As a key point, we have integrated the protein abundance results into the resource Expression Atlas, where they can be accessed and visualized either individually or together with gene expression data coming from transcriptomics datasets. We believe this is a good mechanism to make proteomics data more accessible for life scientists.\", \"doi_y\": \"https://doi.org/10.1021/acs.jproteome.2c00406\", \"publication_date\": \"2022-12-28\"}, {\"work_id\": \"W4322493187\", \"doi_x\": \"10.7554/elife.79408\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Coarsening dynamics can explain meiotic crossover patterning in both the presence and absence of the synaptonemal complex\", \"abstract\": \"The shuffling of genetic material facilitated by meiotic crossovers is a critical driver of genetic variation. Therefore, the number and positions of crossover events must be carefully controlled. In Arabidopsis, an obligate crossover and repression of nearby crossovers on each chromosome pair are abolished in mutants that lack the synaptonemal complex (SC), a conserved protein scaffold. We use mathematical modelling and quantitative super-resolution microscopy to explore and mechanistically explain meiotic crossover pattering in Arabidopsis lines with full, incomplete, or abolished synapsis. For zyp1 mutants, which lack an SC, we develop a coarsening model in which crossover precursors globally compete for a limited pool of the pro-crossover factor HEI10, with dynamic HEI10 exchange mediated through the nucleoplasm. We demonstrate that this model is capable of quantitatively reproducing and predicting zyp1 experimental crossover patterning and HEI10 foci intensity data. Additionally, we find that a model combining both SC- and nucleoplasm-mediated coarsening can explain crossover patterning in wild-type Arabidopsis and in pch2 mutants, which display partial synapsis. Together, our results reveal that regulation of crossover patterning in wild-type Arabidopsis and SC-defective mutants likely acts through the same underlying coarsening mechanism, differing only in the spatial compartments through which the pro-crossover factor diffuses.\", \"doi_y\": \"https://doi.org/10.7554/elife.79408\", \"publication_date\": \"2023-02-27\"}, {\"work_id\": \"W4312196304\", \"doi_x\": \"10.3390/cells11244120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Nuclear Shape-Shifters: Lipid and Protein Dynamics at the Nuclear Envelope\", \"abstract\": \"The nuclear envelope constitutes a selective barrier that segregates chromatin into the nucleus of eukaryotic cells [...].\", \"doi_y\": \"https://doi.org/10.3390/cells11244120\", \"publication_date\": \"2022-12-19\"}, {\"work_id\": \"W4380686120\", \"doi_x\": \"10.1371/journal.pbio.3002121\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NANOG is required to establish the competence for germ-layer differentiation in the basal tetrapod axolotl\", \"abstract\": \"Pluripotency defines the unlimited potential of individual cells of vertebrate embryos, from which all adult somatic cells and germ cells are derived. Understanding how the programming of pluripotency evolved has been obscured in part by a lack of data from lower vertebrates; in model systems such as frogs and zebrafish, the function of the pluripotency genes NANOG and POU5F1 have diverged. Here, we investigated how the axolotl ortholog of NANOG programs pluripotency during development. Axolotl NANOG is absolutely required for gastrulation and germ-layer commitment. We show that in axolotl primitive ectoderm (animal caps; ACs) NANOG and NODAL activity, as well as the epigenetic modifying enzyme DPY30, are required for the mass deposition of H3K4me3 in pluripotent chromatin. We also demonstrate that all 3 protein activities are required for ACs to establish the competency to differentiate toward mesoderm. Our results suggest the ancient function of NANOG may be establishing the competence for lineage differentiation in early cells. These observations provide insights into embryonic development in the tetrapod ancestor from which terrestrial vertebrates evolved.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3002121\", \"publication_date\": \"2023-06-14\"}, {\"work_id\": \"W4319081171\", \"doi_x\": \"10.1101/2023.02.02.526626\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"abstract\": \"Abstract Mitochondrial ATP synthases form rows of dimers, which induce membrane curvature to give cristae their characteristic lamellar or tubular morphology. The angle formed between the central stalks of ATP synthase dimers varies between species. Using cryo-electron tomography and sub-tomogram averaging, we determined the structure of the ATP synthase dimer from the nematode worm C. elegans and show that the dimer angle differs from previously determined structures. The consequences of this species-specific difference at the dimer interface were investigated by comparing C. elegans and S. cerevisiae mitochondrial morphology. We reveal that C. elegans has a larger ATP synthase dimer angle with more lamellar (flatter) cristae when compared to yeast. The underlying cause of this difference was investigated by generating an atomic model of the C. elegans ATP synthase dimer by homology modelling. A comparison of our C. elegans model to an existing S. cerevisiae structure reveals the presence of extensions and rearrangements in C. elegans subunits associated with maintaining the dimer interface. We speculate that increasing dimer angles could provide an advantage for species that inhabit variable-oxygen environments by forming flatter more energetically efficient cristae.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.02.526626\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4362521293\", \"doi_x\": \"10.26508/lsa.202301954\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>arfA</i>antisense RNA regulates MscL excretory activity\", \"abstract\": \"Excretion of cytoplasmic protein (ECP) is a commonly observed phenomenon in bacteria, and this partial extracellular localisation of the intracellular proteome has been implicated in a variety of stress response mechanisms. In response to hypoosmotic shock and ribosome stalling in Escherichia coli, ECP is dependent upon the presence of the large-conductance mechanosensitive channel and the alternative ribosome-rescue factor A gene products. However, it is not known if a mechanistic link exists between the corresponding genes and the respective stress response pathways. Here, we report that the corresponding mscL and arfA genes are commonly co-located on the genomes of Gammaproteobacteria and display overlap in their respective 3' UTR and 3' CDS. We show this unusual genomic arrangement permits an antisense RNA-mediated regulatory control between mscL and arfA, and this modulates MscL excretory activity in E. coli These findings highlight a mechanistic link between osmotic, translational stress responses and ECP in E. coli, further elucidating the previously unknown regulatory function of arfA sRNA.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202301954\", \"publication_date\": \"2023-04-03\"}, {\"work_id\": \"W4210493562\", \"doi_x\": \"10.1038/s41586-022-04460-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"T cell responses to SARS-CoV-2 spike cross-recognize Omicron\", \"abstract\": \"The SARS-CoV-2 Omicron variant (B.1.1.529) has multiple spike protein mutations1,2 that contribute to viral escape from antibody neutralization3-6 and reduce vaccine protection from infection7,8. The extent to which other components of the adaptive response such as T cells may still target Omicron and contribute to protection from severe outcomes is unknown. Here we assessed the ability of T cells to react to Omicron spike protein in participants who were vaccinated with Ad26.CoV2.S or BNT162b2, or unvaccinated convalescent COVID-19 patients (n = 70). Between 70% and 80% of the CD4+ and CD8+ T cell response to spike was maintained across study groups. Moreover, the magnitude of Omicron cross-reactive T cells was similar for Beta (B.1.351) and Delta (B.1.617.2) variants, despite Omicron harbouring considerably more mutations. In patients who were hospitalized with Omicron infections (n = 19), there were comparable T cell responses to ancestral spike, nucleocapsid and membrane proteins to those in patients hospitalized in previous waves dominated by the ancestral, Beta or Delta variants (n = 49). Thus, despite extensive mutations and reduced susceptibility to neutralizing antibodies of Omicron, the majority of T cell responses induced by vaccination or infection cross-recognize the variant. It remains to be determined whether well-preserved T cell immunity to Omicron contributes to protection from severe COVID-19 and is linked to early clinical observations from South Africa and elsewhere9-12.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-04460-3\", \"publication_date\": \"2022-01-31\"}, {\"work_id\": \"W4293089785\", \"doi_x\": \"10.1107/s2059798322007732\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>ModelCraft</i>: an advanced automated model-building pipeline using <i>Buccaneer</i>\", \"abstract\": \"Interactive model building can be a difficult and time-consuming step in the structure-solution process. Automated model-building programs such as Buccaneer often make it quicker and easier by completing most of the model in advance. However, they may fail to do so with low-resolution data or a poor initial model or map. The Buccaneer pipeline is a relatively simple program that iterates Buccaneer with REFMAC to refine the model and update the map. A new pipeline called ModelCraft has been developed that expands on this to include shift-field refinement, machine-learned pruning of incorrect residues, classical density modification, addition of water and dummy atoms, building of nucleic acids and final rebuilding of side chains. Testing was performed on 1180 structures solved by experimental phasing, 1338 structures solved by molecular replacement using homologues and 2030 structures solved by molecular replacement using predicted AlphaFold models. Compared with the previous Buccaneer pipeline, ModelCraft increased the mean completeness of the protein models in the experimental phasing cases from 91% to 95%, the molecular-replacement cases from 50% to 78% and the AlphaFold cases from 82% to 91%.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322007732\", \"publication_date\": \"2022-08-25\"}, {\"work_id\": \"W4317434079\", \"doi_x\": \"10.1083/jcb.202301035\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Mammalian fertilization: Does sperm IZUMO1 mediate fusion as well as adhesion?\", \"abstract\": \"The molecular mechanism of sperm-egg fusion is a long-standing mystery in reproduction. Brukman and colleagues (2022. J. Cell Biol.https://doi.org/10.1083/jcb.202207147) now provide evidence that the sperm surface protein IZUMO1, which is essential for mammalian fertilization, can induce membrane fusion in cultured cells.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202301035\", \"publication_date\": \"2023-01-19\"}, {\"work_id\": \"W4220882361\", \"doi_x\": \"10.1111/1751-7915.14034\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The fluorescent protein iLOV as a reporter for screening of high\\u2010yield production of antimicrobial peptides in <i>Pichia pastoris</i>\", \"abstract\": \"The methylotrophic yeast Pichia pastoris is commonly used for the production of recombinant proteins at scale. The identification of an optimally overexpressing strain following transformation can be time and reagent consuming. Fluorescent reporters like GFP have been used to assist identification of superior producers, but their relatively big size, maturation requirements and narrow temperature range restrict their applications. Here, we introduce the use of iLOV, a flavin-based fluorescent protein, as a fluorescent marker to identify P. pastoris high-yielding strains easily and rapidly. The use of this fluorescent protein as a fusion partner is exemplified by the production of the antimicrobial peptide NI01, a difficult target to overexpress in its native form. iLOV fluorescence correlated well with protein expression level and copy number of the chromosomally integrated gene. An easy and simple medium-throughput plate-based screen directly following transformation is demonstrated for low complexity screening, while a high-throughput method using fluorescence-activated cell sorting (FACS) allowed for comprehensive library screening. Both codon optimization of the iLOV_NI01 fusion cassettes and different integration strategies into the P. pastoris genome were tested to produce and isolate a high-yielding strain. Checking the genetic stability, process reproducibility and following the purification of the active native peptide are eased by visualization of and efficient cleavage from the iLOV reporter. We show that this system can be used for expression and screening of several different antimicrobial peptides recombinantly produced in P. pastoris.\", \"doi_y\": \"https://doi.org/10.1111/1751-7915.14034\", \"publication_date\": \"2022-03-21\"}, {\"work_id\": \"W4382046174\", \"doi_x\": \"10.1016/j.tim.2023.06.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural diversity and modularity of photosynthetic RC\\u2212LH1 complexes\", \"abstract\": \"Bacterial photosynthesis is essential for sustaining life on Earth as it aids in carbon assimilation, atmospheric composition, and ecosystem maintenance. Many bacteria utilize anoxygenic photosynthesis to convert sunlight into chemical energy while producing organic matter. The core machinery of anoxygenic photosynthesis performed by purple photosynthetic bacteria and Chloroflexales is the reaction center\\u2212light-harvesting 1 (RC\\u2212LH1) pigment\\u2013protein supercomplex. In this review, we discuss recent structural studies of RC\\u2212LH1 core complexes based on the advancement in structural biology techniques. These studies have provided fundamental insights into the assembly mechanisms, structural variations, and modularity of RC\\u2212LH1 complexes across different bacterial species, highlighting their functional adaptability. Understanding the natural architectures of RC\\u2212LH1 complexes will facilitate the design and engineering of artificial photosynthetic systems, which can enhance photosynthetic efficiency and potentially find applications in sustainable energy production and carbon capture.\", \"doi_y\": \"https://doi.org/10.1016/j.tim.2023.06.002\", \"publication_date\": \"2024-01-01\"}, {\"work_id\": \"W4287510390\", \"doi_x\": \"10.1038/s41467-022-32004-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"abstract\": \"Carboxysomes are a family of bacterial microcompartments in cyanobacteria and chemoautotrophs. They encapsulate Ribulose 1,5-bisphosphate carboxylase/oxygenase (Rubisco) and carbonic anhydrase catalyzing carbon fixation inside a proteinaceous shell. How Rubisco complexes pack within the carboxysomes is unknown. Using cryo-electron tomography, we determine the distinct 3D organization of Rubisco inside two distant \\u03b1-carboxysomes from a marine \\u03b1-cyanobacterium Cyanobium sp. PCC 7001 where Rubiscos are organized in three concentric layers, and from a chemoautotrophic bacterium Halothiobacillus neapolitanus where they form intertwining spirals. We further resolve the structures of native Rubisco as well as its higher-order assembly at near-atomic resolutions by subtomogram averaging. The structures surprisingly reveal that the authentic intrinsically disordered linker protein CsoS2 interacts with Rubiscos in native carboxysomes but functions distinctively in the two \\u03b1-carboxysomes. In contrast to the uniform Rubisco-CsoS2 association in the Cyanobium \\u03b1-carboxysome, CsoS2 binds only to the Rubiscos close to the shell in the Halo \\u03b1-carboxysome. Our findings provide critical knowledge of the assembly principles of \\u03b1-carboxysomes, which may aid in the rational design and repurposing of carboxysome structures for new functions.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-32004-w\", \"publication_date\": \"2022-07-25\"}, {\"work_id\": \"W4366819790\", \"doi_x\": \"10.1111/tra.12885\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Out of the <scp>ESCPE</scp> room: Emerging roles of endosomal <scp>SNX\\u2010BARs</scp> in receptor transport and host\\u2013pathogen interaction\", \"abstract\": \"Abstract Several functions of the human cell, such as sensing nutrients, cell movement and interaction with the surrounding environment, depend on a myriad of transmembrane proteins and their associated proteins and lipids (collectively termed \\u201ccargoes\\u201d). To successfully perform their tasks, cargo must be sorted and delivered to the right place, at the right time, and in the right amount. To achieve this, eukaryotic cells have evolved a highly organized sorting platform, the endosomal network. Here, a variety of specialized multiprotein complexes sort cargo into itineraries leading to either their degradation or their recycling to various organelles for further rounds of reuse. A key sorting complex is the Endosomal SNX\\u2010BAR Sorting Complex for Promoting Exit (ESCPE\\u20101) that promotes the recycling of an array of cargos to the plasma membrane and/or the trans \\u2010Golgi network. ESCPE\\u20101 recognizes a hydrophobic\\u2010based sorting motif in numerous cargoes and orchestrates their packaging into tubular carriers that pinch off from the endosome and travel to the target organelle. A wide range of pathogens mimic this sorting motif to hijack ESCPE\\u20101 transport to promote their invasion and survival within infected cells. In other instances, ESCPE\\u20101 exerts restrictive functions against pathogens by limiting their replication and infection. In this review, we discuss ESCPE\\u20101 assembly and functions, with a particular focus on recent advances in the understanding of its role in membrane trafficking, cellular homeostasis and host\\u2013pathogen interaction.\", \"doi_y\": \"https://doi.org/10.1111/tra.12885\", \"publication_date\": \"2023-04-23\"}, {\"work_id\": \"W4220702678\", \"doi_x\": \"10.1016/j.jmb.2022.167567\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Venus: Elucidating the Impact of Amino Acid Variants on Protein Function Beyond Structure Destabilisation\", \"abstract\": \"Exploring the functional effect of a non-synonymous coding variant at the protein level requires multiple pieces of information to be interpreted appropriately. This is particularly important when embarking on the study of a potentially pathogenic variant linked to a rare or monogenic disease. Whereas accurate protein stability predictions alone are generally informative, other effects, such as disruption of post-translational modifications or weakened ligand binding, may also contribute to the disease phenotype. Furthermore, consideration of nearby variants that are found in the healthy population may strengthen or refute a given mechanistic hypothesis. Whilst there are several bioinformatics tools available that score a genetic variant in terms of deleteriousness, there is no single tool that assembles multiple effects of a variant on the encoded protein, beyond structural stability, and presents them on the structure for inspection. Venus is a web application which, given a protein substitution, rapidly estimates the predicted effect on protein stability of the variant, flags if the variant affects a post-translational modification site, a predicted linear motif or known annotation, and determines the effect on protein stability of variants which affect nearby residues and have been identified in healthy populations. Venus is built upon Michelanglo and the results can be exported to it, allowing them to be annotated and shared with other researchers. Venus is freely accessible at https://venus.cmd.ox.ac.uk and its source code is openly available at https://github.com/CMD-Oxford/Michelanglo-and-Venus.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2022.167567\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4206153788\", \"doi_x\": \"10.1016/j.cell.2021.12.046\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"abstract\": \"On 24th November 2021, the sequence of a new SARS-CoV-2 viral isolate Omicron-B.1.1.529 was announced, containing far more mutations in Spike (S) than previously reported variants. Neutralization titers of Omicron by sera from vaccinees and convalescent subjects infected with early pandemic Alpha, Beta, Gamma, or Delta are substantially reduced, or the sera failed to neutralize. Titers against Omicron are boosted by third vaccine doses and are high in both vaccinated individuals and those infected by Delta. Mutations in Omicron knock out or substantially reduce neutralization by most of the large panel of potent monoclonal antibodies and antibodies under commercial development. Omicron S has structural changes from earlier viruses and uses mutations that confer tight binding to ACE2 to unleash evolution driven by immune escape. This leads to a large number of mutations in the ACE2 binding site and rebalances receptor affinity to that of earlier pandemic viruses.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2021.12.046\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4385666772\", \"doi_x\": \"10.1016/j.devcel.2023.07.012\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"S-acylation of the Wnt receptor Frizzled-5 by zDHHC5 controls its cellular localization and synaptogenic activity in the rodent hippocampus\", \"abstract\": \"Proper localization of receptors for synaptic organizing factors is crucial for synapse formation. Wnt proteins promote synapse assembly through Frizzled (Fz) receptors. In hippocampal neurons, the surface and synaptic localization of Fz5 is regulated by neuronal activity, but the mechanisms involved remain poorly understood. Here, we report that all Fz receptors can be post-translationally modified by S-acylation and that Fz5 is S-acylated on three C-terminal cysteines by zDHHC5. S-acylation is essential for Fz5 localization to the cell surface, axons, and presynaptic sites. Notably, S-acylation-deficient Fz5 is internalized faster, affecting its association with signalosome components at the cell surface. S-acylation-deficient Fz5 also fails to activate canonical and divergent canonical Wnt pathways. Fz5 S-acylation levels are regulated by the pattern of neuronal activity. In vivo studies demonstrate that S-acylation-deficient Fz5 expression fails to induce presynaptic assembly. Our studies show that S-acylation of Frizzled receptors is a mechanism controlling their localization and function.\", \"doi_y\": \"https://doi.org/10.1016/j.devcel.2023.07.012\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4314935533\", \"doi_x\": \"10.1371/journal.pone.0271016\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Using canavanine resistance to measure mutation rates in Schizosaccharomyces pombe\", \"abstract\": \"We constructed a panel of S . pombe strains expressing DNA polymerase \\u03b5 variants associated with cancer, specifically POLES297F, POLEV411L, POLEL424V, POLES459F, and used these to compare mutation rates determined by canavanine resistance with other selective methods. Canavanine-resistance mutation rates are broadly similar to those seen with reversion of the ade-485 mutation to adenine prototrophy, but lower than 5-fluoroorotic acid (FOA)-resistance rates (inactivation of ura4 + or ura5 + genes). Inactivation of several genes has been associated with canavanine resistance in S . pombe but surprisingly whole genome sequencing showed that 8/8 spontaneous canavanine-resistant mutants have an R175C mutation in the any1/arn1 gene. This gene encodes an \\u03b1-arrestin-like protein involved in mediating Pub1 ubiquitylation of target proteins, and the phenotypic resistance to canavanine by this single mutation is similar to that shown by the original \\u201c can1-1 \\u201d strain, which also has the any1R175C mutation. Some of the spontaneous mutants have additional mutations in arginine transporters, suggesting that this may marginally increase resistance to canavanine. The any1R175C strain showed internalisation of the Cat1 arginine transporter as previously reported, explaining the canavanine-resistance phenotype.\", \"doi_y\": \"https://doi.org/10.1371/journal.pone.0271016\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W4315797308\", \"doi_x\": \"10.3389/fmolb.2022.1074714\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"AlphaFold predicted structure of the Hsp90-like domains of the neurodegeneration linked protein sacsin reveals key residues for ATPase activity\", \"abstract\": \"The ataxia-linked protein sacsin has three regions of partial homology to Hsp90's N-terminal ATP binding domain. Although a crystal structure for this Hsp90-like domain has been reported the precise molecular interactions required for ATP-binding and hydrolysis are unclear and it is debatable whether ATP biding is compatible with these domains. Furthermore, the Identification of a sacsin domain(s) equivalent to the middle domain of Hsp90 has been elusive. Here we present the superimposition of an AlphaFold structure of sacsin with yeast Hsp90, which provides novel insights into sacsin's structure. We identify residues within the sacsin Hsp90-like domains that are required for ATP binding and hydrolysis, including the putative catalytic arginine residues equivalent to that of the Hsp90 middle domain. Importantly, our analysis allows comparison of the Hsp90 middle domain with corresponding sacsin regions and identifies a shorter lid segment, in the sacsin ATP-binding domains, than the one found in the N-terminal domain of Hsp90. Our results show how a realignment of residues in the lid segment of sacsin that are involved in ATP binding can better match equivalent residues seen in Hsp90, which we then corroborated using molecular dynamic simulations. We speculate, from a structural viewpoint, why some ATP competitive inhibitors of Hsp90 may not bind sacsin, while others would. Together our analysis supports the hypothesis that sacsin's function is ATP-driven and would be consistent with it having a role as a super molecular chaperone. We propose that the SR1 regions of sacsin be renamed as HSP-NRD (Hsp90 N-Terminal Repeat Domain; residues 84-324) and the fragment immediately after as HSP-MRD (Hsp90 Middle Repeat Domain; residues 325-518).\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.1074714\", \"publication_date\": \"2023-01-13\"}, {\"work_id\": \"W4281491296\", \"doi_x\": \"10.1038/s41467-022-30608-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"abstract\": \"Bacterial metabolosomes are a family of protein organelles in bacteria. Elucidating how thousands of proteins self-assemble to form functional metabolosomes is essential for understanding their significance in cellular metabolism and pathogenesis. Here we investigate the de novo biogenesis of propanediol-utilization (Pdu) metabolosomes and characterize the roles of the key constituents in generation and intracellular positioning of functional metabolosomes. Our results demonstrate that the Pdu metabolosome undertakes both \\\"Shell first\\\" and \\\"Cargo first\\\" assembly pathways, unlike the \\u03b2-carboxysome structural analog which only involves the \\\"Cargo first\\\" strategy. Shell and cargo assemblies occur independently at the cell poles. The internal cargo core is formed through the ordered assembly of multiple enzyme complexes, and exhibits liquid-like properties within the metabolosome architecture. Our findings provide mechanistic insight into the molecular principles driving bacterial metabolosome assembly and expand our understanding of liquid-like organelle biogenesis.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-30608-w\", \"publication_date\": \"2022-05-25\"}, {\"work_id\": \"W4361002045\", \"doi_x\": \"10.12688/f1000research.122987.2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Improving the predictive power of xenograft and syngeneic anti-tumour studies using mice humanised for pathways of drug metabolism\", \"abstract\": \"<ns4:p>Drug development is an expensive and time-consuming process, with only a small fraction of drugs gaining regulatory approval from the often many thousands of candidates identified during target validation. Once a lead compound has been identified and optimised, they are subject to intensive pre-clinical research to determine their pharmacodynamic, pharmacokinetic and toxicological properties, procedures which inevitably involve significant numbers of animals - mainly mice and rats, but also dogs and monkeys in much smaller numbers and for specific types of drug candidates. Many compounds that emerge from this process, having been shown to be safe and efficacious in pre-clinical studies, subsequently fail to replicate this outcome in clinical trials, therefore wasting time, money and, most importantly, animals.</ns4:p><ns4:p> Due to high rates of metabolism and a differing spectrum of metabolites (some pharmacologically active) in rodents, species differences in drug metabolism can be a major impediment to drug discovery programmes and confound the extrapolation of animal data to humans. To circumvent this, we have developed a complex transgenic mouse model \\u2013 8HUM - which faithfully replicates human Phase I drug metabolism (and its regulation), and which will generate more human-relevant data from fewer animals in a pre-clinical setting and reduce attrition in the clinic.</ns4:p><ns4:p> One key area for the pre-clinical application of animals in an oncology setting \\u2013 almost exclusively mice - is their use in anti-tumour studies. We now further demonstrate the utility of the 8HUM mouse using a murine melanoma cell line as a syngeneic tumour and also present an immunodeficient version 8HUM_Rag2 <ns4:sup>-/-</ns4:sup> - for use in xenograft studies. These models will be of significant benefit not only to Pharma for pre-clinical drug development work, but also throughout the drug efficacy, toxicology, pharmacology, and drug metabolism communities, where fewer animals will be needed to generate more human-relevant data.</ns4:p>\", \"doi_y\": \"https://doi.org/10.12688/f1000research.122987.2\", \"publication_date\": \"2023-03-27\"}, {\"work_id\": \"W4367856209\", \"doi_x\": \"10.1126/sciadv.adg3861\", \"af_direct_citation\": 0, \"af_indirect_citations\": 6, \"title\": \"Resurrection of plant disease resistance proteins via helper NLR bioengineering\", \"abstract\": \"Parasites counteract host immunity by suppressing helper nucleotide binding and leucine-rich repeat (NLR) proteins that function as central nodes in immune receptor networks. Understanding the mechanisms of immunosuppression can lead to strategies for bioengineering disease resistance. Here, we show that a cyst nematode virulence effector binds and inhibits oligomerization of the helper NLR protein NRC2 by physically preventing intramolecular rearrangements required for activation. An amino acid polymorphism at the binding interface between NRC2 and the inhibitor is sufficient for this helper NLR to evade immune suppression, thereby restoring the activity of multiple disease resistance genes. This points to a potential strategy for resurrecting disease resistance in crop genomes.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.adg3861\", \"publication_date\": \"2023-05-03\"}, {\"work_id\": \"W4310874185\", \"doi_x\": \"10.21203/rs.3.rs-2289542/v1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structure-based design of biocatalytic panels for pharmaceuticals synthesis\", \"abstract\": \"Abstract Biocatalysis is emerging as an attractive option for manufacturing pharmaceuticals. However, the identification of enzymes for target transformations of interest requires major screening efforts. Here we report a structure-based computational workflow to prioritize protein sequences by a score based on predicted activities on substrates, thereby reducing resource intensive laboratory-based biocatalyst screening. We selected imine reductases (IREDs) as a class of biocatalysts to illustrate the application of the computational workflow termed IREDFisher. Validation by using published data showed that IREDFisher can retrieve the best enzymes and increase the hit rate by identifying the top 20 ranked sequences. The power of IREDFisher is confirmed by computationally screening 1,400 sequences to identify suitable biocatalysts for five selected reductive amination reactions. IREDFisher is available as a user-friendly web interface that will enable rapid identification of biocatalysts for applications in synthesis and directed evolution studies with minimal time and resource expenditure.\", \"doi_y\": \"https://doi.org/10.21203/rs.3.rs-2289542/v1\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4229452091\", \"doi_x\": \"10.1038/s41587-022-01316-z\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"An open invitation to the Understudied Proteins Initiative\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41587-022-01316-z\", \"publication_date\": \"2022-05-09\"}, {\"work_id\": \"W4225260126\", \"doi_x\": \"10.26508/lsa.202101088\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Refining the domain architecture model of the replication origin firing factor Treslin/TICRR\", \"abstract\": \"Faithful genome duplication requires appropriately controlled replication origin firing. The metazoan origin firing regulation hub Treslin/TICRR and its yeast orthologue Sld3 share the Sld3-Treslin domain and the adjacent TopBP1/Dpb11 interaction domain. We report a revised domain architecture model of Treslin/TICRR. Protein sequence analyses uncovered a conserved Ku70-homologous \\u03b2-barrel fold in the Treslin/TICRR middle domain (M domain) and in Sld3. Thus, the Sld3-homologous Treslin/TICRR core comprises its three central domains, M domain, Sld3-Treslin domain, and TopBP1/Dpb11 interaction domain, flanked by non-conserved terminal domains, the CIT (conserved in Treslins) and the C terminus. The CIT includes a von Willebrand factor type A domain. Unexpectedly, MTBP, Treslin/TICRR, and Ku70/80 share the same N-terminal domain architecture, von Willebrand factor type A and Ku70-like \\u03b2-barrels, suggesting a common ancestry. Binding experiments using mutants and the Sld3\\u2013Sld7 dimer structure suggest that the Treslin/Sld3 and MTBP/Sld7 \\u03b2-barrels engage in homotypic interactions, reminiscent of Ku70-Ku80 dimerization. Cells expressing Treslin/TICRR domain mutants indicate that all Sld3-core domains and the non-conserved terminal domains fulfil important functions during origin firing in human cells. Thus, metazoa-specific and widely conserved molecular processes cooperate during metazoan origin firing.\", \"doi_y\": \"https://doi.org/10.26508/lsa.202101088\", \"publication_date\": \"2022-01-28\"}, {\"work_id\": \"W4220885070\", \"doi_x\": \"10.1038/s41586-022-04534-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structure, substrate recognition and initiation of hyaluronan synthase\", \"abstract\": \"Hyaluronan is an acidic heteropolysaccharide comprising alternating N-acetylglucosamine and glucuronic acid sugars that is ubiquitously expressed in the vertebrate extracellular matrix1. The high-molecular-mass polymer modulates essential physiological processes in health and disease, including cell differentiation, tissue homeostasis and angiogenesis2. Hyaluronan is synthesized by a membrane-embedded processive glycosyltransferase, hyaluronan synthase (HAS), which catalyses the synthesis and membrane translocation of hyaluronan from uridine diphosphate-activated precursors3,4. Here we describe five cryo-electron microscopy structures of a viral HAS homologue at different states during substrate binding and initiation of polymer synthesis. Combined with biochemical analyses and molecular dynamics simulations, our data reveal how HAS selects its substrates, hydrolyses the first substrate to prime the synthesis reaction, opens a hyaluronan-conducting transmembrane channel, ensures alternating substrate polymerization and coordinates hyaluronan inside its transmembrane pore. Our research suggests a detailed model for the formation of an acidic extracellular heteropolysaccharide and provides insights into the biosynthesis of one of the most abundant and essential glycosaminoglycans in the human body.\", \"doi_y\": \"https://doi.org/10.1038/s41586-022-04534-2\", \"publication_date\": \"2022-03-30\"}, {\"work_id\": \"W4385064091\", \"doi_x\": \"10.1371/journal.ppat.1011529\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Thrombin cleavage of the hepatitis E virus polyprotein at multiple conserved locations is required for genome replication\", \"abstract\": \"The genomes of positive-sense RNA viruses encode polyproteins that are essential for mediating viral replication. These viral polyproteins must undergo proteolysis (also termed polyprotein processing) to generate functional protein units. This proteolysis can be performed by virally-encoded proteases as well as host cellular proteases, and is generally believed to be a key step in regulating viral replication. Hepatitis E virus (HEV) is a leading cause of acute viral hepatitis. The positive-sense RNA genome is translated to generate a polyprotein, termed pORF1, which is necessary and sufficient for viral genome replication. However, the mechanism of polyprotein processing in HEV remains to be determined. In this study, we aimed to understand processing of this polyprotein and its role in viral replication using a combination of in vitro translation experiments and HEV sub-genomic replicons. Our data suggest no evidence for a virally-encoded protease or auto-proteolytic activity, as in vitro translation predominantly generates unprocessed viral polyprotein precursors. However, seven cleavage sites within the polyprotein (suggested by bioinformatic analysis) are susceptible to the host cellular protease, thrombin. Using two sub-genomic replicon systems, we demonstrate that mutagenesis of these sites prevents replication, as does pharmacological inhibition of serine proteases including thrombin. Overall, our data supports a model where HEV uses host proteases to support replication and could have evolved to be independent of a virally-encoded protease for polyprotein processing.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011529\", \"publication_date\": \"2023-07-21\"}, {\"work_id\": \"W4316465051\", \"doi_x\": \"10.3390/v15010243\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Omicron-BA.1 Dispersion Rates in Mexico Varied According to the Regional Epidemic Patterns and the Diversity of Local Delta Subvariants\", \"abstract\": \"The Omicron subvariant BA.1 of SARS-CoV-2 was first detected in November 2021 and quickly spread worldwide, displacing the Delta variant. In this work, a characterization of the spread of this variant in Mexico is presented.The time to fixation of BA.1, the diversity of Delta sublineages, the population density, and the level of virus circulation during the inter-wave interval were determined to analyze differences in BA.1 spread.BA.1 began spreading during the first week of December 2021 and became dominant in the next three weeks, causing the fourth COVID-19 epidemiological surge in Mexico. Unlike previous variants, BA.1 did not exhibit a geographically distinct circulation pattern. However, a regional difference in the speed of the replacement of the Delta variant was observed.Viral diversity and the relative abundance of the virus in a particular area around the time of the introduction of a new lineage seem to have influenced the spread dynamics, in addition to population density. Nonetheless, if there is a significant difference in the fitness of the variants, or if the time allowed for the competition is sufficiently long, it seems the fitter virus will eventually become dominant, as observed in the eventual dominance of the BA.1.x variant in Mexico.\", \"doi_y\": \"https://doi.org/10.3390/v15010243\", \"publication_date\": \"2023-01-15\"}, {\"work_id\": \"W4224126867\", \"doi_x\": \"10.1101/2022.04.14.488335\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Parallel recruitment pathways contribute to synaptonemal complex assembly during mammalian meiosis\", \"abstract\": \"Abstract During meiosis, the synaptonemal complex (SC) assembles between paired chromosomes, binding them together in close apposition, and facilitating recombination. SC assembly is thought to occur through the hierarchical zipper-like recruitment of axial elements, followed by transverse filaments and then central elements. However, the rapidity of SC formation in mammals has hitherto hindered investigation of its assembly mechanisms and their relationship with recombination. Using super-resolution imaging of separation-of-function mouse mutants, we show that, contrary to the hierarchical assembly model, central element protein SYCE2 is recruited to recombination sites early in SC assembly, and independently of SYCP1-containing transverse filaments. Further, SYCE2-TEX12 binds DNA in vitro , and SYCE2-containing bridges physically link paired chromosomes at recombination sites prior to transverse filament recruitment and chromosome synapsis. These data suggest that mammals integrate parallel recruitment pathways to assemble a mature SC: one recruiting central element proteins to recombination sites, and another recruiting transverse filaments to chromosomes.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.14.488335\", \"publication_date\": \"2022-04-14\"}, {\"work_id\": \"W4376121194\", \"doi_x\": \"10.1002/anie.202218783\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Darobactin B Stabilises a Lateral\\u2010Closed Conformation of the BAM Complex in <i>E. coli</i> Cells\", \"abstract\": \"Abstract The \\u03b2\\u2010barrel assembly machinery (BAM complex) is essential for outer membrane protein (OMP) folding in Gram\\u2010negative bacteria, and represents a promising antimicrobial target. Several conformational states of BAM have been reported, but all have been obtained under conditions which lack the unique features and complexity of the outer membrane (OM). Here, we use Pulsed Electron\\u2010Electron Double Resonance (PELDOR, or DEER) spectroscopy distance measurements to interrogate the conformational ensemble of the BAM complex in E. coli cells. We show that BAM adopts a broad ensemble of conformations in the OM, while in the presence of the antibiotic darobactin B (DAR\\u2010B), BAM\\u2032s conformational equilibrium shifts to a restricted ensemble consistent with the lateral closed state. Our in\\u2010cell PELDOR findings are supported by new cryoEM structures of BAM in the presence and absence of DAR\\u2010B. This work demonstrates the utility of PELDOR to map conformational changes in BAM within its native cellular environment.\", \"doi_y\": \"https://doi.org/10.1002/anie.202218783\", \"publication_date\": \"2023-05-31\"}, {\"work_id\": \"W4319938166\", \"doi_x\": \"10.1016/j.crstbi.2023.100098\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Structure of the Saccharolobus solfataricus type III-D CRISPR effector\", \"abstract\": \"CRISPR-Cas is a prokaryotic adaptive immune system, classified into six different types, each characterised by a signature protein. Type III systems, classified based on the presence of a Cas10 subunit, are rather diverse multi-subunit assemblies with a range of enzymatic activities and downstream ancillary effectors. The broad array of current biotechnological CRISPR applications is mainly based on proteins classified as Type II, however recent developments established the feasibility and efficacy of multi-protein Type III CRISPR-Cas effector complexes as RNA-targeting tools in eukaryotes. The crenarchaeon Saccharolobus solfataricus has two type III system subtypes (III\\u2013B and III-D). Here, we report the cryo-EM structure of the Csm Type III-D complex from S. solfataricus (SsoCsm), which uses CRISPR RNA to bind target RNA molecules, activating the Cas10 subunit for antiviral defence. The structure reveals the complex organisation, subunit/subunit connectivity and protein/guide RNA interactions of the SsoCsm complex, one of the largest CRISPR effectors known.\", \"doi_y\": \"https://doi.org/10.1016/j.crstbi.2023.100098\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4304142251\", \"doi_x\": \"10.15252/embr.202254856\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cryo\\u2010EM structure of the octameric pore of <i>Clostridium perfringens</i> \\u03b2\\u2010toxin\", \"abstract\": \"Report10 October 2022Open Access Transparent process Cryo-EM structure of the octameric pore of Clostridium perfringens \\u03b2-toxin Julia Bruggisser Julia Bruggisser orcid.org/0000-0001-6449-0115 Institute of Animal Pathology, Vetsuisse-Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, \\u200bInvestigation, Writing - original draft, Writing - review & editing Search for more papers by this author Ioan Iacovache Ioan Iacovache orcid.org/0000-0001-8470-5056 Institute of Anatomy, Medical Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, \\u200bInvestigation, Writing - original draft, Writing - review & editing Search for more papers by this author Samuel C Musson Samuel C Musson orcid.org/0000-0002-2189-554X Department of Physics, Durham University, Durham, UK Contribution: \\u200bInvestigation, Writing - review & editing Search for more papers by this author Matteo T Degiacomi Matteo T Degiacomi orcid.org/0000-0003-4672-471X Department of Physics, Durham University, Durham, UK Contribution: Supervision, \\u200bInvestigation, Writing - review & editing Search for more papers by this author Horst Posthaus Corresponding Author Horst Posthaus [email protected] orcid.org/0000-0002-4579-7493 Institute of Animal Pathology, Vetsuisse-Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, Supervision, Project administration, Writing - review & editing Search for more papers by this author Beno\\u00eet Zuber Corresponding Author Beno\\u00eet Zuber [email protected] orcid.org/0000-0001-7725-5579 Institute of Anatomy, Medical Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, Supervision, Project administration, Writing - review & editing Search for more papers by this author Julia Bruggisser Julia Bruggisser orcid.org/0000-0001-6449-0115 Institute of Animal Pathology, Vetsuisse-Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, \\u200bInvestigation, Writing - original draft, Writing - review & editing Search for more papers by this author Ioan Iacovache Ioan Iacovache orcid.org/0000-0001-8470-5056 Institute of Anatomy, Medical Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, \\u200bInvestigation, Writing - original draft, Writing - review & editing Search for more papers by this author Samuel C Musson Samuel C Musson orcid.org/0000-0002-2189-554X Department of Physics, Durham University, Durham, UK Contribution: \\u200bInvestigation, Writing - review & editing Search for more papers by this author Matteo T Degiacomi Matteo T Degiacomi orcid.org/0000-0003-4672-471X Department of Physics, Durham University, Durham, UK Contribution: Supervision, \\u200bInvestigation, Writing - review & editing Search for more papers by this author Horst Posthaus Corresponding Author Horst Posthaus [email protected] orcid.org/0000-0002-4579-7493 Institute of Animal Pathology, Vetsuisse-Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, Supervision, Project administration, Writing - review & editing Search for more papers by this author Beno\\u00eet Zuber Corresponding Author Beno\\u00eet Zuber [email protected] orcid.org/0000-0001-7725-5579 Institute of Anatomy, Medical Faculty, University of Bern, Bern, Switzerland Contribution: Conceptualization, Supervision, Project administration, Writing - review & editing Search for more papers by this author Author Information Julia Bruggisser1,\\u2020, Ioan Iacovache2,\\u2020, Samuel C Musson3, Matteo T Degiacomi3, Horst Posthaus *,1,\\u2021 and Beno\\u00eet Zuber *,2,\\u2021 1Institute of Animal Pathology, Vetsuisse-Faculty, University of Bern, Bern, Switzerland 2Institute of Anatomy, Medical Faculty, University of Bern, Bern, Switzerland 3Department of Physics, Durham University, Durham, UK \\u2020 These authors contributed equally to this work as first authors. \\u2021 These authors contributed equally to this work as corresponding authors. *Corresponding author. Tel: +41 31 684 23 99; E-mail: [email protected] *Corresponding author. Tel: +41 31 684 84 40; E-mail: [email protected] EMBO Reports (2022)23:e54856https://doi.org/10.15252/embr.202254856 PDFDownload PDF of article text and main figures.PDF PLUSDownload PDF of article text, main figures, expanded view figures and appendix. Peer ReviewDownload a summary of the editorial decision process including editorial decision letters, reviewer comments and author responses to feedback. ToolsAdd to favoritesDownload CitationsTrack CitationsPermissions ShareFacebookTwitterLinked InMendeleyWechatReddit Figures & Info Abstract Clostridium perfringens is one of the most widely distributed and successful pathogens producing an impressive arsenal of toxins. One of the most potent toxins produced is the C. perfringens \\u03b2-toxin (CPB). This toxin is the main virulence factor of type C strains. We describe the cryo-electron microscopy (EM) structure of CPB oligomer. We show that CPB forms homo-octameric pores like the hetero-oligomeric pores of the bi-component leukocidins, with important differences in the receptor binding region and the N-terminal latch domain. Intriguingly, the octameric CPB pore complex contains a second 16-stranded \\u03b2-barrel protrusion atop of the cap domain that is formed by the N-termini of the eight protomers. We propose that CPB, together with the newly identified Epx toxins, is a member a new subclass of the hemolysin-like family. In addition, we show that the \\u03b2-barrel protrusion domain can be modified without affecting the pore-forming ability, thus making the pore particularly attractive for macromolecule sensing and nanotechnology. The cryo-EM structure of the octameric pore of CPB will facilitate future developments in both nanotechnology and basic research. Synopsis Clostridium perfringens \\u03b2-toxin (CPB) is a beta-pore-forming toxin of the hemolysin family and an essential virulence factor of type C strains causing fatal necrotic enteritis in animals and humans. This study reports the cryo-electron microscopy structure of the oligomeric CPB-pore which can facilitate future developments in both nanotechnology and basic research. CPB is the prototype of a novel hemolysin subfamily The N-termini of the eight protomers form a \\u03b2-barrel protrusion atop of the cap domain The membrane binding domain contains two unique flexible loops which are most likely involved in receptor specificity of CPB Introduction One of the most common and evolutionary conserved bacterial virulence mechanisms is the secretion of protein toxins that disrupt cellular membranes by pore formation. Such pore-forming toxins (PFTs) are used by the pathogens to invade, survive, and disseminate in their hosts. Despite their large diversity, bacterial PFTs share common features. Most are secreted as water-soluble monomers and bind to target cells via membrane receptors. Receptor binding leads to an increase in the local concentration, oligomerization, and insertion of a stable pore in the cell membrane. This allows uncontrolled exchanges between the extracellular and intracellular milieus, disturbs cellular homeostasis, and leads to diverse reactions ranging from defense mechanisms to cell death (Dal Peraro & van der Goot, 2016). Because of their nearly universal presence in bacterial pathogens, common structures and mechanisms used by PFTs are promising targets for novel anti-virulence strategies as alternatives or supplementation to increasingly ineffective antibiotic treatments. Moreover, PFTs have gathered much interest in the scientific community beyond bacterial infections. The nano-sized pores that they form are used for \\u201csensing\\u201d biomolecules. Many nanopore applications are based on \\u03b1-hemolysin (Hla), the prototype hemolysin-like \\u03b2-PFT secreted by Staphylococcus aureus (Kasianowicz et al, 1996) as well as aerolysin from Aeromonas hydrophila (Cao et al, 2019). The human and animal pathogen Clostridium perfringens causes many severe diseases such as wound infections, septicemia, food poisoning, enterotoxemia, and enteritis (Songer, 1996, 2010; Kiu & Hall, 2018). The bacterium can produce a large arsenal of exotoxins, in particular PFTs that cross the membrane as a \\u03b2-barrel (\\u03b2-PFT). The largest group among them, with currently 11 known members, are the hemolysin-like \\u03b2-PFTs (Popoff & Bouvet, 2009; Popoff, 2014; Mehdizadeh Gohari et al, 2015; Lacey et al, 2019). Despite their widespread use by the pathogen, their role in diseases is incompletely understood. One of the most potent toxins produced by C. perfringens is the \\u03b2-toxin (CPB; Uzal et al, 2014). CPB is secreted by C. perfringens type C strains and is essential in the pathogenesis of a lethal necrotic enteritis in many animal species and humans (Uzal et al, 2014). CPB contributes to the intestinal damages by targeting endothelial cells and potentially thrombocytes, leading to vascular damage and hemorrhage (Posthaus et al, 2020). Based on sequence homology between mature CPB and other bacterial toxins (Appendix Fig S1 and S2), the toxin is a member of the hemolysin-like family of \\u03b2-PFTs. Within this group, CPB is most closely related to C. perfringens \\u03b4-toxin (46% identity) and NetB (39% identity). In addition, it shows a 41% identity with the newly discovered enterococcal EPX4 toxin (Xiong et al, 2022). It shares lower sequence homology to the staphylococcal toxins Hla (26% identity) and the bi-component leukocidins (Notredame et al, 2000; Robert & Gouet, 2014). Recently, we determined the molecular basis for the specificity of CPB toward endothelial and leukocytic cells by showing that PECAM-1/CD31, an endothelial and leukocytic adhesion molecule, serves as its cellular receptor (Bruggisser et al, 2020; Tarek et al, 2021). So far however, no structural information has been available for CPB. In the present study, we describe the cryo-electron microscopy (EM) structure of CPB in SMA discs, which likely represents the membrane-inserted pore form, at near atomic resolution. We show that CPB forms a homo-oligomeric pore with a novel N-terminal \\u03b2-barrel replacing the typical hemolysin latch domain. We propose that the N-terminal \\u03b2-hairpin stabilizes the monomeric protein in solution and influences the pore conductivity. Our results have important implications for comparative studies on related toxins and the rational design of novel anti-virulence strategies against clostridial diseases. In addition, the unique features of the N-terminal \\u03b2-barrel make CPB an attractive candidate for applications in nanotechnology. Results and Discussion Formation of the CPB oligomeric pore To investigate the pore formed by CPB by single particle cryo-EM, we screened for a suitable detergent for oligomerization. CPB spontaneously assembles into SDS-resistant oligomeric species when stored in solution leading to partial precipitation of the protein. Purification in the presence of cholate followed by detergent removal and exchange with 2.5% SMA (Knowles et al, 2009) led to a homogeneous distribution of pores suitable for single particle cryo-EM (Fig EV1A\\u2013G). Interestingly, the synthetic nanodiscs did not require addition of lipids suggesting that the SMA is able to directly wrap around the \\u03b2-barrel of the CPB pore keeping the complex soluble. Particle distribution, orientation, and sample quality in SMA were better than either in detergent or inserted in protein-based nanodiscs (Fig EV2A\\u2013D). The average diameter of the particles was ~100 \\u00c5. Two-dimensional classification of the particles revealed CPB pores with eight-fold symmetry. Further refinement and postprocessing resulted in an electron density map at an estimated 3.8 \\u00c5 resolution (Figs 1A and C, and EV2E and F). This allowed us to unambiguously build a model for the CPB octameric pore except for two loops in the rim domain (Glu76\\u2013Ser89, Ala283\\u2013Pro287), 4N-terminal amino acids and several amino acids with unresolved side chains (Fig 1B). Figure 1. Structure of the CPB pore Side and top views of the 3.8 \\u00c5 sharpened cryo-EM map of the Clostridium perfringens \\u03b2-toxin (CPB) with one protomer highlighted in purple. The map shows an extended 123 \\u00c5 particle with protrusions on both sides. The diameter of the particles is 97 \\u00c5 with a visible channel of ~ 30 \\u00c5 diameter. Model of the CPB oligomer showing a homo-octameric pore in top view and side view. The dimensions of the two \\u03b2-barrels are indicated as measured from C\\u03b1 to C\\u03b1. One protomer is colored by domains: green\\u2014N-terminal \\u03b2-barrel protrusion; light blue\\u2014\\u03b2-sandwich cap domain; dark blue\\u2014rim domain; red\\u2014stem domain. Magnified view of the aromatic pocket of the rim domain docked in the cryo-EM map showing the density of the aromatic side chains. Residues are indicated in proximity to their respective densities. Cartoon model of the CPB protomer in the oligomer color coded as in (B) compared to a protomer extracted from the Hla oligomer. The N-terminus of Hla is highlighted in purple. Download figure Download PowerPoint Click here to expand this figure. Figure EV1. Oligomerization and solubility of CPB oligomers in different detergents A. Detergent screen showing cholate and deoxycholate as the best candidates for purification and solubilization of Clostridium perfringens \\u03b2-toxin (CPB) pores. Each detergent was added to the CPB sample for 16 h at 4\\u00b0 and then the sample was centrifuged at 15,000 g for 10 min. The pelleted insoluble fraction was recovered in sample buffer and loaded on sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE) together with the soluble fraction and the initial (total) fractions. B. Negative stain of representative CPB oligomers after purification in 30 mM cholate with evenly spread oligomers (top) vs after exchanging cholate with 0.1 mM GDN with aggregates and background particles (bottom). C, D. SDS-PAGE gels of His6-CPB purification in the absence (C) and presence (D) of cholate. 10 \\u03bcl of supernatant (S), flow through (F), and elution (E) fractions were loaded, and gels stained with Coomassie stain. Most oligomers precipitate in the absence of detergent during the purification, whereas after concentration (C) almost all CPB shifted into the oligomeric state in the presence of cholate. E. Table of the detergents used for the screen, their CMC and their concentration. F, G. Characterization of N- and C-terminal tagged CPB constructs by gel electrophoretic analysis under denaturing (F) and native (G) conditions. SDS-PAGE gel analysis of CPB samples (1 \\u03bcg per lane) showing monomeric CPB in the absence of detergent and SDS-resistant oligomers in the presence of cholate. CPB samples containing cholate were boiled for 5 min at 95\\u00b0C (+) or not (\\u2212). Download figure Download PowerPoint Click here to expand this figure. Figure EV2. Cryo-EM of CPB Electron micrograph of a typical field of view of oligomeric Clostridium perfringens \\u03b2-toxin (CPB). Automatically picked particles shown in green. Inset showing characteristic 2D class averages, side, tilted, top. Scale bar is 100 nm. FSC showing the resolution of CPB oligomers in different reconstruction conditions with SMA giving the best orientation of the sample on the grid. Refined cryo-EM maps and angular distribution of particles of CPB reconstituted in SMA, BigCHAP or 2N2 nanodiscs. Representative micrographs of the data in panels B and C. Local resolution estimate performed in RELION showing the cap and rim domains at the highest resolution. Position of the NBP at the extremes of the multibody analysis performed in RELION (in green) compared to the CPB map (gray). Download figure Download PowerPoint Molecular architecture of the CPB pore The cryo-EM analysis of CPB clearly showed that like other members of the hemolysin-like family, the CPB protomer is composed of a cap, a rim, and a stem domain (Johnstone et al, 2021). In addition to the prototypical features of the family members, CPB possesses a second \\u03b2-barrel on top of the cap domain. While the N-terminus of Hla is located inside the cap domain and wraps the vestibule-exposed surface of the adjacent protomer, the CPB N-terminus protrudes from the cap as a short hairpin, which assembles to form a 16-strand \\u03b2-barrel (Fig 1B and D). We termed it the N-terminal \\u03b2-barrel protrusion (NBP). The cap domain consists of a \\u03b2-sandwich composed of two \\u03b2-sheets and short \\u03b1-helices. Two strands extend into the lower part of the molecule making up the rim domain. While the cap domain is one of the most conserved features within the hemolysin-like family, interesting differences are found in the rim domain. Unfolded loops in this domain have been shown to bind the toxin receptor in LukGH leukocidin (CD11b) and phosphatidyl choline in both Hla and \\u03b3-hemolysin (Trstenjak et al, 2020; Olson et al, 1999; Galdiero & Gouaux, 2004; Monma et al, 2004). Interestingly at position 210, CPB is the only family member to have a bulky aromatic residue, a tyrosine, pointing inside a pocket (Fig 1C). This pocket was shown to bind phosphocholine in Hla (Galdiero & Gouaux, 2004). Furthermore, CPB lacks a four-residue stretch which lines the pocket. Among them a tryptophan which is important for phosphocholine binding in staphylococcal hemolysins (Monma et al, 2004). These differences are compatible with the fact that CPB main receptor is a membrane protein, whereas for both \\u03b1- and \\u03b3-hemolysins the main receptor is phosphatidylcholine (Dal Peraro & van der Goot, 2016; Bruggisser et al, 2020). The loops at the base of the rim domain were well resolved except for two stretches of 14 and 5 residues, respectively (Glu76\\u2013Ser89; Ala283\\u2013Pro287). Importantly, these stretches are predicted to be surface exposed (Fig EV3A\\u2013D). (Trstenjak et al, 2020). We compared the protomer structure in CPB and \\u03b3-hemolysin octamers, respectively (Fig EV3E). The rim domain of the two proteins nearly perfectly overlaps except in the membrane-proximal loop region, suggesting that the loops have different functions between the two toxins. This may indicate that this region of CPB could potentially bind its receptor. Indeed, a pairwise flexible alignment of CPB with other hemolysins shows that apart from the NBP the main structural differences are located in the rim domain, in particular in the flexible loops of the rim (Appendix Fig S3). This issue is out of the scope of this paper and will be investigated in subsequent studies. The stem domain is similar to the other family members. It contains a long, curved amphipathic hairpin, which is connected to the cap \\u03b2-sandwich through two short coils forming the transmembrane \\u03b2-barrel upon oligomerization. Overall, the CPB octamer forms a 123 \\u00c5-long ring-like structure with a widest outer diameter of 97 \\u00c5 (Fig 1A and B). The channel runs along the 8-fold symmetry axis. It goes through the 25-\\u00c5 long N-terminal \\u00df-barrel, the central large 72,900 \\u00c53 vestibule, formed by the cap domains, and the transmembrane \\u03b2-barrel. Click here to expand this figure. Figure EV3. Modeling the missing features of the CPB model and structural comparison of CPB and \\u03b3-hemolysin A. Model of Clostridium perfringens \\u03b4-toxin (2YGT) using the same color code as in Fig 1 showing the position of its N-terminus folded back as an additional strand with the inset showing a comparison between the N-terminus of the \\u03b4-toxin (green) and the N-terminus of Hla (black\\u20144YHD). B. Model of a protomer of CPB extracted from the oligomer structure color coded as before. The missing loops in the rim domain are modeled and shown in gray. The missing loops are only shown as visual guide for the number of missing amino acids in the model as the map quality in those regions is not good enough for model building. C, D. Prediction of the CPB monomer structure by AlphaFold (C) and RosettaFold (D) color coded as before. The predicted N-terminus folds as the N-terminus of \\u03b4-toxin. The missing loops in the rim domains fold similarly to our modeling shown in (B) in the case of alphafold algorithm while RosettaFold modeling extends the longer missing loop into a long \\u03b2-sheet (D). E. Monomers extracted from the C. perfringens \\u03b2-toxin (CPB) and \\u03b3-hemolysin (PDB: 3B07) octamers were aligned (STAMP structural alignment), and the Euclidean distance of their paired C\\u03b1 atoms measured. In the graph, the black line reports on the distances measured when using our CPB structure. The red line reports on the mean difference when using all structures extracted from our MD simulations for comparison, and the gray shaded region reports on the standard deviation. The protein rendering is colored according to the mean difference of simulated structures. Download figure Download PowerPoint N-terminal \\u03b2-barrel protrusion To further investigate the NBP and its role, we constructed CPB mutants where the NBP was either deleted (\\u039423CPB) or replaced by the equivalent N-termini of S. aureus Hla (Hla-\\u039423CPB), \\u03b3-hemolysin component B (HlgB-\\u039423CPB), or C. perfringens \\u03b4-toxin (\\u03b4-toxin-\\u039423CPB). While the modification of the N-terminus seemed to affect protein solubility in Escherichia coli, with a much lower soluble protein yield for the chimeras when compared to the WT-CPB, the cholate purified proteins still retained their ability to oligomerize and form pores (Fig 2A and B). 2D class averages of the different mutants showed the lack of the NBP density for the \\u039423CPB mutant as well as a lack of a structured N-terminus for the Hla-\\u039423CPB and HlgB-\\u039423CPB. The N-terminus from \\u03b4-toxin in the \\u03b4-toxin-\\u039423CPB chimera seems to form an NBP similar to the WT-CPB, suggesting that the N-terminus of \\u03b4-toxin oligomer might also adopt a \\u03b2-barrel conformation in its oligomeric form (Figs 2C and EV3A). Interestingly, the ability to form the NBP correlates with the cytotoxic activity. The truncated \\u039423CPB, HlgB-\\u039423CPB, and Hla-\\u039423CPB were inactive, whereas activity was fully restored for the \\u03b4-toxin-\\u039423CPB (Fig 2D). Figure 2. Investigating the N-terminal domain of different hemolysins Characterization of Clostridium perfringens \\u03b2-toxin (CPB) constructs with different N-termini by gel electrophoretic analysis under denaturing (A) and native (B) conditions. Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE) gel analysis of CPB samples in cholate (1 \\u03bcg per lane). CPB samples containing cholate were boiled for 5 min at 95\\u00b0C (+) or not (\\u2212). Coomassie stained Blue native PAGE gel (4\\u201316%) of CPB samples (10 \\u03bcg per lane) showing oligomer formation for all different N-termini constructs. Alignment of CPB N-terminus with N-termini of different hemolysins and cryo-EM 2D classification and average of N-terminally tagged WT CPB (1), \\u039423CPB (2), \\u03b4-toxin N-terminus \\u039423CPB (3), C-terminally tagged WT CPB (4), Hla N-terminus \\u039423CPB (5), and HlgB N-terminus \\u039423CPB (6). Viability of HEK 293FT/CD31-GFP cells (transduced with CD31-GFP) as a percentage of untreated control cells after incubation with indicated concentrations of toxins (24 h, 37\\u00b0C). Data (technical replicates) are represented as means (n = 4) \\u00b1 SD. Download figure Download PowerPoint Channel properties and dynamics To investigate the mechanical and physical properties of the CPB pore, we carried out atomistic molecular dynamics (MD) simulations of it inserted into a lipid bilayer. The CPB channel features four constriction points in its two \\u03b2-barrels (Fig 3A). The narrowest points have a ~ 6 \\u00c5 mean radius and are located within the NBP at the level of the positively charged Arg11 and Lys13. The channel reaches a maximum radius of about 20 \\u00c5 within the vestibule of the cap. Quantifying the root mean square fluctuation (RMSF) of the toxin atoms revealed that the most flexible regions are the NBP and the intracellular turns of the transmembrane \\u03b2-barrel (Fig 3B). The NBP, while maintaining its overall structure, can oscillate off-axis, whereas the pore turns can squeeze off a perfectly circular symmetry contributing to reducing the local pore radius (Fig EV4A\\u2013C). These observations are consistent with EM data obtained by performing a multibody analysis on the CPB particles and could explain the variability observed previously in the channel conductance (Shatursky et al, 2000; Fig EV2F and Movie EV1). Examining charge distribution in the pore lumen revealed a high density of positive charges inside the \\u03b2-barrel protrusion due to Arg11 and Lys13 (Fig 3C). Potential mean force (PMF) profiles for Na+, Ca2+, and Cl\\u2212 estimated via umbrella sampling indicated that this region should constitute an energy barrier for cations (4.6 kcal/mol for Na+). The wide cap vestibule features a balanced amount of positive and negative charges, while the transmembrane \\u03b2-barrel is overall negatively charged, especially due to Glu152 and Glu162. These amino acids, associated with constriction points, determine a highly negative PMF for cations (as low as \\u221224.6 kcal/mol for Ca2+) and a large 10.2 kcal/mol barrier for Cl\\u2212 in agreement with previous reports suggesting that the CPB pore is mostly permeable to cations (Shatursky et al, 2000; Manich et al, 2008). Figure 3. CPB dynamics and NBP measurements Average pore radius in MD simulations. Above, regions with mean radius < 8 \\u00c5 (constriction points) are shown in red, > 8 \\u00c5 and < 10 \\u00c5 in orange, and > 10 \\u00c5 in yellow. In the graph, mean values are shown in red, and standard deviations in gray. Polar amino acids at constriction points are annotated. Clostridium perfringens \\u03b2-toxin (CPB) root mean square fluctuation (RMSF) averaged over the eight CPB chains and three simulation repeats. Above, most mobile regions are shown in yellow, and least mobile in purple. In the graph, mean RMSF values are shown in red, and their standard deviation in gray. The N-terminal \\u03b2-barrel Protrusion (NBP) is the most mobile region and, along with the flexible intracellular mount of transmembrane \\u03b2-barrel, is annotated in the graph. Electrostatic properties of the pore internal cavity. Above, negative regions are shown in red and positive ones in blue. In the graph below, potential mean force profiles along the pore axis for Na+, Ca2+, and Cl\\u2212 ions are shown. The NBP features a small positive region selective to anions, while the transmembrane \\u03b2-barrel is expected to be highly selective to cations. Download figure Download PowerPoint Click here to expand this figure. Figure EV4. In silico flexibility of the CPB channelFor each conformation in our Clostridium perfringens \\u03b2-toxin (CPB) simulation, we extracted the coordinates of C\\u03b1 at each constriction point and fitted them with an ellipse. The constriction point at the intracellular side of the toxin (residues 143 and 147) is the most dynamic, featuring the largest fluctuations in the fitted ellipse. Time evolution of constriction point surface area and aspect ratio of three consecutive 200 ns simulations. Position of all extracted C\\u03b1 coordinates of each constriction point, aligned so that the each fitted ellipse is centered at the origin and oriented so that its major semiaxis is parallel to the x-axis. Black ellipses fitted to these points represent the average constriction points shape. Only the constriction point at the intracellular side is noticeably elliptical, with C\\u03b1 atoms featuring larger deviations from the fitted ellipse. Position of the constriction points shown color coded in the cryo-EM map cross section. Download figure Download PowerPoint Comparison with other hemolysins-like \\u03b2-PFTs In this study, we explored the structure of the membrane inserted oligomeric CPB and showed that it belongs to a new subclass of hemolysin-like \\u03b2-PFTs that contain an additional \\u03b2-barrel domain in its extracellular side. After our first version of the manuscript was released on a preprint server, eight novel enterococcal hemolysin-like \\u03b2-PFTs were reported (Xiong et al, 2022). Interestingly, the structure of two of them was solved and showed that they also adopt an NBP fold atop the cap domain. A structural comparison shows a remarkable structural similarity suggesting that they also belong to the CPB subfamily of hemolysin-like PFTs (Fig EV5A\\u2013C). It is of particular interest that the NBP structure of EPX1 toxin has the same fold as CPB while the NBP of EPX4 toxin is reversed. This supports our mutagenesis results that the NBP \\u03b2-barrel structure can be highly flexible. A detailed comparison of the N-terminal domains with structures of other oligomeric hemolysins is not possible. The N-terminus of the oligomeric \\u03b3-hemolysin is disordered (Yamashita et al, 2011). For NetB, oligomer crystallization was only possible after removing the first 20 amino acids of the protein (Savva et al, 2013). For C. perfringens \\u03b4-toxin, only the monomer structure is available (Huyet et al, 2013)(Savva et al, 2013). In \\u03b4-toxin soluble monomer, unlike S. aureus Hla, the N-terminus adopts a \\u03b2-hairpin conformation, which extends halfway along the cap \\u03b2-sandwich and contacts the pre-stem (Fig EV3A). Because of the high sequence similarity between CPB and C. perfringens \\u03b4-toxin (Appendix Figs S1 and S2), it is reasonable to assume that the N-terminus of CPB adopts a similar conformation in the water-soluble monomer. To gain insight into the putative structure of CPB soluble monomer, we made use of recent advances in protein structure prediction (Baek et al, 2021; Jumper et al, 2021). Both Rosetta and AlphaFold generated predictions similar to the structure of \\u03b4-toxin monomer (Fig EV3C and D) with an N-terminus folded back and forming a \\u03b2-hairpin. Click here to expand this figure. Figure EV5. The \\u03b2-toxin subfamily of hemolysin-like \\u03b2-pore-forming toxins (PFTs) Side by side comparison of protomer structures of CPB, EPX1 (PDB: 7T4E), and EPX4 (PDB: 7T4D) color coded as in Fig 1 (NBP\\u2014green, cap\\u2014light blue, rim\\u2014dark blue, and stem\\u2014red). A magnified view of the NBP region showing that while CPB and EPX1 share the\", \"doi_y\": \"https://doi.org/10.15252/embr.202254856\", \"publication_date\": \"2022-10-10\"}, {\"work_id\": \"W4220732247\", \"doi_x\": \"10.1101/2022.03.07.22271833\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"GWAS and meta-analysis identifies multiple new genetic mechanisms underlying severe Covid-19\", \"abstract\": \"Abstract Pulmonary inflammation drives critical illness in Covid-19, 1;2 creating a clinically homogeneous extreme phenotype, which we have previously shown to be highly efficient for discovery of genetic associations. 3;4 Despite the advanced stage of illness, we have found that immunomodulatory therapies have strong beneficial effects in this group. 1;5 Further genetic discoveries may identify additional therapeutic targets to modulate severe disease. 6 In this new data release from the GenOMICC (Genetics Of Mortality in Critical Care) study we include new microarray genotyping data from additional critically-ill cases in the UK and Brazil, together with cohorts of severe Covid-19 from the ISARIC4C 7 and SCOURGE 8 studies, and meta-analysis with previously-reported data. We find an additional 14 new genetic associations. Many are in potentially druggable targets, in inflammatory signalling (JAK1, PDE4A), monocyte-macrophage differentiation (CSF2), immunometabolism (SLC2A5, AK5), and host factors required for viral entry and replication (TMPRSS2, RAB2A). As with our previous work, these results provide tractable therapeutic targets for modulation of harmful host-mediated inflammation in Covid-19.\", \"doi_y\": \"https://doi.org/10.1101/2022.03.07.22271833\", \"publication_date\": \"2022-03-07\"}, {\"work_id\": \"W4308834822\", \"doi_x\": \"10.1093/protein/gzac013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering enzyme activity using an expanded amino acid alphabet\", \"abstract\": \"Abstract Enzyme design and engineering strategies are typically constrained by the limited size of nature\\u2019s genetic alphabet, comprised of only 20 canonical amino acids. In recent years, site-selective incorporation of non-canonical amino acids (ncAAs) via an expanded genetic code has emerged as a powerful means of inserting new functional components into proteins, with hundreds of structurally diverse ncAAs now available. Here, we highlight how the emergence of an expanded repertoire of amino acids has opened new avenues in enzyme design and engineering. ncAAs have been used to probe complex biological mechanisms, augment enzyme function and, most ambitiously, embed new catalytic mechanisms into protein active sites that would be challenging to access within the constraints of nature\\u2019s genetic code. We predict that the studies reviewed in this article, along with further advances in genetic code expansion technology, will establish ncAA incorporation as an increasingly important tool for biocatalysis in the coming years.\", \"doi_y\": \"https://doi.org/10.1093/protein/gzac013\", \"publication_date\": \"2022-11-12\"}, {\"work_id\": \"W4319302345\", \"doi_x\": \"10.1016/j.semcdb.2023.02.001\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Generative models of morphogenesis in developmental biology\", \"abstract\": \"Understanding the mechanism by which cells coordinate their differentiation and migration is critical to our understanding of many fundamental processes such as wound healing, disease progression, and developmental biology. Mathematical models have been an essential tool for testing and developing our understanding, such as models of cells as soft spherical particles, reaction-diffusion systems that couple cell movement to environmental factors, and multi-scale multi-physics simulations that combine bottom-up rule-based models with continuum laws. However, mathematical models can often be loosely related to data or have so many parameters that model behaviour is weakly constrained. Recent methods in machine learning introduce new means by which models can be derived and deployed. In this review, we discuss examples of mathematical models of aspects of developmental biology, such as cell migration, and how these models can be combined with these recent machine learning methods.\", \"doi_y\": \"https://doi.org/10.1016/j.semcdb.2023.02.001\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4376141439\", \"doi_x\": \"10.1002/ange.202218783\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Darobactin B Stabilises a Lateral\\u2010Closed Conformation of the BAM Complex in <i>E. coli</i> Cells\", \"abstract\": \"Abstract The \\u03b2\\u2010barrel assembly machinery (BAM complex) is essential for outer membrane protein (OMP) folding in Gram\\u2010negative bacteria, and represents a promising antimicrobial target. Several conformational states of BAM have been reported, but all have been obtained under conditions which lack the unique features and complexity of the outer membrane (OM). Here, we use Pulsed Electron\\u2010Electron Double Resonance (PELDOR, or DEER) spectroscopy distance measurements to interrogate the conformational ensemble of the BAM complex in E. coli cells. We show that BAM adopts a broad ensemble of conformations in the OM, while in the presence of the antibiotic darobactin B (DAR\\u2010B), BAM\\u2032s conformational equilibrium shifts to a restricted ensemble consistent with the lateral closed state. Our in\\u2010cell PELDOR findings are supported by new cryoEM structures of BAM in the presence and absence of DAR\\u2010B. This work demonstrates the utility of PELDOR to map conformational changes in BAM within its native cellular environment.\", \"doi_y\": \"https://doi.org/10.1002/ange.202218783\", \"publication_date\": \"2023-05-31\"}, {\"work_id\": \"W4361301517\", \"doi_x\": \"10.1021/acssynbio.3c00064\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Rational Design of Phosphorylation-Responsive Coiled Coil-Peptide Assemblies\", \"abstract\": \"De novo peptides and proteins that switch state in response to chemical and physical cues would advance protein design and synthetic biology. Here we report two designed systems that disassemble and reassemble upon site-specific phosphorylation and dephosphorylation, respectively. As starting points, we use hyperthermostable de novo antiparallel and parallel coiled-coil heterotetramers, i.e., A2B2 systems, to afford control in downstream applications. The switches are incorporated by adding protein kinase A phosphorylation sites, R-R-X-S, with the phosphoacceptor serine residues placed to maximize disruption of the coiled-coil interfaces. The unphosphorylated peptides assemble as designed and unfold reversibly when heated. Addition of kinase to the assembled states unfolds them with half-lives of \\u22645 min. Phosphorylation is reversed by Lambda Protein Phosphatase resulting in tetramer reassembly. We envisage that the new de novo designed coiled-coil components, the switches, and a mechanistic model for them will be useful in synthetic biology, biomaterials, and biotechnology applications.\", \"doi_y\": \"https://doi.org/10.1021/acssynbio.3c00064\", \"publication_date\": \"2023-03-29\"}, {\"work_id\": \"W4228998779\", \"doi_x\": \"10.1101/2022.05.06.490883\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Modelling the within-host spread of SARS-CoV-2 infection, and the subsequent immune response, using a hybrid, multiscale, individual-based model. <i>Part I: Macrophages</i>\", \"abstract\": \"Abstract Individual responses to SARS-CoV-2 infection vary significantly, ranging from mild courses of infection that do not require hospitalisation to the development of disease which not only requires hospitalisation but can be fatal. Whilst many immunological studies have revealed fundamental insights into SARS-CoV-2 infection and COVID-19, mathematical and computational modelling can offer an additional perspective and enhance understanding. The majority of mathematical models for the within-host spread of SARS-CoV-2 infection are ordinary differential equations, which neglect spatial variation. In this article, we present a hybrid, multiscale, individual-based model to study the within-host spread of SARS-CoV-2 infection. The model incorporates epithelial cells (each containing a dynamical model for viral entry and replication), macrophages and a subset of cytokines. We investigate the role of increasing initial viral deposition, increasing delay in type I interferon secretion from epithelial cells (as well as the magnitude of secretion), increasing macrophage virus internalisation rate and macrophage activation, on the spread of infection.\", \"doi_y\": \"https://doi.org/10.1101/2022.05.06.490883\", \"publication_date\": \"2022-05-06\"}, {\"work_id\": \"W4362716961\", \"doi_x\": \"10.1002/adma.202300589\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Multi\\u2010Stimuli Responsive and Mechano\\u2010Actuated Biomimetic Membrane Nanopores Self\\u2010Assembled from DNA\", \"abstract\": \"Abstract In bioinspired design, biological templates are mimicked in structure and function by highly controllable synthetic means. Of interest are static barrel\\u2010like nanopores that enable molecular transport across membranes for use in biosensing, sequencing, and biotechnology. However, biological ion channels offer additional functions such as dynamic changes of the entire pore shape between open and closed states, and triggering of dynamic processes with biochemical and physical stimuli. To better capture this complexity, this report presents multi\\u2010stimuli and mechano\\u2010responsive biomimetic nanopores which are created with DNA nanotechnology. The nanopores switch between open and closed states, whereby specific binding of DNA and protein molecules as stimuli locks the pores in the open state. Furthermore, the physical stimulus of high transmembrane voltage switches the pores into a closed state. In addition, the pore diameters are larger and more tunable than those of natural templates. These multi\\u2010stimuli\\u2010responsive and mechanically actuated nanopores mimic several aspects of complex biological channels yet offer easier control over pore size, shape and stimulus response. The designer pores are expected to be applied in biosensing and synthetic biology.\", \"doi_y\": \"https://doi.org/10.1002/adma.202300589\", \"publication_date\": \"2023-06-02\"}, {\"work_id\": \"W4293900636\", \"doi_x\": \"10.1042/bcj20220154\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The potential role of ischaemia\\u2013reperfusion injury in chronic, relapsing diseases such as rheumatoid arthritis, Long COVID, and ME/CFS: evidence, mechanisms, and therapeutic implications\", \"abstract\": \"Ischaemia\\u2013reperfusion (I\\u2013R) injury, initiated via bursts of reactive oxygen species produced during the reoxygenation phase following hypoxia, is well known in a variety of acute circumstances. We argue here that I\\u2013R injury also underpins elements of the pathology of a variety of chronic, inflammatory diseases, including rheumatoid arthritis, ME/CFS and, our chief focus and most proximally, Long COVID. Ischaemia may be initiated via fibrin amyloid microclot blockage of capillaries, for instance as exercise is started; reperfusion is a necessary corollary when it finishes. We rehearse the mechanistic evidence for these occurrences here, in terms of their manifestation as oxidative stress, hyperinflammation, mast cell activation, the production of marker metabolites and related activities. Such microclot-based phenomena can explain both the breathlessness/fatigue and the post-exertional malaise that may be observed in these conditions, as well as many other observables. The recognition of these processes implies, mechanistically, that therapeutic benefit is potentially to be had from antioxidants, from anti-inflammatories, from iron chelators, and via suitable, safe fibrinolytics, and/or anti-clotting agents. We review the considerable existing evidence that is consistent with this, and with the biochemical mechanisms involved.\", \"doi_y\": \"https://doi.org/10.1042/bcj20220154\", \"publication_date\": \"2022-08-31\"}, {\"work_id\": \"W4290667627\", \"doi_x\": \"10.1371/journal.ppat.1010706\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic evidence of contemporary hybridization between Schistosoma species\", \"abstract\": \"Hybridization between different species of parasites is increasingly being recognised as a major public and veterinary health concern at the interface of infectious diseases biology, evolution, epidemiology and ultimately control. Recent research has revealed that viable hybrids and introgressed lineages between Schistosoma spp. are prevalent across Africa and beyond, including those with zoonotic potential. However, it remains unclear whether these hybrid lineages represent recent hybridization events, suggesting hybridization is ongoing, and/or whether they represent introgressed lineages derived from ancient hybridization events. In human schistosomiasis, investigation is hampered by the inaccessibility of adult-stage worms due to their intravascular location, an issue which can be circumvented by post-mortem of livestock at abattoirs for Schistosoma spp. of known zoonotic potential. To characterise the composition of naturally-occurring schistosome hybrids, we performed whole-genome sequencing of 21 natural livestock infective schistosome isolates. To facilitate this, we also assembled a de novo chromosomal-scale draft assembly of Schistosoma curassoni. Genomic analyses identified isolates of S. bovis, S. curassoni and hybrids between the two species, all of which were early generation hybrids with multiple generations found within the same host. These results show that hybridization is an ongoing process within natural populations with the potential to further challenge elimination efforts against schistosomiasis.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1010706\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4206493948\", \"doi_x\": \"10.1002/cbic.202100688\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular Determinants of Carbocation Cyclisation in Bacterial Monoterpene Synthases\", \"abstract\": \"Abstract Monoterpene synthases are often promiscuous enzymes, yielding product mixtures rather than pure compounds due to the nature of the branched reaction mechanism involving reactive carbocations. Two previously identified bacterial monoterpene synthases, a linalool synthase (bLinS) and a cineole synthase (bCinS), produce nearly pure linalool and cineole from geranyl diphosphate, respectively. We used a combined experimental and computational approach to identify critical residues involved in bacterial monoterpenoid synthesis. Phe77 is essential for bCinS activity, guiding the linear carbocation intermediate towards the formation of the cyclic \\u03b1\\u2010terpinyl intermediate; removal of the aromatic ring results in variants that produce acyclic products only. Computational chemistry confirmed the importance of Phe77 in carbocation stabilisation. Phe74, Phe78 and Phe179 are involved in maintaining the active site shape in bCinS without a specific role for the aromatic ring. Phe295 in bLinS, and the equivalent Ala301 in bCinS, are essential for linalool and cineole formation, respectively. Where Phe295 places steric constraints on the carbocation intermediates, Ala301 is essential for bCinS initial cyclisation and activity. Our multidisciplinary approach gives unique insights into how carefully placed amino acid residues in the active site can direct carbocations down specific paths, by placing steric constraints or offering stabilisation via cation\\u2010\\u03c0 interactions.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202100688\", \"publication_date\": \"2022-01-19\"}, {\"work_id\": \"W4321016016\", \"doi_x\": \"10.1016/j.tips.2022.12.002\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"abstract\": \"Up to 1.5 million people die yearly from fungal disease, but the repertoire of antifungal drug classes is minimal and the incidence of drug resistance is rising rapidly. This dilemma was recently declared by the World Health Organization as a global health emergency, but the discovery of new antifungal drug classes remains excruciatingly slow. This process could be accelerated by focusing on novel targets, such as G protein-coupled receptor (GPCR)-like proteins, that have a high likelihood of being druggable and have well-defined biology and roles in disease. We discuss recent successes in understanding the biology of virulence and in structure determination of yeast GPCRs, and highlight new approaches that might pay significant dividends in the urgent search for novel antifungal drugs.\", \"doi_y\": \"https://doi.org/10.1016/j.tips.2022.12.002\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4321380826\", \"doi_x\": \"10.1039/d2tb02781j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Synthetic engineering of a new biocatalyst encapsulating [NiFe]-hydrogenases for enhanced hydrogen production\", \"abstract\": \"By incorporating [NiFe]-hydrogenases into a proteinaceous carboxysome shell, we generate a novel biocatalyst that has improved production of clean hydrogen, oxygen tolerance, and thermostability, highlighting its great potential in biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1039/d2tb02781j\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4295080505\", \"doi_x\": \"10.1038/s41598-022-18890-6\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Characterisation of HOIP RBR E3 ligase conformational dynamics using integrative modelling\", \"abstract\": \"Abstract Multidomain proteins composed of individual domains connected by flexible linkers pose a challenge for structural studies due to their intrinsic conformational dynamics. Integrated modelling approaches provide a means to characterise protein flexibility by combining experimental measurements with molecular simulations. In this study, we characterise the conformational dynamics of the catalytic RBR domain of the E3 ubiquitin ligase HOIP, which regulates immune and inflammatory signalling pathways. Specifically, we combine small angle X-ray scattering experiments and molecular dynamics simulations to generate weighted conformational ensembles of the HOIP RBR domain using two different approaches based on maximum parsimony and maximum entropy principles. Both methods provide optimised ensembles that are instrumental in rationalising observed differences between SAXS-based solution studies and available crystal structures and highlight the importance of interdomain linker flexibility.\", \"doi_y\": \"https://doi.org/10.1038/s41598-022-18890-6\", \"publication_date\": \"2022-09-08\"}, {\"work_id\": \"W4376121421\", \"doi_x\": \"10.15252/embj.2022112767\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"<scp>FBXL4</scp> suppresses mitophagy by restricting the accumulation of <scp>NIX</scp> and <scp>BNIP3</scp> mitophagy receptors\", \"abstract\": \"Abstract To maintain both mitochondrial quality and quantity, cells selectively remove damaged or excessive mitochondria through mitophagy, which is a specialised form of autophagy. Mitophagy is induced in response to diverse conditions, including hypoxia, cellular differentiation and mitochondrial damage. However, the mechanisms that govern the removal of specific dysfunctional mitochondria under steady\\u2010state conditions to fine\\u2010tune mitochondrial content are not well understood. Here, we report that SCF FBXL4 , an SKP1/CUL1/F\\u2010box protein ubiquitin ligase complex, localises to the mitochondrial outer membrane in unstressed cells and mediates the constitutive ubiquitylation and degradation of the mitophagy receptors NIX and BNIP3 to suppress basal levels of mitophagy. We demonstrate that the pathogenic variants of FBXL4 that cause encephalopathic mtDNA depletion syndrome (MTDPS13) do not efficiently interact with the core SCF ubiquitin ligase machinery or mediate the degradation of NIX and BNIP3. Thus, we reveal a molecular mechanism whereby FBXL4 actively suppresses mitophagy by preventing NIX and BNIP3 accumulation. We propose that the dysregulation of NIX and BNIP3 turnover causes excessive basal mitophagy in FBXL4\\u2010associated mtDNA depletion syndrome.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022112767\", \"publication_date\": \"2023-05-10\"}, {\"work_id\": \"W4385693523\", \"doi_x\": \"10.1073/pnas.2305393120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"abstract\": \"Toxin-antitoxin (TA) systems are a large group of small genetic modules found in prokaryotes and their mobile genetic elements. Type II TAs are encoded as bicistronic (two-gene) operons that encode two proteins: a toxin and a neutralizing antitoxin. Using our tool NetFlax (standing for Network-FlaGs for toxins and antitoxins), we have performed a large-scale bioinformatic analysis of proteinaceous TAs, revealing interconnected clusters constituting a core network of TA-like gene pairs. To understand the structural basis of toxin neutralization by antitoxins, we have predicted the structures of 3,419 complexes with AlphaFold2. Together with mutagenesis and functional assays, our structural predictions provide insights into the neutralizing mechanism of the hyperpromiscuous Panacea antitoxin domain. In antitoxins composed of standalone Panacea, the domain mediates direct toxin neutralization, while in multidomain antitoxins the neutralization is mediated by other domains, such as PAD1, Phd-C, and ZFD. We hypothesize that Panacea acts as a sensor that regulates TA activation. We have experimentally validated 16 NetFlax TA systems and used domain annotations and metabolic labeling assays to predict their potential mechanisms of toxicity (such as membrane disruption, and inhibition of cell division or protein synthesis) as well as biological functions (such as antiphage defense). We have validated the antiphage activity of a RosmerTA system encoded by Gordonia phage Kita, and used fluorescence microscopy to confirm its predicted membrane-depolarizing activity. The interactive version of the NetFlax TA network that includes structural predictions can be accessed at http://netflax.webflags.se/ .\", \"doi_y\": \"https://doi.org/10.1073/pnas.2305393120\", \"publication_date\": \"2023-08-09\"}, {\"work_id\": \"W4293116279\", \"doi_x\": \"10.1093/ve/veac080\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The origins and molecular evolution of SARS-CoV-2 lineage B.1.1.7 in the UK\", \"abstract\": \"The first SARS-CoV-2 variant of concern (VOC) to be designated was lineage B.1.1.7, later labelled by the World Health Organization as Alpha. Originating in early autumn but discovered in December 2020, it spread rapidly and caused large waves of infections worldwide. The Alpha variant is notable for being defined by a long ancestral phylogenetic branch with an increased evolutionary rate, along which only two sequences have been sampled. Alpha genomes comprise a well-supported monophyletic clade within which the evolutionary rate is typical of SARS-CoV-2. The Alpha epidemic continued to grow despite the continued restrictions on social mixing across the UK and the imposition of new restrictions, in particular, the English national lockdown in November 2020. While these interventions succeeded in reducing the absolute number of cases, the impact of these non-pharmaceutical interventions was predominantly to drive the decline of the SARS-CoV-2 lineages that preceded Alpha. We investigate the only two sampled sequences that fall on the branch ancestral to Alpha. We find that one is likely to be a true intermediate sequence, providing information about the order of mutational events that led to Alpha. We explore alternate hypotheses that can explain how Alpha acquired a large number of mutations yet remained largely unobserved in a region of high genomic surveillance: an under-sampled geographical location, a non-human animal population, or a chronically infected individual. We conclude that the latter provides the best explanation of the observed behaviour and dynamics of the variant, although the individual need not be immunocompromised, as persistently infected immunocompetent hosts also display a higher within-host rate of evolution. Finally, we compare the ancestral branches and mutation profiles of other VOCs and find that Delta appears to be an outlier both in terms of the genomic locations of its defining mutations and a lack of the rapid evolutionary rate on its ancestral branch. As new variants, such as Omicron, continue to evolve (potentially through similar mechanisms), it remains important to investigate the origins of other variants to identify ways to potentially disrupt their evolution and emergence.\", \"doi_y\": \"https://doi.org/10.1093/ve/veac080\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4293168376\", \"doi_x\": \"10.1038/s41467-022-32774-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A ribosomally synthesised and post-translationally modified peptide containing a \\u03b2-enamino acid and a macrocyclic motif\", \"abstract\": \"Abstract Ribosomally synthesized and post-translationally modified peptides (RiPPs) are structurally complex natural products with diverse bioactivities. Here we report discovery of a RiPP, kintamdin, for which the structure is determined through spectroscopy, spectrometry and genomic analysis to feature a bis -thioether macrocyclic ring and a \\u03b2-enamino acid residue. Biosynthetic investigation demonstrated that its pathway relies on four dedicated proteins: phosphotransferase KinD, Lyase KinC, kinase homolog KinH and flavoprotein KinI, which share low homologues to enzymes known in other RiPP biosynthesis. During the posttranslational modifications, KinCD is responsible for the formation of the characteristic dehydroamino acid residues including the \\u03b2-enamino acid residue, followed by oxidative decarboxylation on the C-terminal Cys and subsequent cyclization to provide the bis -thioether ring moiety mediated by coordinated action of KinH and KinI. Finally, conserved genomic investigation allows further identification of two kintamdin-like peptides among the kin -like BGCs, suggesting the occurrence of RiPPs from actinobacteria.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-32774-3\", \"publication_date\": \"2022-08-26\"}, {\"work_id\": \"W4280528488\", \"doi_x\": \"10.1146/annurev-virology-100120-010228\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"abstract\": \"Advances in our understanding of prokaryotic antiphage defense mechanisms in the past few years have revealed a multitude of new cyclic nucleotide signaling molecules that play a crucial role in switching infected cells into an antiviral state. Defense pathways including type III CRISPR (clustered regularly interspaced palindromic repeats), CBASS (cyclic nucleotide-based antiphage signaling system), PYCSAR (pyrimidine cyclase system for antiphage resistance), and Thoeris all use cyclic nucleotides as second messengers to activate a diverse range of effector proteins. These effectors typically degrade or disrupt key cellular components such as nucleic acids, membranes, or metabolites, slowing down viral replication kinetics at great cost to the infected cell. Mechanisms to manipulate the levels of cyclic nucleotides are employed by cells to regulate defense pathways and by viruses to subvert them. Here we review the discovery and mechanism of the key pathways, signaling molecules and effectors, parallels and differences between the systems, open questions, and prospects for future research in this area.\", \"doi_y\": \"https://doi.org/10.1146/annurev-virology-100120-010228\", \"publication_date\": \"2022-09-29\"}, {\"work_id\": \"W4305015942\", \"doi_x\": \"10.1016/j.cell.2022.09.025\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"abstract\": \"Neural migration is a critical step during brain development that requires the interactions of cell-surface guidance receptors. Cancer cells often hijack these mechanisms to disseminate. Here, we reveal crystal structures of Uncoordinated-5 receptor D (Unc5D) in complex with morphogen receptor glypican-3 (GPC3), forming an octameric glycoprotein complex. In the complex, four Unc5D molecules pack into an antiparallel bundle, flanked by four GPC3 molecules. Central glycan-glycan interactions are formed by N-linked glycans emanating from GPC3 (N241 in human) and C-mannosylated tryptophans of the Unc5D thrombospondin-like domains. MD simulations, mass spectrometry and structure-based mutants validate the crystallographic data. Anti-GPC3 nanobodies enhance or weaken Unc5-GPC3 binding and, together with mutant proteins, show that Unc5/GPC3 guide migrating pyramidal neurons in the mouse cortex, and cancer cells in an embryonic xenograft neuroblastoma model. The results demonstrate a conserved structural mechanism of cell guidance, where finely balanced Unc5-GPC3 interactions regulate cell migration.\", \"doi_y\": \"https://doi.org/10.1016/j.cell.2022.09.025\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4283821860\", \"doi_x\": \"10.1038/s41591-022-01883-3\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"abstract\": \"Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management.\", \"doi_y\": \"https://doi.org/10.1038/s41591-022-01883-3\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4205120081\", \"doi_x\": \"10.1101/2022.01.14.476343\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"abstract\": \"Abstract Tpp49Aa1 from Lysinibacillus sphaericus is a Toxin_10 family protein that \\u2013 in combination with Cry48Aa1, a 3-domain crystal protein - has potent mosquitocidal activity, specifically against Culex quinquefasciatus mosquitoes. MHz serial femtosecond crystallography at a nano-focused X-ray free electron laser, allowed rapid and high-quality data collection to determine the Tpp49Aa1 structure at 1.62 \\u00c5 resolution from native nanocrystals. This revealed the packing of Tpp49Aa1 within these nanocrystals, isolated from sporulated bacteria, as a homodimer with a large intermolecular interface, shedding light on natural crystallization. Complementary experiments conducted at varied pH also enabled investigations of the early structural events leading up to the dissolution of natural Tpp49Aa1 crystals. Using modelling, we propose a potential interaction between Tpp49Aa1 and Cry48Aa1 that may play a role in their codependency and broaden our understanding of this two-component system. We expand the known target range, demonstrating Tpp49Aa1/Cry48Aa1 susceptibility of larvae from Anopheles stephensi, Aedes albopictus and Culex tarsalis \\u2013 substantially increasing the potential use of this toxin pair in mosquito control. Further functional insights are gained using Culex cell lines to characterise cellular models for future investigations into Cry48Aa1/Tpp49Aa1 mechanism of action and to demonstrate transient detrimental effects of individual toxin components. Significance Statement The Tpp49Aa1/Cry48Aa1 protein pair kills mosquito larvae. Innovative use of nano-focused X-ray free electron laser to match the size of natural Tpp49Aa1 nanocrystals and the highest beam intensity available in any XFEL for high-throughput data collection, allowed structural resolution to 1.62 \\u00c5. Tpp proteins show a range of interactions with different partners to elicit toxicity. To gain insight into Tpp49Aa1, its interaction with Cry48Aa1 was modelled. We also establish cell-based assays of Tpp49Aa1/Cry48Aa1 activity. We expand the known target range to include three more mosquito species: Anopheles stephensi, Aedes albopictus and Culex tarsalis . This study will underpin future Tpp mode of action investigations and aid insecticide optimization against mosquito vectors of emerging diseases such as West Nile Virus and malaria.\", \"doi_y\": \"https://doi.org/10.1101/2022.01.14.476343\", \"publication_date\": \"2022-01-15\"}, {\"work_id\": \"W4323858785\", \"doi_x\": \"10.1002/aenm.202300244\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"\\u03c0 Learning: A Performance\\u2010Informed Framework for Microstructural Electrode Design\", \"abstract\": \"Abstract Designing high\\u2010performance porous electrodes is the key to next\\u2010generation electrochemical energy devices. Current machine\\u2010learning\\u2010based electrode design strategies are mainly orientated toward physical properties; however, the electrochemical performance is the ultimate design objective. Performance\\u2010orientated electrode design is challenging because the current data driven approaches do not accurately extract high\\u2010dimensional features in complex multiphase microstructures. Herein, this work reports a novel performance\\u2010informed deep learning framework, termed \\u03c0 learning, which enables performance\\u2010informed microstructure generation, toward overall performance prediction of candidate electrodes by adding most relevant physical features into the learning process. This is achieved by integrating physics\\u2010informed generative adversarial neural networks (GANs) with convolutional neural networks (CNNs) and with advanced multi\\u2010physics, multi\\u2010scale modeling of 3D porous electrodes. This work demonstrates the advantages of \\u03c0 learning by employing two popular design philosophies: forward and inverse designs, for the design of solid oxide fuel cells electrodes. \\u03c0 learning thus has the potential to unlock performance\\u2010driven learning in the design of next generation porous electrodes for advanced electrochemical energy devices such as fuel cells and batteries.\", \"doi_y\": \"https://doi.org/10.1002/aenm.202300244\", \"publication_date\": \"2023-03-09\"}, {\"work_id\": \"W4321076815\", \"doi_x\": \"10.1021/acscentsci.2c01349\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"O-Linked Sialoglycans Modulate the Proteolysis of SARS-CoV-2 Spike and Likely Contribute to the Mutational Trajectory in Variants of Concern\", \"abstract\": \"The emergence of a polybasic cleavage motif for the protease furin in SARS-CoV-2 spike has been established as a major factor for human viral transmission. The region N-terminal to that motif is extensively mutated in variants of concern (VOCs). Besides furin, spikes from these variants appear to rely on other proteases for maturation, including TMPRSS2. Glycans near the cleavage site have raised questions about proteolytic processing and the consequences of variant-borne mutations. Here, we identify that sialic acid-containing O-linked glycans on Thr678 of SARS-CoV-2 spike influence furin and TMPRSS2 cleavage and posit O-linked glycosylation as a likely driving force for the emergence of VOC mutations. We provide direct evidence that the glycosyltransferase GalNAc-T1 primes glycosylation at Thr678 in the living cell, an event that is suppressed by mutations in the VOCs Alpha, Delta, and Omicron. We found that the sole incorporation of N-acetylgalactosamine did not impact furin activity in synthetic O-glycopeptides, but the presence of sialic acid reduced the furin rate by up to 65%. Similarly, O-glycosylation with a sialylated trisaccharide had a negative impact on TMPRSS2 cleavage. With a chemistry-centered approach, we substantiate O-glycosylation as a major determinant of spike maturation and propose disruption of O-glycosylation as a substantial driving force for VOC evolution.\", \"doi_y\": \"https://doi.org/10.1021/acscentsci.2c01349\", \"publication_date\": \"2023-02-16\"}, {\"work_id\": \"W4385273065\", \"doi_x\": \"10.3389/fnins.2023.1236815\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"How neurons maintain their axons long-term: an integrated view of axon biology and pathology\", \"abstract\": \"Axons are processes of neurons, up to a metre long, that form the essential biological cables wiring nervous systems. They must survive, often far away from their cell bodies and up to a century in humans. This requires self-sufficient cell biology including structural proteins, organelles, and membrane trafficking, metabolic, signalling, translational, chaperone, and degradation machinery\\u2014all maintaining the homeostasis of energy, lipids, proteins, and signalling networks including reactive oxygen species and calcium. Axon maintenance also involves specialised cytoskeleton including the cortical actin-spectrin corset, and bundles of microtubules that provide the highways for motor-driven transport of components and organelles for virtually all the above-mentioned processes. Here, we aim to provide a conceptual overview of key aspects of axon biology and physiology, and the homeostatic networks they form. This homeostasis can be derailed, causing axonopathies through processes of ageing, trauma, poisoning, inflammation or genetic mutations. To illustrate which malfunctions of organelles or cell biological processes can lead to axonopathies, we focus on axonopathy-linked subcellular defects caused by genetic mutations. Based on these descriptions and backed up by our comprehensive data mining of genes linked to neural disorders, we describe the \\u2018dependency cycle of local axon homeostasis\\u2019 as an integrative model to explain why very different causes can trigger very similar axonopathies, providing new ideas that can drive the quest for strategies able to battle these devastating diseases.\", \"doi_y\": \"https://doi.org/10.3389/fnins.2023.1236815\", \"publication_date\": \"2023-07-26\"}, {\"work_id\": \"W4307499980\", \"doi_x\": \"10.1016/j.pt.2022.09.008\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The mystery of massive mitochondrial complexes: the apicomplexan respiratory chain\", \"abstract\": \"The mitochondrial respiratory chain is an essential pathway in most studied eukaryotes due to its roles in respiration and other pathways that depend on mitochondrial membrane potential. Apicomplexans are unicellular eukaryotes whose members have an impact on global health. The respiratory chain is a drug target for some members of this group, notably the malaria-causing Plasmodium spp. This has motivated studies of the respiratory chain in apicomplexan parasites, primarily Toxoplasma gondii and Plasmodium spp. for which experimental tools are most advanced. Studies of the respiratory complexes in these organisms revealed numerous novel features, including expansion of complex size. The divergence of apicomplexan mitochondria from commonly studied models highlights the diversity of mitochondrial form and function across eukaryotic life.\", \"doi_y\": \"https://doi.org/10.1016/j.pt.2022.09.008\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4321117779\", \"doi_x\": \"10.1016/j.jbc.2023.103033\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The ROK kinase N-acetylglucosamine kinase uses a sequential random enzyme mechanism with successive conformational changes upon each substrate binding\", \"abstract\": \"N-acetyl-d-glucosamine (GlcNAc) is a major component of bacterial cell walls. Many organisms recycle GlcNAc from the cell wall or metabolize environmental GlcNAc. The first step in GlcNAc metabolism is phosphorylation to GlcNAc-6-phosphate. In bacteria, the ROK family kinase N-acetylglucosamine kinase (NagK) performs this activity. Although ROK kinases have been studied extensively, no ternary complex showing the two substrates has yet been observed. Here, we solved the structure of NagK from the human pathogen Plesiomonas shigelloides in complex with GlcNAc and the ATP analog AMP-PNP. Surprisingly, PsNagK showed distinct conformational changes associated with the binding of each substrate. Consistent with this, the enzyme showed a sequential random enzyme mechanism. This indicates that the enzyme acts as a coordinated unit responding to each interaction. Our molecular dynamics modeling of catalytic ion binding confirmed the location of the essential catalytic metal. Additionally, site-directed mutagenesis confirmed the catalytic base and that the metal-coordinating residue is essential. Together, this study provides the most comprehensive insight into the activity of a ROK kinase.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.103033\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4285605412\", \"doi_x\": \"10.1016/j.envpol.2022.119757\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genetic sequencing detected the SARS-CoV-2 delta variant in wastewater a month prior to the first COVID-19 case in Ahmedabad (India)\", \"abstract\": \"Wastewater-based genomic surveillance can identify a huge majority of variants shed by the infected individuals within a population, which goes beyond genomic surveillance based on clinical samples (i.e., symptomatic patients only). We analyzed four samples to detect key mutations in the SARS-CoV-2 genome and track circulating variants in Ahmedabad during the first wave (Sep/Nov 2020) and before the second wave (in Feb 2021) of COVID-19 in India. The analysis identified a total of 34 mutations in the spike protein across samples categorized into 23 types. The spike protein mutations were linked to the VOC-21APR-02; B.1.617.2 lineage (Delta variant) with 57% frequency in wastewater samples of Feb 2021. The key spike protein mutations were T19R, L452R, T478K, D614G, & P681R and deletions at 22029 (6 bp), 28248 (6 bp), & 28271 (1 bp). Interestingly, these mutations were not seen in the samples from Sep/Nov 2020 but did appear before the massive second wave of COVID-19 cases, which in India started in early April 2021. In fact, genetic traces of the Delta variant were found in samples of early Feb 2021, more than a month before the first clinically confirmed case of this in March 2021 in Ahmedabad, Gujarat. The present work describes the circulating of SARS-CoV-2 variants in Ahmedabad and confirms the consequential value of wastewater surveillance for the early detection of variants of concerns (VOCs). Such monitoring must be included as a major component of future health protection systems.\", \"doi_y\": \"https://doi.org/10.1016/j.envpol.2022.119757\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4225303574\", \"doi_x\": \"10.1530/raf-21-0078\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Insights from genomic studies on the role of sex steroids in the aetiology of endometriosis\", \"abstract\": \"Endometriosis is a chronic neuro-inflammatory disorder the defining feature of which is the growth of tissue (lesions) that resembles the endometrium outside the uterus. Estimates of prevalence quote rates of ~10% of women of reproductive age, equating to at least 190 million women world-wide. Genetic, hormonal and immunological factors have all been proposed as contributing to risk factors associated with the development of lesions. Twin studies report the heritable component of endometriosis as ~50%. Genome-wide association studies (GWAS) have identified single nucleotide polymorphisms (SNPs) that appear over-represented in patients with endometriosis, particularly those with more extensive disease (stage III/IV). In different sample populations, there has been replication of SNPs near genes involved in oestrogen and other steroid regulated pathways including ESR1 (oestrogen receptor alpha), GREB1, HOXA10, WNT4 and MAPK kinase signalling. Comparisons with GWAS conducted on other patient cohorts have found links with reproductive traits (age at menarche) and disorders (fibroids, endometrial and ovarian cancer) and common co-morbidities (migraine, depression, asthma). In summary, genetic analyses have provided new insights into the hormone-regulated pathways that may contribute to increased risk of developing endometriosis some of which may act in early life. New studies are needed to clarify the relationship between the many SNPs identified, the genes that they regulate and their contribution(s) to development of different forms of endometriosis. We hope that more advanced methods allowing integration between GWAS, epigenetic and tissue expression data will improve risk analysis and reduce diagnositic delay.Endometriosis is a debilitating reproductive disorder affecting ~10% of reproductive-age women, and those assigned female at birth, which causes a range of symptoms including chronic pain and infertility. The reason why some, but not all these individuals, develop the lesions that characterise the disease are poorly understood, but recently attention has focused on genetic risk factors to explain why the incidence is higher in some families. Studies on large cohorts of patients with comparison of their DNA to women without endometriosis or with other disorders have documented changes in genes associated with steroid hormone production or action. The results provide further evidence that endometriosis shares genetic risk factors with other disorders of the reproductive system and a platform for new ideas related to risk, biomarkers and therapies.\", \"doi_y\": \"https://doi.org/10.1530/raf-21-0078\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W4308469543\", \"doi_x\": \"10.1016/j.csbj.2022.11.004\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"abstract\": \"Coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 is an ongoing pandemic that causes significant health/socioeconomic burden. Variants of concern (VOCs) have emerged affecting transmissibility, disease severity and re-infection risk. Studies suggest that the - N-terminal domain (NTD) of the spike protein may have a role in facilitating virus entry via sialic-acid receptor binding. Furthermore, most VOCs include novel NTD variants. Despite global sequence and structure similarity, most sialic-acid binding pockets in NTD vary across coronaviruses. Our work suggests ongoing evolutionary tuning of the sugar-binding pockets and recent analyses have shown that NTD insertions in VOCs tend to lie close to loops. We extended the structural characterisation of these sugar-binding pockets and explored whether variants could enhance sialic acid-binding. We found that recent NTD insertions in VOCs (i.e., Gamma, Delta and Omicron variants) and emerging variants of interest (VOIs) (i.e., Iota, Lambda and Theta variants) frequently lie close to sugar-binding pockets. For some variants, including the recent Omicron VOC, we find increases in predicted sialic acid-binding energy, compared to the original SARS-CoV-2, which may contribute to increased transmission. These binding observations are supported by molecular dynamics simulations (MD). We examined the similarity of NTD across Betacoronaviruses to determine whether the sugar-binding pockets are sufficiently similar to be exploited in drug design. Whilst most pockets are too structurally variable, we detected a previously unknown highly structurally conserved pocket which can be investigated in pursuit of a generic pan-Betacoronavirus drug. Our structure-based analyses help rationalise the effects of VOCs and provide hypotheses for experiments. Our findings suggest a strong need for experimental monitoring of changes in NTD of VOCs.\", \"doi_y\": \"https://doi.org/10.1016/j.csbj.2022.11.004\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4320879233\", \"doi_x\": \"10.1002/cmdc.202200541\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Development of Enterovirus Antiviral Agents That Target the Viral 2C Protein\", \"abstract\": \"Abstract The Enterovirus (EV) genus includes several important human and animal pathogens. EV\\u2010A71, EV\\u2010D68, poliovirus (PV), and coxsackievirus (CV) outbreaks have affected millions worldwide, causing a range of upper respiratory, skin, and neuromuscular diseases, including acute flaccid myelitis, and hand\\u2010foot\\u2010and\\u2010mouth disease. There are no FDA\\u2010approved antiviral therapeutics for these enteroviruses. This study describes novel antiviral compounds targeting the conserved non\\u2010structural viral protein 2C with low micromolar to nanomolar IC 50 values. The selection of resistant mutants resulted in amino acid substitutions in the viral capsid protein, implying these compounds may play a role in inhibiting the interaction of 2C and the capsid protein. The assembly and encapsidation stages of the viral life cycle still need to be fully understood, and the inhibitors reported here could be useful probes in understanding these processes.\", \"doi_y\": \"https://doi.org/10.1002/cmdc.202200541\", \"publication_date\": \"2023-03-10\"}, {\"work_id\": \"W4321456268\", \"doi_x\": \"10.1042/ebc20220158\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structure and function of microbial \\u03b1-<scp>l</scp>-fucosidases: a mini review\", \"abstract\": \"Fucose is a monosaccharide commonly found in mammalian, insect, microbial and plant glycans. The removal of terminal \\u03b1-l-fucosyl residues from oligosaccharides and glycoconjugates is catalysed by \\u03b1-l-fucosidases. To date, glycoside hydrolases (GHs) with exo-fucosidase activity on \\u03b1-l-fucosylated substrates (EC 3.2.1.51, EC 3.2.1.-) have been reported in the GH29, GH95, GH139, GH141 and GH151 families of the Carbohydrate Active Enzymes (CAZy) database. Microbes generally encode several fucosidases in their genomes, often from more than one GH family, reflecting the high diversity of naturally occuring fucosylated structures they encounter. Functionally characterised microbial \\u03b1-l-fucosidases have been shown to act on a range of substrates with \\u03b1-1,2, \\u03b1-1,3, \\u03b1-1,4 or \\u03b1-1,6 fucosylated linkages depending on the GH family and microorganism. Fucosidases show a modular organisation with catalytic domains of GH29 and GH151 displaying a (\\u03b2/\\u03b1)8-barrel fold while GH95 and GH141 show a (\\u03b1/\\u03b1)6 barrel and parallel \\u03b2-helix fold, respectively. A number of crystal structures have been solved in complex with ligands, providing structural basis for their substrate specificity. Fucosidases can also be used in transglycosylation reactions to synthesise oligosaccharides. This mini review provides an overview of the enzymatic and structural properties of microbial \\u03b1-l-fucosidases and some insights into their biological function and biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1042/ebc20220158\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4318049405\", \"doi_x\": \"10.1515/hsz-2022-0314\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Super-resolution microscopy and studies of peroxisomes\", \"abstract\": \"Fluorescence microscopy is an important tool for studying cellular structures such as organelles. Unfortunately, many details in the corresponding images are hidden due to the resolution limit of conventional lens-based far-field microscopy. An example is the study of peroxisomes, where important processes such as molecular organization during protein important can simply not be studied with conventional far-field microscopy methods. A remedy is super-resolution fluorescence microscopy, which is nowadays a well-established technique for the investigation of inner-cellular structures but has so far to a lesser extent been applied to the study of peroxisomes. To help advancing the latter, we here give an overview over the different super-resolution microscopy approaches and their potentials and challenges in cell-biological research, including labelling issues and a focus on studies on peroxisomes. Here, we also highlight experiments beyond simple imaging such as observations of diffusion dynamics of peroxisomal proteins.\", \"doi_y\": \"https://doi.org/10.1515/hsz-2022-0314\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W4306407978\", \"doi_x\": \"10.1002/ange.202212158\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A Vitamin B<sub>2</sub>\\u2010Photocatalysed Approach to Methionine Analogues\", \"abstract\": \"Abstract Access to new non\\u2010canonical amino acid residues is crucial for medicinal chemistry and chemical biology. Analogues of the amino acid methionine have been far less explored\\u2014despite their use in biochemistry, pharmacology and peptide bioconjugation. This is largely due to limited synthetic access. Herein, we exploit a new disconnection to access non\\u2010natural methionines through the development of a photochemical method for the radical \\u03b1\\u2010C\\u2212H functionalization of sulfides with alkenes, in water, using inexpensive and commercially\\u2010available riboflavin (vitamin B 2 ) as a photocatalyst. Our photochemical conditions allow the two\\u2010step synthesis of novel methionine analogues\\u2014by radical addition to unsaturated amino acid derivatives\\u2014and the chemoselective modification of peptide side\\u2010chains to yield non\\u2010natural methionine residues within small peptides. The mechanism of the bio\\u2010inspired flavin photocatalysis has been probed by experimental, DFT and TDDFT studies.\", \"doi_y\": \"https://doi.org/10.1002/ange.202212158\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W4323656741\", \"doi_x\": \"10.1073/pnas.2220921120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Broader functions of TIR domains in Arabidopsis immunity\", \"abstract\": \"TIR domains are NAD-degrading enzymes that function during immune signaling in prokaryotes, plants, and animals. In plants, most TIR domains are incorporated into intracellular immune receptors termed TNLs. In Arabidopsis, TIR-derived small molecules bind and activate EDS1 heterodimers, which in turn activate RNLs, a class of cation channel\\u2013forming immune receptors. RNL activation drives cytoplasmic Ca 2+ influx, transcriptional reprogramming, pathogen resistance, and host cell death. We screened for mutants that suppress an RNL activation mimic allele and identified a TNL, SADR1. Despite being required for the function of an autoactivated RNL, SADR1 is not required for defense signaling triggered by other tested TNLs. SADR1 is required for defense signaling initiated by some transmembrane pattern recognition receptors and contributes to the unbridled spread of cell death in lesion simulating disease 1 . Together with RNLs, SADR1 regulates defense gene expression at infection site borders, likely in a non-cell autonomous manner. RNL mutants that cannot sustain this pattern of gene expression are unable to prevent disease spread beyond localized infection sites, suggesting that this pattern corresponds to a pathogen containment mechanism. SADR1 potentiates RNL-driven immune signaling not only through the activation of EDS1 but also partially independently of EDS1. We studied EDS1-independent TIR function using nicotinamide, an NADase inhibitor. Nicotinamide decreased defense induction from transmembrane pattern recognition receptors and decreased calcium influx, pathogen growth restriction, and host cell death following intracellular immune receptor activation. We demonstrate that TIR domains can potentiate calcium influx and defense and are thus broadly required for Arabidopsis immunity.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2220921120\", \"publication_date\": \"2023-03-09\"}, {\"work_id\": \"W4306641915\", \"doi_x\": \"10.1073/pnas.2210559119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"abstract\": \"Exocytosis plays an important role in plant-microbe interactions, in both pathogenesis and symbiosis. Exo70 proteins are integral components of the exocyst, an octameric complex that mediates tethering of vesicles to membranes in eukaryotes. Although plant Exo70s are known to be targeted by pathogen effectors, the underpinning molecular mechanisms and the impact of this interaction on infection are poorly understood. Here, we show the molecular basis of the association between the effector AVR-Pii of the blast fungus Maganaporthe oryzae and rice Exo70 alleles OsExo70F2 and OsExo70F3, which is sensed by the immune receptor pair Pii via an integrated RIN4/NOI domain. The crystal structure of AVR-Pii in complex with OsExo70F2 reveals that the effector binds to a conserved hydrophobic pocket in Exo70, defining an effector/target binding interface. Structure-guided and random mutagenesis validates the importance of AVR-Pii residues at the Exo70 binding interface to sustain protein association and disease resistance in rice when challenged with fungal strains expressing effector mutants. Furthermore, the structure of AVR-Pii defines a zinc-finger effector fold (ZiF) distinct from the MAX (Magnaporthe Avrs and ToxB-like) fold previously described for a majority of characterized M. oryzae effectors. Our data suggest that blast fungus ZiF effectors bind a conserved Exo70 interface to manipulate plant exocytosis and that these effectors are also baited by plant immune receptors, pointing to new opportunities for engineering disease resistance.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2210559119\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4307770903\", \"doi_x\": \"10.1002/anie.202213053\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"abstract\": \"Abstract Abyssomicin C and its atropisomer are potent inhibitors of bacterial folate metabolism. They possess complex polycyclic structures, and their biosynthesis has been shown to involve several unusual enzymatic transformations. Using a combination of synthesis and in vitro assays we reveal that AbyV, a cytochrome P450 enzyme from the aby gene cluster, catalyses a key late\\u2010stage epoxidation required for the installation of the characteristic ether\\u2010bridged core of abyssomicin C. The X\\u2010ray crystal structure of AbyV has been determined, which in combination with molecular dynamics simulations provides a structural framework for our functional data. This work demonstrates the power of combining selective carbon\\u201013 labelling with NMR spectroscopy as a sensitive tool to interrogate enzyme\\u2010catalysed reactions in vitro with no need for purification.\", \"doi_y\": \"https://doi.org/10.1002/anie.202213053\", \"publication_date\": \"2022-12-08\"}, {\"work_id\": \"W4286488532\", \"doi_x\": \"10.1021/acschemneuro.2c00339\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"5-HT<sub>3</sub> Receptor MX Helix Contributes to Receptor Function\", \"abstract\": \"5-HT3 receptors are members of the family of pentameric ligand-gated ion channels. Each subunit has an extracellular, transmembrane, and intracellular domain. Only part of the intracellular domain structure has been solved, revealing it contains two \\u03b1-helical segments; one, the MA helix, is an extension of M4, while the other, the MX helix, is formed from residues located close to the end of M3. This MX helix is in distinct locations in open and closed receptor structures, suggesting it may play a role in function. Here, we explore this hypothesis using functional responses of Ala-substituted mutant receptors expressed in HEK293 cells. The data show altering many of the MX residues results in a small decrease in EC50 (up to 5-fold), although in one (H232A) this is increased. Radiolabeled ligand binding on selected mutants showed no change in binding affinity, indicating an effect on gating and not binding. In addition, five mutations (P316A, V317A, P318A, D319A, and H323A) initially resulted in nonfunctional receptors, but the function could be rescued by coexpression with a chaperone protein, suggesting a likely role in assembly or folding. Examination of previously obtained MD simulation data shows that the extent of MX encompassed by membrane lipids differs considerably in the open and closed structures, suggesting that lipid-protein interactions in this region could have a major effect on channel opening propensity. We conclude that the MX helix can modulate the function of the receptor and propose that its interactions with membrane lipids play a major role in this.\", \"doi_y\": \"https://doi.org/10.1021/acschemneuro.2c00339\", \"publication_date\": \"2022-07-22\"}, {\"work_id\": \"W4285389980\", \"doi_x\": \"10.1038/s41594-022-00799-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular basis for the regulation of human glycogen synthase by phosphorylation and glucose-6-phosphate\", \"abstract\": \"Glycogen synthase (GYS1) is the central enzyme in muscle glycogen biosynthesis. GYS1 activity is inhibited by phosphorylation of its amino (N) and carboxyl (C) termini, which is relieved by allosteric activation of glucose-6-phosphate (Glc6P). We present cryo-EM structures at 3.0-4.0 \\u00c5 resolution of phosphorylated human GYS1, in complex with a minimal interacting region of glycogenin, in the inhibited, activated and catalytically competent states. Phosphorylations of specific terminal residues are sensed by different arginine clusters, locking the GYS1 tetramer in an inhibited state via intersubunit interactions. The Glc6P activator promotes conformational change by disrupting these interactions and increases the flexibility of GYS1, such that it is poised to adopt a catalytically competent state when the sugar donor UDP-glucose (UDP-glc) binds. We also identify an inhibited-like conformation that has not transitioned into the activated state, in which the locking interaction of phosphorylation with the arginine cluster impedes subsequent conformational changes due to Glc6P binding. Our results address longstanding questions regarding the mechanism of human GYS1 regulation.\", \"doi_y\": \"https://doi.org/10.1038/s41594-022-00799-3\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4320723690\", \"doi_x\": \"10.3389/fcvm.2023.1094563\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Affimers targeting proteins in the cardiomyocyte Z-disc: Novel tools that improve imaging of heart tissue\", \"abstract\": \"Dilated Cardiomyopathy is a common form of heart failure. Determining how this disease affects the structure and organization of cardiomyocytes in the human heart is important in understanding how the heart becomes less effective at contraction. Here we isolated and characterised Affimers (small non-antibody binding proteins) to Z-disc proteins ACTN2 (\\u03b1-actinin-2), ZASP (also known as LIM domain binding protein 3 or LDB3) and the N-terminal region of the giant protein titin (TTN Z1-Z2). These proteins are known to localise in both the sarcomere Z-discs and the transitional junctions, found close to the intercalated discs that connect adjacent cardiomyocytes. We use cryosections of left ventricles from two patients diagnosed with end-stage Dilated Cardiomyopathy who underwent Orthotopic Heart Transplantation and were whole genome sequenced. We describe how Affimers substantially improve the resolution achieved by confocal and STED microscopy compared to conventional antibodies. We quantified the expression of ACTN2, ZASP and TTN proteins in two patients with dilated cardiomyopathy and compared them with a sex- and age-matched healthy donor. The small size of the Affimer reagents, combined with a small linkage error (the distance from the epitope to the dye label covalently bound to the Affimer) revealed new structural details in Z-discs and intercalated discs in the failing samples. Affimers are thus useful for analysis of changes to cardiomyocyte structure and organisation in diseased hearts.\", \"doi_y\": \"https://doi.org/10.3389/fcvm.2023.1094563\", \"publication_date\": \"2023-02-14\"}, {\"work_id\": \"W3198923619\", \"doi_x\": \"10.1002/prot.26237\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"abstract\": \"Critical assessment of structure prediction (CASP) is a community experiment to advance methods of computing three-dimensional protein structure from amino acid sequence. Core components are rigorous blind testing of methods and evaluation of the results by independent assessors. In the most recent experiment (CASP14), deep-learning methods from one research group consistently delivered computed structures rivaling the corresponding experimental ones in accuracy. In this sense, the results represent a solution to the classical protein-folding problem, at least for single proteins. The models have already been shown to be capable of providing solutions for problematic crystal structures, and there are broad implications for the rest of structural biology. Other research groups also substantially improved performance. Here, we describe these results and outline some of the many implications. Other related areas of CASP, including modeling of protein complexes, structure refinement, estimation of model accuracy, and prediction of inter-residue contacts and distances, are also described.\", \"doi_y\": \"https://doi.org/10.1002/prot.26237\", \"publication_date\": \"2021-10-07\"}, {\"work_id\": \"W4385255907\", \"doi_x\": \"10.1111/2041-210x.14166\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Why should we compare morphological and molecular disparity?\", \"abstract\": \"Abstract Indices of morphological disparity seek to summarise the highly multivariate morphological variation across groups of species within clades, time bins or other groups. Morphological variation can be quantified using geometric morphometric, outline or surface\\u2010based methods. These are most effective when morphological differences are relatively modest and there are numerous ubiquitous landmarks and phase aligned features of shape variation. The most disparate samples, such as those across classes and phyla, typically necessitate the use of discrete characters. Unfortunately, such characters are often compiled subjectively in a manner reflecting the level of morphological and taxonomic focus and the intensity of taxon sampling. Sampling intensity is often highly variable within a single data set, especially in repurposed and amalgamated cladistic matrices. Here, we propose indices of molecular disparity analogous to those of morphological disparity. Despite numerous shortcomings discussed here, molecular sequence data can be obtained in a more objective, automated and scalable manner than morphological data. Comparisons of the morphological and molecular disparity of subclades in 16 large data sets suggest that molecular disparity is less susceptible to sampling biases than morphological disparity. Moreover, distance matrices inferred from individual genes tend to correlate strongly with each other and with distances from all concatenated genes. By contrast, morphological and molecular disparity are typically not significantly correlated across subclades, such that comparisons for groups can help to give a fuller picture of their evolution. For example, within mammals, Afrotheria have conspicuously high morphological disparity but modest molecular disparity, suggesting unusually high morphological plasticity. Even more strikingly, the molecular disparity of rodents is over five times that for Artiodactyla, despite having only half of their morphological disparity. These contrasts suggest the differential operation of geometric, biomechanical, ontogenetic and environmental constraints on form. Given the increasing abundance of total evidence datasets in the literature and the widespread and sometimes uncritical repurposing of discrete morphological matrices, we propose the comparison of morphological and molecular disparity as a useful tool to understand subclade evolution more fully.\", \"doi_y\": \"https://doi.org/10.1111/2041-210x.14166\", \"publication_date\": \"2023-07-24\"}, {\"work_id\": \"W4310957810\", \"doi_x\": \"10.1093/nar/gkac1080\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"abstract\": \"Abstract The MGnify platform (https://www.ebi.ac.uk/metagenomics) facilitates the assembly, analysis and archiving of microbiome-derived nucleic acid sequences. The platform provides access to taxonomic assignments and functional annotations for nearly half a million analyses covering metabarcoding, metatranscriptomic, and metagenomic datasets, which are derived from a wide range of different environments. Over the past 3 years, MGnify has not only grown in terms of the number of datasets contained but also increased the breadth of analyses provided, such as the analysis of long-read sequences. The MGnify protein database now exceeds 2.4 billion non-redundant sequences predicted from metagenomic assemblies. This collection is now organised into a relational database making it possible to understand the genomic context of the protein through navigation back to the source assembly and sample metadata, marking a major improvement. To extend beyond the functional annotations already provided in MGnify, we have applied deep learning-based annotation methods. The technology underlying MGnify's Application Programming Interface (API) and website has been upgraded, and we have enabled the ability to perform downstream analysis of the MGnify data through the introduction of a coupled Jupyter Lab environment.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1080\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4293105996\", \"doi_x\": \"10.1101/2022.08.25.505149\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Systems-wide dissection of organic acid assimilation in <i>Pseudomonas aeruginosa</i> reveals a novel path to underground metabolism\", \"abstract\": \"Abstract The human pathogen Pseudomonas aeruginosa (Pa) is one of the most frequent and severe causes of nosocomial infection. This organism is also a major cause of airway infections in people with cystic fibrosis (CF). Pa is known to have a remarkable metabolic plasticity, allowing it to thrive in diverse environmental conditions and ecological niches, yet little is known about the central metabolic pathways which sustain its growth during infection, or precisely how these pathways operate. In this work, we used a combination of \\u2018omics approaches (transcriptomics, proteomics, metabolomics and 13 C-fluxomics) and reverse genetics to provide a systems-level insight into how the infection-relevant organic acids, succinate and propionate, are metabolized by Pa. Moreover, through structural and kinetic analysis of the 2-methylcitrate synthase (PrpC) and its paralogue, citrate synthase (GltA), we show how these two crucial enzymatic steps are interconnected in Pa organic acid assimilation. We found that Pa can rapidly adapt to the loss of GltA function by acquiring mutations in a transcriptional repressor, which then de-represses prpC expression. Our findings provide a clear example of how \\u2018underground metabolism\\u2019, facilitated by enzyme substrate promiscuity, \\u201crewires\\u201d Pa metabolism, allowing it to overcome the loss of a crucial enzyme. This pathogen-specific knowledge is critical for the advancement of a model-driven framework to target bacterial central metabolism.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.25.505149\", \"publication_date\": \"2022-08-25\"}, {\"work_id\": \"W4301366241\", \"doi_x\": \"10.1101/2022.10.04.510756\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Intracellular FUS protein accumulation leads to cytoskeletal, organelle and cellular homeostasis perturbations\", \"abstract\": \"Summary The molecular mechanisms that connect the formation of aberrant cytoplasmic FUS condensates to biological malfunction are incompletely understood. Here, we develop an approach to determine the intracellular FUS viscosity in live mammalian cells and find that ALS-related mutant P525L-FUS forms the most viscous condensates and has impaired cytoskeletal mechanoproperties and increased euchromatin formation. We further show that some of the main cellular organelles, e.g., actin/tubulin, lysosomes, mitochondria, the endoplasmic reticulum, are significantly functionally/structurally impaired in the presence of FUS. These may be related to defects in the tubulin network, as the latter facilitates transport, formation, fusion and fission of organelles. We observe significant increases in lysosomal biogenesis, size and pH; moreover, intracellular FUS accumulation significantly promotes cytoplasmic-to-nuclear translocation of TFEB, i.e., the master gene for inducing autophagy. However, despite these, increased autophagy needed for protein aggregate clearance is not observed to occur. Our study reveals that the formation of highly viscous FUS condensates significantly impacts cytoskeletal/organelle function and cellular homeostasis, which are closely associated with cell ageing. This raises the intriguing question as to whether mutant FUS activates similar cell processes as those during cellular senescence.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.04.510756\", \"publication_date\": \"2022-10-04\"}, {\"work_id\": \"W4381431272\", \"doi_x\": \"10.1039/d3cp00479a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Limitations of non-polarizable force fields in describing anion binding poses in non-polar synthetic hosts\", \"abstract\": \"Transmembrane anion transport by synthetic ionophores has received increasing interest not only because of its relevance for understanding endogenous anion transport, but also because of potential implications for therapeutic routes in disease states where chloride transport is impaired. Computational studies can shed light on the binding recognition process and can deepen our mechanistic understanding of them. However, the ability of molecular mechanics methods to properly capture solvation and binding properties of anions is known to be challenging. Consequently, polarizable models have been suggested to improve the accuracy of such calculations. In this study, we calculate binding free energies for different anions to the synthetic ionophore, biotin[6]uril hexamethyl ester in acetonitrile and to biotin[6]uril hexaacid in water by employing non-polarizable and polarizable force fields. Anion binding shows strong solvent dependency consistent with experimental studies. In water, the binding strengths are iodide > bromide > chloride, and reversed in acetonitrile. These trends are well captured by both classes of force fields. However, the free energy profiles obtained from potential of mean force calculations and preferred binding positions of anions depend on the treatment of electrostatics. Results from simulations using the AMOEBA force-field, which recapitulate the observed binding positions, suggest strong effects from multipoles dominate with a smaller contribution from polarization. The oxidation status of the macrocycle was also found to influence anion recognition in water. Overall, these results have implications for the understanding of anion host interactions not just in synthetic ionophores, but also in narrow cavities of biological ion channels.\", \"doi_y\": \"https://doi.org/10.1039/d3cp00479a\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4282925771\", \"doi_x\": \"10.1101/2022.06.10.22276179\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Saturation genome editing of DDX3X clarifies pathogenicity of germline and somatic variation\", \"abstract\": \"Abstract Loss-of-function of DDX3X is a leading cause of neurodevelopmental disorders (NDD) in females. DDX3X is also a somatically mutated cancer driver gene proposed to have tumour promoting and suppressing effects. We performed saturation genome editing of DDX3X, testing in vitro the functional impact of 12,776 nucleotide variants. We identified 3,432 functionally abnormal variants, in three distinct classes. We trained a machine learning classifier to identify functionally abnormal variants of NDD-relevance. This classifier has at least 97% sensitivity and 99% specificity to detect variants pathogenic for NDD, substantially out-performing in silico predictors, and resolving up to 93% of variants of uncertain significance. Moreover, functionally-abnormal variants could account for almost all of the excess nonsynonymous DDX3X somatic mutations seen in DDX3X - driven cancers. Systematic maps of variant effects generated in experimentally tractable cell types have the potential to transform clinical interpretation of both germline and somatic disease-associated variation.\", \"doi_y\": \"https://doi.org/10.1101/2022.06.10.22276179\", \"publication_date\": \"2022-06-14\"}, {\"work_id\": \"W4307903228\", \"doi_x\": \"10.1016/j.jbc.2022.102659\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Dimers of D76N-\\u03b22-microglobulin display potent antiamyloid aggregation activity\", \"abstract\": \"Self-association of WT \\u03b22-microglobulin (WT-\\u03b22m) into amyloid fibrils is associated with the disorder dialysis related amyloidosis. In the familial variant D76N-\\u03b22m, the single amino acid substitution enhances the aggregation propensity of the protein dramatically and gives rise to a disorder that is independent of renal dysfunction. Numerous biophysical and structural studies on WT- and D76N-\\u03b22m have been performed in order to better understand the structure and dynamics of the native proteins and their different potentials to aggregate into amyloid. However, the structural properties of transient D76N-\\u03b22m oligomers and their role(s) in assembly remained uncharted. Here, we have utilized NMR methods, combined with photo-induced crosslinking, to detect, trap, and structurally characterize transient dimers of D76N-\\u03b22m. We show that the crosslinked D76N-\\u03b22m dimers have different structures from those previously characterized for the on-pathway dimers of \\u0394N6-\\u03b22m and are unable to assemble into amyloid. Instead, the crosslinked D76N-\\u03b22m dimers are potent inhibitors of amyloid formation, preventing primary nucleation and elongation/secondary nucleation when added in substoichiometric amounts with D76N-\\u03b22m monomers. The results highlight the specificity of early protein-protein interactions in amyloid formation and show how mapping these interfaces can inform new strategies to inhibit amyloid assembly.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102659\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4280630195\", \"doi_x\": \"10.7554/elife.79310\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Lighting up protein design\", \"abstract\": \"Using a neural network to predict how green fluorescent proteins respond to genetic mutations illuminates properties that could help design new proteins.\", \"doi_y\": \"https://doi.org/10.7554/elife.79310\", \"publication_date\": \"2022-05-19\"}, {\"work_id\": \"W4205296878\", \"doi_x\": \"10.1101/2022.01.18.476751\", \"af_direct_citation\": 1, \"af_indirect_citations\": 6, \"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"abstract\": \"Abstract In the recent CASP (Critical Assessment of Structure Prediction) competition, AlphaFold2 performed outstandingly. Its worst predictions were for NMR structures, which has two alternative explanations: either the NMR structures were poor, implying that AlphaFold may be more accurate than NMR; or there is a genuine difference between crystal and solution structures. Here, we use the program ANSURR, which measures the accuracy of solution structures, and show that one of the NMR structures was indeed poor. We then compare AlphaFold predictions to NMR structures, and show that AlphaFold tends to be more accurate than NMR ensembles, in particular correctly more rigid in loops. There are however some cases where the NMR ensembles are more accurate. These tend to be dynamic structures where AlphaFold had low confidence. We suggest that AlphaFold could be used as the model for NMR structure refinements, and that AlphaFold structures validated by ANSURR require no further refinement.\", \"doi_y\": \"https://doi.org/10.1101/2022.01.18.476751\", \"publication_date\": \"2022-01-20\"}, {\"work_id\": \"W4322741830\", \"doi_x\": \"10.3390/genes14030615\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Review of Recurrently Mutated Genes in Craniosynostosis Supports Expansion of Diagnostic Gene Panels\", \"abstract\": \"Craniosynostosis, the premature fusion of the cranial sutures, affects ~1 in 2000 children. Although many patients with a genetically determined cause harbor a variant in one of just seven genes or have a chromosomal abnormality, over 60 genes are known to be recurrently mutated, thus comprising a long tail of rarer diagnoses. Genome sequencing for the diagnosis of rare diseases is increasingly used in clinical settings, but analysis of the data is labor intensive and involves a trade-off between achieving high sensitivity or high precision. PanelApp, a crowd-sourced disease-focused set of gene panels, was designed to enable prioritization of variants in known disease genes for a given pathology, allowing enhanced identification of true-positives. For heterogeneous disorders like craniosynostosis, these panels must be regularly updated to ensure that diagnoses are not being missed. We provide a systematic review of genetic literature on craniosynostosis over the last 5 years, including additional results from resequencing a 42-gene panel in 617 affected individuals. We identify 16 genes (representing a 25% uplift) that should be added to the list of bona fide craniosynostosis disease genes and discuss the insights that these new genes provide into pathophysiological mechanisms of craniosynostosis.\", \"doi_y\": \"https://doi.org/10.3390/genes14030615\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4385196381\", \"doi_x\": \"10.1073/pnas.2306046120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"An expandable, modular de novo protein platform for precision redox engineering\", \"abstract\": \"The electron-conducting circuitry of life represents an as-yet untapped resource of exquisite, nanoscale biomolecular engineering. Here, we report the characterization and structure of a de novo diheme \\u201cmaquette\\u201d protein, 4D2, which we subsequently use to create an expanded, modular platform for heme protein design. A well-folded monoheme variant was created by computational redesign, which was then utilized for the experimental validation of continuum electrostatic redox potential calculations. This demonstrates how fundamental biophysical properties can be predicted and fine-tuned. 4D2 was then extended into a tetraheme helical bundle, representing a 7 nm molecular wire. Despite a molecular weight of only 24 kDa, electron cryomicroscopy illustrated a remarkable level of detail, indicating the positioning of the secondary structure and the heme cofactors. This robust, expressible, highly thermostable and readily designable modular platform presents a valuable resource for redox protein design and the future construction of artificial electron-conducting circuitry.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2306046120\", \"publication_date\": \"2023-07-24\"}, {\"work_id\": \"W4303986696\", \"doi_x\": \"10.1128/mbio.02472-22\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Solanimycin: Biosynthesis and Distribution of a New Antifungal Antibiotic Regulated by Two Quorum-Sensing Systems\", \"abstract\": \"The increasing emergence of drug-resistant fungal infections has necessitated a search for new compounds capable of combating fungal pathogens of plants, animals, and humans. Microorganisms represent the main source of antibiotics with applicability in agriculture and in the clinic, but many aspects of their metabolic potential remain to be explored. This report describes the discovery and characterization of a new antifungal compound, solanimycin, produced by a hybrid polyketide/nonribosomal peptide (PKS/NRPS) system in Dickeya solani, the enterobacterial pathogen of potato. Solanimycin was active against a broad range of plant-pathogenic fungi of global economic concern and the human pathogen Candida albicans. The genomic cluster responsible for solanimycin production was defined and analyzed to identify the corresponding biosynthetic proteins, which include four multimodular PKS/NRPS proteins and several tailoring enzymes. Antifungal production in D. solani was enhanced in response to experimental conditions found in infected potato tubers and high-density fungal cultures. Solanimycin biosynthesis was cell density dependent in D. solani and was controlled by both the ExpIR acyl-homoserine lactone and Vfm quorum-sensing systems of the bacterial phytopathogen. The expression of the solanimycin cluster was also regulated at the post-transcriptional level, with the regulator RsmA playing a major role. The solanimycin biosynthetic cluster was conserved across phylogenetically distant bacterial genera, and multiple pieces of evidence support that the corresponding gene clusters were acquired by horizontal gene transfer. Given its potent broad-range antifungal properties, this study suggests that solanimycin and related molecules may have potential utility for agricultural and clinical exploitation. IMPORTANCE Fungal infections represent a major clinical, agricultural, and food security threat worldwide, which is accentuated due to the difficult treatment of these infections. Microorganisms represent a prolific source of antibiotics, and current data support that this enormous biosynthetic potential has been scarcely explored. To improve the performance in the discovery of novel antimicrobials, there is a need to diversify the isolation niches for new antibiotic-producing microorganisms as well as to scrutinize novel phylogenetic positions. With the identification of the antifungal antibiotic solanimycin in a broad diversity of phytopathogenic Dickeya spp., we provide further support for the potential of plant-associated bacteria for the biosynthesis of novel antimicrobials. The complex regulatory networks involved in solanimycin production reflect the high metabolic cost of bacterial secondary metabolism. This metabolic regulatory control makes many antibiotics cryptic under standard laboratory conditions, and mimicking environmental conditions, as shown here, is a strategy to activate cryptic antibiotic clusters.\", \"doi_y\": \"https://doi.org/10.1128/mbio.02472-22\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4381470302\", \"doi_x\": \"10.1128/spectrum.04858-22\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Absence of Staphylococcus aureus in Wild Populations of Fish Supports a Spillover Hypothesis\", \"abstract\": \"Staphylococcus aureus is a human and livestock commensal but also an important pathogen responsible for high human mortality rates and economic losses in farming. Recent studies show that S. aureus is common in wild animals, including fish.\", \"doi_y\": \"https://doi.org/10.1128/spectrum.04858-22\", \"publication_date\": \"2023-08-17\"}, {\"work_id\": \"W4294671716\", \"doi_x\": \"10.1021/acs.molpharmaceut.2c00453\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Understanding the Stabilizing Effect of Histidine on mAb Aggregation: A Molecular Dynamics Study\", \"abstract\": \"Histidine, a widely used buffer in monoclonal antibody (mAb) formulations, is known to reduce antibody aggregation. While experimental studies suggest a nonelectrostatic, nonstructural (relating to secondary structure preservation) origin of the phenomenon, the underlying microscopic mechanism behind the histidine action is still unknown. Understanding this mechanism will help evaluate and predict the stabilizing effect of this buffer under different experimental conditions and for different mAbs. We have used all-atom molecular dynamics simulations and contact-based free energy calculations to investigate molecular-level interactions between the histidine buffer and mAbs, which lead to the observed stability of therapeutic formulations in the presence of histidine. We reformulate the Spatial Aggregation Propensity index by including the buffer-protein interactions. The buffer adsorption on the protein surface leads to lower exposure of the hydrophobic regions to water. Our analysis indicates that the mechanism behind the stabilizing action of histidine is connected to the shielding of the solvent-exposed hydrophobic regions on the protein surface by the buffer molecules.\", \"doi_y\": \"https://doi.org/10.1021/acs.molpharmaceut.2c00453\", \"publication_date\": \"2022-08-10\"}, {\"work_id\": \"W4308634875\", \"doi_x\": \"10.1002/pmic.202200013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mass spectrometry\\u2010based high\\u2010throughput proteomics and its role in biomedical studies and systems biology\", \"abstract\": \"There are multiple reasons why the next generation of biological and medical studies require increasing numbers of samples. Biological systems are dynamic, and the effect of a perturbation depends on the genetic background and environment. As a consequence, many conditions need to be considered to reach generalizable conclusions. Moreover, human population and clinical studies only reach sufficient statistical power if conducted at scale and with precise measurement methods. Finally, many proteins remain without sufficient functional annotations, because they have not been systematically studied under a broad range of conditions. In this review, we discuss the latest technical developments in mass spectrometry (MS)-based proteomics that facilitate large-scale studies by fast and efficient chromatography, fast scanning mass spectrometers, data-independent acquisition (DIA), and new software. We further highlight recent studies which demonstrate how high-throughput (HT) proteomics can be applied to capture biological diversity, to annotate gene functions or to generate predictive and prognostic models for human diseases.\", \"doi_y\": \"https://doi.org/10.1002/pmic.202200013\", \"publication_date\": \"2022-11-18\"}, {\"work_id\": \"W4310948626\", \"doi_x\": \"10.1093/bioinformatics/btac793\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Annotation of biologically relevant ligands in UniProtKB using ChEBI\", \"abstract\": \"Abstract Motivation To provide high quality, computationally tractable annotation of binding sites for biologically relevant (cognate) ligands in UniProtKB using the chemical ontology ChEBI (Chemical Entities of Biological Interest), to better support efforts to study and predict functionally relevant interactions between protein sequences and structures and small molecule ligands. Results We structured the data model for cognate ligand binding site annotations in UniProtKB and performed a complete reannotation of all cognate ligand binding sites using stable unique identifiers from ChEBI, which we now use as the reference vocabulary for all such annotations. We developed improved search and query facilities for cognate ligands in the UniProt website, REST API and SPARQL endpoint that leverage the chemical structure data, nomenclature and classification that ChEBI provides. Availability and implementation Binding site annotations for cognate ligands described using ChEBI are available for UniProtKB protein sequence records in several formats (text, XML and RDF) and are freely available to query and download through the UniProt website (www.uniprot.org), REST API (www.uniprot.org/help/api), SPARQL endpoint (sparql.uniprot.org/) and FTP site (https://ftp.uniprot.org/pub/databases/uniprot/). Supplementary information Supplementary data are available at Bioinformatics online.\", \"doi_y\": \"https://doi.org/10.1093/bioinformatics/btac793\", \"publication_date\": \"2022-12-09\"}, {\"work_id\": \"W4385778709\", \"doi_x\": \"10.14814/phy2.15793\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Stanniocalcin\\u20102 inhibits skeletal muscle growth and is upregulated in functional overload\\u2010induced hypertrophy\", \"abstract\": \"Stanniocalcin-2 (STC2) has recently been implicated in human muscle mass variability by genetic analysis. Biochemically, STC2 inhibits the proteolytic activity of the metalloproteinase PAPP-A, which promotes muscle growth by upregulating the insulin-like growth factor (IGF) axis. The aim was to examine if STC2 affects skeletal muscle mass and to assess how the IGF axis mediates muscle hypertrophy induced by functional overload.We compared muscle mass and muscle fiber morphology between Stc2-/- (n = 21) and wild-type (n = 15) mice. We then quantified IGF1, IGF2, IGF binding proteins -4 and -5 (IGFBP-4, IGFBP-5), PAPP-A and STC2 in plantaris muscles of wild-type mice subjected to 4-week unilateral overload (n = 14).Stc2-/- mice showed up to 10% larger muscle mass compared with wild-type mice. This increase was mediated by greater cross-sectional area of muscle fibers. Overload increased plantaris mass and components of the IGF axis, including quantities of IGF1 (by 2.41-fold, p = 0.0117), IGF2 (1.70-fold, p = 0.0461), IGFBP-4 (1.48-fold, p = 0.0268), PAPP-A (1.30-fold, p = 0.0154) and STC2 (1.28-fold, p = 0.019).Here we provide evidence that STC2 is an inhibitor of muscle growth upregulated, along with other components of the IGF axis, during overload-induced muscle hypertrophy.\", \"doi_y\": \"https://doi.org/10.14814/phy2.15793\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4380291584\", \"doi_x\": \"10.1042/bcj20230183\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Classification of Cushing's syndrome PKAc mutants based upon their ability to bind PKI\", \"abstract\": \"Cushing's syndrome is an endocrine disorder caused by excess production of the stress hormone cortisol. Precision medicine strategies have identified single allele mutations within the PRKACA gene that drive adrenal Cushing's syndrome. These mutations promote perturbations in the catalytic core of protein kinase A (PKAc) that impair autoinhibition by regulatory subunits and compartmentalization via recruitment into AKAP signaling islands. PKAcL205R is found in \\u223c45% of patients, whereas PKAcE31V, PKAcW196R, and L198insW and C199insV insertion mutants are less prevalent. Mass spectrometry, cellular, and biochemical data indicate that Cushing's PKAc variants fall into two categories: those that interact with the heat-stable protein kinase inhibitor PKI, and those that do not. In vitro activity measurements show that wild-type PKAc and W196R activities are strongly inhibited by PKI (IC50 < 1 nM). In contrast, PKAcL205R activity is not blocked by the inhibitor. Immunofluorescent analyses show that the PKI-binding variants wild-type PKAc, E31V, and W196R are excluded from the nucleus and protected against proteolytic processing. Thermal stability measurements reveal that upon co-incubation with PKI and metal-bound nucleotide, the W196R variant tolerates melting temperatures 10\\u00b0C higher than PKAcL205. Structural modeling maps PKI-interfering mutations to a \\u223c20 \\u00c5 diameter area at the active site of the catalytic domain that interfaces with the pseudosubstrate of PKI. Thus, Cushing's kinases are individually controlled, compartmentalized, and processed through their differential association with PKI.\", \"doi_y\": \"https://doi.org/10.1042/bcj20230183\", \"publication_date\": \"2023-06-22\"}, {\"work_id\": \"W4282583104\", \"doi_x\": \"10.1002/cbic.202200171\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"abstract\": \"The carbon backbone of biotin is constructed from the C7 di-acid pimelate, which is converted to an acyl-CoA thioester by an ATP-dependent, pimeloyl-CoA synthetase (PCAS, encoded by BioW). The acyl-thioester is condensed with \\u029f-alanine in a decarboxylative, Claisen-like reaction to form an aminoketone (8-amino-7-oxononanoic acid, AON). This step is catalysed by the pyridoxal 5'-phosphate (PLP)-dependent enzyme (AON synthase, AONS, encoded by BioF). Distinct versions of Bacillus subtilis BioW (BsBioW) and E. coli BioF (EcBioF) display strict substrate specificity. In contrast, a BioW-BioF fusion from Corynebacterium amycolatum (CaBioWF) accepts a wider range of mono- and di-fatty acids. Analysis of the active site of the BsBioW : pimeloyl-adenylate complex suggested a key role for a Phe (F192) residue in the CaBioW domain; a F192Y mutant restored the substrate specificity to pimelate. This surprising substrate flexibility also extends to the CaBioF domain, which accepts \\u029f-alanine, \\u029f-serine and glycine. Structural models of the CaBioWF fusion provide insight into how both domains interact with each other and suggest the presence of an intra-domain tunnel. The CaBioWF fusion catalyses conversion of various fatty acids and amino acids to a range of AON derivatives. Such unexpected, natural broad substrate scope suggests that the CaBioWF fusion is a versatile biocatalyst that can be used to prepare a number of aminoketone analogues.\", \"doi_y\": \"https://doi.org/10.1002/cbic.202200171\", \"publication_date\": \"2022-07-13\"}, {\"work_id\": \"W4307523478\", \"doi_x\": \"10.1101/2022.10.19.22280888\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evidence of epistasis in regions of long-range linkage disequilibrium across five complex diseases in the UK Biobank and eMERGE datasets\", \"abstract\": \"Abstract Leveraging linkage disequilibrium (LD) patterns as representative of population substructure enables the discovery of additive association signals in genome-wide association studies (GWAS). Standard GWAS are well-powered to interrogate additive models; however, new approaches are required to investigate other modes of inheritance such as dominance and epistasis. Epistasis, or non-additive interaction between genes, exists across the genome but often goes undetected due to lack of statistical power. Furthermore, the adoption of LD pruning as customary in standard GWAS excludes detection of sites in LD that may underlie the genetic architecture of complex traits. We hypothesize that uncovering long-range interactions between loci with strong LD due to epistatic selection can elucidate genetic mechanisms underlying common diseases. To investigate this hypothesis, we tested for associations between 23 common diseases and 5,625,845 epistatic SNP-SNP pairs (determined by Ohta\\u2019s D statistics) in long-range LD (&gt; 0.25cM). We identified five significant associations across five disease phenotypes that replicated in two large genotype-phenotype datasets (UK Biobank and eMERGE). The genes that were most likely involved in the replicated associations were 1) members of highly conserved gene families with complex roles in multiple pathways, 2) essential genes, and/or 3) associated in the literature with complex traits that display variable expressivity. These results support the highly pleiotropic and conserved nature of variants in long-range under epistatic selection. Our work supports the hypothesis that epistatic interactions regulate diverse clinical mechanisms and may especially be driving factors in conditions with a wide range of phenotypic outcomes. Significance Current knowledge of genotype-phenotype relationships is largely contingent on traditional univariate approaches to genomic analysis. Yet substantial evidence supports non-additive modes of inheritance and regulation, such as epistasis, as being abundant across the genome. In this genome-wide study, we probe the biomolecular mechanisms underlying complex human diseases by testing the association of pairwise genetic interactions with disease occurrence in large-scale biobank data. Specifically, we tested intrachromosomal and interchrosomal long-range interactions between regions of the genome in high linkage disequilibrium, these regions are typically excluded from genomic analyses. The results from this study suggest that essential gene, members of highly conserved gene families, and phenotypes with variable expressivity, are particularly enriched with epistatic and pleiotropic activity.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.19.22280888\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4385684007\", \"doi_x\": \"10.1098/rsif.2023.0187\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Modelling the viral dynamics of the SARS-CoV-2 Delta and Omicron variants in different cell types\", \"abstract\": \"We use viral kinetic models fitted to viral load data from in vitro studies to explain why the SARS-CoV-2 Omicron variant replicates faster than the Delta variant in nasal cells, but slower than Delta in lung cells, which could explain Omicron's higher transmission potential and lower severity. We find that in both nasal and lung cells, viral infectivity is higher for Omicron but the virus production rate is higher for Delta, with an estimated approximately 200-fold increase in infectivity and 100-fold decrease in virus production when comparing Omicron with Delta in nasal cells. However, the differences are unequal between cell types, and ultimately lead to the basic reproduction number and growth rate being higher for Omicron in nasal cells, and higher for Delta in lung cells. In nasal cells, Omicron alone can enter via a TMPRSS2-independent pathway, but it is primarily increased efficiency of TMPRSS2-dependent entry which accounts for Omicron's increased activity. This work paves the way for using within-host mathematical models to understand the transmission potential and severity of future variants.\", \"doi_y\": \"https://doi.org/10.1098/rsif.2023.0187\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4310464829\", \"doi_x\": \"10.1039/d2cc04823j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Enzyme self-label-bound ATTO700 in single-molecule and super-resolution microscopy\", \"abstract\": \"Near-infrared ATTO700 is evaluated as an acceptor in SNAP- and Halo-tag protein labelling for ensemble and single molecule F\\u00f6rster Resonance Energy Transfer (FRET) and in stimulated emission by depletion (STED) super resolution microscopy.\", \"doi_y\": \"https://doi.org/10.1039/d2cc04823j\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4309247386\", \"doi_x\": \"10.15252/embj.2022111898\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structural and biochemical basis of interdependent <scp>FANCI\\u2010FANCD2</scp> ubiquitination\", \"abstract\": \"Abstract Di\\u2010monoubiquitination of the FANCI\\u2010FANCD2 (ID2) complex is a central and crucial step for the repair of DNA interstrand crosslinks via the Fanconi anaemia pathway. While FANCD2 ubiquitination precedes FANCI ubiquitination, FANCD2 is also deubiquitinated at a faster rate than FANCI, which can result in a FANCI\\u2010ubiquitinated ID2 complex (I Ub D2). Here, we present a 4.1 \\u00c5 cryo\\u2010EM structure of I Ub D2 complex bound to double\\u2010stranded DNA. We show that this complex, like ID2 Ub and I Ub D2 Ub , is also in the closed ID2 conformation and clamps on DNA. The target lysine of FANCD2 (K561) becomes fully exposed in the I Ub D2\\u2010DNA structure and is thus primed for ubiquitination. Similarly, FANCI's target lysine (K523) is also primed for ubiquitination in the ID2 Ub \\u2010DNA complex. The I Ub D2\\u2010DNA complex exhibits deubiquitination resistance, conferred by the presence of DNA and FANCD2. ID2 Ub \\u2010DNA, on the other hand, can be efficiently deubiquitinated by USP1\\u2010UAF1, unless further ubiquitination on FANCI occurs. Therefore, FANCI ubiquitination effectively maintains FANCD2 ubiquitination in two ways: it prevents excessive FANCD2 deubiquitination within an I Ub D2 Ub \\u2010DNA complex, and it enables re\\u2010ubiquitination of FANCD2 within a transient, closed\\u2010on\\u2010DNA, I Ub D2 complex.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111898\", \"publication_date\": \"2022-11-17\"}, {\"work_id\": \"W4385782400\", \"doi_x\": \"10.1016/j.funbio.2023.08.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fungal spore swelling and germination are restricted by the macrophage phagolysosome\", \"abstract\": \"Many species of medically important fungi are prolific in the formation of asexual spores. Spores undergo a process of active swelling and cell wall remodelling before a germ tube is formed and filamentous growth ensues. Highly elongated germ tubes are known to be difficult to phagocytose and pose particular challenges for immune phagocytes. However, the significance of the earliest stages of spore germination during immune cell interactions has not been investigated and yet this is likely to be important for defence against sporogenous fungal pathogens. We show here that macrophages restrict the early phases of the spore germination process of Aspergillus fumigatus and Mucor circinelloides including the initial phase of spore swelling, spore germination and early polarised growth. Macrophages are therefore adept at retarding germination as well as subsequent vegetative growth which is likely to be critical for immune surveillance and protection against sporulating fungi.\", \"doi_y\": \"https://doi.org/10.1016/j.funbio.2023.08.002\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4309657977\", \"doi_x\": \"10.7554/elife.81167\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Revealing druggable cryptic pockets in the Nsp1 of SARS-CoV-2 and other \\u03b2-coronaviruses by simulations and crystallography\", \"abstract\": \"Non-structural protein 1 (Nsp1) is a main pathogenicity factor of \\u03b1 - and \\u03b2 - coronaviruses. Nsp1 of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) suppresses the host gene expression by sterically blocking 40S host ribosomal subunits and promoting host mRNA degradation. This mechanism leads to the downregulation of the translation-mediated innate immune response in host cells, ultimately mediating the observed immune evasion capabilities of SARS-CoV-2. Here, by combining extensive molecular dynamics simulations, fragment screening and crystallography, we reveal druggable pockets in Nsp1. Structural and computational solvent mapping analyses indicate the partial crypticity of these newly discovered and druggable binding sites. The results of fragment-based screening via X-ray crystallography confirm the druggability of the major pocket of Nsp1. Finally, we show how the targeting of this pocket could disrupt the Nsp1-mRNA complex and open a novel avenue to design new inhibitors for other Nsp1s present in homologous \\u03b2 - coronaviruses.\", \"doi_y\": \"https://doi.org/10.7554/elife.81167\", \"publication_date\": \"2022-11-22\"}, {\"work_id\": \"W4283021365\", \"doi_x\": \"10.1002/pmic.202100394\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Advances in ionisation techniques for mass spectrometry\\u2010based omics research\", \"abstract\": \"Abstract Omics analysis by mass spectrometry (MS) is a vast field, with proteomics, metabolomics and lipidomics dominating recent research by exploiting biological MS ionisation techniques. Traditional MS ionisation techniques such as electrospray ionisation have limitations in analyte\\u2010specific sensitivity, modes of sampling and throughput, leading to many researchers investigating new ionisation methods for omics research. In this review, we examine the current landscape of these new ionisation techniques, divided into the three groups of (electro)spray\\u2010based, laser\\u2010based and other miscellaneous ionisation techniques. Due to the wide range of new developments, this review can only provide a starting point for further reading on each ionisation technique, as each have unique benefits, often for specialised applications, which promise beneficial results for different areas in the omics world.\", \"doi_y\": \"https://doi.org/10.1002/pmic.202100394\", \"publication_date\": \"2022-06-26\"}, {\"work_id\": \"W4303628909\", \"doi_x\": \"10.3390/cells11193154\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"A Novel Homozygous Founder Variant of RTN4IP1 in Two Consanguineous Saudi Families\", \"abstract\": \"The genetic architecture of mitochondrial disease continues to expand and currently exceeds more than 350 disease-causing genes. Bi-allelic variants in RTN4IP1, also known as Optic Atrophy-10 (OPA10), lead to early-onset recessive optic neuropathy, atrophy, and encephalopathy in the afflicted patients. The gene is known to encode a mitochondrial ubiquinol oxidoreductase that interacts with reticulon 4 and is thought to be a mitochondrial antioxidant NADPH oxidoreductase. Here, we describe two unrelated consanguineous families from the northern region of Saudi Arabia harboring a missense variant (RTN4IP1:NM_032730.5; c.475G&lt;T, p.Val159Phe) in the gene. Clinically affected individuals presented with intellectual disability, encephalopathy, ataxia, optic atrophy, and seizures. Based on whole exome sequencing and confirmatory Sanger sequencing, the variant was fully segregated with the phenotype in the families, absent among large ethnically matching controls as well as numerous in-house exomes, and predicted to be pathogenic by different in silico classifiers. Structural modeling and immunoblot analyses strongly indicated this variant to be pathogenic. Since the families belong to one of the tribal inhabitants of Saudi Arabia, we postulate that the variant is likely to be a founder. We provide the estimated age of the variant and present data confirming the disease-causality of this founder variant.\", \"doi_y\": \"https://doi.org/10.3390/cells11193154\", \"publication_date\": \"2022-10-07\"}, {\"work_id\": \"W4307953698\", \"doi_x\": \"10.1038/s41579-022-00809-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"SARS-CoV-2 variant evasion of monoclonal antibodies based on in vitro studies\", \"abstract\": \"Monoclonal antibodies (mAbs) offer a treatment option for individuals with severe COVID-19 and are especially important in high-risk individuals where vaccination is not an option. Given the importance of understanding the evolution of resistance to mAbs by SARS-CoV-2, we reviewed the available in vitro neutralization data for mAbs against live variants and viral constructs containing spike mutations of interest. Unfortunately, evasion of mAb-induced protection is being reported with new SARS-CoV-2 variants. The magnitude of neutralization reduction varied greatly among mAb-variant pairs. For example, sotrovimab retained its neutralization capacity against Omicron BA.1 but showed reduced efficacy against BA.2, BA.4 and BA.5, and BA.2.12.1. At present, only bebtelovimab has been reported to retain its efficacy against all SARS-CoV-2 variants considered here. Resistance to mAb neutralization was dominated by the action of epitope single amino acid substitutions in the spike protein. Although not all observed epitope mutations result in increased mAb evasion, amino acid substitutions at non-epitope positions and combinations of mutations also contribute to evasion of neutralization. This Review highlights the implications for the rational design of viral genomic surveillance and factors to consider for the development of novel mAb therapies.\", \"doi_y\": \"https://doi.org/10.1038/s41579-022-00809-7\", \"publication_date\": \"2022-10-28\"}, {\"work_id\": \"W4378782355\", \"doi_x\": \"10.1002/prot.26533\", \"af_direct_citation\": 1, \"af_indirect_citations\": 10, \"title\": \"To split or not to split: <scp>CASP15</scp> targets and their processing into tertiary structure evaluation units\", \"abstract\": \"Abstract Processing of CASP15 targets into evaluation units (EUs) and assigning them to evolutionary\\u2010based prediction classes is presented in this study. The targets were first split into structural domains based on compactness and similarity to other proteins. Models were then evaluated against these domains and their combinations. The domains were joined into larger EUs if predictors\\u2019 performance on the combined units was similar to that on individual domains. Alternatively, if most predictors performed better on the individual domains, then they were retained as EUs. As a result, 112 evaluation units were created from 77 tertiary structure prediction targets. The EUs were assigned to four prediction classes roughly corresponding to target difficulty categories in previous CASPs: TBM (template\\u2010based modeling, easy or hard), FM (free modeling), and the TBM/FM overlap category. More than a third of CASP15 EUs were attributed to the historically most challenging FM class, where homology or structural analogy to proteins of known fold cannot be detected.\", \"doi_y\": \"https://doi.org/10.1002/prot.26533\", \"publication_date\": \"2023-05-31\"}, {\"work_id\": \"W4282929382\", \"doi_x\": \"10.1111/mmi.14953\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"abstract\": \"Gram-negative pathogens like Burkholderia pseudomallei use trimeric autotransporter adhesins such as BpaC as key molecules in their pathogenicity. Our 1.4 \\u00c5 crystal structure of the membrane-proximal part of the BpaC head domain shows that the domain is exclusively made of left-handed parallel \\u03b2-roll repeats. This, the largest such structure solved, has two unique features. First, the core, rather than being composed of the canonical hydrophobic Ile and Val, is made up primarily of the hydrophilic Thr and Asn, with two different solvent channels. Second, comparing BpaC to all other left-handed parallel \\u03b2-roll structures showed that the position of the head domain in the protein correlates with the number and type of charged residues. In BpaC, only negatively charged residues face the solvent-in stark contrast to the primarily positive surface charge of the left-handed parallel \\u03b2-roll \\\"type\\\" protein, YadA. We propose extending the definitions of these head domains to include the BpaC-like head domain as a separate subtype, based on its unusual sequence, position, and charge. We speculate that the function of left-handed parallel \\u03b2-roll structures may differ depending on their position in the structure.\", \"doi_y\": \"https://doi.org/10.1111/mmi.14953\", \"publication_date\": \"2022-07-01\"}, {\"work_id\": \"W4308458589\", \"doi_x\": \"10.1038/s41567-022-01808-4\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Enhanced statistical sampling reveals microscopic complexity in the talin mechanosensor folding energy landscape\", \"abstract\": \"Statistical mechanics can describe the major conformational ensembles determining the equilibrium free-energy landscape of a folding protein. The challenge is to capture the full repertoire of low-occurrence conformations separated by high kinetic barriers that define complex landscapes. Computationally, enhanced sampling methods accelerate the exploration of molecular rare events. However, accessing the entire protein's conformational space in equilibrium experiments requires technological developments to enable extended observation times. We developed single-molecule magnetic tweezers to capture over a million individual transitions as a single talin protein unfolds and refolds under force in equilibrium. When observed at classically-probed timescales, talin folds in an apparently uncomplicated two-state manner. As the sampling time extends from minutes to days, the underlying energy landscape exhibits gradually larger signatures of complexity, involving a finite number of well-defined rare conformations. A fluctuation analysis allows us to propose plausible structures of each low-probability conformational state. The physiological relevance of each distinct conformation can be connected to the binding of the cytoskeletal protein vinculin, suggesting an extra layer of complexity in talin-mediated mechanotransduction. More generally, our experiments directly test the fundamental notion that equilibrium dynamics depend on the observation timescale.\", \"doi_y\": \"https://doi.org/10.1038/s41567-022-01808-4\", \"publication_date\": \"2022-11-07\"}, {\"work_id\": \"W4293237544\", \"doi_x\": \"10.1101/2022.08.25.505150\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Full structure/function analysis of all the pilin subunits in a type 4 pilus: a complex of minor pilins in <i>Streptococcus sanguinis</i> mediates binding to glycans\", \"abstract\": \"Abstract Type 4 filaments (T4F) \\u2013 of which type 4 pili (T4P) are the archetype \\u2013 are a superfamily of filamentous nanomachines nearly ubiquitous in prokaryotes. T4F are polymers of one major pilin that also contain minor pilins whose roles are often poorly understood. Here, we complete the structure/function analysis of the full set of T4P pilins in the opportunistic pathogen Streptococcus sanguinis . We determined the structure of the minor pilin PilA, which is unexpectedly similar to one of the subunits of a tip-located complex of four minor pilins, widely conserved in T4F. We found that PilA interacts and dramatically stabilises the minor pilin PilC. We determined the structure of PilC, showing that it is a modular pilin with a lectin module binding a specific subset of glycans prevalent in the human glycome, the host of S. sanguinis . Altogether, our findings support a model whereby the minor pilins in S. sanguinis T4P form a tip-located complex promoting adhesion to various host receptors. Our findings have general implications for a group of minor pilins widely conserved in T4F.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.25.505150\", \"publication_date\": \"2022-08-25\"}, {\"work_id\": \"W4386001747\", \"doi_x\": \"10.1038/s42003-023-05239-6\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"abstract\": \"Abstract Insulin-like Growth Factor-2 (IGF2) is important for the regulation of human embryonic growth and development, and for adults\\u2019 physiology. Incorrect processing of the IGF2 precursor, pro-IGF2(156), leads to the formation of two IGF2 proforms, big-IGF2(87) and big-IGF2(104). Unprocessed and mainly non-glycosylated IGF2 proforms are found at abnormally high levels in certain diseases, but their mode of action is still unclear. Here, we found that pro-IGF2(156) has the lowest ability to form its inactivating complexes with IGF-Binding Proteins and has higher proliferative properties in cells than IGF2 and other IGF prohormones. We also showed that big-IGF2(104) has a seven-fold higher binding affinity for the IGF2 receptor than IGF2, and that pro-IGF2(87) binds and activates specific receptors and stimulates cell growth similarly to the mature IGF2. The properties of these pro-IGF2 forms, especially of pro-IGF2(156) and big-IGF2(104), indicate them as hormones that may be associated with human diseases related to the accumulation of IGF-2 proforms in the circulation.\", \"doi_y\": \"https://doi.org/10.1038/s42003-023-05239-6\", \"publication_date\": \"2023-08-19\"}, {\"work_id\": \"W4294311651\", \"doi_x\": \"10.1021/acs.biomac.2c00781\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Probing the Internal pH and Permeability of a Carboxysome Shell\", \"abstract\": \"The carboxysome is a protein-based nanoscale organelle in cyanobacteria and many proteobacteria, which encapsulates the key CO2-fixing enzymes ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco) and carbonic anhydrase (CA) within a polyhedral protein shell. The intrinsic self-assembly and architectural features of carboxysomes and the semipermeability of the protein shell provide the foundation for the accumulation of CO2 within carboxysomes and enhanced carboxylation. Here, we develop an approach to determine the interior pH conditions and inorganic carbon accumulation within an \\u03b1-carboxysome shell derived from a chemoautotrophic proteobacterium Halothiobacillus neapolitanus and evaluate the shell permeability. By incorporating a pH reporter, pHluorin2, within empty \\u03b1-carboxysome shells produced in Escherichia coli, we probe the interior pH of the protein shells with and without CA. Our in vivo and in vitro results demonstrate a lower interior pH of \\u03b1-carboxysome shells than the cytoplasmic pH and buffer pH, as well as the modulation of the interior pH in response to changes in external environments, indicating the shell permeability to bicarbonate ions and protons. We further determine the saturated HCO3\\u2013 concentration of 15 mM within \\u03b1-carboxysome shells and show the CA-mediated increase in the interior CO2 level. Uncovering the interior physiochemical microenvironment of carboxysomes is crucial for understanding the mechanisms underlying carboxysomal shell permeability and enhancement of Rubisco carboxylation within carboxysomes. Such fundamental knowledge may inform reprogramming carboxysomes to improve metabolism and recruit foreign enzymes for enhanced catalytical performance.\", \"doi_y\": \"https://doi.org/10.1021/acs.biomac.2c00781\", \"publication_date\": \"2022-09-02\"}, {\"work_id\": \"W4321099736\", \"doi_x\": \"10.1016/j.medj.2023.02.004\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evolution of long-term vaccine-induced and hybrid immunity in healthcare workers after different COVID-19 vaccine regimens\", \"abstract\": \"Both infection and vaccination, alone or in combination, generate antibody and T cell responses against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). However, the maintenance of such responses-and hence protection from disease-requires careful characterization. In a large prospective study of UK healthcare workers (HCWs) (Protective Immunity from T Cells in Healthcare Workers [PITCH], within the larger SARS-CoV-2 Immunity and Reinfection Evaluation [SIREN] study), we previously observed that prior infection strongly affected subsequent cellular and humoral immunity induced after long and short dosing intervals of BNT162b2 (Pfizer/BioNTech) vaccination.Here, we report longer follow-up of 684 HCWs in this cohort over 6-9 months following two doses of BNT162b2 or AZD1222 (Oxford/AstraZeneca) vaccination and up to 6 months following a subsequent mRNA booster vaccination.We make three observations: first, the dynamics of humoral and cellular responses differ; binding and neutralizing antibodies declined, whereas T and memory B cell responses were maintained after the second vaccine dose. Second, vaccine boosting restored immunoglobulin (Ig) G levels; broadened neutralizing activity against variants of concern, including Omicron BA.1, BA.2, and BA.5; and boosted T cell responses above the 6-month level after dose 2. Third, prior infection maintained its impact driving larger and broader T cell responses compared with never-infected people, a feature maintained until 6 months after the third dose.Broadly cross-reactive T cell responses are well maintained over time-especially in those with combined vaccine and infection-induced immunity (\\\"hybrid\\\" immunity)-and may contribute to continued protection against severe disease.Department for Health and Social Care, Medical Research Council.\", \"doi_y\": \"https://doi.org/10.1016/j.medj.2023.02.004\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W3164905191\", \"doi_x\": \"10.1002/prot.26185\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Evaluation of model refinement in <scp>CASP14</scp>\", \"abstract\": \"We report here an assessment of the model refinement category of the 14th round of Critical Assessment of Structure Prediction (CASP14). As before, predictors submitted up to five ranked refinements, along with associated residue-level error estimates, for targets that had a wide range of starting quality. The ability of groups to accurately rank their submissions and to predict coordinate error varied widely. Overall, only four groups out-performed a \\\"na\\u00efve predictor\\\" corresponding to the resubmission of the starting model. Among the top groups, there are interesting differences of approach and in the spread of improvements seen: some methods are more conservative, others more adventurous. Some targets were \\\"double-barreled\\\" for which predictors were offered a high-quality AlphaFold 2 (AF2)-derived prediction alongside another of lower quality. The AF2-derived models were largely unimprovable, many of their apparent errors being found to reside at domain and, especially, crystal lattice contacts. Refinement is shown to have a mixed impact overall on structure-based function annotation methods to predict nucleic acid binding, spot catalytic sites, and dock protein structures.\", \"doi_y\": \"https://doi.org/10.1002/prot.26185\", \"publication_date\": \"2021-07-29\"}, {\"work_id\": \"W4376959094\", \"doi_x\": \"10.1038/s41467-023-38484-8\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"abstract\": \"Siderophores belonging to the ferrichrome family are essential for the viability of fungal species and play a key role for virulence of numerous pathogenic fungi. Despite their biological significance, our understanding of how these iron-chelating cyclic hexapeptides are assembled by non-ribosomal peptide synthetase (NRPS) enzymes remains poorly understood, primarily due to the nonlinearity exhibited by the domain architecture. Herein, we report the biochemical characterization of the SidC NRPS, responsible for construction of the intracellular siderophore ferricrocin. In vitro reconstitution of purified SidC reveals its ability to produce ferricrocin and its structural variant, ferrichrome. Application of intact protein mass spectrometry uncovers several non-canonical events during peptidyl siderophore biosynthesis, including inter-modular loading of amino acid substrates and an adenylation domain capable of poly-amide bond formation. This work expands the scope of NRPS programming, allows biosynthetic assignment of ferrichrome NRPSs, and sets the stage for reprogramming towards novel hydroxamate scaffolds.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-38484-8\", \"publication_date\": \"2023-05-17\"}, {\"work_id\": \"W4313829776\", \"doi_x\": \"10.1002/cpz1.639\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"An <i>Infectious Bursal Disease Virus</i> (IBDV) Reverse Genetics Rescue System and Neutralization Assay in Chicken B Cells\", \"abstract\": \"Infectious bursal disease virus (IBDV) is a major threat to the productivity of the poultry industry due to morbidity, mortality, and immunosuppression that exacerbates secondary infections and reduces the efficacy of vaccination programs. Field strains of IBDV have a preferred tropism for chicken B cells, the majority of which reside in the bursa of Fabricius (BF). IBDV adaptation to adherent cell culture is associated with mutations altering amino acids in the hypervariable region (HVR) of the capsid protein, which affects immunogenicity and virulence. Until recently, this has limited both the application of reverse genetics systems for engineering molecular clones, and the use of in vitro neutralization assays, to cell-culture-adapted strains of IBDV. Here, we describe the rescue of molecular clones of IBDV containing the HVR from diverse field strains, along with a neutralization assay to quantify antibody responses against the rescued viruses, both using chicken B cells. These methods are readily adaptable to any laboratory with molecular biology expertise and negate the need to obtain wild-type strains. \\u00a9 2023 The Authors. Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: A chicken B-cell rescue system for IBDV Basic Protocol 2: A chicken B-cell neutralization assay for IBDV.\", \"doi_y\": \"https://doi.org/10.1002/cpz1.639\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4386593063\", \"doi_x\": \"10.1002/bies.202300111\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evolution of sex\\u2010determination in dioecious plants: From active Y to X/A balance?\", \"abstract\": \"Sex chromosomes in plants have been known for a century, but only recently have we begun to understand the mechanisms behind sex determination in dioecious plants. Here, we discuss evolution of sex determination, focusing on Silene latifolia, where evolution of separate sexes is consistent with the classic \\\"two mutations\\\" model-a loss of function male sterility mutation and a gain of function gynoecium suppression mutation, which turned an ancestral hermaphroditic population into separate males and females. Interestingly, the gynoecium suppression function in S. latifolia evolved via loss of function in at least two sex-linked genes and works via gene dosage balance between sex-linked, and autosomal genes. This system resembles X/A-ratio-based sex determination systems in Drosophila and Rumex, and could represent a steppingstone in the evolution of X/A-ratio-based sex determination from an active Y system.\", \"doi_y\": \"https://doi.org/10.1002/bies.202300111\", \"publication_date\": \"2023-09-11\"}, {\"work_id\": \"W4324376276\", \"doi_x\": \"10.3389/fimmu.2023.1146702\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 antibody responses associate with sex, age and disease severity in previously uninfected people admitted to hospital with COVID-19: An ISARIC4C prospective study\", \"abstract\": \"The SARS-CoV-2 pandemic enables the analysis of immune responses induced against a novel coronavirus infecting immunologically na\\u00efve individuals. This provides an opportunity for analysis of immune responses and associations with age, sex and disease severity. Here we measured an array of solid-phase binding antibody and viral neutralising Ab (nAb) responses in participants (n=337) of the ISARIC4C cohort and characterised their correlation with peak disease severity during acute infection and early convalescence. Overall, the responses in a Double Antigen Binding Assay (DABA) for antibody to the receptor binding domain (anti-RBD) correlated well with IgM as well as IgG responses against viral spike, S1 and nucleocapsid protein (NP) antigens. DABA reactivity also correlated with nAb. As we and others reported previously, there is greater risk of severe disease and death in older men, whilst the sex ratio was found to be equal within each severity grouping in younger people. In older males with severe disease (mean age 68 years), peak antibody levels were found to be delayed by one to two weeks compared with women, and nAb responses were delayed further. Additionally, we demonstrated that solid-phase binding antibody responses reached higher levels in males as measured via DABA and IgM binding against Spike, NP and S1 antigens. In contrast, this was not observed for nAb responses. When measuring SARS-CoV-2 RNA transcripts (as a surrogate for viral shedding) in nasal swabs at recruitment, we saw no significant differences by sex or disease severity status. However, we have shown higher antibody levels associated with low nasal viral RNA indicating a role of antibody responses in controlling viral replication and shedding in the upper airway. In this study, we have shown discernible differences in the humoral immune responses between males and females and these differences associate with age as well as with resultant disease severity.\", \"doi_y\": \"https://doi.org/10.3389/fimmu.2023.1146702\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4308616064\", \"doi_x\": \"10.1093/nar/gkac993\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"InterPro in 2022\", \"abstract\": \"Abstract The InterPro database (https://www.ebi.ac.uk/interpro/) provides an integrative classification of protein sequences into families, and identifies functionally important domains and conserved sites. Here, we report recent developments with InterPro (version 90.0) and its associated software, including updates to data content and to the website. These developments extend and enrich the information provided by InterPro, and provide a more user friendly access to the data. Additionally, we have worked on adding Pfam website features to the InterPro website, as the Pfam website will be retired in late 2022. We also show that InterPro's sequence coverage has kept pace with the growth of UniProtKB. Moreover, we report the development of a card game as a method of engaging the non-scientific community. Finally, we discuss the benefits and challenges brought by the use of artificial intelligence for protein structure prediction.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac993\", \"publication_date\": \"2022-11-09\"}, {\"work_id\": \"W4362520906\", \"doi_x\": \"10.7554/elife.86049\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"abstract\": \"The phylum of Apicomplexa groups intracellular parasites that employ substrate-dependent gliding motility to invade host cells, egress from the infected cells, and cross biological barriers. The glideosome-associated connector (GAC) is a conserved protein essential to this process. GAC facilitates the association of actin filaments with surface transmembrane adhesins and the efficient transmission of the force generated by myosin translocation of actin to the cell surface substrate. Here, we present the crystal structure of Toxoplasma gondii GAC and reveal a unique, supercoiled armadillo repeat region that adopts a closed ring conformation. Characterisation of the solution properties together with membrane and F-actin binding interfaces suggests that GAC adopts several conformations from closed to open and extended. A multi-conformational model for assembly and regulation of GAC within the glideosome is proposed.\", \"doi_y\": \"https://doi.org/10.7554/elife.86049\", \"publication_date\": \"2023-04-04\"}, {\"work_id\": \"W4289528710\", \"doi_x\": \"10.1016/j.celrep.2022.111220\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 spike N-terminal domain modulates TMPRSS2-dependent viral entry and fusogenicity\", \"abstract\": \"The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike N-terminal domain (NTD) remains poorly characterized despite enrichment of mutations in this region across variants of concern (VOCs). Here, we examine the contribution of the NTD to infection and cell-cell fusion by constructing chimeric spikes bearing B.1.617 lineage (Delta and Kappa variants) NTDs and generating spike pseudotyped lentivirus. We find that the Delta NTD on a Kappa or wild-type (WT) background increases S1/S2 cleavage efficiency and virus entry, specifically in lung cells and airway organoids, through use of TMPRSS2. Delta exhibits increased cell-cell fusogenicity that could be conferred to WT and Kappa spikes by Delta NTD transfer. However, chimeras of Omicron BA.1 and BA.2 spikes with a Delta NTD do not show more efficient TMPRSS2 use or fusogenicity. We conclude that the NTD allosterically modulates S1/S2 cleavage and spike-mediated functions in a spike context-dependent manner, and allosteric interactions may be lost when combining regions from more distantly related VOCs.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111220\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4303699445\", \"doi_x\": \"10.3390/ijms231911669\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"New CRISPR Tools to Correct Pathogenic Mutations in Usher Syndrome\", \"abstract\": \"Inherited retinal degenerations are a leading cause of blindness in the UK. Significant advances have been made to tackle this issue in recent years, with a pioneering FDA approved gene therapy treatment (Luxturna\\u00ae), which targets a loss of function mutation in the RPE65 gene. However, there remain notable shortcomings to this form of gene replacement therapy. In particular, the lack of viability for gene sequences exceeding the 4.7 kb adeno-associated virus (AAV) packaging limit or for toxic gain of function mutations. The USH2A gene at ~15.7 kb for instance is too large for AAV delivery: a safe and effective vehicle capable of transducing photoreceptor cells for gene replacement therapy. Usher Syndrome is a clinically and genetically heterogenous deaf-blindness syndrome with autosomal recessive inheritance. The USH2A gene encodes the protein usherin, which localises to the photoreceptor cilium and cochlear hair cells. Mutations in the USH2A gene cause Usher Syndrome type II (USH2), which is the most common subtype of Usher Syndrome and the focus of this review. To date, researchers have been unable to create an efficient, safe editing tool that is small enough to fit inside a single AAV vector for delivery into human cells. This article reviews the potential of CRISPR technology, derived from bacterial defence mechanisms, to overcome these challenges; delivering tools to precisely edit and correct small insertions, deletions and base transitions in USH2A without the need to deliver the full-length gene. Such an ultra-compact therapy could make strides in combating a significant cause of blindness in young people.\", \"doi_y\": \"https://doi.org/10.3390/ijms231911669\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4283514597\", \"doi_x\": \"10.1136/annrheumdis-2022-222460\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Multi-trait and cross-population genome-wide association studies across autoimmune and allergic diseases identify shared and distinct genetic component\", \"abstract\": \"Autoimmune and allergic diseases are outcomes of the dysregulation of the immune system. Our study aimed to elucidate differences or shared components in genetic backgrounds between autoimmune and allergic diseases.We estimated genetic correlation and performed multi-trait and cross-population genome-wide association study (GWAS) meta-analysis of six immune-related diseases: rheumatoid arthritis, Graves' disease, type 1 diabetes for autoimmune diseases and asthma, atopic dermatitis and pollinosis for allergic diseases. By integrating large-scale biobank resources (Biobank Japan and UK biobank), our study included 105 721 cases and 433 663 controls. Newly identified variants were evaluated in 21 778 cases and 712 767 controls for two additional autoimmune diseases: psoriasis and systemic lupus erythematosus. We performed enrichment analyses of cell types and biological pathways to highlight shared and distinct perspectives.Autoimmune and allergic diseases were not only mutually classified based on genetic backgrounds but also they had multiple positive genetic correlations beyond the classifications. Multi-trait GWAS meta-analysis newly identified six allergic disease-associated loci. We identified four loci shared between the six autoimmune and allergic diseases (rs10803431 at PRDM2, OR=1.07, p=2.3\\u00d710-8, rs2053062 at G3BP1, OR=0.90, p=2.9\\u00d710-8, rs2210366 at HBS1L, OR=1.07, p=2.5\\u00d710-8 in Japanese and rs4529910 at POU2AF1, OR=0.96, p=1.9\\u00d710-10 across ancestries). Associations of rs10803431 and rs4529910 were confirmed at the two additional autoimmune diseases. Enrichment analysis demonstrated link to T cells, natural killer cells and various cytokine signals, including innate immune pathways.Our multi-trait and cross-population study should elucidate complex pathogenesis shared components across autoimmune and allergic diseases.\", \"doi_y\": \"https://doi.org/10.1136/annrheumdis-2022-222460\", \"publication_date\": \"2022-06-26\"}, {\"work_id\": \"W4225103662\", \"doi_x\": \"10.1186/s40694-022-00138-7\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"abstract\": \"Non-ribosomal peptide synthetase-like (NRPS-like) enzymes are highly enriched in fungal genomes and can be discriminated into reducing and non-reducing enzymes. Non-reducing NRPS-like enzymes possess a C-terminal thioesterase domain that catalyses the condensation of two identical aromatic \\u03b1-keto acids under the formation of enzyme-specific substrate-interconnecting core structures such as terphenylquinones, furanones, butyrolactones or dioxolanones. Ascocoryne sarcoides produces large quantities of ascocorynin, which structurally resembles a terphenylquinone produced from the condensation of p-hydroxyphenylpyruvate and phenylpyruvate. Since the parallel use of two different substrates by a non-reducing NRPS-like enzyme appeared as highly unusual, we investigated the biosynthesis of ascocorynin in A. sarcoides.Here, we searched the genome of A. sarcoides for genes coding for non-reducing NRPS-like enzymes. A single candidate gene was identified that was termed acyN. Heterologous gene expression confirmed that AcyN is involved in ascocorynin production but only produces the non-hydroxylated precursor polyporic acid. Although acyN is embedded in an ascocorynin biosynthesis gene cluster, a gene encoding a monooxygenase required for the hydroxylation of polyporic acid was not present. Expression analyses of all monooxygenase-encoding genes from A. sarcoides identified a single candidate that showed the same expression pattern as acyN. Accordingly, heterologous co-expression of acyN and the monooxygenase gene resulted in the production of ascocorynin. Structural modelling of the monooxygenase suggests that the hydrophobic substrate polyporic acid enters the monooxygenase from a membrane facing entry site and is converted into the more hydrophilic product ascocorynin, which prevents its re-entry for a second round of hydroxylation.This study characterises the first naturally occurring polyporic acid synthetase from an ascomycete. It confirms the high substrate and product specificity of this non-reducing NRPS-like enzyme and highlights the requirement of a monooxygenase to produce the terphenylquinone ascocorynin.\", \"doi_y\": \"https://doi.org/10.1186/s40694-022-00138-7\", \"publication_date\": \"2022-04-27\"}, {\"work_id\": \"W4225516213\", \"doi_x\": \"10.1093/bib/bbac187\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"abstract\": \"Mutations in human proteins lead to diseases. The structure of these proteins can help understand the mechanism of such diseases and develop therapeutics against them. With improved deep learning techniques, such as RoseTTAFold and AlphaFold, we can predict the structure of proteins even in the absence of structural homologs. We modeled and extracted the domains from 553 disease-associated human proteins without known protein structures or close homologs in the Protein Databank. We noticed that the model quality was higher and the Root mean square deviation (RMSD) lower between AlphaFold and RoseTTAFold models for domains that could be assigned to CATH families as compared to those which could only be assigned to Pfam families of unknown structure or could not be assigned to either. We predicted ligand-binding sites, protein-protein interfaces and conserved residues in these predicted structures. We then explored whether the disease-associated missense mutations were in the proximity of these predicted functional sites, whether they destabilized the protein structure based on ddG calculations or whether they were predicted to be pathogenic. We could explain 80% of these disease-associated mutations based on proximity to functional sites, structural destabilization or pathogenicity. When compared to polymorphisms, a larger percentage of disease-associated missense mutations were buried, closer to predicted functional sites, predicted as destabilizing and pathogenic. Usage of models from the two state-of-the-art techniques provide better confidence in our predictions, and we explain 93 additional mutations based on RoseTTAFold models which could not be explained based solely on AlphaFold models.\", \"doi_y\": \"https://doi.org/10.1093/bib/bbac187\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4303437380\", \"doi_x\": \"10.1101/2022.10.04.510780\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mutation of the ALS/FTD-associated RNA-binding protein FUS alters axonal cytoskeletal organisation\", \"abstract\": \"Summary Aberrant condensation and localisation of the RNA-binding protein fused in sarcoma (FUS) occur in variants of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). ALS is also associated with cytoskeletal defects, genetically and through observations of compromised axonal transport. Here, we asked whether compromised axonal cytoskeletal organisation is an early feature of FUS-associated ALS/FTD. We used an ALS-associated mutant FUS(P525L) and the FTD-mimic hypomethylated FUS, FUS(16R), to investigate the common and distinct cytoskeletal changes found in these two reported Xenopus models. Combining a novel atomic force microscopy (AFM)-based approach for in vitro cytoskeletal characterisation and in vivo axonal branching analysis, we found that mutant FUS reduced actin density in the dynamically remodelling growth cone, and reduced axonal branch complexity. We furthermore found evidence of an axon looping defect for FUS(P525L). Therefore, we show that compromised actin remodelling is potentially an important early event in FUS-associated pathogenesis. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.10.04.510780\", \"publication_date\": \"2022-10-07\"}, {\"work_id\": \"W4311547974\", \"doi_x\": \"10.1038/s41467-022-35454-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Multi-omics identify falling LRRC15 as a COVID-19 severity marker and persistent pro-thrombotic signals in convalescence\", \"abstract\": \"Patients with end-stage kidney disease (ESKD) are at high risk of severe COVID-19. Here, we perform longitudinal blood sampling of ESKD haemodialysis patients with COVID-19, collecting samples pre-infection, serially during infection, and after clinical recovery. Using plasma proteomics, and RNA-sequencing and flow cytometry of immune cells, we identify transcriptomic and proteomic signatures of COVID-19 severity, and find distinct temporal molecular profiles in patients with severe disease. Supervised learning reveals that the plasma proteome is a superior indicator of clinical severity than the PBMC transcriptome. We show that a decreasing trajectory of plasma LRRC15, a proposed co-receptor for SARS-CoV-2, is associated with a more severe clinical course. We observe that two months after the acute infection, patients still display dysregulated gene expression related to vascular, platelet and coagulation pathways, including PF4 (platelet factor 4), which may explain the prolonged thrombotic risk following COVID-19.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-35454-4\", \"publication_date\": \"2022-12-15\"}, {\"work_id\": \"W4366259401\", \"doi_x\": \"10.1021/acs.jcim.2c01510\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Does a Machine-Learned Potential Perform Better Than an Optimally Tuned Traditional Force Field? A Case Study on Fluorohydrins\", \"abstract\": \"We present a comparative study that evaluates the performance of a machine learning potential (ANI-2x), a conventional force field (GAFF), and an optimally tuned GAFF-like force field in the modeling of a set of 10 \\u03b3-fluorohydrins that exhibit a complex interplay between intra- and intermolecular interactions in determining conformer stability. To benchmark the performance of each molecular model, we evaluated their energetic, geometric, and sampling accuracies relative to quantum-mechanical data. This benchmark involved conformational analysis both in the gas phase and chloroform solution. We also assessed the performance of the aforementioned molecular models in estimating nuclear spin-spin coupling constants by comparing their predictions to experimental data available in chloroform. The results and discussion presented in this study demonstrate that ANI-2x tends to predict stronger-than-expected hydrogen bonding and overstabilize global minima and shows problems related to inadequate description of dispersion interactions. Furthermore, while ANI-2x is a viable model for modeling in the gas phase, conventional force fields still play an important role, especially for condensed-phase simulations. Overall, this study highlights the strengths and weaknesses of each model, providing guidelines for the use and future development of force fields and machine learning potentials.\", \"doi_y\": \"https://doi.org/10.1021/acs.jcim.2c01510\", \"publication_date\": \"2023-04-18\"}, {\"work_id\": \"W4324354022\", \"doi_x\": \"10.1103/physrevb.107.l121105\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation energy of the paramagnetic electron gas at the thermodynamic limit\", \"abstract\": \"The variational and diffusion quantum Monte Carlo methods are used to calculate the correlation energy of the paramagnetic three-dimensional homogeneous electron gas at intermediate to high density. Ground state energies in finite cells are determined using Slater-Jastrow-backflow trial wave functions, and finite-size errors are removed using twist-averaged boundary conditions and extrapolation of the energy per particle to the thermodynamic limit of infinite system size. Our correlation energies in the thermodynamic limit are lower (i.e., more negative, and therefore more accurate according to the variational principle) than previous results, and can be used for the parameterization of density functionals to be applied to high-density systems.\", \"doi_y\": \"https://doi.org/10.1103/physrevb.107.l121105\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4308059022\", \"doi_x\": \"10.1182/blood.2021014668\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Single-cell methods in myeloproliferative neoplasms: old questions, new technologies\", \"abstract\": \"Myeloproliferative neoplasms (MPN) are a group of clonal stem cell-derived hematopoietic malignancies driven by aberrant Janus kinase-signal transducer and activator of transcription proteins (JAK/STAT) signaling. Although these are genetically simple diseases, MPNs are phenotypically heterogeneous, reflecting underlying intratumoral heterogeneity driven by the interplay of genetic and nongenetic factors. Their evolution is determined by factors that enable certain cellular subsets to outcompete others. Therefore, techniques that resolve cellular heterogeneity at the single-cell level are ideally placed to provide new insights into MPN biology. With these insights comes the potential to uncover new approaches to predict the clinical course and treat these cancers, ultimately improving outcomes for patients. MPNs present a particularly tractable model of cancer evolution, because most patients present in an early disease phase and only a small proportion progress to aggressive disease. Therefore, it is not surprising that many groundbreaking technological advances in single-cell omics have been pioneered by their application in MPNs. In this review article, we explore how single-cell approaches have provided transformative insights into MPN disease biology, which are broadly applicable across human cancers, and discuss how these studies might be swiftly translated into clinical pathways and may eventually underpin precision medicine.\", \"doi_y\": \"https://doi.org/10.1182/blood.2021014668\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W4285491711\", \"doi_x\": \"10.1073/pnas.2205412119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation between the binding affinity and the conformational entropy of nanobody SARS-CoV-2 spike protein complexes\", \"abstract\": \"Camelid single-domain antibodies, also known as nanobodies, can be readily isolated from na\\u00efve libraries for specific targets but often bind too weakly to their targets to be immediately useful. Laboratory-based genetic engineering methods to enhance their affinity, termed maturation, can deliver useful reagents for different areas of biology and potentially medicine. Using the receptor binding domain (RBD) of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike protein and a na\\u00efve library, we generated closely related nanobodies with micromolar to nanomolar binding affinities. By analyzing the structure-activity relationship using X-ray crystallography, cryoelectron microscopy, and biophysical methods, we observed that higher conformational entropy losses in the formation of the spike protein-nanobody complex are associated with tighter binding. To investigate this, we generated structural ensembles of the different complexes from electron microscopy maps and correlated the conformational fluctuations with binding affinity. This insight guided the engineering of a nanobody with improved affinity for the spike protein.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2205412119\", \"publication_date\": \"2022-07-15\"}, {\"work_id\": \"W4220913467\", \"doi_x\": \"10.1098/rsob.210389\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Reconstitution of an active human CENP-E motor\", \"abstract\": \"CENP-E is a large kinesin motor protein which plays pivotal roles in mitosis by facilitating chromosome capture and alignment, and promoting microtubule flux in the spindle. So far, it has not been possible to obtain active human CENP-E to study its molecular properties. Xenopus CENP-E motor has been characterized in vitro and is used as a model motor; however, its protein sequence differs significantly from human CENP-E. Here, we characterize human CENP-E motility in vitro. Full-length CENP-E exhibits an increase in run length and longer residency times on microtubules when compared to CENP-E motor truncations, indicating that the C-terminal microtubule-binding site enhances the processivity when the full-length motor is active. In contrast with constitutively active human CENP-E truncations, full-length human CENP-E has a reduced microtubule landing rate in vitro, suggesting that the non-motor coiled-coil regions self-regulate motor activity. Together, we demonstrate that human CENP-E is a processive motor, providing a useful tool to study the mechanistic basis for how human CENP-E drives chromosome congression and spindle organization during human cell division.\", \"doi_y\": \"https://doi.org/10.1098/rsob.210389\", \"publication_date\": \"2022-03-01\"}, {\"work_id\": \"W4310112261\", \"doi_x\": \"10.1021/acs.biomac.2c01097\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Poly(vinyl alcohol) Molecular Bottlebrushes Nucleate Ice\", \"abstract\": \"Ice binding proteins (IBP) have evolved to limit the growth of ice but also to promote ice formation by ice-nucleating proteins (INPs). IBPs, which modulate these seemingly distinct processes, often have high sequence similarities, and molecular size/assembly is hypothesized to be a crucial determinant. There are only a few synthetic materials that reproduce INP function, and rational design of ice nucleators has not been achieved due to outstanding questions about the mechanisms of ice binding. Poly(vinyl alcohol) (PVA) is a water-soluble synthetic polymer well known to effectively block ice recrystallization, by binding to ice. Here, we report the synthesis of a polymeric ice nucleator, which mimics the dense assembly of IBPs, using confined ice-binding polymers in a high-molar-mass molecular bottlebrush. Poly(vinyl alcohol)-based molecular bottlebrushes with different side-chain densities were synthesized via a combination of ring-opening metathesis polymerization (ROMP) and reversible addition-fragmentation chain-transfer (RAFT) polymerization, using \\\"grafting-to\\\" and \\\"grafting-through\\\" approaches. The facile preparation of the PVA bottlebrushes was performed via selective hydrolysis of the acetate of the poly(vinyl acetate) (PVAc) side chains of the PVAc bottlebrush precursors. Ice-binding polymer side-chain density was shown to be crucial for nucleation activity, with less dense brushes resulting in colder nucleation than denser brushes. This bio-inspired approach provides a synthetic framework for probing heterogeneous ice nucleation and a route toward defined synthetic nucleators for biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1021/acs.biomac.2c01097\", \"publication_date\": \"2022-11-28\"}, {\"work_id\": \"W4366691203\", \"doi_x\": \"10.3390/microorganisms11041077\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Structural Analysis of Bacillus subtilis Sigma Factors\", \"abstract\": \"Bacteria use an array of sigma factors to regulate gene expression during different stages of their life cycles. Full-length, atomic-level structures of sigma factors have been challenging to obtain experimentally as a result of their many regions of intrinsic disorder. AlphaFold has now supplied plausible full-length models for most sigma factors. Here we discuss the current understanding of the structures and functions of sigma factors in the model organism, Bacillus subtilis, and present an X-ray crystal structure of a region of B. subtilis SigE, a sigma factor that plays a critical role in the developmental process of spore formation.\", \"doi_y\": \"https://doi.org/10.3390/microorganisms11041077\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4200343853\", \"doi_x\": \"10.1101/2021.12.28.474284\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Clathrin-mediated Endocytosis Facilitates Internalization of<i>Magnaporthe oryzae</i>Effectors into Rice Cells\", \"abstract\": \"ABSTRACT Fungi and oomycetes deliver effectors into living plant cells to suppress defenses and control plant processes needed for infection. Little is known about the mechanism by which these pathogens translocate effector proteins across the plasma membrane into the plant cytoplasm. The blast fungus Magnaporthe oryzae secretes cytoplasmic effectors into a specialized biotrophic interfacial complex (BIC) before translocation. Here we show that cytoplasmic effectors within BICs are packaged into vesicles that are occasionally observed in the host cytoplasm. Live cell imaging with fluorescently-labeled rice showed that effector vesicles colocalize with plant plasma membrane and with clathrin light chain-1, a marker for clathrin-mediated endocytosis (CME). Inhibition of CME using Virus-Induced Gene Silencing (VIGS) and chemical treatments results in cytoplasmic effectors in swollen BICs lacking vesicles. In contrast, fluorescent marker co-localization, VIGS and chemical inhibitor studies failed to support a major role for clathrin-independent endocytosis in effector vesicle formation. Localization studies of two novel effectors, and of known effectors after CME inhibition, indicate that cytoplasmic effector translocation occurs underneath appressoria before invasive hyphal growth. Taken together, this study provides evidence that cytoplasmic effector translocation is mediated by clathrin-mediated endocytosis in BICs and suggests a role for M. oryzae effectors in co-opting plant endocytosis.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.28.474284\", \"publication_date\": \"2021-12-29\"}, {\"work_id\": \"W4282925859\", \"doi_x\": \"10.1038/s42003-022-03502-w\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"abstract\": \"Abstract Correct folding of outer membrane proteins (OMPs) into the outer membrane of Gram-negative bacteria depends on delivery of unfolded OMPs to the \\u03b2-barrel assembly machinery (BAM). How unfolded substrates are presented to BAM remains elusive, but the major OMP chaperone SurA is proposed to play a key role. Here, we have used hydrogen deuterium exchange mass spectrometry (HDX-MS), crosslinking, in vitro folding and binding assays and computational modelling to show that the core domain of SurA and one of its two PPIase domains are key to the SurA-BAM interaction and are required for maximal catalysis of OMP folding. We reveal that binding causes changes in BAM and SurA conformation and/or dynamics distal to the sites of binding, including at the BamA \\u03b21-\\u03b216 seam. We propose a model for OMP biogenesis in which SurA plays a crucial role in OMP delivery and primes BAM to accept substrates for folding.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03502-w\", \"publication_date\": \"2022-06-08\"}, {\"work_id\": \"W4365479730\", \"doi_x\": \"10.1111/ele.14218\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Metabolic efficiency reshapes the seminal relationship between pathogen growth rate and virulence\", \"abstract\": \"A cornerstone of classical virulence evolution theories is the assumption that pathogen growth rate is positively correlated with virulence, the amount of damage pathogens inflict on their hosts. Such theories are key for incorporating evolutionary principles into sustainable disease management strategies. Yet, empirical evidence raises doubts over this central assumption underpinning classical theories, thus undermining their generality and predictive power. In this paper, we identify a key component missing from current theories which redefines the growth-virulence relationship in a way that is consistent with data. By modifying the activity of a single metabolic gene, we engineered strains of Magnaporthe oryzae with different nutrient acquisition and growth rates. We conducted in planta infection studies and uncovered an unexpected non-monotonic relationship between growth rate and virulence that is jointly shaped by how growth rate and metabolic efficiency interact. This novel mechanistic framework paves the way for a much-needed new suite of virulence evolution theories.\", \"doi_y\": \"https://doi.org/10.1111/ele.14218\", \"publication_date\": \"2023-04-13\"}, {\"work_id\": \"W4309809681\", \"doi_x\": \"10.1021/acs.analchem.2c03164\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"abstract\": \"Microfluidic water-in-oil emulsion droplets are becoming a mainstay of experimental biology, where they replace the classical test tube. In most applications, such as ultrahigh-throughput directed evolution, the droplet content is identical for all compartmentalized assay reactions. When emulsion droplets are used for kinetics or other functional assays, though, concentration dependencies of initial rates that define Michaelis\\u2013Menten parameters are required. Droplet-on-demand systems satisfy this need, but extracting large amounts of data is challenging. Here, we introduce a multiplexed droplet absorbance detector, which\\u2500coupled to semi-automated droplet generation\\u2500forms a tubing-based droplet-on-demand system able to generate and extract quantitative datasets from defined concentration gradients across multiple series of droplets for multiple time points. The emergence of a product is detected by reading the absorbance of the droplet sets at multiple, adjustable time points by reversing the flow direction after each detection, so that the droplets pass a line scan camera multiple times. Detection multiplexing allows absorbance values at 12 distinct positions to be measured, and enzyme kinetics are recorded for label-free concentration gradients that are composed of about 60 droplets each, covering as many concentrations. With a throughput of around 8640 data points per hour, a 10-fold improvement compared to the previously reported single point detection method is achieved. In a single experiment, 12 full datasets of high-resolution and high-accuracy Michaelis\\u2013Menten kinetics were determined to demonstrate the potential for enzyme characterization for glycosidase substrates covering a range in enzymatic hydrolysis of 7 orders of magnitude in kcat/KM. The straightforward setup, high throughput, excellent data quality, and wide dynamic range that allows coverage of diverse activities suggest that this system may serve as a miniaturized spectrophotometer for detailed analysis of clones emerging from large-scale combinatorial experiments.\", \"doi_y\": \"https://doi.org/10.1021/acs.analchem.2c03164\", \"publication_date\": \"2022-11-23\"}, {\"work_id\": \"W4318828504\", \"doi_x\": \"10.3390/ijms24032682\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"PTCHD1 Binds Cholesterol but Not Sonic Hedgehog, Suggesting a Distinct Cellular Function\", \"abstract\": \"Deleterious mutations in the X-linked Patched domain-containing 1 (PTCHD1) gene may account for up to 1% of autism cases. Despite this, the PTCHD1 protein remains poorly understood. Structural similarities to Patched family proteins point to a role in sterol transport, but this hypothesis has not been verified experimentally. Additionally, PTCHD1 has been suggested to be involved in Hedgehog signalling, but thus far, the experimental results have been conflicting. To enable a variety of biochemical and structural experiments, we developed a method for expressing PTCHD1 in Spodoptera frugiperda cells, solubilising it in glycol-diosgenin, and purifying it to homogeneity. In vitro and in silico experiments show that PTCHD1 function is not interchangeable with Patched 1 (PTCH1) in canonical Hedgehog signalling, since it does not repress Smoothened in Ptch1\\u2212/\\u2212 mouse embryonic fibroblasts and does not bind Sonic Hedgehog. However, we found that PTCHD1 binds cholesterol similarly to PTCH1. Furthermore, we identified 13 PTCHD1-specific protein interactors through co-immunoprecipitation and demonstrated a link to cell stress responses and RNA stress granule formation. Thus, our results support the notion that despite structural similarities to other Patched family proteins, PTCHD1 may have a distinct cellular function.\", \"doi_y\": \"https://doi.org/10.3390/ijms24032682\", \"publication_date\": \"2023-01-31\"}, {\"work_id\": \"W4281667554\", \"doi_x\": \"10.1007/s11120-022-00925-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Quantum chemical elucidation of a sevenfold symmetric bacterial antenna complex\", \"abstract\": \"The light-harvesting complex 2 (LH2) of purple bacteria is one of the most studied photosynthetic antenna complexes. Its symmetric structure and ring-like bacteriochlorophyll arrangement make it an ideal system for theoreticians and spectroscopists. LH2 complexes from most bacterial species are thought to have eightfold or ninefold symmetry, but recently a sevenfold symmetric LH2 structure from the bacterium Mch. purpuratum was solved by Cryo-Electron microscopy. This LH2 also possesses unique near-infrared absorption and circular dichroism (CD) spectral properties. Here we use an atomistic strategy to elucidate the spectral properties of Mch. purpuratum LH2 and understand the differences with the most commonly studied LH2 from Rbl. acidophilus. Our strategy exploits a combination of molecular dynamics simulations, multiscale polarizable quantum mechanics/molecular mechanics calculations, and lineshape simulations. Our calculations reveal that the spectral properties of LH2 complexes are tuned by site energies and exciton couplings, which in turn depend on the structural fluctuations of the bacteriochlorophylls. Our strategy proves effective in reproducing the absorption and CD spectra of the two LH2 complexes, and in uncovering the origin of their differences. This work proves that it is possible to obtain insight into the spectral tuning strategies of purple bacteria by quantitatively simulating the spectral properties of their antenna complexes.\", \"doi_y\": \"https://doi.org/10.1007/s11120-022-00925-8\", \"publication_date\": \"2022-06-08\"}, {\"work_id\": \"W4310422180\", \"doi_x\": \"10.1021/acs.jpcb.2c04983\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Phasor Analysis Reveals Multicomponent Fluorescence Kinetics in the LH2 Complex from <i>Marichromatium purpuratum</i>\", \"abstract\": \"We investigated the fluorescence kinetics of LH2 complexes from Marichromatium purpuratum, the cryo-EM structure of which has been recently elucidated with 2.4 \\u00c5 resolution. The experiments have been carried out as a function of the excitation density by varying both the excitation fluence and the repetition rate of the laser excitation. Instead of the usual multiexponential fitting procedure, we applied the less common phasor formalism for evaluating the transients because this allows for a model-free analysis of the data without a priori knowledge about the number of processes that contribute to a particular decay. For the various excitation conditions, this analysis reproduces consistently three lifetime components with decay times below 100 ps, 500 ps, and 730 ps, which were associated with the quenched state, singlet-triplet annihilation, and fluorescence decay, respectively. Moreover, it reveals that the number of decay components that contribute to the transients depends on whether the excitation wavelength is in resonance with the B800 BChl a molecules or with the carotenoids. Based on the mutual arrangement of the chromophores in their binding pockets, this leads us to conclude that the energy transfer pathways within the LH2 complex of this species differ significantly from each other for exciting either the B800 BChl molecules or the carotenoids. Finally, we speculate whether the illumination with strong laser light converts the LH2 complexes studied here into a quenched conformation that might be related to the development of the non-photochemical quenching mechanism that occurs in higher plants.\", \"doi_y\": \"https://doi.org/10.1021/acs.jpcb.2c04983\", \"publication_date\": \"2022-11-30\"}, {\"work_id\": \"W4210460281\", \"doi_x\": \"10.1093/bioinformatics/btab881\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Current structure predictors are not learning the physics of protein folding\", \"abstract\": \"Motivation. Predicting the native state of a protein has long been considered a gateway problem for understanding protein folding. Recent advances in structural modeling driven by deep learning have achieved unprecedented success at predicting a protein's crystal structure, but it is not clear if these models are learning the physics of how proteins dynamically fold into their equilibrium structure or are just accurate knowledge-based predictors of the final state. Results. In this work, we compare the pathways generated by state-of-the-art protein structure prediction methods to experimental data about protein folding pathways. The methods considered were AlphaFold 2, RoseTTAFold, trRosetta, RaptorX, DMPfold, EVfold, SAINT2 and Rosetta. We find evidence that their simulated dynamics capture some information about the folding pathway, but their predictive ability is worse than a trivial classifier using sequence-agnostic features like chain length. The folding trajectories produced are also uncorrelated with experimental observables such as intermediate structures and the folding rate constant. These results suggest that recent advances in structure prediction do not yet provide an enhanced understanding of protein folding. Availability. The data underlying this article are available in GitHub at https://github.com/oxpig/structure-vs-folding/.Supplementary data are available at Bioinformatics online.\", \"doi_y\": \"https://doi.org/10.1093/bioinformatics/btab881\", \"publication_date\": \"2022-01-31\"}, {\"work_id\": \"W4296400863\", \"doi_x\": \"10.1002/1873-3468.14500\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protein metalation in a nutshell\", \"abstract\": \"Metalation, the acquisition of metals by proteins, must avoid mis\\u2010metalation with tighter binding metals. This is illustrated by four selected proteins that require different metals: all show similar ranked orders of affinity for bioavailable metals, as described in a universal affinity series (the Irving\\u2013Williams series). Crucially, cellular protein metalation occurs in competition with other metal binding sites. The strength of this competition defines the intracellular availability of each metal: its magnitude has been estimated by calibrating a cells' set of DNA\\u2010binding, metal\\u2010sensing, transcriptional regulators. This has established that metal availabilities (as free energies for forming metal complexes) are maintained to the inverse of the universal series. The tightest binding metals are least available. With these availabilities, correct metalation is achieved.\", \"doi_y\": \"https://doi.org/10.1002/1873-3468.14500\", \"publication_date\": \"2022-09-26\"}, {\"work_id\": \"W4283271216\", \"doi_x\": \"10.1371/journal.pgen.1010279\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"P450 gene duplication and divergence led to the evolution of dual novel functions and insecticide cross-resistance in the brown planthopper Nilaparvata lugens\", \"abstract\": \"The sustainable control of many highly damaging insect crop pests and disease vectors is threatened by the evolution of insecticide resistance. As a consequence, strategies have been developed that aim to prevent or delay resistance development by rotating or mixing insecticides with different modes of action (MoA). However, these approaches can be compromised by the emergence of mechanisms that confer cross-resistance to insecticides with different MoA. Despite the applied importance of cross-resistance, its evolutionary underpinnings remain poorly understood. Here we reveal how a single gene evolved the capacity to detoxify two structurally unrelated insecticides with different MoA. Using transgenic approaches we demonstrate that a specific variant of the cytochrome P450 CYP6ER1, previously shown to confer resistance to the neonicotinoid imidacloprid in the brown planthopper, N. lugens, also confers cross-resistance to the phenylpyrazole ethiprole. CYP6ER1 is duplicated in resistant strains, and we show that while the acquisition of mutations in two encoded substrate recognition sites (SRS) of one of the parologs led to resistance to imidacloprid, a different set of mutations, outside of known SRS, are primarily responsible for resistance to ethiprole. Epistatic interactions between these mutations and their genetic background suggest that the evolution of dual resistance from the same gene copy involved functional trade-offs in respect to CYP6ER1 catalytic activity for ethiprole versus imidacloprid. Surprisingly, the mutations leading to ethiprole and imidacloprid resistance do not confer the ability to detoxify the insecticide fipronil, another phenylpyrazole with close structural similarity to ethiprole. Taken together, these findings reveal how gene duplication and divergence can lead to the evolution of multiple novel functions from a single gene. From an applied perspective they also demonstrate how cross-resistance to structurally unrelated insecticides can evolve, and illustrate the difficulty in predicting cross-resistance profiles mediated by metabolic mechanisms.\", \"doi_y\": \"https://doi.org/10.1371/journal.pgen.1010279\", \"publication_date\": \"2022-06-21\"}, {\"work_id\": \"W4297394861\", \"doi_x\": \"10.1101/2022.09.25.509395\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"AF2-multimer guided high accuracy prediction of typical and atypical ATG8 binding motifs\", \"abstract\": \"Abstract Macroautophagy/autophagy is an intracellular degradation process central to cellular homeostasis and defense against pathogens in eukaryotic cells. Regulation of autophagy relies on hierarchical binding of autophagy cargo receptors and adaptors to ATG8/LC3 protein family members. Interactions with ATG8/LC3 are typically facilitated by a conserved, short linear sequence, referred to as the ATG8/LC3 interacting motif/region (AIM/LIR), present in autophagy adaptors and receptors as well as pathogen virulence factors targeting host autophagy machinery. Since the canonical AIM/LIR sequence can be found in many proteins, identifying functional AIM/LIR motifs has proven challenging. Here we show that protein modelling using Alphafold-Multimer (AF2-multimer) identifies both canonical and atypical AIM/LIR motifs with a high level of accuracy. AF2-multimer can be modified to detect additional functional AIM/LIR motifs by using protein sequences with mutations in primary AIM/LIR residues. By combining protein modelling data from AF2-multimer with phylogenetic analysis of protein sequences and protein-protein interaction assays, we demonstrate that AF2-multimer predicts the physiologically relevant AIM motif in the ATG8-interacting protein 2 (ATI-2) as well as the previously uncharacterized non-canonical AIM motif in ATG3 from potato ( Solanum tuberosum ). AF2-multimer also identified the AIM/LIR motifs in pathogen-encoded virulence factors that target ATG8 members in their plant and human hosts, revealing that cross-kingdom ATG8-LIR/AIM associations can also be predicted by AF2-multimer. We conclude that the AF2-guided discovery of autophagy adaptors/receptors will substantially accelerate our understanding of the molecular basis of autophagy in all biological kingdoms.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.25.509395\", \"publication_date\": \"2022-09-26\"}, {\"work_id\": \"W4366992750\", \"doi_x\": \"10.1111/nph.18921\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Manipulation of plant immunity via an <scp>mRNA</scp> decapping pathogen effector\", \"abstract\": \"This article is a Commentary on McCombe et al . (2023), 239 : 222\\u2013239 .\", \"doi_y\": \"https://doi.org/10.1111/nph.18921\", \"publication_date\": \"2023-04-25\"}, {\"work_id\": \"W4313455881\", \"doi_x\": \"10.1101/2022.12.20.521207\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Photovoltaic enzymes by design and evolution\", \"abstract\": \"ABSTRACT The global energy crisis challenges us to develop more efficient strategies for the sustainable production of energy. Given the excellent efficiency of the natural photosynthetic apparatus, biohybrid photovoltaic devices present an attractive solution for solar energy conversion. However, their composition, stability, and complexity can limit their inclusion into photovoltaic devices. Here, we combined computational design and directed evolution to overcome these limitations and create tailor-made photoenzymes. Photo-biocatalysts were designed by introducing photosensitizer binding sites into heme-containing helical bundle proteins. The designed binding sites were specific for the target photosensitizer and readily transplanted into other helical bundles. The best design was highly evolvable and reached nanomolar ligand affinity after mutagenesis and screening. The evolved enzyme generated 2.6 times higher photocurrents than the photosensitizer alone, primarily driven by increased photostability. Evolvability is a unique advantage of our protein-based approach over abiological photovoltaic and will be critical to developing efficient biohybrid systems. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.12.20.521207\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4381548756\", \"doi_x\": \"10.1038/s41467-023-38944-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cyclic di-AMP traps proton-coupled K+ transporters of the KUP family in an inward-occluded conformation\", \"abstract\": \"Cyclic di-AMP is the only known essential second messenger in bacteria and archaea, regulating different proteins indispensable for numerous physiological processes. In particular, it controls various potassium and osmolyte transporters involved in osmoregulation. In Bacillus subtilis, the K+/H+ symporter KimA of the KUP family is inactivated by c-di-AMP. KimA sustains survival at potassium limitation at low external pH by mediating potassium ion uptake. However, at elevated intracellular K+ concentrations, further K+ accumulation would be toxic. In this study, we reveal the molecular basis of how c-di-AMP binding inhibits KimA. We report cryo-EM structures of KimA with bound c-di-AMP in detergent solution and reconstituted in amphipols. By combining structural data with functional assays and molecular dynamics simulations we reveal how c-di-AMP modulates transport. We show that an intracellular loop in the transmembrane domain interacts with c-di-AMP bound to the adjacent cytosolic domain. This reduces the mobility of transmembrane helices at the cytosolic side of the K+ binding site and therefore traps KimA in an inward-occluded conformation.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-38944-1\", \"publication_date\": \"2023-06-21\"}, {\"work_id\": \"W4291801098\", \"doi_x\": \"10.1101/2022.08.12.503723\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evolutionary dynamics of genome size and content during the adaptive radiation of Heliconiini butterflies\", \"abstract\": \"Abstract Heliconius butterflies, a speciose genus of M\\u00fcllerian mimics, represent a classic example of an adaptive radiation that includes a range of derived dietary, life history, physiological and neural traits. However, key lineages within the genus, and across the broader Heliconiini tribe, lack genomic resources, limiting our understanding of how adaptive and neutral processes shaped genome evolution during their radiation. We have generated highly contiguous genome assemblies for nine new Heliconiini, 29 additional reference-assembled genomes, and improve 10 existing assemblies. Altogether, we provide a major new dataset of annotated genomes for a total of 63 species, including 58 species within the Heliconiini tribe. We use this extensive dataset to generate a robust and dated heliconiine phylogeny, describe major patterns of introgression, explore the evolution of genome architecture, and the genomic basis of key innovations in this enigmatic group, including an assessment of the evolution of putative regulatory regions at the Heliconius stem. Our work illustrates how the increased resolution provided by such dense genomic sampling improves our power to generate and test gene-phenotype hypotheses, and precisely characterize how genomes evolve.\", \"doi_y\": \"https://doi.org/10.1101/2022.08.12.503723\", \"publication_date\": \"2022-08-15\"}, {\"work_id\": \"W4321483489\", \"doi_x\": \"10.1016/j.jmb.2023.168021\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"abstract\": \"ModelCIF (github.com/ihmwg/ModelCIF) is a data information framework developed for and by computational structural biologists to enable delivery of Findable, Accessible, Interoperable, and Reusable (FAIR) data to users worldwide. ModelCIF describes the specific set of attributes and metadata associated with macromolecular structures modeled by solely computational methods and provides an extensible data representation for deposition, archiving, and public dissemination of predicted three-dimensional (3D) models of macromolecules. It is an extension of the Protein Data Bank Exchange / macromolecular Crystallographic Information Framework (PDBx/mmCIF), which is the global data standard for representing experimentally-determined 3D structures of macromolecules and associated metadata. The PDBx/mmCIF framework and its extensions (e.g., ModelCIF) are managed by the Worldwide Protein Data Bank partnership (wwPDB, wwpdb.org) in collaboration with relevant community stakeholders such as the wwPDB ModelCIF Working Group (wwpdb.org/task/modelcif). This semantically rich and extensible data framework for representing computed structure models (CSMs) accelerates the pace of scientific discovery. Herein, we describe the architecture, contents, and governance of ModelCIF, and tools and processes for maintaining and extending the data standard. Community tools and software libraries that support ModelCIF are also described.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168021\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4309048142\", \"doi_x\": \"10.1128/mbio.02541-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"abstract\": \"The human pathogen Pseudomonas aeruginosa (Pa) is one of the most frequent and severe causes of nosocomial infection. This organism is also a major cause of airway infections in people with cystic fibrosis (CF). Pa is known to have a remarkable metabolic plasticity, allowing it to thrive under diverse environmental conditions and ecological niches; yet, little is known about the central metabolic pathways that sustain its growth during infection or precisely how these pathways operate. In this work, we used a combination of 'omics approaches (transcriptomics, proteomics, metabolomics, and 13C-fluxomics) and reverse genetics to provide systems-level insight into how the infection-relevant organic acids succinate and propionate are metabolized by Pa. Moreover, through structural and kinetic analysis of the 2-methylcitrate synthase (2-MCS; PrpC) and its paralogue citrate (CIT) synthase (GltA), we show how these two crucial enzymatic steps are interconnected in Pa organic acid assimilation. We found that Pa can rapidly adapt to the loss of GltA function by acquiring mutations in a transcriptional repressor, which then derepresses prpC expression. Our findings provide a clear example of how \\\"underground metabolism,\\\" facilitated by enzyme substrate promiscuity, \\\"rewires\\\" Pa metabolism, allowing it to overcome the loss of a crucial enzyme. This pathogen-specific knowledge is critical for the advancement of a model-driven framework to target bacterial central metabolism. IMPORTANCE Pseudomonas aeruginosa is an opportunistic human pathogen that, due to its unrivalled resistance to antibiotics, ubiquity in the built environment, and aggressiveness in infection scenarios, has acquired the somewhat dubious accolade of being designated a \\\"critical priority pathogen\\\" by the WHO. In this work, we uncover the pathways and mechanisms used by P. aeruginosa to grow on a substrate that is abundant at many infection sites: propionate. We found that if the organism is prevented from metabolizing propionate, the substrate turns from being a convenient nutrient source into a potent poison, preventing bacterial growth. We further show that one of the enzymes involved in these reactions, 2-methylcitrate synthase (PrpC), is promiscuous and can moonlight for another essential enzyme in the cell (citrate synthase). Indeed, mutations that abolish citrate synthase activity (which would normally prevent the cell from growing) can be readily overcome if the cell acquires additional mutations that increase the expression of PrpC. This is a nice example of the evolutionary utility of so-called \\\"underground metabolism.\\\"\", \"doi_y\": \"https://doi.org/10.1128/mbio.02541-22\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4366463013\", \"doi_x\": \"10.1371/journal.pone.0284453\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Obscurin Rho GEF domains are phosphorylated by MST-family kinases but do not exhibit nucleotide exchange factor activity towards Rho GTPases in vitro\", \"abstract\": \"Obscurin is a giant muscle protein (>800 kDa) featuring multiple signalling domains, including an SH3-DH-PH domain triplet from the Trio-subfamily of guanosine nucleotide exchange factors (GEFs). While previous research suggests that these domains can activate the small GTPases RhoA and RhoQ in cells, in vitro characterization of these interactions using biophysical techniques has been hampered by the intrinsic instability of obscurin GEF domains. To study substrate specificity, mechanism and regulation of obscurin GEF function by individual domains, we successfully optimized recombinant production of obscurin GEF domains and found that MST-family kinases phosphorylate the obscurin DH domain at Thr5798. Despite extensive testing of multiple GEF domain fragments, we did not detect any nucleotide exchange activity in vitro against 9 representative small GTPases. Bioinformatic analyses show that obscurin differs from other Trio-subfamily GEFs in several important aspects. While further research is necessary to evaluate obscurin GEF activity in vivo, our results indicate that obscurin has atypical GEF domains that, if catalytically active at all, are subject to complex regulation.\", \"doi_y\": \"https://doi.org/10.1371/journal.pone.0284453\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4280488864\", \"doi_x\": \"10.1101/2022.04.25.489359\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"abstract\": \"Abstract Plant pathogens compromise crop yields. Plants have evolved robust innate immunity that depends in part on intracellular Nucleotide-binding, Leucine Rich-Repeat (NLR) immune receptors that activate defense responses upon detection of pathogen-derived effectors. Most \\u201csensor\\u201d NLRs that detect effectors require the activity of \\u201chelper\\u201d NLRs, but how helper NLRs support sensor NLR function is poorly understood. Many Solanaceae NLRs require the NRC (NLR-Required for Cell death) class of helper NLRs. We show here that Rpi-amr3, a sensor NLR from Solanum americanum , detects AVRamr3 from the potato late blight pathogen, Phytophthora infestans , and activates oligomerization of the helper NLR NRC2 into a high-molecular weight resistosome. The NRC2 resistosome also forms upon recognition of P. infestans effector AVRamr1 by another sensor NLR, Rpi-amr1. The ATP-binding motif of Rpi-amr3 is required for NRC2 resistosome formation, but not for interaction with the cognate effector. The NRC2 resistosome can be activated by AVRamr3 homologs from other Phytophthora species. Mechanistic understanding of NRC resistosome formation will underpin engineering crops with durable disease resistance.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.25.489359\", \"publication_date\": \"2022-04-25\"}, {\"work_id\": \"W4386529680\", \"doi_x\": \"10.1021/jacs.3c04330\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mechanism-Based Redesign of GAP to Activate Oncogenic Ras\", \"abstract\": \"Ras GTPases play a crucial role in cell signaling pathways. Mutations of the Ras gene occur in about one third of cancerous cell lines and are often associated with detrimental clinical prognosis. Hot spot residues Gly12, Gly13, and Gln61 cover 97% of oncogenic mutations, which impair the enzymatic activity in Ras. Using QM/MM free energy calculations, we present a two-step mechanism for the GTP hydrolysis catalyzed by the wild-type Ras.GAP complex. We found that the deprotonation of the catalytic water takes place via the Gln61 as a transient Br\\u00f8nsted base. We also determined the reaction profiles for key oncogenic Ras mutants G12D and G12C using QM/MM minimizations, matching the experimentally observed loss of catalytic activity, thereby validating our reaction mechanism. Using the optimized reaction paths, we devised a fast and accurate procedure to design GAP mutants that activate G12D Ras. We replaced GAP residues near the active site and determined the activation barrier for 190 single mutants. We furthermore built a machine learning for ultrafast screening, by fast prediction of the barrier heights, tested both on the single and double mutations. This work demonstrates that fast and accurate screening can be accomplished via QM/MM reaction path optimizations to design protein sequences with increased catalytic activity. Several GAP mutations are predicted to re-enable catalysis in oncogenic G12D, offering a promising avenue to overcome aberrant Ras-driven signal transduction by activating enzymatic activity instead of inhibition. The outlined computational screening protocol is readily applicable for designing ligands and cofactors analogously.\", \"doi_y\": \"https://doi.org/10.1021/jacs.3c04330\", \"publication_date\": \"2023-09-08\"}, {\"work_id\": \"W4386204040\", \"doi_x\": \"10.1111/febs.16943\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural and biochemical characterisation of the N\\u2010carbamoyl\\u2010\\u03b2\\u2010alanine amidohydrolase from <i>Rhizobium radiobacter</i><scp>MDC</scp> 8606\", \"abstract\": \"N\\u2010carbamoyl\\u2010\\u03b2\\u2010alanine amidohydrolase (C\\u03b2AA) constitutes one of the most important groups of industrially relevant enzymes used in the production of optically pure amino acids and derivatives. In this study, a C\\u03b2AA\\u2010encoding gene from Rhizobium radiobacter strain MDC 8606 was cloned and overexpressed in Escherichia coli . The purified recombinant enzyme (RrC\\u03b2AA) showed a specific activity of 14 U\\u00b7mg \\u22121 using N\\u2010carbamoyl\\u2010\\u03b2\\u2010alanine as a substrate with an optimum activity at 55 \\u00b0C and pH 8.0. In this work, we report also the first prokaryotic C\\u03b2AA structure at a resolution of 2.0 \\u00c5. A discontinuous catalytic domain and a dimerisation domain attached through a flexible hinge region at the domain interface have been revealed. We identify key ligand binding residues, including a conserved glutamic acid (Glu131), histidine (H385) and arginine (Arg291). Our results allowed us to explain the preference of the enzyme for linear carbamoyl substrates, as large and branched carbamoyl substrates cannot fit in the active site of the enzyme. This work envisages the use of RrC\\u03b2AA from R. radiobacter MDC 8606 for the industrial production of L\\u2010\\u03b1\\u2010, L\\u2010\\u03b2\\u2010 and L\\u2010\\u03b3\\u2010amino acids. The structural analysis provides new insights on enzyme\\u2013substrate interaction, which shed light on engineering of C\\u03b2AAs for high catalytic activity and broad substrate specificity.\", \"doi_y\": \"https://doi.org/10.1111/febs.16943\", \"publication_date\": \"2023-09-08\"}, {\"work_id\": \"W4307036039\", \"doi_x\": \"10.7554/elife.57736\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"KAT2-mediated acetylation switches the mode of PALB2 chromatin association to safeguard genome integrity\", \"abstract\": \"The tumour suppressor PALB2 stimulates RAD51-mediated homologous recombination (HR) repair of DNA damage, whilst its steady-state association with active genes protects these loci from replication stress. Here, we report that the lysine acetyltransferases 2A and 2B (KAT2A/2B, also called GCN5/PCAF), two well-known transcriptional regulators, acetylate a cluster of seven lysine residues (7K-patch) within the PALB2 chromatin association motif (ChAM) and, in this way, regulate context-dependent PALB2 binding to chromatin. In unperturbed cells, the 7K-patch is targeted for KAT2A/2B-mediated acetylation, which in turn enhances the direct association of PALB2 with nucleosomes. Importantly, DNA damage triggers a rapid deacetylation of ChAM and increases the overall mobility of PALB2. Distinct missense mutations of the 7K-patch render the mode of PALB2 chromatin binding, making it either unstably chromatin-bound (7Q) or randomly bound with a reduced capacity for mobilisation (7R). Significantly, both of these mutations confer a deficiency in RAD51 foci formation and increase DNA damage in S phase, leading to the reduction of overall cell survival. Thus, our study reveals that acetylation of the ChAM 7K-patch acts as a molecular switch to enable dynamic PALB2 shuttling for HR repair while protecting active genes during DNA replication.\", \"doi_y\": \"https://doi.org/10.7554/elife.57736\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4212951442\", \"doi_x\": \"10.3390/v14020441\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Na\\u00efve Human Macrophages Are Refractory to SARS-CoV-2 Infection and Exhibit a Modest Inflammatory Response Early in Infection\", \"abstract\": \"Involvement of macrophages in the SARS-CoV-2-associated cytokine storm, the excessive secretion of inflammatory/anti-viral factors leading to the acute respiratory distress syndrome (ARDS) in COVID-19 patients, is unclear. In this study, we sought to characterize the interplay between the virus and primary human monocyte-derived macrophages (MDM). MDM were stimulated with recombinant IFN-\\u03b1 and/or infected with either live or UV-inactivated SARS-CoV-2 or with two reassortant influenza viruses containing external genes from the H1N1 PR8 strain and heterologous internal genes from a highly pathogenic avian H5N1 or a low pathogenic human seasonal H1N1 strain. Virus replication was monitored by qRT-PCR for the E viral gene for SARS-CoV-2 or M gene for influenza and TCID50 or plaque assay, and cytokine levels were assessed semiquantitatively with qRT-PCR and a proteome cytokine array. We report that MDM are not susceptible to SARS-CoV-2 whereas both influenza viruses replicated in MDM, albeit abortively. We observed a modest cytokine response in SARS-CoV-2 exposed MDM with notable absence of IFN-\\u03b2 induction, which was instead strongly induced by the influenza viruses. Pre-treatment of MDM with IFN-\\u03b1 enhanced proinflammatory cytokine expression upon exposure to virus. Together, the findings concur that the hyperinflammation observed in SARS-CoV-2 infection is not driven by macrophages.\", \"doi_y\": \"https://doi.org/10.3390/v14020441\", \"publication_date\": \"2022-02-21\"}, {\"work_id\": \"W4318069271\", \"doi_x\": \"10.1128/mbio.00188-23\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Virology under the Microscope\\u2014a Call for Rational Discourse\", \"abstract\": \"Viruses have brought humanity many challenges: respiratory infection, cancer, neurological impairment and immunosuppression to name a few. Virology research over the last 60+ years has responded to reduce this disease burden with vaccines and antivirals. Despite this long history, the COVID-19 pandemic has brought unprecedented attention to the field of virology. Some of this attention is focused on concern about the safe conduct of research with human pathogens. A small but vocal group of individuals has seized upon these concerns - conflating legitimate questions about safely conducting virus-related research with uncertainties over the origins of SARS-CoV-2. The result has fueled public confusion and, in many instances, ill-informed condemnation of virology. With this article, we seek to promote a return to rational discourse. We explain the use of gain-of-function approaches in science, discuss the possible origins of SARS-CoV-2 and outline current regulatory structures that provide oversight for virological research in the United States. By offering our expertise, we - a broad group of working virologists - seek to aid policy makers in navigating these controversial issues. Balanced, evidence-based discourse is essential to addressing public concern while maintaining and expanding much-needed research in virology.\", \"doi_y\": \"https://doi.org/10.1128/mbio.00188-23\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4302276392\", \"doi_x\": \"10.1093/cid/ciac796\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Booster Vaccination Against SARS-CoV-2 Induces Potent Immune Responses in People With Human Immunodeficiency Virus\", \"abstract\": \"People with human immunodeficiency virus (HIV) on antiretroviral therapy (ART) with good CD4 T-cell counts make effective immune responses following vaccination against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). There are few data on longer term responses and the impact of a booster dose.Adults with HIV were enrolled into a single arm open label study. Two doses of ChAdOx1 nCoV-19 were followed 12 months later by a third heterologous vaccine dose. Participants had undetectable viraemia on ART and CD4 counts >350 cells/\\u00b5L. Immune responses to the ancestral strain and variants of concern were measured by anti-spike immunoglobulin G (IgG) enzyme-linked immunosorbent assay (ELISA), MesoScale Discovery (MSD) anti-spike platform, ACE-2 inhibition, activation induced marker (AIM) assay, and T-cell proliferation.In total, 54 participants received 2 doses of ChAdOx1 nCoV-19. 43 received a third dose (42 with BNT162b2; 1 with mRNA-1273) 1 year after the first dose. After the third dose, total anti-SARS-CoV-2 spike IgG titers (MSD), ACE-2 inhibition, and IgG ELISA results were significantly higher compared to Day 182 titers (P < .0001 for all 3). SARS-CoV-2 specific CD4+ T-cell responses measured by AIM against SARS-CoV-2 S1 and S2 peptide pools were significantly increased after a third vaccine compared to 6 months after a first dose, with significant increases in proliferative CD4+ and CD8+ T-cell responses to SARS-CoV-2 S1 and S2 after boosting. Responses to Alpha, Beta, Gamma, and Delta variants were boosted, although to a lesser extent for Omicron.In PWH receiving a third vaccine dose, there were significant increases in B- and T-cell immunity, including to known variants of concern (VOCs).\", \"doi_y\": \"https://doi.org/10.1093/cid/ciac796\", \"publication_date\": \"2022-10-05\"}, {\"work_id\": \"W4291169794\", \"doi_x\": \"10.1111/pim.12946\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Fungal resilience and host\\u2013pathogen interactions: Future perspectives and opportunities\", \"abstract\": \"We are constantly exposed to the threat of fungal infection. The outcome-clearance, commensalism or infection-depends largely on the ability of our innate immune defences to clear infecting fungal cells versus the success of the fungus in mounting compensatory adaptive responses. As each seeks to gain advantage during these skirmishes, the interactions between host and fungal pathogen are complex and dynamic. Nevertheless, simply compromising the physiological robustness of fungal pathogens reduces their ability to evade antifungal immunity, their virulence, and their tolerance against antifungal therapy. In this article I argue that this physiological robustness is based on a 'Resilience Network' which mechanistically links and controls fungal growth, metabolism, stress resistance and drug tolerance. The elasticity of this network probably underlies the phenotypic variability of fungal isolates and the heterogeneity of individual cells within clonal populations. Consequently, I suggest that the definition of the fungal Resilience Network represents an important goal for the future which offers the clear potential to reveal drug targets that compromise drug tolerance and synergise with current antifungal therapies.\", \"doi_y\": \"https://doi.org/10.1111/pim.12946\", \"publication_date\": \"2022-08-27\"}, {\"work_id\": \"W4360813182\", \"doi_x\": \"10.1093/femsec/fiad027\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Compensatory mutations reducing the fitness cost of plasmid carriage occur in plant rhizosphere communities\", \"abstract\": \"Plasmids drive bacterial evolutionary innovation by transferring ecologically important functions between lineages, but acquiring a plasmid often comes at a fitness cost to the host cell. Compensatory mutations, which ameliorate the cost of plasmid carriage, promote plasmid maintenance in simplified laboratory media across diverse plasmid-host associations. Whether such compensatory evolution can occur in more complex communities inhabiting natural environmental niches where evolutionary paths may be more constrained is, however, unclear. Here, we show a substantial fitness cost of carrying the large conjugative plasmid pQBR103 in Pseudomonas fluorescens SBW25 in the plant rhizosphere. This plasmid fitness cost could be ameliorated by compensatory mutations affecting the chromosomal global regulatory system gacA/gacS, which arose rapidly in plant rhizosphere communities and were exclusive to plasmid carriers. These findings expand our understanding of the importance of compensatory evolution in plasmid dynamics beyond simplified lab media. Compensatory mutations contribute to plasmid survival in bacterial populations living within complex microbial communities in their environmental niche.\", \"doi_y\": \"https://doi.org/10.1093/femsec/fiad027\", \"publication_date\": \"2023-03-23\"}, {\"work_id\": \"W4312064800\", \"doi_x\": \"10.1007/s11120-022-00990-z\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Absolute quantification of cellular levels of photosynthesis-related proteins in Synechocystis sp. PCC 6803\", \"abstract\": \"Quantifying cellular components is a basic and important step for understanding how a cell works, how it responds to environmental changes, and for re-engineering cells to produce valuable metabolites and increased biomass. We quantified proteins in the model cyanobacterium Synechocystis sp. PCC 6803 given the general importance of cyanobacteria for global photosynthesis, for synthetic biology and biotechnology research, and their ancestral relationship to the chloroplasts of plants. Four mass spectrometry methods were used to quantify cellular components involved in the biosynthesis of chlorophyll, carotenoid and bilin pigments, membrane assembly, the light reactions of photosynthesis, fixation of carbon dioxide and nitrogen, and hydrogen and sulfur metabolism. Components of biosynthetic pathways, such as those for chlorophyll or for photosystem II assembly, range between 1000 and 10,000 copies per cell, but can be tenfold higher for CO2 fixation enzymes. The most abundant subunits are those for photosystem I, with around 100,000 copies per cell, approximately 2 to fivefold higher than for photosystem II and ATP synthase, and 5-20 fold more than for the cytochrome b6f complex. Disparities between numbers of pathway enzymes, between components of electron transfer chains, and between subunits within complexes indicate possible control points for biosynthetic processes, bioenergetic reactions and for the assembly of multisubunit complexes.\", \"doi_y\": \"https://doi.org/10.1007/s11120-022-00990-z\", \"publication_date\": \"2022-12-21\"}, {\"work_id\": \"W4220875554\", \"doi_x\": \"10.7554/elife.72825\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Allele-specific gene expression can underlie altered transcript abundance in zebrafish mutants\", \"abstract\": \"In model organisms, RNA-sequencing (RNA-seq) is frequently used to assess the effect of genetic mutations on cellular and developmental processes. Typically, animals heterozygous for a mutation are crossed to produce offspring with different genotypes. Resultant embryos are grouped by genotype to compare homozygous mutant embryos to heterozygous and wild-type siblings. Genes that are differentially expressed between the groups are assumed to reveal insights into the pathways affected by the mutation. Here we show that in zebrafish, differentially expressed genes are often over-represented on the same chromosome as the mutation due to different levels of expression of alleles from different genetic backgrounds. Using an incross of haplotype-resolved wild-type fish, we found evidence of widespread allele-specific expression, which appears as differential expression when comparing embryos homozygous for a region of the genome to their siblings. When analysing mutant transcriptomes, this means that the differential expression of genes on the same chromosome as a mutation of interest may not be caused by that mutation. Typically, the genomic location of a differentially expressed gene is not considered when interpreting its importance with respect to the phenotype. This could lead to pathways being erroneously implicated or overlooked due to the noise of spurious differentially expressed genes on the same chromosome as the mutation. These observations have implications for the interpretation of RNA-seq experiments involving outbred animals and non-inbred model organisms.\", \"doi_y\": \"https://doi.org/10.7554/elife.72825\", \"publication_date\": \"2022-02-17\"}, {\"work_id\": \"W4306641762\", \"doi_x\": \"10.1073/pnas.2210109119\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"abstract\": \"The genomes of some purple photosynthetic bacteria contain a multigene puc family encoding a series of \\u03b1- and \\u03b2-polypeptides that together form a heterogeneous antenna of light-harvesting 2 (LH2) complexes. To unravel this complexity, we generated four sets of puc deletion mutants in Rhodopseudomonas palustris , each encoding a single type of pucBA gene pair and enabling the purification of complexes designated as PucA-LH2, PucB-LH2, PucD-LH2, and PucE-LH2. The structures of all four purified LH2 complexes were determined by cryogenic electron microscopy (cryo-EM) at resolutions ranging from 2.7 to 3.6 \\u00c5. Uniquely, each of these complexes contains a hitherto unknown polypeptide, \\u03b3, that forms an extended undulating ribbon that lies in the plane of the membrane and that encloses six of the nine LH2 \\u03b1\\u03b2-subunits. The \\u03b3-subunit, which is located near to the cytoplasmic side of the complex, breaks the C9 symmetry of the LH2 complex and binds six extra bacteriochlorophylls (BChls) that enhance the 800-nm absorption of each complex. The structures show that all four complexes have two complete rings of BChls, conferring absorption bands centered at 800 and 850 nm on the PucA-LH2, PucB-LH2, and PucE-LH2 complexes, but, unusually, the PucD-LH2 antenna has only a single strong near-infared (NIR) absorption peak at 803 nm. Comparison of the cryo-EM structures of these LH2 complexes reveals altered patterns of hydrogen bonds between LH2 \\u03b1\\u03b2-side chains and the bacteriochlorin rings, further emphasizing the major role that H bonds play in spectral tuning of bacterial antenna complexes.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2210109119\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4281909669\", \"doi_x\": \"10.1002/rmv.2373\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Recent insights into SARS\\u2010CoV\\u20102 omicron variant\", \"abstract\": \"The SARS-CoV-2 omicron variant (B.1.1.529) was first identified in Botswana and South Africa, and its emergence has been associated with a steep increase in the number of SARS-CoV-2 infections. The omicron variant has subsequently spread very rapidly across the world, resulting in the World Health Organization classification as a variant of concern on 26 November 2021. Since its emergence, great efforts have been made by research groups around the world that have rapidly responded to fill our gaps in knowledge for this novel variant. A growing body of data has demonstrated that the omicron variant shows high transmissibility, robust binding to human angiotensin-converting enzyme 2 receptor, attenuated viral replication, and causes less severe disease in COVID-19 patients. Further, the variant has high environmental stability, high resistance against most therapeutic antibodies, and partial escape neutralisation by antibodies from convalescent patients or vaccinated individuals. With the pandemic ongoing, there is a need for the distillation of literature from primary research into an accessible format for the community. In this review, we summarise the key discoveries related to the SARS-CoV-2 omicron variant, highlighting the gaps in knowledge that guide the field's ongoing and future work.\", \"doi_y\": \"https://doi.org/10.1002/rmv.2373\", \"publication_date\": \"2022-06-04\"}, {\"work_id\": \"W4383092885\", \"doi_x\": \"10.1016/j.jbc.2023.105014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Heme binding to the SARS-CoV-2 spike glycoprotein\", \"abstract\": \"The target for humoral immunity, SARS-CoV-2 spike glycoprotein, has become the focus of vaccine research and development. Previous work demonstrated that the N-terminal domain (NTD) of SARS-CoV-2 spike binds biliverdin-a product of heme catabolism-causing a strong allosteric effect on the activity of a subset of neutralizing antibodies. Herein, we show that the spike glycoprotein is also able to bind heme (KD = 0.5 \\u00b1 0.2 \\u03bcM). Molecular modeling indicated that the heme group fits well within the same pocket on the SARS-CoV-2 spike NTD. Lined by aromatic and hydrophobic residues (W104, V126, I129, F192, F194, I203, and L226), the pocket provides a suitable environment to stabilize the hydrophobic heme. Mutagenesis of N121 has a substantive effect on heme binding (KD = 3000 \\u00b1 220 \\u03bcM), confirming the pocket as a major heme binding location of the viral glycoprotein. Coupled oxidation experiments in the presence of ascorbate indicated that the SARS-CoV-2 glycoprotein can catalyze the slow conversion of heme to biliverdin. The heme trapping and oxidation activities of the spike may allow the virus to reduce levels of free heme during infection to facilitate evasion of the adaptive and innate immunity.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.105014\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4380365679\", \"doi_x\": \"10.1016/j.str.2023.05.012\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"abstract\": \"\\u2022We show that NMR structures typically lack hydrogen bonds\\u2022We present a protocol for gradual inclusion of hydrogen bond restraints\\u2022The resulting structures are better using many measures of quality\\u2022ANSURR provides a measure for when the NMR structure calculation is completed Protein structures calculated using NMR data are less accurate and less well-defined than they could be. Here we use the program ANSURR to show that this deficiency is at least in part due to a lack of hydrogen bond restraints. We describe a protocol to introduce hydrogen bond restraints into the structure calculation of the SH2 domain from SH2B1 in a systematic and transparent way and show that the structures generated are more accurate and better defined as a result. We also show that ANSURR can be used as a guide to know when the structure calculation is good enough to stop. Protein structures calculated using NMR data are less accurate and less well-defined than they could be. Here we use the program ANSURR to show that this deficiency is at least in part due to a lack of hydrogen bond restraints. We describe a protocol to introduce hydrogen bond restraints into the structure calculation of the SH2 domain from SH2B1 in a systematic and transparent way and show that the structures generated are more accurate and better defined as a result. We also show that ANSURR can be used as a guide to know when the structure calculation is good enough to stop. We recently introduced a new method, ANSURR (Accuracy of NMR structures Using RCI and Rigidity), which permits a simple and robust indication of the accuracy of NMR structures.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar,3Fowler N.J. Williamson M.P. The accuracy of protein structures in solution determined by AlphaFold and NMR.Structure. 2022; 30: 1-9https://doi.org/10.1101/2022.01.18.476751Crossref PubMed Scopus (0) Google Scholar We noted2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar that when estimated in this way, the accuracy of NMR protein structures improved until about 2005 but has not changed much since; and we suggested that one reason for the lack of improvement is that there has been until now no reliable way to calculate accuracy, and therefore to measure the success of technical improvements. With the introduction of ANSURR, we have a tool that increases our confidence in measuring whether new methods are effective. In this paper, we take the first step in implementing this idea, by addressing one experimental restraint that has proved difficult to implement, namely hydrogen bond restraints (HBRs). The most widely used restraints for NMR structure calculation are nuclear Overhauser effects (NOEs). These are easily obtained from nuclear Overhauser effect spectroscopy (NOESY) spectra and have (in principle) a simple interpretation, in that under appropriate conditions, an NOE peak denotes the close distance between two protons, with an intensity proportional to r\\u22126, where r is the internuclear distance.4Neuhaus D. Williamson M.P. The Nuclear Overhauser Effect in Structural and Conformational Analysis, 2nd Edition. Wiley-VCH, 2000Google Scholar Another common restraint is to use backbone chemical shifts to produce restraints on dihedral angles, for example using TALOS-N.5Shen Y. Bax A. Protein backbone and sidechain torsion angles predicted from NMR chemical shifts using artificial neural networks.J. Biomol. NMR. 2013; 56: 227-241https://doi.org/10.1007/s10858-013-9741-yCrossref PubMed Scopus (740) Google Scholar Hydrogen bonds are one of the few other common restraints and are very strong, because the distances are short and have a small range between upper and lower bounds. Restraint violations therefore typically become unfavorable with relatively small changes in geometry. For this reason, HBRs should be applied with caution, because they effectively force the relevant groups to form a hydrogen bond. Hydrogen bonds are more difficult restraints to apply than NOEs because they cannot usually be obtained directly from experimental data. NOE restraints come directly from NOESY spectra and dihedral restraints from backbone chemical shifts. It is possible to observe hydrogen bonds directly from observation of 3J across the hydrogen bond.6Cordier F. Grzesiek S. Direct observation of hydrogen bonds in proteins by interresidue 3hJNC ' scalar couplings.J. Am. Chem. Soc. 1999; 121: 1601-1602https://doi.org/10.1021/ja983945dCrossref Scopus (370) Google Scholar However, the coupling constant is small and for most proteins the signal is too weak to be detected. Therefore the presence of a hydrogen bond has to be inferred. Hydrogen bonds in regular secondary structure can be predicted based on NOE patterns in the vicinity,7W\\u00fcthrich K. Billeter M. Braun W. Polypeptide secondary structure determination by nuclear magnetic resonance observation of short proton-proton distances.J. Mol. Biol. 1984; 180: 715-740https://doi.org/10.1016/0022-2836(84)90034-2Crossref PubMed Scopus (747) Google Scholar but a more secure identification of hydrogen bonds relies on secondary factors such as slow amide exchange of the amide proton8Wagner G. Kumar A. W\\u00fcthrich K. Systematic application of two-dimensional 1H NMR techniques for studies of proteins. 2. Combined use of correlated spectroscopy and nuclear Overhauser spectroscopy for sequential ssignments of backbone resonances and elucidation of polypeptide secondary structures.Eur. J. Biochem. 1981; 114: 375-384https://doi.org/10.1111/j.1432-1033.1981.tb05157.xCrossref PubMed Scopus (272) Google Scholar,9Williamson M.P. Havel T.F. W\\u00fcthrich K. Solution conformation of proteinase inhibitor IIA from bull seminal plasma by 1H nuclear magnetic resonance and distance geometry.J. Mol. Biol. 1985; 182: 295-315Crossref PubMed Scopus (532) Google Scholar or on the amide proton temperature coefficient.10Baxter N.J. Williamson M.P. Temperature dependence of 1H chemical shifts in proteins.J. Biomol. NMR. 1997; 9: 359-369Crossref PubMed Google Scholar Neither of these is completely reliable, meaning that any inference of a hydrogen bond is potentially wrong. Significantly, amide exchange and temperature coefficients identify the amide group but do not identify the carbonyl donor. Therefore for most hydrogen bonds, identification of the donor is based on inferences not direct observation. For these reasons, NMR spectroscopists have generally been cautious in applying HBRs in protein structure calculation. There is no \\u201cstandard protocol,\\u201d but typically HBRs are only applied toward the end of the calculation, the suspicion being that it may not be scientifically justified to use them as part of the standard structure calculation. There has thus emerged a sense that HBRs are not as \\u201crespectable\\u201d as NOEs or dihedral restraints, as a result of which, the details of hydrogen bond restraints are often deeply buried in papers, and it is often very hard to work out from the literature exactly how HBRs were generated and used. This is a pity, because HBRs are such powerful restraints, and therefore very useful if used correctly. The main aim of this work is to propose a justifiable and transparent protocol. We describe a robust iterative approach to the addition of HBRs, monitored using ANSURR, and demonstrate the effect of HBRs on the structure calculation of the SH2 domain from SH2B1. We show that the accuracy of the structures improves as HBRs are added. SH2B1 is a signaling scaffold protein that acts downstream of the insulin and leptin receptors11Maures T.J. Kurzer J.H. Carter-Su C. SH2B1 (SH2-B) and JAK2: a multifunctional adaptor protein and kinase made for each other.Trends Endocrinol. Metabol. 2007; 18: 38-45Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar and regulates the activity of JAK2 kinase. Mutations to its SH2 domain have been shown to disrupt the leptin signaling pathway12Li Z. Zhou Y. Carter-Su C. Myers Jr., M.G. Rui L. SH2B1 enhances leptin signaling by both janus kinase 2 Tyr813 phosphorylation-dependent and -independent mechanisms.Mol. Endocrinol. 2007; 21: 2270-2281https://doi.org/10.1210/me.2007-0111Crossref PubMed Scopus (86) Google Scholar; deletion leads to an obese phenotype, and severely disrupts signaling downstream of the insulin receptor.13Morris D.L. Cho K.W. Zhou Y. Rui L. SH2B1 enhances insulin sensitivity by both stimulating the insulin receptor and inhibiting tyrosine dephosphorylation of insulin receptor substrate proteins.Diabetes. 2009; 58: 2039-2047https://doi.org/10.2337/db08-1388Crossref PubMed Scopus (72) Google Scholar SH2B1 consists of a dimerization domain, a PH domain, and an SH2 domain, which together form less than half of the protein, the rest being intrinsically disordered.11Maures T.J. Kurzer J.H. Carter-Su C. SH2B1 (SH2-B) and JAK2: a multifunctional adaptor protein and kinase made for each other.Trends Endocrinol. Metabol. 2007; 18: 38-45Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar Part of the function of SH2 is to bind to JAK2 kinase via its phosphorylated pTyr813 and modulate its kinase activity.14Rui L. Gunter D.R. Herrington J. Carter-Su C. Differential binding to and regulation of JAK2 by the SH2 domain and N-terminal region of SH2-B \\u03b2.Mol. Cell Biol. 2000; 20: 3168-3177https://doi.org/10.1128/mcb.20.9.3168-3177.2000Crossref PubMed Scopus (0) Google Scholar It is thus a protein with important biological effects. It is also an example of a well-studied domain with a crystal structure. For the purposes of this study this is useful because the crystal structure provides us with a useful guide to accuracy. The aim of this study is to present an iterative approach to introducing hydrogen bond restraints (HBRs) into an NMR structure calculation, and to investigate whether this improves the accuracy of the structure. Our primary measure of accuracy is ANSURR. ANSURR calculates the rigidity of a protein structure using the algorithm Floppy Inclusions and Rigid Substructure Topography (FIRST)15Jacobs D.J. Rader A.J. Kuhn L.A. Thorpe M.F. Protein flexibility predictions using graph theory.Proteins. 2001; 44: 150-165https://doi.org/10.1002/prot.1081Crossref PubMed Scopus (609) Google Scholar and compares it to the rigidity derived from experimental backbone chemical shifts via the random coil index (RCI).16Berjanskii M.V. Wishart D.S. Application of the random coil index to studying protein flexibility.J. Biomol. NMR. 2008; 40: 31-48https://doi.org/10.1007/s10858-007-9208-0Crossref PubMed Scopus (90) Google Scholar The similarity obtained by comparing these two measures provides the accuracy. The comparison uses two measures: correlation, which calculates the extent to which peaks and troughs in rigidity along the sequence match, and is primarily a measure of the correctness of local secondary structure; and RMSD, which compares rigidity along the sequence (Figure S1). Our previous studies1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar showed that although the correlation for NMR structures is often good, the RMSD is often poor, in almost all cases because NMR structures are too floppy in comparison to the rigidity indicated by backbone chemical shifts. We suggested that a major cause of this floppiness is that NMR structures have too few hydrogen bonds, in comparison to either crystal structures2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar or AlphaFold structures.3Fowler N.J. Williamson M.P. The accuracy of protein structures in solution determined by AlphaFold and NMR.Structure. 2022; 30: 1-9https://doi.org/10.1101/2022.01.18.476751Crossref PubMed Scopus (0) Google Scholar This was the rationale behind the present study. Before starting these structure calculations, we therefore undertook a more thorough study of the relationship between hydrogen bond content and accuracy. A set of 215 pairs of published NMR and X-ray structures was assembled (Table S1). This set was used to compute the average hydrogen bond correctness of each NMR ensemble, defined as the average percentage of hydrogen bonds in an X-ray structure that appear in an NMR ensemble. This calculation relies on the assumption that the hydrogen bonds in a crystal structure are also present in solution, which is clearly not completely true, but is a good approximation.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar We computed the average ANSURR score (the sum of correlation and RMSD scores, and thus a ranked score out of 200) for each ensemble and compared this to the average hydrogen bond correctness. Figure 1A shows that NMR structures with higher hydrogen bond correctness tend to be more accurate as measured by ANSURR, and also indicates that the hydrogen bonds in X-ray structures generally represent those that appear in solution. Plotting the number of hydrogen bonds per residue in each ensemble against hydrogen bond correctness (Figure 1B) demonstrates that poor scoring NMR ensembles are primarily inaccurate because they lack hydrogen bonds, rather than because they contain incorrect hydrogen bonds. We conclude that the accuracy of NMR structures will be improved by incorporation of HBRs but only if they are correct. This is important, because it implies that ANSURR scores will not improve on addition of incorrect HBRs. In the subsequent text, we therefore explore how to add HBRs, monitoring the accuracy of the resultant structures using ANSURR but also comparing to other measures. A plasmid for expression of mouse SH2 was constructed and inserted into Escherichia coli. The expressed protein is 118 residues long, with an 8-residue His-tag at the N-terminus, followed by the sequence D9QPLS \\u2026, with D9 corresponding to D519 of mature mouse SH2B. The protein was expressed and labeled uniformly with 15N and 13C. One aim of this project was to test out the automated assignment/structure calculation in FLYA/CYANA.17L\\u00f3pez-M\\u00e9ndez B. G\\u00fcntert P. Automated protein structure determination from NMR spectra.J. Am. Chem. Soc. 2006; 128: 13112-13122Crossref PubMed Scopus (136) Google Scholar The NMR spectrum was therefore assigned automatically using CYANA, as well as manually. The NMR spectra used are listed in Table S2, and the final manually assigned HSQC is shown in Figure S2. A comparison of the two assignments showed that 99% of the automated backbone chemical shift assignments (1HN, 15N, 1H\\u03b1, 13C\\u03bf, 13C\\u03b1, and 13C\\u03b2) of SH2 (9\\u2013118) agreed with the manual assignment. The only resonances that were not assigned correctly by the automated calculation are the amide chemical shifts of Trp17 and Gln61, which are two of the four resonances that had missing signals (but assigned H and N) in the manual assignment. For aliphatic methyl and methylene sidechain groups, the agreement was 97%. On the other hand, many of the NOE-based chemical shift assignments did not match the manual assignment. This includes the amide sidechains of Arg, Asn, and Gln, and many aromatics: only 53% of these were assigned correctly (Figure S3). CYANA continues to check and refine its assignments throughout the structure calculations, and it is possible that some of these incorrect assignments would have been corrected. On the other hand, the accuracy of the automated structure calculation relies heavily on the accuracy of the first iteration, which in turn depends on the accuracy of the initial set of assignments.18G\\u00fcntert P. Buchner L. Combined automated NOE assignment and structure calculation with CYANA.J. Biomol. NMR. 2015; 62: 453-471https://doi.org/10.1007/s10858-015-9924-9Crossref PubMed Scopus (233) Google Scholar The clear implication is that assignments should be checked manually before doing automated structure calculations.19Williamson M.P. Craven C.J. Automated protein structure calculation from NMR data.J. Biomol. NMR. 2009; 43: 131-143Crossref PubMed Scopus (0) Google Scholar A manually checked set of assignments have been deposited in BioMagResBank20Ulrich E.L. Akutsu H. Doreleijers J.F. Harano Y. Ioannidis Y.E. Lin J. Livny M. Mading S. Maziuk D. Miller Z. et al.BioMagResBank. Nucleic Acids Res. 2008; 36: D402-D408Crossref PubMed Scopus (1314) Google Scholar under code 51342. Structure calculation was conducted using the standard automated routines within CYANA. We used the default CYANA calculation of 20 structures. We then repeated this calculation 60 times changing the random seed each time, in order to obtain a better \\u201cconsensus bundle\\u201d of structures,21Buchner L. G\\u00fcntert P. Increased reliability of nuclear magnetic resonance protein structures by consensus structure bundles.Structure. 2015; 23: 425-434https://doi.org/10.1016/j.str.2014.11.014Abstract Full Text Full Text PDF PubMed Scopus (21) Google Scholar giving a total of 1200 calculated structures per iteration. The reason for carrying out so many calculations was to generate a reliable set of HBRs, with good discrimination between correct and incorrect HBRs. The figure of 60 was determined by calculating the ratio of true positive HBRs (that is, HBRs identified that are also present in the crystal structure) to false positive HBRs (HBRs identified that are not in the crystal structure). This ratio improved with the number of calculations, reaching a plateau at 60 (Figure S4). HBRs were identified from these structures and used as restraints for the next iteration, as described below. The initial round of CYANA calculations produced an ensemble with a correct fold but low accuracy (as measured by ANSURR), and had poor convergence, a large RMSD between structures, and poor energies, all symptoms of a weak restraint set. We therefore improved the structures by iteratively and systematically including backbone hydrogen bond restraints, in such a way as to avoid incorrectly biasing the structure calculation (Figure 2). The protocol aims to generate HBRs that are present in a significant fraction of ensemble structures, supported by experimental temperature coefficients and/or amide exchange rates, and are consistent with other restraints. Iteration 6 was introduced as a way of removing any HBRs that were incompatible with the experimental restraints. The complete process identified 47 hydrogen bond restraints, mostly in regular secondary structure. The final set of HBRs is illustrated in Figure 3. Not surprisingly, the first HBRs to be identified came from the helices and from the \\u201ccore\\u201d \\u03b2-strands (residues 41\\u201369). The shorter \\u03b2-sheet HBRs tended to be added later. Many of the HBRs that do not form part of regular secondary structures (for example, HBRs from the amide protons of G14, Y15, G104, and V109) were identified in the final stages. More details are available in Table S3, which also shows that most of the HBRs that were added in an early iteration and subsequently removed are present in the crystal structure. At each stage in the calculation, the structures were analyzed to measure whether the additional HBRs were improving the quality of the ensemble. Our primary analytical tool was ANSURR. Figure 4 shows how ANSURR scores steadily increase as the number of HBRs increases, suggesting that inclusion of hydrogen bond restraints does improve accuracy. The best scoring models in the final iteration 6 approach the accuracy of AlphaFold222Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. Tunyasuvunakool K. Bates R. \\u017d\\u00eddek A. Potapenko A. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589https://doi.org/10.1038/s41586-021-03819-2Crossref PubMed Scopus (6941) Google Scholar and Rosetta-Fold23Baek M. DiMaio F. Anishchenko I. Dauparas J. Ovchinnikov S. Lee G.R. Wang J. Cong Q. Kinch L.N. Schaeffer R.D. et al.Accurate prediction of protein structures and interactions using a three-track neural network.Science. 2021; 373: 871-876https://doi.org/10.1126/science.abj8754Crossref PubMed Scopus (1299) Google Scholar (Figure 4A), although the average values remain worse. Note that the structures shown here come directly from CYANA and have not yet been refined in explicit solvent. The final iteration was designed to remove HBRs that do not have good supporting evidence. This iteration removed 4 out of 51 HBRs, leading to a drop in ANSURR score as well as some of the other measures of accuracy discussed below. The implication is that we probably removed correct HBRs, so this final step may have been over-cautious. Figures 4B\\u20134G depicts the ANSURR output for the best scoring model from each iteration. Figures 4H and 4I show ANSURR output for the X-ray structure (PDB: 5w3r) and the best scoring AlphaFold2 prediction, respectively. The crystal structure has a very good ANSURR score, but appears to be more rigid than the \\u201ctrue\\u201d solution structure around residues 70\\u201390, judging by the rigidities shown in Figure 4H. With this exception, it would appear to be an excellent guide to the hydrogen bonds present in solution. In support, in Figure S5 we present an analysis of the HBRs at each iteration compared to those in the crystal structure, and show that they become progressively closer to those in the crystal structure. We note that the amino acid sequence of the proteins used for NMR and crystallography24McKercher M.A. Guan X. Tan Z. Wuttke D.S. Diversity in peptide recognition by the SH2 domain of SH2B1.Proteins. 2018; 86: 164-176https://doi.org/10.1002/prot.25420Crossref PubMed Scopus (4) Google Scholar (mouse and human, respectively) differ in 5 locations (Figure S6). It is of interest that the region for which NMR does worst is residues 46\\u201355, which is too flexible in the NMR structures but is better in the X-ray structure. In the crystal, there is a bound phosphate here. This is where phosphotyrosine binds. The residues in contact with the phosphate in the crystal structure (residues 47\\u201349) appear to be too rigid, suggesting that the phosphate is more tightly bound in the crystal than in solution, or perhaps that the flexibility according to chemical shifts represents an average of a bound state and unbound state. We have compared backbone chemical shifts for protein in 50 mM Tris buffer and in 50 mM phosphate buffer, pH 6 (Figure S7). It is striking that almost all of the large chemical shift changes surround the location of the phosphate ion in the crystal structure, implying that phosphate binds here in 50 mM phosphate buffer. This may serve to rigidify the region, and explain why the NMR structure, which was determined in tris buffer, has low rigidity here. The previous section showed that inclusion of HBRs leads to a steady improvement in ANSURR score: i.e., a more accurate structure. Here, we report on other commonly used measures of quality, to characterize the structural improvement in more detail. The results are shown in Figure 5. The magnitude of NOE violations increases gradually as HBRs are included, although it remains low throughout (Figure 5A). The calculation of an NMR structure is a balance between forces pulling in different directions: for example, van der Waals violations push atoms apart, while NOE violations pull them together. For this reason, it is generally found that addition of a new type of restraint leads to a slight increase in violations of existing restraints.25Schwieters C.D. Bermejo G.A. Clore G.M. A three-dimensional potential of mean force to improve backbone and sidechain hydrogen bond geometry in Xplor-NIH protein structure determination.Protein Sci. 2020; 29: 100-110https://doi.org/10.1002/pro.3745Crossref PubMed Scopus (9) Google Scholar We therefore do not see this result as surprising or indicative of structural errors. In a similar way, the magnitude of dihedral angle violations has a small but insignificant increase (Figure 5B), as do van der Waals violations (data not shown). The RMSD of backbone atoms to the crystal structure improves on initial addition of HBRs, up to iteration 4 after which there is little change (Figure 5C). This can be understood by the argument that HBRs are excellent for improving local geometry, but make little difference to the positions of backbone atoms. It is widely agreed that the template modeling score (TM-score) is a more accurate measure of global similarity than RMSD,26Zhang Y. Skolnick J. Scoring function for automated assessment of protein structure template quality.Proteins. 2004; 57: 702-710https://doi.org/10.1002/prot.20264Crossref PubMed Scopus (1268) Google Scholar and we see a more clear trend for TM-score than for RMSD (Figure 5D). TM-score slightly increases up until iteration 4. Residual dipolar couplings (RDCs) can be used either to validate the quality of NMR structures or as restraints to improve NMR structures, but not both at the same time (unless steps are taken to separate them into different groups).27Chiliveri S.C. Robertson A.J. Shen Y. Torchia D.A. Bax A. Advances in NMR spectroscopy of weakly aligned biomolecular systems.Chem. Rev. 2022; 122: 9307-9330https://doi.org/10.1021/acs.chemrev.1c00730Crossref PubMed Scopus (6) Google Scholar Here we measured a set of 1DNH as a simple validation measure. Figure 5E shows the RDC Q factor,28Cornilescu G. Marquardt J.L. Ottiger M. Bax A. Validation of protein structure from anisotropic carbonyl chemical shifts in a dilute liquid crystalline phase.J. Am. Chem. Soc. 1998; 120: 6836-6837https://doi.org/10.1021/ja9812610Crossref Scopus (787) Google Scholar which improves up to iteration 4. The chemical shifts of amide proteins should be a sensitive measure of structural accuracy, because they are strongly affected by hydrogen bonding.29Asakura T. Taoka K. Demura M. Williamson M.P. The relationship between amide proton chemical shifts and secondary structure in proteins.J. Biomol. NMR. 1995; 6: 227-236Crossref PubMed Scopus (115) Google Scholar We therefore calculated the expected amide proton shifts using SHIFTX230Han B. Liu Y. Ginzinger S.W. Wishart D.S. SHIFTX2: significantly improved protein chemical shift prediction.J. Biomol. NMR. 2011; 50: 43-57Crossref PubMed Scopus (468) Google Scholar and compared them to the experimental shifts (Figure 5F), which improve steadily as HBRs are added. The distribution of Ramachandran angles also improves slightly up to iteration 4 (Figure 5G). Finally, we have calculated the backbone ensemble RMSD (Figure 5H). This is a measure of precision rather than accuracy, but our survey of PDB structures indicated that it does correlate with accuracy, presumably because adding more (correct) restraints serves to improve both accuracy and precision at the same time2Fowler N.J. Sljoka A. Williamson M.P. The accuracy of NMR protein structures in the Protein Data Bank.Structure. 2021; 29: 1430-1439.e2https://doi.org/10.1016/j.str.2021.07.001Abstract Full Text Full Text PDF PubMed Scopus (3) Google Scholar; and indeed, RMSD decreases steadily as HBRs are added. Thus in summary, the gradual addition of 47 HBRs results in an improvement in accuracy, as measured by a wide range of parameters, with the improvement in accuracy slowing down as the number of HBRs increases. These metrics clearly show that there is overall no increased penalty to the structure quality from their inclusion: in other words, the HBRs are correct and do not conflict with the other structural restraints. The accuracy clearly improves until iteration 4. Iteration 5 results in an improvement on some measures but a deterioration in others, while iteration 6 (in which some HBRs were removed for lack of good evidence) represents a deterioration in almost all measures, the exception being precision (Figure 5H). It is well established that refinement in solvent is required in order to produce good quality structures.1Fowler N.J. Sljoka A. Williamson M.P. A method for validating the accuracy of NMR protein structures.Nat. Commun. 2020; 11: 6321https://doi.org/10.1038/s41467-020-20177-1Crossref PubMed Scopus (24) Google Scholar,31Linge J.P. Williams M.A. Spronk C.A.E.M. Bonvin A.M.J.J. Nilges M. Refinement of protein structures in explicit solvent.Proteins. 2003; 50: 496-506https://doi.org/10.1002/prot.10299Crossref PubMed Scopus (544) Google Scholar The 1,200 structures produced in each iteration above were therefore refined in water using CNS.32Br\\u00fcnger A.T. Adams P.D. Clore G.M. DeLano W.L. Gros P. Grosse-Kunstleve R.W. Jiang J.S. Kuszewski J. Nilges M. Pannu N.S. et al.Crystallography & NMR system: a new software suite for macromolecular structure determination.Acta Crystallogr. D. 1998; 54: 905-921Crossref PubMed Scopus (16979) Google Scholar There is a reasonable correlation between the energies before and after refinement (Figure S8). The distance, dihedral a\", \"doi_y\": \"https://doi.org/10.1016/j.str.2023.05.012\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4312197546\", \"doi_x\": \"10.1101/2022.12.22.521579\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Beta secretase BACE1 drives Systemic Sclerosis fibroblasts activation through \\u03b2-catenin and Notch signalling\", \"abstract\": \"Abstract Background The beta-amyloid precursor protein cleaving enzyme 1 (BACE1) is well known for its role in the development of Alzheimer\\u2019s disease via the generation of \\u03b2-amyloid. Recent publications, including our own, have demonstrated a role for this enzyme in other chronic inflammatory diseases, including type 2 diabetes and cardiovascular disease. However, to date there has been no studies looking into the role of BACE1 in the autoimmune condition Systemic Sclerosis (SSc). The aim of this study was to investigate the role of BACE1 in SSc tissue fibrosis. Methods Patient fibroblasts were obtained from full thickness forearm skin biopsies from healthy and early diffuse SSc patients. BACE1 was inhibited with the small molecule inhibitors M-3 and AZD3839 and siRNA specific to BACE1. Morphogen signalling was activated with recombinant TGF-\\u03b2, Wnt-3a or the smoothened agonist SAG. Xenotransplant mouse model using patient pDC was used to interrogate in vivo expression of BACE1 in fibrosis. Results Here we show that BACE1 protein levels are elevated in SSc patient skin biopsies, as well as dermal fibroblasts and in mouse skin during fibrosis. Inhibition of BACE1 with small molecule inhibitors or siRNA blocked SSc and morphogen mediated fibroblast pro-fibrotic activation. Furthermore, we show that BACE1 regulation of dermal fibroblast activation is dependent on the \\u03b2-catenin and Notch signalling pathway activation. Conclusions The ability of BACE1 to regulate SSc fibroblast activation reveals an exciting new therapeutic target in SSc. Several BACE1 inhibitors, including AZD3839, have been shown to be safe in clinical trials for Alzheimer\\u2019s disease.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.22.521579\", \"publication_date\": \"2022-12-22\"}, {\"work_id\": \"W4312692745\", \"doi_x\": \"10.1101/2022.05.26.493611\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"An amino acid transporter AAT1 plays a pivotal role in chloroquine resistance evolution in malaria parasites\", \"abstract\": \"Malaria parasites break down host hemoglobin into peptides and amino acids in the digestive vacuole for export to the parasite cytoplasm for growth: interrupting this process is central to the mode of action of several antimalarial drugs. Mutations in the chloroquine (CQ) resistance transporter, pfcrt , located in the digestive vacuole membrane, confer CQ resistance in Plasmodium falciparum , but typically affect parasite fitness. However, the role of other parasite loci in the evolution of CQ resistance is unclear. Here we use a combination of population genomics, genetic crosses and gene editing to demonstrate that a second vacuolar transporter plays a key role in both resistance and compensatory evolution. Longitudinal genomic analyses of the Gambian parasites revealed temporal signatures of selection on an amino acid transporter ( pfaat1) S258 L variant which increased from 0-87% in frequency between 1984 and 2014 in parallel with pfcrt1 K76 T . Parasite genetic crosses then identified a chromosome 6 quantitative trait locus containing pfaat1 that is selected by CQ treatment. Gene editing demonstrated that pfaat1 S258 L potentiates CQ-resistance but at a cost of reduced fitness, while pfaat1 F313 S , a common Southeast Asian polymorphism, reduces CQ-resistance while restoring fitness. Our analyses reveal hidden complexity in CQ-resistance evolution, suggesting that pfaat1 may underlie regional differences in the dynamics of resistance evolution, and modulate parasite resistance or fitness by manipulating the balance between both amino acid and drug transport.\", \"doi_y\": \"https://doi.org/10.1101/2022.05.26.493611\", \"publication_date\": \"2022-05-26\"}, {\"work_id\": \"W4386419407\", \"doi_x\": \"10.1002/fes3.498\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Reviving grain quality in wheat through non\\u2010destructive phenotyping techniques like hyperspectral imaging\", \"abstract\": \"Abstract A long\\u2010term goal of breeders and researchers is to develop crop varieties that can resist environmental stressors and produce high yields. However, prioritising yield often compromises improvement of other key traits, including grain quality, which is tedious and time\\u2010consuming to measure because of the frequent involvement of destructive phenotyping methods. Recently, non\\u2010destructive methods such as hyperspectral imaging (HSI) have gained attention in the food industry for studying wheat grain quality. HSI can quantify variations in individual grains, helping to differentiate high\\u2010quality grains from those of low quality. In this review, we discuss the reduction of wheat genetic diversity underlying grain quality traits due to modern breeding, key traits for grain quality, traditional methods for studying grain quality and the application of HSI to study grain quality traits in wheat and its scope in breeding. Our critical review of literature on wheat domestication, grain quality traits and innovative technology introduces approaches that could help improve grain quality in wheat.\", \"doi_y\": \"https://doi.org/10.1002/fes3.498\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4311105850\", \"doi_x\": \"10.1101/2022.12.01.518684\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Joint Multi-Ancestry and Admixed GWAS Reveals the Complex Genetics behind Human Cranial Vault Shape\", \"abstract\": \"Abstract The cranial vault \\u2013 the portion of the skull surrounding the brain and cerebellum \\u2013 is highly variable, clinically relevant, and heritable, yet its genetic architecture remains poorly understood. Here, we conducted a joint multi-ancestry and admixed multivariate GWAS on 3D cranial vault shape extracted from magnetic resonance images of 6,772 children from the ABCD study cohort, identifying 30 genome-wide significant genetic loci and replicating 20 of these signals in 16,947 additional individuals of the UK Biobank. This joint multi-ancestry GWAS was enriched for genetic components of cranial vault shape shared across ancestral groups and yielded a greater discovery than a European-only GWAS. We present supporting evidence for parietal versus frontal bone localization for several of the identified genes based on expression patterns in E15.5 mice. Collectively, our GWAS loci were enriched for processes related to skeletal development and showed elevated activity in cranial neural crest cells, suggesting a role during early craniofacial development. Among the identified genes, were RUNX2 and several of its upstream and downstream actors, highlighting the prominent role of intramembranous ossification \\u2013 which takes place at the cranial sutures \\u2013 in influencing cranial vault shape. We found that mutations in many genes associated with craniosynostosis exert their pathogenicity by modulating the same pathways involved in normal cranial vault development. This was further demonstrated in a non-syndromic sagittal craniosynostosis case-parent trio dataset of 63 probands (n = 189), where our GWAS signals near BMP2, BBS9 , and ZIC2 contributed significantly to disease risk. Moreover, we found strong evidence of overlap with genes influencing the morphology of the face and the brain, suggesting a common genetic architecture connecting these developmentally adjacent structures. Overall, our study provides a comprehensive overview of the genetics underlying normal cranial vault shape and its relevance for understanding modern human craniofacial diversity and the etiology of congenital malformations.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.01.518684\", \"publication_date\": \"2022-12-02\"}, {\"work_id\": \"W4225150583\", \"doi_x\": \"10.1161/circresaha.122.321066\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Junctophillin-2: Coupling Hopes for Cardiac Gene Therapy to Gene Transcription\", \"abstract\": \"HomeCirculation ResearchVol. 130, No. 9Junctophillin-2: Coupling Hopes for Cardiac Gene Therapy to Gene Transcription Free AccessEditorialPDF/EPUBAboutView PDFView EPUBSections ToolsAdd to favoritesDownload citationsTrack citationsPermissions ShareShare onFacebookTwitterLinked InMendeleyReddit Jump toFree AccessEditorialPDF/EPUBJunctophillin-2: Coupling Hopes for Cardiac Gene Therapy to Gene Transcription Peter Wright and Julia Gorelik Peter WrightPeter Wright School of Life and Health Sciences, University of Roehampton, London, United Kingdom (P.W.). National Heart, and Lung Institute (NHLI), Faculty of Medicine, Imperial College London, United Kingdom (P.W., J.G.). Search for more papers by this author and Julia GorelikJulia Gorelik Correspondence to: Julia Gorelik, PhD, Cardiac Section, National Heart, and Lung Institute (NHLI), Faculty of Medicine, Imperial College London, Hammersmith Campus, Du Cane Rd, London W12 0NN, United Kingdom. Email E-mail Address: [email protected] https://orcid.org/0000-0003-1148-9158 National Heart, and Lung Institute (NHLI), Faculty of Medicine, Imperial College London, United Kingdom (P.W., J.G.). Search for more papers by this author Originally published28 Apr 2022https://doi.org/10.1161/CIRCRESAHA.122.321066Circulation Research. 2022;130:1318\\u20131320This article is a commentary on the followingGene Therapy With the N-Terminus of Junctophilin-2 Improves Heart Failure in MiceHeart failure continues to have devasting societal and economic impacts. Current gold-standard pharmacological treatments only provide symptomatic relief, meaning that surgical replacement of failing hearts (through transplant) remains the sole curative intervention.1 In the past decade, hopes were raised that therapies using (adeno and adeno-associated [AAV]) viral vectors\\u2014targeting aberrant cellular genetic programs, would restore a normal cardiac phenotype by modulating the expression of crucial proteins.2 The 2 sets of human clinical trials which attempted these gene therapies, sought to modulate the expression of sarcoplasmic reticulum calcium ATPase3and adenyl cyclase 6.4 These proteins are crucial components of cardiomyocyte excitation-contraction coupling machinery modulating the abbreviation of cellular calcium-induced calcium release and the phosphorylation/activation of calcium-handling proteins, respectively. In the aftermath of these trials which had neutral results, researchers began to question whether directly targeting excitation-contraction coupling was the correct approach. The pathological remodeling of the myocardium during heart failure is driven by the activation of a fetal gene program. This hypertrophic program modulates a panel of genes causing significant changes to cardiomyocyte structure, which are initially compensatory, but deleterious in the medium to long-term. Thus, targeting the transcriptional machinery which controls hypertrophic signaling, has been suggested as an alternative approach for therapies.5Article, see p 1306In this issue, Wang et al6 describe an approach that may bridge the two approaches through the singular behavior of the structural protein JPH-2 (junctophillin 2).The plasma membrane of the adult cardiomyocyte is penetrated at regular intervals by tubular elements, which form part of a wider transverse-axial tubular system. The transverse-axial tubular system permits the transduction of ion currents rapidly and synchronously throughout the cardiomyocyte, where diffusion will not suffice. L-type calcium channels (LTCCs) modulate calcium-induced calcium release at junctional membrane complexes within the transverse-axial tubular system. JPH-2, a scaffolding protein, recruits functional LTCCs to the junctional membrane complexes, associating them with the ryanodine receptors which drive the release of calcium ions through calcium-induced calcium release.7 JPH-2 supports the structural integrity of the transverse-axial tubular system through a cholesterol dependent mechanism and interaction with the cytoskeleton through microtubules.8,9 Alongside, these crucial functions JPH-2 was discovered to have, potentially salutary, transcription factor activity.10 In situations of cellular stress JPH-2 can be cleaved by calpain-1. This process yields an N-terminal peptide (JPH-2NT) which can move to the nucleus and bind to TATA-box elements which has the effect of repressing the prohypertrophic gene program induced by cardiac pathology. This unusual, multifaceted behavior provides an important homeostatic mechanism linking the structures which control excitation-contraction coupling and the processes which govern the induction of prohypertrophic signaling.10Wang et al6 have demonstrated that mice overexpressing JPH-2NT have a significantly milder cardiac phenotype following the induction of cardiac hypertrophy following transverse aortic constriction (TAC). The hearts of mice overexpressing JPH-2NT display much less fibrosis and a preserved cellular TAT network. An investigation of the genetic changes induced by JPH-2NT suggests that this is, in part, due to the reversal of deleterious prohypertrophic signaling programs. Overexpression of JPH-2NT is a promising strategy for cardiac gene therapy as it retains the useful properties of earlier gene therapy approaches (maintaining inotropy and preserving cellular ultrastructure) but also offers the broadband reversal of hypertrophic signaling.Neonatal mice were infected with AAV (2/9) vectors expressing JPH-2NT or a mutant peptide incapable of binding DNA (\\u0394bNLS/ARR). After 9 weeks, the mice were subjected to transaortic constriction to induce cardiac stress and failure due to hypertrophy. After 5 weeks of TAC, mice that received the bona fide JPH-2NT retained a significant degree of cardiac functionality, exhibited far less cardiac remodeling and fibrosis in comparison to mice that received eGFP (enhanced green fluorescent protein) or \\u0394bNLS/ARR. None of the vectors induced aberrant effects in the SHAM control animals. The positive functional effects of JPH-2NT overexpression in TAC animals appeared to correlate with changes in the cardiomyocytes at the ultrastructural level. TAT regularity was preserved in mice that received JPH-2NT. Importantly, the presence of AAV was confirmed in cardiomyocytes infected by the 3 vectors. Only cells infected with AAV-expressing JPH-2NT had preserved TAT integrity (at levels similar to SHAM animals). The phenotype of mice subjected to TAC continued to worsen past 8 weeks in animals infected with AAV-eGFP but animals which received AAV-JPH-2NT escaped the phenotype. An RNA-Seq investigation demonstrated that JPH-2NT infection prevented the induction of hypertrophic and heart failure related gene programs.As discussed, human clinical trials of cardiac gene therapy have so far only targeted 2 genes, with indifferent results. A significant number of other preclinical trials (in mice and large animal species) have shown efficacy of a gene targeting approach, across different phenotypes, but failed to make the transition to human trials.2 Numerous potential reasons for the inability of human trials to meet their primary goals have been posited, including issues with the vector, gene, delivery, human heterogeneity, and the designs of studies themselves. The purview of this study was to investigate a better genetic target. A lack of beneficial effect of protein overexpression or immunogenicity have been speculated to be the reasons for nonefficacy stemming from the target selection. In this instance, JPH-2NT undoubtedly has efficacy in this preclinical model and there were no indications of adverse immune effects. JPH-2NT based therapy offers the potential benefits indicated by early investigations of gene therapy with sarcoplasmic reticulum calcium ATPase in preclinical models. These include the restoration of cardiac function and the maintenance of normal cardiomyocyte ultrastructure.11 As a therapeutic strategy JPH-2NT also offers a broadband modification of the induction of prohypertrophic transcription. Numerous transcriptional regulator targets, such as BRD4, CDK8/9, and HDAC, have been identified for pharmacological therapies but targeting these processes with genetic manipulation offers a more sustainable approach.5This study prepares mice by infecting them with AAV before inducing cardiac damage. Therefore, the phenomena that the article presents is the escape from the cardiac phenotype induced by TAC after JPH-2NT delivery. Clearly, this approach is not currently viable (or perhaps even desirable) in human patients. Therefore, the focus of further studies should be to evaluate whether JPH-2NT application to wild-type mice with preexisting heart failure, has a therapeutic effect. One important finding of the study relates the choice of vector. AAV2 remains the only rationally designed AAV-vector with excellent tropism and transduction efficiency for cardiomyocytes. The liver tropism which seems to ruin the efficacy of other vectors is not apparent in this study.12 A significant amount of the viral vector seems to collect in the skeletal muscle of infected animals. JPH-2 performs extensive functions within skeletal muscle (another type of striated muscle) which are extremely similar to those performed in cardiac muscle.13 The significance of the overexpression of JPH-2 in skeletal muscle induced by this vector should, therefore, be explored. Unlike, cardiac muscle, skeletal muscle hypertrophy seems to invariably associate with improved health outcomes, especially in senescent patients. But the specific effect of JPH-2NT on skeletal muscle hypertrophy is unclear.14 Finally, it must be mentioned that the potential utility of the JPH-2 molecule, with respect to transcription factor manipulation, is not limited to the N terminus, given that a fragment of the C terminus has also been demonstrated to have transcriptional activity.15In summary, promoting beneficial anti-hypertrophic signaling and the preservation of cell structure by overexpressing the JPH-2NT fragment is a promising avenue for cardiac therapy. Targeting JPH-2 related mechanisms recapitulates the useful effects of past gene therapy approaches linking maintenance of excitation-contraction-coupling with a novel broader effect on prohypertrophic gene programs.Article InformationSources of FundingThis work was supported by British Heart Foundation Grant (RG/17/13/33173 to Dr Gorelik).Disclosures None.FootnotesThe opinions expressed in this article are not necessarily those of the editors or of the American Heart Association.For Sources of Funding and Disclosures, see page 1319 & 1320.Correspondence to: Julia Gorelik, PhD, Cardiac Section, National Heart, and Lung Institute (NHLI), Faculty of Medicine, Imperial College London, Hammersmith Campus, Du Cane Rd, London W12 0NN, United Kingdom. Email j.[email protected]ac.ukReferences1. Virani SS, Alonso A, Aparicio HJ, Benjamin EJ, Bittencourt MS, Callaway CW, Carson AP, Chamberlain AM, Cheng S, Delling FN, et al; American Heart Association Council on Epidemiology and Prevention Statistics Committee and Stroke Statistics Subcommittee. Heart disease and stroke statistics-2021 update: a report from the American Heart Association.Circulation. 2021; 143:e254\\u2013e743. doi: 10.1161/CIR.0000000000000950LinkGoogle Scholar2. Ishikawa K, Weber T, Hajjar RJ. Human cardiac gene therapy.Circ Res. 2018; 123:601\\u2013613. doi: 10.1161/CIRCRESAHA.118.311587LinkGoogle Scholar3. Greenberg B, Butler J, Felker GM, Ponikowski P, Voors AA, Desai AS, Barnard D, Bouchard A, Jaski B, Lyon AR, et al. Calcium upregulation by percutaneous administration of gene therapy in patients with cardiac disease (CUPID 2): a randomised, multinational, double-blind, placebo-controlled, phase 2b trial.Lancet. 2016; 387:1178\\u20131186. doi: 10.1016/S0140-6736(16)00082-9CrossrefMedlineGoogle Scholar4. Hammond HK, Penny WF, Traverse JH, Henry TD, Watkins MW, Yancy CW, Sweis RN, Adler ED, Patel AN, Murray DR, et al. Intracoronary gene transfer of adenylyl cyclase 6 in patients with heart failure: a randomized clinical trial.JAMA Cardiol. 2016; 1:163\\u2013171. doi: 10.1001/jamacardio.2016.0008CrossrefMedlineGoogle Scholar5. Minerath RA, Hall DD, Grueter CE. Targeting transcriptional machinery to inhibit enhancer-driven gene expression in heart failure.Heart Fail Rev. 2019; 24:725\\u2013741. doi: 10.1007/s10741-019-09792-3CrossrefMedlineGoogle Scholar6. Wang J, Shi Q, Wang Y, Dawson LW, Ciampa G, Zhao W, Zhang G, Chen B, Weiss RM, Grueter CE, et al. Gene therapy with the N-terminus of Junctophilin-2 improves heart failure in mice.Circ Res. 2022; 130:1306\\u20131317. doi: 10.1161/CIRCRESAHA.121.320680LinkGoogle Scholar7. Gross P, Johnson J, Romero CM, Eaton DM, Poulet C, Sanchez-Alonso J, Lucarelli C, Ross J, Gibb AA, Garbincius JF, et al. Interaction of the joining region in Junctophilin-2 With the L-Type Ca2+ Channel Is pivotal for cardiac dyad assembly and intracellular Ca2+ dynamics.Circ Res. 2021; 128:92\\u2013114. doi: 10.1161/CIRCRESAHA.119.315715LinkGoogle Scholar8. Poulet C, Sanchez-Alonso J, Swiatlowska P, Mouy F, Lucarelli C, Alvarez-Laviada A, Gross P, Terracciano C, Houser S, Gorelik J. Junctophilin-2 tethers T-tubules and recruits functional L-type calcium channels to lipid rafts in adult cardiomyocytes.Cardiovasc Res. 2021; 117:149\\u2013161. doi: 10.1093/cvr/cvaa033CrossrefMedlineGoogle Scholar9. Zhang C, Chen B, Guo A, Zhu Y, Miller JD, Gao S, Yuan C, Kutschke W, Zimmerman K, Weiss RM, et al. Microtubule-mediated defects in junctophilin-2 trafficking contribute to myocyte transverse-tubule remodeling and Ca2+ handling dysfunction in heart failure.Circulation. 2014; 129:1742\\u20131750. doi: 10.1161/CIRCULATIONAHA.113.008452LinkGoogle Scholar10. Ang G, Yihui W, Biyi C, Yunhao W, Jinxiang Y, Liyang Z, Duane H, Jennifer W, Yun S, Qi Z, et al. E-C coupling structural protein junctophilin-2 encodes a stress-adaptive transcription regulator.Science (80-). 2018; 362:eaan3303. doi: 10.1126/science.aan3303CrossrefMedlineGoogle Scholar11. Lyon AR, Nikolaev VO, Miragoli M, Sikkel MB, Paur H, Benard L, Hulot J-S, Kohlbrenner E, Hajjar RJ, Peters NS, et al. Plasticity of surface structures and \\u03b22-adrenergic receptor localization in failing ventricular cardiomyocytes during recovery from heart failure.Circ Hear Fail. 2012; 5:357\\u2013365. doi: 10.1161/CIRCHEARTFAILURE.111.964692LinkGoogle Scholar12. Asokan A, Conway JC, Phillips JL, Li C, Hegge J, Sinnott R, Yadav S, DiPrimio N, Nam HJ, Agbandje-McKenna M, et al. Reengineering a receptor footprint of adeno-associated virus enables selective and systemic gene transfer to muscle.Nat Biotechnol. 2010; 28:79\\u201382. doi: 10.1038/nbt.1599CrossrefMedlineGoogle Scholar13. Perni S. The Builders of the junction: roles of Junctophilin1 and Junctophilin2 in the assembly of the sarcoplasmic reticulum&ndash;plasma membrane junctions in striated muscle.Biomolecules. 2022; 12:109. doi: 10.3390/biom12010109CrossrefMedlineGoogle Scholar14. Springer J, Springer JI, Anker SD. Muscle wasting and sarcopenia in heart failure and beyond: update 2017.ESC Heart Fail. 2017; 4:492\\u2013498. doi: 10.1002/ehf2.12237CrossrefMedlineGoogle Scholar15. Lahiri SK, Quick AP, Samson-Couterie B, Hulsurkar M, Elzenaar I, van Oort RJ, Wehrens XHT. Nuclear localization of a novel calpain-2 mediated junctophilin-2 C-terminal cleavage peptide promotes cardiomyocyte remodeling.Basic Res Cardiol. 2020; 115:49. doi: 10.1007/s00395-020-0807-1CrossrefMedlineGoogle Scholar eLetters(0)eLetters should relate to an article recently published in the journal and are not a forum for providing unpublished data. Comments are reviewed for appropriate use of tone and language. Comments are not peer-reviewed. Acceptable comments are posted to the journal website only. Comments are not published in an issue and are not indexed in PubMed. Comments should be no longer than 500 words and will only be posted online. References are limited to 10. Authors of the article cited in the comment will be invited to reply, as appropriate.Comments and feedback on AHA/ASA Scientific Statements and Guidelines should be directed to the AHA/ASA Manuscript Oversight Committee via its Correspondence page.Sign In to Submit a Response to This Article Previous Back to top Next FiguresReferencesRelatedDetailsRelated articlesGene Therapy With the N-Terminus of Junctophilin-2 Improves Heart Failure in MiceJinxi Wang, et al. Circulation Research. 2022;130:1306-1317 April 29, 2022Vol 130, Issue 9 Advertisement Article InformationMetrics \\u00a9 2022 American Heart Association, Inc.https://doi.org/10.1161/CIRCRESAHA.122.321066PMID: 35482830 Originally publishedApril 28, 2022 Keywordsheart failuresarcoplasmic reticulumphosphorylationgenetic therapyEditorialsphenotypePDF download Advertisement\", \"doi_y\": \"https://doi.org/10.1161/circresaha.122.321066\", \"publication_date\": \"2022-04-29\"}, {\"work_id\": \"W4385349885\", \"doi_x\": \"10.1021/acscatal.3c01906\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Comment on: \\u201cComputer Simulations Reveal an Entirely Entropic Activation Barrier for the Chemical Step in a Designer Enzyme\\u201d\", \"abstract\": \"Activation heat capacity has been proposed as an important factor in enzyme evolution and thermoadaptation. We previously demonstrated that the emergence of curved activity\\u2013temperature profiles during the evolution of a designer enzyme was due to the selective rigidification of its transition state ensemble that induced an activation heat capacity. \\u00c5qvist challenged our findings with molecular dynamics simulations suggesting that a change in the rate-limiting step underlies the experimental observations. As we describe here, \\u00c5qvist\\u2019s model is not consistent with the experimental trends observed for the chemical step of the catalyzed reaction (kcat). We suggest that this discrepancy arises because the simulations performed by \\u00c5qvist were limited by restraints and short simulation times, which do not allow sampling of the motions responsible for the observed activation heat capacity.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c01906\", \"publication_date\": \"2023-07-28\"}, {\"work_id\": \"W4319071338\", \"doi_x\": \"10.1136/jnnp-2022-330504\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>UNC13A</i>in amyotrophic lateral sclerosis: from genetic association to therapeutic target\", \"abstract\": \"Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease with limited treatment options and an incompletely understood pathophysiology. Although genomewide association studies (GWAS) have advanced our understanding of the disease, the precise manner in which risk polymorphisms contribute to disease pathogenesis remains unclear. Of relevance, GWAS have shown that a polymorphism (rs12608932) in the UNC13A gene is associated with risk for both ALS and frontotemporal dementia (FTD). Homozygosity for the C-allele at rs12608932 modifies the ALS phenotype, as these patients are more likely to have bulbar-onset disease, cognitive impairment and FTD at baseline as well as shorter survival. UNC13A is expressed in neuronal tissue and is involved in maintaining synaptic active zones, by enabling the priming and docking of synaptic vesicles. In the absence of functional TDP-43, risk variants in UNC13A lead to the inclusion of a cryptic exon in UNC13A messenger RNA, subsequently leading to nonsense mediated decay, with loss of functional protein. Depletion of UNC13A leads to impaired neurotransmission. Recent discoveries have identified UNC13A as a potential target for therapy development in ALS, with a confirmatory trial with lithium carbonate in UNC13A cases now underway and future approaches with antisense oligonucleotides currently under consideration. Considering UNC13A is a potent phenotypic modifier, it may also impact clinical trial outcomes. This present review describes the path from the initial discovery of UNC13A as a risk gene in ALS to the current therapeutic options being explored and how knowledge of its distinct phenotype needs to be taken into account in future trials.\", \"doi_y\": \"https://doi.org/10.1136/jnnp-2022-330504\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W4310142617\", \"doi_x\": \"10.1093/plcell/koac340\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The small peptide CEP1 and the NIN-like protein NLP1 regulate <i>NRT2.1</i> to mediate root nodule formation across nitrate concentrations\", \"abstract\": \"Legumes acquire fixed nitrogen (N) from the soil and through endosymbiotic association with diazotrophic bacteria. However, establishing and maintaining N2-fixing nodules are expensive for the host plant, relative to taking up N from the soil. Therefore, plants suppress symbiosis when N is plentiful and enhance symbiosis when N is sparse. Here, we show that the nitrate transporter MtNRT2.1 is required for optimal nodule establishment in Medicago truncatula under low-nitrate conditions and the repression of nodulation under high-nitrate conditions. The NIN-like protein (NLP) MtNLP1 is required for MtNRT2.1 expression and regulation of nitrate uptake/transport under low- and high-nitrate conditions. Under low nitrate, the gene encoding the C-terminally encoded peptide (CEP) MtCEP1 was more highly expressed, and the exogenous application of MtCEP1 systemically promoted MtNRT2.1 expression in a compact root architecture 2 (MtCRA2)-dependent manner. The enhancement of nodulation by MtCEP1 and nitrate uptake were both impaired in the Mtnrt2.1 mutant under low nitrate. Our study demonstrates that nitrate uptake by MtNRT2.1 differentially affects nodulation at low- and high-nitrate conditions through the actions of MtCEP1 and MtNLP1.\", \"doi_y\": \"https://doi.org/10.1093/plcell/koac340\", \"publication_date\": \"2022-11-28\"}, {\"work_id\": \"W4304889335\", \"doi_x\": \"10.1016/j.jbc.2022.102589\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Herpes simplex virus 1 protein pUL21 alters ceramide metabolism by activating the interorganelle transport protein CERT\", \"abstract\": \"Herpes simplex virus (HSV)-1 dramatically alters the architecture and protein composition of cellular membranes during infection, but its effects upon membrane lipid composition remain unclear. HSV-1 pUL21 is a virus-encoded protein phosphatase adaptor that promotes dephosphorylation of multiple cellular and virus proteins, including the cellular ceramide (Cer) transport protein CERT. CERT mediates nonvesicular Cer transport from the endoplasmic reticulum to the trans-Golgi network, whereupon Cer is converted to sphingomyelin (SM) and other sphingolipids that play important roles in cellular proliferation, signaling, and membrane trafficking. Here, we use click chemistry to profile the kinetics of sphingolipid metabolism, showing that pUL21-mediated dephosphorylation activates CERT and accelerates Cer-to-SM conversion. Purified pUL21 and full-length CERT interact with submicromolar affinity, and we solve the solution structure of the pUL21 C-terminal domain in complex with the CERT Pleckstrin homology and steroidogenic acute regulatory-related lipid transfer domains using small-angle X-ray scattering. We identify a single amino acid mutation on the surface of pUL21 that disrupts CERT binding in vitro and in cultured cells. This residue is highly conserved across the genus Simplexvirus. In addition, we identify a pUL21 residue essential for binding to HSV-1 pUL16. Sphingolipid profiling demonstrates that Cer-to-SM conversion is severely diminished in the context of HSV-1 infection, a defect that is compounded when infecting with a virus encoding the mutated form of pUL21 that lacks the ability to activate CERT. However, virus replication and spread in cultured keratinocytes or epithelial cells is not significantly altered when pUL21-mediated CERT dephosphorylation is abolished. Collectively, we demonstrate that HSV-1 modifies sphingolipid metabolism via specific protein-protein interactions.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102589\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4307390372\", \"doi_x\": \"10.1038/s41578-022-00488-z\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The role of single-protein elasticity in mechanobiology\", \"abstract\": \"In addition to biochemical signals and genetic considerations, mechanical forces are rapidly emerging as a master regulator of human physiology. Yet the molecular mechanisms that regulate force-induced functionalities across a wide range of scales, encompassing the cell, tissue or organ levels, are comparatively not so well understood. With the advent, development and refining of single molecule nanomechanical techniques, enabling to exquisitely probe the conformational dynamics of individual proteins under the effect of a calibrated force, we have begun to acquire a comprehensive knowledge on the rich plethora of physicochemical principles that regulate the elasticity of single proteins. Here we review the major advances underpinning our current understanding of how the elasticity of single proteins regulates mechanosensing and mechanotransduction. We discuss the present limitations and future challenges of such a prolific and burgeoning field.\", \"doi_y\": \"https://doi.org/10.1038/s41578-022-00488-z\", \"publication_date\": \"2022-10-24\"}, {\"work_id\": \"W4200347144\", \"doi_x\": \"10.1016/j.mcpro.2021.100186\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Subcellular Transcriptomics and Proteomics: A Comparative Methods Review\", \"abstract\": \"The internal environment of cells is molecularly crowded, which requires spatial organization via subcellular compartmentalization. These compartments harbor specific conditions for molecules to perform their biological functions, such as coordination of the cell cycle, cell survival, and growth. This compartmentalization is also not static, with molecules trafficking between these subcellular neighborhoods to carry out their functions. For example, some biomolecules are multifunctional, requiring an environment with differing conditions or interacting partners, and others traffic to export such molecules. Aberrant localization of proteins or RNA species has been linked to many pathological conditions, such as neurological, cancer, and pulmonary diseases. Differential expression studies in transcriptomics and proteomics are relatively common, but the majority have overlooked the importance of subcellular information. In addition, subcellular transcriptomics and proteomics data do not always colocate because of the biochemical processes that occur during and after translation, highlighting the complementary nature of these fields. In this review, we discuss and directly compare the current methods in spatial proteomics and transcriptomics, which include sequencing- and imaging-based strategies, to give the reader an overview of the current tools available. We also discuss current limitations of these strategies as well as future developments in the field of spatial -omics.\", \"doi_y\": \"https://doi.org/10.1016/j.mcpro.2021.100186\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4283205014\", \"doi_x\": \"10.3389/fpubh.2022.873633\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Molecular Epidemiology of AY.28 and AY.104 Delta Sub-lineages in Sri Lanka\", \"abstract\": \"Background The worst SARS-CoV-2 outbreak in Sri Lanka was due to the two Sri Lankan delta sub-lineages AY.28 and AY.104. We proceeded to further characterize the mutations and clinical disease severity of these two sub-lineages. Methods 705 delta SARS-CoV-2 genomes sequenced by our laboratory from mid-May to November 2021 using Illumina and Oxford Nanopore were included in the analysis. The clinical disease severity of 440/705 individuals were further analyzed to determine if infection with either AY.28 or AY.104 was associated with more severe disease. Sub-genomic RNA (sg-RNA) expression was analyzed using periscope. Results AY.28 was the dominant variant throughout the outbreak, accounting for 67.7% of infections during the peak of the outbreak. AY.28 had three lineage defining mutations in the spike protein: A222V (92.80%), A701S (88.06%), and A1078S (92.04%) and seven in the ORF1a: R24C, K634N, P1640L, A2994V, A3209V, V3718A, and T3750I. AY.104 was characterized by the high prevalence of T95I (90.81%) and T572L (65.01%) mutations in the spike protein and by the absence of P1640L (94.28%) in ORF1a with the presence of A1918V (98.58%) mutation. The mean sgRNA expression levels of ORF6 in AY.28 were significantly higher compared to AY.104 ( p &amp;lt; 0.0001) and B.1.617.2 ( p &amp;lt; 0.01). Also, ORF3a showed significantly higher sgRNA expression in AY.28 compared to AY.104 ( p &amp;lt; 0.0001). There was no difference in the clinical disease severity or duration of hospitalization in individuals infected with these sub lineages. Conclusions Therefore, AY.28 and AY.104 appear to have a fitness advantage over the parental delta variant (B.1.617.2), while AY.28 also had a higher expression of sg-RNA compared to other sub-lineages. The clinical implications of these should be further investigated.\", \"doi_y\": \"https://doi.org/10.3389/fpubh.2022.873633\", \"publication_date\": \"2022-06-21\"}, {\"work_id\": \"W4293832947\", \"doi_x\": \"10.1016/j.str.2022.08.004\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Assessing PDB macromolecular crystal structure confidence at the individual amino acid residue level\", \"abstract\": \"Approximately 87% of the more than 190,000 atomic-level three-dimensional (3D) biostructures in the PDB were determined using macromolecular crystallography (MX). Agreement between 3D atomic coordinates and experimental data for >100 million individual amino acid residues occurring within \\u223c150,000 PDB MX structures was analyzed in detail. The real-space correlation coefficient (RSCC) calculated using the 3D atomic coordinates for each residue and experimental-data-derived electron density enables outlier detection of unreliable atomic coordinates (particularly important for poorly resolved side-chain atoms) and ready evaluation of local structure quality by PDB users. For human protein MX structures in PDB, comparisons of the per-residue RSCC metric with AlphaFold2-computed structure model confidence (pLDDT-predicted local distance difference test) document (1) that RSCC values and pLDDT scores are correlated (median correlation coefficient \\u223c0.41), and (2) that experimentally determined MX structures (3.5 \\u00c5 resolution or better) are more reliable than AlphaFold2-computed structure models and should be used preferentially whenever possible.\", \"doi_y\": \"https://doi.org/10.1016/j.str.2022.08.004\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4225009875\", \"doi_x\": \"10.1016/j.jmb.2022.167608\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"GWYRE: A Resource for Mapping Variants onto Experimental and Modeled Structures of Human Protein Complexes\", \"abstract\": \"Rapid progress in structural modeling of proteins and their interactions is powered by advances in knowledge-based methodologies along with better understanding of physical principles of protein structure and function. The pool of structural data for modeling of proteins and protein-protein complexes is constantly increasing due to the rapid growth of protein interaction databases and Protein Data Bank. The GWYRE (Genome Wide PhYRE) project capitalizes on these developments by advancing and applying new powerful modeling methodologies to structural modeling of protein-protein interactions and genetic variation. The methods integrate knowledge-based tertiary structure prediction using Phyre2 and quaternary structure prediction using template-based docking by a full-structure alignment protocol to generate models for binary complexes. The predictions are incorporated in a comprehensive public resource for structural characterization of the human interactome and the location of human genetic variants. The GWYRE resource facilitates better understanding of principles of protein interaction and structure/function relationships. The resource is available at http://www.gwyre.org.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2022.167608\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4367399594\", \"doi_x\": \"10.1016/j.sbi.2023.102600\", \"af_direct_citation\": 1, \"af_indirect_citations\": 13, \"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"abstract\": \"We provide an overview of the methods that can be used for protein structure-based evaluation of missense variants. The algorithms can be broadly divided into those that calculate the difference in free energy (\\u0394\\u0394G) between the wild type and variant structures and those that use structural features to predict the damaging effect of a variant without providing a \\u0394\\u0394G. A wide range of machine learning approaches have been employed to develop those algorithms. We also discuss challenges and opportunities for variant interpretation in view of the recent breakthrough in three-dimensional structural modelling using deep learning.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102600\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4360843077\", \"doi_x\": \"10.1016/j.jmb.2023.168060\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"abstract\": \"In 2019, we released Missense3D which identifies stereochemical features that are disrupted by a missense variant, such as introducing a buried charge. Missense3D analyses the effect of a missense variant on a single structure and thus may fail to identify as damaging surface variants disrupting a protein interface i.e., a protein-protein interaction (PPI) site. Here we present Missense3D-PPI designed to predict missense variants at PPI interfaces. Our development dataset comprised of 1,279 missense variants (pathogenic n = 733, benign n = 546) in 434 proteins and 545 experimental structures of PPI complexes. Benchmarking of Missense3D-PPI was performed after dividing the dataset in training (320 benign and 320 pathogenic variants) and testing (226 benign and 413 pathogenic). Structural features affecting PPI, such as disruption of interchain bonds and introduction of unbalanced charged interface residues, were analysed to assess the impact of the variant at PPI. The performance of Missense3D-PPI was superior to that of Missense3D: sensitivity 44 % versus 8% and accuracy 58% versus 40%, p = 4.23 \\u00d7 10-16. However, the specificity of Missense3D-PPI was lower compared to Missense3D (84% versus 98%). On our dataset, Missense3D-PPI's accuracy was superior to BeAtMuSiC (p = 3.4 \\u00d7 10-5), mCSM-PPI2 (p = 1.5 \\u00d7 10-12) and MutaBind2 (p = 0.0025). Missense3D-PPI represents a valuable tool for predicting the structural effect of missense variants on biological protein networks and is available at the Missense3D web portal (http://missense3d.bc.ic.ac.uk).\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168060\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4310055674\", \"doi_x\": \"10.1107/s2059798322010415\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"abstract\": \"Determination of protein structures typically entails building a model that satisfies the collected experimental observations and its deposition in the Protein Data Bank. Experimental limitations can lead to unavoidable uncertainties during the process of model building, which result in the introduction of errors into the deposited model. Many metrics are available for model validation, but most are limited to consideration of the physico-chemical aspects of the model or its match to the experimental data. The latest advances in the field of deep learning have enabled the increasingly accurate prediction of inter-residue distances, an advance which has played a pivotal role in the recent improvements observed in the field of protein ab initio modelling. Here, new validation methods are presented based on the use of these precise inter-residue distance predictions, which are compared with the distances observed in the protein model. Sequence-register errors are particularly clearly detected and the register shifts required for their correction can be reliably determined. The method is available in the ConKit package (https://www.conkit.org).\", \"doi_y\": \"https://doi.org/10.1107/s2059798322010415\", \"publication_date\": \"2022-11-25\"}, {\"work_id\": \"W4319333664\", \"doi_x\": \"10.1038/s41467-023-36372-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Plasma FIB milling for the determination of structures in situ\", \"abstract\": \"Abstract Structural biology studies inside cells and tissues require methods to thin vitrified specimens to electron transparency. Until now, focused ion beams based on gallium have been used. However, ion implantation, changes to surface chemistry and an inability to access high currents limit gallium application. Here, we show that plasma-coupled ion sources can produce cryogenic lamellae of vitrified human cells in a robust and automated manner, with quality sufficient for pseudo-atomic structure determination. Lamellae were produced in a prototype microscope equipped for long cryogenic run times (&gt; 1 week) and with multi-specimen support fully compatible with modern-day transmission electron microscopes. We demonstrate that plasma ion sources can be used for structural biology within cells, determining a structure in situ to 4.9 \\u00c5, and characterise the resolution dependence on particle distance from the lamella edge. We describe a workflow upon which different plasmas can be examined to further streamline lamella fabrication.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-36372-9\", \"publication_date\": \"2023-02-06\"}, {\"work_id\": \"W4206131125\", \"doi_x\": \"10.1098/rsob.210264\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mapping of a N-terminal \\u03b1-helix domain required for human PINK1 stabilization, Serine228 autophosphorylation and activation in cells\", \"abstract\": \"Autosomal recessive mutations in the PINK1 gene are causal for Parkinson's disease (PD). PINK1 encodes a mitochondrial localized protein kinase that is a master-regulator of mitochondrial quality control pathways. Structural studies to date have elaborated the mechanism of how mutations located within the kinase domain disrupt PINK1 function; however, the molecular mechanism of PINK1 mutations located upstream and downstream of the kinase domain is unknown. We have employed mutagenesis studies to define the minimal region of human PINK1 required for optimal ubiquitin phosphorylation, beginning at residue Ile111. Inspection of the AlphaFold human PINK1 structure model predicts a conserved N-terminal \\u03b1-helical extension (NTE) domain forming an intramolecular interaction with the C-terminal extension (CTE), which we corroborate using hydrogen/deuterium exchange mass spectrometry of recombinant insect PINK1 protein. Cell-based analysis of human PINK1 reveals that PD-associated mutations (e.g. Q126P), located within the NTE : CTE interface, markedly inhibit stabilization of PINK1; autophosphorylation at Serine228 (Ser228) and Ubiquitin Serine65 (Ser65) phosphorylation. Furthermore, we provide evidence that NTE and CTE domain mutants disrupt PINK1 stabilization at the mitochondrial Translocase of outer membrane complex. The clinical relevance of our findings is supported by the demonstration of defective stabilization and activation of endogenous PINK1 in human fibroblasts of a patient with early-onset PD due to homozygous PINK1 Q126P mutations. Overall, we define a functional role of the NTE : CTE interface towards PINK1 stabilization and activation and show that loss of NTE : CTE interactions is a major mechanism of PINK1-associated mutations linked to PD.\", \"doi_y\": \"https://doi.org/10.1098/rsob.210264\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4317436860\", \"doi_x\": \"10.1038/s41467-023-35949-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural basis for a conserved neutralization epitope on the receptor-binding domain of SARS-CoV-2\", \"abstract\": \"Antibody-mediated immunity plays a crucial role in protection against SARS-CoV-2 infection. We isolated a panel of neutralizing anti-receptor-binding domain (RBD) antibodies elicited upon natural infection and vaccination and showed that they recognize an immunogenic patch on the internal surface of the core RBD, which faces inwards and is hidden in the \\\"down\\\" state. These antibodies broadly neutralize wild type (Wuhan-Hu-1) SARS-CoV-2, Beta and Delta variants and some are effective against other sarbecoviruses. We observed a continuum of partially overlapping antibody epitopes from lower to upper part of the inner face of the RBD and some antibodies extend towards the receptor-binding motif. The majority of antibodies are substantially compromised by three mutational hotspots (S371L/F, S373P and S375F) in the lower part of the Omicron BA.1, BA.2 and BA.4/5 RBD. By contrast, antibody IY-2A induces a partial unfolding of this variable region and interacts with a conserved conformational epitope to tolerate all antigenic variations and neutralize diverse sarbecoviruses as well. This finding establishes that antibody recognition is not limited to the normal surface structures on the RBD. In conclusion, the delineation of functionally and structurally conserved RBD epitopes highlights potential vaccine and therapeutic candidates for COVID-19.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-35949-8\", \"publication_date\": \"2023-01-19\"}, {\"work_id\": \"W4383228156\", \"doi_x\": \"10.1038/s41431-023-01422-9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Germline pathogenic variants in HNRNPU are associated with alterations in blood methylome\", \"abstract\": \"HNRNPU encodes a multifunctional RNA-binding protein that plays critical roles in regulating pre-mRNA splicing, mRNA stability, and translation. Aberrant expression and dysregulation of HNRNPU have been implicated in various human diseases, including cancers and neurological disorders. We applied a next generation sequencing based assay (EPIC-NGS) to investigate genome-wide methylation profiling for >2 M CpGs for 7 individuals with a neurodevelopmental disorder associated with HNRNPU germline pathogenic loss-of-function variants. Compared to healthy individuals, 227 HNRNPU-associated differentially methylated positions were detected. Both hyper- and hypomethylation alterations were identified but the former predominated. The identification of a methylation episignature for HNRNPU-associated neurodevelopmental disorder (NDD) implicates HNPRNPU-related chromatin alterations in the aetiopathogenesis of this disorder and suggests that episignature profiling should have clinical utility as a predictor for the pathogenicity of HNRNPU variants of uncertain significance. The detection of a methylation episignaure for HNRNPU-associated NDD is consistent with a recent report of a methylation episignature for HNRNPK-associated NDD.\", \"doi_y\": \"https://doi.org/10.1038/s41431-023-01422-9\", \"publication_date\": \"2023-07-05\"}, {\"work_id\": \"W4308682319\", \"doi_x\": \"10.1093/nar/gkac1019\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"COMBATdb: a database for the COVID-19 Multi-Omics Blood ATlas\", \"abstract\": \"Advances in our understanding of the nature of the immune response to SARS-CoV-2 infection, and how this varies within and between individuals, is important in efforts to develop targeted therapies and precision medicine approaches. Here we present a database for the COvid-19 Multi-omics Blood ATlas (COMBAT) project, COMBATdb (https://db.combat.ox.ac.uk). This enables exploration of multi-modal datasets arising from profiling of patients with different severities of illness admitted to hospital in the first phase of the pandemic in the UK prior to vaccination, compared with community cases, healthy controls, and patients with all-cause sepsis and influenza. These data include whole blood transcriptomics, plasma proteomics, epigenomics, single-cell multi-omics, immune repertoire sequencing, flow and mass cytometry, and cohort metadata. COMBATdb provides access to the processed data in a well-defined framework of samples, cell types and genes/proteins that allows exploration across the assayed modalities, with functionality including browse, search, download, calculation and visualisation via shiny apps. This advances the ability of users to leverage COMBAT datasets to understand the pathogenesis of COVID-19, and the nature of specific and shared features with other infectious diseases.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1019\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W3206058908\", \"doi_x\": \"10.1101/2021.10.17.464712\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Systematic analysis of membrane contact sites in <i>Saccharomyces cerevisiae</i> uncovers modulators of cellular lipid distribution\", \"abstract\": \"Abstract Actively maintained close appositions, or contact sites, between organelle membranes, enable the efficient transfer of biomolecules between the various cellular compartments. Several such sites have been described together with their tethering machinery. Despite these advances we are still far from a comprehensive understanding of the function and regulation of most contact sites. To systematically characterize the proteome of contact sites and support the discovery of new tethers and functional molecules, we established a high throughput screening approach in Saccharomyces cerevisiae based on co-localization imaging. We imaged split fluorescence reporters for six different contact sites, two of which have never been studied before, on the background of 1165 strains expressing a mCherry-tagged yeast protein that have a cellular punctate distribution (a hallmark of contact sites). By scoring both co-localization events and effects on reporter size and abundance, we discovered over 100 new potential contact site residents and effectors in yeast. Focusing on several of the newly identified residents, we identified one set of hits as previously unrecognized homologs to Vps13 and Atg2. These proteins share their lipid transport domain, thus expanding this family of lipid transporters. Analysis of another candidate, Ypr097w, which we now call Lec1 ( L ipid-droplet E rgosterol C ortex 1), revealed that this previously uncharacterized protein dynamically shifts between lipid droplets and the cell cortex, and plays a role in regulation of ergosterol distribution in the cell.\", \"doi_y\": \"https://doi.org/10.1101/2021.10.17.464712\", \"publication_date\": \"2021-10-19\"}, {\"work_id\": \"W4382138469\", \"doi_x\": \"10.1073/pnas.2219036120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Design of the elusive proteinaceous oxygen donor copper site suggests a promising future for copper for MRI contrast agents\", \"abstract\": \"We report the preparation and spectroscopic characterization of a highly elusive copper site bound exclusively to oxygen donor atoms within a protein scaffold. Despite copper generally being considered unsuitable for use in MRI contrast agents, which in the clinic are largely Gd(III) based, the designed copper coiled coil displays relaxivity values equal to, or superior than, those of the Gd(III) analog at clinical field strengths. The creation of this new-to-biology proteinaceous CuO x -binding site demonstrates the power of the de novo peptide design approach to access chemistry for abiological applications, such as for the development of MRI contrast agents.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2219036120\", \"publication_date\": \"2023-06-26\"}, {\"work_id\": \"W4318262988\", \"doi_x\": \"10.1021/acsbiomaterials.2c01116\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Silk Bioconjugates: From Chemistry and Concept to Application\", \"abstract\": \"Medical silks have captured global interest. While silk sutures have a long track record in humans, silk bioconjugates are still in preclinical development. This perspective examines key advances in silk bioconjugation, including the fabrication of silk\\u2013protein conjugates, bioconjugated silk particles, and bioconjugated substrates to enhance cell\\u2013material interactions in two and three dimensions. Many of these systems rely on chemical modification of the silk biopolymer, often using carbodiimide and reactive ester chemistries. However, recent progress in enzyme-mediated and click chemistries has expanded the molecular toolbox to enable biorthogonal, site-specific conjugation in a single step when combined with recombinant silk fibroin tagged with noncanonical amino acids. This perspective outlines key strategies available for chemical modification, compares the resulting silk conjugates to clinical benchmarks, and outlines open questions and areas that require more work. Overall, this assessment highlights a domain of new sunrise capabilities and development opportunities for silk bioconjugates that may ultimately offer new ways of delivering improved healthcare.\", \"doi_y\": \"https://doi.org/10.1021/acsbiomaterials.2c01116\", \"publication_date\": \"2023-01-27\"}, {\"work_id\": \"W4220811381\", \"doi_x\": \"10.1101/2022.04.01.486724\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Impact of 100 LRRK2 variants linked to Parkinson\\u2019s Disease on kinase activity and microtubule binding\", \"abstract\": \"Abstract Mutations enhancing the kinase activity of LRRK2 cause Parkinson\\u2019s disease (PD) and therapies that reduce LRRK2 kinase activity are being tested in clinical trials. Numerous rare variants of unknown clinical significance have been reported, but how the vast majority impact on LRRK2 function is unknown. Here, we investigate 100 LRRK2 variants linked to PD, including previously described pathogenic mutations. We identify 23 LRRK2 variants that robustly stimulate kinase activity, including variants within the N-terminal non-catalytic regions [ARM (E334K, A419V), ANK(R767H), LRR (R1067Q, R1325Q)], as well as variants predicted to destabilise the ROC:COR B interface [ROC (A1442P, V1447M), COR A (R1628P) COR B (S1761R, L1795F)] and COR:COR dimer interface [COR B (R1728H/L)]. Most activating variants decrease LRRK2 biomarker site phosphorylation (pSer935/pSer955/pSer973), consistent with the notion that the active kinase conformation blocks their phosphorylation. We conclude that the impact of variants on kinase activity is best evaluated by deploying a cellular assay of LRRK2-dependent Rab10 substrate phosphorylation, compared to a biochemical kinase assay, as only a minority of activating variants [COR B (Y1699C, R1728H/L, S1761R) and kinase (G2019S, I2020T, T2031S)], enhance in vitro kinase activity of immunoprecipitated LRRK2. Twelve variants including several that activate LRRK2 and have been linked to PD, suppressed microtubule association in the presence of a Type I kinase inhibitor [ARM(M712V), LRR(R1320S), ROC (A1442P, K1468E, S1508R), COR A (A1589S), COR B (Y1699C, R1728H/L) and WD40(R2143M, S2350I, G2385R)]. Our findings will stimulate work to better understand the mechanisms by which variants impact biology and provide rationale for variant carrier inclusion or exclusion in ongoing and future LRRK2 inhibitor clinical trials.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.01.486724\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W4287218243\", \"doi_x\": \"10.3389/fgene.2022.920390\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Incomplete Penetrance and Variable Expressivity: From Clinical Studies to Population Cohorts\", \"abstract\": \"The same genetic variant found in different individuals can cause a range of diverse phenotypes, from no discernible clinical phenotype to severe disease, even among related individuals. Such variants can be said to display incomplete penetrance, a binary phenomenon where the genotype either causes the expected clinical phenotype or it does not, or they can be said to display variable expressivity, in which the same genotype can cause a wide range of clinical symptoms across a spectrum. Both incomplete penetrance and variable expressivity are thought to be caused by a range of factors, including common variants, variants in regulatory regions, epigenetics, environmental factors, and lifestyle. Many thousands of genetic variants have been identified as the cause of monogenic disorders, mostly determined through small clinical studies, and thus, the penetrance and expressivity of these variants may be overestimated when compared to their effect on the general population. With the wealth of population cohort data currently available, the penetrance and expressivity of such genetic variants can be investigated across a much wider contingent, potentially helping to reclassify variants that were previously thought to be completely penetrant. Research into the penetrance and expressivity of such genetic variants is important for clinical classification, both for determining causative mechanisms of disease in the affected population and for providing accurate risk information through genetic counseling. A genotype-based definition of the causes of rare diseases incorporating information from population cohorts and clinical studies is critical for our understanding of incomplete penetrance and variable expressivity. This review examines our current knowledge of the penetrance and expressivity of genetic variants in rare disease and across populations, as well as looking into the potential causes of the variation seen, including genetic modifiers, mosaicism, and polygenic factors, among others. We also considered the challenges that come with investigating penetrance and expressivity.\", \"doi_y\": \"https://doi.org/10.3389/fgene.2022.920390\", \"publication_date\": \"2022-07-25\"}, {\"work_id\": \"W4385802069\", \"doi_x\": \"10.1371/journal.ppat.1011562\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The phenuivirus Toscana virus makes an atypical use of vacuolar acidity to enter host cells\", \"abstract\": \"Toscana virus is a major cause of arboviral disease in humans in the Mediterranean basin during summer. However, early virus-host cell interactions and entry mechanisms remain poorly characterized. Investigating iPSC-derived human neurons and cell lines, we found that virus binding to the cell surface was specific, and 50% of bound virions were endocytosed within 10 min. Virions entered Rab5a+ early endosomes and, subsequently, Rab7a+ and LAMP-1+ late endosomal compartments. Penetration required intact late endosomes and occurred within 30 min following internalization. Virus entry relied on vacuolar acidification, with an optimal pH for viral membrane fusion at pH 5.5. The pH threshold increased to 5.8 with longer pre-exposure of virions to the slightly acidic pH in early endosomes. Strikingly, the particles remained infectious after entering late endosomes with a pH below the fusion threshold. Overall, our study establishes Toscana virus as a late-penetrating virus and reveals an atypical use of vacuolar acidity by this virus to enter host cells.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011562\", \"publication_date\": \"2023-08-14\"}, {\"work_id\": \"W4386769118\", \"doi_x\": \"10.1103/physrevb.108.115134\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlation energy of the spin-polarized electron liquid studied using quantum Monte Carlo simulations\", \"abstract\": \"Variational and diffusion quantum Monte Carlo (VMC and DMC) methods with Slater-Jastrow-backflow trial wave functions are used to study the spin-polarized three-dimensional uniform electron fluid. We report ground state VMC and DMC energies in the density range $0.5\\\\ensuremath{\\\\le}{r}_{\\\\text{s}}\\\\ensuremath{\\\\le}20$. Finite-size errors are corrected using canonical-ensemble twist-averaged boundary conditions and extrapolation of the twist-averaged energy per particle calculated at three system sizes $(N=113, 259, \\\\mathrm{and} 387)$ to the thermodynamic limit of infinite system size. The DMC energies in the thermodynamic limit are used to parametrize a local spin density approximation correlation function for inhomogeneous electron systems.\", \"doi_y\": \"https://doi.org/10.1103/physrevb.108.115134\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4285005484\", \"doi_x\": \"10.1016/j.jbc.2022.102249\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Spectroscopic studies reveal details of substrate-induced conformational changes distant from the active site in isopenicillin N synthase\", \"abstract\": \"Isopenicillin N synthase (IPNS) catalyzes formation of the \\u03b2-lactam and thiazolidine rings of isopenicillin N from its linear tripeptide l-\\u03b4-(\\u03b1-aminoadipoyl)-l-cysteinyl-d-valine (ACV) substrate in an iron- and dioxygen (O2)-dependent four-electron oxidation without precedent in current synthetic chemistry. Recent X-ray free-electron laser studies including time-resolved serial femtosecond crystallography show that binding of O2 to the IPNS-Fe(II)-ACV complex induces unexpected conformational changes in \\u03b1-helices on the surface of IPNS, in particular in \\u03b13 and \\u03b110. However, how substrate binding leads to conformational changes away from the active site is unknown. Here, using detailed 19F NMR and electron paramagnetic resonance experiments with labeled IPNS variants, we investigated motions in \\u03b13 and \\u03b110 induced by binding of ferrous iron, ACV, and the O2 analog nitric oxide, using the less mobile \\u03b16 for comparison. 19F NMR studies were carried out on singly and doubly labeled \\u03b13, \\u03b16, and \\u03b110 variants at different temperatures. In addition, double electron-electron resonance electron paramagnetic resonance analysis was carried out on doubly spin-labeled variants. The combined spectroscopic and crystallographic results reveal that substantial conformational changes in regions of IPNS including \\u03b13 and \\u03b110 are induced by binding of ACV and nitric oxide. Since IPNS is a member of the structural superfamily of 2-oxoglutarate-dependent oxygenases and related enzymes, related conformational changes may be of general importance in nonheme oxygenase catalysis.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102249\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4361190388\", \"doi_x\": \"10.1107/s205979832300181x\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>Buccaneer</i> model building with neural network fragment selection\", \"abstract\": \"Tracing the backbone is a critical step in protein model building, as incorrect tracing leads to poor protein models. Here, a neural network trained to identify unfavourable fragments and remove them from the model-building process in order to improve backbone tracing is presented. Moreover, a decision tree was trained to select an optimal threshold to eliminate unfavourable fragments. The neural network was tested on experimental phasing data sets from the Joint Center for Structural Genomics (JCSG), recently deposited experimental phasing data sets (from 2015 to 2021) and molecular-replacement data sets. The experimental results show that using the neural network in the Buccaneer protein-model-building software can produce significantly more complete protein models than those built using Buccaneer alone. In particular, Buccaneer with the neural network built protein models with a completeness that was at least 5% higher for 25% and 50% of the original and truncated resolution JCSG experimental phasing data sets, respectively, for 28% of the recently collected experimental phasing data sets and for 43% of the molecular-replacement data sets.\", \"doi_y\": \"https://doi.org/10.1107/s205979832300181x\", \"publication_date\": \"2023-03-28\"}, {\"work_id\": \"W4319598121\", \"doi_x\": \"10.3390/antiox12020401\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Methionine Sulfoxide Reductases Suppress the Formation of the [PSI+] Prion and Protein Aggregation in Yeast\", \"abstract\": \"Prions are self-propagating, misfolded forms of proteins associated with various neurodegenerative diseases in mammals and heritable traits in yeast. How prions form spontaneously into infectious amyloid-like structures without underlying genetic changes is poorly understood. Previous studies have suggested that methionine oxidation may underlie the switch from a soluble protein to the prion form. In this current study, we have examined the role of methionine sulfoxide reductases (MXRs) in protecting against de novo formation of the yeast [PSI+] prion, which is the amyloid form of the Sup35 translation termination factor. We show that [PSI+] formation is increased during normal and oxidative stress conditions in mutants lacking either one of the yeast MXRs (Mxr1, Mxr2), which protect against methionine oxidation by reducing the two epimers of methionine-S-sulfoxide. We have identified a methionine residue (Met124) in Sup35 that is important for prion formation, confirming that direct Sup35 oxidation causes [PSI+] prion formation. [PSI+] formation was less pronounced in mutants simultaneously lacking both MXR isoenzymes, and we show that the morphology and biophysical properties of protein aggregates are altered in this mutant. Taken together, our data indicate that methionine oxidation triggers spontaneous [PSI+] prion formation, which can be alleviated by methionine sulfoxide reductases.\", \"doi_y\": \"https://doi.org/10.3390/antiox12020401\", \"publication_date\": \"2023-02-07\"}, {\"work_id\": \"W4322628233\", \"doi_x\": \"10.1186/s40793-023-00465-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"New microbiological insights from the Bowland shale highlight heterogeneity of the hydraulically fractured shale microbiome\", \"abstract\": \"Abstract Background Hydraulically fractured shales offer a window into the deep biosphere, where hydraulic fracturing creates new microbial ecosystems kilometers beneath the surface of the Earth. Studying the microbial communities from flowback fluids that are assumed to inhabit these environments provides insights into their ecophysiology, and in particular their ability to survive in these extreme environments as well as their influence on site operation e.g. via problematic biofouling processes and/or biocorrosion. Over the past decade, research on fractured shale microbiology has focused on wells in North America, with a few additional reported studies conducted in China. To extend the knowledge in this area, we characterized the geochemistry and microbial ecology of two exploratory shale gas wells in the Bowland Shale, UK. We then employed a meta-analysis approach to compare geochemical and 16S rRNA gene sequencing data from our study site with previously published research from geographically distinct formations spanning China, Canada and the USA. Results Our findings revealed that fluids recovered from exploratory wells in the Bowland are characterized by moderate salinity and high microbial diversity. The microbial community was dominated by lineages known to degrade hydrocarbons, including members of Shewanellaceae , Marinobacteraceae, Halomonadaceae and Pseudomonadaceae . Moreover, UK fractured shale communities lacked the usually dominant Halanaerobium lineages. From our meta-analysis, we infer that chloride concentrations play a dominant role in controlling microbial community composition. Spatio-temporal trends were also apparent, with different shale formations giving rise to communities of distinct diversity and composition. Conclusions These findings highlight an unexpected level of compositional heterogeneity across fractured shale formations, which is not only relevant to inform management practices but also provides insight into the ability of diverse microbial consortia to tolerate the extreme conditions characteristic of the engineered deep subsurface.\", \"doi_y\": \"https://doi.org/10.1186/s40793-023-00465-1\", \"publication_date\": \"2023-02-28\"}, {\"work_id\": \"W4289861446\", \"doi_x\": \"10.15252/embj.2022112181\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<scp>ARFs</scp> get the <scp>BioID</scp> treatment: what have we been missing?\", \"abstract\": \"Li et al present the results of a proximity\\u2010interaction screen in mammalian cells for the effector proteins of 25 members of the Arf family of small GTPases. This study has generated an important resource for those working in several areas of cell biology and provided an initial characterisation of two new cellular roles for some of the least well studied members of this family, the regulation of PLD1 by ARL11/14 in phagocytosis, and the regulation of PI4KB by ARL5A/5B in the Golgi.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022112181\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4361006845\", \"doi_x\": \"10.1002/mds.29385\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The New p.<scp>F1700L LRRK2</scp> Variant Causes Parkinson's Disease by Extensively Increasing Kinase Activity\", \"abstract\": \"Variants in LRRK2 represent the most frequent cause of clinically classical monogenic Parkinson's disease (PD).1 Altered LRRK2 protein function boosts neuroinflammation, impairs vesicle trafficking, and affects ciliogenesis within the striatum.2 Because a relevant fraction of the more than 1000 identified LRRK2 variants3 is not pathogenic, determining pathogenicity for single variants is crucial, particularly because LRRK2 kinase inhibitors have entered phase 3 trials.4 Notably, we have already established an analytic workflow to determine kinase activity and decipher the pathogenicity of single LRRK2 variants in vitro5 and in vivo.6 In this letter, we report on a 74-year-old male patient from northern Germany with advanced typical PD (Unified Parkinson's Disease Rating Scale Part III: 33/108 points, Hoehn and Yahr stage 3\\u20134) without relevant tremor. The age at onset was 67 years. The disease course was slowly progressive, he experienced a good response to dopaminergic therapy, and dementia was absent. Family history was suggestive of autosomal dominant inheritance, with the father and two brothers also diagnosed with PD. However, the father was already deceased, one brother was not available for examination, and the second brother died early after giving blood for genetic investigation. Both brothers carried a new p.F1700L (NM_198578.4: c.5098T>C) variant in LRRK2, initially detected by gene panel analysis in the deceased brother and further investigated by Sanger sequencing. The variant is rated as a variant of uncertain significance according to the criteria of the American College of Medical Genetics (assessed by Franklin: https://franklin.genoox.com/) and is not listed in gnomAD (https://gnomad.broadinstitute.org/). In silico prediction suggested pathogenicity based on a Combined Annotation Dependent Depletion (CADD) score (https://cadd.gs.washington.edu/) of 27.3. The variant is located within the C terminus of the Ras of complex protein B scaffolding domain.5 We investigated LRRK2 kinase pathway activity in a heterologous transient overexpression system in HEK293 cells as described previously5 (Fig. 1A). We then analyzed LRRK2 activity in vivo in fresh peripheral blood, simultaneously collected from a p.F1700L carrier and a sex-matched healthy control subject (Fig. 1B). The resulting LRRK2-dependent pRab10Thr73 phosphorylation level mirrors LRRK2 kinase activation status.6 Both experiments demonstrated significant LRRK2 kinase hyperactivation because of the p.F1700L variant. Notably, p.F1700L demonstrated LRRK2 kinase hyperactivation similar to the neighboring p.Y1699C variant, which has the highest degree of kinase activity among all LRRK2 variants investigated thus far in the HEK293 assay (Fig. 1A). Moreover, we found a 7- to 8-fold increase in Rab10 phosphorylation levels in vivo (Fig. 1B). Together, we provide robust evidence for the pathogenicity of the newly identified p.F1700L variant in LRRK2, demonstrating that this substitution is among the variants with the highest kinase activity of all LRRK2 variants investigated to date and impacts protein function more profoundly than, for example, the frequent p.G2019S variant.5, 6 Thus, carriers of a p.F1700L variant should be included in clinical trials targeting LRRK2 kinase activity already initiated or soon commencing.4 Finally, we confirmed the applicability and usefulness of the applied assays to determine the pathogenicity of LRRK2 variants of uncertain significance. Further studies should focus on an association between the degree of kinase activity and penetrance, disease onset, and disease severity. We thank the participants for agreeing to undergo the examination for research purposes. Moreover, we thank the technical support of the MRC Protein Phosphorylation and Ubiquitylation Unit Reagents and Services, as well as the tissue culture team. This study was approved by the ethics committee of the University of L\\u00fcbeck and performed according to the Declaration of Helsinki. (1) Research project: A. Conception, B. Organization, C. Execution; (2) Statistical Analysis: A. Design, B. Execution, C. Review and Critique; (3) Manuscript: A. Writing of the first draft, B. Review and Critique. M.B.: 1A, 1B, 1C, 2C, 3A, 3B. N.P.: 1B, 1C, 2A, 2B, 3B. S.S.: 1B, 1C, 3B. F.H.: 1B, 1C, 3B. G.M.: 1B, 1C, 3B. J.U.: 1B, 1C, 3B. K.L.: 1A, 1B, 1C, 3B. C.K.: 1A, 1B, 1C, 3B. D.R.A.: 1A, 1B, 1C, 3B. J.H.: 1A, 1B, 1C, 3B. E.S.: 1A, 1B, 1C, 2A, 2B, 3A, 3B. Max Borsche: Employment, University of L\\u00fcbeck and University Hospital Schleswig-Holstein, Campus L\\u00fcbeck; Honoraria, Ipsen Pharma; Grants, German Research Foundation (DFG)-funded Clinician Scientist School, University of L\\u00fcbeck. Neringa Pratuseviciute: Employment, Carnegie Trust\\u2013funded PhD student, University of Dundee. Susen Schaake: Employment, University of L\\u00fcbeck and University Hospital Schleswig-Holstein, Campus L\\u00fcbeck. Frauke Hinrichs: Employment, University of L\\u00fcbeck and University Hospital Schleswig-Holstein, Campus L\\u00fcbeck. Gabriel Morel: Employment, Undergraduate placement student at University of Dundee. Jan Uter: Employment, University of L\\u00fcbeck and University Hospital Schleswig-Holstein, Campus L\\u00fcbeck. Katja Lohmann: Employment, University of L\\u00fcbeck; Grants, German Research Foundation (DFG), Parkinson's foundation, Damp Foundation, The Michael J. Fox Foundation (MJFF; GP2 project). Christine Klein: Consultancies, Centogene, Retromer Therapeutics; Employment, University of L\\u00fcbeck and University Hospital Schleswig-Holstein, Campus L\\u00fcbeck; Honoraria, Desitin and Bial; Royalties, Oxford University Press; Grants, DFG, MJFF, Aligning Science Across Parkinson's. Dario R. Alessi: Employment, University of Dundee; Grants, Medical Research Council UK. Johann Hagenah: Employment, Westk\\u00fcstenklinikum Heide. Esther Sammler: Employment, University of Dundee, UK; Honoraria, Bial, MJFF; Grants, MJFF, Chief Scientist Office, Scotland, UK (personal fellowship). The data presented in this study can be received from the corresponding author upon reasonable request.\", \"doi_y\": \"https://doi.org/10.1002/mds.29385\", \"publication_date\": \"2023-03-27\"}, {\"work_id\": \"W4210399978\", \"doi_x\": \"10.1016/j.matbio.2022.01.005\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"abstract\": \"TGF\\u03b2 superfamily members are potent growth factors in the extracellular matrix with essential roles in all aspects of cellular behaviour. Latent TGF\\u03b2 binding proteins (LTBPs) are co-expressed with TGF\\u03b2, essential for correct folding and secretion of the growth factor, to form large latent complexes. These large latent complexes bind extracellular proteins such as fibrillin for sequestration of TGF\\u03b2 in the matrix, essential for normal tissue function, and dysregulated TGF\\u03b2 signalling is a hallmark of many fibrillinopathies. Transglutaminase-2 (TG2) cross-linking of LTBPs is known to play a role in TGF\\u03b2 activation but the underlying molecular mechanisms are not resolved. Here we show that fibrillin is a matrix substrate for TG2 and that TG2 cross-linked complexes can be formed between fibrillin and LTBP-1 and -3, and their latent TGF\\u03b2 complexes. The structure of the fibrillin-LTBP1 complex shows that the two elongated proteins interact in a perpendicular arrangement which would allow them to form distal interactions between the matrix and the cell surface. Formation of the cross-link with fibrillin does not change the interaction between latent TGF\\u03b2 and integrin \\u03b1V\\u03b26 but does increase TGF\\u03b2 activation in cell-based assays. The activating effect may be due to direction of the latent complexes to the cell surface by fibrillin, as competition with heparan sulphate can ameliorate the activating effect. Together, these data support that TGF\\u03b2 activation can be enhanced by covalent tethering of LTBPs to the matrix via fibrillin.\", \"doi_y\": \"https://doi.org/10.1016/j.matbio.2022.01.005\", \"publication_date\": \"2022-03-01\"}, {\"work_id\": \"W4378894922\", \"doi_x\": \"10.1126/sciadv.adh4251\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural basis of purine nucleotide inhibition of human uncoupling protein 1\", \"abstract\": \"Mitochondrial uncoupling protein 1 (UCP1) gives brown adipose tissue of mammals its specialized ability to burn calories as heat for thermoregulation. When activated by fatty acids, UCP1 catalyzes the leak of protons across the mitochondrial inner membrane, short-circuiting the mitochondrion to generate heat, bypassing ATP synthesis. In contrast, purine nucleotides bind and inhibit UCP1, regulating proton leak by a molecular mechanism that is unclear. We present the cryo-electron microscopy structure of the GTP-inhibited state of UCP1, which is consistent with its nonconducting state. The purine nucleotide cross-links the transmembrane helices of UCP1 with an extensive interaction network. Our results provide a structural basis for understanding the specificity and pH dependency of the regulatory mechanism. UCP1 has retained all of the key functional and structural features required for a mitochondrial carrier-like transport mechanism. The analysis shows that inhibitor binding prevents the conformational changes that UCP1 uses to facilitate proton leak.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.adh4251\", \"publication_date\": \"2023-06-02\"}, {\"work_id\": \"W4310965502\", \"doi_x\": \"10.1093/nar/gkac1098\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"EMBL\\u2019s European Bioinformatics Institute (EMBL-EBI) in 2022\", \"abstract\": \"The European Molecular Biology Laboratory's European Bioinformatics Institute (EMBL-EBI) is one of the world's leading sources of public biomolecular data. Based at the Wellcome Genome Campus in Hinxton, UK, EMBL-EBI is one of six sites of the European Molecular Biology Laboratory (EMBL), Europe's only intergovernmental life sciences organisation. This overview summarises the status of services that EMBL-EBI data resources provide to scientific communities globally. The scale, openness, rich metadata and extensive curation of EMBL-EBI added-value databases makes them particularly well-suited as training sets for deep learning, machine learning and artificial intelligence applications, a selection of which are described here. The data resources at EMBL-EBI can catalyse such developments because they offer sustainable, high-quality data, collected in some cases over decades and made openly availability to any researcher, globally. Our aim is for EMBL-EBI data resources to keep providing the foundations for tools and research insights that transform fields across the life sciences.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1098\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4289781746\", \"doi_x\": \"10.1136/bmj-2022-070695\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Risk of covid-19 related deaths for SARS-CoV-2 omicron (B.1.1.529) compared with delta (B.1.617.2): retrospective cohort study\", \"abstract\": \"To assess the risk of covid-19 death after infection with omicron BA.1 compared with delta (B.1.617.2).Retrospective cohort study.England, United Kingdom, from 1 December 2021 to 30 December 2021.1 035 149 people aged 18-100 years who tested positive for SARS-CoV-2 under the national surveillance programme and had an infection identified as omicron BA.1 or delta compatible.The main outcome measure was covid-19 death as identified from death certification records. The exposure of interest was the SARS-CoV-2 variant identified from NHS Test and Trace PCR positive tests taken in the community (pillar 2) and analysed by Lighthouse laboratories. Cause specific Cox proportional hazard regression models (censoring non-covid-19 deaths) were adjusted for sex, age, vaccination status, previous infection, calendar time, ethnicity, index of multiple deprivation rank, household deprivation, university degree, keyworker status, country of birth, main language, region, disability, and comorbidities. Interactions between variant and sex, age, vaccination status, and comorbidities were also investigated.The risk of covid-19 death was 66% lower (95% confidence interval 54% to 75%) for omicron BA.1 compared with delta after adjusting for a wide range of potential confounders. The reduction in the risk of covid-19 death for omicron compared with delta was more pronounced in people aged 18-59 years (number of deaths: delta=46, omicron=11; hazard ratio 0.14, 95% confidence interval 0.07 to 0.27) than in those aged \\u226570 years (number of deaths: delta=113, omicron=135; hazard ratio 0.44, 95% confidence interval 0.32 to 0.61, P<0.0001). No evidence of a difference in risk was found between variant and number of comorbidities.The results support earlier studies showing a reduction in severity of infection with omicron BA.1 compared with delta in terms of hospital admission. This study extends the research to also show a reduction in the risk of covid-19 death for the omicron variant compared with the delta variant.\", \"doi_y\": \"https://doi.org/10.1136/bmj-2022-070695\", \"publication_date\": \"2022-08-02\"}, {\"work_id\": \"W4284886202\", \"doi_x\": \"10.1038/s41564-022-01143-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 Omicron is an immune escape variant with an altered cell entry pathway\", \"abstract\": \"Vaccines based on the spike protein of SARS-CoV-2 are a cornerstone of the public health response to COVID-19. The emergence of hypermutated, increasingly transmissible variants of concern (VOCs) threaten this strategy. Omicron (B.1.1.529), the fifth VOC to be described, harbours multiple amino acid mutations in spike, half of which lie within the receptor-binding domain. Here we demonstrate substantial evasion of neutralization by Omicron BA.1 and BA.2 variants in vitro using sera from individuals vaccinated with ChAdOx1, BNT162b2 and mRNA-1273. These data were mirrored by a substantial reduction in real-world vaccine effectiveness that was partially restored by booster vaccination. The Omicron variants BA.1 and BA.2 did not induce cell syncytia in vitro and favoured a TMPRSS2-independent endosomal entry pathway, these phenotypes mapping to distinct regions of the spike protein. Impaired cell fusion was determined by the receptor-binding domain, while endosomal entry mapped to the S2 domain. Such marked changes in antigenicity and replicative biology may underlie the rapid global spread and altered pathogenicity of the Omicron variant.\", \"doi_y\": \"https://doi.org/10.1038/s41564-022-01143-7\", \"publication_date\": \"2022-07-07\"}, {\"work_id\": \"W4315465219\", \"doi_x\": \"10.1101/2023.01.10.523363\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Comprehensive approach to study branched ubiquitin chains reveals roles for K48-K63 branches in VCP/p97-related processes\", \"abstract\": \"Branched ubiquitin (Ub) chains make up a significant proportion of Ub polymers in human cells and are formed when two or more sites on a single Ub molecule are modified with Ub creating bifurcated architectures. Despite their abundance, we have a poor understanding of the cellular functions of branched Ub signals that stems from a lack of facile tools and methods to study them. Here we develop a comprehensive pipeline to define branched Ub function, using K48-K63-branched chains as a case study. We discover branch-specific binders and, by developing a method that monitors cleavage of linkages within complex polyUb, we discover the VCP/p97-associated ATXN3, and MINDY family deubiquitinases to act as debranching enzymes. By engineering and utilizing a branched K48-K63-Ub chain-specific nanobody, we reveal roles for these chains in VCP/p97-related processes. In summary, we provide a blueprint to investigate branched Ub function that can be readily applied to study other branched chain types. Highlights Assembly of defined branched ubiquitin chains enables identification of specific binding proteins Development of quantitative DUB assay monitoring cleavage of individual Ub linkages within complex ubiquitin chains identifies debranching enzymes Engineering specific, high-affinity nanobody against branched K48-K63 ubiquitin reveals roles in VCP/p97 related processes and DNA damage responses General blueprint of new methods and tools for in-depth characterization of branched ubiquitin chains and their underlying biology\", \"doi_y\": \"https://doi.org/10.1101/2023.01.10.523363\", \"publication_date\": \"2023-01-10\"}, {\"work_id\": \"W4380343410\", \"doi_x\": \"10.1039/d3sm00431g\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA-templated self-assembly of bradykinin into bioactive nanofibrils\", \"abstract\": \"Bradykinin (BK) is a peptide hormone that plays a crucial role in blood pressure control, regulates inflammation in the human body, and has recently been implicated in the pathophysiology of COVID-19. In this study, we report a strategy for fabricating highly ordered 1D nanostructures of BK using DNA fragments as a template for self-assembly. We have combined synchrotron small-angle X-ray scattering and high-resolution microscopy to provide insights into the nanoscale structure of BK-DNA complexes, unveiling the formation of ordered nanofibrils. Fluorescence assays hint that BK is more efficient at displacing minor-groove binders in comparison with base-intercalant dyes, thus, suggesting that interaction with DNA strands is mediated by electrostatic attraction between cationic groups at BK and the high negative electron density of minor-grooves. Our data also revealed an intriguing finding that BK-DNA complexes can induce a limited uptake of nucleotides by HEK-293t cells, which is a feature that has not been previously reported for BK. Moreover, we observed that the complexes retained the native bioactivity of BK, including the ability to modulate Ca2+ response into endothelial HUVEC cells. Overall, the findings presented here demonstrate a promising strategy for the fabrication of fibrillar structures of BK using DNA as a template, which keep bioactivity features of the native peptide and may have implications in the development of nanotherapeutics for hypertension and related disorders.\", \"doi_y\": \"https://doi.org/10.1039/d3sm00431g\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4382010777\", \"doi_x\": \"10.1128/aem.00512-23\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Group Selection as a Basis for Screening Mutagenized Libraries of Public Goods (Bacillus thuringiensis Cry Toxins)\", \"abstract\": \"Insecticidal toxins from the bacterium Bacillus thuringiensis are widely exploited in genetically modified plants. This application creates a demand for novel insecticidal toxins that can be used to better manage resistant pests or control new or recalcitrant target species.\", \"doi_y\": \"https://doi.org/10.1128/aem.00512-23\", \"publication_date\": \"2023-07-26\"}, {\"work_id\": \"W3195161208\", \"doi_x\": \"10.1002/prot.26216\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<scp>Cryo\\u2010EM</scp> targets in <scp>CASP14</scp>\", \"abstract\": \"Structures of seven CASP14 targets were determined using cryo-electron microscopy (cryo-EM) technique with resolution between 2.1 and 3.8 \\u00c5. We provide an evaluation of the submitted models versus the experimental data (cryo-EM density maps) and experimental reference structures built into the maps. The accuracy of models is measured in terms of coordinate-to-density and coordinate-to-coordinate fit. A-posteriori refinement of the most accurate models in their corresponding cryo-EM density resulted in structures that are close to the reference structure, including some regions with better fit to the density. Regions that were found to be less \\\"refineable\\\" correlate well with regions of high diversity between the CASP models and low goodness-of-fit to density in the reference structure.\", \"doi_y\": \"https://doi.org/10.1002/prot.26216\", \"publication_date\": \"2021-09-16\"}, {\"work_id\": \"W4385442689\", \"doi_x\": \"10.1099/mic.0.001373\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Repurposing the atypical type I-G CRISPR system for bacterial genome engineering\", \"abstract\": \"The CRISPR-Cas system functions as a prokaryotic immune system and is highly diverse, with six major types and numerous sub-types. The most abundant are type I CRISPR systems, which utilize a multi-subunit effector, Cascade, and a CRISPR RNA (crRNA) to detect invading DNA species. Detection leads to DNA loading of the Cas3 helicase-nuclease, leading to long-range deletions in the targeted DNA, thus providing immunity against mobile genetic elements (MGE). Here, we focus on the type I-G system, a streamlined, 4-subunit complex with an atypical Cas3 enzyme. We demonstrate that Cas3 helicase activity is not essential for immunity against MGE in vivo and explore applications of the Thioalkalivibrio sulfidiphilus Cascade effector for genome engineering in Escherichia coli . Long-range, bidirectional deletions were observed when the lacZ gene was targeted. Deactivation of the Cas3 helicase activity dramatically altered the types of deletions observed, with small deletions flanked by direct repeats that are suggestive of microhomology mediated end joining. When donor DNA templates were present, both the wild-type and helicase-deficient systems promoted homology-directed repair (HDR), with the latter system providing improvements in editing efficiency, suggesting that a single nick in the target site may promote HDR in E. coli using the type I-G system. These findings open the way for further application of the type I-G CRISPR systems in genome engineering.\", \"doi_y\": \"https://doi.org/10.1099/mic.0.001373\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4292651681\", \"doi_x\": \"10.1073/pnas.2205041119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Repeat sequences limit the effectiveness of lateral gene transfer and favored the evolution of meiotic sex in early eukaryotes\", \"abstract\": \"The transition from prokaryotic lateral gene transfer to eukaryotic meiotic sex is poorly understood. Phylogenetic evidence suggests that it was tightly linked to eukaryogenesis, which involved an unprecedented rise in both genome size and the density of genetic repeats. Expansion of genome size raised the severity of Muller's ratchet, while limiting the effectiveness of lateral gene transfer (LGT) at purging deleterious mutations. In principle, an increase in recombination length combined with higher rates of LGT could solve this problem. Here, we show using a computational model that this solution fails in the presence of genetic repeats prevalent in early eukaryotes. The model demonstrates that dispersed repeat sequences allow ectopic recombination, which leads to the loss of genetic information and curtails the capacity of LGT to prevent mutation accumulation. Increasing recombination length in the presence of repeat sequences exacerbates the problem. Mutational decay can only be resisted with homology along extended sequences of DNA. We conclude that the transition to homologous pairing along linear chromosomes was a key innovation in meiotic sex, which was instrumental in the expansion of eukaryotic genomes and morphological complexity.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2205041119\", \"publication_date\": \"2022-08-22\"}, {\"work_id\": \"W4309832125\", \"doi_x\": \"10.1042/bst20220839\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"ISG15 driven cellular responses to virus infection\", \"abstract\": \"One of the hallmarks of antiviral responses to infection is the production of interferons and subsequently of interferon stimulated genes. Interferon stimulated gene 15 (ISG15) is among the earliest and most abundant proteins induced upon interferon signalling, encompassing versatile functions in host immunity. ISG15 is a ubiquitin like modifier that can be conjugated to substrates in a process analogous to ubiquitylation and referred to as ISGylation. The free unconjugated form can either exist intracellularly or be secreted to function as a cytokine. Interestingly, ISG15 has been reported to be both advantageous and detrimental to the development of immunopathology during infection. This review describes recent findings on the role of ISG15 in antiviral responses in human infection models, with a particular emphasis on autophagy, inflammatory responses and cellular metabolism combined with viral strategies of counteracting them. The field of ISGylation has steadily gained momentum; however much of the previous studies of virus infections conducted in mouse models are in sharp contrast with recent findings in human cells, underscoring the need to summarise our current understanding of its potential antiviral function in humans and identify knowledge gaps which need to be addressed in future studies.\", \"doi_y\": \"https://doi.org/10.1042/bst20220839\", \"publication_date\": \"2022-11-23\"}, {\"work_id\": \"W4223575014\", \"doi_x\": \"10.1016/j.jbc.2022.101932\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"abstract\": \"GPR84 is an immune cell\\u2013expressed, proinflammatory receptor currently being assessed as a therapeutic target in conditions including fibrosis and inflammatory bowel disease. Although it was previously shown that the orthosteric GPR84 activators 2-HTP and 6-OAU promoted its interactions with arrestin-3, a G protein\\u2013biased agonist DL-175 did not. Here, we show that replacement of all 21 serine and threonine residues within i-loop 3 of GPR84, but not the two serines in the C-terminal tail, eliminated the incorporation of [32P] and greatly reduced receptor\\u2013arrestin-3 interactions promoted by 2-HTP. GPR84 was phosphorylated constitutively on residues Ser221 and Ser224, while various other amino acids are phosphorylated in response to 2-HTP. Consistent with this, an antiserum able to identify pSer221/pSer224 recognized GPR84 from cells treated with and without activators, whereas an antiserum able to identify pThr263/pThr264 only recognized GPR84 after exposure to 2-HTP and not DL-175. Two distinct GPR84 antagonists as well as inhibition of G protein\\u2013coupled receptor kinase 2/3 prevented phosphorylation of pThr263/pThr264, but neither strategy affected constitutive phosphorylation of Ser221/Ser224. Furthermore, mutation of residues Thr263 and Thr264 to alanine generated a variant of GPR84 also limited in 2-HTP\\u2013induced interactions with arrestin-2 and -3. By contrast, this mutant was unaffected in its capacity to reduce cAMP levels. Taken together, these results define a key pair of threonine residues, regulated only by subsets of GPR84 small molecule activators and by GRK2/3 that define effective interactions with arrestins and provide novel tools to monitor the phosphorylation and functional status of GPR84. GPR84 is an immune cell\\u2013expressed, proinflammatory receptor currently being assessed as a therapeutic target in conditions including fibrosis and inflammatory bowel disease. Although it was previously shown that the orthosteric GPR84 activators 2-HTP and 6-OAU promoted its interactions with arrestin-3, a G protein\\u2013biased agonist DL-175 did not. Here, we show that replacement of all 21 serine and threonine residues within i-loop 3 of GPR84, but not the two serines in the C-terminal tail, eliminated the incorporation of [32P] and greatly reduced receptor\\u2013arrestin-3 interactions promoted by 2-HTP. GPR84 was phosphorylated constitutively on residues Ser221 and Ser224, while various other amino acids are phosphorylated in response to 2-HTP. Consistent with this, an antiserum able to identify pSer221/pSer224 recognized GPR84 from cells treated with and without activators, whereas an antiserum able to identify pThr263/pThr264 only recognized GPR84 after exposure to 2-HTP and not DL-175. Two distinct GPR84 antagonists as well as inhibition of G protein\\u2013coupled receptor kinase 2/3 prevented phosphorylation of pThr263/pThr264, but neither strategy affected constitutive phosphorylation of Ser221/Ser224. Furthermore, mutation of residues Thr263 and Thr264 to alanine generated a variant of GPR84 also limited in 2-HTP\\u2013induced interactions with arrestin-2 and -3. By contrast, this mutant was unaffected in its capacity to reduce cAMP levels. Taken together, these results define a key pair of threonine residues, regulated only by subsets of GPR84 small molecule activators and by GRK2/3 that define effective interactions with arrestins and provide novel tools to monitor the phosphorylation and functional status of GPR84. GPR84 is a nominally orphan G protein\\u2013coupled receptor (GPCR) that can be activated by both medium-chain fatty acids (MCFAs) and a range of synthetic ligands that act as either orthosteric or allosteric activators (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 2Wojciechowicz M.L. Ma'ayan A. GPR84: An immune response dial?.Nat. Rev. Drug Discov. 2020; 19: 374Crossref PubMed Scopus (11) Google Scholar, 3Chen L.H. Zhang Q. Xie X. Nan F.J. Modulation of the G-protein-coupled receptor 84 (GPR84) by agonists and antagonists.J. Med. Chem. 2020; 63: 15399-15409Crossref PubMed Scopus (6) Google Scholar). The pattern of expression across a variety of both peripheral and central immune cells (4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar, 5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar, 7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar), and that mRNA and protein corresponding to the receptor are strongly upregulated in response to proinflammatory challenges (6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar, 7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar), has promoted interest in GPR84 as a therapeutic target in a range of areas (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 2Wojciechowicz M.L. Ma'ayan A. GPR84: An immune response dial?.Nat. Rev. Drug Discov. 2020; 19: 374Crossref PubMed Scopus (11) Google Scholar). These include both ulcerative colitis and idiopathic pulmonary fibrosis (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar). Although focus for both these indications has been on the development and assessment of antagonist ligands (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 3Chen L.H. Zhang Q. Xie X. Nan F.J. Modulation of the G-protein-coupled receptor 84 (GPR84) by agonists and antagonists.J. Med. Chem. 2020; 63: 15399-15409Crossref PubMed Scopus (6) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar), it has been suggested that agonists of GPR84 might also find use in, for example, the treatment of atherosclerosis (9Gaidarov I. Anthony T. Gatlin J. Chen X. Mills D. Solomon M. Han S. Semple G. Unett D.J. Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor.Pharmacol. Res. 2018; 131: 185-198Crossref PubMed Scopus (23) Google Scholar). No matter the disease area, understanding of the extent of upregulation of the receptor in disease settings and the degree of activation of the receptor in such situations is vital. Progress in the production and characterization of a [3H]radiolabeled agonist (10K\\u00f6se M. Pillaiyar T. Namasivayam V. De Filippo E. Sylvester K. Ulven T. von K\\u00fcgelgen I. M\\u00fcller C.E. An agonist radioligand for the proinflammatory lipid-activated G protein-coupled receptor GPR84 providing structural insights.J. Med. Chem. 2020; 63: 2391-2410Crossref PubMed Scopus (10) Google Scholar) and [3H]radiolabeled antagonists (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar, 11Jenkins L. Marsango S. Mancini S. Mahmud Z.A. Morrison A. McElroy S.P. Bennett K.A. Barnes M. Tobin A.B. Tikhonova I.G. Milligan G. Discovery and characterization of novel antagonists of the proinflammatory orphan receptor GPR84.ACS Pharmacol. Transl. Sci. 2021; 4: 1598-1613Crossref PubMed Scopus (1) Google Scholar) can potentially provide insights into the former of these questions, but other reagents and biosensors will be needed to probe the degree of receptor activation. A broadly applicable feature of GPCRs following agonist occupancy is that phosphorylation of various hydroxy-amino acids within intracellular sequences of the receptor is stimulated and this promotes the effectiveness of interactions of the receptor with arrestin isoforms (12Wanka L. Behr V. Beck-Sickinger A.G. Arrestin-dependent internalization of rhodopsin-like G protein-coupled receptors.Biol. Chem. 2021; 403: 133-149Crossref PubMed Scopus (0) Google Scholar, 13Sulon S.M. Benovic J.L. Targeting G protein-coupled receptor kinases (GRKs) to G protein-coupled receptors.Curr. Opin. Endocr. Metab. Res. 2021; 16: 56-65Crossref PubMed Scopus (2) Google Scholar). This can limit the ability of the receptor to interact with G proteins and hence restrict signaling. It can also potentially promote other distinct signaling cascades and generally promotes movement of the receptor away from the cell surface and into intracellular locations. The patterns and locations of the amino acids that become phosphorylated in an agonist-dependent manner have been extensively mapped for a range of GPCRs (14Mann A. Illing S. Miess E. Schulz S. Different mechanisms of homologous and heterologous mu-opioid receptor phosphorylation.Br. J. Pharmacol. 2015; 172: 311-316Crossref PubMed Scopus (30) Google Scholar, 15Kliewer A. Reinscheid R.K. Schulz S. Emerging paradigms of G protein-coupled receptor dephosphorylation.Trends Pharmacol. Sci. 2017; 38: 621-636Abstract Full Text Full Text PDF PubMed Scopus (14) Google Scholar, 16Kliewer A. Schmiedel F. Sianati S. Bailey A. Bateman J.T. Levitt E.S. Williams J.T. Christie M.J. Schulz S. Phosphorylation-deficient G-protein-biased \\u03bc-opioid receptors improve analgesia and diminish tolerance but worsen opioid side effects.Nat. Commun. 2019; 10: 367Crossref PubMed Scopus (123) Google Scholar, 17Matthees E.S.F. Haider R.S. Hoffmann C. Drube J. differential regulation of GPCRs-are GRK expression levels the key?.Front. Cell Dev. Biol. 2021; 9: 687489Crossref PubMed Scopus (5) Google Scholar, 18Seyedabadi M. Gharghabi M. Gurevich E.V. Gurevich V.V. Receptor-arrestin interactions: The GPCR perspective.Biomolecules. 2021; 11: 218Crossref PubMed Scopus (15) Google Scholar), and both mass spectrometry and mutagenesis have contributed to this effort. Information taken from such studies can then be used to generate antisera able to identify specifically individual phosphorylated amino acids or groups of such amino acids (14Mann A. Illing S. Miess E. Schulz S. Different mechanisms of homologous and heterologous mu-opioid receptor phosphorylation.Br. J. Pharmacol. 2015; 172: 311-316Crossref PubMed Scopus (30) Google Scholar, 15Kliewer A. Reinscheid R.K. Schulz S. Emerging paradigms of G protein-coupled receptor dephosphorylation.Trends Pharmacol. Sci. 2017; 38: 621-636Abstract Full Text Full Text PDF PubMed Scopus (14) Google Scholar, 16Kliewer A. Schmiedel F. Sianati S. Bailey A. Bateman J.T. Levitt E.S. Williams J.T. Christie M.J. Schulz S. Phosphorylation-deficient G-protein-biased \\u03bc-opioid receptors improve analgesia and diminish tolerance but worsen opioid side effects.Nat. Commun. 2019; 10: 367Crossref PubMed Scopus (123) Google Scholar, 19Butcher A.J. Hudson B.D. Shimpukade B. Alvarez-Curto E. Prihandoko R. Ulven T. Milligan G. Tobin A.B. Concomitant action of structural elements and receptor phosphorylation determines arrestin-3 interaction with the free fatty acid receptor FFA4.J. Biol. Chem. 2014; 289: 18451-18465Abstract Full Text Full Text PDF PubMed Scopus (48) Google Scholar, 20Butcher A.J. Bradley S.J. Prihandoko R. Brooke S.M. Mogg A. Bourgognon J.M. Macedo-Hatch T. Edwards J.M. Bottrill A.R. Challiss R.A. Broad L.M. Felder C.C. Tobin A.B. An antibody biosensor establishes the activation of the M1 muscarinic acetylcholine receptor during learning and memory.J. Biol. Chem. 2016; 291: 8862-8875Abstract Full Text Full Text PDF PubMed Scopus (19) Google Scholar). These can then potentially be used as surrogate reagents to detect and demonstrate the extent of receptor activation in situ. Among ligands that activate GPR84, the recently described molecule 3-(2-((4-chloronaphthalen-1-yl)oxy)ethyl)pyridine 1-oxide (DL-175) is reported to be functionally \\u201cbiased,\\u201d in that, although a potent regulator of G protein\\u2013mediated signaling, it is not effective in promoting arrestin interactions (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar) and has distinct characteristics in signaling compared with the agonist 6-n-octylaminouracil (6-OAU) (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar). Because we anticipated that DL-175 and 6-OAU might well produce a different pattern or extent of phosphorylation of GPR84, to obtain a fuller understanding of such potential variation caused by GPR84 activators, herein in addition we have assessed both 2-(hexylthio)pyrimidine-4,6 diol (2-HTP) as a potent orthosteric activator (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar) and the allosteric activator PSB-16771 (22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar). This is a more potent analogue of the originally defined allosteric activator of GPR84, 3,3\\u2032-diindolylmethane (23Takeda S. Yamamoto A. Okada T. Matsumura E. Nose E. Kogure K. Kojima S. Haga T. Identification of surrogate ligands for orphan G protein-coupled receptors.Life Sci. 2003; 74: 367-377Crossref PubMed Scopus (49) Google Scholar). We show that, although both 2-HTP and 6-OAU produce effective phosphorylation of human GPR84 at residues within the third intracellular loop, neither DL-175 nor PSB-16771 does so. Moreover, although each of 2-HTP, 6-OAU, and DL-175 acts as orthosteric agonists of GPR84, we use homology modeling and mutagenesis to provide a molecular basis for their differences in function and bias. In addition, based on mass spectrometry, we identify residues in the third intracellular loop of GPR84 that are phosphorylated constitutively. Production of antisera that identify specific sites of both constitutive and agonist-regulated phosphorylation allowed us to demonstrate that phosphorylation at amino acids Thr263 and/or Thr264 is required to generate highly effective interactions with arrestin-2 and arrestin-3 and that these amino acids become phosphorylated in a GRK2/3-dependent manner. By contrast, neither Ser221 nor Ser224, which are constitutively phosphorylated and not affected by inhibition of GRK2/3, plays a major role in arrestin-2 or arrestin-3-interactions. However, the phosphorylation status of neither of these sets of amino acids impacts on the ability of the receptor to regulate cellular levels of cAMP. Of importance, as GPR84 is markedly upregulated in a wide range of immune cells and in a variety of proinflammatory settings (4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar, 5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 6Recio C. Lucy D. Purvis G.S.D. Iveson P. Zeboudj L. Iqbal A.J. Lin D. O'Callaghan C. Davison L. Griesbach E. Russell A.J. Wynne G.M. Dib L. Monaco C. Greaves D.R. Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages.Front. Immunol. 2018; 9: 1419Crossref PubMed Scopus (51) Google Scholar) the activation status detection of GPR84 by the agonist-regulated antiserum may provide an important biosensor in model systems of GPR84 regulation and function. To study mechanisms of signal transduction induced by activation of the proinflammatory GPCR GPR84 we selected a pair of orthosteric agonists, 2-HTP (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar) (also designated \\u201ccompound 1\\u201d (24Mahmud Z.A. Jenkins L. Ulven T. Lab\\u00e9gu\\u00e8re F. Gosmini R. De Vos S. Hudson B.D. Tikhonova I.G. Milligan G. Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84.Sci. Rep. 2017; 7: 17953Crossref PubMed Scopus (21) Google Scholar), or ZQ-16 (25Liu Y. Zhang Q. Chen L.H. Yang H. Lu W. Xie X. Nan F.J. Design and synthesis of 2-alkylpyrimidine-4,6-diol and 6-alkylpyridine-2,4-diol as potent GPR84 agonists.ACS Med. Chem. Lett. 2016; 7: 579-583Crossref PubMed Scopus (28) Google Scholar)) and 6-OAU (5Suzuki M. Takaishi S. Nagasaki M. Onozawa Y. Iino I. Maeda H. Komai T. Oda T. Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor.J. Biol. Chem. 2013; 288: 10684-10691Abstract Full Text Full Text PDF PubMed Scopus (121) Google Scholar, 26Pillaiyar T. K\\u00f6se M. Namasivayam V. Sylvester K. Borges G. Thimm D. von K\\u00fcgelgen I. M\\u00fcller C.E. 6-(Ar)Alkylamino-Substituted uracil derivatives: Lipid mimetics with potent activity at the orphan G protein-coupled receptor 84 (GPR84).ACS Omega. 2018; 3: 3365-3383Crossref PubMed Scopus (15) Google Scholar), as well as the allosteric activator PSB-16671 (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar) (Fig. 1A). In addition, we selected DL-175, a compound optimized following a virtual screen and reported to display marked G protein signaling compared with arrestin-interaction bias at GPR84 (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar) (Fig. 1A). To initiate studies, we expressed stably in Flp-In TREx 293 cells human GPR84 with enhanced Yellow Fluorescent Protein linked in-frame to the intracellular C-terminal tail of the receptor (hGPR84-eYFP) (24Mahmud Z.A. Jenkins L. Ulven T. Lab\\u00e9gu\\u00e8re F. Gosmini R. De Vos S. Hudson B.D. Tikhonova I.G. Milligan G. Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84.Sci. Rep. 2017; 7: 17953Crossref PubMed Scopus (21) Google Scholar). This cell system allows doxycycline-induced expression of constructs located at the Flp-In TREx locus (27Ward R.J. Alvarez-Curto E. Milligan G. Using the Flp-In\\u2122 T-Rex\\u2122 system to regulate GPCR expression.Methods Mol. Biol. 2011; 746: 21-37Crossref PubMed Scopus (41) Google Scholar). GPR84 is known to be able to interact with pertussis toxin\\u2013sensitive members of the Gi-G protein family (1Marsango S. Barki N. Jenkins L. Tobin A.B. Milligan G. Therapeutic validation of an orphan G protein-coupled receptor: The case of GPR84.Br. J. Pharmacol. 2020; https://doi.org/10.1111/bph.15248Crossref PubMed Scopus (7) Google Scholar, 4Wang J. Wu X. Simonavicius N. Tian H. Ling L. Medium-chain fatty acids as ligands for orphan G protein-coupled receptor GPR84.J. Biol. Chem. 2006; 281: 34457-34464Abstract Full Text Full Text PDF PubMed Scopus (271) Google Scholar). In membranes prepared from such doxycycline-induced cells the ability of each ligand to cause activation of Gi-G proteins was measured via [35S]GTP\\u03b3S binding assays. Each compound promoted an increase in binding of [35S]GTP\\u03b3S in a concentration-dependent fashion with rank order 2-HTP > PSB-16671 > 6-OAU = DL-175 (Fig. 1B), and each ligand displayed similar efficacy at maximally effective concentrations (Fig. 1B). Activation of GPR84 with certain, but not all, ligands is reported to promote interaction with arrestin isoforms (21Lucy D. Purvis G.S.D. Zeboudj L. Chatzopoulou M. Recio C. Bataille C.J.R. Wynne G.M. Greaves D.R. Russell A.J. A biased agonist at immunometabolic receptor GPR84 causes distinct functional effects in macrophages.ACS Chem. Biol. 2019; 14: 2055-2064Crossref PubMed Scopus (9) Google Scholar, 28M\\u00e5rtensson J. Sundqvist M. Manandhar A. Leremias L. Zhang L. Ulven T. Xie X. Bj\\u00f6rkman L. Forsman H. The two formyl peptide receptors differently regulate GPR84-mediated neutrophil NADPH oxidase activity.J. Innate Immun. 2021; 31: 1-15Google Scholar). When hGPR84-eYFP was transiently coexpressed in human embryonic kidney (HEK) 293T cells with arrestin-3 fused to nanoluciferase, 2-HTP effectively promoted proximity and potential interactions between the receptor and this arrestin construct (Fig. 1C). 6-OAU was also able to promote interactions between hGPR84-eYFP and arrestin-3-nanoluciferase. Once more 2-HTP was substantially more potent than 6-OAU, although it was not possible to add 6-OAU at sufficiently high concentrations to allow full concentration\\u2013response curves to be defined (Fig. 1C) because both compounds displayed substantially lower potency than in the [35S]GTP\\u03b3S binding studies (compare Fig. 1, B and C). In contrast, DL-175 did not promote measurable interactions between hGPR84-eYFP and arrestin-3-nanoluciferase at any concentration that was practical to assess (Fig. 1C) and neither did PSB-16671 (Fig. 1C). However, as anticipated for a positive allosteric modulator at GPR84 (7Mancini S.J. Mahmud Z.A. Jenkins L. Bolognini D. Newman R. Barnes M. Edye M.E. McMahon S.B. Tobin A.B. Milligan G. On-target and off-target effects of novel orthosteric and allosteric activators of GPR84.Sci. Rep. 2019; 9: 1861Crossref PubMed Scopus (11) Google Scholar, 22Pillaiyar T. K\\u00f6se M. Sylvester K. Weighardt H. Thimm D. Borges G. F\\u00f6rster I. von K\\u00fcgelgen I. M\\u00fcller C.E. Diindolylmethane derivatives: Potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84.J. Med. Chem. 2017; 60: 3636-3655Crossref PubMed Scopus (55) Google Scholar) PSB-16671 substantially increased (p < 0.0001) the potency of 2-HTP in such arrestin-3 interaction studies (Fig. 1D), and this was also the case for 6-OAU (p < 0.01) (Fig. 1E). By contrast, coaddition of DL-175 did not increase the observed potency of 2-HTP (Fig. 1D) or 6-OAU (Fig. 1E). Interaction of a GPCR with an arrestin is often dependent upon phosphorylation of the receptor that occurs subsequent to agonist occupancy of the receptor (12Wanka L. Behr V. Beck-Sickinger A.G. Arrestin-dependent internalization of rhodopsin-like G protein-coupled receptors.Biol. Chem. 2021; 403: 133-149Crossref PubMed Scopus (0) Google Scholar). To examine this directly we labeled Flp-In TREx 293 cells induced to express hGPR84-eYFP with [32P] orthophosphate and then treated the cells with vehicle or a concentration of 2-HTP (3.7 \\u00d7 10-6 M) determined to produce an EC90 effect in the arrestin-3 interaction BRET assay. Lysates of these cells were immunoprecipitated with an anti-Green Fluorescent Protein (GFP) antiserum (that also identifies eYFP) and then resolved by SDS-PAGE. After drying, such gels were subsequently exposed to X-ray film. No substantial incorporation of [32P] was detected in vehicle-treated samples. However, clear incorporation of [32P] into polypeptide(s) with molecular mass corresponding to some 80 kDa was observed following treatment with 2-HTP, with close to maximal incorporation of [32P] achieved within 5 min of agonist exposure (Fig. 2A). 6-OAU (6.4 \\u00d7 10\\u22125 M) also effectively promoted incorporation of [32P] into hGPR84-eYFP (Fig. 2B). Because neither DL-175 nor PSB-16671 produced effective interactions of hGPR84-eYFP with arrestin-3, and therefore no EC90 concentration could be defined, for [32P]-phosphorylation studies we employed concentrations of these ligands that were 100 times greater than their EC90 concentrations in [35S]GTP\\u03b3S binding studies on the basis that, as shown earlier, 2-HTP and 6-OAU were to this extent less potent in the arrestin-3 interaction studies than in [35S]GTP\\u03b3S binding studies. However, even at these concentrations, no significant incorporation of [32P] into hGPR84-eYFP was observed in samples treated with either DL-175 (9.8 \\u00d7 10\\u22125 M) or PSB-16671 (1.3 \\u00d7 10\\u22125 M) (Fig. 2B). As expected, the ability of 2-HTP to promote phosphorylation of hGPR84-eYFP was prevented by coincubation with either of two chemically distinct GPR84 antagonists, GLPG1205 (8Lab\\u00e9gu\\u00e8re F. Dupont S. Alvey L. Soulas F. Newsome G. Tirera A. Quenehen V. Mai T.T.T. Deprez P. Blanqu\\u00e9 R. Oste L. Le Tallec S. De Vos S. Hagers A. Vandevelde A. et al.Discovery of 9-cyclopropylethynyl-2-((S)-1-[1,4]dioxan-2-ylmethoxy)-6,7-dihydropyrimido[6,1-a]isoquinolin-4-one (GLPG1205), a unique GPR84 negative allosteric modulator undergoing evaluation in a phase II clinical trial.J. Med. Chem. 2020; 63: 13526-13545Crossref PubMed Scopus (10) Google Scholar) and compound 837 (11Jenkins L. Marsango S. Mancini S. Mahmud Z.A. Morrison A. McElroy S.P. Bennett K.A. Barnes M. Tobin A.B. Tikhonova I.G. Milligan G. Discovery and characterization of novel antagonists of the proinflammatory orphan receptor GPR84.ACS Pharmacol. Transl. Sci. 2021; 4: 1598-1613Crossref PubMed Scopus (1) Google Scholar) (each at 1 \\u00d7 10\\u22125 M) (Fig. 2C). Because agonist-mediated phosphorylation of many GPCRs reflects interactions with members of the GRK family we assessed the effect of the GRK2/3 inhibitor compound 101 (1 \\u00d7 10\\u22125 M) (29Thal D.M. Yeow R.Y. Schoenau C. Huber J. Tesmer J.J. Molecular mechanism of selectivity among G protein-coupled receptor kinase 2 inhibitors.Mol. Pharmacol. 2011; 80: 294-303Crossref PubMed Scopus (81) Google Scholar, 30Lowe J.D. Sanderson H.S. Cooke A.E. Ostovar M. Tsisanova E. Withey S.L. Chavkin C. Husbands S.M. Kelly E. Henderson G. Bailey C.P. Role of G Protein-Coupled receptor kinases 2 and 3 in \\u03bc-opioid receptor desensitization and internalization.Mol. Pharmacol. 2015; 88: 347-356Crossref PubMed Scopus (48) Google Scholar). [32P] phosphorylation of hGPR84-eYFP in respons\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.101932\", \"publication_date\": \"2022-05-01\"}, {\"work_id\": \"W4380370147\", \"doi_x\": \"10.1021/jacsau.3c00163\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Polyvalent Nano-Lectin Potently Neutralizes SARS-CoV-2 by Targeting Glycans on the Viral Spike Protein\", \"abstract\": \"Mutations in spike (S) protein epitopes allow SARS-CoV-2 variants to evade antibody responses induced by infection and/or vaccination. In contrast, mutations in glycosylation sites across SARS-CoV-2 variants are very rare, making glycans a potential robust target for developing antivirals. However, this target has not been adequately exploited for SARS-CoV-2, mostly due to intrinsically weak monovalent protein\\u2013glycan interactions. We hypothesize that polyvalent nano-lectins with flexibly linked carbohydrate recognition domains (CRDs) can adjust their relative positions and bind multivalently to S protein glycans, potentially exerting potent antiviral activity. Herein, we displayed the CRDs of DC-SIGN, a dendritic cell lectin known to bind to diverse viruses, polyvalently onto 13 nm gold nanoparticles (named G13-CRD). G13-CRD bound strongly and specifically to target glycan-coated quantum dots with sub-nM Kd. Moreover, G13-CRD neutralized particles pseudotyped with the S proteins of Wuhan Hu-1, B.1, Delta variant and Omicron subvariant BA.1 with low nM EC50. In contrast, natural tetrameric DC-SIGN and its G13 conjugate were ineffective. Further, G13-CRD potently inhibited authentic SARS-CoV-2 B.1 and BA.1, with <10 pM and <10 nM EC50, respectively. These results identify G13-CRD as the 1st polyvalent nano-lectin with broad activity against SARS-CoV-2 variants that merits further exploration as a novel approach to antiviral therapy.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.3c00163\", \"publication_date\": \"2023-06-12\"}, {\"work_id\": \"W4377826683\", \"doi_x\": \"10.1016/j.ceca.2023.102761\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"IP3 receptors: An \\u201celementary\\u201d journey from structure to signals\", \"abstract\": \"Inositol 1,4,5-trisphosphate receptors (IP3Rs) are large tetrameric channels which sit mostly in the membrane of the endoplasmic reticulum (ER) and mediate Ca2+ release from intracellular stores in response to extracellular stimuli in almost all cells. Dual regulation of IP3Rs by IP3 and Ca2+ itself, upstream \\\"licensing\\\", and the arrangement of IP3Rs into small clusters in the ER membrane, allow IP3Rs to generate spatially and temporally diverse Ca2+ signals. The characteristic biphasic regulation of IP3Rs by cytosolic Ca2+ concentration underpins regenerative Ca2+ signals by Ca2+-induced Ca2+-release, while also preventing uncontrolled explosive Ca2+ release. In this way, cells can harness a simple ion such as Ca2+ as a near-universal intracellular messenger to regulate diverse cellular functions, including those with conflicting outcomes such as cell survival and cell death. High-resolution structures of the IP3R bound to IP3 and Ca2+ in different combinations have together started to unravel the workings of this giant channel. Here we discuss, in the context of recently published structures, how the tight regulation of IP3Rs and their cellular geography lead to generation of \\\"elementary\\\" local Ca2+ signals known as Ca2+ \\\"puffs\\\", which form the fundamental bottleneck through which all IP3-mediated cytosolic Ca2+ signals must first pass.\", \"doi_y\": \"https://doi.org/10.1016/j.ceca.2023.102761\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4324129190\", \"doi_x\": \"10.1002/anie.202215719\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering an Oxygen\\u2010Binding Protein for Photocatalytic CO<sub>2</sub>Reductions in Water\", \"abstract\": \"While native CO2 -reducing enzymes display remarkable catalytic efficiency and product selectivity, few artificial biocatalysts have been engineered to allow understanding how the native enzymes work. To address this issue, we report cobalt porphyrin substituted myoglobin (CoMb) as a homogeneous catalyst for photo-driven CO2 to CO conversion in water. The activity and product selectivity were optimized by varying pH and concentrations of the enzyme and the photosensitizer. Up to 2000 TON(CO) was attained at low enzyme concentrations with low product selectivity (15 %), while a product selectivity of 74 % was reached by increasing the enzyme loading but with a compromised TON(CO). The efficiency of CO generation and overall TON(CO) were further improved by introducing positively charged residues (Lys or Arg) near the active stie of CoMb, which demonstrates the value of tuning the enzyme secondary coordination sphere to enhance the CO2 -reducing performance of a protein-based photocatalytic system.\", \"doi_y\": \"https://doi.org/10.1002/anie.202215719\", \"publication_date\": \"2023-04-04\"}, {\"work_id\": \"W4282913766\", \"doi_x\": \"10.1101/2022.06.09.22276196\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Accelerated waning of the humoral response to SARS-CoV-2 vaccines in obesity\", \"abstract\": \"ABSTRACT Obesity is associated with an increased risk of severe Covid-19. However, the effectiveness of SARS-CoV-2 vaccines in people with obesity is unknown. Here we studied the relationship between body mass index (BMI), hospitalization and mortality due to Covid-19 amongst 3.5 million people in Scotland. Vaccinated people with severe obesity (BMI&gt;40 kg/m 2 ) were significantly more likely to experience hospitalization or death from Covid-19. Excess risk increased with time since vaccination. To investigate the underlying mechanisms, we conducted a prospective longitudinal study of the immune response in a clinical cohort of vaccinated people with severe obesity. Compared with normal weight people, six months after their second vaccine dose, significantly more people with severe obesity had unquantifiable titres of neutralizing antibody against authentic SARS-CoV-2 virus, reduced frequencies of antigen-experienced SARS-CoV-2 Spike-binding B cells, and a dissociation between anti-Spike antibody levels and neutralizing capacity. Neutralizing capacity was restored by a third dose of vaccine, but again declined more rapidly in people with severe obesity. We demonstrate that waning of SARS-CoV-2 vaccine-induced humoral immunity is accelerated in people with severe obesity and associated with increased hospitalization and mortality from breakthrough infections. Given the prevalence of obesity, our findings have significant implications for global public health.\", \"doi_y\": \"https://doi.org/10.1101/2022.06.09.22276196\", \"publication_date\": \"2022-06-14\"}, {\"work_id\": \"W4210284779\", \"doi_x\": \"10.3389/fmed.2021.836826\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Viral Load in COVID-19 Patients: Implications for Prognosis and Vaccine Efficacy in the Context of Emerging SARS-CoV-2 Variants\", \"abstract\": \"The worldwide spread of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has caused an unprecedented public health crisis in the 21st century. As the pandemic evolves, the emergence of SARS-CoV-2 has been characterized by the emergence of new variants of concern (VOCs), which resulted in a catastrophic impact on SARS-CoV-2 infection. In light of this, research groups around the world are unraveling key aspects of the associated illness, coronavirus disease 2019 (COVID-19). A cumulative body of data has indicated that the SARS-CoV-2 viral load may be a determinant of the COVID-19 severity. Here we summarize the main characteristics of the emerging variants of SARS-CoV-2, discussing their impact on viral transmissibility, viral load, disease severity, vaccine breakthrough, and lethality among COVID-19 patients. We also provide a rundown of the rapidly expanding scientific evidence from clinical studies and animal models that indicate how viral load could be linked to COVID-19 prognosis and vaccine efficacy among vaccinated individuals, highlighting the differences compared to unvaccinated individuals.\", \"doi_y\": \"https://doi.org/10.3389/fmed.2021.836826\", \"publication_date\": \"2022-01-31\"}, {\"work_id\": \"W4298109826\", \"doi_x\": \"10.3389/fpls.2022.967203\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"abstract\": \"OPINION article Front. Plant Sci., 30 September 2022Sec. Photosynthesis and Photobiology Volume 13 - 2022 | https://doi.org/10.3389/fpls.2022.967203\", \"doi_y\": \"https://doi.org/10.3389/fpls.2022.967203\", \"publication_date\": \"2022-09-30\"}, {\"work_id\": \"W4306152028\", \"doi_x\": \"10.3390/ijms232012237\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"How Arrestins and GRKs Regulate the Function of Long Chain Fatty Acid Receptors\", \"abstract\": \"FFA1 and FFA4, two G protein-coupled receptors that are activated by long chain fatty acids, play crucial roles in mediating many biological functions in the body. As a result, these fatty acid receptors have gained considerable attention due to their potential to be targeted for the treatment of type-2 diabetes. However, the relative contribution of canonical G protein-mediated signalling versus the effects of agonist-induced phosphorylation and interactions with \\u03b2-arrestins have yet to be fully defined. Recently, several reports have highlighted the ability of \\u03b2-arrestins and GRKs to interact with and modulate different functions of both FFA1 and FFA4, suggesting that it is indeed important to consider these interactions when studying the roles of FFA1 and FFA4 in both normal physiology and in different disease settings. Here, we discuss what is currently known and show the importance of understanding fully how \\u03b2-arrestins and GRKs regulate the function of long chain fatty acid receptors.\", \"doi_y\": \"https://doi.org/10.3390/ijms232012237\", \"publication_date\": \"2022-10-13\"}, {\"work_id\": \"W4311502524\", \"doi_x\": \"10.15252/embj.2022111372\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"NDP52 acts as a redox sensor in PINK1/Parkin\\u2010mediated mitophagy\", \"abstract\": \"Abstract Mitophagy, the elimination of mitochondria via the autophagy\\u2010lysosome pathway, is essential for the maintenance of cellular homeostasis. The best characterised mitophagy pathway is mediated by stabilisation of the protein kinase PINK1 and recruitment of the ubiquitin ligase Parkin to damaged mitochondria. Ubiquitinated mitochondrial surface proteins are recognised by autophagy receptors including NDP52 which initiate the formation of an autophagic vesicle around the mitochondria. Damaged mitochondria also generate reactive oxygen species (ROS) which have been proposed to act as a signal for mitophagy, however the mechanism of ROS sensing is unknown. Here we found that oxidation of NDP52 is essential for the efficient PINK1/Parkin\\u2010dependent mitophagy. We identified redox\\u2010sensitive cysteine residues involved in disulphide bond formation and oligomerisation of NDP52 on damaged mitochondria. Oligomerisation of NDP52 facilitates the recruitment of autophagy machinery for rapid mitochondrial degradation. We propose that redox sensing by NDP52 allows mitophagy to function as a mechanism of oxidative stress response.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111372\", \"publication_date\": \"2022-12-14\"}, {\"work_id\": \"W4224310640\", \"doi_x\": \"10.3390/microorganisms10050863\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Identification and Validation of Toxoplasma gondii Mitoribosomal Large Subunit Components\", \"abstract\": \"Mitochondrial ribosomes are fundamental to mitochondrial function, and thus survival, of nearly all eukaryotes. Despite their common ancestry, mitoribosomes have evolved divergent features in different eukaryotic lineages. In apicomplexans, the mitochondrial rRNA is extremely fragmented raising questions about its evolution, protein composition and structure. Apicomplexan mitochondrial translation and the mitoribosomes are essential in all parasites and life stages studied, highlighting mitoribosomes as a promising target for drugs. Still, the apicomplexan mitoribosome is understudied, with one of the obstacles being that its composition is unknown. Here, to facilitate the study of apicomplexan mitoribosomes, we identified and validated components of the mitoribosomal large subunit in the model apicomplexan Toxoplasma gondii.\", \"doi_y\": \"https://doi.org/10.3390/microorganisms10050863\", \"publication_date\": \"2022-04-21\"}, {\"work_id\": \"W4223926836\", \"doi_x\": \"10.1371/journal.pbio.3001601\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"abstract\": \"Coat complexes coordinate cargo recognition through cargo adaptors with biogenesis of transport carriers during integral membrane protein trafficking. Here, we combine biochemical, structural, and cellular analyses to establish the mechanistic basis through which SNX27-Retromer, a major endosomal cargo adaptor, couples to the membrane remodeling endosomal SNX-BAR sorting complex for promoting exit 1 (ESCPE-1). In showing that the SNX27 FERM (4.1/ezrin/radixin/moesin) domain directly binds acidic-Asp-Leu-Phe (aDLF) motifs in the SNX1/SNX2 subunits of ESCPE-1, we propose a handover model where SNX27-Retromer captured cargo proteins are transferred into ESCPE-1 transport carriers to promote endosome-to-plasma membrane recycling. By revealing that assembly of the SNX27:Retromer:ESCPE-1 coat evolved in a stepwise manner during early metazoan evolution, likely reflecting the increasing complexity of endosome-to-plasma membrane recycling from the ancestral opisthokont to modern animals, we provide further evidence of the functional diversification of yeast pentameric Retromer in the recycling of hundreds of integral membrane proteins in metazoans.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001601\", \"publication_date\": \"2022-04-13\"}, {\"work_id\": \"W4381735028\", \"doi_x\": \"10.7554/elife.87178.3\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural and functional properties of the transporter SLC26A6 reveal mechanism of coupled anion exchange\", \"abstract\": \"Members of the SLC26 family constitute a conserved class of anion transport proteins, which encompasses uncoupled transporters with channel-like properties, coupled exchangers and motor proteins. Among the 10 functional paralogs in humans, several participate in the secretion of bicarbonate in exchange with chloride and thus play an important role in maintaining pH homeostasis. Previously, we have elucidated the structure of murine SLC26A9 and defined its function as an uncoupled chloride transporter (Walter et al., 2019). Here we have determined the structure of the closely related human transporter SLC26A6 and characterized it as a coupled exchanger of chloride with bicarbonate and presumably also oxalate. The structure defines an inward-facing conformation of the protein that generally resembles known structures of SLC26A9. The altered anion selectivity between both paralogs is a consequence of a remodeled ion binding site located in the center of a mobile unit of the membrane-inserted domain, which also accounts for differences in the coupling mechanism.\", \"doi_y\": \"https://doi.org/10.7554/elife.87178.3\", \"publication_date\": \"2023-06-23\"}, {\"work_id\": \"W4224227149\", \"doi_x\": \"10.1101/cshperspect.a041390\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The Evolution and Biology of SARS-CoV-2 Variants\", \"abstract\": \"Our understanding of the still unfolding severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic would have been extremely limited without the study of the genetics and evolution of this new human coronavirus. Large-scale genome-sequencing efforts have provided close to real-time tracking of the global spread and diversification of SARS-CoV-2 since its entry into the human population in late 2019. These data have underpinned analysis of its origins, epidemiology, and adaptations to the human population: principally immune evasion and increasing transmissibility. SARS-CoV-2, despite being a new human pathogen, was highly capable of human-to-human transmission. During its rapid spread in humans, SARS-CoV-2 has evolved independent new forms, the so-called \\\"variants of concern,\\\" that are better optimized for human-to-human transmission. The most important adaptation of the bat coronavirus progenitor of both SARS-CoV-1 and SARS-CoV-2 for human infection (and other mammals) is the use of the angiotensin-converting enzyme 2 (ACE2) receptor. Relaxed structural constraints provide plasticity to SARS-related coronavirus spike protein permitting it to accommodate significant amino acid replacements of antigenic consequence without compromising the ability to bind to ACE2. Although the bulk of research has justifiably concentrated on the viral spike protein as the main determinant of antigenic evolution and changes in transmissibility, there is accumulating evidence for the contribution of other regions of the viral proteome to virus-host interaction. Whereas levels of community transmission of recombinants compromising genetically distinct variants are at present low, when divergent variants cocirculate, recombination between SARS-CoV-2 clades is being detected, increasing the risk that viruses with new properties emerge. Applying computational and machine learning methods to genome sequence data sets to generate experimentally verifiable predictions will serve as an early warning system for novel variant surveillance and will be important in future vaccine planning. Omicron, the latest SARS-CoV-2 variant of concern, has focused attention on step change antigenic events, \\\"shift,\\\" as opposed to incremental \\\"drift\\\" changes in antigenicity. Both an increase in transmissibility and antigenic shift in Omicron led to it readily causing infections in the fully vaccinated and/or previously infected. Omicron's virulence, while reduced relative to the variant of concern it replaced, Delta, is very much premised on the past immune exposure of individuals with a clear signal that boosted vaccination protects from severe disease. Currently, SARS-CoV-2 has proven itself to be a dangerous new human respiratory pathogen with an unpredictable evolutionary capacity, leading to a risk of future variants too great not to ensure all regions of the world are screened by viral genome sequencing, protected through available and affordable vaccines, and have non-punitive strategies in place for detecting and responding to novel variants of concern.\", \"doi_y\": \"https://doi.org/10.1101/cshperspect.a041390\", \"publication_date\": \"2022-04-20\"}, {\"work_id\": \"W4283260526\", \"doi_x\": \"10.1038/s41598-022-14411-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Transmission of B.1.617.2 Delta variant between vaccinated healthcare workers\", \"abstract\": \"Abstract Breakthrough infections with SARS-CoV-2 Delta variant have been reported in doubly-vaccinated recipients and as re-infections. Studies of viral spread within hospital settings have highlighted the potential for transmission between doubly-vaccinated patients and health care workers and have highlighted the benefits of high-grade respiratory protection for health care workers. However the extent to which vaccination is preventative of viral spread in health care settings is less well studied. Here, we analysed data from 118 vaccinated health care workers (HCW) across two hospitals in India, constructing two probable transmission networks involving six HCWs in Hospital A and eight HCWs in Hospital B from epidemiological and virus genome sequence data, using a suite of computational approaches. A maximum likelihood reconstruction of transmission involving known cases of infection suggests a high probability that doubly vaccinated HCWs transmitted SARS-CoV-2 between each other and highlights potential cases of virus transmission between individuals who had received two doses of vaccine. Our findings show firstly that vaccination may reduce rates of transmission, supporting the need for ongoing infection control measures even in highly vaccinated populations, and secondly we have described a novel approach to identifying transmissions that is scalable and rapid, without the need for an infection control infrastructure.\", \"doi_y\": \"https://doi.org/10.1038/s41598-022-14411-7\", \"publication_date\": \"2022-06-21\"}, {\"work_id\": \"W4313527937\", \"doi_x\": \"10.1088/2633-1357/acadb9\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Morphology exploration of pollen using deep learning latent space\", \"abstract\": \"Abstract The structure of pollen has evolved depending on its local environment, competition, and ecology. As pollen grains are generally of size 10\\u2013100 microns with nanometre-scale substructure, scanning electron microscopy is an important microscopy technique for imaging and analysis. Here, we use style transfer deep learning to allow exploration of latent w-space of scanning electron microscope images of pollen grains and show the potential for using this technique to understand evolutionary pathways and characteristic structural traits of pollen grains.\", \"doi_y\": \"https://doi.org/10.1088/2633-1357/acadb9\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4200390169\", \"doi_x\": \"10.1101/2021.12.03.471045\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"abstract\": \"Summary On the 24 th November 2021 the sequence of a new SARS CoV-2 viral isolate spreading rapidly in Southern Africa was announced, containing far more mutations in Spike (S) than previously reported variants. Neutralization titres of Omicron by sera from vaccinees and convalescent subjects infected with early pandemic as well as Alpha, Beta, Gamma, Delta are substantially reduced or fail to neutralize. Titres against Omicron are boosted by third vaccine doses and are high in cases both vaccinated and infected by Delta. Mutations in Omicron knock out or substantially reduce neutralization by most of a large panel of potent monoclonal antibodies and antibodies under commercial development. Omicron S has structural changes from earlier viruses, combining mutations conferring tight binding to ACE2 to unleash evolution driven by immune escape, leading to a large number of mutations in the ACE2 binding site which rebalance receptor affinity to that of early pandemic viruses.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.03.471045\", \"publication_date\": \"2021-12-07\"}, {\"work_id\": \"W4317868556\", \"doi_x\": \"10.1073/pnas.2212577120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"SARS-CoV-2 evolution influences GBP and IFITM sensitivity\", \"abstract\": \"SARS-CoV-2 spike requires proteolytic processing for viral entry. A polybasic furin-cleavage site (FCS) in spike, and evolution toward an optimized FCS by dominant variants of concern (VOCs), are linked to enhanced infectivity and transmission. Here we show interferon-inducible restriction factors Guanylate-binding proteins (GBP) 2 and 5 interfere with furin-mediated spike cleavage and inhibit the infectivity of early-lineage isolates Wuhan-Hu-1 and VIC. By contrast, VOCs Alpha and Delta escape restriction by GBP2/5 that we map to the spike substitution D614G present in these VOCs. Despite inhibition of spike cleavage, these viruses remained sensitive to plasma membrane IFITM1, but not endosomal IFITM2 and 3, consistent with a preference for TMPRSS2-dependent plasma membrane entry. Strikingly, we find that Omicron is unique among VOCs, being sensitive to restriction factors GBP2/5, and also IFITM1, 2, and 3. Using chimeric spike mutants, we map the Omicron phenotype and show that the S1 domain determines Omicron's sensitivity to GBP2/5, whereas the S2' domain determines its sensitivity to endosomal IFITM2/3 and preferential use of TMPRSS2-independent entry. We propose that evolution of SARS-CoV-2 for the D614G substitution has allowed for escape from GBP restriction factors, but the selective pressures on Omicron for spike changes that mediate antibody escape, and altered tropism, have come at the expense of increased sensitivity to innate immune restriction factors that target virus entry.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2212577120\", \"publication_date\": \"2023-01-24\"}, {\"work_id\": \"W4312213393\", \"doi_x\": \"10.1101/2022.12.26.22283911\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Exome sequencing reveals aggregates of rare variants in glycosyltransferase and other genes influencing immunoglobulin G and transferrin glycosylation\", \"abstract\": \"Abstract It is often difficult to be certain which genes underlie the effects seen in association studies. However, variants that disrupt the protein, such as predicted loss of function (pLoF) and missense variants, provide a shortcut to identify genes with a clear biological link to the phenotype of interest. Glycosylation is one of the most common post-translationalmodifications of proteins, and an important biomarker of both disease and its progression. Here, we utilised the power of genetic isolates, gene-based aggregation tests and intermediate phenotypes to assess the effect of rare (MAF&lt;5%) pLoF and missense variants from whole exome sequencing on the N-glycome of plasma transferrin (N=1907) and immunoglobulin G (N=4912), and their effect on diseases. We identified significant gene-based associations for transferrin glycosylation at 5 genes (p&lt;8.06\\u00d710 \\u22128 ) and for IgG glycan traits at 4 genes (p&lt;1.19\\u00d710 \\u22127 ). Associations in three of these genes ( FUT8, MGAT3 and RFXAP ) are driven by multiple rare variants simultaneously contributing to protein glycosylation. Association at ST6GAL1 , with a 300-fold up-drifted variant in the Orkney Islands, was detectable by a single-point exome-wide association analysis. Glycome-associated aggregate associations are located in genes already known to have a biological link to protein glycosylation ( FUT6, FUT8 for transferrin; FUT8, MGAT3 and ST6GAL1 for IgG) but also in genes which have not been previously reported (e.g. RFXAP for IgG). To assess the potential impact of rare variants associated with glycosylation on other traits, we queried public repositories of gene-based tests, discovering a potential connection between transferrin glycosylation, MSR1 , galectin-3, insulin-like growth factor 1 and diabetes. However, the exact mechanism behind these connections requires further elucidation.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.26.22283911\", \"publication_date\": \"2022-12-27\"}, {\"work_id\": \"W4378640426\", \"doi_x\": \"10.1210/endocr/bqad075\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Hypothalamic PVN CRH neurons signal through PVN GABA neurons to suppress GnRH pulse generator frequency in female mice\", \"abstract\": \"Corticotropin-releasing hormone (CRH) neurons in the paraventricular nucleus of the hypothalamus (PVN) are central to the stress response. Chemogenetic activation of PVN CRH neurons decreases LH pulse frequency but the mechanism is unknown. In the present study, optogenetic stimulation of PVN CRH neurons suppressed LH pulse frequency in estradiol-replaced ovariectomized CRH-cre mice, and this effect was augmented or attenuated by intra-PVN GABAA or GABAB receptor antagonism, respectively. PVN CRH neurons signal to local GABA neurons, which may provide a possible indirect mechanism by which PVN CRH neurons suppress LH pulse frequency. Optogenetic stimulation of potential PVN GABAergic projection terminals in the hypothalamic arcuate nucleus in ovariectomized estradiol-replaced Vgat-cre-tdTomato mice via an optic fiber implanted in the arcuate nucleus suppressed LH pulse frequency. To further determine whether PVN CRH neurons signal through PVN GABA neurons to suppress LH pulsatility, we combined recombinase mice with intersectional vectors to selectively target these neurons. CRH-cre::Vgat-FlpO mice expressing the stimulatory opsin ChRmine in non-GABAergic CRH neurons alone or in combination with the inhibitory opsin NpHR3.3 in non-CRH-expressing GABA neurons in the PVN were used. Optogenetic stimulation of non-GABAergic CRH neurons suppressed pulsatile LH secretion; however, LH pulse frequency was not affected when CRH neurons were stimulated and PVN GABA neurons were simultaneously inhibited. Together, these studies demonstrate that suppression of LH pulse frequency in response to PVN CRH neuronal activation is mediated by GABAergic signalling intrinsic to the PVN and may incorporate PVN GABAergic projection to the hypothalamic GnRH pulse generator.\", \"doi_y\": \"https://doi.org/10.1210/endocr/bqad075\", \"publication_date\": \"2023-04-17\"}, {\"work_id\": \"W4286560310\", \"doi_x\": \"10.3389/fmolb.2022.908285\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Redesigning Enzymes for Biocatalysis: Exploiting Structural Understanding for Improved Selectivity\", \"abstract\": \"The discovery of new enzymes, alongside the push to make chemical processes more sustainable, has resulted in increased industrial interest in the use of biocatalytic processes to produce high-value and chiral precursor chemicals. Huge strides in protein engineering methodology and in silico tools have facilitated significant progress in the discovery and production of enzymes for biocatalytic processes. However, there are significant gaps in our knowledge of the relationship between enzyme structure and function. This has demonstrated the need for improved computational methods to model mechanisms and understand structure dynamics. Here, we explore efforts to rationally modify enzymes toward changing aspects of their catalyzed chemistry. We highlight examples of enzymes where links between enzyme function and structure have been made, thus enabling rational changes to the enzyme structure to give predictable chemical outcomes. We look at future directions the field could take and the technologies that will enable it.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.908285\", \"publication_date\": \"2022-07-22\"}, {\"work_id\": \"W4308842536\", \"doi_x\": \"10.1016/j.pbi.2022.102311\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"abstract\": \"Crop yield and global food security are under constant threat from plant pathogens with the potential to cause epidemics. Traditional breeding for disease resistance can be too slow to counteract these emerging threats, resulting in the need to retool the plant immune system using bioengineered made-to-order immune receptors. Efforts to engineer immune receptors have focused primarily on nucleotide-binding domain and leucine-rich repeat (NLR) immune receptors and proof-of-principles studies. Based upon a near-exhaustive literature search of previously engineered plant immune systems we distil five emerging principles in the design of bioengineered made-to-order plant NLRs and describe approaches based on other components. These emerging principles are anticipated to assist the functional understanding of plant immune receptors, as well as bioengineering novel disease resistance specificities.\", \"doi_y\": \"https://doi.org/10.1016/j.pbi.2022.102311\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4315490974\", \"doi_x\": \"10.3389/fnume.2022.1058388\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Radiometal chelators for infection diagnostics\", \"abstract\": \"Infection of native tissues or implanted devices is common, but clinical diagnosis is frequently difficult and currently available noninvasive tests perform poorly. Immunocompromised individuals (for example transplant recipients, or those with cancer) are at increased risk. No imaging test in clinical use can specifically identify infection, or accurately differentiate bacterial from fungal infections. Commonly used [ 18 F]fluorodeoxyglucose (18FDG) positron emission computed tomography (PET/CT) is sensitive for infection, but limited by poor specificity because increased glucose uptake may also indicate inflammation or malignancy. Furthermore, this tracer provides no indication of the type of infective agent (bacterial, fungal, or parasitic). Imaging tools that directly and specifically target microbial pathogens are highly desirable to improve noninvasive infection diagnosis and localization. A growing field of research is exploring the utility of radiometals and their chelators (siderophores), which are small molecules that bind radiometals and form a stable complex allowing sequestration by microbes. This radiometal-chelator complex can be directed to a specific microbial target in vivo , facilitating anatomical localization by PET or single photon emission computed tomography. Additionally, bifunctional chelators can further conjugate therapeutic molecules (e.g., peptides, antibiotics, antibodies) while still bound to desired radiometals, combining specific imaging with highly targeted antimicrobial therapy. These novel therapeutics may prove a useful complement to the armamentarium in the global fight against antimicrobial resistance. This review will highlight current state of infection imaging diagnostics and their limitations, strategies to develop infection-specific diagnostics, recent advances in radiometal-based chelators for microbial infection imaging, challenges, and future directions to improve targeted diagnostics and/or therapeutics.\", \"doi_y\": \"https://doi.org/10.3389/fnume.2022.1058388\", \"publication_date\": \"2023-01-09\"}, {\"work_id\": \"W4319460068\", \"doi_x\": \"10.1016/j.jbc.2023.102989\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biochemical and structural basis of sialic acid utilization by gut microbes\", \"abstract\": \"The human gastrointestinal (GI) tract harbors diverse microbial communities collectively known as the gut microbiota that exert a profound impact on human health and disease. The repartition and availability of sialic acid derivatives in the gut have a significant impact on the modulation of gut microbes and host susceptibility to infection and inflammation. Although N-acetylneuraminic acid (Neu5Ac) is the main form of sialic acids in humans, the sialic acid family regroups more than 50 structurally and chemically distinct modified derivatives. In the GI tract, sialic acids are found in the terminal location of mucin glycan chains constituting the mucus layer and also come from human milk oligosaccharides in the infant gut or from meat-based foods in adults. The repartition of sialic acid in the GI tract influences the gut microbiota composition and pathogen colonization. In this review, we provide an update on the mechanisms underpinning sialic acid utilization by gut microbes, focusing on sialidases, transporters, and metabolic enzymes.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.102989\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4362589713\", \"doi_x\": \"10.1016/j.str.2023.03.008\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Single-particle cryo-EM analysis of the shell architecture and internal organization of an intact \\u03b1-carboxysome\", \"abstract\": \"Carboxysomes are proteinaceous bacterial microcompartments that sequester the key enzymes for carbon fixation in cyanobacteria and some proteobacteria. They consist of a virus-like icosahedral shell, encapsulating several enzymes, including ribulose 1,5-bisphosphate carboxylase/oxygenase (RuBisCO), responsible for the first step of the Calvin-Benson-Bassham cycle. Despite their significance in carbon fixation and great bioengineering potentials, the structural understanding of native carboxysomes is currently limited to low-resolution studies. Here, we report the characterization of a native \\u03b1-carboxysome from a marine cyanobacterium by single-particle cryoelectron microscopy (cryo-EM). We have determined the structure of its RuBisCO enzyme, and obtained low-resolution maps of its icosahedral shell, and of its concentric interior organization. Using integrative modeling approaches, we have proposed a complete atomic model of an intact carboxysome, providing insight into its organization and assembly. This is critical for a better understanding of the carbon fixation mechanism and toward repurposing carboxysomes in synthetic biology for biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1016/j.str.2023.03.008\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W3201894557\", \"doi_x\": \"10.1073/pnas.2100198118\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural basis of rotavirus RNA chaperone displacement and RNA annealing\", \"abstract\": \"Rotavirus genomes are distributed between 11 distinct RNA molecules, all of which must be selectively copackaged during virus assembly. This likely occurs through sequence-specific RNA interactions facilitated by the RNA chaperone NSP2. Here, we report that NSP2 autoregulates its chaperone activity through its C-terminal region (CTR) that promotes RNA-RNA interactions by limiting its helix-unwinding activity. Unexpectedly, structural proteomics data revealed that the CTR does not directly interact with RNA, while accelerating RNA release from NSP2. Cryo-electron microscopy reconstructions of an NSP2-RNA complex reveal a highly conserved acidic patch on the CTR, which is poised toward the bound RNA. Virus replication was abrogated by charge-disrupting mutations within the acidic patch but completely restored by charge-preserving mutations. Mechanistic similarities between NSP2 and the unrelated bacterial RNA chaperone Hfq suggest that accelerating RNA dissociation while promoting intermolecular RNA interactions may be a widespread strategy of RNA chaperone recycling.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2100198118\", \"publication_date\": \"2021-10-06\"}, {\"work_id\": \"W4308433996\", \"doi_x\": \"10.1080/21541248.2022.2131313\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Small-molecule inhibitors of P-Rex guanine-nucleotide exchange factors\", \"abstract\": \"P-Rex1 and P-Rex2 are guanine-nucleotide exchange factors (GEFs) that activate Rac small GTPases in response to the stimulation of G protein-coupled receptors and phosphoinositide 3-kinase. P-Rex Rac-GEFs regulate the morphology, adhesion and migration of various cell types, as well as reactive oxygen species production and cell cycle progression. P-Rex Rac-GEFs also have pathogenic roles in the initiation, progression or metastasis of several types of cancer. With one exception, all P-Rex functions are known or assumed to be mediated through their catalytic Rac-GEF activity. Thus, inhibitors of P-Rex Rac-GEF activity would be valuable research tools. We have generated a panel of small-molecule P-Rex inhibitors that target the interface between the catalytic DH domain of P-Rex Rac-GEFs and Rac. Our best-characterized compound, P-Rex inhibitor 1 (PREX-in1), blocks the Rac-GEF activity of full-length P-Rex1 and P-Rex2, and of their isolated catalytic domains, in vitro at low-micromolar concentration, without affecting the activities of several other Rho-GEFs. PREX-in1 blocks the P-Rex1 dependent spreading of PDGF-stimulated endothelial cells and the production of reactive oxygen species in fMLP-stimulated mouse neutrophils. Structure-function analysis revealed critical structural elements of PREX-in1, allowing us to develop derivatives with increased efficacy, the best with an IC50 of 2 \\u00b5M. In summary, we have developed PREX-in1 and derivative small-molecule compounds that will be useful laboratory research tools for the study of P-Rex function. These compounds may also be a good starting point for the future development of more sophisticated drug-like inhibitors aimed at targeting P-Rex Rac-GEFs in cancer.\", \"doi_y\": \"https://doi.org/10.1080/21541248.2022.2131313\", \"publication_date\": \"2022-11-07\"}, {\"work_id\": \"W4285803545\", \"doi_x\": \"10.1101/2022.07.17.500290\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The snapdragon genomes reveal the evolutionary dynamics of the<i>S</i>locus supergene\", \"abstract\": \"Abstract The multi-allelic S -locus, containing a pistil S-RNase and dozens of S -locus F-box ( SLF ), underlies genetic control of self-incompatibility (SI) in Antirrhinum hispanicum . The genus Antirrhinum , harboring such a SI system has been used as a model to study self-incompatibility extensively. However, there have been limited studies on the genomic organization of the S -locus supergene due to a lack of high-quality genomic data. Here, we present the chromosome-level reference and haplotype-resolved genome assemblies of a self-incompatible Antirrhinum hispanicum line, AhS 7 S 8 . Alongside with the draft genome of Misopates orontium , comparative genomics reveals that A.hispanicum diverged from its self-compatible cousin 12.3 million years ago (Mya). Expanded gene families enriched in distinct functional terms implied different evolutionary trajectories of outcrossing and selfing species. For the first time, two complete A.hispanicum S -haplotypes spanning \\u223c1.2Mb and containing 32 SLFs were reconstructed, while most of the SLFs derived from retroelement-mediated proximal or tandem duplication approximately 122 Mya. Moreover, we detected a candidate cis -transcription factor associated with regulating SLF s expression, and two miRNAs may control the expression of this transcription factor. Inter-specific S -locus and intra-specific S -haplotype comparisons revealed the dynamic nature and polymorphism of the S -locus supergene mediated by continuous gene duplication, segmental translocation or loss, and TE-involved transposition events. Our data provides an excellent resource for future research on the evolutionary studies on S-RNase -based self-incompatibility system.\", \"doi_y\": \"https://doi.org/10.1101/2022.07.17.500290\", \"publication_date\": \"2022-07-18\"}, {\"work_id\": \"W4378176816\", \"doi_x\": \"10.1371/journal.ppat.1011037\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protective human IgE responses are promoted by comparable life-cycle dependent Tegument Allergen-Like expression in Schistosoma haematobium and Schistosoma mansoni infection\", \"abstract\": \"Schistosoma haematobium is the most prevalent of the human-infecting schistosome species, causing significant morbidity in endemically exposed populations. Despite this, it has been relatively understudied compared to its fellow species, S. mansoni. Here we provide the first comprehensive characterization of the S. haematobium Tegument Allergen-Like protein family, a key protein family directly linked to protective immunity in S. mansoni infection. Comparable with observations for S. mansoni, parasite phylogenetic analysis and relative gene expression combined with host serological analysis support a cross-reactive relationship between S. haematobium TAL proteins, exposed to the host immune system as adult worms die, and closely related proteins, exposed during penetration by the infecting cercarial and early schistosomulae stages. Specifically, our results strengthen the evidence for host immunity driven by cross-reactivity between family members TAL3 and TAL5, establishing it for the first time for S. haematobium infection. Furthermore, we build upon this relationship to include the involvement of an additional member of the TAL protein family, TAL11 for both schistosome species. Finally, we show a close association between experience of infection and intensity of transmission and the development of protective IgE responses to these antigens, thus improving our knowledge of the mechanisms by which protective host immune responses develop. This knowledge will be critical in understanding how control efforts such as mass drug administration campaigns influence the development of host immunity and subsequent patterns of infection and disease within endemic populations.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011037\", \"publication_date\": \"2023-05-25\"}, {\"work_id\": \"W4308101411\", \"doi_x\": \"10.1101/2022.11.01.514678\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA-PKcs is required for cGAS/STING-dependent viral DNA sensing in human cells\", \"abstract\": \"Abstract To mount an efficient interferon response to virus infection, intracellular pattern recognition receptors (PRRs) sense viral nucleic acids and activate anti-viral gene transcription. The mechanisms by which intracellular DNA and DNA viruses are sensed are relevant not only to antiviral innate immunity, but also to autoinflammation and anti-tumour immunity through initiation of sterile inflammation by self-DNA recognition. The PRRs that directly sense and respond to viral or damaged self-DNA function by signalling to activate interferon regulatory factor (IRF)-dependent type one interferon (IFN-I) transcription. We and others have previously defined DNA-dependent protein kinase (DNA-PK) as an essential component of the DNA-dependent antiviral innate immune system. Here, we show that DNA-PK is essential for STING-dependent IFN-I responses in human cells during stimulation with exogenous DNA and infection with DNA viruses.\", \"doi_y\": \"https://doi.org/10.1101/2022.11.01.514678\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W3215814963\", \"doi_x\": \"10.1098/rstb.2020.0460\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Introduction: the secret lives of microbial mobile genetic elements\", \"abstract\": \"Open AccessMoreSectionsView PDF ToolsAdd to favoritesDownload CitationsTrack Citations ShareShare onFacebookTwitterLinked InRedditEmail Cite this article Hall James P. J., Harrison Ellie and Baltrus David A. 2022Introduction: the secret lives of microbial mobile genetic elementsPhil. Trans. R. Soc. B3772020046020200460http://doi.org/10.1098/rstb.2020.0460SectionOpen AccessIntroductionIntroduction: the secret lives of microbial mobile genetic elements James P. J. Hall James P. J. Hall http://orcid.org/0000-0002-4896-4592 Department of Evolution, Ecology and Behaviour, Institute of Infection, Veterinary and Ecological Sciences, University of Liverpool, Liverpool L69 7ZB, UK [email protected] Google Scholar Find this author on PubMed Search for more papers by this author , Ellie Harrison Ellie Harrison http://orcid.org/0000-0002-2050-4631 Department of Animal Plant Sciences, University of Sheffield, Western Bank, Sheffield S10 1EA, UK Google Scholar Find this author on PubMed Search for more papers by this author and David A. Baltrus David A. Baltrus http://orcid.org/0000-0002-5166-9551 School of Plant Sciences, University of Arizona, Tucson, AZ 85721\\u20100036, USA Google Scholar Find this author on PubMed Search for more papers by this author James P. J. Hall James P. J. Hall http://orcid.org/0000-0002-4896-4592 Department of Evolution, Ecology and Behaviour, Institute of Infection, Veterinary and Ecological Sciences, University of Liverpool, Liverpool L69 7ZB, UK [email protected] Google Scholar Find this author on PubMed Search for more papers by this author , Ellie Harrison Ellie Harrison http://orcid.org/0000-0002-2050-4631 Department of Animal Plant Sciences, University of Sheffield, Western Bank, Sheffield S10 1EA, UK Google Scholar Find this author on PubMed Search for more papers by this author and David A. Baltrus David A. Baltrus http://orcid.org/0000-0002-5166-9551 School of Plant Sciences, University of Arizona, Tucson, AZ 85721\\u20100036, USA Google Scholar Find this author on PubMed Search for more papers by this author Published:29 November 2021https://doi.org/10.1098/rstb.2020.04601. Introduction\\u2018So, Nat'ralists observe, a FleaHath smaller Fleas that on him prey,And these have smaller yet to bite 'em,And so proceed ad infinitum\\u2019\\u223c Jonathan Swift (1733; see http://www.online-literature.com/swift/3515/) Early microscopists, training their lenses on samples, observed individual microbes, separated from one another by membranes and cell walls and dividing by binary fission. These observations developed into an understanding of microbial diversity and ecology that, by and large, placed cellular organisms at the centre. How would our perspective of microbiology be different, had those seventeenth century glass grinders invented the genome sequencer instead? A quarter century of using genomics to examine microbial genomes has illuminated a world rife with mobile genetic elements (MGEs): entities that have evolved to persist and replicate through adaptations that move DNA. The roll call of MGEs is long, diverse and growing [1]. Some MGEs, like transposons and insertion sequences, move DNA between locations within a cell. Others, like conjugative elements and bacteriophages, move DNA between cells. Many MGEs are mosaic or modular in their structure, enabling coalitions of different functions and defying straightforward classification [2]. Regardless, it is clear that MGEs can be both powerful and elusive. Unlike the microbes that carry them, it is difficult to meaningfully visualize an MGE, since in most cases, MGEs are essentially strands of DNA, nested within genomes. But as the principal cause of horizontal gene transfer (HGT)\\u2014a major mode of microbial evolution in which individual microbes get genetic information from sources other than their parents [3]\\u2014MGEs have a huge impact on adaptation and genome structure [4]. Though we have known about MGEs for a long time [5], it is only in recent years, with the explosion of relatively inexpensive sequencing, that we have begun to understand their unique contribution to microbial evolution.The effective invisibility of MGEs has meant that they have often been conceptualized as traits or properties of their microbial hosts. Many MGEs indeed replicate along with their host cells, and can provide benefits that improve host fitness. But MGEs also have an alternative route to success, because the DNA moved by MGEs often encodes the MGE itself, providing an alternative opportunity for MGE replication that is not necessarily shared with other genes in a cell. As a consequence, these autonomous MGEs experience selection occurring at multiple levels [6]. The resulting pressures may not align with those acting on other regions of the genome, and can instead favour adaptations that can generate intragenomic conflicts [7]. For example, traits that promote selfish MGE transmission to other cells can impose a burden on the current host [8,9]. The evolutionary prospects of an autonomous MGE are thus not confined by the borders of the cell, and instead incorporate transmission across hosts and genetic backgrounds.One defining feature of living things is their ability to evolve adaptations to enhance reproductive success and fitness. While not alive in any conventional sense, MGEs, with their own interests and adaptive trajectories transcending those of their hosts, might well be considered to have \\u2018lives\\u2019, hidden within the genomic habitat of their host cells. This perspective\\u2014viewing MGEs as evolving, self-interested, semi-autonomous actors in their own right\\u2014challenges the implicit coherence of the microbial individual, instead revealing genomes to be a contested space of competition and collaboration [10]. What are the implications for understanding MGEs, horizontal gene transfer and microbial evolution?2. In the driving seat: MGEs as vehicles for HGTMGEs are powerful drivers of HGT, and can have far-reaching effects that extend across microbial communities and beyond, to affect animals, plants, human health and disease, industry and the wider environment. Many ecologically-, clinically- and economically important traits are harboured and transmitted by MGEs, ranging from virulence, to resistance, to bioremediation, to metabolism [11\\u201315]. Indeed, the ability to confer beneficial traits on their hosts is the most conspicuous feature of many MGEs. The effects can be clearly seen in the emerging crisis of antimicrobial resistance, which, to a large extent, is driven by MGEs mobilizing antibiotic resistance genes (ARGs) across species and genera [16]. To understand the propensity for ARG transfer, Wang et al. [17] studied the distribution of ARGs across plasmids and chromosomes in three groups of Enterobacteriaceae: Escherichia, Klebsiella and Salmonella. Certain types of resistance gene were more likely to be transmitted by plasmids than others. In particular, genes that caused antibiotic inactivation, or replacement or protection of the antibiotic's target, tended to be more common on plasmids than chromosomes, while resistance genes associated with efflux pumps tended to be more often chromosomally encoded. These patterns reflect broader trends in the susceptibility of genes to undergo HGT, with transfer of genes encoding proteins with multiple interaction partners (such as efflux pumps) proving less successfully transmitted than genes with fewer interactions (such as antibiotic inactivation) [18,19]. The recent global spread of ARGs is perhaps a case study, representative of broader patterns of a general, ubiquitous, ancient process of MGE-mediated gene exchange.While MGEs can spread useful genes, viewing the crucial benefits provided to microbial hosts from the perspective of the MGE itself reveals details and discrepancies that are fundamental to understanding how MGEs cause microbes to adapt. The nitrogen-fixing symbiosis genes of Rhizobia provide an instructive example. In their review on the subject, Wardell et al. [20] explain that the complex, multi-component and exquisitely coevolved symbiosis trait is encoded not as part of the core rhizobial genome but is instead located on mobile plasmids or integrative conjugative elements\\u2014imposing a fitness cost and impeding co-adaptation with the rest of the genome. Yet this phenomenon makes sense when considering the interests of the MGE and the symbiosis genes it carries: symbiosis mobility enables MGEs to take advantage of patchy selection in the heterogeneous soil environment, to sample different genomic environments, and facilitate competition between different symbiosis elements within the intracellular community.Taking an MGE's-eye-view to HGT also helps to explain the wide ranging changes to bacterial phenotypes resulting from plasmid acquisition. The accessory genes carried by plasmids can confer distinct, adaptive traits, but often the effects of plasmids extend much further, to significantly influence the expression of diverse resident genes. Billane et al. [21] argue that, rather than being maladaptive side-effects, these changes could be considered a manipulation of chromosomal genes to ultimately serve plasmid inclusive fitness. Drawing on diverse examples, including plasmids that rewire core metabolism, promote bacterial virulence, shut down interference competition mechanisms, or trigger biofilm production or motility, they show how plasmid-induced changes in bacterial phenotypes can ultimately favour plasmids, either through vertical or horizontal transmission.Overall then, though MGEs do act as vehicles of HGT for adaptive traits, they are self-driving autonomous vehicles. The evolutionary destination to which they are being pushed by selection may not be the same as that of their host cells [22]. Instead, MGEs are better conceptualized as fickle symbionts, existing on a continuum, with interactions that range from mutualistic to antagonistic depending on the genetic and environmental contexts [23].3. Contested spaces: conflicts between MGEs and resident genesMGEs have an intrinsic cost. Under some circumstances, these costs can be outweighed by the benefits of carrying the element, for example, when plasmids harbour accessory genes that are under positive selection [24]. But in most cases, such cargo remains beneficial if it happens to recombine onto the chromosome, leaving a redundant and burdensome MGE [25]. Understanding the source of MGE fitness costs is a major theme in current research. It has long been supposed that the metabolic costs associated with plasmid gene expression, and in particular, translation, make the principal contribution to observed plasmid fitness costs, because of the bioenergetic costs of amino acid biosynthesis and the fact that proteins tend to be more abundant than their corresponding DNA and transcripts [26]. Rodr\\u00edguez-Beltr\\u00e1n and colleagues [27] addressed the role of translation directly, by examining how plasmid fitness costs are affected in cells with translation defects. If translation were the dominant reason for plasmid costs, then cells with either reduced ribosome elongation rates or fewer ribosomes should suffer exacerbated fitness costs when they are inhabited by a plasmid. But this was not the case, and in fact, impeding translation appeared in some cases to reduce plasmid costs. These results suggest that generic mechanisms are less important for plasmid fitness costs than expected, and indicate instead that specific gene conflicts may be the predominant cause, a result that will probably generalize to other types of MGE.The role of specific genes in mediating the negative effects of plasmids is also supported by a study by Smith and colleagues [28]. Here, the authors examine a curious phenomenon that occurs when Pseudomonas species acquire the huge (976 kb) P. syringae plasmid pMPPla107: plasmid-carrying cells become sensitive to an unknown factor secreted by competitors during laboratory growth conditions [29]. The evolutionary reason for this sensitivity is unknown, but it can be abolished by evolution occurring in the laboratory, through a single nonsynonymous substitution in the hypothetical plasmid protein SkaA. It is not clear whether or how the wild-type version of skaA benefits the plasmid, but the gene appears to be essential for plasmid replication. Regardless, the fact that single basepair mutations can have such dramatic effects on costly plasmid-conferred phenotypes speaks to the importance of specific gene functions, and gene interactions within the context of a host cell [18,30]. With large MGEs carrying huge numbers of genes of unknown function and activity, there is great potential for conflicts to arise, and likewise for new adaptive phenotypes to emerge.Compensatory evolution can occur to resolve MGE-associated fitness costs [8]. If such mutations occur to chromosomal genes, then microbes that have compensated otherwise costly transmissible MGEs potentially have a double advantage: First, as the costs of MGE carriage are reduced, compensated cells benefit from enhanced absolute growth rates. And second, transmission of the costly MGE into uncompensated neighbours can increase growth of the donor strain relative to those competitors. In this case, the MGE would effectively act as an agent of \\u2018spiteful\\u2019 interference competition [31], a phenomenon explored by Domingues et al. [32]. Using a theoretical and individual-based modelling framework, they show that compensated plasmid donor cells may indeed use plasmids as weapons during competition within structured communities. In this context, it is interesting to consider that MGEs have provided the raw material for the evolution of other mechanisms of interference competition, including contact-dependent (type IV secretion) and contact-independent (tailocin) toxins [33,34].The burden imposed by MGEs drives the emergence of genome defences, which target horizontally-acquired DNA to prevent its establishment [35]. CRISPR-Cas adaptive immunity is a common defence system in bacteria, and is a particularly rich source of information because CRISPR arrays present specific sequences matching their targets, often identifying the antagonising MGE [36]. Pursey et al. [37] interrogated a large genomic dataset to investigate whether, in repelling incoming MGEs, bacteria encoding CRISPR were also cutting themselves off from the flow of genetic information via HGT. Overall, they identified a negative association between ARGs and CRISPR-Cas, consistent with a model in which genome defences target vectors of ARGs, but in environments where antimicrobial resistance is beneficial, cells compromise their defences and lose CRISPR-Cas to gain the survival benefits of resistance. It could be that recent strong antibiotic selection has thus rendered many genomes prone to MGEs\\u2014an \\u2018immunocompromised state\\u2019 of which MGEs have taken advantage, undergoing rampant transmission, shaping and reshaping microbial genomes in their wake [38]. By opening up genomes to MGEs, anthropogenic antibiotic use may have radical effects on microbial genome evolution beyond simply promoting antimicrobial resistance.The power of MGEs to drive large-scale genome evolution is explored in the context of transposable elements (TEs) by van Dijk et al. [39] in a series of individual-based models that investigate the relationship between TEs and genome streamlining. TEs can wreck their host cells if they insert into an essential gene, and thus highly streamlined genomes are exceptionally susceptible to damaging TE activity. This susceptibility is bad for individual cells, but tends to purge populations of TEs, because TEs also lose out if their host microbe is killed. On the other hand, genomes with redundancy, for example with gene duplications, provide a safer habitat for TEs, because transposition is less likely to disrupt essential genome functions. The predicted consequences are a rock\\u2013paper\\u2013scissors dynamic that can explain long-term associations between TEs and their hosts, as well as broader patterns in genome evolution and compactness.The conflicts between MGEs and resident microbial genes thus extend beyond the generic metabolic burdens of MGE gene expression, and instead emerge from particular functions and evolutionary trade-offs. Characterizing these conflicts, and how they are resolved, will help explain patterns of gene acquisition and loss, and identify key species responsible for harbouring and disseminating MGEs in communities.4. MGEs in an MGE worldAn MGE is seldom alone. Indeed, MGEs live in a world inhabited by other MGEs, and the genome might well be envisaged as an ecosystem in itself, with many co-existing and interacting agents. Many plasmids, for example, inhabit cells with at least one other plasmid [40], and there are diverse mechanisms by which plasmids can interact to affect conjugation, co-infection and fitness costs [41]. The implication of such plasmid \\u2018co-infection\\u2019 is formally investigated in a framework presented by Igler et al. [42]. A salient feature of their models is the emergence of frequency-dependent selection, which varies in nature from positive to negative depending on epistasis between the plasmids. Where plasmids reduce one anothers' costs, the co-infected state is favoured, causing rare plasmids to spread, whereas if plasmids exacerbate one anothers\\u2019 costs then rare plasmids are suppressed by dominant ones. The fact that the presence of other plasmids in a community can dramatically affect plasmid fate underlines the importance of considering the effects of a plasmid in the context of other MGEs.Conflicts and collaborations between MGEs also extend across MGE types. In a series of experiments extended and explored with deterministic modelling, Igler et al. [43] investigate how conjugative plasmids and integrative prophages affect each other's transmission. Though neither the plasmid nor the phage used in their experiments (lambda and RP4) contained systems known to directly interfere with transfer, they found that prophages limited conjugative plasmid spread directly by killing recipients, and suggest that prophages may also inhibit plasmid entry. By contrast, in environments with high rates of phage infection, plasmids can benefit from the superinfection immunity provided by a prophage, and are likely to evolve higher conjugation rates in response. In diverse ecosystems, phage infection drives microbial ecology [15], and this context is likely to shape the activity and behaviour of many MGEs, both directly and indirectly.Indeed, some MGEs have adapted to parasitize phage transmission processes for their own advantage. Satellites are unable to package and transmit themselves, and instead hijack the machinery of phages [44]. P4-like satellites parasitize P2-like phages in Enterobacterales, and de Sousa et al. [45] perform a comprehensive overview of the abundance and genetic diversity of these distinct, broadly distributed, and ancient family of MGEs, related neither to phages nor to plasmids. They identify P4-like elements in almost a third of Enterobacterales genomes, and show phylogenetically that these elements have proliferated by HGT across species. Hyperparasitism\\u2014being parasitic on a parasite\\u2014is a major emerging theme in MGE biology, with Staphylococcus pathogenicity islands (SaPIs), phage-like elements (PLE) and phage-inducible chromsomal islands (PICI) also manifesting this strategy [46]. In some cases, MGE hyperparasites offer powerful evolutionary opportunities to their microbial hosts by providing some defence against a heavy burden of phage infection [47]. Viewing this relationship from the perspective of the hyperparasite reveals conflicting evolutionary pressures as these semi-autonomous, but heavily dependent, elements co-evolve with their host bacteriophage and their broader genomic milieu.Pervasive MGE\\u2013MGE competition can also cast MGE accessory traits in a new light. Virus\\u2013host mutualism, in which viruses confer beneficial traits on their infected hosts, is widespread [48]. One common mechanism, found across bacteria, is where phages encode toxins that facilitate inter-strain interference competition [49]. Sulfolobus islandicus, a hyperthermophilic Archaeal denizen of volcanic hot springs, can be chronically infected with the virus Sulfolobus spindle-shaped virus 9 (SSV9). Chronic infection imposes only a small fitness cost, because SSV9 reproduces by budding from host cells, without lysis. In fact, in communities, SSV9 provides a competitive advantage over uninfected strains [50]. DeWerff et al. [51] demonstrate that this advantage is due to a virus-encoded specific toxin protein that kills competitors. From the perspective of the virus, killing potential hosts seems counterintuitive, but experiments combined with comparative genomics revealed that related spindle-shaped viruses carry different toxin variants, and it was the carriage of a heterologous virus that rendered strains vulnerable to a toxin. This complex network of cross-targeting makes sense in the context of virus\\u2013virus competition, with the host cells caught in the cross-fire.Together, the understanding that MGEs are adapted for interactions with other MGEs, just as much as (if not more than) for dealing with core chromosomal genes, challenges the conception of microbial individuals as the principal agents adapting in microbiomes. Instead, these findings provoke a more pluralistic model, in which chromosomes and MGEs are nodes in an interacting, coevolving network.5. The diversity of the MGE menagerieBy providing a window into the world of MGEs, genome sequencing has cast light on hidden diversity that would otherwise have remained invisible [52]. Santamar\\u00eda et al. [53] focus their attention on the phages of Rhizobium species, which remained poorly understood, despite the crucial ecological and economic importance of this nitrogen-fixing genus. Comparative genomics revealed dozens of viral clusters and the widespread presence of prophage in Rhizobium across species and geographic locations, indicating that some phages have wide distribution and host range while others remained more tightly associated. Interestingly, only a small proportion of Rhizobium prophages were intact, suggesting that within genomes, prophage are subjected to an ongoing process of degradation. Selection at the level of the microbial host often disfavours MGE independence, driving mutations that prevent MGE transmission (such as the loss of genes for packaging) but retaining genes for the benefit of the host [33,54]. As a consequence, such \\u2018domesticated\\u2019 prophage can be shorter than known-to-be active phage. Pattenden and colleagues [55] developed a model that combines genome sequencing data with measurements of growth characteristics, to investigate how the life history traits of host bacteria influence the length and gene content of putative prophages. While fast-growing hosts displayed a clear bimodal distribution in prophage lengths, slower-growing hosts, and those unlikely to be pathogenic, were relatively depleted in the longer, intact prophage and exhibited a lower rate of prophage induction. These properties are likely to reflect the relative stability of those slower-growing strains, which are less likely to experience boom-and-bust growth cycles and the associated stresses. Such patterns illustrate the dynamic evolution of MGEs in the context of organismal genomes, demonstrating how varied selection pressures can dissociate self-reproducing MGEs into constituent modules.Advances in long-read sequencing have also enabled larger, repetitive MGEs to be completely assembled and studied, and recent years have seen an increase in the identification of super-sized MGEs, including megaplasmids, which in some cases represent megabases of DNA and harbour hundreds of genes [56,57]. With plasmid size spanning three orders of magnitude, what factors drive megaplasmids to become so large, and what are the consequences for microbial genome evolution? In their review of the field, Hall et al. [58] suggest that while there is no meaningful size threshold for assigning megaplasmid status, distinct selective pressures can favour and stabilize larger plasmids and their magnified capacity for HGT.Comparative genomics is likewise uncovering the opportunism and mosaicism that have emerged as a defining feature of MGE evolution. The plasmids of the agrobacteria\\u2013rhizobia complex provide a graphic example. Analysing over 4000 plasmid sequences from this agriculturally-, biotechnologically- and ecologically-important group, Weisberg et al. [59] focus on the evolution of the characteristic oncogenic plasmids that are associated with plant disease, finding pervasive hallmarks of recombination and reshuffling that have generated plasmids with new combinations of genes, able to confer pathogenicity to new lineages. In particular, the underexplored accessory plasmids of the complex, which are thought to mainly confer catabolic traits, appear to have contributed key regions and genes, accelerating diversification and extending the functions of virulence plasmids.The diversity of MGEs uncovered to date is likely to represent just the tip of the iceberg, as technological developments, including long read sequencing, contact sequencing and metagenome assemblies continue to reveal and define the biology of new elements [52,57,60,61]. There is surely much to be discovered, particularly from environmental microbes and those yet to be cultured.6. New perspectives on microbial identitiesExposing the secret lives of MGEs, through sequencing, experiments and modelling, is beginning to force a change in the perspective that has dominated microbiology since those first observations of individual microbes hundreds of years ago. Viewing MGEs as more than simply extensions of their host microbes, and instead as adapting, fitness-maximizing agents in their own right, enables us to draw on a wider range of conceptual frameworks for understanding their behaviour. In a thought-provoking opinion piece, Ghaly et al. [62] provide three complementary viewpoints for understanding and controlling MGEs in the context of antimicrobial resistance. MGEs can be conceptualized as biological individuals, as pollutants, and as invasive species, with each of these presentations inviting distinct interventions. For example, by viewing MGEs as biological individuals, attempts could be made to target MGE replication specifically through use of conjugation inhibitors or CRISPR, reducing the collateral damage to their host cells. As pollutants, MGEs could be targeted by adapting methods of wastewater treatment. And as invasive species, we might use tactics used in larger-organism ecology to make communities more stable against invasion. Though focused on controlling the spread of resistance, strategies drawing on such alternative perspectives are more widely applicable, for example, because influencing and controlling MGE activity could facilitate bioaugmentation of microbiomes through introduction of beneficial traits [11].The realization, stemming from those original microscopic studies, that we live in a microbial world pervaded by tiny lives, is a powerful one, and continues to shape our understanding of health, disease and ecosystem function. We believe that understanding MGEs on their own terms, and uncovering their \\u2018secret lives\\u2019 in the process, will ultimately spur new ways of studying, influencing, and comprehending microbiomes, and to advance this perspective, we are proud to present this Theme Issue.Data accessibilityThis article has no additional data.Authors' contributionsAll authors contributed to the final manuscript.Competing interestsWe declare we have no competing interests.FundingE.H. is supported by the RCUK Natural Environment Research Council (NERC) via an independent research fellowship (NE/P017584/1).AcknowledgementsWe are grateful to all authors for their contributions, and to Editor Helen Eaton for support and guidance during the whole process.Authors' profileJames P. J. Hall is a Tenure Track Fellow in the Department of Evolution, Ecology and Behaviour at the University of Liverpool. Having studied evolution in eukaryotes, viruses and bacteria, he became captivated by the multifarious relationships that MGEs have with one another and with their host microorganisms, and his current focus is on the evolutionary ecology of plasmid-mediated HGT in microbial communities.Ellie Harrison is a Tenure Track NERC Fellow in the School of Biosciences at the University of Sheffield. She is interested in the forces that shape microbial communities, often addressing this question at the sub-cellular level, focused on the menagerie of MGEs that exist within and between bacterial genomes. At present, her research focuses on MGEs in Rhizobia: the bacteria that (sometimes) live in the roots of legumes and provide them with lovely nitrogen.David A. Baltrus is an Associate Professor in the School of Plant Sciences at the University of Arizona. His current research broadly focuses on interactions between microbes and (many) other organisms. While the pervasive nature and evolutionary dynamics of MGEs across all these interactions often makes no sense, trying to grasp their importance certainly compels him.FootnotesOne contribution of 18 to a theme issue \\u2018The secret lives of microbial mobile genetic elements\\u2019.\\u00a9 2021 The Authors.Published by the Royal Society under the terms of the Creative Commons Attribution License http://creativecommons.org/licenses/by/4.0/, which permits unrestricted use, provided the original author and source are credited.References1. Chandler M, Gellert M, Lambowitz AM, Rice PA, Sandmeyer SB. 2020Mobile DNA III. Washington, DC: American Society for Microbiology Press. Google Scholar2. Osborn AM, B\\u00f6ltner D. 2002\", \"doi_y\": \"https://doi.org/10.1098/rstb.2020.0460\", \"publication_date\": \"2021-11-29\"}, {\"work_id\": \"W4307843054\", \"doi_x\": \"10.1093/nar/gkac958\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Ensembl 2023\", \"abstract\": \"Abstract Ensembl (https://www.ensembl.org) has produced high-quality genomic resources for vertebrates and model organisms for more than twenty years. During that time, our resources, services and tools have continually evolved in line with both the publicly available genome data and the downstream research and applications that utilise the Ensembl platform. In recent years we have witnessed a dramatic shift in the genomic landscape. There has been a large increase in the number of high-quality reference genomes through global biodiversity initiatives. In parallel, there have been major advances towards pangenome representations of higher species, where many alternative genome assemblies representing different breeds, cultivars, strains and haplotypes are now available. In order to support these efforts and accelerate downstream research, it is our goal at Ensembl to create high-quality annotations, tools and services for species across the tree of life. Here, we report our resources for popular reference genomes, the dramatic growth of our annotations (including haplotypes from the first human pangenome graphs), updates to the Ensembl Variant Effect Predictor (VEP), interactive protein structure predictions from AlphaFold DB, and the beta release of our new website.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac958\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4296712231\", \"doi_x\": \"10.12688/f1000research.122987.1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Improving the predictive power of xenograft and syngeneic anti-tumour studies using mice humanised for pathways of drug metabolism\", \"abstract\": \"Drug development is an expensive and time-consuming process, with only a small fraction of drugs gaining regulatory approval from the often many thousands of candidates identified during target validation. Once a lead compound has been identified and optimised, they are subject to intensive pre-clinical research to determine their pharmacodynamic, pharmacokinetic and toxicological properties, procedures which inevitably involve significant numbers of animals - mainly mice and rats, but also dogs and monkeys in much smaller numbers and for specific types of drug candidates. Many compounds that emerge from this process, having been shown to be safe and efficacious in pre-clinical studies, subsequently fail to replicate this outcome in clinical trials, therefore wasting time, money and, most importantly, animals. The poor predictive power of animal models in pre-clinical studies is predominantly due to lack of efficacy or safety reasons, which in turn can be attributed mainly to the significant species differences in drug metabolism between humans and animals. To circumvent this, we have developed a complex transgenic mouse model - 8HUM - which faithfully replicates human Phase I drug metabolism (and its regulation), and which will generate more human-relevant data [ REFINEMENT] from fewer animals [ REDUCTION] in a pre-clinical setting and reduce attrition in the clinic. One key area for the pre-clinical application of animals in an oncology setting - almost exclusively mice - is their use in anti-tumour studies. We now further demonstrate the utility of the 8HUM mouse using a murine melanoma cell line as a syngeneic tumour and also present an immunodeficient version 8HUM_Rag2 -/- - for use in xenograft studies. These models will be of significant benefit not only to Pharma for pre-clinical drug development work, but also throughout the drug efficacy, toxicology, pharmacology, and drug metabolism communities, where fewer animals will be needed to generate more human-relevant data.\", \"doi_y\": \"https://doi.org/10.12688/f1000research.122987.1\", \"publication_date\": \"2022-09-21\"}, {\"work_id\": \"W4381469587\", \"doi_x\": \"10.1021/jacs.3c02742\", \"af_direct_citation\": 1, \"af_indirect_citations\": 7, \"title\": \"Design of Heme Enzymes with a Tunable Substrate Binding Pocket Adjacent to an Open Metal Coordination Site\", \"abstract\": \"The catalytic versatility of pentacoordinated iron is highlighted by the broad range of natural and engineered activities of heme enzymes such as cytochrome P450s, which position a porphyrin cofactor coordinating a central iron atom below an open substrate binding pocket. This catalytic prowess has inspired efforts to design de novo helical bundle scaffolds that bind porphyrin cofactors. However, such designs lack the large open substrate binding pocket of P450s, and hence, the range of chemical transformations accessible is limited. Here, with the goal of combining the advantages of the P450 catalytic site geometry with the almost unlimited customizability of de novo protein design, we design a high-affinity heme-binding protein, dnHEM1, with an axial histidine ligand, a vacant coordination site for generating reactive intermediates, and a tunable distal pocket for substrate binding. A 1.6 \\u00c5 X-ray crystal structure of dnHEM1 reveals excellent agreement to the design model with key features programmed as intended. The incorporation of distal pocket substitutions converted dnHEM1 into a proficient peroxidase with a stable neutral ferryl intermediate. In parallel, dnHEM1 was redesigned to generate enantiocomplementary carbene transferases for styrene cyclopropanation (up to 93% isolated yield, 5000 turnovers, 97:3 e.r.) by reconfiguring the distal pocket to accommodate calculated transition state models. Our approach now enables the custom design of enzymes containing cofactors adjacent to binding pockets with an almost unlimited variety of shapes and functionalities.\", \"doi_y\": \"https://doi.org/10.1021/jacs.3c02742\", \"publication_date\": \"2023-06-21\"}, {\"work_id\": \"W4313424363\", \"doi_x\": \"10.15252/embj.2022111484\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"abstract\": \"Plant pathogens compromise crop yields. Plants have evolved robust innate immunity that depends in part on intracellular Nucleotide-binding, Leucine rich-Repeat (NLR) immune receptors that activate defense responses upon detection of pathogen-derived effectors. Most \\\"sensor\\\" NLRs that detect effectors require the activity of \\\"helper\\\" NLRs, but how helper NLRs support sensor NLR function is poorly understood. Many Solanaceae NLRs require NRC (NLR-Required for Cell death) class of helper NLRs. We show here that Rpi-amr3, a sensor NLR from Solanum americanum, detects AVRamr3 from the potato late blight pathogen, Phytophthora infestans, and activates oligomerization of helper NLRs NRC2 and NRC4 into high-molecular-weight resistosomes. In contrast, recognition of P. infestans effector AVRamr1 by another sensor NLR Rpi-amr1 induces formation of only the NRC2 resistosome. The activated NRC2 oligomer becomes enriched in membrane fractions. ATP-binding motifs of both Rpi-amr3 and NRC2 are required for NRC2 resistosome formation, but not for the interaction of Rpi-amr3 with its cognate effector. NRC2 resistosome can be activated by Rpi-amr3 upon detection of AVRamr3 homologs from other Phytophthora species. Mechanistic understanding of NRC resistosome formation will underpin engineering crops with durable disease resistance.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111484\", \"publication_date\": \"2023-01-02\"}, {\"work_id\": \"W4382139362\", \"doi_x\": \"10.1038/s41477-023-01447-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Next-generation ABACUS biosensors reveal cellular ABA dynamics driving root growth at low aerial humidity\", \"abstract\": \"The plant hormone abscisic acid (ABA) accumulates under abiotic stress to recast water relations and development. To overcome a lack of high-resolution sensitive reporters, we developed ABACUS2s-next-generation F\\u00f6rster resonance energy transfer (FRET) biosensors for ABA with high affinity, signal-to-noise ratio and orthogonality-that reveal endogenous ABA patterns in Arabidopsis thaliana. We mapped stress-induced ABA dynamics in high resolution to reveal the cellular basis for local and systemic ABA functions. At reduced foliar humidity, root cells accumulated ABA in the elongation zone, the site of phloem-transported ABA unloading. Phloem ABA and root ABA signalling were both essential to maintain root growth at low humidity. ABA coordinates a root response to foliar stresses, enabling plants to maintain foraging of deeper soil for water uptake.\", \"doi_y\": \"https://doi.org/10.1038/s41477-023-01447-4\", \"publication_date\": \"2023-06-26\"}, {\"work_id\": \"W4309473730\", \"doi_x\": \"10.1038/s41467-022-34773-w\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cryo-EM structure of the agonist-bound Hsp90-XAP2-AHR cytosolic complex\", \"abstract\": \"The aryl hydrocarbon receptor (AHR) is a ligand-dependent transcription factor that mediates a broad spectrum of (patho)physiological processes in response to numerous substances including pollutants, natural products and metabolites. However, the scarcity of structural data precludes understanding of how AHR is activated by such diverse compounds. Our 2.85 \\u00c5 structure of the human indirubin-bound AHR complex with the chaperone Hsp90 and the co-chaperone XAP2, reported herein, reveals a closed conformation Hsp90 dimer with AHR threaded through its lumen and XAP2 serving as a brace. Importantly, we disclose the long-awaited structure of the AHR PAS-B domain revealing a unique organisation of the ligand-binding pocket and the structural determinants of ligand-binding specificity and promiscuity of the receptor. By providing structural details of the molecular initiating event leading to AHR activation, our study rationalises almost forty years of biochemical data and provides a framework for future mechanistic studies and structure-guided drug design.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-34773-w\", \"publication_date\": \"2022-11-16\"}, {\"work_id\": \"W4379793796\", \"doi_x\": \"10.1021/acs.bioconjchem.3c00122\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Design and Evolution of Enhanced Peptide\\u2013Peptide Ligation for Modular Transglutaminase Assembly\", \"abstract\": \"Robust and precise tools are needed to enhance the functionality and resilience of synthetic nanoarchitectures. Here, we have employed directed evolution and rational design to build a fast-acting molecular superglue from a bacterial adhesion protein. We have generated the SnoopLigase2 coupling system, a genetically encoded route for efficient transamidation between SnoopTag2 and DogTag2 peptides. Each peptide was selected for rapid reaction by phage display screening. The optimized set allows more than 99% completion and is compatible with diverse buffers, pH values, and temperatures, accelerating the reaction over 1000-fold. SnoopLigase2 directs a specific reaction in the mammalian secretory pathway, allowing covalent display on the plasma membrane. Transglutaminase 2 (TG2) has a network of interactions and substrates amidst the mammalian cell surface and extracellular matrix. We expressed a modified TG2 with resistance to oxidative inactivation and minimal self-reactivity. SnoopLigase2 enables TG2 functionalization with transforming growth factor alpha (TGF\\u03b1) in routes that would be impossible through genetic fusion. The TG2:TGF\\u03b1 conjugate retained transamidase activity, stably anchored TGF\\u03b1 for signal activation in the extracellular environment, and reprogrammed cell behavior. This modular toolbox should create new opportunities for molecular assembly, both for novel biomaterials and complex cellular environments.\", \"doi_y\": \"https://doi.org/10.1021/acs.bioconjchem.3c00122\", \"publication_date\": \"2023-06-08\"}, {\"work_id\": \"W4226229183\", \"doi_x\": \"10.1056/nejmoa2119451\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Covid-19 Vaccine Effectiveness against the Omicron (B.1.1.529) Variant\", \"abstract\": \"A rapid increase in coronavirus disease 2019 (Covid-19) cases due to the omicron (B.1.1.529) variant of severe acute respiratory syndrome coronavirus 2 in highly vaccinated populations has aroused concerns about the effectiveness of current vaccines.We used a test-negative case-control design to estimate vaccine effectiveness against symptomatic disease caused by the omicron and delta (B.1.617.2) variants in England. Vaccine effectiveness was calculated after primary immunization with two doses of BNT162b2 (Pfizer-BioNTech), ChAdOx1 nCoV-19 (AstraZeneca), or mRNA-1273 (Moderna) vaccine and after a booster dose of BNT162b2, ChAdOx1 nCoV-19, or mRNA-1273.Between November 27, 2021, and January 12, 2022, a total of 886,774 eligible persons infected with the omicron variant, 204,154 eligible persons infected with the delta variant, and 1,572,621 eligible test-negative controls were identified. At all time points investigated and for all combinations of primary course and booster vaccines, vaccine effectiveness against symptomatic disease was higher for the delta variant than for the omicron variant. No effect against the omicron variant was noted from 20 weeks after two ChAdOx1 nCoV-19 doses, whereas vaccine effectiveness after two BNT162b2 doses was 65.5% (95% confidence interval [CI], 63.9 to 67.0) at 2 to 4 weeks, dropping to 8.8% (95% CI, 7.0 to 10.5) at 25 or more weeks. Among ChAdOx1 nCoV-19 primary course recipients, vaccine effectiveness increased to 62.4% (95% CI, 61.8 to 63.0) at 2 to 4 weeks after a BNT162b2 booster before decreasing to 39.6% (95% CI, 38.0 to 41.1) at 10 or more weeks. Among BNT162b2 primary course recipients, vaccine effectiveness increased to 67.2% (95% CI, 66.5 to 67.8) at 2 to 4 weeks after a BNT162b2 booster before declining to 45.7% (95% CI, 44.7 to 46.7) at 10 or more weeks. Vaccine effectiveness after a ChAdOx1 nCoV-19 primary course increased to 70.1% (95% CI, 69.5 to 70.7) at 2 to 4 weeks after an mRNA-1273 booster and decreased to 60.9% (95% CI, 59.7 to 62.1) at 5 to 9 weeks. After a BNT162b2 primary course, the mRNA-1273 booster increased vaccine effectiveness to 73.9% (95% CI, 73.1 to 74.6) at 2 to 4 weeks; vaccine effectiveness fell to 64.4% (95% CI, 62.6 to 66.1) at 5 to 9 weeks.Primary immunization with two doses of ChAdOx1 nCoV-19 or BNT162b2 vaccine provided limited protection against symptomatic disease caused by the omicron variant. A BNT162b2 or mRNA-1273 booster after either the ChAdOx1 nCoV-19 or BNT162b2 primary course substantially increased protection, but that protection waned over time. (Funded by the U.K. Health Security Agency.).\", \"doi_y\": \"https://doi.org/10.1056/nejmoa2119451\", \"publication_date\": \"2022-04-21\"}, {\"work_id\": \"W4385683583\", \"doi_x\": \"10.1107/s1600576723005344\", \"af_direct_citation\": 1, \"af_indirect_citations\": 12, \"title\": \"AlphaFold-predicted protein structures and small-angle X-ray scattering: insights from an extended examination of selected data in the Small-Angle Scattering Biological Data Bank\", \"abstract\": \"By providing predicted protein structures from nearly all known protein sequences, the artificial intelligence program AlphaFold (AF) is having a major impact on structural biology. While a stunning accuracy has been achieved for many folding units, predicted unstructured regions and the arrangement of potentially flexible linkers connecting structured domains present challenges. Focusing on single-chain structures without prosthetic groups, an earlier comparison of features derived from small-angle X-ray scattering (SAXS) data taken from the Small-Angle Scattering Biological Data Bank (SASBDB) is extended to those calculated using the corresponding AF-predicted structures. Selected SASBDB entries were carefully examined to ensure that they represented data from monodisperse protein solutions and had sufficient statistical precision and q resolution for reliable structural evaluation. Three examples were identified where there is clear evidence that the single AF-predicted structure cannot account for the experimental SAXS data. Instead, excellent agreement is found with ensemble models generated by allowing for flexible linkers between high-confidence predicted structured domains. A pool of representative structures was generated using a Monte Carlo method that adjusts backbone dihedral allowed angles along potentially flexible regions. A fast ensemble modelling method was employed that optimizes the fit of pair distance distribution functions [P(r) versus r] and intensity profiles [I(q) versus q] computed from the pool to their experimental counterparts. These results highlight the complementarity between AF prediction, solution SAXS and molecular dynamics/conformational sampling for structural modelling of proteins having both structured and flexible regions.\", \"doi_y\": \"https://doi.org/10.1107/s1600576723005344\", \"publication_date\": \"2023-07-20\"}, {\"work_id\": \"W3196998750\", \"doi_x\": \"10.1002/prot.26231\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Modeling SARS\\u2010CoV\\u20102 proteins in the CASP\\u2010commons experiment\", \"abstract\": \"Abstract Critical Assessment of Structure Prediction (CASP) is an organization aimed at advancing the state of the art in computing protein structure from sequence. In the spring of 2020, CASP launched a community project to compute the structures of the most structurally challenging proteins coded for in the SARS\\u2010CoV\\u20102 genome. Forty\\u2010seven research groups submitted over 3000 three\\u2010dimensional models and 700 sets of accuracy estimates on 10 proteins. The resulting models were released to the public. CASP community members also worked together to provide estimates of local and global accuracy and identify structure\\u2010based domain boundaries for some proteins. Subsequently, two of these structures (ORF3a and ORF8) have been solved experimentally, allowing assessment of both model quality and the accuracy estimates. Models from the AlphaFold2 group were found to have good agreement with the experimental structures, with main chain GDT_TS accuracy scores ranging from 63 (a correct topology) to 87 (competitive with experiment).\", \"doi_y\": \"https://doi.org/10.1002/prot.26231\", \"publication_date\": \"2021-10-05\"}, {\"work_id\": \"W4321440644\", \"doi_x\": \"10.1126/scisignal.abm7134\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Disruptions in endocytic traffic contribute to the activation of the NLRP3 inflammasome\", \"abstract\": \"Inflammation driven by the NLRP3 inflammasome is coordinated through multiple signaling pathways and is regulated by subcellular organelles. Here, we tested the hypothesis that NLRP3 senses disrupted endosome trafficking to trigger inflammasome formation and inflammatory cytokine secretion. NLRP3-activating stimuli disrupted endosome trafficking and triggered localization of NLRP3 to vesicles positive for endolysosomal markers and for the inositol lipid PI4P. Chemical disruption of endosome trafficking sensitized macrophages to the NLRP3 activator imiquimod, driving enhanced inflammasome activation and cytokine secretion. Together, these data suggest that NLRP3 can sense disruptions in the trafficking of endosomal cargoes, which may explain in part the spatial activation of the NLRP3 inflammasome. These data highlight mechanisms that could be exploited in the therapeutic targeting of NLRP3.\", \"doi_y\": \"https://doi.org/10.1126/scisignal.abm7134\", \"publication_date\": \"2023-02-21\"}, {\"work_id\": \"W4327614608\", \"doi_x\": \"10.1101/2023.03.15.532816\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Intracellular<i>Plasmodium</i>aquaporin 2 is required for sporozoite production in the mosquito vector and malaria transmission\", \"abstract\": \"Abstract Malaria remains a devastating disease and, with current measures failing to control its transmission, there is a need for novel interventions. A family of proteins that have long been pursued as potential intervention targets are aquaporins which are channels facilitating the movement of water and other solutes across membranes. We identify a new aquaporin in malaria parasites and demonstrate that it is essential for disease transmission through mosquitoes. Disruption of AQP2 in the human parasite Plasmodium falciparum and the rodent parasite Plasmodium berghei blocks sporozoite production inside oocysts established on mosquito midguts, preventing parasite infection of salivary glands and transmission to a new host. In vivo epitope tagging of AQP2 in P. berghei , combined with immunofluorescence assays, reveals that the protein is localized in previously uncharacterized organelles found in the cytoplasm of gametocytes, ookinetes and sporozoites. The number of these organelles varies between individual parasites and lifecycle stages suggesting that they are likely part of a dynamic endolysosomal system. Phylogenetic analysis confirms that AQP2 is unique to malaria and closely related parasites and most closely related to other intracellular aquaporins. Structure prediction analyses identify several unusual features, including a large accessory extracellular loop and an arginine-to-phenylalanine substitution in the selectivity filter principally determining pore function, a unique feature not found in any aquaporins studied to date. This in conjunction with the requirement of AQP2 for malaria transmission suggests that AQP2 may be a fruitful new target of novel antimalarial interventions.\", \"doi_y\": \"https://doi.org/10.1101/2023.03.15.532816\", \"publication_date\": \"2023-03-15\"}, {\"work_id\": \"W4386697797\", \"doi_x\": \"10.1088/1748-605x/acf976\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Human vascularised synovium-on-a-chip: a mechanically stimulated, microfluidic model to investigate synovial inflammation and monocyte recruitment\", \"abstract\": \"Healthy synovium is critical for joint homeostasis. Synovial inflammation (synovitis) is implicated in the onset, progression and symptomatic presentation of arthritic joint diseases such as rheumatoid arthritis and osteoarthritis. Thus, the synovium is a promising target for the development of novel, disease-modifying therapeutics. However, target exploration is hampered by a lack of good pre-clinical models that accurately replicate human physiology and that are developed in a way that allows for widespread uptake. The current study presents a multi-channel, microfluidic, organ-on-a-chip (OOAC) model, comprising a 3D configuration of the human synovium and its associated vasculature, with biomechanical and inflammatory stimulation, built upon a commercially available OOAC platform. Healthy human fibroblast-like synoviocytes (hFLS) were co-cultured with human umbilical vein endothelial cells (HUVECs) with appropriate matrix proteins, separated by a flexible, porous membrane. The model was developed within the Emulate organ-chip platform enabling the application of physiological biomechanical stimulation in the form of fluid shear and cyclic tensile strain. The hFLS exhibited characteristic morphology, cytoskeletal architecture and matrix protein deposition. Synovial inflammation was initiated through the addition of interleukin-1\\u03b2(IL-1\\u03b2) into the synovium channel resulting in the increased secretion of inflammatory and catabolic mediators, interleukin-6 (IL-6), prostaglandin E2 (PGE2), matrix metalloproteinase 1 (MMP-1), as well as the synovial fluid constituent protein, hyaluronan. Enhanced expression of the inflammatory marker, intercellular adhesion molecule-1 (ICAM-1), was observed in HUVECs in the vascular channel, accompanied by increased attachment of circulating monocytes. This vascularised human synovium-on-a-chip model recapitulates a number of the functional characteristics of both healthy and inflamed human synovium. Thus, this model offers the first human synovium organ-chip suitable for widespread adoption to understand synovial joint disease mechanisms, permit the identification of novel therapeutic targets and support pre-clinical testing of therapies.\", \"doi_y\": \"https://doi.org/10.1088/1748-605x/acf976\", \"publication_date\": \"2023-10-10\"}, {\"work_id\": \"W4313299198\", \"doi_x\": \"10.1021/jacs.2c10673\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Ultrahigh-Throughput Directed Evolution of a Metal-Free \\u03b1/\\u03b2-Hydrolase with a Cys-His-Asp Triad into an Efficient Phosphotriesterase\", \"abstract\": \"Finding new mechanistic solutions for biocatalytic challenges is key in the evolutionary adaptation of enzymes, as well as in devising new catalysts. The recent release of man-made substances into the environment provides a dynamic testing ground for observing biocatalytic innovation at play. Phosphate triesters, used as pesticides, have only recently been introduced into the environment, where they have no natural counterpart. Enzymes have rapidly evolved to hydrolyze phosphate triesters in response to this challenge, converging onto the same mechanistic solution, which requires bivalent cations as a cofactor for catalysis. In contrast, the previously identified metagenomic promiscuous hydrolase P91, a homologue of acetylcholinesterase, achieves slow phosphotriester hydrolysis mediated by a metal-independent Cys-His-Asp triad. Here, we probe the evolvability of this new catalytic motif by subjecting P91 to directed evolution. By combining a focused library approach with the ultrahigh throughput of droplet microfluidics, we increase P91's activity by a factor of \\u2248360 (to a kcat/KM of \\u22487 \\u00d7 105 M-1 s-1) in only two rounds of evolution, rivaling the catalytic efficiencies of naturally evolved, metal-dependent phosphotriesterases. Unlike its homologue acetylcholinesterase, P91 does not suffer suicide inhibition; instead, fast dephosphorylation rates make the formation of the covalent adduct rather than its hydrolysis rate-limiting. This step is improved by directed evolution, with intermediate formation accelerated by 2 orders of magnitude. Combining focused, combinatorial libraries with the ultrahigh throughput of droplet microfluidics can be leveraged to identify and enhance mechanistic strategies that have not reached high efficiency in nature, resulting in alternative reagents with novel catalytic machineries.\", \"doi_y\": \"https://doi.org/10.1021/jacs.2c10673\", \"publication_date\": \"2022-12-30\"}, {\"work_id\": \"W4310730332\", \"doi_x\": \"10.1088/2399-6528/aca45e\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Deep learning in airborne particulate matter sensing: a review\", \"abstract\": \"Abstract Airborne particulate matter pollution is a global health problem that affects people from all demographics. To reduce the impact of such pollution and enable mitigation and policy planning, quantifying individuals\\u2019 exposure to pollution is necessary. To achieve this, effective monitoring of airborne particulates is required, through monitoring of pollution hotspots and sources. Furthermore, since pollution is a global problem, which varies from urban areas to city centres, industrial facilities to inside homes, a variety of sensors might be needed. Current sensing techniques either lack species resolution on a world scale, lack real-time capabilities, or are too expensive or too large for mass deployment. However, recent work using deep learning techniques has expanded the capability of current sensors and allowed the development of new techniques that have the potential for worldwide, species specific, real-time monitoring. Here, it is proposed how deep learning can enable sensor design for the development of small, low-cost sensors for real-time monitoring of particulate matter pollution, whilst unlocking the capability for predicting future particulate events and health inference from particulates, for both individuals and the environment in general.\", \"doi_y\": \"https://doi.org/10.1088/2399-6528/aca45e\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4213296721\", \"doi_x\": \"10.3389/fmolb.2022.834011\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"abstract\": \"Since pH sensitivity has a fundamental role in biology, much effort has been committed to establishing physical models to rationalize and predict pH dependence from molecular structures. Two of the key challenges are to accurately calculate ionizable group solvation and hydration and then to apply this modeling to all conformations relevant to the process in question. Explicit solvent methods coupled to molecular dynamics simulation are increasingly complementing lower resolution implicit solvent techniques, but equally, the scale of biological data acquisition leaves a role for high-throughput modeling. Additionally, determination of ranges of structures for a system allows sampling of key stages in solvation. In a review of the area, it is emphasized that pH sensors in biology beyond the most obvious candidate (histidine side chain, with an unshifted p K a near neutral pH) should be considered; that modeling can benefit from other concepts in bioinformatics, in particular modulation of interactions and function in families of homologs; and that it can also be beneficial to incorporate as many experimental structures as possible, to mitigate against small variations in conformation and to analyze larger, functional, conformational changes. These aspects are then demonstrated with new work on the spike protein of SARS-CoV-2, looking at the pH dependence of variants, including prediction of a change in the balance of locked, closed, and open forms at neutral pH for the Omicron variant spike protein.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.834011\", \"publication_date\": \"2022-02-18\"}, {\"work_id\": \"W4303518330\", \"doi_x\": \"10.1371/journal.pcbi.1010611\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Understanding molecular mechanisms and predicting phenotypic effects of pathogenic tubulin mutations\", \"abstract\": \"Cells rely heavily on microtubules for several processes, including cell division and molecular trafficking. Mutations in the different tubulin-\\u03b1 and -\\u03b2 proteins that comprise microtubules have been associated with various diseases and are often dominant, sporadic and congenital. While the earliest reported tubulin mutations affect neurodevelopment, mutations are also associated with other disorders such as bleeding disorders and infertility. We performed a systematic survey of tubulin mutations across all isotypes in order to improve our understanding of how they cause disease, and increase our ability to predict their phenotypic effects. Both protein structural analyses and computational variant effect predictors were very limited in their utility for differentiating between pathogenic and benign mutations. This was even worse for those genes associated with non-neurodevelopmental disorders. We selected tubulin-\\u03b1 and -\\u03b2 disease mutations that were most poorly predicted for experimental characterisation. These mutants co-localise to the mitotic spindle in HeLa cells, suggesting they may exert dominant-negative effects by altering microtubule properties. Our results show that tubulin mutations represent a blind spot for current computational approaches, being much more poorly predicted than mutations in most human disease genes. We suggest that this is likely due to their strong association with dominant-negative and gain-of-function mechanisms.\", \"doi_y\": \"https://doi.org/10.1371/journal.pcbi.1010611\", \"publication_date\": \"2022-10-07\"}, {\"work_id\": \"W4289132939\", \"doi_x\": \"10.1101/2022.07.28.500969\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High throughput steady-state enzyme kinetics measured in a parallel droplet generation and absorbance detection platform\", \"abstract\": \"ABSTRACT Microfluidic water-in-oil emulsion droplets are becoming a mainstay of experimental biology, where they replace the classical test tube. In most applications (e.g. in ultrahigh throughput directed evolution) the droplet content is identical for all compartmentalized assay reactions. When emulsion droplets are used for kinetics or other functional assays, though, concentration dependencies (e.g. of initial rates for Michaelis-Menten plots) are required. Droplet-on-demand systems satisfy this need but extracting large amounts of data is challenging. Here we introduce a multiplexed droplet absorbance detector which, coupled to semi-automated droplet generation, forms a tubing-based droplet-on-demand system able to generate and extract quantitative datasets from defined concentration gradients across multiple series of droplets for multiple time points. The emergence of product is detected by reading the absorbance of the droplet sets at multiple, adjustable time points (reversing the flow direction after each detection, so that the droplets pass a line scan camera multiple times). Detection multiplexing allows absorbance values at twelve distinct positions to be measured and enzyme kinetics are recorded for label-free concentration gradients (composed of about 60 droplets each, covering as many concentrations). With a throughput of around 8640 data points per hour, a 10-fold improvement compared to the previously reported single point detection method is achieved. In a single experiment, twelve full datasets of high-resolution and high accuracy Michaelis-Menten kinetics were determined to demonstrate the potential for enzyme characterization for glycosidase substrates covering a range in enzymatic hydrolysis of seven orders of magnitude in k cat /K M . The straightforward set-up, high throughput, excellent data quality, wide dynamic range that allows coverage of diverse activities suggest that this system may serve as a miniaturized spectrophotometer to for detailed analysis of study clones emerging from large-scale combinatorial experiments. Abstract Figure\", \"doi_y\": \"https://doi.org/10.1101/2022.07.28.500969\", \"publication_date\": \"2022-07-31\"}, {\"work_id\": \"W4366596594\", \"doi_x\": \"10.3389/fcell.2023.1153624\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Regulation of eukaryotic mRNA deadenylation and degradation by the Ccr4-Not complex\", \"abstract\": \"Accurate and precise regulation of gene expression programmes in eukaryotes involves the coordinated control of transcription, mRNA stability and translation. In recent years, significant progress has been made about the role of sequence elements in the 3' untranslated region for the regulation of mRNA degradation, and a model has emerged in which recruitment of the Ccr4-Not complex is the critical step in the regulation of mRNA decay. Recruitment of the Ccr4-Not complex to a target mRNA results in deadenylation mediated by the Caf1 and Ccr4 catalytic subunits of the complex. Following deadenylation, the 5' cap structure is removed, and the mRNA subjected to 5'-3' degradation. Here, the role of the human Ccr4-Not complex in cytoplasmic deadenylation of mRNA is reviewed, with a particular focus on mechanisms of its recruitment to mRNA by sequence motifs in the 3' untranslated region, codon usage, as well as general mechanisms involving the poly(A) tail.\", \"doi_y\": \"https://doi.org/10.3389/fcell.2023.1153624\", \"publication_date\": \"2023-04-20\"}, {\"work_id\": \"W4319459491\", \"doi_x\": \"10.1371/journal.pbio.3001962\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"abstract\": \"Macroautophagy/autophagy is an intracellular degradation process central to cellular homeostasis and defense against pathogens in eukaryotic cells. Regulation of autophagy relies on hierarchical binding of autophagy cargo receptors and adaptors to ATG8/LC3 protein family members. Interactions with ATG8/LC3 are typically facilitated by a conserved, short linear sequence, referred to as the ATG8/LC3 interacting motif/region (AIM/LIR), present in autophagy adaptors and receptors as well as pathogen virulence factors targeting host autophagy machinery. Since the canonical AIM/LIR sequence can be found in many proteins, identifying functional AIM/LIR motifs has proven challenging. Here, we show that protein modelling using Alphafold-Multimer (AF2-multimer) identifies both canonical and atypical AIM/LIR motifs with a high level of accuracy. AF2-multimer can be modified to detect additional functional AIM/LIR motifs by using protein sequences with mutations in primary AIM/LIR residues. By combining protein modelling data from AF2-multimer with phylogenetic analysis of protein sequences and protein\\u2013protein interaction assays, we demonstrate that AF2-multimer predicts the physiologically relevant AIM motif in the ATG8-interacting protein 2 (ATI-2) as well as the previously uncharacterized noncanonical AIM motif in ATG3 from potato ( Solanum tuberosum ). AF2-multimer also identified the AIM/LIR motifs in pathogen-encoded virulence factors that target ATG8 members in their plant and human hosts, revealing that cross-kingdom ATG8-LIR/AIM associations can also be predicted by AF2-multimer. We conclude that the AF2-guided discovery of autophagy adaptors/receptors will substantially accelerate our understanding of the molecular basis of autophagy in all biological kingdoms.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001962\", \"publication_date\": \"2023-02-08\"}, {\"work_id\": \"W4323537719\", \"doi_x\": \"10.1146/annurev-arplant-070522-062509\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Evolution and Evolvability of Photosystem II\", \"abstract\": \"Photosystem II is the water-oxidizing and O 2 -evolving enzyme of photosynthesis. How and when this remarkable enzyme arose are fundamental questions in the history of life that have remained difficult to answer. Here, recent advances in our understanding of the origin and evolution of photosystem II are reviewed and discussed in detail. The evolution of photosystem II indicates that water oxidation originated early in the history of life, long before the diversification of cyanobacteria and other major groups of prokaryotes, challenging and transforming current paradigms on the evolution of photosynthesis. We show that photosystem II has remained virtually unchanged for billions of years, and yet the nonstop duplication process of the D1 subunit of photosystem II, which controls photochemistry and catalysis, has enabled the enzyme to become adaptable to variable environmental conditions and even to innovate enzymatic functions beyond water oxidation. We suggest that this evolvability can be harnessed to develop novel light-powered enzymes with the capacity to carry out complex multistep oxidative transformations for sustainable biocatalysis.\", \"doi_y\": \"https://doi.org/10.1146/annurev-arplant-070522-062509\", \"publication_date\": \"2023-05-22\"}, {\"work_id\": \"W4290928385\", \"doi_x\": \"10.1093/database/baac062\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"A roadmap for the functional annotation of protein families: a community perspective\", \"abstract\": \"Over the last 25 years, biology has entered the genomic era and is becoming a science of 'big data'. Most interpretations of genomic analyses rely on accurate functional annotations of the proteins encoded by more than 500 000 genomes sequenced to date. By different estimates, only half the predicted sequenced proteins carry an accurate functional annotation, and this percentage varies drastically between different organismal lineages. Such a large gap in knowledge hampers all aspects of biological enterprise and, thereby, is standing in the way of genomic biology reaching its full potential. A brainstorming meeting to address this issue funded by the National Science Foundation was held during 3-4 February 2022. Bringing together data scientists, biocurators, computational biologists and experimentalists within the same venue allowed for a comprehensive assessment of the current state of functional annotations of protein families. Further, major issues that were obstructing the field were identified and discussed, which ultimately allowed for the proposal of solutions on how to move forward.\", \"doi_y\": \"https://doi.org/10.1093/database/baac062\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4318159575\", \"doi_x\": \"10.1101/2023.01.25.525455\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Structure of the Commander endosomal trafficking complex linked to X-linked intellectual disability/Ritscher-Schinzel syndrome\", \"abstract\": \"SUMMARY The Commander complex is required for endosomal recycling of diverse transmembrane cargos and is mutated in Ritscher-Schinzel syndrome. It comprises two subassemblies; Retriever composed of VPS35L, VPS26C and VPS29, and the CCC complex which contains ten subunits COMMD1-COMMD10 and two coiled-coil domain-containing (CCDC) proteins CCDC22 and CCDC93. Combining X-ray crystallography, electron cryomicroscopy and in silico predictions we have assembled a complete structural model of Commander. Retriever is distantly related to the endosomal Retromer complex but has unique features preventing the shared VPS29 subunit from interacting with Retromer-associated factors. The COMMD proteins form a distinctive hetero-decameric ring stabilised by extensive interactions with CCDC22 and CCDC93. These adopt a coiled-coil structure that connects the CCC and Retriever assemblies and recruits a sixteenth subunit, DENND10, to form the complete Commander complex. The structure allows mapping of disease-causing mutations and reveals the molecular features required for the function of this evolutionarily conserved trafficking machinery.\", \"doi_y\": \"https://doi.org/10.1101/2023.01.25.525455\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W4289028821\", \"doi_x\": \"10.1101/2022.07.29.22278171\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A power-based sliding window approach to evaluate the clinical impact of rare genetic variants\", \"abstract\": \"Abstract Systematic determination of rare and novel variant pathogenicity remains a major challenge, even when there is an established association between a gene and phenotype. Here we present Power Window (PW), a novel sliding window technique that identifies the clinically impactful regions of a gene using population-scale clinico-genomic datasets. By sizing windows based on the number of variant carriers, rather than the number of variants or nucleotides, statistical power is held constant during analysis, enabling the localization of clinical impact as well as the removal of unassociated gene regions. This method can be used to focus on: specific variant types such as loss of function (LoF) or other coding; parts of a gene, such as those expressed in different tissues; or isolating gene regions with opposite directions of effect. Using a training set of 300K exomes from the UKBiobank (UKB), we developed PW-based LoF and coding models for well-established gene-disease associations and tested their accuracy in two additional cohorts (128k exomes from the UKB and 30k exomes from the Healthy Nevada Project (HNP)). The significant PW models retained a mean of 64% of the rare variant carriers in each gene (range 16-98%), with quantitative traits showing a mean effect size improvement of 48% compared to aggregating rare variants across the entire gene, and the odds ratios for binary traits improving by a mean of 2.4-fold. PW showcases that EHR-based statistical analyses can accurately distinguish between novel coding variants that will have high phenotypic penetrance in a population and those that will not, unlocking new potential for population genetic screening.\", \"doi_y\": \"https://doi.org/10.1101/2022.07.29.22278171\", \"publication_date\": \"2022-07-31\"}, {\"work_id\": \"W4386732685\", \"doi_x\": \"10.1038/s41467-023-41201-0\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Pro-phagocytic function and structural basis of GPR84 signaling\", \"abstract\": \"Abstract GPR84 is a unique orphan G protein-coupled receptor (GPCR) that can be activated by endogenous medium-chain fatty acids (MCFAs). The signaling of GPR84 is largely pro-inflammatory, which can augment inflammatory response, and GPR84 also functions as a pro-phagocytic receptor to enhance phagocytic activities of macrophages. In this study, we show that the activation of GPR84 by the synthetic agonist 6-OAU can synergize with the blockade of CD47 on cancer cells to induce phagocytosis of cancer cells by macrophages. We also determine a high-resolution structure of the GPR84-G i signaling complex with 6-OAU. This structure reveals an occluded binding pocket for 6-OAU, the molecular basis of receptor activation involving non-conserved structural motifs of GPR84, and an unusual G i -coupling interface. Together with computational docking and simulations studies, this structure also suggests a mechanism for the high selectivity of GPR84 for MCFAs and a potential routes of ligand binding and dissociation. These results provide a framework for understanding GPR84 signaling and developing new drugs targeting GPR84.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-41201-0\", \"publication_date\": \"2023-09-14\"}, {\"work_id\": \"W4323546397\", \"doi_x\": \"10.1038/s41586-023-05781-7\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"abstract\": \"Abstract Diverse aerobic bacteria use atmospheric H 2 as an energy source for growth and survival 1 . This globally significant process regulates the composition of the atmosphere, enhances soil biodiversity and drives primary production in extreme environments 2,3 . Atmospheric H 2 oxidation is attributed to uncharacterized members of the [NiFe] hydrogenase superfamily 4,5 . However, it remains unresolved how these enzymes overcome the extraordinary catalytic challenge of oxidizing picomolar levels of H 2 amid ambient levels of the catalytic poison O 2 and how the derived electrons are transferred to the respiratory chain 1 . Here we determined the cryo-electron microscopy structure of the Mycobacterium smegmatis hydrogenase Huc and investigated its mechanism. Huc is a highly efficient oxygen-insensitive enzyme that couples oxidation of atmospheric H 2 to the hydrogenation of the respiratory electron carrier menaquinone. Huc uses narrow hydrophobic gas channels to selectively bind atmospheric H 2 at the expense of O 2 , and 3 [3Fe\\u20134S] clusters modulate the properties of the enzyme so that atmospheric H 2 oxidation is energetically feasible. The Huc catalytic subunits form an octameric 833 kDa complex around a membrane-associated stalk, which transports and reduces menaquinone 94 \\u00c5 from the membrane. These findings provide a mechanistic basis for the biogeochemically and ecologically important process of atmospheric H 2 oxidation, uncover a mode of energy coupling dependent on long-range quinone transport, and pave the way for the development of catalysts that oxidize H 2 in ambient air.\", \"doi_y\": \"https://doi.org/10.1038/s41586-023-05781-7\", \"publication_date\": \"2023-03-08\"}, {\"work_id\": \"W4225312695\", \"doi_x\": \"10.1016/j.tcb.2022.03.011\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"A novel superfamily of bridge-like lipid transfer proteins\", \"abstract\": \"Lipid transfer proteins mediate nonvesicular transport of lipids at membrane contact sites to regulate the lipid composition of organelle membranes. Recently, a new type of bridge-like lipid transfer protein has emerged; these proteins contain a long hydrophobic groove and can mediate bulk transport of lipids between organelles. Here, we review recent insights into the structure of these proteins and identify a repeating modular unit that we propose to name the repeating \\u03b2-groove (RBG) domain. This new structural understanding conceptually unifies all the RBG domain-containing lipid transfer proteins as members of an RBG protein superfamily. We also examine the biological functions of these lipid transporters in normal physiology and disease and speculate on the evolutionary origins of RBG proteins in bacteria.\", \"doi_y\": \"https://doi.org/10.1016/j.tcb.2022.03.011\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4386248168\", \"doi_x\": \"10.1098/rsob.230166\", \"af_direct_citation\": 0, \"af_indirect_citations\": 5, \"title\": \"A unifying mechanism for protein transport through the core bacterial Sec machinery\", \"abstract\": \"Encapsulation and compartmentalization are fundamental to the evolution of cellular life, but they also pose a challenge: how to partition the molecules that perform biological functions\\u2014the proteins\\u2014across impermeable barriers into sub-cellular organelles, and to the outside. The solution lies in the evolution of specialized machines, translocons, found in every biological membrane, which act both as gate and gatekeeper across and into membrane bilayers. Understanding how these translocons operate at the molecular level has been a long-standing ambition of cell biology, and one that is approaching its denouement; particularly in the case of the ubiquitous Sec system. In this review, we highlight the fruits of recent game-changing technical innovations in structural biology, biophysics and biochemistry to present a largely complete mechanism for the bacterial version of the core Sec machinery. We discuss the merits of our model over alternative proposals and identify the remaining open questions. The template laid out by the study of the Sec system will be of immense value for probing the many other translocons found in diverse biological membranes, towards the ultimate goal of altering or impeding their functions for pharmaceutical or biotechnological purposes.\", \"doi_y\": \"https://doi.org/10.1098/rsob.230166\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4383483939\", \"doi_x\": \"10.1128/msphere.00170-23\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Growth in a biofilm promotes conjugation of a <i>bla</i> <sub>NDM-1</sub> -bearing plasmid between <i>Klebsiella pneumoniae</i> strains\", \"abstract\": \"Antimicrobial resistance (AMR) is a growing problem, especially in Gram-negative Enterobacteriaceae such as Klebsiella pneumoniae. Horizontal transfer of conjugative plasmids contributes to AMR gene dissemination. Bacteria such as K. pneumoniae commonly exist in biofilms, yet most studies focus on planktonic cultures. Here we studied the transfer of a multi-drug resistance plasmid in planktonic and biofilm populations of K. pneumoniae. We determined plasmid transfer from a clinical isolate, CPE16, which carried four plasmids, including the 119-kbp blaNDM-1-bearing F-type plasmid pCPE16_3, in planktonic and biofilm conditions. We found that transfer frequency of pCPE16_3 in a biofilm was orders-of-magnitude higher than between planktonic cells. In 5/7 sequenced transconjugants (TCs) multiple plasmids had transferred. Plasmid acquisition had no detectable growth impact on TCs. Gene expression of the recipient and a transconjugant was investigated by RNA-sequencing in three lifestyles: planktonic exponential growth, planktonic stationary phase, and biofilm. We found that lifestyle had a substantial impact on chromosomal gene expression, and plasmid carriage affected chromosomal gene expression most in stationary planktonic and biofilm lifestyles. Furthermore, expression of plasmid genes was lifestyle-dependent, with distinct signatures across the three conditions. Our study shows that growth in biofilm greatly increased the risk of conjugative transfer of a carbapenem resistance plasmid in K. pneumoniae without fitness costs and minimal transcriptional rearrangements, thus highlighting the importance of biofilms in the spread of AMR in this opportunistic pathogen. IMPORTANCE Carbapenem-resistant K. pneumoniae is particularly problematic in hospital settings. Carbapenem resistance genes can transfer between bacteria via plasmid conjugation. Alongside drug resistance, K. pneumoniae can form biofilms on hospital surfaces, at infection sites and on implanted devices. Biofilms are naturally protected and can be inherently more tolerant to antimicrobials than their free-floating counterparts. There have been indications that plasmid transfer may be more likely in biofilm populations, thus creating a conjugation \\\"hotspot\\\". However, there is no clear consensus on the effect of the biofilm lifestyle on plasmid transfer. Therefore, we aimed to explore the transfer of a plasmid in planktonic and biofilm conditions, and the impact of plasmid acquisition on a new bacterial host. Our data show transfer of a resistance plasmid is increased in a biofilm, which may be a significant contributing factor to the rapid dissemination of resistance plasmids in K. pneumoniae.\", \"doi_y\": \"https://doi.org/10.1128/msphere.00170-23\", \"publication_date\": \"2023-08-24\"}, {\"work_id\": \"W4321598216\", \"doi_x\": \"10.3389/frqst.2023.1128576\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"NP-hard but no longer hard to solve? Using quantum computing to tackle optimization problems\", \"abstract\": \"In the last decade, public and industrial research funding has moved quantum computing from the early promises of Shor's algorithm through experiments to the era of noisy intermediate scale quantum devices (NISQ) for solving real-world problems. It is likely that quantum methods can efficiently solve certain (NP-)hard optimization problems where classical approaches fail. In our perspective, we examine the field of quantum optimization where we solve optimisation problems using quantum computers. We demonstrate this through a proper use case and discuss the current quality of quantum computers, their solver capabilities, and benchmarking difficulties. Although we show a proof-of-concept rather than a full benchmark, we use the results to emphasize the importance of using appropriate metrics when comparing quantum and classical methods. We conclude with discussion on some recent quantum optimization breakthroughs and the current status and future directions.\", \"doi_y\": \"https://doi.org/10.3389/frqst.2023.1128576\", \"publication_date\": \"2023-02-23\"}, {\"work_id\": \"W4205290562\", \"doi_x\": \"10.1371/journal.ppat.1010202\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Mutational analysis of Aedes aegypti Dicer 2 provides insights into the biogenesis of antiviral exogenous small interfering RNAs\", \"abstract\": \"The exogenous small interfering RNA (exo-siRNA) pathway is a key antiviral mechanism in the Aedes aegypti mosquito, a widely distributed vector of human-pathogenic arboviruses. This pathway is induced by virus-derived double-stranded RNAs (dsRNA) that are cleaved by the ribonuclease Dicer 2 (Dcr2) into predominantly 21 nucleotide (nt) virus-derived small interfering RNAs (vsiRNAs). These vsiRNAs are used by the effector protein Argonaute 2 within the RNA-induced silencing complex to cleave target viral RNA. Dcr2 contains several domains crucial for its activities, including helicase and RNase III domains. In Drosophila melanogaster Dcr2, the helicase domain has been associated with binding to dsRNA with blunt-ended termini and a processive siRNA production mechanism, while the platform-PAZ domains bind dsRNA with 3\\u2019 overhangs and subsequent distributive siRNA production. Here we analyzed the contributions of the helicase and RNase III domains in Ae . aegypti Dcr2 to antiviral activity and to the exo-siRNA pathway. Conserved amino acids in the helicase and RNase III domains were identified to investigate Dcr2 antiviral activity in an Ae . aegypti -derived Dcr2 knockout cell line by reporter assays and infection with mosquito-borne Semliki Forest virus ( Togaviridae , Alphavirus ). Functionally relevant amino acids were found to be conserved in haplotype Dcr2 sequences from field-derived Ae . aegypti across different continents. The helicase and RNase III domains were critical for silencing activity and 21 nt vsiRNA production, with RNase III domain activity alone determined to be insufficient for antiviral activity. Analysis of 21 nt vsiRNA sequences (produced by functional Dcr2) to assess the distribution and phasing along the viral genome revealed diverse yet highly consistent vsiRNA pools, with predominantly short or long sequence overlaps including 19 nt overlaps (the latter representing most likely true Dcr2 cleavage products). Combined with the importance of the Dcr2 helicase domain, this suggests that the majority of 21 nt vsiRNAs originate by processive cleavage. This study sheds new light on Ae . aegypti Dcr2 functions and properties in this important arbovirus vector species.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1010202\", \"publication_date\": \"2022-01-06\"}, {\"work_id\": \"W4310909615\", \"doi_x\": \"10.1109/tevc.2022.3220187\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Evolutionary Multiobjective Clustering Over Multiple Conflicting Data Views\", \"abstract\": \"Multiview data analysis provides an effective means to integrate the distinct information sources which are inherent to many applications. Data clustering in a multiview setting specifically aims to identify the most appropriate grouping for a collection of entities, where those entities (or their relationships) can be described from multiple perspectives. Leveraging recent advances in multiobjective clustering, we propose a new evolutionary method to tackle this challenge. Designed around a flexible and unbiased solution representation, together with strategies based on the minimum spanning tree and neighborhood relations, our algorithm optimizes multiple objectives simultaneously to effectively explore the space of candidate tradeoffs between the data views. Through a series of experiments, we investigate the suitability of our proposal in the context of a bioinformatics application, clustering of plausible protein structures, and a diverse set of synthetic problems. The specific case of two data views is considered in this article. The evaluation with respect to a variety of reference approaches demonstrates the effectiveness of our method in discovering high-quality partitions in a multiview setting. Robustness against unreliable data sources and the ability to automatically determine the number of clusters are additional advantages evidenced by the results obtained.\", \"doi_y\": \"https://doi.org/10.1109/tevc.2022.3220187\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4368362681\", \"doi_x\": \"10.1107/s2059798323002413\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>GEMMI</i> and <i>Servalcat</i> restrain <i>REFMAC</i>5\", \"abstract\": \"Macromolecular refinement uses experimental data together with prior chemical knowledge (usually digested into geometrical restraints) to optimally fit an atomic structural model into experimental data, while ensuring that the model is chemically plausible. In the CCP4 suite this chemical knowledge is stored in a Monomer Library, which comprises a set of restraint dictionaries. To use restraints in refinement, the model is analysed and template restraints from the dictionary are used to infer (i) restraints between concrete atoms and (ii) the positions of riding hydrogen atoms. Recently, this mundane process has been overhauled. This was also an opportunity to enhance the Monomer Library with new features, resulting in a small improvement in REFMAC5 refinement. Importantly, the overhaul of this part of CCP4 has increased flexibility and eased experimentation, opening up new possibilities.\", \"doi_y\": \"https://doi.org/10.1107/s2059798323002413\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4283593702\", \"doi_x\": \"10.1111/tpj.15879\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Meiosis in allopolyploid <i>Arabidopsis suecica</i>\", \"abstract\": \"Polyploidy is a major force shaping eukaryote evolution but poses challenges for meiotic chromosome segregation. As a result, first-generation polyploids often suffer from more meiotic errors and lower fertility than established wild polyploid populations. How established polyploids adapt their meiotic behaviour to ensure genome stability and accurate chromosome segregation remains an active research question. We present here a cytological description of meiosis in the model allopolyploid species Arabidopsis suecica (2n = 4x = 26). In large part meiosis in A. suecica is diploid-like, with normal synaptic progression and no evidence of synaptic partner exchanges. Some abnormalities were seen at low frequency, including univalents at metaphase I, anaphase bridges and aneuploidy at metaphase II; however, we saw no evidence of crossover formation occurring between non-homologous chromosomes. The crossover number in A. suecica is similar to the combined number reported from its diploid parents Arabidopsis thaliana (2n = 2x = 10) and Arabidopsis arenosa (2n = 2x = 16), with an average of approximately 1.75 crossovers per chromosome pair. This contrasts with naturally evolved autotetraploid A. arenosa, where accurate chromosome segregation is achieved by restricting crossovers to approximately 1 per chromosome pair. Although an autotetraploid donor is hypothesized to have contributed the A. arenosa subgenome to A. suecica, A. suecica harbours diploid A. arenosa variants of key meiotic genes. These multiple lines of evidence suggest that meiosis in the recently evolved allopolyploid A. suecica is essentially diploid like, with meiotic adaptation following a very different trajectory to that described for autotetraploid A. arenosa.\", \"doi_y\": \"https://doi.org/10.1111/tpj.15879\", \"publication_date\": \"2022-07-22\"}, {\"work_id\": \"W3208999135\", \"doi_x\": \"10.1016/j.jmb.2021.167336\", \"af_direct_citation\": 1, \"af_indirect_citations\": 7, \"title\": \"The AlphaFold Database of Protein Structures: A Biologist\\u2019s Guide\", \"abstract\": \"AlphaFold, the deep learning algorithm developed by DeepMind, recently released the three-dimensional models of the whole human proteome to the scientific community. Here we discuss the advantages, limitations and the still unsolved challenges of the AlphaFold models from the perspective of a biologist, who may not be an expert in structural biology.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2021.167336\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4320855862\", \"doi_x\": \"10.7554/elife.82015\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural insights into actin isoforms\", \"abstract\": \"Actin isoforms organize into distinct networks that are essential for the normal function of eukaryotic cells. Despite a high level of sequence and structure conservation, subtle differences in their design principles determine the interaction with myosin motors and actin-binding proteins. Therefore, identifying how the structure of actin isoforms relates to function is important for our understanding of normal cytoskeletal physiology. Here, we report the high-resolution structures of filamentous skeletal muscle \\u03b1-actin (3.37 \\u00c5), cardiac muscle \\u03b1-actin (3.07 \\u00c5), \\u00df-actin (2.99 \\u00c5), and \\u03b3-actin (3.38 \\u00c5) in the Mg2+\\u00b7ADP state with their native post-translational modifications. The structures revealed isoform-specific conformations of the N-terminus that shift closer to the filament surface upon myosin binding, thereby establishing isoform-specific interfaces. Collectively, the structures of single-isotype, post-translationally modified bare skeletal muscle \\u03b1-actin, cardiac muscle \\u03b1-actin, \\u00df-actin, and \\u03b3-actin reveal general principles, similarities, and differences between isoforms. They complement the repertoire of known actin structures and allow for a comprehensive understanding of in vitro and in vivo functions of actin isoforms.The protein actin is important for many fundamental processes in biology, from contracting muscle to dividing a cell in two. As actin is involved in such a variety of roles, human cells have slightly different versions of the protein, known as isoforms. For example, alpha-actin is vital for contracting muscle, while beta- and gamma-actin drive cellular processes in non-muscle cells. In order to carry out its various functions, actin interacts with many other proteins inside the cell, such as myosin motors which power muscle contraction. These interactions rely on the precise chain of building blocks, known as amino acids, that make up the actin isoforms; even subtle alterations in this sequence can influence the behavior of the protein. However, it is not clear how differences in the amino acid sequence of the actin isoforms impact actin\\u2019s interactions with other proteins. Arora et al. addressed this by studying the structure of four human actin isoforms using a technique called cryo-electron microscopy, where the proteins are flash-frozen and bombarded with electrons. These experiments showed where differences between the amino acid chains of each isoform were located in the protein. Arora et al. then compared their structures with previous work showing the structure of actin bound to myosin. This revealed that the tail-end of the protein (known as the N-terminus) differed in shape between the four isoforms, and this variation may influence how actin binds to others proteins in the cell. These results are an important foundation for further work on actin and how it interacts with other proteins. The structures could help researchers design new tools that can be used to target specific isoforms of actin in different types of laboratory experiments.\", \"doi_y\": \"https://doi.org/10.7554/elife.82015\", \"publication_date\": \"2023-02-15\"}, {\"work_id\": \"W4210916320\", \"doi_x\": \"10.1002/anie.202117324\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Carboxyl Methyltransferase Catalysed Formation of Mono\\u2010 and Dimethyl Esters under Aqueous Conditions: Application in Cascade Biocatalysis\", \"abstract\": \"Carboxyl methyltransferase (CMT) enzymes catalyse the biomethylation of carboxylic acids under aqueous conditions and have potential for use in synthetic enzyme cascades. Herein we report that the enzyme FtpM from Aspergillus fumigatus can methylate a broad range of aromatic mono- and dicarboxylic acids in good to excellent conversions. The enzyme shows high regioselectivity on its natural substrate fumaryl-l-tyrosine, trans, trans-muconic acid and a number of the dicarboxylic acids tested. Dicarboxylic acids are generally better substrates than monocarboxylic acids, although some substituents are able to compensate for the absence of a second acid group. For dicarboxylic acids, the second methylation shows strong pH dependency with an optimum at pH 5.5-6. Potential for application in industrial biotechnology was demonstrated in a cascade for the production of a bioplastics precursor (FDME) from bioderived 5-hydroxymethylfurfural (HMF).\", \"doi_y\": \"https://doi.org/10.1002/anie.202117324\", \"publication_date\": \"2022-02-16\"}, {\"work_id\": \"W4319063252\", \"doi_x\": \"10.1371/journal.pbio.3001959\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"LRRC15 mediates an accessory interaction with the SARS-CoV-2 spike protein\", \"abstract\": \"The interactions between Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) and human host factors enable the virus to propagate infections that lead to Coronavirus Disease 2019 (COVID-19). The spike protein is the largest structural component of the virus and mediates interactions essential for infection, including with the primary angiotensin-converting enzyme 2 (ACE2) receptor. We performed two independent cell-based systematic screens to determine whether there are additional proteins by which the spike protein of SARS-CoV-2 can interact with human cells. We discovered that in addition to ACE2, expression of LRRC15 also causes spike protein binding. This interaction is distinct from other known spike attachment mechanisms such as heparan sulfates or lectin receptors. Measurements of orthologous coronavirus spike proteins implied the interaction was functionally restricted to SARS-CoV-2 by accessibility. We localized the interaction to the C-terminus of the S1 domain and showed that LRRC15 shares recognition of the ACE2 receptor binding domain. From analyzing proteomics and single-cell transcriptomics, we identify LRRC15 expression as being common in human lung vasculature cells and fibroblasts. Levels of LRRC15 were greatly elevated by inflammatory signals in the lungs of COVID-19 patients. Although infection assays demonstrated that LRRC15 alone is not sufficient to permit viral entry, we present evidence that it can modulate infection of human cells. This unexpected interaction merits further investigation to determine how SARS-CoV-2 exploits host LRRC15 and whether it could account for any of the distinctive features of COVID-19.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3001959\", \"publication_date\": \"2023-02-03\"}, {\"work_id\": \"W2978205448\", \"doi_x\": \"10.1101/793943\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Energy landscape steering in SecYEG mediates dynamic coupling in ATP driven protein translocation\", \"abstract\": \"Abstract The Sec translocon is a highly conserved membrane complex for transport of polypeptides across, or into, lipid bilayers. In bacteria, the core protein-channel SecYEG resides in the inner-membrane, through which secretion is powered by the cytosolic ATPase SecA. Here, we use single-molecule fluorescence to interrogate the dynamic state of SecYEG throughout the hydrolytic cycle of SecA. We show that the SecYEG channel fluctuates between open and closed states faster ( \\u223c 20-fold during transport) than ATP turnover; while the nucleotide status of SecA modulates the rates of opening and closure. Interestingly, a SecY variant (PrlA4), exhibiting faster protein transport, but unaffected ATPase rates, increases the dwell time in the open state, facilitating pre-protein diffusion through the pore; thereby improving the efficiency of translocation. Thus, contrary to prevailing structure-based models, SecYEG plays an integral part in the translocation mechanism through dynamic allosteric coupling in which SecA \\u2018steers\\u2019 the energy landscape of the protein-channel.\", \"doi_y\": \"https://doi.org/10.1101/793943\", \"publication_date\": \"2019-10-05\"}, {\"work_id\": \"W4323276421\", \"doi_x\": \"10.1016/j.jbc.2023.104579\", \"af_direct_citation\": 1, \"af_indirect_citations\": 11, \"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"abstract\": \"Protein science is being transformed by powerful computational methods for structure prediction and design: AlphaFold2 can predict many natural protein structures from sequence, and other AI methods are enabling the de novo design of new structures. This raises a question: how much do we understand the underlying sequence-to-structure/function relationships being captured by these methods? This perspective presents our current understanding of one class of protein assembly, the \\u03b1-helical coiled coils. At first sight, these are straightforward: sequence repeats of hydrophobic (h) and polar (p) residues, (hpphppp)n, direct the folding and assembly of amphipathic \\u03b1 helices into bundles. However, many different bundles are possible: they can have two or more helices (different oligomers); the helices can have parallel, antiparallel, or mixed arrangements (different topologies); and the helical sequences can be the same (homomers) or different (heteromers). Thus, sequence-to-structure relationships must be present within the hpphppp repeats to distinguish these states. I discuss the current understanding of this problem at three levels: first, physics gives a parametric framework to generate the many possible coiled-coil backbone structures. Second, chemistry provides a means to explore and deliver sequence-to-structure relationships. Third, biology shows how coiled coils are adapted and functionalized in nature, inspiring applications of coiled coils in synthetic biology. I argue that the chemistry is largely understood; the physics is partly solved, though the considerable challenge of predicting even relative stabilities of different coiled-coil states remains; but there is much more to explore in the biology and synthetic biology of coiled coils. Protein science is being transformed by powerful computational methods for structure prediction and design: AlphaFold2 can predict many natural protein structures from sequence, and other AI methods are enabling the de novo design of new structures. This raises a question: how much do we understand the underlying sequence-to-structure/function relationships being captured by these methods? This perspective presents our current understanding of one class of protein assembly, the \\u03b1-helical coiled coils. At first sight, these are straightforward: sequence repeats of hydrophobic (h) and polar (p) residues, (hpphppp)n, direct the folding and assembly of amphipathic \\u03b1 helices into bundles. However, many different bundles are possible: they can have two or more helices (different oligomers); the helices can have parallel, antiparallel, or mixed arrangements (different topologies); and the helical sequences can be the same (homomers) or different (heteromers). Thus, sequence-to-structure relationships must be present within the hpphppp repeats to distinguish these states. I discuss the current understanding of this problem at three levels: first, physics gives a parametric framework to generate the many possible coiled-coil backbone structures. Second, chemistry provides a means to explore and deliver sequence-to-structure relationships. Third, biology shows how coiled coils are adapted and functionalized in nature, inspiring applications of coiled coils in synthetic biology. I argue that the chemistry is largely understood; the physics is partly solved, though the considerable challenge of predicting even relative stabilities of different coiled-coil states remains; but there is much more to explore in the biology and synthetic biology of coiled coils. As a graduate student in the late 1980s, I was drawn to the challenge of the protein-folding problem and to using protein design as a means of testing our understanding of protein structure and folding. Even then, we suspected that solutions to these problems would come through bioinformatics. However, we did not realize how long it would take for solutions to come or the form that they would take in terms of combining big data, large computer power, and artificial intelligence, specifically, machine learning. Of course, I refer to the recent successes AlphaFold2 and RoseTTAfold in predicting protein structure from sequence alone (1AlQuraishi M. AlphaFold at CASP13.Bioinformatics. 2019; 35: 4862-4865Crossref PubMed Scopus (153) Google Scholar, 2Baek M. DiMaio F. Anishchenko I. Dauparas J. Ovchinnikov S. Lee G.R. et al.Accurate prediction of protein structures and interactions using a three-track neural network.Science. 2021; 373: 871-876Crossref PubMed Scopus (1186) Google Scholar, 3Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (6259) Google Scholar, 4Varadi M. Anyango S. Deshpande M. Nair S. Natassia C. Yordanova G. et al.AlphaFold protein structure database: massively expanding the structural coverage of protein-sequence space with high-accuracy models.Nucl. Acids Res. 2022; 50: D439-D444Crossref PubMed Scopus (1001) Google Scholar). Like many, when I look at an AlphaFold2 model for a complex protein assembly, I am awestruck by the structural solutions that it finds. However, while AlphaFold2 and RoseTTAfold provide solutions to the protein-folding problem\\u2014and, to be sure, these and other methods will improve to provide models for ever-more complex protein structures and assemblies\\u2014they are just that, solutions. In themselves, they do not necessarily, at least not at present, provide an understanding of the physics and chemistry that drives and directs protein folding and assembly, which, in turn, are responsible for protein function. Back in the 1980s and 1990s, it was the desire to understand protein folding that drove interest in the field; it was not just to find solutions to the problem per se. In short, we wanted to understand the physico-chemical principles that underpin protein structure, folding, assembly, and stability. In other words, we sought to decipher the underlying sequence-to-structure relationships for these properties. This perspective is about how far we have progressed in understanding protein folding and design in these terms. Well, at least for one particular protein structure\\u2014the \\u03b1-helical coiled coil. At first sight, this is a relatively straightforward peptide and protein assembly in which two or more \\u03b1-helical chains wrap around each other to form supercoiled or rope-like structures, Figure 1. At that time, coiled-coil\\u2013forming leucine-zipper peptides, Figure 1A, became models for protein folding and design (5Landschulz W.H. Johnson P.F. Mcknight S.L. The leucine zipper - a hypothetical structure common to a new class of DNA-binding proteins.Science. 1988; 240: 1759-1764Crossref PubMed Google Scholar, 6O\\u2019Shea E.K. Klemm J.D. Kim P.S. Alber T. X-ray structure of the GCN4 leucine zipper, a 2-stranded, parallel coiled coil.Science. 1991; 254: 539-544Crossref PubMed Google Scholar, 7O\\u2019Shea E.K. Rutkowski R. Kim P.S. Evidence that the leucine zipper is a coiled coil.Science. 1989; 243: 538-542Crossref PubMed Google Scholar). I was drawn to the folding and design of leucine zippers as a post-doc with Tom Alber in the early 1990s. This was because of the apparent simplicity of these coiled coils and the thought that unless we could understand such simple protein folds and assemblies, we would have no hope with more complex globular proteins. Therefore, I adopted coiled coils as a model for developing an understanding of sequence-to-structure relationships in proteins. An aim of this perspective is to capture some of that journey, which has been contributed to by many scientists in many groups over the past few decades. However, it is not a complete review of coiled-coil structure, biology, or even design. Such an article would be redundant because many excellent reviews are already available (8Lupas A. Coiled coils: new structures and new functions.Trends Biochem. Sci. 1996; 21: 375-382Abstract Full Text PDF PubMed Scopus (0) Google Scholar, 9Mason J.M. Arndt K.M. Coiled coil domains: stability, specificity, and biological implications.Chembiochem. 2004; 5: 170-176Crossref PubMed Scopus (546) Google Scholar, 10Burkhard P. Stetefeld J. Strelkov S.V. Coiled coils: a highly versatile protein folding motif.Trends Cell Biol. 2001; 11: 82-88Abstract Full Text Full Text PDF PubMed Scopus (850) Google Scholar, 11Rose A. Meier I. Scaffolds, levers, rods and springs: diverse cellular functions of long coiled-coil proteins.Cell Mol. Life Sci. 2004; 61: 1996-2009Crossref PubMed Google Scholar, 12Woolfson D.N. The design of coiled-coil structures and assemblies.Adv. Protein Chem. 2005; 70: 79-112Crossref PubMed Scopus (477) Google Scholar, 13Lupas A.N. Bassler J. Coiled coils - a model system for the 21st century.Trends Biochem. Sci. 2017; 42: 130-140Abstract Full Text Full Text PDF PubMed Scopus (127) Google Scholar, 14Woolfson D.N. Coiled-coil design: updated and upgraded.Subcell Biochem. 2017; 82: 35-61Crossref PubMed Scopus (90) Google Scholar), as are others covering newer topics such as the applications of natural and designed coiled coils in biotechnology and synthetic biology (15Lapenta F. Aupic J. Strmsek Z. Jerala R. Coiled coil protein origami: from modular design principles towards biotechnological applications.Chem. Soc. Rev. 2018; 47: 3530-3542Crossref PubMed Google Scholar, 16Beesley J.L. Woolfson D.N. The de novo design of alpha-helical peptides for supramolecular self-assembly.Curr. Opin. Biotech. 2019; 58: 175-182Crossref PubMed Scopus (0) Google Scholar, 17Rink W.M. Thomas F. De novo designed alpha-helical coiled-coil peptides as scaffolds for chemical reactions.Chem-eur J. 2019; 25: 1665-1677Crossref PubMed Scopus (9) Google Scholar, 18Utterstrom J. Naeimipour S. Selegard R. Aili D. Coiled coil-based therapeutics and drug delivery systems.Adv. Drug Deliv. Rev. 2021; 170: 26-43Crossref PubMed Scopus (0) Google Scholar). Instead, I want to give some sense of three things: first, of the journey and the joy of discovery, which have led to our current understanding of this relatively straightforward protein structure; second, that in some respects\\u2014for instance, the physics and chemistry of coiled-coil folding and assembly\\u2014our understanding is complete or very near to it; and third, despite this understanding, we still have much to learn, particularly on the biology of coiled coils. This article is my perspective on the amassed understanding of coiled-coil proteins. Indeed, it closes some of my own research questions; namely, what are the sequence-to-structure relationships that govern coiled-coil folding and assembly, and how do these allow us to design de novo coiled coils with confidence? However, as some research chapters close, others open. For coiled-coil research, new challenges include the following: achieving fully quantitative (free-energy) predictions for coiled-coil structure, stability, and partner selection; gaining a deeper understanding of coiled-coil dynamics and plasticity and how this relates to coiled-coil function. And, as one of the best understood protein folds, how can de novo coiled-coil peptides and proteins be used in biotechnology and synthetic biology to address real-world applications, for instance, in health and the environment. I am certain that the methods, principles, and understanding that have been developed over the past few decades will provide a foundation for these and other endeavors. And to come full circle, it is clear that new artificial intelligence/machine learning\\u2013based methods for protein-structure prediction will contribute here. Indeed, I see these tools as fantastic hypothesis generators for structural molecular and cell biology, including for processes that involve relatively simple, but adaptable and versatile coiled coils. For more complete historical perspectives on the conceptual origins of \\u03b1-helical coiled coils, please see reviews by Squire and Parry and by Lupas et al. (19Lupas A.N. Gruber M. The structure of alpha-helical coiled coils.Adv. Protein Chem. 2005; 70: 37-78Crossref PubMed Scopus (578) Google Scholar, 20Lupas A.N. Bassler J. Dunin-Horkawicz S. The structure and topology of alpha-helical coiled coils.Subcell Biochem. 2017; 82: 95-129Crossref PubMed Scopus (59) Google Scholar, 21Squire J.M. Parry D.A. Fibrous protein structures: hierarchy, history and heroes.Subcell Biochem. 2017; 82: 1-33Crossref PubMed Scopus (8) Google Scholar). The coiled coil came out of physics: Francis Crick\\u2019s description of the coiled coil, as he first named it, is mathematical (22Crick F.H.C. The fourier transform of a coiled-coil.Acta Crystallogr. 1953; 6: 685-689Crossref Google Scholar, 23Crick F.H.C. The packing of alpha-helices - simple coiled-coils.Acta Crystallogr. 1953; 6: 689-697Crossref Google Scholar). Moreover, at that time in the early 1950s, there was little experimental data or confirmed details for any protein structure. So, what followed\\u2014namely, the first description of any protein structure, the concepts of helical nets and knobs-into-holes (KIH) packing, heptad repeats, and what we now refer to as the Crick Equations\\u2014was extremely insightful. Incidentally, Crick published this work in the same year that he and James Watson proposed the double helix for the structure of DNA (24Watson J.D. Crick F.H.C. Molecular structure of nucleic acids - a structure for deoxyribose nucleic acid.Nature. 1953; 171: 737-738Crossref PubMed Scopus (0) Google Scholar). Crick started with Linus Pauling\\u2019s \\u03b1 helix and its regularity (Fig. 2A) (25Pauling L. Corey R.B. Branson H.R. The structure of proteins - 2 hydrogen-bonded helical configurations of the polypeptide chain.Pro. Natl. Acad. Sci. U. S. A. 1951; 37: 205-211Crossref PubMed Google Scholar). As a consequence of steric constraints in polypeptide chains\\u2014as later formalized by Ramachandran in his famous plot (26Ramachandran G.N. Ramakrishnan C. Sasisekharan V. Stereochemistry of polypeptide chain configurations.J. Mol. Biol. 1963; 7: 95-99Crossref PubMed Scopus (2715) Google Scholar, 27Ramachandran G.N. Venkatachalam C.M. Krimm S. Stereochemical criteria for polypeptide and protein chain conformations .3. Helical and hydrogen-bonded polypeptide chains.Biophys. J. 1966; 6: 849-872Abstract Full Text PDF PubMed Google Scholar)\\u2014the \\u03b1 helix has precisely 3.6 residues per turn (Fig. 2A). Crick reasoned that two or more such helices could interact tightly via seams of interacting side chains spaced three and four residues apart along polypeptide chains\\u2014i.e., with an average spacing of 3.5 residues\\u2014to match the 3.6 residues per turn as closely as possible. Now, we annotate these repeats abcdefg with the key interacting side chains falling at the a and d sites. Crick visualized this with helical-net diagrams (Fig. 2B). Highlighting the 3,4 spacing on one of these reveals the seam as the line of connected diamond shapes. Two such seams can interlace to bring the helices into intimate contact (Fig. 2C): the diamonds on one helix form \\u2018holes\\u2019 into which side chains from the other helix can slot. In this way, Crick coined the terms \\u2018heptad repeat\\u2019 (from the Greek for seven) and KIH packing for these sequence and structural features, respectively. These are now the hallmarks for coiled-coil peptides and proteins, and they provide a firm basis for understanding them. The term \\u2018coiled coil\\u2019 is a consequence of these sequence patterns and the structural packing. This is because 3.5 is less than 3.6. As a result, on a helical net, the seam slants to the left (Fig. 2B). Thus, for two nets to interlace requires one to be offset at an angle to the other (Fig. 2C). In a 3D \\u03b1 helix, the seam precesses around the surface of the helix with the opposite sense to the handedness of the helix. Thus, for two \\u03b1 helices to maintain contact, they must wrap around each other like the strands of a rope. As \\u03b1 helices made from l-amino acids are right handed, heptad-based coiled coils are left-handed ropes. This overall assembly\\u2014which is a quaternary structure and not a tertiary structure\\u2014is also helical. Strictly, it is a superhelix. This is why Crick called the envisaged structures coiled coils; \\u201ccoils\\u201d refers to the \\u03b1 helices, and \\u201ccoiled\\u201d refers to the superhelix. From Pauling\\u2019s \\u03b1-helical parameters\\u20143.6 residues per turn and a rise per residues of 1.5 \\u00c5\\u2014Crick predicted that the crossing angle between the two helices would be \\u224820\\u02da, which gives a superhelical pitch of \\u2248126 residues or \\u2248186 \\u00c5 (Box 1). (Note: these values correspond to those observed in experimentally determined coiled coils (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar)). This is formalized in the Crick Equations. Thus, the coiled coil is inherently parametric, which makes it physics, predictable, modelable and, as, we will see, ultimately designable (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar). Indeed, this formalism has led to a swathe of coiled-coil modeling and design programs (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar, 30Harbury P.B. Tidor B. Kim P.S. Repacking protein cores with backbone freedom - structure prediction for coiled coils.Proc. Natl. Acad. Sci. U. S. A. 1995; 92: 8408-8412Crossref PubMed Scopus (0) Google Scholar, 31Offer G. Hicks M.R. Woolfson D.N. Generalized crick equations for modeling noncanonical coiled coils.J. Struct. Biol. 2002; 137: 41-53Crossref PubMed Scopus (51) Google Scholar, 32Dunin-Horkawicz S. Lupas A.N. Measuring the conformational space of square four-helical bundles with the program samCC.J. Struct. Biol. 2010; 170: 226-235Crossref PubMed Scopus (33) Google Scholar, 33Huang P.S. Oberdorfer G. Xu C.F. Pei X.Y. Nannenga B.L. Rogers J.M. et al.High thermodynamic stability of parametrically designed helical bundles.Science. 2014; 346: 481-485Crossref PubMed Scopus (192) Google Scholar, 34Thomson A.R. Wood C.W. Burton A.J. Bartlett G.J. Sessions R.B. Brady R.L. et al.Computational design of water-soluble alpha-helical barrels.Science. 2014; 346: 485-488Crossref PubMed Scopus (0) Google Scholar, 35Wood C.W. Heal J.W. Thomson A.R. Bartlett G.J. Ibarra A.A. Brady R.L. et al.ISAMBARD: an open-source computational environment for biomolecular analysis, modelling and design.Bioinformatics. 2017; 33: 3043-3050Crossref PubMed Scopus (33) Google Scholar, 36Wood C.W. Woolfson D.N. CCBuilder 2.0: powerful and accessible coiled-coil modeling.Protein Sci. 2018; 27: 103-111Crossref PubMed Scopus (69) Google Scholar). Several of these are accessible and easy-to-use computational tools on the internet (Box 2). Some allow coiled-coil backbones to be built quickly and accurately (28Grigoryan G. DeGrado W.F. Probing designability via a generalized model of helical bundle geometry.J. Mol. Biol. 2011; 405: 1079-1100Crossref PubMed Scopus (148) Google Scholar, 31Offer G. Hicks M.R. Woolfson D.N. Generalized crick equations for modeling noncanonical coiled coils.J. Struct. Biol. 2002; 137: 41-53Crossref PubMed Scopus (51) Google Scholar), and others allow full atomistic modeling of coiled coils including side chains (29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar, 33Huang P.S. Oberdorfer G. Xu C.F. Pei X.Y. Nannenga B.L. Rogers J.M. et al.High thermodynamic stability of parametrically designed helical bundles.Science. 2014; 346: 481-485Crossref PubMed Scopus (192) Google Scholar, 35Wood C.W. Heal J.W. Thomson A.R. Bartlett G.J. Ibarra A.A. Brady R.L. et al.ISAMBARD: an open-source computational environment for biomolecular analysis, modelling and design.Bioinformatics. 2017; 33: 3043-3050Crossref PubMed Scopus (33) Google Scholar, 36Wood C.W. Woolfson D.N. CCBuilder 2.0: powerful and accessible coiled-coil modeling.Protein Sci. 2018; 27: 103-111Crossref PubMed Scopus (69) Google Scholar). Furthermore, there are tools for assessing the quality of such models and experimental structures parametrically and energetically (32Dunin-Horkawicz S. Lupas A.N. Measuring the conformational space of square four-helical bundles with the program samCC.J. Struct. Biol. 2010; 170: 226-235Crossref PubMed Scopus (33) Google Scholar, 37Walshaw J. Woolfson D.N. Socket: a program for identifying and analysing coiled-coil motifs within protein structures.J. Mol. Biol. 2001; 307: 1427-1450Crossref PubMed Scopus (306) Google Scholar, 38Strelkov S.V. Burkhard P. Analysis of alpha-helical coiled coils with the program TWISTER reveals a structural mechanism for stutter compensation.J. Struct. Biol. 2002; 137: 54-64Crossref PubMed Scopus (212) Google Scholar, 39Kumar P. Woolfson D.N. Socket2: a program for locating, visualizing and analyzing coiled-coil interfaces in protein structures.Bioinformatics. 2021; 37: 4575-4577Crossref PubMed Scopus (12) Google Scholar). This has made coiled-coil modeling, engineering, and design accessible to nonexperts, which is a significant advance. In turn, this has enabled the exploration of coiled-coil assemblies beyond the confines of natural proteins (34Thomson A.R. Wood C.W. Burton A.J. Bartlett G.J. Sessions R.B. Brady R.L. et al.Computational design of water-soluble alpha-helical barrels.Science. 2014; 346: 485-488Crossref PubMed Scopus (0) Google Scholar, 40Dawson W.M. Martin F.J.O. Rhys G.G. Shelley K.L. Brady R.L. Woolfson D.N. Coiled coils 9-to-5: rational de novo design of alpha-helical barrels with tunable oligomeric states.Chem. Sci. 2021; 12: 6923-6928Crossref PubMed Google Scholar).Box 1From \\u03b1-helical to coiled-coil parametersThe following exercise can be done by considering projections of the \\u03b1 helix as helical nets or helical wheels.First, I need to dispel a key misconception made by students and professors alike. Pauling\\u2019s \\u03b1 helix has 3.6 residues per turn with a very low tolerance of variation (29Wood C.W. Bruning M. Ibarra A.A. Bartlett G.J. Thomson A.R. Sessions R.B. et al.CCBuilder: an interactive web-based tool for building, designing and assessing coiled-coil protein assemblies.Bioinformatics. 2014; 30: 3029-3035Crossref PubMed Scopus (87) Google Scholar); indeed, this is one of the few fixed physical constants in biology (52Kumar P. Paterson N.G. Clayden J. Woolfson D.N. De novo design of discrete, stable 310-helix peptide assemblies.Nature. 2022; 607: 387-392Crossref PubMed Scopus (0) Google Scholar). This is because the energy well for the \\u03b1 helix is narrow and deep (53Kuster D.J. Liu C.Y. Fang Z. Ponder J.W. Marshall G.R. High-resolution crystal structures of protein helices reconciled with three-centered hydrogen bonds and multipole electrostatics.PLoS One. 2015; 10: e0123146Crossref PubMed Scopus (23) Google Scholar), and small deviations from \\u03b1-helical parameters incur large energy penalties. Indeed, the nearby 310 and \\u03c0 helixes are rare in protein structures and difficult to design (52Kumar P. Paterson N.G. Clayden J. Woolfson D.N. De novo design of discrete, stable 310-helix peptide assemblies.Nature. 2022; 607: 387-392Crossref PubMed Scopus (0) Google Scholar). As a result, the \\u03b1 helix does not somehow collapse to 3.5 residues per turn in coiled coils. If it did, we would not have coiled coils: the average sequence repeat of 3.5 and a helical repeat of 3.5 would match, and the helices would pack straight and not wrap around each other. I suspect that the convenience of 7-residue helical wheels (Fig. 2D) rather than the more accurate 18-residues helical wheels (Fig. 2E) has contributed to this misconception. Crick\\u2019s helical nets are a more faithful mapping of an \\u03b1-helical surface in 2D (Fig. 2B).With the C\\u03b1 atoms of an \\u03b1 helix projected on an 18-residue helical wheel, successive residues are separated by 100\\u02da. Thus, the seven residues of a heptad repeat would span out 700\\u02da. This is 20\\u02da short of two full turns (2 \\u00d7 360\\u02da = 720\\u02da and 7.2 residues) of the \\u03b1 helix. 20\\u02da goes into 360\\u02da 18 times. Therefore, 18 heptad repeats are required for the interacting seam to make one complete revolution of a helix and to bring the helices back into sync. This defines the pitch of the coiled coil. Hence, the pitch is 18 \\u00d7 7 = 126 residues. Given that the \\u03b1 helix has a rise per residue of 1.5 \\u00c5, these would span 189 \\u00c5 of a straight \\u03b1 helix. However, each \\u03b1 helix is inclined by \\u224810\\u02da relative to the superhelical axis. Therefore, the rise per residue along this coiled-coil axis is 1.5 \\u00d7 cos(10\\u02da) = 1.48 \\u00c5, and the ideal superhelical pitch is \\u2248186 \\u00c5 (Fig. 2F). In addition to the rise per residue and the coiled-coil pitch, just two other parameters are needed to define and generate regular coiled-coil backbones. These are the radius of the coiled coil and the interface or Crick angle, Figure 2G.Finally for this Box, we should also credit Pauling. Crick considered only 7-residue repeats. However, Pauling et al. considered other repeats in \\u03b1-helical conformations, specifically, those with 11, 15, and 18 residues (25Pauling L. Corey R.B. Branson H.R. The structure of proteins - 2 hydrogen-bonded helical configurations of the polypeptide chain.Pro. Natl. Acad. Sci. U. S. A. 1951; 37: 205-211Crossref PubMed Google Scholar, 54Pauling L. Corey R.B. The structure of synthetic polypeptides.Proc. Natl. Acad. Sci. U. S. A. 1951; 37: 241-250Crossref PubMed Scopus (0) Google Scholar). Along with 7-residue repeats, these are compatible with combinations of 3- and 4-residue spacings; namely, 3-4, 3-4-4, 3-4-4-4, and 3-4-3-4-4, respectively. However, they have different average spacings between the interacting residues of 3.5, 3.67, 3.75, and 3.6, respectively. As the \\u03b1-helical structural repeat is fixed at 3.6 residues per turn, when these repeats are realized in packed coiled coils, they lead to different superhelical twists. These can also be calculated using the considerations laid out above. They result in further left-handed, some right-handed, and even \\u201cstraight\\u201d coiled coils (19Lupas A.N. Gruber M. The structure of alpha-helical coiled coils.Adv. Protein Chem. 2005; 70: 37-78Crossref PubMed Scopus (578) Google Scholar, 55Brown J.H. Cohen C. Parry D.A.D. Heptad breaks in alpha-helical coiled coils: stutters and stammers.Proteins-Struct. Funct. Genet. 1996; 26: 134-145Crossref PubMed Scopus (0) Google Scholar, 56Hicks M.R. Holberton D.V. Kowalczyk C. Woolfson D.N. Coiled-coil assembly by peptides with non-heptad sequence motifs.Fold Des. 1997; 2: 149-158Abstract Full Text Full Text PDF PubMed Google Scholar, 57Harbury P.B. Plecs J.J. Tidor B. Alber T. Kim P.S. High-resolution protein design with backbone freedom.Science. 1998; 282: 1462-1467Crossref PubMed Google Scholar, 58Hicks M.R. Walshaw J. Woolfson D.N. Investigating the tolerance of coiled-coil peptides to nonheptad sequence inserts.J. Struct. Biol. 2002; 137: 73-81Crossref PubMed Scopus (26) Google Scholar).Box 2Tools and resources for predicting, building, analyzing, and visualizing coiled-coil structuresPredictionSeveral tools for predicting coiled-coil structure from sequence are brought together and implemented at Andrei Lupas\\u2019s Max Planck Institute Bioinformatics Toolkit (59Gabler F. Nam S.Z. Till S. Mirdita M. Steinegger M. Soding J. et al.Protein sequence analysis using the MPI bioinformatics toolkit.Curr. Protoc. Bioinform. 2020; 72: e108Crossref PubMed Scopus (196) Google Scholar) (https://toolkit.tuebingen.mpg.de/). These include the following: Lupas\\u2019s original COILS program (60Lupas A. Vandyke M. Stock J. Predicting coiled coils from protein sequences.Science. 1991; 252: 1162-1164Crossref PubMed Google Scholar) (now PCOIL), DeepCoil (61Ludwiczak J. Winski A. Szczepaniak K. Alva V. Dunin-Horkawicz S. DeepCoil-a fast and accurate prediction of coiled-coil domains in protein sequences.Bioinformatics. 2019; 35: 2790-2795Crossref PubMed Scopus (0) Google Scholar), and MARCOIL (62Delorenzi M. Speed T. An HMM model for coiled-coil domains and a comparison with PSSM-based predictions.Bioinformatics. 2002; 18: 617-625Crossref PubMed Google Scholar). Regarding predicting oligomeric states from sequence, in light of recent advances in coiled-coil design and protein-structure prediction, generally, there is considerable room for improvement here. However, the following are currently available: LOGICOIL (63Vincent T.L. Green P.J. Woolfson D.N. LOGICOIL-multi-state prediction of coiled-coil oligomeric state.Bioinformatics. 2013; 29: 69-76Crossref PubMed Scopus (0) Google Scholar) (http://coiledcoils.chm.bris.ac.uk/LOGICOIL/) and Multicoil2 (64Trigg J. Gutwin K. Keating A.E. Berger B. Multicoil2: predicting coiled coils and their oligomerization states from sequence in the twilight zone.PLoS One. 2011; 6: e23519Crossref PubMed Scopus (60) Google Scholar) (http://cb.csail.mit.edu/cb/multicoil2/cgi-bin/multicoil2.cgi). AlphaFold2 (3Jumper J. Evans R. Pritzel A. Green T. Figurnov M. Ronneberger O. et al.Highly accurate protein structure prediction with AlphaFold.Nature. 2021; 596: 583-589Crossref PubMed Scopus (6259) Google Scholar) also predi\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.104579\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4310059371\", \"doi_x\": \"10.1038/s41467-022-34972-5\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"abstract\": \"Abstract The Klebsiella jumbo myophage \\u03d5Kp24 displays an unusually complex arrangement of tail fibers interacting with a host cell. In this study, we combine cryo-electron microscopy methods, protein structure prediction methods, molecular simulations, microbiological and machine learning approaches to explore the capsid, tail, and tail fibers of \\u03d5Kp24. We determine the structure of the capsid and tail at 4.1 \\u00c5 and 3.0 \\u00c5 resolution. We observe the tail fibers are branched and rearranged dramatically upon cell surface attachment. This complex configuration involves fourteen putative tail fibers with depolymerase activity that provide \\u03d5Kp24 with the ability to infect a broad panel of capsular polysaccharide (CPS) types of Klebsiella pneumoniae . Our study provides structural and functional insight into how \\u03d5Kp24 adapts to the variable surfaces of capsulated bacterial pathogens, which is useful for the development of phage therapy approaches against pan-drug resistant K. pneumoniae strains.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-34972-5\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4297475121\", \"doi_x\": \"10.1002/pro.4442\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"CSM\\u2010peptides: A computational approach to rapid identification of therapeutic peptides\", \"abstract\": \"Abstract Peptides are attractive alternatives for the development of new therapeutic strategies due to their versatility and low complexity of synthesis. Increasing interest in these molecules has led to the creation of large collections of experimentally characterized therapeutic peptides, which greatly contributes to development of data\\u2010driven computational approaches. Here we propose CSM\\u2010peptides, a novel machine learning method for rapid identification of eight different types of therapeutic peptides: anti\\u2010angiogenic, anti\\u2010bacterial, anti\\u2010cancer, anti\\u2010inflammatory, anti\\u2010viral, cell\\u2010penetrating, quorum sensing, and surface binding. Our method has shown to outperform existing approaches, achieving an AUC of up to 0.92 on independent blind tests, and consistent performance on cross\\u2010validation. We anticipate CSM\\u2010peptides to be of great value in helping screening large libraries to identify novel peptides with therapeutic potential and have made it freely available as a user\\u2010friendly web server and Application Programming Interface at https://biosig.lab.uq.edu.au/csm_peptides .\", \"doi_y\": \"https://doi.org/10.1002/pro.4442\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W3202009088\", \"doi_x\": \"10.1098/rsob.210222\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Intrinsically disordered proteins: modes of binding with emphasis on disordered domains\", \"abstract\": \"Our notions of protein function have long been determined by the protein structure-function paradigm. However, the idea that protein function is dictated by a prerequisite complementarity of shapes at the binding interface is becoming increasingly challenged. Interactions involving intrinsically disordered proteins (IDPs) have indicated a significant degree of disorder present in the bound state, ranging from static disorder to complete disorder, termed 'random fuzziness'. This review assesses the anatomy of an IDP and relates how its intrinsic properties permit promiscuity and allow for the various modes of interaction. Furthermore, a mechanistic overview of the types of disordered domains is detailed, while also relating to a recent example and the kinetic and thermodynamic principles governing its formation.\", \"doi_y\": \"https://doi.org/10.1098/rsob.210222\", \"publication_date\": \"2021-10-01\"}, {\"work_id\": \"W4306353420\", \"doi_x\": \"10.1093/nar/gkac896\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"abstract\": \"RNA G-quadruplex (rG4) is a vital RNA tertiary structure motif that involves the base pairs on both Hoogsteen and Watson-Crick faces of guanines. rG4 is of great importance in the post-transcriptional regulation of gene expression. Experimental technologies have advanced to identify in vitro and in vivo rG4s across diverse transcriptomes. Building on these recent advances, here we present G4Atlas, the first transcriptome-wide G-quadruplex database, in which we have collated, classified, and visualized transcriptome rG4 experimental data, generated from rG4-seq, chemical profiling and ligand-binding methods. Our comprehensive database includes transcriptome-wide rG4s generated from 82 experimental treatments and 238 samples across ten species. In addition, we have also included RNA secondary structure prediction information across both experimentally identified and unidentified rG4s to enable users to display any potential competitive folding between rG4 and RNA secondary structures. As such, G4Atlas will enable users to explore the general functions of rG4s in diverse biological processes. In addition, G4Atlas lays the foundation for further data-driven deep learning algorithms to examine rG4 structural features.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac896\", \"publication_date\": \"2022-10-16\"}, {\"work_id\": \"W4378211582\", \"doi_x\": \"10.1016/j.jmb.2023.168160\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Computational Resources for Molecular Biology 2023\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168160\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4309506674\", \"doi_x\": \"10.1093/nar/gkac1052\", \"af_direct_citation\": 0, \"af_indirect_citations\": 5, \"title\": \"UniProt: the Universal Protein Knowledgebase in 2023\", \"abstract\": \"The aim of the UniProt Knowledgebase is to provide users with a comprehensive, high-quality and freely accessible set of protein sequences annotated with functional information. In this publication we describe enhancements made to our data processing pipeline and to our website to adapt to an ever-increasing information content. The number of sequences in UniProtKB has risen to over 227 million and we are working towards including a reference proteome for each taxonomic group. We continue to extract detailed annotations from the literature to update or create reviewed entries, while unreviewed entries are supplemented with annotations provided by automated systems using a variety of machine-learning techniques. In addition, the scientific community continues their contributions of publications and annotations to UniProt entries of their interest. Finally, we describe our new website (https://www.uniprot.org/), designed to enhance our users' experience and make our data easily accessible to the research community. This interface includes access to AlphaFold structures for more than 85% of all entries as well as improved visualisations for subcellular localisation of proteins.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac1052\", \"publication_date\": \"2022-11-21\"}, {\"work_id\": \"W4319842310\", \"doi_x\": \"10.1039/d2cb00231k\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A primer to directed evolution: current methodologies and future directions\", \"abstract\": \"This review summarises the methods available for directed evolution, including mutagenesis and variant selection techniques. The advantages and disadvantages of each technique are presented, and future challenges in the field are discussed.\", \"doi_y\": \"https://doi.org/10.1039/d2cb00231k\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385263863\", \"doi_x\": \"10.1002/prot.26545\", \"af_direct_citation\": 1, \"af_indirect_citations\": 16, \"title\": \"Protein target highlights in <scp>CASP15</scp>: Analysis of models by structure providers\", \"abstract\": \"Abstract We present an in\\u2010depth analysis of selected CASP15 targets, focusing on their biological and functional significance. The authors of the structures identify and discuss key protein features and evaluate how effectively these aspects were captured in the submitted predictions. While the overall ability to predict three\\u2010dimensional protein structures continues to impress, reproducing uncommon features not previously observed in experimental structures is still a challenge. Furthermore, instances with conformational flexibility and large multimeric complexes highlight the need for novel scoring strategies to better emphasize biologically relevant structural regions. Looking ahead, closer integration of computational and experimental techniques will play a key role in determining the next challenges to be unraveled in the field of structural molecular biology.\", \"doi_y\": \"https://doi.org/10.1002/prot.26545\", \"publication_date\": \"2023-07-26\"}, {\"work_id\": \"W4313340527\", \"doi_x\": \"10.1038/s41467-022-35475-z\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Mechanism of human Lig1 regulation by PCNA in Okazaki fragment sealing\", \"abstract\": \"During lagging strand synthesis, DNA Ligase 1 (Lig1) cooperates with the sliding clamp PCNA to seal the nicks between Okazaki fragments generated by Pol \\u03b4 and Flap endonuclease 1 (FEN1). We present several cryo-EM structures combined with functional assays, showing that human Lig1 recruits PCNA to nicked DNA using two PCNA-interacting motifs (PIPs) located at its disordered N-terminus (PIPN-term) and DNA binding domain (PIPDBD). Once Lig1 and PCNA assemble as two-stack rings encircling DNA, PIPN-term is released from PCNA and only PIPDBD is required for ligation to facilitate the substrate handoff from FEN1. Consistently, we observed that PCNA forms a defined complex with FEN1 and nicked DNA, and it recruits Lig1 to an unoccupied monomer creating a toolbelt that drives the transfer of DNA to Lig1. Collectively, our results provide a structural model on how PCNA regulates FEN1 and Lig1 during Okazaki fragments maturation.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-35475-z\", \"publication_date\": \"2022-12-20\"}, {\"work_id\": \"W4386442752\", \"doi_x\": \"10.1021/acscatal.3c02278\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Structure-Based Design of Small Imine Reductase Panels for Target Substrates\", \"abstract\": \"Biocatalysis is important in the discovery, development, and manufacture of pharmaceuticals. However, the identification of enzymes for target transformations of interest requires major screening efforts. Here, we report a structure-based computational workflow to prioritize protein sequences by a score based on predicted activities on substrates, thereby reducing a resource-intensive laboratory-based biocatalyst screening. We selected imine reductases (IREDs) as a class of biocatalysts to illustrate the application of the computational workflow termed IREDFisher. Validation by using published data showed that IREDFisher can retrieve the best enzymes and increase the hit rate by identifying the top 20 ranked sequences. The power of IREDFisher is confirmed by computationally screening 1400 sequences for chosen reductive amination reactions with different levels of complexity. Highly active IREDs were identified by only testing 20 samples in vitro. Our speed test shows that it only takes 90 min to rank 85 sequences from user input and 30 min for the established IREDFisher database containing 591 IRED sequences. IREDFisher is available as a user-friendly web interface (https://enzymeevolver.com/IREDFisher). IREDFisher enables the rapid discovery of IREDs for applications in synthesis and directed evolution studies, with minimal time and resource expenditure. Future use of the workflow with other enzyme families could be implemented following the modification of the workflow scoring function.\", \"doi_y\": \"https://doi.org/10.1021/acscatal.3c02278\", \"publication_date\": \"2023-09-05\"}, {\"work_id\": \"W4304974831\", \"doi_x\": \"10.1038/s41591-022-02078-6\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Human leukocyte antigen alleles associate with COVID-19 vaccine immunogenicity and risk of breakthrough infection\", \"abstract\": \"Abstract Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) vaccine immunogenicity varies between individuals, and immune responses correlate with vaccine efficacy. Using data from 1,076 participants enrolled in ChAdOx1 nCov-19 vaccine efficacy trials in the United Kingdom, we found that inter-individual variation in normalized antibody responses against SARS-CoV-2 spike and its receptor-binding domain (RBD) at 28 days after first vaccination shows genome-wide significant association with major histocompatibility complex (MHC) class II alleles. The most statistically significant association with higher levels of anti-RBD antibody was HLA-DQB1*06 ( P = 3.2 \\u00d7 10 \\u22129 ), which we replicated in 1,677 additional vaccinees. Individuals carrying HLA-DQB1*06 alleles were less likely to experience PCR-confirmed breakthrough infection during the ancestral SARS-CoV-2 virus and subsequent Alpha variant waves compared to non-carriers (hazard ratio = 0.63, 0.42\\u20130.93, P = 0.02). We identified a distinct spike-derived peptide that is predicted to bind differentially to HLA-DQB1*06 compared to other similar alleles, and we found evidence of increased spike-specific memory B cell responses in HLA-DQB1*06 carriers at 84 days after first vaccination. Our results demonstrate association of HLA type with Coronavirus Disease 2019 (COVID-19) vaccine antibody response and risk of breakthrough infection, with implications for future vaccine design and implementation.\", \"doi_y\": \"https://doi.org/10.1038/s41591-022-02078-6\", \"publication_date\": \"2022-10-13\"}, {\"work_id\": \"W4385443358\", \"doi_x\": \"10.1021/acs.jmedchem.3c01114\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"5-Substituted Pyridine-2,4-dicarboxylate Derivatives Have Potential for Selective Inhibition of Human Jumonji-C Domain-Containing Protein 5\", \"abstract\": \"Jumonji-C domain-containing protein 5 (JMJD5) is a 2-oxoglutarate (2OG)-dependent oxygenase that plays important roles in development, circadian rhythm, and cancer through unclear mechanisms. JMJD5 has been reported to have activity as a histone protease, as an N\\u03b5-methyl lysine demethylase, and as an arginine residue hydroxylase. Small-molecule JMJD5-selective inhibitors will be useful for investigating its (patho)physiological roles. Following the observation that the broad-spectrum 2OG oxygenase inhibitor pyridine-2,4-dicarboxylic acid (2,4-PDCA) is a 2OG-competing JMJD5 inhibitor, we report that 5-aminoalkyl-substituted 2,4-PDCA derivatives are potent JMJD5 inhibitors manifesting selectivity for JMJD5 over other human 2OG oxygenases. Crystallographic analyses with five inhibitors imply induced fit binding and reveal that the 2,4-PDCA C5 substituent orients into the JMJD5 substrate-binding pocket. Cellular studies indicate that the lead compounds display similar phenotypes as reported for clinically observed JMJD5 variants, which have a reduced catalytic activity compared to wild-type JMJD5.\", \"doi_y\": \"https://doi.org/10.1021/acs.jmedchem.3c01114\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4200558169\", \"doi_x\": \"10.1101/2021.12.20.473413\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrated view and comparative analysis of baseline protein expression in mouse and rat tissues\", \"abstract\": \"Abstract The increasingly large amount of proteomics data in the public domain enables, among other applications, the combined analyses of datasets to create comparative protein expression maps covering different organisms and different biological conditions. Here we have reanalysed public proteomics datasets from mouse and rat tissues (14 and 9 datasets, respectively), to assess baseline protein abundance. Overall, the aggregated dataset contained 23 individual datasets, including a total of 211 samples coming from 34 different tissues across 14 organs, comprising 9 mouse and 3 rat strains, respectively. In all cases, we studied the distribution of canonical proteins between the different organs. The number of canonical proteins per dataset ranged from 273 (tendon) and 9,715 (liver) in mouse, and from 101 (tendon) and 6,130 (kidney) in rat. Then, we studied how protein abundances compared across different datasets and organs for both species. As a key point we carried out a comparative analysis of protein expression between mouse, rat and human tissues. We observed a high level of correlation of protein expression among orthologs between all three species in brain, kidney, heart and liver samples, whereas the correlation of protein expression was generally slightly lower between organs within the same species. Protein expression results have been integrated into the resource Expression Atlas for widespread dissemination. Author summary We have reanalysed 23 baseline mass spectrometry-based public proteomics datasets stored in the PRIDE database. Overall, the aggregated dataset contained 211 samples, coming from 34 different tissues across 14 organs, comprising 9 mouse and 3 rat strains, respectively. We analysed the distribution of protein expression across organs in both species. We also studied how protein abundances compared across different datasets and organs for both species. Then we performed gene ontology and pathway enrichment analyses to identify enriched biological processes and pathways across organs. We also carried out a comparative analysis of baseline protein expression across mouse, rat and human tissues, observing a high level of expression correlation among orthologs in all three species, in brain, kidney, heart and liver samples. To disseminate these findings, we have integrated the protein expression results into the resource Expression Atlas.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.20.473413\", \"publication_date\": \"2021-12-21\"}, {\"work_id\": \"W4306412156\", \"doi_x\": \"10.1038/s41564-022-01215-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The cell biology of archaea\", \"abstract\": \"The past decade has revealed the diversity and ubiquity of archaea in nature, with a growing number of studies highlighting their importance in ecology, biotechnology and even human health. Myriad lineages have been discovered, which expanded the phylogenetic breadth of archaea and revealed their central role in the evolutionary origins of eukaryotes. These discoveries, coupled with advances that enable the culturing and live imaging of archaeal cells under extreme environments, have underpinned a better understanding of their biology. In this Review we focus on the shape, internal organization and surface structures that are characteristic of archaeal cells as well as membrane remodelling, cell growth and division. We also highlight some of the technical challenges faced and discuss how new and improved technologies will help address many of the key unanswered questions.\", \"doi_y\": \"https://doi.org/10.1038/s41564-022-01215-8\", \"publication_date\": \"2022-10-17\"}, {\"work_id\": \"W4384663047\", \"doi_x\": \"10.1038/s41467-023-39615-x\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Conformational restriction shapes the inhibition of a multidrug efflux adaptor protein\", \"abstract\": \"Membrane efflux pumps play a major role in bacterial multidrug resistance. The tripartite multidrug efflux pump system from Escherichia coli, AcrAB-TolC, is a target for inhibition to lessen resistance development and restore antibiotic efficacy, with homologs in other ESKAPE pathogens. Here, we rationalize a mechanism of inhibition against the periplasmic adaptor protein, AcrA, using a combination of hydrogen/deuterium exchange mass spectrometry, cellular efflux assays, and molecular dynamics simulations. We define the structural dynamics of AcrA and find that an inhibitor can inflict long-range stabilisation across all four of its domains, whereas an interacting efflux substrate has minimal effect. Our results support a model where an inhibitor forms a molecular wedge within a cleft between the lipoyl and \\u03b1\\u03b2 barrel domains of AcrA, diminishing its conformational transmission of drug-evoked signals from AcrB to TolC. This work provides molecular insights into multidrug adaptor protein function which could be valuable for developing antimicrobial therapeutics.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-39615-x\", \"publication_date\": \"2023-07-18\"}, {\"work_id\": \"W4309862298\", \"doi_x\": \"10.7554/elife.76387\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"abstract\": \"EROS (essential for reactive oxygen species) protein is indispensable for expression of gp91phox, the catalytic core of the phagocyte NADPH oxidase. EROS deficiency in humans is a novel cause of the severe immunodeficiency, chronic granulomatous disease, but its mechanism of action was unknown until now. We elucidate the role of EROS, showing it acts at the earliest stages of gp91phox maturation. It binds the immature 58 kDa gp91phox directly, preventing gp91phox degradation and allowing glycosylation via the oligosaccharyltransferase machinery and the incorporation of the heme prosthetic groups essential for catalysis. EROS also regulates the purine receptors P2X7 and P2X1 through direct interactions, and P2X7 is almost absent in EROS-deficient mouse and human primary cells. Accordingly, lack of murine EROS results in markedly abnormal P2X7 signalling, inflammasome activation, and T cell responses. The loss of both ROS and P2X7 signalling leads to resistance to influenza infection in mice. Our work identifies EROS as a highly selective chaperone for key proteins in innate and adaptive immunity and a rheostat for immunity to infection. It has profound implications for our understanding of immune physiology, ROS dysregulation, and possibly gene therapy.\", \"doi_y\": \"https://doi.org/10.7554/elife.76387\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4322764113\", \"doi_x\": \"10.1126/science.abn4116\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NLR immune receptor\\u2013nanobody fusions confer plant disease resistance\", \"abstract\": \"Plant pathogens cause recurrent epidemics, threatening crop yield and global food security. Efforts to retool the plant immune system have been limited to modifying natural components and can be nullified by the emergence of new pathogen strains. Made-to-order synthetic plant immune receptors provide an opportunity to tailor resistance to pathogen genotypes present in the field. In this work, we show that plant nucleotide-binding, leucine-rich repeat immune receptors (NLRs) can be used as scaffolds for nanobody (single-domain antibody fragment) fusions that bind fluorescent proteins (FPs). These fusions trigger immune responses in the presence of the corresponding FP and confer resistance against plant viruses expressing FPs. Because nanobodies can be raised against most molecules, immune receptor\\u2013nanobody fusions have the potential to generate resistance against plant pathogens and pests delivering effectors inside host cells.\", \"doi_y\": \"https://doi.org/10.1126/science.abn4116\", \"publication_date\": \"2023-03-03\"}, {\"work_id\": \"W4313237556\", \"doi_x\": \"10.7554/elife.82207\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A choline-releasing glycerophosphodiesterase essential for phosphatidylcholine biosynthesis and blood stage development in the malaria parasite\", \"abstract\": \"The malaria parasite Plasmodium falciparum synthesizes significant amounts of phospholipids to meet the demands of replication within red blood cells. De novo phosphatidylcholine (PC) biosynthesis via the Kennedy pathway is essential, requiring choline that is primarily sourced from host serum lysophosphatidylcholine (lysoPC). LysoPC also acts as an environmental sensor to regulate parasite sexual differentiation. Despite these critical roles for host lysoPC, the enzyme(s) involved in its breakdown to free choline for PC synthesis are unknown. Here, we show that a parasite glycerophosphodiesterase (PfGDPD) is indispensable for blood stage parasite proliferation. Exogenous choline rescues growth of PfGDPD-null parasites, directly linking PfGDPD function to choline incorporation. Genetic ablation of PfGDPD reduces choline uptake from lysoPC, resulting in depletion of several PC species in the parasite, whilst purified PfGDPD releases choline from glycerophosphocholine in vitro. Our results identify PfGDPD as a choline-releasing glycerophosphodiesterase that mediates a critical step in PC biosynthesis and parasite survival.Malaria kills over half a million people every year worldwide. A single-celled parasite called Plasmodium falciparum is responsible for the most lethal form of the disease. This malaria-causing agent is carried by mosquitos which transmit the parasite to humans through their bite. Once in the bloodstream, the parasite enters red blood cells and starts to replicate so it can go on to infect other cells. Like our cells, P. falciparum is surrounded by a membrane, and further membranes surround a number of its internal compartments. To make these protective coats, the parasite has to gather a nutrient called choline to form an important building block in the membrane. The parasite gets most of its choline by absorbing and digesting a molecule known as lysoPC found in the bloodstream of its host. However, it was unclear precisely how the parasite achieves this. To address this question, Ramaprasad, Burda et al. used genetic and metabolomic approaches to study how P. falciparum breaks down lysoPC. The experiments found that mutant parasites that are unable to make an enzyme called GDPD were able to infect red blood cells, but failed to grow properly once inside the cells. The mutant parasites took up less choline and, as a result, also made fewer membrane building blocks. The team were able to rescue the mutant parasites by supplying them with large quantities of choline, which allowed them to resume growing. Taken together, the findings of Ramaprasad, Burda et al. suggest that P. falciparum uses GDPD to extract choline from lysoPC when it is living in red blood cells. More and more P. falciparum parasites are becoming resistant to many of the drugs currently being used to treat malaria. One solution is to develop new therapies that target different molecules in the parasite. Since it performs such a vital role, GDPD may have the potential to be a future drug target.\", \"doi_y\": \"https://doi.org/10.7554/elife.82207\", \"publication_date\": \"2022-12-28\"}, {\"work_id\": \"W4306835822\", \"doi_x\": \"10.1101/2022.10.19.512840\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The N-terminal executioner domains of NLR immune receptors are functionally conserved across major plant lineages\", \"abstract\": \"ABSTRACT Nucleotide-binding domain and leucine-rich repeat (NLR) proteins are a prominent class of intracellular immune receptors that are present across diverse plant lineages. However, our understanding of plant NLR structure and function is limited to the evolutionarily young flowering plant clade (angiosperms). Here, we describe an extended spectrum of NLR diversity across major plant lineages and demonstrate functional conservation of N-terminal \\u2018executioner\\u2019 domains that trigger immune responses. We show that broadly distributed CC (coiled-coil) and TIR (toll/interleukin-1 receptor) domains retain executioner function through trans-lineage activation of immune-related cell death in the model angiosperm Nicotiana benthamiana . Further examination of a CC subfamily specific to non-flowering lineages uncovered an essential N-terminal MAEPL motif with functional similarity to resistosome-forming CC-NLRs. Ectopic activation of the MAEPL-type CC in the divergent liverwort Marchantia polymorpha led to profound growth inhibition, defense gene activation, and signatures of cell death resembling CC activity in flowering plants. Moreover, comparative macroevolutionary transcriptomics in Marchantia and Nicotiana identified conserved CC responsive genes, providing further insight into the core aspects of CC function shared between flowering and non-flowering plants. Our findings highlight the need to understand NLR structure and function across the full spectrum of plant diversity.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.19.512840\", \"publication_date\": \"2022-10-19\"}, {\"work_id\": \"W4282926916\", \"doi_x\": \"10.1093/nar/gkac381\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"CSM-Potential: mapping protein interactions and biological ligands in 3D space using geometric deep learning\", \"abstract\": \"Recent advances in protein structural modelling have enabled the accurate prediction of the holo 3D structures of almost any protein, however protein function is intrinsically linked to the interactions it makes. While a number of computational approaches have been proposed to explore potential biological interactions, they have been limited to specific interactions, and have not been readily accessible for non-experts or use in bioinformatics pipelines. Here we present CSM-Potential, a geometric deep learning approach to identify regions of a protein surface that are likely to mediate protein-protein and protein-ligand interactions in order to provide a link between 3D structure and biological function. Our method has shown robust performance, outperforming existing methods for both predictive tasks. By assessing the performance of CSM-Potential on independent blind tests, we show that our method was able to achieve ROC AUC values of up to 0.81 for the identification of potential protein-protein binding sites, and up to 0.96 accuracy on biological ligand classification. Our method is freely available as a user-friendly and easy-to-use web server and API at http://biosig.unimelb.edu.au/csm_potential.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac381\", \"publication_date\": \"2022-05-24\"}, {\"work_id\": \"W4213244137\", \"doi_x\": \"10.1080/19420862.2021.2020082\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Current advances in biopharmaceutical informatics: guidelines, impact and challenges in the computational developability assessment of antibody therapeutics\", \"abstract\": \"Therapeutic monoclonal antibodies and their derivatives are key components of clinical pipelines in the global biopharmaceutical industry. The availability of large datasets of antibody sequences, structures, and biophysical properties is increasingly enabling the development of predictive models and computational tools for the \\\"developability assessment\\\" of antibody drug candidates. Here, we provide an overview of the antibody informatics tools applicable to the prediction of developability issues such as stability, aggregation, immunogenicity, and chemical degradation. We further evaluate the opportunities and challenges of using biopharmaceutical informatics for drug discovery and optimization. Finally, we discuss the potential of developability guidelines based on in silico metrics that can be used for the assessment of antibody stability and manufacturability.\", \"doi_y\": \"https://doi.org/10.1080/19420862.2021.2020082\", \"publication_date\": \"2022-02-01\"}, {\"work_id\": \"W4386499094\", \"doi_x\": \"10.1016/j.jgsce.2023.205104\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The rise of the machines: A state-of-the-art technical review on process modelling and machine learning within hydrogen production with carbon capture\", \"abstract\": \"This study aims to present a compendious yet technical scrutiny of the current trends in process modelling as well as the implementation of machine learning within combined hydrogen production and carbon capture (i.e. blue hydrogen). The paper is intended to accurately portray the role that machine learning is anticipated to play within research and development in blue hydrogen production in the forthcoming years. This covers the implementation of machine learning at both material and process development levels. The paper provides a concise overview of the current trends in blue hydrogen production, as well as an intro to machine learning and process modelling within the same context. We have reinforced our paper by first summarising a brief description of the key \\u201ctools\\u201d used in machine learning and process modelling, before painstakingly examining the implementation of these techniques in blue hydrogen production and the less-discovered merits and de-merits. Ultimately, the paper depicts a clear picture of the advancements in machine learning and the major role it is expected to play in accelerating research and development in blue hydrogen production on both material and process development fronts. The paper strives to shed some light on the key advantages that machine learning has to offer in blue hydrogen for future research work.\", \"doi_y\": \"https://doi.org/10.1016/j.jgsce.2023.205104\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4361766435\", \"doi_x\": \"10.1039/d2sm01225a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Diversity of viscoelastic properties of an engineered muscle-inspired protein hydrogel\", \"abstract\": \"Through a multi-modal rheology approach, we show that muscle-inspired hydrogels exhibit a surprising diversity of viscoelastic response from power-law rheology at low strains to stiffening and energy dissipation at high strains.\", \"doi_y\": \"https://doi.org/10.1039/d2sm01225a\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385819122\", \"doi_x\": \"10.1039/d3cb00107e\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Exploring a <i>Streptomyces</i> wax synthase using acyl-SNACs as donor substrates\", \"abstract\": \"The demand of fragrance and food industries for short/branched wax esters is increasing due to their rich scent and low toxicity. Wax synthase and acyl-CoA:diacylglycerol O-acyltransferase (WS/DGAT) are a family of bacterial enzymes capable of catalysing the production of wax esters. Here, we report that a WS/DGAT from Streptomyces coelicolor is able to mediate the reactions between alcohol acceptors and synthetic acyl-donor mimics, acyl-SNACs. The enzyme displayed considerable substrate tolerance towards acyl-donors with structural diversity. Structural modelling-guided site directed mutagenesis resulted in a variant, L25F, the catalytic efficiency of which was improved toward aromatic, short-linear, and branched acyl-donors compared to the wild type.\", \"doi_y\": \"https://doi.org/10.1039/d3cb00107e\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385972446\", \"doi_x\": \"10.1107/s1600576723006428\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A versatile approach to high-density microcrystals in lipidic cubic phase for room-temperature serial crystallography\", \"abstract\": \"Serial crystallography has emerged as an important tool for structural studies of integral membrane proteins. The ability to collect data from micrometre-sized weakly diffracting crystals at room temperature with minimal radiation damage has opened many new opportunities in time-resolved studies and drug discovery. However, the production of integral membrane protein microcrystals in lipidic cubic phase at the desired crystal density and quantity is challenging. This paper introduces VIALS (versatile approach to high-density microcrystals in lipidic cubic phase for serial crystallography), a simple, fast and efficient method for preparing hundreds of microlitres of high-density microcrystals suitable for serial X-ray diffraction experiments at both synchrotron and free-electron laser sources. The method is also of great benefit for rational structure-based drug design as it facilitates in situ crystal soaking and rapid determination of many co-crystal structures. Using the VIALS approach, room-temperature structures are reported of (i) the archaerhodopsin-3 protein in its dark-adapted state and 110 ns photocycle intermediate, determined to 2.2 and 1.7 \\u00c5, respectively, and (ii) the human A2A adenosine receptor in complex with two different ligands determined to a resolution of 3.5 \\u00c5.\", \"doi_y\": \"https://doi.org/10.1107/s1600576723006428\", \"publication_date\": \"2023-08-18\"}, {\"work_id\": \"W4308653678\", \"doi_x\": \"10.1038/s41467-022-34610-0\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Cryo-EM reveals the architecture of the PELP1-WDR18 molecular scaffold\", \"abstract\": \"PELP1 (Proline-, Glutamic acid-, Leucine-rich protein 1) is a large scaffolding protein that functions in many cellular pathways including steroid receptor (SR) coactivation, heterochromatin maintenance, and ribosome biogenesis. PELP1 is a proto-oncogene whose expression is upregulated in many human cancers, but how the PELP1 scaffold coordinates its diverse cellular functions is poorly understood. Here we show that PELP1 serves as the central scaffold for the human Rix1 complex whose members include WDR18, TEX10, and SENP3. We reconstitute the mammalian Rix1 complex and identified a stable sub-complex comprised of the conserved PELP1 Rix1 domain and WDR18. We determine a 2.7 \\u00c5 cryo-EM structure of the subcomplex revealing an interconnected tetrameric assembly and the architecture of PELP1's signaling motifs, including eleven LxxLL motifs previously implicated in SR signaling and coactivation of Estrogen Receptor alpha (ER\\u03b1) mediated transcription. However, the structure shows that none of these motifs is in a conformation that would support SR binding. Together this work establishes that PELP1 scaffolds the Rix1 complex, and association with WDR18 may direct PELP1's activity away from SR coactivation.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-34610-0\", \"publication_date\": \"2022-11-09\"}, {\"work_id\": \"W4306790272\", \"doi_x\": \"10.1093/nar/gkac904\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Role of NS2 specific RNA binding and phosphorylation in liquid\\u2013liquid phase separation and virus assembly\", \"abstract\": \"Liquid-liquid phase separation (LLPS) has assumed a prominent role in biological cell systems, where it underpins the formation of subcellular compartments necessary for cell function. We investigated the underlying mechanism of LLPS in virus infected cells, where virus inclusion bodies are formed by an RNA-binding phosphoprotein (NS2) of Bluetongue virus to serve as sites for subviral particle assembly and virus maturation. We show that NS2 undergoes LLPS that is dependent on protein phosphorylation and RNA-binding and that LLPS occurrence is accompanied by a change in protein secondary structure. Site-directed mutagenesis identified two critical arginine residues in NS2 responsible for specific RNA binding and thus for NS2-RNA complex driven LLPS. Reverse genetics identified the same residues as essential for VIB assembly in infected cells and virus viability. Our findings suggest that a specific arginine-RNA interaction in the context of a phosphorylated state drives LLPS in this, and possibly other, virus infections.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac904\", \"publication_date\": \"2022-10-19\"}, {\"work_id\": \"W4297252158\", \"doi_x\": \"10.1016/j.dnarep.2022.103407\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The role of histone H3K36me3 writers, readers and erasers in maintaining genome stability\", \"abstract\": \"Histone Post-Translational Modifications (PTMs) play fundamental roles in mediating DNA-related processes such as transcription, replication and repair. The histone mark H3K36me3 and its associated methyltransferase SETD2 (Set2 in yeast) are archetypical in this regard, performing critical roles in each of these DNA transactions. Here, we present an overview of H3K36me3 regulation and the roles of its writers, readers and erasers in maintaining genome stability through facilitating DNA double-strand break (DSB) repair, checkpoint signalling and replication stress responses. Further, we consider how loss of SETD2 and H3K36me3, frequently observed in a number of different cancer types, can be specifically targeted in the clinic through exploiting loss of particular genome stability functions.\", \"doi_y\": \"https://doi.org/10.1016/j.dnarep.2022.103407\", \"publication_date\": \"2022-11-01\"}, {\"work_id\": \"W4220725479\", \"doi_x\": \"10.1128/aem.00255-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"abstract\": \"The methylotrophic OM43 clade are Gammaproteobacteria that comprise some of the smallest free-living cells known and have highly streamlined genomes. OM43 represents an important microbial link between marine primary production and remineralization of carbon back to the atmosphere. Bacteriophages shape microbial communities and are major drivers of mortality and global marine biogeochemistry. Recent cultivation efforts have brought the first viruses infecting members of the OM43 clade into culture. Here, we characterize a novel myophage infecting OM43 called Melnitz. Melnitz was isolated independently from water samples from a subtropical ocean gyre (Sargasso Sea) and temperate coastal (Western English Channel) systems. Metagenomic recruitment from global ocean viromes confirmed that Melnitz is globally ubiquitous, congruent with patterns of host abundance. Bacteria with streamlined genomes such as OM43 and the globally dominant SAR11 clade use riboswitches as an efficient method to regulate metabolism. Melnitz encodes a two-piece tmRNA (ssrA), controlled by a glutamine riboswitch, providing evidence that riboswitch use also occurs for regulation during phage infection of streamlined heterotrophs. Virally encoded tRNAs and ssrA found in Melnitz were phylogenetically more closely related to those found within the alphaproteobacterial SAR11 clade and their associated myophages than those within their gammaproteobacterial hosts. This suggests the possibility of an ancestral host transition event between SAR11 and OM43. Melnitz and a related myophage that infects SAR11 were unable to infect hosts of the SAR11 and OM43, respectively, suggesting host transition rather than a broadening of host range. IMPORTANCE Isolation and cultivation of viruses are the foundations on which the mechanistic understanding of virus-host interactions and parameterization of bioinformatic tools for viral ecology are based. This study isolated and characterized the first myophage known to infect the OM43 clade, expanding our knowledge of this understudied group of microbes. The nearly identical genomes of four strains of Melnitz isolated from different marine provinces and the global abundance estimations from metagenomic data suggest that this viral population is globally ubiquitous. Genome analysis revealed several unusual features in Melnitz and related genomes recovered from viromes, such as a curli operon and virally encoded tmRNA controlled by a glutamine riboswitch, neither of which are found in the host. Further phylogenetic analysis of shared genes indicates that this group of viruses infecting the gammaproteobacterial OM43 shares a recent common ancestor with viruses infecting the abundant alphaproteobacterial SAR11 clade. Host ranges are affected by compatible cell surface receptors, successful circumvention of superinfection exclusion systems, and the presence of required accessory proteins, which typically limits phages to singular narrow groups of closely related bacterial hosts. This study provides intriguing evidence that for streamlined heterotrophic bacteria, virus-host transitioning may not be necessarily restricted to phylogenetically related hosts but is a function of shared physical and biochemical properties of the cell.\", \"doi_y\": \"https://doi.org/10.1128/aem.00255-22\", \"publication_date\": \"2022-04-12\"}, {\"work_id\": \"W4361266887\", \"doi_x\": \"10.1073/pnas.2218823120\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Altered plasma membrane abundance of the sulfatide-binding protein NF155 links glycosphingolipid imbalances to demyelination\", \"abstract\": \"Myelin is a multilayered membrane that tightly wraps neuronal axons, enabling efficient, high-speed signal propagation. The axon and myelin sheath form tight contacts, mediated by specific plasma membrane proteins and lipids, and disruption of these contacts causes devastating demyelinating diseases. Using two cell-based models of demyelinating sphingolipidoses, we demonstrate that altered lipid metabolism changes the abundance of specific plasma membrane proteins. These altered membrane proteins have known roles in cell adhesion and signaling, with several implicated in neurological diseases. The cell surface abundance of the adhesion molecule neurofascin (NFASC), a protein critical for the maintenance of myelin-axon contacts, changes following disruption to sphingolipid metabolism. This provides a direct molecular link between altered lipid abundance and myelin stability. We show that the NFASC isoform NF155, but not NF186, interacts directly and specifically with the sphingolipid sulfatide via multiple binding sites and that this interaction requires the full-length extracellular domain of NF155. We demonstrate that NF155 adopts an S-shaped conformation and preferentially binds sulfatide-containing membranes in cis , with important implications for protein arrangement in the tight axon-myelin space. Our work links glycosphingolipid imbalances to disturbance of membrane protein abundance and demonstrates how this may be driven by direct protein\\u2013lipid interactions, providing a mechanistic framework to understand the pathogenesis of galactosphingolipidoses.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2218823120\", \"publication_date\": \"2023-03-30\"}, {\"work_id\": \"W4307417454\", \"doi_x\": \"10.1101/2022.10.23.22281420\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Age-dependent topic modelling of comorbidities in UK Biobank identifies disease subtypes with differential genetic risk\", \"abstract\": \"Abstract The analysis of longitudinal data from electronic health records (EHR) has potential to improve clinical diagnoses and enable personalised medicine, motivating efforts to identify disease subtypes from age-dependent patient comorbidity information. Here, we introduce an age-dependent topic modelling (ATM) method that provides a low-rank representation of longitudinal records of hundreds of distinct diseases in large EHR data sets. The model learns, and assigns to each individual, topic weights for several disease topics, each of which reflects a set of diseases that tend to co-occur within individuals as a function of age. Simulations show that ATM attains high accuracy in distinguishing distinct age-dependent comorbidity profiles. We applied ATM to 282,957 UK Biobank samples, analysing 1,726,144 disease diagnoses spanning all 348 diseases with \\u22651,000 independent occurrences in the Hospital Episode Statistics (HES) data, identifying 10 disease topics under the optimal model fit. Analysis of an independent cohort, All of Us, with 211,908 samples and 3,098,771 disease diagnoses spanning 233 of the 348 UK Biobank diseases produced highly concordant findings. In UK Biobank we identified 52 diseases with heterogeneous comorbidity profiles (\\u2265500 occurrences assigned to each of \\u22652 topics), including breast cancer, type 2 diabetes (T2D), hypertension, and hypercholesterolemia. For most of these diseases, topic assignments were highly age-dependent, suggesting differences in disease aetiology for early-onset vs. late-onset disease. We defined subtypes of the 52 heterogeneous diseases based on the topic assignments, and compared genetic risk across subtypes using polygenic risk scores (PRS). We identified 18 disease subtypes whose PRS differed significantly from other subtypes of the same disease, including a subtype of T2D characterised by cardiovascular comorbidities and a subtype of asthma characterised by dermatological comorbidities. We further identified specific variants underlying these differences such as a T2D-associated SNP in the HMGA2 locus that has a higher odds ratio in the top quartile of cardiovascular topic weight (1.18\\u00b10.02) compared to the bottom quartile (1.00\\u00b10.02) (P=3 \\u00d7 10 - 7 for difference, FDR = 0.0002 &lt; 0.1). In conclusion, ATM identifies disease subtypes with differential genome-wide and locus-specific genetic risk profiles.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.23.22281420\", \"publication_date\": \"2022-10-25\"}, {\"work_id\": \"W4386757821\", \"doi_x\": \"10.1107/s2059798323006514\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"X-ray structure of the metastable SEPT14\\u2013SEPT7 coiled coil reveals a hendecad region crucial for heterodimerization\", \"abstract\": \"Septins are membrane-associated, GTP-binding proteins that are present in most eukaryotes. They polymerize to play important roles as scaffolds and/or diffusion barriers as part of the cytoskeleton. \\u03b1-Helical coiled-coil domains are believed to contribute to septin assembly, and those observed in both human SEPT6 and SEPT8 form antiparallel homodimers. These are not compatible with their parallel heterodimeric organization expected from the current model for protofilament assembly, but they could explain the interfilament cross-bridges observed by microscopy. Here, the first structure of a heterodimeric septin coiled coil is presented, that between SEPT14 and SEPT7; the former is a SEPT6/SEPT8 homolog. This new structure is parallel, with two long helices that are axially shifted by a full helical turn with reference to their sequence alignment. The structure also has unusual knobs-into-holes packing of side chains. Both standard seven-residue (heptad) and the less common 11-residue (hendecad) repeats are present, creating two distinct regions with opposite supercoiling, which gives rise to an overall straight coiled coil. Part of the hendecad region is required for heterodimerization and therefore may be crucial for selective septin recognition. These unconventional sequences and structural features produce a metastable heterocomplex that nonetheless has enough specificity to promote correct protofilament assembly. For instance, the lack of supercoiling may facilitate unzipping and transitioning to the antiparallel homodimeric state.\", \"doi_y\": \"https://doi.org/10.1107/s2059798323006514\", \"publication_date\": \"2023-09-15\"}, {\"work_id\": \"W4386544136\", \"doi_x\": \"10.1038/s43705-023-00299-5\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Slow growing bacteria survive bacteriophage in isolation\", \"abstract\": \"The interactions between bacteria and bacteriophage have important roles in the global ecosystem; in turn changes in environmental parameters affect the interactions between bacteria and phage. However, there is a lack of knowledge on whether clonal bacterial populations harbour different phenotypes that respond to phage in distinct ways and whether the abundance of such phenotypes within bacterial populations is affected by variations in environmental parameters. Here we study the impact of variations in nutrient availability, bacterial growth rate and phage abundance on the interactions between the phage T4 and individual Escherichia coli cells confined in spatial refuges. Surprisingly, we found that fast growing bacteria survive together with all of their clonal kin cells, whereas slow growing bacteria survive in isolation. We also discovered that the number of bacteria that survive in isolation decreases at increasing phage doses possibly due to lysis inhibition in the presence of secondary adsorptions. We further show that these changes in the phenotypic composition of the E. coli population have important consequences on the bacterial and phage population dynamics and should therefore be considered when investigating bacteria-phage interactions in ecological, health or food production settings in structured environments.\", \"doi_y\": \"https://doi.org/10.1038/s43705-023-00299-5\", \"publication_date\": \"2023-09-08\"}, {\"work_id\": \"W4280631615\", \"doi_x\": \"10.1101/2022.04.26.489605\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural basis of human LRRK2 membrane recruitment and activation\", \"abstract\": \"Summary Mutations in LRRK2 are the most common genetic cause of late-onset Parkinson\\u2019s disease (PD). LRRK2 encodes the leucine-rich repeat kinase 2 (LRRK2), whose kinase activity is regulated by Rab29, a membrane-anchored GTPase. However, molecular mechanisms underlying Rab29-dependent recruitment and activation of LRRK2 remain unclear. Here we report cryo-EM structures of LRRK2\\u2013Rab29 complexes in three oligomeric states, illustrating snapshots of key steps during LRRK2 membrane recruitment and activation. Rab29 binds to the ARM domain of LRRK2, and disruption at the interface abrogates LRRK2 kinase activity. Activation of LRRK2 is underpinned by the formation of an unexpected Rab29-induced super-assembly containing two central kinase-active and two peripheral kinase-inactive LRRK2 protomers. Central protomers undergo pronounced oligomerization-associated rearrangements and adopt an active conformation. Our work reveals the structural mechanism for LRRK2\\u2019s spatial regulation controlled by Rab GTPases, provides mechanistic insights into pathogenic mutations and identifies new opportunities to design LRRK2 inhibitors for PD treatment.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.26.489605\", \"publication_date\": \"2022-04-26\"}, {\"work_id\": \"W4378078262\", \"doi_x\": \"10.1039/d3sm00363a\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Self-assembled aggregates based on cationic amphiphilic peptides: structural insight\", \"abstract\": \"Short and ultra-short peptides have recently emerged as suitable building blocks for the fabrication of self-assembled innovative materials. Peptide aggregation is strictly related to the amino acids composing the sequence and their capability to establish intermolecular interactions. Additional structural and functional properties can also be achieved by peptide derivatization (e.g. with polymeric moieties, alkyl chains or other organic molecules). For instance, peptide amphiphiles (PAs), containing one or more alkyl tails on the backbone, have a propensity to form highly ordered nanostructures like nanotapes, twisted helices, nanotubes and cylindrical nanostructures. Further lateral interactions among peptides can also promote hydrogelation. Here we report the synthesis and the aggregation behaviour of four PAs containing cationic tetra- or hexa-peptides (C19-VAGK, C19-K1, C19-K2 and C19-K3) derivatized with a nonadecanoic alkyl chain. In their acetylated (Ac-) or fluorenylated (Fmoc-) versions, these peptides previously demonstrated the ability to form biocompatible hydrogels potentially suitable as extracellular matrices for tissue engineering or diagnostic MRI applications. In the micromolar range, PAs self-assemble in aqueous solution into nanotapes, or small clusters, resulting in high biocompatibility on HaCat cells up to 72 hours of incubation. Moreover, C19-VAGK also forms a gel at a concentration of 5 wt%.\", \"doi_y\": \"https://doi.org/10.1039/d3sm00363a\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4284992638\", \"doi_x\": \"10.1016/j.jbc.2022.102247\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structural features of the protein kinase domain and targeted binding by small-molecule inhibitors\", \"abstract\": \"Protein kinases are key components in cellular signaling pathways as they carry out the phosphorylation of proteins, primarily on Ser, Thr, and Tyr residues. The catalytic activity of protein kinases is regulated, and they can be thought of as molecular switches that are controlled through protein-protein interactions and post-translational modifications. Protein kinases exhibit diverse structural mechanisms of regulation and have been fascinating subjects for structural biologists from the first crystal structure of a protein kinase over 30 years ago, to recent insights into kinase assemblies enabled by the breakthroughs in cryo-EM. Protein kinases are high-priority targets for drug discovery in oncology and other disease settings, and kinase inhibitors have transformed the outcomes of specific groups of patients. Most kinase inhibitors are ATP competitive, deriving potency by occupying the deep hydrophobic pocket at the heart of the kinase domain. Selectivity of inhibitors depends on exploiting differences between the amino acids that line the ATP site and exploring the surrounding pockets that are present in inactive states of the kinase. More recently, allosteric pockets outside the ATP site are being targeted to achieve high selectivity and to overcome resistance to current therapeutics. Here, we review the key regulatory features of the protein kinase family, describe the different types of kinase inhibitors, and highlight examples where the understanding of kinase regulatory mechanisms has gone hand in hand with the development of inhibitors.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102247\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4311717877\", \"doi_x\": \"10.1021/jacsau.2c00529\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biocatalytic Approaches to Building Blocks for Enzymatic and Chemical Glycan Synthesis\", \"abstract\": \"While the field of biocatalysis has bloomed over the past 20\\u201330 years, advances in the understanding and improvement of carbohydrate-active enzymes, in particular, the sugar nucleotides involved in glycan building block biosynthesis, have progressed relatively more slowly. This perspective highlights the need for further insight into substrate promiscuity and the use of biocatalysis fundamentals (rational design, directed evolution, immobilization) to expand substrate scopes toward such carbohydrate building block syntheses and/or to improve enzyme stability, kinetics, or turnover. Further, it explores the growing premise of using biocatalysis to provide simple, cost-effective access to stereochemically defined carbohydrate materials, which can undergo late-stage chemical functionalization or automated glycan synthesis/polymerization.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.2c00529\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4296344173\", \"doi_x\": \"10.1101/2022.09.16.508329\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genetic, Clinical Underpinnings of Brain Change Along Two Neuroanatomical Dimensions of Clinically-defined Alzheimer\\u2019s Disease\", \"abstract\": \"Abstract Alzheimer\\u2019s disease (AD) is associated with heterogeneous atrophy patterns. We employed a semi-supervised clustering technique known as Surreal-GAN, through which we identified two dominant dimensions of brain atrophy in symptomatic mild cognitive impairment (MCI) and AD patients: the \\u201cdiffuse-AD\\u201d (R1) dimension shows widespread brain atrophy, and the \\u201cMTL-AD\\u201d (R2) dimension displays focal medial temporal lobe (MTL) atrophy. Critically, only R2 was associated with widely known sporadic AD genetic risk factors (e.g., APOE \\u03b54 ) in MCI and AD patients at baseline. We then independently detected the presence of the two dimensions in the early stages by deploying the trained model in the general population and two cognitively unimpaired cohorts of asymptomatic participants. In the general population, genome-wide association studies found 77 genes unrelated to APOE differentially associated with R1 and R2. Functional analyses revealed that these genes were overrepresented in differentially expressed gene sets in organs beyond the brain (R1 and R2), including the heart (R1) and the pituitary gland, muscle, and kidney (R2). These genes were enriched in biological pathways implicated in dendritic cells (R2), macrophage functions (R1), and cancer (R1 and R2). Several of them were \\u201cdruggable genes\\u201d for cancer (R1), inflammation (R1), cardiovascular diseases (R1), and diseases of the nervous system (R2). The longitudinal progression showed that APOE \\u03b54 , amyloid, and tau were associated with R2 at early asymptomatic stages, but this longitudinal association occurs only at late symptomatic stages in R1. Our findings deepen our understanding of the multifaceted pathogenesis of AD beyond the brain. In early asymptomatic stages, the two dimensions are associated with diverse pathological mechanisms, including cardiovascular diseases, inflammation, and hormonal dysfunction \\u2013 driven by genes different from APOE \\u2013 which may collectively contribute to the early pathogenesis of AD.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.16.508329\", \"publication_date\": \"2022-09-19\"}, {\"work_id\": \"W4311547358\", \"doi_x\": \"10.1042/bcj20220474\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"abstract\": \"Pseudokinases, so named because they lack one or more conserved canonical amino acids that define their catalytically active relatives, have evolved a variety of biological functions in both prokaryotic and eukaryotic organisms. Human PSKH2 is closely related to the canonical kinase PSKH1, which maps to the CAMK family of protein kinases. Primates encode PSKH2 in the form of a pseudokinase, which is predicted to be catalytically inactive due to loss of the invariant catalytic Asp residue. Although the biological role(s) of vertebrate PSKH2 proteins remains unclear, we previously identified species-level adaptions in PSKH2 that have led to the appearance of kinase or pseudokinase variants in vertebrate genomes alongside a canonical PSKH1 paralog. In this paper we confirm that, as predicted, PSKH2 lacks detectable protein phosphotransferase activity, and exploit structural informatics, biochemistry and cellular proteomics to begin to characterise vertebrate PSKH2 orthologues. AlphaFold 2-based structural analysis predicts functional roles for both the PSKH2 N- and C-regions that flank the pseudokinase domain core, and cellular truncation analysis confirms that the N-terminal domain, which contains a conserved myristoylation site, is required for both stable human PSKH2 expression and localisation to a membrane-rich subcellular fraction containing mitochondrial proteins. Using mass spectrometry-based proteomics, we confirm that human PSKH2 is part of a cellular mitochondrial protein network, and that its expression is regulated through client-status within the HSP90/Cdc37 molecular chaperone system. HSP90 interactions are mediated through binding to the PSKH2 C-terminal tail, leading us to predict that this region might act as both a cis and trans regulatory element, driving outputs linked to the PSKH2 pseudokinase domain that are important for functional signalling.\", \"doi_y\": \"https://doi.org/10.1042/bcj20220474\", \"publication_date\": \"2023-01-23\"}, {\"work_id\": \"W4220769299\", \"doi_x\": \"10.1101/2022.03.07.483185\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Moonlighting proteins activate transformation in multidrug-resistant Streptococcus pneumoniae epigenetic phase variants\", \"abstract\": \"Abstract Despite enabling Streptococcus pneumoniae to acquire antibiotic resistance and evade vaccine-induced immunity, transformation occurs at variable rates across pneumococci. Phase variants of isolate RMV7, distinguished by altered methylation patterns driven by the translocating variable restriction-modification ( tvr ) locus, differed significantly in their transformation efficiencies and biofilm thicknesses. These differences were replicated when the corresponding tvr alleles were introduced into an RMV7 derivative lacking the locus. RNA-seq identified differential expression of the type 1 pilus, causing the variation in biofilm formation, and inhibition of competence induction in the less transformable variant, RMV7 domi . This was partly attributable to lower expression of ManLMN in RMV7 domi , which promoted competence induction through importing N -acetylglucosamine. This effect was potentiated by orthologues of the gram-negative competence regulatory machinery. Furthermore, a phage-related chromosomal island was more active in RMV7 domi , which inhibited transformation by increasing expression of the stress response proteins ClpP and HrcA. However, HrcA increased competence induction in the other variant, with its effects depending on Ca 2+ supplementation or heat shock. Hence the heterogeneity in transformation efficiency likely reflects the diverse signalling pathways by which it is affected. This regulatory complexity will modulate population-wide responses to synchronising quorum sensing signals to produce co-ordinated yet stochastic \\u201cbet hedging\\u201d behaviour.\", \"doi_y\": \"https://doi.org/10.1101/2022.03.07.483185\", \"publication_date\": \"2022-03-07\"}, {\"work_id\": \"W4307842580\", \"doi_x\": \"10.1016/j.gim.2022.09.010\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Brain monoamine vesicular transport disease caused by homozygous SLC18A2 variants: A study in 42 affected individuals\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.gim.2022.09.010\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4386596810\", \"doi_x\": \"10.1038/s41467-023-40921-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Building block aspect ratio controls assembly, architecture, and mechanics of synthetic and natural protein networks\", \"abstract\": \"Abstract Fibrous networks constructed from high aspect ratio protein building blocks are ubiquitous in nature. Despite this ubiquity, the functional advantage of such building blocks over globular proteins is not understood. To answer this question, we engineered hydrogel network building blocks with varying numbers of protein L domains to control the aspect ratio. The mechanical and structural properties of photochemically crosslinked protein L networks were then characterised using shear rheology and small angle neutron scattering. We show that aspect ratio is a crucial property that defines network architecture and mechanics, by shifting the formation from translationally diffusion dominated to rotationally diffusion dominated. Additionally, we demonstrate that a similar transition is observed in the model living system: fibrin blood clot networks. The functional advantages of this transition are increased mechanical strength and the rapid assembly of homogenous networks above a critical protein concentration, crucial for in vivo biological processes such as blood clotting. In addition, manipulating aspect ratio also provides a parameter in the design of future bio-mimetic and bio-inspired materials.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40921-7\", \"publication_date\": \"2023-09-11\"}, {\"work_id\": \"W4309336604\", \"doi_x\": \"10.1016/j.pt.2022.10.003\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Is the incidence of congenital toxoplasmosis declining?\", \"abstract\": \"Prenatal infection with the protozoan parasite Toxoplasma gondii can cause congenital toxoplasmosis (CT), an often fatal or lifelong-disabling condition. Several studies of human populations have reported temporal decreases in seroprevalence, suggesting declining CT incidence. However, the consistency of this trend among diverse populations remains unclear, as does its implication for prenatal screening programmes, the major intervention against CT. Using temporally resolved data on the seroprevalence of T. gondii in various countries, we discuss how the parasite\\u2019s changing epidemiology may affect trends in CT incidence in varying and counterintuitive ways. We argue that parasite stage-specific serology could be helpful for understanding underlying causes of secular changes in seroprevalence. Furthermore, we highlight the importance of updating cost-effectiveness estimates of screening programmes, accounting for neuropsychiatric sequelae.\", \"doi_y\": \"https://doi.org/10.1016/j.pt.2022.10.003\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4309284852\", \"doi_x\": \"10.3389/fncel.2022.1014629\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The MeshCODE to scale\\u2014visualising synaptic binary information\", \"abstract\": \"The Mercator projection map of the world provides a useful, but distorted, view of the relative scale of countries. Current cellular models suffer from a similar distortion. Here, we undertook an in-depth structural analysis of the molecular dimensions in the cell\\u2019s computational machinery, the MeshCODE, that is assembled from a meshwork of binary switches in the scaffolding proteins talin and vinculin. Talin contains a series of force-dependent binary switches and each domain switching state introduces quantised step-changes in talin length on a micrometre scale. The average dendritic spine is 1 \\u03bcm in diameter so this analysis identifies a plausible Gearbox-like mechanism for dynamic regulation of synaptic function, whereby the positioning of enzymes and substrates relative to each other, mechanically-encoded by the MeshCODE switch patterns, might control synaptic transmission. Based on biophysical rules and experimentally derived distances, this analysis yields a novel perspective on biological digital information.\", \"doi_y\": \"https://doi.org/10.3389/fncel.2022.1014629\", \"publication_date\": \"2022-11-18\"}, {\"work_id\": \"W4285678530\", \"doi_x\": \"10.1016/j.chemolab.2022.104616\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Probabilistic machine learning based soft-sensors for product quality prediction in batch processes\", \"abstract\": \"The major promise of the 4th industrial revolution is encompassed by the utilization of real-time process data to inform operational decision-making. Research focus relating to the monitoring of batch process end product qualities has typically been dominated by the use of latent variable models. In this work, we combined latent variable modeling with probabilistic machine learning methods to construct novel soft-sensors, which are able to synchronously capture nonlinearities expressed within process data, as well as identify accurate uncertainty estimates for predictions. Specifically, we explored the combination of multiway projection to latent structures (MPLS) with Gaussian processes (GPs), Bayesian neural networks (BNNs) and heteroscedastic noise neural networks (HNNs). We demonstrated performance of the soft-sensors for industrial batch process quality control, which involves estimating the end viscosity of different variants of a non-newtonian liquid product manufactured over different periods. Through experimental validation, it is concluded that the use of the MPLS-HNN soft-sensor provides particular promise for industrial batch process monitoring given its high accuracy and reliability, as well as ease for practical implementation.\", \"doi_y\": \"https://doi.org/10.1016/j.chemolab.2022.104616\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4290695246\", \"doi_x\": \"10.1021/acsinfecdis.2c00204\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Two Years into the COVID-19 Pandemic: Lessons Learned\", \"abstract\": \"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a highly transmissible and virulent human-infecting coronavirus that emerged in late December 2019 in Wuhan, China, causing a respiratory disease called coronavirus disease 2019 (COVID-19), which has massively impacted global public health and caused widespread disruption to daily life. The crisis caused by COVID-19 has mobilized scientists and public health authorities across the world to rapidly improve our knowledge about this devastating disease, shedding light on its management and control, and spawned the development of new countermeasures. Here we provide an overview of the state of the art of knowledge gained in the last 2 years about the virus and COVID-19, including its origin and natural reservoir hosts, viral etiology, epidemiology, modes of transmission, clinical manifestations, pathophysiology, diagnosis, treatment, prevention, emerging variants, and vaccines, highlighting important differences from previously known highly pathogenic coronaviruses. We also discuss selected key discoveries from each topic and underline the gaps of knowledge for future investigations.\", \"doi_y\": \"https://doi.org/10.1021/acsinfecdis.2c00204\", \"publication_date\": \"2022-08-08\"}, {\"work_id\": \"W4282839494\", \"doi_x\": \"10.1093/ve/veac054\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Conserved recombination patterns across coronavirus subgenera\", \"abstract\": \"Recombination contributes to the genetic diversity found in coronaviruses and is known to be a prominent mechanism whereby they evolve. It is apparent, both from controlled experiments and in genome sequences sampled from nature, that patterns of recombination in coronaviruses are non-random and that this is likely attributable to a combination of sequence features that favour the occurrence of recombination break points at specific genomic sites, and selection disfavouring the survival of recombinants within which favourable intra-genome interactions have been disrupted. Here we leverage available whole-genome sequence data for six coronavirus subgenera to identify specific patterns of recombination that are conserved between multiple subgenera and then identify the likely factors that underlie these conserved patterns. Specifically, we confirm the non-randomness of recombination break points across all six tested coronavirus subgenera, locate conserved recombination hot- and cold-spots, and determine that the locations of transcriptional regulatory sequences are likely major determinants of conserved recombination break-point hotspot locations. We find that while the locations of recombination break points are not uniformly associated with degrees of nucleotide sequence conservation, they display significant tendencies in multiple coronavirus subgenera to occur in low guanine-cytosine content genome regions, in non-coding regions, at the edges of genes, and at sites within the Spike gene that are predicted to be minimally disruptive of Spike protein folding. While it is apparent that sequence features such as transcriptional regulatory sequences are likely major determinants of where the template-switching events that yield recombination break points most commonly occur, it is evident that selection against misfolded recombinant proteins also strongly impacts observable recombination break-point distributions in coronavirus genomes sampled from nature.\", \"doi_y\": \"https://doi.org/10.1093/ve/veac054\", \"publication_date\": \"2022-06-14\"}, {\"work_id\": \"W4286462093\", \"doi_x\": \"10.1101/2022.07.21.500943\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of Complexes III and IV\", \"abstract\": \"Abstract The mitochondrial electron transport chain comprises a series of protein complexes embedded in the inner mitochondrial membrane that generate a proton motive force via oxidative phosphorylation, ultimately generating ATP. These protein complexes can oligomerize to form larger structures called supercomplexes. Cardiolipin (CL), a conical lipid, unique within eukaryotes to the inner mitochondrial membrane, has proven essential in maintaining the stability and function of supercomplexes. Monolysocardiolipin (MLCL) is a CL variant that accumulates in people with Barth syndrome (BTHS). BTHS is caused by defects in CL biosynthesis and characterised by abnormal mitochondrial bioenergetics and destabilised supercomplexes. However, the mechanisms by which MLCL causes pathogenesis remain unclear. Here, multiscale molecular dynamics characterise the interactions of CL and MLCL with yeast and mammalian mitochondrial supercomplexes containing Complex III (CIII) and Complex IV (CIV). Coarse-grained simulations reveal that both CL and MLCL bind to sites at the interface between CIII and CIV of the supercomplex. Free energy perturbation calculations show that MLCL interaction is weaker than that of CL and suggest that interaction with CIV drives this difference. Atomistic contact analyses show that, although interaction with CIII is similar for CL and MLCL, CIV makes more contacts with CL than MLCL, demonstrating that CL is a more successful \\u201cglue\\u201d between the two complexes. Simulations of the human CIII2CIV supercomplex show that this interface site is maintained between species. Our study suggests that MLCL accumulation in people with BTHS disrupts supercomplex stability by formation of relatively weak interactions at the interface lipid binding site.\", \"doi_y\": \"https://doi.org/10.1101/2022.07.21.500943\", \"publication_date\": \"2022-07-22\"}, {\"work_id\": \"W4378528560\", \"doi_x\": \"10.3390/ijms24119359\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A Unique Mode of Coenzyme A Binding to the Nucleotide Binding Pocket of Human Metastasis Suppressor NME1\", \"abstract\": \"Coenzyme A (CoA) is a key cellular metabolite which participates in diverse metabolic pathways, regulation of gene expression and the antioxidant defense mechanism. Human NME1 (hNME1), which is a moonlighting protein, was identified as a major CoA-binding protein. Biochemical studies showed that hNME1 is regulated by CoA through both covalent and non-covalent binding, which leads to a decrease in the hNME1 nucleoside diphosphate kinase (NDPK) activity. In this study, we expanded the knowledge on previous findings by focusing on the non-covalent mode of CoA binding to the hNME1. With X-ray crystallography, we solved the CoA bound structure of hNME1 (hNME1-CoA) and determined the stabilization interactions CoA forms within the nucleotide-binding site of hNME1. A hydrophobic patch stabilizing the CoA adenine ring, while salt bridges and hydrogen bonds stabilizing the phosphate groups of CoA were observed. With molecular dynamics studies, we extended our structural analysis by characterizing the hNME1-CoA structure and elucidating possible orientations of the pantetheine tail, which is absent in the X-ray structure due to its flexibility. Crystallographic studies suggested the involvement of arginine 58 and threonine 94 in mediating specific interactions with CoA. Site-directed mutagenesis and CoA-based affinity purifications showed that arginine 58 mutation to glutamate (R58E) and threonine 94 mutation to aspartate (T94D) prevent hNME1 from binding to CoA. Overall, our results reveal a unique mode by which hNME1 binds CoA, which differs significantly from that of ADP binding: the \\u03b1- and \\u03b2-phosphates of CoA are oriented away from the nucleotide-binding site, while 3'-phosphate faces catalytic histidine 118 (H118). The interactions formed by the CoA adenine ring and phosphate groups contribute to the specific mode of CoA binding to hNME1.\", \"doi_y\": \"https://doi.org/10.3390/ijms24119359\", \"publication_date\": \"2023-05-27\"}, {\"work_id\": \"W4280558961\", \"doi_x\": \"10.1016/j.jmb.2022.167625\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"Computational Resources for Molecular Biology 2022\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2022.167625\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4306407977\", \"doi_x\": \"10.1002/anie.202212158\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A Vitamin B<sub>2</sub>\\u2010Photocatalysed Approach to Methionine Analogues\", \"abstract\": \"Access to new non-canonical amino acid residues is crucial for medicinal chemistry and chemical biology. Analogues of the amino acid methionine have been far less explored-despite their use in biochemistry, pharmacology and peptide bioconjugation. This is largely due to limited synthetic access. Herein, we exploit a new disconnection to access non-natural methionines through the development of a photochemical method for the radical \\u03b1-C-H functionalization of sulfides with alkenes, in water, using inexpensive and commercially-available riboflavin (vitamin B2 ) as a photocatalyst. Our photochemical conditions allow the two-step synthesis of novel methionine analogues-by radical addition to unsaturated amino acid derivatives-and the chemoselective modification of peptide side-chains to yield non-natural methionine residues within small peptides. The mechanism of the bio-inspired flavin photocatalysis has been probed by experimental, DFT and TDDFT studies.\", \"doi_y\": \"https://doi.org/10.1002/anie.202212158\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W4381326180\", \"doi_x\": \"10.1098/rsob.230058\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The structural basis of the talin\\u2013KANK1 interaction that coordinates the actin and microtubule cytoskeletons at focal adhesions\", \"abstract\": \"Adhesion between cells and the extracellular matrix is mediated by heterodimeric (\\u03b1\\u03b2) integrin receptors that are intracellularly linked to the contractile actomyosin machinery. One of the proteins that control this link is talin, which organizes cytosolic signalling proteins into discrete complexes on \\u03b2-integrin tails referred to as focal adhesions (FAs). The adapter protein KANK1 binds to talin in the region of FAs known as the adhesion belt. Here, we adapted a non-covalent crystallographic chaperone to resolve the talin-KANK1 complex. This structure revealed that the talin binding KN region of KANK1 contains a novel motif where a \\u03b2-hairpin stabilizes the \\u03b1-helical region, explaining both its specific interaction with talin R7 and high affinity. Single point mutants in KANK1 identified from the structure abolished the interaction and enabled us to examine KANK1 enrichment in the adhesion belt. Strikingly, in cells expressing a constitutively active form of vinculin that keeps the FA structure intact even in the presence of myosin inhibitors, KANK1 localizes throughout the entire FA structure even when actomyosin tension is released. We propose a model whereby actomyosin forces on talin eliminate KANK1 from talin binding in the centre of FAs while retaining it at the adhesion periphery.\", \"doi_y\": \"https://doi.org/10.1098/rsob.230058\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4283325295\", \"doi_x\": \"10.1073/pnas.2205850119\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Phylogenomic and functional characterization of an evolutionary conserved cytochrome P450-based insecticide detoxification mechanism in bees\", \"abstract\": \"The regulatory process for assessing the risks of pesticides to bees relies heavily on the use of the honeybee, Apis mellifera, as a model for other bee species. However, the validity of using A. mellifera as a surrogate for other Apis and non-Apis bees in pesticide risk assessment has been questioned. Related to this line of research, recent work on A. mellifera has shown that specific P450 enzymes belonging to the CYP9Q subfamily act as critically important determinants of insecticide sensitivity in this species by efficiently detoxifying certain insecticide chemotypes. However, the extent to which the presence of functional orthologs of these enzymes is conserved across the diversity of bees is unclear. Here we used a phylogenomic approach to identify > 100 putative CYP9Q functional orthologs across 75 bee species encompassing all major bee families. Functional analysis of 26 P450s from 20 representative bee species revealed that P450-mediated detoxification of certain systemic insecticides, including the neonicotinoid thiacloprid and the butenolide flupyradifurone, is conserved across all major bee pollinator families. However, our analyses also reveal that CYP9Q-related genes are not universal to all bee species, with some Megachilidae species lacking such genes. Thus, our results reveal an evolutionary conserved capacity to metabolize certain insecticides across all major bee families while identifying a small number of bee species where this function may have been lost. Furthermore, they illustrate the potential of a toxicogenomic approach to inform pesticide risk assessment for nonmanaged bee species by predicting the capability of bee pollinator species to break down synthetic insecticides.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2205850119\", \"publication_date\": \"2022-06-21\"}, {\"work_id\": \"W4382278861\", \"doi_x\": \"10.1038/s41433-023-02619-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"ChatGPT in ophthalmology: the dawn of a new era?\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41433-023-02619-4\", \"publication_date\": \"2023-06-27\"}, {\"work_id\": \"W4313433587\", \"doi_x\": \"10.1007/978-3-031-23220-6_4\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"AI Benchmarking for Science: Efforts from the MLCommons Science Working Group\", \"abstract\": \"With machine learning (ML) becoming a transformative tool for science, the scientific community needs a clear catalogue of ML techniques, and their relative benefits on various scientific problems, if they were to make significant advances in science using AI. Although this comes under the purview of benchmarking, conventional benchmarking initiatives are focused on performance, and as such, science, often becomes a secondary criteria. In this paper, we describe a community effort from a working group, namely, MLCommons Science Working Group, in developing science-specific AI benchmarking for the international scientific community. Since the inception of the working group in 2020, the group has worked very collaboratively with a number of national laboratories, academic institutions and industries, across the world, and has developed four science-specific AI benchmarks. We will describe the overall process, the resulting benchmarks along with some initial results. We foresee that this initiative is likely to be very transformative for the AI for Science, and for performance-focused communities.\", \"doi_y\": \"https://doi.org/10.1007/978-3-031-23220-6_4\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4307492594\", \"doi_x\": \"10.1093/nar/gkac925\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"abstract\": \"Abstract Type I CRISPR systems are the most common CRISPR type found in bacteria. They use a multisubunit effector, guided by crRNA, to detect and bind dsDNA targets, forming an R-loop and recruiting the Cas3 enzyme to facilitate target DNA destruction, thus providing immunity against mobile genetic elements. Subtypes have been classified into families A-G, with type I-G being the least well understood. Here, we report the composition, structure and function of the type I-G Cascade CRISPR effector from Thioalkalivibrio sulfidiphilus, revealing key new molecular details. The unique Csb2 subunit processes pre-crRNA, remaining bound to the 3\\u2032 end of the mature crRNA, and seven Cas7 subunits form the backbone of the effector. Cas3 associates stably with the effector complex via the Cas8g subunit and is important for target DNA recognition. Structural analysis by cryo-Electron Microscopy reveals a strikingly curved backbone conformation with Cas8g spanning the belly of the structure. These biochemical and structural insights shed new light on the diversity of type I systems and open the way to applications in genome engineering.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac925\", \"publication_date\": \"2022-10-28\"}, {\"work_id\": \"W4312093219\", \"doi_x\": \"10.1101/2022.12.21.520923\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"CHC22 clathrin membrane recruitment uses SNX5 in bipartite interaction with secretory tether p115\", \"abstract\": \"Abstract The two clathrin isoforms, CHC17 and CHC22, mediate separate intracellular transport routes. CHC17 performs endocytosis and housekeeping membrane traffic in all cells. CHC22, expressed most highly in skeletal muscle, transports the glucose transporter GLUT4 from the endoplasmic-reticulum-to-Golgi intermediate compartment (ERGIC) to an intracellular GLUT4 storage compartment (GSC) from where GLUT4 is mobilized to the plasma membrane by insulin. Here, the molecular determinants distinguishing CHC22 from CHC17 trafficking are defined. The C-terminal trimerization domain of CHC22 binds SNX5, which also binds the ERGIC tether p115. SNX5, and the functionally redundant SNX6, are required for CHC22 localization independently of their participation in the endosomal ESCPE-1 complex. In tandem, an isoform-specific patch in the CHC22 N-terminal domain separately mediates binding to p115. This unconventional mode of clathrin recruitment, involving interactions at both heavy chain termini, is required for CHC22 targeting to ERGIC membranes and for GLUT4 traffic to the GSC, defining a bipartite mechanism regulating a key pathway in human glucose metabolism. Summary statement CHC22 clathrin is recruited to the secretory pathway for GLUT4 transporter trafficking by binding the p115 tether via SNX5 association with the CHC22 C-terminus and an isoform-specific patch on its N-terminal domain, with both interactions required for insulin-responsive GLUT4 translocation.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.21.520923\", \"publication_date\": \"2022-12-21\"}, {\"work_id\": \"W4225281512\", \"doi_x\": \"10.1101/2022.04.28.489841\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"abstract\": \"Abstract Despite continuing progress in kinesin enzyme mechanochemistry and emerging understanding of the cargo recognition machinery, it is not known how these functions are coupled and controlled by the alpha-helical coiled coils encoded by a large component of kinesin protein sequences. Here, we combine computational structure prediction with single-particle negative stain electron microscopy to reveal the coiled-coil architecture of heterotetrameric kinesin-1, in its compact state. An unusual flexion in the scaffold enables folding of the complex, bringing the kinesin heavy chain-light chain interface into close apposition with a tetrameric assembly formed from the region of the molecule previously assumed to be the folding hinge. This framework for autoinhibition is required to uncover how engagement of cargo and other regulatory factors drive kinesin-1 activation. Summary statement Integration of computational structure prediction with electron microscopy reveals the coiled-coil architecture of the autoinhibited compact conformer of the microtubule motor, kinesin-1.\", \"doi_y\": \"https://doi.org/10.1101/2022.04.28.489841\", \"publication_date\": \"2022-04-29\"}, {\"work_id\": \"W4301210157\", \"doi_x\": \"10.1101/2022.06.09.495448\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"PKC isoforms activate LRRK1 kinase by phosphorylating conserved residues (Ser1064, Ser1074 and Thr1075) within the COR<sub>B</sub> GTPase domain\", \"abstract\": \"Abstract Leucine-rich-repeat-kinase 1 (LRRK1) and its homologue LRRK2 are multidomain kinases possessing a ROC-COR A -COR B containing GTPase domain and phosphorylate distinct Rab proteins. LRRK1 loss of function mutations cause the bone disorder osteosclerotic metaphyseal dysplasia, whereas LRRK2 missense mutations that enhance kinase activity cause Parkinson\\u2019s disease. Previous work suggested that LRRK1 but not LRRK2, is activated via a Protein Kinase C (PKC)-dependent mechanism. Here we demonstrate that phosphorylation and activation of LRRK1 in HEK293 cells is blocked by PKC inhibitors including LXS-196 (Darovasertib), a compound that has entered clinical trials. We show multiple PKC isoforms phosphorylate and activate recombinant LRRK1 in a manner reversed by phosphatase treatment. PKC\\u03b1 unexpectedly does not activate LRRK1 by phosphorylating the kinase domain, but instead phosphorylates a cluster of conserved residues (Ser1064, Ser1074 and Thr1075) located within a region of the COR B domain of the GTPase domain. These residues are positioned at the equivalent region of the LRRK2 DK helix reported to stabilize the kinase domain \\u03b1C-helix in the active conformation. Thr1075 represents an optimal PKC site phosphorylation motif and its mutation to Ala, blocked PKC-mediated activation of LRRK1. A triple Glu mutation of Ser1064/Ser1074/Thr1075 to mimic phosphorylation, enhanced LRRK1 kinase activity ~3-fold. From analysis of available structures, we postulate that phosphorylation of Ser1064, Ser1074 and Thr1075 activates LRRK1 by promoting interaction and stabilization of the aC-helix on the kinase domain. This study provides new fundamental insights into the mechanism controlling LRRK1 activity and reveals a novel unexpected activation mechanism.\", \"doi_y\": \"https://doi.org/10.1101/2022.06.09.495448\", \"publication_date\": \"2022-06-09\"}, {\"work_id\": \"W4200435200\", \"doi_x\": \"10.1002/ajmg.a.62611\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Truncating and zinc\\u2010finger variants in <scp><i>GLI2</i></scp> are associated with hypopituitarism\", \"abstract\": \"Variants in transcription factor GLI2 have been associated with hypopituitarism and structural brain abnormalities, occasionally including holoprosencephaly (HPE). Substantial phenotypic variability and nonpenetrance have been described, posing difficulties in the counseling of affected families. We present three individuals with novel likely pathogenic GLI2 variants, two with truncating and one with a de novo missense variant p.(Ser548Leu), and review the literature for comprehensive phenotypic descriptions of individuals with confirmed pathogenic (a) intragenic GLI2 variants and (b) chromosome 2q14.2 deletions encompassing only GLI2. We show that most of the 31 missense variants previously reported as pathogenic are likely benign or, at most, low-risk variants. Four Zn-finger variants: p.(Arg479Gly), p.(Arg516Pro), p.(Gly518Lys), and p.(Tyr575His) were classified as likely pathogenic, and three other variants as possibly pathogenic: p.(Pro253Ser), p.(Ala593Val), and p.(Pro1243Leu). We analyze the phenotypic descriptions of 60 individuals with pathogenic GLI2 variants and evidence a morbidity spectrum that includes hypopituitarism (58%), HPE (6%) or other brain structure abnormalities (15%), orofacial clefting (17%) and dysmorphic facial features (35%). We establish that truncating and Zn-finger variants in GLI2 are associated with a high risk of hypopituitarism, and that a solitary median maxillary central incisor is part of the GLI2-related phenotypic variability. The most prevalent phenotypic feature is post-axial polydactyly (65%) which is also the mildest phenotypic expression of the condition, reported in many parents of individuals with systemic findings. Our approach clarifies clinical risks and the important messages to discuss in counseling for a pathogenic GLI2 variant.\", \"doi_y\": \"https://doi.org/10.1002/ajmg.a.62611\", \"publication_date\": \"2021-12-17\"}, {\"work_id\": \"W4323348220\", \"doi_x\": \"10.1083/jcb.202209010\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>TLN1</i> contains a cancer-associated cassette exon that alters talin-1 mechanosensitivity\", \"abstract\": \"Talin-1 is the core mechanosensitive adapter protein linking integrins to the cytoskeleton. The TLN1 gene is comprised of 57 exons that encode the 2,541 amino acid TLN1 protein. TLN1 was previously considered to be expressed as a single isoform. However, through differential pre-mRNA splicing analysis, we discovered a cancer-enriched, non-annotated 51-nucleotide exon in TLN1 between exons 17 and 18, which we refer to as exon 17b. TLN1 is comprised of an N-terminal FERM domain, linked to 13 force-dependent switch domains, R1-R13. Inclusion of exon 17b introduces an in-frame insertion of 17 amino acids immediately after Gln665 in the region between R1 and R2 which lowers the force required to open the R1-R2 switches potentially altering downstream mechanotransduction. Biochemical analysis of this isoform revealed enhanced vinculin binding, and cells expressing this variant show altered adhesion dynamics and motility. Finally, we showed that the TGF-\\u03b2/SMAD3 signaling pathway regulates this isoform switch. Future studies will need to consider the balance of these two TLN1 isoforms.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202209010\", \"publication_date\": \"2023-03-06\"}, {\"work_id\": \"W4308274664\", \"doi_x\": \"10.1101/2022.11.03.514279\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Cryo-electron microscopy of the f1 filamentous phage reveals a new paradigm in viral infection and assembly\", \"abstract\": \"Abstract Phages are viruses that infect bacteria and dominate every ecosystem on our planet. As well as impacting microbial ecology, physiology and evolution, phages are exploited as tools in molecular biology and biotechnology. This is particularly true for the Ff (f1, fd or M13) phages, which represent a widely distributed group of filamentous viruses. Over nearly five decades, Ff has seen an extraordinary range of applications, including in phage display and nanotechnology. However, the complete structure of the phage capsid and consequently the mechanisms of infection and assembly remain largely mysterious. Using cryo-electron microscopy and a highly efficient system for production of short Ff-derived nanorods, we have determined the first structure of a filamentous virus, including the filament tips. Structure combined with mutagenesis was employed to identify domains of the phage that are important in bacterial attack and for release of new phage progeny. These data allow new models to be proposed for the phage lifecycle and will undoubtedly enable the development of novel biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1101/2022.11.03.514279\", \"publication_date\": \"2022-11-04\"}, {\"work_id\": \"W4315754439\", \"doi_x\": \"10.1128/jvi.00872-22\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"abstract\": \"The interferon system is part of the innate immune response that inhibits viruses and other pathogens. This system emerged approximately 500 million years ago in early vertebrates.\", \"doi_y\": \"https://doi.org/10.1128/jvi.00872-22\", \"publication_date\": \"2023-01-31\"}, {\"work_id\": \"W4296502374\", \"doi_x\": \"10.1039/d2sc04479j\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"abstract\": \"Rules for designing 4-helix bundles are defined, tested, and used to generate de novo peptide assemblies and a single-chain protein.\", \"doi_y\": \"https://doi.org/10.1039/d2sc04479j\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4321116997\", \"doi_x\": \"10.1016/j.jmb.2023.168016\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"MGnify Genomes: A Resource for Biome-specific Microbial Genome Catalogues\", \"abstract\": \"An increasingly common output arising from the analysis of shotgun metagenomic datasets is the generation of metagenome-assembled genomes (MAGs), with tens of thousands of MAGs now described in the literature. However, the discovery and comparison of these MAG collections is hampered by the lack of uniformity in their generation, annotation and storage. To address this, we have developed MGnify Genomes, a growing collection of biome-specific non-redundant microbial genome catalogues generated using MAGs and publicly available isolate genomes. Genomes within a biome-specific catalogue are organised into species clusters. For species that contain multiple conspecific genomes, the highest quality genome is selected as the representative, always prioritising an isolate genome over a MAG. The species representative sequences and annotations can be visualised on the MGnify website and the full catalogue and associated analysis outputs can be downloaded from MGnify servers. A suite of online search tools is provided allowing users to compare their own sequences, ranging from a gene to sets of genomes, against the catalogues. Seven biomes are available currently, comprising over 300,000 genomes that represent 11,048 non-redundant species, and include 36 taxonomic classes not currently represented by cultured genomes. MGnify Genomes is available at https://www.ebi.ac.uk/metagenomics/browse/genomes/.\", \"doi_y\": \"https://doi.org/10.1016/j.jmb.2023.168016\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4296807180\", \"doi_x\": \"10.1242/jcs.260288\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"abstract\": \"ABSTRACT Tropomyosins are structurally conserved \\u03b1-helical coiled-coil proteins that bind along the length of filamentous actin (F-actin) in fungi and animals. Tropomyosins play essential roles in the stability of actin filaments and in regulating myosin II contractility. Despite the crucial role of tropomyosin in actin cytoskeletal regulation, in vivo investigations of tropomyosin are limited, mainly due to the suboptimal live-cell imaging tools currently available. Here, we report on an mNeonGreen (mNG)-tagged tropomyosin, with native promoter and linker length configuration, that clearly reports tropomyosin dynamics in Schizosaccharomyces pombe (Cdc8), Schizosaccharomyces japonicus (Cdc8) and Saccharomyces cerevisiae (Tpm1 and Tpm2). We also describe a fluorescent probe to visualize mammalian tropomyosin (TPM2 isoform). Finally, we generated a camelid nanobody against S. pombe Cdc8, which mimics the localization of mNG\\u2013Cdc8 in vivo. Using these tools, we report the presence of tropomyosin in previously unappreciated patch-like structures in fission and budding yeasts, show flow of tropomyosin (F-actin) cables to the cytokinetic actomyosin ring and identify rearrangements of the actin cytoskeleton during mating. These powerful tools and strategies will aid better analyses of tropomyosin and F-actin cables in vivo.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260288\", \"publication_date\": \"2022-09-15\"}, {\"work_id\": \"W4307563852\", \"doi_x\": \"10.1085/jgp.202213210\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"abstract\": \"The flux of ions through a channel is most commonly regulated by changes that result in steric occlusion of its pore. However, ion permeation can also be prevented by formation of a desolvation barrier created by hydrophobic residues that line the pore. As a result of relatively minor structural changes, confined hydrophobic regions in channels may undergo transitions between wet and dry states to gate the pore closed without physical constriction of the permeation pathway. This concept is referred to as hydrophobic gating, and many examples of this process have been demonstrated. However, the term is also now being used in a much broader context that often deviates from its original meaning. In this Viewpoint, we explore the formal definition of a hydrophobic gate, discuss examples of this process compared with other gating mechanisms that simply exploit hydrophobic residues and/or lipids in steric closure of the pore, and describe the best practice for identification of a hydrophobic gate.\", \"doi_y\": \"https://doi.org/10.1085/jgp.202213210\", \"publication_date\": \"2022-10-26\"}, {\"work_id\": \"W4225822689\", \"doi_x\": \"10.1242/jcs.259234\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Visualising the cytoskeletal machinery in neuronal growth cones using cryo-electron tomography\", \"abstract\": \"ABSTRACT Neurons extend axons to form the complex circuitry of the mature brain. This depends on the coordinated response and continuous remodelling of the microtubule and F-actin networks in the axonal growth cone. Growth cone architecture remains poorly understood at nanoscales. We therefore investigated mouse hippocampal neuron growth cones using cryo-electron tomography to directly visualise their three-dimensional subcellular architecture with molecular detail. Our data showed that the hexagonal arrays of actin bundles that form filopodia penetrate and terminate deep within the growth cone interior. We directly observed the modulation of these and other growth cone actin bundles by alteration of individual F-actin helical structures. Microtubules with blunt, slightly flared or gently curved ends predominated in the growth cone, frequently contained lumenal particles and exhibited lattice defects. Investigation of the effect of absence of doublecortin, a neurodevelopmental cytoskeleton regulator, on growth cone cytoskeleton showed no major anomalies in overall growth cone organisation or in F-actin subpopulations. However, our data suggested that microtubules sustained more structural defects, highlighting the importance of microtubule integrity during growth cone migration.\", \"doi_y\": \"https://doi.org/10.1242/jcs.259234\", \"publication_date\": \"2022-04-01\"}, {\"work_id\": \"W4318071688\", \"doi_x\": \"10.1042/bcj20220480\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"abstract\": \"Gram-negative bacteria are surrounded by two protein-rich membranes with a peptidoglycan layer sandwiched between them. Together they form the envelope (or cell wall), crucial for energy production, lipid biosynthesis, structural integrity, and for protection against physical and chemical environmental challenges. To achieve envelope biogenesis, periplasmic and outer-membrane proteins (OMPs) must be transported from the cytosol and through the inner-membrane, via the ubiquitous SecYEG protein-channel. Emergent proteins either fold in the periplasm or cross the peptidoglycan (PG) layer towards the outer-membrane for insertion through the \\u03b2-barrel assembly machinery (BAM). Trafficking of hydrophobic proteins through the periplasm is particularly treacherous given the high protein density and the absence of energy (ATP or chemiosmotic potential). Numerous molecular chaperones assist in the prevention and recovery from aggregation, and of these SurA is known to interact with BAM, facilitating delivery to the outer-membrane. However, it is unclear how proteins emerging from the Sec-machinery are received and protected from aggregation and proteolysis prior to an interaction with SurA. Through biochemical analysis and electron microscopy we demonstrate the binding capabilities of the unoccupied and substrate-engaged SurA to the inner-membrane translocation machinery complex of SecYEG-SecDF-YidC - aka the holo-translocon (HTL). Supported by AlphaFold predictions, we suggest a role for periplasmic domains of SecDF in chaperone recruitment to the protein translocation exit site in SecYEG. We propose that this immediate interaction with the enlisted chaperone helps to prevent aggregation and degradation of nascent envelope proteins, facilitating their safe passage to the periplasm and outer-membrane.\", \"doi_y\": \"https://doi.org/10.1042/bcj20220480\", \"publication_date\": \"2023-02-27\"}, {\"work_id\": \"W4292295879\", \"doi_x\": \"10.1039/d2cs00387b\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Discovery of small molecule ligands for the von Hippel-Lindau (VHL) E3 ligase and their use as inhibitors and PROTAC degraders\", \"abstract\": \"The von Hippel-Lindau (VHL) Cullin RING E3 ligase is an essential enzyme in the ubiquitin-proteasome system that recruits substrates such as the hypoxia inducible factor for ubiquitination and subsequent proteasomal degradation. The ubiquitin-proteasome pathway can be hijacked toward non-native neo-substrate proteins using proteolysis targeting chimeras (PROTACs), bifunctional molecules designed to simultaneously bind to an E3 ligase and a target protein to induce target ubiquitination and degradation. The availability of high-quality small-molecule ligands with good binding affinity for E3 ligases is fundamental for PROTAC development. Lack of good E3 ligase ligands as starting points to develop PROTAC degraders was initially a stumbling block to the development of the field. Herein, the journey towards the design of small-molecule ligands binding to VHL is presented. We cover the structure-based design of VHL ligands, their application as inhibitors in their own right, and their implementation into rationally designed, potent PROTAC degraders of various target proteins. We highlight the key findings and learnings that have provided strong foundations for the remarkable development of targeted protein degradation, and that offer a blueprint for designing new ligands for E3 ligases beyond VHL.\", \"doi_y\": \"https://doi.org/10.1039/d2cs00387b\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4322620417\", \"doi_x\": \"10.1016/j.sbi.2023.102565\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Recent structural advances in bacterial chemotaxis signalling\", \"abstract\": \"Bacterial chemosensory arrays have served as a model system for in-situ structure determination, clearly cataloguing the improvement of cryo-electron tomography (cryoET) over the past decade. In recent years, this has culminated in an accurately fitted atomistic model for the full-length core signalling unit (CSU) and numerous insights into the function of the transmembrane receptors responsible for signal transduction. Here, we review the achievements of the latest structural advances in bacterial chemosensory arrays and the developments which have made such advances possible.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102565\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4285091781\", \"doi_x\": \"10.3390/v14071522\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Women in the European Virus Bioinformatics Center\", \"abstract\": \"Viruses are the cause of a considerable burden to human, animal and plant health, while on the other hand playing an important role in regulating entire ecosystems. The power of new sequencing technologies combined with new tools for processing \\u201cBig Data\\u201d offers unprecedented opportunities to answer fundamental questions in virology. Virologists have an urgent need for virus-specific bioinformatics tools. These developments have led to the formation of the European Virus Bioinformatics Center, a network of experts in virology and bioinformatics who are joining forces to enable extensive exchange and collaboration between these research areas. The EVBC strives to provide talented researchers with a supportive environment free of gender bias, but the gender gap in science, especially in math-intensive fields such as computer science, persists. To bring more talented women into research and keep them there, we need to highlight role models to spark their interest, and we need to ensure that female scientists are not kept at lower levels but are given the opportunity to lead the field. Here we showcase the work of the EVBC and highlight the achievements of some outstanding women experts in virology and viral bioinformatics.\", \"doi_y\": \"https://doi.org/10.3390/v14071522\", \"publication_date\": \"2022-07-12\"}, {\"work_id\": \"W3178087467\", \"doi_x\": \"10.1002/prot.26171\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"High\\u2010accuracy protein structure prediction in <scp>CASP14</scp>\", \"abstract\": \"The application of state-of-the-art deep-learning approaches to the protein modeling problem has expanded the \\\"high-accuracy\\\" category in CASP14 to encompass all targets. Building on the metrics used for high-accuracy assessment in previous CASPs, we evaluated the performance of all groups that submitted models for at least 10 targets across all difficulty classes, and judged the usefulness of those produced by AlphaFold2 (AF2) as molecular replacement search models with AMPLE. Driven by the qualitative diversity of the targets submitted to CASP, we also introduce DipDiff as a new measure for the improvement in backbone geometry provided by a model versus available templates. Although a large leap in high-accuracy is seen due to AF2, the second-best method in CASP14 out-performed the best in CASP13, illustrating the role of community-based benchmarking in the development and evolution of the protein structure prediction field.\", \"doi_y\": \"https://doi.org/10.1002/prot.26171\", \"publication_date\": \"2021-07-14\"}, {\"work_id\": \"W4377010500\", \"doi_x\": \"10.1039/d3nh00080j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Anomalous DNA hybridisation kinetics on gold nanorods revealed <i>via</i> a dual single-molecule imaging and optoplasmonic sensing platform\", \"abstract\": \"Observing the hybridisation kinetics of DNA probes immobilised on plasmonic nanoparticles is key in plamon enhanced fluorescence detection from weak emitting species, and refractive index based single-molecule detection on optoplasmonic sensors. The role of the local field in providing plasmonic signal enhancements for single-molecule detection has been studied in great detail. Nevertheless, few studies have compared the experimental results in both techniques for single-molecule studies. Here we developed the first optical setup that integrates optoplasmonic and DNA-PAINT based detection of oligonucleotides to compare these sub-platforms and provide complementary insights into single-molecule processes. We record the fluorescence and optoplasmonic sensor signals for individual, transient hybridisation events. The hybridisation events are observed in the same sample cell and over a prolonged time (i.e. towards high binding site occupancies). A decrease in the association rate over the measurement duration is reported. Our dual optoplasmonic sensing and imaging platform offers insight into the observed phenomenon, revealing that irreversible hybridisation events accumulate over detected step signals in optoplasmonic sensing. Our results point to novel physicochemical mechanisms that result in the stabilisation of DNA hybridisation on optically-excited plasmonic nanoparticles.\", \"doi_y\": \"https://doi.org/10.1039/d3nh00080j\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385620824\", \"doi_x\": \"10.1111/jeu.12994\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<scp>AAK1</scp>\\u2010like: A putative pseudokinase with potential roles in cargo uptake in bloodstream form <i>Trypanosoma brucei</i> parasites\", \"abstract\": \"Abstract Selection and internalization of cargo via clathrin\\u2010mediated endocytosis requires adaptor protein complexes. One complex, AP\\u20102, acts during cargo selection at the plasma membrane. African trypanosomes lack all components of the AP\\u20102 complex, except for a recently identified orthologue of the AP\\u20102\\u2010associated protein kinase 1, AAK1. In characterized eukaryotes, AAK1 phosphorylates the \\u03bc2 subunit of the AP\\u20102 complex to enhance cargo recognition and uptake into clathrin\\u2010coated vesicles. Here, we show that kinetoplastids encode not one, but two AAK1 orthologues: one (AAK1L2) is absent from salivarian trypanosomes, while the other (AAK1L1) lacks important kinase\\u2010specific residues in a range of trypanosomes. These AAK1L1 and AAK1L2 novelties reinforce suggestions of functional divergence in endocytic uptake within salivarian trypanosomes. Despite this, we show that AAK1L1 null mutant Trypanosoma brucei , while viable, display slowed proliferation, morphological abnormalities including swelling of the flagellar pocket, and altered cargo uptake. In summary, our data suggest an unconventional role for a putative pseudokinase during endocytosis and/or vesicular trafficking in T. brucei , independent of AP\\u20102.\", \"doi_y\": \"https://doi.org/10.1111/jeu.12994\", \"publication_date\": \"2023-08-07\"}, {\"work_id\": \"W4226169846\", \"doi_x\": \"10.1128/jvi.02024-21\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"<i>Birnaviridae</i> Virus Factories Show Features of Liquid-Liquid Phase Separation and Are Distinct from Paracrystalline Arrays of Virions Observed by Electron Microscopy\", \"abstract\": \"To gain more information about the nature of Birnaviridae virus factories (VFs), we used a recombinant infectious bursal disease virus (IBDV) expressing split-GFP11 tagged to the polymerase (VP1) that we have previously shown is a marker for VFs in infected cells expressing GFP1-10. We found that VFs colocalized with 5-ethynyl uridine in the presence of actinomycin, demonstrating they contained newly synthesized viral RNA, and VFs were visible in infected cells that were fixed and permeabilized with digitonin, demonstrating that they were not membrane bound. Fluorescence recovery after photobleaching (FRAP) a region of interest within the VFs occurred rapidly, recovering from approximately 25% to 87% the original intensity over 146 s, and VFs were dissolved by 1,6-hexanediol treatment, demonstrating they showed properties consistent with liquid-liquid phase separation. There was a lower colocalization of the VF GFP signal with the capsid protein VP2 (Manders' coefficient [MC] 0.6), compared to VP3 (MC, 0.9), which prompted us to investigate the VF ultrastructure by transmission electron microscopy (TEM). In infected cells, paracrystalline arrays (PAs) of virions were observed in the cytoplasm, as well as discrete electron dense regions. Using correlative light and electron microscopy (CLEM), we observed that the electron dense regions correlated with the GFP signal of the VFs, which were distinct from the PAs. In summary, Birnaviridae VFs contain newly synthesized viral RNA, are not bound by a membrane, show properties consistent with liquid-liquid phase separation, and are distinct from the PAs observed by TEM. IMPORTANCE Members of the Birnaviridae infect birds, fish and insects, and are responsible for diseases of significant economic importance to the poultry industry and aquaculture. Despite their importance, how they replicate in cells remains poorly understood. Here, we show that the Birnaviridae virus factories are not membrane bound, demonstrate properties consistent with liquid-liquid phase separation, and are distinct from the paracrystalline arrays of virions observed by transmission electron microscopy, enhancing our fundamental knowledge of virus replication that could be used to develop strategies to control disease, or optimize their therapeutic application.\", \"doi_y\": \"https://doi.org/10.1128/jvi.02024-21\", \"publication_date\": \"2022-03-23\"}, {\"work_id\": \"W4210823904\", \"doi_x\": \"10.1101/2022.02.07.478301\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Identification of Unusual Oxysterols Biosynthesised in Human Pregnancy by Charge-Tagging and Liquid Chromatography - Mass Spectrometry\", \"abstract\": \"Abstract The aim of this study was to identify sterols, oxysterols and any down-stream metabolites in placenta, umbilical cord blood plasma, maternal plasma and amniotic fluid to enhance our knowledge of the involvement of these molecules in pregnancy. We confirm the identification of 20S- hydroxycholesterol in human placenta, previously reported in a single publication, and propose a pathway from 22R-hydroxycholesterol to a C 27 bile acid of probable structure 3\\u03b2,20R,22R- trihydroxycholest-5-en-(25R)26-oic acid. The pathway is evident not only in placenta, but pathway intermediates are also found in umbilical cord plasma, maternal plasma and amniotic fluid but not non-pregnant women.\", \"doi_y\": \"https://doi.org/10.1101/2022.02.07.478301\", \"publication_date\": \"2022-02-08\"}, {\"work_id\": \"W4382728082\", \"doi_x\": \"10.1016/j.molcel.2023.06.013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Molecular basis for the reversible ADP-ribosylation of guanosine bases\", \"abstract\": \"Modification of nucleic acids by ADP-ribosylation is catalyzed by various ADP-ribosyltransferases, including the DarT enzyme. The latter is part of the bacterial toxin-antitoxin (TA) system DarTG, which was shown to provide control of DNA replication and bacterial growth as well as protection against bacteriophages. Two subfamilies have been identified, DarTG1 and DarTG2, which are distinguished by their associated antitoxins. While DarTG2 catalyzes reversible ADP-ribosylation of thymidine bases employing a macrodomain as antitoxin, the DNA ADP-ribosylation activity of DarTG1 and the biochemical function of its antitoxin, a NADAR domain, are as yet unknown. Using structural and biochemical approaches, we show that DarT1-NADAR is a TA system for reversible ADP-ribosylation of guanosine bases. DarT1 evolved the ability to link ADP-ribose to the guanine amino group, which is specifically hydrolyzed by NADAR. We show that guanine de-ADP-ribosylation is also conserved among eukaryotic and non-DarT-associated NADAR members, indicating a wide distribution of reversible guanine modifications beyond DarTG systems.\", \"doi_y\": \"https://doi.org/10.1016/j.molcel.2023.06.013\", \"publication_date\": \"2023-07-01\"}, {\"work_id\": \"W4382652024\", \"doi_x\": \"10.1002/cbic.202300250\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Application of a Synthetic Ferredoxin\\u2010Inspired [4Fe4S]\\u2010Peptide Maquette as the Redox Partner for a [FeFe]\\u2010Hydrogenase\", \"abstract\": \"'Bacterial-type' ferredoxins host a cubane [4Fe4S]2+/+ cluster that enables these proteins to mediate electron transfer and facilitate a broad range of biological processes. Peptide maquettes based on the conserved cluster-forming motif have previously been reported and used to model the ferredoxins. Herein we explore the integration of a [4Fe4S]-peptide maquette into a H2 -powered electron transport chain. While routinely formed under anaerobic conditions, we illustrate by electron paramagnetic resonance (EPR) analysis that these maquettes can be reconstituted under aerobic conditions by using photoactivated NADH to reduce the cluster at 240 K. Attempts to tune the redox properties of the iron-sulfur cluster by introducing an Fe-coordinating selenocysteine residue were also explored. To demonstrate the integration of these artificial metalloproteins into a semi-synthetic electron transport chain, we utilize a ferredoxin-inspired [4Fe4S]-peptide maquette as the redox partner in the hydrogenase-mediated oxidation of H2 .\", \"doi_y\": \"https://doi.org/10.1002/cbic.202300250\", \"publication_date\": \"2023-08-13\"}, {\"work_id\": \"W4320724796\", \"doi_x\": \"10.12688/wellcomeopenres.18937.1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Structure of the transmembrane protein 2 (TMEM2) ectodomain and its lack of hyaluronidase activity\", \"abstract\": \"<ns4:p><ns4:bold>Background: </ns4:bold>Hyaluronic acid (HA) is a major polysaccharide component of the extracellular matrix. HA has essential functions in tissue architecture and the regulation of cell behaviour. HA turnover needs to be finely balanced. Increased HA degradation is associated with cancer, inflammation, and other pathological situations. Transmembrane protein 2 (TMEM2) is a cell surface protein that has been reported to degrade HA into ~5 kDa fragments and play an essential role in systemic HA turnover.</ns4:p><ns4:p> <ns4:bold>Methods: </ns4:bold>We produced the soluble TMEM2 ectodomain (residues 106-1383; sTMEM2) in human embryonic kidney cells (HEK293) and determined its structure using X-ray crystallography. We tested sTMEM2 hyaluronidase activity using fluorescently labelled HA and size fractionation of reaction products.</ns4:p><ns4:p> <ns4:bold>Results: </ns4:bold>Our crystal structure of sTMEM2 confirms a remarkably accurate prediction by AlphaFold. sTMEM2 contains a parallel \\u03b2-helix typical of other polysaccharide-degrading enzymes, but an active site cannot be assigned with confidence. A lectin-like domain is inserted into the \\u03b2-helix and predicted to be functional in carbohydrate binding. A second lectin-like domain at the C-terminus is unlikely to bind carbohydrates. Unexpectedly, we were unable to observe any HA degradation by sTMEM2. Our negative results set an upper limit for k<ns4:sub>cat</ns4:sub> of approximately 10<ns4:sup>-5</ns4:sup> min<ns4:sup>-1</ns4:sup>.</ns4:p><ns4:p> <ns4:bold>Conclusions: </ns4:bold>Although sTMEM2 contains domain types consistent with its suggested role in TMEM2 degradation, its hyaluronidase activity was undetectable. HA degradation by TMEM2 may require additional proteins and/or localisation at the cell surface.</ns4:p>\", \"doi_y\": \"https://doi.org/10.12688/wellcomeopenres.18937.1\", \"publication_date\": \"2023-02-14\"}, {\"work_id\": \"W4365443842\", \"doi_x\": \"10.1002/pro.4640\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Engineering mutually orthogonal <scp>PylRS</scp>/<scp>tRNA</scp> pairs for dual encoding of functional histidine analogues\", \"abstract\": \"The availability of an expanded genetic code opens exciting new opportunities in enzyme design and engineering. In this regard histidine analogues have proven particularly versatile, serving as ligands to augment metalloenzyme function and as catalytic nucleophiles in designed enzymes. The ability to genetically encode multiple functional residues could greatly expand the range of chemistry accessible within enzyme active sites. Here, we develop mutually orthogonal translation components to selectively encode two structurally similar histidine analogues. Transplanting known mutations from a promiscuous Methanosarcina mazei pyrrolysyl-tRNA synthetase (MmPylRSIFGFF ) into a single domain PylRS from Methanomethylophilus alvus (MaPylRSIFGFF ) provided a variant with improved efficiency and specificity for 3-methyl-L-histidine (MeHis) incorporation. The MaPylRSIFGFF clone was further characterized using in vitro biochemical assays and x-ray crystallography. We subsequently engineered the orthogonal MmPylRS for activity and selectivity for 3-(3-pyridyl)-L-alanine (3-Pyr), which was used in combination with MaPylRSIFGFF to produce proteins containing both 3-Pyr and MeHis. Given the versatile roles played by histidine in enzyme mechanisms, we anticipate that the tools developed within this study will underpin the development of enzymes with new and enhanced functions.\", \"doi_y\": \"https://doi.org/10.1002/pro.4640\", \"publication_date\": \"2023-04-25\"}, {\"work_id\": \"W4368617765\", \"doi_x\": \"10.1186/s13024-023-00619-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Opinion: more mouse models and more translation needed for ALS\", \"abstract\": \"Abstract Amyotrophic lateral sclerosis is a complex disorder most of which is \\u2018sporadic\\u2019 of unknown origin but approximately 10% is familial, arising from single mutations in any of more than 30 genes. Thus, there are more than 30 familial ALS subtypes, with different, often unknown, molecular pathologies leading to a complex constellation of clinical phenotypes. We have mouse models for many genetic forms of the disorder, but these do not, on their own, necessarily show us the key pathological pathways at work in human patients. To date, we have no models for the 90% of ALS that is \\u2018sporadic\\u2019. Potential therapies have been developed mainly using a limited set of mouse models, and through lack of alternatives, in the past these have been tested on patients regardless of aetiology. Cancer researchers have undertaken therapy development with similar challenges; they have responded by producing complex mouse models that have transformed understanding of pathological processes, and they have implemented patient stratification in multi-centre trials, leading to the effective translation of basic research findings to the clinic. ALS researchers have successfully adopted this combined approach, and now to increase our understanding of key disease pathologies, and our rate of progress for moving from mouse models to mechanism to ALS therapies we need more, innovative, complex mouse models to address specific questions.\", \"doi_y\": \"https://doi.org/10.1186/s13024-023-00619-2\", \"publication_date\": \"2023-05-04\"}, {\"work_id\": \"W4320709386\", \"doi_x\": \"10.1021/acsami.2c21187\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Surface-Functionalized Metal\\u2013Organic Frameworks for Binding Coronavirus Proteins\", \"abstract\": \"Since the outbreak of SARS-CoV-2, a multitude of strategies have been explored for the means of protection and shielding against virus particles: filtration equipment (PPE) has been widely used in daily life. In this work, we explore another approach in the form of deactivating coronavirus particles through selective binding onto the surface of metal-organic frameworks (MOFs) to further the fight against the transmission of respiratory viruses. MOFs are attractive materials in this regard, as their rich pore and surface chemistry can easily be modified on demand. The surfaces of three MOFs, UiO-66(Zr), UiO-66-NH2(Zr), and UiO-66-NO2(Zr), have been functionalized with repurposed antiviral agents, namely, folic acid, nystatin, and tenofovir, to enable specific interactions with the external spike protein of the SARS virus. Protein binding studies revealed that this surface modification significantly improved the binding affinity toward glycosylated and non-glycosylated proteins for all three MOFs. Additionally, the pores for the surface-functionalized MOFs can adsorb water, making them suitable for locally dehydrating microbial aerosols. Our findings highlight the immense potential of MOFs in deactivating respiratory coronaviruses to be better equipped to fight future pandemics.\", \"doi_y\": \"https://doi.org/10.1021/acsami.2c21187\", \"publication_date\": \"2023-02-14\"}, {\"work_id\": \"W4315643752\", \"doi_x\": \"10.1101/2023.01.05.522703\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Growth in a biofilm promotes conjugation of a<i>bla</i><sub>NDM-1</sub>-bearing plasmid between<i>Klebsiella pneumoniae</i>strains\", \"abstract\": \"Abstract Antimicrobial resistance (AMR) is a growing problem, especially in Gram-negative Enterobacteriaceae such as Klebsiella pneumoniae . Horizontal transfer of conjugative plasmids contributes to AMR gene dissemination. Bacteria such as K. pneumoniae commonly exist in biofilms, yet most studies focus on planktonic cultures. Here we studied the transfer of a multidrug resistance plasmid in planktonic and biofilm populations of K. pneumoniae . We determined plasmid transfer from a clinical isolate, CPE16, which carried four plasmids, including the 119-kbp bla NDM-1 -bearing F-type plasmid pCPE16_3, in planktonic and biofilm conditions. We found that transfer frequency of pCPE16_3 in a biofilm was orders-of-magnitude higher than between planktonic cells. In 5/7 sequenced transconjugants multiple plasmids had transferred. Plasmid acquisition had no detectable growth impact on transconjugants. Gene expression of the recipient and a transconjugant was investigated by RNA-sequencing in three lifestyles: planktonic exponential growth, planktonic stationary phase, and biofilm. We found that lifestyle had a substantial impact on chromosomal gene expression, and plasmid carriage affected chromosomal gene expression most in stationary planktonic and biofilm lifestyles. Furthermore, expression of plasmid genes was lifestyle-dependent, with unique signatures across the three conditions. Our study shows that growth in biofilm greatly increased the risk of conjugative transfer of a carbapenem resistance plasmid in K. pneumoniae without fitness costs and minimal transcriptional rearrangements, thus highlighting the importance of biofilms in the spread of AMR in this opportunistic pathogen. Importance Carbapenem-resistant K. pneumoniae is particularly problematic in hospital settings. Carbapenem resistance genes can transfer between bacteria via plasmid conjugation. Alongside drug resistance, K. pneumoniae can form biofilms on hospital surfaces, at infection sites and on implanted devices. Biofilms are naturally protected and can be inherently more tolerant to antimicrobials than their free-floating counterparts. There have been indications that plasmid transfer may be more likely in biofilm populations, thus creating a conjugation \\u2018hotspot\\u2019. However, there is no clear consensus on the effect of the biofilm lifestyle on plasmid transfer. Therefore, we aimed to explore the relationship between plasmid transfer and biofilms, and the impact of plasmid acquisition on the host bacterial cell. Our data show resistance plasmid transfer is greatly increased in a biofilm versus planktonic growth, which may be a significant contributing factor to the rapid dissemination of resistance plasmids in K. pneumoniae .\", \"doi_y\": \"https://doi.org/10.1101/2023.01.05.522703\", \"publication_date\": \"2023-01-11\"}, {\"work_id\": \"W4362722036\", \"doi_x\": \"10.1016/j.ceca.2023.102740\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The role of Ca2+ signalling in the pathology of exocrine pancreas\", \"abstract\": \"Exocrine pancreas has been the field of many successful studies in pancreatic physiology and pathology. However, related disease - acute pancreatitis (AP) is still takes it toll with more than 100,000 related deaths worldwide per year. In spite of significant scientific progress and several human trials currently running for AP, there is still no specific treatment in the clinic. Studies of the mechanism of initiation of AP have identified two crucial conditions: sustained elevations of cytoplasmic calcium concentration (Ca2+ plateau) and significantly reduced intracellular energy (ATP depletion). These hallmarks are interdependent, i.e., Ca2+ plateau increase energy demand for its clearance while energy production is greatly affected by the pathology. Result of long standing Ca2+ plateau is destabilisation of the secretory granules and premature activation of the digestive enzymes leading to necrotic cell death. Main attempts so far to break the vicious circle of cell death have been concentrated on reduction of Ca2+ overload or reduction of ATP depletion. This review will summarise these approaches, including recent developments of potential therapies for AP.\", \"doi_y\": \"https://doi.org/10.1016/j.ceca.2023.102740\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4365514760\", \"doi_x\": \"10.7554/elife.87178\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural and functional properties of the transporter SLC26A6 reveal mechanism of coupled anion exchange\", \"abstract\": \"Members of the SLC26 family constitute a conserved class of anion transport proteins, which encompasses uncoupled transporters with channel-like properties, coupled exchangers and motor proteins. Among the 10 functional paralogs in humans, several participate in the secretion of bicarbonate in exchange with chloride and thus play an important role in maintaining pH homeostasis. Previously, we have elucidated the structure of murine SLC26A9 and defined its function as an uncoupled chloride transporter (Walter et al., 2019). Here we have determined the structure of the closely related human transporter SLC26A6 and characterized it as a coupled exchanger of chloride with bicarbonate and presumably also oxalate. The structure defines an inward-facing conformation of the protein that generally resembles known structures of SLC26A9. The altered anion selectivity between both paralogs is a consequence of a remodeled ion binding site located in the center of a mobile unit of the membrane-inserted domain, which also accounts for differences in the coupling mechanism.\", \"doi_y\": \"https://doi.org/10.7554/elife.87178\", \"publication_date\": \"2023-06-23\"}, {\"work_id\": \"W4384120486\", \"doi_x\": \"10.1242/jcs.261097\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"UMAD1 contributes to ESCRT-III dynamic subunit turnover during cytokinetic abscission\", \"abstract\": \"ABSTRACT Abscission is the final stage of cytokinesis whereby the midbody, a thin intercellular bridge, is resolved to separate the daughter cells. Cytokinetic abscission is mediated by the endosomal sorting complex required for transport (ESCRT), a conserved membrane remodelling machinery. The midbody organiser CEP55 recruits early acting ESCRT factors such as ESCRT-I and ALIX (also known as PDCD6IP), which subsequently initiate the formation of ESCRT-III polymers that sever the midbody. We now identify UMAD1 as an ESCRT-I subunit that facilitates abscission. UMAD1 selectively associates with VPS37C and VPS37B, supporting the formation of cytokinesis-specific ESCRT-I assemblies. TSG101 recruits UMAD1 to the site of midbody abscission, to stabilise the CEP55\\u2013ESCRT-I interaction. We further demonstrate that the UMAD1\\u2013ESCRT-I interaction facilitates the final step of cytokinesis. Paradoxically, UMAD1 and ALIX co-depletion has synergistic effects on abscission, whereas ESCRT-III recruitment to the midbody is not inhibited. Importantly, we find that both UMAD1 and ALIX are required for the dynamic exchange of ESCRT-III subunits at the midbody. Therefore, UMAD1 reveals a key functional connection between ESCRT-I and ESCRT-III that is required for cytokinesis.\", \"doi_y\": \"https://doi.org/10.1242/jcs.261097\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4385064620\", \"doi_x\": \"10.1038/s41467-023-40125-z\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Mapping diversity in African trypanosomes using high resolution spatial proteomics\", \"abstract\": \"African trypanosomes are dixenous eukaryotic parasites that impose a significant human and veterinary disease burden on sub-Saharan Africa. Diversity between species and life-cycle stages is concomitant with distinct host and tissue tropisms within this group. Here, the spatial proteomes of two African trypanosome species, Trypanosoma brucei and Trypanosoma congolense, are mapped across two life-stages. The four resulting datasets provide evidence of expression of approximately 5500 proteins per cell-type. Over 2500 proteins per cell-type are classified to specific subcellular compartments, providing four comprehensive spatial proteomes. Comparative analysis reveals key routes of parasitic adaptation to different biological niches and provides insight into the molecular basis for diversity within and between these pathogen species.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40125-z\", \"publication_date\": \"2023-07-21\"}, {\"work_id\": \"W4322615507\", \"doi_x\": \"10.1146/annurev-arplant-070122-032532\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"abstract\": \"Chloroplasts are the defining plant organelles with responsibility for photosynthesis and other vital functions. To deliver these functions, they possess a complex proteome comprising thousands of largely nucleus-encoded proteins. Composition of the proteome is controlled by diverse processes affecting protein translocation and degradation-our focus here. Most chloroplast proteins are imported from the cytosol via multiprotein translocons in the outer and inner envelope membranes (the TOC and TIC complexes, respectively), or via one of several noncanonical pathways, and then sorted by different systems to organellar subcompartments. Chloroplast proteolysis is equally complex, involving the concerted action of internal proteases of prokaryotic origin and the nucleocytosolic ubiquitin-proteasome system (UPS). The UPS degrades unimported proteins in the cytosol and chloroplast-resident proteins via chloroplast-associated protein degradation (CHLORAD). The latter targets the TOC apparatus to regulate protein import, as well as numerous internal proteins directly, to reconfigure chloroplast functions in response to developmental and environmental signals.\", \"doi_y\": \"https://doi.org/10.1146/annurev-arplant-070122-032532\", \"publication_date\": \"2023-05-22\"}, {\"work_id\": \"W4308878080\", \"doi_x\": \"10.3389/fendo.2022.1031013\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Identification of unusual oxysterols biosynthesised in human pregnancy by charge-tagging and liquid chromatography - mass spectrometry\", \"abstract\": \"The aim of this study was to identify oxysterols and any down-stream metabolites in placenta, umbilical cord blood plasma, maternal plasma and amniotic fluid to enhance our knowledge of the involvement of these molecules in pregnancy. We confirm the identification of 20S-hydroxycholesterol in human placenta, previously reported in a single publication, and propose a pathway from 22R-hydroxycholesterol to a C 27 bile acid of probable structure 3\\u03b2,20R,22R-trihydroxycholest-5-en-(25R)26-oic acid. The pathway is evident not only in placenta, but pathway intermediates are also found in umbilical cord plasma, maternal plasma and amniotic fluid but not non-pregnant women.\", \"doi_y\": \"https://doi.org/10.3389/fendo.2022.1031013\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W4291825011\", \"doi_x\": \"10.1038/s41419-022-05146-4\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"EPIKOL, a chromatin-focused CRISPR/Cas9-based screening platform, to identify cancer-specific epigenetic vulnerabilities\", \"abstract\": \"Dysregulation of the epigenome due to alterations in chromatin modifier proteins commonly contribute to malignant transformation. To interrogate the roles of epigenetic modifiers in cancer cells, we generated an epigenome-wide CRISPR-Cas9 knockout library (EPIKOL) that targets a wide-range of epigenetic modifiers and their cofactors. We conducted eight screens in two different cancer types and showed that EPIKOL performs with high efficiency in terms of sgRNA distribution and depletion of essential genes. We discovered novel epigenetic modifiers that regulate triple-negative breast cancer (TNBC) and prostate cancer cell fitness. We confirmed the growth-regulatory functions of individual candidates, including SS18L2 and members of the NSL complex (KANSL2, KANSL3, KAT8) in TNBC cells. Overall, we show that EPIKOL, a focused sgRNA library targeting ~800 genes, can reveal epigenetic modifiers that are essential for cancer cell fitness under in vitro and in vivo conditions and enable the identification of novel anti-cancer targets. Due to its comprehensive epigenome-wide targets and relatively high number of sgRNAs per gene, EPIKOL will facilitate studies examining functional roles of epigenetic modifiers in a wide range of contexts, such as screens in primary cells, patient-derived xenografts as well as in vivo models.\", \"doi_y\": \"https://doi.org/10.1038/s41419-022-05146-4\", \"publication_date\": \"2022-08-16\"}, {\"work_id\": \"W4225416470\", \"doi_x\": \"10.1016/j.tips.2022.04.003\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NLRP3 and pyroptosis blockers for treating inflammatory diseases\", \"abstract\": \"The nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain-containing protein 3 (NLRP3) inflammasome is a key mediator of pathological inflammation in many diseases.There are several drug-like small molecules under clinical development as novel therapeutics targeting the NLRP3 inflammasome.Pyroptosis is a unique approach to suppress chronic inflammation.Understanding the mechanism of action by which small molecules block the NLRP3 inflammasome allows better molecular optimization.Different clinical trials are running using agents that inhibit the NLRP3 inflammasome. The nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain-containing protein 3 (NLRP3) inflammasome has emerged as a key mediator of pathological inflammation in many diseases and is an exciting drug target. Here, we review the molecular basis of NLRP3 inhibition by drug-like small molecules under development as novel therapeutics. We also summarize recent strategies to block pyroptosis as a novel approach to suppress chronic inflammation. Major recent developments in this area include the elucidation of mechanisms of action (MoAs) by which small molecules block NLRP3 inflammasome assembly and gasdermin D (GSDMD)-induced pyroptosis. We also discuss the status of clinical trials using agents that block specific components of the NLRP3 pathway, including their potential clinical applications for the treatment of many diseases. The nucleotide-binding oligomerization domain, leucine-rich repeat and pyrin domain-containing protein 3 (NLRP3) inflammasome has emerged as a key mediator of pathological inflammation in many diseases and is an exciting drug target. Here, we review the molecular basis of NLRP3 inhibition by drug-like small molecules under development as novel therapeutics. We also summarize recent strategies to block pyroptosis as a novel approach to suppress chronic inflammation. Major recent developments in this area include the elucidation of mechanisms of action (MoAs) by which small molecules block NLRP3 inflammasome assembly and gasdermin D (GSDMD)-induced pyroptosis. We also discuss the status of clinical trials using agents that block specific components of the NLRP3 pathway, including their potential clinical applications for the treatment of many diseases. NLRP3 (see Glossary) inflammasome is an intracellular protein complex that has emerged as a key mediator of inflammation in many pathologies. NLRP3 activation by microbes and danger signals (Figure 1) leads to the release of the proinflammatory cytokines IL-1\\u03b2 and IL-18, as well as to GSDMD-mediated pyroptosis, a proinflammatory type of cell death (Box 1).Box 1Pyroptosis: a gasdermin-meditated type of inflammatory cell deathPyroptosis is a type of programmed necrotic cell death characterized by the permeabilization of the plasma membrane by gasdermin protein family members, leading to the subsequent release of intracellular contents (for recent review, see [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]). The presence of cytosolic constituents in the extracellular milieu is a signal that the innate immune system recognizes through specific receptors that elicit an inflammatory response aimed to repair tissue damage. For example, proinflammatory cytokines of the IL-1 family, such as the alarmin IL-1\\u03b2, lack a signal peptide and, thus, their cellular release requires permeabilization of the plasma membrane [112.Chan A.H. Schroder K. Inflammasome signaling and regulation of interleukin-1 family cytokines.J. Exp. Med. 2020; 217: 1-10Crossref Google Scholar]. Gasdermins are an evolutionary conserved family of proteins that share two well-defined structural domains: an N-terminal domain with intrinsic pore-forming properties and a C-terminal domain that represses the pore-forming activities of the N-terminal domain, connected by a linker sequence [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]. Linker cleavage of gasdermin proteins liberates the N-terminal fragment, which oligomerizes and forms pores in cellular membranes, including the plasma membrane. Several distinct proteases cleave the linker of specific gasdermin family members. GSDMD is processed by caspase-1/-4/-5/-8/-11 and neutrophil elastase, whereas GSDMB is processed by granzyme A, and GSDME is cleaved by caspase-3 [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar,113.Demarco B. et al.Caspase-8-dependent gasdermin D cleavage promotes antimicrobial defense but confers susceptibility to TNF-induced lethality.Sci. Adv. 2020; 6: eabc3465Crossref PubMed Scopus (8) Google Scholar,114.Zhou Z. et al.Granzyme A from cytotoxic lymphocytes cleaves GSDMB to trigger pyroptosis in target cells.Science. 2020; 368: eaaz7548Crossref PubMed Scopus (267) Google Scholar]. GSDMD plasma membrane pores are positively regulated by ROS [40.Evavold C.L. et al.Control of gasdermin D oligomerization and pyroptosis by the Ragulator-Rag-mTORC1 pathway.Cell. 2021; 184: 4495-4511Abstract Full Text Full Text PDF PubMed Google Scholar] and negatively regulated by the scission of damaged membranes by the endosomal sorting complexes required for transport (ESCRT) machinery [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]. GSDMD pores are a negatively charged conduit for the direct passage of mature IL-1 cytokines [115.Xia S. et al.Gasdermin D pore structure reveals preferential release of mature interleukin-1.Nature. 2021; 593: 607-611Crossref PubMed Scopus (85) Google Scholar], which have a neutral or positively charged surface [37.Monteleone M. et al.Interleukin-1\\u03b2 maturation triggers its relocation to the plasma membrane for gasdermin-D-dependent and -independent secretion.Cell Rep. 2018; 24: 1425-1433Abstract Full Text Full Text PDF PubMed Scopus (147) Google Scholar]. In many settings, the extent of plasma membrane rupture induced by GSDMD pores in the dying cells is regulated by the protein ninjurin-1 [116.Kayagaki N. et al.NINJ1 mediates plasma membrane rupture during lytic cell death.Nature. 2021; 591: 131-136Crossref PubMed Scopus (99) Google Scholar], leading to the release of intracellular contents, including HMGB1, inflammasome oligomers, and mitochondrial DNA, or even inducing the expulsion of neutrophil nuclear DNA leading to NETosis [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar,39.Torre-Minguela C. et al.Gasdermins mediate cellular release of mitochondrial DNA during pyroptosis and apoptosis.FASEB J. 2021; 35e21757Crossref PubMed Scopus (7) Google Scholar,115.Xia S. et al.Gasdermin D pore structure reveals preferential release of mature interleukin-1.Nature. 2021; 593: 607-611Crossref PubMed Scopus (85) Google Scholar,117.Tan G. et al.HMGB1 released from GSDME-mediated pyroptotic epithelial cells participates in the tumorigenesis of colitis-associated colorectal cancer through the ERK1/2 pathway.J. Hematol. Oncol. 2020; 13: 149Crossref PubMed Scopus (40) Google Scholar]. Pyroptosis is a type of programmed necrotic cell death characterized by the permeabilization of the plasma membrane by gasdermin protein family members, leading to the subsequent release of intracellular contents (for recent review, see [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]). The presence of cytosolic constituents in the extracellular milieu is a signal that the innate immune system recognizes through specific receptors that elicit an inflammatory response aimed to repair tissue damage. For example, proinflammatory cytokines of the IL-1 family, such as the alarmin IL-1\\u03b2, lack a signal peptide and, thus, their cellular release requires permeabilization of the plasma membrane [112.Chan A.H. Schroder K. Inflammasome signaling and regulation of interleukin-1 family cytokines.J. Exp. Med. 2020; 217: 1-10Crossref Google Scholar]. Gasdermins are an evolutionary conserved family of proteins that share two well-defined structural domains: an N-terminal domain with intrinsic pore-forming properties and a C-terminal domain that represses the pore-forming activities of the N-terminal domain, connected by a linker sequence [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]. Linker cleavage of gasdermin proteins liberates the N-terminal fragment, which oligomerizes and forms pores in cellular membranes, including the plasma membrane. Several distinct proteases cleave the linker of specific gasdermin family members. GSDMD is processed by caspase-1/-4/-5/-8/-11 and neutrophil elastase, whereas GSDMB is processed by granzyme A, and GSDME is cleaved by caspase-3 [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar,113.Demarco B. et al.Caspase-8-dependent gasdermin D cleavage promotes antimicrobial defense but confers susceptibility to TNF-induced lethality.Sci. Adv. 2020; 6: eabc3465Crossref PubMed Scopus (8) Google Scholar,114.Zhou Z. et al.Granzyme A from cytotoxic lymphocytes cleaves GSDMB to trigger pyroptosis in target cells.Science. 2020; 368: eaaz7548Crossref PubMed Scopus (267) Google Scholar]. GSDMD plasma membrane pores are positively regulated by ROS [40.Evavold C.L. et al.Control of gasdermin D oligomerization and pyroptosis by the Ragulator-Rag-mTORC1 pathway.Cell. 2021; 184: 4495-4511Abstract Full Text Full Text PDF PubMed Google Scholar] and negatively regulated by the scission of damaged membranes by the endosomal sorting complexes required for transport (ESCRT) machinery [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]. GSDMD pores are a negatively charged conduit for the direct passage of mature IL-1 cytokines [115.Xia S. et al.Gasdermin D pore structure reveals preferential release of mature interleukin-1.Nature. 2021; 593: 607-611Crossref PubMed Scopus (85) Google Scholar], which have a neutral or positively charged surface [37.Monteleone M. et al.Interleukin-1\\u03b2 maturation triggers its relocation to the plasma membrane for gasdermin-D-dependent and -independent secretion.Cell Rep. 2018; 24: 1425-1433Abstract Full Text Full Text PDF PubMed Scopus (147) Google Scholar]. In many settings, the extent of plasma membrane rupture induced by GSDMD pores in the dying cells is regulated by the protein ninjurin-1 [116.Kayagaki N. et al.NINJ1 mediates plasma membrane rupture during lytic cell death.Nature. 2021; 591: 131-136Crossref PubMed Scopus (99) Google Scholar], leading to the release of intracellular contents, including HMGB1, inflammasome oligomers, and mitochondrial DNA, or even inducing the expulsion of neutrophil nuclear DNA leading to NETosis [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar,39.Torre-Minguela C. et al.Gasdermins mediate cellular release of mitochondrial DNA during pyroptosis and apoptosis.FASEB J. 2021; 35e21757Crossref PubMed Scopus (7) Google Scholar,115.Xia S. et al.Gasdermin D pore structure reveals preferential release of mature interleukin-1.Nature. 2021; 593: 607-611Crossref PubMed Scopus (85) Google Scholar,117.Tan G. et al.HMGB1 released from GSDME-mediated pyroptotic epithelial cells participates in the tumorigenesis of colitis-associated colorectal cancer through the ERK1/2 pathway.J. Hematol. Oncol. 2020; 13: 149Crossref PubMed Scopus (40) Google Scholar]. The past 5 years have seen rapid progress in the development of small-molecule inhibitors targeting the NLRP3 inflammasome pathway, with several NLRP3 and GSDMD inhibitors alleviating pathology in more than 100 preclinical disease models [1.Corcoran S.E. et al.Pharmacological inhibition of the nod-like receptor family pyrin domain containing 3 inflammasome with MCC950.Pharmacol. Rev. 2021; 73: 968-1000Crossref PubMed Google Scholar,2.Mangan M.S.J. et al.Targeting the NLRP3 inflammasome in inflammatory diseases.Nat. Rev. Drug Discov. 2018; 17: 588-606Crossref PubMed Google Scholar] (Tables 1 and 2 and Figure 2). These promising findings have fueled the rapid progression of NLRP3 inhibitors through human trials toward the clinic (Box 2), with GSDMD inhibitors closely following. Thus, it is both important and timely to review the landscape of emerging inflammasome drug candidates and their potential clinical applications. It is becoming particularly pressing to understand which inflammasome inhibitors to prioritize for trials in specific disease indications. Here, we discuss priority disease areas for NLRP3 inflammasome blockers, as well as the advantages and disadvantages of directly blocking the inflammasome (via NLRP3 inhibition) versus suppressing GSDMD-induced pyroptosis.Table 1NLRP3 inhibitorsaAbbreviations: AKT kinase, protein kinase B; DSS, dextran sulfate sodium; EAE, experimental autoimmune encephalomyelitis; FAK1, focal adhesion kinase 1; HSF1, heat shock factor 1; Hsp72, heat shock protein 72; JAK, Janus kinase; MAPK, mitogen-activated protein kinase; N/A, not applicable; NAFLD, nonalcoholic fatty liver disease; ND, Not determined ; PGE2, prostaglandin E2; RPL10, ribosomal protein L10; STAT, signal transducer and activator of transcription; SYK, spleen tyrosine kinase; TBI, traumatic brain injury; TGF-\\u03b2, transforming growth factor-\\u03b2; TRPV2, transient receptor potential cation channel subfamily V member 2; USP, ubiquitin-specific protease. InhibitorMoAIC50bIC50 values are included for IL-1\\u03b2 release from mouse macrophages stimulated with LPS and ATP or nigericin.Off-target effects (IC50)cKnown off-target inhibitory effects are summarized with IC50 values presented where possible.NLRP3-associated disease modeldDisease models in which inhibitors have been effective are summarized.MCC950Binds to NACHT domain, locking NLRP3 in an inactive conformationInhibits ATPase activity7.5 nMCarbonic anhydrase 2 (11 \\u03bcM)AD, PD, EAE, ischemic stroke, TBI, T2DM, age-related metabolic syndrome, diabetes-related co-morbidities: nephropathy, endothelial dysfunction, atherosclerosis, retinopathy, encephalopathy, cognitive impairment, ischemia\\u2013reperfusion injury, liver injury; NAFLD, NASH, liver fibrosis, IBD, colitis, and models reviewed in [1.Corcoran S.E. et al.Pharmacological inhibition of the nod-like receptor family pyrin domain containing 3 inflammasome with MCC950.Pharmacol. Rev. 2021; 73: 968-1000Crossref PubMed Google Scholar]CY-09Binds to Walker A site in NACHT domainInhibits ATPase activity6 \\u03bcMCytochrome P450 enzymes 1A2 (19 \\u03bcM), 2C9 (8 \\u03bcM), 2C19 (>50 \\u03bcM), 2D6 (>50 \\u03bcM), 3A4 (26 \\u03bcM)Stroke, T2DM, diabetic retinopathy, diabetic liver injury, NAFLD, peritonitis, Muckle\\u2013Wells syndrome, pain, osteoarthritis, myocardial infarction, epilepsy, osteosarcoma, acute kidney injury, hemolytic uremic syndromeBay-11-7082Inhibits ATPase activityBinding site unknown~3 \\u03bcMIKK (10 \\u03bcM), USP7 (0.19 \\u03bcM), USP21 (0.96 \\u03bcM)TBI, spinal cord injury, T2DM, IBD, psoriasis, lupus nephritis, acute kidney injury, acute lung injury, wound healingParthenolideInhibits ATPase activityBinding site unknown1.4 \\u03bcMIKK-\\u03b2 (7.5 \\u03bcM), caspase-1 (10 \\u03bcM), Hsp72 (>10 \\u03bcM), FAK1 (>10 \\u03bcM), RPL10 (>10 \\u03bcM), JAK2 (4 \\u03bcM)N/ABOT-4-oneInhibits ATPase activityBinding site unknown1.28 \\u03bcMIKK-\\u03b2 (~10 \\u03bcM)Peritonitis, dermatitis, psoriasis, arthritisMNSInhibits ATPase activityBinding site unknown~3 \\u03bcMSyk (2.5 \\u03bcM), Src kinase (29.3 \\u03bcM), p97 (1.7 \\u03bcM)Burn wound healing, airway inflammationOLT1177Inhibits ATPase activityBinding site unknownNDIn vivo NF-\\u03baB, I\\u03baB, IKKIncreases oxidative metabolism in vivoAD, EAE, colitis, arthritis, allergic asthma, acute kidney injuryINF39Inhibits ATPase activityBinding site unknown~10 \\u03bcMUnknownSpinal cord injury, IBD, colitis, drug-induced liver injury, pancreatitis, myocardial infarctionHS-203873Inhibits ATPase activity by competing with ATP binding~50 \\u03bcMUnknownN/ATranilastBinds to NACHT disrupting NLRP3\\u2013NLRP3 interactions and blocking oligomerization~25 \\u03bcMNumerous, e.g., TRPV2 (<75 \\u03bcM), inhibition of histamine (100 \\u03bcM), PGE2 (30 \\u03bcM), and cytokine release (TGF-\\u03b2, IL-6, IL-8, IL-5), collagen synthesisT2DM, NASH, colitis, peritonitis, Muckle\\u2013Wells syndrome, atherosclerosis, nanoparticle cerebral toxicity, myocardial infarctionOridoninCovalently modifies Cys279 in NACHT and attenuates NLRP3\\u2013NEK7 interaction0.75 \\u03bcMNumerous, e.g., AKT1 (8.4 \\u03bcM), AKT2 (8.9 \\u03bcM), HSF1, cMyc, NF-\\u03baB, JAK/STAT, p38 MAPKTBI, T2DM, alcohol-induced hepatotoxicity, liver fibrosis, IBD, ischemia\\u2013reperfusion injury, peritonitis, gouty arthritis, myocardial infarction, hemolytic uremic syndrome, acute lung injury, acute liver injury, ventilator-induced lung injury, hearing loss, pleurisyRRx-001Binds to Cys409 in NACHT and attenuates NLRP3\\u2013NEK7 interaction117 nMNumerous, e.g., inhibition of glucose 6-phosphate dehydrogenase and NF-\\u03baB, hemoglobin binding, apoptosisEAE, DSS colitis, allergic asthmaJC-171Attenuates NLRP3\\u2013ASC interaction. Binding site unknown8.45 \\u03bcMUnknownEAEa Abbreviations: AKT kinase, protein kinase B; DSS, dextran sulfate sodium; EAE, experimental autoimmune encephalomyelitis; FAK1, focal adhesion kinase 1; HSF1, heat shock factor 1; Hsp72, heat shock protein 72; JAK, Janus kinase; MAPK, mitogen-activated protein kinase; N/A, not applicable; NAFLD, nonalcoholic fatty liver disease; ND, Not determined ; PGE2, prostaglandin E2; RPL10, ribosomal protein L10; STAT, signal transducer and activator of transcription; SYK, spleen tyrosine kinase; TBI, traumatic brain injury; TGF-\\u03b2, transforming growth factor-\\u03b2; TRPV2, transient receptor potential cation channel subfamily V member 2; USP, ubiquitin-specific protease.b IC50 values are included for IL-1\\u03b2 release from mouse macrophages stimulated with LPS and ATP or nigericin.c Known off-target inhibitory effects are summarized with IC50 values presented where possible.d Disease models in which inhibitors have been effective are summarized. Open table in a new tab Table 2GSDMD inhibitorsaAbbreviations: GAPDH, glyceraldehyde 3-phosphate dehydrogenase; N/A, not applicable; Nrf2, nuclear factor erythroid 2-related factor 2. InhibitorMoAIC50bIC50 values are included for inflammasome-stimulated cell death.Off-target effects (IC50)cKnown off-target inhibitory effects are summarized, with IC50 values presented where possible.GSDMD-associated disease modeldDisease models in which inhibitors have been effective are summarized.NecrosulfonamideBinds directly to C191 blocking GSDMD oligomerization~10 \\u03bcMMLKL (0.2 \\u03bcM)LPS-induced sepsis, LPS-induced pulmonary fibrosisDimethyl fumarateSuccinates C191, blocking GSDMD oligomerization<10 \\u03bcMNumerous, e.g., aldehyde dehydrogenase (0.15\\u20131.45 \\u03bcM), dopamine beta-hydroxylase (<10 \\u03bcM), caspase-1 (~0.25 \\u03bcM), caspase-3 (~2 \\u03bcM), GSDME (<10 \\u03bcM)LPS-induced sepsis, familial Mediterranean fever, autoimmune encephalitisDisulfiramModifies C191, blocking GSDMD oligomerization~10 \\u03bcMNumerous, e.g., activates Nrf2, inhibits NFkB signaling, GAPDH, cyclooxygenase 2, NLRP3 signalingLPS-induced sepsis, pneumonia, renal fibrosis, gouty arthritisAntioxidants and mTORC1 inhibitorsBlock ROS-induced GSDMD oligomerizationN/AN/AN/Aa Abbreviations: GAPDH, glyceraldehyde 3-phosphate dehydrogenase; N/A, not applicable; Nrf2, nuclear factor erythroid 2-related factor 2.b IC50 values are included for inflammasome-stimulated cell death.c Known off-target inhibitory effects are summarized, with IC50 values presented where possible.d Disease models in which inhibitors have been effective are summarized. Open table in a new tab Box 2Early clinical trials of NLRP3 blockersFive companies have initiated clinical trials with specific NLRP3 inhibitors, although the results from most of these are not yet published. IFM Tre was acquired by Novartis in 2019, and their molecule IFM 2427 (later named DFV890) has reportedly completed Phase I safety trials. DFV890 will be tested in Phase II trials for the CAPS subtype familial cold autoinflammatory syndrome (NCT04868968) and knee osteoarthritis (NCT04886258). DFV890 has completed a Phase II trial for coronavirus disease 2019 (COVID-19) pneumonia (NCT04382053). DFV890 administrated to patients with impaired respiratory function because of COVID-19 pneumonia had no effect on disease severity as measured by the Acute Physiology and Chronic Health Evaluation II score, serum C-reactive protein levels, clinical status, or requirement for mechanical ventilation. Inflazome tested two molecules in Phase I trials. Inzomelid (IZD174; NCT04086602) is a brain-penetrant molecule, whereas Somalix (IZD334) is peripherally restricted and was tested in a Phase I and a Phase IB trial in CAPS (NCT04015076). No results from these trials have been released, but Roche acquired Inflazome in 2020i and Phase II trials might be initiated in the near future. In June 2020, NodTheraii announced a Phase I trial with their NLRP3 inhibitor NT-0167, but no further information on this trial is available. Recently, NodThera announced that they initiated a Phase I safety trial with a lead NLRP3 inhibitor NT-0796 (ACTRN12621001082897)iii,iv. Zydus Cadila developed MCC950-derivative molecules [20.Agarwal S. et al.Identification of a novel orally bioavailable NLRP3 inflammasome inhibitor.Bioorg. Med. Chem. Lett. 2020; 30127571Crossref Scopus (5) Google Scholar,36.Kennedy C.R. et al.A probe for NLRP3 inflammasome inhibitor MCC950 identifies carbonic anhydrase 2 as a novel target.ACS Chem. Biol. 2021; 16: 982-990Crossref PubMed Scopus (6) Google Scholar] and has initiated a Phase I safety trial for their molecule ZYIL1 (NCT04972188)v,vi. In addition, OLT1177 (aka dapansutrile) is being developed by Olatec Therapeutics and has concluded a Phase IB trial (NCT03534297) with no serious reported adverse events, in which it improved left ventricular ejection fraction in patients with heart failure and reduced ejection fraction. Dapansutrile was also tested in a Phase IIA trial in which it reduced joint pain during gout flares (EudraCT 2016-000943-14) [96.Wohlford G.F. et al.Phase 1B, randomized, double-blinded, dose escalation, single-center, repeat dose safety and pharmacodynamics study of the oral NLRP3 inhibitor dapansutrile in subjects with NYHA II-III systolic heart failure.J. Cardiovasc. Pharmacol. 2020; 77: 49-60Crossref PubMed Scopus (26) Google Scholar,118.Kl\\u00fcck V. et al.Dapansutrile, an oral selective NLRP3 inflammasome inhibitor, for treatment of gout flares: an open-label, dose-adaptive, proof-of-concept, phase 2a trial.Lancet Rheumatol. 2020; 2: e270-e280Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar]. Five companies have initiated clinical trials with specific NLRP3 inhibitors, although the results from most of these are not yet published. IFM Tre was acquired by Novartis in 2019, and their molecule IFM 2427 (later named DFV890) has reportedly completed Phase I safety trials. DFV890 will be tested in Phase II trials for the CAPS subtype familial cold autoinflammatory syndrome (NCT04868968) and knee osteoarthritis (NCT04886258). DFV890 has completed a Phase II trial for coronavirus disease 2019 (COVID-19) pneumonia (NCT04382053). DFV890 administrated to patients with impaired respiratory function because of COVID-19 pneumonia had no effect on disease severity as measured by the Acute Physiology and Chronic Health Evaluation II score, serum C-reactive protein levels, clinical status, or requirement for mechanical ventilation. Inflazome tested two molecules in Phase I trials. Inzomelid (IZD174; NCT04086602) is a brain-penetrant molecule, whereas Somalix (IZD334) is peripherally restricted and was tested in a Phase I and a Phase IB trial in CAPS (NCT04015076). No results from these trials have been released, but Roche acquired Inflazome in 2020i and Phase II trials might be initiated in the near future. In June 2020, NodTheraii announced a Phase I trial with their NLRP3 inhibitor NT-0167, but no further information on this trial is available. Recently, NodThera announced that they initiated a Phase I safety trial with a lead NLRP3 inhibitor NT-0796 (ACTRN12621001082897)iii,iv. Zydus Cadila developed MCC950-derivative molecules [20.Agarwal S. et al.Identification of a novel orally bioavailable NLRP3 inflammasome inhibitor.Bioorg. Med. Chem. Lett. 2020; 30127571Crossref Scopus (5) Google Scholar,36.Kennedy C.R. et al.A probe for NLRP3 inflammasome inhibitor MCC950 identifies carbonic anhydrase 2 as a novel target.ACS Chem. Biol. 2021; 16: 982-990Crossref PubMed Scopus (6) Google Scholar] and has initiated a Phase I safety trial for their molecule ZYIL1 (NCT04972188)v,vi. In addition, OLT1177 (aka dapansutrile) is being developed by Olatec Therapeutics and has concluded a Phase IB trial (NCT03534297) with no serious reported adverse events, in which it improved left ventricular ejection fraction in patients with heart failure and reduced ejection fraction. Dapansutrile was also tested in a Phase IIA trial in which it reduced joint pain during gout flares (EudraCT 2016-000943-14) [96.Wohlford G.F. et al.Phase 1B, randomized, double-blinded, dose escalation, single-center, repeat dose safety and pharmacodynamics study of the oral NLRP3 inhibitor dapansutrile in subjects with NYHA II-III systolic heart failure.J. Cardiovasc. Pharmacol. 2020; 77: 49-60Crossref PubMed Scopus (26) Google Scholar,118.Kl\\u00fcck V. et al.Dapansutrile, an oral selective NLRP3 inflammasome inhibitor, for treatment of gout flares: an open-label, dose-adaptive, proof-of-concept, phase 2a trial.Lancet Rheumatol. 2020; 2: e270-e280Abstract Full Text Full Text PDF PubMed Scopus (0) Google Scholar]. The inflammasome pathway is an important innate immune effector mechanism triggered in response to infection and tissue injury. Inflammasomes elicit an inflammatory response that aims to eliminate invading pathogens and repair damaged tissues, restoring homeostasis. Inflammasome assembly involves a series of protein oligomerization events induced after the recognition of specific molecular patterns from pathogens, cell damage, or altered homeostatic conditions, resulting in the formation of large multiprotein signaling platforms that activate the protease caspase-1 (Figure 1). The best-described inflammasomes are formed by the following sensor proteins: (i) nucleotide-binding oligomerization domain with leucine-rich repeat (NLR) and pyrin domain-containing 1 (NLRP1); (ii) NLRP3 (Figure 1); (iii) NLRP6; (iv) NLR family apoptosis inhibitory protein (NAIP), which functions in concert with the NLR family, caspase activation and recruitment domain (CARD)-containing 4 (NLRC4); (v) absent in melanoma 2 (AIM2); and (vi) Pyrin (PYD) [3.Christgen S. et al.Toward targeting inflammasomes: insights into their regulation and activation.Cell Res. 2020; 30: 315-327Crossref PubMed Scopus (87) Google Scholar]. These inflammasome sensor proteins recognize distinct perturbations of homeostasis and form oligomers able to recruit the adaptor protein apoptosis-associated speck-like protein containing a CARD (ASC) into the inflammasome through PYD\\u2013PYD or CARD\\u2013CARD homotypic domain interactions (Figure 1). ASC then oligomerizes to form helical filaments in which the zymogen pro-caspase-1 is recruited via CARD\\u2013CARD domain interactions to favor its activation [3.Christgen S. et al.Toward targeting inflammasomes: insights into their regulation and activation.Cell Res. 2020; 30: 315-327Crossref PubMed Scopus (87) Google Scholar]. Caspase-1 subsequently induces the proteolytic maturation of proinflammatory cytokines of the IL-1 family, such as IL-1\\u03b2 and IL-18 (Figure 1) [3.Christgen S. et al.Toward targeting inflammasomes: insights into their regulation and activation.Cell Res. 2020; 30: 315-327Crossref PubMed Scopus (87) Google Scholar]. Caspase-1 also processes the protein GSDMD, which releases its lytic N-terminal domain (N-GSDMD), which binds to the plasma membrane and oligomerizes to form pores, allowing the release of mature IL-1\\u03b2 and IL-18 (Figure 1). If the burden of plasma membrane pores is sufficiently high, this results in a proinflammatory necrotic type of cell death called pyroptosis (Box 1) [4.Broz P. et al.The gasdermins, a protein family executing cell death and inflammation.Nat. Rev. Immunol. 2020; 20: 143-157Crossref PubMed Scopus (355) Google Scholar]. During pyroptosis, the release of intracellular contents, including inflammasome oligomers, amplifies the inflammatory response and contributes to inflammatory pathology [5.Baroja-Mazo A. et al.The NLRP3 inflammasome is released as a particulate danger signal that amplifies the inflammatory response.Nat. Immunol. 2014; 15: 738-748Crossref PubMed Scopus (506) Google Scholar]. Among inflammasomes, the NLRP3 inflammasome is the most promiscuous, because it can be activated in response to diverse triggers, including pathogen- and danger-associated molecular patterns. Activators of the NLRP3 inflammasome include an array of viral and bacterial proteins (e.g., bacterial toxins, such as nigericin or viroporins), intracellular molecules released into the extracellular fluid upon cellular damage (su\", \"doi_y\": \"https://doi.org/10.1016/j.tips.2022.04.003\", \"publication_date\": \"2022-08-01\"}, {\"work_id\": \"W4307527792\", \"doi_x\": \"10.1101/2022.10.19.512731\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Next-generation ABACUS biosensors reveal cellular ABA dynamics driving root growth at low aerial humidity\", \"abstract\": \"Abstract The plant hormone abscisic acid (ABA) accumulates under abiotic stress to recast water relations and development. To overcome a lack of high-resolution, sensitive reporters, we developed ABACUS2s, next-generation FRET biosensors for ABA with high affinity, signal-to-noise ratio and orthogonality, that reveal endogenous ABA patterns in Arabidopsis thaliana. We mapped stress-induced ABA dynamics in high-resolution to reveal the cellular basis for local and systemic ABA functions. At reduced foliar humidity, roots cells accumulated ABA in the elongation zone, the site of phloem transported ABA unloading. Phloem ABA and root ABA signalling were both essential to maintain root growth at low humidity. ABA coordinates a robust system to maintain root growth in response to foliar stresses, enabling plants to maintain foraging of deeper soil for water uptake\", \"doi_y\": \"https://doi.org/10.1101/2022.10.19.512731\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4322486715\", \"doi_x\": \"10.1016/j.mib.2023.102282\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"How do interactions between mobile genetic elements affect horizontal gene transfer?\", \"abstract\": \"Horizontal gene transfer is central to bacterial adaptation and is facilitated by mobile genetic elements (MGEs). Increasingly, MGEs are being studied as agents with their own interests and adaptations, and the interactions MGEs have with one another are recognised as having a powerful effect on the flow of traits between microbes. Collaborations and conflicts between MGEs are nuanced and can both promote and inhibit the acquisition of new genetic material, shaping the maintenance of newly acquired genes and the dissemination of important adaptive traits through microbiomes. We review recent studies that shed light on this dynamic and oftentimes interlaced interplay, highlighting the importance of genome defence systems in mediating MGE\\u2013MGE conflicts, and outlining the consequences for evolutionary change, that resonate from the molecular to microbiome and ecosystem levels.\", \"doi_y\": \"https://doi.org/10.1016/j.mib.2023.102282\", \"publication_date\": \"2023-06-01\"}, {\"work_id\": \"W4225115349\", \"doi_x\": \"10.1136/bmjmed-2021-000040\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Covid-19: virology, variants, and vaccines\", \"abstract\": \"As of 25 January 2022, over 349 million individuals have received a confirmed diagnosis of covid-19, with over 5.59 million confirmed deaths associated with the SARS-CoV-2 virus. The covid-19 pandemic has prompted an extensive global effort to study the molecular evolution of the virus and develop vaccines to prevent its spread. Although rigorous determination of SARS-CoV-2 infectivity remains elusive, owing to the continuous evolution of the virus, steps have been made to understand its genome, structure, and emerging genetic mutations. The SARS-CoV-2 genome is composed of several open reading frames and structural proteins, including the spike protein, which is essential for entry into host cells. As of 25 January 2022, the World Health Organization has reported five variants of concern, two variants of interest, and three variants under monitoring. Additional sublineages have since been identified, and are being monitored. The mutations harboured in these variants confer an increased transmissibility, severity of disease, and escape from neutralising antibodies compared with the primary strain. The current vaccine strategy, including booster doses, provides protection from severe disease. As of 24 January 2022, 33 vaccines have been approved for use in 197 countries. In this review, we discuss the genetics, structure, and transmission methods of SARS-CoV-2 and its variants, highlighting how mutations provide enhanced abilities to spread and inflict disease. This review also outlines the vaccines currently in use around the world, providing evidence for every vaccine's immunogenicity and effectiveness.\", \"doi_y\": \"https://doi.org/10.1136/bmjmed-2021-000040\", \"publication_date\": \"2022-03-01\"}, {\"work_id\": \"W4386019616\", \"doi_x\": \"10.15252/embr.202357495\", \"af_direct_citation\": 0, \"af_indirect_citations\": 7, \"title\": \"NLR receptors in plant immunity: making sense of the alphabet soup\", \"abstract\": \"Plants coordinately use cell-surface and intracellular immune receptors to perceive pathogens and mount an immune response. Intracellular events of pathogen recognition are largely mediated by immune receptors of the nucleotide binding and leucine rich-repeat (NLR) classes. Upon pathogen perception, NLRs trigger a potent broad-spectrum immune reaction, usually accompanied by a form of programmed cell death termed the hypersensitive response. Some plant NLRs act as multifunctional singleton receptors which combine pathogen detection and immune signaling. However, NLRs can also function in higher order pairs and networks of functionally specialized interconnected receptors. In this article, we cover the basic aspects of plant NLR biology with an emphasis on NLR networks. We highlight some of the recent advances in NLR structure, function, and activation and discuss emerging topics such as modulator NLRs, pathogen suppression of NLRs, and NLR bioengineering. Multi-disciplinary approaches are required to disentangle how these NLR immune receptor pairs and networks function and evolve. Answering these questions holds the potential to deepen our understanding of the plant immune system and unlock a new era of disease resistance breeding.\", \"doi_y\": \"https://doi.org/10.15252/embr.202357495\", \"publication_date\": \"2023-08-21\"}, {\"work_id\": \"W4283802940\", \"doi_x\": \"10.3389/fphy.2022.910381\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Extending the Capabilities of Data-Driven Reduced-Order Models to Make Predictions for Unseen Scenarios: Applied to Flow Around Buildings\", \"abstract\": \"We present a data-driven or non-intrusive reduced-order model (NIROM) which is capable of making predictions for a significantly larger domain than the one used to generate the snapshots or training data. This development relies on the combination of a novel way of sampling the training data (which frees the NIROM from its dependency on the original problem domain) and a domain decomposition approach (which partitions unseen geometries in a manner consistent with the sub-sampling approach). The method extends current capabilities of reduced-order models to generalise, i.e., to make predictions for unseen scenarios. The method is applied to a 2D test case which simulates the chaotic time-dependent flow of air past buildings at a moderate Reynolds number using a computational fluid dynamics (CFD) code. The procedure for 3D problems is similar, however, a 2D test case is considered sufficient here, as a proof-of-concept. The reduced-order model consists of a sampling technique to obtain the snapshots; a convolutional autoencoder for dimensionality reduction; an adversarial network for prediction; all set within a domain decomposition framework. The autoencoder is chosen for dimensionality reduction as it has been demonstrated in the literature that these networks can compress information more efficiently than traditional (linear) approaches based on singular value decomposition. In order to keep the predictions realistic, properties of adversarial networks are exploited. To demonstrate its ability to generalise, once trained, the method is applied to a larger domain which has a different arrangement of buildings. Statistical properties of the flows from the reduced-order model are compared with those from the CFD model in order to establish how realistic the predictions are.\", \"doi_y\": \"https://doi.org/10.3389/fphy.2022.910381\", \"publication_date\": \"2022-07-04\"}, {\"work_id\": \"W4210726793\", \"doi_x\": \"10.1093/plcell/koac039\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Innovation and appropriation in mycorrhizal and rhizobial Symbioses\", \"abstract\": \"Most land plants benefit from endosymbiotic interactions with mycorrhizal fungi, including legumes and some nonlegumes that also interact with endosymbiotic nitrogen (N)-fixing bacteria to form nodules. In addition to these helpful interactions, plants are continuously exposed to would-be pathogenic microbes: discriminating between friends and foes is a major determinant of plant survival. Recent breakthroughs have revealed how some key signals from pathogens and symbionts are distinguished. Once this checkpoint has been passed and a compatible symbiont is recognized, the plant coordinates the sequential development of two types of specialized structures in the host. The first serves to mediate infection, and the second, which appears later, serves as sophisticated intracellular nutrient exchange interfaces. The overlap in both the signaling pathways and downstream infection components of these symbioses reflects their evolutionary relatedness and the common requirements of these two interactions. However, the different outputs of the symbioses, phosphate uptake versus N fixation, require fundamentally different components and physical environments and necessitated the recruitment of different master regulators, NODULE INCEPTION-LIKE PROTEINS, and PHOSPHATE STARVATION RESPONSES, for nodulation and mycorrhization, respectively.\", \"doi_y\": \"https://doi.org/10.1093/plcell/koac039\", \"publication_date\": \"2022-02-14\"}, {\"work_id\": \"W4221136828\", \"doi_x\": \"10.1038/s41576-022-00465-w\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA methylation-based predictors of health: applications and statistical considerations\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41576-022-00465-w\", \"publication_date\": \"2022-03-18\"}, {\"work_id\": \"W4385421713\", \"doi_x\": \"10.3390/molecules28155753\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Computational Investigation of Mechanisms for pH Modulation of Human Chloride Channels\", \"abstract\": \"Many transmembrane proteins are modulated by intracellular or extracellular pH. Investigation of pH dependence generally proceeds by mutagenesis of a wide set of amino acids, guided by properties such as amino-acid conservation and structure. Prediction of pKas can streamline this process, allowing rapid and effective identification of amino acids of interest with respect to pH dependence. Commencing with the calcium-activated chloride channel bestrophin 1, the carboxylate ligand structure around calcium sites relaxes in the absence of calcium, consistent with a measured lack of pH dependence. By contrast, less relaxation in the absence of calcium in TMEM16A, and maintenance of elevated carboxylate sidechain pKas, is suggested to give rise to pH-dependent chloride channel activity. This hypothesis, modulation of calcium/proton coupling and pH-dependent activity through the extent of structural relaxation, is shown to apply to the well-characterised cytosolic proteins calmodulin (pH-independent) and calbindin D9k (pH-dependent). Further application of destabilised, ionisable charge sites, or electrostatic frustration, is made to other human chloride channels (that are not calcium-activated), ClC-2, GABAA, and GlyR. Experimentally determined sites of pH modulation are readily identified. Structure-based tools for pKa prediction are freely available, allowing users to focus on mutagenesis studies, construct hypothetical proton pathways, and derive hypotheses such as the model for control of pH-dependent calcium activation through structural flexibility. Predicting altered pH dependence for mutations in ion channel disorders can support experimentation and, ultimately, clinical intervention.\", \"doi_y\": \"https://doi.org/10.3390/molecules28155753\", \"publication_date\": \"2023-07-30\"}, {\"work_id\": \"W4386424782\", \"doi_x\": \"10.1016/j.tem.2023.08.007\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pre-RNA splicing in metabolic homeostasis and liver disease\", \"abstract\": \"The liver plays a key role in sensing nutritional and hormonal inputs to maintain metabolic homeostasis. Recent studies into pre-mRNA splicing and alternative splicing (AS) and their effects on gene expression have revealed considerable transcriptional complexity in the liver, both in health and disease. While the contribution of these mechanisms to cell and tissue identity is widely accepted, their role in physiological and pathological contexts within tissues is just beginning to be appreciated. In this review, we showcase recent studies on the splicing and AS of key genes in metabolic pathways in the liver, the effect of metabolic signals on the spliceosome, and therapeutic intervention points based on RNA splicing.\", \"doi_y\": \"https://doi.org/10.1016/j.tem.2023.08.007\", \"publication_date\": \"2023-12-01\"}, {\"work_id\": \"W4381054504\", \"doi_x\": \"10.1002/anie.202305694\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Infrared Multiphoton Dissociation Enables Top\\u2010Down Characterization of Membrane Protein Complexes and G Protein\\u2010Coupled Receptors\", \"abstract\": \"Membrane proteins are challenging to analyze by native mass spectrometry (MS) as their hydrophobic nature typically requires stabilization in detergent micelles that are removed prior to analysis via collisional activation. There is however a practical limit to the amount of energy which can be applied, which often precludes subsequent characterization by top-down MS. To overcome this barrier, we have applied a modified Orbitrap Eclipse Tribrid mass spectrometer coupled to an infrared laser within a high-pressure linear ion trap. We show how tuning the intensity and time of incident photons enables liberation of membrane proteins from detergent micelles. Specifically, we relate the ease of micelle removal to the infrared absorption of detergents in both condensed and gas phases. Top-down MS via infrared multiphoton dissociation (IRMPD), results in good sequence coverage enabling unambiguous identification of membrane proteins and their complexes. By contrasting and comparing the fragmentation patterns of the ammonia channel with two class A GPCRs, we identify successive cleavage of adjacent amino acids within transmembrane domains. Using gas-phase molecular dynamics simulations, we show that areas prone to fragmentation maintain aspects of protein structure at increasing temperatures. Altogether, we propose a rationale to explain why and where in the protein fragment ions are generated.\", \"doi_y\": \"https://doi.org/10.1002/anie.202305694\", \"publication_date\": \"2023-07-25\"}, {\"work_id\": \"W3214785274\", \"doi_x\": \"10.1093/nar/gkab1127\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The European Bioinformatics Institute (EMBL-EBI) in 2021\", \"abstract\": \"The European Bioinformatics Institute (EMBL-EBI) maintains a comprehensive range of freely available and up-to-date molecular data resources, which includes over 40 resources covering every major data type in the life sciences. This year's service update for EMBL-EBI includes new resources, PGS Catalog and AlphaFold DB, and updates on existing resources, including the COVID-19 Data Platform, trRosetta and RoseTTAfold models introduced in Pfam and InterPro, and the launch of Genome Integrations with Function and Sequence by UniProt and Ensembl. Furthermore, we highlight projects through which EMBL-EBI has contributed to the development of community-driven data standards and guidelines, including the Recommended Metadata for Biological Images (REMBI), and the BioModels Reproducibility Scorecard. Training is one of EMBL-EBI's core missions and a key component of the provision of bioinformatics services to users: this year's update includes many of the improvements that have been developed to EMBL-EBI's online training offering.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkab1127\", \"publication_date\": \"2021-11-25\"}, {\"work_id\": \"W4313291879\", \"doi_x\": \"10.1101/2022.12.07.519495\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"TranceptEVE: Combining Family-specific and Family-agnostic Models of Protein Sequences for Improved Fitness Prediction\", \"abstract\": \"Abstract Modeling the fitness landscape of protein sequences has historically relied on training models on family-specific sets of homologous sequences called Multiple Sequence Alignments. Many proteins are however difficult to align or have shallow alignments which limits the potential scope of alignment-based methods. Not subject to these limitations, large protein language models trained on non-aligned sequences across protein families have achieved increasingly high predictive performance \\u2013 but have not yet fully bridged the gap with their alignment-based counterparts. In this work, we introduce TranceptEVE \\u2013 a hybrid method between family-specific and family-agnostic models that seeks to build on the relative strengths from each approach. Our method gracefully adapts to the depth of the alignment, fully relying on its autoregressive transformer when dealing with shallow alignments and leaning more heavily on the family-specific models for proteins with deeper alignments. Besides its broader application scope, it achieves state-of-the-art performance for mutation effects prediction, both in terms of correlation with experimental assays and with clinical annotations from ClinVar.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.07.519495\", \"publication_date\": \"2022-12-10\"}, {\"work_id\": \"W4385655467\", \"doi_x\": \"10.1371/journal.pbio.3002222\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Functional unknomics: Systematic screening of conserved genes of unknown function\", \"abstract\": \"The human genome encodes approximately 20,000 proteins, many still uncharacterised. It has become clear that scientific research tends to focus on well-studied proteins, leading to a concern that poorly understood genes are unjustifiably neglected. To address this, we have developed a publicly available and customisable \\\"Unknome database\\\" that ranks proteins based on how little is known about them. We applied RNA interference (RNAi) in Drosophila to 260 unknown genes that are conserved between flies and humans. Knockdown of some genes resulted in loss of viability, and functional screening of the rest revealed hits for fertility, development, locomotion, protein quality control, and resilience to stress. CRISPR/Cas9 gene disruption validated a component of Notch signalling and 2 genes contributing to male fertility. Our work illustrates the importance of poorly understood genes, provides a resource to accelerate future research, and highlights a need to support database curation to ensure that misannotation does not erode our awareness of our own ignorance.\", \"doi_y\": \"https://doi.org/10.1371/journal.pbio.3002222\", \"publication_date\": \"2023-08-08\"}, {\"work_id\": \"W4315752818\", \"doi_x\": \"10.1242/jcs.260462\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Multiple interactions of the dynein-2 complex with the IFT-B complex are required for effective intraflagellar transport\", \"abstract\": \"The dynein-2 complex must be transported anterogradely within cilia to then drive retrograde trafficking of the intraflagellar transport (IFT) machinery containing IFT-A and IFT-B complexes. Here, we screened for potential interactions between the dynein-2 and IFT-B complexes and found multiple interactions among the dynein-2 and IFT-B subunits. In particular, WDR60 (also known as DYNC2I1) and the DYNC2H1-DYNC2LI1 dimer from dynein-2, and IFT54 (also known as TRAF3IP1) and IFT57 from IFT-B contribute to the dynein-2-IFT-B interactions. WDR60 interacts with IFT54 via a conserved region N-terminal to its light chain-binding regions. Expression of the WDR60 constructs in WDR60-knockout (KO) cells revealed that N-terminal truncation mutants lacking the IFT54-binding site fail to rescue abnormal phenotypes of WDR60-KO cells, such as aberrant accumulation of the IFT machinery around the ciliary tip and on the distal side of the transition zone. However, a WDR60 construct specifically lacking just the IFT54-binding site substantially restored the ciliary defects. In line with the current docking model of dynein-2 with the anterograde IFT trains, these results indicate that extensive interactions involving multiple subunits from the dynein-2 and IFT-B complexes participate in their connection.\", \"doi_y\": \"https://doi.org/10.1242/jcs.260462\", \"publication_date\": \"2023-02-07\"}, {\"work_id\": \"W4206635163\", \"doi_x\": \"10.3390/genes13010158\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Inorganic Nitrogen Transport and Assimilation in Pea (Pisum sativum)\", \"abstract\": \"The genome sequences of several legume species are now available allowing the comparison of the nitrogen (N) transporter inventories with non-legume species. A survey of the genes encoding inorganic N transporters and the sensing and assimilatory families in pea, revealed similar numbers of genes encoding the primary N assimilatory enzymes to those in other types of plants. Interestingly, we find that pea and Medicago truncatula have fewer members of the NRT2 nitrate transporter family. We suggest that this difference may result from a decreased dependency on soil nitrate acquisition, as legumes have the capacity to derive N from a symbiotic relationship with diazotrophs. Comparison with M. truncatula, indicates that only one of three NRT2s in pea is likely to be functional, possibly indicating less N uptake before nodule formation and N-fixation starts. Pea seeds are large, containing generous amounts of N-rich storage proteins providing a reserve that helps seedling establishment and this may also explain why fewer high affinity nitrate transporters are required. The capacity for nitrate accumulation in the vacuole is another component of assimilation, as it can provide a storage reservoir that supplies the plant when soil N is depleted. Comparing published pea tissue nitrate concentrations with other plants, we find that there is less accumulation of nitrate, even in non-nodulated plants, and that suggests a lower capacity for vacuolar storage. The long-distance transported form of organic N in the phloem is known to be specialized in legumes, with increased amounts of organic N molecules transported, like ureides, allantoin, asparagine and amides in pea. We suggest that, in general, the lower tissue and phloem nitrate levels compared with non-legumes may also result in less requirement for high affinity nitrate transporters. The pattern of N transporter and assimilatory enzyme distribution in pea is discussed and compared with non-legumes with the aim of identifying future breeding targets.\", \"doi_y\": \"https://doi.org/10.3390/genes13010158\", \"publication_date\": \"2022-01-17\"}, {\"work_id\": \"W4220723714\", \"doi_x\": \"10.1111/febs.16422\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The intricate balance between microRNA\\u2010induced mRNA decay and translational repression\", \"abstract\": \"Post-transcriptional regulation of messenger RNAs (mRNAs) (i.e., mechanisms that control translation, stability and localization) is a critical focal point in spatiotemporal regulation of gene expression in response to changes in environmental conditions. The human genome encodes ~ 2000 microRNAs (miRNAs), each of which could control the expression of hundreds of protein-coding mRNAs by inducing translational repression and/or promoting mRNA decay. While mRNA degradation is a terminal event, translational repression is reversible and can be employed for rapid response to internal or external cues. Recent years have seen significant progress in our understanding of how miRNAs induce degradation or translational repression of the target mRNAs. Here, we review the recent findings that illustrate the cellular machinery that contributes to miRNA-induced silencing, with a focus on the factors that could influence translational repression vs. decay.\", \"doi_y\": \"https://doi.org/10.1111/febs.16422\", \"publication_date\": \"2022-03-11\"}, {\"work_id\": \"W4386783247\", \"doi_x\": \"10.1016/j.sbi.2023.102696\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Time resolved applications for Cryo-EM; approaches, challenges and future directions\", \"abstract\": \"Developments within the cryo-EM field have allowed us to generate higher-resolution \\u201cstatic\\u201d structures and pull out different conformational states which exist at equilibrium within the sample. Moreover, to trap non-equilibrium states and determine conformations that are present after a defined period of time (typically in the ms time frame) new approaches have been developed for the application of time-resolved cryo-EM. Here we give an overview of these different approaches and the limitations and strengths of each whilst identifying some of the current challenges to achieve higher resolutions and trap states within faster time frames. Time-resolved applications may play an important role in the ever-expanding toolkit of cryo-EM and open up new possibilities in both single particle and tomographic studies.\", \"doi_y\": \"https://doi.org/10.1016/j.sbi.2023.102696\", \"publication_date\": \"2023-12-01\"}, {\"work_id\": \"W4288051747\", \"doi_x\": \"10.1038/s41589-022-01088-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A new dawn beyond lysine ubiquitination\", \"abstract\": \"The ubiquitin system has become synonymous with the modification of lysine residues. However, the substrate scope and diversity of the conjugation machinery have been underappreciated, bringing us to an epoch in ubiquitin system research. The striking discoveries of metazoan enzymes dedicated toward serine and threonine ubiquitination have revealed the important role of nonlysine ubiquitination in endoplasmic reticulum\\u2013associated degradation, immune signaling and neuronal processes, while reports of nonproteinaceous substrates have extended ubiquitination beyond the proteome. Bacterial effectors that bypass the canonical ubiquitination machinery and form unprecedented linkage chemistry further redefine long-standing dogma. While chemical biology approaches have advanced our understanding of the canonical ubiquitin system, further study of noncanonical ubiquitination has been hampered by a lack of suitable tools. This Perspective aims to consolidate and contextualize recent discoveries and to propose potential applications of chemical biology, which will be instrumental in unraveling this new frontier of ubiquitin research. Emerging evidence shows that ubiquitination can occur on nonlysine residues and nonproteinaceous substrates. This perspective summarizes the recent discoveries in noncanonical ubiquitination and advocates the development of chemical biology tools for new substrate identification and mechanistic dissection.\", \"doi_y\": \"https://doi.org/10.1038/s41589-022-01088-2\", \"publication_date\": \"2022-07-27\"}, {\"work_id\": \"W3199364264\", \"doi_x\": \"10.3390/molecules26185629\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"The Transporter-Mediated Cellular Uptake and Efflux of Pharmaceutical Drugs and Biotechnology Products: How and Why Phospholipid Bilayer Transport Is Negligible in Real Biomembranes\", \"abstract\": \"Over the years, my colleagues and I have come to realise that the likelihood of pharmaceutical drugs being able to diffuse through whatever unhindered phospholipid bilayer may exist in intact biological membranes in vivo is vanishingly low. This is because (i) most real biomembranes are mostly protein, not lipid, (ii) unlike purely lipid bilayers that can form transient aqueous channels, the high concentrations of proteins serve to stop such activity, (iii) natural evolution long ago selected against transport methods that just let any undesirable products enter a cell, (iv) transporters have now been identified for all kinds of molecules (even water) that were once thought not to require them, (v) many experiments show a massive variation in the uptake of drugs between different cells, tissues, and organisms, that cannot be explained if lipid bilayer transport is significant or if efflux were the only differentiator, and (vi) many experiments that manipulate the expression level of individual transporters as an independent variable demonstrate their role in drug and nutrient uptake (including in cytotoxicity or adverse drug reactions). This makes such transporters valuable both as a means of targeting drugs (not least anti-infectives) to selected cells or tissues and also as drug targets. The same considerations apply to the exploitation of substrate uptake and product efflux transporters in biotechnology. We are also beginning to recognise that transporters are more promiscuous, and antiporter activity is much more widespread, than had been realised, and that such processes are adaptive (i.e., were selected by natural evolution). The purpose of the present review is to summarise the above, and to rehearse and update readers on recent developments. These developments lead us to retain and indeed to strengthen our contention that for transmembrane pharmaceutical drug transport \\u201cphospholipid bilayer transport is negligible\\u201d.\", \"doi_y\": \"https://doi.org/10.3390/molecules26185629\", \"publication_date\": \"2021-09-16\"}, {\"work_id\": \"W4285386858\", \"doi_x\": \"10.1038/s41564-022-01163-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Neutralizing antibody activity against 21 SARS-CoV-2 variants in older adults vaccinated with BNT162b2\", \"abstract\": \"SARS-CoV-2 variants may threaten the effectiveness of vaccines and antivirals to mitigate serious COVID-19 disease. This is of most concern in clinically vulnerable groups such as older adults. We analysed 72 sera samples from 37 individuals, aged 70-89 years, vaccinated with two doses of BNT162b2 (Pfizer-BioNTech) 3 weeks apart, for neutralizing antibody responses to wildtype SARS-CoV-2. Between 3 and 20 weeks after the second vaccine dose, neutralizing antibody titres fell 4.9-fold to a median titre of 21.3 (neutralization dose 80%), with 21.6% of individuals having no detectable neutralizing antibodies at the later time point. Next, we examined neutralization of 21 distinct SARS-CoV-2 variant spike proteins with these sera, and confirmed substantial antigenic escape, especially for the Omicron (B.1.1.529, BA.1/BA.2), Beta (B.1.351), Delta (B.1.617.2), Theta (P.3), C.1.2 and B.1.638 spike variants. By combining pseudotype neutralization with specific receptor-binding domain (RBD) enzyme-linked immunosorbent assays, we showed that changes to position 484 in the spike RBD were mainly responsible for SARS-CoV-2 neutralizing antibody escape. Nineteen sera from the same individuals boosted with a third dose of BNT162b2 contained higher neutralizing antibody titres, providing cross-protection against Omicron BA.1 and BA.2. Despite SARS-CoV-2 immunity waning over time in older adults, booster vaccines can elicit broad neutralizing antibodies against a large number of SARS-CoV-2 variants in this clinically vulnerable cohort.\", \"doi_y\": \"https://doi.org/10.1038/s41564-022-01163-3\", \"publication_date\": \"2022-07-14\"}, {\"work_id\": \"W4300817772\", \"doi_x\": \"10.1007/s10858-022-00403-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Optimizing frequency sampling in CEST experiments\", \"abstract\": \"For the past decade chemical exchange saturation transfer (CEST) experiments have been successfully applied to study exchange processes in biomolecules involving sparsely populated, transiently formed conformers. Initial implementations focused on extensive sampling of the CEST frequency domain, requiring significant measurement times. Here we show that the lengthy sampling schemes often used are not optimal and that reduced frequency sampling schedules can be developed without a priori knowledge of the exchange parameters, that only depend on the chosen B1 field, and, to a lesser extent, on the intrinsic transverse relaxation rates of ground state spins. The reduced sampling approach described here can be used synergistically with other methods for reducing measurement times such as those that excite multiple frequencies in the CEST dimension simultaneously, or make use of non-uniform sampling of indirectly detected time domains, to further decrease measurement times. The proposed approach is validated by analysis of simulated and experimental datasets.\", \"doi_y\": \"https://doi.org/10.1007/s10858-022-00403-2\", \"publication_date\": \"2022-10-04\"}, {\"work_id\": \"W4296551186\", \"doi_x\": \"10.1016/j.bbabio.2022.148919\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Formate hydrogenlyase, formic acid translocation and hydrogen production: dynamic membrane biology during fermentation\", \"abstract\": \"Formate hydrogenlyase-1 (FHL-1) is a complex-I-like enzyme that is commonly found in gram-negative bacteria. The enzyme comprises a peripheral arm and a membrane arm but is not involved in quinone reduction. Instead, FHL-1 couples formate oxidation to the reduction of protons to molecular hydrogen (H2). Escherichia coli produces FHL-1 under fermentative conditions where it serves to detoxify formic acid in the environment. The membrane biology and bioenergetics surrounding E. coli FHL-1 have long held fascination. Here, we review recent work on understanding the molecular basis of formic acid efflux and influx. We also consider the structure and function of E. coli FHL-1, its relationship with formate transport, and pay particular attention to the molecular interface between the peripheral arm and the membrane arm. Finally, we highlight the interesting phenotype of genetic mutation of the ND1 Loop, which is located at that interface.\", \"doi_y\": \"https://doi.org/10.1016/j.bbabio.2022.148919\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385566485\", \"doi_x\": \"10.1016/j.ijbiomac.2023.126066\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Obtaining high yield recombinant Enterococcus faecium nicotinate nucleotide adenylyltransferase for X-ray crystallography and biophysical studies\", \"abstract\": \"Nicotinate nucleotide adenylyltransferase (NNAT) has been a significant research focus on druggable targets, given its indispensability in the biosynthesis of NAD+, which is crucial to the survival of bacterial pathogens. However, no information is available on the structure-function of Enterococcus faecium NNAT (EfNNAT). This study established the expression and purification protocol for obtaining a high-yield recombinant EfNNAT using the E. coli expression system and a single-step IMAC purification method. Approximately 101 mg of EfNNAT was obtained per 7.8 g of wet E. coli cells, estimated to be over 98 % pure. We further characterized the biophysical structure and determined the three-dimensional structure of the EfNNAT. Biophysical studies revealed a dimeric protein with a higher \\u03b1-helical composition. The highly stable protein crystalizes in multiple conditions, yielding high-quality crystals diffracting between 1.78 and 2.80 \\u00c5. Two high-resolution crystal structures of EfNNAT in its native and adenine-bound forms were determined at 1.90 \\u00c5 and 1.82 \\u00c5, respectively. The X-ray structures of the EfNNAT revealed the presence of phosphate and sulfate ions occupying and interacting with conserved amino acid residues within the putative substrate binding site, hence providing insight into the probable substrate preference of EfNNAT and, consequently, why EfNNAT may not prefer \\u03b2-nicotinamide mononucleotide as a substrate. With the accessibility to high-resolution structures of EfNNAT, further structural evaluation and drug-based screening can be achieved. Hence, we anticipate that this study will provide the basis for the discovery of structure-based inhibitors against this enzyme.\", \"doi_y\": \"https://doi.org/10.1016/j.ijbiomac.2023.126066\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4294875221\", \"doi_x\": \"10.1038/s42003-022-03886-9\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Coiled-coil structure of meiosis protein TEX12 and conformational regulation by its C-terminal tip\", \"abstract\": \"Meiosis protein TEX12 is an essential component of the synaptonemal complex (SC), which mediates homologous chromosome synapsis. It is also recruited to centrosomes in meiosis, and aberrantly in certain cancers, leading to centrosome dysfunction. Within the SC, TEX12 forms an intertwined complex with SYCE2 that undergoes fibrous assembly, driven by TEX12's C-terminal tip. However, we hitherto lack structural information regarding SYCE2-independent functions of TEX12. Here, we report X-ray crystal structures of TEX12 mutants in three distinct conformations, and utilise solution light and X-ray scattering to determine its wild-type dimeric four-helical coiled-coil structure. TEX12 undergoes conformational change upon C-terminal tip mutations, indicating that the sequence responsible for driving SYCE2-TEX12 assembly within the SC also controls the oligomeric state and conformation of isolated TEX12. Our findings provide the structural basis for SYCE2-independent roles of TEX12, including the possible regulation of SC assembly, and its known functions in meiotic centrosomes and cancer.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03886-9\", \"publication_date\": \"2022-09-07\"}, {\"work_id\": \"W4386138222\", \"doi_x\": \"10.1038/s41467-023-40483-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Structural basis of peptidoglycan synthesis by E. coli RodA-PBP2 complex\", \"abstract\": \"Peptidoglycan (PG) is an essential structural component of the bacterial cell wall that is synthetized during cell division and elongation. PG forms an extracellular polymer crucial for cellular viability, the synthesis of which is the target of many antibiotics. PG assembly requires a glycosyltransferase (GT) to generate a glycan polymer using a Lipid II substrate, which is then crosslinked to the existing PG via a transpeptidase (TP) reaction. A Shape, Elongation, Division and Sporulation (SEDS) GT enzyme and a Class B Penicillin Binding Protein (PBP) form the core of the multi-protein complex required for PG assembly. Here we used single particle cryo-electron microscopy to determine the structure of a cell elongation-specific E. coli RodA-PBP2 complex. We combine this information with biochemical, genetic, spectroscopic, and computational analyses to identify the Lipid II binding sites and propose a mechanism for Lipid II polymerization. Our data suggest a hypothesis for the movement of the glycan strand from the Lipid II polymerization site of RodA towards the TP site of PBP2, functionally linking these two central enzymatic activities required for cell wall peptidoglycan biosynthesis.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-40483-8\", \"publication_date\": \"2023-08-24\"}, {\"work_id\": \"W4384282319\", \"doi_x\": \"10.1021/acscentsci.3c00395\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Proximity-Based Modalities for Biology and Medicine\", \"abstract\": \"Molecular proximity orchestrates biological function, and blocking existing proximities is an established therapeutic strategy. By contrast, strengthening or creating neoproximity with chemistry enables modulation of biological processes with high selectivity and has the potential to substantially expand the target space. A plethora of proximity-based modalities to target proteins via diverse approaches have recently emerged, opening opportunities for biopharmaceutical innovation. This Outlook outlines the diverse mechanisms and molecules based on induced proximity, including protein degraders, blockers, and stabilizers, inducers of protein post-translational modifications, and agents for cell therapy, and discusses opportunities and challenges that the field must address to mature and unlock translation in biology and medicine.\", \"doi_y\": \"https://doi.org/10.1021/acscentsci.3c00395\", \"publication_date\": \"2023-07-14\"}, {\"work_id\": \"W4308923992\", \"doi_x\": \"10.1016/j.jlr.2022.100313\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Genetic Mimicry Analysis Reveals the Specific Lipases Targeted by the ANGPTL3-ANGPTL8 Complex and ANGPTL4\", \"abstract\": \"Angiopoietin-like proteins, ANGPTL3, ANGPTL4, and ANGPTL8, are involved in regulating plasma lipids. In vitro and animal-based studies point to LPL and endothelial lipase (EL, LIPG) as key targets of ANGPTLs. To examine the ANGPTL mechanisms for plasma lipid modulation in humans, we pursued a genetic mimicry analysis of enhancing or suppressing variants in the LPL, LIPG, lipase C hepatic type (LIPC), ANGPTL3, ANGPTL4, and ANGPTL8 genes using data on 248 metabolic parameters derived from over 110,000 nonfasted individuals in the UK Biobank and validated in over 13,000 overnight fasted individuals from 11 other European populations. ANGPTL4 suppression was highly concordant with LPL enhancement but not HL or EL, suggesting ANGPTL4 impacts plasma metabolic parameters exclusively via LPL. The LPL-independent effects of ANGPTL3 suppression on plasma metabolic parameters showed a striking inverse resemblance with EL suppression, suggesting ANGPTL3 not only targets LPL but also targets EL. Investigation of the impact of the ANGPTL3-ANGPTL8 complex on plasma metabolite traits via the ANGPTL8 R59W substitution as an instrumental variable showed a much higher concordance between R59W and EL activity than between R59W and LPL activity, suggesting the R59W substitution more strongly affects EL inhibition than LPL inhibition. Meanwhile, when using a rare and deleterious protein-truncating ANGPTL8 variant as an instrumental variable, the ANGPTL3-ANGPTL8 complex was very LPL specific. In conclusion, our analysis provides strong human genetic evidence that the ANGPTL3-ANGPTL8 complex regulates plasma metabolic parameters, which is achieved by impacting LPL and EL. By contrast, ANGPTL4 influences plasma metabolic parameters exclusively via LPL.\", \"doi_y\": \"https://doi.org/10.1016/j.jlr.2022.100313\", \"publication_date\": \"2023-01-01\"}, {\"work_id\": \"W4385516203\", \"doi_x\": \"10.1007/s00418-023-02216-2\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Providing open imaging data at scale: An EMBL-EBI perspective\", \"abstract\": \"Abstract Biological imaging is one of the primary tools by which we understand living systems across scales from atoms to organisms. Rapid advances in imaging technology have increased both the spatial and temporal resolutions at which we examine those systems, as well as enabling visualisation of larger tissue volumes. These advances have huge potential but also generate ever increasing amounts of imaging data that must be stored and analysed. Public image repositories provide a critical scientific service through open data provision, supporting reproducibility of scientific results, access to reference imaging datasets and reuse of data for new scientific discovery and acceleration of image analysis methods development. The scale and scope of imaging data provides both challenges and opportunities for open sharing of image data. In this article, we provide a perspective influenced by decades of provision of open data resources for biological information, suggesting areas to focus on and a path towards global interoperability.\", \"doi_y\": \"https://doi.org/10.1007/s00418-023-02216-2\", \"publication_date\": \"2023-08-03\"}, {\"work_id\": \"W4320893582\", \"doi_x\": \"10.21203/rs.3.rs-2535247/v1\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Pro-phagocytic function and structural basis of GPR84 signaling\", \"abstract\": \"GPR84 is a unique orphan G protein-coupled receptor (GPCR) that can be activated by endogenous medium-chain fatty acids (MCFAs). The signaling of GPR84 is largely pro-inflammatory, which can augment inflammatory response, and GPR84 also functions as a pro-phagocytic receptor to enhance the phagocytic activities of macrophages. In this study, we first showed that the activation of GPR84 by the synthetic agonist 6-OAU could synergize with the blockade of CD47 on cancer cells to induce phagocytosis of cancer cells by macrophages. Then, we determined a high-resolution structure of the GPR84-Gi signaling complex with 6-OAU. This structure revealed a completely occluded binding pocket for 6-OAU, the molecular basis of receptor activation involving non-conserved structural motifs of GPR84, and an unusual Gi-coupling interface. Together with computational docking and simulations studies, our structure also suggested the mechanism for the high selectivity of GPR84 for MCFAs and the potential routes of ligand binding and dissociation. Our results provide a framework for understanding GPR84 signaling and developing new drugs targeting GPR84.\", \"doi_y\": \"https://doi.org/10.21203/rs.3.rs-2535247/v1\", \"publication_date\": \"2023-02-15\"}, {\"work_id\": \"W4312224088\", \"doi_x\": \"10.1038/s41467-022-35375-2\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"abstract\": \"Abstract Histone modifications are deposited by chromatin modifying enzymes and read out by proteins that recognize the modified state. BRD4-NUT is an oncogenic fusion protein of the acetyl lysine reader BRD4 that binds to the acetylase p300 and enables formation of long-range intra- and interchromosomal interactions. We here examine how acetylation reading and writing enable formation of such interactions. We show that NUT contains an acidic transcriptional activation domain that binds to the TAZ2 domain of p300. We use NMR to investigate the structure of the complex and found that the TAZ2 domain has an autoinhibitory role for p300. NUT-TAZ2 interaction or mutations found in cancer that interfere with autoinhibition by TAZ2 allosterically activate p300. p300 activation results in a self-organizing, acetylation-dependent feed-forward reaction that enables long-range interactions by bromodomain multivalent acetyl-lysine binding. We discuss the implications for chromatin organisation, gene regulation and dysregulation in disease.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-35375-2\", \"publication_date\": \"2022-12-15\"}, {\"work_id\": \"W4317473799\", \"doi_x\": \"10.1016/j.media.2023.102743\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Handcrafted Histological Transformer (H2T): Unsupervised representation of whole slide images\", \"abstract\": \"Diagnostic, prognostic and therapeutic decision-making of cancer in pathology clinics can now be carried out based on analysis of multi-gigapixel tissue images, also known as whole-slide images (WSIs). Recently, deep convolutional neural networks (CNNs) have been proposed to derive unsupervised WSI representations; these are attractive as they rely less on expert annotation which is cumbersome. However, a major trade-off is that higher predictive power generally comes at the cost of interpretability, posing a challenge to their clinical use where transparency in decision-making is generally expected. To address this challenge, we present a handcrafted framework based on deep CNN for constructing holistic WSI-level representations. Building on recent findings about the internal working of the Transformer in the domain of natural language processing, we break down its processes and handcraft them into a more transparent framework that we term as the Handcrafted Histological Transformer or H2T. Based on our experiments involving various datasets consisting of a total of 10,042 WSIs, the results demonstrate that H2T based holistic WSI-level representations offer competitive performance compared to recent state-of-the-art methods and can be readily utilized for various downstream analysis tasks. Finally, our results demonstrate that the H2T framework can be up to 14 times faster than the Transformer models.\", \"doi_y\": \"https://doi.org/10.1016/j.media.2023.102743\", \"publication_date\": \"2023-04-01\"}, {\"work_id\": \"W4385366065\", \"doi_x\": \"10.3390/jof9080787\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"The Yeast Protein Kinase Sch9 Functions as a Central Nutrient-Responsive Hub That Calibrates Metabolic and Stress-Related Responses\", \"abstract\": \"Yeast cells are equipped with different nutrient signaling pathways that enable them to sense the availability of various nutrients and adjust metabolism and growth accordingly. These pathways are part of an intricate network since most of them are cross-regulated and subject to feedback regulation at different levels. In yeast, a central role is played by Sch9, a protein kinase that functions as a proximal effector of the conserved growth-regulatory TORC1 complex to mediate information on the availability of free amino acids. However, recent studies established that Sch9 is more than a TORC1-effector as its activity is tuned by several other kinases. This allows Sch9 to function as an integrator that aligns different input signals to achieve accuracy in metabolic responses and stress-related molecular adaptations. In this review, we highlight the latest findings on the structure and regulation of Sch9, as well as its role as a nutrient-responsive hub that impacts on growth and longevity of yeast cells. Given that most key players impinging on Sch9 are well-conserved, we also discuss how studies on Sch9 can be instrumental to further elucidate mechanisms underpinning healthy aging in mammalians.\", \"doi_y\": \"https://doi.org/10.3390/jof9080787\", \"publication_date\": \"2023-07-26\"}, {\"work_id\": \"W4308368958\", \"doi_x\": \"10.15252/embj.2022111179\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"Structure and functional mapping of the KRAB\\u2010KAP1 repressor complex\", \"abstract\": \"Abstract Transposable elements are a genetic reservoir from which new genes and regulatory elements can emerge. However, expression of transposable elements can be pathogenic and is therefore tightly controlled. KRAB domain\\u2010containing zinc finger proteins (KRAB\\u2010ZFPs) recruit the co\\u2010repressor KRAB\\u2010associated protein 1 (KAP1/TRIM28) to regulate many transposable elements, but how KRAB\\u2010ZFPs and KAP1 interact remains unclear. Here, we report the crystal structure of the KAP1 tripartite motif (TRIM) in complex with the KRAB domain from a human KRAB\\u2010ZFP, ZNF93. Structure\\u2010guided mutations in the KAP1\\u2010KRAB binding interface abolished repressive activity in an epigenetic transcriptional silencing assay. Deposition of H3K9me3 over thousands of loci is lost genome\\u2010wide in cells expressing a KAP1 variant with mutations that abolish KRAB binding. Our work identifies and functionally validates the KRAB\\u2010KAP1 molecular interface, which is critical for a central transcriptional control axis in vertebrates. In addition, the structure\\u2010based prediction of KAP1 recruitment efficiency will enable optimization of KRABs used in CRISPRi.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111179\", \"publication_date\": \"2022-11-07\"}, {\"work_id\": \"W4382930913\", \"doi_x\": \"10.1073/pnas.2304714120\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Phase separation and molecular ordering of the prion-like domain of the Arabidopsis thermosensory protein EARLY FLOWERING 3\", \"abstract\": \"Liquid-liquid phase separation (LLPS) is an important mechanism enabling the dynamic compartmentalization of macromolecules, including complex polymers such as proteins and nucleic acids, and occurs as a function of the physicochemical environment. In the model plant, Arabidopsis thaliana, LLPS by the protein EARLY FLOWERING3 (ELF3) occurs in a temperature-sensitive manner and controls thermoresponsive growth. ELF3 contains a largely unstructured prion-like domain (PrLD) that acts as a driver of LLPS in vivo and in vitro. The PrLD contains a poly-glutamine (polyQ) tract, whose length varies across natural Arabidopsis accessions. Here, we use a combination of biochemical, biophysical, and structural techniques to investigate the dilute and condensed phases of the ELF3 PrLD with varying polyQ lengths. We demonstrate that the dilute phase of the ELF3 PrLD forms a monodisperse higher-order oligomer that does not depend on the presence of the polyQ sequence. This species undergoes LLPS in a pH- and temperature-sensitive manner and the polyQ region of the protein tunes the initial stages of phase separation. The liquid phase rapidly undergoes aging and forms a hydrogel as shown by fluorescence and atomic force microscopies. Furthermore, we demonstrate that the hydrogel assumes a semiordered structure as determined by small-angle X-ray scattering, electron microscopy, and X-ray diffraction. These experiments demonstrate a rich structural landscape for a PrLD protein and provide a framework to describe the structural and biophysical properties of biomolecular condensates.\", \"doi_y\": \"https://doi.org/10.1073/pnas.2304714120\", \"publication_date\": \"2023-07-03\"}, {\"work_id\": \"W4298086600\", \"doi_x\": \"10.1039/d2cc03638j\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Deracemisation and stereoinversion by a nanoconfined bidirectional enzyme cascade: dual control by electrochemistry and selective metal ion activation\", \"abstract\": \"The unique ability of the 'electrochemical leaf' (e-Leaf) to drive and control nanoconfined enzyme cascades bidirectionally, while directly monitoring their rate in real-time as electrical current, is exploited to achieve deracemisation and stereoinversion of secondary alcohols using a single electrode in one pot. Two alcohol dehydrogenase enzymes with opposing enantioselectivities, from Thermoanaerobacter ethanolicus (selective for S) and Lactobacillus kefir (selective for R) are driven bidirectionally via coupling to the fast and quasi-reversible interconversion of NADP+/NADPH catalysed by ferredoxin NADP+ reductase - all enzymes being co-entrapped in a nanoporous indium tin oxide electrode. Activity of the Lactobacillus kefir enzyme depends on the binding of a non-catalytic Mg2+, allowing it to be switched off after an oxidative half-cycle, by adding EDTA - the S-selective enzyme, with a tightly-bound Zn2+, remaining fully active. Racemate \\u2192 S or R \\u2192 S conversions are thus achieved in high yield with unprecedented ease.\", \"doi_y\": \"https://doi.org/10.1039/d2cc03638j\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4310954939\", \"doi_x\": \"10.1111/jipb.13422\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"DNA methylation dynamics during germline development\", \"abstract\": \"DNA methylation plays essential homeostatic functions in eukaryotic genomes. In animals, DNA methylation is also developmentally regulated and, in turn, regulates development. In the past two decades, huge research effort has endorsed the understanding that DNA methylation plays a similar role in plant development, especially during sexual reproduction. The power of whole-genome sequencing and cell isolation techniques, as well as bioinformatics tools, have enabled recent studies to reveal dynamic changes in DNA methylation during germline development. Furthermore, the combination of these technological advances with genetics, developmental biology and cell biology tools has revealed functional methylation reprogramming events that control gene and transposon activities in flowering plant germlines. In this review, we discuss the major advances in our knowledge of DNA methylation dynamics during male and female germline development in flowering plants.\", \"doi_y\": \"https://doi.org/10.1111/jipb.13422\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4311935814\", \"doi_x\": \"10.1101/2022.12.16.520778\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Quantifying constraint in the human mitochondrial genome\", \"abstract\": \"Abstract Mitochondrial DNA (mtDNA) has an important, yet often overlooked, role in health and disease. Constraint models quantify the removal of deleterious variation from the population by selection, representing a powerful tool for identifying genetic variation underlying human phenotypes 1\\u20134 . However, a constraint model for the mtDNA has not been developed, due to its unique features. Here we describe the development of a mitochondrial constraint model and its application to the Genome Aggregation Database (gnomAD), a large-scale population dataset reporting mtDNA variation across 56,434 humans 5 . Our results demonstrate strong depletion of expected variation, suggesting most deleterious mtDNA variants remain undiscovered. To aid their identification, we compute constraint metrics for every mitochondrial protein, tRNA, and rRNA gene, revealing a spectrum of intolerance to variation. We characterize the most constrained regions within genes via regional constraint, and positions across the entire mtDNA via local constraint, showing their enrichment in pathogenic variation and functionally critical sites, including topological clustering in 3D protein and RNA structures. Notably, we identify constraint at often overlooked sites, such as rRNAs and non-coding regions. Lastly, we demonstrate how these metrics can improve the discovery of mtDNA variation underlying rare and common human phenotypes.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.16.520778\", \"publication_date\": \"2022-12-19\"}, {\"work_id\": \"W4206620032\", \"doi_x\": \"10.1016/j.addr.2021.114098\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Advancing pharmacy and healthcare with virtual digital technologies\", \"abstract\": \"Digitalisation of the healthcare sector promises to revolutionise patient healthcare globally. From the different technologies, virtual tools including artificial intelligence, blockchain, virtual, and augmented reality, to name but a few, are providing significant benefits to patients and the pharmaceutical sector alike, ranging from improving access to clinicians and medicines, as well as improving real-time diagnoses and treatments. Indeed, it is envisioned that such technologies will communicate together in real-time, as well as with their physical counterparts, to create a large-scale, cyber healthcare system. Despite the significant benefits that virtual-based digital health technologies can bring to patient care, a number of challenges still remain, ranging from data security to acceptance within the healthcare sector. This review provides a timely account of the benefits and challenges of virtual health interventions, as well an outlook on how such technologies can be transitioned from research-focused towards real-world healthcare and pharmaceutical applications to transform treatment pathways for patients worldwide.\", \"doi_y\": \"https://doi.org/10.1016/j.addr.2021.114098\", \"publication_date\": \"2022-03-01\"}, {\"work_id\": \"W4379768914\", \"doi_x\": \"10.1038/s41467-023-37826-w\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Generation of SARS-CoV-2 escape mutations by monoclonal antibody therapy\", \"abstract\": \"COVID-19 patients at risk of severe disease may be treated with neutralising monoclonal antibodies (mAbs). To minimise virus escape from neutralisation these are administered as combinations e.g. casirivimab+imdevimab or, for antibodies targeting relatively conserved regions, individually e.g. sotrovimab. Unprecedented genomic surveillance of SARS-CoV-2 in the UK has enabled a genome-first approach to detect emerging drug resistance in Delta and Omicron cases treated with casirivimab+imdevimab and sotrovimab respectively. Mutations occur within the antibody epitopes and for casirivimab+imdevimab multiple mutations are present on contiguous raw reads, simultaneously affecting both components. Using surface plasmon resonance and pseudoviral neutralisation assays we demonstrate these mutations reduce or completely abrogate antibody affinity and neutralising activity, suggesting they are driven by immune evasion. In addition, we show that some mutations also reduce the neutralising activity of vaccine-induced serum.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-37826-w\", \"publication_date\": \"2023-06-07\"}, {\"work_id\": \"W4307407432\", \"doi_x\": \"10.1107/s2059798322009986\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"abstract\": \"Crystallographic fragment screens provide an efficient and effective way to identify small-molecule ligands of a crystallized protein. Due to their low molecular weight, such hits tend to have low, often unquantifiable, affinity for their target, complicating the twin challenges of validating the hits as authentic solution-phase ligands of the target and identifying the `best' hit(s) for further elaboration. In this article, approaches that address these challenges are assessed. Using retrospective analysis of a recent ATAD2 hit-identification campaign, alongside other examples of successful fragment-screening campaigns, it is suggested that hit validation and prioritization are best achieved by a `triangulation' approach in which the results of multiple available biochemical and biophysical techniques are correlated to develop qualitative structure-activity relationships (SARs). Such qualitative SARs may indeed be the only means by which to navigate a project through the tunnel of uncertainty that prevails before on-scale biophysical, biochemical and/or biological measurements become possible.\", \"doi_y\": \"https://doi.org/10.1107/s2059798322009986\", \"publication_date\": \"2022-10-27\"}, {\"work_id\": \"W4296323389\", \"doi_x\": \"10.15252/embj.2022111015\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"abstract\": \"Abstract Protein UFMylation, i.e., post\\u2010translational modification with ubiquitin\\u2010fold modifier 1 (UFM1), is essential for cellular and endoplasmic reticulum homeostasis. Despite its biological importance, we have a poor understanding of how UFM1 is conjugated onto substrates. Here, we use a rebuilding approach to define the minimal requirements of protein UFMylation. We find that the reported cognate E3 ligase UFL1 is inactive on its own and instead requires the adaptor protein UFBP1 to form an active E3 ligase complex. Structure predictions suggest the UFL1/UFBP1 complex to be made up of winged helix (WH) domain repeats. We show that UFL1/UFBP1 utilizes a scaffold\\u2010type E3 ligase mechanism that activates the UFM1\\u2010conjugating E2 enzyme, UFC1, for aminolysis. Further, we characterize a second adaptor protein CDK5RAP3 that binds to and forms an integral part of the ligase complex. Unexpectedly, we find that CDK5RAP3 inhibits UFL1/UFBP1 ligase activity in vitro . Results from reconstituting ribosome UFMylation suggest that CDK5RAP3 functions as a substrate adaptor that directs UFMylation to the ribosomal protein RPL26. In summary, our reconstitution approach reveals the biochemical basis of UFMylation and regulatory principles of this atypical E3 ligase complex.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111015\", \"publication_date\": \"2022-09-19\"}, {\"work_id\": \"W4317690277\", \"doi_x\": \"10.1111/febs.16730\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"A guide to UFMylation, an emerging posttranslational modification\", \"abstract\": \"Ubiquitin Fold Modifier-1 (UFM1) is a ubiquitin-like modifier (UBL) that is posttranslationally attached to lysine residues on substrates via a dedicated system of enzymes conserved in most eukaryotes. Despite the structural similarity between UFM1 and ubiquitin, the UFMylation machinery employs unique mechanisms that ensure fidelity. While physiological triggers and consequences of UFMylation are not entirely clear, its biological importance is epitomized by mutations in the UFMylation pathway in human pathophysiology including musculoskeletal and neurodevelopmental diseases. Some of these diseases can be explained by the increased endoplasmic reticulum (ER) stress and disrupted translational homeostasis observed upon loss of UFMylation. The roles of UFM1 in these processes likely stem from its function at the ER where ribosomes are UFMylated in response to translational stalling. In addition, UFMylation has been implicated in other cellular processes including DNA damage response and telomere maintenance. Hence, the study of UFM1 pathway mechanics and its biological function will reveal insights into fundamental cell biology and is likely to afford new therapeutic opportunities for the benefit of human health. To this end, we herein provide a comprehensive guide to the current state of knowledge of UFM1 biogenesis, conjugation, and function with an emphasis on the underlying mechanisms.\", \"doi_y\": \"https://doi.org/10.1111/febs.16730\", \"publication_date\": \"2023-02-08\"}, {\"work_id\": \"W4378070558\", \"doi_x\": \"10.1021/acssynbio.3c00231\", \"af_direct_citation\": 1, \"af_indirect_citations\": 5, \"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"abstract\": \"Synthetic biology applications would benefit from protein modules of reduced complexity that function orthogonally to cellular components. As many subcellular processes depend on peptide\\u2013protein or protein\\u2013protein interactions, de novo designed polypeptides that can bring together other proteins controllably are particularly useful. Thanks to established sequence-to-structure relationships, helical bundles provide good starting points for such designs. Typically, however, such designs are tested in vitro and function in cells is not guaranteed. Here, we describe the design, characterization, and application of de novo helical hairpins that heterodimerize to form 4-helix bundles in cells. Starting from a rationally designed homodimer, we construct a library of helical hairpins and identify complementary pairs using bimolecular fluorescence complementation in E. coli. We characterize some of the pairs using biophysics and X-ray crystallography to confirm heterodimeric 4-helix bundles. Finally, we demonstrate the function of an exemplar pair in regulating transcription in both E. coli and mammalian cells.\", \"doi_y\": \"https://doi.org/10.1021/acssynbio.3c00231\", \"publication_date\": \"2023-05-24\"}, {\"work_id\": \"W4311838198\", \"doi_x\": \"10.1101/2022.12.03.22282974\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Non-generalizability of biomarkers for mortality in SARS-CoV-2: a meta-analyses series\", \"abstract\": \"ABSTRACT Rationale Sophisticated prognostic scores have been proposed for SARS-CoV-2 but do not always perform consistently. We conducted these meta-analyses to uncover why and to investigate the impact of vaccination and variants. Methods We searched the PubMed database for the keywords \\u2018SARS-CoV-2\\u2019 with \\u2018biomarker\\u2019 and \\u2018mortality\\u2019 for the baseline tranche (01/12/2020-30/06/2021) and either \\u2018SARS-CoV-2\\u2019 or \\u2018Covid19\\u2019 with \\u2018biomarker\\u2019 and either \\u2018vaccination\\u2019 or \\u2018variant\\u2019 from 01/12/2020 to 31/10/2023. To aggregate the data, the meta library in R was used, and a random effects model fitted to obtain pooled AUCs and 95% confidence intervals for the European/North American, Asian, and overall datasets. Results Biomarker effectiveness varies significantly in different continents. Admission CRP levels were a good prognostic marker for mortality due to wild-type virus in Asian countries, with a pooled area under curve (AUC) of 0.83 (95%CI 0.80-0.85), but only an average predictor of mortality in Europe/North America, with a pooled AUC of 0.67 (95%CI 0.63-0.71, P&lt; 0.0001). We observed the same pattern for D-dimer and IL-6. This variability explains why the proposed prognostic scores did not perform evenly. Notably, urea and troponin had pooled AUCs \\u22650.78 regardless of location, implying that end-organ damage at presentation is a key prognostic factor. The inflammatory biomarkers (CRP, D-dimer and IL-6) have generally declined in effectiveness in the vaccinated and variant cohorts. We note a significant lag from the pandemic advent to data availability and this has no doubt impacted on patient care. Conclusions Biomarker efficacies vary considerably by region. It is imperative that the infrastructure for collecting clinical data should be put in place ahead of a future pandemic.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.03.22282974\", \"publication_date\": \"2022-12-07\"}, {\"work_id\": \"W4313280753\", \"doi_x\": \"10.15252/embj.2022111519\", \"af_direct_citation\": 0, \"af_indirect_citations\": 4, \"title\": \"Sensor <scp>NLR</scp> immune proteins activate oligomerization of their <scp>NRC</scp> helpers in response to plant pathogens\", \"abstract\": \"Abstract Nucleotide\\u2010binding domain leucine\\u2010rich repeat (NLR) immune receptors are important components of plant and metazoan innate immunity that can function as individual units or as pairs or networks. Upon activation, NLRs form multiprotein complexes termed resistosomes or inflammasomes. Although metazoan paired NLRs, such as NAIP/NLRC4, form hetero\\u2010complexes upon activation, the molecular mechanisms underpinning activation of plant paired NLRs, especially whether they associate in resistosome hetero\\u2010complexes, is unknown. In asterid plant species, the NLR required for cell death (NRC) immune receptor network is composed of multiple resistance protein sensors and downstream helpers that confer immunity against diverse plant pathogens. Here, we show that pathogen effector\\u2010activation of the NLR proteins Rx (confers virus resistance), and Bs2 (confers bacterial resistance) leads to oligomerization of their helper NLR, NRC2. Activated Rx does not oligomerize or enter into a stable complex with the NRC2 oligomer and remains cytoplasmic. In contrast, activated NRC2 oligomers accumulate in membrane\\u2010associated puncta. We propose an activation\\u2010and\\u2010release model for NLRs in the NRC immune receptor network. This points to a distinct activation model compared with mammalian paired NLRs.\", \"doi_y\": \"https://doi.org/10.15252/embj.2022111519\", \"publication_date\": \"2022-12-29\"}, {\"work_id\": \"W4376134661\", \"doi_x\": \"10.1038/s41591-023-02343-2\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Accelerated waning of the humoral response to COVID-19 vaccines in obesity\", \"abstract\": \"Obesity is associated with an increased risk of severe Coronavirus Disease 2019 (COVID-19) infection and mortality. COVID-19 vaccines reduce the risk of serious COVID-19 outcomes; however, their effectiveness in people with obesity is incompletely understood. We studied the relationship among body mass index (BMI), hospitalization and mortality due to COVID-19 among 3.6 million people in Scotland using the Early Pandemic Evaluation and Enhanced Surveillance of COVID-19 (EAVE II) surveillance platform. We found that vaccinated individuals with severe obesity (BMI > 40 kg/m2) were 76% more likely to experience hospitalization or death from COVID-19 (adjusted rate ratio of 1.76 (95% confidence interval (CI), 1.60-1.94). We also conducted a prospective longitudinal study of a cohort of 28 individuals with severe obesity compared to 41 control individuals with normal BMI (BMI 18.5-24.9 kg/m2). We found that 55% of individuals with severe obesity had unquantifiable titers of neutralizing antibody against authentic severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus compared to 12% of individuals with normal BMI (P = 0.0003) 6 months after their second vaccine dose. Furthermore, we observed that, for individuals with severe obesity, at any given anti-spike and anti-receptor-binding domain (RBD) antibody level, neutralizing capacity was lower than that of individuals with a normal BMI. Neutralizing capacity was restored by a third dose of vaccine but again declined more rapidly in people with severe obesity. We demonstrate that waning of COVID-19 vaccine-induced humoral immunity is accelerated in individuals with severe obesity. As obesity is associated with increased hospitalization and mortality from breakthrough infections, our findings have implications for vaccine prioritization policies.\", \"doi_y\": \"https://doi.org/10.1038/s41591-023-02343-2\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4379928262\", \"doi_x\": \"10.1371/journal.ppat.1011433\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The host phylogeny determines viral infectivity and replication across Staphylococcus host species\", \"abstract\": \"Virus host shifts, where a virus transmits to and infects a novel host species, are a major source of emerging infectious disease. Genetic similarity between eukaryotic host species has been shown to be an important determinant of the outcome of virus host shifts, but it is unclear if this is the case for prokaryotes where anti-virus defences can be transmitted by horizontal gene transfer and evolve rapidly. Here, we measure the susceptibility of 64 strains of Staphylococcaceae bacteria (48 strains of Staphylococcus aureus and 16 non-S. aureus species spanning 2 genera) to the bacteriophage ISP, which is currently under investigation for use in phage therapy. Using three methods-plaque assays, optical density (OD) assays, and quantitative (q)PCR-we find that the host phylogeny explains a large proportion of the variation in susceptibility to ISP across the host panel. These patterns were consistent in models of only S. aureus strains and models with a single representative from each Staphylococcaceae species, suggesting that these phylogenetic effects are conserved both within and among host species. We find positive correlations between susceptibility assessed using OD and qPCR and variable correlations between plaque assays and either OD or qPCR, suggesting that plaque assays alone may be inadequate to assess host range. Furthermore, we demonstrate that the phylogenetic relationships between bacterial hosts can generally be used to predict the susceptibility of bacterial strains to phage infection when the susceptibility of closely related hosts is known, although this approach produced large prediction errors in multiple strains where phylogeny was uninformative. Together, our results demonstrate the ability of bacterial host evolutionary relatedness to explain differences in susceptibility to phage infection, with implications for the development of ISP both as a phage therapy treatment and as an experimental system for the study of virus host shifts.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011433\", \"publication_date\": \"2023-06-08\"}, {\"work_id\": \"W4281738349\", \"doi_x\": \"10.1016/j.patter.2022.100496\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Systematic indication extension for drugs using patient stratification insights generated by combinatorial analytics\", \"abstract\": \"Indication extension or repositioning of drugs can, if done well, provide a faster, cheaper, and derisked route to the approval of new therapies, creating new options to address pockets of unmet medical need for patients and offering the potential for significant commercial and clinical benefits. We look at the promises and challenges of different repositioning strategies and the disease insights and scalability that new high-resolution patient stratification methodologies can bring. This is exemplified by a systematic analysis of all development candidates and on-market drugs, which identified 477 indication extension opportunities across 30 chronic disease areas, each supported by patient stratification biomarkers. This illustrates the potential that new artificial intelligence (AI) and combinatorial analytics methods have to enhance the rate and cost of innovation across the drug discovery industry.\", \"doi_y\": \"https://doi.org/10.1016/j.patter.2022.100496\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4221067052\", \"doi_x\": \"10.1016/s1473-3099(22)00129-3\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Neutralisation sensitivity of the SARS-CoV-2 omicron (B.1.1.529) variant: a cross-sectional study\", \"abstract\": \"The SARS-CoV-2 omicron (B.1.1.529) variant, which was first identified in November, 2021, spread rapidly in many countries, with a spike protein highly diverged from previously known variants, and raised concerns that this variant might evade neutralising antibody responses. We therefore aimed to characterise the sensitivity of the omicron variant to neutralisation.For this cross-sectional study, we cloned the sequence encoding the omicron spike protein from a diagnostic sample to establish an omicron pseudotyped virus neutralisation assay. We quantified the neutralising antibody ID50 (the reciprocal dilution that produces 50% inhibition) against the omicron spike protein, and the fold-change in ID50 relative to the spike of wild-type SARS-CoV-2 (ie, the pandemic founder variant), for one convalescent reference plasma pool (WHO International Standard for anti-SARS-CoV-2 immunoglobulin [20/136]), three reference serum pools from vaccinated individuals, and two cohorts from Stockholm, Sweden: one comprising previously infected hospital workers (17 sampled in November, 2021, after vaccine rollout and nine in June or July, 2020, before vaccination) and one comprising serum from 40 randomly sampled blood donors donated during week 48 (Nov 29-Dec 5) of 2021. Furthermore, we assessed the neutralisation of omicron by five clinically relevant monoclonal antibodies (mAbs).Neutralising antibody responses in reference sample pools sampled shortly after infection or vaccination were substantially less potent against the omicron variant than against wild-type SARS-CoV-2 (seven-fold to 42-fold reduction in ID50 titres). Similarly, for sera obtained before vaccination in 2020 from a cohort of convalescent hospital workers, neutralisation of the omicron variant was low to undetectable (all ID50 titres <20). However, in serum samples obtained in 2021 from two cohorts in Stockholm, substantial cross-neutralisation of the omicron variant was observed. Sera from 17 hospital workers after infection and subsequent vaccination had a reduction in average potency of only five-fold relative to wild-type SARS-CoV-2 (geometric mean ID50 titre 495 vs 105), and two donors had no reduction in potency. A similar pattern was observed in randomly sampled blood donors (n=40), who had an eight-fold reduction in average potency against the omicron variant compared with wild-type SARS-CoV-2 (geometric mean ID50 titre 369 vs 45). We found that the omicron variant was resistant to neutralisation (50% inhibitory concentration [IC50] >10 \\u03bcg/mL) by mAbs casirivimab (REGN-10933), imdevimab (REGN-10987), etesevimab (Ly-CoV016), and bamlanivimab (Ly-CoV555), which form part of antibody combinations used in the clinic to treat COVID-19. However, S309, the parent of sotrovimab, retained most of its activity, with only an approximately two-fold reduction in potency against the omicron variant compared with ancestral D614G SARS-CoV-2 (IC50 0\\u00b71-0\\u00b72 \\u03bcg/mL).These data highlight the extensive, but incomplete, evasion of neutralising antibody responses by the omicron variant, and suggest that boosting with licensed vaccines might be sufficient to raise neutralising antibody titres to protective levels.European Union Horizon 2020 research and innovation programme, European and Developing Countries Clinical Trials Partnership, SciLifeLab, and the Erling-Persson Foundation.\", \"doi_y\": \"https://doi.org/10.1016/s1473-3099(22)00129-3\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4313340328\", \"doi_x\": \"10.1002/pro.4558\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Live\\u2010cell super\\u2010resolution imaging of actin using <scp>LifeAct</scp>\\u201014 with a <scp>PAINT</scp>\\u2010based approach\", \"abstract\": \"We present direct-LIVE-PAINT, an easy-to-implement approach for the nanoscopic imaging of protein structures in live cells using labeled binding peptides. We demonstrate the feasibility of direct-LIVE-PAINT with an actin-binding peptide fused to EGFP, the location of which can be accurately determined as it transiently binds to actin filaments. We show that direct-LIVE-PAINT can be used to image actin structures below the diffraction-limit of light and have used it to observe the dynamic nature of actin in live cells. We envisage a similar approach could be applied to imaging other proteins within live mammalian cells.\", \"doi_y\": \"https://doi.org/10.1002/pro.4558\", \"publication_date\": \"2023-01-26\"}, {\"work_id\": \"W4386692072\", \"doi_x\": \"10.1021/acsomega.3c05670\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Proteinoid Microspheres as Protoneural Networks\", \"abstract\": \"Proteinoids, also known as thermal proteins, possess a fascinating ability to generate microspheres that exhibit electrical spikes resembling the action potentials of neurons. These spiking microspheres, referred to as protoneurons, hold the potential to assemble into proto-nanobrains. In our study, we investigate the feasibility of utilizing a promising electrochemical technique called differential pulse voltammetry (DPV) to interface with proteinoid nanobrains. We evaluate DPV's suitability by examining critical parameters such as selectivity, sensitivity, and linearity of the electrochemical responses. The research systematically explores the influence of various operational factors, including pulse width, pulse amplitude, scan rate, and scan time. Encouragingly, our findings indicate that DPV exhibits significant potential as an efficient electrochemical interface for proteinoid nanobrains. This technology opens up new avenues for developing artificial neural networks with broad applications across diverse fields of research.\", \"doi_y\": \"https://doi.org/10.1021/acsomega.3c05670\", \"publication_date\": \"2023-09-12\"}, {\"work_id\": \"W4381489574\", \"doi_x\": \"10.1038/s41586-023-06179-1\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Structure and function of the RAD51B\\u2013RAD51C\\u2013RAD51D\\u2013XRCC2 tumour suppressor\", \"abstract\": \"Homologous recombination is a fundamental process of life. It is required for the protection and restart of broken replication forks, the repair of chromosome breaks and the exchange of genetic material during meiosis. Individuals with mutations in key recombination genes, such as BRCA2 (also known as FANCD1), or the RAD51 paralogues RAD51B, RAD51C (also known as FANCO), RAD51D, XRCC2 (also known as FANCU) and XRCC3, are predisposed to breast, ovarian and prostate cancers1-10 and the cancer-prone syndrome Fanconi anaemia11-13. The BRCA2 tumour suppressor protein-the product of BRCA2-is well characterized, but the cellular functions of the RAD51 paralogues remain unclear. Genetic knockouts display growth defects, reduced RAD51 focus formation, spontaneous chromosome abnormalities, sensitivity to PARP inhibitors and replication fork defects14,15, but the precise molecular roles of RAD51 paralogues in fork stability, DNA repair and cancer avoidance remain unknown. Here we used cryo-electron microscopy, AlphaFold2 modelling and structural proteomics to determine the structure of the RAD51B-RAD51C-RAD51D-XRCC2 complex (BCDX2), revealing that RAD51C-RAD51D-XRCC2 mimics three RAD51 protomers aligned within a nucleoprotein filament, whereas RAD51B is highly dynamic. Biochemical and single-molecule analyses showed that BCDX2 stimulates the nucleation and extension of RAD51 filaments-which are essential for recombinational DNA repair-in reactions that depend on the coupled ATPase activities of RAD51B and RAD51C. Our studies demonstrate that BCDX2 orchestrates RAD51 assembly on single stranded DNA for replication fork protection and double strand break repair, in reactions that are critical for tumour avoidance.\", \"doi_y\": \"https://doi.org/10.1038/s41586-023-06179-1\", \"publication_date\": \"2023-06-21\"}, {\"work_id\": \"W4200264790\", \"doi_x\": \"10.1101/2021.12.12.472321\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Insights into the role of lipoteichoic acids and MprF function in <i>Bacillus subtilis</i>\", \"abstract\": \"Abstract Gram-positive bacterial cells are protected from the environment by a cell envelope which comprises of layers of peptidoglycan that maintain the cell shape and teichoic acids polymers whose biological function remains unclear. In Bacillus subtilis , loss of all Class A Penicillin-Binding Proteins (aPBPs) which function in peptidoglycan synthesis is conditionally lethal. Here we show that this lethality is associated with an alteration of the lipoteichoic acids (LTA) and the accumulation of the major autolysin LytE in the cell wall. Our analysis provides further evidence that the length and abundance of LTA acts to regulate the cellular level and activity of autolytic enzymes, specifically LytE. Importantly, we identify a novel function for the aminoacyl-phosphatidylglycerol synthase MprF in the modulation of LTA biosynthesis in B. subtilis and Staphylococcus aureus . This finding has implications for our understanding of antimicrobial resistance (particularly daptomycin) in clinically relevant bacteria and the involvement of MprF in the virulence of pathogens, such as methicillin resistant S. aureus .\", \"doi_y\": \"https://doi.org/10.1101/2021.12.12.472321\", \"publication_date\": \"2021-12-13\"}, {\"work_id\": \"W4361279418\", \"doi_x\": \"10.22331/q-2023-03-30-966\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Characterization of variational quantum algorithms using free fermions\", \"abstract\": \"We study variational quantum algorithms from the perspective of free fermions. By deriving the explicit structure of the associated Lie algebras, we show that the Quantum Approximate Optimization Algorithm (QAOA) on a one-dimensional lattice \\u2013 with and without decoupled angles \\u2013 is able to prepare all fermionic Gaussian states respecting the symmetries of the circuit. Leveraging these results, we numerically study the interplay between these symmetries and the locality of the target state, and find that an absence of symmetries makes nonlocal states easier to prepare. An efficient classical simulation of Gaussian states, with system sizes up to<mml:math xmlns:mml=\\\"http://www.w3.org/1998/Math/MathML\\\"><mml:mn>80</mml:mn></mml:math>and deep circuits, is employed to study the behavior of the circuit when it is overparameterized. In this regime of optimization, we find that the number of iterations to converge to the solution scales linearly with system size. Moreover, we observe that the number of iterations to converge to the solution decreases exponentially with the depth of the circuit, until it saturates at a depth which is quadratic in system size. Finally, we conclude that the improvement in the optimization can be explained in terms of better local linear approximations provided by the gradients.\", \"doi_y\": \"https://doi.org/10.22331/q-2023-03-30-966\", \"publication_date\": \"2023-03-30\"}, {\"work_id\": \"W4313545505\", \"doi_x\": \"10.1186/s12915-022-01498-7\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Increased interaction between endoplasmic reticulum and mitochondria following sleep deprivation\", \"abstract\": \"Prolonged cellular activity may overload cell function, leading to high rates of protein synthesis and accumulation of misfolded or unassembled proteins, which cause endoplasmic reticulum (ER) stress and activate the unfolded protein response (UPR) to re-establish normal protein homeostasis. Previous molecular work has demonstrated that sleep deprivation (SD) leads to ER stress in neurons, with a number of ER-specific proteins being upregulated to maintain optimal cellular proteostasis. It is still not clear which cellular processes activated by sleep deprivation lead to ER- stress, but increased cellular metabolism, higher request for protein synthesis, and over production of oxygen radicals have been proposed as potential contributing factors. Here, we investigate the transcriptional and ultrastructural ER and mitochondrial modifications induced by sleep loss.We used gene expression analysis in mouse forebrains to show that SD was associated with significant transcriptional modifications of genes involved in ER stress but also in ER-mitochondria interaction, calcium homeostasis, and mitochondrial respiratory activity. Using electron microscopy, we also showed that SD was associated with a general increase in the density of ER cisternae in pyramidal neurons of the motor cortex. Moreover, ER cisternae established new contact sites with mitochondria, the so-called mitochondria associated membranes (MAMs), important hubs for molecule shuttling, such as calcium and lipids, and for the modulation of ATP production and redox state. Finally, we demonstrated that Drosophila male mutant flies (elav > linker), in which the number of MAMs had been genetically increased, showed a reduction in the amount and consolidation of sleep without alterations in the homeostatic sleep response to SD.We provide evidence that sleep loss induces ER stress characterized by increased crosstalk between ER and mitochondria. MAMs formation associated with SD could represent a key phenomenon for the modulation of multiple cellular processes that ensure appropriate responses to increased cell metabolism. In addition, MAMs establishment may play a role in the regulation of sleep under baseline conditions.\", \"doi_y\": \"https://doi.org/10.1186/s12915-022-01498-7\", \"publication_date\": \"2023-01-04\"}, {\"work_id\": \"W4309858933\", \"doi_x\": \"10.1080/21505594.2022.2150452\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pathogenicity and virulence of <i>Clostridioides difficile</i>\", \"abstract\": \"Clostridioides difficile is the most common cause of nosocomial antibiotic-associated diarrhea, and is responsible for a spectrum of diseases characterized by high levels of recurrence, morbidity, and mortality. Treatment is complex, since antibiotics constitute both the main treatment and the major risk factor for infection. Worryingly, resistance to multiple antibiotics is becoming increasingly widespread, leading to the classification of this pathogen as an urgent threat to global health. As a consummate opportunist, C. difficile is well equipped for promoting disease, owing to its arsenal of virulence factors: transmission of this anaerobe is highly efficient due to the formation of robust endospores, and an array of adhesins promote gut colonization. C. difficile produces multiple toxins acting upon gut epithelia, resulting in manifestations typical of diarrheal disease, and severe inflammation in a subset of patients. This review focuses on such virulence factors, as well as the importance of antimicrobial resistance and genome plasticity in enabling pathogenesis and persistence of this important pathogen.\", \"doi_y\": \"https://doi.org/10.1080/21505594.2022.2150452\", \"publication_date\": \"2023-01-04\"}, {\"work_id\": \"W4308197865\", \"doi_x\": \"10.3390/ijms232113455\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Generation and Characterization of Native and Sialic Acid-Deficient IgE\", \"abstract\": \"Efficient characterization of IgE antibodies and their glycan structures is required for understanding their function in allergy and in the emerging AllergoOncology field for antibody immunotherapy. We report the generation, glyco-profiling and functional analysis of native and sialic acid-deficient glyco-engineered human IgE. The antibodies produced from human embryonic kidney cells were purified via a human IgE class-specific affinity matrix and structural integrity was confirmed by SDS-PAGE and size-exclusion chromatography (SEC). Purified IgEs specific for the tumor-associated antigens Chondroitin Sulfate Proteoglycan 4 (CSPG4-IgE) and Human Epidermal Growth Factor Receptor 2 (HER2-IgE) were devoid of by-products such as free light chains. Using neuraminidase-A, we generated sialic acid-deficient CSPG4-IgE as example glyco-engineered antibody. Comparative glycan analyses of native and glyco-engineered IgEs by Hydrophilic interaction liquid chromatography (HILIC)-high performance liquid chromatography (HPLC) indicated loss of sialic acid terminal residues and differential glycan profiles. Native and glyco-engineered CSPG4-IgEs recognized Fc receptors on the surface of human Fc\\u03b5RI-expressing rat basophilic leukemia RBL-SX38 cells, and of CD23/Fc\\u03b5RII-expressing human RPMI-8866 B-lymphocytes and bound to CSPG4-expressing A2058 human melanoma cells, confirming Fab-mediated recognition. When cross-linked on the cell surface, both IgEs triggered RBL-SX38 degranulation. We demonstrate efficient generation and functional competence of recombinant native and sialic acid-deficient IgEs.\", \"doi_y\": \"https://doi.org/10.3390/ijms232113455\", \"publication_date\": \"2022-11-03\"}, {\"work_id\": \"W4306746309\", \"doi_x\": \"10.1016/j.celrep.2022.111522\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genomic landscape of drug response reveals mediators of anthelmintic resistance\", \"abstract\": \"Like other pathogens, parasitic helminths can rapidly evolve resistance to drug treatment. Understanding the genetic basis of anthelmintic drug resistance in parasitic nematodes is key to tracking its spread and improving the efficacy and sustainability of parasite control. Here, we use an in vivo genetic cross between drug-susceptible and multi-drug-resistant strains of Haemonchus contortus in a natural host-parasite system to simultaneously map resistance loci for the three major classes of anthelmintics. This approach identifies new alleles for resistance to benzimidazoles and levamisole and implicates the transcription factor cky-1 in ivermectin resistance. This gene is within a locus under selection in ivermectin-resistant populations worldwide; expression analyses and functional validation using knockdown experiments support that cky-1 is associated with ivermectin survival. Our work demonstrates the feasibility of high-resolution forward genetics in a parasitic nematode and identifies variants for the development of molecular diagnostics to combat drug resistance in the field.\", \"doi_y\": \"https://doi.org/10.1016/j.celrep.2022.111522\", \"publication_date\": \"2022-10-01\"}, {\"work_id\": \"W4308864026\", \"doi_x\": \"10.7554/elife.74602\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Systematic analysis of membrane contact sites in Saccharomyces cerevisiae uncovers modulators of cellular lipid distribution\", \"abstract\": \"Actively maintained close appositions between organelle membranes, also known as contact sites, enable the efficient transfer of biomolecules between cellular compartments. Several such sites have been described as well as their tethering machineries. Despite these advances we are still far from a comprehensive understanding of the function and regulation of most contact sites. To systematically characterize contact site proteomes, we established a high-throughput screening approach in Saccharomyces cerevisiae based on co-localization imaging. We imaged split fluorescence reporters for six different contact sites, several of which are poorly characterized, on the background of 1165 strains expressing a mCherry-tagged yeast protein that has a cellular punctate distribution (a hallmark of contact sites), under regulation of the strong TEF2 promoter. By scoring both co-localization events and effects on reporter size and abundance, we discovered over 100 new potential contact site residents and effectors in yeast. Focusing on several of the newly identified residents, we identified three homologs of Vps13 and Atg2 that are residents of multiple contact sites. These proteins share their lipid transport domain, thus expanding this family of lipid transporters. Analysis of another candidate, Ypr097w, which we now call Lec1 ( L ipid-droplet E rgosterol C ortex 1), revealed that this previously uncharacterized protein dynamically shifts between lipid droplets and the cell cortex, and plays a role in regulation of ergosterol distribution in the cell. Overall, our analysis expands the universe of contact site residents and effectors and creates a rich database to mine for new functions, tethers, and regulators.\", \"doi_y\": \"https://doi.org/10.7554/elife.74602\", \"publication_date\": \"2022-11-10\"}, {\"work_id\": \"W4311325293\", \"doi_x\": \"10.3389/fmolb.2022.960248\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Crystal polymorphism in fragment-based lead discovery of ligands of the catalytic domain of UGGT, the glycoprotein folding quality control checkpoint\", \"abstract\": \"None of the current data processing pipelines for X-ray crystallography fragment-based lead discovery (FBLD) consults all the information available when deciding on the lattice and symmetry (i.e., the polymorph) of each soaked crystal. Often, X-ray crystallography FBLD pipelines either choose the polymorph based on cell volume and point-group symmetry of the X-ray diffraction data or leave polymorph attribution to manual intervention on the part of the user. Thus, when the FBLD crystals belong to more than one crystal polymorph, the discovery pipeline can be plagued by space group ambiguity, especially if the polymorphs at hand are variations of the same lattice and, therefore, difficult to tell apart from their morphology and/or their apparent crystal lattices and point groups. In the course of a fragment-based lead discovery effort aimed at finding ligands of the catalytic domain of UDP-glucose glycoprotein glucosyltransferase (UGGT), we encountered a mixture of trigonal crystals and pseudotrigonal triclinic crystals-with the two lattices closely related. In order to resolve that polymorphism ambiguity, we have written and described here a series of Unix shell scripts called CoALLA (crystal polymorph and ligand likelihood-based assignment). The CoALLA scripts are written in Unix shell and use autoPROC for data processing, CCP4-Dimple/REFMAC5 and BUSTER for refinement, and RHOFIT for ligand docking. The choice of the polymorph is effected by carrying out (in each of the known polymorphs) the tasks of diffraction data indexing, integration, scaling, and structural refinement. The most likely polymorph is then chosen as the one with the best structure refinement Rfree statistic. The CoALLA scripts further implement a likelihood-based ligand assignment strategy, starting with macromolecular refinement and automated water addition, followed by removal of the water molecules that appear to be fitting ligand density, and a final round of refinement after random perturbation of the refined macromolecular model, in order to obtain unbiased difference density maps for automated ligand placement. We illustrate the use of CoALLA to discriminate between H3 and P1 crystals used for an FBLD effort to find fragments binding to the catalytic domain of Chaetomium thermophilum UGGT.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.960248\", \"publication_date\": \"2022-12-14\"}, {\"work_id\": \"W4315644321\", \"doi_x\": \"10.1016/j.ijpara.2022.12.002\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Evaluation of genome skimming to detect and characterise human and livestock helminths\", \"abstract\": \"The identification of gastrointestinal helminth infections of humans and livestock almost exclusively relies on the detection of eggs or larvae in faeces, followed by manual counting and morphological characterisation to differentiate species using microscopy-based techniques. However, molecular approaches based on the detection and quantification of parasite DNA are becoming more prevalent, increasing the sensitivity, specificity and throughput of diagnostic assays. High-throughput sequencing, from single PCR targets through to the analysis of whole genomes, offers significant promise towards providing information-rich data that may add value beyond traditional and conventional molecular approaches; however, thus far, its utility has not been fully explored to detect helminths in faecal samples. In this study, low-depth whole genome sequencing, i.e. genome skimming, has been applied to detect and characterise helminth diversity in a set of helminth-infected human and livestock faecal material. The strengths and limitations of this approach are evaluated using three methods to characterise and differentiate metagenomic sequencing data based on (i) mapping to whole mitochondrial genomes, (ii) whole genome assemblies, and (iii) a comprehensive internal transcribed spacer 2 (ITS2) database, together with validation using quantitative PCR (qPCR). Our analyses suggest that genome skimming can successfully identify most single and multi-species infections reported by qPCR and can provide sufficient coverage within some samples to resolve consensus mitochondrial genomes, thus facilitating phylogenetic analyses of selected genera, e.g. Ascaris spp. Key to this approach is both the availability and integrity of helminth reference genomes, some of which are currently contaminated with bacterial and host sequences. The success of genome skimming of faecal DNA is dependent on the availability of vouchered sequences of helminths spanning both taxonomic and geographic diversity, together with methods to detect or amplify minute quantities of parasite nucleic acids in mixed samples.\", \"doi_y\": \"https://doi.org/10.1016/j.ijpara.2022.12.002\", \"publication_date\": \"2023-02-01\"}, {\"work_id\": \"W4321447089\", \"doi_x\": \"10.1242/bio.059719\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The factory, the antenna and the scaffold: the three-way interplay between the Golgi, cilium and extracellular matrix underlying tissue function\", \"abstract\": \"ABSTRACT The growth and development of healthy tissues is dependent on the construction of a highly specialised extracellular matrix (ECM) to provide support for cell growth and migration and to determine the biomechanical properties of the tissue. These scaffolds are composed of extensively glycosylated proteins which are secreted and assembled into well-ordered structures that can hydrate, mineralise, and store growth factors as required. The proteolytic processing and glycosylation of ECM components is vital to their function. These modifications are under the control of the Golgi apparatus, an intracellular factory hosting spatially organised, protein-modifying enzymes. Regulation also requires a cellular antenna, the cilium, which integrates extracellular growth signals and mechanical cues to inform ECM production. Consequently, mutations in either Golgi or ciliary genes frequently lead to connective tissue disorders. The individual importance of each of these organelles to ECM function is well-studied. However, emerging evidence points towards a more tightly linked system of interdependence between the Golgi, cilium and ECM. This review examines how the interplay between all three compartments underpins healthy tissue. As an example, it will look at several members of the golgin family of Golgi-resident proteins whose loss is detrimental to connective tissue function. This perspective will be important for many future studies looking to dissect the cause and effect of mutations impacting tissue integrity.\", \"doi_y\": \"https://doi.org/10.1242/bio.059719\", \"publication_date\": \"2023-02-15\"}, {\"work_id\": \"W4365135071\", \"doi_x\": \"10.7554/elife.66765\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Enhanced functional detection of synaptic calcium-permeable AMPA receptors using intracellular NASPM\", \"abstract\": \"Calcium-permeable AMPA-type glutamate receptors (CP-AMPARs) contribute to many forms of synaptic plasticity and pathology. They can be distinguished from GluA2-containing calcium-impermeable AMPARs by the inward rectification of their currents, which reflects voltage-dependent channel block by intracellular spermine. However, the efficacy of this weakly permeant blocker is differentially altered by the presence of AMPAR auxiliary subunits - including transmembrane AMPAR regulatory proteins, cornichons, and GSG1L - which are widely expressed in neurons and glia. This complicates the interpretation of rectification as a measure of CP-AMPAR expression. Here, we show that the inclusion of the spider toxin analog 1-naphthylacetyl spermine (NASPM) in the intracellular solution results in a complete block of GluA1-mediated outward currents irrespective of the type of associated auxiliary subunit. In neurons from GluA2-knockout mice expressing only CP-AMPARs, intracellular NASPM, unlike spermine, completely blocks outward synaptic currents. Thus, our results identify a functional measure of CP-AMPARs, that is unaffected by their auxiliary subunit content.\", \"doi_y\": \"https://doi.org/10.7554/elife.66765\", \"publication_date\": \"2023-04-12\"}, {\"work_id\": \"W4223432929\", \"doi_x\": \"10.1093/nar/gkac250\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"3DLigandSite: structure-based prediction of protein\\u2013ligand binding sites\", \"abstract\": \"3DLigandSite is a web tool for the prediction of ligand-binding sites in proteins. Here, we report a significant update since the first release of 3DLigandSite in 2010. The overall methodology remains the same, with candidate binding sites in proteins inferred using known binding sites in related protein structures as templates. However, the initial structural modelling step now uses the newly available structures from the AlphaFold database or alternatively Phyre2 when AlphaFold structures are not available. Further, a sequence-based search using HHSearch has been introduced to identify template structures with bound ligands that are used to infer the ligand-binding residues in the query protein. Finally, we introduced a machine learning element as the final prediction step, which improves the accuracy of predictions and provides a confidence score for each residue predicted to be part of a binding site. Validation of 3DLigandSite on a set of 6416 binding sites obtained 92% recall at 75% precision for non-metal binding sites and 52% recall at 75% precision for metal binding sites. 3DLigandSite is available at https://www.wass-michaelislab.org/3dligandsite. Users submit either a protein sequence or structure. Results are displayed in multiple formats including an interactive Mol* molecular visualization of the protein and the predicted binding sites.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkac250\", \"publication_date\": \"2022-04-12\"}, {\"work_id\": \"W4210588499\", \"doi_x\": \"10.3389/fmolb.2022.820792\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The Energy Landscape Perspective: Encoding Structure and Function for Biomolecules\", \"abstract\": \"The energy landscape perspective is outlined with particular reference to biomolecules that perform multiple functions. We associate these multifunctional molecules with multifunnel energy landscapes, illustrated by some selected examples, where understanding the organisation of the landscape has provided new insight into function. Conformational selection and induced fit may provide alternative routes to realisation of multifunctionality, exploiting the possibility of environmental control and distinct binding modes.\", \"doi_y\": \"https://doi.org/10.3389/fmolb.2022.820792\", \"publication_date\": \"2022-01-27\"}, {\"work_id\": \"W4309865319\", \"doi_x\": \"10.1042/bst20220191\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Mechanisms of <i>E. coli</i> chemotaxis signaling pathways visualized using cryoET and computational approaches\", \"abstract\": \"Chemotaxis signaling pathways enable bacteria to sense and respond to their chemical environment and, in some species, are critical for lifestyle processes such as biofilm formation and pathogenesis. The signal transduction underlying chemotaxis behavior is mediated by large, highly ordered protein complexes known as chemosensory arrays. For nearly two decades, cryo-electron tomography (cryoET) has been used to image chemosensory arrays, providing an increasingly detailed understanding of their structure and function. In this mini-review, we provide an overview of the use of cryoET to study chemosensory arrays, including imaging strategies, key results, and outstanding questions. We further discuss the application of molecular modeling and simulation techniques to complement structure determination efforts and provide insight into signaling mechanisms. We close the review with a brief outlook, highlighting promising future directions for the field.\", \"doi_y\": \"https://doi.org/10.1042/bst20220191\", \"publication_date\": \"2022-11-24\"}, {\"work_id\": \"W4313421799\", \"doi_x\": \"10.1101/2022.12.16.22283452\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Genome-wide association study reveals loci with sex-specific effects on plasma bile acids\", \"abstract\": \"Abstract Bile acids are essential for food digestion and nutrient absorption, but also act as signalling molecules involved in hepatobiliary diseases, gastrointestinal disorders and carcinogenesis. While many studies have focused on the genetic determinants of blood metabolites, research focusing specifically on genetic regulation of bile acids in the general population is currently lacking. Here we investigate the genetic architecture of primary and secondary bile acids in blood plasma, reporting associations with both common and rare variants. By performing genome-wide association analysis (GWAS) of plasma blood levels of 18 bile acids (N = 4923) we identify two significantly associated loci, a common variant mapping to SLCO1B1 (encoding a liver bilirubin and drug transporter) and a rare variant in PRKG1 (encoding soluble cyclic GMP-dependent protein kinase). For these loci, in the sex-stratified GWAS (N\\u2642 = 820, N\\u2640 = 1088), we observe sex-specific effects ( SLCO1B1 \\u03b2 \\u2642 = -0.51, P = 2.30\\u00d710 \\u221213 , \\u03b2\\u2640 = -0.3, P = 9.90\\u00d710 \\u221207 ; PRKG1 \\u03b2 \\u2642 = -0.18, P = 1.80\\u00d710 \\u221201 , \\u03b2 \\u2640 = -0.79, P = 8.30\\u00d710 \\u221211 ), corroborating the contribution of sex to bile acid variability. Using gene-based aggregate tests and whole exome sequencing, we identify rare pLoF and missense variants potentially associated with bile acid levels in 3 genes ( OR1G1, SART1 and SORCS2 ), some of which have been linked with liver diseases.\", \"doi_y\": \"https://doi.org/10.1101/2022.12.16.22283452\", \"publication_date\": \"2022-12-19\"}, {\"work_id\": \"W3194187547\", \"doi_x\": \"10.1002/prot.26223\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Computational models in the service of X\\u2010ray and <scp>cryo\\u2010</scp>electron microscopy structure determination\", \"abstract\": \"Critical assessment of structure prediction (CASP) conducts community experiments to determine the state of the art in computing protein structure from amino acid sequence. The process relies on the experimental community providing information about not yet public or about to be solved structures, for use as targets. For some targets, the experimental structure is not solved in time for use in CASP. Calculated structure accuracy improved dramatically in this round, implying that models should now be much more useful for resolving many sorts of experimental difficulties. To test this, selected models for seven unsolved targets were provided to the experimental groups. These models were from the AlphaFold2 group, who overall submitted the most accurate predictions in CASP14. Four targets were solved with the aid of the models, and, additionally, the structure of an already solved target was improved. An a posteriori analysis showed that, in some cases, models from other groups would also be effective. This paper provides accounts of the successful application of models to structure determination, including molecular replacement for X-ray crystallography, backbone tracing and sequence positioning in a cryo-electron microscopy structure, and correction of local features. The results suggest that, in future, there will be greatly increased synergy between computational and experimental approaches to structure determination.\", \"doi_y\": \"https://doi.org/10.1002/prot.26223\", \"publication_date\": \"2021-09-06\"}, {\"work_id\": \"W4353015072\", \"doi_x\": \"10.1083/jcb.202204042\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The meiotic LINC complex component KASH5 is an activating adaptor for cytoplasmic dynein\", \"abstract\": \"Cytoplasmic dynein-driven movement of chromosomes during prophase I of mammalian meiosis is essential for synapsis and genetic exchange. Dynein connects to chromosome telomeres via KASH5 and SUN1 or SUN2, which together span the nuclear envelope. Here, we show that KASH5 promotes dynein motility in vitro, and cytosolic KASH5 inhibits dynein\\u2019s interphase functions. KASH5 interacts with a dynein light intermediate chain (DYNC1LI1 or DYNC1LI2) via a conserved helix in the LIC C-terminal, and this region is also needed for dynein\\u2019s recruitment to other cellular membranes. KASH5\\u2019s N-terminal EF-hands are essential as the interaction with dynein is disrupted by mutation of key calcium-binding residues, although it is not regulated by cellular calcium levels. Dynein can be recruited to KASH5 at the nuclear envelope independently of dynactin, while LIS1 is essential for dynactin incorporation into the KASH5\\u2013dynein complex. Altogether, we show that the transmembrane protein KASH5 is an activating adaptor for dynein and shed light on the hierarchy of assembly of KASH5\\u2013dynein\\u2013dynactin complexes.\", \"doi_y\": \"https://doi.org/10.1083/jcb.202204042\", \"publication_date\": \"2023-03-22\"}, {\"work_id\": \"W4284881539\", \"doi_x\": \"10.1038/s41467-022-31686-6\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Loss-of-function, gain-of-function and dominant-negative mutations have profoundly different effects on protein structure\", \"abstract\": \"Most known pathogenic mutations occur in protein-coding regions of DNA and change the way proteins are made. Taking protein structure into account has therefore provided great insight into the molecular mechanisms underlying human genetic disease. While there has been much focus on how mutations can disrupt protein structure and thus cause a loss of function (LOF), alternative mechanisms, specifically dominant-negative (DN) and gain-of-function (GOF) effects, are less understood. Here, we investigate the protein-level effects of pathogenic missense mutations associated with different molecular mechanisms. We observe striking differences between recessive vs dominant, and LOF vs non-LOF mutations, with dominant, non-LOF disease mutations having much milder effects on protein structure, and DN mutations being highly enriched at protein interfaces. We also find that nearly all computational variant effect predictors, even those based solely on sequence conservation, underperform on non-LOF mutations. However, we do show that non-LOF mutations could potentially be identified by their tendency to cluster in three-dimensional space. Overall, our work suggests that many pathogenic mutations that act via DN and GOF mechanisms are likely being missed by current variant prioritisation strategies, but that there is considerable scope to improve computational predictions through consideration of molecular disease mechanisms.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-31686-6\", \"publication_date\": \"2022-07-06\"}, {\"work_id\": \"W4200461734\", \"doi_x\": \"10.1101/2021.12.06.471394\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Insertions in the SARS-CoV-2 Spike N-Terminal Domain May Aid COVID-19 Transmission\", \"abstract\": \"Abstract Coronavirus disease 2019 (COVID-19) caused by SARS-CoV-2 is an ongoing pandemic that causes significant health/socioeconomic burden. Variants of concern (VOCs) have emerged which may affect transmissibility, disease severity and re-infection risk. Most studies focus on the receptor-binding domain (RBD) of the Spike protein. However, some studies suggest that the Spike N-terminal domain (NTD) may have a role in facilitating virus entry via sialic-acid receptor binding. Furthermore, most VOCs include novel NTD variants. Recent analyses demonstrated that NTD insertions in VOCs tend to lie close to loop regions likely to be involved in binding sialic acids. We extended the structural characterisation of these putative sugar binding pockets and explored whether variants could enhance the binding to sialic acids and therefore to the host membrane, thereby contributing to increased transmissibility. We found that recent NTD insertions in VOCs (i.e., Gamma, Delta and Omicron variants) and emerging variants of interest (VOIs) (i.e., Iota, Lambda, Theta variants) frequently lie close to known and putative sugar-binding pockets. For some variants, including the recent Omicron VOC, we find increases in predicted sialic acid binding energy, compared to the original SARS-CoV-2, which may contribute to increased transmission. We examined the similarity of NTD across a range of related Betacoronaviruses to determine whether the putative sugar-binding pockets are sufficiently similar to be exploited in drug design. Despite global sequence and structure similarity, most sialic-acid binding pockets of NTD vary across related coronaviruses. Typically, SARS-CoV-2 possesses additional loops in these pockets that increase contact with polysaccharides. Our work suggests ongoing evolutionary tuning of the sugar-binding pockets in the virus. Whilst three of the pockets are too structurally variable to be amenable to pan Betacoronavirus drug design, we detected a fourth pocket that is highly structurally conserved and could therefore be investigated in pursuit of a generic drug. Our structure-based analyses help rationalise the effects of VOCs and provide hypotheses for experiments. For example, the Omicron variant, which has increased binding to sialic acids in pocket 3, has a rather unique insertion near pocket 3. Our work suggests a strong need for experimental monitoring of VOC changes in NTD.\", \"doi_y\": \"https://doi.org/10.1101/2021.12.06.471394\", \"publication_date\": \"2021-12-07\"}, {\"work_id\": \"W4318042259\", \"doi_x\": \"10.3390/biology12020181\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Macromolecular Crowding Is Surprisingly Unable to Deform the Structure of a Model Biomolecular Condensate\", \"abstract\": \"The crowded interior of a living cell makes performing experiments on simpler in vitro systems attractive. Although these reveal interesting phenomena, their biological relevance can be questionable. A topical example is the phase separation of intrinsically disordered proteins into biomolecular condensates, which is proposed to underlie the membrane-less compartmentalization of many cellular functions. How a cell reliably controls biochemical reactions in compartments open to the compositionally-varying cytoplasm is an important question for understanding cellular homeostasis. Computer simulations are often used to study the phase behavior of model biomolecular condensates, but the number of relevant parameters increases as the number of protein components increases. It is unfeasible to exhaustively simulate such models for all parameter combinations, although interesting phenomena are almost certainly hidden in their high-dimensional parameter space. Here, we have studied the phase behavior of a model biomolecular condensate in the presence of a polymeric crowding agent. We used a novel compute framework to execute dozens of simultaneous simulations spanning the protein/crowder concentration space. We then combined the results into a graphical representation for human interpretation, which provided an efficient way to search the model\\u2019s high-dimensional parameter space. We found that steric repulsion from the crowder drives a near-critical system across the phase boundary, but the molecular arrangement within the resulting biomolecular condensate is rather insensitive to the crowder concentration and molecular weight. We propose that a cell may use the local cytoplasmic concentration to assist the formation of biomolecular condensates, while relying on the dense phase to reliably provide a stable, structured, fluid milieu for cellular biochemistry despite being open to its changing environment.\", \"doi_y\": \"https://doi.org/10.3390/biology12020181\", \"publication_date\": \"2023-01-25\"}, {\"work_id\": \"W4379157466\", \"doi_x\": \"10.1016/j.tplants.2023.05.008\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"How to convert host plants into nonhosts\", \"abstract\": \"Recent research demonstrates that undermining interactions between pathogen effectors and their host target proteins can reduce infection. As more effector\\u2013target pairs are identified, their structures and interaction surfaces exposed, and there is the possibility of making multiple edits to diverse plant genomes, the desire to convert crops to nonhosts could become reality.\", \"doi_y\": \"https://doi.org/10.1016/j.tplants.2023.05.008\", \"publication_date\": \"2023-08-01\"}, {\"work_id\": \"W4385254430\", \"doi_x\": \"10.1016/j.jbc.2023.105086\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Efficient NADPH-dependent dehalogenation afforded by a self-sufficient reductive dehalogenase\", \"abstract\": \"Reductive dehalogenases are corrinoid and iron-sulfur cluster-containing enzymes that catalyze the reductive removal of a halogen atom. The oxygen-sensitive and membrane-associated nature of the respiratory reductive dehalogenases has hindered their detailed kinetic study. In contrast, the evolutionarily related catabolic reductive dehalogenases are oxygen tolerant, with those that are naturally fused to a reductase domain with similarity to phthalate dioxygenase presenting attractive targets for further study. We present efficient heterologous expression of a self-sufficient catabolic reductive dehalogenase from Jhaorihella thermophila in Escherichia coli. Combining the use of maltose-binding protein as a solubility-enhancing tag with the btuCEDFB cobalamin uptake system affords up to 40% cobalamin occupancy and a full complement of iron-sulfur clusters. The enzyme is able to efficiently perform NADPH-dependent dehalogenation of brominated and iodinated phenolic compounds, including the flame retardant tetrabromobisphenol, under both anaerobic and aerobic conditions. NADPH consumption is tightly coupled to product formation. Surprisingly, corresponding chlorinated compounds only act as competitive inhibitors. Electron paramagnetic resonance spectroscopy reveals loss of the Co(II) signal observed in the resting state of the enzyme under steady-state conditions, suggesting accumulation of Co(I)/(III) species prior to the rate-limiting step. In vivo reductive debromination activity is readily observed, and when the enzyme is expressed in E. coli strain W, supports growth on 3-bromo-4-hydroxyphenylacetic as a sole carbon source. This demonstrates the potential for catabolic reductive dehalogenases for future application in bioremediation.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2023.105086\", \"publication_date\": \"2023-09-01\"}, {\"work_id\": \"W4297458009\", \"doi_x\": \"10.7554/elife.83135\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"The SPARC complex defines RNAPII promoters in Trypanosoma brucei\", \"abstract\": \"Kinetoplastids are a highly divergent lineage of eukaryotes with unusual mechanisms for regulating gene expression. We previously surveyed 65 putative chromatin factors in the kinetoplastid Trypanosoma brucei. Our analyses revealed that the predicted histone methyltransferase SET27 and the Chromodomain protein CRD1 are tightly concentrated at RNAPII transcription start regions (TSRs). Here, we report that SET27 and CRD1, together with four previously uncharacterized constituents, form the SET27 promoter-associated regulatory complex (SPARC), which is specifically enriched at TSRs. SET27 loss leads to aberrant RNAPII recruitment to promoter sites, accumulation of polyadenylated transcripts upstream of normal transcription start sites, and conversion of some normally unidirectional promoters to bidirectional promoters. Transcriptome analysis in the absence of SET27 revealed upregulated mRNA expression in the vicinity of SPARC peaks within the main body of chromosomes in addition to derepression of genes encoding variant surface glycoproteins (VSGs) located in subtelomeric regions. These analyses uncover a novel chromatin-associated complex required to establish accurate promoter position and directionality.\", \"doi_y\": \"https://doi.org/10.7554/elife.83135\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W4280621285\", \"doi_x\": \"10.1016/j.isci.2022.104389\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Integrating CRISPR/Cas systems with programmable DNA nanostructures for delivery and beyond\", \"abstract\": \"Precise genome editing with CRISPR/Cas paves the way for many biochemical, biotechnological, and medical applications, and consequently, it may enable treatment of already known and still-to-be-found genetic diseases. Meanwhile, another rapidly emerging field\\u2014structural DNA nanotechnology\\u2014provides a customizable and modular platform for accurate positioning of nanoscopic materials, for e.g., biomedical uses. This addressability has just recently been applied in conjunction with the newly developed gene engineering tools to enable impactful, programmable nanotechnological applications. As of yet, self-assembled DNA nanostructures have been mainly employed to enhance and direct the delivery of CRISPR/Cas, but lately the groundwork has also been laid out for other intriguing and complex functions. These recent advances will be described in this perspective.\", \"doi_y\": \"https://doi.org/10.1016/j.isci.2022.104389\", \"publication_date\": \"2022-06-01\"}, {\"work_id\": \"W4310931455\", \"doi_x\": \"10.1016/j.semcdb.2022.11.015\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"TALE transcription factors: Cofactors no more\", \"abstract\": \"Exd/PBX, Hth/MEIS and PREP proteins belong to the TALE (three-amino-acid loop extension) superclass of transcription factors (TFs) with an atypical homedomain (HD). Originally discovered as \\\"cofactors\\\" to HOX proteins, revisiting their traditional role in light of genome-wide experiments reveals a strong and reproducible pattern of HOX and TALE co-occupancy across diverse embryonic tissues. While confirming that TALE increases HOX specificity and selectivity in vivo, this wider outlook also reveals novel aspects of HOX:TALE collaboration, namely that HOX TFs generally require pre-bound TALE factors to access their functional binding sites in vivo. In contrast to the restricted expression domains of HOX TFs, TALE factors are largely ubiquitous, and PBX and PREP are expressed at the earliest developmental stages. PBX and MEIS control development of many organs and tissues and their dysregulation is associated with congenital disease and cancer. Accordingly, many instances of TALE cooperation with non HOX TFs have been documented in various systems. The model that emerges from these studies is that TALE TFs create a permissive chromatin platform that is selected by tissue-restricted TFs for binding. In turn, HOX and other tissue-restricted TFs selectively convert a ubiquitous pool of low affinity TALE binding events into high confidence, tissue-restricted binding events associated with transcriptional activation. As a result, TALE:TF complexes are associated with active chromatin and domain/lineage-specific gene activity. TALE ubiquitous expression and broad genomic occupancy, as well as the increasing examples of TALE tissue-specific partners, reveal a universal and obligatory role for TALE in the control of tissue and lineage-specific transcriptional programs, beyond their initial discovery as HOX co-factors.\", \"doi_y\": \"https://doi.org/10.1016/j.semcdb.2022.11.015\", \"publication_date\": \"2024-01-01\"}, {\"work_id\": \"W4224945247\", \"doi_x\": \"10.3389/fphar.2022.861311\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"NMDA Receptor Antagonists Increase the Release of GLP-1 From Gut Endocrine Cells\", \"abstract\": \"Type 2 diabetes mellitus (T2DM) remains one of the most pressing health issues facing modern society. Several antidiabetic drugs are currently in clinical use to treat hyperglycaemia, but there is a need for new treatments that effectively restore pancreatic islet function in patients. Recent studies reported that both murine and human pancreatic islets exhibit enhanced insulin release and \\u03b2-cell viability in response to N-methyl-D-aspartate (NMDA) receptor antagonists. Furthermore, oral administration of dextromethorphan, an over-the-counter NMDA receptor antagonist, to diabetic patients in a small clinical trial showed improved glucose tolerance and increased insulin release. However, the effects of NMDA receptor antagonists on the secretion of the incretin hormone GLP-1 was not tested, and nothing is known regarding how NMDA receptor antagonists may alter the secretion of gut hormones. This study demonstrates for the first time that, similar to \\u03b2-cells, the NMDA receptor antagonist MK-801 increases the release of GLP-1 from a murine L-cell enteroendocrine model cell line, GLUTag cells. Furthermore, we report the 3\\u2032 mRNA expression profiling of GLUTag cells, with a specific focus on glutamate-activated receptors. We conclude that if NMDA receptor antagonists are to be pursued as an alternative, orally administered treatment for T2DM, it is essential that the effects of these drugs on the release of gut hormones, and specifically the incretin hormones, are fully investigated.\", \"doi_y\": \"https://doi.org/10.3389/fphar.2022.861311\", \"publication_date\": \"2022-04-27\"}, {\"work_id\": \"W4367551418\", \"doi_x\": \"10.1098/rstb.2022.0160\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Cardiomyocyte electrophysiology and its modulation: current views and future prospects\", \"abstract\": \"Normal and abnormal cardiac rhythms are of key physiological and clinical interest. This introductory article begins from Sylvio Weidmann's key historic 1950s microelectrode measurements of cardiac electrophysiological activity and Singh & Vaughan Williams's classification of cardiotropic targets. It then proceeds to introduce the insights into cardiomyocyte function and its regulation that subsequently emerged and their therapeutic implications. We recapitulate the resulting view that surface membrane electrophysiological events underlying cardiac excitation and its initiation, conduction and recovery constitute the final common path for the cellular mechanisms that impinge upon this normal or abnormal cardiac electrophysiological activity. We then consider progress in the more recently characterized successive regulatory hierarchies involving Ca2+ homeostasis, excitation-contraction coupling and autonomic G-protein signalling and their often reciprocal interactions with the surface membrane events, and their circadian rhythms. Then follow accounts of longer-term upstream modulation processes involving altered channel expression, cardiomyocyte energetics and hypertrophic and fibrotic cardiac remodelling. Consideration of these developments introduces each of the articles in this Phil. Trans. B theme issue. The findings contained in these articles translate naturally into recent classifications of cardiac electrophysiological targets and drug actions, thereby encouraging future iterations of experimental cardiac electrophysiological discovery, and testing directed towards clinical management. This article is part of the theme issue 'The heartbeat: its molecular basis and physiological mechanisms'.\", \"doi_y\": \"https://doi.org/10.1098/rstb.2022.0160\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4307357511\", \"doi_x\": \"10.1101/2022.10.21.513200\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A systems approach evaluating the impact of SARS-CoV-2 variant of concern mutations on CD8+ T cell responses\", \"abstract\": \"Abstract T cell recognition of SARS-CoV-2 antigens after vaccination and/or natural infection has played a central role in resolving SARS-CoV-2 infections and generating adaptive immune memory. However, the clinical impact of SARS-CoV-2-specific T cell responses is variable and the mechanisms underlying T cell interaction with target antigens are not fully understood. This is especially true given the virus\\u2019 rapid evolution, which leads to new variants with immune escape capacity. In this study, we used the Omicron variant as a model organism and took a systems approach to evaluate the impact of mutations on CD8+ T cell immunogenicity. We computed an \\u2018immunogenicity potential\\u2019 score for each SARS-CoV-2 peptide antigen from the ancestral strain and Omicron, capturing both antigen presentation and T cell recognition probabilities. By comparing ancestral vs. Omicron immunogenicity scores, we reveal a divergent and heterogeneous landscape of impact for CD8+ T cell recognition of mutated targets in Omicron variants. While T cell recognition of Omicron peptides is broadly preserved, we observed mutated peptides with deteriorated immunogenicity that may assist breakthrough infection in some individuals. We then combined our scoring scheme with an in-silico mutagenesis, to characterise the position- and residue-specific theoretical mutational impact on immunogenicity. While we predict many escape trajectories from the theoretical landscape of substitutions, our study suggests that Omicron mutations in T cell epitopes did not develop under cell-mediated pressure. Our study provides a generalisable platform for fostering a deeper understanding of existing and novel variant impact on antigen-specific vaccine- and/or infection-induced T cell immunity.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.21.513200\", \"publication_date\": \"2022-10-21\"}, {\"work_id\": \"W4309466761\", \"doi_x\": \"10.1242/jcs.259769\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"The mechanical cell \\u2013 the role of force dependencies in synchronising protein interaction networks\", \"abstract\": \"ABSTRACT The role of mechanical signals in the proper functioning of organisms is increasingly recognised, and every cell senses physical forces and responds to them. These forces are generated both from outside the cell or via the sophisticated force-generation machinery of the cell, the cytoskeleton. All regions of the cell are connected via mechanical linkages, enabling the whole cell to function as a mechanical system. In this Review, we define some of the key concepts of how this machinery functions, highlighting the critical requirement for mechanosensory proteins, and conceptualise the coupling of mechanical linkages to mechanochemical switches that enables forces to be converted into biological signals. These mechanical couplings provide a mechanism for how mechanical crosstalk might coordinate the entire cell, its neighbours, extending into whole collections of cells, in tissues and in organs, and ultimately in the coordination and operation of entire organisms. Consequently, many diseases manifest through defects in this machinery, which we map onto schematics of the mechanical linkages within a cell. This mapping approach paves the way for the identification of additional linkages between mechanosignalling pathways and so might identify treatments for diseases, where mechanical connections are affected by mutations or where individual force-regulated components are defective.\", \"doi_y\": \"https://doi.org/10.1242/jcs.259769\", \"publication_date\": \"2022-11-15\"}, {\"work_id\": \"W4292241897\", \"doi_x\": \"10.1016/j.chom.2022.07.014\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Structure of trimeric pre-fusion rabies virus glycoprotein in complex with two protective antibodies\", \"abstract\": \"Rabies virus (RABV) causes lethal encephalitis and is responsible for approximately 60,000 deaths per year. As the sole virion-surface protein, the rabies virus glycoprotein (RABV-G) mediates host-cell entry. RABV-G's pre-fusion trimeric conformation displays epitopes bound by protective neutralizing antibodies that can be induced by vaccination or passively administered for post-exposure prophylaxis. We report a 2.8-\\u00c5 structure of a RABV-G trimer in the pre-fusion conformation, in complex with two neutralizing and protective monoclonal antibodies, 17C7 and 1112-1, that recognize distinct epitopes. One of these antibodies is a licensed prophylactic (17C7, Rabishield), which we show locks the protein in pre-fusion conformation. Targeted mutations can similarly stabilize RABV-G in the pre-fusion conformation, a key step toward structure-guided vaccine design. These data reveal the higher-order architecture of a key therapeutic target and the structural basis of neutralization by antibodies binding two key antigenic sites, and this will facilitate the development of improved vaccines and prophylactic antibodies.\", \"doi_y\": \"https://doi.org/10.1016/j.chom.2022.07.014\", \"publication_date\": \"2022-09-01\"}, {\"work_id\": \"W4306857359\", \"doi_x\": \"10.1016/j.jbc.2022.102624\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"A bovine antibody possessing an ultralong complementarity-determining region CDRH3 targets a highly conserved epitope in sarbecovirus spike proteins\", \"abstract\": \"Broadly neutralizing antibodies have huge potential as novel antiviral therapeutics due to their ability to recognize highly conserved epitopes that are seldom mutated in viral variants. A subset of bovine antibodies possess an ultralong complementarity-determining region (CDR)H3 that is highly adept at recognizing such conserved epitopes, but their reactivity against Sarbecovirus Spike proteins has not been explored previously. Here, we use a SARS-na\\u00efve library to isolate a broadly reactive bovine CDRH3 that binds the receptor-binding domain of SARS-CoV, SARS-CoV-2, and all SARS-CoV-2 variants. We show further that it neutralizes viruses pseudo-typed with SARS-CoV Spike, but this is not by competition with angiotensin-converting enzyme 2 (ACE2) binding. Instead, using differential hydrogen-deuterium exchange mass spectrometry, we demonstrate that it recognizes the major site of vulnerability of Sarbecoviruses. This glycan-shielded cryptic epitope becomes available only transiently via interdomain movements of the Spike protein such that antibody binding triggers destruction of the prefusion complex. This proof of principle study demonstrates the power of in vitro expressed bovine antibodies with ultralong CDRH3s for the isolation of novel, broadly reactive tools to combat emerging pathogens and to identify key epitopes for vaccine development.\", \"doi_y\": \"https://doi.org/10.1016/j.jbc.2022.102624\", \"publication_date\": \"2022-12-01\"}, {\"work_id\": \"W4220937155\", \"doi_x\": \"10.15252/embj.2021109700\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"HOIL\\u20101 ubiquitin ligase activity targets unbranched glucosaccharides and is required to prevent polyglucosan accumulation\", \"abstract\": \"HOIL-1, a component of the linear ubiquitin chain assembly complex (LUBAC), ubiquitylates serine and threonine residues in proteins by esterification. Here, we report that mice expressing an E3 ligase-inactive HOIL-1[C458S] mutant accumulate polyglucosan in brain, heart and other organs, indicating that HOIL-1's E3 ligase activity is essential to prevent these toxic polysaccharide deposits from accumulating. We found that HOIL-1 monoubiquitylates glycogen and \\u03b11:4-linked maltoheptaose in vitro and identify the C6 hydroxyl moiety of glucose as the site of ester-linked ubiquitylation. The monoubiquitylation of maltoheptaose was accelerated > 100-fold by the interaction of Met1-linked or Lys63-linked ubiquitin oligomers with the RBR domain of HOIL-1. HOIL-1 also transferred pre-formed ubiquitin oligomers to maltoheptaose en bloc, producing polyubiquitylated maltoheptaose in one catalytic step. The Sharpin and HOIP components of LUBAC, but not HOIL-1, bound to unbranched and infrequently branched glucose polymers in vitro, but not to highly branched mammalian glycogen, suggesting a potential function in targeting HOIL-1 to unbranched glucosaccharides in cells. We suggest that monoubiquitylation of unbranched glucosaccharides may initiate their removal from cells, preventing precipitation as polyglucosan.\", \"doi_y\": \"https://doi.org/10.15252/embj.2021109700\", \"publication_date\": \"2022-03-11\"}, {\"work_id\": \"W4367297818\", \"doi_x\": \"10.3390/biology12050664\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Organelle Membrane Extensions in Mammalian Cells\", \"abstract\": \"Organelles within eukaryotic cells are not isolated static compartments, instead being morphologically diverse and highly dynamic in order to respond to cellular needs and carry out their diverse and cooperative functions. One phenomenon exemplifying this plasticity, and increasingly gaining attention, is the extension and retraction of thin tubules from organelle membranes. While these protrusions have been observed in morphological studies for decades, their formation, properties and functions are only beginning to be understood. In this review, we provide an overview of what is known and still to be discovered about organelle membrane protrusions in mammalian cells, focusing on the best-characterised examples of these membrane extensions arising from peroxisomes (ubiquitous organelles involved in lipid metabolism and reactive oxygen species homeostasis) and mitochondria. We summarise the current knowledge on the diversity of peroxisomal/mitochondrial membrane extensions, as well as the molecular mechanisms by which they extend and retract, necessitating dynamic membrane remodelling, pulling forces and lipid flow. We also propose broad cellular functions for these membrane extensions in inter-organelle communication, organelle biogenesis, metabolism and protection, and finally present a mathematical model that suggests that extending protrusions is the most efficient way for an organelle to explore its surroundings.\", \"doi_y\": \"https://doi.org/10.3390/biology12050664\", \"publication_date\": \"2023-04-27\"}, {\"work_id\": \"W4378746466\", \"doi_x\": \"10.1021/acs.jmedchem.3c00555\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"PINK1-Dependent Mitophagy Inhibits Elevated Ubiquitin Phosphorylation Caused by Mitochondrial Damage\", \"abstract\": \"Ubiquitin phosphorylation by the mitochondrial protein kinase PTEN-induced kinase 1 (PINK1), upon mitochondrial depolarization, is an important intermediate step in the recycling of damaged mitochondria via mitophagy. As mutations in PINK1 can cause early-onset Parkinson's disease (PD), there has been a growing interest in small-molecule activators of PINK1-mediated mitophagy as potential PD treatments. Herein, we show that N6-substituted adenosines, such as N6-(2-furanylmethyl)adenosine (known as kinetin riboside) and N6-benzyladenosine, activate PINK1 in HeLa cells and induce PINK1-dependent mitophagy in primary mouse fibroblasts. Interestingly, pre-treatment of HeLa cells and astrocytes with these compounds inhibited elevated ubiquitin phosphorylation that is induced by established mitochondrial depolarizing agents, carbonyl cyanide m-chlorophenyl-hydrazine and niclosamide. Together, this highlights N6-substituted adenosines as progenitor PINK1 activators that could potentially be developed, in the future, as treatments for aged and sporadic PD patients who have elevated phosphorylated ubiquitin levels in the brain.\", \"doi_y\": \"https://doi.org/10.1021/acs.jmedchem.3c00555\", \"publication_date\": \"2023-05-30\"}, {\"work_id\": \"W4385953765\", \"doi_x\": \"10.1002/cctc.202300930\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"Expanding the Substrate Scope of <i>N</i>\\u2010 and <i>O</i>\\u2010Methyltransferases from Plants for Chemoselective Alkylation**\", \"abstract\": \"Abstract Methylation reactions are of significant interest when generating pharmaceutically active molecules and building blocks for other applications. Synthetic methylating reagents are often toxic and unselective due to their high reactivity. S \\u2010Adenosyl\\u2010 l \\u2010methionine (SAM)\\u2010dependent methyltransferases (MTs) present a chemoselective and environmentally friendly alternative. The anthranilate N \\u2010MT from Ruta graveolens ( Rg ANMT) is involved in acridone alkaloid biosynthesis, methylating anthranilate. Although it is known to methylate substrates only at the N\\u2010 position, the closest relatives with respect to amino acid sequence similarities of over 60 % are O \\u2010MTs catalysing the methylation reaction of caffeate and derivatives containing only hydroxyl groups (CaOMTs). In this study, we investigated the substrate range of Rg ANMT and a CaOMT from Prunus persica ( Pp CaOMT) using compounds with both, an amino\\u2010 and hydroxyl group (aminophenols) as possible methyl group acceptors. For both enzymes, the reaction was highly chemoselective. Furthermore, generating cofactor derivatives in situ enabled the transfer of other alkyl chains onto the aminophenols, leading to an enlarged pool of products. Selected MT reactions were performed at a preparative biocatalytic scale in in vitro and in vivo experiments resulting in yields of up to 62 %.\", \"doi_y\": \"https://doi.org/10.1002/cctc.202300930\", \"publication_date\": \"2023-09-08\"}, {\"work_id\": \"W4282933962\", \"doi_x\": \"10.1038/s41467-022-31048-2\", \"af_direct_citation\": 1, \"af_indirect_citations\": 3, \"title\": \"The structure of EXTL3 helps to explain the different roles of bi-domain exostosins in heparan sulfate synthesis\", \"abstract\": \"Abstract Heparan sulfate is a highly modified O -linked glycan that performs diverse physiological roles in animal tissues. Though quickly modified, it is initially synthesised as a polysaccharide of alternating \\u03b2- d -glucuronosyl and N -acetyl-\\u03b1- d -glucosaminyl residues by exostosins. These enzymes generally possess two glycosyltransferase domains (GT47 and GT64)\\u2014each thought to add one type of monosaccharide unit to the backbone. Although previous structures of murine exostosin-like 2 (EXTL2) provide insight into the GT64 domain, the rest of the bi-domain architecture is yet to be characterised; hence, how the two domains co-operate is unknown. Here, we report the structure of human exostosin-like 3 (EXTL3) in apo and UDP-bound forms. We explain the ineffectiveness of EXTL3\\u2019s GT47 domain to transfer \\u03b2- d -glucuronosyl units, and we observe that, in general, the bi-domain architecture would preclude a processive mechanism of backbone extension. We therefore propose that heparan sulfate backbone polymerisation occurs by a simple dissociative mechanism.\", \"doi_y\": \"https://doi.org/10.1038/s41467-022-31048-2\", \"publication_date\": \"2022-06-08\"}, {\"work_id\": \"W4220772739\", \"doi_x\": \"10.1101/2022.03.19.484997\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Correlations between human mating partners: a comprehensive meta-analysis of 22 traits and raw data analysis of 133 traits in the UK Biobank\", \"abstract\": \"Abstract Positive correlations between human mating partners are consistently observed across traits. Such correlations can increase phenotypic variation and, to the extent that they reflect genetic similarity in co-parents, can also increase prevalence for rare phenotypes and bias estimates in genetic designs. We conducted the largest set of meta-analyses on human partner correlations to date, incorporating 480 partner correlations across 22 traits. We also calculated 133 trait correlations between up to 79,074 male-female couples in the UK Biobank (UKB). Estimates of the mean meta-analyzed correlations ranged from r meta =.08 for extraversion to r meta = .58 for political values. UKB correlations ranged from r UKB =-.18 for chronotype to r UKB =.87 for birth year. Overall, attitudes, education, and substance use traits mostly showed the highest correlations, while psychological and biological traits generally yielded lower but still positive correlations. We observed high between-study heterogeneity for most meta-analyzed traits, likely because of both systematic differences between samples and true differences in partner correlations across populations.\", \"doi_y\": \"https://doi.org/10.1101/2022.03.19.484997\", \"publication_date\": \"2022-03-20\"}, {\"work_id\": \"W4302024092\", \"doi_x\": \"10.1101/2022.10.03.510624\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Computational investigation of mechanisms for pH modulation of human chloride channels\", \"abstract\": \"Abstract Many transmembrane proteins are modulated by intracellular or extracellular pH. Investigation of pH-dependence generally proceeds by mutagenesis of a wide set of amino acids, guided by properties such as amino acid conservation and structure. This study shows how web-based calculation of pKas allows rapid and effective identification of amino acids of interest with respect to pH-dependence. Commencing with the calcium-activated chloride channel bestrophin 1, the carboxylate ligand structure around calcium sites relaxes in the absence of calcium, consistent with a measured lack of pH-dependence. By contrast, less relaxation in the absence of calcium in TMEM16A, and maintenance of elevated carboxylate sidechain pKas, is suggested to give rise to pH-dependent chloride channel activity. This hypothesis, modulation of calcium/proton coupling and pH-dependent activity through extent of structural relaxation, is shown to apply to the well-characterised cytosolic proteins calmodulin (pH-independent) and calbindin D 9k (pH-dependent). Further application of destabilised ionisable charge sites, or electrostatic frustration, is made to other human chloride channels (that are not calcium-activated), ClC-2, GABA A , and GlyR. Experimentally-determined sites of pH modulation are readily identified. The structure-based tool is freely available, allowing users to focus mutagenesis studies, construct hypothetical proton pathways, and derive hypotheses such as the model for control of pH-dependent calcium activation through structural flexibility. Predicting altered pH-dependence for mutations in ion channel disorders can support experimentation and, ultimately, clinical intervention.\", \"doi_y\": \"https://doi.org/10.1101/2022.10.03.510624\", \"publication_date\": \"2022-10-05\"}, {\"work_id\": \"W4385492078\", \"doi_x\": \"10.1016/j.pt.2023.07.002\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Meiosis in Plasmodium: how does it work?\", \"abstract\": \"Meiosis is sexual cell division, a process in eukaryotes whereby haploid gametes are produced. Compared to canonical model eukaryotes, meiosis in apicomplexan parasites appears to diverge from the process with respect to the molecular mechanisms involved; the biology of Plasmodium meiosis, and its regulation by means of post-translational modification, are largely unexplored. Here, we discuss the impact of technological advances in cell biology, evolutionary bioinformatics, and genome-wide functional studies on our understanding of meiosis in the Apicomplexa. These parasites, including Plasmodium falciparum, Toxoplasma gondii, and Eimeria spp., have significant socioeconomic impact on human and animal health. Understanding this key stage during the parasite's life cycle may well reveal attractive targets for therapeutic intervention.\", \"doi_y\": \"https://doi.org/10.1016/j.pt.2023.07.002\", \"publication_date\": \"2023-10-01\"}, {\"work_id\": \"W4297521014\", \"doi_x\": \"10.1101/2022.09.27.509720\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Hybridisation has shaped a recent radiation of grass-feeding aphids\", \"abstract\": \"Abstract Aphids are common crop pests. These insects reproduce by facultative parthenogenesis involving several rounds of clonal reproduction interspersed with an occasional sexual cycle. Furthermore, clonal aphids give birth to live apterous or winged young that are already pregnant. Together, these qualities enable rapid population growth and have facilitated the colonisation of crops globally. In several cases so-called \\u201csuper clones\\u201d have come to dominate agricultural systems. However, the extent to which the sexual stage of the aphid life cycle has shaped global pest populations has remained largely unclear, as have the origins of successful lineages. Here, we used chromosome-scale genome assemblies to disentangle the evolution of two of the most significant global pests of cereals \\u2013 the English ( Sitobion avenae ) and Indian ( Sitobion miscanthi ) grain aphids. We found that genome-wide divergence between S. avenae and S. miscanthi is low and that S. avenae sensu stricto is part of a larger cryptic species complex that includes multiple diverged S. miscanthi lineages. Moreover, comparison of haplotype-resolved assemblies reveals that the S. miscanthi isolate used for genome sequencing is likely a hybrid, with one of its diploid genome copies being closely related to S. avenae (\\u223c0.5% divergence) and the second being substantially more divergent (&gt; 1%). Analyses of genotyping-by-sequencing (GBS) data of grain aphids from the UK and China revealed that S. avenae and S. miscanthi are part of a species complex with many highly differentiated lineages that predate the origins of agriculture. The complex consists of hybrid lineages that display a tangled history of hybridisation and genetic introgression. These data demonstrate that hybridisation has substantially contributed to grain aphid diversity, and hence, to the evolutionary potential of this important pest species.\", \"doi_y\": \"https://doi.org/10.1101/2022.09.27.509720\", \"publication_date\": \"2022-09-28\"}, {\"work_id\": \"W4289941695\", \"doi_x\": \"10.1177/10943420221113513\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Intelligent resolution: Integrating Cryo-EM with AI-driven multi-resolution simulations to observe the severe acute respiratory syndrome coronavirus-2 replication-transcription machinery in action\", \"abstract\": \"The severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) replication transcription complex (RTC) is a multi-domain protein responsible for replicating and transcribing the viral mRNA inside a human cell. Attacking RTC function with pharmaceutical compounds is a pathway to treating COVID-19. Conventional tools, e.g. cryo-electron microscopy and all-atom molecular dynamics (AAMD), do not provide sufficiently high resolution or timescale to capture important dynamics of this molecular machine. Consequently, we develop an innovative workflow that bridges the gap between these resolutions, using mesoscale fluctuating finite element analysis (FFEA) continuum simulations and a hierarchy of AI-methods that continually learn and infer features for maintaining consistency between AAMD and FFEA simulations. We leverage a multi-site distributed workflow manager to orchestrate AI, FFEA, and AAMD jobs, providing optimal resource utilization across HPC centers. Our study provides unprecedented access to study the SARS-CoV-2 RTC machinery, while providing general capability for AI-enabled multi-resolution simulations at scale.\", \"doi_y\": \"https://doi.org/10.1177/10943420221113513\", \"publication_date\": \"2022-08-05\"}, {\"work_id\": \"W4296793671\", \"doi_x\": \"10.7554/elife.79771\", \"af_direct_citation\": 1, \"af_indirect_citations\": 4, \"title\": \"A feed-forward pathway drives LRRK2 kinase membrane recruitment and activation\", \"abstract\": \"Activating mutations in the leucine-rich repeat kinase 2 (LRRK2) cause Parkinson's disease, and previously we showed that activated LRRK2 phosphorylates a subset of Rab GTPases (Steger et al., 2017). Moreover, Golgi-associated Rab29 can recruit LRRK2 to the surface of the Golgi and activate it there for both auto- and Rab substrate phosphorylation. Here, we define the precise Rab29 binding region of the LRRK2 Armadillo domain between residues 360-450 and show that this domain, termed 'site #1,' can also bind additional LRRK2 substrates, Rab8A and Rab10. Moreover, we identify a distinct, N-terminal, higher-affinity interaction interface between LRRK2 phosphorylated Rab8 and Rab10 termed 'site #2' that can retain LRRK2 on membranes in cells to catalyze multiple, subsequent phosphorylation events. Kinase inhibitor washout experiments demonstrate that rapid recovery of kinase activity in cells depends on the ability of LRRK2 to associate with phosphorylated Rab proteins, and phosphorylated Rab8A stimulates LRRK2 phosphorylation of Rab10 in vitro. Reconstitution of purified LRRK2 recruitment onto planar lipid bilayers decorated with Rab10 protein demonstrates cooperative association of only active LRRK2 with phospho-Rab10-containing membrane surfaces. These experiments reveal a feed-forward pathway that provides spatial control and membrane activation of LRRK2 kinase activity.\", \"doi_y\": \"https://doi.org/10.7554/elife.79771\", \"publication_date\": \"2022-09-23\"}, {\"work_id\": \"W4297088219\", \"doi_x\": \"10.1038/s42003-022-03857-0\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Transcriptomic plasticity of the hypothalamic osmoregulatory control centre of the Arabian dromedary camel\", \"abstract\": \"Water conservation is vital for life in the desert. The dromedary camel (Camelus dromedarius) produces low volumes of highly concentrated urine, more so when water is scarce, to conserve body water. Two hormones, arginine vasopressin and oxytocin, both produced in the supraoptic nucleus, the core hypothalamic osmoregulatory control centre, are vital for this adaptive process, but the mechanisms that enable the camel supraoptic nucleus to cope with osmotic stress are not known. To investigate the central control of water homeostasis in the camel, we first build three dimensional models of the camel supraoptic nucleus based on the expression of the vasopressin and oxytocin mRNAs in order to facilitate sampling. We then compare the transcriptomes of the supraoptic nucleus under control and water deprived conditions and identified genes that change in expression due to hyperosmotic stress. By comparing camel and rat datasets, we have identified common elements of the water deprivation transcriptomic response network, as well as elements, such as extracellular matrix remodelling and upregulation of angiotensinogen expression, that appear to be unique to the dromedary camel and that may be essential adaptations necessary for life in the desert.\", \"doi_y\": \"https://doi.org/10.1038/s42003-022-03857-0\", \"publication_date\": \"2022-09-23\"}, {\"work_id\": \"W4386498874\", \"doi_x\": \"10.1038/s41467-023-41211-y\", \"af_direct_citation\": 0, \"af_indirect_citations\": 3, \"title\": \"Intrinsically disordered CsoS2 acts as a general molecular thread for \\u03b1-carboxysome shell assembly\", \"abstract\": \"Carboxysomes are a paradigm of self-assembling proteinaceous organelles found in nature, offering compartmentalisation of enzymes and pathways to enhance carbon fixation. In \\u03b1-carboxysomes, the disordered linker protein CsoS2 plays an essential role in carboxysome assembly and Rubisco encapsulation. Its mechanism of action, however, is not fully understood. Here we synthetically engineer \\u03b1-carboxysome shells using minimal shell components and determine cryoEM structures of these to decipher the principle of shell assembly and encapsulation. The structures reveal that the intrinsically disordered CsoS2 C-terminus is well-structured and acts as a universal \\\"molecular thread\\\" stitching through multiple shell protein interfaces. We further uncover in CsoS2 a highly conserved repetitive key interaction motif, [IV]TG, which is critical to the shell assembly and architecture. Our study provides a general mechanism for the CsoS2-governed carboxysome shell assembly and cargo encapsulation and further advances synthetic engineering of carboxysomes for diverse biotechnological applications.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-41211-y\", \"publication_date\": \"2023-09-07\"}, {\"work_id\": \"W4309600450\", \"doi_x\": \"10.1371/journal.pone.0268149\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Monoallelic variants resulting in substitutions of MAB21L1 Arg51 Cause Aniridia and microphthalmia\", \"abstract\": \"Classical aniridia is a congenital and progressive panocular disorder almost exclusively caused by heterozygous loss-of-function variants at the PAX6 locus. We report nine individuals from five families with severe aniridia and/or microphthalmia (with no detectable PAX6 mutation) with ultrarare monoallelic missense variants altering the Arg51 codon of MAB21L1 . These mutations occurred de novo in 3/5 families, with the remaining families being compatible with autosomal dominant inheritance. Mice engineered to carry the p.Arg51Leu change showed a highly-penetrant optic disc anomaly in heterozygous animals with severe microphthalmia in homozygotes. Substitutions of the same codon (Arg51) in MAB21L2 , a close homolog of MAB21L1 , cause severe ocular and skeletal malformations in humans and mice. The predicted nucleotidyltransferase function of MAB21L1 could not be demonstrated using purified protein with a variety of nucleotide substrates and oligonucleotide activators. Induced expression of GFP-tagged wildtype and mutant MAB21L1 in human cells caused only modest transcriptional changes. Mass spectrometry of immunoprecipitated protein revealed that both mutant and wildtype MAB21L1 associate with transcription factors that are known regulators of PAX6 (MEIS1, MEIS2 and PBX1) and with poly(A) RNA binding proteins. Arg51 substitutions reduce the association of wild-type MAB21L1 with TBL1XR1, a component of the NCoR complex. We found limited evidence for mutation-specific interactions with MSI2/Musashi-2, an RNA-binding proteins with effects on many different developmental pathways. Given that biallelic loss-of-function variants in MAB21L1 result in a milder eye phenotype we suggest that Arg51-altering monoallelic variants most plausibly perturb eye development via a gain-of-function mechanism.\", \"doi_y\": \"https://doi.org/10.1371/journal.pone.0268149\", \"publication_date\": \"2022-11-22\"}, {\"work_id\": \"W4312193693\", \"doi_x\": \"10.1016/j.carbpol.2022.120482\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biofilm inspired fabrication of functional bacterial cellulose through ex-situ and in-situ approaches\", \"abstract\": \"Bacterial cellulose (BC) has been explored for use in a range of applications including tissue engineering and textiles. BC can be produced from waste streams, but sustainable approaches are needed for functionalisation. To this end, BslA, a B. subtilis biofilm protein was produced recombinantly with and without a cellulose binding module (CBM) and the cell free extract was used to treat BC either ex-situ, through drip coating or in-situ, by incorporating during fermentation. The results showed that ex-situ modified BC increased the hydrophobicity and water contact angle reached 120\\u00b0. In-situ experiments led to a BC film morphological change and mechanical testing demonstrated that addition of BslA with CBM resulted in a stronger, more elastic material. This study presents a nature inspired approach to functionalise BC using a biofilm hydrophobin, and we demonstrate that recombinant proteins could be effective and sustainable molecules for functionalisation of BC materials.\", \"doi_y\": \"https://doi.org/10.1016/j.carbpol.2022.120482\", \"publication_date\": \"2023-03-01\"}, {\"work_id\": \"W4379598190\", \"doi_x\": \"10.1093/femsre/fuad026\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"The role of glycoconjugates as receptors for insecticidal proteins\", \"abstract\": \"Bacillus thuringiensis (Bt) proteins are an environmentally safe and effective alternative to chemical pesticides and have been used as biopesticides, with great commercial success, for over 50 years. Global agricultural production is predicted to require a 70% increase until 2050 to provide for an increasing population. In addition to agriculture, Bt proteins are utilized to control human vectors of disease-namely mosquitoes-which account for >700 000 deaths annually. The evolution of resistance to Bt pesticial toxins threatens the progression of sustainable agriculture. Whilst Bt protein toxins are heavily utilized, the exact mechanisms behind receptor binding and toxicity are unknown. It is critical to gain a better understanding of these mechanisms in order to engineer novel toxin variants and to predict, and prevent, future resistance evolution. This review focuses on the role of carbohydrate binding in the toxicity of the most utilized group of Bt pesticidal proteins-three domain Cry (3D-Cry) toxins.\", \"doi_y\": \"https://doi.org/10.1093/femsre/fuad026\", \"publication_date\": \"2023-06-02\"}, {\"work_id\": \"W4320481144\", \"doi_x\": \"10.1101/2023.02.12.528179\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Light sensitive short hypocotyl (LSH) confer symbiotic nodule identity in the legume Medicago truncatula\", \"abstract\": \"Summary Legumes grow specialized root nodules that are distinct from lateral roots in morphology and function, with nodules intracellularly hosting beneficial nitrogen-fixing bacteria that provide the plant with a nitrogen source. We have previously shown that a lateral root-like program underpins nodule initiation, but there must be additional developmental programs that confer nodule identity. Here, we show that two members of the LIGHT SENSITIVE SHORT HYPOCOTYL (LSH) transcription factor family, predominantly known to define organ boundaries and meristem complexity in the shoot, function as regulators of nodule organ identity. LSH1/LSH2 function upstream of and together with the known nodule regulators Nuclear Factor Y - A1 and NODULE ROOT1/2. The principal outcome of LSH1/LSH2 function is the production of cells able to accommodate nitrogen-fixing bacteria, the unique nodule feature. We conclude that the coordinate recruitment of a pre-existing primordium identity program, in parallel to a root initiation program, underpins the divergence between lateral roots and nodules.\", \"doi_y\": \"https://doi.org/10.1101/2023.02.12.528179\", \"publication_date\": \"2023-02-13\"}, {\"work_id\": \"W4386470019\", \"doi_x\": \"10.1371/journal.pntd.0011618\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Exploring the antimicrobial peptidome of nematodes through phylum-spanning in silico analyses highlights novel opportunities for pathogen control\", \"abstract\": \"Antimicrobial Peptides (AMPs) are key constituents of the invertebrate innate immune system and provide critical protection against microbial threat. Nematodes display diverse life strategies where they are exposed to heterogenous, microbe rich, environments highlighting their need for an innate immune system. Within the Ecdysozoa, arthropod AMPs have been well characterised, however nematode-derived AMP knowledge is limited. In this study the distribution and abundance of putative AMP-encoding genes was examined in 134 nematode genomes providing the most comprehensive profile of AMP candidates within phylum Nematoda. Through genome and transcriptome analyses we reveal that phylum Nematoda is a rich source of putative AMP diversity and demonstrate (i) putative AMP group profiles that are influenced by nematode lifestyle where free-living nematodes appear to display enriched putative AMP profiles relative to parasitic species; (ii) major differences in the putative AMP profiles between nematode clades where Clade 9/V and 10/IV species possess expanded putative AMP repertoires; (iii) AMP groups with highly restricted profiles (e.g. Cecropins and Diapausins) and others [e.g. Nemapores and Glycine Rich Secreted Peptides (GRSPs)] which are more widely distributed; (iv) complexity in the distribution and abundance of CS\\u03b1\\u03b2 subgroup members; and (v) that putative AMPs are expressed in host-facing life stages and biofluids of key nematode parasites. These data indicate that phylum Nematoda displays diversity in putative AMPs and underscores the need for functional characterisation to reveal their role and importance to nematode biology and host-nematode-microbiome interactions.\", \"doi_y\": \"https://doi.org/10.1371/journal.pntd.0011618\", \"publication_date\": \"2023-09-06\"}, {\"work_id\": \"W4310433496\", \"doi_x\": \"10.1021/jacsau.2c00481\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Biocatalytic Synthesis of Antiviral Nucleosides, Cyclic Dinucleotides, and Oligonucleotide Therapies\", \"abstract\": \"Nucleosides, nucleotides, and oligonucleotides modulate diverse cellular processes ranging from protein production to cell signaling. It is therefore unsurprising that synthetic analogues of nucleosides and their derivatives have emerged as a versatile class of drug molecules for the treatment of a wide range of disease areas. Despite their great therapeutic potential, the dense arrangements of functional groups and stereogenic centers present in nucleic acid analogues pose a considerable synthetic challenge, especially in the context of large-scale manufacturing. Commonly employed synthetic methods rely on extensive protecting group manipulations, which compromise step-economy and result in high process mass intensities. Biocatalytic approaches have the potential to address these limitations, enabling the development of more streamlined, selective, and sustainable synthetic routes. Here we review recent achievements in the biocatalytic manufacturing of nucleosides and cyclic dinucleotides along with progress in developing enzymatic strategies to produce oligonucleotide therapies. We also highlight opportunities for innovations that are needed to facilitate widespread adoption of these biocatalytic methods across the pharmaceutical industry.\", \"doi_y\": \"https://doi.org/10.1021/jacsau.2c00481\", \"publication_date\": \"2022-11-30\"}, {\"work_id\": \"W4313486378\", \"doi_x\": \"10.1126/sciadv.add3479\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"abstract\": \"Gene transcription is carried out by RNA polymerase (RNAP) and requires the conversion of the initial closed promoter complex, where DNA is double stranded, to a transcription-competent open promoter complex, where DNA is opened up. In bacteria, RNAP relies on \\u03c3 factors for its promoter specificities. Using a special form of sigma factor (\\u03c354), which forms a stable closed complex and requires its activator that belongs to the AAA+ ATPases (ATPases associated with diverse cellular activities), we obtained cryo-electron microscopy structures of transcription initiation complexes that reveal a previously unidentified process of DNA melting opening. The \\u03c354 amino terminus threads through the locally opened up DNA and then becomes enclosed by the AAA+ hexameric ring in the activator-bound intermediate complex. Our structures suggest how ATP hydrolysis by the AAA+ activator could remove the \\u03c354 inhibition while helping to open up DNA, using \\u03c354 amino-terminal peptide as a pry bar.\", \"doi_y\": \"https://doi.org/10.1126/sciadv.add3479\", \"publication_date\": \"2022-12-23\"}, {\"work_id\": \"W4366831064\", \"doi_x\": \"10.1016/j.biosystems.2023.104905\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Metabolic Control Analysis of triacylglycerol accumulation in oilseed rape\", \"abstract\": \"The increasing global demand for vegetable oils will only be met if there are significant improvements in the productivity of the major oil crops, such as oilseed rape. Metabolic engineering offers the prospect of further gains in yield beyond that already achieved by breeding and selection but requires guidance as to the changes that need to be made. Metabolic Control Analysis, through measurement and estimation of flux control coefficients, can indicate which enzymes have the most influence on a desired flux. Some experiments have previously reported flux control coefficients for oil accumulation in the seeds of oilseed rape, and others have measured control coefficient distributions for multi-enzyme segments of oil synthesis in seed embryo metabolism measured in vitro. In addition, other reported manipulations of oil accumulation contain results that are exploited further here to calculate previously unknown flux control coefficients. These results are then assembled within a framework that allows an integrated interpretation of the controls on oil accumulation from the assimilation of CO2 to deposition of oil in the seed. The analysis shows that the control is distributed to an extent that the gains from amplifying any single target are necessarily limited, but there are candidates for joint amplification that are likely to act synergistically to produce much more significant gains.\", \"doi_y\": \"https://doi.org/10.1016/j.biosystems.2023.104905\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4286670068\", \"doi_x\": \"10.1101/2022.07.21.501023\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Learning from pre-pandemic data to forecast viral escape\", \"abstract\": \"Summary Effective pandemic preparedness relies on anticipating viral mutations that are able to evade host immune responses in order to facilitate vaccine and therapeutic design. However, current strategies for viral evolution prediction are not available early in a pandemic \\u2013 experimental approaches require host polyclonal antibodies to test against and existing computational methods draw heavily from current strain prevalence to make reliable predictions of variants of concern. To address this, we developed EVEscape, a generalizable, modular framework that combines fitness predictions from a deep learning model of historical sequences with biophysical structural information. EVEscape quantifies the viral escape potential of mutations at scale and has the advantage of being applicable before surveillance sequencing, experimental scans, or 3D structures of antibody complexes are available. We demonstrate that EVEscape, trained on sequences available prior to 2020, is as accurate as high-throughput experimental scans at anticipating pandemic variation for SARS-CoV-2 and is generalizable to other viruses including Influenza, HIV, and understudied viruses with pandemic potential such as Lassa and Nipah. We provide continually updated escape scores for all current strains of SARS-CoV-2 and predict likely additional mutations to forecast emerging strains as a tool for ongoing vaccine development ( evescape.org ).\", \"doi_y\": \"https://doi.org/10.1101/2022.07.21.501023\", \"publication_date\": \"2022-07-22\"}, {\"work_id\": \"W4380870307\", \"doi_x\": \"10.1093/nar/gkad514\", \"af_direct_citation\": 1, \"af_indirect_citations\": 10, \"title\": \"Updated protein domain annotation of the PARP protein family sheds new light on biological function\", \"abstract\": \"Abstract AlphaFold2 and related computational tools have greatly aided studies of structural biology through their ability to accurately predict protein structures. In the present work, we explored AF2 structural models of the 17 canonical members of the human PARP protein family and supplemented this analysis with new experiments and an overview of recent published data. PARP proteins are typically involved in the modification of proteins and nucleic acids through mono or poly(ADP-ribosyl)ation, but this function can be modulated by the presence of various auxiliary protein domains. Our analysis provides a comprehensive view of the structured domains and long intrinsically disordered regions within human PARPs, offering a revised basis for understanding the function of these proteins. Among other functional insights, the study provides a model of PARP1 domain dynamics in the DNA-free and DNA-bound states and enhances the connection between ADP-ribosylation and RNA biology and between ADP-ribosylation and ubiquitin-like modifications by predicting putative RNA-binding domains and E2-related RWD domains in certain PARPs. In line with the bioinformatic analysis, we demonstrate for the first time PARP14\\u2019s RNA-binding capability and RNA ADP-ribosylation activity in vitro. While our insights align with existing experimental data and are probably accurate, they need further validation through experiments.\", \"doi_y\": \"https://doi.org/10.1093/nar/gkad514\", \"publication_date\": \"2023-06-16\"}, {\"work_id\": \"W4376872501\", \"doi_x\": \"10.1038/s41467-023-38275-1\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Protection against SARS-CoV-2 Omicron BA.4/5 variant following booster vaccination or breakthrough infection in the UK\", \"abstract\": \"Following primary SARS-CoV-2 vaccination, whether boosters or breakthrough infections provide greater protection against SARS-CoV-2 infection is incompletely understood. Here we investigated SARS-CoV-2 antibody correlates of protection against new Omicron BA.4/5 (re-)infections and anti-spike IgG antibody trajectories after a third/booster vaccination or breakthrough infection following second vaccination in 154,149 adults \\u226518 y from the United Kingdom general population. Higher antibody levels were associated with increased protection against Omicron BA.4/5 infection and breakthrough infections were associated with higher levels of protection at any given antibody level than boosters. Breakthrough infections generated similar antibody levels to boosters, and the subsequent antibody declines were slightly slower than after boosters. Together our findings show breakthrough infection provides longer-lasting protection against further infections than booster vaccinations. Our findings, considered alongside the risks of severe infection and long-term consequences of infection, have important implications for vaccine policy.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-38275-1\", \"publication_date\": \"2023-05-16\"}, {\"work_id\": \"W4362676364\", \"doi_x\": \"10.1016/j.mcpro.2023.100544\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Translational and Posttranslational Dynamics in a Model Peptidergic System\", \"abstract\": \"The cell bodies of hypothalamic magnocellular neurones are densely packed in the hypothalamic supraoptic nucleus, whereas their axons project to the anatomically discrete posterior pituitary gland. We have taken advantage of this unique anatomical structure to establish proteome and phosphoproteome dynamics in neuronal cell bodies and axonal terminals in response to physiological stimulation. We have found that proteome and phosphoproteome responses to neuronal stimulation are very different between somatic and axonal neuronal compartments, indicating the need of each cell domain to differentially adapt. In particular, changes in the phosphoproteome in the cell body are involved in the reorganization of the cytoskeleton and in axonal terminals the regulation of synaptic and secretory processes. We have identified that prohormone precursors including vasopressin and oxytocin are phosphorylated in axonal terminals and are hyperphosphorylated following stimulation. By multiomic integration of transcriptome and proteomic data, we identify changes to proteins present in afferent inputs to this nucleus.\", \"doi_y\": \"https://doi.org/10.1016/j.mcpro.2023.100544\", \"publication_date\": \"2023-05-01\"}, {\"work_id\": \"W4379516072\", \"doi_x\": \"10.1099/mic.0.001342\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Speaking the host language: how Salmonella effector proteins manipulate the host\", \"abstract\": \"Salmonella injects over 40 virulence factors, termed effectors, into host cells to subvert diverse host cellular processes. Of these 40 Salmonella effectors, at least 25 have been described as mediating eukaryotic-like, biochemical post-translational modifications (PTMs) of host proteins, altering the outcome of infection. The downstream changes mediated by an effector's enzymatic activity range from highly specific to multifunctional, and altogether their combined action impacts the function of an impressive array of host cellular processes, including signal transduction, membrane trafficking, and both innate and adaptive immune responses. Salmonella and related Gram-negative pathogens have been a rich resource for the discovery of unique enzymatic activities, expanding our understanding of host signalling networks, bacterial pathogenesis as well as basic biochemistry. In this review, we provide an up-to-date assessment of host manipulation mediated by the Salmonella type III secretion system injectosome, exploring the cellular effects of diverse effector activities with a particular focus on PTMs and the implications for infection outcomes. We also highlight activities and functions of numerous effectors that remain poorly characterized.\", \"doi_y\": \"https://doi.org/10.1099/mic.0.001342\", \"publication_date\": \"2023-06-23\"}, {\"work_id\": \"W4318250429\", \"doi_x\": \"10.1038/s41576-022-00572-8\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Pharmacogenomics: current status and future perspectives\", \"abstract\": \"\", \"doi_y\": \"https://doi.org/10.1038/s41576-022-00572-8\", \"publication_date\": \"2023-01-27\"}, {\"work_id\": \"W4321787068\", \"doi_x\": \"10.1038/s41467-023-36610-0\", \"af_direct_citation\": 1, \"af_indirect_citations\": 2, \"title\": \"Enzyme adaptation to habitat thermal legacy shapes the thermal plasticity of marine microbiomes\", \"abstract\": \"Microbial communities respond to temperature with physiological adaptation and compositional turnover. Whether thermal selection of enzymes explains marine microbiome plasticity in response to temperature remains unresolved. By quantifying the thermal behaviour of seven functionally-independent enzyme classes (esterase, extradiol dioxygenase, phosphatase, beta-galactosidase, nuclease, transaminase, and aldo-keto reductase) in native proteomes of marine sediment microbiomes from the Irish Sea to the southern Red Sea, we record a significant effect of the mean annual temperature (MAT) on enzyme response in all cases. Activity and stability profiles of 228 esterases and 5 extradiol dioxygenases from sediment and seawater across 70 locations worldwide validate this thermal pattern. Modelling the esterase phase transition temperature as a measure of structural flexibility confirms the observed relationship with MAT. Furthermore, when considering temperature variability in sites with non-significantly different MATs, the broadest range of enzyme thermal behaviour and the highest growth plasticity of the enriched heterotrophic bacteria occur in samples with the widest annual thermal variability. These results indicate that temperature-driven enzyme selection shapes microbiome thermal plasticity and that thermal variability finely tunes such processes and should be considered alongside MAT in forecasting microbial community thermal response.\", \"doi_y\": \"https://doi.org/10.1038/s41467-023-36610-0\", \"publication_date\": \"2023-02-24\"}, {\"work_id\": \"W4303614299\", \"doi_x\": \"10.3390/biom12101425\", \"af_direct_citation\": 1, \"af_indirect_citations\": 13, \"title\": \"Protein Data Bank: A Comprehensive Review of 3D Structure Holdings and Worldwide Utilization by Researchers, Educators, and Students\", \"abstract\": \"The Research Collaboratory for Structural Bioinformatics Protein Data Bank (RCSB PDB), funded by the United States National Science Foundation, National Institutes of Health, and Department of Energy, supports structural biologists and Protein Data Bank (PDB) data users around the world. The RCSB PDB, a founding member of the Worldwide Protein Data Bank (wwPDB) partnership, serves as the US data center for the global PDB archive housing experimentally-determined three-dimensional (3D) structure data for biological macromolecules. As the wwPDB-designated Archive Keeper, RCSB PDB is also responsible for the security of PDB data and weekly update of the archive. RCSB PDB serves tens of thousands of data depositors (using macromolecular crystallography, nuclear magnetic resonance spectroscopy, electron microscopy, and micro-electron diffraction) annually working on all permanently inhabited continents. RCSB PDB makes PDB data available from its research-focused web portal at no charge and without usage restrictions to many millions of PDB data consumers around the globe. It also provides educators, students, and the general public with an introduction to the PDB and related training materials through its outreach and education-focused web portal. This review article describes growth of the PDB, examines evolution of experimental methods for structure determination viewed through the lens of the PDB archive, and provides a detailed accounting of PDB archival holdings and their utilization by researchers, educators, and students worldwide.\", \"doi_y\": \"https://doi.org/10.3390/biom12101425\", \"publication_date\": \"2022-10-04\"}, {\"work_id\": \"W4385516047\", \"doi_x\": \"10.1371/journal.ppat.1011545\", \"af_direct_citation\": 0, \"af_indirect_citations\": 2, \"title\": \"Deep mutagenesis scanning using whole trimeric SARS-CoV-2 spike highlights the importance of NTD-RBD interactions in determining spike phenotype\", \"abstract\": \"New variants of SARS-CoV-2 are continually emerging with mutations in spike associated with increased transmissibility and immune escape. Phenotypic maps can inform the prediction of concerning mutations from genomic surveillance, however most of these maps currently derive from studies using monomeric RBD, while spike is trimeric, and contains additional domains. These maps may fail to reflect interdomain interactions in the prediction of phenotypes. To try to improve on this, we developed a platform for deep mutational scanning using whole trimeric spike. We confirmed a previously reported epistatic effect within the RBD affecting ACE2 binding, that highlights the importance of updating the base spike sequence for future mutational scanning studies. Using post vaccine sera, we found that the immune response of vaccinated individuals was highly focused on one or two epitopes in the RBD and that single point mutations at these positions can account for most of the immune escape mediated by the Omicron BA.1 RBD. However, unexpectedly we found that the BA.1 RBD alone does not account for the high level of antigenic escape by BA.1 spike. We show that the BA.1 NTD amplifies the immune evasion of its associated RBD. BA.1 NTD reduces neutralistion by RBD directed monoclonal antibodies, and impacts ACE2 interaction. NTD variation is thus an important mechanism of immune evasion by SARS-CoV-2. Such effects are not seen when pre-stabilized spike proteins are used, suggesting the interdomain effects require protein mobility to express their phenotype.\", \"doi_y\": \"https://doi.org/10.1371/journal.ppat.1011545\", \"publication_date\": \"2023-08-03\"}, {\"work_id\": \"W4205929130\", \"doi_x\": \"10.1038/s41592-021-01361-7\", \"af_direct_citation\": 1, \"af_indirect_citations\": 8, \"title\": \"A paradigm shift in structural biology\", \"abstract\": \"The release of protein structure predictions from AlphaFold will increase the number of protein structural models by almost three orders of magnitude. Structural biology and bioinformatics will never be the same, and the need for incisive experimental approaches will be greater than ever. Combining these advances in structure prediction with recent advances in cryo-electron microscopy suggests a new paradigm for structural biology.\", \"doi_y\": \"https://doi.org/10.1038/s41592-021-01361-7\", \"publication_date\": \"2022-01-01\"}, {\"work_id\": \"W4205593626\", \"doi_x\": \"10.1101/2022.01.17.22269412\", \"af_direct_citation\": 1, \"af_indirect_citations\": 1, \"title\": \"EFCAB4B (CRACR2A) genetic variants associated with COVID-19 fatality\", \"abstract\": \"Abstract The coronavirus disease 2019 (COVID-19) pandemic, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has resulted in more than 235 million cases worldwide and 4.8 million deaths (October 2021). Severe COVID-19 is characterised in part by vascular thrombosis and a cytokine storm due to increased plasma concentrations of factors secreted from endothelial and T-cells. Here, using patient data recorded in the UK Biobank, we demonstrate the importance of variations in Rab46 (CRACR2A) with clinical outcomes. Using logistic regression analysis, we determined that three single nucleotide polymorphisms (SNPs) in the gene EFCAB4B cause missense mutations in Rab46, which are associated with COVID-19 fatality independently of risk factors. All three SNPs cause changes in amino acid residues that are highly conserved across species, indicating their importance in protein structure and function. Two SNPs, rs17836273 (A98T) and rs36030417 (H212Q), cause amino acid substitutions in important functional domains: the EF-hand and coiled-coil domain respectively. By using molecular modelling, we suggest that the substitution of threonine at position 98 causes structural changes in the EF-hand calcium binding domain. Since Rab46 is a Rab GTPase that regulates both endothelial cell secretion and T-cell signalling, these missense variations may play a role in the molecular mechanisms underlying the thrombotic and inflammatory characteristics observed in patients with severe COVID-19 outcomes.\", \"doi_y\": \"https://doi.org/10.1101/2022.01.17.22269412\", \"publication_date\": \"2022-01-17\"}, {\"work_id\": \"W4383217220\", \"doi_x\": \"10.3389/fpls.2023.1230278\", \"af_direct_citation\": 0, \"af_indirect_citations\": 1, \"title\": \"Mechanisms of prezygotic post-pollination reproductive barriers in plants\", \"abstract\": \"Hybridisation between individuals of different species can lead to maladapted or inviable progeny due to genetic incompatibilities between diverging species. On the other hand, mating with close relatives, or self-fertilisation may lead to inbreeding depression. Thus, both too much or too little divergence may lead to problems and the organisms have to carefully choose mating partners to avoid both of these pitfalls. In plants this choice occurs at many stages during reproduction, but pollen-pistil interactions play a particularly important role in avoiding inbreeding and hybridisation with other species. Interestingly, the mechanisms involved in avoidance of selfing and interspecific hybridisation may work via shared molecular pathways, as self-incompatible species tend to be more 'choosy' with heterospecific pollen compared to self-compatible ones. This review discusses various prezygotic post-pollination barriers to interspecific hybridisation, with a focus on the mechanisms of pollen-pistil interactions and their role in the maintenance of species integrity.\", \"doi_y\": \"https://doi.org/10.3389/fpls.2023.1230278\", \"publication_date\": \"2023-07-05\"}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "indirect_citations = citations[citations.af_indirect_citations > 0]\n",
    "\n",
    "indirect_citations_chart = alt.Chart(indirect_citations).mark_bar().encode(\n",
    "    x=alt.X('af_indirect_citations:Q', title='Number of Papers'),\n",
    "    y=alt.Y('title:N', title='Paper Title', sort='-x'),\n",
    "    tooltip=['title', 'af_indirect_citations']\n",
    ").properties(\n",
    "    title='Count of Papers that Indirectly Cite Alphafold',\n",
    "    height=6400\n",
    ")\n",
    "\n",
    "indirect_citations_chart"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
      ],
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<style>\n",
       "  #altair-viz-47c7b85e548a473480330e523c35cf24.vega-embed {\n",
       "    width: 100%;\n",
       "    display: flex;\n",
       "  }\n",
       "\n",
       "  #altair-viz-47c7b85e548a473480330e523c35cf24.vega-embed details,\n",
       "  #altair-viz-47c7b85e548a473480330e523c35cf24.vega-embed details summary {\n",
       "    position: relative;\n",
       "  }\n",
       "</style>\n",
       "<div id=\"altair-viz-47c7b85e548a473480330e523c35cf24\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-47c7b85e548a473480330e523c35cf24\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-47c7b85e548a473480330e523c35cf24\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm/vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm/vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm/vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"5.16.3\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 300, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-b432ea6aebcfdc329c1638c81b23adf0\"}, \"mark\": {\"type\": \"bar\"}, \"encoding\": {\"color\": {\"field\": \"type\", \"scale\": {\"domain\": [\"Indirect\", \"Direct\"], \"range\": [\"blue\", \"red\"]}, \"title\": \"Citation Type\", \"type\": \"nominal\"}, \"tooltip\": [{\"field\": \"title\", \"type\": \"nominal\"}, {\"aggregate\": \"sum\", \"field\": \"count\", \"type\": \"quantitative\"}, {\"field\": \"type\", \"type\": \"nominal\"}], \"x\": {\"field\": \"count\", \"title\": \"Number of Citations\", \"type\": \"quantitative\"}, \"y\": {\"field\": \"title\", \"sort\": \"-x\", \"title\": \"Paper Title\", \"type\": \"nominal\"}}, \"height\": 6400, \"title\": \"Count of Papers Citing Alphafold (Direct and Indirect)\", \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.16.3.json\", \"datasets\": {\"data-b432ea6aebcfdc329c1638c81b23adf0\": [{\"title\": \"Protein target highlights in <scp>CASP15</scp>: Analysis of models by structure providers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 16}, {\"title\": \"Protein Data Bank: A Comprehensive Review of 3D Structure Holdings and Worldwide Utilization by Researchers, Educators, and Students\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 13}, {\"title\": \"AlphaFold-predicted protein structures and small-angle X-ray scattering: insights from an extended examination of selected data in the Small-Angle Scattering Biological Data Bank\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 12}, {\"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 11}, {\"title\": \"Updated protein domain annotation of the PARP protein family sheds new light on biological function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 10}, {\"title\": \"3D-Beacons: decreasing the gap between protein sequences and structures through a federated network of protein structure data resources\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 10}, {\"title\": \"To split or not to split: <scp>CASP15</scp> targets and their processing into tertiary structure evaluation units\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 10}, {\"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 9}, {\"title\": \"Improvement of protein tertiary and quaternary structure predictions using the ReFOLD refinement method and the AlphaFold2 recycling process\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 9}, {\"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Lumenal components of cytoplasmic microtubules\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Pro-phagocytic function and structural basis of GPR84 signaling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Computational Resources for Molecular Biology 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Computational Resources for Molecular Biology 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Comprehensive Collection and Prediction of ABC Transmembrane Protein Structures in the AI Era of Structural Biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Sequence Analysis and Structural Predictions of Lipid Transfer Bridges in the Repeating Beta Groove (RBG) Superfamily Reveal Past and Present Domain Variations Affecting Form, Function and Interactions of VPS13, ATG2, SHIP164, Hobbit and Tweek\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Codon language embeddings provide strong signals for protein engineering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"A paradigm shift in structural biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 8}, {\"title\": \"Large protein complex interfaces have evolved to promote cotranslational assembly\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"Design of Heme Enzymes with a Tunable Substrate Binding Pocket Adjacent to an Open Metal Coordination Site\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"Protein diversification through post-translational modifications, alternative splicing, and gene duplication\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"NLR receptors in plant immunity: making sense of the alphabet soup\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"<scp>RCSB</scp> Protein Data bank: Tools for visualizing and understanding biological macromolecules in <scp>3D</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"The AlphaFold Database of Protein Structures: A Biologist\\u2019s Guide\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 7}, {\"title\": \"Resurrection of plant disease resistance proteins via helper NLR bioengineering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 6}, {\"title\": \"OAF: a new member of the BRICHOS family\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Assessing PDB macromolecular crystal structure confidence at the individual amino acid residue level\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Reciprocal best structure hits: using AlphaFold models to discover distant homologues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Estimation of model accuracy in <scp>CASP15</scp> using the M<scp>odFOLDdock</scp> server\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Characterizing and explaining impact of disease-associated mutations in proteins without known structures or structural homologues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"A unifying mechanism for protein transport through the core bacterial Sec machinery\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"UniProt: the Universal Protein Knowledgebase in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 5}, {\"title\": \"Mechanism of primer synthesis by Primase-Polymerases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Slow growing bacteria survive bacteriophage in isolation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"How do interactions between mobile genetic elements affect horizontal gene transfer?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"X-ray structure of the metastable SEPT14\\u2013SEPT7 coiled coil reveals a hendecad region crucial for heterodimerization\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Systematic analysis of membrane contact sites in Saccharomyces cerevisiae uncovers modulators of cellular lipid distribution\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"A feed-forward pathway drives LRRK2 kinase membrane recruitment and activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"A novel superfamily of bridge-like lipid transfer proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Sensor <scp>NLR</scp> immune proteins activate oligomerization of their <scp>NRC</scp> helpers in response to plant pathogens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Expanding the Substrate Scope of <i>N</i>\\u2010 and <i>O</i>\\u2010Methyltransferases from Plants for Chemoselective Alkylation**\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"How neurons maintain their axons long-term: an integrated view of axon biology and pathology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structural Analysis of Bacillus subtilis Sigma Factors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Deep Learning Concepts and Applications for Synthetic Biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"PKC isoforms activate LRRK1 kinase by phosphorylating conserved residues (Ser1064, Ser1074 and Thr1075) within the COR<sub>B</sub> GTPase domain\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"The Transporter-Mediated Cellular Uptake and Efflux of Pharmaceutical Drugs and Biotechnology Products: How and Why Phospholipid Bilayer Transport Is Negligible in Real Biomembranes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Decoding Catalysis by Terpene Synthases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Annotation of biologically relevant ligands in UniProtKB using ChEBI\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structure of the Commander endosomal trafficking complex linked to X-linked intellectual disability/Ritscher-Schinzel syndrome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Plant and prokaryotic TIR domains generate distinct cyclic ADPR NADase products\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Impact of 100 LRRK2 variants linked to Parkinson\\u2019s Disease on kinase activity and microtubule binding\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"KinFams: De-Novo Classification of Protein Kinases Using CATH Functional Units\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structural basis of peptidoglycan synthesis by E. coli RodA-PBP2 complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 4}, {\"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Estimating clinical risk in gene regions from population sequencing cohort data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"SARS-CoV-2 variant evasion of monoclonal antibodies based on in vitro studies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"SARS-CoV-2 variant biology: immune escape, transmission and fitness\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Evaluation of model refinement in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure and functional mapping of the KRAB\\u2010KAP1 repressor complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure and functional mapping of the KRAB-KAP1 repressor complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"An expandable, modular de novo protein platform for precision redox engineering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure determination and analysis of titin A-band fibronectin type III domains provides insights for disease-linked variants and protein oligomerisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Computational models in the service of X\\u2010ray and <scp>cryo\\u2010</scp>electron microscopy structure determination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Design of the elusive proteinaceous oxygen donor copper site suggests a promising future for copper for MRI contrast agents\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure-Based Design of Small Imine Reductase Panels for Target Substrates\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"The increasing role of structural proteomics in cyanobacteria\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Coiled-coil structure of meiosis protein TEX12 and conformational regulation by its C-terminal tip\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Rgs1 is a regulator of effector gene expression during plant infection by the rice blast fungus <i>Magnaporthe oryzae</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"The structure of EXTL3 helps to explain the different roles of bi-domain exostosins in heparan sulfate synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Proximity-Based Modalities for Biology and Medicine\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"NDP52 acts as a redox sensor in PINK1/Parkin\\u2010mediated mitophagy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Intrinsically disordered CsoS2 acts as a general molecular thread for \\u03b1-carboxysome shell assembly\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"PTCHD1 Binds Cholesterol but Not Sonic Hedgehog, Suggesting a Distinct Cellular Function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Photovoltaic enzymes by design and evolution\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Regulation of eukaryotic mRNA deadenylation and degradation by the Ccr4-Not complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"IFT-A structure reveals carriages for membrane protein transport into cilia\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"How to convert host plants into nonhosts\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Tropomyosin 1-I/C co-ordinates kinesin-1 and dynein motors during<i>oskar</i>mRNA transport\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"A new class of biological ion-driven rotary molecular motors with 5:2 symmetry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"TranceptEVE: Combining Family-specific and Family-agnostic Models of Protein Sequences for Improved Fitness Prediction\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Pitfalls of machine learning models for protein-protein interactions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Redesigning Enzymes for Biocatalysis: Exploiting Structural Understanding for Improved Selectivity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"A roadmap for the functional annotation of protein families: a community perspective\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 3}, {\"title\": \"Structure and function of the RAD51B\\u2013RAD51C\\u2013RAD51D\\u2013XRCC2 tumour suppressor\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mammalian fertilization: Does sperm IZUMO1 mediate fusion as well as adhesion?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"NP-hard but no longer hard to solve? Using quantum computing to tackle optimization problems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Loss-of-function, gain-of-function and dominant-negative mutations have profoundly different effects on protein structure\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Next-generation ABACUS biosensors reveal cellular ABA dynamics driving root growth at low aerial humidity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Cryo-EM structure of the Smc5/6 holo-complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Lighting up protein design\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Learning from pre-pandemic data to forecast viral escape\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure of the priming arabinosyltransferase AftA required for AG biosynthesis of <i>Mycobacterium tuberculosis</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure and function of microbial \\u03b1-<scp>l</scp>-fucosidases: a mini review\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Cryo-electron microscopy of the f1 filamentous phage reveals a new paradigm in viral infection and assembly\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structural mass spectrometry approaches to understand multidrug efflux systems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Crystal polymorphism in fragment-based lead discovery of ligands of the catalytic domain of UGGT, the glycoprotein folding quality control checkpoint\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Current structure predictors are not learning the physics of protein folding\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Investigating Native Metal Ion Binding Sites in Mammalian Histidine-Rich Glycoprotein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Functional unknomics: Systematic screening of conserved genes of unknown function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure of the Saccharolobus solfataricus type III-D CRISPR effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure of the transmembrane protein 2 (TMEM2) ectodomain and its apparent lack of hyaluronidase activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Intracellular<i>Plasmodium</i>aquaporin 2 is required for sporozoite production in the mosquito vector and malaria transmission\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structure of the transmembrane protein 2 (TMEM2) ectodomain and its lack of hyaluronidase activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Molecular Epidemiology of AY.28 and AY.104 Delta Sub-lineages in Sri Lanka\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Molecular architecture of the autoinhibited kinesin-1 lambda particle\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Modeling SARS\\u2010CoV\\u20102 proteins in the CASP\\u2010commons experiment\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"ChatGPT in ophthalmology: the dawn of a new era?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"TMEM161B regulates cerebral cortical gyration, Sonic Hedgehog signaling, and ciliary structure in the developing central nervous system\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Multi-omics identify falling LRRC15 as a COVID-19 severity marker and persistent pro-thrombotic signals in convalescence\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Meeting report \\u2013 Cell dynamics: host\\u2013pathogen interface\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanoenzymatic reactions for the hydrolysis of PET\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Comprehensive approach to study branched ubiquitin chains reveals roles for K48-K63 branches in VCP/p97-related processes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Systematic analysis of membrane contact sites in <i>Saccharomyces cerevisiae</i> uncovers modulators of cellular lipid distribution\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanistic basis for potent neutralization of Sin Nombre hantavirus by a human monoclonal antibody\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Multiple interactions of the dynein-2 complex with the IFT-B complex are required for effective intraflagellar transport\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanisms of <i>E. coli</i> chemotaxis signaling pathways visualized using cryoET and computational approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Mechanism of human Lig1 regulation by PCNA in Okazaki fragment sealing\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Review of Recurrently Mutated Genes in Craniosynostosis Supports Expansion of Diagnostic Gene Panels\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Considerations for defining +80 Da mass shifts in mass spectrometry-based proteomics: phosphorylation and beyond\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Darobactin B Stabilises a Lateral\\u2010Closed Conformation of the BAM Complex in <i>E. coli</i> Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Growth in a biofilm promotes conjugation of a <i>bla</i> <sub>NDM-1</sub> -bearing plasmid between <i>Klebsiella pneumoniae</i> strains\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Single-particle cryo-EM analysis of the shell architecture and internal organization of an intact \\u03b1-carboxysome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"EMBL\\u2019s European Bioinformatics Institute (EMBL-EBI) in 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"How Pyroptosis Contributes to Inflammation and Fibroblast-Macrophage Cross-Talk in Rheumatoid Arthritis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"High-throughput process development from gene cloning to protein production\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Efficient NADPH-dependent dehalogenation afforded by a self-sufficient reductive dehalogenase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"GWYRE: A Resource for Mapping Variants onto Experimental and Modeled Structures of Human Protein Complexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Germline pathogenic variants in HNRNPU are associated with alterations in blood methylome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Role of NS2 specific RNA binding and phosphorylation in liquid\\u2013liquid phase separation and virus assembly\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Single-particle cryo-EM analysis of the shell architecture and internal organization of an intact \\u03b1-carboxysome\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Herpes simplex virus 1 protein pUL21 alters ceramide metabolism by activating the interorganelle transport protein CERT\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Energy landscape steering in SecYEG mediates dynamic coupling in ATP driven protein translocation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"HOIL\\u20101 ubiquitin ligase activity targets unbranched glucosaccharides and is required to prevent polyglucosan accumulation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Sensor NLR immune proteins activate oligomerization of their NRC helper\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Enhanced statistical sampling reveals microscopic complexity in the talin mechanosensor folding energy landscape\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Growth in a biofilm promotes conjugation of a<i>bla</i><sub>NDM-1</sub>-bearing plasmid between<i>Klebsiella pneumoniae</i>strains\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Enzyme adaptation to habitat thermal legacy shapes the thermal plasticity of marine microbiomes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Dynamics and Molecular Interactions of GPI-Anchored CD59\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Reconstitution of an active human CENP-E motor\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Evolutionary Multiobjective Clustering Over Multiple Conflicting Data Views\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Phase separation and molecular ordering of the prion-like domain of the Arabidopsis thermosensory protein EARLY FLOWERING 3\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Incomplete Penetrance and Variable Expressivity: From Clinical Studies to Population Cohorts\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Deep mutagenesis scanning using whole trimeric SARS-CoV-2 spike highlights the importance of NTD-RBD interactions in determining spike phenotype\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Improving the predictive power of xenograft and syngeneic anti-tumour studies using mice humanised for pathways of drug metabolism\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Deregulations of RNA Pol II Subunits in Cancer\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Potent cross-reactive antibodies following Omicron breakthrough in vaccinees\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Precisely patterned nanofibers made from extendable protein multiplexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Development of Enterovirus Antiviral Agents That Target the Viral 2C Protein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structural and functional properties of the transporter SLC26A6 reveal mechanism of coupled anion exchange\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Dimers of D76N-\\u03b22-microglobulin display potent antiamyloid aggregation activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Prediction of protein structures, functions and interactions using the IntFOLD7, MultiFOLD and ModFOLDdock servers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structural and functional properties of the transporter SLC26A6 reveal mechanism of coupled anion exchange\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Discovery of small molecule ligands for the von Hippel-Lindau (VHL) E3 ligase and their use as inhibitors and PROTAC degraders\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Structural and biochemical basis of interdependent <scp>FANCI\\u2010FANCD2</scp> ubiquitination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Pro-phagocytic function and structural basis of GPR84 signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"10q26 \\u2013 The enigma in age-related macular degeneration\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Modelling the viral dynamics of the SARS-CoV-2 Delta and Omicron variants in different cell types\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The metabolic control of DNA replication: mechanism and function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Broader functions of TIR domains in Arabidopsis immunity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"CHC22 clathrin membrane recruitment uses SNX5 in bipartite interaction with secretory tether p115\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"AF2-multimer guided high accuracy prediction of typical and atypical ATG8 binding motifs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"<i>N</i>-Methyl deuterated rhodamines for protein labelling in sensitive fluorescence microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The accuracy of protein structures in solution determined by AlphaFold and NMR\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"The mystery of massive mitochondrial complexes: the apicomplexan respiratory chain\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Unifying the known and unknown microbial coding sequence space\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Absolute quantification of cellular levels of photosynthesis-related proteins in Synechocystis sp. PCC 6803\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"AlphaFold predicted structure of the Hsp90-like domains of the neurodegeneration linked protein sacsin reveals key residues for ATPase activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The N-terminal executioner domains of NLR immune receptors are functionally conserved across major plant lineages\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Versatile Product Detection via Coupled Assays for Ultrahigh-Throughput Screening of Carbohydrate-Active Enzymes in Microfluidic Droplets\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"<i>arfA</i>antisense RNA regulates MscL excretory activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Altered plasma membrane abundance of the sulfatide-binding protein NF155 links glycosphingolipid imbalances to demyelination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A choline-releasing glycerophosphodiesterase essential for phosphatidylcholine biosynthesis and blood stage development in the malaria parasite\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Accurate prediction of protein structures and interactions using a three-track neural network\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Beyond transcription: compelling open questions in plant RNA biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A guide to UFMylation, an emerging posttranslational modification\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Two Years into the COVID-19 Pandemic: Lessons Learned\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The road to fully programmable protein catalysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The ROK kinase N-acetylglucosamine kinase uses a sequential random enzyme mechanism with successive conformational changes upon each substrate binding\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Advancing pharmacy and healthcare with virtual digital technologies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The Role of Cytochrome P450 AbyV in the Final Stages of Abyssomicin C Biosynthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"The SPARC complex defines RNAPII promoters in Trypanosoma brucei\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"<scp>FBXL4</scp> suppresses mitophagy by restricting the accumulation of <scp>NIX</scp> and <scp>BNIP3</scp> mitophagy receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Altered plasma membrane abundance of the sulfatide-binding protein NF155 links glycosphingolipid imbalances to demyelination\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"Affimers targeting proteins in the cardiomyocyte Z-disc: Novel tools that improve imaging of heart tissue\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The Gene Ontology knowledgebase in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A Vitamin B<sub>2</sub>\\u2010Photocatalysed Approach to Methionine Analogues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"\\u03b2-1,4-Xylan backbone synthesis in higher plants: How complex can it be?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"3DLigandSite: structure-based prediction of protein\\u2013ligand binding sites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Thermophilic Carboxylesterases from Hydrothermal Vents of the Volcanic Island of Ischia Active on Synthetic and Biobased Polymers and Mycotoxins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The fluorescent protein iLOV as a reporter for screening of high\\u2010yield production of antimicrobial peptides in <i>Pichia pastoris</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Antibody escape of SARS-CoV-2 Omicron BA.4 and BA.5 from vaccine and BA.1 serum\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Understanding the Stabilizing Effect of Histidine on mAb Aggregation: A Molecular Dynamics Study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Understanding molecular mechanisms and predicting phenotypic effects of pathogenic tubulin mutations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The mechanical cell \\u2013 the role of force dependencies in synchronising protein interaction networks\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The crystal structure of vaccinia virus protein E2 and perspectives on the prediction of novel viral protein folds\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 2}, {\"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Thrombin cleavage of the hepatitis E virus polyprotein at multiple conserved locations is required for genome replication\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The crystal structure of vaccinia virus protein E2 and perspectives on the prediction of novel viral protein folds\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A Novel Homozygous Founder Variant of RTN4IP1 in Two Consanguineous Saudi Families\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The Evolution and Biology of SARS-CoV-2 Variants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The meiotic LINC complex component KASH5 is an activating adaptor for cytoplasmic dynein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"The cell biology of archaea\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"CSM-Potential: mapping protein interactions and biological ligands in 3D space using geometric deep learning\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 2}, {\"title\": \"Plasmids manipulate bacterial behaviour through translational regulatory crosstalk\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Potential Role of the Host-Derived Cell-Wall Binding Domain of Endolysin CD16/50L as a Molecular Anchor in Preservation of Uninfected Clostridioides difficile for New Rounds of Phage Infection\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plant vernalization proteins contain unusual PHD superdomains without histone H3 binding activity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"UVC inactivation of pathogenic samples suitable for cryo-EM analysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Phenome-wide association study of loci harboring de novo tandem repeat mutations in UK Biobank exomes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plant proteostasis: a proven and promising target for crop improvement\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Photopharmacology of Ion Channels through the Light of the Computational Microscope\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Precisely patterned nanofibers made from extendable protein multiplexes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Ultrahigh-Throughput Directed Evolution of a Metal-Free \\u03b1/\\u03b2-Hydrolase with a Cys-His-Asp Triad into an Efficient Phosphotriesterase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Pre-RNA splicing in metabolic homeostasis and liver disease\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Prediction of protein structures, functions and interactions using the IntFOLD7, MultiFOLD and ModFOLDdock servers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plant immune receptor pathways as a united front against pathogens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pollen Coat Proteomes of Arabidopsis thaliana, Arabidopsis lyrata, and Brassica oleracea Reveal Remarkable Diversity of Small Cysteine-Rich Proteins at the Pollen-Stigma Interface\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Understanding a protein fold: The physics, chemistry, and biology of \\u03b1-helical coiled coils\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Post-transcriptional control of fungal cell wall synthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Polyvalent Nano-Lectin Potently Neutralizes SARS-CoV-2 by Targeting Glycans on the Viral Spike Protein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Poly(vinyl alcohol) Molecular Bottlebrushes Nucleate Ice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Phylogenomic and functional characterization of an evolutionary conserved cytochrome P450-based insecticide detoxification mechanism in bees\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Unconventional secretion of \\u03b1-synuclein mediated by palmitoylated DNAJC5 oligomers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plants to mine metals and remediate land\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Plasma FIB milling for the determination of structures in situ\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Poly ADP-Ribose Signaling is Dysregulated in Huntington\\u2019s Disease Patients\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ultrahigh-Throughput Directed Evolution of a Metal-Free \\u03b1/\\u03b2-Hydrolase with a Cys-His-Asp Triad into an Efficient Phosphotriesterase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Arabidopsis <scp> <i>WRR4A</i> </scp> and <scp> <i>WRR4B</i> </scp> paralogous <scp>NLR</scp> proteins both confer recognition of multiple <i>Albugo candida</i> effectors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Truncating and zinc\\u2010finger variants in <scp><i>GLI2</i></scp> are associated with hypopituitarism\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"UMAD1 contributes to ESCRT-III dynamic subunit turnover during cytokinetic abscission\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Towards Reverse Vaccinology for Bovine TB: High Throughput Expression of Full Length Recombinant Mycobacterium bovis Proteins\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Protein target highlights in <scp>CASP15</scp>: Analysis of models by structure providers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Total In Vitro Biosynthesis of the Thioamitide Thioholgamide and Investigation of the Pathway\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Proteinoid Microspheres as Protoneural Networks\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Proteomic repository data submission, dissemination, and reuse: key messages\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"To split or not to split: <scp>CASP15</scp> targets and their processing into tertiary structure evaluation units\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Time resolved applications for Cryo-EM; approaches, challenges and future directions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Proteomics Standards Initiative at Twenty Years: Current Activities and Future Work\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Providing open imaging data at scale: An EMBL-EBI perspective\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Providing open imaging data at scale: An EMBL-EBI perspective\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Thioalbamide inhibits FoF1-ATPase in breast cancer cells and reduces tumor proliferation and invasiveness in breast cancer in vivo models\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pseudotyped Bat Coronavirus RaTG13 is efficiently neutralised by convalescent sera from SARS-CoV-2 infected patients\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pulse dipolar EPR for determining nanomolar binding affinities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Purriato is a conserved small open reading frame gene that interacts with the CASA pathway to regulate muscle homeostasis and epithelial tissue growth in Drosophila\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Quality by Design for enabling RNA platform production processes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Quantifying constraint in the human mitochondrial genome\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Quantifying constraint in the human mitochondrial genome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Quantum chemical elucidation of a sevenfold symmetric bacterial antenna complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"RLOps: Development Life-Cycle of Reinforcement Learning Aided Open RAN\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"RNA-viromics reveals diverse communities of soil RNA viruses with the potential to affect grassland ecosystems across multiple trophic levels\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"ROS production and signalling in chloroplasts: cornerstones and evolving concepts\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Radiometal chelators for infection diagnostics\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Toward Glycomaterials with Selectivity as Well as Affinity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Protein structure-based evaluation of missense variants: Resources, challenges and future directions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Protein metalation in a nutshell\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Predictive Engineering of Class I Terpene Synthases Using Experimental and Computational Approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Primase-polymerases: how to make a primer from scratch\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"UMAD1 contributes to ESCRT-III dynamic subunit turnover during cytokinetic abscission\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Two Distinct Thermodynamic Gradients for Cellular Metalation of Vitamin B<sub>12</sub>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Understanding the evolution of immune genes in jawed vertebrates\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Tropomyosin 1-I/C co-ordinates kinesin-1 and dynein motors during<i>oskar</i>mRNA transport\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Transmission of B.1.617.2 Delta variant between vaccinated healthcare workers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"ProSight Native: Defining Protein Complex Composition from Native Top-Down Mass Spectrometry Data\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Probabilistic machine learning based soft-sensors for product quality prediction in batch processes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Probing heavy metal binding to phycobiliproteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Translational and Posttranslational Dynamics in a Model Peptidergic System\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Probing the Internal pH and Permeability of a Carboxysome Shell\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Processing of NODULE INCEPTION controls the transition to nitrogen fixation in root nodules\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Protection against SARS-CoV-2 Omicron BA.4/5 variant following booster vaccination or breakthrough infection in the UK\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protective human IgE responses are promoted by comparable life-cycle dependent Tegument Allergen-Like expression in Schistosoma haematobium and Schistosoma mansoni infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protective human IgE responses are promoted by comparable life-cycle dependent Tegument Allergen-Like protein expression in<i>Schistosoma haematobium</i>and<i>Schistosoma mansoni</i>infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protein Data Bank: A Comprehensive Review of 3D Structure Holdings and Worldwide Utilization by Researchers, Educators, and Students\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Transcriptomic plasticity of the hypothalamic osmoregulatory control centre of the Arabian dromedary camel\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Towards Reverse Vaccinology for Bovine TB: High Throughput Expression of Full Length Recombinant Mycobacterium bovis Proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protein disulfide isomerases \\u2013 a way to tackle malaria\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Protein diversification through post-translational modifications, alternative splicing, and gene duplication\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Phasor Analysis Reveals Multicomponent Fluorescence Kinetics in the LH2 Complex from <i>Marichromatium purpuratum</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Understudied proteins: opportunities and challenges for functional proteomics\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Pharmacogenomics: current status and future perspectives\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mutations in TIC100 impair and repair chloroplast protein import and impact retrograde signalling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mutations in the chloroplast inner envelope protein TIC100 impair and repair chloroplast protein import and impact retrograde signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"NANOG is required to establish the competence for germ-layer differentiation in the basal tetrapod axolotl\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Visualising the cytoskeletal machinery in neuronal growth cones using cryo-electron tomography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"NIH4215: A mutation-prone thiamine auxotrophic clinical Aspergillus fumigatus isolate\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"NLR immune receptor\\u2013nanobody fusions confer plant disease resistance\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"NLRP3 and pyroptosis blockers for treating inflammatory diseases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"NMDA Receptor Antagonists Increase the Release of GLP-1 From Gut Endocrine Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Virology under the Microscope\\u2014a Call for Rational Discourse\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Viral Load in COVID-19 Patients: Implications for Prognosis and Vaccine Efficacy in the Context of Emerging SARS-CoV-2 Variants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Native Mass Spectrometry of Complexes Formed by Molecular Glues Reveals Stoichiometric Rearrangement of E3 Ligases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Na\\u00efve Human Macrophages Are Refractory to SARS-CoV-2 Infection and Exhibit a Modest Inflammatory Response Early in Infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Near atmospheric carbon dioxide activates plant ubiquitin cross-linking\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Neutralisation sensitivity of the SARS-CoV-2 omicron (B.1.1.529) variant: a cross-sectional study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Neutralizing antibody activity against 21 SARS-CoV-2 variants in older adults vaccinated with BNT162b2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"New CRISPR Tools to Correct Pathogenic Mutations in Usher Syndrome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"New microbiological insights from the Bowland shale highlight heterogeneity of the hydraulically fractured shale microbiome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Versatile Chemo-Biocatalytic Cascade Driven by a Thermophilic and Irreversible C\\u2013C Bond-Forming \\u03b1-Oxoamine Synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Next-generation protein-based materials capture and preserve projectiles from supersonic impacts\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Non canonical scaffold-type ligase complex mediates protein UFMylation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Non-canonical <i>Staphylococcus aureus</i> pathogenicity island repression\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"When is a hydrophobic gate not a hydrophobic gate?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mutational analysis of Aedes aegypti Dicer 2 provides insights into the biogenesis of antiviral exogenous small interfering RNAs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Perspective: Structure determination of protein-ligand complexes at room temperature using X-ray diffraction approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mutational analysis of Aedes aegypti Dicer 2 provides insights into the biogenesis of antiviral exogenous small interfering RNAs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Molecular and in vivo phenotyping of missense variants of the human glucagon receptor\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"pdCSM-GPCR: predicting potent GPCR ligands with graph-based signatures\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"mRNA vaccination drives differential mucosal neutralizing antibody profiles in na\\u00efve and SARS-CoV-2 previously-infected individuals\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"mPPases create a conserved anionic membrane fingerprint as identified via multi-scale simulations\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Molecular basis for the regulation of human glycogen synthase by phosphorylation and glucose-6-phosphate\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Molecular basis for the reversible ADP-ribosylation of guanosine bases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Monoallelic variants resulting in substitutions of MAB21L1 Arg51 Cause Aniridia and microphthalmia\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Moonlighting proteins activate transformation in multidrug-resistant Streptococcus pneumoniae epigenetic phase variants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Morphology exploration of pollen using deep learning latent space\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mucin networks: Dynamic structural assemblies controlling mucus function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"mNG-tagged fusion proteins and nanobodies to visualize tropomyosins in yeast and mammalian cells\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Multi-organ functions of yolk sac during human early development\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"basicsynbio and the BASIC SEVA collection: software and vectors for an established DNA assembly method\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Zebrafish as a model for cardiac disease; Cryo-EM structure of native cardiac thin filaments from Danio Rerio\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"X-ray structure of the metastable SEPT14\\u2013SEPT7 coiled coil reveals a hendecad region crucial for heterodimerization\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Multi-trait and cross-population genome-wide association studies across autoimmune and allergic diseases identify shared and distinct genetic component\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Women in the European Virus Bioinformatics Center\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Multi\\u2010Stimuli Responsive and Mechano\\u2010Actuated Biomimetic Membrane Nanopores Self\\u2010Assembled from DNA\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mutation of the ALS/FTD-associated RNA-binding protein FUS alters axonal cytoskeletal organisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Why should we compare morphological and molecular disparity?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Non-generalizability of biomarkers for mortality in SARS-CoV-2: a meta-analyses series\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Non-glycosylated IGF2 prohormones are more mitogenic than native IGF2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Venus: Elucidating the Impact of Amino Acid Variants on Protein Function Beyond Structure Destabilisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Venus: Elucidating the Impact of Amino Acid Variants on Protein Function Beyond Structure Destabilisation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Out of the <scp>ESCPE</scp> room: Emerging roles of endosomal <scp>SNX\\u2010BARs</scp> in receptor transport and host\\u2013pathogen interaction\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Unifying the known and unknown microbial coding sequence space\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Overview of physiological, biochemical, and regulatory aspects of nitrogen fixation in <i>Azotobacter vinelandii</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"P450 gene duplication and divergence led to the evolution of dual novel functions and insecticide cross-resistance in the brown planthopper Nilaparvata lugens\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"P450 gene duplication and divergence led to the evolution of dual novel functions and insecticide cross-resistance in the brown planthopper Nilaparvata lugens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PDBe-KB: collaboratively defining the biological context of structural data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PDBench: evaluating computational methods for protein-sequence design\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PDBx/mmCIF Ecosystem: Foundational Semantic Tools for Structural Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PI(3,5)P2 and NAADP: Team players or lone warriors? \\u2013 New insights into TPC activation modes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PINK1-Dependent Mitophagy Inhibits Elevated Ubiquitin Phosphorylation Caused by Mitochondrial Damage\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"PK1 from <i>Drosophila</i> obscurin is an inactive pseudokinase with scaffolding properties\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PKC isoforms activate LRRK1 kinase by phosphorylating conserved residues (Ser1064, Ser1074 and Thr1075) within the COR<sub>B</sub> GTPase domain\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Real space in cryo-EM: the future is local\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PTPRD and DCC Are Novel BACE1 Substrates Differentially Expressed in Alzheimer\\u2019s Disease: A Data Mining and Bioinformatics Study\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"PTPRD and DCC Are Novel BACE1 Substrates Differentially Expressed in Alzheimer\\u2019s Disease: A Data Mining and Bioinformatics Study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Parallel recruitment pathways contribute to synaptonemal complex assembly during mammalian meiosis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pathogen protein modularity enables elaborate mimicry of a host phosphatase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Pathogen protein modularity enables elaborate mimicry of a host phosphatase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Pathogenicity and virulence of <i>Clostridioides difficile</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Organelle Membrane Extensions in Mammalian Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Optimizing frequency sampling in CEST experiments\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Unravelling mechanisms that govern meiotic crossover formation in wheat\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Updated protein domain annotation of the PARP protein family sheds new light on biological function\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Novel variants provide differential stabilisation of human equilibrative nucleoside transporter 1 states\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Nuclear Shape-Shifters: Lipid and Protein Dynamics at the Nuclear Envelope\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Using deep-learning predictions of inter-residue distances for model validation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Nutrient-Response Pathways in Healthspan and Lifespan Regulation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"O-Linked Sialoglycans Modulate the Proteolysis of SARS-CoV-2 Spike and Likely Contribute to the Mutational Trajectory in Variants of Concern\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"OAF: a new member of the BRICHOS family\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Using canavanine resistance to measure mutation rates in Schizosaccharomyces pombe\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Obscurin Rho GEF domains are phosphorylated by MST-family kinases but do not exhibit nucleotide exchange factor activity towards Rho GTPases in vitro\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Obscurin Rho GEF domains are phosphorylated by MST-family kinases but do not exhibit nucleotide exchange factor activity towards Rho GTPases in vitro\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Optimized expression and purification of adipose triglyceride lipase improved hydrolytic and transacylation activities in vitro\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Obtaining high yield recombinant Enterococcus faecium nicotinate nucleotide adenylyltransferase for X-ray crystallography and biophysical studies\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Obtaining high yield recombinant Enterococcus faecium nicotinate nucleotide adenylyltransferase for X-ray crystallography and biophysical studies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Omicron-BA.1 Dispersion Rates in Mexico Varied According to the Regional Epidemic Patterns and the Diversity of Local Delta Subvariants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Unveiling the Essential Role of Arkadia\\u2019s Non-RING Elements in the Ubiquitination Process\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"On-demand cell-autonomous gene therapy for brain circuit disorders\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Open Force Field BespokeFit: Automating Bespoke Torsion Parametrization at Scale\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Opinion: more mouse models and more translation needed for ALS\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Rational Design of Phosphorylation-Responsive Coiled Coil-Peptide Assemblies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Regulation of eukaryotic mRNA deadenylation and degradation by the Ccr4-Not complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Recent insights into SARS\\u2010CoV\\u20102 omicron variant\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The ROK kinase N-acetylglucosamine kinase uses a sequential random enzyme mechanism with successive conformational changes upon each substrate binding\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural landscapes of PPI interfaces\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural mass spectrometry approaches to understand multidrug efflux systems\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The SPARC complex defines RNAPII promoters in Trypanosoma brucei\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure Determination of Microtubules and Pili: Past, Present, and Future Directions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The SHDRA syndrome-associated gene <i>TMEM260</i> encodes a protein-specific O-mannosyltransferase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure and assembly of cargo Rubisco in two native \\u03b1-carboxysomes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure and function of the RAD51B\\u2013RAD51C\\u2013RAD51D\\u2013XRCC2 tumour suppressor\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure and functional mapping of the KRAB-KAP1 repressor complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure and functional mapping of the KRAB\\u2010KAP1 repressor complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Physical Basis for pH Sensitivity in Biomolecular Structure and Function, With Application to the Spike Protein of SARS-CoV-2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure and mechanism of the type I-G CRISPR effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The PCDDB (Protein Circular Dichroism Data Bank): A Bioinformatics Resource for Protein Characterisations and Methods Development\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Novel Application of Geometric Morphometrics with Principal Component Analysis to Existing G Protein-Coupled Receptor (GPCR) Structures\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure determination and analysis of titin A-band fibronectin type III domains provides insights for disease-linked variants and protein oligomerisation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure of the Commander endosomal trafficking complex linked to X-linked intellectual disability/Ritscher-Schinzel syndrome\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Novel Application of Geometric Morphometrics with Principal Component Analysis to Existing G Protein-Coupled Receptor (GPCR) Structures\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure of the Saccharolobus solfataricus type III-D CRISPR effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural landscapes of PPI interfaces\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural insights into the mechanism of adaptive ribosomal modification by <i>Pseudomonas</i><scp>RimK</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure of the malaria vaccine candidate Pfs48/45 and its recognition by transmission blocking antibodies\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Yeast Protein Kinase Sch9 Functions as a Central Nutrient-Responsive Hub That Calibrates Metabolic and Stress-Related Responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The archetypal gene transfer agent RcGTA is regulated via direct interaction with the enigmatic RNA polymerase omega subunit\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for membrane attack complex inhibition by CD59\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for proton coupled cystine transport by cystinosin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of Qng1-mediated salvage of the micronutrient queuine from queuosine-5\\u2032-monophosphate as the biological substrate\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The antibody response to SARS-CoV-2 Beta underscores the antigenic distance to other variants\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of human LRRK2 membrane recruitment and activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of human LRRK2 membrane recruitment and activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis of lipopolysaccharide maturation by the O-antigen ligase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of lipoprotein recognition by the bacterial Lol trafficking chaperone LolA\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis of purine nucleotide inhibition of human uncoupling protein 1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis of purine nucleotide inhibition of human uncoupling protein 1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis of rotavirus RNA chaperone displacement and RNA annealing\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Yeast Protein Kinase Sch9 Functions as a Central Nutrient-Responsive Hub That Calibrates Metabolic and Stress-Related Responses\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural determinants of DNA recognition by the NO sensor NsrR and related Rrf2-type [FeS]-transcription factors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural diversity and modularity of photosynthetic RC\\u2212LH1 complexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Transporter-Mediated Cellular Uptake and Efflux of Pharmaceutical Drugs and Biotechnology Products: How and Why Phospholipid Bilayer Transport Is Negligible in Real Biomembranes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural features of the protein kinase domain and targeted binding by small-molecule inhibitors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural insights into actin isoforms\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural insights into p300 regulation and acetylation-dependent genome organisation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The New p.<scp>F1700L LRRK2</scp> Variant Causes Parkinson's Disease by Extensively Increasing Kinase Activity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The New Era of High-Throughput Nanoelectrochemistry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Recent structural advances in bacterial chemotaxis signalling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The DNA replication initiation protein DnaD recognises a specific strand of the <i>Bacillus subtilis</i> chromosome origin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The European Bioinformatics Institute (EMBL-EBI) in 2021\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systems-wide dissection of organic acid assimilation in <i>Pseudomonas aeruginosa</i> reveals a novel path to underground metabolism\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The European Bioinformatics Institute (EMBL-EBI) in 2021\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Systems-wide dissection of organic acid assimilation in <i>Pseudomonas aeruginosa</i> reveals a novel path to underground metabolism\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"T cell responses to SARS-CoV-2 spike cross-recognize Omicron\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"TALE transcription factors: Cofactors no more\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"TIES 2.0: A Dual-Topology Open Source Relative Binding Free Energy Builder with Web Portal\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"TMEM161B regulates cerebral cortical gyration, Sonic Hedgehog signaling, and ciliary structure in the developing central nervous system\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Energy Landscape Perspective: Encoding Structure and Function for Biomolecules\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"TRAIP resolves DNA replication-transcription conflicts during the S-phase of unperturbed cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The C\\u2010terminal head domain of <i>Burkholderia pseudomallei</i> <scp>BpaC</scp> has a striking hydrophilic core with an extensive solvent network\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Teaching old drugs new tricks\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Crystal Structure of Bacillus thuringiensis Tpp80Aa1 and Its Interaction with Galactose-Containing Glycolipids\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The <i>CCP</i>4 suite: integrative software for macromolecular crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Chromatin Regulator HMGA1a Undergoes Phase Separation in the Nucleus**\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The AlphaFold Database of Protein Structures: A Biologist\\u2019s Guide\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Chromatin Regulator HMGA1a Undergoes Phase Separation in the Nucleus**\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Angiotensin AT<sub>2</sub>Receptor: From a Binding Site to a Novel Therapeutic Target\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systems-Wide Dissection of Organic Acid Assimilation in Pseudomonas aeruginosa Reveals a Novel Path To Underground Metabolism\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systematic indication extension for drugs using patient stratification insights generated by combinatorial analytics\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Substrate specificity of branched chain amino acid aminotransferases: The substitution of glycine to serine in the active site determines the substrate specificity for \\u03b1-ketoglutarate\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure of the<i>Lysinibacillus sphaericus</i>Tpp49Aa1 pesticidal protein elucidated from natural crystals using MHz-SFX\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure of trimeric pre-fusion rabies virus glycoprotein in complex with two protective antibodies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure, substrate recognition and initiation of hyaluronan synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The MeshCODE to scale\\u2014visualising synaptic binary information\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The MeshCODE to scale\\u2014visualising synaptic binary information\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structure-based design of biocatalytic panels for pharmaceuticals synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structure-function analysis of MmpL7-mediated lipid transport in mycobacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structures of nonsense-mediated mRNA decay factors UPF3B and UPF3A in complex with UPF2 reveal molecular basis for competitive binding and for neurodevelopmental disorder-causing mutation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Subcellular Transcriptomics and Proteomics: A Comparative Methods Review\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Inseparable Relationship Between Cholesterol and Hedgehog Signaling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systematic analysis of membrane contact sites in Saccharomyces cerevisiae uncovers modulators of cellular lipid distribution\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Super-resolution microscopy and studies of peroxisomes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Supercharged PGR5-dependent cyclic electron transfer compensates for mis-regulated chloroplast ATP synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Supramolecular organization and dynamics of mannosylated phosphatidylinositol lipids in the mycobacterial plasma membrane\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Surface-Functionalized Metal\\u2013Organic Frameworks for Binding Coronavirus Proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Synthetic Sansanmycin Analogues as Potent <i>Mycobacterium tuberculosis</i> Translocase I Inhibitors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The IUPHAR/BPS guide to PHARMACOLOGY in 2022: curating pharmacology for COVID-19, malaria and antibacterials\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Synthetic engineering of a new biocatalyst encapsulating [NiFe]-hydrogenases for enhanced hydrogen production\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Systematic analysis of membrane contact sites in <i>Saccharomyces cerevisiae</i> uncovers modulators of cellular lipid distribution\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The Evolution and Evolvability of Photosystem II\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis for binding of the renal carcinoma target hypoxia\\u2010inducible factor 2\\u03b1 to prolyl hydroxylase domain 2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis for bacterial energy extraction from atmospheric hydrogen\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Risk of covid-19 related deaths for SARS-CoV-2 omicron (B.1.1.529) compared with delta (B.1.617.2): retrospective cohort study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The structural basis of hyperpromiscuity in a core combinatorial network of type II toxin\\u2013antitoxin and related phage defense systems\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Root angle is controlled by<i>EGT1</i>in cereal crops employing an antigravitropic mechanism\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"S-acylation of the Wnt receptor Frizzled-5 by zDHHC5 controls its cellular localization and synaptogenic activity in the rodent hippocampus\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-COV-2 antibody responses to AZD1222 vaccination in West Africa\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The snapdragon genomes reveal the evolutionary dynamics of the<i>S</i>locus supergene\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron is an immune escape variant with an altered cell entry pathway\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron spike mediated immune escape and tropism shift\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 Omicron-B.1.1.529 leads to widespread escape from neutralizing antibody responses\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 antibody responses associate with sex, age and disease severity in previously uninfected people admitted to hospital with COVID-19: An ISARIC4C prospective study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The structural basis of the talin\\u2013KANK1 interaction that coordinates the actin and microtubule cytoskeletons at focal adhesions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 evolution influences GBP and IFITM sensitivity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 impairs interferon production via NSP2-induced repression of mRNA translation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The small peptide CEP1 and the NIN-like protein NLP1 regulate <i>NRT2.1</i> to mediate root nodule formation across nitrate concentrations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SARS-CoV-2 impairs interferon production via NSP2-induced repression of mRNA translation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The science of the host\\u2013virus network\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"SARS-CoV-2 spike N-terminal domain modulates TMPRSS2-dependent viral entry and fusogenicity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The role of single-protein elasticity in mechanobiology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The role of histone H3K36me3 writers, readers and erasers in maintaining genome stability\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"SNX27\\u2013Retromer directly binds ESCPE-1 to transfer cargo proteins during endosomal recycling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Reviving grain quality in wheat through non\\u2010destructive phenotyping techniques like hyperspectral imaging\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Reverse transcriptases prime DNA synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Molecular Determinants of Carbocation Cyclisation in Bacterial Monoterpene Synthases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Refining the domain architecture model of the replication origin firing factor Treslin/TICRR\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Reciprocal best structure hits: using AlphaFold models to discover distant homologues\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Reconstitution of an active human CENP-E motor\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Thermophilic Carboxylesterases from Hydrothermal Vents of the Volcanic Island of Ischia Active on Synthetic and Biobased Polymers and Mycotoxins\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Recovery of Lutacidiplasmatales archaeal order genomes suggests convergent evolution in Thermoplasmatota\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Redeployment of automated <i>MrBUMP</i> search-model identification for map fitting in cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Redesigning Enzymes for Biocatalysis: Exploiting Structural Understanding for Improved Selectivity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The<i>Solanum americanum</i>pangenome and effectoromics reveal new resistance genes against potato late blight\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The \\u03b2I domain promotes active \\u03b21 integrin clustering into mature adhesion sites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Rediscovering orbital mechanics with machine learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Refining the domain architecture model of the replication origin firing factor Treslin/TICRR\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Revealing druggable cryptic pockets in the Nsp1 of SARS-CoV-2 and other \\u03b2-coronaviruses by simulations and crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The value of genotype-specific reference for transcriptome analyses in barley\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The Beta secretase BACE1 drives Systemic Sclerosis fibroblasts activation through \\u03b2-catenin and Notch signalling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The transcriptional landscape of plant infection by the rice blast fungus <i>Magnaporthe oryzae</i> reveals distinct families of temporally co-regulated and structurally conserved effectors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Repeat sequences limit the effectiveness of lateral gene transfer and favored the evolution of meiotic sex in early eukaryotes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Repurposing the atypical type I-G CRISPR system for bacterial genome engineering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The structure of EXTL3 helps to explain the different roles of bi-domain exostosins in heparan sulfate synthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Retinal cadherins and the retinal cadherinopathies: Current concepts and future directions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Revealing druggable cryptic pockets in the Nsp1 of SARS-CoV-2 and other \\u03b2-coronaviruses by simulations and crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The structural basis of the talin\\u2013KANK1 interaction that coordinates the actin and microtubule cytoskeletons at focal adhesions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The role of glycoconjugates as receptors for insecticidal proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Saturation genome editing of DDX3X clarifies pathogenicity of germline and somatic variation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The role of Ca2+ signalling in the pathology of exocrine pancreas\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The increasing role of structural proteomics in cyanobacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The metabolic control of DNA replication: mechanism and function\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Spectral decomposition of atomic structures in heterogeneous cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Spectroscopic studies reveal details of substrate-induced conformational changes distant from the active site in isopenicillin N synthase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Spontaneously established syntrophic yeast communities improve bioproduction\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Stanniocalcin\\u20102 inhibits skeletal muscle growth and is upregulated in functional overload\\u2010induced hypertrophy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The limitations of the current protein classification tools in identifying lipolytic features in putative bacterial lipase sequences\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Streptomyces coelicolor macrodomain hydrolase SCO6735 cleaves thymidine-linked ADP-ribosylation of DNA\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The intricate balance between microRNA\\u2010induced mRNA decay and translational repression\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural Conservation of Insulin/IGF Signalling Axis at the Insulin Receptors Level in<i>Drosophila</i>and humans\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural and biochemical characterisation of the N\\u2010carbamoyl\\u2010\\u03b2\\u2010alanine amidohydrolase from <i>Rhizobium radiobacter</i><scp>MDC</scp> 8606\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Seipin concentrates distinct neutral lipids via interactions with their acyl chain carboxyl esters\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The host phylogeny determines viral infectivity and replication across Staphylococcus host species\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural and energetic analyses of SARS-CoV-2 N-terminal domain characterise sugar binding pockets and suggest putative impacts of variants on COVID-19 transmission\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The helper NLR immune protein NRC3 mediates the hypersensitive cell death caused by the cell-surface receptor Cf-4\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The factory, the antenna and the scaffold: the three-way interplay between the Golgi, cilium and extracellular matrix underlying tissue function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural and regulatory insights into the glideosome-associated connector from Toxoplasma gondii\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The evolution of selective autophagy as a mechanism of oxidative stress response\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Structural basis for SMAC-mediated antagonism of caspase inhibition by the giant ubiquitin ligase BIRC6\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The consequence of ATP synthase dimer angle on mitochondrial morphology studied by cryo-electron tomography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Speaking the host language: how Salmonella effector proteins manipulate the host\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Solanimycin: Biosynthesis and Distribution of a New Antifungal Antibiotic Regulated by Two Quorum-Sensing Systems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Small-molecule inhibitors of P-Rex guanine-nucleotide exchange factors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Small molecule-mediated targeting of microRNAs for drug discovery: Experiments, computational techniques, and disease implications\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Selection Analysis Identifies Clusters of Unusual Mutational Changes in Omicron Lineage BA.1 That Likely Impact Spike Function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Selective phosphorylation of threonine residues defines GPR84\\u2013arrestin interactions of biased ligands\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Self-assembled aggregates based on cationic amphiphilic peptides: structural insight\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The road to fully programmable protein catalysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The rise of the machines: A state-of-the-art technical review on process modelling and machine learning within hydrogen production with carbon capture\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Sequence Analysis and Structural Predictions of Lipid Transfer Bridges in the Repeating Beta Groove (RBG) Superfamily Reveal Past and Present Domain Variations Affecting Form, Function and Interactions of VPS13, ATG2, SHIP164, Hobbit and Tweek\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The potential role of ischaemia\\u2013reperfusion injury in chronic, relapsing diseases such as rheumatoid arthritis, Long COVID, and ME/CFS: evidence, mechanisms, and therapeutic implications\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Severity of omicron variant of concern and effectiveness of vaccine boosters against symptomatic disease in Scotland (EAVE II): a national cohort study with nested test-negative design\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Shapeshifting tau: from intrinsically disordered to paired-helical filaments\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Shuffled ATG8 interacting motifs form an ancestral bridge between UFMylation and autophagy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The phenuivirus Toscana virus makes an atypical use of vacuolar acidity to enter host cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Silk Bioconjugates: From Chemistry and Concept to Application\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Single-cell RNA sequencing reveals transcriptional changes of human choroidal and retinal pigment epithelium cells during fetal development, in healthy adult and intermediate age-related macular degeneration\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Single-cell methods in myeloproliferative neoplasms: old questions, new technologies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The phase separated CO<sub>2</sub>-fixing pyrenoid proteome determined by TurboID\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"The patterned assembly and stepwise Vps4-mediated disassembly of composite ESCRT-III polymers drives archaeal cell division\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"The origins and molecular evolution of SARS-CoV-2 lineage B.1.1.7 in the UK\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Site-specific encoding of photoactivity and photoreactivity into antibody fragments\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Structural basis for a conserved neutralization epitope on the receptor-binding domain of SARS-CoV-2\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"KinFams: De-Novo Classification of Protein Kinases Using CATH Functional Units\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Modelling the within-host spread of SARS-CoV-2 infection, and the subsequent immune response, using a hybrid, multiscale, individual-based model. <i>Part I: Macrophages</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Coiled-coil structure of meiosis protein TEX12 and conformational regulation by its C-terminal tip\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterization of a glycan-binding complex of minor pilins completes the analysis of <i>Streptococcus sanguinis</i> type 4 pili subunits\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterization of the molecular mechanisms of silicon uptake in coccolithophores\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterization of variational quantum algorithms using free fermions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterizing and explaining impact of disease-associated mutations in proteins without known structures or structural homologues\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterizing and explaining the impact of disease-associated mutations in proteins without known structures or structural homologs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"ChemGAPP; A Package for Chemical Genomics Analysis and Phenotypic Profiling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Chloroplast Proteostasis: Import, Sorting, Ubiquitination, and Proteolysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Circadian regulation of pulmonary disease: the importance of timing\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Circumventing the packaging limit of AAV-mediated gene replacement therapy for neurological disorders\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Classification of Cushing's syndrome PKAc mutants based upon their ability to bind PKI\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Clathrin-mediated Endocytosis Facilitates Internalization of<i>Magnaporthe oryzae</i>Effectors into Rice Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Coarsening dynamics can explain meiotic crossover patterning in both the presence and absence of the synaptonemal complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Combined docking and machine learning identify key molecular determinants of ligand pharmacological activity on \\u03b22 adrenoceptor\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterisation of ascocorynin biosynthesis in the purple jellydisc fungus Ascocoryne sarcoides\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Comment on: \\u201cComputer Simulations Reveal an Entirely Entropic Activation Barrier for the Chemical Step in a Designer Enzyme\\u201d\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Comparative analysis of the risks of hospitalisation and death associated with SARS-CoV-2 omicron (B.1.1.529) and delta (B.1.617.2) variants in England: a cohort study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Compensatory mutations reducing the fitness cost of plasmid carriage occur in plant rhizosphere communities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Comprehensive Collection and Prediction of ABC Transmembrane Protein Structures in the AI Era of Structural Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational Investigation of Mechanisms for pH Modulation of Human Chloride Channels\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational Investigation of Mechanisms for pH Modulation of Human Chloride Channels\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Computational Resources for Molecular Biology 2022\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational Resources for Molecular Biology 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational investigation of aphid odorant receptor structure and binding function\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational investigation of mechanisms for pH modulation of human chloride channels\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Computational investigation of mechanisms for pH modulation of human chloride channels\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Computational modelling of diatom silicic acid transporters predicts a conserved fold with implications for their function and evolution\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Characterisation of the antiviral RNA interference response to Toscana virus in sand fly cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Characterisation of HOIP RBR E3 ligase conformational dynamics using integrative modelling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Bicyclic Picomolar OGA Inhibitors Enable Chemoproteomic Mapping of Its Endogenous Post-translational Modifications\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CD4+ and CD8+ T cell and antibody correlates of protection against Delta vaccine breakthrough infection: A nested case-control study within the PITCH study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Bimodular architecture of bacterial effector SAP05 drives ubiquitin-independent targeted protein degradation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"BioWF: A Naturally\\u2010Fused, Di\\u2010Domain Biocatalyst from Biotin Biosynthesis Displays an Unexpectedly Broad Substrate Scope\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Biocatalytic Approaches to Building Blocks for Enzymatic and Chemical Glycan Synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biocatalytic Synthesis of Antiviral Nucleosides, Cyclic Dinucleotides, and Oligonucleotide Therapies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biochemical and structural basis of sialic acid utilization by gut microbes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biofilm inspired fabrication of functional bacterial cellulose through ex-situ and in-situ approaches\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Biogenesis of a bacterial metabolosome for propanediol utilization\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Booster Vaccination Against SARS-CoV-2 Induces Potent Immune Responses in People With Human Immunodeficiency Virus\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Brain monoamine vesicular transport disease caused by homozygous SLC18A2 variants: A study in 42 affected individuals\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Building block aspect ratio controls assembly, architecture, and mechanics of synthetic and natural protein networks\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"C16orf72/HAPSTR1/TAPR1 functions with BRCA1/Senataxin to modulate replication-associated R-loops and confer resistance to PARP disruption\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"CATHe: detection of remote homologues for CATH superfamilies using embeddings from protein language models\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CHC22 clathrin membrane recruitment uses SNX5 in bipartite interaction with secretory tether p115\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cefotaxime Exposure Selects Mutations within the CA-Domain of <i>envZ</i> Which Promote Antibiotic Resistance but Repress Biofilm Formation in Salmonella\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CMTr mediated 2\\u2032-<i>O</i>-ribose methylation status of cap-adjacent nucleotides across animals\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"COMBATdb: a database for the COVID-19 Multi-Omics Blood ATlas\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"COVID-19-Induced Myocarditis: Pathophysiological Roles of ACE2 and Toll-like Receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"CSM-Potential: mapping protein interactions and biological ligands in 3D space using geometric deep learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CSM\\u2010peptides: A computational approach to rapid identification of therapeutic peptides\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cancer-driving mutations are enriched in genic regions intolerant to germline variation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Carboxyl Methyltransferase Catalysed Formation of Mono\\u2010 and Dimethyl Esters under Aqueous Conditions: Application in Cascade Biocatalysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Carboxyl Methyltransferase Catalysed Formation of Mono\\u2010 and Dimethyl Esters under Aqueous Conditions: Application in Cascade Biocatalysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of Complexes III and IV\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cardiolipin, and not monolysocardiolipin, preferentially binds to the interface of complexes III and IV\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cardiomyocyte electrophysiology and its modulation: current views and future prospects\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cation ATPase (ATP4) Orthologue Replacement in the Malaria Parasite Plasmodium knowlesi Reveals Species-Specific Responses to ATP4-Targeting Drugs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Conformational changes and CO2-induced channel gating in connexin26\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Conformational restriction shapes the inhibition of a multidrug efflux adaptor protein\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Conformational restriction shapes the inhibition of a multidrug efflux adaptor protein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA methylation dynamics during germline development\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cryo\\u2010EM structure of the octameric pore of <i>Clostridium perfringens</i> \\u03b2\\u2010toxin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo\\u2010EM structure of the octameric pore of <i>Clostridium perfringens</i> \\u03b2\\u2010toxin\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Crystal structures of the human IgD Fab reveal insights into CH1 domain diversity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Current advances in biopharmaceutical informatics: guidelines, impact and challenges in the computational developability assessment of antibody therapeutics\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Current advances in biopharmaceutical informatics: guidelines, impact and challenges in the computational developability assessment of antibody therapeutics\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Current structure predictors are not learning the physics of protein folding\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cyclic ADP ribose isomers: Production, chemical structures, and immune signaling\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cyclic Nucleotide Signaling in Phage Defense and Counter-Defense\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cyclic di-AMP traps proton-coupled K+ transporters of the KUP family in an inward-occluded conformation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cyclic nucleotide-induced helical structure activates a TIR immune effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cysteine synthase: multiple structures of a key enzyme in cysteine synthesis and a potential drug target for Chagas disease and leishmaniasis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA methylation-based predictors of health: applications and statistical considerations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Conformational transitions and allosteric modulation in a heteromeric glycine receptor\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"DNA synthesis technologies to close the gene writing gap\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA-PKcs is required for cGAS/STING-dependent viral DNA sensing in human cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"DNA-templated self-assembly of bradykinin into bioactive nanofibrils\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Decoding Catalysis by Terpene Synthases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Deep Learning Concepts and Applications for Synthetic Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Deep learning in airborne particulate matter sensing: a review\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Deep learning predicts DNA methylation regulatory variants in the human brain and elucidates the genetics of psychiatric disorders\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Deep learning predicts DNA methylation regulatory variants in the human brain and elucidates the genetics of psychiatric disorders\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Deep-Learning-Assisted Focused Ion Beam Nanofabrication\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Deracemisation and stereoinversion by a nanoconfined bidirectional enzyme cascade: dual control by electrochemistry and selective metal ion activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Design and Evolution of Enhanced Peptide\\u2013Peptide Ligation for Modular Transglutaminase Assembly\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Design and Selection of Heterodimerizing Helical Hairpins for Synthetic Biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CryoEM structure of the outer membrane secretin channel pIV from the f1 filamentous bacteriophage\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"CryoEM reveals that ribosomes in microsporidian spores are locked in a dimeric hibernating state\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-electron microscopy of the f1 filamentous phage reveals a new paradigm in viral infection and assembly\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Conserved and Divergent Features of pH Sensing in Major Fungal Pathogens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Conserved recombination patterns across coronavirus subgenera\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Contrasting self-recognition rejection systems for self-incompatibility in Brassica and Papaver\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Contrasting self-recognition rejection systems for self-incompatibility in Brassica and Papaver\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Contribution of domain structure to the function of the yeast DEDD family exoribonuclease and RNase T functional homolog, Rex1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Controlled Bioactive Delivery Using Degradable Electroactive Polymers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Controlling contacts\\u2014Molecular mechanisms to regulate organelle membrane tethering\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlation between the binding affinity and the conformational entropy of nanobody SARS-CoV-2 spike protein complexes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlation energy of the paramagnetic electron gas at the thermodynamic limit\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlation energy of the spin-polarized electron liquid studied using quantum Monte Carlo simulations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Correlations between human mating partners: a comprehensive meta-analysis of 22 traits and raw data analysis of 133 traits in the UK Biobank\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Coupled Natural Fusion Enzymes in a Novel Biocatalytic Cascade Convert Fatty Acids to Amines\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Coupled Natural Fusion Enzymes in a Novel Biocatalytic Cascade Convert Fatty Acids to Amines\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Covid-19 Vaccine Effectiveness against the Omicron (B.1.1.529) Variant\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Covid-19: virology, variants, and vaccines\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Co\\u2010evolutionary adaptations of <i>Acinetobacter baumannii</i> and a clinical carbapenemase\\u2010encoding plasmid during carbapenem exposure\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Critical assessment of methods of protein structure prediction (CASP)\\u2014Round <scp>XIV</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM reconstruction of the human 40S ribosomal subunit at 2.15 \\u00c5 resolution\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM reveals the architecture of the PELP1-WDR18 molecular scaffold\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM reveals the architecture of the PELP1-WDR18 molecular scaffold\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the agonist-bound Hsp90-XAP2-AHR cytosolic complex\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the four-subunit <i>Rhodobacter sphaeroides</i> cytochrome <i>bc</i> <sub>1</sub> complex in styrene maleic acid nanodiscs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structure of the human Kv3.1 channel reveals gating control by the cytoplasmic T1 domain\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structures of human fucosidase FucA1 reveal insight into substrate recognition and catalysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Cryo-EM structures of light-harvesting 2 complexes from <i>Rhodopseudomonas palustris</i> reveal the molecular origin of absorption tuning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Bimodular architecture of bacterial effector SAP05 drives ubiquitin-independent targeted protein degradation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Biased agonism and polymorphic variation at the GLP-1 receptor: Implications for the development of personalised therapeutics\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Modelling the structure of Short Gastrulation and generation of a toolkit for studying its function in <i>Drosophila</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A feed-forward pathway drives LRRK2 kinase membrane recruitment and activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Novel and Ubiquitous Marine Methylophage Provides Insights into Viral-Host Coevolution and Possible Host-Range Expansion in Streamlined Marine Heterotrophic Bacteria\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A Nuclear Export Signal in KHNYN Required for Its Antiviral Activity Evolved as ZAP Emerged in Tetrapods\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Quantitative Description for Optical Mass Measurement of Single Biomolecules\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Unique Mode of Coenzyme A Binding to the Nucleotide Binding Pocket of Human Metastasis Suppressor NME1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A bioinformatic workflow for in silico secretome prediction with the lignocellulose degrading ascomycete fungus <i>Parascedosporium putredinis</i> NO1\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A blast fungus zinc-finger fold effector binds to a hydrophobic pocket in host Exo70 proteins to modulate immune recognition in rice\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A bovine antibody possessing an ultralong complementarity-determining region CDRH3 targets a highly conserved epitope in sarbecovirus spike proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A choline-releasing glycerophosphodiesterase essential for phosphatidylcholine biosynthesis and blood stage development in the malaria parasite\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A common TMPRSS2 variant has a protective effect against severe COVID-19\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A designed photoenzyme for enantioselective [2+2] cycloadditions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A dynamic duo\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A founder<i>UMOD</i>variant is a common cause of hereditary nephropathy in the British population\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A Feed-forward Pathway Drives LRRK2 kinase Membrane Recruitment and Activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A genome-wide association study identifies distinct variants associated with pulmonary function among European and African ancestries from the UK Biobank\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A guide to UFMylation, an emerging posttranslational modification\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A hybrid and poly-polish workflow for the complete and accurate assembly of phage genomes: a case study of ten przondoviruses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A hyperpromiscuous antitoxin protein domain for the neutralization of diverse toxin domains\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A late-stage assembly checkpoint of the human mitochondrial ribosome large subunit\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A machine learning framework for discovery and enrichment of metagenomics metadata from open access publications\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A new dawn beyond lysine ubiquitination\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A new dimethylsulfoniopropionate lyase of the cupin superfamily in marine bacteria\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A non\\u2010canonical scaffold\\u2010type <scp>E3</scp> ligase complex mediates protein <scp>UFMylation</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel fold for acyltransferase-3 (AT3) proteins provides a framework for transmembrane acyl-group transfer\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel mutant allele of <i>AtCNGC15</i> reveals a dual function of nuclear calcium release in the root meristem\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel mutant allele of <i>AtCNGC15</i> reveals a dual function of nuclear calcium release in the root meristem\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A HIF independent oxygen-sensitive pathway for controlling cholesterol synthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A Feed-forward Pathway Drives LRRK2 kinase Membrane Recruitment and Activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Biallelic variants in OGDH encoding oxoglutarate dehydrogenase lead to a neurodevelopmental disorder characterized by global developmental delay, movement disorder, and metabolic abnormalities\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>In situ</i> structure of a dimeric hibernating ribosome from a eukaryotic intracellular pathogen\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"3DLigandSite: structure-based prediction of protein\\u2013ligand binding sites\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"5-HT<sub>3</sub> Receptor MX Helix Contributes to Receptor Function\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"5-Substituted Pyridine-2,4-dicarboxylate Derivatives Have Potential for Selective Inhibition of Human Jumonji-C Domain-Containing Protein 5\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<b>Stabilisation of the RirA [4Fe-4S] cluster results in loss of iron-sensing function</b>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<b>Stabilisation of the RirA [4Fe-4S] cluster results in loss of iron-sensing function</b>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>Birnaviridae</i> Virus Factories Show Features of Liquid-Liquid Phase Separation and Are Distinct from Paracrystalline Arrays of Virions Observed by Electron Microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>Buccaneer</i> model building with neural network fragment selection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>CCP</i>4 Cloud for structure determination and project management in macromolecular crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>Dapper:</i> Deploying Service Function Chains in the Programmable Data Plane Via Deep Reinforcement Learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>GEMMI</i> and <i>Servalcat</i> restrain <i>REFMAC</i>5\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>IFT74</i>variants cause skeletal ciliopathy and motile cilia defects in mice and humans\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>In situ, in vivo</i>, and <i>in vitro</i> approaches for studying AMR plasmid conjugation in the gut microbiome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>ModelCraft</i>: an advanced automated model-building pipeline using <i>Buccaneer</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A 2.2 \\u00c5 cryoEM structure of a quinol-dependent NO Reductase shows close similarity to respiratory oxidases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>MrParse</i>: finding homologues in the PDB and the EBI AlphaFold database for molecular replacement and more\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>TLN1</i> contains a cancer-associated cassette exon that alters talin-1 mechanosensitivity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>Toxoplasma</i> ceramide synthases: a curious case of gene duplication, divergence and key functionality\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<i>UNC13A</i>in amyotrophic lateral sclerosis: from genetic association to therapeutic target\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<i>findMySequence</i>: a neural-network-based approach for identification of unknown proteins in X-ray crystallography and cryo-EM\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<scp>AAK1</scp>\\u2010like: A putative pseudokinase with potential roles in cargo uptake in bloodstream form <i>Trypanosoma brucei</i> parasites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<scp>ARFs</scp> get the <scp>BioID</scp> treatment: what have we been missing?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<scp>Cryo\\u2010EM</scp> reveals mechanisms of angiotensin I\\u2010converting enzyme allostery and dimerization\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<scp>Cryo\\u2010EM</scp> targets in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"<scp>FBXL4</scp> suppresses mitophagy by restricting the accumulation of <scp>NIX</scp> and <scp>BNIP3</scp> mitophagy receptors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<scp>PDBe</scp> and <scp>PDBe\\u2010KB</scp>: Providing high\\u2010quality, up\\u2010to\\u2010date and integrated resources of macromolecular structures to support basic and applied research and education\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"<scp>RCSB</scp> Protein Data bank: Tools for visualizing and understanding biological macromolecules in <scp>3D</scp>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel superfamily of bridge-like lipid transfer proteins\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A novel, essential<i>trans</i>-splicing protein connects the nematode SL1 snRNP to the CBC-ARS2 complex\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A paradigm shift in structural biology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Application of antibiotic-derived fluorescent probes to bacterial studies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Altered TMPRSS2 usage by SARS-CoV-2 Omicron impacts infectivity and fusogenicity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"An <i>Infectious Bursal Disease Virus</i> (IBDV) Reverse Genetics Rescue System and Neutralization Assay in Chicken B Cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"An RNA-binding switch drives ribosome biogenesis and tumorigenesis downstream of RAS oncogene\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"An allelic series of starch-branching enzyme mutants in pea (Pisum sativum L.) reveals complex relationships with seed starch phenotypes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"An amino acid transporter AAT1 plays a pivotal role in chloroquine resistance evolution in malaria parasites\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"An amino acid transporter AAT1 plays a pivotal role in chloroquine resistance evolution in malaria parasites\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"An eggshell-localised annexin plays a key role in the coordination of the life cycle of a plant-parasitic nematode with its host\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"An open invitation to the Understudied Proteins Initiative\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"An open-source molecular builder and free energy preparation workflow\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Annotation of biologically relevant ligands in UniProtKB using ChEBI\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Anomalous DNA hybridisation kinetics on gold nanorods revealed <i>via</i> a dual single-molecule imaging and optoplasmonic sensing platform\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Application of a Synthetic Ferredoxin\\u2010Inspired [4Fe4S]\\u2010Peptide Maquette as the Redox Partner for a [FeFe]\\u2010Hydrogenase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Are the integrin binding motifs within SARS CoV-2 spike protein and MHC class II alleles playing the key role in COVID-19?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A phase-separated CO2-fixing pyrenoid proteome determined by TurboID in <i>Chlamydomonas reinhardtii</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Are the integrin binding motifs within SARS CoV-2 spike protein and MHC class II alleles playing the key role in COVID-19?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ascendancy of semi-synthetic biomaterials from design towards democratization\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Assembling membraneless organelles from de novo designed proteins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Assessing PDB macromolecular crystal structure confidence at the individual amino acid residue level\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Assessing the Pathogenicity of In-Frame CACNA1F Indel Variants Using Structural Modeling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Assessing the utility of <scp>CASP14</scp> models for molecular replacement\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Asymmetric Periodic Boundary Conditions for All-Atom Molecular Dynamics and Coarse-Grained Simulations of Nucleic Acids\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Attenuation hotspots in neurotropic human astroviruses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Bacillus subtilis YtpP and Thioredoxin A Are New Players in the Coenzyme-A-Mediated Defense Mechanism against Cellular Stress\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Bacteriophages benefit from mobilizing pathogenicity islands encoding immune systems against competitors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Benchmarking strategies for cross-species integration of single-cell RNA sequencing data\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"AlphaFold2-multimer guided high-accuracy prediction of typical and atypical ATG8-binding motifs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AlphaFold-predicted protein structures and small-angle X-ray scattering: insights from an extended examination of selected data in the Small-Angle Scattering Biological Data Bank\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A potato late blight resistance gene protects against multiple Phytophthora species by recognizing a broadly conserved RXLR-WY effector\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A power-based sliding window approach to evaluate the clinical impact of rare genetic variants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A primer to directed evolution: current methodologies and future directions\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A review of molecular representation in the age of machine learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A ribosomally synthesised and post-translationally modified peptide containing a \\u03b2-enamino acid and a macrocyclic motif\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A roadmap for the functional annotation of protein families: a community perspective\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A robust deep learning platform to predict CD8+ T-cell epitopes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"A systems approach evaluating the impact of SARS-CoV-2 variant of concern mutations on CD8+ T cell responses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A versatile approach to high-density microcrystals in lipidic cubic phase for room-temperature serial crystallography\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"A versatile cortical pattern-forming circuit based on Rho, F-actin, Ect2, and RGA-3/4\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AF2-multimer guided high accuracy prediction of typical and atypical ATG8 binding motifs\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AI Benchmarking for Science: Efforts from the MLCommons Science Working Group\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"AI Benchmarking for Science: Efforts from the MLCommons Science Working Group\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Absence of Staphylococcus aureus in Wild Populations of Fish Supports a Spillover Hypothesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Accelerated waning of the humoral response to COVID-19 vaccines in obesity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Accelerated waning of the humoral response to SARS-CoV-2 vaccines in obesity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Activator-induced conformational changes regulate division-associated peptidoglycan amidases\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Advances in ionisation techniques for mass spectrometry\\u2010based omics research\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Advances in methods to analyse cardiolipin and their clinical applications\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Advancing pharmacy and healthcare with virtual digital technologies\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Affinity-enhanced RNA-binding domains as tools to understand RNA recognition\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Age-dependent topic modelling of comorbidities in UK Biobank identifies disease subtypes with differential genetic risk\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Allele-specific gene expression can underlie altered transcript abundance in zebrafish mutants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"AlphaFold predicted structure of the Hsp90-like domains of the neurodegeneration linked protein sacsin reveals key residues for ATPase activity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Design of Heme Enzymes with a Tunable Substrate Binding Pocket Adjacent to an Open Metal Coordination Site\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Detecting and quantifying clonal selection in somatic stem cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Infrared Multiphoton Dissociation Enables Top\\u2010Down Characterization of Membrane Protein Complexes and G Protein\\u2010Coupled Receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Innovation and appropriation in mycorrhizal and rhizobial Symbioses\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Inorganic Nitrogen Transport and Assimilation in Pea (Pisum sativum)\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Insertions in the SARS-CoV-2 Spike N-Terminal Domain May Aid COVID-19 Transmission\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Insertions in the SARS-CoV-2 Spike N-Terminal Domain May Aid COVID-19 Transmission\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Insights from genomic studies on the role of sex steroids in the aetiology of endometriosis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Insights into the Roles of Lipoteichoic Acids and MprF in Bacillus subtilis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Insights into the role of lipoteichoic acids and MprF function in <i>Bacillus subtilis</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Integrated View of Baseline Protein Expression in Human Tissues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Integrated view and comparative analysis of baseline protein expression in mouse and rat tissues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Integrating CRISPR/Cas systems with programmable DNA nanostructures for delivery and beyond\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Integrating artificial intelligence into an ophthalmologist\\u2019s workflow: obstacles and opportunities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Intelligent host engineering for metabolic flux optimisation in biotechnology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Inducible apelin receptor knockdown reduces differentiation efficiency and contractility of hESC-derived cardiomyocytes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Intelligent resolution: Integrating Cryo-EM with AI-driven multi-resolution simulations to observe the severe acute respiratory syndrome coronavirus-2 replication-transcription machinery in action\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intelligent resolution: Integrating Cryo-EM with AI-driven multi-resolution simulations to observe the severe acute respiratory syndrome coronavirus-2 replication-transcription machinery in action\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"InterPro in 2022\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"InterPro in 2022\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Interaction of the periplasmic chaperone SurA with the inner membrane protein secretion (SEC) machinery\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Interrogation of an Enzyme Library Reveals the Catalytic Plasticity of Naturally Evolved [4+2] Cyclases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intracellular FUS protein accumulation leads to cytoskeletal, organelle and cellular homeostasis perturbations\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Intracellular<i>Plasmodium</i>aquaporin 2 is required for sporozoite production in the mosquito vector and malaria transmission\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Intrinsically disordered proteins: modes of binding with emphasis on disordered domains\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Introduction: the secret lives of microbial mobile genetic elements\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Investigating Native Metal Ion Binding Sites in Mammalian Histidine-Rich Glycoprotein\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Inferring disease architecture and predictive ability with LDpred2-auto\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Increased interaction between endoplasmic reticulum and mitochondria following sleep deprivation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Developing novel antifungals: lessons from G protein-coupled receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Identification and Validation of Toxoplasma gondii Mitoribosomal Large Subunit Components\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"How to convert host plants into nonhosts\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Human UFSP1 is an active protease that regulates UFM1 maturation and UFMylation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Human leukocyte antigen alleles associate with COVID-19 vaccine immunogenicity and risk of breakthrough infection\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Human leukocyte antigen alleles associate with COVID-19 vaccine immunogenicity and risk of breakthrough infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Human vascularised synovium-on-a-chip: a mechanically stimulated, microfluidic model to investigate synovial inflammation and monocyte recruitment\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Hybridisation has shaped a recent radiation of grass-feeding aphids\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Hypothalamic PVN CRH neurons signal through PVN GABA neurons to suppress GnRH pulse generator frequency in female mice\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Hypoxia inducible factors regulate infectious SARS-CoV-2, epithelial damage and respiratory symptoms in a hamster COVID-19 model\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"IFT-A structure reveals carriages for membrane protein transport into cilia\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"INTERACTION OF THE PERIPLASMIC CHAPERONE SURA WITH THE INNER MEMBRANE PROTEIN SECRETION (SEC) MACHINERY\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"IP3 receptors: An \\u201celementary\\u201d journey from structure to signals\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"ISG15 driven cellular responses to virus infection\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Identification of Unusual Oxysterols Biosynthesised in Human Pregnancy by Charge-Tagging and Liquid Chromatography - Mass Spectrometry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Identification of the target and mode of action for the prokaryotic nucleotide excision repair inhibitor ATBC\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Identification of unusual oxysterols biosynthesised in human pregnancy by charge-tagging and liquid chromatography - mass spectrometry\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Immune-mediated inflammation across disease boundaries: breaking down research silos\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Immunoglobulin germline gene polymorphisms influence the function of SARS-CoV-2 neutralizing antibodies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Impact of Type II LRRK2 inhibitors on signaling and mitophagy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Implicating genes, pleiotropy, and sexual dimorphism at blood lipid loci through multi-ancestry meta-analysis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Improved methodology for protein NMR structure calculation using hydrogen bond restraints and ANSURR validation: The SH2 domain of SH2B1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Improvement of protein tertiary and quaternary structure predictions using the ReFOLD refinement method and the AlphaFold2 recycling process\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Improving carbon monoxide tolerance of Cupriavidus necator H16 through adaptive laboratory evolution\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Improving helminth genome resources in the post-genomic era\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Improving helminth genome resources in the post-genomic era\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Improving photosynthesis through multidisciplinary efforts: The next frontier of photosynthesis research\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Investigating the Structure\\u2013Activity Relationship of 1,2,4-Triazine G-Protein-Coupled Receptor 84 (GPR84) Antagonists\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Investigating the potential for a limited quantum speedup on protein lattice problems\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Manipulation of plant immunity via an <scp>mRNA</scp> decapping pathogen effector\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mapping HDX-MS Data to Protein Conformations through Training Ensemble-Based Models\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mapping diversity in African trypanosomes using high resolution spatial proteomics\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mapping diversity in African trypanosomes using high resolution spatial proteomics\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mapping of a N-terminal \\u03b1-helix domain required for human PINK1 stabilization, Serine228 autophosphorylation and activation in cells\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mass spectrometry\\u2010based high\\u2010throughput proteomics and its role in biomedical studies and systems biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mating pair stabilization mediates bacterial conjugation species specificity\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanism of glycogen synthase inactivation and interaction with glycogenin\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanism of human Lig1 regulation by PCNA in Okazaki fragment sealing\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanism-Based Redesign of GAP to Activate Oncogenic Ras\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mechanisms of <i>E. coli</i> chemotaxis signaling pathways visualized using cryoET and computational approaches\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms of DNA opening revealed in AAA+ transcription complex structures\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms of prezygotic post-pollination reproductive barriers in plants\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Ion Mobility Mass Spectrometry (IM-MS) for Structural Biology: Insights Gained by Measuring Mass, Charge, and Collision Cross Section\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Mechanisms underlying TARP modulation of the GluA1/2-\\u03b38 AMPA receptor\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Meiosis in Plasmodium: how does it work?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Meiosis in Plasmodium: how does it work?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Meiosis in allopolyploid <i>Arabidopsis suecica</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Metabolic Control Analysis of triacylglycerol accumulation in oilseed rape\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Metabolic efficiency reshapes the seminal relationship between pathogen growth rate and virulence\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Methionine Sulfoxide Reductases Suppress the Formation of the [PSI+] Prion and Protein Aggregation in Yeast\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Mining the human gut microbiome identifies mycobacterial d-arabinan degrading enzymes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Missense3D-PPI: A Web Resource to Predict the Impact of Missense Variants at Protein Interfaces Using 3D Structural Data\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"ModelCIF: An Extension of PDBx/mmCIF Data Representation for Computed Structure Models\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Modelling Membrane Reshaping by Staged Polymerization of ESCRT-III Filaments\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Modelling the structure of Short Gastrulation and generation of a toolkit for studying its function in <i>Drosophila</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Macromolecular crowding is surprisingly unable to deform the structure of a model biomolecular condensate\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Macromolecular Crowding Is Surprisingly Unable to Deform the Structure of a Model Biomolecular Condensate\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Machine learning using multi-modal data predicts the production of selective laser sintered 3D printed drug products\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Machine learning applications in macromolecular X-ray crystallography\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Ion\\u2010Selective Microporous Polymer Membranes with Hydrogen\\u2010Bond and Salt\\u2010Bridge Networks for Aqueous Organic Redox Flow Batteries\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Is the incidence of congenital toxoplasmosis declining?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Joint Multi-Ancestry and Admixed GWAS Reveals the Complex Genetics behind Human Cranial Vault Shape\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Junctophillin-2: Coupling Hopes for Cardiac Gene Therapy to Gene Transcription\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"KAT2-mediated acetylation switches the mode of PALB2 chromatin association to safeguard genome integrity\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"3D-Beacons: decreasing the gap between protein sequences and structures through a federated network of protein structure data resources\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"LRRC15 mediates an accessory interaction with the SARS-CoV-2 spike protein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Large-scale deorphanization of<i>Nematostella vectensis</i>neuropeptide GPCRs supports the independent expansion of bilaterian and cnidarian peptidergic systems\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Latent TGF\\u03b2 complexes are transglutaminase cross-linked to fibrillin to facilitate TGF\\u03b2 activation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Lateral flow test engineering and lessons learned from COVID-19\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Lessons from natural flight for aviation: then, now and tomorrow\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Light sensitive short hypocotyl (LSH) confer symbiotic nodule identity in the legume Medicago truncatula\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Lighting up protein design\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Limitations of non-polarizable force fields in describing anion binding poses in non-polar synthetic hosts\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Live enteroviruses, but not other viruses, detected in human pancreas at the onset of type 1 diabetes in the DiViD study\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Live\\u2010cell super\\u2010resolution imaging of actin using <scp>LifeAct</scp>\\u201014 with a <scp>PAINT</scp>\\u2010based approach\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Live\\u2010cell super\\u2010resolution imaging of actin using <scp>LifeAct</scp>\\u201014 with a <scp>PAINT</scp>\\u2010based approach\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Loss-of-function, gain-of-function and dominant-negative mutations have profoundly different effects on protein structure\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"MGnify Genomes: A Resource for Biome-specific Microbial Genome Catalogues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"MGnify: the microbiome sequence data analysis resource in 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Machine learning activation energies of chemical reactions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"How Arrestins and GRKs Regulate the Function of Long Chain Fatty Acid Receptors\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High\\u2010accuracy protein structure prediction in <scp>CASP14</scp>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-throughput property-driven generative design of functional organic molecules\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evolution of long-term vaccine-induced and hybrid immunity in healthcare workers after different COVID-19 vaccine regimens\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Enhanced functional detection of synaptic calcium-permeable AMPA receptors using intracellular NASPM\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Enhanced statistical sampling reveals microscopic complexity in the talin mechanosensor folding energy landscape\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Ensembl 2023\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Ensembl 2023\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Environmental and genetic predictors of human cardiovascular ageing\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Enzymatic Glyco-Modification of Synthetic Membrane Systems\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Enzyme adaptation to habitat thermal legacy shapes the thermal plasticity of marine microbiomes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Enzyme self-label-bound ATTO700 in single-molecule and super-resolution microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evaluating the Breadth of Neutralizing Antibody Responses Elicited by Infectious Bursal Disease Virus Genogroup A1 Strains Using a Novel Chicken B-Cell Rescue System and Neutralization Assay\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evaluation of Nafamostat as Chemoprophylaxis for SARS-CoV-2 Infection in Hamsters\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evaluation of genome skimming to detect and characterise human and livestock helminths\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evidence of epistasis in regions of long-range linkage disequilibrium across five complex diseases in the UK Biobank and eMERGE datasets\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evolution of sex\\u2010determination in dioecious plants: From active Y to X/A balance?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fluorescent Proteins: Crystallization, Structural Determination, and Nonnatural Amino Acid Incorporation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evolutionary and cellular analysis of the \\u2018dark\\u2019 pseudokinase PSKH2\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Evolutionary dynamics of genome size and content during the adaptive radiation of Heliconiini butterflies\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Evolutionary remodelling of N\\u2010terminal domain loops fine\\u2010tunes <scp>SARS\\u2010CoV</scp>\\u20102 spike\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Ex situ-armus: experimental models for combating respiratory dysfunction\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Exiting the tunnel of uncertainty: crystal soak to validated hit\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exome sequencing reveals aggregates of rare variants in glycosyltransferase and other genes influencing immunoglobulin G and transferrin glycosylation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Expanding the Substrate Scope of <i>N</i>\\u2010 and <i>O</i>\\u2010Methyltransferases from Plants for Chemoselective Alkylation**\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Exploring a <i>Streptomyces</i> wax synthase using acyl-SNACs as donor substrates\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Exploring the antimicrobial peptidome of nematodes through phylum-spanning in silico analyses highlights novel opportunities for pathogen control\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Extending the Capabilities of Data-Driven Reduced-Order Models to Make Predictions for Unseen Scenarios: Applied to Flow Around Buildings\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fibrillin microfibril structure identifies long-range effects of inherited pathogenic mutations affecting a key regulatory latent TGF\\u03b2-binding site\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Engineering \\u03b1-carboxysomes into plant chloroplasts to support autotrophic photosynthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Engineering mutually orthogonal <scp>PylRS</scp>/<scp>tRNA</scp> pairs for dual encoding of functional histidine analogues\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Engineering enzyme activity using an expanded amino acid alphabet\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Engineering an Oxygen\\u2010Binding Protein for Photocatalytic CO<sub>2</sub>Reductions in Water\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Diene incorporation by a dehydratase domain variant in modular polyketide synthases\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dipolar Order Parameters in Large Systems With Fast Spinning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Discovery and characterization of non-canonical E2 conjugating enzymes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Discovery and characterization of non-canonical E2 conjugating enzymes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Disruptions in endocytic traffic contribute to the activation of the NLRP3 inflammasome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Diversity of viscoelastic properties of an engineered muscle-inspired protein hydrogel\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Does a Machine-Learned Potential Perform Better Than an Optimally Tuned Traditional Force Field? A Case Study on Fluorohydrins\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Drug resistance-associated mutations in <i>Plasmodium</i> UBP-1 disrupt ubiquitin hydrolysis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dynamic interplay between the periplasmic chaperone SurA and the BAM complex in outer membrane protein folding\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dynamic regulation of RAS and RAS signaling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Dynamics of the secreted frizzled related protein Sizzled and potential implications for binding to bone morphogenetic protein-1 (BMP-1)\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"E2 Partner Tunes the Ubiquitylation Specificity of Arkadia E3 Ubiquitin Ligase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"EFCAB4B (CRACR2A) genetic variants associated with COVID-19 fatality\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"EFCAB4B (CRACR2A) genetic variants associated with COVID-19 fatality\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"EPIKOL, a chromatin-focused CRISPR/Cas9-based screening platform, to identify cancer-specific epigenetic vulnerabilities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"EROS is a selective chaperone regulating the phagocyte NADPH oxidase and purinergic signalling\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Editorial: Experiments and Simulations: A Pas de Deux to Unravel Biological Function\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Effector-dependent activation and oligomerization of NRC helper NLRs by Rpi-amr3 and Rpi-amr1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Effector\\u2010dependent activation and oligomerization of plant <scp>NRC</scp> class helper <scp>NLRs</scp> by sensor <scp>NLR</scp> immune receptors Rpi\\u2010amr3 and Rpi\\u2010amr1\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Effects of Dynamical Degrees of Freedom on Magnetic Compass Sensitivity: A Comparison of Plant and Avian Cryptochromes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Elucidating the molecular programming of a nonlinear non-ribosomal peptide synthetase responsible for fungal siderophore biosynthesis\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Emerging principles in the design of bioengineered made-to-order plant immune receptors\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Endogenous GFP tagging in the diatom<i>Thalassiosira pseudonana</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Energy Landscapes and Heat Capacity Signatures for Monomers and Dimers of Amyloid-Forming Hexapeptides\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"First Insights into the Venom Composition of Two Ecuadorian Coral Snakes\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Force-Generation by the Trans-Envelope Tol-Pal System\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"HOIL-1-catalysed ubiquitylation of unbranched glucosaccharides and its activation by ubiquitin oligomers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Genetically encoded imaging tools for investigating cell dynamics at a glance\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Genome-wide association study reveals loci with sex-specific effects on plasma bile acids\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic and functional analysis of phage\\u2010mediated horizontal gene transfer in <i>Pseudomonas syringae</i> on the plant surface\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic evidence of contemporary hybridization between Schistosoma species\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic landscape of drug response reveals mediators of anthelmintic resistance\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genomic variation during culture-adaptation of genetically complex<i>Plasmodium falciparum</i>clinical isolates\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Glycosaminoglycans: What Remains To Be Deciphered?\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Graph Pattern Matching in GQL and SQL/PGQ\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Group Selection as a Basis for Screening Mutagenized Libraries of Public Goods (Bacillus thuringiensis Cry Toxins)\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Growth-coupled enzyme engineering through manipulation of redox cofactor regeneration\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"H3K4 methylation by SETD1A/BOD1L facilitates RIF1-dependent NHEJ\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"HOIL\\u20101 ubiquitin ligase activity targets unbranched glucosaccharides and is required to prevent polyglucosan accumulation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Formate hydrogenlyase, formic acid translocation and hydrogen production: dynamic membrane biology during fermentation\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"HPV E6 inhibits E6AP to regulate epithelial homeostasis by modulating keratinocyte differentiation commitment and YAP1 activation\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Handcrafted Histological Transformer (H2T): Unsupervised representation of whole slide images\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Handcrafted Histological Transformer (H2T): Unsupervised representation of whole slide images\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Head-to-tail polymerization by VEL proteins underpins cold-induced Polycomb silencing in flowering control\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Heme binding to the SARS-CoV-2 spike glycoprotein\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Herbicolin A production and its modulation by quorum sensing in a<i>Pantoea agglomerans</i>rhizobacterium bioactive against a broad spectrum of plant\\u2010pathogenic fungi\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High quality de novo genome assembly of the non-conventional yeast Kazachstania bulderi describes a potential low pH production host for biorefineries\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"High throughput steady-state enzyme kinetics measured in a parallel droplet generation and absorbance detection platform\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"High-Throughput Steady-State Enzyme Kinetics Measured in a Parallel Droplet Generation and Absorbance Detection Platform\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-resolution promoter interaction analysis in Type 3 Innate Lymphoid Cells implicates Batten Disease gene<i>CLN3</i>in Crohn\\u2019s Disease aetiology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"High-resolution reconstruction of a Jumbo-bacteriophage infecting capsulated bacteria using hyperbranched tail fibers\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Genetic, Clinical Underpinnings of Brain Change Along Two Neuroanatomical Dimensions of Clinically-defined Alzheimer\\u2019s Disease\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic sequencing detected the SARS-CoV-2 delta variant in wastewater a month prior to the first COVID-19 case in Ahmedabad (India)\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic and cellular characterization of MscS-like putative channels in the filamentous fungus <i>Aspergillus nidulans</i>\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Fragment-Based Drug Discovery by NMR. Where Are the Successes and Where can It Be Improved?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"From peptides to proteins: coiled-coil tetramers to single-chain 4-helix bundles\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Frontiers in metalloprotein crystallography and cryogenic electron microscopy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Full structure/function analysis of all the pilin subunits in a type 4 pilus: a complex of minor pilins in <i>Streptococcus sanguinis</i> mediates binding to glycans\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Full structure/function analysis of all the pilin subunits in a type 4 pilus: a complex of minor pilins in <i>Streptococcus sanguinis</i> mediates binding to glycans\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Full-length antithrombin frameshift variant with aberrant C-terminus causes endoplasmic reticulum retention with a dominant-negative effect\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Function and phylogeny support the independent evolution of acid-sensing ion channels in the Placozoa\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Functional metagenomic screening identifies an unexpected \\u03b2-glucuronidase\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fungal resilience and host\\u2013pathogen interactions: Future perspectives and opportunities\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Fungal spore swelling and germination are restricted by the macrophage phagolysosome\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"G protein\\u2013receptor kinases 5/6 are the key regulators of G protein\\u2013coupled receptor 35\\u2013arrestin interactions\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"G4Atlas: a comprehensive transcriptome-wide G-quadruplex database\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"GOLDEN2\\u2010LIKE transcription factors: A golden ticket to improve crops?\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"GPC3-Unc5 receptor complex structure and role in cell migration\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"GWAS and meta-analysis identifies multiple new genetic mechanisms underlying severe Covid-19\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"GWYRE: A Resource for Mapping Variants onto Experimental and Modeled Structures of Human Protein Complexes\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Generating images of hydrated pollen grains using deep learning\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Generation and Characterization of Native and Sialic Acid-Deficient IgE\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Generation of SARS-CoV-2 escape mutations by monoclonal antibody therapy\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Generative models of morphogenesis in developmental biology\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"Genetic Mimicry Analysis Reveals the Specific Lipases Targeted by the ANGPTL3-ANGPTL8 Complex and ANGPTL4\", \"type\": \"Direct\", \"color\": \"red\", \"count\": 1}, {\"title\": \"Genetic Mimicry Analysis Reveals the Specific Lipases Targeted by the ANGPTL3-ANGPTL8 Complex and ANGPTL4\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}, {\"title\": \"\\u03c0 Learning: A Performance\\u2010Informed Framework for Microstructural Electrode Design\", \"type\": \"Indirect\", \"color\": \"blue\", \"count\": 1}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "\u001b[1;35malt.Chart\u001b[0m\u001b[1m(\u001b[0m\u001b[33m...\u001b[0m\u001b[1m)\u001b[0m"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Create two separate DataFrames for indirect and direct citations\n",
    "indirect_df = citations[['title', 'af_indirect_citations']].rename(columns={'af_indirect_citations': 'count'})\n",
    "indirect_df['type'] = 'Indirect'\n",
    "\n",
    "direct_df = citations[['title', 'af_direct_citation']].rename(columns={'af_direct_citation': 'count'})\n",
    "direct_df['type'] = 'Direct'\n",
    "\n",
    "# Concatenate the two DataFrames\n",
    "combined_df = pd.concat([indirect_df, direct_df])\n",
    "\n",
    "# keep those that have at least one citation either directly or indirectly\n",
    "combined_df = combined_df[combined_df['count'] > 0]\n",
    "\n",
    "# Add a color column based on the type\n",
    "combined_df['color'] = combined_df['type'].map({'Indirect': 'blue', 'Direct': 'red'})\n",
    "\n",
    "# Group by title and type, and sum the counts\n",
    "grouped_df = combined_df.groupby(['title', 'type', 'color']).sum().reset_index()\n",
    "\n",
    "# Sort by the summed count\n",
    "grouped_df = grouped_df.sort_values(by='count', ascending=False)\n",
    "\n",
    "# Create the chart\n",
    "chart = alt.Chart(grouped_df).mark_bar().encode(\n",
    "    x=alt.X('count:Q', title='Number of Citations'),\n",
    "    y=alt.Y('title:N', title='Paper Title', sort='-x'),\n",
    "    color=alt.Color('type:N', title='Citation Type', scale=alt.Scale(domain=['Indirect', 'Direct'], range=['blue', 'red'])),\n",
    "    tooltip=['title', 'sum(count)', 'type']\n",
    ").properties(\n",
    "    title='Count of Papers Citing Alphafold (Direct and Indirect)',\n",
    "    height=6400\n",
    ")\n",
    "\n",
    "chart\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".igl_alphafold",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
